{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "e10046fe",
   "metadata": {},
   "source": [
    "# Mapping CoBinding matrix and SE matrix"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "c7d47277",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "    <div class=\"bk-root\">\n",
       "        <a href=\"https://bokeh.org\" target=\"_blank\" class=\"bk-logo bk-logo-small bk-logo-notebook\"></a>\n",
       "        <span id=\"1003\">Loading BokehJS ...</span>\n",
       "    </div>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/javascript": [
       "\n",
       "(function(root) {\n",
       "  function now() {\n",
       "    return new Date();\n",
       "  }\n",
       "\n",
       "  const force = true;\n",
       "\n",
       "  if (typeof root._bokeh_onload_callbacks === \"undefined\" || force === true) {\n",
       "    root._bokeh_onload_callbacks = [];\n",
       "    root._bokeh_is_loading = undefined;\n",
       "  }\n",
       "\n",
       "  const JS_MIME_TYPE = 'application/javascript';\n",
       "  const HTML_MIME_TYPE = 'text/html';\n",
       "  const EXEC_MIME_TYPE = 'application/vnd.bokehjs_exec.v0+json';\n",
       "  const CLASS_NAME = 'output_bokeh rendered_html';\n",
       "\n",
       "  /**\n",
       "   * Render data to the DOM node\n",
       "   */\n",
       "  function render(props, node) {\n",
       "    const script = document.createElement(\"script\");\n",
       "    node.appendChild(script);\n",
       "  }\n",
       "\n",
       "  /**\n",
       "   * Handle when an output is cleared or removed\n",
       "   */\n",
       "  function handleClearOutput(event, handle) {\n",
       "    const cell = handle.cell;\n",
       "\n",
       "    const id = cell.output_area._bokeh_element_id;\n",
       "    const server_id = cell.output_area._bokeh_server_id;\n",
       "    // Clean up Bokeh references\n",
       "    if (id != null && id in Bokeh.index) {\n",
       "      Bokeh.index[id].model.document.clear();\n",
       "      delete Bokeh.index[id];\n",
       "    }\n",
       "\n",
       "    if (server_id !== undefined) {\n",
       "      // Clean up Bokeh references\n",
       "      const cmd_clean = \"from bokeh.io.state import curstate; print(curstate().uuid_to_server['\" + server_id + \"'].get_sessions()[0].document.roots[0]._id)\";\n",
       "      cell.notebook.kernel.execute(cmd_clean, {\n",
       "        iopub: {\n",
       "          output: function(msg) {\n",
       "            const id = msg.content.text.trim();\n",
       "            if (id in Bokeh.index) {\n",
       "              Bokeh.index[id].model.document.clear();\n",
       "              delete Bokeh.index[id];\n",
       "            }\n",
       "          }\n",
       "        }\n",
       "      });\n",
       "      // Destroy server and session\n",
       "      const cmd_destroy = \"import bokeh.io.notebook as ion; ion.destroy_server('\" + server_id + \"')\";\n",
       "      cell.notebook.kernel.execute(cmd_destroy);\n",
       "    }\n",
       "  }\n",
       "\n",
       "  /**\n",
       "   * Handle when a new output is added\n",
       "   */\n",
       "  function handleAddOutput(event, handle) {\n",
       "    const output_area = handle.output_area;\n",
       "    const output = handle.output;\n",
       "\n",
       "    // limit handleAddOutput to display_data with EXEC_MIME_TYPE content only\n",
       "    if ((output.output_type != \"display_data\") || (!Object.prototype.hasOwnProperty.call(output.data, EXEC_MIME_TYPE))) {\n",
       "      return\n",
       "    }\n",
       "\n",
       "    const toinsert = output_area.element.find(\".\" + CLASS_NAME.split(' ')[0]);\n",
       "\n",
       "    if (output.metadata[EXEC_MIME_TYPE][\"id\"] !== undefined) {\n",
       "      toinsert[toinsert.length - 1].firstChild.textContent = output.data[JS_MIME_TYPE];\n",
       "      // store reference to embed id on output_area\n",
       "      output_area._bokeh_element_id = output.metadata[EXEC_MIME_TYPE][\"id\"];\n",
       "    }\n",
       "    if (output.metadata[EXEC_MIME_TYPE][\"server_id\"] !== undefined) {\n",
       "      const bk_div = document.createElement(\"div\");\n",
       "      bk_div.innerHTML = output.data[HTML_MIME_TYPE];\n",
       "      const script_attrs = bk_div.children[0].attributes;\n",
       "      for (let i = 0; i < script_attrs.length; i++) {\n",
       "        toinsert[toinsert.length - 1].firstChild.setAttribute(script_attrs[i].name, script_attrs[i].value);\n",
       "        toinsert[toinsert.length - 1].firstChild.textContent = bk_div.children[0].textContent\n",
       "      }\n",
       "      // store reference to server id on output_area\n",
       "      output_area._bokeh_server_id = output.metadata[EXEC_MIME_TYPE][\"server_id\"];\n",
       "    }\n",
       "  }\n",
       "\n",
       "  function register_renderer(events, OutputArea) {\n",
       "\n",
       "    function append_mime(data, metadata, element) {\n",
       "      // create a DOM node to render to\n",
       "      const toinsert = this.create_output_subarea(\n",
       "        metadata,\n",
       "        CLASS_NAME,\n",
       "        EXEC_MIME_TYPE\n",
       "      );\n",
       "      this.keyboard_manager.register_events(toinsert);\n",
       "      // Render to node\n",
       "      const props = {data: data, metadata: metadata[EXEC_MIME_TYPE]};\n",
       "      render(props, toinsert[toinsert.length - 1]);\n",
       "      element.append(toinsert);\n",
       "      return toinsert\n",
       "    }\n",
       "\n",
       "    /* Handle when an output is cleared or removed */\n",
       "    events.on('clear_output.CodeCell', handleClearOutput);\n",
       "    events.on('delete.Cell', handleClearOutput);\n",
       "\n",
       "    /* Handle when a new output is added */\n",
       "    events.on('output_added.OutputArea', handleAddOutput);\n",
       "\n",
       "    /**\n",
       "     * Register the mime type and append_mime function with output_area\n",
       "     */\n",
       "    OutputArea.prototype.register_mime_type(EXEC_MIME_TYPE, append_mime, {\n",
       "      /* Is output safe? */\n",
       "      safe: true,\n",
       "      /* Index of renderer in `output_area.display_order` */\n",
       "      index: 0\n",
       "    });\n",
       "  }\n",
       "\n",
       "  // register the mime type if in Jupyter Notebook environment and previously unregistered\n",
       "  if (root.Jupyter !== undefined) {\n",
       "    const events = require('base/js/events');\n",
       "    const OutputArea = require('notebook/js/outputarea').OutputArea;\n",
       "\n",
       "    if (OutputArea.prototype.mime_types().indexOf(EXEC_MIME_TYPE) == -1) {\n",
       "      register_renderer(events, OutputArea);\n",
       "    }\n",
       "  }\n",
       "\n",
       "  \n",
       "  if (typeof (root._bokeh_timeout) === \"undefined\" || force === true) {\n",
       "    root._bokeh_timeout = Date.now() + 5000;\n",
       "    root._bokeh_failed_load = false;\n",
       "  }\n",
       "\n",
       "  const NB_LOAD_WARNING = {'data': {'text/html':\n",
       "     \"<div style='background-color: #fdd'>\\n\"+\n",
       "     \"<p>\\n\"+\n",
       "     \"BokehJS does not appear to have successfully loaded. If loading BokehJS from CDN, this \\n\"+\n",
       "     \"may be due to a slow or bad network connection. Possible fixes:\\n\"+\n",
       "     \"</p>\\n\"+\n",
       "     \"<ul>\\n\"+\n",
       "     \"<li>re-rerun `output_notebook()` to attempt to load from CDN again, or</li>\\n\"+\n",
       "     \"<li>use INLINE resources instead, as so:</li>\\n\"+\n",
       "     \"</ul>\\n\"+\n",
       "     \"<code>\\n\"+\n",
       "     \"from bokeh.resources import INLINE\\n\"+\n",
       "     \"output_notebook(resources=INLINE)\\n\"+\n",
       "     \"</code>\\n\"+\n",
       "     \"</div>\"}};\n",
       "\n",
       "  function display_loaded() {\n",
       "    const el = document.getElementById(\"1003\");\n",
       "    if (el != null) {\n",
       "      el.textContent = \"BokehJS is loading...\";\n",
       "    }\n",
       "    if (root.Bokeh !== undefined) {\n",
       "      if (el != null) {\n",
       "        el.textContent = \"BokehJS \" + root.Bokeh.version + \" successfully loaded.\";\n",
       "      }\n",
       "    } else if (Date.now() < root._bokeh_timeout) {\n",
       "      setTimeout(display_loaded, 100)\n",
       "    }\n",
       "  }\n",
       "\n",
       "\n",
       "  function run_callbacks() {\n",
       "    try {\n",
       "      root._bokeh_onload_callbacks.forEach(function(callback) {\n",
       "        if (callback != null)\n",
       "          callback();\n",
       "      });\n",
       "    } finally {\n",
       "      delete root._bokeh_onload_callbacks\n",
       "    }\n",
       "    console.debug(\"Bokeh: all callbacks have finished\");\n",
       "  }\n",
       "\n",
       "  function load_libs(css_urls, js_urls, callback) {\n",
       "    if (css_urls == null) css_urls = [];\n",
       "    if (js_urls == null) js_urls = [];\n",
       "\n",
       "    root._bokeh_onload_callbacks.push(callback);\n",
       "    if (root._bokeh_is_loading > 0) {\n",
       "      console.debug(\"Bokeh: BokehJS is being loaded, scheduling callback at\", now());\n",
       "      return null;\n",
       "    }\n",
       "    if (js_urls == null || js_urls.length === 0) {\n",
       "      run_callbacks();\n",
       "      return null;\n",
       "    }\n",
       "    console.debug(\"Bokeh: BokehJS not loaded, scheduling load and callback at\", now());\n",
       "    root._bokeh_is_loading = css_urls.length + js_urls.length;\n",
       "\n",
       "    function on_load() {\n",
       "      root._bokeh_is_loading--;\n",
       "      if (root._bokeh_is_loading === 0) {\n",
       "        console.debug(\"Bokeh: all BokehJS libraries/stylesheets loaded\");\n",
       "        run_callbacks()\n",
       "      }\n",
       "    }\n",
       "\n",
       "    function on_error(url) {\n",
       "      console.error(\"failed to load \" + url);\n",
       "    }\n",
       "\n",
       "    for (let i = 0; i < css_urls.length; i++) {\n",
       "      const url = css_urls[i];\n",
       "      const element = document.createElement(\"link\");\n",
       "      element.onload = on_load;\n",
       "      element.onerror = on_error.bind(null, url);\n",
       "      element.rel = \"stylesheet\";\n",
       "      element.type = \"text/css\";\n",
       "      element.href = url;\n",
       "      console.debug(\"Bokeh: injecting link tag for BokehJS stylesheet: \", url);\n",
       "      document.body.appendChild(element);\n",
       "    }\n",
       "\n",
       "    for (let i = 0; i < js_urls.length; i++) {\n",
       "      const url = js_urls[i];\n",
       "      const element = document.createElement('script');\n",
       "      element.onload = on_load;\n",
       "      element.onerror = on_error.bind(null, url);\n",
       "      element.async = false;\n",
       "      element.src = url;\n",
       "      console.debug(\"Bokeh: injecting script tag for BokehJS library: \", url);\n",
       "      document.head.appendChild(element);\n",
       "    }\n",
       "  };\n",
       "\n",
       "  function inject_raw_css(css) {\n",
       "    const element = document.createElement(\"style\");\n",
       "    element.appendChild(document.createTextNode(css));\n",
       "    document.body.appendChild(element);\n",
       "  }\n",
       "\n",
       "  \n",
       "  const js_urls = [\"https://cdn.bokeh.org/bokeh/release/bokeh-2.4.2.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-gl-2.4.2.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.4.2.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.4.2.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-mathjax-2.4.2.min.js\"];\n",
       "  const css_urls = [];\n",
       "  \n",
       "\n",
       "  const inline_js = [\n",
       "    function(Bokeh) {\n",
       "      Bokeh.set_log_level(\"info\");\n",
       "    },\n",
       "    function(Bokeh) {\n",
       "    \n",
       "    \n",
       "    }\n",
       "  ];\n",
       "\n",
       "  function run_inline_js() {\n",
       "    \n",
       "    if (root.Bokeh !== undefined || force === true) {\n",
       "      \n",
       "    for (let i = 0; i < inline_js.length; i++) {\n",
       "      inline_js[i].call(root, root.Bokeh);\n",
       "    }\n",
       "    if (force === true) {\n",
       "        display_loaded();\n",
       "      }} else if (Date.now() < root._bokeh_timeout) {\n",
       "      setTimeout(run_inline_js, 100);\n",
       "    } else if (!root._bokeh_failed_load) {\n",
       "      console.log(\"Bokeh: BokehJS failed to load within specified timeout.\");\n",
       "      root._bokeh_failed_load = true;\n",
       "    } else if (force !== true) {\n",
       "      const cell = $(document.getElementById(\"1003\")).parents('.cell').data().cell;\n",
       "      cell.output_area.append_execute_result(NB_LOAD_WARNING)\n",
       "    }\n",
       "\n",
       "  }\n",
       "\n",
       "  if (root._bokeh_is_loading === 0) {\n",
       "    console.debug(\"Bokeh: BokehJS loaded, going straight to plotting\");\n",
       "    run_inline_js();\n",
       "  } else {\n",
       "    load_libs(css_urls, js_urls, function() {\n",
       "      console.debug(\"Bokeh: BokehJS plotting callback run at\", now());\n",
       "      run_inline_js();\n",
       "    });\n",
       "  }\n",
       "}(window));"
      ],
      "application/vnd.bokehjs_load.v0+json": "\n(function(root) {\n  function now() {\n    return new Date();\n  }\n\n  const force = true;\n\n  if (typeof root._bokeh_onload_callbacks === \"undefined\" || force === true) {\n    root._bokeh_onload_callbacks = [];\n    root._bokeh_is_loading = undefined;\n  }\n\n  \n\n  \n  if (typeof (root._bokeh_timeout) === \"undefined\" || force === true) {\n    root._bokeh_timeout = Date.now() + 5000;\n    root._bokeh_failed_load = false;\n  }\n\n  const NB_LOAD_WARNING = {'data': {'text/html':\n     \"<div style='background-color: #fdd'>\\n\"+\n     \"<p>\\n\"+\n     \"BokehJS does not appear to have successfully loaded. If loading BokehJS from CDN, this \\n\"+\n     \"may be due to a slow or bad network connection. Possible fixes:\\n\"+\n     \"</p>\\n\"+\n     \"<ul>\\n\"+\n     \"<li>re-rerun `output_notebook()` to attempt to load from CDN again, or</li>\\n\"+\n     \"<li>use INLINE resources instead, as so:</li>\\n\"+\n     \"</ul>\\n\"+\n     \"<code>\\n\"+\n     \"from bokeh.resources import INLINE\\n\"+\n     \"output_notebook(resources=INLINE)\\n\"+\n     \"</code>\\n\"+\n     \"</div>\"}};\n\n  function display_loaded() {\n    const el = document.getElementById(\"1003\");\n    if (el != null) {\n      el.textContent = \"BokehJS is loading...\";\n    }\n    if (root.Bokeh !== undefined) {\n      if (el != null) {\n        el.textContent = \"BokehJS \" + root.Bokeh.version + \" successfully loaded.\";\n      }\n    } else if (Date.now() < root._bokeh_timeout) {\n      setTimeout(display_loaded, 100)\n    }\n  }\n\n\n  function run_callbacks() {\n    try {\n      root._bokeh_onload_callbacks.forEach(function(callback) {\n        if (callback != null)\n          callback();\n      });\n    } finally {\n      delete root._bokeh_onload_callbacks\n    }\n    console.debug(\"Bokeh: all callbacks have finished\");\n  }\n\n  function load_libs(css_urls, js_urls, callback) {\n    if (css_urls == null) css_urls = [];\n    if (js_urls == null) js_urls = [];\n\n    root._bokeh_onload_callbacks.push(callback);\n    if (root._bokeh_is_loading > 0) {\n      console.debug(\"Bokeh: BokehJS is being loaded, scheduling callback at\", now());\n      return null;\n    }\n    if (js_urls == null || js_urls.length === 0) {\n      run_callbacks();\n      return null;\n    }\n    console.debug(\"Bokeh: BokehJS not loaded, scheduling load and callback at\", now());\n    root._bokeh_is_loading = css_urls.length + js_urls.length;\n\n    function on_load() {\n      root._bokeh_is_loading--;\n      if (root._bokeh_is_loading === 0) {\n        console.debug(\"Bokeh: all BokehJS libraries/stylesheets loaded\");\n        run_callbacks()\n      }\n    }\n\n    function on_error(url) {\n      console.error(\"failed to load \" + url);\n    }\n\n    for (let i = 0; i < css_urls.length; i++) {\n      const url = css_urls[i];\n      const element = document.createElement(\"link\");\n      element.onload = on_load;\n      element.onerror = on_error.bind(null, url);\n      element.rel = \"stylesheet\";\n      element.type = \"text/css\";\n      element.href = url;\n      console.debug(\"Bokeh: injecting link tag for BokehJS stylesheet: \", url);\n      document.body.appendChild(element);\n    }\n\n    for (let i = 0; i < js_urls.length; i++) {\n      const url = js_urls[i];\n      const element = document.createElement('script');\n      element.onload = on_load;\n      element.onerror = on_error.bind(null, url);\n      element.async = false;\n      element.src = url;\n      console.debug(\"Bokeh: injecting script tag for BokehJS library: \", url);\n      document.head.appendChild(element);\n    }\n  };\n\n  function inject_raw_css(css) {\n    const element = document.createElement(\"style\");\n    element.appendChild(document.createTextNode(css));\n    document.body.appendChild(element);\n  }\n\n  \n  const js_urls = [\"https://cdn.bokeh.org/bokeh/release/bokeh-2.4.2.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-gl-2.4.2.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.4.2.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.4.2.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-mathjax-2.4.2.min.js\"];\n  const css_urls = [];\n  \n\n  const inline_js = [\n    function(Bokeh) {\n      Bokeh.set_log_level(\"info\");\n    },\n    function(Bokeh) {\n    \n    \n    }\n  ];\n\n  function run_inline_js() {\n    \n    if (root.Bokeh !== undefined || force === true) {\n      \n    for (let i = 0; i < inline_js.length; i++) {\n      inline_js[i].call(root, root.Bokeh);\n    }\n    if (force === true) {\n        display_loaded();\n      }} else if (Date.now() < root._bokeh_timeout) {\n      setTimeout(run_inline_js, 100);\n    } else if (!root._bokeh_failed_load) {\n      console.log(\"Bokeh: BokehJS failed to load within specified timeout.\");\n      root._bokeh_failed_load = true;\n    } else if (force !== true) {\n      const cell = $(document.getElementById(\"1003\")).parents('.cell').data().cell;\n      cell.output_area.append_execute_result(NB_LOAD_WARNING)\n    }\n\n  }\n\n  if (root._bokeh_is_loading === 0) {\n    console.debug(\"Bokeh: BokehJS loaded, going straight to plotting\");\n    run_inline_js();\n  } else {\n    load_libs(css_urls, js_urls, function() {\n      console.debug(\"Bokeh: BokehJS plotting callback run at\", now());\n      run_inline_js();\n    });\n  }\n}(window));"
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The autoreload extension is already loaded. To reload it, use:\n",
      "  %reload_ext autoreload\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "import pandas as pd\n",
    "import sys\n",
    "sys.path.insert(0, '../..')\n",
    "import itertools\n",
    "from scipy import stats\n",
    "\n",
    "import numpy as np\n",
    "from natsort import natsorted, ns\n",
    "\n",
    "from genepy.epigenetics import chipseq as chip\n",
    "from genepy.utils import helper\n",
    "from genepy.utils import plot\n",
    "from genepy.google import gcp\n",
    "import igv\n",
    "import SimpSOM as sps\n",
    "from scipy import stats\n",
    "\n",
    "import seaborn as sns\n",
    "from matplotlib import cm\n",
    "from matplotlib import pyplot as plt\n",
    "from IPython.display import IFrame\n",
    "import seaborn as sns\n",
    "from bokeh.plotting import *\n",
    "import igv\n",
    "\n",
    "import numba\n",
    "from numba import jit\n",
    "\n",
    "from scipy.cluster.hierarchy import linkage, leaves_list\n",
    "from sklearn.cluster import AgglomerativeClustering, DBSCAN, KMeans, OPTICS\n",
    "from sklearn.mixture import GaussianMixture\n",
    "from sklearn.manifold import MDS, TSNE\n",
    "from sklearn.decomposition import PCA\n",
    "from sklearn.preprocessing import scale\n",
    "from IPython.display import IFrame\n",
    "\n",
    "from pybedtools import BedTool\n",
    "import pyBigWig\n",
    "\n",
    "output_notebook()\n",
    "%load_ext autoreload\n",
    "%autoreload 2"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f6d6f6d0",
   "metadata": {},
   "source": [
    "# Cobinding mapping"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "955a3d98",
   "metadata": {},
   "outputs": [],
   "source": [
    "project=\"Cobinding_SE\"\n",
    "\n",
    "cobinding_version=\"v3_remove_single_150\"\n",
    "se_matrix=\"SE.6868.xlsx\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "86636a0f",
   "metadata": {},
   "outputs": [],
   "source": [
    "cobinding_project=\"Cobinding_ChIP\"\n",
    "cobinding_matrix=\"merged.bed.gz\""
   ]
  },
  {
   "cell_type": "markdown",
   "id": "33ce262b",
   "metadata": {},
   "source": [
    "results > Cobinding_ChIP > v3_remove_single_150 > merged.bed.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "965afffa",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "# open cobinding matrix\n",
    "bindings= pd.read_csv('../results/'+cobinding_project+'/'+cobinding_version+'/'+cobinding_matrix, \n",
    "                      sep='\\t', index_col=None, header=0,\n",
    "                      low_memory=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "2b6fbe86",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>chrom</th>\n",
       "      <th>start</th>\n",
       "      <th>end</th>\n",
       "      <th>peak_number</th>\n",
       "      <th>foldchange</th>\n",
       "      <th>-log10pvalue</th>\n",
       "      <th>-log10qvalue</th>\n",
       "      <th>relative_summit_pos</th>\n",
       "      <th>CEBPA</th>\n",
       "      <th>HOXA9</th>\n",
       "      <th>...</th>\n",
       "      <th>TFAP4</th>\n",
       "      <th>RUNX2</th>\n",
       "      <th>MAX</th>\n",
       "      <th>IRF8</th>\n",
       "      <th>MYB</th>\n",
       "      <th>ZEB2</th>\n",
       "      <th>RXRA</th>\n",
       "      <th>SPI1</th>\n",
       "      <th>MEF2C</th>\n",
       "      <th>FLAG_MEF2D</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>chr1</td>\n",
       "      <td>9936</td>\n",
       "      <td>10300</td>\n",
       "      <td>0</td>\n",
       "      <td>10.264422</td>\n",
       "      <td>31242.919925</td>\n",
       "      <td>20942.926956</td>\n",
       "      <td>107</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>20.13070</td>\n",
       "      <td>5.750405</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>8.51315</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.86739</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>chr1</td>\n",
       "      <td>15524</td>\n",
       "      <td>15801</td>\n",
       "      <td>1</td>\n",
       "      <td>3.326400</td>\n",
       "      <td>9.112810</td>\n",
       "      <td>7.086980</td>\n",
       "      <td>277</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>chr1</td>\n",
       "      <td>29182</td>\n",
       "      <td>29649</td>\n",
       "      <td>2</td>\n",
       "      <td>15.734000</td>\n",
       "      <td>43.992500</td>\n",
       "      <td>40.043900</td>\n",
       "      <td>145</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>chr1</td>\n",
       "      <td>180636</td>\n",
       "      <td>182055</td>\n",
       "      <td>3</td>\n",
       "      <td>6.703357</td>\n",
       "      <td>106618.940101</td>\n",
       "      <td>24609.011579</td>\n",
       "      <td>144</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>6.51873</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>4.68577</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>chr1</td>\n",
       "      <td>183251</td>\n",
       "      <td>183855</td>\n",
       "      <td>4</td>\n",
       "      <td>3.734692</td>\n",
       "      <td>20.225199</td>\n",
       "      <td>5.923574</td>\n",
       "      <td>67</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>4.93771</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00000</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows × 37 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "  chrom   start     end  peak_number  foldchange   -log10pvalue  -log10qvalue  \\\n",
       "0  chr1    9936   10300            0   10.264422   31242.919925  20942.926956   \n",
       "1  chr1   15524   15801            1    3.326400       9.112810      7.086980   \n",
       "2  chr1   29182   29649            2   15.734000      43.992500     40.043900   \n",
       "3  chr1  180636  182055            3    6.703357  106618.940101  24609.011579   \n",
       "4  chr1  183251  183855            4    3.734692      20.225199      5.923574   \n",
       "\n",
       "   relative_summit_pos  CEBPA  HOXA9  ...     TFAP4     RUNX2  MAX  IRF8  MYB  \\\n",
       "0                  107    0.0    0.0  ...  20.13070  5.750405  0.0   0.0  0.0   \n",
       "1                  277    0.0    0.0  ...   0.00000  0.000000  0.0   0.0  0.0   \n",
       "2                  145    0.0    0.0  ...   0.00000  0.000000  0.0   0.0  0.0   \n",
       "3                  144    0.0    0.0  ...   6.51873  0.000000  0.0   0.0  0.0   \n",
       "4                   67    0.0    0.0  ...   4.93771  0.000000  0.0   0.0  0.0   \n",
       "\n",
       "   ZEB2     RXRA  SPI1  MEF2C  FLAG_MEF2D  \n",
       "0   0.0  8.51315   0.0    0.0     3.86739  \n",
       "1   0.0  0.00000   0.0    0.0     0.00000  \n",
       "2   0.0  0.00000   0.0    0.0     0.00000  \n",
       "3   0.0  0.00000   0.0    0.0     4.68577  \n",
       "4   0.0  0.00000   0.0    0.0     0.00000  \n",
       "\n",
       "[5 rows x 37 columns]"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bindings.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "6f567f20",
   "metadata": {},
   "outputs": [],
   "source": [
    "# rename FLAG_MEF2D to MEF2D\n",
    "bindings.rename(columns = {\"FLAG_MEF2D\": \"MEF2D\"}, inplace=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "c30c28af",
   "metadata": {},
   "outputs": [],
   "source": [
    "# getting the crcs\n",
    "crcs = helper.fileToList(\"../data/CRCs_31.txt\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "ce340a96",
   "metadata": {},
   "outputs": [],
   "source": [
    "# open SE matrix\n",
    "superenhancers = pd.read_excel(\"../data/superenhancer/\"+se_matrix,\n",
    "                               header=None, index_col=None)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "92b3d265",
   "metadata": {},
   "outputs": [],
   "source": [
    "# get only region and gene cols\n",
    "superenhancers = superenhancers.iloc[:,[0,1,2,3,4,11]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "d861c1dc",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "      <th>3</th>\n",
       "      <th>4</th>\n",
       "      <th>11</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>SE_N</td>\n",
       "      <td>REGION_ID</td>\n",
       "      <td>START.HG19</td>\n",
       "      <td>STOP.HG19</td>\n",
       "      <td>CHROM</td>\n",
       "      <td>ALL_GENES</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_1_lociStitched</td>\n",
       "      <td>11850225</td>\n",
       "      <td>12017830</td>\n",
       "      <td>chr12</td>\n",
       "      <td>APOLD1 CDKN1B CLEC4E CREBL2 ETV6 GPR19 MANSC1 ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_2_lociStitched</td>\n",
       "      <td>30753792</td>\n",
       "      <td>30869320</td>\n",
       "      <td>chr4</td>\n",
       "      <td>PCDH7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>3</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_3_lociStitched</td>\n",
       "      <td>18375636</td>\n",
       "      <td>18489680</td>\n",
       "      <td>chr3</td>\n",
       "      <td>SATB1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>4</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_4_lociStitched</td>\n",
       "      <td>134685054</td>\n",
       "      <td>134795163</td>\n",
       "      <td>chr5</td>\n",
       "      <td>C5orf20 H2AFY PITX1 SLC25A48 TIFAB</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "     0                                           1           2          3   \\\n",
       "0  SE_N                                   REGION_ID  START.HG19  STOP.HG19   \n",
       "1     1  1_merged_PDX_cell_lines_AML_1_lociStitched    11850225   12017830   \n",
       "2     2  1_merged_PDX_cell_lines_AML_2_lociStitched    30753792   30869320   \n",
       "3     3  1_merged_PDX_cell_lines_AML_3_lociStitched    18375636   18489680   \n",
       "4     4  1_merged_PDX_cell_lines_AML_4_lociStitched   134685054  134795163   \n",
       "\n",
       "      4                                                  11  \n",
       "0  CHROM                                          ALL_GENES  \n",
       "1  chr12  APOLD1 CDKN1B CLEC4E CREBL2 ETV6 GPR19 MANSC1 ...  \n",
       "2   chr4                                              PCDH7  \n",
       "3   chr3                                              SATB1  \n",
       "4   chr5                 C5orf20 H2AFY PITX1 SLC25A48 TIFAB  "
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "superenhancers.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "6f05e0fc",
   "metadata": {},
   "outputs": [],
   "source": [
    "# set column names\n",
    "superenhancers.columns = superenhancers.iloc[0]\n",
    "superenhancers.drop(superenhancers.index[0], inplace=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "a767de84",
   "metadata": {},
   "outputs": [],
   "source": [
    "# rename chrom info to match bindings\n",
    "superenhancers.rename(columns = {\"START.HG19\": \"start\", \"STOP.HG19\": \"end\", \"CHROM\": \"chrom\"}, inplace=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "c1bf2e41",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>SE_N</th>\n",
       "      <th>REGION_ID</th>\n",
       "      <th>start</th>\n",
       "      <th>end</th>\n",
       "      <th>chrom</th>\n",
       "      <th>ALL_GENES</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_1_lociStitched</td>\n",
       "      <td>11850225</td>\n",
       "      <td>12017830</td>\n",
       "      <td>chr12</td>\n",
       "      <td>APOLD1 CDKN1B CLEC4E CREBL2 ETV6 GPR19 MANSC1 ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_2_lociStitched</td>\n",
       "      <td>30753792</td>\n",
       "      <td>30869320</td>\n",
       "      <td>chr4</td>\n",
       "      <td>PCDH7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>3</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_3_lociStitched</td>\n",
       "      <td>18375636</td>\n",
       "      <td>18489680</td>\n",
       "      <td>chr3</td>\n",
       "      <td>SATB1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>4</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_4_lociStitched</td>\n",
       "      <td>134685054</td>\n",
       "      <td>134795163</td>\n",
       "      <td>chr5</td>\n",
       "      <td>C5orf20 H2AFY PITX1 SLC25A48 TIFAB</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>5</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_5_lociStitched</td>\n",
       "      <td>9863066</td>\n",
       "      <td>9968827</td>\n",
       "      <td>chr12</td>\n",
       "      <td>CD69 CLEC12A CLEC2D CLECL1 KLRF1 KLRF2 M6PR</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "0 SE_N                                   REGION_ID      start        end  \\\n",
       "1    1  1_merged_PDX_cell_lines_AML_1_lociStitched   11850225   12017830   \n",
       "2    2  1_merged_PDX_cell_lines_AML_2_lociStitched   30753792   30869320   \n",
       "3    3  1_merged_PDX_cell_lines_AML_3_lociStitched   18375636   18489680   \n",
       "4    4  1_merged_PDX_cell_lines_AML_4_lociStitched  134685054  134795163   \n",
       "5    5  1_merged_PDX_cell_lines_AML_5_lociStitched    9863066    9968827   \n",
       "\n",
       "0  chrom                                          ALL_GENES  \n",
       "1  chr12  APOLD1 CDKN1B CLEC4E CREBL2 ETV6 GPR19 MANSC1 ...  \n",
       "2   chr4                                              PCDH7  \n",
       "3   chr3                                              SATB1  \n",
       "4   chr5                 C5orf20 H2AFY PITX1 SLC25A48 TIFAB  \n",
       "5  chr12        CD69 CLEC12A CLEC2D CLECL1 KLRF1 KLRF2 M6PR  "
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "superenhancers.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "014f2274",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>chrom</th>\n",
       "      <th>start</th>\n",
       "      <th>end</th>\n",
       "      <th>peak_number</th>\n",
       "      <th>foldchange</th>\n",
       "      <th>-log10pvalue</th>\n",
       "      <th>-log10qvalue</th>\n",
       "      <th>relative_summit_pos</th>\n",
       "      <th>CEBPA</th>\n",
       "      <th>HOXA9</th>\n",
       "      <th>...</th>\n",
       "      <th>TFAP4</th>\n",
       "      <th>RUNX2</th>\n",
       "      <th>MAX</th>\n",
       "      <th>IRF8</th>\n",
       "      <th>MYB</th>\n",
       "      <th>ZEB2</th>\n",
       "      <th>RXRA</th>\n",
       "      <th>SPI1</th>\n",
       "      <th>MEF2C</th>\n",
       "      <th>MEF2D</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>chr1</td>\n",
       "      <td>9936</td>\n",
       "      <td>10300</td>\n",
       "      <td>0</td>\n",
       "      <td>10.264422</td>\n",
       "      <td>31242.919925</td>\n",
       "      <td>20942.926956</td>\n",
       "      <td>107</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>20.13070</td>\n",
       "      <td>5.750405</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>8.51315</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.86739</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>chr1</td>\n",
       "      <td>15524</td>\n",
       "      <td>15801</td>\n",
       "      <td>1</td>\n",
       "      <td>3.326400</td>\n",
       "      <td>9.112810</td>\n",
       "      <td>7.086980</td>\n",
       "      <td>277</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>chr1</td>\n",
       "      <td>29182</td>\n",
       "      <td>29649</td>\n",
       "      <td>2</td>\n",
       "      <td>15.734000</td>\n",
       "      <td>43.992500</td>\n",
       "      <td>40.043900</td>\n",
       "      <td>145</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>chr1</td>\n",
       "      <td>180636</td>\n",
       "      <td>182055</td>\n",
       "      <td>3</td>\n",
       "      <td>6.703357</td>\n",
       "      <td>106618.940101</td>\n",
       "      <td>24609.011579</td>\n",
       "      <td>144</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>6.51873</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>4.68577</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>chr1</td>\n",
       "      <td>183251</td>\n",
       "      <td>183855</td>\n",
       "      <td>4</td>\n",
       "      <td>3.734692</td>\n",
       "      <td>20.225199</td>\n",
       "      <td>5.923574</td>\n",
       "      <td>67</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>4.93771</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00000</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00000</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows × 37 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "  chrom   start     end  peak_number  foldchange   -log10pvalue  -log10qvalue  \\\n",
       "0  chr1    9936   10300            0   10.264422   31242.919925  20942.926956   \n",
       "1  chr1   15524   15801            1    3.326400       9.112810      7.086980   \n",
       "2  chr1   29182   29649            2   15.734000      43.992500     40.043900   \n",
       "3  chr1  180636  182055            3    6.703357  106618.940101  24609.011579   \n",
       "4  chr1  183251  183855            4    3.734692      20.225199      5.923574   \n",
       "\n",
       "   relative_summit_pos  CEBPA  HOXA9  ...     TFAP4     RUNX2  MAX  IRF8  MYB  \\\n",
       "0                  107    0.0    0.0  ...  20.13070  5.750405  0.0   0.0  0.0   \n",
       "1                  277    0.0    0.0  ...   0.00000  0.000000  0.0   0.0  0.0   \n",
       "2                  145    0.0    0.0  ...   0.00000  0.000000  0.0   0.0  0.0   \n",
       "3                  144    0.0    0.0  ...   6.51873  0.000000  0.0   0.0  0.0   \n",
       "4                   67    0.0    0.0  ...   4.93771  0.000000  0.0   0.0  0.0   \n",
       "\n",
       "   ZEB2     RXRA  SPI1  MEF2C    MEF2D  \n",
       "0   0.0  8.51315   0.0    0.0  3.86739  \n",
       "1   0.0  0.00000   0.0    0.0  0.00000  \n",
       "2   0.0  0.00000   0.0    0.0  0.00000  \n",
       "3   0.0  0.00000   0.0    0.0  4.68577  \n",
       "4   0.0  0.00000   0.0    0.0  0.00000  \n",
       "\n",
       "[5 rows x 37 columns]"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bindings.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0a816f1e",
   "metadata": {},
   "source": [
    "# Map cobinding matrix to SE regions"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b16b881c",
   "metadata": {},
   "source": [
    "map cobinding peaks to SE regions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "35111404",
   "metadata": {},
   "outputs": [],
   "source": [
    "window = 150\n",
    "merging_version = \"simpleMerge\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "896a053c",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(6868, 6)"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "superenhancers.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "f278fa72",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(383704, 37)"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bindings.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "1720845c",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>SE_N</th>\n",
       "      <th>REGION_ID</th>\n",
       "      <th>start</th>\n",
       "      <th>end</th>\n",
       "      <th>chrom</th>\n",
       "      <th>ALL_GENES</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>1201</th>\n",
       "      <td>1243</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_1243_lociStitched</td>\n",
       "      <td>88518136</td>\n",
       "      <td>88536335</td>\n",
       "      <td>1</td>\n",
       "      <td>ZC3H18 ZFPM1 ZNF469</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "0     SE_N                                      REGION_ID     start       end  \\\n",
       "1201  1243  1_merged_PDX_cell_lines_AML_1243_lociStitched  88518136  88536335   \n",
       "\n",
       "0    chrom            ALL_GENES  \n",
       "1201     1  ZC3H18 ZFPM1 ZNF469  "
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# one region has chrom as 1 (int)\n",
    "superenhancers.loc[superenhancers[\"chrom\"] == 1,]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "6ca23a42",
   "metadata": {},
   "outputs": [],
   "source": [
    "SEs = superenhancers.sort_values(by=[\"chrom\", \"start\", \"end\"])\n",
    "# replace 1 with chr1\n",
    "SEs[\"chrom\"] = SEs[\"chrom\"].replace([1], \"chr1\")\n",
    "# sort SE peaks by chr and start\n",
    "SEs = SEs.sort_values(by=[\"chrom\", \"start\", \"end\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "id": "aae78f5e",
   "metadata": {},
   "outputs": [],
   "source": [
    "# SE regions\n",
    "conscensus = SEs.sort_values(by=[\"chrom\", \"start\", \"end\"]).reset_index(drop=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "id": "0bf4f4cb",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(383704, 37)"
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# sort binding peaks by chr and start\n",
    "peaks = bindings.sort_values(by=[\"chrom\", \"start\", \"end\"]).reset_index(drop=True)\n",
    "# name peak regions\n",
    "peaks[\"peak_number\"] = peaks[\"peak_number\"].apply(lambda row: \"cobinding_peak_\"+str(row+1))\n",
    "peaks.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "id": "8745c59e",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(382746, 37)"
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# remove weird chromosomes\n",
    "peaks = peaks[~peaks[\"chrom\"].str.contains(\"_\")]\n",
    "# set coords to int\n",
    "peaks.start = peaks.start.astype(int)\n",
    "peaks.end = peaks.end.astype(int)\n",
    "peaks.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "id": "f54839e2",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(382027, 37)"
      ]
     },
     "execution_count": 36,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# get only peaks w/ same chromosomes as conscensus\n",
    "peaks = peaks[peaks[\"chrom\"].isin(conscensus[\"chrom\"])]\n",
    "peaks.shape"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a800a7d0",
   "metadata": {},
   "source": [
    "__overlapping testing__"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 163,
   "id": "347cd5c1",
   "metadata": {},
   "outputs": [],
   "source": [
    "mergedpeaksdict = {}\n",
    "peaknumber=0\n",
    "prev_start=conscensus.iloc[0][\"start\"]\n",
    "prev_end=conscensus.iloc[0][\"end\"]\n",
    "prev_chrom=conscensus.iloc[0][\"chrom\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 211,
   "id": "bfb63ae2",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(31340, 37)\n",
      "['GFI1', 'RUNX1', 'SP1', 'E2F3', 'STAT5B', 'TFAP4', 'RUNX2', 'RXRA', 'MEF2D']\n",
      "['MYC']\n",
      "['PLAGL2']\n",
      "['MEIS1', 'GFI1', 'RUNX1', 'SP1', 'ZNF281', 'STAT5B', 'MYC', 'TFAP4', 'MEF2D']\n",
      "['ZNF281', 'MYC', 'TFAP4']\n",
      "(31340, 37)\n",
      "['GFI1', 'RUNX1', 'SP1', 'E2F3', 'STAT5B', 'TFAP4', 'RUNX2', 'RXRA', 'MEF2D']\n",
      "['MYC']\n",
      "['PLAGL2']\n",
      "['MEIS1', 'GFI1', 'RUNX1', 'SP1', 'ZNF281', 'STAT5B', 'MYC', 'TFAP4', 'MEF2D']\n",
      "['ZNF281', 'MYC', 'TFAP4']\n"
     ]
    }
   ],
   "source": [
    "mergedpeaksdict={}\n",
    "for idx, row in conscensus.iloc[:2].iterrows():\n",
    "    peak_dict = {}\n",
    "    value = peaks.loc[peaks[\"chrom\"] == row.chrom]\n",
    "    print(values.shape)\n",
    "\n",
    "    for i, b in value.iloc[:5].iterrows():\n",
    "        if b.start<row.start:\n",
    "            print(b.index[8:][b.iloc[8:] != 0].tolist())\n",
    "            peak_dict[b.peak_number] = b.index[8:][b.iloc[8:] != 0].tolist()\n",
    "    mergedpeaksdict[row.REGION_ID] = peak_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 212,
   "id": "55e54407",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'1_merged_PDX_cell_lines_AML_7731_lociStitched': {'cobinding_peak_1': ['GFI1',\n",
       "   'RUNX1',\n",
       "   'SP1',\n",
       "   'E2F3',\n",
       "   'STAT5B',\n",
       "   'TFAP4',\n",
       "   'RUNX2',\n",
       "   'RXRA',\n",
       "   'MEF2D'],\n",
       "  'cobinding_peak_2': ['MYC'],\n",
       "  'cobinding_peak_3': ['PLAGL2'],\n",
       "  'cobinding_peak_4': ['MEIS1',\n",
       "   'GFI1',\n",
       "   'RUNX1',\n",
       "   'SP1',\n",
       "   'ZNF281',\n",
       "   'STAT5B',\n",
       "   'MYC',\n",
       "   'TFAP4',\n",
       "   'MEF2D'],\n",
       "  'cobinding_peak_5': ['ZNF281', 'MYC', 'TFAP4']},\n",
       " '1_merged_PDX_cell_lines_AML_1367_lociStitched': {'cobinding_peak_1': ['GFI1',\n",
       "   'RUNX1',\n",
       "   'SP1',\n",
       "   'E2F3',\n",
       "   'STAT5B',\n",
       "   'TFAP4',\n",
       "   'RUNX2',\n",
       "   'RXRA',\n",
       "   'MEF2D'],\n",
       "  'cobinding_peak_2': ['MYC'],\n",
       "  'cobinding_peak_3': ['PLAGL2'],\n",
       "  'cobinding_peak_4': ['MEIS1',\n",
       "   'GFI1',\n",
       "   'RUNX1',\n",
       "   'SP1',\n",
       "   'ZNF281',\n",
       "   'STAT5B',\n",
       "   'MYC',\n",
       "   'TFAP4',\n",
       "   'MEF2D'],\n",
       "  'cobinding_peak_5': ['ZNF281', 'MYC', 'TFAP4']}}"
      ]
     },
     "execution_count": 212,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "mergedpeaksdict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 218,
   "id": "ac4dbb0a",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>1_merged_PDX_cell_lines_AML_7731_lociStitched</th>\n",
       "      <th>1_merged_PDX_cell_lines_AML_1367_lociStitched</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>cobinding_peak_1</th>\n",
       "      <td>[GFI1, RUNX1, SP1, E2F3, STAT5B, TFAP4, RUNX2,...</td>\n",
       "      <td>[GFI1, RUNX1, SP1, E2F3, STAT5B, TFAP4, RUNX2,...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cobinding_peak_2</th>\n",
       "      <td>[MYC]</td>\n",
       "      <td>[MYC]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cobinding_peak_3</th>\n",
       "      <td>[PLAGL2]</td>\n",
       "      <td>[PLAGL2]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cobinding_peak_4</th>\n",
       "      <td>[MEIS1, GFI1, RUNX1, SP1, ZNF281, STAT5B, MYC,...</td>\n",
       "      <td>[MEIS1, GFI1, RUNX1, SP1, ZNF281, STAT5B, MYC,...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>cobinding_peak_5</th>\n",
       "      <td>[ZNF281, MYC, TFAP4]</td>\n",
       "      <td>[ZNF281, MYC, TFAP4]</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                      1_merged_PDX_cell_lines_AML_7731_lociStitched  \\\n",
       "cobinding_peak_1  [GFI1, RUNX1, SP1, E2F3, STAT5B, TFAP4, RUNX2,...   \n",
       "cobinding_peak_2                                              [MYC]   \n",
       "cobinding_peak_3                                           [PLAGL2]   \n",
       "cobinding_peak_4  [MEIS1, GFI1, RUNX1, SP1, ZNF281, STAT5B, MYC,...   \n",
       "cobinding_peak_5                               [ZNF281, MYC, TFAP4]   \n",
       "\n",
       "                      1_merged_PDX_cell_lines_AML_1367_lociStitched  \n",
       "cobinding_peak_1  [GFI1, RUNX1, SP1, E2F3, STAT5B, TFAP4, RUNX2,...  \n",
       "cobinding_peak_2                                              [MYC]  \n",
       "cobinding_peak_3                                           [PLAGL2]  \n",
       "cobinding_peak_4  [MEIS1, GFI1, RUNX1, SP1, ZNF281, STAT5B, MYC,...  \n",
       "cobinding_peak_5                               [ZNF281, MYC, TFAP4]  "
      ]
     },
     "execution_count": 218,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pd.DataFrame.from_dict(mergedpeaksdict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "97feb918",
   "metadata": {},
   "outputs": [],
   "source": [
    "mergedpeaksdict={}\n",
    "for idx, row in conscensus.iloc[:2].iterrows():\n",
    "    peak_dict = {}\n",
    "    value = peaks.loc[peaks[\"chrom\"] == row.chrom]\n",
    "    print(values.shape)\n",
    "\n",
    "    for i, b in value.iloc[:5].iterrows():\n",
    "        if b.start<row.start:\n",
    "            print(b.index[8:][b.iloc[8:] != 0].tolist())\n",
    "            peak_dict[b.peak_number] = b.index[8:][b.iloc[8:] != 0].tolist()\n",
    "    mergedpeaksdict[row.REGION_ID] = peak_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 224,
   "id": "ba81127d",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>SE_N</th>\n",
       "      <th>REGION_ID</th>\n",
       "      <th>start</th>\n",
       "      <th>end</th>\n",
       "      <th>chrom</th>\n",
       "      <th>ALL_GENES</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>7731</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_7731_lociStitched</td>\n",
       "      <td>1076094</td>\n",
       "      <td>1083036</td>\n",
       "      <td>chr1</td>\n",
       "      <td>C1orf159 MIR200A MIR200B MIR429 TTLL10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1367</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_1367_lociStitched</td>\n",
       "      <td>1138179</td>\n",
       "      <td>1155728</td>\n",
       "      <td>chr1</td>\n",
       "      <td>B3GALT6 FAM132A MIR200A MIR200B MIR429 SDF4 SM...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>8547</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_8547_lociStitched</td>\n",
       "      <td>1365724</td>\n",
       "      <td>1372115</td>\n",
       "      <td>chr1</td>\n",
       "      <td>ATAD3A ATAD3B ATAD3C AURKAIP1 CCNL2 DVL1 LOC14...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>16159</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_16159_lociStitched</td>\n",
       "      <td>1589785</td>\n",
       "      <td>1591611</td>\n",
       "      <td>chr1</td>\n",
       "      <td>ATAD3A CDK11B MIB2 MMP23A MMP23B MORN1 RER1 SL...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>4799</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_4799_lociStitched</td>\n",
       "      <td>1706094</td>\n",
       "      <td>1715621</td>\n",
       "      <td>chr1</td>\n",
       "      <td>GABRD GNB1 NADK SDF4 SLC35E2</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "0   SE_N                                       REGION_ID    start      end  \\\n",
       "0   7731   1_merged_PDX_cell_lines_AML_7731_lociStitched  1076094  1083036   \n",
       "1   1367   1_merged_PDX_cell_lines_AML_1367_lociStitched  1138179  1155728   \n",
       "2   8547   1_merged_PDX_cell_lines_AML_8547_lociStitched  1365724  1372115   \n",
       "3  16159  1_merged_PDX_cell_lines_AML_16159_lociStitched  1589785  1591611   \n",
       "4   4799   1_merged_PDX_cell_lines_AML_4799_lociStitched  1706094  1715621   \n",
       "\n",
       "0 chrom                                          ALL_GENES  \n",
       "0  chr1             C1orf159 MIR200A MIR200B MIR429 TTLL10  \n",
       "1  chr1  B3GALT6 FAM132A MIR200A MIR200B MIR429 SDF4 SM...  \n",
       "2  chr1  ATAD3A ATAD3B ATAD3C AURKAIP1 CCNL2 DVL1 LOC14...  \n",
       "3  chr1  ATAD3A CDK11B MIB2 MMP23A MMP23B MORN1 RER1 SL...  \n",
       "4  chr1                       GABRD GNB1 NADK SDF4 SLC35E2  "
      ]
     },
     "execution_count": 224,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "conscensus.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e0a65d44",
   "metadata": {},
   "source": [
    "__overlapping conditions__\n",
    "* peak overlaps SE start (peak.start < SE.start and peak.end > SE.start)\n",
    "* peak overlaps SE end (peak.start > SE.start and peak.end > SE.end)\n",
    "* peak within SE (peak.start > SE.start and peak.end < SE.end)\n",
    "* peak larger than SE (peak.start < SE.start and peak.end > SE.end)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 330,
   "id": "3a31adb4",
   "metadata": {},
   "outputs": [],
   "source": [
    "# presence df\n",
    "res_cols = conscensus.columns.tolist()+[\"N_PEAKS\"]+peaks.columns.tolist()[8:]\n",
    "res_rows = []\n",
    "\n",
    "# all peaks df columns\n",
    "split_cols = conscensus.columns.tolist()+[peaks.columns.tolist()[3]]+peaks.columns.tolist()[8:]\n",
    "# rows of df with all peaks\n",
    "split_rows = []\n",
    "\n",
    "for idx, row in conscensus.iloc[:2].iterrows():\n",
    "    # get only peaks w/ same chromosome\n",
    "    peaks_chr = peaks.loc[peaks[\"chrom\"] == row.chrom]\n",
    "    # make list of 0s for each tf +1 for # peaks\n",
    "    merged_res = np.zeros(len(mergedres_cols[6:]))\n",
    "    # for each peak\n",
    "    for i, peak in peaks_chr.iloc[80:90].iterrows():\n",
    "        # peak overlaps SE start or peak is larger than SE\n",
    "        # peak on or greater than start or peak within\n",
    "        if (peak.start<row.start and peak.end>row.start) \\\n",
    "        or (peak.start>=row.start and peak.start<row.end):\n",
    "            merged_res = np.add(merged_res, [1]+[1 if x!= 0 else 0 for x in peak[8:].tolist()])\n",
    "            split_rows.append(row.tolist()+[peak.peak_number]+peak[8:].tolist())\n",
    "    res_rows.append(row.tolist()+merged_res.tolist())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 331,
   "id": "33f48e37",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(2, 36)\n",
      "(1, 36)\n"
     ]
    }
   ],
   "source": [
    "# SEs with merged peaks\n",
    "res = pd.DataFrame(res_rows, columns=res_cols)\n",
    "print(res.shape)\n",
    "# SEs per merged peak\n",
    "res_split = pd.DataFrame(split_rows, columns=split_cols)\n",
    "print(res_split.shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 332,
   "id": "6106a013",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>SE_N</th>\n",
       "      <th>REGION_ID</th>\n",
       "      <th>start</th>\n",
       "      <th>end</th>\n",
       "      <th>chrom</th>\n",
       "      <th>ALL_GENES</th>\n",
       "      <th>N_PEAKS</th>\n",
       "      <th>CEBPA</th>\n",
       "      <th>HOXA9</th>\n",
       "      <th>PLAGL2</th>\n",
       "      <th>...</th>\n",
       "      <th>TFAP4</th>\n",
       "      <th>RUNX2</th>\n",
       "      <th>MAX</th>\n",
       "      <th>IRF8</th>\n",
       "      <th>MYB</th>\n",
       "      <th>ZEB2</th>\n",
       "      <th>RXRA</th>\n",
       "      <th>SPI1</th>\n",
       "      <th>MEF2C</th>\n",
       "      <th>MEF2D</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>7731</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_7731_lociStitched</td>\n",
       "      <td>1076094</td>\n",
       "      <td>1083036</td>\n",
       "      <td>chr1</td>\n",
       "      <td>C1orf159 MIR200A MIR200B MIR429 TTLL10</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1367</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_1367_lociStitched</td>\n",
       "      <td>1138179</td>\n",
       "      <td>1155728</td>\n",
       "      <td>chr1</td>\n",
       "      <td>B3GALT6 FAM132A MIR200A MIR200B MIR429 SDF4 SM...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows × 36 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "   SE_N                                      REGION_ID    start      end  \\\n",
       "0  7731  1_merged_PDX_cell_lines_AML_7731_lociStitched  1076094  1083036   \n",
       "1  1367  1_merged_PDX_cell_lines_AML_1367_lociStitched  1138179  1155728   \n",
       "\n",
       "  chrom                                          ALL_GENES  N_PEAKS  CEBPA  \\\n",
       "0  chr1             C1orf159 MIR200A MIR200B MIR429 TTLL10      1.0    0.0   \n",
       "1  chr1  B3GALT6 FAM132A MIR200A MIR200B MIR429 SDF4 SM...      0.0    0.0   \n",
       "\n",
       "   HOXA9  PLAGL2  ...  TFAP4  RUNX2  MAX  IRF8  MYB  ZEB2  RXRA  SPI1  MEF2C  \\\n",
       "0    0.0     0.0  ...    0.0    0.0  1.0   0.0  0.0   0.0   0.0   0.0    0.0   \n",
       "1    0.0     0.0  ...    0.0    0.0  0.0   0.0  0.0   0.0   0.0   0.0    0.0   \n",
       "\n",
       "   MEF2D  \n",
       "0    0.0  \n",
       "1    0.0  \n",
       "\n",
       "[2 rows x 36 columns]"
      ]
     },
     "execution_count": 332,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "res"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 333,
   "id": "4c0106e6",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>SE_N</th>\n",
       "      <th>REGION_ID</th>\n",
       "      <th>start</th>\n",
       "      <th>end</th>\n",
       "      <th>chrom</th>\n",
       "      <th>ALL_GENES</th>\n",
       "      <th>peak_number</th>\n",
       "      <th>CEBPA</th>\n",
       "      <th>HOXA9</th>\n",
       "      <th>PLAGL2</th>\n",
       "      <th>...</th>\n",
       "      <th>TFAP4</th>\n",
       "      <th>RUNX2</th>\n",
       "      <th>MAX</th>\n",
       "      <th>IRF8</th>\n",
       "      <th>MYB</th>\n",
       "      <th>ZEB2</th>\n",
       "      <th>RXRA</th>\n",
       "      <th>SPI1</th>\n",
       "      <th>MEF2C</th>\n",
       "      <th>MEF2D</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>7731</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_7731_lociStitched</td>\n",
       "      <td>1076094</td>\n",
       "      <td>1083036</td>\n",
       "      <td>chr1</td>\n",
       "      <td>C1orf159 MIR200A MIR200B MIR429 TTLL10</td>\n",
       "      <td>cobinding_peak_84</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>8.739785</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows × 36 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "   SE_N                                      REGION_ID    start      end  \\\n",
       "0  7731  1_merged_PDX_cell_lines_AML_7731_lociStitched  1076094  1083036   \n",
       "\n",
       "  chrom                               ALL_GENES        peak_number  CEBPA  \\\n",
       "0  chr1  C1orf159 MIR200A MIR200B MIR429 TTLL10  cobinding_peak_84    0.0   \n",
       "\n",
       "   HOXA9  PLAGL2  ...  TFAP4  RUNX2       MAX  IRF8  MYB  ZEB2  RXRA  SPI1  \\\n",
       "0    0.0     0.0  ...    0.0    0.0  8.739785   0.0  0.0   0.0   0.0   0.0   \n",
       "\n",
       "   MEF2C  MEF2D  \n",
       "0    0.0    0.0  \n",
       "\n",
       "[1 rows x 36 columns]"
      ]
     },
     "execution_count": 333,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "res_split"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 286,
   "id": "030ef71c",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2 5\n",
      "overlapping\n",
      "5 7\n",
      "overlapping\n",
      "7 9\n",
      "overlapping\n",
      "2 10\n",
      "overlapping\n",
      "0 3\n",
      "not overlapping\n",
      "9 12\n",
      "not overlapping\n"
     ]
    }
   ],
   "source": [
    "# SE = [4, 8]\n",
    "for x,y in [[2,5], [5,7], [7,9], [2, 10], [0,3], [9,12]]:\n",
    "    print(x, y)\n",
    "    if (x<=4 and y>4) or (x>=4 and x<8):\n",
    "        print(\"overlapping\")\n",
    "    else:\n",
    "        print(\"not overlapping\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 292,
   "id": "ca2cef51",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1076094 1083036\n"
     ]
    }
   ],
   "source": [
    "print(conscensus.iloc[0].start, conscensus.iloc[0].end)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 311,
   "id": "22c0a504",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['no', 'no', 'no', 'yes', 'no', 'no', 'no', 'no', 'no', 'no']"
      ]
     },
     "execution_count": 311,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "[\"yes\" if (x.start<conscensus.iloc[0].start and x.end>conscensus.iloc[0].start) or\n",
    " (x.start>=conscensus.iloc[0].start and x.start<conscensus.iloc[0].end) else \"no\" \n",
    " for i,x in peaks_chr.iloc[80:90].iterrows()]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 310,
   "id": "343be42b",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1076094 1083036\n",
      "\n",
      "\n",
      "1067690 1068155\n",
      "not overlapping\n",
      "1068551 1069388\n",
      "not overlapping\n",
      "1074770 1075098\n",
      "not overlapping\n",
      "1079409 1080385\n",
      "overlapping end or within\n",
      "1091300 1091850\n",
      "not overlapping\n",
      "1097970 1098314\n",
      "not overlapping\n",
      "1103921 1104665\n",
      "not overlapping\n",
      "1112303 1112734\n",
      "not overlapping\n",
      "1115570 1116445\n",
      "not overlapping\n",
      "1117255 1117850\n",
      "not overlapping\n"
     ]
    }
   ],
   "source": [
    "print(conscensus.iloc[0].start, conscensus.iloc[0].end)\n",
    "print(\"\\n\")\n",
    "\n",
    "for i,x in peaks_chr.iloc[80:90].iterrows():\n",
    "    print(x.start, x.end)\n",
    "    if (x.start < conscensus.iloc[0].start and x.end > conscensus.iloc[0].start):\n",
    "        print(\"overlapping start\")\n",
    "        \n",
    "    elif (x.start >= conscensus.iloc[0].start and x.start <= conscensus.iloc[0].end):\n",
    "        print(\"overlapping end or within\")\n",
    "    \n",
    "    else:\n",
    "        print(\"not overlapping\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "11cad56b",
   "metadata": {},
   "source": [
    "__run over both matricies__"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "id": "31d97536",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SE regions: 6868\n",
      "Binding peaks: 382027\n"
     ]
    }
   ],
   "source": [
    "print(\"SE regions: {}\".format(conscensus.shape[0]))\n",
    "print(\"Binding peaks: {}\".format(peaks.shape[0]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 336,
   "id": "5b300083",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SE regions: 6868\n",
      "Binding peaks: 382027\n"
     ]
    }
   ],
   "source": [
    "print(\"SE regions: {}\".format(conscensus.shape[0]))\n",
    "print(\"Binding peaks: {}\".format(peaks.shape[0]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "id": "d45d182e",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "36"
      ]
     },
     "execution_count": 41,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(conscensus.columns.tolist()+[\"N_PEAKS\"]+peaks.columns.tolist()[8:])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "id": "af84c294",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "36"
      ]
     },
     "execution_count": 42,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(conscensus.columns.tolist()+[peaks.columns.tolist()[3]]+peaks.columns.tolist()[8:])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "61472f12",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1000 SEs mapped to\n",
      "2000 SEs mapped to\n",
      "3000 SEs mapped to\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "\n",
    "# presence df\n",
    "res_cols = conscensus.columns.tolist()+[\"N_PEAKS\"]+peaks.columns.tolist()[8:]\n",
    "res_rows = []\n",
    "\n",
    "# all peaks df columns\n",
    "split_cols = conscensus.columns.tolist()+[peaks.columns.tolist()[3]]+peaks.columns.tolist()[8:]\n",
    "# rows of df with all peaks\n",
    "split_rows = []\n",
    "\n",
    "ix = 0\n",
    "for idx, row in conscensus.iterrows():\n",
    "    # get only peaks w/ same chromosome\n",
    "    peaks_chr = peaks.loc[peaks[\"chrom\"] == row.chrom]\n",
    "    # make list of 0s for each tf +1 for # peaks\n",
    "    merged_res = np.zeros(len(res_cols[6:]))\n",
    "    # for each peak\n",
    "    i = 0\n",
    "    for i, peak in peaks_chr.iterrows():\n",
    "        # peak overlaps SE start or peak is larger than SE\n",
    "        # peak on or greater than start or peak within\n",
    "        if (peak.start<row.start and peak.end>row.start) \\\n",
    "        or (peak.start>=row.start and peak.start<row.end):\n",
    "            merged_res = np.add(merged_res, [1]+[1 if x!= 0 else 0 for x in peak[8:].tolist()])\n",
    "            split_rows.append(row.tolist()+[peak.peak_number]+peak[8:].tolist())\n",
    "            i = i+1\n",
    "        else:\n",
    "            i = i+1\n",
    "    #print(\"Binding peaks mapped: {}\".format(row.REGION_ID))\n",
    "    res_rows.append(row.tolist()+merged_res.tolist())\n",
    "    ix = ix+1\n",
    "    if (ix%1000 == 0): print(\"{} SEs mapped to\".format(ix))\n",
    "\n",
    "\n",
    "# SEs with merged peaks\n",
    "res = pd.DataFrame(res_rows, columns=res_cols)\n",
    "print(res.shape)\n",
    "# SEs per merged peak\n",
    "res_split = pd.DataFrame(split_rows, columns=split_cols)\n",
    "print(res_split.shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 356,
   "id": "0925b5fe",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1000 SEs mapped to\n",
      "2000 SEs mapped to\n",
      "3000 SEs mapped to\n",
      "4000 SEs mapped to\n",
      "5000 SEs mapped to\n",
      "(5868, 36)\n",
      "(13701, 36)\n",
      "CPU times: user 1h 48min 22s, sys: 312 ms, total: 1h 48min 23s\n",
      "Wall time: 1h 48min 23s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "\n",
    "# presence df\n",
    "res_cols = conscensus.columns.tolist()+[\"N_PEAKS\"]+peaks.columns.tolist()[8:]\n",
    "res_rows = []\n",
    "\n",
    "# all peaks df columns\n",
    "split_cols = conscensus.columns.tolist()+[peaks.columns.tolist()[3]]+peaks.columns.tolist()[8:]\n",
    "# rows of df with all peaks\n",
    "split_rows = []\n",
    "\n",
    "ix = 0\n",
    "for idx, row in conscensus.iloc[1000:].iterrows():\n",
    "    # get only peaks w/ same chromosome\n",
    "    peaks_chr = peaks.loc[peaks[\"chrom\"] == row.chrom]\n",
    "    # make list of 0s for each tf +1 for # peaks\n",
    "    merged_res = np.zeros(len(mergedres_cols[6:]))\n",
    "    # for each peak\n",
    "    i = 0\n",
    "    for i, peak in peaks_chr.iterrows():\n",
    "        # peak overlaps SE start or peak is larger than SE\n",
    "        # peak on or greater than start or peak within\n",
    "        if (peak.start<row.start and peak.end>row.start) \\\n",
    "        or (peak.start>=row.start and peak.start<row.end):\n",
    "            merged_res = np.add(merged_res, [1]+[1 if x!= 0 else 0 for x in peak[8:].tolist()])\n",
    "            split_rows.append(row.tolist()+[peak.peak_number]+peak[8:].tolist())\n",
    "            i = i+1\n",
    "        else:\n",
    "            i = i+1\n",
    "    #print(\"Binding peaks mapped: {}\".format(row.REGION_ID))\n",
    "    res_rows.append(row.tolist()+merged_res.tolist())\n",
    "    ix = ix+1\n",
    "    if (ix%1000 == 0): print(\"{} SEs mapped to\".format(ix))\n",
    "\n",
    "\n",
    "# SEs with merged peaks\n",
    "res = pd.DataFrame(res_rows, columns=res_cols)\n",
    "print(res.shape)\n",
    "# SEs per merged peak\n",
    "res_split = pd.DataFrame(split_rows, columns=split_cols)\n",
    "print(res_split.shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 350,
   "id": "e3374d9b",
   "metadata": {},
   "outputs": [],
   "source": [
    "# first 1000 SE regions\n",
    "res_base = res\n",
    "res_split_base = res_split"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 351,
   "id": "7c531ef7",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(1000, 36)\n",
      "(2090, 36)\n"
     ]
    }
   ],
   "source": [
    "print(res_base.shape)\n",
    "print(res_split_base.shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 359,
   "id": "382a2476",
   "metadata": {},
   "outputs": [],
   "source": [
    "# last 5868 SE regions\n",
    "res_new = res\n",
    "res_split_new = res_split"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 364,
   "id": "04c819d2",
   "metadata": {},
   "outputs": [],
   "source": [
    "# combine all SE regions\n",
    "res = pd.concat([res_base, res_new])\n",
    "res_split = pd.concat([res_split_base, res_split_new])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 365,
   "id": "ba204f29",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(6868, 36)\n",
      "(15791, 36)\n"
     ]
    }
   ],
   "source": [
    "print(res.shape)\n",
    "print(res_split.shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 371,
   "id": "04316dc7",
   "metadata": {},
   "outputs": [],
   "source": [
    "# save to csv\n",
    "res.to_csv(\"../results/\"+cobinding_project+\"/\"+cobinding_version+\"/\"+cobinding_version+\n",
    "           \"_SE_matrix_merged.csv.gz\", index=False)\n",
    "res_split.to_csv(\"../results/\"+cobinding_project+\"/\"+cobinding_version+\"/\"+cobinding_version+\n",
    "                 \"_SE_matrix_merged_split.csv.gz\", index=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "fa7c205a",
   "metadata": {},
   "source": [
    "## Map genes to SE"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8a461040",
   "metadata": {},
   "source": [
    "* res = cobinding mapped to SE regions\n",
    "* SEs = sorted SE matrix"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 375,
   "id": "bd3cfd2f",
   "metadata": {},
   "outputs": [],
   "source": [
    "SEs = SEs.sort_values(by=[\"chrom\", \"start\", \"end\"]).reset_index(drop=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 378,
   "id": "9299b0de",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>SE_N</th>\n",
       "      <th>REGION_ID</th>\n",
       "      <th>start</th>\n",
       "      <th>end</th>\n",
       "      <th>chrom</th>\n",
       "      <th>ALL_GENES</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>7731</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_7731_lociStitched</td>\n",
       "      <td>1076094</td>\n",
       "      <td>1083036</td>\n",
       "      <td>chr1</td>\n",
       "      <td>C1orf159 MIR200A MIR200B MIR429 TTLL10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1367</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_1367_lociStitched</td>\n",
       "      <td>1138179</td>\n",
       "      <td>1155728</td>\n",
       "      <td>chr1</td>\n",
       "      <td>B3GALT6 FAM132A MIR200A MIR200B MIR429 SDF4 SM...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>8547</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_8547_lociStitched</td>\n",
       "      <td>1365724</td>\n",
       "      <td>1372115</td>\n",
       "      <td>chr1</td>\n",
       "      <td>ATAD3A ATAD3B ATAD3C AURKAIP1 CCNL2 DVL1 LOC14...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>16159</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_16159_lociStitched</td>\n",
       "      <td>1589785</td>\n",
       "      <td>1591611</td>\n",
       "      <td>chr1</td>\n",
       "      <td>ATAD3A CDK11B MIB2 MMP23A MMP23B MORN1 RER1 SL...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>4799</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_4799_lociStitched</td>\n",
       "      <td>1706094</td>\n",
       "      <td>1715621</td>\n",
       "      <td>chr1</td>\n",
       "      <td>GABRD GNB1 NADK SDF4 SLC35E2</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "0   SE_N                                       REGION_ID    start      end  \\\n",
       "0   7731   1_merged_PDX_cell_lines_AML_7731_lociStitched  1076094  1083036   \n",
       "1   1367   1_merged_PDX_cell_lines_AML_1367_lociStitched  1138179  1155728   \n",
       "2   8547   1_merged_PDX_cell_lines_AML_8547_lociStitched  1365724  1372115   \n",
       "3  16159  1_merged_PDX_cell_lines_AML_16159_lociStitched  1589785  1591611   \n",
       "4   4799   1_merged_PDX_cell_lines_AML_4799_lociStitched  1706094  1715621   \n",
       "\n",
       "0 chrom                                          ALL_GENES  \n",
       "0  chr1             C1orf159 MIR200A MIR200B MIR429 TTLL10  \n",
       "1  chr1  B3GALT6 FAM132A MIR200A MIR200B MIR429 SDF4 SM...  \n",
       "2  chr1  ATAD3A ATAD3B ATAD3C AURKAIP1 CCNL2 DVL1 LOC14...  \n",
       "3  chr1  ATAD3A CDK11B MIB2 MMP23A MMP23B MORN1 RER1 SL...  \n",
       "4  chr1                       GABRD GNB1 NADK SDF4 SLC35E2  "
      ]
     },
     "execution_count": 378,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "SEs.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b5835825",
   "metadata": {},
   "source": [
    "seperate all genes and get list of unique genes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 382,
   "id": "9a1c932c",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0            [C1orf159, MIR200A, MIR200B, MIR429, TTLL10]\n",
       "1       [B3GALT6, FAM132A, MIR200A, MIR200B, MIR429, S...\n",
       "2       [ATAD3A, ATAD3B, ATAD3C, AURKAIP1, CCNL2, DVL1...\n",
       "3       [ATAD3A, CDK11B, MIB2, MMP23A, MMP23B, MORN1, ...\n",
       "4                      [GABRD, GNB1, NADK, SDF4, SLC35E2]\n",
       "                              ...                        \n",
       "6863    [ARHGAP4, HCFC1, IRAK1, MIR3202-1, MIR3202-2, ...\n",
       "6864           [DNASE1L1, EMD, FLNA, RPL10, SNORA70, TAZ]\n",
       "6865                      [DKC1, GAB3, SNORA36A, SNORA56]\n",
       "6866    [DKC1, GAB3, LOC100132963, MPP1, SNORA36A, SNO...\n",
       "6867                                               [IL9R]\n",
       "Name: ALL_GENES, Length: 6868, dtype: object"
      ]
     },
     "execution_count": 382,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "SEs['ALL_GENES'].astype(str).apply(lambda x: x.split(\" \"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 399,
   "id": "9428ab3e",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "71\n",
      "CPU times: user 0 ns, sys: 0 ns, total: 0 ns\n",
      "Wall time: 1.23 ms\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "\n",
    "unique_genes = []\n",
    "for idx, row in SEs.iloc[:20].iterrows():\n",
    "    region = row[\"REGION_ID\"]\n",
    "    gene_list = row[\"ALL_GENES\"]\n",
    "    for gene in gene_list.split(\" \"):\n",
    "        if gene not in unique_genes:\n",
    "            unique_genes.append(gene)\n",
    "\n",
    "print(len(unique_genes))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 403,
   "id": "7d5a8916",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['ACAP3',\n",
       " 'ACTRT2',\n",
       " 'ARHGEF16',\n",
       " 'ATAD3A',\n",
       " 'ATAD3B',\n",
       " 'ATAD3C',\n",
       " 'AURKAIP1',\n",
       " 'B3GALT6',\n",
       " 'C1orf159',\n",
       " 'C1orf174',\n",
       " 'C1orf86',\n",
       " 'C1orf93',\n",
       " 'CALML6',\n",
       " 'CCDC27',\n",
       " 'CCNL2',\n",
       " 'CDK11B',\n",
       " 'CEP104',\n",
       " 'DFFB',\n",
       " 'DVL1',\n",
       " 'FAM132A',\n",
       " 'FLJ42875',\n",
       " 'GABRD',\n",
       " 'GNB1',\n",
       " 'HES5',\n",
       " 'KIAA0495',\n",
       " 'LOC100128003',\n",
       " 'LOC100129534',\n",
       " 'LOC115110',\n",
       " 'LOC148413',\n",
       " 'LOC441869',\n",
       " 'LRRC47',\n",
       " 'MEGF6',\n",
       " 'MIB2',\n",
       " 'MIR200A',\n",
       " 'MIR200B',\n",
       " 'MIR4251',\n",
       " 'MIR429',\n",
       " 'MIR551A',\n",
       " 'MMEL1',\n",
       " 'MMP23A',\n",
       " 'MMP23B',\n",
       " 'MORN1',\n",
       " 'MRPL20',\n",
       " 'NADK',\n",
       " 'PANK4',\n",
       " 'PEX10',\n",
       " 'PRDM16',\n",
       " 'PRKCZ',\n",
       " 'PUSL1',\n",
       " 'RER1',\n",
       " 'SAMD11',\n",
       " 'SDF4',\n",
       " 'SKI',\n",
       " 'SLC35E2',\n",
       " 'SLC35E2B',\n",
       " 'SMIM1',\n",
       " 'SSU72',\n",
       " 'THAP3',\n",
       " 'TMEM52',\n",
       " 'TMEM88B',\n",
       " 'TNFRSF14',\n",
       " 'TNFRSF18',\n",
       " 'TNFRSF25',\n",
       " 'TNFRSF4',\n",
       " 'TP73',\n",
       " 'TPRG1L',\n",
       " 'TTLL10',\n",
       " 'UBE2J2',\n",
       " 'VWA1',\n",
       " 'WDR8',\n",
       " 'WRAP73']"
      ]
     },
     "execution_count": 403,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sorted(unique_genes)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 406,
   "id": "78b574e6",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>SE_N</th>\n",
       "      <th>REGION_ID</th>\n",
       "      <th>start</th>\n",
       "      <th>end</th>\n",
       "      <th>chrom</th>\n",
       "      <th>ALL_GENES</th>\n",
       "      <th>N_PEAKS</th>\n",
       "      <th>CEBPA</th>\n",
       "      <th>HOXA9</th>\n",
       "      <th>PLAGL2</th>\n",
       "      <th>...</th>\n",
       "      <th>TFAP4</th>\n",
       "      <th>RUNX2</th>\n",
       "      <th>MAX</th>\n",
       "      <th>IRF8</th>\n",
       "      <th>MYB</th>\n",
       "      <th>ZEB2</th>\n",
       "      <th>RXRA</th>\n",
       "      <th>SPI1</th>\n",
       "      <th>MEF2C</th>\n",
       "      <th>MEF2D</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>7731</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_7731_lociStitched</td>\n",
       "      <td>1076094</td>\n",
       "      <td>1083036</td>\n",
       "      <td>chr1</td>\n",
       "      <td>C1orf159 MIR200A MIR200B MIR429 TTLL10</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows × 36 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "   SE_N                                      REGION_ID    start      end  \\\n",
       "0  7731  1_merged_PDX_cell_lines_AML_7731_lociStitched  1076094  1083036   \n",
       "\n",
       "  chrom                               ALL_GENES  N_PEAKS  CEBPA  HOXA9  \\\n",
       "0  chr1  C1orf159 MIR200A MIR200B MIR429 TTLL10      1.0    0.0    0.0   \n",
       "\n",
       "   PLAGL2  ...  TFAP4  RUNX2  MAX  IRF8  MYB  ZEB2  RXRA  SPI1  MEF2C  MEF2D  \n",
       "0     0.0  ...    0.0    0.0  1.0   0.0  0.0   0.0   0.0   0.0    0.0    0.0  \n",
       "\n",
       "[1 rows x 36 columns]"
      ]
     },
     "execution_count": 406,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "res.iloc[:1]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 408,
   "id": "719696d1",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['CEBPA',\n",
       " 'E2F3',\n",
       " 'FLI1',\n",
       " 'FOSL2',\n",
       " 'GFI1',\n",
       " 'HHEX',\n",
       " 'IRF8',\n",
       " 'LYL1',\n",
       " 'MEF2C',\n",
       " 'MEF2D',\n",
       " 'MEIS1',\n",
       " 'MYB',\n",
       " 'PLAGL2',\n",
       " 'RUNX1',\n",
       " 'RUNX2',\n",
       " 'RXRA',\n",
       " 'SP1',\n",
       " 'SPI1',\n",
       " 'SREBF1',\n",
       " 'STAT5B',\n",
       " 'TFAP4',\n",
       " 'ZEB2',\n",
       " 'ZFPM1',\n",
       " 'ZNF281',\n",
       " 'GATA2',\n",
       " 'MAX',\n",
       " 'MYC',\n",
       " 'ETV6',\n",
       " 'HOXA9',\n",
       " 'LMO2',\n",
       " 'ZMYND']"
      ]
     },
     "execution_count": 408,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "crcs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 423,
   "id": "32209171",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['CEBPA',\n",
       " 'HOXA9',\n",
       " 'PLAGL2',\n",
       " 'MEIS1',\n",
       " 'ETV6',\n",
       " 'GFI1',\n",
       " 'RUNX1',\n",
       " 'FOSL2',\n",
       " 'FLI1',\n",
       " 'GATA2',\n",
       " 'SP1',\n",
       " 'E2F3',\n",
       " 'ZNF281',\n",
       " 'SREBF1',\n",
       " 'LMO2',\n",
       " 'STAT5B',\n",
       " 'ZMYND8',\n",
       " 'LYL1',\n",
       " 'MYC',\n",
       " 'TFAP4',\n",
       " 'RUNX2',\n",
       " 'MAX',\n",
       " 'IRF8',\n",
       " 'MYB',\n",
       " 'ZEB2',\n",
       " 'RXRA',\n",
       " 'SPI1',\n",
       " 'MEF2C',\n",
       " 'MEF2D']"
      ]
     },
     "execution_count": 423,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bindings.columns.tolist()[-29:]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 459,
   "id": "7d371110",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>SE_N</th>\n",
       "      <th>REGION_ID</th>\n",
       "      <th>start</th>\n",
       "      <th>end</th>\n",
       "      <th>chrom</th>\n",
       "      <th>ALL_GENES</th>\n",
       "      <th>N_PEAKS</th>\n",
       "      <th>CEBPA</th>\n",
       "      <th>HOXA9</th>\n",
       "      <th>PLAGL2</th>\n",
       "      <th>...</th>\n",
       "      <th>TFAP4</th>\n",
       "      <th>RUNX2</th>\n",
       "      <th>MAX</th>\n",
       "      <th>IRF8</th>\n",
       "      <th>MYB</th>\n",
       "      <th>ZEB2</th>\n",
       "      <th>RXRA</th>\n",
       "      <th>SPI1</th>\n",
       "      <th>MEF2C</th>\n",
       "      <th>MEF2D</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>7731</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_7731_lociStitched</td>\n",
       "      <td>1076094</td>\n",
       "      <td>1083036</td>\n",
       "      <td>chr1</td>\n",
       "      <td>C1orf159 MIR200A MIR200B MIR429 TTLL10</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows × 36 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "   SE_N                                      REGION_ID    start      end  \\\n",
       "0  7731  1_merged_PDX_cell_lines_AML_7731_lociStitched  1076094  1083036   \n",
       "\n",
       "  chrom                               ALL_GENES  N_PEAKS  CEBPA  HOXA9  \\\n",
       "0  chr1  C1orf159 MIR200A MIR200B MIR429 TTLL10      1.0    0.0    0.0   \n",
       "\n",
       "   PLAGL2  ...  TFAP4  RUNX2  MAX  IRF8  MYB  ZEB2  RXRA  SPI1  MEF2C  MEF2D  \n",
       "0     0.0  ...    0.0    0.0  1.0   0.0  0.0   0.0   0.0   0.0    0.0    0.0  \n",
       "\n",
       "[1 rows x 36 columns]"
      ]
     },
     "execution_count": 459,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "res.iloc[:1]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 460,
   "id": "4e734c5a",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['N_PEAKS',\n",
       " 'CEBPA',\n",
       " 'HOXA9',\n",
       " 'PLAGL2',\n",
       " 'MEIS1',\n",
       " 'ETV6',\n",
       " 'GFI1',\n",
       " 'RUNX1',\n",
       " 'FOSL2',\n",
       " 'FLI1',\n",
       " 'GATA2',\n",
       " 'SP1',\n",
       " 'E2F3',\n",
       " 'ZNF281',\n",
       " 'SREBF1',\n",
       " 'LMO2',\n",
       " 'STAT5B',\n",
       " 'ZMYND8',\n",
       " 'LYL1',\n",
       " 'MYC',\n",
       " 'TFAP4',\n",
       " 'RUNX2',\n",
       " 'MAX',\n",
       " 'IRF8',\n",
       " 'MYB',\n",
       " 'ZEB2',\n",
       " 'RXRA',\n",
       " 'SPI1',\n",
       " 'MEF2C',\n",
       " 'MEF2D']"
      ]
     },
     "execution_count": 460,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "res.columns.tolist()[6:]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 470,
   "id": "3ff3c7a6",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1000 SEs processed\n",
      "Total genes: 1707\n",
      "Genes in df: 1707\n",
      "CPU times: user 196 ms, sys: 0 ns, total: 196 ms\n",
      "Wall time: 194 ms\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "\n",
    "# presence df\n",
    "df_cols = res.columns.tolist()[-30:] #[\"GENE\", \"CRC\"]\n",
    "res_dict = {}\n",
    "unique_genes = []\n",
    "\n",
    "ix = 0\n",
    "for idx, row in res.iloc[:1000].iterrows():\n",
    "    \n",
    "    # make list of 0s for each tf +1 for # peaks\n",
    "    #merged_res = np.zeros(len(mergedres_cols[-29:]))\n",
    "    \n",
    "    region = row[\"REGION_ID\"]\n",
    "    gene_list = str(row[\"ALL_GENES\"]).split(\" \")\n",
    "    for gene in gene_list:\n",
    "        if gene not in unique_genes:\n",
    "            unique_genes.append(gene)\n",
    "        if gene in res_dict:\n",
    "            res_dict[gene] = np.add(res_dict[gene], row[6:].tolist())\n",
    "        else:\n",
    "            res_dict[gene] = row[6:].tolist()\n",
    "\n",
    "    ix = ix+1\n",
    "    if (ix%1000 == 0): print(\"{} SEs processed\".format(ix))\n",
    "\n",
    "\n",
    "# SE genes with peaks\n",
    "df = pd.DataFrame.from_dict(res_dict, orient=\"index\", columns=df_cols)\n",
    "print(\"Total genes: {}\".format(len(unique_genes)))\n",
    "print(\"Genes in df: {}\".format(df.shape[0]))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "99fd047f",
   "metadata": {},
   "source": [
    "Time measures - SEs (genes)\n",
    "* 10 (52) - CPU: 4ms / Wall: 5.46ms\n",
    "* 100 (235) - CPU: 20ms / Wall: 21.5ms\n",
    "* 500 (1072) - CPU: 112ms / Wall: 111ms\n",
    "* 1000 (1707) - CPU: 196ms / Wall: 194ms"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 471,
   "id": "ac5d0e75",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>N_PEAKS</th>\n",
       "      <th>CEBPA</th>\n",
       "      <th>HOXA9</th>\n",
       "      <th>PLAGL2</th>\n",
       "      <th>MEIS1</th>\n",
       "      <th>ETV6</th>\n",
       "      <th>GFI1</th>\n",
       "      <th>RUNX1</th>\n",
       "      <th>FOSL2</th>\n",
       "      <th>FLI1</th>\n",
       "      <th>...</th>\n",
       "      <th>TFAP4</th>\n",
       "      <th>RUNX2</th>\n",
       "      <th>MAX</th>\n",
       "      <th>IRF8</th>\n",
       "      <th>MYB</th>\n",
       "      <th>ZEB2</th>\n",
       "      <th>RXRA</th>\n",
       "      <th>SPI1</th>\n",
       "      <th>MEF2C</th>\n",
       "      <th>MEF2D</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>C1orf159</th>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>MIR200A</th>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>MIR200B</th>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>MIR429</th>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>TTLL10</th>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows × 30 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "          N_PEAKS  CEBPA  HOXA9  PLAGL2  MEIS1  ETV6  GFI1  RUNX1  FOSL2  \\\n",
       "C1orf159      1.0    0.0    0.0     0.0    0.0   0.0   0.0    1.0    0.0   \n",
       "MIR200A       3.0    0.0    0.0     0.0    0.0   0.0   0.0    3.0    0.0   \n",
       "MIR200B       3.0    0.0    0.0     0.0    0.0   0.0   0.0    3.0    0.0   \n",
       "MIR429        3.0    0.0    0.0     0.0    0.0   0.0   0.0    3.0    0.0   \n",
       "TTLL10        3.0    0.0    0.0     0.0    0.0   0.0   0.0    3.0    0.0   \n",
       "\n",
       "          FLI1  ...  TFAP4  RUNX2  MAX  IRF8  MYB  ZEB2  RXRA  SPI1  MEF2C  \\\n",
       "C1orf159   0.0  ...    0.0    0.0  1.0   0.0  0.0   0.0   0.0   0.0    0.0   \n",
       "MIR200A    1.0  ...    0.0    0.0  3.0   0.0  1.0   1.0   1.0   0.0    0.0   \n",
       "MIR200B    1.0  ...    0.0    0.0  3.0   0.0  1.0   1.0   1.0   0.0    0.0   \n",
       "MIR429     1.0  ...    0.0    0.0  3.0   0.0  1.0   1.0   1.0   0.0    0.0   \n",
       "TTLL10     1.0  ...    0.0    0.0  3.0   0.0  1.0   1.0   1.0   0.0    0.0   \n",
       "\n",
       "          MEF2D  \n",
       "C1orf159    0.0  \n",
       "MIR200A     0.0  \n",
       "MIR200B     0.0  \n",
       "MIR429      0.0  \n",
       "TTLL10      0.0  \n",
       "\n",
       "[5 rows x 30 columns]"
      ]
     },
     "execution_count": 471,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 486,
   "id": "0a8f70c0",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "ename": "ValueError",
     "evalue": "Cannot mask with non-boolean array containing NA / NaN values",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mValueError\u001b[0m                                Traceback (most recent call last)",
      "\u001b[0;32m/tmp/ipykernel_14808/3869692415.py\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0msuperenhancers\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mloc\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0msuperenhancers\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m'ALL_GENES'\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mstr\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mcontains\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"CEBPA\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcase\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mFalse\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;32m~/anaconda3/envs/slamseq/lib/python3.8/site-packages/pandas/core/indexing.py\u001b[0m in \u001b[0;36m__getitem__\u001b[0;34m(self, key)\u001b[0m\n\u001b[1;32m    929\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    930\u001b[0m             \u001b[0mmaybe_callable\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mcom\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mapply_if_callable\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mkey\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mobj\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 931\u001b[0;31m             \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_getitem_axis\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mmaybe_callable\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0maxis\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0maxis\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    932\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    933\u001b[0m     \u001b[0;32mdef\u001b[0m \u001b[0m_is_scalar_access\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mkey\u001b[0m\u001b[0;34m:\u001b[0m \u001b[0mtuple\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/envs/slamseq/lib/python3.8/site-packages/pandas/core/indexing.py\u001b[0m in \u001b[0;36m_getitem_axis\u001b[0;34m(self, key, axis)\u001b[0m\n\u001b[1;32m   1141\u001b[0m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_validate_key\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mkey\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0maxis\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1142\u001b[0m             \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_get_slice_axis\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mkey\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0maxis\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0maxis\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1143\u001b[0;31m         \u001b[0;32melif\u001b[0m \u001b[0mcom\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mis_bool_indexer\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mkey\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1144\u001b[0m             \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_getbool_axis\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mkey\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0maxis\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0maxis\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1145\u001b[0m         \u001b[0;32melif\u001b[0m \u001b[0mis_list_like_indexer\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mkey\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/envs/slamseq/lib/python3.8/site-packages/pandas/core/common.py\u001b[0m in \u001b[0;36mis_bool_indexer\u001b[0;34m(key)\u001b[0m\n\u001b[1;32m    137\u001b[0m                     \u001b[0;31m# Don't raise on e.g. [\"A\", \"B\", np.nan], see\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    138\u001b[0m                     \u001b[0;31m#  test_loc_getitem_list_of_labels_categoricalindex_with_na\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 139\u001b[0;31m                     \u001b[0;32mraise\u001b[0m \u001b[0mValueError\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mna_msg\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    140\u001b[0m                 \u001b[0;32mreturn\u001b[0m \u001b[0;32mFalse\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    141\u001b[0m             \u001b[0;32mreturn\u001b[0m \u001b[0;32mTrue\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mValueError\u001b[0m: Cannot mask with non-boolean array containing NA / NaN values"
     ]
    }
   ],
   "source": [
    "superenhancers.loc[superenhancers['ALL_GENES'].str.contains(\"CEBPA\", case=False)]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 493,
   "id": "e3846840",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0\n",
       "SE_N                                                        40\n",
       "REGION_ID          1_merged_PDX_cell_lines_AML_40_lociStitched\n",
       "start                                                 33739117\n",
       "end                                                   33795715\n",
       "chrom                                                    chr19\n",
       "ALL_GENES    CCNE1 CEBPA GRAMD1A HSPB6 KCTD15 LOC80054 LRP3...\n",
       "Name: 40, dtype: object"
      ]
     },
     "execution_count": 493,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "superenhancers.iloc[39]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 490,
   "id": "54ce7aa7",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "40      True\n",
       "1426     NaN\n",
       "2087     NaN\n",
       "2503     NaN\n",
       "3121     NaN\n",
       "4068     NaN\n",
       "5187     NaN\n",
       "5938     NaN\n",
       "6394     NaN\n",
       "Name: ALL_GENES, dtype: object"
      ]
     },
     "execution_count": 490,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "superenhancers['ALL_GENES'].str.contains(\"CEBPA\", case=False)[superenhancers['ALL_GENES'].str.contains(\"CEBPA\", case=False) != False]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 499,
   "id": "ed451788",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>SE_N</th>\n",
       "      <th>REGION_ID</th>\n",
       "      <th>start</th>\n",
       "      <th>end</th>\n",
       "      <th>chrom</th>\n",
       "      <th>ALL_GENES</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>40</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_40_lociStitched</td>\n",
       "      <td>33739117</td>\n",
       "      <td>33795715</td>\n",
       "      <td>chr19</td>\n",
       "      <td>CCNE1 CEBPA GRAMD1A HSPB6 KCTD15 LOC80054 LRP3...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1426</th>\n",
       "      <td>1497</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_1497_lociStitched</td>\n",
       "      <td>45960034</td>\n",
       "      <td>45976885</td>\n",
       "      <td>chr10</td>\n",
       "      <td>43898</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2087</th>\n",
       "      <td>2264</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_2264_lociStitched</td>\n",
       "      <td>164508601</td>\n",
       "      <td>164522551</td>\n",
       "      <td>chr4</td>\n",
       "      <td>43891</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2503</th>\n",
       "      <td>2818</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_2818_lociStitched</td>\n",
       "      <td>75560044</td>\n",
       "      <td>75572674</td>\n",
       "      <td>chr17</td>\n",
       "      <td>44083</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3121</th>\n",
       "      <td>3663</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_3663_lociStitched</td>\n",
       "      <td>77906409</td>\n",
       "      <td>77917457</td>\n",
       "      <td>chr4</td>\n",
       "      <td>44085</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4068</th>\n",
       "      <td>5156</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_5156_lociStitched</td>\n",
       "      <td>164527064</td>\n",
       "      <td>164536207</td>\n",
       "      <td>chr4</td>\n",
       "      <td>43891</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5187</th>\n",
       "      <td>7529</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_7529_lociStitched</td>\n",
       "      <td>164582088</td>\n",
       "      <td>164589162</td>\n",
       "      <td>chr4</td>\n",
       "      <td>43891</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5938</th>\n",
       "      <td>9659</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_9659_lociStitched</td>\n",
       "      <td>77926134</td>\n",
       "      <td>77931811</td>\n",
       "      <td>chr4</td>\n",
       "      <td>44085</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6394</th>\n",
       "      <td>11386</td>\n",
       "      <td>1_merged_PDX_cell_lines_AML_11386_lociStitched</td>\n",
       "      <td>164476769</td>\n",
       "      <td>164481461</td>\n",
       "      <td>chr4</td>\n",
       "      <td>43891</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "0      SE_N                                       REGION_ID      start  \\\n",
       "40       40     1_merged_PDX_cell_lines_AML_40_lociStitched   33739117   \n",
       "1426   1497   1_merged_PDX_cell_lines_AML_1497_lociStitched   45960034   \n",
       "2087   2264   1_merged_PDX_cell_lines_AML_2264_lociStitched  164508601   \n",
       "2503   2818   1_merged_PDX_cell_lines_AML_2818_lociStitched   75560044   \n",
       "3121   3663   1_merged_PDX_cell_lines_AML_3663_lociStitched   77906409   \n",
       "4068   5156   1_merged_PDX_cell_lines_AML_5156_lociStitched  164527064   \n",
       "5187   7529   1_merged_PDX_cell_lines_AML_7529_lociStitched  164582088   \n",
       "5938   9659   1_merged_PDX_cell_lines_AML_9659_lociStitched   77926134   \n",
       "6394  11386  1_merged_PDX_cell_lines_AML_11386_lociStitched  164476769   \n",
       "\n",
       "0           end  chrom                                          ALL_GENES  \n",
       "40     33795715  chr19  CCNE1 CEBPA GRAMD1A HSPB6 KCTD15 LOC80054 LRP3...  \n",
       "1426   45976885  chr10                                              43898  \n",
       "2087  164522551   chr4                                              43891  \n",
       "2503   75572674  chr17                                              44083  \n",
       "3121   77917457   chr4                                              44085  \n",
       "4068  164536207   chr4                                              43891  \n",
       "5187  164589162   chr4                                              43891  \n",
       "5938   77931811   chr4                                              44085  \n",
       "6394  164481461   chr4                                              43891  "
      ]
     },
     "execution_count": 499,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "superenhancers.iloc[[39,1425,2086,2502,3120,4067,5186,5937,6393],:]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 483,
   "id": "bab9a106",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "31\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "['CEBPA',\n",
       " 'E2F3',\n",
       " 'FLI1',\n",
       " 'FOSL2',\n",
       " 'GFI1',\n",
       " 'HHEX',\n",
       " 'IRF8',\n",
       " 'LYL1',\n",
       " 'MEF2C',\n",
       " 'MEF2D',\n",
       " 'MEIS1',\n",
       " 'MYB',\n",
       " 'PLAGL2',\n",
       " 'RUNX1',\n",
       " 'RUNX2',\n",
       " 'RXRA',\n",
       " 'SP1',\n",
       " 'SPI1',\n",
       " 'SREBF1',\n",
       " 'STAT5B',\n",
       " 'TFAP4',\n",
       " 'ZEB2',\n",
       " 'ZFPM1',\n",
       " 'ZNF281',\n",
       " 'GATA2',\n",
       " 'MAX',\n",
       " 'MYC',\n",
       " 'ETV6',\n",
       " 'HOXA9',\n",
       " 'LMO2',\n",
       " 'ZMYND']"
      ]
     },
     "execution_count": 483,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "print(len(crcs))\n",
    "crcs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 496,
   "id": "10d31db8",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>N_PEAKS</th>\n",
       "      <th>CEBPA</th>\n",
       "      <th>HOXA9</th>\n",
       "      <th>PLAGL2</th>\n",
       "      <th>MEIS1</th>\n",
       "      <th>ETV6</th>\n",
       "      <th>GFI1</th>\n",
       "      <th>RUNX1</th>\n",
       "      <th>FOSL2</th>\n",
       "      <th>FLI1</th>\n",
       "      <th>...</th>\n",
       "      <th>TFAP4</th>\n",
       "      <th>RUNX2</th>\n",
       "      <th>MAX</th>\n",
       "      <th>IRF8</th>\n",
       "      <th>MYB</th>\n",
       "      <th>ZEB2</th>\n",
       "      <th>RXRA</th>\n",
       "      <th>SPI1</th>\n",
       "      <th>MEF2C</th>\n",
       "      <th>MEF2D</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>C1orf159</th>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>MIR200A</th>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>MIR200B</th>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>MIR429</th>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>TTLL10</th>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows × 30 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "          N_PEAKS  CEBPA  HOXA9  PLAGL2  MEIS1  ETV6  GFI1  RUNX1  FOSL2  \\\n",
       "C1orf159      1.0    0.0    0.0     0.0    0.0   0.0   0.0    1.0    0.0   \n",
       "MIR200A       3.0    0.0    0.0     0.0    0.0   0.0   0.0    3.0    0.0   \n",
       "MIR200B       3.0    0.0    0.0     0.0    0.0   0.0   0.0    3.0    0.0   \n",
       "MIR429        3.0    0.0    0.0     0.0    0.0   0.0   0.0    3.0    0.0   \n",
       "TTLL10        3.0    0.0    0.0     0.0    0.0   0.0   0.0    3.0    0.0   \n",
       "\n",
       "          FLI1  ...  TFAP4  RUNX2  MAX  IRF8  MYB  ZEB2  RXRA  SPI1  MEF2C  \\\n",
       "C1orf159   0.0  ...    0.0    0.0  1.0   0.0  0.0   0.0   0.0   0.0    0.0   \n",
       "MIR200A    1.0  ...    0.0    0.0  3.0   0.0  1.0   1.0   1.0   0.0    0.0   \n",
       "MIR200B    1.0  ...    0.0    0.0  3.0   0.0  1.0   1.0   1.0   0.0    0.0   \n",
       "MIR429     1.0  ...    0.0    0.0  3.0   0.0  1.0   1.0   1.0   0.0    0.0   \n",
       "TTLL10     1.0  ...    0.0    0.0  3.0   0.0  1.0   1.0   1.0   0.0    0.0   \n",
       "\n",
       "          MEF2D  \n",
       "C1orf159    0.0  \n",
       "MIR200A     0.0  \n",
       "MIR200B     0.0  \n",
       "MIR429      0.0  \n",
       "TTLL10      0.0  \n",
       "\n",
       "[5 rows x 30 columns]"
      ]
     },
     "execution_count": 496,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 498,
   "id": "521d846b",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "N_PEAKS    3.0\n",
       "CEBPA      0.0\n",
       "HOXA9      0.0\n",
       "PLAGL2     0.0\n",
       "MEIS1      0.0\n",
       "ETV6       0.0\n",
       "GFI1       0.0\n",
       "RUNX1      3.0\n",
       "FOSL2      0.0\n",
       "FLI1       1.0\n",
       "GATA2      0.0\n",
       "SP1        1.0\n",
       "E2F3       0.0\n",
       "ZNF281     1.0\n",
       "SREBF1     0.0\n",
       "LMO2       1.0\n",
       "STAT5B     0.0\n",
       "ZMYND8     1.0\n",
       "LYL1       0.0\n",
       "MYC        2.0\n",
       "TFAP4      0.0\n",
       "RUNX2      0.0\n",
       "MAX        3.0\n",
       "IRF8       0.0\n",
       "MYB        1.0\n",
       "ZEB2       1.0\n",
       "RXRA       1.0\n",
       "SPI1       0.0\n",
       "MEF2C      0.0\n",
       "MEF2D      0.0\n",
       "Name: MIR200A, dtype: float64"
      ]
     },
     "execution_count": 498,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.loc[\"MIR200A\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 476,
   "id": "448135f3",
   "metadata": {},
   "outputs": [],
   "source": [
    "crc_genes = pd.Series([1 if gene not in crcs else 0 for gene in df.index.tolist()], index=df.index.tolist())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 484,
   "id": "4da23537",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['43898',\n",
       " 'ABCA4',\n",
       " 'ABCB10',\n",
       " 'ABCD3',\n",
       " 'ABI1',\n",
       " 'ABL2',\n",
       " 'ACAP3',\n",
       " 'ACBD3',\n",
       " 'ACBD5',\n",
       " 'ACBD7',\n",
       " 'ACOT11',\n",
       " 'ACOT7',\n",
       " 'ACTA2',\n",
       " 'ACTR1A',\n",
       " 'ACTRT2',\n",
       " 'ADAM15',\n",
       " 'ADAMTS14',\n",
       " 'ADAMTS4',\n",
       " 'ADAMTSL4',\n",
       " 'ADAR',\n",
       " 'ADARB2',\n",
       " 'ADC',\n",
       " 'ADCK3',\n",
       " 'ADCY10',\n",
       " 'ADD3',\n",
       " 'ADIPOR1',\n",
       " 'ADK',\n",
       " 'ADORA1',\n",
       " 'ADORA3',\n",
       " 'ADPRHL2',\n",
       " 'ADRA2A',\n",
       " 'AFAP1L2',\n",
       " 'AGAP4',\n",
       " 'AGAP7',\n",
       " 'AGMAT',\n",
       " 'AGO1',\n",
       " 'AGO3',\n",
       " 'AGTRAP',\n",
       " 'AHCYL1',\n",
       " 'AHDC1',\n",
       " 'AK2',\n",
       " 'AK4',\n",
       " 'AK5',\n",
       " 'AKNAD1',\n",
       " 'AKR1A1',\n",
       " 'AKR1C1',\n",
       " 'AKR1E2',\n",
       " 'AKR7A2',\n",
       " 'AKR7A3',\n",
       " 'AKR7L',\n",
       " 'AKT3',\n",
       " 'ALDH18A1',\n",
       " 'ALDH4A1',\n",
       " 'ALDH9A1',\n",
       " 'ALG14',\n",
       " 'ALOX5',\n",
       " 'AMIGO1',\n",
       " 'AMPD2',\n",
       " 'ANAPC16',\n",
       " 'ANGPTL1',\n",
       " 'ANKRD1',\n",
       " 'ANKRD16',\n",
       " 'ANKRD22',\n",
       " 'ANKRD26',\n",
       " 'ANKRD34A',\n",
       " 'ANKRD36BP1',\n",
       " 'ANP32E',\n",
       " 'ANXA2P3',\n",
       " 'ANXA7',\n",
       " 'ANXA9',\n",
       " 'AP4B1',\n",
       " 'APBB1IP',\n",
       " 'APH1A',\n",
       " 'APITD1',\n",
       " 'APITD1-CORT',\n",
       " 'APOA2',\n",
       " 'APOBEC4',\n",
       " 'AQP10',\n",
       " 'ARF1',\n",
       " 'ARHGAP19',\n",
       " 'ARHGAP22',\n",
       " 'ARHGAP30',\n",
       " 'ARHGEF10L',\n",
       " 'ARHGEF11',\n",
       " 'ARHGEF16',\n",
       " 'ARHGEF2',\n",
       " 'ARID1A',\n",
       " 'ARID4B',\n",
       " 'ARID5B',\n",
       " 'ARL3',\n",
       " 'ARL5B',\n",
       " 'ARL8A',\n",
       " 'ARMC4',\n",
       " 'ARNT',\n",
       " 'ARPC5',\n",
       " 'ARTN',\n",
       " 'ASAH2',\n",
       " 'ASAP3',\n",
       " 'ASB13',\n",
       " 'ASCC1',\n",
       " 'ASH1L',\n",
       " 'ATAD1',\n",
       " 'ATAD3A',\n",
       " 'ATAD3B',\n",
       " 'ATAD3C',\n",
       " 'ATF3',\n",
       " 'ATF6',\n",
       " 'ATG4C',\n",
       " 'ATP13A2',\n",
       " 'ATP1A1',\n",
       " 'ATP1A1OS',\n",
       " 'ATP1B1',\n",
       " 'ATP2B4',\n",
       " 'ATP5F1',\n",
       " 'ATP6V0B',\n",
       " 'ATP8B2',\n",
       " 'ATPIF1',\n",
       " 'ATXN7L2',\n",
       " 'AUNIP',\n",
       " 'AURKAIP1',\n",
       " 'AVPI1',\n",
       " 'AVPR1B',\n",
       " 'B3GALNT2',\n",
       " 'B3GALT6',\n",
       " 'B4GALT2',\n",
       " 'B4GALT3',\n",
       " 'BAMBI',\n",
       " 'BBIP1',\n",
       " 'BCAR3',\n",
       " 'BCAS2',\n",
       " 'BCL2L15',\n",
       " 'BEST4',\n",
       " 'BGLAP',\n",
       " 'BLNK',\n",
       " 'BLOC1S2',\n",
       " 'BMI1',\n",
       " 'BMP8A',\n",
       " 'BMP8B',\n",
       " 'BMS1',\n",
       " 'BNIPL',\n",
       " 'BOLA1',\n",
       " 'BPNT1',\n",
       " 'BRP44',\n",
       " 'BSDC1',\n",
       " 'BTAF1',\n",
       " 'BTBD19',\n",
       " 'BTBD8',\n",
       " 'BTF3L4',\n",
       " 'BTG2',\n",
       " 'BTRC',\n",
       " 'C10orf10',\n",
       " 'C10orf105',\n",
       " 'C10orf107',\n",
       " 'C10orf111',\n",
       " 'C10orf114',\n",
       " 'C10orf131',\n",
       " 'C10orf140',\n",
       " 'C10orf18',\n",
       " 'C10orf2',\n",
       " 'C10orf25',\n",
       " 'C10orf26',\n",
       " 'C10orf32',\n",
       " 'C10orf47',\n",
       " 'C10orf53',\n",
       " 'C10orf54',\n",
       " 'C10orf55',\n",
       " 'C10orf58',\n",
       " 'C10orf76',\n",
       " 'C10orf88',\n",
       " 'C1orf100',\n",
       " 'C1orf105',\n",
       " 'C1orf109',\n",
       " 'C1orf116',\n",
       " 'C1orf122',\n",
       " 'C1orf127',\n",
       " 'C1orf135',\n",
       " 'C1orf141',\n",
       " 'C1orf144',\n",
       " 'C1orf159',\n",
       " 'C1orf162',\n",
       " 'C1orf167',\n",
       " 'C1orf174',\n",
       " 'C1orf180',\n",
       " 'C1orf183',\n",
       " 'C1orf186',\n",
       " 'C1orf187',\n",
       " 'C1orf189',\n",
       " 'C1orf192',\n",
       " 'C1orf194',\n",
       " 'C1orf200',\n",
       " 'C1orf21',\n",
       " 'C1orf212',\n",
       " 'C1orf213',\n",
       " 'C1orf216',\n",
       " 'C1orf220',\n",
       " 'C1orf228',\n",
       " 'C1orf25',\n",
       " 'C1orf26',\n",
       " 'C1orf35',\n",
       " 'C1orf38',\n",
       " 'C1orf43',\n",
       " 'C1orf49',\n",
       " 'C1orf50',\n",
       " 'C1orf51',\n",
       " 'C1orf52',\n",
       " 'C1orf53',\n",
       " 'C1orf54',\n",
       " 'C1orf55',\n",
       " 'C1orf56',\n",
       " 'C1orf59',\n",
       " 'C1orf64',\n",
       " 'C1orf66',\n",
       " 'C1orf69',\n",
       " 'C1orf74',\n",
       " 'C1orf83',\n",
       " 'C1orf85',\n",
       " 'C1orf86',\n",
       " 'C1orf87',\n",
       " 'C1orf9',\n",
       " 'C1orf91',\n",
       " 'C1orf93',\n",
       " 'C1orf97',\n",
       " 'C4BPA',\n",
       " 'C4BPB',\n",
       " 'CA14',\n",
       " 'CA6',\n",
       " 'CACHD1',\n",
       " 'CACNA1S',\n",
       " 'CACNB2',\n",
       " 'CACYBP',\n",
       " 'CALML3',\n",
       " 'CALML5',\n",
       " 'CALML6',\n",
       " 'CAMK1D',\n",
       " 'CAMK2G',\n",
       " 'CAMTA1',\n",
       " 'CAP1',\n",
       " 'CAPN2',\n",
       " 'CAPZA1',\n",
       " 'CAPZB',\n",
       " 'CASC2',\n",
       " 'CASP9',\n",
       " 'CASZ1',\n",
       " 'CBARA1',\n",
       " 'CC2D1B',\n",
       " 'CCDC109A',\n",
       " 'CCDC147',\n",
       " 'CCDC17',\n",
       " 'CCDC18',\n",
       " 'CCDC19',\n",
       " 'CCDC21',\n",
       " 'CCDC27',\n",
       " 'CCDC28B',\n",
       " 'CCDC30',\n",
       " 'CCDC6',\n",
       " 'CCNJ',\n",
       " 'CCNL2',\n",
       " 'CCNY',\n",
       " 'CD101',\n",
       " 'CD164L2',\n",
       " 'CD1A',\n",
       " 'CD1B',\n",
       " 'CD1C',\n",
       " 'CD1E',\n",
       " 'CD2',\n",
       " 'CD244',\n",
       " 'CD247',\n",
       " 'CD34',\n",
       " 'CD46',\n",
       " 'CD48',\n",
       " 'CD52',\n",
       " 'CD53',\n",
       " 'CD55',\n",
       " 'CD58',\n",
       " 'CD84',\n",
       " 'CDA',\n",
       " 'CDC123',\n",
       " 'CDC14A',\n",
       " 'CDC20',\n",
       " 'CDC42',\n",
       " 'CDC42BPA',\n",
       " 'CDC42SE1',\n",
       " 'CDC73',\n",
       " 'CDCA8',\n",
       " 'CDH23',\n",
       " 'CDK11B',\n",
       " 'CDKN2C',\n",
       " 'CDNF',\n",
       " 'CELA2A',\n",
       " 'CELA2B',\n",
       " 'CELA3A',\n",
       " 'CELF2',\n",
       " 'CELSR2',\n",
       " 'CENPL',\n",
       " 'CEP104',\n",
       " 'CEP170',\n",
       " 'CEP350',\n",
       " 'CEP85',\n",
       " 'CEPT1',\n",
       " 'CERS2',\n",
       " 'CFLP1',\n",
       " 'CGN',\n",
       " 'CH25H',\n",
       " 'CHAT',\n",
       " 'CHD5',\n",
       " 'CHI3L1',\n",
       " 'CHI3L2',\n",
       " 'CHIT1',\n",
       " 'CHML',\n",
       " 'CHRM3',\n",
       " 'CHRNB2',\n",
       " 'CHTOP',\n",
       " 'CHUK',\n",
       " 'CIART',\n",
       " 'CISD1',\n",
       " 'CITED4',\n",
       " 'CKS1B',\n",
       " 'CLCA1',\n",
       " 'CLCA4',\n",
       " 'CLCC1',\n",
       " 'CLCN6',\n",
       " 'CLIC4',\n",
       " 'CLK2',\n",
       " 'CLSPN',\n",
       " 'CLSTN1',\n",
       " 'CMPK1',\n",
       " 'CNIH3',\n",
       " 'CNIH4',\n",
       " 'CNKSR1',\n",
       " 'CNNM1',\n",
       " 'CNR2',\n",
       " 'CNST',\n",
       " 'COA6',\n",
       " 'COL24A1',\n",
       " 'COL8A2',\n",
       " 'COL9A2',\n",
       " 'COMMD3',\n",
       " 'COMTD1',\n",
       " 'CORT',\n",
       " 'COX15',\n",
       " 'COX20',\n",
       " 'CR1',\n",
       " 'CR1L',\n",
       " 'CR2',\n",
       " 'CRABP2',\n",
       " 'CREB3L4',\n",
       " 'CREG1',\n",
       " 'CREM',\n",
       " 'CROCC',\n",
       " 'CROCCP2',\n",
       " 'CRP',\n",
       " 'CRTC2',\n",
       " 'CSDE1',\n",
       " 'CSF1',\n",
       " 'CSGALNACT2',\n",
       " 'CTH',\n",
       " 'CTNNBIP1',\n",
       " 'CTPS',\n",
       " 'CTPS1',\n",
       " 'CTRC',\n",
       " 'CTSE',\n",
       " 'CTSK',\n",
       " 'CTSS',\n",
       " 'CTTNBP2NL',\n",
       " 'CUBN',\n",
       " 'CUEDC2',\n",
       " 'CUL2',\n",
       " 'CUTC',\n",
       " 'CWF19L1',\n",
       " 'CYB5R1',\n",
       " 'CYCSP52',\n",
       " 'CYP17A1',\n",
       " 'CYP4A22',\n",
       " 'DAP3',\n",
       " 'DARS2',\n",
       " 'DBT',\n",
       " 'DCAF6',\n",
       " 'DCAF8',\n",
       " 'DCDC2B',\n",
       " 'DCLRE1A',\n",
       " 'DCLRE1B',\n",
       " 'DCLRE1C',\n",
       " 'DCST1',\n",
       " 'DCST2',\n",
       " 'DDI2',\n",
       " 'DDIT4',\n",
       " 'DDOST',\n",
       " 'DDX20',\n",
       " 'DDX21',\n",
       " 'DDX50',\n",
       " 'DEDD',\n",
       " 'DEGS1',\n",
       " 'DENND1B',\n",
       " 'DENND2D',\n",
       " 'DENND4B',\n",
       " 'DFFA',\n",
       " 'DFFB',\n",
       " 'DHDDS',\n",
       " 'DHRS3',\n",
       " 'DHTKD1',\n",
       " 'DLG5',\n",
       " 'DLGAP3',\n",
       " 'DMRTB1',\n",
       " 'DNAJB12',\n",
       " 'DNAJB4',\n",
       " 'DNAJC1',\n",
       " 'DNAJC11',\n",
       " 'DNAJC16',\n",
       " 'DNAJC6',\n",
       " 'DNAJC8',\n",
       " 'DNAJC9',\n",
       " 'DNASE2B',\n",
       " 'DNM3',\n",
       " 'DNTT',\n",
       " 'DNTTIP2',\n",
       " 'DOCK7',\n",
       " 'DPH5',\n",
       " 'DPM3',\n",
       " 'DPYD',\n",
       " 'DR1',\n",
       " 'DRAM2',\n",
       " 'DSTYK',\n",
       " 'DTL',\n",
       " 'DUSP10',\n",
       " 'DUSP12',\n",
       " 'DUSP23',\n",
       " 'DUSP27',\n",
       " 'DVL1',\n",
       " 'DYRK3',\n",
       " 'E2F2',\n",
       " 'ECE1',\n",
       " 'ECHDC2',\n",
       " 'ECHDC3',\n",
       " 'ECM1',\n",
       " 'EDEM3',\n",
       " 'EFCAB2',\n",
       " 'EFHD2',\n",
       " 'EFNA1',\n",
       " 'EFNA3',\n",
       " 'EFNA4',\n",
       " 'EGLN1',\n",
       " 'EIF2B3',\n",
       " 'EIF2C4',\n",
       " 'EIF3A',\n",
       " 'EIF3I',\n",
       " 'EIF4G3',\n",
       " 'EIF5AL1',\n",
       " 'ELK4',\n",
       " 'ELOVL1',\n",
       " 'EMC1',\n",
       " 'EMX2',\n",
       " 'EMX2OS',\n",
       " 'ENAH',\n",
       " 'ENO1',\n",
       " 'ENSA',\n",
       " 'ENTPD1',\n",
       " 'ENTPD7',\n",
       " 'EPB41',\n",
       " 'EPC1',\n",
       " 'EPHB2',\n",
       " 'EPHX1',\n",
       " 'EPS15',\n",
       " 'EPS8L3',\n",
       " 'ERI3',\n",
       " 'ERLIN1',\n",
       " 'ERMAP',\n",
       " 'ERO1LB',\n",
       " 'ESPN',\n",
       " 'ETNK2',\n",
       " 'ETV3',\n",
       " 'ETV3L',\n",
       " 'EVA1B',\n",
       " 'EVI5',\n",
       " 'EXO1',\n",
       " 'EXO5',\n",
       " 'EXOC6',\n",
       " 'EXOC8',\n",
       " 'EXOSC1',\n",
       " 'EXTL1',\n",
       " 'EXTL2',\n",
       " 'EYA3',\n",
       " 'F11R',\n",
       " 'F3',\n",
       " 'FAAH',\n",
       " 'FAF1',\n",
       " 'FAIM3',\n",
       " 'FAM102B',\n",
       " 'FAM107B',\n",
       " 'FAM129A',\n",
       " 'FAM132A',\n",
       " 'FAM159A',\n",
       " 'FAM160B1',\n",
       " 'FAM167B',\n",
       " 'FAM171A1',\n",
       " 'FAM176B',\n",
       " 'FAM177B',\n",
       " 'FAM189B',\n",
       " 'FAM20B',\n",
       " 'FAM229A',\n",
       " 'FAM35A',\n",
       " 'FAM36A',\n",
       " 'FAM43B',\n",
       " 'FAM45A',\n",
       " 'FAM45B',\n",
       " 'FAM46C',\n",
       " 'FAM54B',\n",
       " 'FAM58B',\n",
       " 'FAM63A',\n",
       " 'FAM69A',\n",
       " 'FAM71A',\n",
       " 'FAM72A',\n",
       " 'FAM76A',\n",
       " 'FAS',\n",
       " 'FAS-AS1',\n",
       " 'FASLG',\n",
       " 'FBXL15',\n",
       " 'FBXO18',\n",
       " 'FBXO28',\n",
       " 'FBXO42',\n",
       " 'FBXO44',\n",
       " 'FBXW4',\n",
       " 'FCAMR',\n",
       " 'FCER1A',\n",
       " 'FCER1G',\n",
       " 'FCGR1A',\n",
       " 'FCGR2A',\n",
       " 'FCGR2C',\n",
       " 'FCN3',\n",
       " 'FCRL3',\n",
       " 'FCRL4',\n",
       " 'FCRL5',\n",
       " 'FCRLA',\n",
       " 'FCRLB',\n",
       " 'FDPS',\n",
       " 'FGF8',\n",
       " 'FGGY',\n",
       " 'FGR',\n",
       " 'FH',\n",
       " 'FHAD1',\n",
       " 'FHL3',\n",
       " 'FLAD1',\n",
       " 'FLJ27354',\n",
       " 'FLJ32224',\n",
       " 'FLJ37453',\n",
       " 'FLJ40434',\n",
       " 'FLJ42875',\n",
       " 'FLJ45983',\n",
       " 'FLVCR1',\n",
       " 'FMOD',\n",
       " 'FNBP1L',\n",
       " 'FNDC5',\n",
       " 'FNDC7',\n",
       " 'FOXJ3',\n",
       " 'FRAT1',\n",
       " 'FRAT2',\n",
       " 'FUBP1',\n",
       " 'FUCA1',\n",
       " 'FXYD4',\n",
       " 'FZD8',\n",
       " 'G0S2',\n",
       " 'GABPB2',\n",
       " 'GABRD',\n",
       " 'GADD45A',\n",
       " 'GALE',\n",
       " 'GALNT2',\n",
       " 'GAS5',\n",
       " 'GATA3',\n",
       " 'GATAD2B',\n",
       " 'GBA',\n",
       " 'GBAP1',\n",
       " 'GBP1',\n",
       " 'GBP2',\n",
       " 'GBP4',\n",
       " 'GBP5',\n",
       " 'GBP7',\n",
       " 'GCLM',\n",
       " 'GCSAML',\n",
       " 'GDI2',\n",
       " 'GFI1',\n",
       " 'GGPS1',\n",
       " 'GIPC2',\n",
       " 'GJA4',\n",
       " 'GJB3',\n",
       " 'GJC2',\n",
       " 'GLMN',\n",
       " 'GLUD1',\n",
       " 'GLUL',\n",
       " 'GMEB1',\n",
       " 'GNAI3',\n",
       " 'GNAT2',\n",
       " 'GNB1',\n",
       " 'GNG12',\n",
       " 'GNG4',\n",
       " 'GNG5',\n",
       " 'GNL2',\n",
       " 'GOLPH3L',\n",
       " 'GOT1',\n",
       " 'GPAM',\n",
       " 'GPATCH3',\n",
       " 'GPBP1L1',\n",
       " 'GPN2',\n",
       " 'GPR137B',\n",
       " 'GPR153',\n",
       " 'GPR157',\n",
       " 'GPR161',\n",
       " 'GPR3',\n",
       " 'GPR37L1',\n",
       " 'GPSM2',\n",
       " 'GPX7',\n",
       " 'GRK5',\n",
       " 'GRRP1',\n",
       " 'GSTM2',\n",
       " 'GSTM3',\n",
       " 'GSTM4',\n",
       " 'GSTO2',\n",
       " 'GTPBP4',\n",
       " 'GUK1',\n",
       " 'H3F3A',\n",
       " 'H6PD',\n",
       " 'HAX1',\n",
       " 'HDAC1',\n",
       " 'HDGF',\n",
       " 'HECTD3',\n",
       " 'HENMT1',\n",
       " 'HERC4',\n",
       " 'HES5',\n",
       " 'HEYL',\n",
       " 'HFE2',\n",
       " 'HHEX',\n",
       " 'HIAT1',\n",
       " 'HIF1AN',\n",
       " 'HIPK1',\n",
       " 'HIST2H2AA3',\n",
       " 'HIST2H2AB',\n",
       " 'HIST2H2AC',\n",
       " 'HIST2H2BC',\n",
       " 'HIST2H2BE',\n",
       " 'HIST2H2BF',\n",
       " 'HIST2H3C',\n",
       " 'HIST2H3D',\n",
       " 'HIST2H4B',\n",
       " 'HIVEP3',\n",
       " 'HLX',\n",
       " 'HMCN1',\n",
       " 'HMGN2',\n",
       " 'HNRNPF',\n",
       " 'HNRNPH3',\n",
       " 'HNRNPR',\n",
       " 'HNRNPU',\n",
       " 'HOOK1',\n",
       " 'HP1BP3',\n",
       " 'HPDL',\n",
       " 'HPS1',\n",
       " 'HPS6',\n",
       " 'HPSE2',\n",
       " 'HS2ST1',\n",
       " 'HSD11B1',\n",
       " 'HSPA14',\n",
       " 'HSPA6',\n",
       " 'HSPB7',\n",
       " 'HSPC157',\n",
       " 'HSPG2',\n",
       " 'HYI',\n",
       " 'IARS2',\n",
       " 'IBA57',\n",
       " 'ID3',\n",
       " 'IDE',\n",
       " 'IFFO2',\n",
       " 'IFI16',\n",
       " 'IFI44',\n",
       " 'IFI44L',\n",
       " 'IFI6',\n",
       " 'IFIT1B',\n",
       " 'IFIT2',\n",
       " 'IFIT3',\n",
       " 'IFNLR1',\n",
       " 'IGSF3',\n",
       " 'IGSF9',\n",
       " 'IKBKE',\n",
       " 'IL10',\n",
       " 'IL12RB2',\n",
       " 'IL15RA',\n",
       " 'IL19',\n",
       " 'IL22RA1',\n",
       " 'IL23R',\n",
       " 'IL24',\n",
       " 'IL28RA',\n",
       " 'IL2RA',\n",
       " 'IL6R',\n",
       " 'ILDR2',\n",
       " 'ILF2',\n",
       " 'INPP5B',\n",
       " 'INSL5',\n",
       " 'INSRR',\n",
       " 'INTS3',\n",
       " 'INTS7',\n",
       " 'IPMK',\n",
       " 'IPO9',\n",
       " 'IPP',\n",
       " 'IQCC',\n",
       " 'IQGAP3',\n",
       " 'IRF2BP2',\n",
       " 'IRF6',\n",
       " 'ISG20L2',\n",
       " 'ITGB1',\n",
       " 'ITLN1',\n",
       " 'ITPKB',\n",
       " 'ITPRIP',\n",
       " 'IVNS1ABP',\n",
       " 'JMJD1C',\n",
       " 'JMJD4',\n",
       " 'JTB',\n",
       " 'JUN',\n",
       " 'KAZ',\n",
       " 'KAZALD1',\n",
       " 'KAZN',\n",
       " 'KCNA3',\n",
       " 'KCNAB2',\n",
       " 'KCNIP2',\n",
       " 'KCNK1',\n",
       " 'KCNQ4',\n",
       " 'KCNT2',\n",
       " 'KDM1A',\n",
       " 'KDM4A',\n",
       " 'KDM5B',\n",
       " 'KHDRBS1',\n",
       " 'KIAA0040',\n",
       " 'KIAA0090',\n",
       " 'KIAA0319L',\n",
       " 'KIAA0495',\n",
       " 'KIAA0562',\n",
       " 'KIAA0907',\n",
       " 'KIAA1217',\n",
       " 'KIAA1274',\n",
       " 'KIAA1279',\n",
       " 'KIAA1324',\n",
       " 'KIAA1383',\n",
       " 'KIAA1462',\n",
       " 'KIAA1522',\n",
       " 'KIAA1614',\n",
       " 'KIAA2013',\n",
       " 'KIF11',\n",
       " 'KIF14',\n",
       " 'KIF17',\n",
       " 'KIF1B',\n",
       " 'KIF20B',\n",
       " 'KIF21B',\n",
       " 'KIF26B',\n",
       " 'KIF2C',\n",
       " 'KIF5B',\n",
       " 'KIFAP3',\n",
       " 'KILLIN',\n",
       " 'KIN',\n",
       " 'KLF6',\n",
       " 'KLHDC9',\n",
       " 'KLHL12',\n",
       " 'KLHL20',\n",
       " 'KLHL21',\n",
       " 'KLLN',\n",
       " 'KMO',\n",
       " 'KPNA6',\n",
       " 'KRTCAP2',\n",
       " 'KTI12',\n",
       " 'LAMB3',\n",
       " 'LAMTOR2',\n",
       " 'LAPTM5',\n",
       " 'LARP4B',\n",
       " 'LASS2',\n",
       " 'LAX1',\n",
       " 'LBR',\n",
       " 'LCK',\n",
       " 'LCOR',\n",
       " 'LDB1',\n",
       " 'LDLRAD2',\n",
       " 'LDLRAP1',\n",
       " 'LEFTY1',\n",
       " 'LENEP',\n",
       " 'LEPR',\n",
       " 'LEPROT',\n",
       " 'LGR6',\n",
       " 'LGTN',\n",
       " 'LHX4',\n",
       " 'LIN9',\n",
       " 'LIX1L',\n",
       " 'LMNA',\n",
       " 'LMO4',\n",
       " 'LMX1A',\n",
       " 'LOC100128003',\n",
       " 'LOC100128071',\n",
       " 'LOC100128098',\n",
       " 'LOC100129196',\n",
       " 'LOC100129269',\n",
       " 'LOC100129534',\n",
       " 'LOC100130557',\n",
       " 'LOC100131060',\n",
       " 'LOC100131564',\n",
       " 'LOC100133612',\n",
       " 'LOC100302401',\n",
       " 'LOC100499489',\n",
       " 'LOC100506046',\n",
       " 'LOC115110',\n",
       " 'LOC127841',\n",
       " 'LOC148413',\n",
       " 'LOC148696',\n",
       " 'LOC148709',\n",
       " 'LOC149086',\n",
       " 'LOC149134',\n",
       " 'LOC149620',\n",
       " 'LOC220930',\n",
       " 'LOC220980',\n",
       " 'LOC282997',\n",
       " 'LOC283050',\n",
       " 'LOC283070',\n",
       " 'LOC284551',\n",
       " 'LOC284632',\n",
       " 'LOC339524',\n",
       " 'LOC339529',\n",
       " 'LOC387646',\n",
       " 'LOC388588',\n",
       " 'LOC388692',\n",
       " 'LOC401980',\n",
       " 'LOC439949',\n",
       " 'LOC440600',\n",
       " 'LOC440926',\n",
       " 'LOC441869',\n",
       " 'LOC642864',\n",
       " 'LOC645676',\n",
       " 'LOC646471',\n",
       " 'LOC646627',\n",
       " 'LOC646870',\n",
       " 'LOC653566',\n",
       " 'LOC728989',\n",
       " 'LOC730102',\n",
       " 'LOC730227',\n",
       " 'LOC84989',\n",
       " 'LPAR3',\n",
       " 'LPHN2',\n",
       " 'LRIF1',\n",
       " 'LRIG2',\n",
       " 'LRP8',\n",
       " 'LRRC20',\n",
       " 'LRRC38',\n",
       " 'LRRC41',\n",
       " 'LRRC47',\n",
       " 'LRRC7',\n",
       " 'LRRC8B',\n",
       " 'LRRC8C',\n",
       " 'LRRC8D',\n",
       " 'LRRN2',\n",
       " 'LSM10',\n",
       " 'LUZP1',\n",
       " 'LY9',\n",
       " 'LYPLAL1',\n",
       " 'LYST',\n",
       " 'LYZL2',\n",
       " 'LZIC',\n",
       " 'LZTS2',\n",
       " 'MACF1',\n",
       " 'MAD2L2',\n",
       " 'MAEL',\n",
       " 'MAGI3',\n",
       " 'MAGOH',\n",
       " 'MAN1C1',\n",
       " 'MANEAL',\n",
       " 'MAP3K6',\n",
       " 'MAP3K8',\n",
       " 'MAP7D1',\n",
       " 'MAPK8',\n",
       " 'MAPKAPK2',\n",
       " 'MARC2',\n",
       " 'MARCKSL1',\n",
       " 'MASTL',\n",
       " 'MATN1',\n",
       " 'MCL1',\n",
       " 'MCM10',\n",
       " 'MCOLN2',\n",
       " 'MCOLN3',\n",
       " 'MCU',\n",
       " 'MDM4',\n",
       " 'MDS2',\n",
       " 'MEAF6',\n",
       " 'MECR',\n",
       " 'MED8',\n",
       " 'MEF2D',\n",
       " 'MEGF6',\n",
       " 'MEX3A',\n",
       " 'MFAP2',\n",
       " 'MFN2',\n",
       " 'MFSD2A',\n",
       " 'MGEA5',\n",
       " 'MIA3',\n",
       " 'MIB2',\n",
       " 'MICU1',\n",
       " 'MIER1',\n",
       " 'MINPP1',\n",
       " 'MIR101-1',\n",
       " 'MIR1256',\n",
       " 'MIR1265',\n",
       " 'MIR1296',\n",
       " 'MIR1537',\n",
       " 'MIR181A1',\n",
       " 'MIR181B1',\n",
       " 'MIR190B',\n",
       " 'MIR1915',\n",
       " 'MIR194-1',\n",
       " 'MIR197',\n",
       " 'MIR1976',\n",
       " 'MIR200A',\n",
       " 'MIR200B',\n",
       " 'MIR215',\n",
       " 'MIR29B2',\n",
       " 'MIR29C',\n",
       " 'MIR30C1',\n",
       " 'MIR30E',\n",
       " 'MIR3122',\n",
       " 'MIR3124',\n",
       " 'MIR3620',\n",
       " 'MIR3658',\n",
       " 'MIR3671',\n",
       " 'MIR3917',\n",
       " 'MIR4251',\n",
       " 'MIR4257',\n",
       " 'MIR4258',\n",
       " 'MIR4260',\n",
       " 'MIR429',\n",
       " 'MIR551A',\n",
       " 'MIR608',\n",
       " 'MIR760',\n",
       " 'MIR761',\n",
       " 'MIR92B',\n",
       " 'MIXL1',\n",
       " 'MKX',\n",
       " 'MLLT10',\n",
       " 'MLLT11',\n",
       " 'MMACHC',\n",
       " 'MMEL1',\n",
       " 'MMP23A',\n",
       " 'MMP23B',\n",
       " 'MMS19',\n",
       " 'MNDA',\n",
       " 'MORN1',\n",
       " 'MOV10',\n",
       " 'MPC2',\n",
       " 'MPL',\n",
       " 'MPP7',\n",
       " 'MPZ',\n",
       " 'MPZL1',\n",
       " 'MRPL20',\n",
       " 'MRPL24',\n",
       " 'MRPL37',\n",
       " 'MRPL43',\n",
       " 'MRPL55',\n",
       " 'MRPL9',\n",
       " 'MRPS15',\n",
       " 'MRPS16',\n",
       " 'MRPS21',\n",
       " 'MRTO4',\n",
       " 'MSMB',\n",
       " 'MST1P2',\n",
       " 'MSTO1',\n",
       " 'MTF1',\n",
       " 'MTF2',\n",
       " 'MTFR1L',\n",
       " 'MTHFR',\n",
       " 'MTMR11',\n",
       " 'MTMR9LP',\n",
       " 'MTOR',\n",
       " 'MTPAP',\n",
       " 'MTX1',\n",
       " 'MUC1',\n",
       " 'MUL1',\n",
       " 'MXI1',\n",
       " 'MYBPH',\n",
       " 'MYCBP',\n",
       " 'MYCL',\n",
       " 'MYCL1',\n",
       " 'MYOF',\n",
       " 'MYSM1',\n",
       " 'MYST4',\n",
       " 'NADK',\n",
       " 'NANOS1',\n",
       " 'NASP',\n",
       " 'NAV1',\n",
       " 'NBL1',\n",
       " 'NBPF1',\n",
       " 'NBPF10',\n",
       " 'NBPF11',\n",
       " 'NBPF14',\n",
       " 'NCDN',\n",
       " 'NCF2',\n",
       " 'NCOA4',\n",
       " 'NCRNA00081',\n",
       " 'NCRNA00168',\n",
       " 'NCRNA00184',\n",
       " 'NCRNA00201',\n",
       " 'NCRNA00245',\n",
       " 'NCRNA00263',\n",
       " 'NCRNA00272',\n",
       " 'NDST2',\n",
       " 'NDUFB8',\n",
       " ...]"
      ]
     },
     "execution_count": 484,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#[1 if x not in crcs else 0 for x in df.index.tolist()]\n",
    "[x for x in sorted(df.index.tolist())]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 456,
   "id": "3105aa45",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>MAX</th>\n",
       "      <th>MEF2C</th>\n",
       "      <th>MEF2D</th>\n",
       "      <th>MEIS1</th>\n",
       "      <th>MYB</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>GENE1</th>\n",
       "      <td>1</td>\n",
       "      <td>2</td>\n",
       "      <td>2</td>\n",
       "      <td>2</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GENE2</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GENE3</th>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>2</td>\n",
       "      <td>1</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GENE4</th>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>2</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GENE5</th>\n",
       "      <td>2</td>\n",
       "      <td>2</td>\n",
       "      <td>1</td>\n",
       "      <td>2</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GENE6</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "       MAX  MEF2C  MEF2D  MEIS1  MYB\n",
       "GENE1    1      2      2      2    3\n",
       "GENE2    0      0      1      1    1\n",
       "GENE3    1      1      2      1    2\n",
       "GENE4    1      1      0      2    1\n",
       "GENE5    2      2      1      2    2\n",
       "GENE6    1      0      0      1    0"
      ]
     },
     "execution_count": 456,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "mydict = {}\n",
    "for idx, row in test.iterrows():\n",
    "    region = row[\"SE\"]\n",
    "    genes = row[\"GENE\"]\n",
    "    #counts = row[2:].tolist()\n",
    "    for gene in genes:\n",
    "        if gene in mydict:\n",
    "            #print(row[2:])\n",
    "            mydict[gene] = np.add(mydict[gene], row[2:].tolist())\n",
    "        else:\n",
    "            mydict[gene] = row[2:].tolist()\n",
    "\n",
    "pd.DataFrame.from_dict(mydict, orient=\"index\", columns=test.columns.tolist()[2:])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 452,
   "id": "fe148ab7",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>SE</th>\n",
       "      <th>GENE</th>\n",
       "      <th>MAX</th>\n",
       "      <th>MEF2C</th>\n",
       "      <th>MEF2D</th>\n",
       "      <th>MEIS1</th>\n",
       "      <th>MYB</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>SE1</td>\n",
       "      <td>[GENE1, GENE2, GENE3]</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>SE2</td>\n",
       "      <td>[GENE4, GENE5, GENE6]</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>SE3</td>\n",
       "      <td>[GENE1, GENE3, GENE5]</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>SE4</td>\n",
       "      <td>[GENE2, GENE4, GENE6]</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>SE5</td>\n",
       "      <td>[GENE1, GENE4, GENE5]</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    SE                   GENE  MAX  MEF2C  MEF2D  MEIS1  MYB\n",
       "0  SE1  [GENE1, GENE2, GENE3]    0      0      1      0    1\n",
       "1  SE2  [GENE4, GENE5, GENE6]    1      0      0      0    0\n",
       "2  SE3  [GENE1, GENE3, GENE5]    1      1      1      1    1\n",
       "3  SE4  [GENE2, GENE4, GENE6]    0      0      0      1    0\n",
       "4  SE5  [GENE1, GENE4, GENE5]    0      1      0      1    1"
      ]
     },
     "execution_count": 452,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data = [[\"SE1\",[\"GENE1\", \"GENE2\", \"GENE3\"],0,0,1,0,1],\n",
    "        [\"SE2\",[\"GENE4\", \"GENE5\", \"GENE6\"],1,0,0,0,0],\n",
    "        [\"SE3\",[\"GENE1\", \"GENE3\", \"GENE5\"],1,1,1,1,1],\n",
    "        [\"SE4\",[\"GENE2\", \"GENE4\", \"GENE6\"],0,0,0,1,0],\n",
    "        [\"SE5\",[\"GENE1\", \"GENE4\", \"GENE5\"],0,1,0,1,1]]\n",
    "test = pd.DataFrame(data, columns=[\"SE\",\"GENE\",\"MAX\",\"MEF2C\", \"MEF2D\",\"MEIS1\",\"MYB\"])\n",
    "test"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 451,
   "id": "b5092abb",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>MAX</th>\n",
       "      <th>MEF2C</th>\n",
       "      <th>MEF2D</th>\n",
       "      <th>MEIS1</th>\n",
       "      <th>MYB</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>GENE1</th>\n",
       "      <td>1</td>\n",
       "      <td>2</td>\n",
       "      <td>2</td>\n",
       "      <td>2</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GENE2</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GENE3</th>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>2</td>\n",
       "      <td>1</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GENE4</th>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>2</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GENE5</th>\n",
       "      <td>2</td>\n",
       "      <td>2</td>\n",
       "      <td>1</td>\n",
       "      <td>2</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>GENE6</th>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "       MAX  MEF2C  MEF2D  MEIS1  MYB\n",
       "GENE1    1      2      2      2    3\n",
       "GENE2    0      0      1      1    1\n",
       "GENE3    1      1      2      1    2\n",
       "GENE4    1      1      0      2    1\n",
       "GENE5    2      2      1      2    2\n",
       "GENE6    1      0      0      1    0"
      ]
     },
     "execution_count": 451,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pd.DataFrame.from_dict({\"GENE1\":[1,2,2,2,3], \"GENE2\":[0,0,1,1,1], \"GENE3\":[1,1,2,1,2], \n",
    "                        \"GENE4\":[1,1,0,2,1], \"GENE5\":[2,2,1,2,2], \"GENE6\":[1,0,0,1,0]},\n",
    "                        orient=\"index\", columns=test.columns.tolist()[2:])"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8010a2b7",
   "metadata": {},
   "source": [
    "# check Jost's SE matrix for genes"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a07453e7",
   "metadata": {},
   "source": [
    "Issues with genes in certain regions. They do not have any assigned genes. It changed to a number. Might be an xlsx issue."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 501,
   "id": "300df3fd",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/monika/anaconda3/envs/slamseq/lib/python3.8/site-packages/IPython/core/interactiveshell.py:3441: DtypeWarning: Columns (10,11) have mixed types.Specify dtype option on import or set low_memory=False.\n",
      "  exec(code_obj, self.user_global_ns, self.user_ns)\n"
     ]
    }
   ],
   "source": [
    "jk = pd.read_csv(\"/home/monika/AMLproject/data/superenhancer/superenhancer_matrix_jk.csv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 502,
   "id": "21a71b9a",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0</th>\n",
       "      <th>CHROM</th>\n",
       "      <th>START.HG19</th>\n",
       "      <th>STOP.HG19</th>\n",
       "      <th>START.HG38</th>\n",
       "      <th>STOP.HG38</th>\n",
       "      <th>PROXIMAL_GENES</th>\n",
       "      <th>OVERLAP_GENES</th>\n",
       "      <th>CLOSEST_GENE</th>\n",
       "      <th>ABC_assignment ([Corr] &gt;0.3 in cell lines)</th>\n",
       "      <th>...</th>\n",
       "      <th>EOL1.4</th>\n",
       "      <th>Pearson Dep/SE</th>\n",
       "      <th>MLL-nonMLL Dep</th>\n",
       "      <th>p, MLL-nonMLL Dep</th>\n",
       "      <th>Average Dep score</th>\n",
       "      <th>Average SE score cell lines</th>\n",
       "      <th>Average SE score primary</th>\n",
       "      <th>Correlation to MEF2D (pAML)</th>\n",
       "      <th>Correlation to IRF8</th>\n",
       "      <th>Correlation to BCL2 dependency</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>chr1</td>\n",
       "      <td>188208220</td>\n",
       "      <td>188212962</td>\n",
       "      <td>188239089.0</td>\n",
       "      <td>188243831.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>5</td>\n",
       "      <td>2</td>\n",
       "      <td>0,0379143000748641</td>\n",
       "      <td>0,0920239347507136</td>\n",
       "      <td>-0,87233174050688</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>chr14</td>\n",
       "      <td>31595175</td>\n",
       "      <td>31600330</td>\n",
       "      <td>31125969.0</td>\n",
       "      <td>31131124.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>HECTD1</td>\n",
       "      <td>HECTD1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>3</td>\n",
       "      <td>0,195477662590484</td>\n",
       "      <td>-0,144763956280788</td>\n",
       "      <td>-0,851385789021602</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2</td>\n",
       "      <td>chr2</td>\n",
       "      <td>120107154</td>\n",
       "      <td>120126341</td>\n",
       "      <td>119349578.0</td>\n",
       "      <td>119368765.0</td>\n",
       "      <td>DBI</td>\n",
       "      <td>DBI C2orf76</td>\n",
       "      <td>C2orf76</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>18</td>\n",
       "      <td>19</td>\n",
       "      <td>0,671370252609654</td>\n",
       "      <td>0,408822184660836</td>\n",
       "      <td>-0,83625252406249</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>3</td>\n",
       "      <td>chr11</td>\n",
       "      <td>14575144</td>\n",
       "      <td>14578998</td>\n",
       "      <td>14553598.0</td>\n",
       "      <td>14557452.0</td>\n",
       "      <td>PSMA1</td>\n",
       "      <td>PSMA1</td>\n",
       "      <td>PSMA1</td>\n",
       "      <td>PDE3B</td>\n",
       "      <td>...</td>\n",
       "      <td>-2,20256386635</td>\n",
       "      <td>-0,105259020156355</td>\n",
       "      <td>0,0340497751521212</td>\n",
       "      <td>0,661128178413146</td>\n",
       "      <td>-2,07385252593529</td>\n",
       "      <td>3</td>\n",
       "      <td>3</td>\n",
       "      <td>0,203617619318475</td>\n",
       "      <td>-0,0106547504639564</td>\n",
       "      <td>-0,832003040497709</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>4</td>\n",
       "      <td>chr11</td>\n",
       "      <td>73169511</td>\n",
       "      <td>73175223</td>\n",
       "      <td>73458466.0</td>\n",
       "      <td>73464178.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>FAM168A</td>\n",
       "      <td>RELT</td>\n",
       "      <td>PAAF1</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>6</td>\n",
       "      <td>7</td>\n",
       "      <td>0,467617461610967</td>\n",
       "      <td>0,251036001503711</td>\n",
       "      <td>-0,825472555837567</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18139</th>\n",
       "      <td>18139</td>\n",
       "      <td>chrX</td>\n",
       "      <td>1619536</td>\n",
       "      <td>1627373</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ASMTL</td>\n",
       "      <td>P2RY8</td>\n",
       "      <td>P2RY8</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>4</td>\n",
       "      <td>-0,161001239494127</td>\n",
       "      <td>0,00635055497410523</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18140</th>\n",
       "      <td>18140</td>\n",
       "      <td>chrX</td>\n",
       "      <td>1581299</td>\n",
       "      <td>1591684</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ASMTL</td>\n",
       "      <td>P2RY8</td>\n",
       "      <td>ASMTL</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>7</td>\n",
       "      <td>-0,189069300511941</td>\n",
       "      <td>-0,209916254940075</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18141</th>\n",
       "      <td>18141</td>\n",
       "      <td>chrX</td>\n",
       "      <td>1392194</td>\n",
       "      <td>1394458</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CSF2RA MIR3690</td>\n",
       "      <td>CSF2RA</td>\n",
       "      <td>CSF2RA</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>-0,218221505891962</td>\n",
       "      <td>0,0480840763877521</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18142</th>\n",
       "      <td>18142</td>\n",
       "      <td>chrX</td>\n",
       "      <td>1653122</td>\n",
       "      <td>1657699</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>P2RY8</td>\n",
       "      <td>P2RY8</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>8</td>\n",
       "      <td>-0,268990863330352</td>\n",
       "      <td>-0,163071456921095</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18143</th>\n",
       "      <td>18143</td>\n",
       "      <td>chrX</td>\n",
       "      <td>1596345</td>\n",
       "      <td>1604399</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ASMTL</td>\n",
       "      <td>P2RY8</td>\n",
       "      <td>ASMTL</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>6</td>\n",
       "      <td>-0,277081706404252</td>\n",
       "      <td>-0,210993586508389</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>18144 rows × 157 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "       Unnamed: 0  CHROM  START.HG19  STOP.HG19   START.HG38    STOP.HG38  \\\n",
       "0               0   chr1   188208220  188212962  188239089.0  188243831.0   \n",
       "1               1  chr14    31595175   31600330   31125969.0   31131124.0   \n",
       "2               2   chr2   120107154  120126341  119349578.0  119368765.0   \n",
       "3               3  chr11    14575144   14578998   14553598.0   14557452.0   \n",
       "4               4  chr11    73169511   73175223   73458466.0   73464178.0   \n",
       "...           ...    ...         ...        ...          ...          ...   \n",
       "18139       18139   chrX     1619536    1627373          NaN          NaN   \n",
       "18140       18140   chrX     1581299    1591684          NaN          NaN   \n",
       "18141       18141   chrX     1392194    1394458          NaN          NaN   \n",
       "18142       18142   chrX     1653122    1657699          NaN          NaN   \n",
       "18143       18143   chrX     1596345    1604399          NaN          NaN   \n",
       "\n",
       "       PROXIMAL_GENES OVERLAP_GENES CLOSEST_GENE  \\\n",
       "0                 NaN           NaN          NaN   \n",
       "1                 NaN        HECTD1       HECTD1   \n",
       "2                 DBI   DBI C2orf76      C2orf76   \n",
       "3               PSMA1         PSMA1        PSMA1   \n",
       "4                 NaN       FAM168A         RELT   \n",
       "...               ...           ...          ...   \n",
       "18139           ASMTL         P2RY8        P2RY8   \n",
       "18140           ASMTL         P2RY8        ASMTL   \n",
       "18141  CSF2RA MIR3690        CSF2RA       CSF2RA   \n",
       "18142             NaN         P2RY8        P2RY8   \n",
       "18143           ASMTL         P2RY8        ASMTL   \n",
       "\n",
       "      ABC_assignment ([Corr] >0.3 in cell lines)  ...          EOL1.4  \\\n",
       "0                                            NaN  ...             NaN   \n",
       "1                                            NaN  ...             NaN   \n",
       "2                                            NaN  ...             NaN   \n",
       "3                                        PDE3B    ...  -2,20256386635   \n",
       "4                                        PAAF1    ...             NaN   \n",
       "...                                          ...  ...             ...   \n",
       "18139                                        NaN  ...             NaN   \n",
       "18140                                        NaN  ...             NaN   \n",
       "18141                                        NaN  ...             NaN   \n",
       "18142                                        NaN  ...             NaN   \n",
       "18143                                        NaN  ...             NaN   \n",
       "\n",
       "           Pearson Dep/SE      MLL-nonMLL Dep  p, MLL-nonMLL Dep  \\\n",
       "0                     NaN                 NaN                NaN   \n",
       "1                     NaN                 NaN                NaN   \n",
       "2                     NaN                 NaN                NaN   \n",
       "3      -0,105259020156355  0,0340497751521212  0,661128178413146   \n",
       "4                     NaN                 NaN                NaN   \n",
       "...                   ...                 ...                ...   \n",
       "18139                 NaN                 NaN                NaN   \n",
       "18140                 NaN                 NaN                NaN   \n",
       "18141                 NaN                 NaN                NaN   \n",
       "18142                 NaN                 NaN                NaN   \n",
       "18143                 NaN                 NaN                NaN   \n",
       "\n",
       "       Average Dep score  Average SE score cell lines  \\\n",
       "0                    NaN                            5   \n",
       "1                    NaN                            2   \n",
       "2                    NaN                           18   \n",
       "3      -2,07385252593529                            3   \n",
       "4                    NaN                            6   \n",
       "...                  ...                          ...   \n",
       "18139                NaN                            0   \n",
       "18140                NaN                            0   \n",
       "18141                NaN                            0   \n",
       "18142                NaN                            0   \n",
       "18143                NaN                            0   \n",
       "\n",
       "       Average SE score primary  Correlation to MEF2D (pAML)  \\\n",
       "0                             2           0,0379143000748641   \n",
       "1                             3            0,195477662590484   \n",
       "2                            19            0,671370252609654   \n",
       "3                             3            0,203617619318475   \n",
       "4                             7            0,467617461610967   \n",
       "...                         ...                          ...   \n",
       "18139                         4           -0,161001239494127   \n",
       "18140                         7           -0,189069300511941   \n",
       "18141                         1           -0,218221505891962   \n",
       "18142                         8           -0,268990863330352   \n",
       "18143                         6           -0,277081706404252   \n",
       "\n",
       "       Correlation to IRF8  Correlation to BCL2 dependency  \n",
       "0       0,0920239347507136               -0,87233174050688  \n",
       "1       -0,144763956280788              -0,851385789021602  \n",
       "2        0,408822184660836               -0,83625252406249  \n",
       "3      -0,0106547504639564              -0,832003040497709  \n",
       "4        0,251036001503711              -0,825472555837567  \n",
       "...                    ...                             ...  \n",
       "18139  0,00635055497410523                             NaN  \n",
       "18140   -0,209916254940075                             NaN  \n",
       "18141   0,0480840763877521                             NaN  \n",
       "18142   -0,163071456921095                             NaN  \n",
       "18143   -0,210993586508389                             NaN  \n",
       "\n",
       "[18144 rows x 157 columns]"
      ]
     },
     "execution_count": 502,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "jk"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "9b328e9c",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.11"
  },
  "toc": {
   "base_numbering": 1,
   "nav_menu": {},
   "number_sections": false,
   "sideBar": true,
   "skip_h1_title": false,
   "title_cell": "Table of Contents",
   "title_sidebar": "Contents",
   "toc_cell": false,
   "toc_position": {
    "height": "calc(100% - 180px)",
    "left": "10px",
    "top": "150px",
    "width": "191px"
   },
   "toc_section_display": true,
   "toc_window_display": true
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
