{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 295,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "    <div class=\"bk-root\">\n",
       "        <a href=\"https://bokeh.org\" target=\"_blank\" class=\"bk-logo bk-logo-small bk-logo-notebook\"></a>\n",
       "        <span id=\"1004\">Loading BokehJS ...</span>\n",
       "    </div>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/javascript": [
       "\n",
       "(function(root) {\n",
       "  function now() {\n",
       "    return new Date();\n",
       "  }\n",
       "\n",
       "  var force = true;\n",
       "\n",
       "  if (typeof root._bokeh_onload_callbacks === \"undefined\" || force === true) {\n",
       "    root._bokeh_onload_callbacks = [];\n",
       "    root._bokeh_is_loading = undefined;\n",
       "  }\n",
       "\n",
       "  var JS_MIME_TYPE = 'application/javascript';\n",
       "  var HTML_MIME_TYPE = 'text/html';\n",
       "  var EXEC_MIME_TYPE = 'application/vnd.bokehjs_exec.v0+json';\n",
       "  var CLASS_NAME = 'output_bokeh rendered_html';\n",
       "\n",
       "  /**\n",
       "   * Render data to the DOM node\n",
       "   */\n",
       "  function render(props, node) {\n",
       "    var script = document.createElement(\"script\");\n",
       "    node.appendChild(script);\n",
       "  }\n",
       "\n",
       "  /**\n",
       "   * Handle when an output is cleared or removed\n",
       "   */\n",
       "  function handleClearOutput(event, handle) {\n",
       "    var cell = handle.cell;\n",
       "\n",
       "    var id = cell.output_area._bokeh_element_id;\n",
       "    var server_id = cell.output_area._bokeh_server_id;\n",
       "    // Clean up Bokeh references\n",
       "    if (id != null && id in Bokeh.index) {\n",
       "      Bokeh.index[id].model.document.clear();\n",
       "      delete Bokeh.index[id];\n",
       "    }\n",
       "\n",
       "    if (server_id !== undefined) {\n",
       "      // Clean up Bokeh references\n",
       "      var cmd = \"from bokeh.io.state import curstate; print(curstate().uuid_to_server['\" + server_id + \"'].get_sessions()[0].document.roots[0]._id)\";\n",
       "      cell.notebook.kernel.execute(cmd, {\n",
       "        iopub: {\n",
       "          output: function(msg) {\n",
       "            var id = msg.content.text.trim();\n",
       "            if (id in Bokeh.index) {\n",
       "              Bokeh.index[id].model.document.clear();\n",
       "              delete Bokeh.index[id];\n",
       "            }\n",
       "          }\n",
       "        }\n",
       "      });\n",
       "      // Destroy server and session\n",
       "      var cmd = \"import bokeh.io.notebook as ion; ion.destroy_server('\" + server_id + \"')\";\n",
       "      cell.notebook.kernel.execute(cmd);\n",
       "    }\n",
       "  }\n",
       "\n",
       "  /**\n",
       "   * Handle when a new output is added\n",
       "   */\n",
       "  function handleAddOutput(event, handle) {\n",
       "    var output_area = handle.output_area;\n",
       "    var output = handle.output;\n",
       "\n",
       "    // limit handleAddOutput to display_data with EXEC_MIME_TYPE content only\n",
       "    if ((output.output_type != \"display_data\") || (!output.data.hasOwnProperty(EXEC_MIME_TYPE))) {\n",
       "      return\n",
       "    }\n",
       "\n",
       "    var toinsert = output_area.element.find(\".\" + CLASS_NAME.split(' ')[0]);\n",
       "\n",
       "    if (output.metadata[EXEC_MIME_TYPE][\"id\"] !== undefined) {\n",
       "      toinsert[toinsert.length - 1].firstChild.textContent = output.data[JS_MIME_TYPE];\n",
       "      // store reference to embed id on output_area\n",
       "      output_area._bokeh_element_id = output.metadata[EXEC_MIME_TYPE][\"id\"];\n",
       "    }\n",
       "    if (output.metadata[EXEC_MIME_TYPE][\"server_id\"] !== undefined) {\n",
       "      var bk_div = document.createElement(\"div\");\n",
       "      bk_div.innerHTML = output.data[HTML_MIME_TYPE];\n",
       "      var script_attrs = bk_div.children[0].attributes;\n",
       "      for (var i = 0; i < script_attrs.length; i++) {\n",
       "        toinsert[toinsert.length - 1].firstChild.setAttribute(script_attrs[i].name, script_attrs[i].value);\n",
       "        toinsert[toinsert.length - 1].firstChild.textContent = bk_div.children[0].textContent\n",
       "      }\n",
       "      // store reference to server id on output_area\n",
       "      output_area._bokeh_server_id = output.metadata[EXEC_MIME_TYPE][\"server_id\"];\n",
       "    }\n",
       "  }\n",
       "\n",
       "  function register_renderer(events, OutputArea) {\n",
       "\n",
       "    function append_mime(data, metadata, element) {\n",
       "      // create a DOM node to render to\n",
       "      var toinsert = this.create_output_subarea(\n",
       "        metadata,\n",
       "        CLASS_NAME,\n",
       "        EXEC_MIME_TYPE\n",
       "      );\n",
       "      this.keyboard_manager.register_events(toinsert);\n",
       "      // Render to node\n",
       "      var props = {data: data, metadata: metadata[EXEC_MIME_TYPE]};\n",
       "      render(props, toinsert[toinsert.length - 1]);\n",
       "      element.append(toinsert);\n",
       "      return toinsert\n",
       "    }\n",
       "\n",
       "    /* Handle when an output is cleared or removed */\n",
       "    events.on('clear_output.CodeCell', handleClearOutput);\n",
       "    events.on('delete.Cell', handleClearOutput);\n",
       "\n",
       "    /* Handle when a new output is added */\n",
       "    events.on('output_added.OutputArea', handleAddOutput);\n",
       "\n",
       "    /**\n",
       "     * Register the mime type and append_mime function with output_area\n",
       "     */\n",
       "    OutputArea.prototype.register_mime_type(EXEC_MIME_TYPE, append_mime, {\n",
       "      /* Is output safe? */\n",
       "      safe: true,\n",
       "      /* Index of renderer in `output_area.display_order` */\n",
       "      index: 0\n",
       "    });\n",
       "  }\n",
       "\n",
       "  // register the mime type if in Jupyter Notebook environment and previously unregistered\n",
       "  if (root.Jupyter !== undefined) {\n",
       "    var events = require('base/js/events');\n",
       "    var OutputArea = require('notebook/js/outputarea').OutputArea;\n",
       "\n",
       "    if (OutputArea.prototype.mime_types().indexOf(EXEC_MIME_TYPE) == -1) {\n",
       "      register_renderer(events, OutputArea);\n",
       "    }\n",
       "  }\n",
       "\n",
       "  \n",
       "  if (typeof (root._bokeh_timeout) === \"undefined\" || force === true) {\n",
       "    root._bokeh_timeout = Date.now() + 5000;\n",
       "    root._bokeh_failed_load = false;\n",
       "  }\n",
       "\n",
       "  var NB_LOAD_WARNING = {'data': {'text/html':\n",
       "     \"<div style='background-color: #fdd'>\\n\"+\n",
       "     \"<p>\\n\"+\n",
       "     \"BokehJS does not appear to have successfully loaded. If loading BokehJS from CDN, this \\n\"+\n",
       "     \"may be due to a slow or bad network connection. Possible fixes:\\n\"+\n",
       "     \"</p>\\n\"+\n",
       "     \"<ul>\\n\"+\n",
       "     \"<li>re-rerun `output_notebook()` to attempt to load from CDN again, or</li>\\n\"+\n",
       "     \"<li>use INLINE resources instead, as so:</li>\\n\"+\n",
       "     \"</ul>\\n\"+\n",
       "     \"<code>\\n\"+\n",
       "     \"from bokeh.resources import INLINE\\n\"+\n",
       "     \"output_notebook(resources=INLINE)\\n\"+\n",
       "     \"</code>\\n\"+\n",
       "     \"</div>\"}};\n",
       "\n",
       "  function display_loaded() {\n",
       "    var el = document.getElementById(\"1004\");\n",
       "    if (el != null) {\n",
       "      el.textContent = \"BokehJS is loading...\";\n",
       "    }\n",
       "    if (root.Bokeh !== undefined) {\n",
       "      if (el != null) {\n",
       "        el.textContent = \"BokehJS \" + root.Bokeh.version + \" successfully loaded.\";\n",
       "      }\n",
       "    } else if (Date.now() < root._bokeh_timeout) {\n",
       "      setTimeout(display_loaded, 100)\n",
       "    }\n",
       "  }\n",
       "\n",
       "\n",
       "  function run_callbacks() {\n",
       "    try {\n",
       "      root._bokeh_onload_callbacks.forEach(function(callback) {\n",
       "        if (callback != null)\n",
       "          callback();\n",
       "      });\n",
       "    } finally {\n",
       "      delete root._bokeh_onload_callbacks\n",
       "    }\n",
       "    console.debug(\"Bokeh: all callbacks have finished\");\n",
       "  }\n",
       "\n",
       "  function load_libs(css_urls, js_urls, callback) {\n",
       "    if (css_urls == null) css_urls = [];\n",
       "    if (js_urls == null) js_urls = [];\n",
       "\n",
       "    root._bokeh_onload_callbacks.push(callback);\n",
       "    if (root._bokeh_is_loading > 0) {\n",
       "      console.debug(\"Bokeh: BokehJS is being loaded, scheduling callback at\", now());\n",
       "      return null;\n",
       "    }\n",
       "    if (js_urls == null || js_urls.length === 0) {\n",
       "      run_callbacks();\n",
       "      return null;\n",
       "    }\n",
       "    console.debug(\"Bokeh: BokehJS not loaded, scheduling load and callback at\", now());\n",
       "    root._bokeh_is_loading = css_urls.length + js_urls.length;\n",
       "\n",
       "    function on_load() {\n",
       "      root._bokeh_is_loading--;\n",
       "      if (root._bokeh_is_loading === 0) {\n",
       "        console.debug(\"Bokeh: all BokehJS libraries/stylesheets loaded\");\n",
       "        run_callbacks()\n",
       "      }\n",
       "    }\n",
       "\n",
       "    function on_error() {\n",
       "      console.error(\"failed to load \" + url);\n",
       "    }\n",
       "\n",
       "    for (var i = 0; i < css_urls.length; i++) {\n",
       "      var url = css_urls[i];\n",
       "      const element = document.createElement(\"link\");\n",
       "      element.onload = on_load;\n",
       "      element.onerror = on_error;\n",
       "      element.rel = \"stylesheet\";\n",
       "      element.type = \"text/css\";\n",
       "      element.href = url;\n",
       "      console.debug(\"Bokeh: injecting link tag for BokehJS stylesheet: \", url);\n",
       "      document.body.appendChild(element);\n",
       "    }\n",
       "\n",
       "    const hashes = {\"https://cdn.bokeh.org/bokeh/release/bokeh-2.0.2.min.js\": \"ufR9RFnRs6lniiaFvtJziE0YeidtAgBRH6ux2oUItHw5WTvE1zuk9uzhUU/FJXDp\", \"https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.0.2.min.js\": \"8QM/PGWBT+IssZuRcDcjzwIh1mkOmJSoNMmyYDZbCfXJg3Ap1lEvdVgFuSAwhb/J\", \"https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.0.2.min.js\": \"Jm8cH3Rg0P6UeZhVY5cLy1WzKajUT9KImCY+76hEqrcJt59/d8GPvFHjCkYgnSIn\", \"https://cdn.bokeh.org/bokeh/release/bokeh-gl-2.0.2.min.js\": \"Ozhzj+SI7ywm74aOI/UajcWz+C0NjsPunEVyVIrxzYkB+jA+2tUw8x5xJCbVtK5I\"};\n",
       "\n",
       "    for (var i = 0; i < js_urls.length; i++) {\n",
       "      var url = js_urls[i];\n",
       "      var element = document.createElement('script');\n",
       "      element.onload = on_load;\n",
       "      element.onerror = on_error;\n",
       "      element.async = false;\n",
       "      element.src = url;\n",
       "      if (url in hashes) {\n",
       "        element.crossOrigin = \"anonymous\";\n",
       "        element.integrity = \"sha384-\" + hashes[url];\n",
       "      }\n",
       "      console.debug(\"Bokeh: injecting script tag for BokehJS library: \", url);\n",
       "      document.head.appendChild(element);\n",
       "    }\n",
       "  };\n",
       "\n",
       "  function inject_raw_css(css) {\n",
       "    const element = document.createElement(\"style\");\n",
       "    element.appendChild(document.createTextNode(css));\n",
       "    document.body.appendChild(element);\n",
       "  }\n",
       "\n",
       "  \n",
       "  var js_urls = [\"https://cdn.bokeh.org/bokeh/release/bokeh-2.0.2.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.0.2.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.0.2.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-gl-2.0.2.min.js\"];\n",
       "  var css_urls = [];\n",
       "  \n",
       "\n",
       "  var inline_js = [\n",
       "    function(Bokeh) {\n",
       "      Bokeh.set_log_level(\"info\");\n",
       "    },\n",
       "    function(Bokeh) {\n",
       "    \n",
       "    \n",
       "    }\n",
       "  ];\n",
       "\n",
       "  function run_inline_js() {\n",
       "    \n",
       "    if (root.Bokeh !== undefined || force === true) {\n",
       "      \n",
       "    for (var i = 0; i < inline_js.length; i++) {\n",
       "      inline_js[i].call(root, root.Bokeh);\n",
       "    }\n",
       "    if (force === true) {\n",
       "        display_loaded();\n",
       "      }} else if (Date.now() < root._bokeh_timeout) {\n",
       "      setTimeout(run_inline_js, 100);\n",
       "    } else if (!root._bokeh_failed_load) {\n",
       "      console.log(\"Bokeh: BokehJS failed to load within specified timeout.\");\n",
       "      root._bokeh_failed_load = true;\n",
       "    } else if (force !== true) {\n",
       "      var cell = $(document.getElementById(\"1004\")).parents('.cell').data().cell;\n",
       "      cell.output_area.append_execute_result(NB_LOAD_WARNING)\n",
       "    }\n",
       "\n",
       "  }\n",
       "\n",
       "  if (root._bokeh_is_loading === 0) {\n",
       "    console.debug(\"Bokeh: BokehJS loaded, going straight to plotting\");\n",
       "    run_inline_js();\n",
       "  } else {\n",
       "    load_libs(css_urls, js_urls, function() {\n",
       "      console.debug(\"Bokeh: BokehJS plotting callback run at\", now());\n",
       "      run_inline_js();\n",
       "    });\n",
       "  }\n",
       "}(window));"
      ],
      "application/vnd.bokehjs_load.v0+json": "\n(function(root) {\n  function now() {\n    return new Date();\n  }\n\n  var force = true;\n\n  if (typeof root._bokeh_onload_callbacks === \"undefined\" || force === true) {\n    root._bokeh_onload_callbacks = [];\n    root._bokeh_is_loading = undefined;\n  }\n\n  \n\n  \n  if (typeof (root._bokeh_timeout) === \"undefined\" || force === true) {\n    root._bokeh_timeout = Date.now() + 5000;\n    root._bokeh_failed_load = false;\n  }\n\n  var NB_LOAD_WARNING = {'data': {'text/html':\n     \"<div style='background-color: #fdd'>\\n\"+\n     \"<p>\\n\"+\n     \"BokehJS does not appear to have successfully loaded. If loading BokehJS from CDN, this \\n\"+\n     \"may be due to a slow or bad network connection. Possible fixes:\\n\"+\n     \"</p>\\n\"+\n     \"<ul>\\n\"+\n     \"<li>re-rerun `output_notebook()` to attempt to load from CDN again, or</li>\\n\"+\n     \"<li>use INLINE resources instead, as so:</li>\\n\"+\n     \"</ul>\\n\"+\n     \"<code>\\n\"+\n     \"from bokeh.resources import INLINE\\n\"+\n     \"output_notebook(resources=INLINE)\\n\"+\n     \"</code>\\n\"+\n     \"</div>\"}};\n\n  function display_loaded() {\n    var el = document.getElementById(\"1004\");\n    if (el != null) {\n      el.textContent = \"BokehJS is loading...\";\n    }\n    if (root.Bokeh !== undefined) {\n      if (el != null) {\n        el.textContent = \"BokehJS \" + root.Bokeh.version + \" successfully loaded.\";\n      }\n    } else if (Date.now() < root._bokeh_timeout) {\n      setTimeout(display_loaded, 100)\n    }\n  }\n\n\n  function run_callbacks() {\n    try {\n      root._bokeh_onload_callbacks.forEach(function(callback) {\n        if (callback != null)\n          callback();\n      });\n    } finally {\n      delete root._bokeh_onload_callbacks\n    }\n    console.debug(\"Bokeh: all callbacks have finished\");\n  }\n\n  function load_libs(css_urls, js_urls, callback) {\n    if (css_urls == null) css_urls = [];\n    if (js_urls == null) js_urls = [];\n\n    root._bokeh_onload_callbacks.push(callback);\n    if (root._bokeh_is_loading > 0) {\n      console.debug(\"Bokeh: BokehJS is being loaded, scheduling callback at\", now());\n      return null;\n    }\n    if (js_urls == null || js_urls.length === 0) {\n      run_callbacks();\n      return null;\n    }\n    console.debug(\"Bokeh: BokehJS not loaded, scheduling load and callback at\", now());\n    root._bokeh_is_loading = css_urls.length + js_urls.length;\n\n    function on_load() {\n      root._bokeh_is_loading--;\n      if (root._bokeh_is_loading === 0) {\n        console.debug(\"Bokeh: all BokehJS libraries/stylesheets loaded\");\n        run_callbacks()\n      }\n    }\n\n    function on_error() {\n      console.error(\"failed to load \" + url);\n    }\n\n    for (var i = 0; i < css_urls.length; i++) {\n      var url = css_urls[i];\n      const element = document.createElement(\"link\");\n      element.onload = on_load;\n      element.onerror = on_error;\n      element.rel = \"stylesheet\";\n      element.type = \"text/css\";\n      element.href = url;\n      console.debug(\"Bokeh: injecting link tag for BokehJS stylesheet: \", url);\n      document.body.appendChild(element);\n    }\n\n    const hashes = {\"https://cdn.bokeh.org/bokeh/release/bokeh-2.0.2.min.js\": \"ufR9RFnRs6lniiaFvtJziE0YeidtAgBRH6ux2oUItHw5WTvE1zuk9uzhUU/FJXDp\", \"https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.0.2.min.js\": \"8QM/PGWBT+IssZuRcDcjzwIh1mkOmJSoNMmyYDZbCfXJg3Ap1lEvdVgFuSAwhb/J\", \"https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.0.2.min.js\": \"Jm8cH3Rg0P6UeZhVY5cLy1WzKajUT9KImCY+76hEqrcJt59/d8GPvFHjCkYgnSIn\", \"https://cdn.bokeh.org/bokeh/release/bokeh-gl-2.0.2.min.js\": \"Ozhzj+SI7ywm74aOI/UajcWz+C0NjsPunEVyVIrxzYkB+jA+2tUw8x5xJCbVtK5I\"};\n\n    for (var i = 0; i < js_urls.length; i++) {\n      var url = js_urls[i];\n      var element = document.createElement('script');\n      element.onload = on_load;\n      element.onerror = on_error;\n      element.async = false;\n      element.src = url;\n      if (url in hashes) {\n        element.crossOrigin = \"anonymous\";\n        element.integrity = \"sha384-\" + hashes[url];\n      }\n      console.debug(\"Bokeh: injecting script tag for BokehJS library: \", url);\n      document.head.appendChild(element);\n    }\n  };\n\n  function inject_raw_css(css) {\n    const element = document.createElement(\"style\");\n    element.appendChild(document.createTextNode(css));\n    document.body.appendChild(element);\n  }\n\n  \n  var js_urls = [\"https://cdn.bokeh.org/bokeh/release/bokeh-2.0.2.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.0.2.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.0.2.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-gl-2.0.2.min.js\"];\n  var css_urls = [];\n  \n\n  var inline_js = [\n    function(Bokeh) {\n      Bokeh.set_log_level(\"info\");\n    },\n    function(Bokeh) {\n    \n    \n    }\n  ];\n\n  function run_inline_js() {\n    \n    if (root.Bokeh !== undefined || force === true) {\n      \n    for (var i = 0; i < inline_js.length; i++) {\n      inline_js[i].call(root, root.Bokeh);\n    }\n    if (force === true) {\n        display_loaded();\n      }} else if (Date.now() < root._bokeh_timeout) {\n      setTimeout(run_inline_js, 100);\n    } else if (!root._bokeh_failed_load) {\n      console.log(\"Bokeh: BokehJS failed to load within specified timeout.\");\n      root._bokeh_failed_load = true;\n    } else if (force !== true) {\n      var cell = $(document.getElementById(\"1004\")).parents('.cell').data().cell;\n      cell.output_area.append_execute_result(NB_LOAD_WARNING)\n    }\n\n  }\n\n  if (root._bokeh_is_loading === 0) {\n    console.debug(\"Bokeh: BokehJS loaded, going straight to plotting\");\n    run_inline_js();\n  } else {\n    load_libs(css_urls, js_urls, function() {\n      console.debug(\"Bokeh: BokehJS plotting callback run at\", now());\n      run_inline_js();\n    });\n  }\n}(window));"
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The autoreload extension is already loaded. To reload it, use:\n",
      "  %reload_ext autoreload\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "import pandas as pd\n",
    "import sys\n",
    "import numpy as np\n",
    "import sys\n",
    "sys.path.insert(0, '../..')\n",
    "import itertools\n",
    "\n",
    "from JKBio import TerraFunction as terra\n",
    "from ccle_processing.src.CCLE_postp_function import *\n",
    "from JKBio import Helper as h\n",
    "from JKBio.epigenetics import ChIP_helper as chip\n",
    "\n",
    "import dalmatian as dm\n",
    "import pyBigWig\n",
    "\n",
    "from bokeh.plotting import *\n",
    "from IPython.display import IFrame\n",
    "import igv\n",
    "from biomart import BiomartServer\n",
    "import io\n",
    "\n",
    "from sklearn.manifold import MDS, TSNE\n",
    "from sklearn.decomposition import PCA\n",
    "from sklearn.preprocessing import scale\n",
    "from sklearn.cluster import AgglomerativeClustering\n",
    "from sklearn.mixture import GaussianMixture\n",
    "\n",
    "output_notebook()\n",
    "%load_ext autoreload\n",
    "%autoreload 2"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# ChIP"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "project=\"IRF2BP2_degron_ChIP\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!gsutil ls gs://amlproject/Chip/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# install bwa\n",
    "! mkdir ../../ref\n",
    "! curl ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/001/215/GCF_000001215.4_Release_6_plus_ISO1_MT/GCF_000001215.4_Release_6_plus_ISO1_MT_genomic.fna.gz -o ../../ref/reference_droso.fna.gz\n",
    "! gunzip ref/reference_droso.fna.gz\n",
    "! bwa index -a bwtsw ../../ref/reference_droso.fna"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## V1"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "rename1 = {\n",
    " 'gs://transfer-amlproject/20191211_10_MP7613_S8_R1_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp779-MV411_IRF2BP2_DMSO_6h-FLAG_IRF2BP2-r1_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_10_MP7613_S8_R2_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp779-MV411_IRF2BP2_DMSO_6h-FLAG_IRF2BP2-r1_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_11_MP7613_S9_R1_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp780-MV411_IRF2BP2_DMSO_6h-MED1-r1_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_11_MP7613_S9_R2_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp780-MV411_IRF2BP2_DMSO_6h-MED1-r1_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_12_MP7613_S10_R1_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp781-MV411_IRF2BP2_DMSO_6h-POLII_total-r1_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_12_MP7613_S10_R2_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp781-MV411_IRF2BP2_DMSO_6h-POLII_total-r1_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_13_MP7613_S11_R1_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp782-MV411_IRF2BP2_DMSO_6h-POLII_S2-r1_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_13_MP7613_S11_R2_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp782-MV411_IRF2BP2_DMSO_6h-POLII_S2-r1_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_14_MP7613_S12_R1_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp783-MV411_IRF2BP2_DMSO_6h-POLII_S5-r1_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_14_MP7613_S12_R2_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp783-MV411_IRF2BP2_DMSO_6h-POLII_S5-r1_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_15_MP7613_S13_R1_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp784-MV411_IRF2BP2_DMSO_6h-MYC-r1_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_15_MP7613_S13_R2_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp784-MV411_IRF2BP2_DMSO_6h-MYC-r1_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_16_MP7613_S14_R1_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp785-MV411_IRF2BP2_DMSO_6h-MYB-r1_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_16_MP7613_S14_R2_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp785-MV411_IRF2BP2_DMSO_6h-MYB-r1_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_1_MP7613_S1_R1_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp786-MV411_IRF2BP2_DMSO_6h-SPI1-r1_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_1_MP7613_S1_R2_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp786-MV411_IRF2BP2_DMSO_6h-SPI1-r1_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_2_MP7613_S2_R1_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp787-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r1_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_2_MP7613_S2_R2_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp787-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r1_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_3_MP7613_S3_R1_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp788-MV411_IRF2BP_VHL_6h-MED1-r1_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_3_MP7613_S3_R2_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp788-MV411_IRF2BP_VHL_6h-MED1-r1_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_4_MP7613_S4_R1_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp789-MV411_IRF2BP_VHL_6h-POLII_total-r1_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_4_MP7613_S4_R2_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp789-MV411_IRF2BP_VHL_6h-POLII_total-r1_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_5R_MP7613_S15_R1_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp790-MV411_IRF2BP_VHL_6h-POLII_S2-r1_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_5R_MP7613_S15_R2_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp790-MV411_IRF2BP_VHL_6h-POLII_S2-r1_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_6R_MP7613_S16_R1_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp791-MV411_IRF2BP_VHL_6h-POLII_S5-r1_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_6R_MP7613_S16_R2_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp791-MV411_IRF2BP_VHL_6h-POLII_S5-r1_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_7_MP7613_S5_R1_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp792-MV411_IRF2BP_VHL_6h-MYC-r1_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_7_MP7613_S5_R2_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp792-MV411_IRF2BP_VHL_6h-MYC-r1_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_8_MP7613_S6_R1_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp793-MV411_IRF2BP_VHL_6h-MYB-r1_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_8_MP7613_S6_R2_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp793-MV411_IRF2BP_VHL_6h-MYB-r1_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_9_MP7613_S7_R1_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp794-MV411_IRF2BP_VHL_6h-SPI1-r1_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_9_MP7613_S7_R2_001.fastq.gz':\"gs://amlproject/Chip/IRF2BP2_degraded_rep1/fastqs/mp794-MV411_IRF2BP_VHL_6h-SPI1-r1_2.fastq.gz\"}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for k, val in rename1.items():\n",
    "    os.system('gsutil mv '+k+' '+val)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! gsutil -m cp -r gs://amlproject/Chip/IRF2BP2_degraded_rep1 ../../data/\n",
    "inputfastq=\"gs://amlproject/Chip/fastqs/mp99-MV411-INPUT-r1.fastq.gz\"\n",
    "! gsutil cp $inputfastq ../../data/IRF2BP2_degraded_rep1/fastqs/\n",
    "a = ! ls ../../data/IRF2BP2_degraded_rep1/fastqs/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from gsheets import Sheets\n",
    "sheets = Sheets.from_files('~/.client_secret.json', '~/.storage.json')\n",
    "url=\"https://docs.google.com/spreadsheets/d/1yFLjYB1McU530JnLgL0QIMAKIkVl3kl0_LCHje2gk8U\"\n",
    "gsheet = sheets.get(url).sheets[2].to_frame()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "gsheet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = {\n",
    "\"fastq1\": [],\n",
    "\"fastq2\": [],\n",
    "\"antibody\": [],\n",
    "\"group\": [],\n",
    "\"replicate\": [],\n",
    "\"control\": []\n",
    "}\n",
    "for val in helper.grouped(a[:-1],2):\n",
    "    row = gsheet[gsheet.id==val[0].split('/')[-1].split('_')[0]]\n",
    "    df['group'].append((row.id.values[0] + '_'+ row.name_replicate.values[0] +\"_\"+row.protein.values[0]))\n",
    "    df['replicate'].append(1)\n",
    "    df['fastq1'].append(val[0])\n",
    "    #df['fastq2'].append(val[1])\n",
    "    df['antibody'].append(row.protein.values[0])\n",
    "    df['control'].append(\"INPUT\")\n",
    "df['group'].append('INPUT')\n",
    "df['replicate'].append(1)\n",
    "df['fastq1'].append(a[-1])\n",
    "#df['fastq2'].append(val[1])\n",
    "df['antibody'].append(\"INPUT\")\n",
    "df['control'].append(\"INPUT\")\n",
    "df = pd.DataFrame(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df[df.columns[[3,4,0,1,2,5]]].to_csv('../nextflow/design_IRF2BP2_degraded_rep1.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#process chips\n",
    "! sudo ./nextflow run nf-core/chipseq --single_end --seq_center 'DFCI' --email 'jkobject@gmail.com' --narrow_peak --input ../nextflow/design_IRF2BP2_degraded_rep1.csv --genome GRCh38 --skip_preseq --max_cpus 24 -profile docker -w work"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#!gsutil cp results/* ../../data/IRF2BP2_degraded_rep1/ && sudo rm -r results && sudo rm -r work"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# get scaling values\n",
    "mappedreads, umappedreads_norm, mapped = chip.getSpikeInControlScales(refgenome=\"../../ref/reference_droso.fna\",FastQfolder='../../data/IRF2BP2_degraded_rep1/fastqs/',pairedEnd=True, cores=8)\n",
    "mappedreads, umappedreads_norm, mapped"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "computing scales from the excel sheet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [[562285,1496707],\n",
    "[1686168,7198567],\n",
    "[3642441,2612624],\n",
    "[3992589,3474812],\n",
    "[3347901,3829477],\n",
    "[6181136,989703],\n",
    "[7523840,4173047],\n",
    "[922482,1195857]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [[1.0, 0.3756814126], #IRF2BP2\n",
    "[1.0, 0.2342366196], #MED1\n",
    "[0.5546432407, 1.0], #MYB\n",
    "[0.1601166841, 1.0], #MYC\n",
    "[0.870315477, 1.0], #POL2S2\n",
    "[1.0, 0.8742449687], #POL2S5\n",
    "[0.7172728398, 1.0], #POL2TOT\n",
    "[1.0, 0.7713982525]] #SPI1"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### on scalled data\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bams = !ls ../../data/IRF2BP2_degraded_rep1/diffData/*treat_pileup.bdg\n",
    "bams = ['_'.join(i.split('/')[-1].split('_')[:-2]) for i in bams]\n",
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# diffPeak on scalled data\n",
    "sizes = [220, 191, 211, 204, 285, 222, 228, 194]\n",
    "for i in range(int(len(bams)/2)):\n",
    "    name1 = bams[i]\n",
    "    name2 = bams[i+8]\n",
    "    print(name1,name2)\n",
    "    print(chip.diffPeak(name1,name2, directory= \"../../data/IRF2BP2_degraded_rep1/diffData/\", res_directory='../../data/IRF2BP2_degraded_rep1/diffPeaks/', scaling1=scales[i][1], scaling2=scales[i][0], size=size[i]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# diffPeak on scalled data\n",
    "for i in range(int(len(bams[1:])/2)-1):\n",
    "    name1 = bams[1+i]\n",
    "    name2 = bams[9+i]\n",
    "    chip.fullDiffPeak(name1,name2, bams[0], scaling = scales[i],compute_size=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [1.0,\n",
    "1.0,\n",
    "0.5546432407,\n",
    "0.1601166841,\n",
    "0.870315477,\n",
    "1.0,\n",
    "0.7172728398,\n",
    "1.0,\n",
    "0.3756814126,\n",
    "0.2342366196,\n",
    "1.0,\n",
    "1.0,\n",
    "1.0,\n",
    "0.8742449687,\n",
    "1.0,\n",
    "0.7713982525]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "chip.bigWigFrom(bams[1:], genome='GRCh38',scaling=scales)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mv ../../data/recalib_bigwig_rep1/* ../../data/IRF2BP2_degraded_rep1/recalib_bigwig/\n",
    "bw = ! ls ../../data/IRF2BP2_degraded_rep1/recalib_bigwig/*.bw\n",
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "cond1peak = ! ls ../../data/IRF2BP2_degraded_rep1/diffPeaks/*cond1.bed\n",
    "cond2peak = ! ls ../../data/IRF2BP2_degraded_rep1/diffPeaks/*cond2.bed\n",
    "commonpeak = ! ls ../../data/IRF2BP2_degraded_rep1/diffPeaks/*common.bed\n",
    "commonpeak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"FLAG_IRF2BP2\",\"MED1\",\"MYB\",\"MYC\",\"POLII_S2\",\"POLII_S5\",\"POLII_total\",\"SPI1\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for i in range(int(len(bw)/2)):\n",
    "    if i<0:\n",
    "        continue\n",
    "    name1 = bw[i]\n",
    "    name2 = bw[8+i]\n",
    "    peak = [cond1peak[i], commonpeak[i], cond2peak[i]]\n",
    "    for val in peak:\n",
    "        chip.dropWeirdChromosomes(val)\n",
    "    name = names[i]\n",
    "    print(name1,name2)\n",
    "    chip.getPeaksAt(peak, [name1, name2], bigwignames=['DMSO', 'VHL'], peaknames=['DMSO_peaks', 'common', 'VHL_peaks'], window=3000, refpoint='center', folder=\"\", title=name, numthreads=7, torecompute=True, name='../../data/IRF2BP2_degraded_rep1/diffPeaks/'+name+'_mat.pdf', withDeeptools=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "peaks = ! ls ../../data/IRF2BP2_degraded_rep1/bwa/mergedLibrary/macs/narrowPeak/*.narrowPeak\n",
    "peaks"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"FLAG_IRF2BP2\",\"MED1\",\"MYB\",\"MYC\",\"POLII_S2\",\"POLII_S5\",\"POLII_total\",\"SPI1\"]\n",
    "names.extend([i+'_VHL' for i in names])\n",
    "names"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for i in range(len(bw)):\n",
    "    chip.getPeaksAt(peaks[i], bw[i], window=3000, folder=\"\", title=names[i], numthreads=7, torecompute=True, onlyProfile=True, refpoint='center', name='../../data/IRF2BP2_degraded_rep1/diffPeaks/'+names[i]+'_mat_profile.pdf', withDeeptools=True)\n",
    "    chip.getPeaksAt(peaks[i], bw[i], window=3000, folder=\"\", title=names[i], numthreads=7, torecompute=True, refpoint='center',onlyProfile=True,name='../../data/IRF2BP2_degraded_rep1/diffPeaks/'+names[i]+'_mat_profile_clust3.pdf', withDeeptools=True, cluster=3)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### on unscalled data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bams = !ls ../../data/IRF2BP2_degraded_rep1/bwa/mergedLibrary/mp*.bam\n",
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mkdir ../../data/IRF2BP2_degraded_rep1/diffPeaks_unscaled"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bams= list(bams)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "size"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# on unscalled data \n",
    "for i in range(int((len(bams)-1)/2)-1):\n",
    "    if i < 0:\n",
    "        continue\n",
    "    name1 = bams[1+i]\n",
    "    name2 = bams[9+i]\n",
    "    chip.fullDiffPeak(name1,name2, control1=bams[0], size=size[i], compute_size=False, directory = \"../../data/IRF2BP2_degraded_rep1/diffData_unscaled/\", res_directory = \"../../data/IRF2BP2_degraded_rep1/diffPeaks_unscaled/\",pairedend=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bw = ! ls ../../data/IRF2BP2_degraded_rep1/bwa/mergedLibrary/bigwig/*.bigWig"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "commonpeak = ! ls ../../data/IRF2BP2_degraded_rep1/diffPeaks_unscaled/*common.bed\n",
    "cond1peak = ! ls ../../data/IRF2BP2_degraded_rep1/diffPeaks_unscaled/*cond1.bed\n",
    "cond2peak = ! ls ../../data/IRF2BP2_degraded_rep1/diffPeaks_unscaled/*cond2.bed\n",
    "cond2peak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"FLAG_IRF2BP2\",\"MED1\",\"POLII_total\",\"POLII_S2\",\"POLII_S5\",\"MYC\",\"MYB\",\"SPI1\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for i in range(int(len(bw[1:])/2)-1):\n",
    "    name1 = bw[1+i]\n",
    "    name2 = bw[9+i]\n",
    "    peak = [cond1peak[i], commonpeak[i], cond2peak[i]]\n",
    "    for val in peak:\n",
    "        chip.dropWeirdChromosomes(val)\n",
    "    name = names[i]\n",
    "    chip.getPeaksAt(peak, [name1, name2], bigwignames=['DMSO', 'VHL'], peaknames=['DMSO_peaks', 'common', 'VHL_peaks'], window=3000, title=name, numthreads=7, refpoint='center', folder=\"../../data/IRF2BP2_degraded_rep1/bwa/mergedLibrary/bigwig/\", torecompute=True, name='../../data/IRF2BP2_degraded_rep1/diffPeaks_unscaled/'+name+'_mat.pdf', withDeeptools=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! gsutil -m cp -r ../../data/IRF2BP2_degraded_rep1 gs://amlproject/Chip/\n",
    "# rm -r ../../data/IRF2BP2_degraded_rep1"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## v2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# mkdir ../../data/IRF2BP2_degraded_rep2/ && mkdir ../../data/IRF2BP2_degraded_rep2/fastqs && gsutil -m cp \"gs://transfer-amlproject/20191219_MP7659*\" ../../data/IRF2BP2_degraded_rep2/fastqs/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "rename = {\n",
    "\"20191219_MP7659_1_S1_R1_001.fastq.gz\":\"mp795-MV411_IRF2BP2_DMSO_6h-FLAG_IRF2BP2-r2_R1.fastq.gz\",\n",
    "\"20191219_MP7659_1_S1_R2_001.fastq.gz\":\"mp795-MV411_IRF2BP2_DMSO_6h-FLAG_IRF2BP2-r2_R2.fastq.gz\",\n",
    "\"20191219_MP7659_2_S2_R1_001.fastq.gz\":\"mp796-MV411_IRF2BP2_DMSO_6h-MED1-r2_R1.fastq.gz\",\n",
    "\"20191219_MP7659_2_S2_R2_001.fastq.gz\":\"mp796-MV411_IRF2BP2_DMSO_6h-MED1-r2_R2.fastq.gz\",\n",
    "\"20191219_MP7659_3_S3_R1_001.fastq.gz\":\"mp797-MV411_IRF2BP2_DMSO_6h-POLII_total-r2_R1.fastq.gz\",\n",
    "\"20191219_MP7659_3_S3_R2_001.fastq.gz\":\"mp797-MV411_IRF2BP2_DMSO_6h-POLII_total-r2_R2.fastq.gz\",\n",
    "\"20191219_MP7659_4_S4_R1_001.fastq.gz\":\"mp798-MV411_IRF2BP2_DMSO_6h-POLII_S2-r2_R1.fastq.gz\",\n",
    "\"20191219_MP7659_4_S4_R2_001.fastq.gz\":\"mp798-MV411_IRF2BP2_DMSO_6h-POLII_S2-r2_R2.fastq.gz\",\n",
    "\"20191219_MP7659_5_S5_R1_001.fastq.gz\":\"mp799-MV411_IRF2BP2_DMSO_6h-POLII_S5-r2_R1.fastq.gz\",\n",
    "\"20191219_MP7659_5_S5_R2_001.fastq.gz\":\"mp799-MV411_IRF2BP2_DMSO_6h-POLII_S5-r2_R2.fastq.gz\",\n",
    "\"20191219_MP7659_6_S6_R1_001.fastq.gz\":\"mp800-MV411_IRF2BP2_DMSO_6h-MYC-r2_R1.fastq.gz\",\n",
    "\"20191219_MP7659_6_S6_R2_001.fastq.gz\":\"mp800-MV411_IRF2BP2_DMSO_6h-MYC-r2_R2.fastq.gz\",\n",
    "\"20191219_MP7659_7_S7_R1_001.fastq.gz\":\"mp801-MV411_IRF2BP2_DMSO_6h-MYB-r2_R1.fastq.gz\",\n",
    "\"20191219_MP7659_7_S7_R2_001.fastq.gz\":\"mp801-MV411_IRF2BP2_DMSO_6h-MYB-r2_R2.fastq.gz\",\n",
    "\"20191219_MP7659_8_S8_R1_001.fastq.gz\":\"mp802-MV411_IRF2BP2_DMSO_6h-SPI1-r2_R1.fastq.gz\",\n",
    "\"20191219_MP7659_8_S8_R2_001.fastq.gz\":\"mp802-MV411_IRF2BP2_DMSO_6h-SPI1-r2_R2.fastq.gz\",\n",
    "\"20191219_MP7659_9_S9_R1_001.fastq.gz\":\"mp803-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r2_R1.fastq.gz\",\n",
    "\"20191219_MP7659_9_S9_R2_001.fastq.gz\":\"mp803-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r2_R2.fastq.gz\",\n",
    "\"20191219_MP7659_10_S10_R1_001.fastq.gz\":\"mp804-MV411_IRF2BP_VHL_6h-MED1-r2_R1.fastq.gz\",\n",
    "\"20191219_MP7659_10_S10_R2_001.fastq.gz\":\"mp804-MV411_IRF2BP_VHL_6h-MED1-r2_R2.fastq.gz\",\n",
    "\"20191219_MP7659_11_S11_R1_001.fastq.gz\":\"mp805-MV411_IRF2BP_VHL_6h-POLII_total-r2_R1.fastq.gz\",\n",
    "\"20191219_MP7659_11_S11_R2_001.fastq.gz\":\"mp805-MV411_IRF2BP_VHL_6h-POLII_total-r2_R2.fastq.gz\",\n",
    "\"20191219_MP7659_12_S12_R1_001.fastq.gz\":\"mp806-MV411_IRF2BP_VHL_6h-POLII_S2-r2_R1.fastq.gz\",\n",
    "\"20191219_MP7659_12_S12_R2_001.fastq.gz\":\"mp806-MV411_IRF2BP_VHL_6h-POLII_S2-r2_R2.fastq.gz\",\n",
    "\"20191219_MP7659_13_S13_R1_001.fastq.gz\":\"mp807-MV411_IRF2BP_VHL_6h-POLII_S5-r2_R1.fastq.gz\",\n",
    "\"20191219_MP7659_13_S13_R2_001.fastq.gz\":\"mp807-MV411_IRF2BP_VHL_6h-POLII_S5-r2_R2.fastq.gz\",\n",
    "\"20191219_MP7659_14_S14_R1_001.fastq.gz\":\"mp808-MV411_IRF2BP_VHL_6h-MYC-r2_R1.fastq.gz\",\n",
    "\"20191219_MP7659_14_S14_R2_001.fastq.gz\":\"mp808-MV411_IRF2BP_VHL_6h-MYC-r2_R2.fastq.gz\",\n",
    "\"20191219_MP7659_15_S15_R1_001.fastq.gz\":\"mp809-MV411_IRF2BP_VHL_6h-MYB-r2_R1.fastq.gz\",\n",
    "\"20191219_MP7659_15_S15_R2_001.fastq.gz\":\"mp809-MV411_IRF2BP_VHL_6h-MYB-r2_R2.fastq.gz\",\n",
    "\"20191219_MP7659_16_S16_R1_001.fastq.gz\":\"mp810-MV411_IRF2BP_VHL_6h-SPI1-r2_R1.fastq.gz\",\n",
    "\"20191219_MP7659_16_S16_R2_001.fastq.gz\":\"mp810-MV411_IRF2BP_VHL_6h-SPI1-r2_R2.fastq.gz\"\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for k,v in rename.items():\n",
    "    ! mv ../../data/IRF2BP2_degraded_rep2/fastqs/$k ../../data/IRF2BP2_degraded_rep2/fastqs/$v\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "inputfastq=\"gs://amlproject/Chip/fastqs/mp99-MV411-INPUT-r1.fastq.gz\"\n",
    "! gsutil cp $inputfastq ../../data/IRF2BP2_degraded_rep2/fastqs\n",
    "a = ! ls ../../data/IRF2BP2_degraded_rep2/fastqs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from gsheets import Sheets\n",
    "sheets = Sheets.from_files('~/.client_secret.json', '~/.storage.json')\n",
    "url=\"https://docs.google.com/spreadsheets/d/1yFLjYB1McU530JnLgL0QIMAKIkVl3kl0_LCHje2gk8U\"\n",
    "gsheet = sheets.get(url).sheets[2].to_frame()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = {\n",
    "\"fastq1\": [],\n",
    "\"fastq2\": [],\n",
    "\"antibody\": [],\n",
    "\"group\": [],\n",
    "\"replicate\": [],\n",
    "\"control\": []\n",
    "}\n",
    "for val in helper.grouped(a[:-1],2):\n",
    "    row = gsheet[gsheet.id==val[0].split('/')[-1].split('_')[0]]\n",
    "    df['group'].append((row.id.values[0] + '_'+ row.name_replicate.values[0] +\"_\"+row.protein.values[0]))\n",
    "    df['replicate'].append(1)\n",
    "    df['fastq1'].append(val[0])\n",
    "    #df['fastq2'].append(val[1])\n",
    "    df['antibody'].append(row.protein.values[0])\n",
    "    df['control'].append(\"INPUT\")\n",
    "df['group'].append('INPUT')\n",
    "df['replicate'].append(1)\n",
    "df['fastq1'].append(a[-1])\n",
    "#df['fastq2'].append(val[1])\n",
    "df['antibody'].append(\"INPUT\")\n",
    "df['control'].append(\"INPUT\")\n",
    "df = pd.DataFrame(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df[df.columns[[3,4,0,1,2,5]]].to_csv('../nextflow/IRF2BP2_degraded_rep2_design.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#process chips\n",
    "! sudo ./nextflow run nf-core/chipseq --single_end --seq_center 'DFCI' --email 'jkobject@gmail.com' --narrow_peak --input ../nextflow/IRF2BP2_degraded_rep2_design.csv --genome GRCh38 --skip_preseq --max_cpus 24 -profile docker -w work"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#!gsutil cp results/* ../../data/IRF2BP2_degraded_rep2/ && sudo rm -r work && sudo rm -r results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# get scaling values\n",
    "mappedreads, umappedreads_norm, mapped = chip.getSpikeInControlScales(refgenome=\"../../ref/reference_droso.fna\",FastQfolder='../../data/IRF2BP2_degraded_rep2/fastqs/',pairedEnd=True, cores=8)\n",
    "mappedreads, umappedreads_norm, mapped"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "computing scales from the excel sheet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [[508878,637972],\n",
    "[1929129,11595],\n",
    "[669536,429562],\n",
    "[1272730,810802],\n",
    "[743859,893304],\n",
    "[312888,1154119],\n",
    "[1086031,880901],\n",
    "[850181,1019640]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [[1.0, 0.7976494266],\n",
    "[0.006010484524, 1.0],\n",
    "[0.6415816327, 1.0],\n",
    "[0.6370573492, 1.0],\n",
    "[1.0, 0.8327053276],\n",
    "[1.0, 0.2711054926],\n",
    "[0.8111195721, 1.0],\n",
    "[1.0, 0.8338050685]]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### on scalled data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# on scaled data\n",
    "bams = ! ls ../../IRF2BP2_degraded_rep2/bwa/mergedLibrary/*.bam\n",
    "for i in range(int(len(bams[1:])/2)-1):\n",
    "    name1 = bams[1+val]\n",
    "    name2 = bams[9+val]\n",
    "    chip.fullDiffPeak(name1,name2,control1='../../data/IRF2BP2_degraded_rep2/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam', directory = \"../../data/IRF2BP2_degraded_rep2/diffData_unscaled/\", res_directory = \"../../data/IRF2BP2_degraded_rep2/diffPeaks_unscaled/\", scaling = scales[val][::-1])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bams = ! ls ../../data/IRF2BP2_degraded_rep2/bwa/mergedLibrary/*.bam\n",
    "bams = [i.split('/')[-1].split('.')[0] for i in bams]\n",
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# diffPeak on scalled data\n",
    "sizes = [220, 191, 228, 285, 222, 204, 211, 194]\n",
    "for i in range(int(len(bams[1:])/2)-1):\n",
    "    name1 = bams[1+i]\n",
    "    name2 = bams[9+i]\n",
    "    chip.diffPeak(name1,name2, directory= \"../../data/IRF2BP2_degraded_rep2/diffData/\", res_directory='../../data/IRF2BP2_degraded_rep2/diffPeaks/', scaling1=scales[i][1], scaling2=scales[i][0], size=size[i])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [1.0,\n",
    "0.006010484524,\n",
    "0.6415816327,\n",
    "0.6370573492,\n",
    "1.0,\n",
    "1.0,\n",
    "0.8111195721,\n",
    "1.0,\n",
    "0.7976494266,\n",
    "1.0,\n",
    "1.0,\n",
    "1.0,\n",
    "0.8327053276,\n",
    "0.2711054926,\n",
    "1.0,\n",
    "0.8338050685]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "chip.bigWigFrom(bams[1:],genome='GRCh38',scaling=scales)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mv ../../data/recalib_bigwig_rep2/* ../../data/IRF2BP2_degraded_rep2/recalib_bigwig/\n",
    "bw = ! ls ../../data/IRF2BP2_degraded_rep2/recalib_bigwig/*.bw\n",
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "cond1peak = ! ls ../../data/IRF2BP2_degraded_rep2/diffPeaks/*cond1.bed\n",
    "cond2peak = ! ls ../../data/IRF2BP2_degraded_rep2/diffPeaks/*cond2.bed\n",
    "commonpeak = ! ls ../../data/IRF2BP2_degraded_rep2/diffPeaks/*common.bed\n",
    "commonpeak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"FLAG_IRF2BP2\",\"MED1\",\"POLII_total\",\"POLII_S2\",\"POLII_S5\",\"MYC\",\"MYB\",\"SPI1\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for i in range(int(len(bw)/2)):\n",
    "    name1 = bw[i]\n",
    "    name2 = bw[8+i]\n",
    "    peak = [cond1peak[i], commonpeak[i], cond2peak[i]]\n",
    "    for val in peak:\n",
    "        chip.dropWeirdChromosomes(val)\n",
    "    name = names[i]\n",
    "    chip.getPeaksAt(peak, [name1, name2], bigwignames=['DMSO', 'VHL'], peaknames=['DMSO_peaks', 'common', 'VHL_peaks'], window=3000, folder=\"\", title=name, numthreads=20, name='../../data/IRF2BP2_degraded_rep2/diffPeaks/'+name+'_mat.pdf', refpoint='center', withDeeptools=True, torecompute= True,)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "peaks = ! ls ../../data/IRF2BP2_degraded_rep2/bwa/mergedLibrary/macs/narrowPeak/*.narrowPeak\n",
    "peaks"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"FLAG_IRF2BP2\",\"MED1\",\"POLII_total\",\"POLII_S2\",\"POLII_S5\",\"MYC\",\"MYB\",\"SPI1\"]\n",
    "names.extend([i+'_VHL' for i in names])\n",
    "names"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for i in range(len(bw)-1):\n",
    "    chip.getPeaksAt(peaks[i], bw[1+i], window=3000, folder=\"\", title=names[i], numthreads=7, torecompute=True, onlyProfile=True, name='../../data/IRF2BP2_degraded_rep2/diffPeaks/'+names[i]+'_mat_profile.pdf', refpoint='center', withDeeptools=True)\n",
    "    chip.getPeaksAt(peaks[i], bw[1+i], window=3000, folder=\"\", title=names[i], numthreads=7, torecompute=True, refpoint='center', onlyProfile=True, name='../../data/IRF2BP2_degraded_rep2/diffPeaks/'+names[i]+'_mat_profile_clust3.pdf', withDeeptools=True, cluster=3)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### unscalled"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bams = !ls ../../data/IRF2BP2_degraded_rep2/bwa/mergedLibrary/mp*.bam\n",
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!mkdir  ../../data/IRF2BP2_degraded_rep2/diffPeaks_unscaled\n",
    "bw = ! ../../data/IRF2BP2_degraded_rep2/bwa/mergedLibrary/bigwig/*.bigWig"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# on unscalled data \n",
    "for i in range(int(len(bams)/2)):\n",
    "    if i <0:\n",
    "        continue\n",
    "    name1 = bams[i]\n",
    "    name2 = bams[8+i]\n",
    "    chip.fullDiffPeak(name1,name2, control1='../../data/results3/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam', directory = \"../../data/IRF2BP2_degraded_rep2/diffData_unscaled/\", res_directory = \"../../data/IRF2BP2_degraded_rep2/diffPeaks_unscaled/\", pairedend=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "commonpeak = ! ls ../../data/IRF2BP2_degraded_rep2/diffPeaks_unscaled/*common.bed\n",
    "cond1peak = ! ls ../../data/IRF2BP2_degraded_rep2/diffPeaks_unscaled/*cond1.bed\n",
    "cond2peak = ! ls ../../data/IRF2BP2_degraded_rep2/diffPeaks_unscaled/*cond2.bed\n",
    "cond2peak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"FLAG_IRF2BP2\",\"MED1\",\"POLII_total\",\"POLII_S2\",\"POLII_S5\",\"MYC\",\"MYB\",\"SPI1\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for i in range(int(len(bw[1:])/2)):\n",
    "    name1 = bw[1+i]\n",
    "    name2 = bw[9+i]\n",
    "    peak = [cond1peak[i], commonpeak[i], cond2peak[i]]\n",
    "    for val in peak:\n",
    "        chip.dropWeirdChromosomes(val)\n",
    "    name = names[i]\n",
    "    chip.getPeaksAt(peak, [name1, name2], bigwignames=['DMSO', 'VHL'], peaknames=['DMSO_peaks', 'common', 'VHL_peaks'], window=3000, title=name, numthreads=7, torecompute=True, refpoint='center', folder=\"\", name='../../data/IRF2BP2_degraded_rep2/diffPeaks_unscaled/'+name+'_mat.pdf', withDeeptools=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! gsutil -m cp -r \"../../data/IRF2BP2_degraded_rep2\" gs://amlproject/Chip/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## v3"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! gsutil mv gs://transfer-amlproject/*MP7781*  gs://transfer-amlproject/IRF2BP2_v3/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mkdir ../../data/IRF2BP2_degraded_rep3 && mkdir ../../data/IRF2BP2_degraded_rep3/fastqs && gsutil -m cp gs://transfer-amlproject/IRF2BP2_v3/* ../../data/IRF2BP2_degraded_rep3/fastqs/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "rename = {\n",
    "\"20200203_1_MP7781_S67_R1_001.fastq.gz\":\"mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1.fastq.gz\",\n",
    "\"20200203_1_MP7781_S67_R2_001.fastq.gz\":\"mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R2.fastq.gz\",\n",
    "\"20200203_2_MP7781_S68_R1_001.fastq.gz\":\"mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1.fastq.gz\",\n",
    "\"20200203_2_MP7781_S68_R2_001.fastq.gz\":\"mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R2.fastq.gz\",\n",
    "\"20200203_3_MP7781_S69_R1_001.fastq.gz\":\"mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1.fastq.gz\",\n",
    "\"20200203_3_MP7781_S69_R2_001.fastq.gz\":\"mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R2.fastq.gz\",\n",
    "\"20200203_4_MP7781_S70_R1_001.fastq.gz\":\"mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1.fastq.gz\",\n",
    "\"20200203_4_MP7781_S70_R2_001.fastq.gz\":\"mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R2.fastq.gz\",\n",
    "\"20200203_5_MP7781_S71_R2_001.fastq.gz\":\"mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R2.fastq.gz\",\n",
    "\"20200203_5_MP7781_S71_R1_001.fastq.gz\":\"mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1.fastq.gz\",\n",
    "\"20200203_6_MP7781_S72_R1_001.fastq.gz\":\"mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1.fastq.gz\",\n",
    "\"20200203_6_MP7781_S72_R2_001.fastq.gz\":\"mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R2.fastq.gz\",\n",
    "\"20200203_7_MP7781_S73_R1_001.fastq.gz\":\"mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1.fastq.gz\",\n",
    "\"20200203_7_MP7781_S73_R2_001.fastq.gz\":\"mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R2.fastq.gz\",\n",
    "\"20200203_8_MP7781_S74_R1_001.fastq.gz\":\"mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1.fastq.gz\",\n",
    "\"20200203_8_MP7781_S74_R2_001.fastq.gz\":\"mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R2.fastq.gz\",\n",
    "\"20200203_9_MP7781_S75_R1_001.fastq.gz\":\"mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1.fastq.gz\",\n",
    "\"20200203_9_MP7781_S75_R2_001.fastq.gz\":\"mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R2.fastq.gz\",\n",
    "\"20200203_10_MP7781_S76_R2_001.fastq.gz\":\"mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R2.fastq.gz\",\n",
    "\"20200203_10_MP7781_S76_R1_001.fastq.gz\":\"mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1.fastq.gz\",\n",
    "\"20200203_11_MP7781_S77_R1_001.fastq.gz\":\"mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1.fastq.gz\",\n",
    "\"20200203_11_MP7781_S77_R2_001.fastq.gz\":\"mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R2.fastq.gz\",\n",
    "\"20200203_12_MP7781_S78_R1_001.fastq.gz\":\"mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1.fastq.gz\",\n",
    "\"20200203_12_MP7781_S78_R2_001.fastq.gz\":\"mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R2.fastq.gz\",\n",
    "\"20200203_13_MP7781_S79_R1_001.fastq.gz\":\"mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1.fastq.gz\",\n",
    "\"20200203_13_MP7781_S79_R2_001.fastq.gz\":\"mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R2.fastq.gz\",\n",
    "\"20200203_14_MP7781_S80_R2_001.fastq.gz\":\"mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R2.fastq.gz\",\n",
    "\"20200203_14_MP7781_S80_R1_001.fastq.gz\":\"mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1.fastq.gz\",\n",
    "\"20200203_Input_MP7781_S81_R1_001.fastq.gz\":\"mp845-MV411_IRF2BP2_-INPUT-r1_R1.fastq.gz\",\n",
    "\"20200203_Input_MP7781_S81_R2_001.fastq.gz\":\"mp845-MV411_IRF2BP2_-INPUT-r1_R2.fastq.gz\"\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for k,v in rename.items():\n",
    "    ! mv ../../data/IRF2BP2_degraded_rep3/fastqs/$k ../../data/IRF2BP2_degraded_rep3/fastqs/$v"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from gsheets import Sheets\n",
    "sheets = Sheets.from_files('~/.client_secret.json', '~/.storage.json')\n",
    "url=\"https://docs.google.com/spreadsheets/d/1yFLjYB1McU530JnLgL0QIMAKIkVl3kl0_LCHje2gk8U\"\n",
    "gsheet = sheets.get(url).sheets[2].to_frame()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "a = ! ls ../../data/IRF2BP2_degraded_rep3/fastqs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "gsheet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = {\n",
    "\"fastq_1\": [],\n",
    "\"fastq_2\": [],\n",
    "\"antibody\": [],\n",
    "\"group\": [],\n",
    "\"replicate\": [],\n",
    "\"control\": []\n",
    "}\n",
    "for val in helper.grouped(a[:-2],2):\n",
    "    row = gsheet[gsheet.id==val[0].split('/')[-1].split('-')[0]]\n",
    "    df['group'].append(row['name'].values[0])\n",
    "    df['replicate'].append(1)\n",
    "    df['fastq_1'].append(val[0])\n",
    "    df['fastq_2'].append(val[1])\n",
    "    df['antibody'].append(row['protein'].values[0])\n",
    "    df['control'].append(\"INPUT\")\n",
    "df['group'].append('INPUT')\n",
    "df['replicate'].append(1)\n",
    "df['fastq_1'].append(a[-2])\n",
    "df['fastq_2'].append(a[-1])\n",
    "df['antibody'].append(\"\")\n",
    "df['control'].append(\"\")\n",
    "df = pd.DataFrame(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df[df.columns[[3,4,0,1,2,5]]].to_csv('../nextflow/IRF2BP2_degraded_rep3_design.csv',index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#process chips\n",
    "! sudo ../nextflow run nf-core/chipseq --paired_end --seq_center 'DFCI' --email \\\n",
    "'jkobject@gmail.com' --narrow_peak --input ../nextflow/../../data/IRF2BP2_degraded_rep3_design.csv --genome GRCh38 --skip_preseq \\\n",
    "--max_cpus 24 -profile docker -w work"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "ls"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!cp -r results/* ../../data/IRF2BP2_degraded_rep3/ && sudo rm -r work && sudo rm -r results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "ls ../TrimGalore-0.6.5/trim_galore"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# get scaling values\n",
    "mappedreads, umappedreads_norm = chip.getSpikeInControlScales(refgenome=\"../../ref/reference_droso.fna\",\n",
    "                                                                      fastQfolder='../../data/IRF2BP2_degraded_rep3/fastqs',\n",
    "                                                                      pairedEnd=True, cores=8,\n",
    "                                                                      tofilter=False,\n",
    "                                                                      totrim=False,\n",
    "                                                                      tomap=False,\n",
    "                                                                      pathtotrim_galore=\"../TrimGalore-0.6.5/trim_galore\")\n",
    "mappedreads, umappedreads_norm"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "computing scales from the excel sheet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [[536923,632558],\n",
    "[601370,681405],\n",
    "[2402198,1676203],\n",
    "[417892,216192],\n",
    "[1544590,1350802],\n",
    "[1174994,1376726],\n",
    "[289635,240366]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales= [[1.0, 0.8488122828],\n",
    "[1.0, 0.8825441551],\n",
    "[0.6977788675, 1.0],\n",
    "[0.5173394083, 1.0],\n",
    "[0.8745375796, 1.0],\n",
    "[1.0, 0.8534697536],\n",
    "[0.8298927961, 1.0]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bams = ! ls ../../data/IRF2BP2_degraded_rep3/bwa/mergedLibrary/*.bam #../../data/results3/bwa/mergedLibrary/*.bam\n",
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bams[7]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bams = [bam1.split('/')[-1].split('.')[0] for bam1 in bams]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### on scalled data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# diffPeak on scaled data\n",
    "size=[206, 218, 189, 194, 217, 217, 176]\n",
    "for i in range(int(len(bams)/2)):\n",
    "    if i<0:\n",
    "        continue\n",
    "    name1 = bams[i]\n",
    "    name2 = bams[7+i]\n",
    "    print(name1,name2)\n",
    "    chip.diffPeak(name1, name2, directory= \"../../data/IRF2BP2_degraded_rep3/diffData/\", res_directory='../../data/IRF2BP2_degraded_rep3/diffPeaks/', scaling1=scales[i][1], scaling2=scales[i][0], size=size[i])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "os.popen('for i in $(ls ../../data/IRF2BP2_degraded_rep3/diffPeaks/*.bed); \\\n",
    "            do echo $(wc -l $i); \\\n",
    "            done').read().split('\\n')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# diffPeak on scaled data\n",
    "for i in range(int(len(bams)/2)):\n",
    "    if i <0:\n",
    "        continue\n",
    "    name1 = bams[i]\n",
    "    name2 = bams[7+i]\n",
    "    print(name1,name2)\n",
    "    chip.fullDiffPeak(name1,name2, control1='../../data/IRF2BP2_degraded_rep3/INPUT_R1.mLb.clN.sorted.bam', directory = \"../../data/IRF2BP2_degraded_rep3/diffData/\", res_directory = \"../../data/IRF2BP2_degraded_rep3/diffPeaks/\", isTF=True, compute_size=True, pairedend=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [1.0,\n",
    "1.0,\n",
    "0.6977788675,\n",
    "0.5173394083,\n",
    "0.8745375796,\n",
    "1.0,\n",
    "0.8298927961,\n",
    "0.8488122828,\n",
    "0.8825441551,\n",
    "1.0,\n",
    "1.0,\n",
    "1.0,\n",
    "0.8534697536,\n",
    "1.0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "chip.bigWigFrom(bams[1:], \n",
    "                genome='GRCh38',scaling=scales,\n",
    "               numthreads=8)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mv diffPeaks ../../data/IRF2BP2_degraded_rep3\n",
    "! mv diffData ../../data/IRF2BP2_degraded_rep3"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!mv bigwig ../../data/recalib_bigwig_3 "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "os.popen('for i in $(ls ../../data/IRF2BP2_degraded_rep3/diffPeaks); \\\n",
    "            do echo $(wc -l \"../../data/IRF2BP2_degraded_rep3/diffPeaks/\"$i); \\\n",
    "            done').read().split('\\n')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bw = ! ls ../../data/recalib_bigwig_3/*.bw\n",
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "cond1peak = ! ls ../../data/IRF2BP2_degraded_rep3/diffPeaks/*cond1.bed\n",
    "cond2peak = ! ls ../../data/IRF2BP2_degraded_rep3/diffPeaks/*cond2.bed\n",
    "commonpeak = ! ls ../../data/IRF2BP2_degraded_rep3/diffPeaks/*common.bed\n",
    "cond1peak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"CDK8\",\"BRD4\",\"IRF8\",\"SMC1\",\"MED1\",\"ZEB2\",\"CEBPA\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for i in range(int(len(bw)/2)):\n",
    "    if i<0:\n",
    "        continue\n",
    "    name1 = bw[i]\n",
    "    name2 = bw[7+i]\n",
    "    peak = [cond1peak[i], commonpeak[i], cond2peak[i]]\n",
    "    #for val in peak:\n",
    "        #chip.dropWeirdChromosomes(val)\n",
    "    name = names[i]\n",
    "    chip.getPeaksAt(peak, [name1, name2], bigwignames=['DMSO', 'VHL'], refpoint='center', peaknames=['DMSO_peaks', 'common', 'VHL_peaks'], window=3000, folder=\"\", title=name, numthreads=14, name='../../data/IRF2BP2_degraded_rep3/diffPeaks/'+name+'_mat.pdf', withDeeptools=True, torecompute=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"CDK8\",\"BRD4\",\"IRF8\",\"SMC1\",\"MED1\",\"ZEB2\",\"CEBPA\"]\n",
    "names.extend([i+'_VHL' for i in names])\n",
    "names"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "peaks = ! ls ../../data/IRF2BP2_degraded_rep3/bwa/mergedLibrary/macs/narrowPeak/*.narrowPeak\n",
    "peaks"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for i in range(len(bw)):\n",
    "    chip.getPeaksAt(peaks[i], bw[i], window=3000, folder=\"\", title=names[i], numthreads=7, torecompute=True, onlyProfile=True, refpoint='center', name='../../data/IRF2BP2_degraded_rep3/diffPeaks/'+names[i]+'_mat_profile.pdf', withDeeptools=True)\n",
    "    chip.getPeaksAt(peaks[i], bw[i], window=3000, folder=\"\", title=names[i], numthreads=7, torecompute=True, refpoint='center', onlyProfile=True,name='../../data/IRF2BP2_degraded_rep3/diffPeaks/'+names[i]+'_mat_profile_clust3.pdf', withDeeptools=True, cluster=3)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### on unscalled data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bams = !ls ../../data/IRF2BP2_degraded_rep3/bwa/mergedLibrary/mp*.bam\n",
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mkdir ../../data/IRF2BP2_degraded_rep3/diffPeaks_unscaled"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#on unscalled data \n",
    "for i in range(int(len(bams)/2)):\n",
    "    if i < 0:\n",
    "        continue\n",
    "    name1 = bams[i]\n",
    "    name2 = bams[7+i]\n",
    "    print(name1,name2)\n",
    "    chip.fullDiffPeak(name1,name2, control1='../../data/results3/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam', directory = \"../../data/IRF2BP2_degraded_rep3/diffData_unscaled/\", res_directory = \"../../data/IRF2BP2_degraded_rep3/diffPeaks_unscaled/\",pairedend=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bw = ! ls ../../data/IRF2BP2_degraded_rep3/bwa/mergedLibrary/bigwig/*.bigWig\n",
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "cond1peak = ! ls ../../data/IRF2BP2_degraded_rep3/diffPeaks_unscaled/*cond1.bed\n",
    "cond2peak = ! ls ../../data/IRF2BP2_degraded_rep3/diffPeaks_unscaled/*cond2.bed\n",
    "commonpeak = ! ls ../../data/IRF2BP2_degraded_rep3/diffPeaks_unscaled/*common.bed\n",
    "cond1peak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"CDK8\",\"BRD4\",\"IRF8\",\"SMC1\",\"MED1\",\"ZEB2\",\"CEBPA\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for i in range(int((len(bw)-1)/2)):\n",
    "    if i <0:\n",
    "        continue\n",
    "    name1 = bw[1+i]\n",
    "    name2 = bw[8+i]\n",
    "    peak = [cond1peak[i], commonpeak[i], cond2peak[i]]\n",
    "    #for val in peak:\n",
    "        #chip.dropWeirdChromosomes(val)\n",
    "    name = names[i]\n",
    "    print(name1,name2)\n",
    "    chip.getPeaksAt(peak, [name1, name2], bigwignames=['DMSO', 'VHL'], peaknames=['DMSO_peaks', 'common', 'VHL_peaks'], window=3000, folder=\"\", title=name, numthreads=14, torecompute=True, refpoint=\"center\", name='../../data/IRF2BP2_degraded_rep3/diffPeaks_unscaled/'+name+'_mat.pdf', withDeeptools=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! gsutil -m cp -r \"../../data/IRF2BP2_degraded_rep3/\" gs://amlproject/Chip/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## histones V1&2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bamfolder=\"../../data/IRF2BP2_degraded_histones/fastqs/\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mkdir ../../data/IRF2BP2_degraded_histones && mkdir ../../data/IRF2BP2_degraded_histones/fastqs && gsutil cp \"gs://transfer-amlproject/*.fastq.gz\" $bamfolder"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! gsutil -m cp gs://transfer-amlproject/*MP7693* gs://transfer-amlproject/IRF2BP2_hist/ && gsutil -m rm gs://transfer-amlproject/*MP7693*"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "\n",
    "### analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "rename = {\n",
    "    \"20200103_1_MP7693_S6_R1_001.fastq.gz\":\"mp811-MV411_IRF2BP2_DMSO_6h-H3K27ac-r1_R1.fastq.gz\",\n",
    "    \"20200103_1_MP7693_S6_R2_001.fastq.gz\":\"mp811-MV411_IRF2BP2_DMSO_6h-H3K27ac-r1.fastq.gz\",\n",
    "    \"20200103_2_MP7693_S7_R1_001.fastq.gz\":\"mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2_R1.fastq.gz\",\n",
    "    \"20200103_2_MP7693_S7_R2_001.fastq.gz\":\"mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2.fastq.gz\",\n",
    "    \"20200103_3_MP7693_S8_R1_001.fastq.gz\":\"mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1_R1.fastq.gz\",\n",
    "    \"20200103_3_MP7693_S8_R2_001.fastq.gz\":\"mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1.fastq.gz\",\n",
    "    \"20200103_4_MP7693_S9_R1_001.fastq.gz\":\"mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2_R1.fastq.gz\",\n",
    "    \"20200103_4_MP7693_S9_R2_001.fastq.gz\":\"mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2.fastq.gz\",\n",
    "    \"20200103_5_MP7693_S10_R1_001.fastq.gz\":\"mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1_R1.fastq.gz\",\n",
    "    \"20200103_5_MP7693_S10_R2_001.fastq.gz\":\"mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1.fastq.gz\",\n",
    "    \"20200103_6_MP7693_S11_R1_001.fastq.gz\":\"mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2_R1.fastq.gz\",\n",
    "    \"20200103_6_MP7693_S11_R2_001.fastq.gz\":\"mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2.fastq.gz\",\n",
    "    \"20200103_7_MP7693_S12_R1_001.fastq.gz\":\"mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1_R1.fastq.gz\",\n",
    "    \"20200103_7_MP7693_S12_R2_001.fastq.gz\":\"mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1.fastq.gz\",\n",
    "    \"20200103_8_MP7693_S13_R1_001.fastq.gz\":\"mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2_R1.fastq.gz\",\n",
    "    \"20200103_8_MP7693_S13_R2_001.fastq.gz\":\"mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2.fastq.gz\",\n",
    "    \"20200103_9_MP7693_S14_R1_001.fastq.gz\":\"mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1_R1.fastq.gz\",\n",
    "    \"20200103_9_MP7693_S14_R2_001.fastq.gz\":\"mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1.fastq.gz\",\n",
    "    \"20200103_10_MP7693_S15_R1_001.fastq.gz\":\"mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2_R1.fastq.gz\",\n",
    "    \"20200103_10_MP7693_S15_R2_001.fastq.gz\":\"mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2.fastq.gz\",\n",
    "    \"20200103_11_MP7693_S16_R1_001.fastq.gz\":\"mp821-MV411_IRF2BP2_VHL_6h-H3K27ac-r1_R1.fastq.gz\",\n",
    "    \"20200103_11_MP7693_S16_R2_001.fastq.gz\":\"mp821-MV411_IRF2BP2_VHL_6h-H3K27ac-r1.fastq.gz\",\n",
    "    \"20200103_12_MP7693_S17_R1_001.fastq.gz\":\"mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2_R1.fastq.gz\",\n",
    "    \"20200103_12_MP7693_S17_R2_001.fastq.gz\":\"mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2.fastq.gz\",\n",
    "    \"20200103_13_MP7693_S18_R1_001.fastq.gz\":\"mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1_R1.fastq.gz\",\n",
    "    \"20200103_13_MP7693_S18_R2_001.fastq.gz\":\"mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1.fastq.gz\",\n",
    "    \"20200103_14_MP7693_S19_R1_001.fastq.gz\":\"mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2_R1.fastq.gz\",\n",
    "    \"20200103_14_MP7693_S19_R2_001.fastq.gz\":\"mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2.fastq.gz\",\n",
    "    \"20200103_15_MP7693_S20_R1_001.fastq.gz\":\"mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1_R1.fastq.gz\",\n",
    "    \"20200103_15_MP7693_S20_R2_001.fastq.gz\":\"mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1.fastq.gz\",\n",
    "    \"20200103_16_MP7693_S21_R1_001.fastq.gz\":\"mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2_R1.fastq.gz\",\n",
    "    \"20200103_16_MP7693_S21_R2_001.fastq.gz\":\"mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2.fastq.gz\",\n",
    "    \"20200103_17_MP7693_S22_R1_001.fastq.gz\":\"mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1_R1.fastq.gz\",\n",
    "    \"20200103_17_MP7693_S22_R2_001.fastq.gz\":\"mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1.fastq.gz\",\n",
    "    \"20200103_18_MP7693_S23_R1_001.fastq.gz\":\"mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2_R1.fastq.gz\",\n",
    "    \"20200103_18_MP7693_S23_R2_001.fastq.gz\":\"mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2.fastq.gz\",\n",
    "    \"20200103_19_MP7693_S24_R1_001.fastq.gz\":\"mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1_R1.fastq.gz\",\n",
    "    \"20200103_19_MP7693_S24_R2_001.fastq.gz\":\"mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1.fastq.gz\",\n",
    "    \"20200103_20_MP7693_S25_R1_001.fastq.gz\":\"mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2_R1.fastq.gz\",\n",
    "    \"20200103_20_MP7693_S25_R2_001.fastq.gz\":\"mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2.fastq.gz\"\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! gsutil -m cp gs://transfer-amlproject/IRF2BP2_hist/* ../../data/IRF2BP2_degraded_histones/fastqs/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for k,v in rename.items():\n",
    "    ! mv $bamfolder$k $bamfolder$v"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "inputfastq=\"gs://amlproject/Chip/fastqs/mp99-MV411-INPUT-r1.fastq.gz\"\n",
    "! gsutil cp $inputfastq $bamfolder"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "a = ! ls $bamfolder"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from gsheets import Sheets\n",
    "sheets = Sheets.from_files('~/.client_secret.json', '~/.storage.json')\n",
    "url=\"https://docs.google.com/spreadsheets/d/1yFLjYB1McU530JnLgL0QIMAKIkVl3kl0_LCHje2gk8U\"\n",
    "gsheet = sheets.get(url).sheets[2].to_frame()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = {\n",
    "\"fastq1\": [],\n",
    "\"fastq2\": [],\n",
    "\"antibody\": [],\n",
    "\"group\": [],\n",
    "\"replicate\": [],\n",
    "\"control\": []\n",
    "}\n",
    "for val in helper.grouped(a[:-1],2):\n",
    "    row = gsheet[gsheet.id==val[0].split('/')[-1].split('_')[0]]\n",
    "    df['group'].append((row.id.values[0] + '_'+ row.name_replicate.values[0] +\"_\"+row.protein.values[0]))\n",
    "    df['replicate'].append(1)\n",
    "    df['fastq1'].append(val[0])\n",
    "    #df['fastq2'].append(val[1])\n",
    "    df['antibody'].append(row.protein.values[0])\n",
    "    df['control'].append(\"INPUT\")\n",
    "df['group'].append('INPUT')\n",
    "df['replicate'].append(1)\n",
    "df['fastq1'].append(a[-1])\n",
    "#df['fastq2'].append(val[1])\n",
    "df['antibody'].append(\"INPUT\")\n",
    "df['control'].append(\"INPUT\")\n",
    "df = pd.DataFrame(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df[df.columns[[3,4,0,1,2,5]]].to_csv('../nextflow/IRF2BP2_degraded_histones_design.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#process chips\n",
    " ! sudo ./nextflow run nf-core/chipseq --single_end --seq_center 'DFCI' --email 'jkobject@gmail.com' --narrow_peak --input ../nextflow/IRF2BP2_degraded_histones_design.csv --genome GRCh38 --skip_preseq --max_cpus 24 -profile docker -w work"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mv results/* ../../data/IRF2BP2_degraded_histones/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bams = ! ls ../../data/IRF2BP2_degraded_hist/bwa/mergedLibrary/*.bam\n",
    "bams = [i.split('/')[-1].split('.')[0] for i in bams]\n",
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [[191079,278272],\n",
    "[274625,494562],\n",
    "[1094016,2067804],\n",
    "[1328914,969565],\n",
    "[237779,132422],\n",
    "[162971,174092],\n",
    "[115788,187078],\n",
    "[134269,495924],\n",
    "[171890,225315],\n",
    "[188370,199911]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [[1.0,0.6866626897],\n",
    "[1.0,0.5552893267],\n",
    "[1.0,0.5290714207],\n",
    "[0.7295919826,1.0],\n",
    "[0.5569120906,1.0],\n",
    "[1.0,0.9361199825],\n",
    "[1.0,0.6189290029],\n",
    "[1.0,0.2707451142],\n",
    "[1.0,0.762887513],\n",
    "[1.0,0.9422693098]]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### on scalled data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# diffPeak on scaled data\n",
    "size=[206, 213, 47, 272, 229, 235, 190, 196, 287, 288]\n",
    "for i in range(int(len(bams)/2)):\n",
    "    name1 = bams[1+i]\n",
    "    name2 = bams[11+i]\n",
    "    print(name1,name2)\n",
    "    chip.diffPeak(name1, name2, directory= \"../../data/IRF2BP2_degraded_hist/diffData/\", res_directory='../../data/IRF2BP2_degraded_hist/diffPeaks/', scaling1=scales[i][1], scaling2=scales[i][0], size=size[i])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# diffPeak on scaled data\n",
    "for i in range(int(len(bams)/2)):\n",
    "    name1 = bams[1+i]\n",
    "    name2 = bams[11+i]\n",
    "    print(name1,name2)\n",
    "    chip.fullDiffPeak(name1,name2, bams[0], scaling = scales[i], directory='../../data/IRF2BP2_degraded_hist/diffData/',\n",
    "res_directory=\"../../data/IRF2BP2_degraded_hist/diffPeaks/\", isTF=False, compute_size=True, pairedend=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [1.0,\n",
    "1.0,\n",
    "1.0,\n",
    "0.7295919826,\n",
    "0.5569120906,\n",
    "1.0,\n",
    "1.0,\n",
    "1.0,\n",
    "1.0,\n",
    "1.0,\n",
    "0.6866626897,\n",
    "0.5552893267,\n",
    "0.5290714207,\n",
    "1.0,\n",
    "1.0,\n",
    "0.9361199825,\n",
    "0.6189290029,\n",
    "0.2707451142,\n",
    "0.762887513,\n",
    "0.9422693098,]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "chip.bigWigFrom(bams[1:],genome='GRCh38',scaling=scales)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mv ../../recalib_bigwig_hist/* ../../data/IRF2BP2_degraded_histones/recalib_bigwig/\n",
    "bw = ! ls ../../data/IRF2BP2_degraded_histones/recalib_bigwig/*.bw\n",
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! rm ../../data/IRF2BP2_degraded_hist/diffData/mp*_R1*"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "cond1peak = ! ls ../../data/IRF2BP2_degraded_hist/diffPeaks/*cond1.bed\n",
    "cond2peak = ! ls ../../data/IRF2BP2_degraded_hist/diffPeaks/*cond2.bed\n",
    "commonpeak = ! ls ../../data/IRF2BP2_degraded_hist/diffPeaks/*common.bed\n",
    "cond1peak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"H3K27ac\", \"H3K27ac_v2\",\"H3K27me3\",\"H3K27me3_v2\",\"H3K4me1\",\"H3K4me1_v2\", \"H3K4me3\", \"H3K4me3_v2\", \"H3K79me2\", \"H3K79me2_v2\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for i in range(int(len(bw)/2)):\n",
    "    if i < 0 : \n",
    "        continue\n",
    "    print(i)\n",
    "    name1 = bw[i]\n",
    "    name2 = bw[10+i]\n",
    "    peak = [cond1peak[i], commonpeak[i], cond2peak[i]]\n",
    "    #for val in peak:\n",
    "        #chip.dropWeirdChromosomes(val)\n",
    "    name = names[i]\n",
    "    print(name1,name2)\n",
    "    chip.getPeaksAt(peak, [name1, name2], torecompute= True, bigwignames=['DMSO', 'VHL'], peaknames=['DMSO_peaks', 'common', 'VHL_peaks'], window=3000, folder=\"\", title=name, numthreads=7, refpoint='center', name='../../data/IRF2BP2_degraded_hist/diffPeaks/'+name+'_mat.pdf', withDeeptools=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"H3K27ac\", \"H3K27ac_v2\",\"H3K27me3\",\"H3K27me3_v2\",\"H3K4me1\",\"H3K4me1_v2\", \"H3K4me3\", \"H3K4me3_v2\", \"H3K79me2\", \"H3K79me2_v2\"]\n",
    "names.extend([i+'_VHL' for i in names])\n",
    "names"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "peaks = ! ls ../../data/IRF2BP2_degraded_histones/bwa/mergedLibrary/macs/broadPeak/*.broadPeak \n",
    "peaks"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for i in range(len(bw)):\n",
    "    chip.getPeaksAt(peaks[i], bw[i], window=3000, folder=\"\", title=names[i], numthreads=7, torecompute=True, onlyProfile=True, refpoint='center', name='../../data/IRF2BP2_degraded_hist/diffPeaks/'+names[i]+'_mat_profile.pdf', withDeeptools=True)\n",
    "    chip.getPeaksAt(peaks[i], bw[i], window=3000, folder=\"\", title=names[i], numthreads=7, torecompute=True, refpoint='center', onlyProfile=True,name='../../data/IRF2BP2_degraded_hist/diffPeaks/'+names[i]+'_mat_profile_clust3.pdf', withDeeptools=True, cluster=3)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### on unscalled data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bams = !ls ../../data/IRF2BP2_degraded_hist/bwa/mergedLibrary/mp*.bam\n",
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bw = ! ls ../../data/IRF2BP2_degraded_histones/bwa/mergedLibrary/bigwig/*.bigWig\n",
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# on unscalled data \n",
    "for i in range(int((len(bams)-1)/2)):\n",
    "    name1 = bams[1+i]\n",
    "    name2 = bams[11+i]\n",
    "    chip.fullDiffPeak(name1,name2, control1='../../data/IRF2BP2_degraded_hist/INPUT_R1.mLb.clN.sorted.bam', directory = \"../../data/IRF2BP2_degraded_hist/diffData_unscaled/\", res_directory = \"../../data/IRF2BP2_degraded_hist/diffPeaks_unscaled/\",isTF=False, compute_size=True, pairedend=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "cond1peak = ! ls ../../data/IRF2BP2_degraded_hist/diffPeaks_unscaled/*cond1.bed\n",
    "cond2peak = ! ls ../../data/IRF2BP2_degraded_hist/diffPeaks_unscaled/*cond2.bed\n",
    "commonpeak = ! ls ../../data/IRF2BP2_degraded_hist/diffPeaks_unscaled/*common.bed\n",
    "cond2peak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"H3K27ac\", \"H3K27ac_v2\",\"H3K27me3\",\"H3K27me3_v2\",\"H3K4me1\",\"H3K4me1_v2\", \"H3K4me3\", \"H3K4me3_v2\", \"H3K79me2\", \"H3K79me2_v2\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for i in range(int(len(bw)/2)):\n",
    "    if i < 0:\n",
    "        continue\n",
    "    name1 = bw[1+i]\n",
    "    name2 = bw[11+i]\n",
    "    peak = [cond1peak[i], commonpeak[i], cond2peak[i]]\n",
    "    #for val in peak:\n",
    "        #chip.dropWeirdChromosomes(val)\n",
    "    name = names[i]\n",
    "    print(name1,name2)\n",
    "    chip.getPeaksAt(peak, [name1, name2], bigwignames=['DMSO', 'VHL'], peaknames=['DMSO_peaks', 'common', 'VHL_peaks'], window=3000, folder=\"\", title=name, numthreads=10, refpoint='center', name='../../data/IRF2BP2_degraded_hist/diffPeaks_unscaled/'+name+'_mat.pdf', withDeeptools=True, torecompute=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! gsutil -m cp -r \"../../data/IRF2BP2_degraded_hist/\" gs://amlproject/Chip/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "## on local\n",
    "! scp runningchip2.us-east1-b.aml-chipseq-rnaseq-259419:\"~/data/IRF2BP2_degraded_rep3/diffPeaks_unscaled/*.pdf\" unscaled/v3/\n",
    "! scp runningchip2.us-east1-b.aml-chipseq-rnaseq-259419:\"~/data/IRF2BP2_degraded_rep3/diffPeaks/*.pdf\" scaled/v3/\n",
    "! scp runningchip2.us-east1-b.aml-chipseq-rnaseq-259419:\"~/data/IRF2BP2_degraded_rep1/diffPeaks_unscaled/*.pdf\" unscaled/v3/\n",
    "! scp runningchip2.us-east1-b.aml-chipseq-rnaseq-259419:\"~/data/IRF2BP2_degraded_rep1/diffPeaks/*.pdf\" scaled/v1/\n",
    "! scp runningchip2.us-east1-b.aml-chipseq-rnaseq-259419:\"~/data/IRF2BP2_degraded_hist/diffPeaks_unscaled/*.pdf\" unscaled/v1/\n",
    "! scp runningchip2.us-east1-b.aml-chipseq-rnaseq-259419:\"~/data/IRF2BP2_degraded_hist/diffPeaks/*.pdf\" unscaled/v1/\n",
    "! scp runningchip2.us-east1-b.aml-chipseq-rnaseq-259419:\"~/data/IRF2BP2_degraded_rep2/diffPeaks_unscaled/*.pdf\" unscaled/v2/\n",
    "! scp runningchip2.us-east1-b.aml-chipseq-rnaseq-259419:\"~/data/IRF2BP2_degraded_rep2/diffPeaks/*.pdf\" scaled/v2/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## v4"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! gsutil -m mv gs://transfer-amlproject/*MP7868*  gs://transfer-amlproject/IRF2BP2_v4/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mkdir ../../data/IRF2BP2_degraded_rep4 && mkdir ../../data/IRF2BP2_degraded_rep4/fastqs && gsutil -m cp gs://transfer-amlproject/IRF2BP2_v4/* ../../data/IRF2BP2_degraded_rep4/fastqs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "a = ! ls ../../data/IRF2BP2_degraded_rep4/fastqs\n",
    "a"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "rename = {\n",
    "\"20200302_1_MP7868_S51_R1_001.fastq.gz\":\"mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1.fastq.gz\", \n",
    "\"20200302_1_MP7868_S51_R2_001.fastq.gz\":\"mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R2.fastq.gz\", \n",
    "\"20200302_2_MP7868_S52_R1_001.fastq.gz\":\"mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1.fastq.gz\", \n",
    "\"20200302_2_MP7868_S52_R2_001.fastq.gz\":\"mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R2.fastq.gz\", \n",
    "\"20200302_3_MP7868_S53_R1_001.fastq.gz\":\"mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1.fastq.gz\", \n",
    "\"20200302_3_MP7868_S53_R2_001.fastq.gz\":\"mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R2.fastq.gz\",\n",
    "\"20200302_4_MP7868_S54_R1_001.fastq.gz\":\"mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1.fastq.gz\",\n",
    "\"20200302_4_MP7868_S54_R2_001.fastq.gz\":\"mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R2.fastq.gz\",\n",
    "\"20200302_5_MP7868_S55_R1_001.fastq.gz\":\"mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1.fastq.gz\",\n",
    "\"20200302_5_MP7868_S55_R2_001.fastq.gz\":\"mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R2.fastq.gz\",\n",
    "\"20200302_6_MP7868_S56_R1_001.fastq.gz\":\"mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1.fastq.gz\",\n",
    "\"20200302_6_MP7868_S56_R2_001.fastq.gz\":\"mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R2.fastq.gz\",\n",
    "\"20200302_7_MP7868_S57_R1_001.fastq.gz\":\"mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1.fastq.gz\",\n",
    "\"20200302_7_MP7868_S57_R2_001.fastq.gz\":\"mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R2.fastq.gz\",\n",
    "\"20200302_8_MP7868_S58_R1_001.fastq.gz\":\"mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1.fastq.gz\",\n",
    "\"20200302_8_MP7868_S58_R2_001.fastq.gz\":\"mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R2.fastq.gz\",\n",
    "\"20200302_9_MP7868_S59_R1_001.fastq.gz\":\"mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1.fastq.gz\",\n",
    "\"20200302_9_MP7868_S59_R2_001.fastq.gz\":\"mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R2.fastq.gz\", \n",
    "\"20200302_10_MP7868_S60_R1_001.fastq.gz\":\"mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1.fastq.gz\",\n",
    "\"20200302_10_MP7868_S60_R2_001.fastq.gz\":\"mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R2.fastq.gz\",\n",
    "\"20200302_11_MP7868_S61_R1_001.fastq.gz\":\"mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1.fastq.gz\",\n",
    "\"20200302_11_MP7868_S61_R2_001.fastq.gz\":\"mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R2.fastq.gz\",\n",
    "\"20200302_12_MP7868_S62_R1_001.fastq.gz\":\"mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1.fastq.gz\",\n",
    "\"20200302_12_MP7868_S62_R2_001.fastq.gz\":\"mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R2.fastq.gz\"\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for k,v in rename.items():\n",
    "    ! mv ../../data/IRF2BP2_degraded_rep4/fastqs/$k ../../data/IRF2BP2_degraded_rep4/fastqs/$v"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from gsheets import Sheets\n",
    "sheets = Sheets.from_files('~/.client_secret.json', '~/.storage.json')\n",
    "url=\"https://docs.google.com/spreadsheets/d/1yFLjYB1McU530JnLgL0QIMAKIkVl3kl0_LCHje2gk8U\"\n",
    "gsheet = sheets.get(url).sheets[2].to_frame()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "gsheet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = {\n",
    "\"fastq_1\": [],\n",
    "\"fastq_2\": [],\n",
    "\"antibody\": [],\n",
    "\"group\": [],\n",
    "\"replicate\": [],\n",
    "\"control\": []\n",
    "}\n",
    "for val in h.grouped(a,2):\n",
    "    row = gsheet[gsheet.id==val[0].split('/')[-1].split('-')[0]]\n",
    "    df['group'].append(row['name'].values[0])\n",
    "    df['replicate'].append(1)\n",
    "    df['fastq_1'].append(val[0])\n",
    "    df['fastq_2'].append(val[1])\n",
    "    df['antibody'].append(row['protein'].values[0])\n",
    "    df['control'].append(\"INPUT\")\n",
    "df['group'].append('INPUT')\n",
    "df['replicate'].append(1)\n",
    "df['fastq_1'].append('IRF2BP2_degraded_rep3/mp845-MV411_IRF2BP2_-INPUT-r1_R1.fastq.gz')\n",
    "df['fastq_2'].append('IRF2BP2_degraded_rep3/mp845-MV411_IRF2BP2_-INPUT-r1_R2.fastq.gz')\n",
    "df['antibody'].append(\"\")\n",
    "df['control'].append(\"\")\n",
    "df = pd.DataFrame(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df[df.columns[[3,4,0,1,2,5]]].to_csv('../nextflow/IRF2BP2_degraded_rep4_design.csv',index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#process chips\n",
    "! sudo ../nextflow run nf-core/chipseq --paired_end --seq_center 'DFCI' --email \\\n",
    "'jkobject@gmail.com' --narrow_peak --input ../nextflow/IRF2BP2_degraded_rep4_design.csv --genome GRCh38 --skip_preseq \\\n",
    "--max_cpus 24 -profile docker -w work"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!cp -r results/* ../../data/IRF2BP2_degraded_rep4/ && sudo rm -r work && sudo rm -r results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "ls ../../TrimGalore-0.6.5/trim_galore"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# get scaling values\n",
    "mappedreads, umappedreads_norm = chip.getSpikeInControlScales(refgenome=\"../../data/ref/reference_droso.fna\",\n",
    "fastQfolder='../../data/IRF2BP2_degraded_rep4/fastqs/',\n",
    "pairedEnd=True, cores=12,\n",
    "tofilter=True,\n",
    "totrim=False,\n",
    "tomap=True,\n",
    "results=\"../../data/IRF2BP2_degraded_rep4/\",\n",
    "pathtotrim_galore=\"../../TrimGalore-0.6.5/trim_galore\")\n",
    "mappedreads, umappedreads_norm"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "computing scales from the excel sheet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bams = ! ls ../../data/IRF2BP2_degraded_rep4/bwa/mergedLibrary/*.bam\n",
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bams[6]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### on scalled data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bams = [bam1.split('/')[-1].split('.')[0] for bam1 in bams]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# diffPeak on scaled data\n",
    "size=[ 208, 214, 207, 234, 296, 231]\n",
    "for i in range(int(len(bams)/2)-1):\n",
    "    name1 = bams[1+i]\n",
    "    name2 = bams[7+i]\n",
    "    print(name1,name2)\n",
    "    chip.diffPeak(name1, name2, directory= \"../../data/IRF2BP2_degraded_rep4/diffData/\", res_directory='../../data/IRF2BP2_degraded_rep4/diffPeaks/', scaling1=scales[i][1], scaling2=scales[i][0], size=size[i])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# diffPeak on scaled data\n",
    "for i in range(int(len(bams[1:])/2)):\n",
    "    name1 = bams[1+i]\n",
    "    name2 = bams[7+i]\n",
    "    chip.fullDiffPeak(name1,name2, control1='../../data/IRF2BP2_degraded_rep3/bwa/mergedLibrary/mp845-MV411_IRF2BP2-INPUT-r1.mLb.clN.sorted.bam', directory = \"../../data/IRF2BP2_degraded_rep4/diffData/\", res_directory = \"../../data/IRF2BP2_degraded_rep4/diffPeaks/\",pairedend=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [1.0,\n",
    "1.0,\n",
    "1.0,\n",
    "1.0,\n",
    "1.0,\n",
    "0.626304048,\n",
    "0.2628507876,\n",
    "0.9021192519,\n",
    "0.1869653476,\n",
    "0.5500321887,\n",
    "0.6046056203,\n",
    "1.0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "chip.bigWigFrom(bams[1:], \n",
    "                genome='GRCh38',scaling=scales,\n",
    "               numthreads=8)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!mv bigwig ../../data/IRF2BP2_degraded_rep4/recalib_bigwig/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "os.popen('for i in $(ls ../../data/IRF2BP2_degraded_rep4/diffPeaks/*.bed); \\\n",
    "            do echo $(wc -l $i); \\\n",
    "            done').read().split('\\n')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bw = ! ls ../../data/IRF2BP2_degraded_rep4/recalib_bigwig/*.bw\n",
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "cond1peak = ! ls ../../data/IRF2BP2_degraded_rep4/diffPeaks/*cond1.bed\n",
    "cond2peak = ! ls ../../data/IRF2BP2_degraded_rep4/diffPeaks/*cond2.bed\n",
    "commonpeak = ! ls ../../data/IRF2BP2_degraded_rep4/diffPeaks/*common.bed\n",
    "cond1peak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"MED1\",\"MED1_v2\",\"FLAG_IRF2BP2\",\"POLII_total\",\"POLII_S2\",\"POLII_S5\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for i in range(int(len(bw)/2)):\n",
    "    if i <0:\n",
    "        continue\n",
    "    name1 = bw[i]\n",
    "    name2 = bw[i+6]\n",
    "    peak = [cond1peak[i], commonpeak[i], cond2peak[i]]\n",
    "   # for val in peak:\n",
    "      #  chip.dropWeirdChromosomes(val)\n",
    "    name = names[i]\n",
    "    print(name1,name2)\n",
    "    chip.getPeaksAt(peak, [name1, name2], bigwignames=['DMSO', 'VHL'], peaknames=['DMSO_peaks', 'common', 'VHL_peaks'], window=3000, folder=\"\", title=name, numthreads=7, refpoint=\"center\", name='../../data/IRF2BP2_degraded_rep4/diffPeaks/'+name+'_mat.pdf', withDeeptools=True, torecompute=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "peaks = ! ls ../../data/results4/bwa/mergedLibrary/macs/narrowPeak/*.narrowPeak\n",
    "peaks"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"MED1\",\"MED1_v2\",\"FLAG_IRF2BP2\",\"POLII_total\",\"POLII_S2\",\"POLII_S5\"]\n",
    "names.extend([i+'_VHL' for i in names])\n",
    "names"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for i in range(len(bw)):\n",
    "    chip.getPeaksAt(peaks[i], bw[i], window=3000, folder=\"\", title=names[i], numthreads=7, torecompute=True, onlyProfile=True, name='../../data/IRF2BP2_degraded_rep4/diffPeaks/'+names[i]+'_mat_profile.pdf', withDeeptools=True, refpoint=\"center\")\n",
    "    chip.getPeaksAt(peaks[i], bw[i], window=3000, folder=\"\", title=names[i], numthreads=7, torecompute=True, onlyProfile=True,name='../../data/IRF2BP2_degraded_rep4/diffPeaks/'+names[i]+'_mat_profile_clust3.pdf', withDeeptools=True, cluster=3, refpoint=\"center\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### on unscalled data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bams = !ls ../../data/results4/bwa/mergedLibrary/mp*.bam\n",
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mkdir ../../data/IRF2BP2_degraded_rep4/diffPeaks_unscaled"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#on unscalled data \n",
    "for i in range(int(len(bams)/2)):\n",
    "    if i < 5:\n",
    "        continue\n",
    "    name1 = bams[i]\n",
    "    name2 = bams[6+i]\n",
    "    print(name1,name2)\n",
    "    chip.fullDiffPeak(name1,name2, control1='../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam', directory = \"../../data/IRF2BP2_degraded_rep4/diffData_unscaled/\", res_directory = \"../../data/IRF2BP2_degraded_rep4/diffPeaks_unscaled/\",pairedend=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bw = ! ls ../../data/results4/bwa/mergedLibrary/bigwig/*.bigWig\n",
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "cond1peak = ! ls ../../data/IRF2BP2_degraded_rep4/diffPeaks_unscaled/*cond1.bed\n",
    "cond2peak = ! ls ../../data/IRF2BP2_degraded_rep4/diffPeaks_unscaled/*cond2.bed\n",
    "commonpeak = ! ls ../../data/IRF2BP2_degraded_rep4/diffPeaks_unscaled/*common.bed\n",
    "commonpeak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"MED1\",\"MED1_v2\",\"FLAG_IRF2BP2\",\"POLII_total\",\"POLII_S2\",\"POLII_S5\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for i in range(int((len(bw)-1)/2)):\n",
    "    name1 = bw[1+i]\n",
    "    name2 = bw[7+i]\n",
    "    peak = [cond1peak[i], commonpeak[i], cond2peak[i]]\n",
    "    #for val in peak:\n",
    "     #   chip.dropWeirdChromosomes(val)\n",
    "    name = names[i]\n",
    "    print(name1,name2)\n",
    "    chip.getPeaksAt(peak, [name1, name2], bigwignames=['DMSO', 'VHL'], peaknames=['DMSO_peaks', 'common', 'VHL_peaks'], window=3000, folder=\"\", title=name, numthreads=7, torecompute=True, refpoint='center', name='../../data/IRF2BP2_degraded_rep4/diffPeaks_unscaled/'+name+'_mat.pdf', withDeeptools=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! gsutil -m cp -r \"../../data/IRF2BP2_degraded_rep4\" gs://amlproject/Chip/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Copying data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [],
   "source": [
    "mkdir ../results/$project/diffPeaks_unscaled/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [],
   "source": [
    "! cp ../../data/*/diffPeaks_unscaled/*.bed ../results/$project/diffPeaks_unscaled/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [],
   "source": [
    "mkdir ../results/$project/diffPeaks_scaled/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [],
   "source": [
    "! cp ../../data/*/diffPeaks/*.bed ../results/$project/diffPeaks_scaled/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## v5"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "project=\"additional_degraded_v1\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_10_MP8095_S220_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_10_MP8095_S220_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_11_MP8095_S221_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_11_MP8095_S221_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_12_MP8095_S222_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_12_MP8095_S222_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_13_MP8095_S223_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_13_MP8095_S223_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_14_MP8095_S224_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_14_MP8095_S224_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_15_MP8095_S225_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_15_MP8095_S225_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_16_MP8095_S226_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_16_MP8095_S226_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_1S_MP8095_S209_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_1S_MP8095_S209_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_1_MP8095_S211_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_1_MP8095_S211_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_2S_MP8095_S210_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_2S_MP8095_S210_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_2_MP8095_S212_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_2_MP8095_S212_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_3_MP8095_S213_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_3_MP8095_S213_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_4_MP8095_S214_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_4_MP8095_S214_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_5_MP8095_S215_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_5_MP8095_S215_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_6_MP8095_S216_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_6_MP8095_S216_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_7_MP8095_S217_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_7_MP8095_S217_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_8_MP8095_S218_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_8_MP8095_S218_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_9_MP8095_S219_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/20200723_9_MP8095_S219_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/multiqc_report.html\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/Reports/\r\n",
      "gs://transfer-amlproject/200723_MP8095_fastq/multiqc_data/\r\n"
     ]
    }
   ],
   "source": [
    "! gsutil ls gs://transfer-amlproject/200723_MP8095_fastq/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [],
   "source": [
    "mkdir ../data/$project/qc/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/multiqc_report.html...\n",
      "/ [1/1 files][  1.5 MiB/  1.5 MiB] 100% Done                                    \n",
      "Operation completed over 1 objects/1.5 MiB.                                      \n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_10_MP8095_S220_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_10_MP8095_S220_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_10_MP8095_S220_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_10_MP8095_S220_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_11_MP8095_S221_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_11_MP8095_S221_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_11_MP8095_S221_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_11_MP8095_S221_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_12_MP8095_S222_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_12_MP8095_S222_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_12_MP8095_S222_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_12_MP8095_S222_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_13_MP8095_S223_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_13_MP8095_S223_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_13_MP8095_S223_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_13_MP8095_S223_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_14_MP8095_S224_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_14_MP8095_S224_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_14_MP8095_S224_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_14_MP8095_S224_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_15_MP8095_S225_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_15_MP8095_S225_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_15_MP8095_S225_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_15_MP8095_S225_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_16_MP8095_S226_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_1S_MP8095_S209_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_16_MP8095_S226_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_1_MP8095_S211_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_16_MP8095_S226_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_16_MP8095_S226_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_1S_MP8095_S209_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_1_MP8095_S211_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_1S_MP8095_S209_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_2S_MP8095_S210_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_1_MP8095_S211_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_2S_MP8095_S210_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_2S_MP8095_S210_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_1_MP8095_S211_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_1S_MP8095_S209_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_2S_MP8095_S210_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_2_MP8095_S212_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_2_MP8095_S212_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_2_MP8095_S212_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_2_MP8095_S212_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_3_MP8095_S213_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_3_MP8095_S213_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_3_MP8095_S213_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_3_MP8095_S213_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_4_MP8095_S214_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_4_MP8095_S214_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_4_MP8095_S214_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_4_MP8095_S214_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_5_MP8095_S215_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_5_MP8095_S215_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_5_MP8095_S215_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_5_MP8095_S215_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_6_MP8095_S216_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_6_MP8095_S216_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_6_MP8095_S216_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_6_MP8095_S216_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_7_MP8095_S217_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_7_MP8095_S217_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_7_MP8095_S217_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_8_MP8095_S218_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_7_MP8095_S217_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_8_MP8095_S218_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_8_MP8095_S218_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_8_MP8095_S218_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_9_MP8095_S219_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_9_MP8095_S219_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_9_MP8095_S219_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/Reports/20200723_9_MP8095_S219_R2_001_fastqc.html...\n",
      "\\ [72/72 files][ 38.4 MiB/ 38.4 MiB] 100% Done                                  \n",
      "Operation completed over 72 objects/38.4 MiB.                                    \n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/multiqc_data/multiqc.log...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/multiqc_data/multiqc_data.json...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/multiqc_data/multiqc_fastqc.txt...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/multiqc_data/multiqc_general_stats.txt...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/multiqc_data/multiqc_qualimap_bamqc_genome_results.txt...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/multiqc_data/multiqc_sources.txt...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/ [6/6 files][  2.2 MiB/  2.2 MiB] 100% Done                                    \n",
      "Operation completed over 6 objects/2.2 MiB.                                      \n"
     ]
    }
   ],
   "source": [
    "!gsutil -m cp gs://transfer-amlproject/200723_MP8095_fastq/multiqc_report.html ../data/$project/qc/\n",
    "!gsutil -m cp -r gs://transfer-amlproject/200723_MP8095_fastq/Reports/ ../data/$project/qc/\n",
    "!gsutil -m cp -r gs://transfer-amlproject/200723_MP8095_fastq/multiqc_data/ ../data/$project/qc/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Omitting prefix \"gs://transfer-amlproject/200723_MP8095_fastq/Reports/\". (Did you mean to do cp -r?)\n",
      "Omitting prefix \"gs://transfer-amlproject/200723_MP8095_fastq/multiqc_data/\". (Did you mean to do cp -r?)\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_10_MP8095_S220_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_10_MP8095_S220_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_11_MP8095_S221_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_11_MP8095_S221_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_12_MP8095_S222_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_12_MP8095_S222_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_13_MP8095_S223_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_13_MP8095_S223_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_14_MP8095_S224_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_14_MP8095_S224_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_15_MP8095_S225_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_15_MP8095_S225_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_16_MP8095_S226_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_16_MP8095_S226_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_1S_MP8095_S209_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_1S_MP8095_S209_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_1_MP8095_S211_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_1_MP8095_S211_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_2_MP8095_S212_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_2S_MP8095_S210_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_2S_MP8095_S210_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_2_MP8095_S212_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_3_MP8095_S213_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_3_MP8095_S213_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_4_MP8095_S214_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_4_MP8095_S214_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_5_MP8095_S215_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_5_MP8095_S215_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_8_MP8095_S218_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_7_MP8095_S217_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_6_MP8095_S216_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_6_MP8095_S216_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_7_MP8095_S217_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_8_MP8095_S218_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_9_MP8095_S219_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/20200723_9_MP8095_S219_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200723_MP8095_fastq/multiqc_report.html [Content-Type=text/html]...\n",
      "- [37/37 files][ 65.6 GiB/ 65.6 GiB] 100% Done                                  \n",
      "Operation completed over 37 objects/65.6 GiB.                                    \n"
     ]
    }
   ],
   "source": [
    "! gsutil -m cp gs://transfer-amlproject/200723_MP8095_fastq/*  gs://transfer-amlproject/$project/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_10_MP8095_S220_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_10_MP8095_S220_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_11_MP8095_S221_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_11_MP8095_S221_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_12_MP8095_S222_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_13_MP8095_S223_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_12_MP8095_S222_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_13_MP8095_S223_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_14_MP8095_S224_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_14_MP8095_S224_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_15_MP8095_S225_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_15_MP8095_S225_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_16_MP8095_S226_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_16_MP8095_S226_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_1S_MP8095_S209_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_1S_MP8095_S209_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_1_MP8095_S211_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_1_MP8095_S211_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_2S_MP8095_S210_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_2S_MP8095_S210_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_2_MP8095_S212_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_2_MP8095_S212_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_3_MP8095_S213_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_3_MP8095_S213_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_4_MP8095_S214_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_4_MP8095_S214_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_5_MP8095_S215_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_5_MP8095_S215_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_6_MP8095_S216_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_6_MP8095_S216_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_7_MP8095_S217_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_9_MP8095_S219_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/multiqc_report.html...  \n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_8_MP8095_S218_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_7_MP8095_S217_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_8_MP8095_S218_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/additional_degraded_v1/20200723_9_MP8095_S219_R1_001.fastq.gz...\n",
      "- [37/37 files][ 65.6 GiB/ 65.6 GiB] 100% Done  79.8 MiB/s ETA 00:00:00         \n",
      "Operation completed over 37 objects/65.6 GiB.                                    \n"
     ]
    }
   ],
   "source": [
    "! mkdir ../../data/$project && mkdir ../../data/$project/fastqs && gsutil -m cp gs://transfer-amlproject/$project/* ../../data/$project/fastqs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {},
   "outputs": [],
   "source": [
    "! rm ../../data/$project/fastqs/multiqc_report.html"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['mp858-MV411_RNP_AAVS1-H3K27AC-r1_R1_001.fastq.gz',\n",
       " 'mp858-MV411_RNP_AAVS1-H3K27AC-r1_R2_001.fastq.gz',\n",
       " 'mp859-MV411_RNP_AAVS1-H3K27AC-r2_R1_001.fastq.gz',\n",
       " 'mp859-MV411_RNP_AAVS1-H3K27AC-r2_R2_001.fastq.gz',\n",
       " 'mp860-MV411_RNP_RUNX1-H3K27AC-r1_R1_001.fastq.gz',\n",
       " 'mp860-MV411_RNP_RUNX1-H3K27AC-r1_R2_001.fastq.gz',\n",
       " 'mp861-MV411_RNP_RUNX1-H3K27AC-r2_R1_001.fastq.gz',\n",
       " 'mp861-MV411_RNP_RUNX1-H3K27AC-r2_R2_001.fastq.gz',\n",
       " 'mp862-MV411_RNP_RUNX2-H3K27AC-r1_R1_001.fastq.gz',\n",
       " 'mp862-MV411_RNP_RUNX2-H3K27AC-r1_R2_001.fastq.gz',\n",
       " 'mp863-MV411_RNP_RUNX2-H3K27AC-r2_R1_001.fastq.gz',\n",
       " 'mp863-MV411_RNP_RUNX2-H3K27AC-r2_R2_001.fastq.gz',\n",
       " 'mp864-MV411_RNP_RUNX1_RUNX2-H3K27AC-r1_R1_001.fastq.gz',\n",
       " 'mp864-MV411_RNP_RUNX1_RUNX2-H3K27AC-r1_R2_001.fastq.gz',\n",
       " 'mp865-MV411_RNP_RUNX1_RUNX2-H3K27AC-r2_R1_001.fastq.gz',\n",
       " 'mp865-MV411_RNP_RUNX1_RUNX2-H3K27AC-r2_R2_001.fastq.gz',\n",
       " 'mp866-MV411_RNP_MEF2D-H3K27AC-r1_R1_001.fastq.gz',\n",
       " 'mp866-MV411_RNP_MEF2D-H3K27AC-r1_R2_001.fastq.gz',\n",
       " 'mp867-MV411_RNP_MEF2D-H3K27AC-r2_R1_001.fastq.gz',\n",
       " 'mp867-MV411_RNP_MEF2D-H3K27AC-r2_R2_001.fastq.gz',\n",
       " 'mp868-MV411_RNP_IRF8-H3K27AC-r1_R1_001.fastq.gz',\n",
       " 'mp868-MV411_RNP_IRF8-H3K27AC-r1_R2_001.fastq.gz',\n",
       " 'mp869-MV411_RNP_IRF8-H3K27AC-r2_R1_001.fastq.gz',\n",
       " 'mp869-MV411_RNP_IRF8-H3K27AC-r2_R2_001.fastq.gz',\n",
       " 'mp870-MV411_RNP_MYB-H3K27AC-r1_R1_001.fastq.gz',\n",
       " 'mp870-MV411_RNP_MYB-H3K27AC-r1_R2_001.fastq.gz',\n",
       " 'mp871-MV411_RNP_MYB-H3K27AC-r2_R1_001.fastq.gz',\n",
       " 'mp871-MV411_RNP_MYB-H3K27AC-r2_R2_001.fastq.gz',\n",
       " 'mp872-MV411_RNP_SPI1-H3K27AC-r1_R1_001.fastq.gz',\n",
       " 'mp872-MV411_RNP_SPI1-H3K27AC-r1_R2_001.fastq.gz',\n",
       " 'mp873-MV411_RNP_SPI1-H3K27AC-r2_R1_001.fastq.gz',\n",
       " 'mp873-MV411_RNP_SPI1-H3K27AC-r2_R2_001.fastq.gz']"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "a = ! ls ../../data/$project/fastqs\n",
    "a"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 67,
   "metadata": {},
   "outputs": [],
   "source": [
    "rename = {\n",
    "\"20200723_1_MP8095_S211\": \"mp858-MV411_RNP_AAVS1-H3K27AC-r1\",\n",
    "\"20200723_2_MP8095_S212\": \"mp859-MV411_RNP_AAVS1-H3K27AC-r2\",\n",
    "\"20200723_3_MP8095_S213\": \"mp860-MV411_RNP_RUNX1-H3K27AC-r1\",\n",
    "\"20200723_4_MP8095_S214\": \"mp861-MV411_RNP_RUNX1-H3K27AC-r2\",\n",
    "\"20200723_5_MP8095_S215\": \"mp862-MV411_RNP_RUNX2-H3K27AC-r1\",\n",
    "\"20200723_6_MP8095_S216\": \"mp863-MV411_RNP_RUNX2-H3K27AC-r2\",\n",
    "\"20200723_7_MP8095_S217\": \"mp864-MV411_RNP_RUNX1_RUNX2-H3K27AC-r1\",\n",
    "\"20200723_8_MP8095_S218\": \"mp865-MV411_RNP_RUNX1_RUNX2-H3K27AC-r2\",\n",
    "\"20200723_9_MP8095_S219\": \"mp866-MV411_RNP_MEF2D-H3K27AC-r1\",\n",
    "\"20200723_10_MP8095_S220\": \"mp867-MV411_RNP_MEF2D-H3K27AC-r2\",\n",
    "\"20200723_11_MP8095_S221\": \"mp868-MV411_RNP_IRF8-H3K27AC-r1\",\n",
    "\"20200723_12_MP8095_S222\": \"mp869-MV411_RNP_IRF8-H3K27AC-r2\",\n",
    "\"20200723_13_MP8095_S223\": \"mp870-MV411_RNP_MYB-H3K27AC-r1\",\n",
    "\"20200723_14_MP8095_S224\": \"mp871-MV411_RNP_MYB-H3K27AC-r2\",\n",
    "\"20200723_15_MP8095_S225\": \"mp872-MV411_RNP_SPI1-H3K27AC-r1\",\n",
    "\"20200723_16_MP8095_S226\": \"mp873-MV411_RNP_SPI1-H3K27AC-r2\",\n",
    "\"20200723_1S_MP8095_S209\": \"mp874-MV411_MEF2D_NT_SC_63-FLAG_MEF2D-r2\",\n",
    "\"20200723_2S_MP8095_S210\": \"mp875-MV411_MEF2C_NT-FLAG_MEF2C-r1\"}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {},
   "outputs": [],
   "source": [
    "for val in a:\n",
    "    rep = val\n",
    "    for k,v in rename.items():\n",
    "        rep = rep.replace(k,v)\n",
    "    !mv ../../data/$project/fastqs/$val ../../data/$project/fastqs/$rep"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 69,
   "metadata": {},
   "outputs": [],
   "source": [
    "from gsheets import Sheets\n",
    "sheets = Sheets.from_files('~/.client_secret.json', '~/.storage.json')\n",
    "url=\"https://docs.google.com/spreadsheets/d/1yFLjYB1McU530JnLgL0QIMAKIkVl3kl0_LCHje2gk8U\"\n",
    "gsheet = sheets.get(url).sheets[2].to_frame()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>id</th>\n",
       "      <th>cell line</th>\n",
       "      <th>replicate</th>\n",
       "      <th>protein</th>\n",
       "      <th>quality</th>\n",
       "      <th>paired_end</th>\n",
       "      <th>matching input name</th>\n",
       "      <th>processed</th>\n",
       "      <th>name</th>\n",
       "      <th>previous name</th>\n",
       "      <th>...</th>\n",
       "      <th>ratio to droso</th>\n",
       "      <th>unique mapped reads(droso)</th>\n",
       "      <th>scaling factor</th>\n",
       "      <th>Total QC</th>\n",
       "      <th>folderNarrow</th>\n",
       "      <th>folderCompensated</th>\n",
       "      <th>folderQC</th>\n",
       "      <th>folderBroad</th>\n",
       "      <th>folder Bigwig</th>\n",
       "      <th>folder diffPeaks</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>mp100</td>\n",
       "      <td>U937</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "      <td>NaN</td>\n",
       "      <td>n</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Y</td>\n",
       "      <td>mp100-U937-INPUT-r1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://storage.cloud.google.com/amlproject/Ch...</td>\n",
       "      <td>https://console.cloud.google.com/storage/brows...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://console.cloud.google.com/storage/brows...</td>\n",
       "      <td>https://console.cloud.google.com/storage/brows...</td>\n",
       "      <td>https://console.cloud.google.com/storage/brows...</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>mp101</td>\n",
       "      <td>NOMO1</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "      <td>NaN</td>\n",
       "      <td>n</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Y</td>\n",
       "      <td>mp101-NOMO1-INPUT-r1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>mp102</td>\n",
       "      <td>UT7</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "      <td>NaN</td>\n",
       "      <td>n</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Y</td>\n",
       "      <td>mp102-UT7-INPUT-r1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>mp106</td>\n",
       "      <td>MV411</td>\n",
       "      <td>1</td>\n",
       "      <td>MYB</td>\n",
       "      <td>x</td>\n",
       "      <td>n</td>\n",
       "      <td>INPUT_MV411</td>\n",
       "      <td>Y</td>\n",
       "      <td>mp106-MV411-MYB-r1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>mp109</td>\n",
       "      <td>M6</td>\n",
       "      <td>1</td>\n",
       "      <td>CEBPA</td>\n",
       "      <td>x</td>\n",
       "      <td>n</td>\n",
       "      <td>INPUT_M6</td>\n",
       "      <td>Y</td>\n",
       "      <td>mp109-M6-CEBPA-r1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>299</th>\n",
       "      <td>mp871</td>\n",
       "      <td>MV411_RNP_MYB</td>\n",
       "      <td>2</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>NaN</td>\n",
       "      <td>y</td>\n",
       "      <td>INPUT_MV411</td>\n",
       "      <td>NaN</td>\n",
       "      <td>mp871-MV411_RNP_MYB-H3K27AC-r2</td>\n",
       "      <td>20200723_14_MP8095_S224</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>300</th>\n",
       "      <td>mp872</td>\n",
       "      <td>MV411_RNP_SPI1</td>\n",
       "      <td>1</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>NaN</td>\n",
       "      <td>y</td>\n",
       "      <td>INPUT_MV411</td>\n",
       "      <td>NaN</td>\n",
       "      <td>mp872-MV411_RNP_SPI1-H3K27AC-r1</td>\n",
       "      <td>20200723_15_MP8095_S225</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>301</th>\n",
       "      <td>mp873</td>\n",
       "      <td>MV411_RNP_SPI1</td>\n",
       "      <td>2</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>NaN</td>\n",
       "      <td>y</td>\n",
       "      <td>INPUT_MV411</td>\n",
       "      <td>NaN</td>\n",
       "      <td>mp873-MV411_RNP_SPI1-H3K27AC-r2</td>\n",
       "      <td>20200723_16_MP8095_S226</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>302</th>\n",
       "      <td>mp874</td>\n",
       "      <td>MV411_MEF2D_NT_SC_63</td>\n",
       "      <td>2</td>\n",
       "      <td>FLAG_MEF2D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>y</td>\n",
       "      <td>INPUT_MV411</td>\n",
       "      <td>NaN</td>\n",
       "      <td>mp874-MV411_MEF2D_NT_SC_63-FLAG_MEF2D-r2</td>\n",
       "      <td>20200723_1S_MP8095_S209</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>303</th>\n",
       "      <td>mp875</td>\n",
       "      <td>MV411_MEF2C_NT</td>\n",
       "      <td>1</td>\n",
       "      <td>FLAG_MEF2C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>y</td>\n",
       "      <td>INPUT_MV411</td>\n",
       "      <td>NaN</td>\n",
       "      <td>mp875-MV411_MEF2C_NT-FLAG_MEF2C-r1</td>\n",
       "      <td>20200723_2S_MP8095_S210</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>304 rows  30 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "        id             cell line  replicate     protein quality paired_end  \\\n",
       "0    mp100                  U937          1       INPUT     NaN          n   \n",
       "1    mp101                 NOMO1          1       INPUT     NaN          n   \n",
       "2    mp102                   UT7          1       INPUT     NaN          n   \n",
       "3    mp106                 MV411          1         MYB       x          n   \n",
       "4    mp109                    M6          1       CEBPA       x          n   \n",
       "..     ...                   ...        ...         ...     ...        ...   \n",
       "299  mp871         MV411_RNP_MYB          2     H3K27AC     NaN          y   \n",
       "300  mp872        MV411_RNP_SPI1          1     H3K27AC     NaN          y   \n",
       "301  mp873        MV411_RNP_SPI1          2     H3K27AC     NaN          y   \n",
       "302  mp874  MV411_MEF2D_NT_SC_63          2  FLAG_MEF2D     NaN          y   \n",
       "303  mp875        MV411_MEF2C_NT          1  FLAG_MEF2C     NaN          y   \n",
       "\n",
       "    matching input name processed                                      name  \\\n",
       "0                   NaN         Y                       mp100-U937-INPUT-r1   \n",
       "1                   NaN         Y                      mp101-NOMO1-INPUT-r1   \n",
       "2                   NaN         Y                        mp102-UT7-INPUT-r1   \n",
       "3           INPUT_MV411         Y                        mp106-MV411-MYB-r1   \n",
       "4              INPUT_M6         Y                         mp109-M6-CEBPA-r1   \n",
       "..                  ...       ...                                       ...   \n",
       "299         INPUT_MV411       NaN            mp871-MV411_RNP_MYB-H3K27AC-r2   \n",
       "300         INPUT_MV411       NaN           mp872-MV411_RNP_SPI1-H3K27AC-r1   \n",
       "301         INPUT_MV411       NaN           mp873-MV411_RNP_SPI1-H3K27AC-r2   \n",
       "302         INPUT_MV411       NaN  mp874-MV411_MEF2D_NT_SC_63-FLAG_MEF2D-r2   \n",
       "303         INPUT_MV411       NaN        mp875-MV411_MEF2C_NT-FLAG_MEF2C-r1   \n",
       "\n",
       "               previous name  ... ratio to droso unique mapped reads(droso)  \\\n",
       "0                        NaN  ...            NaN                        NaN   \n",
       "1                        NaN  ...            NaN                        NaN   \n",
       "2                        NaN  ...            NaN                        NaN   \n",
       "3                        NaN  ...            NaN                        NaN   \n",
       "4                        NaN  ...            NaN                        NaN   \n",
       "..                       ...  ...            ...                        ...   \n",
       "299  20200723_14_MP8095_S224  ...            NaN                        NaN   \n",
       "300  20200723_15_MP8095_S225  ...            NaN                        NaN   \n",
       "301  20200723_16_MP8095_S226  ...            NaN                        NaN   \n",
       "302  20200723_1S_MP8095_S209  ...            NaN                        NaN   \n",
       "303  20200723_2S_MP8095_S210  ...            NaN                        NaN   \n",
       "\n",
       "    scaling factor                                           Total QC  \\\n",
       "0              NaN  https://storage.cloud.google.com/amlproject/Ch...   \n",
       "1              NaN                                                NaN   \n",
       "2              NaN                                                NaN   \n",
       "3              NaN                                                NaN   \n",
       "4              NaN                                                NaN   \n",
       "..             ...                                                ...   \n",
       "299            NaN                                                NaN   \n",
       "300            NaN                                                NaN   \n",
       "301            NaN                                                NaN   \n",
       "302            NaN                                                NaN   \n",
       "303            NaN                                                NaN   \n",
       "\n",
       "                                          folderNarrow folderCompensated  \\\n",
       "0    https://console.cloud.google.com/storage/brows...               NaN   \n",
       "1                                                  NaN               NaN   \n",
       "2                                                  NaN               NaN   \n",
       "3                                                  NaN               NaN   \n",
       "4                                                  NaN               NaN   \n",
       "..                                                 ...               ...   \n",
       "299                                                NaN               NaN   \n",
       "300                                                NaN               NaN   \n",
       "301                                                NaN               NaN   \n",
       "302                                                NaN               NaN   \n",
       "303                                                NaN               NaN   \n",
       "\n",
       "                                              folderQC  \\\n",
       "0    https://console.cloud.google.com/storage/brows...   \n",
       "1                                                  NaN   \n",
       "2                                                  NaN   \n",
       "3                                                  NaN   \n",
       "4                                                  NaN   \n",
       "..                                                 ...   \n",
       "299                                                NaN   \n",
       "300                                                NaN   \n",
       "301                                                NaN   \n",
       "302                                                NaN   \n",
       "303                                                NaN   \n",
       "\n",
       "                                           folderBroad  \\\n",
       "0    https://console.cloud.google.com/storage/brows...   \n",
       "1                                                  NaN   \n",
       "2                                                  NaN   \n",
       "3                                                  NaN   \n",
       "4                                                  NaN   \n",
       "..                                                 ...   \n",
       "299                                                NaN   \n",
       "300                                                NaN   \n",
       "301                                                NaN   \n",
       "302                                                NaN   \n",
       "303                                                NaN   \n",
       "\n",
       "                                         folder Bigwig  folder diffPeaks  \n",
       "0    https://console.cloud.google.com/storage/brows...               NaN  \n",
       "1                                                  NaN               NaN  \n",
       "2                                                  NaN               NaN  \n",
       "3                                                  NaN               NaN  \n",
       "4                                                  NaN               NaN  \n",
       "..                                                 ...               ...  \n",
       "299                                                NaN               NaN  \n",
       "300                                                NaN               NaN  \n",
       "301                                                NaN               NaN  \n",
       "302                                                NaN               NaN  \n",
       "303                                                NaN               NaN  \n",
       "\n",
       "[304 rows x 30 columns]"
      ]
     },
     "execution_count": 70,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gsheet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 71,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['mp858-MV411_RNP_AAVS1-H3K27AC-r1_R1_001.fastq.gz',\n",
       " 'mp858-MV411_RNP_AAVS1-H3K27AC-r1_R2_001.fastq.gz',\n",
       " 'mp859-MV411_RNP_AAVS1-H3K27AC-r2_R1_001.fastq.gz',\n",
       " 'mp859-MV411_RNP_AAVS1-H3K27AC-r2_R2_001.fastq.gz',\n",
       " 'mp860-MV411_RNP_RUNX1-H3K27AC-r1_R1_001.fastq.gz',\n",
       " 'mp860-MV411_RNP_RUNX1-H3K27AC-r1_R2_001.fastq.gz',\n",
       " 'mp861-MV411_RNP_RUNX1-H3K27AC-r2_R1_001.fastq.gz',\n",
       " 'mp861-MV411_RNP_RUNX1-H3K27AC-r2_R2_001.fastq.gz',\n",
       " 'mp862-MV411_RNP_RUNX2-H3K27AC-r1_R1_001.fastq.gz',\n",
       " 'mp862-MV411_RNP_RUNX2-H3K27AC-r1_R2_001.fastq.gz',\n",
       " 'mp863-MV411_RNP_RUNX2-H3K27AC-r2_R1_001.fastq.gz',\n",
       " 'mp863-MV411_RNP_RUNX2-H3K27AC-r2_R2_001.fastq.gz',\n",
       " 'mp864-MV411_RNP_RUNX1_RUNX2-H3K27AC-r1_R1_001.fastq.gz',\n",
       " 'mp864-MV411_RNP_RUNX1_RUNX2-H3K27AC-r1_R2_001.fastq.gz',\n",
       " 'mp865-MV411_RNP_RUNX1_RUNX2-H3K27AC-r2_R1_001.fastq.gz',\n",
       " 'mp865-MV411_RNP_RUNX1_RUNX2-H3K27AC-r2_R2_001.fastq.gz',\n",
       " 'mp866-MV411_RNP_MEF2D-H3K27AC-r1_R1_001.fastq.gz',\n",
       " 'mp866-MV411_RNP_MEF2D-H3K27AC-r1_R2_001.fastq.gz',\n",
       " 'mp867-MV411_RNP_MEF2D-H3K27AC-r2_R1_001.fastq.gz',\n",
       " 'mp867-MV411_RNP_MEF2D-H3K27AC-r2_R2_001.fastq.gz',\n",
       " 'mp868-MV411_RNP_IRF8-H3K27AC-r1_R1_001.fastq.gz',\n",
       " 'mp868-MV411_RNP_IRF8-H3K27AC-r1_R2_001.fastq.gz',\n",
       " 'mp869-MV411_RNP_IRF8-H3K27AC-r2_R1_001.fastq.gz',\n",
       " 'mp869-MV411_RNP_IRF8-H3K27AC-r2_R2_001.fastq.gz',\n",
       " 'mp870-MV411_RNP_MYB-H3K27AC-r1_R1_001.fastq.gz',\n",
       " 'mp870-MV411_RNP_MYB-H3K27AC-r1_R2_001.fastq.gz',\n",
       " 'mp871-MV411_RNP_MYB-H3K27AC-r2_R1_001.fastq.gz',\n",
       " 'mp871-MV411_RNP_MYB-H3K27AC-r2_R2_001.fastq.gz',\n",
       " 'mp872-MV411_RNP_SPI1-H3K27AC-r1_R1_001.fastq.gz',\n",
       " 'mp872-MV411_RNP_SPI1-H3K27AC-r1_R2_001.fastq.gz',\n",
       " 'mp873-MV411_RNP_SPI1-H3K27AC-r2_R1_001.fastq.gz',\n",
       " 'mp873-MV411_RNP_SPI1-H3K27AC-r2_R2_001.fastq.gz',\n",
       " 'mp874-MV411_MEF2D_NT_SC_63-FLAG_MEF2D-r2_R1_001.fastq.gz',\n",
       " 'mp874-MV411_MEF2D_NT_SC_63-FLAG_MEF2D-r2_R2_001.fastq.gz',\n",
       " 'mp875-MV411_MEF2C_NT-FLAG_MEF2C-r1_R1_001.fastq.gz',\n",
       " 'mp875-MV411_MEF2C_NT-FLAG_MEF2C-r1_R2_001.fastq.gz']"
      ]
     },
     "execution_count": 71,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "a = ! ls ../../data/$project/fastqs\n",
    "a"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 73,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying file://../../data/additional_degraded_v1/fastqs/mp874-MV411_MEF2D_NT_SC_63-FLAG_MEF2D-r2_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "==> NOTE: You are uploading one or more large file(s), which would run          \n",
      "significantly faster if you enable parallel composite uploads. This\n",
      "feature can be enabled by editing the\n",
      "\"parallel_composite_upload_threshold\" value in your .boto\n",
      "configuration file. However, note that if you do this large files will\n",
      "be uploaded as `composite objects\n",
      "<https://cloud.google.com/storage/docs/composite-objects>`_,which\n",
      "means that any user who downloads such objects will need to have a\n",
      "compiled crcmod installed (see \"gsutil help crcmod\"). This is because\n",
      "without a compiled crcmod, computing checksums on composite objects is\n",
      "so slow that gsutil disables downloads of composite objects.\n",
      "\n",
      "/ [1 files][  1.7 GiB/  1.7 GiB]   98.9 MiB/s                                   \n",
      "Operation completed over 1 objects/1.7 GiB.                                      \n",
      "Copying file://../../data/additional_degraded_v1/fastqs/mp874-MV411_MEF2D_NT_SC_63-FLAG_MEF2D-r2_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "==> NOTE: You are uploading one or more large file(s), which would run          \n",
      "significantly faster if you enable parallel composite uploads. This\n",
      "feature can be enabled by editing the\n",
      "\"parallel_composite_upload_threshold\" value in your .boto\n",
      "configuration file. However, note that if you do this large files will\n",
      "be uploaded as `composite objects\n",
      "<https://cloud.google.com/storage/docs/composite-objects>`_,which\n",
      "means that any user who downloads such objects will need to have a\n",
      "compiled crcmod installed (see \"gsutil help crcmod\"). This is because\n",
      "without a compiled crcmod, computing checksums on composite objects is\n",
      "so slow that gsutil disables downloads of composite objects.\n",
      "\n",
      "/ [1 files][  1.7 GiB/  1.7 GiB]  104.6 MiB/s                                   \n",
      "Operation completed over 1 objects/1.7 GiB.                                      \n",
      "Copying file://../../data/additional_degraded_v1/fastqs/mp875-MV411_MEF2C_NT-FLAG_MEF2C-r1_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "==> NOTE: You are uploading one or more large file(s), which would run          \n",
      "significantly faster if you enable parallel composite uploads. This\n",
      "feature can be enabled by editing the\n",
      "\"parallel_composite_upload_threshold\" value in your .boto\n",
      "configuration file. However, note that if you do this large files will\n",
      "be uploaded as `composite objects\n",
      "<https://cloud.google.com/storage/docs/composite-objects>`_,which\n",
      "means that any user who downloads such objects will need to have a\n",
      "compiled crcmod installed (see \"gsutil help crcmod\"). This is because\n",
      "without a compiled crcmod, computing checksums on composite objects is\n",
      "so slow that gsutil disables downloads of composite objects.\n",
      "\n",
      "| [1 files][  2.1 GiB/  2.1 GiB]   95.0 MiB/s                                   \n",
      "Operation completed over 1 objects/2.1 GiB.                                      \n",
      "Copying file://../../data/additional_degraded_v1/fastqs/mp875-MV411_MEF2C_NT-FLAG_MEF2C-r1_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "==> NOTE: You are uploading one or more large file(s), which would run          \n",
      "significantly faster if you enable parallel composite uploads. This\n",
      "feature can be enabled by editing the\n",
      "\"parallel_composite_upload_threshold\" value in your .boto\n",
      "configuration file. However, note that if you do this large files will\n",
      "be uploaded as `composite objects\n",
      "<https://cloud.google.com/storage/docs/composite-objects>`_,which\n",
      "means that any user who downloads such objects will need to have a\n",
      "compiled crcmod installed (see \"gsutil help crcmod\"). This is because\n",
      "without a compiled crcmod, computing checksums on composite objects is\n",
      "so slow that gsutil disables downloads of composite objects.\n",
      "\n",
      "- [1 files][  2.1 GiB/  2.1 GiB]  110.6 MiB/s                                   \n",
      "Operation completed over 1 objects/2.1 GiB.                                      \n"
     ]
    }
   ],
   "source": [
    "for val in a[-4:]:\n",
    "    !gsutil cp ../../data/$project/fastqs/$val gs://amlproject/Chip/fastqs/\n",
    "    !rm ../../data/$project/fastqs/$val"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 76,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "found 2 files to rename\n"
     ]
    }
   ],
   "source": [
    "gcp.patternRN({'mp845-MV411_IRF2BP2_-INPUT-r1':'mp845-MV411-INPUT-r2'},'gs://amlproject/Chip/',wildcards=['**','.*'], test=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 87,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = {\n",
    "\"fastq_1\": [],\n",
    "\"fastq_2\": [],\n",
    "\"antibody\": [],\n",
    "\"group\": [],\n",
    "\"replicate\": [],\n",
    "\"control\": []\n",
    "}\n",
    "for val in h.grouped(a[:-4],2):\n",
    "    row = gsheet[gsheet.id==val[0].split('/')[-1].split('-')[0]]\n",
    "    df['group'].append(\"MV4\"+row['name'].values[0].split('-r')[0].split('-MV4')[1])\n",
    "    df['replicate'].append(row['name'].values[0].split('-r')[1])\n",
    "    df['fastq_1'].append(project+\"/fastqs/\"+val[0])\n",
    "    df['fastq_2'].append(project+\"/fastqs/\"+val[1])\n",
    "    df['antibody'].append(row['protein'].values[0])\n",
    "    df['control'].append(\"INPUT\")\n",
    "df['group'].append('INPUT')\n",
    "df['replicate'].append(1)\n",
    "df['fastq_1'].append('ref/mp845-MV411-INPUT-r2_R1.fastq.gz')\n",
    "df['fastq_2'].append('ref/mp845-MV411-INPUT-r2_R2.fastq.gz')\n",
    "df['antibody'].append(\"\")\n",
    "df['control'].append(\"\")\n",
    "df = pd.DataFrame(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 88,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>fastq_1</th>\n",
       "      <th>fastq_2</th>\n",
       "      <th>antibody</th>\n",
       "      <th>group</th>\n",
       "      <th>replicate</th>\n",
       "      <th>control</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>additional_degraded_v1/fastqs/mp858-MV411_RNP_...</td>\n",
       "      <td>additional_degraded_v1/fastqs/mp858-MV411_RNP_...</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>MV411_RNP_AAVS1-H3K27AC</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>additional_degraded_v1/fastqs/mp859-MV411_RNP_...</td>\n",
       "      <td>additional_degraded_v1/fastqs/mp859-MV411_RNP_...</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>MV411_RNP_AAVS1-H3K27AC</td>\n",
       "      <td>2</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>additional_degraded_v1/fastqs/mp860-MV411_RNP_...</td>\n",
       "      <td>additional_degraded_v1/fastqs/mp860-MV411_RNP_...</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>MV411_RNP_RUNX1-H3K27AC</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>additional_degraded_v1/fastqs/mp861-MV411_RNP_...</td>\n",
       "      <td>additional_degraded_v1/fastqs/mp861-MV411_RNP_...</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>MV411_RNP_RUNX1-H3K27AC</td>\n",
       "      <td>2</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>additional_degraded_v1/fastqs/mp862-MV411_RNP_...</td>\n",
       "      <td>additional_degraded_v1/fastqs/mp862-MV411_RNP_...</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>MV411_RNP_RUNX2-H3K27AC</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>additional_degraded_v1/fastqs/mp863-MV411_RNP_...</td>\n",
       "      <td>additional_degraded_v1/fastqs/mp863-MV411_RNP_...</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>MV411_RNP_RUNX2-H3K27AC</td>\n",
       "      <td>2</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>additional_degraded_v1/fastqs/mp864-MV411_RNP_...</td>\n",
       "      <td>additional_degraded_v1/fastqs/mp864-MV411_RNP_...</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>MV411_RNP_RUNX1_RUNX2-H3K27AC</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>additional_degraded_v1/fastqs/mp865-MV411_RNP_...</td>\n",
       "      <td>additional_degraded_v1/fastqs/mp865-MV411_RNP_...</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>MV411_RNP_RUNX1_RUNX2-H3K27AC</td>\n",
       "      <td>2</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>additional_degraded_v1/fastqs/mp866-MV411_RNP_...</td>\n",
       "      <td>additional_degraded_v1/fastqs/mp866-MV411_RNP_...</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>MV411_RNP_MEF2D-H3K27AC</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>additional_degraded_v1/fastqs/mp867-MV411_RNP_...</td>\n",
       "      <td>additional_degraded_v1/fastqs/mp867-MV411_RNP_...</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>MV411_RNP_MEF2D-H3K27AC</td>\n",
       "      <td>2</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>additional_degraded_v1/fastqs/mp868-MV411_RNP_...</td>\n",
       "      <td>additional_degraded_v1/fastqs/mp868-MV411_RNP_...</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>MV411_RNP_IRF8-H3K27AC</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>additional_degraded_v1/fastqs/mp869-MV411_RNP_...</td>\n",
       "      <td>additional_degraded_v1/fastqs/mp869-MV411_RNP_...</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>MV411_RNP_IRF8-H3K27AC</td>\n",
       "      <td>2</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>additional_degraded_v1/fastqs/mp870-MV411_RNP_...</td>\n",
       "      <td>additional_degraded_v1/fastqs/mp870-MV411_RNP_...</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>MV411_RNP_MYB-H3K27AC</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>additional_degraded_v1/fastqs/mp871-MV411_RNP_...</td>\n",
       "      <td>additional_degraded_v1/fastqs/mp871-MV411_RNP_...</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>MV411_RNP_MYB-H3K27AC</td>\n",
       "      <td>2</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>additional_degraded_v1/fastqs/mp872-MV411_RNP_...</td>\n",
       "      <td>additional_degraded_v1/fastqs/mp872-MV411_RNP_...</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>MV411_RNP_SPI1-H3K27AC</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>additional_degraded_v1/fastqs/mp873-MV411_RNP_...</td>\n",
       "      <td>additional_degraded_v1/fastqs/mp873-MV411_RNP_...</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>MV411_RNP_SPI1-H3K27AC</td>\n",
       "      <td>2</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>ref/mp845-MV411-INPUT-r2_R1.fastq.gz</td>\n",
       "      <td>ref/mp845-MV411-INPUT-r2_R2.fastq.gz</td>\n",
       "      <td></td>\n",
       "      <td>INPUT</td>\n",
       "      <td>1</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                              fastq_1  \\\n",
       "0   additional_degraded_v1/fastqs/mp858-MV411_RNP_...   \n",
       "1   additional_degraded_v1/fastqs/mp859-MV411_RNP_...   \n",
       "2   additional_degraded_v1/fastqs/mp860-MV411_RNP_...   \n",
       "3   additional_degraded_v1/fastqs/mp861-MV411_RNP_...   \n",
       "4   additional_degraded_v1/fastqs/mp862-MV411_RNP_...   \n",
       "5   additional_degraded_v1/fastqs/mp863-MV411_RNP_...   \n",
       "6   additional_degraded_v1/fastqs/mp864-MV411_RNP_...   \n",
       "7   additional_degraded_v1/fastqs/mp865-MV411_RNP_...   \n",
       "8   additional_degraded_v1/fastqs/mp866-MV411_RNP_...   \n",
       "9   additional_degraded_v1/fastqs/mp867-MV411_RNP_...   \n",
       "10  additional_degraded_v1/fastqs/mp868-MV411_RNP_...   \n",
       "11  additional_degraded_v1/fastqs/mp869-MV411_RNP_...   \n",
       "12  additional_degraded_v1/fastqs/mp870-MV411_RNP_...   \n",
       "13  additional_degraded_v1/fastqs/mp871-MV411_RNP_...   \n",
       "14  additional_degraded_v1/fastqs/mp872-MV411_RNP_...   \n",
       "15  additional_degraded_v1/fastqs/mp873-MV411_RNP_...   \n",
       "16               ref/mp845-MV411-INPUT-r2_R1.fastq.gz   \n",
       "\n",
       "                                              fastq_2 antibody  \\\n",
       "0   additional_degraded_v1/fastqs/mp858-MV411_RNP_...  H3K27AC   \n",
       "1   additional_degraded_v1/fastqs/mp859-MV411_RNP_...  H3K27AC   \n",
       "2   additional_degraded_v1/fastqs/mp860-MV411_RNP_...  H3K27AC   \n",
       "3   additional_degraded_v1/fastqs/mp861-MV411_RNP_...  H3K27AC   \n",
       "4   additional_degraded_v1/fastqs/mp862-MV411_RNP_...  H3K27AC   \n",
       "5   additional_degraded_v1/fastqs/mp863-MV411_RNP_...  H3K27AC   \n",
       "6   additional_degraded_v1/fastqs/mp864-MV411_RNP_...  H3K27AC   \n",
       "7   additional_degraded_v1/fastqs/mp865-MV411_RNP_...  H3K27AC   \n",
       "8   additional_degraded_v1/fastqs/mp866-MV411_RNP_...  H3K27AC   \n",
       "9   additional_degraded_v1/fastqs/mp867-MV411_RNP_...  H3K27AC   \n",
       "10  additional_degraded_v1/fastqs/mp868-MV411_RNP_...  H3K27AC   \n",
       "11  additional_degraded_v1/fastqs/mp869-MV411_RNP_...  H3K27AC   \n",
       "12  additional_degraded_v1/fastqs/mp870-MV411_RNP_...  H3K27AC   \n",
       "13  additional_degraded_v1/fastqs/mp871-MV411_RNP_...  H3K27AC   \n",
       "14  additional_degraded_v1/fastqs/mp872-MV411_RNP_...  H3K27AC   \n",
       "15  additional_degraded_v1/fastqs/mp873-MV411_RNP_...  H3K27AC   \n",
       "16               ref/mp845-MV411-INPUT-r2_R2.fastq.gz            \n",
       "\n",
       "                            group replicate control  \n",
       "0         MV411_RNP_AAVS1-H3K27AC         1   INPUT  \n",
       "1         MV411_RNP_AAVS1-H3K27AC         2   INPUT  \n",
       "2         MV411_RNP_RUNX1-H3K27AC         1   INPUT  \n",
       "3         MV411_RNP_RUNX1-H3K27AC         2   INPUT  \n",
       "4         MV411_RNP_RUNX2-H3K27AC         1   INPUT  \n",
       "5         MV411_RNP_RUNX2-H3K27AC         2   INPUT  \n",
       "6   MV411_RNP_RUNX1_RUNX2-H3K27AC         1   INPUT  \n",
       "7   MV411_RNP_RUNX1_RUNX2-H3K27AC         2   INPUT  \n",
       "8         MV411_RNP_MEF2D-H3K27AC         1   INPUT  \n",
       "9         MV411_RNP_MEF2D-H3K27AC         2   INPUT  \n",
       "10         MV411_RNP_IRF8-H3K27AC         1   INPUT  \n",
       "11         MV411_RNP_IRF8-H3K27AC         2   INPUT  \n",
       "12          MV411_RNP_MYB-H3K27AC         1   INPUT  \n",
       "13          MV411_RNP_MYB-H3K27AC         2   INPUT  \n",
       "14         MV411_RNP_SPI1-H3K27AC         1   INPUT  \n",
       "15         MV411_RNP_SPI1-H3K27AC         2   INPUT  \n",
       "16                          INPUT         1          "
      ]
     },
     "execution_count": 88,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 89,
   "metadata": {},
   "outputs": [],
   "source": [
    "df[df.columns[[3,4,0,1,2,5]]].to_csv('../nextflow/additional_degraded_v1_design.csv',index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "TIMESTAMP          \tDURATION      \tRUN NAME              \tSTATUS\tREVISION ID\tSESSION ID                          \tCOMMAND                                                                                                                                                                                                                                                          \r\n",
      "2020-02-10 22:39:18\t8m 25s        \tstupefied_crick       \tERR   \t21be314954 \t76ea5df0-153c-4e71-a59d-52c6112fda84\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --narrow_peak --input design.csv --genome GRCh38 --skip_preseq --max_cpus 24 -profile docker -w work                                                                      \r\n",
      "2020-02-10 22:40:37\t1m 6s         \tridiculous_hilbert    \tERR   \t21be314954 \t75004903-035d-4504-ab80-cab74b5acac4\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --narrow_peak --input design.csv --genome GRCh38 --skip_preseq --max_cpus 24 -profile docker -w work                                                                      \r\n",
      "2020-02-10 22:41:52\t1m 18s        \tintergalactic_mercator\tERR   \t21be314954 \tbf13ed87-be57-46c0-8a99-3939bba1a794\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --narrow_peak --input design.csv --genome GRCh38 --skip_preseq --max_cpus 24 -profile docker -w work                                                                      \r\n",
      "2020-02-10 22:52:31\t1d 15h 39m 50s\tgrave_baekeland       \tOK    \t21be314954 \tb554f82a-f7d1-42fd-8421-543518d2aa84\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --narrow_peak --input design.csv --genome GRCh38 --skip_preseq --max_cpus 24 -profile docker -w work                                                                      \r\n",
      "2020-03-10 17:24:20\t4h 47m 8s     \tfestering_lovelace    \tERR   \t21be314954 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --narrow_peak --input design_rep4.csv --genome GRCh38 --skip_preseq --max_cpus 24 -profile docker -w work                                                                 \r\n",
      "2020-03-11 15:16:55\t16s           \tcompassionate_raman   \tERR   \t21be314954 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --narrow_peak --input design_rep4.csv --genome GRCh38 --skip_preseq --max_cpus 24 -profile docker -w work -resume festering_lovelace                                      \r\n",
      "2020-03-11 15:17:44\t19h 57m 30s   \tsilly_thompson        \tOK    \t21be314954 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --narrow_peak --input design_rep4.csv --genome GRCh38 --skip_preseq --max_cpus 24 -profile docker -w work -resume festering_lovelace                                      \r\n",
      "2020-05-27 15:08:20\t-             \tridiculous_bardeen    \t-     \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh37 -resume                                                                                                                                                                        \r\n",
      "2020-05-27 15:09:40\t-             \tmagical_mirzakhani    \t-     \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh37 -resume                                                                                                                                                                        \r\n",
      "2020-05-27 16:54:22\t6m 55s        \tnice_tesla            \tERR   \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh37 -resume                                                                                                                                                                        \r\n",
      "2020-05-27 17:01:54\t1m 32s        \tagitated_bose         \tERR   \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh37 -resume                                                                                                                                                                        \r\n",
      "2020-05-27 17:03:49\t-             \tconfident_pauling     \t-     \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh38 -resume --genome --max_cpus 16                                                                                                                                                 \r\n",
      "2020-05-27 17:04:17\t-             \thopeful_watson        \t-     \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh38 -resume --max_cpus 16                                                                                                                                                          \r\n",
      "2020-05-27 17:42:01\t-             \telegant_liskov        \t-     \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh38 -resume --max_cpus 12                                                                                                                                                          \r\n",
      "2020-05-27 18:23:13\t3h 1m 56s     \tcurious_engelbart     \tERR   \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh38 -resume --max_cpus 12                                                                                                                                                          \r\n",
      "2020-05-27 21:25:55\t4m 43s        \tmad_bell              \tERR   \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh38 -resume --max_cpus 6                                                                                                                                                           \r\n",
      "2020-05-28 13:08:34\t11h 21m 20s   \tspecial_heyrovsky     \tOK    \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh38 -resume --max_cpus 6                                                                                                                                                           \r\n",
      "2020-08-04 17:36:59\t1m 59s        \tbackstabbing_hilbert  \tERR   \t21be314954 \t360ae07b-fb63-4b50-b834-7ba2af75b988\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --broad_peak --input ../AMLproject/nextflow/additional_degraded_v1_design.csv --genome GRCh38 --skip_preseq --max_cpus 8 -profile docker -w work                          \r\n",
      "2020-08-04 17:40:57\t1m 14s        \tfurious_liskov        \tERR   \t21be314954 \tb16ce591-36eb-4bc2-901b-840bbe4c73bd\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --broad_peak --input ../AMLproject/nextflow/additional_degraded_v1_design.csv --genome GRCh38 --skip_preseq --max_cpus 8 -profile docker -w work                          \r\n",
      "2020-08-04 17:42:40\t17.5s         \tridiculous_kilby      \tERR   \t21be314954 \tc3ff2f08-8186-4048-b643-1d615eb7dcc1\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --broad_peak --input ../AMLproject/nextflow/additional_degraded_v1_design.csv --genome GRCh38 --skip_preseq --max_cpus 6 -profile docker -w work                          \r\n",
      "2020-08-04 18:23:46\t6.1s          \tlethal_celsius        \tERR   \t21be314954 \t58d0cab0-e38f-4cd7-97a9-d897c5d2b66d\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --broad_peak --input ../AMLproject/nextflow/additional_degraded_v1_design.csv --genome GRCh38 --skip_preseq --max_cpus 6 -profile docker -w work                          \r\n",
      "2020-08-04 18:23:57\t4h 44m 38s    \tinfallible_woese      \tERR   \t21be314954 \t9d191f62-68e0-4e0b-8c13-2e7c1b32ca2f\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --broad_peak --input ../AMLproject/nextflow/additional_degraded_v1_design.csv --genome GRCh38 --skip_preseq --max_cpus 16 -profile docker -w work                         \r\n",
      "2020-08-04 23:21:36\t16h 4m 40s    \texotic_bartik         \tERR   \t21be314954 \t9d191f62-68e0-4e0b-8c13-2e7c1b32ca2f\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --broad_peak --input ../AMLproject/nextflow/additional_degraded_v1_design.csv --genome GRCh38 --skip_preseq --max_cpus 16 -profile docker -w work -resume infallible_woese\r\n"
     ]
    }
   ],
   "source": [
    "! cd ../../data/ && sudo ../nextflow log ## to get access to the previous runs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "N E X T F L O W  ~  version 19.10.0\n",
      "Launching `nf-core/chipseq` [wise_legentil] - revision: 21be314954 [master]\n",
      "NOTE: Your local project version looks outdated - a different revision is available in the remote repository [0f487ed76d]\n",
      "-\u001b[2m--------------------------------------------------\u001b[0m-\n",
      "                                        \u001b[0;32m,--.\u001b[0;30m/\u001b[0;32m,-.\u001b[0m\n",
      "\u001b[0;34m        ___     __   __   __   ___     \u001b[0;32m/,-._.--~'\u001b[0m\n",
      "\u001b[0;34m  |\\ | |__  __ /  ` /  \\ |__) |__         \u001b[0;33m}  {\u001b[0m\n",
      "\u001b[0;34m  | \\| |       \\__, \\__/ |  \\ |___     \u001b[0;32m\\`-._,-`-,\u001b[0m\n",
      "                                        \u001b[0;32m`._,._,'\u001b[0m\n",
      "\u001b[0;35m  nf-core/chipseq v1.1.0\u001b[0m\n",
      "-\u001b[2m--------------------------------------------------\u001b[0m-\n",
      "Run Name            : wise_legentil\n",
      "Data Type           : Paired-End\n",
      "Design File         : ../AMLproject/nextflow/additional_degraded_v1_design.csv\n",
      "Genome              : GRCh38\n",
      "Fasta File          : s3://ngi-igenomes/igenomes//Homo_sapiens/NCBI/GRCh38/Sequence/WholeGenomeFasta/genome.fa\n",
      "GTF File            : s3://ngi-igenomes/igenomes//Homo_sapiens/NCBI/GRCh38/Annotation/Genes/genes.gtf\n",
      "Gene BED File       : s3://ngi-igenomes/igenomes//Homo_sapiens/NCBI/GRCh38/Annotation/Genes/genes.bed\n",
      "BWA Index           : s3://ngi-igenomes/igenomes//Homo_sapiens/NCBI/GRCh38/Sequence/BWAIndex/genome.fa\n",
      "Blacklist BED       : /root/.nextflow/assets/nf-core/chipseq/assets/blacklists/hg38-blacklist.bed\n",
      "MACS2 Genome Size   : 2.7e9\n",
      "Min Consensus Reps  : 1\n",
      "MACS2 Narrow Peaks  : No\n",
      "MACS2 Broad Cutoff  : 0.1\n",
      "Trim R1             : 0 bp\n",
      "Trim R2             : 0 bp\n",
      "Trim 3' R1          : 0 bp\n",
      "Trim 3' R2          : 0 bp\n",
      "NextSeq Trim        : 0 bp\n",
      "Sequencing Center   : DFCI\n",
      "Fingerprint Bins    : 500000\n",
      "Save Genome Index   : No\n",
      "Skip Preseq         : Yes\n",
      "Max Resources       : 128 GB memory, 16 cpus, 10d time per job\n",
      "Container           : docker - nfcore/chipseq:1.1.0\n",
      "Output Dir          : ./results\n",
      "Launch Dir          : /home/jeremie/data\n",
      "Working Dir         : /home/jeremie/data/work\n",
      "Script Dir          : /root/.nextflow/assets/nf-core/chipseq\n",
      "User                : root\n",
      "Config Profile      : docker\n",
      "E-mail Address      : jkobject@gmail.com\n",
      "E-mail on failure   : false\n",
      "MultiQC Max Size    : 25 MB\n",
      "-\u001b[2m--------------------------------------------------\u001b[0m-\n",
      "[-        ] process > CheckDesign      -\u001b[K\n",
      "[-        ] process > MakeTSSBED       -\u001b[K\n",
      "[-        ] process > MakeGenomeFilter -\u001b[K\n",
      "[-        ] process > FastQC           -\u001b[K\n",
      "[-        ] process > TrimGalore       -\u001b[K\n",
      "[-        ] process > BWAMem           -\u001b[K\n",
      "\u001b[7A\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[-        ] process > MakeTSSBED                     -\u001b[K\n",
      "[-        ] process > MakeGenomeFilter               -\u001b[K\n",
      "[-        ] process > FastQC                         -\u001b[K\n",
      "[-        ] process > TrimGalore                     -\u001b[K\n",
      "[-        ] process > BWAMem                         -\u001b[K\n",
      "[-        ] process > SortBAM                        -\u001b[K\n",
      "[-        ] process > MergeBAM                       -\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "\u001b[20A\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[-        ] process > MakeGenomeFilter               -\u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[ee/468381] process > TrimGalore (MV411_RNP_RUNX1... [100%] 7 of 7, cached: 7\u001b[K\n",
      "[-        ] process > BWAMem                         -\u001b[K\n",
      "[-        ] process > SortBAM                        -\u001b[K\n",
      "[-        ] process > MergeBAM                       -\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[-        ] process > get_software_versions          -\u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "\u001b[26A\n",
      "executor >  local (1)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[-        ] process > MakeGenomeFilter               -\u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[-        ] process > BWAMem                         -\u001b[K\n",
      "[-        ] process > SortBAM                        -\u001b[K\n",
      "[-        ] process > MergeBAM                       -\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (1)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[-        ] process > MakeGenomeFilter               -\u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[8f/3989b4] process > BWAMem (MV411_RNP_AAVS1-H3K... [100%] 1 of 1, cached: 1\u001b[K\n",
      "[-        ] process > SortBAM                        -\u001b[K\n",
      "[-        ] process > MergeBAM                       -\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (1)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[-        ] process > MergeBAM                       -\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (3)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/8c0a93] process > MergeBAM (MV411_RNP_RUNX2-H... [ 85%] 11 of 13, cached: 11\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (3)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/8c0a93] process > MergeBAM (MV411_RNP_RUNX2-H... [ 85%] 11 of 13, cached: 11\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (4)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/8c0a93] process > MergeBAM (MV411_RNP_RUNX2-H... [ 86%] 12 of 14, cached: 11\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (5)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[c6/4b33eb] process > MergeBAM (MV411_RNP_MEF2D-H... [ 87%] 13 of 15, cached: 11\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (6)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[fc/cf6140] process > MergeBAM (MV411_RNP_SPI1-H3... [ 88%] 14 of 16, cached: 11\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (7)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[c6/4b33eb] process > MergeBAM (MV411_RNP_MEF2D-H... [ 88%] 15 of 17, cached: 11\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (8)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[12/a07a72] process > MergeBAM (MV411_RNP_MYB-H3K... [ 94%] 16 of 17, cached: 11\u001b[K\n",
      "[84/574208] process > MergeBAMFilter (MV411_RNP_S... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (9)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[66/a24586] process > MergeBAMFilter (MV411_RNP_R... [  0%] 0 of 2\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (10)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/83ab58] process > MergeBAMFilter (MV411_RNP_M... [ 33%] 1 of 3\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (11)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[84/574208] process > MergeBAMFilter (MV411_RNP_S... [ 50%] 2 of 4\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (12)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/83ab58] process > MergeBAMFilter (MV411_RNP_M... [ 60%] 3 of 5\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (13)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[60/4767d1] process > MergeBAMFilter (MV411_RNP_R... [ 67%] 4 of 6\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (14)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[62/3caa87] process > MergeBAMFilter (MV411_RNP_I... [ 57%] 4 of 7\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (14)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[60/4767d1] process > MergeBAMFilter (MV411_RNP_R... [ 71%] 5 of 7\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (15)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[e1/d4ac46] process > MergeBAMFilter (MV411_RNP_A... [ 63%] 5 of 8\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (15)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[9f/a848c2] process > MergeBAMFilter (INPUT_R1)      [ 75%] 6 of 8\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (16)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[ff/e09463] process > MergeBAMFilter (MV411_RNP_M... [ 78%] 7 of 9\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (17)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[77/a464ff] process > MergeBAMFilter (MV411_RNP_I... [ 80%] 8 of 10\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (18)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[ff/e09463] process > MergeBAMFilter (MV411_RNP_M... [ 82%] 9 of 11\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (18)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[77/a464ff] process > MergeBAMFilter (MV411_RNP_I... [ 91%] 10 of 11\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (19)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[e2/12ed6b] process > MergeBAMFilter (MV411_RNP_R... [ 83%] 10 of 12\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (20)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[07/b34e79] process > MergeBAMFilter (MV411_RNP_R... [ 85%] 11 of 13\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (21)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[9a/a02fdf] process > MergeBAMFilter (MV411_RNP_S... [ 86%] 12 of 14\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (22)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[65/4e0731] process > MergeBAMFilter (MV411_RNP_M... [ 87%] 13 of 15\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (23)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[e0/a67acd] process > MergeBAMFilter (MV411_RNP_M... [ 88%] 14 of 16\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (24)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [ 88%] 15 of 17\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (25)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[e0/a67acd] process > MergeBAMFilter (MV411_RNP_M... [ 94%] 16 of 17\u001b[K\n",
      "[21/51df00] process > MergeBAMRemoveOrphan (MV411... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (26)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[e0/a67acd] process > MergeBAMFilter (MV411_RNP_M... [ 94%] 16 of 17\u001b[K\n",
      "[21/51df00] process > MergeBAMRemoveOrphan (MV411... [ 50%] 1 of 2\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (27)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[30/1ef981] process > MergeBAMRemoveOrphan (MV411... [ 33%] 1 of 3\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (28)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[57/199f77] process > MergeBAMRemoveOrphan (MV411... [ 50%] 2 of 4\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (29)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[30/1ef981] process > MergeBAMRemoveOrphan (MV411... [ 60%] 3 of 5\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (30)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[bf/2620e0] process > MergeBAMRemoveOrphan (MV411... [ 67%] 4 of 6\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (31)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[c7/9d9b0a] process > MergeBAMRemoveOrphan (MV411... [ 71%] 5 of 7\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (32)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[70/ef513d] process > MergeBAMRemoveOrphan (INPUT... [ 75%] 6 of 8\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (33)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[63/892aee] process > MergeBAMRemoveOrphan (MV411... [ 78%] 7 of 9\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (34)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[b3/9db05d] process > MergeBAMRemoveOrphan (MV411... [ 80%] 8 of 10\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (35)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[9f/c2beee] process > MergeBAMRemoveOrphan (MV411... [ 82%] 9 of 11\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (36)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[22/110bf7] process > MergeBAMRemoveOrphan (MV411... [ 83%] 10 of 12\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (37)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[1b/49d44f] process > MergeBAMRemoveOrphan (MV411... [ 85%] 11 of 13\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (38)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[ba/00ce48] process > MergeBAMRemoveOrphan (MV411... [ 79%] 11 of 14\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (38)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[08/ccd392] process > MergeBAMRemoveOrphan (MV411... [ 86%] 12 of 14\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (39)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[9a/fa4609] process > MergeBAMRemoveOrphan (MV411... [ 87%] 13 of 15\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (40)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[04/2a4108] process > MergeBAMRemoveOrphan (MV411... [ 81%] 13 of 16\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (40)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[ba/00ce48] process > MergeBAMRemoveOrphan (MV411... [ 88%] 14 of 16\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (41)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[c0/52c2db] process > MergeBAMRemoveOrphan (MV411... [ 94%] 15 of 16\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[68/d5d1f4] process > PhantomPeakQualTools (MV411... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (42)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[04/2a4108] process > MergeBAMRemoveOrphan (MV411... [100%] 16 of 16\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[41/1dc1b3] process > BigWig (MV411_RNP_RUNX1_RUN... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[68/d5d1f4] process > PhantomPeakQualTools (MV411... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (43)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[04/2a4108] process > MergeBAMRemoveOrphan (MV411... [100%] 16 of 16\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[18/e3205c] process > CollectMultipleMetrics (MV4... [  0%] 0 of 1\u001b[K\n",
      "[41/1dc1b3] process > BigWig (MV411_RNP_RUNX1_RUN... [100%] 1 of 1\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[68/d5d1f4] process > PhantomPeakQualTools (MV411... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (44)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [ 94%] 16 of 17\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[18/e3205c] process > CollectMultipleMetrics (MV4... [  0%] 0 of 1\u001b[K\n",
      "[41/1dc1b3] process > BigWig (MV411_RNP_RUNX1_RUN... [100%] 1 of 1\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[68/d5d1f4] process > PhantomPeakQualTools (MV411... [100%] 1 of 1\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (45)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [ 94%] 16 of 17\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[18/e3205c] process > CollectMultipleMetrics (MV4... [100%] 1 of 1\u001b[K\n",
      "[fa/7ca520] process > BigWig (MV411_RNP_SPI1-H3K2... [ 50%] 1 of 2\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[68/d5d1f4] process > PhantomPeakQualTools (MV411... [100%] 1 of 1\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (46)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [ 94%] 16 of 17\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[18/e3205c] process > CollectMultipleMetrics (MV4... [100%] 1 of 1\u001b[K\n",
      "[fa/7ca520] process > BigWig (MV411_RNP_SPI1-H3K2... [100%] 2 of 2\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[c1/b1a581] process > PhantomPeakQualTools (MV411... [ 50%] 1 of 2\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (47)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[ea/949a3e] process > CollectMultipleMetrics (MV4... [ 50%] 1 of 2\u001b[K\n",
      "[fa/7ca520] process > BigWig (MV411_RNP_SPI1-H3K2... [100%] 2 of 2\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[c1/b1a581] process > PhantomPeakQualTools (MV411... [ 50%] 1 of 2\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (48)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[ea/949a3e] process > CollectMultipleMetrics (MV4... [100%] 2 of 2\u001b[K\n",
      "[55/063141] process > BigWig (MV411_RNP_MEF2D-H3K... [ 67%] 2 of 3\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[c1/b1a581] process > PhantomPeakQualTools (MV411... [ 50%] 1 of 2\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (49)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[ea/949a3e] process > CollectMultipleMetrics (MV4... [100%] 2 of 2\u001b[K\n",
      "[55/063141] process > BigWig (MV411_RNP_MEF2D-H3K... [100%] 3 of 3\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[9d/c498be] process > PhantomPeakQualTools (MV411... [ 33%] 1 of 3\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (50)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[96/83ce50] process > CollectMultipleMetrics (MV4... [ 67%] 2 of 3\u001b[K\n",
      "[55/063141] process > BigWig (MV411_RNP_MEF2D-H3K... [100%] 3 of 3\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[c1/b1a581] process > PhantomPeakQualTools (MV411... [ 67%] 2 of 3\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (51)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[96/83ce50] process > CollectMultipleMetrics (MV4... [100%] 3 of 3\u001b[K\n",
      "[55/063141] process > BigWig (MV411_RNP_MEF2D-H3K... [100%] 3 of 3\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[65/278ab6] process > PhantomPeakQualTools (MV411... [ 50%] 2 of 4\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (52)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[96/83ce50] process > CollectMultipleMetrics (MV4... [100%] 3 of 3\u001b[K\n",
      "[c4/0c2b62] process > BigWig (MV411_RNP_AAVS1-H3K... [ 75%] 3 of 4\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[9d/c498be] process > PhantomPeakQualTools (MV411... [ 75%] 3 of 4\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (53)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[24/913f78] process > CollectMultipleMetrics (MV4... [ 75%] 3 of 4\u001b[K\n",
      "[c4/0c2b62] process > BigWig (MV411_RNP_AAVS1-H3K... [100%] 4 of 4\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[9d/c498be] process > PhantomPeakQualTools (MV411... [ 75%] 3 of 4\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (54)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[24/913f78] process > CollectMultipleMetrics (MV4... [ 75%] 3 of 4\u001b[K\n",
      "[c4/0c2b62] process > BigWig (MV411_RNP_AAVS1-H3K... [100%] 4 of 4\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[65/278ab6] process > PhantomPeakQualTools (MV411... [ 80%] 4 of 5\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (55)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[24/913f78] process > CollectMultipleMetrics (MV4... [100%] 4 of 4\u001b[K\n",
      "[26/8106f4] process > BigWig (MV411_RNP_RUNX1-H3K... [ 80%] 4 of 5\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[65/278ab6] process > PhantomPeakQualTools (MV411... [ 80%] 4 of 5\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (56)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[36/03ab9d] process > CollectMultipleMetrics (MV4... [ 80%] 4 of 5\u001b[K\n",
      "[26/8106f4] process > BigWig (MV411_RNP_RUNX1-H3K... [100%] 5 of 5\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[65/278ab6] process > PhantomPeakQualTools (MV411... [ 80%] 4 of 5\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (57)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[36/03ab9d] process > CollectMultipleMetrics (MV4... [100%] 5 of 5\u001b[K\n",
      "[26/8106f4] process > BigWig (MV411_RNP_RUNX1-H3K... [100%] 5 of 5\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[41/509b9d] process > PhantomPeakQualTools (INPUT... [ 67%] 4 of 6\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (58)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[36/03ab9d] process > CollectMultipleMetrics (MV4... [100%] 5 of 5\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[b6/1c51c9] process > PhantomPeakQualTools (MV411... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (59)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[36/03ab9d] process > CollectMultipleMetrics (MV4... [100%] 5 of 5\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[b6/1c51c9] process > PhantomPeakQualTools (MV411... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[df/af3c12] process > MACSCallPeak (MV411_RNP_AAV... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (60)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[36/03ab9d] process > CollectMultipleMetrics (MV4... [100%] 5 of 5\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[b6/1c51c9] process > PhantomPeakQualTools (MV411... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[df/af3c12] process > MACSCallPeak (MV411_RNP_AAV... [ 50%] 1 of 2\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (60)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[36/03ab9d] process > CollectMultipleMetrics (MV4... [100%] 5 of 5\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[b6/1c51c9] process > PhantomPeakQualTools (MV411... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[df/af3c12] process > MACSCallPeak (MV411_RNP_AAV... [ 50%] 1 of 2\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "Staging foreign file: s3://ngi-igenomes/igenomes/Homo_sapiens/NCBI/GRCh38/Annotation/Genes/genes.gtf\u001b[K\n",
      "\u001b[29A\n",
      "executor >  local (61)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[36/03ab9d] process > CollectMultipleMetrics (MV4... [100%] 5 of 5\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[41/509b9d] process > PhantomPeakQualTools (INPUT... [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[ec/449ea1] process > MACSCallPeak (MV411_RNP_RUN... [ 33%] 1 of 3\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[K\n",
      "\u001b[29A\n",
      "executor >  local (62)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[36/03ab9d] process > CollectMultipleMetrics (MV4... [100%] 5 of 5\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[41/509b9d] process > PhantomPeakQualTools (INPUT... [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[ca/eb457d] process > MACSCallPeak (MV411_RNP_MEF... [ 50%] 2 of 4\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (63)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[36/03ab9d] process > CollectMultipleMetrics (MV4... [100%] 5 of 5\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[41/509b9d] process > PhantomPeakQualTools (INPUT... [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[ec/449ea1] process > MACSCallPeak (MV411_RNP_RUN... [ 60%] 3 of 5\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (64)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[6f/0c32f5] process > CollectMultipleMetrics (INP... [ 83%] 5 of 6\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[41/509b9d] process > PhantomPeakQualTools (INPUT... [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[99/4577c5] process > MACSCallPeak (MV411_RNP_RUN... [ 80%] 4 of 5\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (65)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[6f/0c32f5] process > CollectMultipleMetrics (INP... [100%] 6 of 6\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[8c/6222bd] process > PhantomPeakQualTools (MV411... [ 86%] 6 of 7\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[99/4577c5] process > MACSCallPeak (MV411_RNP_RUN... [ 80%] 4 of 5\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (66)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[6f/0c32f5] process > CollectMultipleMetrics (INP... [100%] 6 of 6\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[8c/6222bd] process > PhantomPeakQualTools (MV411... [ 86%] 6 of 7\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[74/d6d890] process > MACSCallPeak (MV411_RNP_SPI... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (67)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[6f/0c32f5] process > CollectMultipleMetrics (INP... [100%] 6 of 6\u001b[K\n",
      "[7f/a5a2c1] process > BigWig (MV411_RNP_AAVS1-H3K... [ 86%] 6 of 7\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[8c/6222bd] process > PhantomPeakQualTools (MV411... [100%] 7 of 7\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[74/d6d890] process > MACSCallPeak (MV411_RNP_SPI... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (68)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[77/85e402] process > CollectMultipleMetrics (MV4... [ 86%] 6 of 7\u001b[K\n",
      "[7f/a5a2c1] process > BigWig (MV411_RNP_AAVS1-H3K... [100%] 7 of 7\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[8c/6222bd] process > PhantomPeakQualTools (MV411... [100%] 7 of 7\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[74/d6d890] process > MACSCallPeak (MV411_RNP_SPI... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (69)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[77/85e402] process > CollectMultipleMetrics (MV4... [100%] 7 of 7\u001b[K\n",
      "[83/f433d6] process > BigWig (MV411_RNP_MYB-H3K27... [ 88%] 7 of 8\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[8c/6222bd] process > PhantomPeakQualTools (MV411... [100%] 7 of 7\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[74/d6d890] process > MACSCallPeak (MV411_RNP_SPI... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (70)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[77/85e402] process > CollectMultipleMetrics (MV4... [100%] 7 of 7\u001b[K\n",
      "[83/f433d6] process > BigWig (MV411_RNP_MYB-H3K27... [100%] 8 of 8\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[83/e5aeda] process > PhantomPeakQualTools (MV411... [ 88%] 7 of 8\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[74/d6d890] process > MACSCallPeak (MV411_RNP_SPI... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (71)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[77/85e402] process > CollectMultipleMetrics (MV4... [100%] 7 of 7\u001b[K\n",
      "[83/f433d6] process > BigWig (MV411_RNP_MYB-H3K27... [100%] 8 of 8\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[83/e5aeda] process > PhantomPeakQualTools (MV411... [ 88%] 7 of 8\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[c5/1a276a] process > MACSCallPeak (MV411_RNP_MYB... [ 71%] 5 of 7\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (71)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[77/85e402] process > CollectMultipleMetrics (MV4... [100%] 7 of 7\u001b[K\n",
      "[83/f433d6] process > BigWig (MV411_RNP_MYB-H3K27... [100%] 8 of 8\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[83/e5aeda] process > PhantomPeakQualTools (MV411... [ 88%] 7 of 8\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[aa/9e4d91] process > MACSCallPeak (MV411_RNP_AAV... [ 86%] 6 of 7\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (72)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[f7/870fa3] process > CollectMultipleMetrics (MV4... [ 88%] 7 of 8\u001b[K\n",
      "[83/f433d6] process > BigWig (MV411_RNP_MYB-H3K27... [100%] 8 of 8\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[83/e5aeda] process > PhantomPeakQualTools (MV411... [100%] 8 of 8\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[aa/9e4d91] process > MACSCallPeak (MV411_RNP_AAV... [ 86%] 6 of 7\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (73)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[f7/870fa3] process > CollectMultipleMetrics (MV4... [100%] 8 of 8\u001b[K\n",
      "[83/f433d6] process > BigWig (MV411_RNP_MYB-H3K27... [100%] 8 of 8\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[bc/873392] process > PhantomPeakQualTools (MV411... [ 89%] 8 of 9\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[aa/9e4d91] process > MACSCallPeak (MV411_RNP_AAV... [ 86%] 6 of 7\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (74)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[f7/870fa3] process > CollectMultipleMetrics (MV4... [100%] 8 of 8\u001b[K\n",
      "[be/8c5392] process > BigWig (MV411_RNP_IRF8-H3K2... [ 89%] 8 of 9\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[bc/873392] process > PhantomPeakQualTools (MV411... [ 89%] 8 of 9\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[c5/1a276a] process > MACSCallPeak (MV411_RNP_MYB... [100%] 7 of 7\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (75)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[f7/870fa3] process > CollectMultipleMetrics (MV4... [100%] 8 of 8\u001b[K\n",
      "[be/8c5392] process > BigWig (MV411_RNP_IRF8-H3K2... [ 89%] 8 of 9\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[bc/873392] process > PhantomPeakQualTools (MV411... [100%] 9 of 9\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[47/040371] process > MACSCallPeak (MV411_RNP_IRF... [ 88%] 7 of 8\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (76)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[eb/85c928] process > CollectMultipleMetrics (MV4... [ 89%] 8 of 9\u001b[K\n",
      "[be/8c5392] process > BigWig (MV411_RNP_IRF8-H3K2... [100%] 9 of 9\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[bc/873392] process > PhantomPeakQualTools (MV411... [100%] 9 of 9\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[47/040371] process > MACSCallPeak (MV411_RNP_IRF... [ 88%] 7 of 8\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (77)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[eb/85c928] process > CollectMultipleMetrics (MV4... [100%] 9 of 9\u001b[K\n",
      "[be/8c5392] process > BigWig (MV411_RNP_IRF8-H3K2... [100%] 9 of 9\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/001577] process > PhantomPeakQualTools (MV411... [ 90%] 9 of 10\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[47/040371] process > MACSCallPeak (MV411_RNP_IRF... [ 88%] 7 of 8\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (78)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[eb/85c928] process > CollectMultipleMetrics (MV4... [100%] 9 of 9\u001b[K\n",
      "[4e/c46492] process > BigWig (MV411_RNP_RUNX1_RUN... [ 90%] 9 of 10\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/001577] process > PhantomPeakQualTools (MV411... [100%] 10 of 10\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[47/040371] process > MACSCallPeak (MV411_RNP_IRF... [ 88%] 7 of 8\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (79)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[eb/85c928] process > CollectMultipleMetrics (MV4... [100%] 9 of 9\u001b[K\n",
      "[4e/c46492] process > BigWig (MV411_RNP_RUNX1_RUN... [100%] 10 of 10\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/001577] process > PhantomPeakQualTools (MV411... [100%] 10 of 10\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[3f/dba7c2] process > MACSCallPeak (MV411_RNP_RUN... [ 78%] 7 of 9\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (80)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[56/e75566] process > CollectMultipleMetrics (MV4... [ 90%] 9 of 10\u001b[K\n",
      "[4e/c46492] process > BigWig (MV411_RNP_RUNX1_RUN... [100%] 10 of 10\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/001577] process > PhantomPeakQualTools (MV411... [100%] 10 of 10\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[47/040371] process > MACSCallPeak (MV411_RNP_IRF... [ 89%] 8 of 9\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (81)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[56/e75566] process > CollectMultipleMetrics (MV4... [100%] 10 of 10\u001b[K\n",
      "[60/3850df] process > BigWig (MV411_RNP_IRF8-H3K2... [ 91%] 10 of 11\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/001577] process > PhantomPeakQualTools (MV411... [100%] 10 of 10\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[47/040371] process > MACSCallPeak (MV411_RNP_IRF... [ 89%] 8 of 9\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (82)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[56/e75566] process > CollectMultipleMetrics (MV4... [100%] 10 of 10\u001b[K\n",
      "[60/3850df] process > BigWig (MV411_RNP_IRF8-H3K2... [100%] 11 of 11\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[1c/07bf5e] process > PhantomPeakQualTools (MV411... [ 91%] 10 of 11\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[47/040371] process > MACSCallPeak (MV411_RNP_IRF... [ 89%] 8 of 9\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (83)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[56/e75566] process > CollectMultipleMetrics (MV4... [100%] 10 of 10\u001b[K\n",
      "[60/3850df] process > BigWig (MV411_RNP_IRF8-H3K2... [100%] 11 of 11\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[1c/07bf5e] process > PhantomPeakQualTools (MV411... [ 91%] 10 of 11\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[3f/dba7c2] process > MACSCallPeak (MV411_RNP_RUN... [ 90%] 9 of 10\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (84)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[af/724a3d] process > CollectMultipleMetrics (MV4... [ 91%] 10 of 11\u001b[K\n",
      "[60/3850df] process > BigWig (MV411_RNP_IRF8-H3K2... [100%] 11 of 11\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[1c/07bf5e] process > PhantomPeakQualTools (MV411... [100%] 11 of 11\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[3f/dba7c2] process > MACSCallPeak (MV411_RNP_RUN... [ 90%] 9 of 10\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (85)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[af/724a3d] process > CollectMultipleMetrics (MV4... [100%] 11 of 11\u001b[K\n",
      "[f7/5fa48c] process > BigWig (MV411_RNP_RUNX2-H3K... [ 92%] 11 of 12\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[1c/07bf5e] process > PhantomPeakQualTools (MV411... [100%] 11 of 11\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[3f/dba7c2] process > MACSCallPeak (MV411_RNP_RUN... [ 90%] 9 of 10\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (86)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[af/724a3d] process > CollectMultipleMetrics (MV4... [100%] 11 of 11\u001b[K\n",
      "[f7/5fa48c] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 12 of 12\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[21/517648] process > PhantomPeakQualTools (MV411... [ 92%] 11 of 12\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[3f/dba7c2] process > MACSCallPeak (MV411_RNP_RUN... [ 90%] 9 of 10\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (87)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[af/724a3d] process > CollectMultipleMetrics (MV4... [100%] 11 of 11\u001b[K\n",
      "[f7/5fa48c] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 12 of 12\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[21/517648] process > PhantomPeakQualTools (MV411... [ 92%] 11 of 12\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[51/147018] process > MACSCallPeak (MV411_RNP_IRF... [ 91%] 10 of 11\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (88)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[be/e4f2a1] process > CollectMultipleMetrics (MV4... [ 92%] 11 of 12\u001b[K\n",
      "[f7/5fa48c] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 12 of 12\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[21/517648] process > PhantomPeakQualTools (MV411... [100%] 12 of 12\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[51/147018] process > MACSCallPeak (MV411_RNP_IRF... [ 91%] 10 of 11\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (89)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[be/e4f2a1] process > CollectMultipleMetrics (MV4... [100%] 12 of 12\u001b[K\n",
      "[eb/3a1856] process > BigWig (MV411_RNP_RUNX1-H3K... [ 92%] 12 of 13\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[21/517648] process > PhantomPeakQualTools (MV411... [100%] 12 of 12\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[51/147018] process > MACSCallPeak (MV411_RNP_IRF... [ 91%] 10 of 11\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (90)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[be/e4f2a1] process > CollectMultipleMetrics (MV4... [100%] 12 of 12\u001b[K\n",
      "[eb/3a1856] process > BigWig (MV411_RNP_RUNX1-H3K... [100%] 13 of 13\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[21/517648] process > PhantomPeakQualTools (MV411... [100%] 12 of 12\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[d8/0be403] process > MACSCallPeak (MV411_RNP_RUN... [ 83%] 10 of 12\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (91)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[be/e4f2a1] process > CollectMultipleMetrics (MV4... [100%] 12 of 12\u001b[K\n",
      "[eb/3a1856] process > BigWig (MV411_RNP_RUNX1-H3K... [100%] 13 of 13\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[ce/6b94f7] process > PhantomPeakQualTools (MV411... [ 92%] 12 of 13\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[99/a486f8] process > MACSCallPeak (MV411_RNP_RUN... [ 92%] 11 of 12\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (92)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[58/9c859d] process > CollectMultipleMetrics (MV4... [ 92%] 12 of 13\u001b[K\n",
      "[eb/3a1856] process > BigWig (MV411_RNP_RUNX1-H3K... [100%] 13 of 13\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[ce/6b94f7] process > PhantomPeakQualTools (MV411... [ 92%] 12 of 13\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[d8/0be403] process > MACSCallPeak (MV411_RNP_RUN... [100%] 12 of 12\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (93)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[58/9c859d] process > CollectMultipleMetrics (MV4... [100%] 13 of 13\u001b[K\n",
      "[eb/3a1856] process > BigWig (MV411_RNP_RUNX1-H3K... [100%] 13 of 13\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[1b/b9c016] process > PhantomPeakQualTools (MV411... [ 86%] 12 of 14\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[d8/0be403] process > MACSCallPeak (MV411_RNP_RUN... [100%] 12 of 12\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (94)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[58/9c859d] process > CollectMultipleMetrics (MV4... [100%] 13 of 13\u001b[K\n",
      "[eb/3a1856] process > BigWig (MV411_RNP_RUNX1-H3K... [100%] 13 of 13\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[ce/6b94f7] process > PhantomPeakQualTools (MV411... [ 93%] 13 of 14\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[46/17f258] process > MACSCallPeak (MV411_RNP_SPI... [ 92%] 12 of 13\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (95)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[58/9c859d] process > CollectMultipleMetrics (MV4... [100%] 13 of 13\u001b[K\n",
      "[e7/594109] process > BigWig (MV411_RNP_SPI1-H3K2... [ 93%] 13 of 14\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[1b/b9c016] process > PhantomPeakQualTools (MV411... [100%] 14 of 14\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[46/17f258] process > MACSCallPeak (MV411_RNP_SPI... [ 92%] 12 of 13\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (96)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[39/7b314d] process > CollectMultipleMetrics (MV4... [ 93%] 13 of 14\u001b[K\n",
      "[e7/594109] process > BigWig (MV411_RNP_SPI1-H3K2... [100%] 14 of 14\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[1b/b9c016] process > PhantomPeakQualTools (MV411... [100%] 14 of 14\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[46/17f258] process > MACSCallPeak (MV411_RNP_SPI... [ 92%] 12 of 13\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (97)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[39/7b314d] process > CollectMultipleMetrics (MV4... [100%] 14 of 14\u001b[K\n",
      "[e7/594109] process > BigWig (MV411_RNP_SPI1-H3K2... [100%] 14 of 14\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[1b/b9c016] process > PhantomPeakQualTools (MV411... [100%] 14 of 14\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[fa/6c4132] process > MACSCallPeak (MV411_RNP_MEF... [ 86%] 12 of 14\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (98)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[39/7b314d] process > CollectMultipleMetrics (MV4... [100%] 14 of 14\u001b[K\n",
      "[e1/c25e9c] process > BigWig (MV411_RNP_MEF2D-H3K... [ 93%] 14 of 15\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[1b/b9c016] process > PhantomPeakQualTools (MV411... [100%] 14 of 14\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[46/17f258] process > MACSCallPeak (MV411_RNP_SPI... [ 93%] 13 of 14\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (99)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[39/7b314d] process > CollectMultipleMetrics (MV4... [100%] 14 of 14\u001b[K\n",
      "[e1/c25e9c] process > BigWig (MV411_RNP_MEF2D-H3K... [100%] 15 of 15\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[d7/e2fcc8] process > PhantomPeakQualTools (MV411... [ 93%] 14 of 15\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[46/17f258] process > MACSCallPeak (MV411_RNP_SPI... [ 93%] 13 of 14\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (100)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[49/670ca2] process > CollectMultipleMetrics (MV4... [ 93%] 14 of 15\u001b[K\n",
      "[e1/c25e9c] process > BigWig (MV411_RNP_MEF2D-H3K... [100%] 15 of 15\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[d7/e2fcc8] process > PhantomPeakQualTools (MV411... [100%] 15 of 15\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[46/17f258] process > MACSCallPeak (MV411_RNP_SPI... [ 93%] 13 of 14\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (101)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[49/670ca2] process > CollectMultipleMetrics (MV4... [ 93%] 14 of 15\u001b[K\n",
      "[e1/c25e9c] process > BigWig (MV411_RNP_MEF2D-H3K... [100%] 15 of 15\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/292df8] process > PhantomPeakQualTools (MV411... [ 94%] 15 of 16\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[fa/6c4132] process > MACSCallPeak (MV411_RNP_MEF... [100%] 14 of 14\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (102)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[49/670ca2] process > CollectMultipleMetrics (MV4... [100%] 15 of 15\u001b[K\n",
      "[9c/f2630b] process > BigWig (MV411_RNP_MYB-H3K27... [ 94%] 15 of 16\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/292df8] process > PhantomPeakQualTools (MV411... [ 94%] 15 of 16\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[fa/6c4132] process > MACSCallPeak (MV411_RNP_MEF... [100%] 14 of 14\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (103)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[49/670ca2] process > CollectMultipleMetrics (MV4... [100%] 15 of 15\u001b[K\n",
      "[9c/f2630b] process > BigWig (MV411_RNP_MYB-H3K27... [100%] 16 of 16\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/292df8] process > PhantomPeakQualTools (MV411... [ 94%] 15 of 16\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 93%] 14 of 15\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (104)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[85/5ed89a] process > CollectMultipleMetrics (MV4... [ 94%] 15 of 16\u001b[K\n",
      "[9c/f2630b] process > BigWig (MV411_RNP_MYB-H3K27... [100%] 16 of 16\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/292df8] process > PhantomPeakQualTools (MV411... [100%] 16 of 16\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 93%] 14 of 15\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (105)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[85/5ed89a] process > CollectMultipleMetrics (MV4... [100%] 16 of 16\u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [ 94%] 16 of 17\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/292df8] process > PhantomPeakQualTools (MV411... [100%] 16 of 16\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 93%] 14 of 15\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (106)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[85/5ed89a] process > CollectMultipleMetrics (MV4... [100%] 16 of 16\u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [ 94%] 16 of 17\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 93%] 14 of 15\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (107)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[85/5ed89a] process > CollectMultipleMetrics (MV4... [100%] 16 of 16\u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [ 94%] 16 of 17\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 94%] 15 of 16\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (108)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [ 94%] 16 of 17\u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 94%] 15 of 16\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (109)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 94%] 15 of 16\u001b[K\n",
      "[92/f4e285] process > AnnotatePeaks (MV411_RNP_AA... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (110)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 94%] 15 of 16\u001b[K\n",
      "[3d/7fd60b] process > AnnotatePeaks (MV411_RNP_ME... [ 50%] 1 of 2\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (111)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 94%] 15 of 16\u001b[K\n",
      "[aa/43437e] process > AnnotatePeaks (MV411_RNP_RU... [ 67%] 2 of 3\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (112)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 94%] 15 of 16\u001b[K\n",
      "[a0/3b8b5a] process > AnnotatePeaks (MV411_RNP_RU... [ 75%] 3 of 4\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (113)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 94%] 15 of 16\u001b[K\n",
      "[d9/83574c] process > AnnotatePeaks (MV411_RNP_SP... [ 80%] 4 of 5\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (114)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 94%] 15 of 16\u001b[K\n",
      "[d9/83574c] process > AnnotatePeaks (MV411_RNP_SP... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (115)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[d5/3a754a] process > AnnotatePeaks (MV411_RNP_MY... [ 71%] 5 of 7\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (117)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[46/342adc] process > AnnotatePeaks (MV411_RNP_IR... [ 75%] 6 of 8\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (118)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[48/756ae7] process > AnnotatePeaks (MV411_RNP_RU... [ 78%] 7 of 9\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (119)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[46/342adc] process > AnnotatePeaks (MV411_RNP_IR... [ 80%] 8 of 10\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (120)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[54/d0a95c] process > AnnotatePeaks (MV411_RNP_RU... [ 82%] 9 of 11\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (121)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[08/d35aca] process > AnnotatePeaks (MV411_RNP_RU... [ 83%] 10 of 12\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (122)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[c2/ac16d8] process > AnnotatePeaks (MV411_RNP_SP... [ 85%] 11 of 13\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (123)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[82/d0e0fc] process > AnnotatePeaks (MV411_RNP_ME... [ 86%] 12 of 14\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (124)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[0e/c24b0a] process > AnnotatePeaks (MV411_RNP_MY... [ 87%] 13 of 15\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (125)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [ 88%] 14 of 16\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (125)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[0e/c24b0a] process > AnnotatePeaks (MV411_RNP_MY... [ 94%] 15 of 16\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (126)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[bc/e561cf] process > PlotFingerprint (MV411_RNP_... [  0%] 0 of 1\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (127)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[bc/e561cf] process > PlotFingerprint (MV411_RNP_... [ 50%] 1 of 2\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (128)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[3f/9aec63] process > PlotFingerprint (MV411_RNP_... [ 67%] 2 of 3\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (129)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[8c/531d81] process > PlotFingerprint (MV411_RNP_... [ 75%] 3 of 4\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (129)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[8c/531d81] process > PlotFingerprint (MV411_RNP_... [ 75%] 3 of 4\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (130)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[20/afbec0] process > PlotFingerprint (MV411_RNP_... [ 80%] 4 of 5\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (131)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[20/afbec0] process > PlotFingerprint (MV411_RNP_... [ 83%] 5 of 6\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (132)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[5e/ac3cd1] process > PlotFingerprint (MV411_RNP_... [ 86%] 6 of 7\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (133)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[8f/17b4a2] process > PlotFingerprint (MV411_RNP_... [ 88%] 7 of 8\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (134)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[48/6bdc90] process > PlotFingerprint (MV411_RNP_... [ 89%] 8 of 9\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (135)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[48/6bdc90] process > PlotFingerprint (MV411_RNP_... [ 90%] 9 of 10\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (136)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[2b/144a84] process > PlotFingerprint (MV411_RNP_... [ 91%] 10 of 11\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (137)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[5c/b9836e] process > PlotFingerprint (MV411_RNP_... [ 92%] 11 of 12\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (138)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[72/264c6a] process > PlotFingerprint (MV411_RNP_... [ 92%] 12 of 13\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (139)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[8e/ba2fa1] process > PlotFingerprint (MV411_RNP_... [ 93%] 13 of 14\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (140)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[11/798ccb] process > PlotFingerprint (MV411_RNP_... [ 93%] 14 of 15\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (141)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[24/32de0c] process > PlotProfile (MV411_RNP_RUNX... [  0%] 0 of 1\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[11/798ccb] process > PlotFingerprint (MV411_RNP_... [100%] 15 of 15\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (142)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[24/32de0c] process > PlotProfile (MV411_RNP_RUNX... [ 50%] 1 of 2\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[11/798ccb] process > PlotFingerprint (MV411_RNP_... [100%] 15 of 15\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (143)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[e1/6ab2d8] process > PlotProfile (MV411_RNP_SPI1... [100%] 2 of 2\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [ 94%] 15 of 16\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (144)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[c5/fb716e] process > PlotProfile (MV411_RNP_MEF2... [ 67%] 2 of 3\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (145)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[da/b297d7] process > PlotProfile (MV411_RNP_AAVS... [ 75%] 3 of 4\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (146)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[da/b297d7] process > PlotProfile (MV411_RNP_AAVS... [ 80%] 4 of 5\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (147)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[e2/e0d2e6] process > PlotProfile (MV411_RNP_RUNX... [ 83%] 5 of 6\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (148)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[2c/37fe88] process > PlotProfile (MV411_RNP_AAVS... [ 86%] 6 of 7\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (149)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[2c/37fe88] process > PlotProfile (MV411_RNP_AAVS... [ 88%] 7 of 8\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (150)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[b8/ec5d9c] process > PlotProfile (MV411_RNP_MYB-... [ 89%] 8 of 9\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (151)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[de/2f9acd] process > PlotProfile (MV411_RNP_IRF8... [ 90%] 9 of 10\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (152)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[43/0ba3d9] process > PlotProfile (MV411_RNP_IRF8... [ 91%] 10 of 11\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (153)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[43/0ba3d9] process > PlotProfile (MV411_RNP_IRF8... [ 92%] 11 of 12\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (154)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[78/ac55d3] process > PlotProfile (MV411_RNP_RUNX... [ 92%] 12 of 13\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (155)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[05/0cacbc] process > PlotProfile (MV411_RNP_SPI1... [ 93%] 13 of 14\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (156)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[05/0cacbc] process > PlotProfile (MV411_RNP_SPI1... [ 93%] 14 of 15\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (157)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[28/273a25] process > PlotProfile (MV411_RNP_MEF2... [ 94%] 15 of 16\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (158)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[c2/bcd9cc] process > PlotProfile (MV411_RNP_MYB-... [ 94%] 16 of 17\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (160)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [  0%] 0 of 1\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (161)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [100%] 1 of 1 \u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[5e/b631fb] process > ConsensusPeakSetAnnotate (H... [  0%] 0 of 1\u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (161)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [100%] 1 of 1 \u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[5e/b631fb] process > ConsensusPeakSetAnnotate (H... [100%] 1 of 1 \u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (161)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [100%] 1 of 1 \u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[5e/b631fb] process > ConsensusPeakSetAnnotate (H... [100%] 1 of 1 \u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (162)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [100%] 1 of 1 \u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[5e/b631fb] process > ConsensusPeakSetAnnotate (H... [100%] 1 of 1 \u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[2b/af351b] process > MultiQC                        [  0%] 0 of 1\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (163)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [100%] 1 of 1 \u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[5e/b631fb] process > ConsensusPeakSetAnnotate (H... [100%] 1 of 1 \u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [100%] 1 of 1 \u001b[K\n",
      "[d4/87dc9d] process > IGV                            [  0%] 0 of 1\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[2b/af351b] process > MultiQC                        [  0%] 0 of 1\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (163)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [100%] 1 of 1 \u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[5e/b631fb] process > ConsensusPeakSetAnnotate (H... [100%] 1 of 1 \u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [100%] 1 of 1 \u001b[K\n",
      "[d4/87dc9d] process > IGV                            [100%] 1 of 1 \u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[2b/af351b] process > MultiQC                        [  0%] 0 of 1\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (163)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [100%] 1 of 1 \u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[5e/b631fb] process > ConsensusPeakSetAnnotate (H... [100%] 1 of 1 \u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [100%] 1 of 1 \u001b[K\n",
      "[d4/87dc9d] process > IGV                            [100%] 1 of 1 \u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[2b/af351b] process > MultiQC                        [100%] 1 of 1 \u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (163)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [100%] 1 of 1 \u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[5e/b631fb] process > ConsensusPeakSetAnnotate (H... [100%] 1 of 1 \u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [100%] 1 of 1 \u001b[K\n",
      "[d4/87dc9d] process > IGV                            [100%] 1 of 1 \u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[2b/af351b] process > MultiQC                        [100%] 1 of 1 \u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[31mFailed to invoke `workflow.onComplete` event handler\u001b[K\n",
      "\u001b[K\n",
      " -- Check script '/root/.nextflow/assets/nf-core/chipseq/main.nf' at line: 1610 or see '.nextflow.log' file for more details\u001b[39m\u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[31A\n",
      "executor >  local (163)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [100%] 1 of 1 \u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[5e/b631fb] process > ConsensusPeakSetAnnotate (H... [100%] 1 of 1 \u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [100%] 1 of 1 \u001b[K\n",
      "[d4/87dc9d] process > IGV                            [100%] 1 of 1 \u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[2b/af351b] process > MultiQC                        [100%] 1 of 1 \u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[31mFailed to invoke `workflow.onComplete` event handler\u001b[K\n",
      "\u001b[K\n",
      " -- Check script '/root/.nextflow/assets/nf-core/chipseq/main.nf' at line: 1610 or see '.nextflow.log' file for more details\u001b[39m\u001b[K\n",
      "\u001b[32;1mCompleted at: 06-Aug-2020 19:20:40\n",
      "Duration    : 1d 3h 52m 44s\n",
      "CPU hours   : 538.2 (38.8% cached)\n",
      "Succeeded   : 163\n",
      "Cached      : 83\n",
      "\u001b[22;39m\u001b[K\n",
      "\n"
     ]
    }
   ],
   "source": [
    "#process chips\n",
    "! cd ../../data/ && sudo ../nextflow run nf-core/chipseq --paired_end --seq_center 'DFCI' --email 'jkobject@gmail.com' --input ../AMLproject/nextflow/additional_degraded_v1_design.csv --genome GRCh38 --skip_preseq --max_cpus 16 -profile docker -w work -resume exotic_bartik"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "^C\r\n"
     ]
    }
   ],
   "source": [
    "!cp -r ../../data/results/* ../../data/$project/ && sudo rm -r ../data/results && sudo rm -r ..data/work"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "using all files from folder\n",
      "need to be name_*1, name_*2\n",
      "[('mp858-MV411_RNP_AAVS1-H3K27AC-r1_R1_001.fastq.gz', 'mp858-MV411_RNP_AAVS1-H3K27AC-r1_R2_001.fastq.gz'), ('mp859-MV411_RNP_AAVS1-H3K27AC-r2_R1_001.fastq.gz', 'mp859-MV411_RNP_AAVS1-H3K27AC-r2_R2_001.fastq.gz'), ('mp860-MV411_RNP_RUNX1-H3K27AC-r1_R1_001.fastq.gz', 'mp860-MV411_RNP_RUNX1-H3K27AC-r1_R2_001.fastq.gz'), ('mp861-MV411_RNP_RUNX1-H3K27AC-r2_R1_001.fastq.gz', 'mp861-MV411_RNP_RUNX1-H3K27AC-r2_R2_001.fastq.gz'), ('mp862-MV411_RNP_RUNX2-H3K27AC-r1_R1_001.fastq.gz', 'mp862-MV411_RNP_RUNX2-H3K27AC-r1_R2_001.fastq.gz'), ('mp863-MV411_RNP_RUNX2-H3K27AC-r2_R1_001.fastq.gz', 'mp863-MV411_RNP_RUNX2-H3K27AC-r2_R2_001.fastq.gz'), ('mp864-MV411_RNP_RUNX1_RUNX2-H3K27AC-r1_R1_001.fastq.gz', 'mp864-MV411_RNP_RUNX1_RUNX2-H3K27AC-r1_R2_001.fastq.gz'), ('mp865-MV411_RNP_RUNX1_RUNX2-H3K27AC-r2_R1_001.fastq.gz', 'mp865-MV411_RNP_RUNX1_RUNX2-H3K27AC-r2_R2_001.fastq.gz'), ('mp866-MV411_RNP_MEF2D-H3K27AC-r1_R1_001.fastq.gz', 'mp866-MV411_RNP_MEF2D-H3K27AC-r1_R2_001.fastq.gz'), ('mp867-MV411_RNP_MEF2D-H3K27AC-r2_R1_001.fastq.gz', 'mp867-MV411_RNP_MEF2D-H3K27AC-r2_R2_001.fastq.gz'), ('mp868-MV411_RNP_IRF8-H3K27AC-r1_R1_001.fastq.gz', 'mp868-MV411_RNP_IRF8-H3K27AC-r1_R2_001.fastq.gz'), ('mp869-MV411_RNP_IRF8-H3K27AC-r2_R1_001.fastq.gz', 'mp869-MV411_RNP_IRF8-H3K27AC-r2_R2_001.fastq.gz'), ('mp870-MV411_RNP_MYB-H3K27AC-r1_R1_001.fastq.gz', 'mp870-MV411_RNP_MYB-H3K27AC-r1_R2_001.fastq.gz'), ('mp871-MV411_RNP_MYB-H3K27AC-r2_R1_001.fastq.gz', 'mp871-MV411_RNP_MYB-H3K27AC-r2_R2_001.fastq.gz'), ('mp872-MV411_RNP_SPI1-H3K27AC-r1_R1_001.fastq.gz', 'mp872-MV411_RNP_SPI1-H3K27AC-r1_R2_001.fastq.gz'), ('mp873-MV411_RNP_SPI1-H3K27AC-r2_R1_001.fastq.gz', 'mp873-MV411_RNP_SPI1-H3K27AC-r2_R2_001.fastq.gz')]\n",
      "\n",
      "\n",
      "trimming\n",
      "\n",
      "\n",
      "['../../TrimGalore-0.6.5/trim_galore --paired --fastqc --gzip ../../data/additional_degraded_v1/fastqs/mp858-MV411_RNP_AAVS1-H3K27AC-r1_R1_001.fastq.gz ../../data/additional_degraded_v1/fastqs/mp858-MV411_RNP_AAVS1-H3K27AC-r1_R2_001.fastq.gz -o ../../data/additional_degraded_v1/', '../../TrimGalore-0.6.5/trim_galore --paired --fastqc --gzip ../../data/additional_degraded_v1/fastqs/mp859-MV411_RNP_AAVS1-H3K27AC-r2_R1_001.fastq.gz ../../data/additional_degraded_v1/fastqs/mp859-MV411_RNP_AAVS1-H3K27AC-r2_R2_001.fastq.gz -o ../../data/additional_degraded_v1/', '../../TrimGalore-0.6.5/trim_galore --paired --fastqc --gzip ../../data/additional_degraded_v1/fastqs/mp860-MV411_RNP_RUNX1-H3K27AC-r1_R1_001.fastq.gz ../../data/additional_degraded_v1/fastqs/mp860-MV411_RNP_RUNX1-H3K27AC-r1_R2_001.fastq.gz -o ../../data/additional_degraded_v1/', '../../TrimGalore-0.6.5/trim_galore --paired --fastqc --gzip ../../data/additional_degraded_v1/fastqs/mp861-MV411_RNP_RUNX1-H3K27AC-r2_R1_001.fastq.gz ../../data/additional_degraded_v1/fastqs/mp861-MV411_RNP_RUNX1-H3K27AC-r2_R2_001.fastq.gz -o ../../data/additional_degraded_v1/', '../../TrimGalore-0.6.5/trim_galore --paired --fastqc --gzip ../../data/additional_degraded_v1/fastqs/mp862-MV411_RNP_RUNX2-H3K27AC-r1_R1_001.fastq.gz ../../data/additional_degraded_v1/fastqs/mp862-MV411_RNP_RUNX2-H3K27AC-r1_R2_001.fastq.gz -o ../../data/additional_degraded_v1/', '../../TrimGalore-0.6.5/trim_galore --paired --fastqc --gzip ../../data/additional_degraded_v1/fastqs/mp863-MV411_RNP_RUNX2-H3K27AC-r2_R1_001.fastq.gz ../../data/additional_degraded_v1/fastqs/mp863-MV411_RNP_RUNX2-H3K27AC-r2_R2_001.fastq.gz -o ../../data/additional_degraded_v1/', '../../TrimGalore-0.6.5/trim_galore --paired --fastqc --gzip ../../data/additional_degraded_v1/fastqs/mp864-MV411_RNP_RUNX1_RUNX2-H3K27AC-r1_R1_001.fastq.gz ../../data/additional_degraded_v1/fastqs/mp864-MV411_RNP_RUNX1_RUNX2-H3K27AC-r1_R2_001.fastq.gz -o ../../data/additional_degraded_v1/', '../../TrimGalore-0.6.5/trim_galore --paired --fastqc --gzip ../../data/additional_degraded_v1/fastqs/mp865-MV411_RNP_RUNX1_RUNX2-H3K27AC-r2_R1_001.fastq.gz ../../data/additional_degraded_v1/fastqs/mp865-MV411_RNP_RUNX1_RUNX2-H3K27AC-r2_R2_001.fastq.gz -o ../../data/additional_degraded_v1/', '../../TrimGalore-0.6.5/trim_galore --paired --fastqc --gzip ../../data/additional_degraded_v1/fastqs/mp866-MV411_RNP_MEF2D-H3K27AC-r1_R1_001.fastq.gz ../../data/additional_degraded_v1/fastqs/mp866-MV411_RNP_MEF2D-H3K27AC-r1_R2_001.fastq.gz -o ../../data/additional_degraded_v1/', '../../TrimGalore-0.6.5/trim_galore --paired --fastqc --gzip ../../data/additional_degraded_v1/fastqs/mp867-MV411_RNP_MEF2D-H3K27AC-r2_R1_001.fastq.gz ../../data/additional_degraded_v1/fastqs/mp867-MV411_RNP_MEF2D-H3K27AC-r2_R2_001.fastq.gz -o ../../data/additional_degraded_v1/', '../../TrimGalore-0.6.5/trim_galore --paired --fastqc --gzip ../../data/additional_degraded_v1/fastqs/mp868-MV411_RNP_IRF8-H3K27AC-r1_R1_001.fastq.gz ../../data/additional_degraded_v1/fastqs/mp868-MV411_RNP_IRF8-H3K27AC-r1_R2_001.fastq.gz -o ../../data/additional_degraded_v1/', '../../TrimGalore-0.6.5/trim_galore --paired --fastqc --gzip ../../data/additional_degraded_v1/fastqs/mp869-MV411_RNP_IRF8-H3K27AC-r2_R1_001.fastq.gz ../../data/additional_degraded_v1/fastqs/mp869-MV411_RNP_IRF8-H3K27AC-r2_R2_001.fastq.gz -o ../../data/additional_degraded_v1/', '../../TrimGalore-0.6.5/trim_galore --paired --fastqc --gzip ../../data/additional_degraded_v1/fastqs/mp870-MV411_RNP_MYB-H3K27AC-r1_R1_001.fastq.gz ../../data/additional_degraded_v1/fastqs/mp870-MV411_RNP_MYB-H3K27AC-r1_R2_001.fastq.gz -o ../../data/additional_degraded_v1/', '../../TrimGalore-0.6.5/trim_galore --paired --fastqc --gzip ../../data/additional_degraded_v1/fastqs/mp871-MV411_RNP_MYB-H3K27AC-r2_R1_001.fastq.gz ../../data/additional_degraded_v1/fastqs/mp871-MV411_RNP_MYB-H3K27AC-r2_R2_001.fastq.gz -o ../../data/additional_degraded_v1/', '../../TrimGalore-0.6.5/trim_galore --paired --fastqc --gzip ../../data/additional_degraded_v1/fastqs/mp872-MV411_RNP_SPI1-H3K27AC-r1_R1_001.fastq.gz ../../data/additional_degraded_v1/fastqs/mp872-MV411_RNP_SPI1-H3K27AC-r1_R2_001.fastq.gz -o ../../data/additional_degraded_v1/', '../../TrimGalore-0.6.5/trim_galore --paired --fastqc --gzip ../../data/additional_degraded_v1/fastqs/mp873-MV411_RNP_SPI1-H3K27AC-r2_R1_001.fastq.gz ../../data/additional_degraded_v1/fastqs/mp873-MV411_RNP_SPI1-H3K27AC-r2_R2_001.fastq.gz -o ../../data/additional_degraded_v1/']\n",
      "\n",
      "\n",
      "mapping\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "filtering\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "counting\n",
      "\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "({'mp858-MV411_RNP_AAVS1-H3K27AC-r1_R1_001_val_1': 0.6644280581189461,\n",
       "  'mp859-MV411_RNP_AAVS1-H3K27AC-r2_R1_001_val_1': 0.5408483387497395,\n",
       "  'mp860-MV411_RNP_RUNX1-H3K27AC-r1_R1_001_val_1': 0.9306982320552297,\n",
       "  'mp861-MV411_RNP_RUNX1-H3K27AC-r2_R1_001_val_1': 0.4595249033344818,\n",
       "  'mp862-MV411_RNP_RUNX2-H3K27AC-r1_R1_001_val_1': 1.0,\n",
       "  'mp863-MV411_RNP_RUNX2-H3K27AC-r2_R1_001_val_1': 0.45660285309024545,\n",
       "  'mp864-MV411_RNP_RUNX1_RUNX2-H3K27AC-r1_R1_001_val_1': 0.500947891528239,\n",
       "  'mp865-MV411_RNP_RUNX1_RUNX2-H3K27AC-r2_R1_001_val_1': 0.3734932073757382,\n",
       "  'mp866-MV411_RNP_MEF2D-H3K27AC-r1_R1_001_val_1': 0.37678768740112,\n",
       "  'mp867-MV411_RNP_MEF2D-H3K27AC-r2_R1_001_val_1': 0.397505335892133,\n",
       "  'mp868-MV411_RNP_IRF8-H3K27AC-r1_R1_001_val_1': 0.20011420092694476,\n",
       "  'mp869-MV411_RNP_IRF8-H3K27AC-r2_R1_001_val_1': 0.2631428229278934,\n",
       "  'mp870-MV411_RNP_MYB-H3K27AC-r1_R1_001_val_1': 0.5015211299972058,\n",
       "  'mp871-MV411_RNP_MYB-H3K27AC-r2_R1_001_val_1': 0.32322855597358185,\n",
       "  'mp872-MV411_RNP_SPI1-H3K27AC-r1_R1_001_val_1': 0.3334831285861435,\n",
       "  'mp873-MV411_RNP_SPI1-H3K27AC-r2_R1_001_val_1': 0.46990096093195277},\n",
       " {'mp858-MV411_RNP_AAVS1-H3K27AC-r1_R1_001_val_1': 1445243,\n",
       "  'mp859-MV411_RNP_AAVS1-H3K27AC-r2_R1_001_val_1': 1775470,\n",
       "  'mp860-MV411_RNP_RUNX1-H3K27AC-r1_R1_001_val_1': 1031763,\n",
       "  'mp861-MV411_RNP_RUNX1-H3K27AC-r2_R1_001_val_1': 2089680,\n",
       "  'mp862-MV411_RNP_RUNX2-H3K27AC-r1_R1_001_val_1': 960260,\n",
       "  'mp863-MV411_RNP_RUNX2-H3K27AC-r2_R1_001_val_1': 2103053,\n",
       "  'mp864-MV411_RNP_RUNX1_RUNX2-H3K27AC-r1_R1_001_val_1': 1916886,\n",
       "  'mp865-MV411_RNP_RUNX1_RUNX2-H3K27AC-r2_R1_001_val_1': 2571024,\n",
       "  'mp866-MV411_RNP_MEF2D-H3K27AC-r1_R1_001_val_1': 2548544,\n",
       "  'mp867-MV411_RNP_MEF2D-H3K27AC-r2_R1_001_val_1': 2415716,\n",
       "  'mp868-MV411_RNP_IRF8-H3K27AC-r1_R1_001_val_1': 4798560,\n",
       "  'mp869-MV411_RNP_IRF8-H3K27AC-r2_R1_001_val_1': 3649197,\n",
       "  'mp870-MV411_RNP_MYB-H3K27AC-r1_R1_001_val_1': 1914695,\n",
       "  'mp871-MV411_RNP_MYB-H3K27AC-r2_R1_001_val_1': 2970839,\n",
       "  'mp872-MV411_RNP_SPI1-H3K27AC-r1_R1_001_val_1': 2879486,\n",
       "  'mp873-MV411_RNP_SPI1-H3K27AC-r2_R1_001_val_1': 2043537})"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# get scaling values\n",
    "norm, mapped = h.getSpikeInControlScales(refgenome=\"../../data/ref/reference_droso.fna\",\n",
    "fastQfolder='../../data/'+project+'/fastqs/',\n",
    "pairedEnd=True, cores=12,\n",
    "tofilter=True,\n",
    "totrim=True,\n",
    "tomap=True,\n",
    "results=\"../../data/\"+project+\"/\",\n",
    "pathtotrim_galore=\"../../TrimGalore-0.6.5/trim_galore\")\n",
    "norm, mapped"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "computing scales from the excel sheet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 119,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [0.3011826465, # 'MV411_RNP_IRF8-H3K27AC_R1',\n",
    "0.4865371752, # 'MV411_RNP_IRF8-H3K27AC_R2',\n",
    "0.5670857556, # 'MV411_RNP_MEF2D-H3K27AC_R1',\n",
    "0.7349663619, # 'MV411_RNP_MEF2D-H3K27AC_R2',\n",
    "0.7548163023, # 'MV411_RNP_MYB-H3K27AC_R1',\n",
    "0.5976325206, # 'MV411_RNP_MYB-H3K27AC_R2',\n",
    "1.400750948, # 'MV411_RNP_RUNX1-H3K27AC_R1',\n",
    "0.849637265, # 'MV411_RNP_RUNX1-H3K27AC_R2',\n",
    "0.7539535476, # 'MV411_RNP_RUNX1_RUNX2-H3K27AC_R1',\n",
    "0.6905692051, # 'MV411_RNP_RUNX1_RUNX2-H3K27AC_R2',\n",
    "1.50505384, # 'MV411_RNP_RUNX2-H3K27AC_R1',\n",
    "0.8442345485, # 'MV411_RNP_RUNX2-H3K27AC_R2',\n",
    "0.5019100631, # 'MV411_RNP_SPI1-H3K27AC_R1',\n",
    "0.8688220473, # 'MV411_RNP_SPI1-H3K27AC_R2'\n",
    "         ]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 78,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/additional_degraded_v1/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_IRF8-H3K27AC_R1.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_IRF8-H3K27AC_R2.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_MEF2D-H3K27AC_R1.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_MEF2D-H3K27AC_R2.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_MYB-H3K27AC_R1.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_MYB-H3K27AC_R2.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX1-H3K27AC_R1.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX1-H3K27AC_R2.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX1_RUNX2-H3K27AC_R1.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX1_RUNX2-H3K27AC_R2.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX2-H3K27AC_R1.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX2-H3K27AC_R2.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_SPI1-H3K27AC_R1.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_SPI1-H3K27AC_R2.mLb.clN.sorted.bam']"
      ]
     },
     "execution_count": 78,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bams = ! ls ../../data/$project/bwa/mergedLibrary/*.bam\n",
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_MEF2D-H3K27AC_R2.mLb.clN.sorted.bam'"
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bams[6]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### on scalled data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mkdir ../../data/$project/diffPeaks/ && ! mkdir ../../data/$project/diffData/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mkdir ../../data/$project/droso_aligned"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mv ../../data/$project/mp* ../../data/$project/droso_aligned"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "wigs = ! ls ../../data/$project/bwa/mergedLibrary/bigwig/*.bigWig"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INPUT_R1.mLb.clN.bigWig\r\n",
      "MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bdg\r\n",
      "MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bigWig\r\n",
      "MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bdg\r\n",
      "MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bigWig\r\n",
      "MV411_RNP_IRF8-H3K27AC_R1.mLb.clN.bdg\r\n",
      "MV411_RNP_IRF8-H3K27AC_R1.mLb.clN.bigWig\r\n",
      "MV411_RNP_IRF8-H3K27AC_R2.mLb.clN.bdg\r\n",
      "MV411_RNP_IRF8-H3K27AC_R2.mLb.clN.bigWig\r\n",
      "MV411_RNP_MEF2D-H3K27AC_R1.mLb.clN.bdg\r\n",
      "MV411_RNP_MEF2D-H3K27AC_R1.mLb.clN.bigWig\r\n",
      "MV411_RNP_MEF2D-H3K27AC_R2.mLb.clN.bdg\r\n",
      "MV411_RNP_MEF2D-H3K27AC_R2.mLb.clN.bigWig\r\n",
      "MV411_RNP_MYB-H3K27AC_R1.mLb.clN.bdg\r\n",
      "MV411_RNP_MYB-H3K27AC_R1.mLb.clN.bigWig\r\n",
      "MV411_RNP_MYB-H3K27AC_R2.mLb.clN.bdg\r\n",
      "MV411_RNP_MYB-H3K27AC_R2.mLb.clN.bigWig\r\n",
      "MV411_RNP_RUNX1-H3K27AC_R1.mLb.clN.bdg\r\n",
      "MV411_RNP_RUNX1-H3K27AC_R1.mLb.clN.bigWig\r\n",
      "MV411_RNP_RUNX1-H3K27AC_R2.mLb.clN.bdg\r\n",
      "MV411_RNP_RUNX1-H3K27AC_R2.mLb.clN.bigWig\r\n",
      "MV411_RNP_RUNX1_RUNX2-H3K27AC_R1.mLb.clN.bdg\r\n",
      "MV411_RNP_RUNX1_RUNX2-H3K27AC_R1.mLb.clN.bigWig\r\n",
      "MV411_RNP_RUNX1_RUNX2-H3K27AC_R2.mLb.clN.bdg\r\n",
      "MV411_RNP_RUNX1_RUNX2-H3K27AC_R2.mLb.clN.bigWig\r\n",
      "MV411_RNP_RUNX2-H3K27AC_R1.mLb.clN.bdg\r\n",
      "MV411_RNP_RUNX2-H3K27AC_R1.mLb.clN.bigWig\r\n",
      "MV411_RNP_RUNX2-H3K27AC_R2.mLb.clN.bdg\r\n",
      "MV411_RNP_RUNX2-H3K27AC_R2.mLb.clN.bigWig\r\n",
      "MV411_RNP_SPI1-H3K27AC_R1.mLb.clN.bdg\r\n",
      "MV411_RNP_SPI1-H3K27AC_R1.mLb.clN.bigWig\r\n",
      "MV411_RNP_SPI1-H3K27AC_R2.mLb.clN.bdg\r\n",
      "MV411_RNP_SPI1-H3K27AC_R2.mLb.clN.bigWig\r\n",
      "\u001b[0m\u001b[01;34mscale\u001b[0m/\r\n"
     ]
    }
   ],
   "source": [
    "ls ../../data/$project/bwa/mergedLibrary/bigwig/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 80,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_IRF8-H3K27AC_R2.mLb.clN.bigWig [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_MYB-H3K27AC_R1.mLb.clN.bigWig [Content-Type=application/octet-stream]...\n",
      "==> NOTE: You are uploading one or more large file(s), which would run\n",
      "significantly faster if you enable parallel composite uploads. This\n",
      "feature can be enabled by editing the\n",
      "\"parallel_composite_upload_threshold\" value in your .boto\n",
      "configuration file. However, note that if you do this large files will\n",
      "be uploaded as `composite objects\n",
      "<https://cloud.google.com/storage/docs/composite-objects>`_,which\n",
      "means that any user who downloads such objects will need to have a\n",
      "compiled crcmod installed (see \"gsutil help crcmod\"). This is because\n",
      "without a compiled crcmod, computing checksums on composite objects is\n",
      "so slow that gsutil disables downloads of composite objects.\n",
      "\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX1_RUNX2-H3K27AC_R1.mLb.clN.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_MEF2D-H3K27AC_R2.mLb.clN.bigWig [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX2-H3K27AC_R2.mLb.clN.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bigWig [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX1-H3K27AC_R2.mLb.clN.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_MEF2D-H3K27AC_R1.mLb.clN.bigWig [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_MYB-H3K27AC_R2.mLb.clN.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_IRF8-H3K27AC_R1.mLb.clN.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_MEF2D-H3K27AC_R2.mLb.clN.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX2-H3K27AC_R1.mLb.clN.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX1_RUNX2-H3K27AC_R1.mLb.clN.bigWig [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_MEF2D-H3K27AC_R1.mLb.clN.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_SPI1-H3K27AC_R1.mLb.clN.bigWig [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX1_RUNX2-H3K27AC_R2.mLb.clN.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bigWig [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX1_RUNX2-H3K27AC_R2.mLb.clN.bigWig [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX1-H3K27AC_R2.mLb.clN.bigWig [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_IRF8-H3K27AC_R1.mLb.clN.bigWig [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_MYB-H3K27AC_R2.mLb.clN.bigWig [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX2-H3K27AC_R2.mLb.clN.bigWig [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX2-H3K27AC_R1.mLb.clN.bigWig [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_SPI1-H3K27AC_R1.mLb.clN.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_SPI1-H3K27AC_R2.mLb.clN.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX1-H3K27AC_R1.mLb.clN.bigWig [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX1-H3K27AC_R1.mLb.clN.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_SPI1-H3K27AC_R2.mLb.clN.bigWig [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_IRF8-H3K27AC_R2.mLb.clN.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_MYB-H3K27AC_R1.mLb.clN.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/scale/MV411_RNP_MEF2D-H3K27AC_R1.mLb.clN.scale_factor.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/scale/MV411_RNP_RUNX1-H3K27AC_R1.mLb.clN.scale_factor.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bigWig [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/scale/INPUT_R1.mLb.clN.scale_factor.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/scale/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.scale_factor.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/scale/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.scale_factor.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/scale/MV411_RNP_RUNX1_RUNX2-H3K27AC_R1.mLb.clN.scale_factor.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/scale/MV411_RNP_RUNX2-H3K27AC_R1.mLb.clN.scale_factor.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/scale/MV411_RNP_RUNX1_RUNX2-H3K27AC_R2.mLb.clN.scale_factor.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/scale/MV411_RNP_IRF8-H3K27AC_R2.mLb.clN.scale_factor.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/scale/MV411_RNP_RUNX1-H3K27AC_R2.mLb.clN.scale_factor.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/scale/MV411_RNP_MYB-H3K27AC_R2.mLb.clN.scale_factor.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/scale/MV411_RNP_IRF8-H3K27AC_R1.mLb.clN.scale_factor.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/scale/MV411_RNP_RUNX2-H3K27AC_R2.mLb.clN.scale_factor.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/scale/MV411_RNP_MEF2D-H3K27AC_R2.mLb.clN.scale_factor.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/scale/MV411_RNP_SPI1-H3K27AC_R2.mLb.clN.scale_factor.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/scale/MV411_RNP_SPI1-H3K27AC_R1.mLb.clN.scale_factor.txt [Content-Type=text/plain]...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying file://../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/scale/MV411_RNP_MYB-H3K27AC_R1.mLb.clN.scale_factor.txt [Content-Type=text/plain]...\n",
      "/ [50/50 files][ 22.6 GiB/ 22.6 GiB] 100% Done  83.7 MiB/s ETA 00:00:00         \n",
      "Operation completed over 50 objects/22.6 GiB.                                    \n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp862-MV411_RNP_RUNX2-H3K27AC-r1_R1_001_val_1.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp860-MV411_RNP_RUNX1-H3K27AC-r1_R1_001.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp870-MV411_RNP_MYB-H3K27AC-r1_R1_001.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "==> NOTE: You are uploading one or more large file(s), which would run          \n",
      "significantly faster if you enable parallel composite uploads. This\n",
      "feature can be enabled by editing the\n",
      "\"parallel_composite_upload_threshold\" value in your .boto\n",
      "configuration file. However, note that if you do this large files will\n",
      "be uploaded as `composite objects\n",
      "<https://cloud.google.com/storage/docs/composite-objects>`_,which\n",
      "means that any user who downloads such objects will need to have a\n",
      "compiled crcmod installed (see \"gsutil help crcmod\"). This is because\n",
      "without a compiled crcmod, computing checksums on composite objects is\n",
      "so slow that gsutil disables downloads of composite objects.\n",
      "\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp866-MV411_RNP_MEF2D-H3K27AC-r1_R1_001.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp873-MV411_RNP_SPI1-H3K27AC-r2_R1_001.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp869-MV411_RNP_IRF8-H3K27AC-r2_R1_001.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp870-MV411_RNP_MYB-H3K27AC-r1_R1_001_val_1.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp858-MV411_RNP_AAVS1-H3K27AC-r1_R1_001_val_1.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp858-MV411_RNP_AAVS1-H3K27AC-r1_R1_001_val_1.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp869-MV411_RNP_IRF8-H3K27AC-r2_R1_001_val_1.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp860-MV411_RNP_RUNX1-H3K27AC-r1_R1_001.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp861-MV411_RNP_RUNX1-H3K27AC-r2_R2_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp861-MV411_RNP_RUNX1-H3K27AC-r2_R1_001.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp873-MV411_RNP_SPI1-H3K27AC-r2_R1_001.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp872-MV411_RNP_SPI1-H3K27AC-r1_R1_001.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp870-MV411_RNP_MYB-H3K27AC-r1_R2_001_val_2.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp863-MV411_RNP_RUNX2-H3K27AC-r2_R1_001.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp859-MV411_RNP_AAVS1-H3K27AC-r2_R1_001_val_1.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp872-MV411_RNP_SPI1-H3K27AC-r1_R2_001_val_2.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp864-MV411_RNP_RUNX1_RUNX2-H3K27AC-r1_R1_001.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp862-MV411_RNP_RUNX2-H3K27AC-r1_R1_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp862-MV411_RNP_RUNX2-H3K27AC-r1_R1_001_val_1.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp863-MV411_RNP_RUNX2-H3K27AC-r2_R1_001.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp871-MV411_RNP_MYB-H3K27AC-r2_R1_001_val_1.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp858-MV411_RNP_AAVS1-H3K27AC-r1_R2_001_val_2.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp864-MV411_RNP_RUNX1_RUNX2-H3K27AC-r1_R1_001_val_1.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp867-MV411_RNP_MEF2D-H3K27AC-r2_R1_001_val_1.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp868-MV411_RNP_IRF8-H3K27AC-r1_R1_001.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp862-MV411_RNP_RUNX2-H3K27AC-r1_R1_001.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp864-MV411_RNP_RUNX1_RUNX2-H3K27AC-r1_R1_001_val_1.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp870-MV411_RNP_MYB-H3K27AC-r1_R1_001.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp862-MV411_RNP_RUNX2-H3K27AC-r1_R2_001_val_2.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp867-MV411_RNP_MEF2D-H3K27AC-r2_R1_001_val_1.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp864-MV411_RNP_RUNX1_RUNX2-H3K27AC-r1_R1_001.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp867-MV411_RNP_MEF2D-H3K27AC-r2_R1_001_val_1.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp859-MV411_RNP_AAVS1-H3K27AC-r2_R1_001.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp866-MV411_RNP_MEF2D-H3K27AC-r1_R1_001_val_1.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp859-MV411_RNP_AAVS1-H3K27AC-r2_R2_001_val_2.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp858-MV411_RNP_AAVS1-H3K27AC-r1_R1_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp869-MV411_RNP_IRF8-H3K27AC-r2_R2_001_val_2.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp871-MV411_RNP_MYB-H3K27AC-r2_R1_001.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp871-MV411_RNP_MYB-H3K27AC-r2_R1_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp870-MV411_RNP_MYB-H3K27AC-r1_R2_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp865-MV411_RNP_RUNX1_RUNX2-H3K27AC-r2_R1_001_val_1.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp869-MV411_RNP_IRF8-H3K27AC-r2_R2_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp863-MV411_RNP_RUNX2-H3K27AC-r2_R1_001_val_1.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp864-MV411_RNP_RUNX1_RUNX2-H3K27AC-r1_R2_001_val_2.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp872-MV411_RNP_SPI1-H3K27AC-r1_R1_001_val_1.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp867-MV411_RNP_MEF2D-H3K27AC-r2_R1_001.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp868-MV411_RNP_IRF8-H3K27AC-r1_R1_001_val_1.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp862-MV411_RNP_RUNX2-H3K27AC-r1_R1_001.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp863-MV411_RNP_RUNX2-H3K27AC-r2_R1_001.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp865-MV411_RNP_RUNX1_RUNX2-H3K27AC-r2_R1_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp866-MV411_RNP_MEF2D-H3K27AC-r1_R1_001_val_1.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp871-MV411_RNP_MYB-H3K27AC-r2_R1_001.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp866-MV411_RNP_MEF2D-H3K27AC-r1_R1_001.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp868-MV411_RNP_IRF8-H3K27AC-r1_R1_001_val_1.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp864-MV411_RNP_RUNX1_RUNX2-H3K27AC-r1_R1_001_val_1.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp858-MV411_RNP_AAVS1-H3K27AC-r1_R1_001.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp860-MV411_RNP_RUNX1-H3K27AC-r1_R2_001_val_2.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp863-MV411_RNP_RUNX2-H3K27AC-r2_R1_001_val_1.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp866-MV411_RNP_MEF2D-H3K27AC-r1_R1_001_val_1.fq.gz [Content-Type=application/octet-stream]...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp873-MV411_RNP_SPI1-H3K27AC-r2_R1_001_val_1.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp865-MV411_RNP_RUNX1_RUNX2-H3K27AC-r2_R1_001_val_1.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp858-MV411_RNP_AAVS1-H3K27AC-r1_R1_001_val_1.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp870-MV411_RNP_MYB-H3K27AC-r1_R1_001_val_1.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp871-MV411_RNP_MYB-H3K27AC-r2_R2_001_val_2.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp858-MV411_RNP_AAVS1-H3K27AC-r1_R1_001.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp869-MV411_RNP_IRF8-H3K27AC-r2_R1_001.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp861-MV411_RNP_RUNX1-H3K27AC-r2_R1_001_val_1.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp869-MV411_RNP_IRF8-H3K27AC-r2_R1_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp863-MV411_RNP_RUNX2-H3K27AC-r2_R1_001.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp866-MV411_RNP_MEF2D-H3K27AC-r1_R1_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp865-MV411_RNP_RUNX1_RUNX2-H3K27AC-r2_R1_001_val_1.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp859-MV411_RNP_AAVS1-H3K27AC-r2_R1_001.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp861-MV411_RNP_RUNX1-H3K27AC-r2_R1_001_val_1.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp873-MV411_RNP_SPI1-H3K27AC-r2_R2_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp862-MV411_RNP_RUNX2-H3K27AC-r1_R1_001.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp870-MV411_RNP_MYB-H3K27AC-r1_R1_001.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp869-MV411_RNP_IRF8-H3K27AC-r2_R1_001_val_1.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp868-MV411_RNP_IRF8-H3K27AC-r1_R1_001.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp862-MV411_RNP_RUNX2-H3K27AC-r1_R2_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp872-MV411_RNP_SPI1-H3K27AC-r1_R1_001.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp867-MV411_RNP_MEF2D-H3K27AC-r2_R1_001_val_1.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp872-MV411_RNP_SPI1-H3K27AC-r1_R2_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp865-MV411_RNP_RUNX1_RUNX2-H3K27AC-r2_R1_001.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp864-MV411_RNP_RUNX1_RUNX2-H3K27AC-r1_R1_001.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp873-MV411_RNP_SPI1-H3K27AC-r2_R1_001_val_1.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp859-MV411_RNP_AAVS1-H3K27AC-r2_R1_001_val_1.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp863-MV411_RNP_RUNX2-H3K27AC-r2_R1_001_val_1.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp869-MV411_RNP_IRF8-H3K27AC-r2_R1_001.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp865-MV411_RNP_RUNX1_RUNX2-H3K27AC-r2_R1_001.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp859-MV411_RNP_AAVS1-H3K27AC-r2_R1_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp864-MV411_RNP_RUNX1_RUNX2-H3K27AC-r1_R1_001_val_1.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp866-MV411_RNP_MEF2D-H3K27AC-r1_R1_001_val_1.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp872-MV411_RNP_SPI1-H3K27AC-r1_R1_001.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp861-MV411_RNP_RUNX1-H3K27AC-r2_R1_001.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp873-MV411_RNP_SPI1-H3K27AC-r2_R1_001_val_1.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp867-MV411_RNP_MEF2D-H3K27AC-r2_R1_001.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp869-MV411_RNP_IRF8-H3K27AC-r2_R1_001_val_1.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp871-MV411_RNP_MYB-H3K27AC-r2_R1_001_val_1.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp860-MV411_RNP_RUNX1-H3K27AC-r1_R2_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp862-MV411_RNP_RUNX2-H3K27AC-r1_R1_001_val_1.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp866-MV411_RNP_MEF2D-H3K27AC-r1_R1_001.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp872-MV411_RNP_SPI1-H3K27AC-r1_R1_001_val_1.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp867-MV411_RNP_MEF2D-H3K27AC-r2_R2_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp873-MV411_RNP_SPI1-H3K27AC-r2_R1_001_val_1.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp863-MV411_RNP_RUNX2-H3K27AC-r2_R2_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp868-MV411_RNP_IRF8-H3K27AC-r1_R2_001_val_2.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp867-MV411_RNP_MEF2D-H3K27AC-r2_R2_001_val_2.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp864-MV411_RNP_RUNX1_RUNX2-H3K27AC-r1_R1_001_val_1.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp861-MV411_RNP_RUNX1-H3K27AC-r2_R1_001.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp867-MV411_RNP_MEF2D-H3K27AC-r2_R1_001.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp871-MV411_RNP_MYB-H3K27AC-r2_R1_001_val_1.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp868-MV411_RNP_IRF8-H3K27AC-r1_R1_001_val_1.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp867-MV411_RNP_MEF2D-H3K27AC-r2_R1_001.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp859-MV411_RNP_AAVS1-H3K27AC-r2_R1_001_val_1.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp865-MV411_RNP_RUNX1_RUNX2-H3K27AC-r2_R1_001.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp872-MV411_RNP_SPI1-H3K27AC-r1_R1_001_val_1.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp864-MV411_RNP_RUNX1_RUNX2-H3K27AC-r1_R1_001.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp868-MV411_RNP_IRF8-H3K27AC-r1_R1_001.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp868-MV411_RNP_IRF8-H3K27AC-r1_R1_001_val_1.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp873-MV411_RNP_SPI1-H3K27AC-r2_R1_001.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp868-MV411_RNP_IRF8-H3K27AC-r1_R1_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp860-MV411_RNP_RUNX1-H3K27AC-r1_R1_001_val_1.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp866-MV411_RNP_MEF2D-H3K27AC-r1_R2_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp869-MV411_RNP_IRF8-H3K27AC-r2_R1_001.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp864-MV411_RNP_RUNX1_RUNX2-H3K27AC-r1_R2_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp863-MV411_RNP_RUNX2-H3K27AC-r2_R1_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp860-MV411_RNP_RUNX1-H3K27AC-r1_R1_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp867-MV411_RNP_MEF2D-H3K27AC-r2_R1_001_val_1.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp866-MV411_RNP_MEF2D-H3K27AC-r1_R2_001_val_2.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp862-MV411_RNP_RUNX2-H3K27AC-r1_R1_001_val_1.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp872-MV411_RNP_SPI1-H3K27AC-r1_R1_001_val_1.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp865-MV411_RNP_RUNX1_RUNX2-H3K27AC-r2_R1_001_val_1.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp863-MV411_RNP_RUNX2-H3K27AC-r2_R1_001_val_1.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp860-MV411_RNP_RUNX1-H3K27AC-r1_R1_001.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp861-MV411_RNP_RUNX1-H3K27AC-r2_R1_001_val_1.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp873-MV411_RNP_SPI1-H3K27AC-r2_R1_001_val_1.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp873-MV411_RNP_SPI1-H3K27AC-r2_R2_001_val_2.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp870-MV411_RNP_MYB-H3K27AC-r1_R1_001_val_1.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp872-MV411_RNP_SPI1-H3K27AC-r1_R1_001_val_1.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp867-MV411_RNP_MEF2D-H3K27AC-r2_R1_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp868-MV411_RNP_IRF8-H3K27AC-r1_R1_001.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp861-MV411_RNP_RUNX1-H3K27AC-r2_R1_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp858-MV411_RNP_AAVS1-H3K27AC-r1_R1_001.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp865-MV411_RNP_RUNX1_RUNX2-H3K27AC-r2_R1_001_val_1.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp858-MV411_RNP_AAVS1-H3K27AC-r1_R1_001_val_1.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp872-MV411_RNP_SPI1-H3K27AC-r1_R1_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp858-MV411_RNP_AAVS1-H3K27AC-r1_R1_001_val_1.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp871-MV411_RNP_MYB-H3K27AC-r2_R1_001_val_1.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp860-MV411_RNP_RUNX1-H3K27AC-r1_R1_001_val_1.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp858-MV411_RNP_AAVS1-H3K27AC-r1_R1_001.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp859-MV411_RNP_AAVS1-H3K27AC-r2_R1_001_val_1.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp873-MV411_RNP_SPI1-H3K27AC-r2_R1_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp869-MV411_RNP_IRF8-H3K27AC-r2_R1_001_val_1.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp859-MV411_RNP_AAVS1-H3K27AC-r2_R1_001_val_1.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp861-MV411_RNP_RUNX1-H3K27AC-r2_R1_001.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp871-MV411_RNP_MYB-H3K27AC-r2_R2_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp859-MV411_RNP_AAVS1-H3K27AC-r2_R1_001.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp861-MV411_RNP_RUNX1-H3K27AC-r2_R1_001_val_1.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp870-MV411_RNP_MYB-H3K27AC-r1_R1_001_val_1.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp871-MV411_RNP_MYB-H3K27AC-r2_R1_001.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp860-MV411_RNP_RUNX1-H3K27AC-r1_R1_001_val_1.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp865-MV411_RNP_RUNX1_RUNX2-H3K27AC-r2_R2_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp868-MV411_RNP_IRF8-H3K27AC-r1_R1_001_val_1.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp863-MV411_RNP_RUNX2-H3K27AC-r2_R2_001_val_2.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp861-MV411_RNP_RUNX1-H3K27AC-r2_R1_001_val_1.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp870-MV411_RNP_MYB-H3K27AC-r1_R1_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp861-MV411_RNP_RUNX1-H3K27AC-r2_R2_001_val_2.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp872-MV411_RNP_SPI1-H3K27AC-r1_R1_001.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp862-MV411_RNP_RUNX2-H3K27AC-r1_R1_001_val_1.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp870-MV411_RNP_MYB-H3K27AC-r1_R1_001.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp868-MV411_RNP_IRF8-H3K27AC-r1_R2_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp864-MV411_RNP_RUNX1_RUNX2-H3K27AC-r1_R1_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp870-MV411_RNP_MYB-H3K27AC-r1_R1_001_val_1.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp859-MV411_RNP_AAVS1-H3K27AC-r2_R1_001.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp862-MV411_RNP_RUNX2-H3K27AC-r1_R1_001.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp871-MV411_RNP_MYB-H3K27AC-r2_R1_001.sorted.bam.idxstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp866-MV411_RNP_MEF2D-H3K27AC-r1_R1_001_val_1.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp858-MV411_RNP_AAVS1-H3K27AC-r1_R2_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp865-MV411_RNP_RUNX1_RUNX2-H3K27AC-r2_R2_001_val_2.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp859-MV411_RNP_AAVS1-H3K27AC-r2_R2_001.fastq.gz_trimming_report.txt [Content-Type=text/plain]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp865-MV411_RNP_RUNX1_RUNX2-H3K27AC-r2_R1_001.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp860-MV411_RNP_RUNX1-H3K27AC-r1_R1_001_val_1.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp866-MV411_RNP_MEF2D-H3K27AC-r1_R1_001.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp863-MV411_RNP_RUNX2-H3K27AC-r2_R1_001_val_1.fq.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp873-MV411_RNP_SPI1-H3K27AC-r2_R1_001.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp860-MV411_RNP_RUNX1-H3K27AC-r1_R1_001.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp871-MV411_RNP_MYB-H3K27AC-r2_R1_001_val_1.sorted.bam.bai [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp860-MV411_RNP_RUNX1-H3K27AC-r1_R1_001_val_1.sorted.bam [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/droso_aligned/mp869-MV411_RNP_IRF8-H3K27AC-r2_R1_001_val_1.sorted.bam.flagstat [Content-Type=application/octet-stream]...\n",
      "\\ [192/192 files][102.8 GiB/102.8 GiB] 100% Done  95.1 MiB/s ETA 00:00:00       \n",
      "Operation completed over 192 objects/102.8 GiB.                                  \n",
      "Copying file://../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_RUNX1-H3K27AC_R1.bw [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_SPI1-H3K27AC_R2.bw [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_MYB-H3K27AC_R2.bw [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_RUNX1_RUNX2-H3K27AC_R1.bw [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_RUNX1_RUNX2-H3K27AC_R2.bw [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_IRF8-H3K27AC_R2.bw [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_RUNX2-H3K27AC_R2.bw [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_IRF8-H3K27AC_R1.bw [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_MEF2D-H3K27AC_R1.bw [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_RUNX1-H3K27AC_R2.bw [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_RUNX2-H3K27AC_R1.bw [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_SPI1-H3K27AC_R1.bw [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_MEF2D-H3K27AC_R2.bw [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_MYB-H3K27AC_R1.bw [Content-Type=application/octet-stream]...\n",
      "/ [14/14 files][  1.5 GiB/  1.5 GiB] 100% Done                                  \n",
      "Operation completed over 14 objects/1.5 GiB.                                     \n"
     ]
    }
   ],
   "source": [
    "! gsutil -m cp -r ../../data/$project/bwa/mergedLibrary/bigwig/ gs://amlproject/Chip/$project/bwa/mergedLibrary/\n",
    "! gsutil -m cp -r ../../data/$project/droso_aligned gs://amlproject/Chip/$project/\n",
    "! gsutil -m cp -r ../../data/$project/recalib_bigwig gs://amlproject/Chip/$project/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "metadata": {},
   "outputs": [],
   "source": [
    "for val in wigs:\n",
    "    bedg = val[:-6]+'bdg'\n",
    "    ! bigWigToBedGraph $val $bedg"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bdg',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bdg',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_IRF8-H3K27AC_R1.mLb.clN.bdg',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_IRF8-H3K27AC_R2.mLb.clN.bdg',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_MEF2D-H3K27AC_R1.mLb.clN.bdg',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_MEF2D-H3K27AC_R2.mLb.clN.bdg',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_MYB-H3K27AC_R1.mLb.clN.bdg',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_MYB-H3K27AC_R2.mLb.clN.bdg',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX1-H3K27AC_R1.mLb.clN.bdg',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX1-H3K27AC_R2.mLb.clN.bdg',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX1_RUNX2-H3K27AC_R1.mLb.clN.bdg',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX1_RUNX2-H3K27AC_R2.mLb.clN.bdg',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX2-H3K27AC_R1.mLb.clN.bdg',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX2-H3K27AC_R2.mLb.clN.bdg',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_SPI1-H3K27AC_R1.mLb.clN.bdg',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_SPI1-H3K27AC_R2.mLb.clN.bdg']"
      ]
     },
     "execution_count": 66,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bdg=! ls ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/*.bdg\n",
    "bdg"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 73,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bdg ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_IRF8-H3K27AC_R1.mLb.clN.bdg\n",
      "doing differential peak binding\n",
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bdg --c1 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg --t2 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_IRF8-H3K27AC_R1.mLb.clN.bdg --c2 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg --d1 1 --d2 0.3011826465 -g 60 -l 240 --o-prefix MV411_RNP_AAVS1-H3K27AC_R1_vs_MV411_RNP_IRF8-H3K27AC_R1 --outdir ../../data/additional_degraded_v1/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Wed, 09 Sep 2020 20:39:48: Read and build treatment 1 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 20:40:17: Read and build control 1 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 20:42:24: Read and build treatment 2 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 20:43:07: Read and build control 2 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 20:49:35: Write peaks... \\nINFO  @ Wed, 09 Sep 2020 20:49:35: Done \\n')\n",
      "../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bdg ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_IRF8-H3K27AC_R2.mLb.clN.bdg\n",
      "doing differential peak binding\n",
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bdg --c1 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg --t2 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_IRF8-H3K27AC_R2.mLb.clN.bdg --c2 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg --d1 1 --d2 0.4865371752 -g 60 -l 240 --o-prefix MV411_RNP_AAVS1-H3K27AC_R2_vs_MV411_RNP_IRF8-H3K27AC_R2 --outdir ../../data/additional_degraded_v1/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Wed, 09 Sep 2020 20:49:35: Read and build treatment 1 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 20:50:15: Read and build control 1 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 20:52:29: Read and build treatment 2 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 20:53:09: Read and build control 2 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 20:59:54: Write peaks... \\nINFO  @ Wed, 09 Sep 2020 20:59:54: Done \\n')\n",
      "../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bdg ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_MEF2D-H3K27AC_R1.mLb.clN.bdg\n",
      "doing differential peak binding\n",
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bdg --c1 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg --t2 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_MEF2D-H3K27AC_R1.mLb.clN.bdg --c2 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg --d1 1 --d2 0.5670857556 -g 60 -l 240 --o-prefix MV411_RNP_AAVS1-H3K27AC_R1_vs_MV411_RNP_MEF2D-H3K27AC_R1 --outdir ../../data/additional_degraded_v1/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Wed, 09 Sep 2020 20:59:55: Read and build treatment 1 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 21:00:30: Read and build control 1 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 21:02:59: Read and build treatment 2 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 21:03:46: Read and build control 2 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 21:10:24: Write peaks... \\nINFO  @ Wed, 09 Sep 2020 21:10:24: Done \\n')\n",
      "../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bdg ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_MEF2D-H3K27AC_R2.mLb.clN.bdg\n",
      "doing differential peak binding\n",
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bdg --c1 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg --t2 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_MEF2D-H3K27AC_R2.mLb.clN.bdg --c2 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg --d1 1 --d2 0.7349663619 -g 60 -l 240 --o-prefix MV411_RNP_AAVS1-H3K27AC_R2_vs_MV411_RNP_MEF2D-H3K27AC_R2 --outdir ../../data/additional_degraded_v1/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Wed, 09 Sep 2020 21:10:25: Read and build treatment 1 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 21:11:04: Read and build control 1 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 21:13:13: Read and build treatment 2 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 21:13:57: Read and build control 2 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 21:20:31: Write peaks... \\nINFO  @ Wed, 09 Sep 2020 21:20:31: Done \\n')\n",
      "../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bdg ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_MYB-H3K27AC_R1.mLb.clN.bdg\n",
      "doing differential peak binding\n",
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bdg --c1 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg --t2 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_MYB-H3K27AC_R1.mLb.clN.bdg --c2 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg --d1 1 --d2 0.7548163023 -g 60 -l 240 --o-prefix MV411_RNP_AAVS1-H3K27AC_R1_vs_MV411_RNP_MYB-H3K27AC_R1 --outdir ../../data/additional_degraded_v1/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Wed, 09 Sep 2020 21:20:32: Read and build treatment 1 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 21:21:01: Read and build control 1 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 21:23:03: Read and build treatment 2 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 21:23:33: Read and build control 2 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 21:29:31: Write peaks... \\nINFO  @ Wed, 09 Sep 2020 21:29:31: Done \\n')\n",
      "../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bdg ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_MYB-H3K27AC_R2.mLb.clN.bdg\n",
      "doing differential peak binding\n",
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bdg --c1 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg --t2 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_MYB-H3K27AC_R2.mLb.clN.bdg --c2 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg --d1 1 --d2 0.5976325206 -g 60 -l 240 --o-prefix MV411_RNP_AAVS1-H3K27AC_R2_vs_MV411_RNP_MYB-H3K27AC_R2 --outdir ../../data/additional_degraded_v1/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Wed, 09 Sep 2020 21:29:32: Read and build treatment 1 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 21:30:16: Read and build control 1 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 21:32:36: Read and build treatment 2 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 21:33:28: Read and build control 2 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 21:40:26: Write peaks... \\nINFO  @ Wed, 09 Sep 2020 21:40:26: Done \\n')\n",
      "../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bdg ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX1-H3K27AC_R1.mLb.clN.bdg\n",
      "doing differential peak binding\n",
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bdg --c1 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg --t2 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX1-H3K27AC_R1.mLb.clN.bdg --c2 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg --d1 1 --d2 1.400750948 -g 60 -l 240 --o-prefix MV411_RNP_AAVS1-H3K27AC_R1_vs_MV411_RNP_RUNX1-H3K27AC_R1 --outdir ../../data/additional_degraded_v1/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Wed, 09 Sep 2020 21:40:27: Read and build treatment 1 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 21:41:00: Read and build control 1 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 21:43:11: Read and build treatment 2 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 21:43:45: Read and build control 2 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 21:50:19: Write peaks... \\nINFO  @ Wed, 09 Sep 2020 21:50:19: Done \\n')\n",
      "../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bdg ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX1-H3K27AC_R2.mLb.clN.bdg\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bdg --c1 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg --t2 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX1-H3K27AC_R2.mLb.clN.bdg --c2 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg --d1 1 --d2 0.849637265 -g 60 -l 240 --o-prefix MV411_RNP_AAVS1-H3K27AC_R2_vs_MV411_RNP_RUNX1-H3K27AC_R2 --outdir ../../data/additional_degraded_v1/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Wed, 09 Sep 2020 21:50:20: Read and build treatment 1 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 21:51:00: Read and build control 1 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 21:53:11: Read and build treatment 2 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 21:54:06: Read and build control 2 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 22:01:04: Write peaks... \\nINFO  @ Wed, 09 Sep 2020 22:01:04: Done \\n')\n",
      "../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bdg ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX1_RUNX2-H3K27AC_R1.mLb.clN.bdg\n",
      "doing differential peak binding\n",
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bdg --c1 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg --t2 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX1_RUNX2-H3K27AC_R1.mLb.clN.bdg --c2 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg --d1 1 --d2 0.7539535476 -g 60 -l 240 --o-prefix MV411_RNP_AAVS1-H3K27AC_R1_vs_MV411_RNP_RUNX1_RUNX2-H3K27AC_R1 --outdir ../../data/additional_degraded_v1/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Wed, 09 Sep 2020 22:01:05: Read and build treatment 1 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 22:01:38: Read and build control 1 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 22:03:48: Read and build treatment 2 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 22:04:16: Read and build control 2 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 22:10:28: Write peaks... \\nINFO  @ Wed, 09 Sep 2020 22:10:28: Done \\n')\n",
      "../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bdg ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX1_RUNX2-H3K27AC_R2.mLb.clN.bdg\n",
      "doing differential peak binding\n",
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bdg --c1 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg --t2 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX1_RUNX2-H3K27AC_R2.mLb.clN.bdg --c2 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg --d1 1 --d2 0.6905692051 -g 60 -l 240 --o-prefix MV411_RNP_AAVS1-H3K27AC_R2_vs_MV411_RNP_RUNX1_RUNX2-H3K27AC_R2 --outdir ../../data/additional_degraded_v1/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Wed, 09 Sep 2020 22:10:29: Read and build treatment 1 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 22:11:09: Read and build control 1 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 22:13:27: Read and build treatment 2 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 22:14:02: Read and build control 2 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 22:20:26: Write peaks... \\nINFO  @ Wed, 09 Sep 2020 22:20:26: Done \\n')\n",
      "../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bdg ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX2-H3K27AC_R1.mLb.clN.bdg\n",
      "doing differential peak binding\n",
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bdg --c1 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg --t2 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX2-H3K27AC_R1.mLb.clN.bdg --c2 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg --d1 1 --d2 1.50505384 -g 60 -l 240 --o-prefix MV411_RNP_AAVS1-H3K27AC_R1_vs_MV411_RNP_RUNX2-H3K27AC_R1 --outdir ../../data/additional_degraded_v1/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Wed, 09 Sep 2020 22:20:26: Read and build treatment 1 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 22:20:56: Read and build control 1 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 22:23:00: Read and build treatment 2 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 22:23:44: Read and build control 2 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 22:29:31: Write peaks... \\nINFO  @ Wed, 09 Sep 2020 22:29:31: Done \\n')\n",
      "../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bdg ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX2-H3K27AC_R2.mLb.clN.bdg\n",
      "doing differential peak binding\n",
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bdg --c1 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg --t2 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_RUNX2-H3K27AC_R2.mLb.clN.bdg --c2 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg --d1 1 --d2 0.8442345485 -g 60 -l 240 --o-prefix MV411_RNP_AAVS1-H3K27AC_R2_vs_MV411_RNP_RUNX2-H3K27AC_R2 --outdir ../../data/additional_degraded_v1/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Wed, 09 Sep 2020 22:29:31: Read and build treatment 1 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 22:30:10: Read and build control 1 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 22:32:16: Read and build treatment 2 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 22:33:08: Read and build control 2 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 22:40:00: Write peaks... \\nINFO  @ Wed, 09 Sep 2020 22:40:00: Done \\n')\n",
      "../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bdg ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_SPI1-H3K27AC_R1.mLb.clN.bdg\n",
      "doing differential peak binding\n",
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bdg --c1 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg --t2 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_SPI1-H3K27AC_R1.mLb.clN.bdg --c2 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg --d1 1 --d2 0.5019100631 -g 60 -l 240 --o-prefix MV411_RNP_AAVS1-H3K27AC_R1_vs_MV411_RNP_SPI1-H3K27AC_R1 --outdir ../../data/additional_degraded_v1/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Wed, 09 Sep 2020 22:40:01: Read and build treatment 1 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 22:40:31: Read and build control 1 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 22:42:37: Read and build treatment 2 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 22:43:15: Read and build control 2 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 22:49:18: Write peaks... \\nINFO  @ Wed, 09 Sep 2020 22:49:18: Done \\n')\n",
      "../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bdg ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_SPI1-H3K27AC_R2.mLb.clN.bdg\n",
      "doing differential peak binding\n",
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bdg --c1 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg --t2 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/MV411_RNP_SPI1-H3K27AC_R2.mLb.clN.bdg --c2 ../../data/additional_degraded_v1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bdg --d1 1 --d2 0.8688220473 -g 60 -l 240 --o-prefix MV411_RNP_AAVS1-H3K27AC_R2_vs_MV411_RNP_SPI1-H3K27AC_R2 --outdir ../../data/additional_degraded_v1/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Wed, 09 Sep 2020 22:49:19: Read and build treatment 1 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 22:49:58: Read and build control 1 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 22:52:03: Read and build treatment 2 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 22:52:47: Read and build control 2 bedGraph... \\nINFO  @ Wed, 09 Sep 2020 22:59:14: Write peaks... \\nINFO  @ Wed, 09 Sep 2020 22:59:14: Done \\n')\n"
     ]
    }
   ],
   "source": [
    "# diffPeak on scaled data \n",
    "size= 240\n",
    "for i in range(int(len(bdg)-3)):\n",
    "    name1 = bdg[1+i%2]\n",
    "    name2 = bdg[3+i]\n",
    "    print(name1,name2)\n",
    "    print(chip.diffPeak(name1, name2, control1=bdg[0], control2=bdg[0], res_directory=\"../../data/\"+project+\"/diffPeaks/\", scaling1=1, scaling2=scales[i], size=size))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 148,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/additional_degraded_v1/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_IRF8-H3K27AC_R1.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_IRF8-H3K27AC_R2.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_MEF2D-H3K27AC_R1.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_MEF2D-H3K27AC_R2.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_MYB-H3K27AC_R1.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_MYB-H3K27AC_R2.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX1-H3K27AC_R1.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX1-H3K27AC_R2.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX1_RUNX2-H3K27AC_R1.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX1_RUNX2-H3K27AC_R2.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX2-H3K27AC_R1.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX2-H3K27AC_R2.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_SPI1-H3K27AC_R1.mLb.clN.sorted.bam',\n",
       " '../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_SPI1-H3K27AC_R2.mLb.clN.sorted.bam']"
      ]
     },
     "execution_count": 148,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 154,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX1-H3K27AC_R2.mLb.clN.sorted.bam'"
      ]
     },
     "execution_count": 154,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bams[10]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 155,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.sorted.bam ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX1_RUNX2-H3K27AC_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.sorted.bam and ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX1_RUNX2-H3K27AC_R1.mLb.clN.sorted.bam\n",
      "using default|given size\n",
      "computing the scaling values\n",
      "b'INFO  @ Mon, 14 Sep 2020 15:06:40: \\n# Command line: callpeak -B -t ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.sorted.bam -c ../../data/additional_degraded_v1/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 240 -n MV411_RNP_AAVS1-H3K27AC_R1 --outdir ../../data/additional_degraded_v1/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_RNP_AAVS1-H3K27AC_R1\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/additional_degraded_v1/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Mon, 14 Sep 2020 15:06:40: #1 read fragment files... \\nINFO  @ Mon, 14 Sep 2020 15:06:40: #1 read treatment fragments... \\nINFO  @ Mon, 14 Sep 2020 15:06:45:  1000000 \\nINFO  @ Mon, 14 Sep 2020 15:06:50:  2000000 \\nINFO  @ Mon, 14 Sep 2020 15:06:55:  3000000 \\nINFO  @ Mon, 14 Sep 2020 15:06:59:  4000000 \\nINFO  @ Mon, 14 Sep 2020 15:07:04:  5000000 \\nINFO  @ Mon, 14 Sep 2020 15:07:09:  6000000 \\nINFO  @ Mon, 14 Sep 2020 15:07:14:  7000000 \\nINFO  @ Mon, 14 Sep 2020 15:07:18:  8000000 \\nINFO  @ Mon, 14 Sep 2020 15:07:23:  9000000 \\nINFO  @ Mon, 14 Sep 2020 15:07:28:  10000000 \\nINFO  @ Mon, 14 Sep 2020 15:07:32:  11000000 \\nINFO  @ Mon, 14 Sep 2020 15:07:37:  12000000 \\nINFO  @ Mon, 14 Sep 2020 15:07:42:  13000000 \\nINFO  @ Mon, 14 Sep 2020 15:07:47:  14000000 \\nINFO  @ Mon, 14 Sep 2020 15:07:51:  15000000 \\nINFO  @ Mon, 14 Sep 2020 15:07:56:  16000000 \\nINFO  @ Mon, 14 Sep 2020 15:08:01:  17000000 \\nINFO  @ Mon, 14 Sep 2020 15:08:05:  18000000 \\nINFO  @ Mon, 14 Sep 2020 15:08:10:  19000000 \\nINFO  @ Mon, 14 Sep 2020 15:08:15:  20000000 \\nINFO  @ Mon, 14 Sep 2020 15:08:18: 20629699 fragments have been read. \\nINFO  @ Mon, 14 Sep 2020 15:08:30: #1.2 read input fragments... \\nINFO  @ Mon, 14 Sep 2020 15:08:34:  1000000 \\nINFO  @ Mon, 14 Sep 2020 15:08:38:  2000000 \\nINFO  @ Mon, 14 Sep 2020 15:08:41:  3000000 \\nINFO  @ Mon, 14 Sep 2020 15:08:45:  4000000 \\nINFO  @ Mon, 14 Sep 2020 15:08:49:  5000000 \\nINFO  @ Mon, 14 Sep 2020 15:08:53:  6000000 \\nINFO  @ Mon, 14 Sep 2020 15:08:57:  7000000 \\nINFO  @ Mon, 14 Sep 2020 15:09:01:  8000000 \\nINFO  @ Mon, 14 Sep 2020 15:09:04:  9000000 \\nINFO  @ Mon, 14 Sep 2020 15:09:08:  10000000 \\nINFO  @ Mon, 14 Sep 2020 15:09:12:  11000000 \\nINFO  @ Mon, 14 Sep 2020 15:09:16:  12000000 \\nINFO  @ Mon, 14 Sep 2020 15:09:20:  13000000 \\nINFO  @ Mon, 14 Sep 2020 15:09:24:  14000000 \\nINFO  @ Mon, 14 Sep 2020 15:09:28:  15000000 \\nINFO  @ Mon, 14 Sep 2020 15:09:32:  16000000 \\nINFO  @ Mon, 14 Sep 2020 15:09:36:  17000000 \\nINFO  @ Mon, 14 Sep 2020 15:09:39:  18000000 \\nINFO  @ Mon, 14 Sep 2020 15:09:43:  19000000 \\nINFO  @ Mon, 14 Sep 2020 15:09:47:  20000000 \\nINFO  @ Mon, 14 Sep 2020 15:09:51:  21000000 \\nINFO  @ Mon, 14 Sep 2020 15:09:55:  22000000 \\nINFO  @ Mon, 14 Sep 2020 15:09:58:  23000000 \\nINFO  @ Mon, 14 Sep 2020 15:10:02:  24000000 \\nINFO  @ Mon, 14 Sep 2020 15:10:06:  25000000 \\nINFO  @ Mon, 14 Sep 2020 15:10:10:  26000000 \\nINFO  @ Mon, 14 Sep 2020 15:10:14:  27000000 \\nINFO  @ Mon, 14 Sep 2020 15:10:18:  28000000 \\nINFO  @ Mon, 14 Sep 2020 15:10:21:  29000000 \\nINFO  @ Mon, 14 Sep 2020 15:10:25:  30000000 \\nINFO  @ Mon, 14 Sep 2020 15:10:29:  31000000 \\nINFO  @ Mon, 14 Sep 2020 15:10:33:  32000000 \\nINFO  @ Mon, 14 Sep 2020 15:10:37:  33000000 \\nINFO  @ Mon, 14 Sep 2020 15:10:40:  34000000 \\nINFO  @ Mon, 14 Sep 2020 15:10:44:  35000000 \\nINFO  @ Mon, 14 Sep 2020 15:10:48:  36000000 \\nINFO  @ Mon, 14 Sep 2020 15:10:52:  37000000 \\nINFO  @ Mon, 14 Sep 2020 15:10:56:  38000000 \\nINFO  @ Mon, 14 Sep 2020 15:10:59:  39000000 \\nINFO  @ Mon, 14 Sep 2020 15:11:03:  40000000 \\nINFO  @ Mon, 14 Sep 2020 15:11:07:  41000000 \\nINFO  @ Mon, 14 Sep 2020 15:11:11:  42000000 \\nINFO  @ Mon, 14 Sep 2020 15:11:15:  43000000 \\nINFO  @ Mon, 14 Sep 2020 15:11:19:  44000000 \\nINFO  @ Mon, 14 Sep 2020 15:11:23:  45000000 \\nINFO  @ Mon, 14 Sep 2020 15:11:27:  46000000 \\nINFO  @ Mon, 14 Sep 2020 15:11:30:  47000000 \\nINFO  @ Mon, 14 Sep 2020 15:11:34:  48000000 \\nINFO  @ Mon, 14 Sep 2020 15:11:38:  49000000 \\nINFO  @ Mon, 14 Sep 2020 15:11:42:  50000000 \\nINFO  @ Mon, 14 Sep 2020 15:11:46:  51000000 \\nINFO  @ Mon, 14 Sep 2020 15:11:49:  52000000 \\nINFO  @ Mon, 14 Sep 2020 15:11:53:  53000000 \\nINFO  @ Mon, 14 Sep 2020 15:11:57:  54000000 \\nINFO  @ Mon, 14 Sep 2020 15:12:01:  55000000 \\nINFO  @ Mon, 14 Sep 2020 15:12:05:  56000000 \\nINFO  @ Mon, 14 Sep 2020 15:12:08:  57000000 \\nINFO  @ Mon, 14 Sep 2020 15:12:12:  58000000 \\nINFO  @ Mon, 14 Sep 2020 15:12:16:  59000000 \\nINFO  @ Mon, 14 Sep 2020 15:12:20:  60000000 \\nINFO  @ Mon, 14 Sep 2020 15:12:24:  61000000 \\nINFO  @ Mon, 14 Sep 2020 15:12:28:  62000000 \\nINFO  @ Mon, 14 Sep 2020 15:12:32:  63000000 \\nINFO  @ Mon, 14 Sep 2020 15:12:36:  64000000 \\nINFO  @ Mon, 14 Sep 2020 15:12:39:  65000000 \\nINFO  @ Mon, 14 Sep 2020 15:12:43:  66000000 \\nINFO  @ Mon, 14 Sep 2020 15:12:46: 66660428 fragments have been read. \\nINFO  @ Mon, 14 Sep 2020 15:13:30: #1 mean fragment size is determined as 309.7 bp from treatment \\nINFO  @ Mon, 14 Sep 2020 15:13:30: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Mon, 14 Sep 2020 15:13:30: #1 fragment size = 309.7 \\nINFO  @ Mon, 14 Sep 2020 15:13:30: #1  total fragments in treatment: 20629699 \\nINFO  @ Mon, 14 Sep 2020 15:13:30: #1 user defined the maximum fragments... \\nINFO  @ Mon, 14 Sep 2020 15:13:30: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 14 Sep 2020 15:14:03: #1  fragments after filtering in treatment: 20624221 \\nINFO  @ Mon, 14 Sep 2020 15:14:03: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 14 Sep 2020 15:14:03: #1  total fragments in control: 66660428 \\nINFO  @ Mon, 14 Sep 2020 15:14:03: #1 user defined the maximum fragments... \\nINFO  @ Mon, 14 Sep 2020 15:14:03: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 14 Sep 2020 15:15:48: #1  fragments after filtering in control: 66632618 \\nINFO  @ Mon, 14 Sep 2020 15:15:48: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 14 Sep 2020 15:15:48: #1 finished! \\nINFO  @ Mon, 14 Sep 2020 15:15:48: #2 Build Peak Model... \\nINFO  @ Mon, 14 Sep 2020 15:15:48: #2 Skipped... \\nINFO  @ Mon, 14 Sep 2020 15:15:48: #3 Call peaks... \\nINFO  @ Mon, 14 Sep 2020 15:15:48: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 14 Sep 2020 15:25:19: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 14 Sep 2020 15:25:19: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup.bdg \\nINFO  @ Mon, 14 Sep 2020 15:25:19: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_RNP_AAVS1-H3K27AC_R1_control_lambda.bdg \\nINFO  @ Mon, 14 Sep 2020 15:25:19: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 14 Sep 2020 15:25:19: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 14 Sep 2020 15:31:10: #4 Write output xls file... ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R1_peaks.xls \\nINFO  @ Mon, 14 Sep 2020 15:31:10: #4 Write peak in narrowPeak format file... ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R1_peaks.narrowPeak \\nINFO  @ Mon, 14 Sep 2020 15:31:10: #4 Write summits bed file... ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R1_summits.bed \\nINFO  @ Mon, 14 Sep 2020 15:31:11: Done! \\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'INFO  @ Mon, 14 Sep 2020 15:31:11: \\n# Command line: callpeak -B -t ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX1_RUNX2-H3K27AC_R1.mLb.clN.sorted.bam -c ../../data/additional_degraded_v1/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 240 -n MV411_RNP_RUNX1_RUNX2-H3K27AC_R1 --outdir ../../data/additional_degraded_v1/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_RNP_RUNX1_RUNX2-H3K27AC_R1\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX1_RUNX2-H3K27AC_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/additional_degraded_v1/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Mon, 14 Sep 2020 15:31:11: #1 read fragment files... \\nINFO  @ Mon, 14 Sep 2020 15:31:11: #1 read treatment fragments... \\nINFO  @ Mon, 14 Sep 2020 15:31:16:  1000000 \\nINFO  @ Mon, 14 Sep 2020 15:31:20:  2000000 \\nINFO  @ Mon, 14 Sep 2020 15:31:25:  3000000 \\nINFO  @ Mon, 14 Sep 2020 15:31:29:  4000000 \\nINFO  @ Mon, 14 Sep 2020 15:31:34:  5000000 \\nINFO  @ Mon, 14 Sep 2020 15:31:39:  6000000 \\nINFO  @ Mon, 14 Sep 2020 15:31:43:  7000000 \\nINFO  @ Mon, 14 Sep 2020 15:31:48:  8000000 \\nINFO  @ Mon, 14 Sep 2020 15:31:53:  9000000 \\nINFO  @ Mon, 14 Sep 2020 15:31:57:  10000000 \\nINFO  @ Mon, 14 Sep 2020 15:32:02:  11000000 \\nINFO  @ Mon, 14 Sep 2020 15:32:07:  12000000 \\nINFO  @ Mon, 14 Sep 2020 15:32:11:  13000000 \\nINFO  @ Mon, 14 Sep 2020 15:32:16:  14000000 \\nINFO  @ Mon, 14 Sep 2020 15:32:20:  15000000 \\nINFO  @ Mon, 14 Sep 2020 15:32:25:  16000000 \\nINFO  @ Mon, 14 Sep 2020 15:32:29:  17000000 \\nINFO  @ Mon, 14 Sep 2020 15:32:34:  18000000 \\nINFO  @ Mon, 14 Sep 2020 15:32:38: 18809805 fragments have been read. \\nINFO  @ Mon, 14 Sep 2020 15:32:49: #1.2 read input fragments... \\nINFO  @ Mon, 14 Sep 2020 15:32:53:  1000000 \\nINFO  @ Mon, 14 Sep 2020 15:32:56:  2000000 \\nINFO  @ Mon, 14 Sep 2020 15:33:00:  3000000 \\nINFO  @ Mon, 14 Sep 2020 15:33:04:  4000000 \\nINFO  @ Mon, 14 Sep 2020 15:33:08:  5000000 \\nINFO  @ Mon, 14 Sep 2020 15:33:11:  6000000 \\nINFO  @ Mon, 14 Sep 2020 15:33:15:  7000000 \\nINFO  @ Mon, 14 Sep 2020 15:33:19:  8000000 \\nINFO  @ Mon, 14 Sep 2020 15:33:22:  9000000 \\nINFO  @ Mon, 14 Sep 2020 15:33:27:  10000000 \\nINFO  @ Mon, 14 Sep 2020 15:33:36:  11000000 \\nINFO  @ Mon, 14 Sep 2020 15:33:40:  12000000 \\nINFO  @ Mon, 14 Sep 2020 15:33:44:  13000000 \\nINFO  @ Mon, 14 Sep 2020 15:33:48:  14000000 \\nINFO  @ Mon, 14 Sep 2020 15:33:52:  15000000 \\nINFO  @ Mon, 14 Sep 2020 15:33:56:  16000000 \\nINFO  @ Mon, 14 Sep 2020 15:33:59:  17000000 \\nINFO  @ Mon, 14 Sep 2020 15:34:03:  18000000 \\nINFO  @ Mon, 14 Sep 2020 15:34:07:  19000000 \\nINFO  @ Mon, 14 Sep 2020 15:34:11:  20000000 \\nINFO  @ Mon, 14 Sep 2020 15:34:15:  21000000 \\nINFO  @ Mon, 14 Sep 2020 15:34:19:  22000000 \\nINFO  @ Mon, 14 Sep 2020 15:34:23:  23000000 \\nINFO  @ Mon, 14 Sep 2020 15:34:26:  24000000 \\nINFO  @ Mon, 14 Sep 2020 15:34:30:  25000000 \\nINFO  @ Mon, 14 Sep 2020 15:34:34:  26000000 \\nINFO  @ Mon, 14 Sep 2020 15:34:38:  27000000 \\nINFO  @ Mon, 14 Sep 2020 15:34:42:  28000000 \\nINFO  @ Mon, 14 Sep 2020 15:34:46:  29000000 \\nINFO  @ Mon, 14 Sep 2020 15:34:50:  30000000 \\nINFO  @ Mon, 14 Sep 2020 15:34:53:  31000000 \\nINFO  @ Mon, 14 Sep 2020 15:34:57:  32000000 \\nINFO  @ Mon, 14 Sep 2020 15:35:01:  33000000 \\nINFO  @ Mon, 14 Sep 2020 15:35:05:  34000000 \\nINFO  @ Mon, 14 Sep 2020 15:35:09:  35000000 \\nINFO  @ Mon, 14 Sep 2020 15:35:13:  36000000 \\nINFO  @ Mon, 14 Sep 2020 15:35:16:  37000000 \\nINFO  @ Mon, 14 Sep 2020 15:35:20:  38000000 \\nINFO  @ Mon, 14 Sep 2020 15:35:24:  39000000 \\nINFO  @ Mon, 14 Sep 2020 15:35:28:  40000000 \\nINFO  @ Mon, 14 Sep 2020 15:35:32:  41000000 \\nINFO  @ Mon, 14 Sep 2020 15:35:36:  42000000 \\nINFO  @ Mon, 14 Sep 2020 15:35:40:  43000000 \\nINFO  @ Mon, 14 Sep 2020 15:35:44:  44000000 \\nINFO  @ Mon, 14 Sep 2020 15:35:48:  45000000 \\nINFO  @ Mon, 14 Sep 2020 15:35:52:  46000000 \\nINFO  @ Mon, 14 Sep 2020 15:35:56:  47000000 \\nINFO  @ Mon, 14 Sep 2020 15:36:00:  48000000 \\nINFO  @ Mon, 14 Sep 2020 15:36:03:  49000000 \\nINFO  @ Mon, 14 Sep 2020 15:36:07:  50000000 \\nINFO  @ Mon, 14 Sep 2020 15:36:11:  51000000 \\nINFO  @ Mon, 14 Sep 2020 15:36:15:  52000000 \\nINFO  @ Mon, 14 Sep 2020 15:36:19:  53000000 \\nINFO  @ Mon, 14 Sep 2020 15:36:23:  54000000 \\nINFO  @ Mon, 14 Sep 2020 15:36:27:  55000000 \\nINFO  @ Mon, 14 Sep 2020 15:36:30:  56000000 \\nINFO  @ Mon, 14 Sep 2020 15:36:34:  57000000 \\nINFO  @ Mon, 14 Sep 2020 15:36:38:  58000000 \\nINFO  @ Mon, 14 Sep 2020 15:36:42:  59000000 \\nINFO  @ Mon, 14 Sep 2020 15:36:46:  60000000 \\nINFO  @ Mon, 14 Sep 2020 15:36:50:  61000000 \\nINFO  @ Mon, 14 Sep 2020 15:36:54:  62000000 \\nINFO  @ Mon, 14 Sep 2020 15:36:58:  63000000 \\nINFO  @ Mon, 14 Sep 2020 15:37:01:  64000000 \\nINFO  @ Mon, 14 Sep 2020 15:37:05:  65000000 \\nINFO  @ Mon, 14 Sep 2020 15:37:09:  66000000 \\nINFO  @ Mon, 14 Sep 2020 15:37:12: 66660428 fragments have been read. \\nINFO  @ Mon, 14 Sep 2020 15:37:56: #1 mean fragment size is determined as 326.7 bp from treatment \\nINFO  @ Mon, 14 Sep 2020 15:37:56: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Mon, 14 Sep 2020 15:37:56: #1 fragment size = 326.7 \\nINFO  @ Mon, 14 Sep 2020 15:37:56: #1  total fragments in treatment: 18809805 \\nINFO  @ Mon, 14 Sep 2020 15:37:56: #1 user defined the maximum fragments... \\nINFO  @ Mon, 14 Sep 2020 15:37:56: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 14 Sep 2020 15:38:27: #1  fragments after filtering in treatment: 18802227 \\nINFO  @ Mon, 14 Sep 2020 15:38:27: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 14 Sep 2020 15:38:27: #1  total fragments in control: 66660428 \\nINFO  @ Mon, 14 Sep 2020 15:38:27: #1 user defined the maximum fragments... \\nINFO  @ Mon, 14 Sep 2020 15:38:27: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 14 Sep 2020 15:40:13: #1  fragments after filtering in control: 66632618 \\nINFO  @ Mon, 14 Sep 2020 15:40:13: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 14 Sep 2020 15:40:13: #1 finished! \\nINFO  @ Mon, 14 Sep 2020 15:40:13: #2 Build Peak Model... \\nINFO  @ Mon, 14 Sep 2020 15:40:13: #2 Skipped... \\nINFO  @ Mon, 14 Sep 2020 15:40:13: #3 Call peaks... \\nINFO  @ Mon, 14 Sep 2020 15:40:13: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 14 Sep 2020 15:49:19: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 14 Sep 2020 15:49:19: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_RNP_RUNX1_RUNX2-H3K27AC_R1_treat_pileup.bdg \\nINFO  @ Mon, 14 Sep 2020 15:49:19: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_RNP_RUNX1_RUNX2-H3K27AC_R1_control_lambda.bdg \\nINFO  @ Mon, 14 Sep 2020 15:49:19: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 14 Sep 2020 15:49:19: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 14 Sep 2020 15:55:08: #4 Write output xls file... ../../data/additional_degraded_v1/diffData/MV411_RNP_RUNX1_RUNX2-H3K27AC_R1_peaks.xls \\nINFO  @ Mon, 14 Sep 2020 15:55:08: #4 Write peak in narrowPeak format file... ../../data/additional_degraded_v1/diffData/MV411_RNP_RUNX1_RUNX2-H3K27AC_R1_peaks.narrowPeak \\nINFO  @ Mon, 14 Sep 2020 15:55:09: #4 Write summits bed file... ../../data/additional_degraded_v1/diffData/MV411_RNP_RUNX1_RUNX2-H3K27AC_R1_summits.bed \\nINFO  @ Mon, 14 Sep 2020 15:55:09: Done! \\n'\n",
      "15549704 18802227\n",
      "doing differential peak binding\n",
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup.bdg --c1 ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R1_control_lambda.bdg --t2 ../../data/additional_degraded_v1/diffData/MV411_RNP_RUNX1_RUNX2-H3K27AC_R1_treat_pileup.bdg --c2 ../../data/additional_degraded_v1/diffData/MV411_RNP_RUNX1_RUNX2-H3K27AC_R1_control_lambda.bdg --d1 15549704 --d2 18802227 -g 60 -l 240 --o-prefix MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_RUNX1_RUNX2-H3K27AC_R1_treat_pileup --outdir ../../data/additional_degraded_v1/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Mon, 14 Sep 2020 15:55:09: Read and build treatment 1 bedGraph... \\nINFO  @ Mon, 14 Sep 2020 15:55:40: Read and build control 1 bedGraph... \\nINFO  @ Mon, 14 Sep 2020 15:58:59: Read and build treatment 2 bedGraph... \\nINFO  @ Mon, 14 Sep 2020 15:59:27: Read and build control 2 bedGraph... \\nINFO  @ Mon, 14 Sep 2020 16:11:36: Write peaks... \\nINFO  @ Mon, 14 Sep 2020 16:11:36: Done \\n')\n",
      "../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.sorted.bam ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX1_RUNX2-H3K27AC_R2.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.sorted.bam and ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX1_RUNX2-H3K27AC_R2.mLb.clN.sorted.bam\n",
      "using default|given size\n",
      "computing the scaling values\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'INFO  @ Mon, 14 Sep 2020 16:11:37: \\n# Command line: callpeak -B -t ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.sorted.bam -c ../../data/additional_degraded_v1/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 240 -n MV411_RNP_AAVS1-H3K27AC_R2 --outdir ../../data/additional_degraded_v1/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_RNP_AAVS1-H3K27AC_R2\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/additional_degraded_v1/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Mon, 14 Sep 2020 16:11:37: #1 read fragment files... \\nINFO  @ Mon, 14 Sep 2020 16:11:37: #1 read treatment fragments... \\nINFO  @ Mon, 14 Sep 2020 16:11:42:  1000000 \\nINFO  @ Mon, 14 Sep 2020 16:11:47:  2000000 \\nINFO  @ Mon, 14 Sep 2020 16:11:51:  3000000 \\nINFO  @ Mon, 14 Sep 2020 16:11:56:  4000000 \\nINFO  @ Mon, 14 Sep 2020 16:12:01:  5000000 \\nINFO  @ Mon, 14 Sep 2020 16:12:06:  6000000 \\nINFO  @ Mon, 14 Sep 2020 16:12:10:  7000000 \\nINFO  @ Mon, 14 Sep 2020 16:12:15:  8000000 \\nINFO  @ Mon, 14 Sep 2020 16:12:20:  9000000 \\nINFO  @ Mon, 14 Sep 2020 16:12:25:  10000000 \\nINFO  @ Mon, 14 Sep 2020 16:12:29:  11000000 \\nINFO  @ Mon, 14 Sep 2020 16:12:34:  12000000 \\nINFO  @ Mon, 14 Sep 2020 16:12:39:  13000000 \\nINFO  @ Mon, 14 Sep 2020 16:12:43:  14000000 \\nINFO  @ Mon, 14 Sep 2020 16:12:48:  15000000 \\nINFO  @ Mon, 14 Sep 2020 16:12:53:  16000000 \\nINFO  @ Mon, 14 Sep 2020 16:12:58:  17000000 \\nINFO  @ Mon, 14 Sep 2020 16:13:02:  18000000 \\nINFO  @ Mon, 14 Sep 2020 16:13:07:  19000000 \\nINFO  @ Mon, 14 Sep 2020 16:13:12:  20000000 \\nINFO  @ Mon, 14 Sep 2020 16:13:16:  21000000 \\nINFO  @ Mon, 14 Sep 2020 16:13:21:  22000000 \\nINFO  @ Mon, 14 Sep 2020 16:13:26:  23000000 \\nINFO  @ Mon, 14 Sep 2020 16:13:30:  24000000 \\nINFO  @ Mon, 14 Sep 2020 16:13:35:  25000000 \\nINFO  @ Mon, 14 Sep 2020 16:13:40:  26000000 \\nINFO  @ Mon, 14 Sep 2020 16:13:44:  27000000 \\nINFO  @ Mon, 14 Sep 2020 16:13:47: 27510868 fragments have been read. \\nINFO  @ Mon, 14 Sep 2020 16:14:05: #1.2 read input fragments... \\nINFO  @ Mon, 14 Sep 2020 16:14:09:  1000000 \\nINFO  @ Mon, 14 Sep 2020 16:14:13:  2000000 \\nINFO  @ Mon, 14 Sep 2020 16:14:17:  3000000 \\nINFO  @ Mon, 14 Sep 2020 16:14:21:  4000000 \\nINFO  @ Mon, 14 Sep 2020 16:14:24:  5000000 \\nINFO  @ Mon, 14 Sep 2020 16:14:28:  6000000 \\nINFO  @ Mon, 14 Sep 2020 16:14:32:  7000000 \\nINFO  @ Mon, 14 Sep 2020 16:14:36:  8000000 \\nINFO  @ Mon, 14 Sep 2020 16:14:40:  9000000 \\nINFO  @ Mon, 14 Sep 2020 16:14:44:  10000000 \\nINFO  @ Mon, 14 Sep 2020 16:14:48:  11000000 \\nINFO  @ Mon, 14 Sep 2020 16:14:52:  12000000 \\nINFO  @ Mon, 14 Sep 2020 16:14:55:  13000000 \\nINFO  @ Mon, 14 Sep 2020 16:14:59:  14000000 \\nINFO  @ Mon, 14 Sep 2020 16:15:03:  15000000 \\nINFO  @ Mon, 14 Sep 2020 16:15:07:  16000000 \\nINFO  @ Mon, 14 Sep 2020 16:15:10:  17000000 \\nINFO  @ Mon, 14 Sep 2020 16:15:14:  18000000 \\nINFO  @ Mon, 14 Sep 2020 16:15:18:  19000000 \\nINFO  @ Mon, 14 Sep 2020 16:15:22:  20000000 \\nINFO  @ Mon, 14 Sep 2020 16:15:25:  21000000 \\nINFO  @ Mon, 14 Sep 2020 16:15:29:  22000000 \\nINFO  @ Mon, 14 Sep 2020 16:15:33:  23000000 \\nINFO  @ Mon, 14 Sep 2020 16:15:37:  24000000 \\nINFO  @ Mon, 14 Sep 2020 16:15:40:  25000000 \\nINFO  @ Mon, 14 Sep 2020 16:15:44:  26000000 \\nINFO  @ Mon, 14 Sep 2020 16:15:48:  27000000 \\nINFO  @ Mon, 14 Sep 2020 16:15:52:  28000000 \\nINFO  @ Mon, 14 Sep 2020 16:15:55:  29000000 \\nINFO  @ Mon, 14 Sep 2020 16:15:59:  30000000 \\nINFO  @ Mon, 14 Sep 2020 16:16:03:  31000000 \\nINFO  @ Mon, 14 Sep 2020 16:16:07:  32000000 \\nINFO  @ Mon, 14 Sep 2020 16:16:10:  33000000 \\nINFO  @ Mon, 14 Sep 2020 16:16:14:  34000000 \\nINFO  @ Mon, 14 Sep 2020 16:16:18:  35000000 \\nINFO  @ Mon, 14 Sep 2020 16:16:22:  36000000 \\nINFO  @ Mon, 14 Sep 2020 16:16:25:  37000000 \\nINFO  @ Mon, 14 Sep 2020 16:16:29:  38000000 \\nINFO  @ Mon, 14 Sep 2020 16:16:33:  39000000 \\nINFO  @ Mon, 14 Sep 2020 16:16:37:  40000000 \\nINFO  @ Mon, 14 Sep 2020 16:16:40:  41000000 \\nINFO  @ Mon, 14 Sep 2020 16:16:44:  42000000 \\nINFO  @ Mon, 14 Sep 2020 16:16:48:  43000000 \\nINFO  @ Mon, 14 Sep 2020 16:16:52:  44000000 \\nINFO  @ Mon, 14 Sep 2020 16:16:56:  45000000 \\nINFO  @ Mon, 14 Sep 2020 16:16:59:  46000000 \\nINFO  @ Mon, 14 Sep 2020 16:17:03:  47000000 \\nINFO  @ Mon, 14 Sep 2020 16:17:07:  48000000 \\nINFO  @ Mon, 14 Sep 2020 16:17:11:  49000000 \\nINFO  @ Mon, 14 Sep 2020 16:17:14:  50000000 \\nINFO  @ Mon, 14 Sep 2020 16:17:18:  51000000 \\nINFO  @ Mon, 14 Sep 2020 16:17:22:  52000000 \\nINFO  @ Mon, 14 Sep 2020 16:17:26:  53000000 \\nINFO  @ Mon, 14 Sep 2020 16:17:30:  54000000 \\nINFO  @ Mon, 14 Sep 2020 16:17:33:  55000000 \\nINFO  @ Mon, 14 Sep 2020 16:17:37:  56000000 \\nINFO  @ Mon, 14 Sep 2020 16:17:41:  57000000 \\nINFO  @ Mon, 14 Sep 2020 16:17:45:  58000000 \\nINFO  @ Mon, 14 Sep 2020 16:17:48:  59000000 \\nINFO  @ Mon, 14 Sep 2020 16:17:52:  60000000 \\nINFO  @ Mon, 14 Sep 2020 16:17:56:  61000000 \\nINFO  @ Mon, 14 Sep 2020 16:18:00:  62000000 \\nINFO  @ Mon, 14 Sep 2020 16:18:05:  63000000 \\nINFO  @ Mon, 14 Sep 2020 16:18:09:  64000000 \\nINFO  @ Mon, 14 Sep 2020 16:18:13:  65000000 \\nINFO  @ Mon, 14 Sep 2020 16:18:17:  66000000 \\nINFO  @ Mon, 14 Sep 2020 16:18:19: 66660428 fragments have been read. \\nINFO  @ Mon, 14 Sep 2020 16:19:03: #1 mean fragment size is determined as 304.3 bp from treatment \\nINFO  @ Mon, 14 Sep 2020 16:19:03: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Mon, 14 Sep 2020 16:19:03: #1 fragment size = 304.3 \\nINFO  @ Mon, 14 Sep 2020 16:19:03: #1  total fragments in treatment: 27510868 \\nINFO  @ Mon, 14 Sep 2020 16:19:03: #1 user defined the maximum fragments... \\nINFO  @ Mon, 14 Sep 2020 16:19:03: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 14 Sep 2020 16:19:47: #1  fragments after filtering in treatment: 27503766 \\nINFO  @ Mon, 14 Sep 2020 16:19:47: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 14 Sep 2020 16:19:47: #1  total fragments in control: 66660428 \\nINFO  @ Mon, 14 Sep 2020 16:19:47: #1 user defined the maximum fragments... \\nINFO  @ Mon, 14 Sep 2020 16:19:47: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 14 Sep 2020 16:21:33: #1  fragments after filtering in control: 66632618 \\nINFO  @ Mon, 14 Sep 2020 16:21:33: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 14 Sep 2020 16:21:33: #1 finished! \\nINFO  @ Mon, 14 Sep 2020 16:21:33: #2 Build Peak Model... \\nINFO  @ Mon, 14 Sep 2020 16:21:33: #2 Skipped... \\nINFO  @ Mon, 14 Sep 2020 16:21:33: #3 Call peaks... \\nINFO  @ Mon, 14 Sep 2020 16:21:33: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 14 Sep 2020 16:31:04: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 14 Sep 2020 16:31:04: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup.bdg \\nINFO  @ Mon, 14 Sep 2020 16:31:04: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_RNP_AAVS1-H3K27AC_R2_control_lambda.bdg \\nINFO  @ Mon, 14 Sep 2020 16:31:04: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 14 Sep 2020 16:31:04: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 14 Sep 2020 16:37:05: #4 Write output xls file... ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R2_peaks.xls \\nINFO  @ Mon, 14 Sep 2020 16:37:06: #4 Write peak in narrowPeak format file... ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R2_peaks.narrowPeak \\nINFO  @ Mon, 14 Sep 2020 16:37:06: #4 Write summits bed file... ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R2_summits.bed \\nINFO  @ Mon, 14 Sep 2020 16:37:06: Done! \\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'INFO  @ Mon, 14 Sep 2020 16:37:06: \\n# Command line: callpeak -B -t ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX1_RUNX2-H3K27AC_R2.mLb.clN.sorted.bam -c ../../data/additional_degraded_v1/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 240 -n MV411_RNP_RUNX1_RUNX2-H3K27AC_R2 --outdir ../../data/additional_degraded_v1/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_RNP_RUNX1_RUNX2-H3K27AC_R2\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX1_RUNX2-H3K27AC_R2.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/additional_degraded_v1/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Mon, 14 Sep 2020 16:37:06: #1 read fragment files... \\nINFO  @ Mon, 14 Sep 2020 16:37:06: #1 read treatment fragments... \\nINFO  @ Mon, 14 Sep 2020 16:37:11:  1000000 \\nINFO  @ Mon, 14 Sep 2020 16:37:15:  2000000 \\nINFO  @ Mon, 14 Sep 2020 16:37:20:  3000000 \\nINFO  @ Mon, 14 Sep 2020 16:37:25:  4000000 \\nINFO  @ Mon, 14 Sep 2020 16:37:29:  5000000 \\nINFO  @ Mon, 14 Sep 2020 16:37:34:  6000000 \\nINFO  @ Mon, 14 Sep 2020 16:37:39:  7000000 \\nINFO  @ Mon, 14 Sep 2020 16:37:43:  8000000 \\nINFO  @ Mon, 14 Sep 2020 16:37:48:  9000000 \\nINFO  @ Mon, 14 Sep 2020 16:37:53:  10000000 \\nINFO  @ Mon, 14 Sep 2020 16:37:57:  11000000 \\nINFO  @ Mon, 14 Sep 2020 16:38:02:  12000000 \\nINFO  @ Mon, 14 Sep 2020 16:38:07:  13000000 \\nINFO  @ Mon, 14 Sep 2020 16:38:11:  14000000 \\nINFO  @ Mon, 14 Sep 2020 16:38:16:  15000000 \\nINFO  @ Mon, 14 Sep 2020 16:38:21:  16000000 \\nINFO  @ Mon, 14 Sep 2020 16:38:29:  17000000 \\nINFO  @ Mon, 14 Sep 2020 16:38:35:  18000000 \\nINFO  @ Mon, 14 Sep 2020 16:38:39:  19000000 \\nINFO  @ Mon, 14 Sep 2020 16:38:40: 19080460 fragments have been read. \\nINFO  @ Mon, 14 Sep 2020 16:38:51: #1.2 read input fragments... \\nINFO  @ Mon, 14 Sep 2020 16:38:55:  1000000 \\nINFO  @ Mon, 14 Sep 2020 16:38:59:  2000000 \\nINFO  @ Mon, 14 Sep 2020 16:39:03:  3000000 \\nINFO  @ Mon, 14 Sep 2020 16:39:06:  4000000 \\nINFO  @ Mon, 14 Sep 2020 16:39:10:  5000000 \\nINFO  @ Mon, 14 Sep 2020 16:39:14:  6000000 \\nINFO  @ Mon, 14 Sep 2020 16:39:17:  7000000 \\nINFO  @ Mon, 14 Sep 2020 16:39:21:  8000000 \\nINFO  @ Mon, 14 Sep 2020 16:39:25:  9000000 \\nINFO  @ Mon, 14 Sep 2020 16:39:29:  10000000 \\nINFO  @ Mon, 14 Sep 2020 16:39:32:  11000000 \\nINFO  @ Mon, 14 Sep 2020 16:39:36:  12000000 \\nINFO  @ Mon, 14 Sep 2020 16:39:40:  13000000 \\nINFO  @ Mon, 14 Sep 2020 16:39:44:  14000000 \\nINFO  @ Mon, 14 Sep 2020 16:39:48:  15000000 \\nINFO  @ Mon, 14 Sep 2020 16:39:51:  16000000 \\nINFO  @ Mon, 14 Sep 2020 16:39:55:  17000000 \\nINFO  @ Mon, 14 Sep 2020 16:39:59:  18000000 \\nINFO  @ Mon, 14 Sep 2020 16:40:03:  19000000 \\nINFO  @ Mon, 14 Sep 2020 16:40:07:  20000000 \\nINFO  @ Mon, 14 Sep 2020 16:40:10:  21000000 \\nINFO  @ Mon, 14 Sep 2020 16:40:14:  22000000 \\nINFO  @ Mon, 14 Sep 2020 16:40:18:  23000000 \\nINFO  @ Mon, 14 Sep 2020 16:40:22:  24000000 \\nINFO  @ Mon, 14 Sep 2020 16:40:26:  25000000 \\nINFO  @ Mon, 14 Sep 2020 16:40:29:  26000000 \\nINFO  @ Mon, 14 Sep 2020 16:40:33:  27000000 \\nINFO  @ Mon, 14 Sep 2020 16:40:37:  28000000 \\nINFO  @ Mon, 14 Sep 2020 16:40:41:  29000000 \\nINFO  @ Mon, 14 Sep 2020 16:40:44:  30000000 \\nINFO  @ Mon, 14 Sep 2020 16:40:48:  31000000 \\nINFO  @ Mon, 14 Sep 2020 16:40:52:  32000000 \\nINFO  @ Mon, 14 Sep 2020 16:40:56:  33000000 \\nINFO  @ Mon, 14 Sep 2020 16:41:00:  34000000 \\nINFO  @ Mon, 14 Sep 2020 16:41:03:  35000000 \\nINFO  @ Mon, 14 Sep 2020 16:41:07:  36000000 \\nINFO  @ Mon, 14 Sep 2020 16:41:11:  37000000 \\nINFO  @ Mon, 14 Sep 2020 16:41:14:  38000000 \\nINFO  @ Mon, 14 Sep 2020 16:41:18:  39000000 \\nINFO  @ Mon, 14 Sep 2020 16:41:22:  40000000 \\nINFO  @ Mon, 14 Sep 2020 16:41:26:  41000000 \\nINFO  @ Mon, 14 Sep 2020 16:41:30:  42000000 \\nINFO  @ Mon, 14 Sep 2020 16:41:34:  43000000 \\nINFO  @ Mon, 14 Sep 2020 16:41:37:  44000000 \\nINFO  @ Mon, 14 Sep 2020 16:41:41:  45000000 \\nINFO  @ Mon, 14 Sep 2020 16:41:45:  46000000 \\nINFO  @ Mon, 14 Sep 2020 16:41:49:  47000000 \\nINFO  @ Mon, 14 Sep 2020 16:41:53:  48000000 \\nINFO  @ Mon, 14 Sep 2020 16:41:56:  49000000 \\nINFO  @ Mon, 14 Sep 2020 16:42:00:  50000000 \\nINFO  @ Mon, 14 Sep 2020 16:42:04:  51000000 \\nINFO  @ Mon, 14 Sep 2020 16:42:08:  52000000 \\nINFO  @ Mon, 14 Sep 2020 16:42:12:  53000000 \\nINFO  @ Mon, 14 Sep 2020 16:42:15:  54000000 \\nINFO  @ Mon, 14 Sep 2020 16:42:19:  55000000 \\nINFO  @ Mon, 14 Sep 2020 16:42:23:  56000000 \\nINFO  @ Mon, 14 Sep 2020 16:42:27:  57000000 \\nINFO  @ Mon, 14 Sep 2020 16:42:31:  58000000 \\nINFO  @ Mon, 14 Sep 2020 16:42:35:  59000000 \\nINFO  @ Mon, 14 Sep 2020 16:42:38:  60000000 \\nINFO  @ Mon, 14 Sep 2020 16:42:42:  61000000 \\nINFO  @ Mon, 14 Sep 2020 16:42:46:  62000000 \\nINFO  @ Mon, 14 Sep 2020 16:42:50:  63000000 \\nINFO  @ Mon, 14 Sep 2020 16:42:54:  64000000 \\nINFO  @ Mon, 14 Sep 2020 16:42:57:  65000000 \\nINFO  @ Mon, 14 Sep 2020 16:43:01:  66000000 \\nINFO  @ Mon, 14 Sep 2020 16:43:04: 66660428 fragments have been read. \\nINFO  @ Mon, 14 Sep 2020 16:43:48: #1 mean fragment size is determined as 317.8 bp from treatment \\nINFO  @ Mon, 14 Sep 2020 16:43:48: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Mon, 14 Sep 2020 16:43:48: #1 fragment size = 317.8 \\nINFO  @ Mon, 14 Sep 2020 16:43:48: #1  total fragments in treatment: 19080460 \\nINFO  @ Mon, 14 Sep 2020 16:43:48: #1 user defined the maximum fragments... \\nINFO  @ Mon, 14 Sep 2020 16:43:48: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 14 Sep 2020 16:44:18: #1  fragments after filtering in treatment: 19070372 \\nINFO  @ Mon, 14 Sep 2020 16:44:18: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 14 Sep 2020 16:44:18: #1  total fragments in control: 66660428 \\nINFO  @ Mon, 14 Sep 2020 16:44:18: #1 user defined the maximum fragments... \\nINFO  @ Mon, 14 Sep 2020 16:44:18: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 14 Sep 2020 16:46:03: #1  fragments after filtering in control: 66632618 \\nINFO  @ Mon, 14 Sep 2020 16:46:03: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 14 Sep 2020 16:46:03: #1 finished! \\nINFO  @ Mon, 14 Sep 2020 16:46:03: #2 Build Peak Model... \\nINFO  @ Mon, 14 Sep 2020 16:46:03: #2 Skipped... \\nINFO  @ Mon, 14 Sep 2020 16:46:03: #3 Call peaks... \\nINFO  @ Mon, 14 Sep 2020 16:46:03: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 14 Sep 2020 16:55:17: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 14 Sep 2020 16:55:17: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_RNP_RUNX1_RUNX2-H3K27AC_R2_treat_pileup.bdg \\nINFO  @ Mon, 14 Sep 2020 16:55:17: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_RNP_RUNX1_RUNX2-H3K27AC_R2_control_lambda.bdg \\nINFO  @ Mon, 14 Sep 2020 16:55:17: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 14 Sep 2020 16:55:17: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 14 Sep 2020 17:01:24: #4 Write output xls file... ../../data/additional_degraded_v1/diffData/MV411_RNP_RUNX1_RUNX2-H3K27AC_R2_peaks.xls \\nINFO  @ Mon, 14 Sep 2020 17:01:24: #4 Write peak in narrowPeak format file... ../../data/additional_degraded_v1/diffData/MV411_RNP_RUNX1_RUNX2-H3K27AC_R2_peaks.narrowPeak \\nINFO  @ Mon, 14 Sep 2020 17:01:25: #4 Write summits bed file... ../../data/additional_degraded_v1/diffData/MV411_RNP_RUNX1_RUNX2-H3K27AC_R2_summits.bed \\nINFO  @ Mon, 14 Sep 2020 17:01:25: Done! \\n'\n",
      "18993253 19070372\n",
      "doing differential peak binding\n",
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup.bdg --c1 ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R2_control_lambda.bdg --t2 ../../data/additional_degraded_v1/diffData/MV411_RNP_RUNX1_RUNX2-H3K27AC_R2_treat_pileup.bdg --c2 ../../data/additional_degraded_v1/diffData/MV411_RNP_RUNX1_RUNX2-H3K27AC_R2_control_lambda.bdg --d1 18993253 --d2 19070372 -g 60 -l 240 --o-prefix MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_RUNX1_RUNX2-H3K27AC_R2_treat_pileup --outdir ../../data/additional_degraded_v1/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Mon, 14 Sep 2020 17:01:25: Read and build treatment 1 bedGraph... \\nINFO  @ Mon, 14 Sep 2020 17:02:05: Read and build control 1 bedGraph... \\nINFO  @ Mon, 14 Sep 2020 17:05:25: Read and build treatment 2 bedGraph... \\nINFO  @ Mon, 14 Sep 2020 17:05:54: Read and build control 2 bedGraph... \\nINFO  @ Mon, 14 Sep 2020 17:18:44: Write peaks... \\nINFO  @ Mon, 14 Sep 2020 17:18:44: Done \\n')\n",
      "../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.sorted.bam ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX2-H3K27AC_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.sorted.bam and ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX2-H3K27AC_R1.mLb.clN.sorted.bam\n",
      "using default|given size\n",
      "computing the scaling values\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'INFO  @ Mon, 14 Sep 2020 17:18:45: \\n# Command line: callpeak -B -t ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.sorted.bam -c ../../data/additional_degraded_v1/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 240 -n MV411_RNP_AAVS1-H3K27AC_R1 --outdir ../../data/additional_degraded_v1/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_RNP_AAVS1-H3K27AC_R1\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/additional_degraded_v1/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Mon, 14 Sep 2020 17:18:45: #1 read fragment files... \\nINFO  @ Mon, 14 Sep 2020 17:18:45: #1 read treatment fragments... \\nINFO  @ Mon, 14 Sep 2020 17:18:50:  1000000 \\nINFO  @ Mon, 14 Sep 2020 17:18:55:  2000000 \\nINFO  @ Mon, 14 Sep 2020 17:19:00:  3000000 \\nINFO  @ Mon, 14 Sep 2020 17:19:04:  4000000 \\nINFO  @ Mon, 14 Sep 2020 17:19:09:  5000000 \\nINFO  @ Mon, 14 Sep 2020 17:19:14:  6000000 \\nINFO  @ Mon, 14 Sep 2020 17:19:19:  7000000 \\nINFO  @ Mon, 14 Sep 2020 17:19:23:  8000000 \\nINFO  @ Mon, 14 Sep 2020 17:19:28:  9000000 \\nINFO  @ Mon, 14 Sep 2020 17:19:33:  10000000 \\nINFO  @ Mon, 14 Sep 2020 17:19:37:  11000000 \\nINFO  @ Mon, 14 Sep 2020 17:19:42:  12000000 \\nINFO  @ Mon, 14 Sep 2020 17:19:47:  13000000 \\nINFO  @ Mon, 14 Sep 2020 17:19:51:  14000000 \\nINFO  @ Mon, 14 Sep 2020 17:19:56:  15000000 \\nINFO  @ Mon, 14 Sep 2020 17:20:00:  16000000 \\nINFO  @ Mon, 14 Sep 2020 17:20:05:  17000000 \\nINFO  @ Mon, 14 Sep 2020 17:20:09:  18000000 \\nINFO  @ Mon, 14 Sep 2020 17:20:14:  19000000 \\nINFO  @ Mon, 14 Sep 2020 17:20:19:  20000000 \\nINFO  @ Mon, 14 Sep 2020 17:20:22: 20629699 fragments have been read. \\nINFO  @ Mon, 14 Sep 2020 17:20:34: #1.2 read input fragments... \\nINFO  @ Mon, 14 Sep 2020 17:20:38:  1000000 \\nINFO  @ Mon, 14 Sep 2020 17:20:42:  2000000 \\nINFO  @ Mon, 14 Sep 2020 17:20:46:  3000000 \\nINFO  @ Mon, 14 Sep 2020 17:20:50:  4000000 \\nINFO  @ Mon, 14 Sep 2020 17:20:53:  5000000 \\nINFO  @ Mon, 14 Sep 2020 17:20:57:  6000000 \\nINFO  @ Mon, 14 Sep 2020 17:21:01:  7000000 \\nINFO  @ Mon, 14 Sep 2020 17:21:05:  8000000 \\nINFO  @ Mon, 14 Sep 2020 17:21:09:  9000000 \\nINFO  @ Mon, 14 Sep 2020 17:21:13:  10000000 \\nINFO  @ Mon, 14 Sep 2020 17:21:16:  11000000 \\nINFO  @ Mon, 14 Sep 2020 17:21:20:  12000000 \\nINFO  @ Mon, 14 Sep 2020 17:21:24:  13000000 \\nINFO  @ Mon, 14 Sep 2020 17:21:27:  14000000 \\nINFO  @ Mon, 14 Sep 2020 17:21:31:  15000000 \\nINFO  @ Mon, 14 Sep 2020 17:21:35:  16000000 \\nINFO  @ Mon, 14 Sep 2020 17:21:39:  17000000 \\nINFO  @ Mon, 14 Sep 2020 17:21:42:  18000000 \\nINFO  @ Mon, 14 Sep 2020 17:21:46:  19000000 \\nINFO  @ Mon, 14 Sep 2020 17:21:50:  20000000 \\nINFO  @ Mon, 14 Sep 2020 17:21:53:  21000000 \\nINFO  @ Mon, 14 Sep 2020 17:21:57:  22000000 \\nINFO  @ Mon, 14 Sep 2020 17:22:01:  23000000 \\nINFO  @ Mon, 14 Sep 2020 17:22:04:  24000000 \\nINFO  @ Mon, 14 Sep 2020 17:22:08:  25000000 \\nINFO  @ Mon, 14 Sep 2020 17:22:12:  26000000 \\nINFO  @ Mon, 14 Sep 2020 17:22:15:  27000000 \\nINFO  @ Mon, 14 Sep 2020 17:22:19:  28000000 \\nINFO  @ Mon, 14 Sep 2020 17:22:23:  29000000 \\nINFO  @ Mon, 14 Sep 2020 17:22:26:  30000000 \\nINFO  @ Mon, 14 Sep 2020 17:22:30:  31000000 \\nINFO  @ Mon, 14 Sep 2020 17:22:34:  32000000 \\nINFO  @ Mon, 14 Sep 2020 17:22:37:  33000000 \\nINFO  @ Mon, 14 Sep 2020 17:22:41:  34000000 \\nINFO  @ Mon, 14 Sep 2020 17:22:45:  35000000 \\nINFO  @ Mon, 14 Sep 2020 17:22:48:  36000000 \\nINFO  @ Mon, 14 Sep 2020 17:22:53:  37000000 \\nINFO  @ Mon, 14 Sep 2020 17:22:58:  38000000 \\nINFO  @ Mon, 14 Sep 2020 17:23:01:  39000000 \\nINFO  @ Mon, 14 Sep 2020 17:23:05:  40000000 \\nINFO  @ Mon, 14 Sep 2020 17:23:09:  41000000 \\nINFO  @ Mon, 14 Sep 2020 17:23:13:  42000000 \\nINFO  @ Mon, 14 Sep 2020 17:23:16:  43000000 \\nINFO  @ Mon, 14 Sep 2020 17:23:20:  44000000 \\nINFO  @ Mon, 14 Sep 2020 17:23:24:  45000000 \\nINFO  @ Mon, 14 Sep 2020 17:23:27:  46000000 \\nINFO  @ Mon, 14 Sep 2020 17:23:31:  47000000 \\nINFO  @ Mon, 14 Sep 2020 17:23:35:  48000000 \\nINFO  @ Mon, 14 Sep 2020 17:23:38:  49000000 \\nINFO  @ Mon, 14 Sep 2020 17:23:42:  50000000 \\nINFO  @ Mon, 14 Sep 2020 17:23:46:  51000000 \\nINFO  @ Mon, 14 Sep 2020 17:23:50:  52000000 \\nINFO  @ Mon, 14 Sep 2020 17:23:54:  53000000 \\nINFO  @ Mon, 14 Sep 2020 17:23:57:  54000000 \\nINFO  @ Mon, 14 Sep 2020 17:24:01:  55000000 \\nINFO  @ Mon, 14 Sep 2020 17:24:05:  56000000 \\nINFO  @ Mon, 14 Sep 2020 17:24:09:  57000000 \\nINFO  @ Mon, 14 Sep 2020 17:24:12:  58000000 \\nINFO  @ Mon, 14 Sep 2020 17:24:16:  59000000 \\nINFO  @ Mon, 14 Sep 2020 17:24:20:  60000000 \\nINFO  @ Mon, 14 Sep 2020 17:24:23:  61000000 \\nINFO  @ Mon, 14 Sep 2020 17:24:27:  62000000 \\nINFO  @ Mon, 14 Sep 2020 17:24:31:  63000000 \\nINFO  @ Mon, 14 Sep 2020 17:24:35:  64000000 \\nINFO  @ Mon, 14 Sep 2020 17:24:38:  65000000 \\nINFO  @ Mon, 14 Sep 2020 17:24:42:  66000000 \\nINFO  @ Mon, 14 Sep 2020 17:24:45: 66660428 fragments have been read. \\nINFO  @ Mon, 14 Sep 2020 17:25:30: #1 mean fragment size is determined as 309.7 bp from treatment \\nINFO  @ Mon, 14 Sep 2020 17:25:30: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Mon, 14 Sep 2020 17:25:30: #1 fragment size = 309.7 \\nINFO  @ Mon, 14 Sep 2020 17:25:30: #1  total fragments in treatment: 20629699 \\nINFO  @ Mon, 14 Sep 2020 17:25:30: #1 user defined the maximum fragments... \\nINFO  @ Mon, 14 Sep 2020 17:25:30: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 14 Sep 2020 17:26:04: #1  fragments after filtering in treatment: 20624221 \\nINFO  @ Mon, 14 Sep 2020 17:26:04: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 14 Sep 2020 17:26:04: #1  total fragments in control: 66660428 \\nINFO  @ Mon, 14 Sep 2020 17:26:04: #1 user defined the maximum fragments... \\nINFO  @ Mon, 14 Sep 2020 17:26:04: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 14 Sep 2020 17:27:56: #1  fragments after filtering in control: 66632618 \\nINFO  @ Mon, 14 Sep 2020 17:27:56: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 14 Sep 2020 17:27:56: #1 finished! \\nINFO  @ Mon, 14 Sep 2020 17:27:56: #2 Build Peak Model... \\nINFO  @ Mon, 14 Sep 2020 17:27:56: #2 Skipped... \\nINFO  @ Mon, 14 Sep 2020 17:27:56: #3 Call peaks... \\nINFO  @ Mon, 14 Sep 2020 17:27:56: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 14 Sep 2020 17:37:10: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 14 Sep 2020 17:37:10: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup.bdg \\nINFO  @ Mon, 14 Sep 2020 17:37:10: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_RNP_AAVS1-H3K27AC_R1_control_lambda.bdg \\nINFO  @ Mon, 14 Sep 2020 17:37:10: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 14 Sep 2020 17:37:10: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 14 Sep 2020 17:43:02: #4 Write output xls file... ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R1_peaks.xls \\nINFO  @ Mon, 14 Sep 2020 17:43:02: #4 Write peak in narrowPeak format file... ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R1_peaks.narrowPeak \\nINFO  @ Mon, 14 Sep 2020 17:43:02: #4 Write summits bed file... ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R1_summits.bed \\nINFO  @ Mon, 14 Sep 2020 17:43:02: Done! \\n'\n",
      "b'INFO  @ Mon, 14 Sep 2020 17:43:02: \\n# Command line: callpeak -B -t ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX2-H3K27AC_R1.mLb.clN.sorted.bam -c ../../data/additional_degraded_v1/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 240 -n MV411_RNP_RUNX2-H3K27AC_R1 --outdir ../../data/additional_degraded_v1/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_RNP_RUNX2-H3K27AC_R1\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX2-H3K27AC_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/additional_degraded_v1/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Mon, 14 Sep 2020 17:43:02: #1 read fragment files... \\nINFO  @ Mon, 14 Sep 2020 17:43:02: #1 read treatment fragments... \\nINFO  @ Mon, 14 Sep 2020 17:43:07:  1000000 \\nINFO  @ Mon, 14 Sep 2020 17:43:15:  2000000 \\nINFO  @ Mon, 14 Sep 2020 17:43:20:  3000000 \\nINFO  @ Mon, 14 Sep 2020 17:43:25:  4000000 \\nINFO  @ Mon, 14 Sep 2020 17:43:29:  5000000 \\nINFO  @ Mon, 14 Sep 2020 17:43:34:  6000000 \\nINFO  @ Mon, 14 Sep 2020 17:43:38:  7000000 \\nINFO  @ Mon, 14 Sep 2020 17:43:43:  8000000 \\nINFO  @ Mon, 14 Sep 2020 17:43:47:  9000000 \\nINFO  @ Mon, 14 Sep 2020 17:43:52:  10000000 \\nINFO  @ Mon, 14 Sep 2020 17:43:57:  11000000 \\nINFO  @ Mon, 14 Sep 2020 17:44:01:  12000000 \\nINFO  @ Mon, 14 Sep 2020 17:44:06:  13000000 \\nINFO  @ Mon, 14 Sep 2020 17:44:10:  14000000 \\nINFO  @ Mon, 14 Sep 2020 17:44:15:  15000000 \\nINFO  @ Mon, 14 Sep 2020 17:44:19:  16000000 \\nINFO  @ Mon, 14 Sep 2020 17:44:24:  17000000 \\nINFO  @ Mon, 14 Sep 2020 17:44:28:  18000000 \\nINFO  @ Mon, 14 Sep 2020 17:44:33:  19000000 \\nINFO  @ Mon, 14 Sep 2020 17:44:37:  20000000 \\nINFO  @ Mon, 14 Sep 2020 17:44:42:  21000000 \\nINFO  @ Mon, 14 Sep 2020 17:44:45: 21637131 fragments have been read. \\nINFO  @ Mon, 14 Sep 2020 17:44:58: #1.2 read input fragments... \\nINFO  @ Mon, 14 Sep 2020 17:45:02:  1000000 \\nINFO  @ Mon, 14 Sep 2020 17:45:06:  2000000 \\nINFO  @ Mon, 14 Sep 2020 17:45:09:  3000000 \\nINFO  @ Mon, 14 Sep 2020 17:45:13:  4000000 \\nINFO  @ Mon, 14 Sep 2020 17:45:17:  5000000 \\nINFO  @ Mon, 14 Sep 2020 17:45:21:  6000000 \\nINFO  @ Mon, 14 Sep 2020 17:45:24:  7000000 \\nINFO  @ Mon, 14 Sep 2020 17:45:28:  8000000 \\nINFO  @ Mon, 14 Sep 2020 17:45:32:  9000000 \\nINFO  @ Mon, 14 Sep 2020 17:45:36:  10000000 \\nINFO  @ Mon, 14 Sep 2020 17:45:39:  11000000 \\nINFO  @ Mon, 14 Sep 2020 17:45:43:  12000000 \\nINFO  @ Mon, 14 Sep 2020 17:45:47:  13000000 \\nINFO  @ Mon, 14 Sep 2020 17:45:51:  14000000 \\nINFO  @ Mon, 14 Sep 2020 17:45:55:  15000000 \\nINFO  @ Mon, 14 Sep 2020 17:45:59:  16000000 \\nINFO  @ Mon, 14 Sep 2020 17:46:03:  17000000 \\nINFO  @ Mon, 14 Sep 2020 17:46:06:  18000000 \\nINFO  @ Mon, 14 Sep 2020 17:46:10:  19000000 \\nINFO  @ Mon, 14 Sep 2020 17:46:14:  20000000 \\nINFO  @ Mon, 14 Sep 2020 17:46:18:  21000000 \\nINFO  @ Mon, 14 Sep 2020 17:46:22:  22000000 \\nINFO  @ Mon, 14 Sep 2020 17:46:26:  23000000 \\nINFO  @ Mon, 14 Sep 2020 17:46:29:  24000000 \\nINFO  @ Mon, 14 Sep 2020 17:46:33:  25000000 \\nINFO  @ Mon, 14 Sep 2020 17:46:37:  26000000 \\nINFO  @ Mon, 14 Sep 2020 17:46:41:  27000000 \\nINFO  @ Mon, 14 Sep 2020 17:46:44:  28000000 \\nINFO  @ Mon, 14 Sep 2020 17:46:48:  29000000 \\nINFO  @ Mon, 14 Sep 2020 17:46:52:  30000000 \\nINFO  @ Mon, 14 Sep 2020 17:46:56:  31000000 \\nINFO  @ Mon, 14 Sep 2020 17:47:00:  32000000 \\nINFO  @ Mon, 14 Sep 2020 17:47:04:  33000000 \\nINFO  @ Mon, 14 Sep 2020 17:47:08:  34000000 \\nINFO  @ Mon, 14 Sep 2020 17:47:11:  35000000 \\nINFO  @ Mon, 14 Sep 2020 17:47:15:  36000000 \\nINFO  @ Mon, 14 Sep 2020 17:47:19:  37000000 \\nINFO  @ Mon, 14 Sep 2020 17:47:22:  38000000 \\nINFO  @ Mon, 14 Sep 2020 17:47:26:  39000000 \\nINFO  @ Mon, 14 Sep 2020 17:47:30:  40000000 \\nINFO  @ Mon, 14 Sep 2020 17:47:34:  41000000 \\nINFO  @ Mon, 14 Sep 2020 17:47:38:  42000000 \\nINFO  @ Mon, 14 Sep 2020 17:47:42:  43000000 \\nINFO  @ Mon, 14 Sep 2020 17:47:46:  44000000 \\nINFO  @ Mon, 14 Sep 2020 17:47:49:  45000000 \\nINFO  @ Mon, 14 Sep 2020 17:47:53:  46000000 \\nINFO  @ Mon, 14 Sep 2020 17:47:57:  47000000 \\nINFO  @ Mon, 14 Sep 2020 17:48:01:  48000000 \\nINFO  @ Mon, 14 Sep 2020 17:48:05:  49000000 \\nINFO  @ Mon, 14 Sep 2020 17:48:09:  50000000 \\nINFO  @ Mon, 14 Sep 2020 17:48:13:  51000000 \\nINFO  @ Mon, 14 Sep 2020 17:48:16:  52000000 \\nINFO  @ Mon, 14 Sep 2020 17:48:20:  53000000 \\nINFO  @ Mon, 14 Sep 2020 17:48:24:  54000000 \\nINFO  @ Mon, 14 Sep 2020 17:48:28:  55000000 \\nINFO  @ Mon, 14 Sep 2020 17:48:32:  56000000 \\nINFO  @ Mon, 14 Sep 2020 17:48:36:  57000000 \\nINFO  @ Mon, 14 Sep 2020 17:48:39:  58000000 \\nINFO  @ Mon, 14 Sep 2020 17:48:43:  59000000 \\nINFO  @ Mon, 14 Sep 2020 17:48:47:  60000000 \\nINFO  @ Mon, 14 Sep 2020 17:48:51:  61000000 \\nINFO  @ Mon, 14 Sep 2020 17:48:55:  62000000 \\nINFO  @ Mon, 14 Sep 2020 17:48:59:  63000000 \\nINFO  @ Mon, 14 Sep 2020 17:49:02:  64000000 \\nINFO  @ Mon, 14 Sep 2020 17:49:06:  65000000 \\nINFO  @ Mon, 14 Sep 2020 17:49:10:  66000000 \\nINFO  @ Mon, 14 Sep 2020 17:49:13: 66660428 fragments have been read. \\nINFO  @ Mon, 14 Sep 2020 17:49:57: #1 mean fragment size is determined as 309.1 bp from treatment \\nINFO  @ Mon, 14 Sep 2020 17:49:57: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Mon, 14 Sep 2020 17:49:57: #1 fragment size = 309.1 \\nINFO  @ Mon, 14 Sep 2020 17:49:57: #1  total fragments in treatment: 21637131 \\nINFO  @ Mon, 14 Sep 2020 17:49:57: #1 user defined the maximum fragments... \\nINFO  @ Mon, 14 Sep 2020 17:49:57: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 14 Sep 2020 17:50:32: #1  fragments after filtering in treatment: 21632631 \\nINFO  @ Mon, 14 Sep 2020 17:50:32: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 14 Sep 2020 17:50:32: #1  total fragments in control: 66660428 \\nINFO  @ Mon, 14 Sep 2020 17:50:32: #1 user defined the maximum fragments... \\nINFO  @ Mon, 14 Sep 2020 17:50:32: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 14 Sep 2020 17:52:19: #1  fragments after filtering in control: 66632618 \\nINFO  @ Mon, 14 Sep 2020 17:52:19: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 14 Sep 2020 17:52:19: #1 finished! \\nINFO  @ Mon, 14 Sep 2020 17:52:19: #2 Build Peak Model... \\nINFO  @ Mon, 14 Sep 2020 17:52:19: #2 Skipped... \\nINFO  @ Mon, 14 Sep 2020 17:52:19: #3 Call peaks... \\nINFO  @ Mon, 14 Sep 2020 17:52:19: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 14 Sep 2020 18:01:32: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 14 Sep 2020 18:01:32: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_RNP_RUNX2-H3K27AC_R1_treat_pileup.bdg \\nINFO  @ Mon, 14 Sep 2020 18:01:32: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_RNP_RUNX2-H3K27AC_R1_control_lambda.bdg \\nINFO  @ Mon, 14 Sep 2020 18:01:32: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 14 Sep 2020 18:01:32: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 14 Sep 2020 18:07:35: #4 Write output xls file... ../../data/additional_degraded_v1/diffData/MV411_RNP_RUNX2-H3K27AC_R1_peaks.xls \\nINFO  @ Mon, 14 Sep 2020 18:07:35: #4 Write peak in narrowPeak format file... ../../data/additional_degraded_v1/diffData/MV411_RNP_RUNX2-H3K27AC_R1_peaks.narrowPeak \\nINFO  @ Mon, 14 Sep 2020 18:07:36: #4 Write summits bed file... ../../data/additional_degraded_v1/diffData/MV411_RNP_RUNX2-H3K27AC_R1_summits.bed \\nINFO  @ Mon, 14 Sep 2020 18:07:36: Done! \\n'\n",
      "31040563 21632631\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup.bdg --c1 ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R1_control_lambda.bdg --t2 ../../data/additional_degraded_v1/diffData/MV411_RNP_RUNX2-H3K27AC_R1_treat_pileup.bdg --c2 ../../data/additional_degraded_v1/diffData/MV411_RNP_RUNX2-H3K27AC_R1_control_lambda.bdg --d1 31040563 --d2 21632631 -g 60 -l 240 --o-prefix MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_RUNX2-H3K27AC_R1_treat_pileup --outdir ../../data/additional_degraded_v1/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Mon, 14 Sep 2020 18:07:36: Read and build treatment 1 bedGraph... \\nINFO  @ Mon, 14 Sep 2020 18:08:07: Read and build control 1 bedGraph... \\nINFO  @ Mon, 14 Sep 2020 18:11:23: Read and build treatment 2 bedGraph... \\nINFO  @ Mon, 14 Sep 2020 18:11:54: Read and build control 2 bedGraph... \\nINFO  @ Mon, 14 Sep 2020 18:22:21: Write peaks... \\nINFO  @ Mon, 14 Sep 2020 18:22:21: Done \\n')\n",
      "../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.sorted.bam ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX2-H3K27AC_R2.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.sorted.bam and ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX2-H3K27AC_R2.mLb.clN.sorted.bam\n",
      "using default|given size\n",
      "computing the scaling values\n",
      "b'INFO  @ Mon, 14 Sep 2020 18:22:22: \\n# Command line: callpeak -B -t ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.sorted.bam -c ../../data/additional_degraded_v1/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 240 -n MV411_RNP_AAVS1-H3K27AC_R2 --outdir ../../data/additional_degraded_v1/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_RNP_AAVS1-H3K27AC_R2\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/additional_degraded_v1/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Mon, 14 Sep 2020 18:22:22: #1 read fragment files... \\nINFO  @ Mon, 14 Sep 2020 18:22:22: #1 read treatment fragments... \\nINFO  @ Mon, 14 Sep 2020 18:22:26:  1000000 \\nINFO  @ Mon, 14 Sep 2020 18:22:31:  2000000 \\nINFO  @ Mon, 14 Sep 2020 18:22:36:  3000000 \\nINFO  @ Mon, 14 Sep 2020 18:22:41:  4000000 \\nINFO  @ Mon, 14 Sep 2020 18:22:45:  5000000 \\nINFO  @ Mon, 14 Sep 2020 18:22:50:  6000000 \\nINFO  @ Mon, 14 Sep 2020 18:22:55:  7000000 \\nINFO  @ Mon, 14 Sep 2020 18:22:59:  8000000 \\nINFO  @ Mon, 14 Sep 2020 18:23:04:  9000000 \\nINFO  @ Mon, 14 Sep 2020 18:23:09:  10000000 \\nINFO  @ Mon, 14 Sep 2020 18:23:13:  11000000 \\nINFO  @ Mon, 14 Sep 2020 18:23:18:  12000000 \\nINFO  @ Mon, 14 Sep 2020 18:23:23:  13000000 \\nINFO  @ Mon, 14 Sep 2020 18:23:27:  14000000 \\nINFO  @ Mon, 14 Sep 2020 18:23:32:  15000000 \\nINFO  @ Mon, 14 Sep 2020 18:23:37:  16000000 \\nINFO  @ Mon, 14 Sep 2020 18:23:43:  17000000 \\nINFO  @ Mon, 14 Sep 2020 18:23:47:  18000000 \\nINFO  @ Mon, 14 Sep 2020 18:23:52:  19000000 \\nINFO  @ Mon, 14 Sep 2020 18:23:57:  20000000 \\nINFO  @ Mon, 14 Sep 2020 18:24:01:  21000000 \\nINFO  @ Mon, 14 Sep 2020 18:24:06:  22000000 \\nINFO  @ Mon, 14 Sep 2020 18:24:11:  23000000 \\nINFO  @ Mon, 14 Sep 2020 18:24:15:  24000000 \\nINFO  @ Mon, 14 Sep 2020 18:24:20:  25000000 \\nINFO  @ Mon, 14 Sep 2020 18:24:25:  26000000 \\nINFO  @ Mon, 14 Sep 2020 18:24:29:  27000000 \\nINFO  @ Mon, 14 Sep 2020 18:24:32: 27510868 fragments have been read. \\nINFO  @ Mon, 14 Sep 2020 18:24:49: #1.2 read input fragments... \\nINFO  @ Mon, 14 Sep 2020 18:24:52:  1000000 \\nINFO  @ Mon, 14 Sep 2020 18:24:56:  2000000 \\nINFO  @ Mon, 14 Sep 2020 18:25:00:  3000000 \\nINFO  @ Mon, 14 Sep 2020 18:25:04:  4000000 \\nINFO  @ Mon, 14 Sep 2020 18:25:07:  5000000 \\nINFO  @ Mon, 14 Sep 2020 18:25:11:  6000000 \\nINFO  @ Mon, 14 Sep 2020 18:25:15:  7000000 \\nINFO  @ Mon, 14 Sep 2020 18:25:18:  8000000 \\nINFO  @ Mon, 14 Sep 2020 18:25:22:  9000000 \\nINFO  @ Mon, 14 Sep 2020 18:25:26:  10000000 \\nINFO  @ Mon, 14 Sep 2020 18:25:29:  11000000 \\nINFO  @ Mon, 14 Sep 2020 18:25:33:  12000000 \\nINFO  @ Mon, 14 Sep 2020 18:25:37:  13000000 \\nINFO  @ Mon, 14 Sep 2020 18:25:40:  14000000 \\nINFO  @ Mon, 14 Sep 2020 18:25:44:  15000000 \\nINFO  @ Mon, 14 Sep 2020 18:25:48:  16000000 \\nINFO  @ Mon, 14 Sep 2020 18:25:51:  17000000 \\nINFO  @ Mon, 14 Sep 2020 18:25:55:  18000000 \\nINFO  @ Mon, 14 Sep 2020 18:25:59:  19000000 \\nINFO  @ Mon, 14 Sep 2020 18:26:02:  20000000 \\nINFO  @ Mon, 14 Sep 2020 18:26:06:  21000000 \\nINFO  @ Mon, 14 Sep 2020 18:26:10:  22000000 \\nINFO  @ Mon, 14 Sep 2020 18:26:13:  23000000 \\nINFO  @ Mon, 14 Sep 2020 18:26:17:  24000000 \\nINFO  @ Mon, 14 Sep 2020 18:26:21:  25000000 \\nINFO  @ Mon, 14 Sep 2020 18:26:24:  26000000 \\nINFO  @ Mon, 14 Sep 2020 18:26:28:  27000000 \\nINFO  @ Mon, 14 Sep 2020 18:26:32:  28000000 \\nINFO  @ Mon, 14 Sep 2020 18:26:35:  29000000 \\nINFO  @ Mon, 14 Sep 2020 18:26:39:  30000000 \\nINFO  @ Mon, 14 Sep 2020 18:26:43:  31000000 \\nINFO  @ Mon, 14 Sep 2020 18:26:47:  32000000 \\nINFO  @ Mon, 14 Sep 2020 18:26:50:  33000000 \\nINFO  @ Mon, 14 Sep 2020 18:26:54:  34000000 \\nINFO  @ Mon, 14 Sep 2020 18:26:58:  35000000 \\nINFO  @ Mon, 14 Sep 2020 18:27:01:  36000000 \\nINFO  @ Mon, 14 Sep 2020 18:27:05:  37000000 \\nINFO  @ Mon, 14 Sep 2020 18:27:09:  38000000 \\nINFO  @ Mon, 14 Sep 2020 18:27:12:  39000000 \\nINFO  @ Mon, 14 Sep 2020 18:27:16:  40000000 \\nINFO  @ Mon, 14 Sep 2020 18:27:20:  41000000 \\nINFO  @ Mon, 14 Sep 2020 18:27:24:  42000000 \\nINFO  @ Mon, 14 Sep 2020 18:27:27:  43000000 \\nINFO  @ Mon, 14 Sep 2020 18:27:31:  44000000 \\nINFO  @ Mon, 14 Sep 2020 18:27:35:  45000000 \\nINFO  @ Mon, 14 Sep 2020 18:27:39:  46000000 \\nINFO  @ Mon, 14 Sep 2020 18:27:43:  47000000 \\nINFO  @ Mon, 14 Sep 2020 18:27:47:  48000000 \\nINFO  @ Mon, 14 Sep 2020 18:27:51:  49000000 \\nINFO  @ Mon, 14 Sep 2020 18:27:55:  50000000 \\nINFO  @ Mon, 14 Sep 2020 18:27:58:  51000000 \\nINFO  @ Mon, 14 Sep 2020 18:28:02:  52000000 \\nINFO  @ Mon, 14 Sep 2020 18:28:06:  53000000 \\nINFO  @ Mon, 14 Sep 2020 18:28:09:  54000000 \\nINFO  @ Mon, 14 Sep 2020 18:28:13:  55000000 \\nINFO  @ Mon, 14 Sep 2020 18:28:17:  56000000 \\nINFO  @ Mon, 14 Sep 2020 18:28:21:  57000000 \\nINFO  @ Mon, 14 Sep 2020 18:28:25:  58000000 \\nINFO  @ Mon, 14 Sep 2020 18:28:28:  59000000 \\nINFO  @ Mon, 14 Sep 2020 18:28:32:  60000000 \\nINFO  @ Mon, 14 Sep 2020 18:28:36:  61000000 \\nINFO  @ Mon, 14 Sep 2020 18:28:40:  62000000 \\nINFO  @ Mon, 14 Sep 2020 18:28:44:  63000000 \\nINFO  @ Mon, 14 Sep 2020 18:28:47:  64000000 \\nINFO  @ Mon, 14 Sep 2020 18:28:51:  65000000 \\nINFO  @ Mon, 14 Sep 2020 18:28:55:  66000000 \\nINFO  @ Mon, 14 Sep 2020 18:28:57: 66660428 fragments have been read. \\nINFO  @ Mon, 14 Sep 2020 18:29:42: #1 mean fragment size is determined as 304.3 bp from treatment \\nINFO  @ Mon, 14 Sep 2020 18:29:42: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Mon, 14 Sep 2020 18:29:42: #1 fragment size = 304.3 \\nINFO  @ Mon, 14 Sep 2020 18:29:42: #1  total fragments in treatment: 27510868 \\nINFO  @ Mon, 14 Sep 2020 18:29:42: #1 user defined the maximum fragments... \\nINFO  @ Mon, 14 Sep 2020 18:29:42: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 14 Sep 2020 18:30:26: #1  fragments after filtering in treatment: 27503766 \\nINFO  @ Mon, 14 Sep 2020 18:30:26: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 14 Sep 2020 18:30:26: #1  total fragments in control: 66660428 \\nINFO  @ Mon, 14 Sep 2020 18:30:26: #1 user defined the maximum fragments... \\nINFO  @ Mon, 14 Sep 2020 18:30:26: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 14 Sep 2020 18:32:13: #1  fragments after filtering in control: 66632618 \\nINFO  @ Mon, 14 Sep 2020 18:32:13: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 14 Sep 2020 18:32:13: #1 finished! \\nINFO  @ Mon, 14 Sep 2020 18:32:13: #2 Build Peak Model... \\nINFO  @ Mon, 14 Sep 2020 18:32:13: #2 Skipped... \\nINFO  @ Mon, 14 Sep 2020 18:32:13: #3 Call peaks... \\nINFO  @ Mon, 14 Sep 2020 18:32:13: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 14 Sep 2020 18:41:34: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 14 Sep 2020 18:41:34: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup.bdg \\nINFO  @ Mon, 14 Sep 2020 18:41:34: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_RNP_AAVS1-H3K27AC_R2_control_lambda.bdg \\nINFO  @ Mon, 14 Sep 2020 18:41:34: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 14 Sep 2020 18:41:34: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 14 Sep 2020 18:47:36: #4 Write output xls file... ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R2_peaks.xls \\nINFO  @ Mon, 14 Sep 2020 18:47:36: #4 Write peak in narrowPeak format file... ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R2_peaks.narrowPeak \\nINFO  @ Mon, 14 Sep 2020 18:47:36: #4 Write summits bed file... ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R2_summits.bed \\nINFO  @ Mon, 14 Sep 2020 18:47:36: Done! \\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'INFO  @ Mon, 14 Sep 2020 18:47:37: \\n# Command line: callpeak -B -t ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX2-H3K27AC_R2.mLb.clN.sorted.bam -c ../../data/additional_degraded_v1/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 240 -n MV411_RNP_RUNX2-H3K27AC_R2 --outdir ../../data/additional_degraded_v1/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_RNP_RUNX2-H3K27AC_R2\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_RUNX2-H3K27AC_R2.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/additional_degraded_v1/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Mon, 14 Sep 2020 18:47:37: #1 read fragment files... \\nINFO  @ Mon, 14 Sep 2020 18:47:37: #1 read treatment fragments... \\nINFO  @ Mon, 14 Sep 2020 18:47:41:  1000000 \\nINFO  @ Mon, 14 Sep 2020 18:47:46:  2000000 \\nINFO  @ Mon, 14 Sep 2020 18:47:51:  3000000 \\nINFO  @ Mon, 14 Sep 2020 18:47:55:  4000000 \\nINFO  @ Mon, 14 Sep 2020 18:48:01:  5000000 \\nINFO  @ Mon, 14 Sep 2020 18:48:06:  6000000 \\nINFO  @ Mon, 14 Sep 2020 18:48:10:  7000000 \\nINFO  @ Mon, 14 Sep 2020 18:48:15:  8000000 \\nINFO  @ Mon, 14 Sep 2020 18:48:19:  9000000 \\nINFO  @ Mon, 14 Sep 2020 18:48:24:  10000000 \\nINFO  @ Mon, 14 Sep 2020 18:48:28:  11000000 \\nINFO  @ Mon, 14 Sep 2020 18:48:33:  12000000 \\nINFO  @ Mon, 14 Sep 2020 18:48:37:  13000000 \\nINFO  @ Mon, 14 Sep 2020 18:48:42:  14000000 \\nINFO  @ Mon, 14 Sep 2020 18:48:46:  15000000 \\nINFO  @ Mon, 14 Sep 2020 18:48:51:  16000000 \\nINFO  @ Mon, 14 Sep 2020 18:48:55:  17000000 \\nINFO  @ Mon, 14 Sep 2020 18:49:00:  18000000 \\nINFO  @ Mon, 14 Sep 2020 18:49:04:  19000000 \\nINFO  @ Mon, 14 Sep 2020 18:49:09:  20000000 \\nINFO  @ Mon, 14 Sep 2020 18:49:13:  21000000 \\nINFO  @ Mon, 14 Sep 2020 18:49:18:  22000000 \\nINFO  @ Mon, 14 Sep 2020 18:49:22:  23000000 \\nINFO  @ Mon, 14 Sep 2020 18:49:27:  24000000 \\nINFO  @ Mon, 14 Sep 2020 18:49:31:  25000000 \\nINFO  @ Mon, 14 Sep 2020 18:49:36:  26000000 \\nINFO  @ Mon, 14 Sep 2020 18:49:40:  27000000 \\nINFO  @ Mon, 14 Sep 2020 18:49:45:  28000000 \\nINFO  @ Mon, 14 Sep 2020 18:49:49:  29000000 \\nINFO  @ Mon, 14 Sep 2020 18:49:54:  30000000 \\nINFO  @ Mon, 14 Sep 2020 18:49:58:  31000000 \\nINFO  @ Mon, 14 Sep 2020 18:50:03:  32000000 \\nINFO  @ Mon, 14 Sep 2020 18:50:07:  33000000 \\nINFO  @ Mon, 14 Sep 2020 18:50:12:  34000000 \\nINFO  @ Mon, 14 Sep 2020 18:50:16:  35000000 \\nINFO  @ Mon, 14 Sep 2020 18:50:21:  36000000 \\nINFO  @ Mon, 14 Sep 2020 18:50:25:  37000000 \\nINFO  @ Mon, 14 Sep 2020 18:50:30:  38000000 \\nINFO  @ Mon, 14 Sep 2020 18:50:32: 38388234 fragments have been read. \\nINFO  @ Mon, 14 Sep 2020 18:50:56: #1.2 read input fragments... \\nINFO  @ Mon, 14 Sep 2020 18:51:00:  1000000 \\nINFO  @ Mon, 14 Sep 2020 18:51:03:  2000000 \\nINFO  @ Mon, 14 Sep 2020 18:51:07:  3000000 \\nINFO  @ Mon, 14 Sep 2020 18:51:11:  4000000 \\nINFO  @ Mon, 14 Sep 2020 18:51:14:  5000000 \\nINFO  @ Mon, 14 Sep 2020 18:51:18:  6000000 \\nINFO  @ Mon, 14 Sep 2020 18:51:22:  7000000 \\nINFO  @ Mon, 14 Sep 2020 18:51:26:  8000000 \\nINFO  @ Mon, 14 Sep 2020 18:51:29:  9000000 \\nINFO  @ Mon, 14 Sep 2020 18:51:33:  10000000 \\nINFO  @ Mon, 14 Sep 2020 18:51:37:  11000000 \\nINFO  @ Mon, 14 Sep 2020 18:51:40:  12000000 \\nINFO  @ Mon, 14 Sep 2020 18:51:44:  13000000 \\nINFO  @ Mon, 14 Sep 2020 18:51:48:  14000000 \\nINFO  @ Mon, 14 Sep 2020 18:51:52:  15000000 \\nINFO  @ Mon, 14 Sep 2020 18:51:55:  16000000 \\nINFO  @ Mon, 14 Sep 2020 18:51:59:  17000000 \\nINFO  @ Mon, 14 Sep 2020 18:52:04:  18000000 \\nINFO  @ Mon, 14 Sep 2020 18:52:11:  19000000 \\nINFO  @ Mon, 14 Sep 2020 18:52:14:  20000000 \\nINFO  @ Mon, 14 Sep 2020 18:52:18:  21000000 \\nINFO  @ Mon, 14 Sep 2020 18:52:22:  22000000 \\nINFO  @ Mon, 14 Sep 2020 18:52:26:  23000000 \\nINFO  @ Mon, 14 Sep 2020 18:52:29:  24000000 \\nINFO  @ Mon, 14 Sep 2020 18:52:33:  25000000 \\nINFO  @ Mon, 14 Sep 2020 18:52:36:  26000000 \\nINFO  @ Mon, 14 Sep 2020 18:52:40:  27000000 \\nINFO  @ Mon, 14 Sep 2020 18:52:44:  28000000 \\nINFO  @ Mon, 14 Sep 2020 18:52:47:  29000000 \\nINFO  @ Mon, 14 Sep 2020 18:52:51:  30000000 \\nINFO  @ Mon, 14 Sep 2020 18:52:55:  31000000 \\nINFO  @ Mon, 14 Sep 2020 18:52:58:  32000000 \\nINFO  @ Mon, 14 Sep 2020 18:53:02:  33000000 \\nINFO  @ Mon, 14 Sep 2020 18:53:06:  34000000 \\nINFO  @ Mon, 14 Sep 2020 18:53:09:  35000000 \\nINFO  @ Mon, 14 Sep 2020 18:53:13:  36000000 \\nINFO  @ Mon, 14 Sep 2020 18:53:16:  37000000 \\nINFO  @ Mon, 14 Sep 2020 18:53:20:  38000000 \\nINFO  @ Mon, 14 Sep 2020 18:53:24:  39000000 \\nINFO  @ Mon, 14 Sep 2020 18:53:27:  40000000 \\nINFO  @ Mon, 14 Sep 2020 18:53:31:  41000000 \\nINFO  @ Mon, 14 Sep 2020 18:53:35:  42000000 \\nINFO  @ Mon, 14 Sep 2020 18:53:38:  43000000 \\nINFO  @ Mon, 14 Sep 2020 18:53:42:  44000000 \\nINFO  @ Mon, 14 Sep 2020 18:53:46:  45000000 \\nINFO  @ Mon, 14 Sep 2020 18:53:50:  46000000 \\nINFO  @ Mon, 14 Sep 2020 18:53:53:  47000000 \\nINFO  @ Mon, 14 Sep 2020 18:53:57:  48000000 \\nINFO  @ Mon, 14 Sep 2020 18:54:00:  49000000 \\nINFO  @ Mon, 14 Sep 2020 18:54:04:  50000000 \\nINFO  @ Mon, 14 Sep 2020 18:54:08:  51000000 \\nINFO  @ Mon, 14 Sep 2020 18:54:12:  52000000 \\nINFO  @ Mon, 14 Sep 2020 18:54:15:  53000000 \\nINFO  @ Mon, 14 Sep 2020 18:54:19:  54000000 \\nINFO  @ Mon, 14 Sep 2020 18:54:23:  55000000 \\nINFO  @ Mon, 14 Sep 2020 18:54:26:  56000000 \\nINFO  @ Mon, 14 Sep 2020 18:54:30:  57000000 \\nINFO  @ Mon, 14 Sep 2020 18:54:34:  58000000 \\nINFO  @ Mon, 14 Sep 2020 18:54:38:  59000000 \\nINFO  @ Mon, 14 Sep 2020 18:54:41:  60000000 \\nINFO  @ Mon, 14 Sep 2020 18:54:45:  61000000 \\nINFO  @ Mon, 14 Sep 2020 18:54:49:  62000000 \\nINFO  @ Mon, 14 Sep 2020 18:54:52:  63000000 \\nINFO  @ Mon, 14 Sep 2020 18:54:56:  64000000 \\nINFO  @ Mon, 14 Sep 2020 18:55:00:  65000000 \\nINFO  @ Mon, 14 Sep 2020 18:55:04:  66000000 \\nINFO  @ Mon, 14 Sep 2020 18:55:06: 66660428 fragments have been read. \\nINFO  @ Mon, 14 Sep 2020 18:55:50: #1 mean fragment size is determined as 286.5 bp from treatment \\nINFO  @ Mon, 14 Sep 2020 18:55:50: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Mon, 14 Sep 2020 18:55:50: #1 fragment size = 286.5 \\nINFO  @ Mon, 14 Sep 2020 18:55:50: #1  total fragments in treatment: 38388234 \\nINFO  @ Mon, 14 Sep 2020 18:55:50: #1 user defined the maximum fragments... \\nINFO  @ Mon, 14 Sep 2020 18:55:50: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 14 Sep 2020 18:56:56: #1  fragments after filtering in treatment: 38378814 \\nINFO  @ Mon, 14 Sep 2020 18:56:56: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 14 Sep 2020 18:56:56: #1  total fragments in control: 66660428 \\nINFO  @ Mon, 14 Sep 2020 18:56:56: #1 user defined the maximum fragments... \\nINFO  @ Mon, 14 Sep 2020 18:56:56: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 14 Sep 2020 18:58:45: #1  fragments after filtering in control: 66632618 \\nINFO  @ Mon, 14 Sep 2020 18:58:45: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 14 Sep 2020 18:58:45: #1 finished! \\nINFO  @ Mon, 14 Sep 2020 18:58:45: #2 Build Peak Model... \\nINFO  @ Mon, 14 Sep 2020 18:58:45: #2 Skipped... \\nINFO  @ Mon, 14 Sep 2020 18:58:45: #3 Call peaks... \\nINFO  @ Mon, 14 Sep 2020 18:58:45: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 14 Sep 2020 19:09:00: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 14 Sep 2020 19:09:00: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_RNP_RUNX2-H3K27AC_R2_treat_pileup.bdg \\nINFO  @ Mon, 14 Sep 2020 19:09:00: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_RNP_RUNX2-H3K27AC_R2_control_lambda.bdg \\nINFO  @ Mon, 14 Sep 2020 19:09:00: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 14 Sep 2020 19:09:00: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 14 Sep 2020 19:15:32: #4 Write output xls file... ../../data/additional_degraded_v1/diffData/MV411_RNP_RUNX2-H3K27AC_R2_peaks.xls \\nINFO  @ Mon, 14 Sep 2020 19:15:32: #4 Write peak in narrowPeak format file... ../../data/additional_degraded_v1/diffData/MV411_RNP_RUNX2-H3K27AC_R2_peaks.narrowPeak \\nINFO  @ Mon, 14 Sep 2020 19:15:32: #4 Write summits bed file... ../../data/additional_degraded_v1/diffData/MV411_RNP_RUNX2-H3K27AC_R2_summits.bed \\nINFO  @ Mon, 14 Sep 2020 19:15:32: Done! \\n'\n",
      "23219629 38378814\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup.bdg --c1 ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R2_control_lambda.bdg --t2 ../../data/additional_degraded_v1/diffData/MV411_RNP_RUNX2-H3K27AC_R2_treat_pileup.bdg --c2 ../../data/additional_degraded_v1/diffData/MV411_RNP_RUNX2-H3K27AC_R2_control_lambda.bdg --d1 23219629 --d2 38378814 -g 60 -l 240 --o-prefix MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_RUNX2-H3K27AC_R2_treat_pileup --outdir ../../data/additional_degraded_v1/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Mon, 14 Sep 2020 19:15:32: Read and build treatment 1 bedGraph... \\nINFO  @ Mon, 14 Sep 2020 19:16:11: Read and build control 1 bedGraph... \\nINFO  @ Mon, 14 Sep 2020 19:19:27: Read and build treatment 2 bedGraph... \\nINFO  @ Mon, 14 Sep 2020 19:20:15: Read and build control 2 bedGraph... \\nINFO  @ Mon, 14 Sep 2020 19:33:33: Write peaks... \\nINFO  @ Mon, 14 Sep 2020 19:33:33: Done \\n')\n",
      "../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.sorted.bam ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_SPI1-H3K27AC_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.sorted.bam and ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_SPI1-H3K27AC_R1.mLb.clN.sorted.bam\n",
      "using default|given size\n",
      "computing the scaling values\n",
      "b'INFO  @ Mon, 14 Sep 2020 19:33:34: \\n# Command line: callpeak -B -t ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.sorted.bam -c ../../data/additional_degraded_v1/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 240 -n MV411_RNP_AAVS1-H3K27AC_R1 --outdir ../../data/additional_degraded_v1/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_RNP_AAVS1-H3K27AC_R1\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/additional_degraded_v1/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Mon, 14 Sep 2020 19:33:34: #1 read fragment files... \\nINFO  @ Mon, 14 Sep 2020 19:33:34: #1 read treatment fragments... \\nINFO  @ Mon, 14 Sep 2020 19:33:39:  1000000 \\nINFO  @ Mon, 14 Sep 2020 19:33:44:  2000000 \\nINFO  @ Mon, 14 Sep 2020 19:33:49:  3000000 \\nINFO  @ Mon, 14 Sep 2020 19:33:53:  4000000 \\nINFO  @ Mon, 14 Sep 2020 19:33:58:  5000000 \\nINFO  @ Mon, 14 Sep 2020 19:34:03:  6000000 \\nINFO  @ Mon, 14 Sep 2020 19:34:07:  7000000 \\nINFO  @ Mon, 14 Sep 2020 19:34:12:  8000000 \\nINFO  @ Mon, 14 Sep 2020 19:34:17:  9000000 \\nINFO  @ Mon, 14 Sep 2020 19:34:21:  10000000 \\nINFO  @ Mon, 14 Sep 2020 19:34:26:  11000000 \\nINFO  @ Mon, 14 Sep 2020 19:34:31:  12000000 \\nINFO  @ Mon, 14 Sep 2020 19:34:35:  13000000 \\nINFO  @ Mon, 14 Sep 2020 19:34:40:  14000000 \\nINFO  @ Mon, 14 Sep 2020 19:34:44:  15000000 \\nINFO  @ Mon, 14 Sep 2020 19:34:49:  16000000 \\nINFO  @ Mon, 14 Sep 2020 19:34:54:  17000000 \\nINFO  @ Mon, 14 Sep 2020 19:34:58:  18000000 \\nINFO  @ Mon, 14 Sep 2020 19:35:03:  19000000 \\nINFO  @ Mon, 14 Sep 2020 19:35:07:  20000000 \\nINFO  @ Mon, 14 Sep 2020 19:35:11: 20629699 fragments have been read. \\nINFO  @ Mon, 14 Sep 2020 19:35:23: #1.2 read input fragments... \\nINFO  @ Mon, 14 Sep 2020 19:35:27:  1000000 \\nINFO  @ Mon, 14 Sep 2020 19:35:30:  2000000 \\nINFO  @ Mon, 14 Sep 2020 19:35:34:  3000000 \\nINFO  @ Mon, 14 Sep 2020 19:35:38:  4000000 \\nINFO  @ Mon, 14 Sep 2020 19:35:42:  5000000 \\nINFO  @ Mon, 14 Sep 2020 19:35:46:  6000000 \\nINFO  @ Mon, 14 Sep 2020 19:35:50:  7000000 \\nINFO  @ Mon, 14 Sep 2020 19:35:53:  8000000 \\nINFO  @ Mon, 14 Sep 2020 19:35:57:  9000000 \\nINFO  @ Mon, 14 Sep 2020 19:36:01:  10000000 \\nINFO  @ Mon, 14 Sep 2020 19:36:05:  11000000 \\nINFO  @ Mon, 14 Sep 2020 19:36:09:  12000000 \\nINFO  @ Mon, 14 Sep 2020 19:36:13:  13000000 \\nINFO  @ Mon, 14 Sep 2020 19:36:17:  14000000 \\nINFO  @ Mon, 14 Sep 2020 19:36:20:  15000000 \\nINFO  @ Mon, 14 Sep 2020 19:36:24:  16000000 \\nINFO  @ Mon, 14 Sep 2020 19:36:28:  17000000 \\nINFO  @ Mon, 14 Sep 2020 19:36:32:  18000000 \\nINFO  @ Mon, 14 Sep 2020 19:36:35:  19000000 \\nINFO  @ Mon, 14 Sep 2020 19:36:39:  20000000 \\nINFO  @ Mon, 14 Sep 2020 19:36:43:  21000000 \\nINFO  @ Mon, 14 Sep 2020 19:36:47:  22000000 \\nINFO  @ Mon, 14 Sep 2020 19:36:51:  23000000 \\nINFO  @ Mon, 14 Sep 2020 19:36:54:  24000000 \\nINFO  @ Mon, 14 Sep 2020 19:36:58:  25000000 \\nINFO  @ Mon, 14 Sep 2020 19:37:02:  26000000 \\nINFO  @ Mon, 14 Sep 2020 19:37:06:  27000000 \\nINFO  @ Mon, 14 Sep 2020 19:37:09:  28000000 \\nINFO  @ Mon, 14 Sep 2020 19:37:13:  29000000 \\nINFO  @ Mon, 14 Sep 2020 19:37:17:  30000000 \\nINFO  @ Mon, 14 Sep 2020 19:37:21:  31000000 \\nINFO  @ Mon, 14 Sep 2020 19:37:25:  32000000 \\nINFO  @ Mon, 14 Sep 2020 19:37:28:  33000000 \\nINFO  @ Mon, 14 Sep 2020 19:37:32:  34000000 \\nINFO  @ Mon, 14 Sep 2020 19:37:36:  35000000 \\nINFO  @ Mon, 14 Sep 2020 19:37:40:  36000000 \\nINFO  @ Mon, 14 Sep 2020 19:37:44:  37000000 \\nINFO  @ Mon, 14 Sep 2020 19:37:47:  38000000 \\nINFO  @ Mon, 14 Sep 2020 19:37:51:  39000000 \\nINFO  @ Mon, 14 Sep 2020 19:37:55:  40000000 \\nINFO  @ Mon, 14 Sep 2020 19:37:59:  41000000 \\nINFO  @ Mon, 14 Sep 2020 19:38:03:  42000000 \\nINFO  @ Mon, 14 Sep 2020 19:38:07:  43000000 \\nINFO  @ Mon, 14 Sep 2020 19:38:10:  44000000 \\nINFO  @ Mon, 14 Sep 2020 19:38:14:  45000000 \\nINFO  @ Mon, 14 Sep 2020 19:38:18:  46000000 \\nINFO  @ Mon, 14 Sep 2020 19:38:22:  47000000 \\nINFO  @ Mon, 14 Sep 2020 19:38:26:  48000000 \\nINFO  @ Mon, 14 Sep 2020 19:38:29:  49000000 \\nINFO  @ Mon, 14 Sep 2020 19:38:33:  50000000 \\nINFO  @ Mon, 14 Sep 2020 19:38:37:  51000000 \\nINFO  @ Mon, 14 Sep 2020 19:38:41:  52000000 \\nINFO  @ Mon, 14 Sep 2020 19:38:45:  53000000 \\nINFO  @ Mon, 14 Sep 2020 19:38:49:  54000000 \\nINFO  @ Mon, 14 Sep 2020 19:38:52:  55000000 \\nINFO  @ Mon, 14 Sep 2020 19:38:56:  56000000 \\nINFO  @ Mon, 14 Sep 2020 19:39:00:  57000000 \\nINFO  @ Mon, 14 Sep 2020 19:39:04:  58000000 \\nINFO  @ Mon, 14 Sep 2020 19:39:08:  59000000 \\nINFO  @ Mon, 14 Sep 2020 19:39:11:  60000000 \\nINFO  @ Mon, 14 Sep 2020 19:39:15:  61000000 \\nINFO  @ Mon, 14 Sep 2020 19:39:19:  62000000 \\nINFO  @ Mon, 14 Sep 2020 19:39:23:  63000000 \\nINFO  @ Mon, 14 Sep 2020 19:39:27:  64000000 \\nINFO  @ Mon, 14 Sep 2020 19:39:31:  65000000 \\nINFO  @ Mon, 14 Sep 2020 19:39:34:  66000000 \\nINFO  @ Mon, 14 Sep 2020 19:39:37: 66660428 fragments have been read. \\nINFO  @ Mon, 14 Sep 2020 19:40:21: #1 mean fragment size is determined as 309.7 bp from treatment \\nINFO  @ Mon, 14 Sep 2020 19:40:21: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Mon, 14 Sep 2020 19:40:21: #1 fragment size = 309.7 \\nINFO  @ Mon, 14 Sep 2020 19:40:21: #1  total fragments in treatment: 20629699 \\nINFO  @ Mon, 14 Sep 2020 19:40:21: #1 user defined the maximum fragments... \\nINFO  @ Mon, 14 Sep 2020 19:40:21: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 14 Sep 2020 19:40:53: #1  fragments after filtering in treatment: 20624221 \\nINFO  @ Mon, 14 Sep 2020 19:40:53: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 14 Sep 2020 19:40:53: #1  total fragments in control: 66660428 \\nINFO  @ Mon, 14 Sep 2020 19:40:53: #1 user defined the maximum fragments... \\nINFO  @ Mon, 14 Sep 2020 19:40:53: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 14 Sep 2020 19:42:39: #1  fragments after filtering in control: 66632618 \\nINFO  @ Mon, 14 Sep 2020 19:42:39: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 14 Sep 2020 19:42:39: #1 finished! \\nINFO  @ Mon, 14 Sep 2020 19:42:39: #2 Build Peak Model... \\nINFO  @ Mon, 14 Sep 2020 19:42:39: #2 Skipped... \\nINFO  @ Mon, 14 Sep 2020 19:42:39: #3 Call peaks... \\nINFO  @ Mon, 14 Sep 2020 19:42:39: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 14 Sep 2020 19:51:58: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 14 Sep 2020 19:51:58: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup.bdg \\nINFO  @ Mon, 14 Sep 2020 19:51:58: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_RNP_AAVS1-H3K27AC_R1_control_lambda.bdg \\nINFO  @ Mon, 14 Sep 2020 19:51:58: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 14 Sep 2020 19:51:58: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 14 Sep 2020 19:57:51: #4 Write output xls file... ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R1_peaks.xls \\nINFO  @ Mon, 14 Sep 2020 19:57:51: #4 Write peak in narrowPeak format file... ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R1_peaks.narrowPeak \\nINFO  @ Mon, 14 Sep 2020 19:57:51: #4 Write summits bed file... ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R1_summits.bed \\nINFO  @ Mon, 14 Sep 2020 19:57:51: Done! \\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'INFO  @ Mon, 14 Sep 2020 19:57:52: \\n# Command line: callpeak -B -t ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_SPI1-H3K27AC_R1.mLb.clN.sorted.bam -c ../../data/additional_degraded_v1/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 240 -n MV411_RNP_SPI1-H3K27AC_R1 --outdir ../../data/additional_degraded_v1/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_RNP_SPI1-H3K27AC_R1\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_SPI1-H3K27AC_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/additional_degraded_v1/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Mon, 14 Sep 2020 19:57:52: #1 read fragment files... \\nINFO  @ Mon, 14 Sep 2020 19:57:52: #1 read treatment fragments... \\nINFO  @ Mon, 14 Sep 2020 19:57:57:  1000000 \\nINFO  @ Mon, 14 Sep 2020 19:58:01:  2000000 \\nINFO  @ Mon, 14 Sep 2020 19:58:06:  3000000 \\nINFO  @ Mon, 14 Sep 2020 19:58:10:  4000000 \\nINFO  @ Mon, 14 Sep 2020 19:58:15:  5000000 \\nINFO  @ Mon, 14 Sep 2020 19:58:20:  6000000 \\nINFO  @ Mon, 14 Sep 2020 19:58:24:  7000000 \\nINFO  @ Mon, 14 Sep 2020 19:58:29:  8000000 \\nINFO  @ Mon, 14 Sep 2020 19:58:33:  9000000 \\nINFO  @ Mon, 14 Sep 2020 19:58:38:  10000000 \\nINFO  @ Mon, 14 Sep 2020 19:58:42:  11000000 \\nINFO  @ Mon, 14 Sep 2020 19:58:47:  12000000 \\nINFO  @ Mon, 14 Sep 2020 19:58:52:  13000000 \\nINFO  @ Mon, 14 Sep 2020 19:58:56:  14000000 \\nINFO  @ Mon, 14 Sep 2020 19:59:01:  15000000 \\nINFO  @ Mon, 14 Sep 2020 19:59:05:  16000000 \\nINFO  @ Mon, 14 Sep 2020 19:59:10:  17000000 \\nINFO  @ Mon, 14 Sep 2020 19:59:14:  18000000 \\nINFO  @ Mon, 14 Sep 2020 19:59:19:  19000000 \\nINFO  @ Mon, 14 Sep 2020 19:59:23:  20000000 \\nINFO  @ Mon, 14 Sep 2020 19:59:28:  21000000 \\nINFO  @ Mon, 14 Sep 2020 19:59:33:  22000000 \\nINFO  @ Mon, 14 Sep 2020 19:59:37:  23000000 \\nINFO  @ Mon, 14 Sep 2020 19:59:42:  24000000 \\nINFO  @ Mon, 14 Sep 2020 19:59:46:  25000000 \\nINFO  @ Mon, 14 Sep 2020 19:59:51:  26000000 \\nINFO  @ Mon, 14 Sep 2020 19:59:55:  27000000 \\nINFO  @ Mon, 14 Sep 2020 19:59:57: 27446663 fragments have been read. \\nINFO  @ Mon, 14 Sep 2020 20:00:14: #1.2 read input fragments... \\nINFO  @ Mon, 14 Sep 2020 20:00:18:  1000000 \\nINFO  @ Mon, 14 Sep 2020 20:00:22:  2000000 \\nINFO  @ Mon, 14 Sep 2020 20:00:25:  3000000 \\nINFO  @ Mon, 14 Sep 2020 20:00:29:  4000000 \\nINFO  @ Mon, 14 Sep 2020 20:00:33:  5000000 \\nINFO  @ Mon, 14 Sep 2020 20:00:36:  6000000 \\nINFO  @ Mon, 14 Sep 2020 20:00:40:  7000000 \\nINFO  @ Mon, 14 Sep 2020 20:00:44:  8000000 \\nINFO  @ Mon, 14 Sep 2020 20:00:48:  9000000 \\nINFO  @ Mon, 14 Sep 2020 20:00:51:  10000000 \\nINFO  @ Mon, 14 Sep 2020 20:00:55:  11000000 \\nINFO  @ Mon, 14 Sep 2020 20:00:59:  12000000 \\nINFO  @ Mon, 14 Sep 2020 20:01:03:  13000000 \\nINFO  @ Mon, 14 Sep 2020 20:01:08:  14000000 \\nINFO  @ Mon, 14 Sep 2020 20:01:12:  15000000 \\nINFO  @ Mon, 14 Sep 2020 20:01:15:  16000000 \\nINFO  @ Mon, 14 Sep 2020 20:01:19:  17000000 \\nINFO  @ Mon, 14 Sep 2020 20:01:23:  18000000 \\nINFO  @ Mon, 14 Sep 2020 20:01:26:  19000000 \\nINFO  @ Mon, 14 Sep 2020 20:01:30:  20000000 \\nINFO  @ Mon, 14 Sep 2020 20:01:34:  21000000 \\nINFO  @ Mon, 14 Sep 2020 20:01:37:  22000000 \\nINFO  @ Mon, 14 Sep 2020 20:01:41:  23000000 \\nINFO  @ Mon, 14 Sep 2020 20:01:45:  24000000 \\nINFO  @ Mon, 14 Sep 2020 20:01:49:  25000000 \\nINFO  @ Mon, 14 Sep 2020 20:01:52:  26000000 \\nINFO  @ Mon, 14 Sep 2020 20:01:56:  27000000 \\nINFO  @ Mon, 14 Sep 2020 20:02:00:  28000000 \\nINFO  @ Mon, 14 Sep 2020 20:02:03:  29000000 \\nINFO  @ Mon, 14 Sep 2020 20:02:07:  30000000 \\nINFO  @ Mon, 14 Sep 2020 20:02:11:  31000000 \\nINFO  @ Mon, 14 Sep 2020 20:02:15:  32000000 \\nINFO  @ Mon, 14 Sep 2020 20:02:18:  33000000 \\nINFO  @ Mon, 14 Sep 2020 20:02:22:  34000000 \\nINFO  @ Mon, 14 Sep 2020 20:02:26:  35000000 \\nINFO  @ Mon, 14 Sep 2020 20:02:29:  36000000 \\nINFO  @ Mon, 14 Sep 2020 20:02:33:  37000000 \\nINFO  @ Mon, 14 Sep 2020 20:02:36:  38000000 \\nINFO  @ Mon, 14 Sep 2020 20:02:40:  39000000 \\nINFO  @ Mon, 14 Sep 2020 20:02:44:  40000000 \\nINFO  @ Mon, 14 Sep 2020 20:02:48:  41000000 \\nINFO  @ Mon, 14 Sep 2020 20:02:52:  42000000 \\nINFO  @ Mon, 14 Sep 2020 20:02:55:  43000000 \\nINFO  @ Mon, 14 Sep 2020 20:02:59:  44000000 \\nINFO  @ Mon, 14 Sep 2020 20:03:03:  45000000 \\nINFO  @ Mon, 14 Sep 2020 20:03:07:  46000000 \\nINFO  @ Mon, 14 Sep 2020 20:03:10:  47000000 \\nINFO  @ Mon, 14 Sep 2020 20:03:14:  48000000 \\nINFO  @ Mon, 14 Sep 2020 20:03:18:  49000000 \\nINFO  @ Mon, 14 Sep 2020 20:03:22:  50000000 \\nINFO  @ Mon, 14 Sep 2020 20:03:25:  51000000 \\nINFO  @ Mon, 14 Sep 2020 20:03:29:  52000000 \\nINFO  @ Mon, 14 Sep 2020 20:03:33:  53000000 \\nINFO  @ Mon, 14 Sep 2020 20:03:37:  54000000 \\nINFO  @ Mon, 14 Sep 2020 20:03:40:  55000000 \\nINFO  @ Mon, 14 Sep 2020 20:03:44:  56000000 \\nINFO  @ Mon, 14 Sep 2020 20:03:48:  57000000 \\nINFO  @ Mon, 14 Sep 2020 20:03:52:  58000000 \\nINFO  @ Mon, 14 Sep 2020 20:03:56:  59000000 \\nINFO  @ Mon, 14 Sep 2020 20:03:59:  60000000 \\nINFO  @ Mon, 14 Sep 2020 20:04:03:  61000000 \\nINFO  @ Mon, 14 Sep 2020 20:04:07:  62000000 \\nINFO  @ Mon, 14 Sep 2020 20:04:11:  63000000 \\nINFO  @ Mon, 14 Sep 2020 20:04:15:  64000000 \\nINFO  @ Mon, 14 Sep 2020 20:04:18:  65000000 \\nINFO  @ Mon, 14 Sep 2020 20:04:22:  66000000 \\nINFO  @ Mon, 14 Sep 2020 20:04:25: 66660428 fragments have been read. \\nINFO  @ Mon, 14 Sep 2020 20:05:09: #1 mean fragment size is determined as 245.9 bp from treatment \\nINFO  @ Mon, 14 Sep 2020 20:05:09: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Mon, 14 Sep 2020 20:05:09: #1 fragment size = 245.9 \\nINFO  @ Mon, 14 Sep 2020 20:05:09: #1  total fragments in treatment: 27446663 \\nINFO  @ Mon, 14 Sep 2020 20:05:09: #1 user defined the maximum fragments... \\nINFO  @ Mon, 14 Sep 2020 20:05:09: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 14 Sep 2020 20:05:55: #1  fragments after filtering in treatment: 27433416 \\nINFO  @ Mon, 14 Sep 2020 20:05:55: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 14 Sep 2020 20:05:55: #1  total fragments in control: 66660428 \\nINFO  @ Mon, 14 Sep 2020 20:05:55: #1 user defined the maximum fragments... \\nINFO  @ Mon, 14 Sep 2020 20:05:55: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 14 Sep 2020 20:07:42: #1  fragments after filtering in control: 66632618 \\nINFO  @ Mon, 14 Sep 2020 20:07:42: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 14 Sep 2020 20:07:42: #1 finished! \\nINFO  @ Mon, 14 Sep 2020 20:07:42: #2 Build Peak Model... \\nINFO  @ Mon, 14 Sep 2020 20:07:42: #2 Skipped... \\nINFO  @ Mon, 14 Sep 2020 20:07:42: #3 Call peaks... \\nINFO  @ Mon, 14 Sep 2020 20:07:42: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 14 Sep 2020 20:16:59: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 14 Sep 2020 20:16:59: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_RNP_SPI1-H3K27AC_R1_treat_pileup.bdg \\nINFO  @ Mon, 14 Sep 2020 20:16:59: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_RNP_SPI1-H3K27AC_R1_control_lambda.bdg \\nINFO  @ Mon, 14 Sep 2020 20:16:59: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 14 Sep 2020 20:16:59: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 14 Sep 2020 20:23:05: #4 Write output xls file... ../../data/additional_degraded_v1/diffData/MV411_RNP_SPI1-H3K27AC_R1_peaks.xls \\nINFO  @ Mon, 14 Sep 2020 20:23:05: #4 Write peak in narrowPeak format file... ../../data/additional_degraded_v1/diffData/MV411_RNP_SPI1-H3K27AC_R1_peaks.narrowPeak \\nINFO  @ Mon, 14 Sep 2020 20:23:06: #4 Write summits bed file... ../../data/additional_degraded_v1/diffData/MV411_RNP_SPI1-H3K27AC_R1_summits.bed \\nINFO  @ Mon, 14 Sep 2020 20:23:06: Done! \\n'\n",
      "10351504 27433416\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup.bdg --c1 ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R1_control_lambda.bdg --t2 ../../data/additional_degraded_v1/diffData/MV411_RNP_SPI1-H3K27AC_R1_treat_pileup.bdg --c2 ../../data/additional_degraded_v1/diffData/MV411_RNP_SPI1-H3K27AC_R1_control_lambda.bdg --d1 10351504 --d2 27433416 -g 60 -l 240 --o-prefix MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_SPI1-H3K27AC_R1_treat_pileup --outdir ../../data/additional_degraded_v1/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Mon, 14 Sep 2020 20:23:06: Read and build treatment 1 bedGraph... \\nINFO  @ Mon, 14 Sep 2020 20:23:40: Read and build control 1 bedGraph... \\nINFO  @ Mon, 14 Sep 2020 20:27:11: Read and build treatment 2 bedGraph... \\nINFO  @ Mon, 14 Sep 2020 20:27:52: Read and build control 2 bedGraph... \\nINFO  @ Mon, 14 Sep 2020 20:40:46: Write peaks... \\nINFO  @ Mon, 14 Sep 2020 20:40:46: Done \\n')\n",
      "../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.sorted.bam ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_SPI1-H3K27AC_R2.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.sorted.bam and ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_SPI1-H3K27AC_R2.mLb.clN.sorted.bam\n",
      "using default|given size\n",
      "computing the scaling values\n",
      "b'INFO  @ Mon, 14 Sep 2020 20:40:48: \\n# Command line: callpeak -B -t ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.sorted.bam -c ../../data/additional_degraded_v1/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 240 -n MV411_RNP_AAVS1-H3K27AC_R2 --outdir ../../data/additional_degraded_v1/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_RNP_AAVS1-H3K27AC_R2\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/additional_degraded_v1/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Mon, 14 Sep 2020 20:40:48: #1 read fragment files... \\nINFO  @ Mon, 14 Sep 2020 20:40:48: #1 read treatment fragments... \\nINFO  @ Mon, 14 Sep 2020 20:40:52:  1000000 \\nINFO  @ Mon, 14 Sep 2020 20:40:57:  2000000 \\nINFO  @ Mon, 14 Sep 2020 20:41:01:  3000000 \\nINFO  @ Mon, 14 Sep 2020 20:41:06:  4000000 \\nINFO  @ Mon, 14 Sep 2020 20:41:11:  5000000 \\nINFO  @ Mon, 14 Sep 2020 20:41:15:  6000000 \\nINFO  @ Mon, 14 Sep 2020 20:41:20:  7000000 \\nINFO  @ Mon, 14 Sep 2020 20:41:25:  8000000 \\nINFO  @ Mon, 14 Sep 2020 20:41:30:  9000000 \\nINFO  @ Mon, 14 Sep 2020 20:41:34:  10000000 \\nINFO  @ Mon, 14 Sep 2020 20:41:41:  11000000 \\nINFO  @ Mon, 14 Sep 2020 20:41:46:  12000000 \\nINFO  @ Mon, 14 Sep 2020 20:41:51:  13000000 \\nINFO  @ Mon, 14 Sep 2020 20:41:56:  14000000 \\nINFO  @ Mon, 14 Sep 2020 20:42:00:  15000000 \\nINFO  @ Mon, 14 Sep 2020 20:42:05:  16000000 \\nINFO  @ Mon, 14 Sep 2020 20:42:10:  17000000 \\nINFO  @ Mon, 14 Sep 2020 20:42:14:  18000000 \\nINFO  @ Mon, 14 Sep 2020 20:42:19:  19000000 \\nINFO  @ Mon, 14 Sep 2020 20:42:24:  20000000 \\nINFO  @ Mon, 14 Sep 2020 20:42:28:  21000000 \\nINFO  @ Mon, 14 Sep 2020 20:42:33:  22000000 \\nINFO  @ Mon, 14 Sep 2020 20:42:38:  23000000 \\nINFO  @ Mon, 14 Sep 2020 20:42:42:  24000000 \\nINFO  @ Mon, 14 Sep 2020 20:42:47:  25000000 \\nINFO  @ Mon, 14 Sep 2020 20:42:51:  26000000 \\nINFO  @ Mon, 14 Sep 2020 20:42:56:  27000000 \\nINFO  @ Mon, 14 Sep 2020 20:42:59: 27510868 fragments have been read. \\nINFO  @ Mon, 14 Sep 2020 20:43:16: #1.2 read input fragments... \\nINFO  @ Mon, 14 Sep 2020 20:43:19:  1000000 \\nINFO  @ Mon, 14 Sep 2020 20:43:23:  2000000 \\nINFO  @ Mon, 14 Sep 2020 20:43:27:  3000000 \\nINFO  @ Mon, 14 Sep 2020 20:43:30:  4000000 \\nINFO  @ Mon, 14 Sep 2020 20:43:34:  5000000 \\nINFO  @ Mon, 14 Sep 2020 20:43:38:  6000000 \\nINFO  @ Mon, 14 Sep 2020 20:43:41:  7000000 \\nINFO  @ Mon, 14 Sep 2020 20:43:45:  8000000 \\nINFO  @ Mon, 14 Sep 2020 20:43:49:  9000000 \\nINFO  @ Mon, 14 Sep 2020 20:43:53:  10000000 \\nINFO  @ Mon, 14 Sep 2020 20:43:56:  11000000 \\nINFO  @ Mon, 14 Sep 2020 20:44:00:  12000000 \\nINFO  @ Mon, 14 Sep 2020 20:44:04:  13000000 \\nINFO  @ Mon, 14 Sep 2020 20:44:07:  14000000 \\nINFO  @ Mon, 14 Sep 2020 20:44:11:  15000000 \\nINFO  @ Mon, 14 Sep 2020 20:44:15:  16000000 \\nINFO  @ Mon, 14 Sep 2020 20:44:18:  17000000 \\nINFO  @ Mon, 14 Sep 2020 20:44:22:  18000000 \\nINFO  @ Mon, 14 Sep 2020 20:44:26:  19000000 \\nINFO  @ Mon, 14 Sep 2020 20:44:29:  20000000 \\nINFO  @ Mon, 14 Sep 2020 20:44:33:  21000000 \\nINFO  @ Mon, 14 Sep 2020 20:44:37:  22000000 \\nINFO  @ Mon, 14 Sep 2020 20:44:40:  23000000 \\nINFO  @ Mon, 14 Sep 2020 20:44:44:  24000000 \\nINFO  @ Mon, 14 Sep 2020 20:44:48:  25000000 \\nINFO  @ Mon, 14 Sep 2020 20:44:51:  26000000 \\nINFO  @ Mon, 14 Sep 2020 20:44:55:  27000000 \\nINFO  @ Mon, 14 Sep 2020 20:44:59:  28000000 \\nINFO  @ Mon, 14 Sep 2020 20:45:02:  29000000 \\nINFO  @ Mon, 14 Sep 2020 20:45:06:  30000000 \\nINFO  @ Mon, 14 Sep 2020 20:45:10:  31000000 \\nINFO  @ Mon, 14 Sep 2020 20:45:13:  32000000 \\nINFO  @ Mon, 14 Sep 2020 20:45:17:  33000000 \\nINFO  @ Mon, 14 Sep 2020 20:45:21:  34000000 \\nINFO  @ Mon, 14 Sep 2020 20:45:24:  35000000 \\nINFO  @ Mon, 14 Sep 2020 20:45:28:  36000000 \\nINFO  @ Mon, 14 Sep 2020 20:45:32:  37000000 \\nINFO  @ Mon, 14 Sep 2020 20:45:35:  38000000 \\nINFO  @ Mon, 14 Sep 2020 20:45:39:  39000000 \\nINFO  @ Mon, 14 Sep 2020 20:45:43:  40000000 \\nINFO  @ Mon, 14 Sep 2020 20:45:46:  41000000 \\nINFO  @ Mon, 14 Sep 2020 20:45:50:  42000000 \\nINFO  @ Mon, 14 Sep 2020 20:45:54:  43000000 \\nINFO  @ Mon, 14 Sep 2020 20:45:57:  44000000 \\nINFO  @ Mon, 14 Sep 2020 20:46:01:  45000000 \\nINFO  @ Mon, 14 Sep 2020 20:46:05:  46000000 \\nINFO  @ Mon, 14 Sep 2020 20:46:09:  47000000 \\nINFO  @ Mon, 14 Sep 2020 20:46:12:  48000000 \\nINFO  @ Mon, 14 Sep 2020 20:46:16:  49000000 \\nINFO  @ Mon, 14 Sep 2020 20:46:20:  50000000 \\nINFO  @ Mon, 14 Sep 2020 20:46:23:  51000000 \\nINFO  @ Mon, 14 Sep 2020 20:46:27:  52000000 \\nINFO  @ Mon, 14 Sep 2020 20:46:31:  53000000 \\nINFO  @ Mon, 14 Sep 2020 20:46:34:  54000000 \\nINFO  @ Mon, 14 Sep 2020 20:46:38:  55000000 \\nINFO  @ Mon, 14 Sep 2020 20:46:42:  56000000 \\nINFO  @ Mon, 14 Sep 2020 20:46:46:  57000000 \\nINFO  @ Mon, 14 Sep 2020 20:46:49:  58000000 \\nINFO  @ Mon, 14 Sep 2020 20:46:53:  59000000 \\nINFO  @ Mon, 14 Sep 2020 20:46:57:  60000000 \\nINFO  @ Mon, 14 Sep 2020 20:47:01:  61000000 \\nINFO  @ Mon, 14 Sep 2020 20:47:04:  62000000 \\nINFO  @ Mon, 14 Sep 2020 20:47:08:  63000000 \\nINFO  @ Mon, 14 Sep 2020 20:47:12:  64000000 \\nINFO  @ Mon, 14 Sep 2020 20:47:16:  65000000 \\nINFO  @ Mon, 14 Sep 2020 20:47:19:  66000000 \\nINFO  @ Mon, 14 Sep 2020 20:47:22: 66660428 fragments have been read. \\nINFO  @ Mon, 14 Sep 2020 20:48:06: #1 mean fragment size is determined as 304.3 bp from treatment \\nINFO  @ Mon, 14 Sep 2020 20:48:06: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Mon, 14 Sep 2020 20:48:06: #1 fragment size = 304.3 \\nINFO  @ Mon, 14 Sep 2020 20:48:06: #1  total fragments in treatment: 27510868 \\nINFO  @ Mon, 14 Sep 2020 20:48:06: #1 user defined the maximum fragments... \\nINFO  @ Mon, 14 Sep 2020 20:48:06: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 14 Sep 2020 20:48:50: #1  fragments after filtering in treatment: 27503766 \\nINFO  @ Mon, 14 Sep 2020 20:48:50: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 14 Sep 2020 20:48:50: #1  total fragments in control: 66660428 \\nINFO  @ Mon, 14 Sep 2020 20:48:50: #1 user defined the maximum fragments... \\nINFO  @ Mon, 14 Sep 2020 20:48:50: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 14 Sep 2020 20:50:36: #1  fragments after filtering in control: 66632618 \\nINFO  @ Mon, 14 Sep 2020 20:50:36: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 14 Sep 2020 20:50:36: #1 finished! \\nINFO  @ Mon, 14 Sep 2020 20:50:36: #2 Build Peak Model... \\nINFO  @ Mon, 14 Sep 2020 20:50:36: #2 Skipped... \\nINFO  @ Mon, 14 Sep 2020 20:50:36: #3 Call peaks... \\nINFO  @ Mon, 14 Sep 2020 20:50:36: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 14 Sep 2020 21:00:00: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 14 Sep 2020 21:00:00: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup.bdg \\nINFO  @ Mon, 14 Sep 2020 21:00:00: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_RNP_AAVS1-H3K27AC_R2_control_lambda.bdg \\nINFO  @ Mon, 14 Sep 2020 21:00:00: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 14 Sep 2020 21:00:00: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 14 Sep 2020 21:06:00: #4 Write output xls file... ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R2_peaks.xls \\nINFO  @ Mon, 14 Sep 2020 21:06:00: #4 Write peak in narrowPeak format file... ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R2_peaks.narrowPeak \\nINFO  @ Mon, 14 Sep 2020 21:06:00: #4 Write summits bed file... ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R2_summits.bed \\nINFO  @ Mon, 14 Sep 2020 21:06:01: Done! \\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'INFO  @ Mon, 14 Sep 2020 21:06:01: \\n# Command line: callpeak -B -t ../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_SPI1-H3K27AC_R2.mLb.clN.sorted.bam -c ../../data/additional_degraded_v1/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 240 -n MV411_RNP_SPI1-H3K27AC_R2 --outdir ../../data/additional_degraded_v1/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_RNP_SPI1-H3K27AC_R2\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/additional_degraded_v1/bwa/mergedLibrary/MV411_RNP_SPI1-H3K27AC_R2.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/additional_degraded_v1/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Mon, 14 Sep 2020 21:06:01: #1 read fragment files... \\nINFO  @ Mon, 14 Sep 2020 21:06:01: #1 read treatment fragments... \\nINFO  @ Mon, 14 Sep 2020 21:06:06:  1000000 \\nINFO  @ Mon, 14 Sep 2020 21:06:10:  2000000 \\nINFO  @ Mon, 14 Sep 2020 21:06:15:  3000000 \\nINFO  @ Mon, 14 Sep 2020 21:06:19:  4000000 \\nINFO  @ Mon, 14 Sep 2020 21:06:24:  5000000 \\nINFO  @ Mon, 14 Sep 2020 21:06:28:  6000000 \\nINFO  @ Mon, 14 Sep 2020 21:06:33:  7000000 \\nINFO  @ Mon, 14 Sep 2020 21:06:38:  8000000 \\nINFO  @ Mon, 14 Sep 2020 21:06:42:  9000000 \\nINFO  @ Mon, 14 Sep 2020 21:06:47:  10000000 \\nINFO  @ Mon, 14 Sep 2020 21:06:51:  11000000 \\nINFO  @ Mon, 14 Sep 2020 21:06:56:  12000000 \\nINFO  @ Mon, 14 Sep 2020 21:07:00:  13000000 \\nINFO  @ Mon, 14 Sep 2020 21:07:05:  14000000 \\nINFO  @ Mon, 14 Sep 2020 21:07:09:  15000000 \\nINFO  @ Mon, 14 Sep 2020 21:07:14:  16000000 \\nINFO  @ Mon, 14 Sep 2020 21:07:18:  17000000 \\nINFO  @ Mon, 14 Sep 2020 21:07:23:  18000000 \\nINFO  @ Mon, 14 Sep 2020 21:07:27:  19000000 \\nINFO  @ Mon, 14 Sep 2020 21:07:32:  20000000 \\nINFO  @ Mon, 14 Sep 2020 21:07:36:  21000000 \\nINFO  @ Mon, 14 Sep 2020 21:07:41:  22000000 \\nINFO  @ Mon, 14 Sep 2020 21:07:45:  23000000 \\nINFO  @ Mon, 14 Sep 2020 21:07:50:  24000000 \\nINFO  @ Mon, 14 Sep 2020 21:07:54:  25000000 \\nINFO  @ Mon, 14 Sep 2020 21:07:59:  26000000 \\nINFO  @ Mon, 14 Sep 2020 21:08:03:  27000000 \\nINFO  @ Mon, 14 Sep 2020 21:08:05: 27322557 fragments have been read. \\nINFO  @ Mon, 14 Sep 2020 21:08:21: #1.2 read input fragments... \\nINFO  @ Mon, 14 Sep 2020 21:08:25:  1000000 \\nINFO  @ Mon, 14 Sep 2020 21:08:29:  2000000 \\nINFO  @ Mon, 14 Sep 2020 21:08:33:  3000000 \\nINFO  @ Mon, 14 Sep 2020 21:08:36:  4000000 \\nINFO  @ Mon, 14 Sep 2020 21:08:40:  5000000 \\nINFO  @ Mon, 14 Sep 2020 21:08:44:  6000000 \\nINFO  @ Mon, 14 Sep 2020 21:08:47:  7000000 \\nINFO  @ Mon, 14 Sep 2020 21:08:51:  8000000 \\nINFO  @ Mon, 14 Sep 2020 21:08:55:  9000000 \\nINFO  @ Mon, 14 Sep 2020 21:08:58:  10000000 \\nINFO  @ Mon, 14 Sep 2020 21:09:02:  11000000 \\nINFO  @ Mon, 14 Sep 2020 21:09:06:  12000000 \\nINFO  @ Mon, 14 Sep 2020 21:09:09:  13000000 \\nINFO  @ Mon, 14 Sep 2020 21:09:13:  14000000 \\nINFO  @ Mon, 14 Sep 2020 21:09:17:  15000000 \\nINFO  @ Mon, 14 Sep 2020 21:09:20:  16000000 \\nINFO  @ Mon, 14 Sep 2020 21:09:24:  17000000 \\nINFO  @ Mon, 14 Sep 2020 21:09:28:  18000000 \\nINFO  @ Mon, 14 Sep 2020 21:09:32:  19000000 \\nINFO  @ Mon, 14 Sep 2020 21:09:35:  20000000 \\nINFO  @ Mon, 14 Sep 2020 21:09:39:  21000000 \\nINFO  @ Mon, 14 Sep 2020 21:09:42:  22000000 \\nINFO  @ Mon, 14 Sep 2020 21:09:46:  23000000 \\nINFO  @ Mon, 14 Sep 2020 21:09:50:  24000000 \\nINFO  @ Mon, 14 Sep 2020 21:09:53:  25000000 \\nINFO  @ Mon, 14 Sep 2020 21:09:57:  26000000 \\nINFO  @ Mon, 14 Sep 2020 21:10:02:  27000000 \\nINFO  @ Mon, 14 Sep 2020 21:10:08:  28000000 \\nINFO  @ Mon, 14 Sep 2020 21:10:11:  29000000 \\nINFO  @ Mon, 14 Sep 2020 21:10:15:  30000000 \\nINFO  @ Mon, 14 Sep 2020 21:10:19:  31000000 \\nINFO  @ Mon, 14 Sep 2020 21:10:22:  32000000 \\nINFO  @ Mon, 14 Sep 2020 21:10:26:  33000000 \\nINFO  @ Mon, 14 Sep 2020 21:10:29:  34000000 \\nINFO  @ Mon, 14 Sep 2020 21:10:33:  35000000 \\nINFO  @ Mon, 14 Sep 2020 21:10:37:  36000000 \\nINFO  @ Mon, 14 Sep 2020 21:10:40:  37000000 \\nINFO  @ Mon, 14 Sep 2020 21:10:44:  38000000 \\nINFO  @ Mon, 14 Sep 2020 21:10:48:  39000000 \\nINFO  @ Mon, 14 Sep 2020 21:10:51:  40000000 \\nINFO  @ Mon, 14 Sep 2020 21:10:55:  41000000 \\nINFO  @ Mon, 14 Sep 2020 21:10:59:  42000000 \\nINFO  @ Mon, 14 Sep 2020 21:11:02:  43000000 \\nINFO  @ Mon, 14 Sep 2020 21:11:06:  44000000 \\nINFO  @ Mon, 14 Sep 2020 21:11:10:  45000000 \\nINFO  @ Mon, 14 Sep 2020 21:11:13:  46000000 \\nINFO  @ Mon, 14 Sep 2020 21:11:17:  47000000 \\nINFO  @ Mon, 14 Sep 2020 21:11:21:  48000000 \\nINFO  @ Mon, 14 Sep 2020 21:11:24:  49000000 \\nINFO  @ Mon, 14 Sep 2020 21:11:28:  50000000 \\nINFO  @ Mon, 14 Sep 2020 21:11:32:  51000000 \\nINFO  @ Mon, 14 Sep 2020 21:11:35:  52000000 \\nINFO  @ Mon, 14 Sep 2020 21:11:39:  53000000 \\nINFO  @ Mon, 14 Sep 2020 21:11:43:  54000000 \\nINFO  @ Mon, 14 Sep 2020 21:11:46:  55000000 \\nINFO  @ Mon, 14 Sep 2020 21:11:50:  56000000 \\nINFO  @ Mon, 14 Sep 2020 21:11:54:  57000000 \\nINFO  @ Mon, 14 Sep 2020 21:11:57:  58000000 \\nINFO  @ Mon, 14 Sep 2020 21:12:01:  59000000 \\nINFO  @ Mon, 14 Sep 2020 21:12:05:  60000000 \\nINFO  @ Mon, 14 Sep 2020 21:12:08:  61000000 \\nINFO  @ Mon, 14 Sep 2020 21:12:12:  62000000 \\nINFO  @ Mon, 14 Sep 2020 21:12:16:  63000000 \\nINFO  @ Mon, 14 Sep 2020 21:12:20:  64000000 \\nINFO  @ Mon, 14 Sep 2020 21:12:23:  65000000 \\nINFO  @ Mon, 14 Sep 2020 21:12:27:  66000000 \\nINFO  @ Mon, 14 Sep 2020 21:12:30: 66660428 fragments have been read. \\nINFO  @ Mon, 14 Sep 2020 21:13:13: #1 mean fragment size is determined as 297.0 bp from treatment \\nINFO  @ Mon, 14 Sep 2020 21:13:13: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Mon, 14 Sep 2020 21:13:13: #1 fragment size = 297.0 \\nINFO  @ Mon, 14 Sep 2020 21:13:13: #1  total fragments in treatment: 27322557 \\nINFO  @ Mon, 14 Sep 2020 21:13:13: #1 user defined the maximum fragments... \\nINFO  @ Mon, 14 Sep 2020 21:13:13: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 14 Sep 2020 21:13:54: #1  fragments after filtering in treatment: 27314390 \\nINFO  @ Mon, 14 Sep 2020 21:13:54: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 14 Sep 2020 21:13:54: #1  total fragments in control: 66660428 \\nINFO  @ Mon, 14 Sep 2020 21:13:54: #1 user defined the maximum fragments... \\nINFO  @ Mon, 14 Sep 2020 21:13:54: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 14 Sep 2020 21:15:39: #1  fragments after filtering in control: 66632618 \\nINFO  @ Mon, 14 Sep 2020 21:15:39: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 14 Sep 2020 21:15:39: #1 finished! \\nINFO  @ Mon, 14 Sep 2020 21:15:39: #2 Build Peak Model... \\nINFO  @ Mon, 14 Sep 2020 21:15:39: #2 Skipped... \\nINFO  @ Mon, 14 Sep 2020 21:15:39: #3 Call peaks... \\nINFO  @ Mon, 14 Sep 2020 21:15:39: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 14 Sep 2020 21:24:59: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 14 Sep 2020 21:24:59: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_RNP_SPI1-H3K27AC_R2_treat_pileup.bdg \\nINFO  @ Mon, 14 Sep 2020 21:24:59: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_RNP_SPI1-H3K27AC_R2_control_lambda.bdg \\nINFO  @ Mon, 14 Sep 2020 21:24:59: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 14 Sep 2020 21:24:59: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 14 Sep 2020 21:31:16: #4 Write output xls file... ../../data/additional_degraded_v1/diffData/MV411_RNP_SPI1-H3K27AC_R2_peaks.xls \\nINFO  @ Mon, 14 Sep 2020 21:31:16: #4 Write peak in narrowPeak format file... ../../data/additional_degraded_v1/diffData/MV411_RNP_SPI1-H3K27AC_R2_peaks.narrowPeak \\nINFO  @ Mon, 14 Sep 2020 21:31:16: #4 Write summits bed file... ../../data/additional_degraded_v1/diffData/MV411_RNP_SPI1-H3K27AC_R2_summits.bed \\nINFO  @ Mon, 14 Sep 2020 21:31:16: Done! \\n'\n",
      "23895878 27314390\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup.bdg --c1 ../../data/additional_degraded_v1/diffData/MV411_RNP_AAVS1-H3K27AC_R2_control_lambda.bdg --t2 ../../data/additional_degraded_v1/diffData/MV411_RNP_SPI1-H3K27AC_R2_treat_pileup.bdg --c2 ../../data/additional_degraded_v1/diffData/MV411_RNP_SPI1-H3K27AC_R2_control_lambda.bdg --d1 23895878 --d2 27314390 -g 60 -l 240 --o-prefix MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_SPI1-H3K27AC_R2_treat_pileup --outdir ../../data/additional_degraded_v1/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Mon, 14 Sep 2020 21:31:16: Read and build treatment 1 bedGraph... \\nINFO  @ Mon, 14 Sep 2020 21:31:56: Read and build control 1 bedGraph... \\nINFO  @ Mon, 14 Sep 2020 21:35:20: Read and build treatment 2 bedGraph... \\nINFO  @ Mon, 14 Sep 2020 21:36:00: Read and build control 2 bedGraph... \\nINFO  @ Mon, 14 Sep 2020 21:49:10: Write peaks... \\nINFO  @ Mon, 14 Sep 2020 21:49:10: Done \\n')\n"
     ]
    }
   ],
   "source": [
    "# diffPeak on scaled data (full reprocessing)\n",
    "size= 240\n",
    "for i in range(len(bams)-3):\n",
    "    if i < 8:\n",
    "        continue\n",
    "    bam1 = bams[1+i%2]\n",
    "    bam2 = bams[3+i]\n",
    "    print(bam1,bam2)\n",
    "    print(chip.fullDiffPeak(bam1,bam2, control1=bams[0], compute_size=False, size=size, scaling=[scales[i],1], directory = \"../../data/\"+project+\"/diffData/\", res_directory = \"../../data/\"+project+\"/diffPeaks/\",pairedend=True))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [],
   "source": [
    "initscales = ! cat ../../data/$project/bwa/mergedLibrary/bigwig/scale/*.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 75,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [val*float(initscales[3+i]) for i, val in enumerate(scales)]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 76,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[0.0055422425878344,\n",
       " 0.0095582660753916,\n",
       " 0.0099217323799776,\n",
       " 0.0123312656199582,\n",
       " 0.01928382044626971,\n",
       " 0.00920760471838008,\n",
       " 0.028572797987493598,\n",
       " 0.00965697715399,\n",
       " 0.01991590914591828,\n",
       " 0.01796301710614069,\n",
       " 0.034577257425544,\n",
       " 0.0109291227710616,\n",
       " 0.00907162286248202,\n",
       " 0.015778937847629487]"
      ]
     },
     "execution_count": 76,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "scales"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 79,
   "metadata": {},
   "outputs": [],
   "source": [
    "chip.bigWigFrom(bams[3:], \n",
    "                genome='GRCh38',scaling=scales,\n",
    "               numthreads=8)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 88,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "MV411_RNP_IRF8-H3K27AC_R1.bw   MV411_RNP_RUNX1-H3K27AC_R2.bw\r\n",
      "MV411_RNP_IRF8-H3K27AC_R2.bw   MV411_RNP_RUNX1_RUNX2-H3K27AC_R1.bw\r\n",
      "MV411_RNP_MEF2D-H3K27AC_R1.bw  MV411_RNP_RUNX1_RUNX2-H3K27AC_R2.bw\r\n",
      "MV411_RNP_MEF2D-H3K27AC_R2.bw  MV411_RNP_RUNX2-H3K27AC_R1.bw\r\n",
      "MV411_RNP_MYB-H3K27AC_R1.bw    MV411_RNP_RUNX2-H3K27AC_R2.bw\r\n",
      "MV411_RNP_MYB-H3K27AC_R2.bw    MV411_RNP_SPI1-H3K27AC_R1.bw\r\n",
      "MV411_RNP_RUNX1-H3K27AC_R1.bw  MV411_RNP_SPI1-H3K27AC_R2.bw\r\n"
     ]
    }
   ],
   "source": [
    "ls ../../data/$project/recalib_bigwig/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 84,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "!mv bigwig/* ../../data/$project/recalib_bigwig/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 91,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "!cp ../../data/$project/bwa/mergedLibrary/bigwig/MV411_RNP_AAVS1-*.bigWig ../../data/$project/recalib_bigwig/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 156,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[autoreload of JKBio.epigenetics.ChIP_helper failed: Traceback (most recent call last):\n",
      "  File \"/home/jeremie/.local/lib/python3.8/site-packages/IPython/extensions/autoreload.py\", line 245, in check\n",
      "    superreload(m, reload, self.old_objects)\n",
      "  File \"/home/jeremie/.local/lib/python3.8/site-packages/IPython/extensions/autoreload.py\", line 394, in superreload\n",
      "    module = reload(module)\n",
      "  File \"/usr/lib/python3.8/imp.py\", line 314, in reload\n",
      "    return importlib.reload(module)\n",
      "  File \"/usr/lib/python3.8/importlib/__init__.py\", line 169, in reload\n",
      "    _bootstrap._exec(spec, module)\n",
      "  File \"<frozen importlib._bootstrap>\", line 604, in _exec\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 779, in exec_module\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 916, in get_code\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 846, in source_to_code\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"../../JKBio/epigenetics/ChIP_helper.py\", line 346\n",
      "    if overlap\n",
      "             ^\n",
      "SyntaxError: invalid syntax\n",
      "]\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "['10158 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_IRF8-H3K27AC_R1_treat_pileup_c3.0_common.bed',\n",
       " '44772 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_IRF8-H3K27AC_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '138 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_IRF8-H3K27AC_R1_treat_pileup_c3.0_cond2.bed',\n",
       " '43929 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_MEF2D-H3K27AC_R1_treat_pileup_c3.0_common.bed',\n",
       " '31829 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_MEF2D-H3K27AC_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '13 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_MEF2D-H3K27AC_R1_treat_pileup_c3.0_cond2.bed',\n",
       " '42999 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_MYB-H3K27AC_R1_treat_pileup_c3.0_common.bed',\n",
       " '23545 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_MYB-H3K27AC_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '5259 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_MYB-H3K27AC_R1_treat_pileup_c3.0_cond2.bed',\n",
       " '52098 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_RUNX1-H3K27AC_R1_treat_pileup_c3.0_common.bed',\n",
       " '168 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_RUNX1-H3K27AC_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '37772 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_RUNX1-H3K27AC_R1_treat_pileup_c3.0_cond2.bed',\n",
       " '50223 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_RUNX1_RUNX2-H3K27AC_R1_treat_pileup_c3.0_common.bed',\n",
       " '16230 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_RUNX1_RUNX2-H3K27AC_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '7922 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_RUNX1_RUNX2-H3K27AC_R1_treat_pileup_c3.0_cond2.bed',\n",
       " '45482 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_RUNX2-H3K27AC_R1_treat_pileup_c3.0_common.bed',\n",
       " '52 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_RUNX2-H3K27AC_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '38062 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_RUNX2-H3K27AC_R1_treat_pileup_c3.0_cond2.bed',\n",
       " '23338 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_SPI1-H3K27AC_R1_treat_pileup_c3.0_common.bed',\n",
       " '42073 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_SPI1-H3K27AC_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '1059 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_SPI1-H3K27AC_R1_treat_pileup_c3.0_cond2.bed',\n",
       " '28031 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_IRF8-H3K27AC_R2_treat_pileup_c3.0_common.bed',\n",
       " '50350 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_IRF8-H3K27AC_R2_treat_pileup_c3.0_cond1.bed',\n",
       " '1071 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_IRF8-H3K27AC_R2_treat_pileup_c3.0_cond2.bed',\n",
       " '56671 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_MEF2D-H3K27AC_R2_treat_pileup_c3.0_common.bed',\n",
       " '29448 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_MEF2D-H3K27AC_R2_treat_pileup_c3.0_cond1.bed',\n",
       " '282 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_MEF2D-H3K27AC_R2_treat_pileup_c3.0_cond2.bed',\n",
       " '40863 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_MYB-H3K27AC_R2_treat_pileup_c3.0_common.bed',\n",
       " '38109 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_MYB-H3K27AC_R2_treat_pileup_c3.0_cond1.bed',\n",
       " '2441 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_MYB-H3K27AC_R2_treat_pileup_c3.0_cond2.bed',\n",
       " '55958 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_RUNX1-H3K27AC_R2_treat_pileup_c3.0_common.bed',\n",
       " '9557 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_RUNX1-H3K27AC_R2_treat_pileup_c3.0_cond1.bed',\n",
       " '1470 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_RUNX1-H3K27AC_R2_treat_pileup_c3.0_cond2.bed',\n",
       " '56368 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_RUNX1_RUNX2-H3K27AC_R2_treat_pileup_c3.0_common.bed',\n",
       " '27500 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_RUNX1_RUNX2-H3K27AC_R2_treat_pileup_c3.0_cond1.bed',\n",
       " '4263 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_RUNX1_RUNX2-H3K27AC_R2_treat_pileup_c3.0_cond2.bed',\n",
       " '57640 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_RUNX2-H3K27AC_R2_treat_pileup_c3.0_common.bed',\n",
       " '13148 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_RUNX2-H3K27AC_R2_treat_pileup_c3.0_cond1.bed',\n",
       " '236 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_RUNX2-H3K27AC_R2_treat_pileup_c3.0_cond2.bed',\n",
       " '45275 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_SPI1-H3K27AC_R2_treat_pileup_c3.0_common.bed',\n",
       " '27804 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_SPI1-H3K27AC_R2_treat_pileup_c3.0_cond1.bed',\n",
       " '10103 ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_SPI1-H3K27AC_R2_treat_pileup_c3.0_cond2.bed',\n",
       " '']"
      ]
     },
     "execution_count": 156,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "os.popen('for i in $(ls ../../data/'+project+'/diffPeaks/*.bed); \\\n",
    "            do echo $(wc -l $i); \\\n",
    "            done').read().split('\\n')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 240,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bigWig',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bigWig',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_IRF8-H3K27AC_R1.bw',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_IRF8-H3K27AC_R2.bw',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_MEF2D-H3K27AC_R1.bw',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_MEF2D-H3K27AC_R2.bw',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_MYB-H3K27AC_R1.bw',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_MYB-H3K27AC_R2.bw',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_RUNX1-H3K27AC_R1.bw',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_RUNX1-H3K27AC_R2.bw',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_RUNX1_RUNX2-H3K27AC_R1.bw',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_RUNX1_RUNX2-H3K27AC_R2.bw',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_RUNX2-H3K27AC_R1.bw',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_RUNX2-H3K27AC_R2.bw',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_SPI1-H3K27AC_R1.bw',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_SPI1-H3K27AC_R2.bw']"
      ]
     },
     "execution_count": 240,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bw = ! ls ../../data/additional_degraded_v1/recalib_bigwig/*\n",
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 167,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mkdir: cannot create directory ../results/additional_degraded_v1/: File exists\n",
      "mkdir: cannot create directory ../results/additional_degraded_v1/plots: File exists\n"
     ]
    }
   ],
   "source": [
    "!mkdir ../results/$project/\n",
    "!mkdir ../results/$project/plots\n",
    "!mkdir ../results/$project/plots/heatmaps/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# GENOME WIDE comparison"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 242,
   "metadata": {},
   "outputs": [],
   "source": [
    "peaks = ! ls ../../data/$project/bwa/mergedLibrary/macs/broadPeak/*.broadPeak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 241,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"AAVS1\", \"AAVS1_v2\", \"IRF8\",\"IRF8_v2\",\"MEF2D\",\"MEF2D_v2\",\"MYB\",\"MYB_v2\",\"RUNX1\",\"RUNX1_v2\",\"RUNX1_RUNX2\",\"RUNX1_RUNX2_v2\",\"RUNX2\",\"RUNX2_v2\", \"SPI1\",\"SPI1_v2\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 245,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IRF8\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_IRF8-H3K27AC_R1.bw ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/additional_degraded_v1/bwa/mergedLibrary/macs/broadPeak/MV411_RNP_AAVS1-H3K27AC_R1_peaks.broadPeak --missingDataAsZero --outFileName ../../data/additional_degraded_v1/IRF8_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/additional_degraded_v1/IRF8_mat.gz --outFileName ../../data/additional_degraded_v1/IRF8_mat.pdf --refPointLabel center -max 4 --legendLocation lower-left --regionsLabel Macs2_Peaks --samplesLabel IRF8 AAVS1 --plotTitle IRF8_vs_AAVS1', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270335v1\\t      1048\\nchrUn_KI270394v1\\t       970\\nchrUn_KI270338v1\\t      1428\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270385v1\\t       990\\nchrUn_KI270372v1\\t      1650\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270752v1\\t     27745\\nchrUn_KI270364v1\\t      2855\\nchrUn_KI270381v1\\t      1930\\nchrUn_KI270383v1\\t      1750\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270371v1\\t      2805\\nchrUn_KI270375v1\\t      2378\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270386v1\\t      1788\\nchrUn_KI270376v1\\t      1136\\nchrUn_KI270340v1\\t      1428\\nchrUn_KI270388v1\\t      1216\\nchrUn_KI270387v1\\t      1537\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270379v1\\t      1045\\nchrUn_KI270392v1\\t       971\\n         chrEBV\\t    171823\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270384v1\\t      1658\\nchrUn_KI270378v1\\t      1048\\nchrUn_KI270395v1\\t      1143\\nchrUn_KI270334v1\\t      1368\\nchrUn_KI270548v1\\t      1599\\nchrUn_KI270373v1\\t      1451\\nchrUn_KI270329v1\\t      1040\\nchrUn_KI270530v1\\t      2168\\n')\n",
      "IRF8_v2\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_IRF8-H3K27AC_R2.bw ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/additional_degraded_v1/bwa/mergedLibrary/macs/broadPeak/MV411_RNP_AAVS1-H3K27AC_R1_peaks.broadPeak --missingDataAsZero --outFileName ../../data/additional_degraded_v1/IRF8_v2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/additional_degraded_v1/IRF8_v2_mat.gz --outFileName ../../data/additional_degraded_v1/IRF8_v2_mat.pdf --refPointLabel center -max 4 --legendLocation lower-left --regionsLabel Macs2_Peaks --samplesLabel IRF8_v2 AAVS1 --plotTitle IRF8_v2_vs_AAVS1', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270378v1\\t      1048\\nchrUn_KI270373v1\\t      1451\\nchrUn_KI270381v1\\t      1930\\nchrUn_KI270371v1\\t      2805\\nchrUn_KI270364v1\\t      2855\\nchrUn_KI270340v1\\t      1428\\nchrUn_KI270385v1\\t       990\\nchrUn_KI270338v1\\t      1428\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270329v1\\t      1040\\nchrUn_KI270379v1\\t      1045\\nchrUn_KI270334v1\\t      1368\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270375v1\\t      2378\\nchrUn_KI270384v1\\t      1658\\nchrUn_KI270372v1\\t      1650\\nchrUn_KI270335v1\\t      1048\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270752v1\\t     27745\\nchrUn_KI270388v1\\t      1216\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270386v1\\t      1788\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270392v1\\t       971\\nchrUn_KI270383v1\\t      1750\\nchrUn_KI270394v1\\t       970\\nchrUn_KI270387v1\\t      1537\\n         chrEBV\\t    171823\\nchrUn_KI270395v1\\t      1143\\nchrUn_KI270376v1\\t      1136\\nchrUn_KI270548v1\\t      1599\\nchrUn_KI270530v1\\t      2168\\nchrUn_KI270528v1\\t      2983\\n')\n",
      "MEF2D\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_MEF2D-H3K27AC_R1.bw ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/additional_degraded_v1/bwa/mergedLibrary/macs/broadPeak/MV411_RNP_AAVS1-H3K27AC_R1_peaks.broadPeak --missingDataAsZero --outFileName ../../data/additional_degraded_v1/MEF2D_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/additional_degraded_v1/MEF2D_mat.gz --outFileName ../../data/additional_degraded_v1/MEF2D_mat.pdf --refPointLabel center -max 4 --legendLocation lower-left --regionsLabel Macs2_Peaks --samplesLabel MEF2D AAVS1 --plotTitle MEF2D_vs_AAVS1', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270386v1\\t      1788\\nchrUn_KI270394v1\\t       970\\nchrUn_KI270329v1\\t      1040\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270385v1\\t       990\\nchrUn_KI270395v1\\t      1143\\nchrUn_KI270375v1\\t      2378\\nchrUn_KI270392v1\\t       971\\nchrUn_KI270384v1\\t      1658\\nchrUn_KI270371v1\\t      2805\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270378v1\\t      1048\\nchrUn_KI270381v1\\t      1930\\nchrUn_KI270548v1\\t      1599\\nchrUn_KI270364v1\\t      2855\\nchrUn_KI270412v1\\t      1179\\n         chrEBV\\t    171823\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270334v1\\t      1368\\nchrUn_KI270340v1\\t      1428\\nchrUn_KI270752v1\\t     27745\\nchrUn_KI270335v1\\t      1048\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270372v1\\t      1650\\nchrUn_KI270338v1\\t      1428\\nchrUn_KI270388v1\\t      1216\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270387v1\\t      1537\\nchrUn_KI270376v1\\t      1136\\nchrUn_KI270530v1\\t      2168\\nchrUn_KI270383v1\\t      1750\\nchrUn_KI270379v1\\t      1045\\nchrUn_KI270373v1\\t      1451\\n')\n",
      "MEF2D_v2\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_MEF2D-H3K27AC_R2.bw ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/additional_degraded_v1/bwa/mergedLibrary/macs/broadPeak/MV411_RNP_AAVS1-H3K27AC_R1_peaks.broadPeak --missingDataAsZero --outFileName ../../data/additional_degraded_v1/MEF2D_v2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/additional_degraded_v1/MEF2D_v2_mat.gz --outFileName ../../data/additional_degraded_v1/MEF2D_v2_mat.pdf --refPointLabel center -max 4 --legendLocation lower-left --regionsLabel Macs2_Peaks --samplesLabel MEF2D_v2 AAVS1 --plotTitle MEF2D_v2_vs_AAVS1', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270530v1\\t      2168\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270548v1\\t      1599\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270372v1\\t      1650\\nchrUn_KI270386v1\\t      1788\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270752v1\\t     27745\\nchrUn_KI270371v1\\t      2805\\nchrUn_KI270338v1\\t      1428\\nchrUn_KI270387v1\\t      1537\\nchrUn_KI270379v1\\t      1045\\n         chrEBV\\t    171823\\nchrUn_KI270383v1\\t      1750\\nchrUn_KI270395v1\\t      1143\\nchrUn_KI270388v1\\t      1216\\nchrUn_KI270378v1\\t      1048\\nchrUn_KI270384v1\\t      1658\\nchrUn_KI270364v1\\t      2855\\nchrUn_KI270373v1\\t      1451\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270392v1\\t       971\\nchrUn_KI270334v1\\t      1368\\nchrUn_KI270376v1\\t      1136\\nchrUn_KI270329v1\\t      1040\\nchrUn_KI270375v1\\t      2378\\nchrUn_KI270335v1\\t      1048\\nchrUn_KI270385v1\\t       990\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270381v1\\t      1930\\nchrUn_KI270340v1\\t      1428\\nchrUn_KI270394v1\\t       970\\n')\n",
      "MYB\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_MYB-H3K27AC_R1.bw ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/additional_degraded_v1/bwa/mergedLibrary/macs/broadPeak/MV411_RNP_AAVS1-H3K27AC_R1_peaks.broadPeak --missingDataAsZero --outFileName ../../data/additional_degraded_v1/MYB_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/additional_degraded_v1/MYB_mat.gz --outFileName ../../data/additional_degraded_v1/MYB_mat.pdf --refPointLabel center -max 4 --legendLocation lower-left --regionsLabel Macs2_Peaks --samplesLabel MYB AAVS1 --plotTitle MYB_vs_AAVS1', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270329v1\\t      1040\\nchrUn_KI270381v1\\t      1930\\nchrUn_KI270387v1\\t      1537\\nchrUn_KI270530v1\\t      2168\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270372v1\\t      1650\\nchrUn_KI270334v1\\t      1368\\nchrUn_KI270364v1\\t      2855\\nchrUn_KI270548v1\\t      1599\\nchrUn_KI270371v1\\t      2805\\nchrUn_KI270384v1\\t      1658\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270378v1\\t      1048\\nchrUn_KI270338v1\\t      1428\\nchrUn_KI270394v1\\t       970\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270379v1\\t      1045\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270373v1\\t      1451\\nchrUn_KI270383v1\\t      1750\\nchrUn_KI270335v1\\t      1048\\nchrUn_KI270392v1\\t       971\\n         chrEBV\\t    171823\\nchrUn_KI270752v1\\t     27745\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270340v1\\t      1428\\nchrUn_KI270375v1\\t      2378\\nchrUn_KI270385v1\\t       990\\nchrUn_KI270386v1\\t      1788\\nchrUn_KI270376v1\\t      1136\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270395v1\\t      1143\\nchrUn_KI270388v1\\t      1216\\n')\n",
      "MYB_v2\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_MYB-H3K27AC_R2.bw ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/additional_degraded_v1/bwa/mergedLibrary/macs/broadPeak/MV411_RNP_AAVS1-H3K27AC_R1_peaks.broadPeak --missingDataAsZero --outFileName ../../data/additional_degraded_v1/MYB_v2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/additional_degraded_v1/MYB_v2_mat.gz --outFileName ../../data/additional_degraded_v1/MYB_v2_mat.pdf --refPointLabel center -max 4 --legendLocation lower-left --regionsLabel Macs2_Peaks --samplesLabel MYB_v2 AAVS1 --plotTitle MYB_v2_vs_AAVS1', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270381v1\\t      1930\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270372v1\\t      1650\\nchrUn_KI270394v1\\t       970\\nchrUn_KI270340v1\\t      1428\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270395v1\\t      1143\\nchrUn_KI270378v1\\t      1048\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270376v1\\t      1136\\nchrUn_KI270329v1\\t      1040\\nchrUn_KI270338v1\\t      1428\\nchrUn_KI270334v1\\t      1368\\nchrUn_KI270387v1\\t      1537\\nchrUn_KI270386v1\\t      1788\\nchrUn_KI270530v1\\t      2168\\nchrUn_KI270383v1\\t      1750\\nchrUn_KI270392v1\\t       971\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270379v1\\t      1045\\nchrUn_KI270384v1\\t      1658\\nchrUn_KI270371v1\\t      2805\\nchrUn_KI270752v1\\t     27745\\nchrUn_KI270335v1\\t      1048\\n         chrEBV\\t    171823\\nchrUn_KI270364v1\\t      2855\\nchrUn_KI270375v1\\t      2378\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270548v1\\t      1599\\nchrUn_KI270388v1\\t      1216\\nchrUn_KI270385v1\\t       990\\nchrUn_KI270373v1\\t      1451\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270374v1\\t      2656\\n')\n",
      "RUNX1\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_RUNX1-H3K27AC_R1.bw ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/additional_degraded_v1/bwa/mergedLibrary/macs/broadPeak/MV411_RNP_AAVS1-H3K27AC_R1_peaks.broadPeak --missingDataAsZero --outFileName ../../data/additional_degraded_v1/RUNX1_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/additional_degraded_v1/RUNX1_mat.gz --outFileName ../../data/additional_degraded_v1/RUNX1_mat.pdf --refPointLabel center -max 4 --legendLocation lower-left --regionsLabel Macs2_Peaks --samplesLabel RUNX1 AAVS1 --plotTitle RUNX1_vs_AAVS1', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270548v1\\t      1599\\nchrUn_KI270388v1\\t      1216\\nchrUn_KI270372v1\\t      1650\\nchrUn_KI270329v1\\t      1040\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270383v1\\t      1750\\nchrUn_KI270752v1\\t     27745\\nchrUn_KI270334v1\\t      1368\\nchrUn_KI270364v1\\t      2855\\nchrUn_KI270384v1\\t      1658\\nchrUn_KI270335v1\\t      1048\\nchrUn_KI270340v1\\t      1428\\nchrUn_KI270376v1\\t      1136\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270381v1\\t      1930\\nchrUn_KI270371v1\\t      2805\\nchrUn_KI270375v1\\t      2378\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270378v1\\t      1048\\n         chrEBV\\t    171823\\nchrUn_KI270385v1\\t       990\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270387v1\\t      1537\\nchrUn_KI270386v1\\t      1788\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270394v1\\t       970\\nchrUn_KI270395v1\\t      1143\\nchrUn_KI270373v1\\t      1451\\nchrUn_KI270530v1\\t      2168\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270379v1\\t      1045\\nchrUn_KI270338v1\\t      1428\\nchrUn_KI270392v1\\t       971\\n')\n",
      "RUNX1_v2\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_RUNX1-H3K27AC_R2.bw ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/additional_degraded_v1/bwa/mergedLibrary/macs/broadPeak/MV411_RNP_AAVS1-H3K27AC_R1_peaks.broadPeak --missingDataAsZero --outFileName ../../data/additional_degraded_v1/RUNX1_v2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/additional_degraded_v1/RUNX1_v2_mat.gz --outFileName ../../data/additional_degraded_v1/RUNX1_v2_mat.pdf --refPointLabel center -max 4 --legendLocation lower-left --regionsLabel Macs2_Peaks --samplesLabel RUNX1_v2 AAVS1 --plotTitle RUNX1_v2_vs_AAVS1', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270387v1\\t      1537\\nchrUn_KI270548v1\\t      1599\\nchrUn_KI270378v1\\t      1048\\nchrUn_KI270373v1\\t      1451\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270383v1\\t      1750\\nchrUn_KI270394v1\\t       970\\n         chrEBV\\t    171823\\nchrUn_KI270395v1\\t      1143\\nchrUn_KI270335v1\\t      1048\\nchrUn_KI270329v1\\t      1040\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270334v1\\t      1368\\nchrUn_KI270376v1\\t      1136\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270381v1\\t      1930\\nchrUn_KI270392v1\\t       971\\nchrUn_KI270372v1\\t      1650\\nchrUn_KI270364v1\\t      2855\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270340v1\\t      1428\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270385v1\\t       990\\nchrUn_KI270752v1\\t     27745\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270375v1\\t      2378\\nchrUn_KI270530v1\\t      2168\\nchrUn_KI270379v1\\t      1045\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270386v1\\t      1788\\nchrUn_KI270384v1\\t      1658\\nchrUn_KI270388v1\\t      1216\\nchrUn_KI270371v1\\t      2805\\nchrUn_KI270338v1\\t      1428\\n')\n",
      "RUNX1_RUNX2\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_RUNX1_RUNX2-H3K27AC_R1.bw ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/additional_degraded_v1/bwa/mergedLibrary/macs/broadPeak/MV411_RNP_AAVS1-H3K27AC_R1_peaks.broadPeak --missingDataAsZero --outFileName ../../data/additional_degraded_v1/RUNX1_RUNX2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/additional_degraded_v1/RUNX1_RUNX2_mat.gz --outFileName ../../data/additional_degraded_v1/RUNX1_RUNX2_mat.pdf --refPointLabel center -max 4 --legendLocation lower-left --regionsLabel Macs2_Peaks --samplesLabel RUNX1_RUNX2 AAVS1 --plotTitle RUNX1_RUNX2_vs_AAVS1', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270334v1\\t      1368\\nchrUn_KI270364v1\\t      2855\\nchrUn_KI270335v1\\t      1048\\nchrUn_KI270340v1\\t      1428\\nchrUn_KI270329v1\\t      1040\\nchrUn_KI270375v1\\t      2378\\nchrUn_KI270373v1\\t      1451\\nchrUn_KI270386v1\\t      1788\\nchrUn_KI270376v1\\t      1136\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270752v1\\t     27745\\nchrUn_KI270394v1\\t       970\\nchrUn_KI270384v1\\t      1658\\nchrUn_KI270379v1\\t      1045\\nchrUn_KI270378v1\\t      1048\\nchrUn_KI270392v1\\t       971\\nchrUn_KI270338v1\\t      1428\\nchrUn_KI270530v1\\t      2168\\n         chrEBV\\t    171823\\nchrUn_KI270548v1\\t      1599\\nchrUn_KI270371v1\\t      2805\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270372v1\\t      1650\\nchrUn_KI270388v1\\t      1216\\nchrUn_KI270385v1\\t       990\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270395v1\\t      1143\\nchrUn_KI270383v1\\t      1750\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270381v1\\t      1930\\nchrUn_KI270387v1\\t      1537\\nchrUn_KI270389v1\\t      1298\\n')\n",
      "RUNX1_RUNX2_v2\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_RUNX1_RUNX2-H3K27AC_R2.bw ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/additional_degraded_v1/bwa/mergedLibrary/macs/broadPeak/MV411_RNP_AAVS1-H3K27AC_R1_peaks.broadPeak --missingDataAsZero --outFileName ../../data/additional_degraded_v1/RUNX1_RUNX2_v2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/additional_degraded_v1/RUNX1_RUNX2_v2_mat.gz --outFileName ../../data/additional_degraded_v1/RUNX1_RUNX2_v2_mat.pdf --refPointLabel center -max 4 --legendLocation lower-left --regionsLabel Macs2_Peaks --samplesLabel RUNX1_RUNX2_v2 AAVS1 --plotTitle RUNX1_RUNX2_v2_vs_AAVS1', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270381v1\\t      1930\\nchrUn_KI270375v1\\t      2378\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270372v1\\t      1650\\nchrUn_KI270752v1\\t     27745\\nchrUn_KI270329v1\\t      1040\\nchrUn_KI270385v1\\t       990\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270395v1\\t      1143\\nchrUn_KI270548v1\\t      1599\\nchrUn_KI270378v1\\t      1048\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270384v1\\t      1658\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270373v1\\t      1451\\nchrUn_KI270394v1\\t       970\\nchrUn_KI270340v1\\t      1428\\nchrUn_KI270371v1\\t      2805\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270334v1\\t      1368\\nchrUn_KI270530v1\\t      2168\\n         chrEBV\\t    171823\\nchrUn_KI270338v1\\t      1428\\nchrUn_KI270376v1\\t      1136\\nchrUn_KI270386v1\\t      1788\\nchrUn_KI270387v1\\t      1537\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270392v1\\t       971\\nchrUn_KI270383v1\\t      1750\\nchrUn_KI270379v1\\t      1045\\nchrUn_KI270335v1\\t      1048\\nchrUn_KI270364v1\\t      2855\\nchrUn_KI270388v1\\t      1216\\nchrUn_KI270374v1\\t      2656\\n')\n",
      "RUNX2\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_RUNX2-H3K27AC_R1.bw ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/additional_degraded_v1/bwa/mergedLibrary/macs/broadPeak/MV411_RNP_AAVS1-H3K27AC_R1_peaks.broadPeak --missingDataAsZero --outFileName ../../data/additional_degraded_v1/RUNX2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/additional_degraded_v1/RUNX2_mat.gz --outFileName ../../data/additional_degraded_v1/RUNX2_mat.pdf --refPointLabel center -max 4 --legendLocation lower-left --regionsLabel Macs2_Peaks --samplesLabel RUNX2 AAVS1 --plotTitle RUNX2_vs_AAVS1', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270395v1\\t      1143\\nchrUn_KI270394v1\\t       970\\n         chrEBV\\t    171823\\nchrUn_KI270384v1\\t      1658\\nchrUn_KI270376v1\\t      1136\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270338v1\\t      1428\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270530v1\\t      2168\\nchrUn_KI270383v1\\t      1750\\nchrUn_KI270375v1\\t      2378\\nchrUn_KI270548v1\\t      1599\\nchrUn_KI270364v1\\t      2855\\nchrUn_KI270334v1\\t      1368\\nchrUn_KI270387v1\\t      1537\\nchrUn_KI270371v1\\t      2805\\nchrUn_KI270335v1\\t      1048\\nchrUn_KI270372v1\\t      1650\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270329v1\\t      1040\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270373v1\\t      1451\\nchrUn_KI270752v1\\t     27745\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270340v1\\t      1428\\nchrUn_KI270392v1\\t       971\\nchrUn_KI270386v1\\t      1788\\nchrUn_KI270388v1\\t      1216\\nchrUn_KI270385v1\\t       990\\nchrUn_KI270381v1\\t      1930\\nchrUn_KI270378v1\\t      1048\\nchrUn_KI270379v1\\t      1045\\nchrUn_KI270393v1\\t      1308\\n')\n",
      "RUNX2_v2\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_RUNX2-H3K27AC_R2.bw ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/additional_degraded_v1/bwa/mergedLibrary/macs/broadPeak/MV411_RNP_AAVS1-H3K27AC_R1_peaks.broadPeak --missingDataAsZero --outFileName ../../data/additional_degraded_v1/RUNX2_v2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/additional_degraded_v1/RUNX2_v2_mat.gz --outFileName ../../data/additional_degraded_v1/RUNX2_v2_mat.pdf --refPointLabel center -max 4 --legendLocation lower-left --regionsLabel Macs2_Peaks --samplesLabel RUNX2_v2 AAVS1 --plotTitle RUNX2_v2_vs_AAVS1', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270387v1\\t      1537\\nchrUn_KI270385v1\\t       990\\nchrUn_KI270375v1\\t      2378\\nchrUn_KI270329v1\\t      1040\\nchrUn_KI270548v1\\t      1599\\nchrUn_KI270372v1\\t      1650\\nchrUn_KI270340v1\\t      1428\\nchrUn_KI270335v1\\t      1048\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270383v1\\t      1750\\nchrUn_KI270386v1\\t      1788\\nchrUn_KI270376v1\\t      1136\\nchrUn_KI270388v1\\t      1216\\n         chrEBV\\t    171823\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270334v1\\t      1368\\nchrUn_KI270530v1\\t      2168\\nchrUn_KI270381v1\\t      1930\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270384v1\\t      1658\\nchrUn_KI270338v1\\t      1428\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270395v1\\t      1143\\nchrUn_KI270378v1\\t      1048\\nchrUn_KI270373v1\\t      1451\\nchrUn_KI270371v1\\t      2805\\nchrUn_KI270394v1\\t       970\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270364v1\\t      2855\\nchrUn_KI270392v1\\t       971\\nchrUn_KI270379v1\\t      1045\\nchrUn_KI270752v1\\t     27745\\n')\n",
      "SPI1\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_SPI1-H3K27AC_R1.bw ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/additional_degraded_v1/bwa/mergedLibrary/macs/broadPeak/MV411_RNP_AAVS1-H3K27AC_R1_peaks.broadPeak --missingDataAsZero --outFileName ../../data/additional_degraded_v1/SPI1_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/additional_degraded_v1/SPI1_mat.gz --outFileName ../../data/additional_degraded_v1/SPI1_mat.pdf --refPointLabel center -max 4 --legendLocation lower-left --regionsLabel Macs2_Peaks --samplesLabel SPI1 AAVS1 --plotTitle SPI1_vs_AAVS1', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270383v1\\t      1750\\nchrUn_KI270548v1\\t      1599\\nchrUn_KI270395v1\\t      1143\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270379v1\\t      1045\\nchrUn_KI270373v1\\t      1451\\nchrUn_KI270394v1\\t       970\\nchrUn_KI270388v1\\t      1216\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270378v1\\t      1048\\nchrUn_KI270412v1\\t      1179\\n         chrEBV\\t    171823\\nchrUn_KI270381v1\\t      1930\\nchrUn_KI270752v1\\t     27745\\nchrUn_KI270387v1\\t      1537\\nchrUn_KI270338v1\\t      1428\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270530v1\\t      2168\\nchrUn_KI270375v1\\t      2378\\nchrUn_KI270340v1\\t      1428\\nchrUn_KI270392v1\\t       971\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270385v1\\t       990\\nchrUn_KI270364v1\\t      2855\\nchrUn_KI270334v1\\t      1368\\nchrUn_KI270335v1\\t      1048\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270376v1\\t      1136\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270329v1\\t      1040\\nchrUn_KI270384v1\\t      1658\\nchrUn_KI270372v1\\t      1650\\nchrUn_KI270371v1\\t      2805\\nchrUn_KI270386v1\\t      1788\\n')\n",
      "SPI1_v2\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_SPI1-H3K27AC_R2.bw ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/additional_degraded_v1/bwa/mergedLibrary/macs/broadPeak/MV411_RNP_AAVS1-H3K27AC_R1_peaks.broadPeak --missingDataAsZero --outFileName ../../data/additional_degraded_v1/SPI1_v2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/additional_degraded_v1/SPI1_v2_mat.gz --outFileName ../../data/additional_degraded_v1/SPI1_v2_mat.pdf --refPointLabel center -max 4 --legendLocation lower-left --regionsLabel Macs2_Peaks --samplesLabel SPI1_v2 AAVS1 --plotTitle SPI1_v2_vs_AAVS1', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270752v1\\t     27745\\nchrUn_KI270378v1\\t      1048\\nchrUn_KI270338v1\\t      1428\\nchrUn_KI270384v1\\t      1658\\nchrUn_KI270372v1\\t      1650\\nchrUn_KI270373v1\\t      1451\\nchrUn_KI270548v1\\t      1599\\nchrUn_KI270381v1\\t      1930\\nchrUn_KI270383v1\\t      1750\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270385v1\\t       990\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270379v1\\t      1045\\nchrUn_KI270371v1\\t      2805\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270392v1\\t       971\\n         chrEBV\\t    171823\\nchrUn_KI270386v1\\t      1788\\nchrUn_KI270340v1\\t      1428\\nchrUn_KI270329v1\\t      1040\\nchrUn_KI270388v1\\t      1216\\nchrUn_KI270375v1\\t      2378\\nchrUn_KI270394v1\\t       970\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270335v1\\t      1048\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270364v1\\t      2855\\nchrUn_KI270387v1\\t      1537\\nchrUn_KI270376v1\\t      1136\\nchrUn_KI270334v1\\t      1368\\nchrUn_KI270530v1\\t      2168\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270395v1\\t      1143\\n')\n"
     ]
    }
   ],
   "source": [
    "for i, val in enumerate(bw):\n",
    "    if i <2:\n",
    "        continue\n",
    "    name = names[i]\n",
    "    print(name)\n",
    "    chip.getPeaksAt(peaks[0], bigwigs = [val,bw[0]], bigwignames= [name,\"AAVS1\"],peaknames=['Macs2_Peaks'], window=3000, folder=\"\", title=name+\"_vs_AAVS1\", numthreads=8, refpoint=\"center\", name='../../data/'+project+'/'+name+'_mat.pdf', withDeeptools=True, torecompute=True, vmax=4, legendLoc=\"lower-left\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 247,
   "metadata": {},
   "outputs": [],
   "source": [
    "! cp ../../data/additional_degraded_v1/*.pdf ../results/$project/plots/scaled/heatmaps/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 190,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_IRF8-H3K27AC_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_MEF2D-H3K27AC_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_MYB-H3K27AC_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_RUNX1-H3K27AC_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_RUNX1_RUNX2-H3K27AC_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_RUNX2-H3K27AC_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_SPI1-H3K27AC_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_IRF8-H3K27AC_R2_treat_pileup_c3.0_cond1.bed',\n",
       " '../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_MEF2D-H3K27AC_R2_treat_pileup_c3.0_cond1.bed',\n",
       " '../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_MYB-H3K27AC_R2_treat_pileup_c3.0_cond1.bed',\n",
       " '../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_RUNX1-H3K27AC_R2_treat_pileup_c3.0_cond1.bed',\n",
       " '../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_RUNX1_RUNX2-H3K27AC_R2_treat_pileup_c3.0_cond1.bed',\n",
       " '../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_RUNX2-H3K27AC_R2_treat_pileup_c3.0_cond1.bed',\n",
       " '../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_SPI1-H3K27AC_R2_treat_pileup_c3.0_cond1.bed']"
      ]
     },
     "execution_count": 190,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "cond1peak = ! ls ../../data/$project/diffPeaks/*cond1.bed\n",
    "cond2peak = ! ls ../../data/$project/diffPeaks/*cond2.bed\n",
    "commonpeak = ! ls ../../data/$project/diffPeaks/*common.bed\n",
    "cond1peak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 234,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"IRF8\", \"MEF2D\", \"MYB\",\"RUNX1\",\"RUNX1_RUNX2\",\"RUNX2\", \"SPI1\", \"IRF8_v2\", \"MEF2D_v2\", \"MYB_v2\", \"RUNX1_v2\", \"RUNX1_RUNX2_v2\", \"RUNX2_v2\", \"SPI1_v2\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 222,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_AAVS1-H3K27AC_R1.mLb.clN.bigWig',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bigWig',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_IRF8-H3K27AC_R1.bw',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_IRF8-H3K27AC_R2.bw',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_MEF2D-H3K27AC_R1.bw',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_MEF2D-H3K27AC_R2.bw',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_MYB-H3K27AC_R1.bw',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_MYB-H3K27AC_R2.bw',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_RUNX1-H3K27AC_R1.bw',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_RUNX1-H3K27AC_R2.bw',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_RUNX1_RUNX2-H3K27AC_R1.bw',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_RUNX1_RUNX2-H3K27AC_R2.bw',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_RUNX2-H3K27AC_R1.bw',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_RUNX2-H3K27AC_R2.bw',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_SPI1-H3K27AC_R1.bw',\n",
       " '../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_SPI1-H3K27AC_R2.bw']"
      ]
     },
     "execution_count": 222,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 216,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_SPI1-H3K27AC_R2.bw'"
      ]
     },
     "execution_count": 216,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bw[(6*2)+3]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 218,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "7"
      ]
     },
     "execution_count": 218,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "int(len(bw[2:])/2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 219,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bigWig'"
      ]
     },
     "execution_count": 219,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bw[round(7/6)]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 238,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bigWig ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_MEF2D-H3K27AC_R2.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bigWig ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_MEF2D-H3K27AC_R2.bw  --referencePoint center --regionsFileName ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_MEF2D-H3K27AC_R2_treat_pileup_c3.0_cond1.bed ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_MEF2D-H3K27AC_R2_treat_pileup_c3.0_common.bed ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_MEF2D-H3K27AC_R2_treat_pileup_c3.0_cond2.bed --missingDataAsZero --outFileName ../../data/additional_degraded_v1/diffPeaks/MEF2D_v2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/additional_degraded_v1/diffPeaks/MEF2D_v2_mat.gz --outFileName ../../data/additional_degraded_v1/diffPeaks/MEF2D_v2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks --samplesLabel DMSO VHL --plotTitle MEF2D_v2', returncode=0, stdout=b'', stderr=b\"\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270376v1\\t      1136\\nchrUn_KI270414v1\\t      2489\\nchrUn_KI270386v1\\t      1788\\nchrUn_KI270379v1\\t      1045\\nchrUn_KI270387v1\\t      1537\\nchrUn_KI270316v1\\t      1444\\nchrUn_KI270394v1\\t       970\\nchrUn_KI270381v1\\t      1930\\nchrUn_KI270373v1\\t      1451\\nchrUn_KI270338v1\\t      1428\\nchrUn_KI270340v1\\t      1428\\nchrUn_KI270579v1\\t     31033\\nchrUn_KI270335v1\\t      1048\\nchrUn_KI270364v1\\t      2855\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270425v1\\t      1884\\nchrUn_KI270548v1\\t      1599\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270383v1\\t      1750\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270371v1\\t      2805\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270334v1\\t      1368\\nchrUn_KI270375v1\\t      2378\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270581v1\\t      7046\\nchrUn_KI270372v1\\t      1650\\n         chrEBV\\t    171823\\nWARNING: Group 'MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_MEF2D-H3K27AC_R2_treat_pileup_c3.0_cond2.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n",
      "../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bigWig ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_MYB-H3K27AC_R2.bw\n",
      "CompletedProcess(args='plotHeatmap --matrixFile ../../data/additional_degraded_v1/diffPeaks/MYB_v2_mat.gz --outFileName ../../data/additional_degraded_v1/diffPeaks/MYB_v2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks --samplesLabel DMSO VHL --plotTitle MYB_v2', returncode=0, stdout=b'', stderr=b'')\n",
      "../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bigWig ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_RUNX1-H3K27AC_R2.bw\n",
      "CompletedProcess(args='plotHeatmap --matrixFile ../../data/additional_degraded_v1/diffPeaks/RUNX1_v2_mat.gz --outFileName ../../data/additional_degraded_v1/diffPeaks/RUNX1_v2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks --samplesLabel DMSO VHL --plotTitle RUNX1_v2', returncode=0, stdout=b'', stderr=b'')\n",
      "../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bigWig ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_RUNX1_RUNX2-H3K27AC_R2.bw\n",
      "CompletedProcess(args='plotHeatmap --matrixFile ../../data/additional_degraded_v1/diffPeaks/RUNX1_RUNX2_v2_mat.gz --outFileName ../../data/additional_degraded_v1/diffPeaks/RUNX1_RUNX2_v2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks --samplesLabel DMSO VHL --plotTitle RUNX1_RUNX2_v2', returncode=0, stdout=b'', stderr=b'')\n",
      "../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bigWig ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_RUNX2-H3K27AC_R2.bw\n",
      "CompletedProcess(args='plotHeatmap --matrixFile ../../data/additional_degraded_v1/diffPeaks/RUNX2_v2_mat.gz --outFileName ../../data/additional_degraded_v1/diffPeaks/RUNX2_v2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks --samplesLabel DMSO VHL --plotTitle RUNX2_v2', returncode=0, stdout=b'', stderr=b\"WARNING: Group 'MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_RUNX2-H3K27AC_R2_treat_pileup_c3.0_cond2.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n",
      "../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bigWig ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_SPI1-H3K27AC_R2.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_AAVS1-H3K27AC_R2.mLb.clN.bigWig ../../data/additional_degraded_v1/recalib_bigwig/MV411_RNP_SPI1-H3K27AC_R2.bw  --referencePoint center --regionsFileName ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_SPI1-H3K27AC_R2_treat_pileup_c3.0_cond1.bed ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_SPI1-H3K27AC_R2_treat_pileup_c3.0_common.bed ../../data/additional_degraded_v1/diffPeaks/MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_SPI1-H3K27AC_R2_treat_pileup_c3.0_cond2.bed --missingDataAsZero --outFileName ../../data/additional_degraded_v1/diffPeaks/SPI1_v2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/additional_degraded_v1/diffPeaks/SPI1_v2_mat.gz --outFileName ../../data/additional_degraded_v1/diffPeaks/SPI1_v2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks --samplesLabel DMSO VHL --plotTitle SPI1_v2', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270383v1\\t      1750\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270376v1\\t      1136\\nchrUn_KI270373v1\\t      1451\\nchrUn_KI270338v1\\t      1428\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270581v1\\t      7046\\nchrUn_KI270375v1\\t      2378\\nchrUn_KI270372v1\\t      1650\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270340v1\\t      1428\\n         chrEBV\\t    171823\\nchrUn_KI270387v1\\t      1537\\nchrUn_KI270364v1\\t      2855\\nchrUn_KI270334v1\\t      1368\\nchrUn_KI270381v1\\t      1930\\nchrUn_KI270386v1\\t      1788\\nchrUn_KI270379v1\\t      1045\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270394v1\\t       970\\nchrUn_KI270425v1\\t      1884\\nchrUn_KI270414v1\\t      2489\\nchrUn_KI270316v1\\t      1444\\nchrUn_KI270371v1\\t      2805\\nchrUn_KI270548v1\\t      1599\\nchrUn_KI270335v1\\t      1048\\nchrUn_KI270579v1\\t     31033\\n')\n"
     ]
    }
   ],
   "source": [
    "for i in range(int(len(bw[2:]))):\n",
    "    if i <8:\n",
    "        continue\n",
    "    cl = len(bw[2:])\n",
    "    name1 = bw[round(i/(cl-1))]\n",
    "    name2 = bw[((i%int(cl/2))*2)+2+round(i/(cl-1))]\n",
    "    peak = [cond1peak[i], commonpeak[i], cond2peak[i]]\n",
    "   # for val in peak:\n",
    "      #  chip.dropWeirdChromosomes(val)\n",
    "    name = names[i]\n",
    "    print(name1,name2)\n",
    "    chip.getPeaksAt(peak, [name1, name2], bigwignames=['DMSO', 'VHL'], peaknames=['DMSO_peaks', 'common', 'VHL_peaks'], window=3000, folder=\"\", title=name, numthreads=7, refpoint=\"center\", name='../../data/'+project+'/diffPeaks/'+name+'_mat.pdf', withDeeptools=True, torecompute=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 139,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_IRF8-H3K27AC_R1_treat_pileup_c3.0_common.bed\r\n",
      "MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_IRF8-H3K27AC_R1_treat_pileup_c3.0_cond1.bed\r\n",
      "MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_IRF8-H3K27AC_R1_treat_pileup_c3.0_cond2.bed\r\n",
      "MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_MEF2D-H3K27AC_R1_treat_pileup_c3.0_common.bed\r\n",
      "MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_MEF2D-H3K27AC_R1_treat_pileup_c3.0_cond1.bed\r\n",
      "MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_MEF2D-H3K27AC_R1_treat_pileup_c3.0_cond2.bed\r\n",
      "MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_MYB-H3K27AC_R1_treat_pileup_c3.0_common.bed\r\n",
      "MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_MYB-H3K27AC_R1_treat_pileup_c3.0_cond1.bed\r\n",
      "MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_MYB-H3K27AC_R1_treat_pileup_c3.0_cond2.bed\r\n",
      "MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_RUNX1-H3K27AC_R1_treat_pileup_c3.0_common.bed\r\n",
      "MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_RUNX1-H3K27AC_R1_treat_pileup_c3.0_cond1.bed\r\n",
      "MV411_RNP_AAVS1-H3K27AC_R1_treat_pileup_vs_MV411_RNP_RUNX1-H3K27AC_R1_treat_pileup_c3.0_cond2.bed\r\n",
      "MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_IRF8-H3K27AC_R2_treat_pileup_c3.0_common.bed\r\n",
      "MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_IRF8-H3K27AC_R2_treat_pileup_c3.0_cond1.bed\r\n",
      "MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_IRF8-H3K27AC_R2_treat_pileup_c3.0_cond2.bed\r\n",
      "MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_MEF2D-H3K27AC_R2_treat_pileup_c3.0_common.bed\r\n",
      "MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_MEF2D-H3K27AC_R2_treat_pileup_c3.0_cond1.bed\r\n",
      "MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_MEF2D-H3K27AC_R2_treat_pileup_c3.0_cond2.bed\r\n",
      "MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_MYB-H3K27AC_R2_treat_pileup_c3.0_common.bed\r\n",
      "MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_MYB-H3K27AC_R2_treat_pileup_c3.0_cond1.bed\r\n",
      "MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_MYB-H3K27AC_R2_treat_pileup_c3.0_cond2.bed\r\n",
      "MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_RUNX1-H3K27AC_R2_treat_pileup_c3.0_common.bed\r\n",
      "MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_RUNX1-H3K27AC_R2_treat_pileup_c3.0_cond1.bed\r\n",
      "MV411_RNP_AAVS1-H3K27AC_R2_treat_pileup_vs_MV411_RNP_RUNX1-H3K27AC_R2_treat_pileup_c3.0_cond2.bed\r\n"
     ]
    }
   ],
   "source": [
    "! mkdir ../results/additional_degraded_v1/plots/scaled/diffPeaks/\n",
    "! cp ../../data/additional_degraded_v1/diffPeaks/*.pdf ../results/additional_degraded_v1/plots/scaled/diffPeaks/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### on unscalled data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bams = !ls ../../data/$project/bwa/mergedLibrary/mp*.bam\n",
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mkdir ../../data/$project/diffPeaks_unscaled"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#on unscalled data \n",
    "for i in range(int(len(bams)/2)):\n",
    "    if i < 5:\n",
    "        continue\n",
    "    name1 = bams[i]\n",
    "    name2 = bams[6+i]\n",
    "    print(name1,name2)\n",
    "    chip.fullDiffPeak(name1,name2, control1='../../data/'+project+'/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam', directory = \"../../data/\"+project+\"/diffData_unscaled/\", res_directory = \"../../data/\"+project+\"/diffPeaks_unscaled/\",pairedend=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bw = ! ls ../../data/$project/bwa/mergedLibrary/bigwig/*.bigWig\n",
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "cond1peak = ! ls ../../data/$project/diffPeaks_unscaled/*cond1.bed\n",
    "cond2peak = ! ls ../../data/$project/diffPeaks_unscaled/*cond2.bed\n",
    "commonpeak = ! ls ../../data/$project/diffPeaks_unscaled/*common.bed\n",
    "commonpeak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"MED1\",\"MED1_v2\",\"FLAG_IRF2BP2\",\"POLII_total\",\"POLII_S2\",\"POLII_S5\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for i in range(int((len(bw)-1)/2)):\n",
    "    name1 = bw[1+i]\n",
    "    name2 = bw[7+i]\n",
    "    peak = [cond1peak[i], commonpeak[i], cond2peak[i]]\n",
    "    #for val in peak:\n",
    "     #   chip.dropWeirdChromosomes(val)\n",
    "    name = names[i]\n",
    "    print(name1,name2)\n",
    "    chip.getPeaksAt(peak, [name1, name2], bigwignames=['DMSO', 'VHL'], peaknames=['DMSO_peaks', 'common', 'VHL_peaks'], window=3000, folder=\"\", title=name, numthreads=7, torecompute=True, refpoint='center', name='../../data/'+project+'/diffPeaks_unscaled/'+name+'_mat.pdf', withDeeptools=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! gsutil -m cp -r \"../../data/additional_degraded_v1\" gs://amlproject/Chip/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! cp ../../$project/*_mat.pdf ../results/$project/unscaled/\n",
    "! cp ../../$project/*_mat.pdf ../results/$project/scaled/\n",
    "\n",
    "! cp ../../$project/*_mat.pdf ../results/$project/unscaled/\n",
    "! cp ../../$project/*_mat.pdf ../results/$project/unscaled/\n",
    "\n",
    "! cp -r ../data/$project/bwa/mergedLibrary/deepTools/plot/* ../results/$project/plots"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## v6"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "project=\"chipseq_MEF2D_degraded\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 250,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gs://transfer-amlproject/201001_MP8262_fastq/20201001_FLAG_DMSO_MP8262_S93_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/201001_MP8262_fastq/20201001_FLAG_DMSO_MP8262_S93_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/201001_MP8262_fastq/20201001_FLAG_VHL_MP8262_S99_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/201001_MP8262_fastq/20201001_FLAG_VHL_MP8262_S99_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/201001_MP8262_fastq/20201001_IRF8_DMSO_MP8262_S88_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/201001_MP8262_fastq/20201001_IRF8_DMSO_MP8262_S88_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/201001_MP8262_fastq/20201001_IRF8_VHL_MP8262_S94_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/201001_MP8262_fastq/20201001_IRF8_VHL_MP8262_S94_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/201001_MP8262_fastq/20201001_MED1_DMSO_MP8262_S91_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/201001_MP8262_fastq/20201001_MED1_DMSO_MP8262_S91_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/201001_MP8262_fastq/20201001_MED1_VHL_MP8262_S97_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/201001_MP8262_fastq/20201001_MED1_VHL_MP8262_S97_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/201001_MP8262_fastq/20201001_MEF2C_DMSO_MP8262_S90_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/201001_MP8262_fastq/20201001_MEF2C_DMSO_MP8262_S90_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/201001_MP8262_fastq/20201001_MEF2C_VHL_MP8262_S96_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/201001_MP8262_fastq/20201001_MEF2C_VHL_MP8262_S96_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/201001_MP8262_fastq/20201001_MYC_DMSO_MP8262_S89_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/201001_MP8262_fastq/20201001_MYC_DMSO_MP8262_S89_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/201001_MP8262_fastq/20201001_MYC_VHL_MP8262_S95_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/201001_MP8262_fastq/20201001_MYC_VHL_MP8262_S95_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/201001_MP8262_fastq/20201001_POL_II_Total_DMSO_MP8262_S92_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/201001_MP8262_fastq/20201001_POL_II_Total_DMSO_MP8262_S92_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/201001_MP8262_fastq/20201001_POL_II_Total_VHL_MP8262_S98_R1_001.fastq.gz\r\n",
      "gs://transfer-amlproject/201001_MP8262_fastq/20201001_POL_II_Total_VHL_MP8262_S98_R2_001.fastq.gz\r\n",
      "gs://transfer-amlproject/201001_MP8262_fastq/multiqc_report.html\r\n",
      "gs://transfer-amlproject/201001_MP8262_fastq/Reports/\r\n",
      "gs://transfer-amlproject/201001_MP8262_fastq/multiqc_data/\r\n"
     ]
    }
   ],
   "source": [
    "! gsutil ls gs://transfer-amlproject/201001_MP8262_fastq/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 252,
   "metadata": {},
   "outputs": [],
   "source": [
    "mkdir ../data/$project/ && mkdir ../data/$project/qc/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 253,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/multiqc_report.html...\n",
      "/ [1/1 files][  1.4 MiB/  1.4 MiB] 100% Done                                    \n",
      "Operation completed over 1 objects/1.4 MiB.                                      \n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_FLAG_DMSO_MP8262_S93_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_FLAG_DMSO_MP8262_S93_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_FLAG_DMSO_MP8262_S93_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_FLAG_DMSO_MP8262_S93_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_FLAG_VHL_MP8262_S99_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_FLAG_VHL_MP8262_S99_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_FLAG_VHL_MP8262_S99_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_IRF8_DMSO_MP8262_S88_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_IRF8_DMSO_MP8262_S88_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_IRF8_VHL_MP8262_S94_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_FLAG_VHL_MP8262_S99_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_IRF8_DMSO_MP8262_S88_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_IRF8_DMSO_MP8262_S88_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_IRF8_VHL_MP8262_S94_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_IRF8_VHL_MP8262_S94_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_IRF8_VHL_MP8262_S94_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_MED1_DMSO_MP8262_S91_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_MED1_DMSO_MP8262_S91_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_MED1_DMSO_MP8262_S91_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_MED1_DMSO_MP8262_S91_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_MED1_VHL_MP8262_S97_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_MED1_VHL_MP8262_S97_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_MED1_VHL_MP8262_S97_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_MED1_VHL_MP8262_S97_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_MEF2C_DMSO_MP8262_S90_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_MEF2C_DMSO_MP8262_S90_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_MEF2C_DMSO_MP8262_S90_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_MEF2C_DMSO_MP8262_S90_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_MEF2C_VHL_MP8262_S96_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_MEF2C_VHL_MP8262_S96_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_MEF2C_VHL_MP8262_S96_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_MEF2C_VHL_MP8262_S96_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_MYC_DMSO_MP8262_S89_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_MYC_DMSO_MP8262_S89_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_MYC_DMSO_MP8262_S89_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_MYC_DMSO_MP8262_S89_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_MYC_VHL_MP8262_S95_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_MYC_VHL_MP8262_S95_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_MYC_VHL_MP8262_S95_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_MYC_VHL_MP8262_S95_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_POL_II_Total_DMSO_MP8262_S92_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_POL_II_Total_DMSO_MP8262_S92_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_POL_II_Total_DMSO_MP8262_S92_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_POL_II_Total_DMSO_MP8262_S92_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_POL_II_Total_VHL_MP8262_S98_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_POL_II_Total_VHL_MP8262_S98_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_POL_II_Total_VHL_MP8262_S98_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/Reports/20201001_POL_II_Total_VHL_MP8262_S98_R2_001_fastqc.zip...\n",
      "- [48/48 files][ 25.3 MiB/ 25.3 MiB] 100% Done                                  \n",
      "Operation completed over 48 objects/25.3 MiB.                                    \n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/multiqc_data/multiqc.log...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/multiqc_data/multiqc_data.json...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/multiqc_data/multiqc_fastqc.txt...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/multiqc_data/multiqc_general_stats.txt...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/multiqc_data/multiqc_qualimap_bamqc_genome_results.txt...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/multiqc_data/multiqc_sources.txt...\n",
      "/ [6/6 files][  1.4 MiB/  1.4 MiB] 100% Done                                    \n",
      "Operation completed over 6 objects/1.4 MiB.                                      \n"
     ]
    }
   ],
   "source": [
    "!gsutil -m cp gs://transfer-amlproject/201001_MP8262_fastq/multiqc_report.html ../data/$project/qc/\n",
    "!gsutil -m cp -r gs://transfer-amlproject/201001_MP8262_fastq/Reports/ ../data/$project/qc/\n",
    "!gsutil -m cp -r gs://transfer-amlproject/201001_MP8262_fastq/multiqc_data/ ../data/$project/qc/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 254,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Omitting prefix \"gs://transfer-amlproject/201001_MP8262_fastq/Reports/\". (Did you mean to do cp -r?)\n",
      "Omitting prefix \"gs://transfer-amlproject/201001_MP8262_fastq/multiqc_data/\". (Did you mean to do cp -r?)\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/20201001_FLAG_DMSO_MP8262_S93_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/20201001_FLAG_DMSO_MP8262_S93_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/20201001_FLAG_VHL_MP8262_S99_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/20201001_FLAG_VHL_MP8262_S99_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/20201001_IRF8_DMSO_MP8262_S88_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/20201001_IRF8_VHL_MP8262_S94_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/20201001_IRF8_DMSO_MP8262_S88_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/20201001_IRF8_VHL_MP8262_S94_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/20201001_MED1_DMSO_MP8262_S91_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/20201001_MED1_DMSO_MP8262_S91_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/20201001_MED1_VHL_MP8262_S97_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/20201001_MED1_VHL_MP8262_S97_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/20201001_MEF2C_DMSO_MP8262_S90_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/20201001_MEF2C_DMSO_MP8262_S90_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/20201001_MEF2C_VHL_MP8262_S96_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/20201001_MEF2C_VHL_MP8262_S96_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/20201001_MYC_DMSO_MP8262_S89_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/20201001_MYC_DMSO_MP8262_S89_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/20201001_MYC_VHL_MP8262_S95_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/20201001_MYC_VHL_MP8262_S95_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/20201001_POL_II_Total_DMSO_MP8262_S92_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/20201001_POL_II_Total_DMSO_MP8262_S92_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/20201001_POL_II_Total_VHL_MP8262_S98_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/20201001_POL_II_Total_VHL_MP8262_S98_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/201001_MP8262_fastq/multiqc_report.html [Content-Type=text/html]...\n",
      "- [25/25 files][176.6 GiB/176.6 GiB] 100% Done                                  \n",
      "Operation completed over 25 objects/176.6 GiB.                                   \n"
     ]
    }
   ],
   "source": [
    "! gsutil -m cp gs://transfer-amlproject/201001_MP8262_fastq/*  gs://transfer-amlproject/$project/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 255,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying gs://transfer-amlproject/chipseq_MEF2D_degraded/20201001_FLAG_DMSO_MP8262_S93_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEF2D_degraded/20201001_FLAG_DMSO_MP8262_S93_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEF2D_degraded/20201001_FLAG_VHL_MP8262_S99_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEF2D_degraded/20201001_FLAG_VHL_MP8262_S99_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEF2D_degraded/20201001_IRF8_DMSO_MP8262_S88_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEF2D_degraded/20201001_IRF8_DMSO_MP8262_S88_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEF2D_degraded/20201001_IRF8_VHL_MP8262_S94_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEF2D_degraded/20201001_IRF8_VHL_MP8262_S94_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEF2D_degraded/20201001_MED1_DMSO_MP8262_S91_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEF2D_degraded/20201001_MED1_DMSO_MP8262_S91_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEF2D_degraded/20201001_MED1_VHL_MP8262_S97_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEF2D_degraded/20201001_MEF2C_DMSO_MP8262_S90_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEF2D_degraded/20201001_MEF2C_DMSO_MP8262_S90_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEF2D_degraded/20201001_MED1_VHL_MP8262_S97_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEF2D_degraded/20201001_MEF2C_VHL_MP8262_S96_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEF2D_degraded/20201001_MEF2C_VHL_MP8262_S96_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEF2D_degraded/20201001_MYC_DMSO_MP8262_S89_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEF2D_degraded/20201001_MYC_DMSO_MP8262_S89_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEF2D_degraded/20201001_MYC_VHL_MP8262_S95_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEF2D_degraded/20201001_MYC_VHL_MP8262_S95_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEF2D_degraded/20201001_POL_II_Total_DMSO_MP8262_S92_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEF2D_degraded/20201001_POL_II_Total_DMSO_MP8262_S92_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEF2D_degraded/20201001_POL_II_Total_VHL_MP8262_S98_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEF2D_degraded/20201001_POL_II_Total_VHL_MP8262_S98_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEF2D_degraded/multiqc_report.html...  \n",
      "| [25/25 files][176.6 GiB/176.6 GiB] 100% Done  33.0 MiB/s ETA 00:00:00         \n",
      "Operation completed over 25 objects/176.6 GiB.                                   \n"
     ]
    }
   ],
   "source": [
    "! mkdir ../../data/$project && mkdir ../../data/$project/fastqs && gsutil -m cp gs://transfer-amlproject/$project/* ../../data/$project/fastqs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 256,
   "metadata": {},
   "outputs": [],
   "source": [
    "! rm ../../data/$project/fastqs/multiqc_report.html"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 257,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['20201001_FLAG_DMSO_MP8262_S93_R1_001.fastq.gz',\n",
       " '20201001_FLAG_DMSO_MP8262_S93_R2_001.fastq.gz',\n",
       " '20201001_FLAG_VHL_MP8262_S99_R1_001.fastq.gz',\n",
       " '20201001_FLAG_VHL_MP8262_S99_R2_001.fastq.gz',\n",
       " '20201001_IRF8_DMSO_MP8262_S88_R1_001.fastq.gz',\n",
       " '20201001_IRF8_DMSO_MP8262_S88_R2_001.fastq.gz',\n",
       " '20201001_IRF8_VHL_MP8262_S94_R1_001.fastq.gz',\n",
       " '20201001_IRF8_VHL_MP8262_S94_R2_001.fastq.gz',\n",
       " '20201001_MED1_DMSO_MP8262_S91_R1_001.fastq.gz',\n",
       " '20201001_MED1_DMSO_MP8262_S91_R2_001.fastq.gz',\n",
       " '20201001_MED1_VHL_MP8262_S97_R1_001.fastq.gz',\n",
       " '20201001_MED1_VHL_MP8262_S97_R2_001.fastq.gz',\n",
       " '20201001_MEF2C_DMSO_MP8262_S90_R1_001.fastq.gz',\n",
       " '20201001_MEF2C_DMSO_MP8262_S90_R2_001.fastq.gz',\n",
       " '20201001_MEF2C_VHL_MP8262_S96_R1_001.fastq.gz',\n",
       " '20201001_MEF2C_VHL_MP8262_S96_R2_001.fastq.gz',\n",
       " '20201001_MYC_DMSO_MP8262_S89_R1_001.fastq.gz',\n",
       " '20201001_MYC_DMSO_MP8262_S89_R2_001.fastq.gz',\n",
       " '20201001_MYC_VHL_MP8262_S95_R1_001.fastq.gz',\n",
       " '20201001_MYC_VHL_MP8262_S95_R2_001.fastq.gz',\n",
       " '20201001_POL_II_Total_DMSO_MP8262_S92_R1_001.fastq.gz',\n",
       " '20201001_POL_II_Total_DMSO_MP8262_S92_R2_001.fastq.gz',\n",
       " '20201001_POL_II_Total_VHL_MP8262_S98_R1_001.fastq.gz',\n",
       " '20201001_POL_II_Total_VHL_MP8262_S98_R2_001.fastq.gz']"
      ]
     },
     "execution_count": 257,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "a = ! ls ../../data/$project/fastqs\n",
    "a"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 258,
   "metadata": {},
   "outputs": [],
   "source": [
    "rename = {\n",
    "\"20201001_FLAG_DMSO_MP8262_S93\": \"mp881-MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D-r3\",\n",
    "\"20201001_FLAG_VHL_MP8262_S99\": \"mp882-MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D-r1\",\n",
    "\"20201001_IRF8_DMSO_MP8262_S88\": \"mp883-MV411_MEF2D_NT_SC_63_DMSO-IFR8-r1\",\n",
    "\"20201001_IRF8_VHL_MP8262_S94\": \"mp884-MV411_MEF2D_NT_SC_63_VHL-IFR8-r1\",\n",
    "\"20201001_MED1_DMSO_MP8262_S91\": \"mp885-MV411_MEF2D_NT_SC_63_DMSO-MED1-r1\",\n",
    "\"20201001_MED1_VHL_MP8262_S97\": \"mp886-MV411_MEF2D_NT_SC_63_VHL-MED1-r1\",\n",
    "\"20201001_MEF2C_DMSO_MP8262_S90\": \"mp887-MV411_MEF2D_NT_SC_63_DMSO-MEF2C-r1\",\n",
    "\"20201001_MEF2C_VHL_MP8262_S96\": \"mp888-MV411_MEF2D_NT_SC_63_VHL-MEF2C-r1\",\n",
    "\"20201001_MYC_DMSO_MP8262_S89\": \"mp889-MV411_MEF2D_NT_SC_63_DMSO-MYC-r1\",\n",
    "\"20201001_MYC_VHL_MP8262_S95\": \"mp890-MV411_MEF2D_NT_SC_63_VHL-MYC-r1\",\n",
    "\"20201001_POL_II_Total_DMSO_MP8262_S92\": \"mp891-MV411_MEF2D_NT_SC_63_DMSO-POLII-r1\",\n",
    "\"20201001_POL_II_Total_VHL_MP8262_S98\": \"mp892-MV411_MEF2D_NT_SC_63_VHL-POLII-r1\",}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 259,
   "metadata": {},
   "outputs": [],
   "source": [
    "for val in a:\n",
    "    rep = val\n",
    "    for k,v in rename.items():\n",
    "        rep = rep.replace(k,v)\n",
    "    !mv ../../data/$project/fastqs/$val ../../data/$project/fastqs/$rep"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 260,
   "metadata": {},
   "outputs": [],
   "source": [
    "from gsheets import Sheets\n",
    "sheets = Sheets.from_files('~/.client_secret.json', '~/.storage.json')\n",
    "url=\"https://docs.google.com/spreadsheets/d/1yFLjYB1McU530JnLgL0QIMAKIkVl3kl0_LCHje2gk8U\"\n",
    "gsheet = sheets.get(url).sheets[2].to_frame()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>id</th>\n",
       "      <th>cell line</th>\n",
       "      <th>replicate</th>\n",
       "      <th>protein</th>\n",
       "      <th>quality</th>\n",
       "      <th>paired_end</th>\n",
       "      <th>matching input name</th>\n",
       "      <th>processed</th>\n",
       "      <th>name</th>\n",
       "      <th>previous name</th>\n",
       "      <th>...</th>\n",
       "      <th>ratio to droso</th>\n",
       "      <th>unique mapped reads(droso)</th>\n",
       "      <th>scaling factor</th>\n",
       "      <th>Total QC</th>\n",
       "      <th>folderNarrow</th>\n",
       "      <th>folderCompensated</th>\n",
       "      <th>folderQC</th>\n",
       "      <th>folderBroad</th>\n",
       "      <th>folder Bigwig</th>\n",
       "      <th>folder diffPeaks</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>mp100</td>\n",
       "      <td>U937</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "      <td>NaN</td>\n",
       "      <td>n</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Y</td>\n",
       "      <td>mp100-U937-INPUT-r1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://storage.cloud.google.com/amlproject/Ch...</td>\n",
       "      <td>https://console.cloud.google.com/storage/brows...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://console.cloud.google.com/storage/brows...</td>\n",
       "      <td>https://console.cloud.google.com/storage/brows...</td>\n",
       "      <td>https://console.cloud.google.com/storage/brows...</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>mp101</td>\n",
       "      <td>NOMO1</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "      <td>NaN</td>\n",
       "      <td>n</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Y</td>\n",
       "      <td>mp101-NOMO1-INPUT-r1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>mp102</td>\n",
       "      <td>UT7</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "      <td>NaN</td>\n",
       "      <td>n</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Y</td>\n",
       "      <td>mp102-UT7-INPUT-r1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>mp106</td>\n",
       "      <td>MV411</td>\n",
       "      <td>1</td>\n",
       "      <td>MYB</td>\n",
       "      <td>x</td>\n",
       "      <td>n</td>\n",
       "      <td>INPUT_MV411</td>\n",
       "      <td>Y</td>\n",
       "      <td>mp106-MV411-MYB-r1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>mp109</td>\n",
       "      <td>M6</td>\n",
       "      <td>1</td>\n",
       "      <td>CEBPA</td>\n",
       "      <td>x</td>\n",
       "      <td>n</td>\n",
       "      <td>INPUT_M6</td>\n",
       "      <td>Y</td>\n",
       "      <td>mp109-M6-CEBPA-r1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>299</th>\n",
       "      <td>mp871</td>\n",
       "      <td>MV411_RNP_MYB</td>\n",
       "      <td>2</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>NaN</td>\n",
       "      <td>y</td>\n",
       "      <td>INPUT_MV411</td>\n",
       "      <td>NaN</td>\n",
       "      <td>mp871-MV411_RNP_MYB-H3K27AC-r2</td>\n",
       "      <td>20200723_14_MP8095_S224</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>300</th>\n",
       "      <td>mp872</td>\n",
       "      <td>MV411_RNP_SPI1</td>\n",
       "      <td>1</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>NaN</td>\n",
       "      <td>y</td>\n",
       "      <td>INPUT_MV411</td>\n",
       "      <td>NaN</td>\n",
       "      <td>mp872-MV411_RNP_SPI1-H3K27AC-r1</td>\n",
       "      <td>20200723_15_MP8095_S225</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>301</th>\n",
       "      <td>mp873</td>\n",
       "      <td>MV411_RNP_SPI1</td>\n",
       "      <td>2</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>NaN</td>\n",
       "      <td>y</td>\n",
       "      <td>INPUT_MV411</td>\n",
       "      <td>NaN</td>\n",
       "      <td>mp873-MV411_RNP_SPI1-H3K27AC-r2</td>\n",
       "      <td>20200723_16_MP8095_S226</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>302</th>\n",
       "      <td>mp874</td>\n",
       "      <td>MV411_MEF2D_NT_SC_63</td>\n",
       "      <td>2</td>\n",
       "      <td>FLAG_MEF2D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>y</td>\n",
       "      <td>INPUT_MV411</td>\n",
       "      <td>NaN</td>\n",
       "      <td>mp874-MV411_MEF2D_NT_SC_63-FLAG_MEF2D-r2</td>\n",
       "      <td>20200723_1S_MP8095_S209</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>303</th>\n",
       "      <td>mp875</td>\n",
       "      <td>MV411_MEF2C_NT</td>\n",
       "      <td>1</td>\n",
       "      <td>FLAG_MEF2C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>y</td>\n",
       "      <td>INPUT_MV411</td>\n",
       "      <td>NaN</td>\n",
       "      <td>mp875-MV411_MEF2C_NT-FLAG_MEF2C-r1</td>\n",
       "      <td>20200723_2S_MP8095_S210</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>304 rows  30 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "        id             cell line  replicate     protein quality paired_end  \\\n",
       "0    mp100                  U937          1       INPUT     NaN          n   \n",
       "1    mp101                 NOMO1          1       INPUT     NaN          n   \n",
       "2    mp102                   UT7          1       INPUT     NaN          n   \n",
       "3    mp106                 MV411          1         MYB       x          n   \n",
       "4    mp109                    M6          1       CEBPA       x          n   \n",
       "..     ...                   ...        ...         ...     ...        ...   \n",
       "299  mp871         MV411_RNP_MYB          2     H3K27AC     NaN          y   \n",
       "300  mp872        MV411_RNP_SPI1          1     H3K27AC     NaN          y   \n",
       "301  mp873        MV411_RNP_SPI1          2     H3K27AC     NaN          y   \n",
       "302  mp874  MV411_MEF2D_NT_SC_63          2  FLAG_MEF2D     NaN          y   \n",
       "303  mp875        MV411_MEF2C_NT          1  FLAG_MEF2C     NaN          y   \n",
       "\n",
       "    matching input name processed                                      name  \\\n",
       "0                   NaN         Y                       mp100-U937-INPUT-r1   \n",
       "1                   NaN         Y                      mp101-NOMO1-INPUT-r1   \n",
       "2                   NaN         Y                        mp102-UT7-INPUT-r1   \n",
       "3           INPUT_MV411         Y                        mp106-MV411-MYB-r1   \n",
       "4              INPUT_M6         Y                         mp109-M6-CEBPA-r1   \n",
       "..                  ...       ...                                       ...   \n",
       "299         INPUT_MV411       NaN            mp871-MV411_RNP_MYB-H3K27AC-r2   \n",
       "300         INPUT_MV411       NaN           mp872-MV411_RNP_SPI1-H3K27AC-r1   \n",
       "301         INPUT_MV411       NaN           mp873-MV411_RNP_SPI1-H3K27AC-r2   \n",
       "302         INPUT_MV411       NaN  mp874-MV411_MEF2D_NT_SC_63-FLAG_MEF2D-r2   \n",
       "303         INPUT_MV411       NaN        mp875-MV411_MEF2C_NT-FLAG_MEF2C-r1   \n",
       "\n",
       "               previous name  ... ratio to droso unique mapped reads(droso)  \\\n",
       "0                        NaN  ...            NaN                        NaN   \n",
       "1                        NaN  ...            NaN                        NaN   \n",
       "2                        NaN  ...            NaN                        NaN   \n",
       "3                        NaN  ...            NaN                        NaN   \n",
       "4                        NaN  ...            NaN                        NaN   \n",
       "..                       ...  ...            ...                        ...   \n",
       "299  20200723_14_MP8095_S224  ...            NaN                        NaN   \n",
       "300  20200723_15_MP8095_S225  ...            NaN                        NaN   \n",
       "301  20200723_16_MP8095_S226  ...            NaN                        NaN   \n",
       "302  20200723_1S_MP8095_S209  ...            NaN                        NaN   \n",
       "303  20200723_2S_MP8095_S210  ...            NaN                        NaN   \n",
       "\n",
       "    scaling factor                                           Total QC  \\\n",
       "0              NaN  https://storage.cloud.google.com/amlproject/Ch...   \n",
       "1              NaN                                                NaN   \n",
       "2              NaN                                                NaN   \n",
       "3              NaN                                                NaN   \n",
       "4              NaN                                                NaN   \n",
       "..             ...                                                ...   \n",
       "299            NaN                                                NaN   \n",
       "300            NaN                                                NaN   \n",
       "301            NaN                                                NaN   \n",
       "302            NaN                                                NaN   \n",
       "303            NaN                                                NaN   \n",
       "\n",
       "                                          folderNarrow folderCompensated  \\\n",
       "0    https://console.cloud.google.com/storage/brows...               NaN   \n",
       "1                                                  NaN               NaN   \n",
       "2                                                  NaN               NaN   \n",
       "3                                                  NaN               NaN   \n",
       "4                                                  NaN               NaN   \n",
       "..                                                 ...               ...   \n",
       "299                                                NaN               NaN   \n",
       "300                                                NaN               NaN   \n",
       "301                                                NaN               NaN   \n",
       "302                                                NaN               NaN   \n",
       "303                                                NaN               NaN   \n",
       "\n",
       "                                              folderQC  \\\n",
       "0    https://console.cloud.google.com/storage/brows...   \n",
       "1                                                  NaN   \n",
       "2                                                  NaN   \n",
       "3                                                  NaN   \n",
       "4                                                  NaN   \n",
       "..                                                 ...   \n",
       "299                                                NaN   \n",
       "300                                                NaN   \n",
       "301                                                NaN   \n",
       "302                                                NaN   \n",
       "303                                                NaN   \n",
       "\n",
       "                                           folderBroad  \\\n",
       "0    https://console.cloud.google.com/storage/brows...   \n",
       "1                                                  NaN   \n",
       "2                                                  NaN   \n",
       "3                                                  NaN   \n",
       "4                                                  NaN   \n",
       "..                                                 ...   \n",
       "299                                                NaN   \n",
       "300                                                NaN   \n",
       "301                                                NaN   \n",
       "302                                                NaN   \n",
       "303                                                NaN   \n",
       "\n",
       "                                         folder Bigwig  folder diffPeaks  \n",
       "0    https://console.cloud.google.com/storage/brows...               NaN  \n",
       "1                                                  NaN               NaN  \n",
       "2                                                  NaN               NaN  \n",
       "3                                                  NaN               NaN  \n",
       "4                                                  NaN               NaN  \n",
       "..                                                 ...               ...  \n",
       "299                                                NaN               NaN  \n",
       "300                                                NaN               NaN  \n",
       "301                                                NaN               NaN  \n",
       "302                                                NaN               NaN  \n",
       "303                                                NaN               NaN  \n",
       "\n",
       "[304 rows x 30 columns]"
      ]
     },
     "execution_count": 70,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gsheet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 261,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['mp881-MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D-r3_R1_001.fastq.gz',\n",
       " 'mp881-MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D-r3_R2_001.fastq.gz',\n",
       " 'mp882-MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D-r1_R1_001.fastq.gz',\n",
       " 'mp882-MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D-r1_R2_001.fastq.gz',\n",
       " 'mp883-MV411_MEF2D_NT_SC_63_DMSO-IFR8-r1_R1_001.fastq.gz',\n",
       " 'mp883-MV411_MEF2D_NT_SC_63_DMSO-IFR8-r1_R2_001.fastq.gz',\n",
       " 'mp884-MV411_MEF2D_NT_SC_63_VHL-IFR8-r1_R1_001.fastq.gz',\n",
       " 'mp884-MV411_MEF2D_NT_SC_63_VHL-IFR8-r1_R2_001.fastq.gz',\n",
       " 'mp885-MV411_MEF2D_NT_SC_63_DMSO-MED1-r1_R1_001.fastq.gz',\n",
       " 'mp885-MV411_MEF2D_NT_SC_63_DMSO-MED1-r1_R2_001.fastq.gz',\n",
       " 'mp886-MV411_MEF2D_NT_SC_63_VHL-MED1-r1_R1_001.fastq.gz',\n",
       " 'mp886-MV411_MEF2D_NT_SC_63_VHL-MED1-r1_R2_001.fastq.gz',\n",
       " 'mp887-MV411_MEF2D_NT_SC_63_DMSO-MEF2C-r1_R1_001.fastq.gz',\n",
       " 'mp887-MV411_MEF2D_NT_SC_63_DMSO-MEF2C-r1_R2_001.fastq.gz',\n",
       " 'mp888-MV411_MEF2D_NT_SC_63_VHL-MEF2C-r1_R1_001.fastq.gz',\n",
       " 'mp888-MV411_MEF2D_NT_SC_63_VHL-MEF2C-r1_R2_001.fastq.gz',\n",
       " 'mp889-MV411_MEF2D_NT_SC_63_DMSO-MYC-r1_R1_001.fastq.gz',\n",
       " 'mp889-MV411_MEF2D_NT_SC_63_DMSO-MYC-r1_R2_001.fastq.gz',\n",
       " 'mp890-MV411_MEF2D_NT_SC_63_VHL-MYC-r1_R1_001.fastq.gz',\n",
       " 'mp890-MV411_MEF2D_NT_SC_63_VHL-MYC-r1_R2_001.fastq.gz',\n",
       " 'mp891-MV411_MEF2D_NT_SC_63_DMSO-POLII-r1_R1_001.fastq.gz',\n",
       " 'mp891-MV411_MEF2D_NT_SC_63_DMSO-POLII-r1_R2_001.fastq.gz',\n",
       " 'mp892-MV411_MEF2D_NT_SC_63_VHL-POLII-r1_R1_001.fastq.gz',\n",
       " 'mp892-MV411_MEF2D_NT_SC_63_VHL-POLII-r1_R2_001.fastq.gz']"
      ]
     },
     "execution_count": 261,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "a = ! ls ../../data/$project/fastqs\n",
    "a"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 262,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = {\n",
    "\"fastq_1\": [],\n",
    "\"fastq_2\": [],\n",
    "\"antibody\": [],\n",
    "\"group\": [],\n",
    "\"replicate\": [],\n",
    "\"control\": []\n",
    "}\n",
    "for val in h.grouped(a,2):\n",
    "    row = gsheet[gsheet.id==val[0].split('/')[-1].split('-')[0]]\n",
    "    df['group'].append(\"MV4\"+row['name'].values[0].split('-r')[0].split('-MV4')[1])\n",
    "    df['replicate'].append(row['name'].values[0].split('-r')[1])\n",
    "    df['fastq_1'].append(project+\"/fastqs/\"+val[0])\n",
    "    df['fastq_2'].append(project+\"/fastqs/\"+val[1])\n",
    "    df['antibody'].append(row['protein'].values[0])\n",
    "    df['control'].append(\"INPUT\")\n",
    "df['group'].append('INPUT')\n",
    "df['replicate'].append(1)\n",
    "df['fastq_1'].append('ref/mp845-MV411-INPUT-r2_R1.fastq.gz')\n",
    "df['fastq_2'].append('ref/mp845-MV411-INPUT-r2_R2.fastq.gz')\n",
    "df['antibody'].append(\"\")\n",
    "df['control'].append(\"\")\n",
    "df = pd.DataFrame(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 265,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['chipseq_MEF2D_degraded/fastqs/mp881-MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D-r3_R1_001.fastq.gz',\n",
       " 'chipseq_MEF2D_degraded/fastqs/mp881-MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D-r3_R2_001.fastq.gz',\n",
       " 'FLAG_MEF2D',\n",
       " 'MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D',\n",
       " '3',\n",
       " 'INPUT']"
      ]
     },
     "execution_count": 265,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.loc[0].tolist()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "group,replicate,fastq_1,fastq_2,antibody,control\r\n",
      "MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D,1,chipseq_MEF2D_degraded/fastqs/mp881-MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D-r3_R1_001.fastq.gz,chipseq_MEF2D_degraded/fastqs/mp881-MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D-r3_R2_001.fastq.gz,FLAG_MEF2D,INPUT\r\n",
      "MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D,1,chipseq_MEF2D_degraded/fastqs/mp882-MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D-r1_R1_001.fastq.gz,chipseq_MEF2D_degraded/fastqs/mp882-MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D-r1_R2_001.fastq.gz,FLAG_MEF2D,INPUT\r\n",
      "MV411_MEF2D_NT_SC_63_DMSO-IFR8,1,chipseq_MEF2D_degraded/fastqs/mp883-MV411_MEF2D_NT_SC_63_DMSO-IFR8-r1_R1_001.fastq.gz,chipseq_MEF2D_degraded/fastqs/mp883-MV411_MEF2D_NT_SC_63_DMSO-IFR8-r1_R2_001.fastq.gz,IFR8,INPUT\r\n",
      "MV411_MEF2D_NT_SC_63_VHL-IFR8,1,chipseq_MEF2D_degraded/fastqs/mp884-MV411_MEF2D_NT_SC_63_VHL-IFR8-r1_R1_001.fastq.gz,chipseq_MEF2D_degraded/fastqs/mp884-MV411_MEF2D_NT_SC_63_VHL-IFR8-r1_R2_001.fastq.gz,IFR8,INPUT\r\n",
      "MV411_MEF2D_NT_SC_63_DMSO-MED1,1,chipseq_MEF2D_degraded/fastqs/mp885-MV411_MEF2D_NT_SC_63_DMSO-MED1-r1_R1_001.fastq.gz,chipseq_MEF2D_degraded/fastqs/mp885-MV411_MEF2D_NT_SC_63_DMSO-MED1-r1_R2_001.fastq.gz,MED1,INPUT\r\n",
      "MV411_MEF2D_NT_SC_63_VHL-MED1,1,chipseq_MEF2D_degraded/fastqs/mp886-MV411_MEF2D_NT_SC_63_VHL-MED1-r1_R1_001.fastq.gz,chipseq_MEF2D_degraded/fastqs/mp886-MV411_MEF2D_NT_SC_63_VHL-MED1-r1_R2_001.fastq.gz,MED1,INPUT\r\n",
      "MV411_MEF2D_NT_SC_63_DMSO-MEF2C,1,chipseq_MEF2D_degraded/fastqs/mp887-MV411_MEF2D_NT_SC_63_DMSO-MEF2C-r1_R1_001.fastq.gz,chipseq_MEF2D_degraded/fastqs/mp887-MV411_MEF2D_NT_SC_63_DMSO-MEF2C-r1_R2_001.fastq.gz,MEF2C,INPUT\r\n",
      "MV411_MEF2D_NT_SC_63_VHL-MEF2C,1,chipseq_MEF2D_degraded/fastqs/mp888-MV411_MEF2D_NT_SC_63_VHL-MEF2C-r1_R1_001.fastq.gz,chipseq_MEF2D_degraded/fastqs/mp888-MV411_MEF2D_NT_SC_63_VHL-MEF2C-r1_R2_001.fastq.gz,MEF2C,INPUT\r\n",
      "MV411_MEF2D_NT_SC_63_DMSO-MYC,1,chipseq_MEF2D_degraded/fastqs/mp889-MV411_MEF2D_NT_SC_63_DMSO-MYC-r1_R1_001.fastq.gz,chipseq_MEF2D_degraded/fastqs/mp889-MV411_MEF2D_NT_SC_63_DMSO-MYC-r1_R2_001.fastq.gz,MYC,INPUT\r\n",
      "MV411_MEF2D_NT_SC_63_VHL-MYC,1,chipseq_MEF2D_degraded/fastqs/mp890-MV411_MEF2D_NT_SC_63_VHL-MYC-r1_R1_001.fastq.gz,chipseq_MEF2D_degraded/fastqs/mp890-MV411_MEF2D_NT_SC_63_VHL-MYC-r1_R2_001.fastq.gz,MYC,INPUT\r\n",
      "MV411_MEF2D_NT_SC_63_DMSO-POLII,1,chipseq_MEF2D_degraded/fastqs/mp891-MV411_MEF2D_NT_SC_63_DMSO-POLII-r1_R1_001.fastq.gz,chipseq_MEF2D_degraded/fastqs/mp891-MV411_MEF2D_NT_SC_63_DMSO-POLII-r1_R2_001.fastq.gz,POLII,INPUT\r\n",
      "MV411_MEF2D_NT_SC_63_VHL-POLII,1,chipseq_MEF2D_degraded/fastqs/mp892-MV411_MEF2D_NT_SC_63_VHL-POLII-r1_R1_001.fastq.gz,chipseq_MEF2D_degraded/fastqs/mp892-MV411_MEF2D_NT_SC_63_VHL-POLII-r1_R2_001.fastq.gz,POLII,INPUT\r\n",
      "INPUT,1,ref/mp845-MV411-INPUT-r2_R1.fastq.gz,ref/mp845-MV411-INPUT-r2_R2.fastq.gz,,\r\n"
     ]
    }
   ],
   "source": [
    "cat ../nextflow/chipseq_MEF2D_degraded_design.csv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 267,
   "metadata": {},
   "outputs": [],
   "source": [
    "df[df.columns[[3,4,0,1,2,5]]].to_csv('../nextflow/'+project+'_design.csv',index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "TIMESTAMP          \tDURATION      \tRUN NAME              \tSTATUS\tREVISION ID\tSESSION ID                          \tCOMMAND                                                                                                                                                                                                                                                          \r\n",
      "2020-02-10 22:39:18\t8m 25s        \tstupefied_crick       \tERR   \t21be314954 \t76ea5df0-153c-4e71-a59d-52c6112fda84\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --narrow_peak --input design.csv --genome GRCh38 --skip_preseq --max_cpus 24 -profile docker -w work                                                                      \r\n",
      "2020-02-10 22:40:37\t1m 6s         \tridiculous_hilbert    \tERR   \t21be314954 \t75004903-035d-4504-ab80-cab74b5acac4\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --narrow_peak --input design.csv --genome GRCh38 --skip_preseq --max_cpus 24 -profile docker -w work                                                                      \r\n",
      "2020-02-10 22:41:52\t1m 18s        \tintergalactic_mercator\tERR   \t21be314954 \tbf13ed87-be57-46c0-8a99-3939bba1a794\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --narrow_peak --input design.csv --genome GRCh38 --skip_preseq --max_cpus 24 -profile docker -w work                                                                      \r\n",
      "2020-02-10 22:52:31\t1d 15h 39m 50s\tgrave_baekeland       \tOK    \t21be314954 \tb554f82a-f7d1-42fd-8421-543518d2aa84\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --narrow_peak --input design.csv --genome GRCh38 --skip_preseq --max_cpus 24 -profile docker -w work                                                                      \r\n",
      "2020-03-10 17:24:20\t4h 47m 8s     \tfestering_lovelace    \tERR   \t21be314954 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --narrow_peak --input design_rep4.csv --genome GRCh38 --skip_preseq --max_cpus 24 -profile docker -w work                                                                 \r\n",
      "2020-03-11 15:16:55\t16s           \tcompassionate_raman   \tERR   \t21be314954 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --narrow_peak --input design_rep4.csv --genome GRCh38 --skip_preseq --max_cpus 24 -profile docker -w work -resume festering_lovelace                                      \r\n",
      "2020-03-11 15:17:44\t19h 57m 30s   \tsilly_thompson        \tOK    \t21be314954 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --narrow_peak --input design_rep4.csv --genome GRCh38 --skip_preseq --max_cpus 24 -profile docker -w work -resume festering_lovelace                                      \r\n",
      "2020-05-27 15:08:20\t-             \tridiculous_bardeen    \t-     \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh37 -resume                                                                                                                                                                        \r\n",
      "2020-05-27 15:09:40\t-             \tmagical_mirzakhani    \t-     \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh37 -resume                                                                                                                                                                        \r\n",
      "2020-05-27 16:54:22\t6m 55s        \tnice_tesla            \tERR   \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh37 -resume                                                                                                                                                                        \r\n",
      "2020-05-27 17:01:54\t1m 32s        \tagitated_bose         \tERR   \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh37 -resume                                                                                                                                                                        \r\n",
      "2020-05-27 17:03:49\t-             \tconfident_pauling     \t-     \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh38 -resume --genome --max_cpus 16                                                                                                                                                 \r\n",
      "2020-05-27 17:04:17\t-             \thopeful_watson        \t-     \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh38 -resume --max_cpus 16                                                                                                                                                          \r\n",
      "2020-05-27 17:42:01\t-             \telegant_liskov        \t-     \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh38 -resume --max_cpus 12                                                                                                                                                          \r\n",
      "2020-05-27 18:23:13\t3h 1m 56s     \tcurious_engelbart     \tERR   \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh38 -resume --max_cpus 12                                                                                                                                                          \r\n",
      "2020-05-27 21:25:55\t4m 43s        \tmad_bell              \tERR   \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh38 -resume --max_cpus 6                                                                                                                                                           \r\n",
      "2020-05-28 13:08:34\t11h 21m 20s   \tspecial_heyrovsky     \tOK    \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh38 -resume --max_cpus 6                                                                                                                                                           \r\n",
      "2020-08-04 17:36:59\t1m 59s        \tbackstabbing_hilbert  \tERR   \t21be314954 \t360ae07b-fb63-4b50-b834-7ba2af75b988\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --broad_peak --input ../AMLproject/nextflow/additional_degraded_v1_design.csv --genome GRCh38 --skip_preseq --max_cpus 8 -profile docker -w work                          \r\n",
      "2020-08-04 17:40:57\t1m 14s        \tfurious_liskov        \tERR   \t21be314954 \tb16ce591-36eb-4bc2-901b-840bbe4c73bd\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --broad_peak --input ../AMLproject/nextflow/additional_degraded_v1_design.csv --genome GRCh38 --skip_preseq --max_cpus 8 -profile docker -w work                          \r\n",
      "2020-08-04 17:42:40\t17.5s         \tridiculous_kilby      \tERR   \t21be314954 \tc3ff2f08-8186-4048-b643-1d615eb7dcc1\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --broad_peak --input ../AMLproject/nextflow/additional_degraded_v1_design.csv --genome GRCh38 --skip_preseq --max_cpus 6 -profile docker -w work                          \r\n",
      "2020-08-04 18:23:46\t6.1s          \tlethal_celsius        \tERR   \t21be314954 \t58d0cab0-e38f-4cd7-97a9-d897c5d2b66d\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --broad_peak --input ../AMLproject/nextflow/additional_degraded_v1_design.csv --genome GRCh38 --skip_preseq --max_cpus 6 -profile docker -w work                          \r\n",
      "2020-08-04 18:23:57\t4h 44m 38s    \tinfallible_woese      \tERR   \t21be314954 \t9d191f62-68e0-4e0b-8c13-2e7c1b32ca2f\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --broad_peak --input ../AMLproject/nextflow/additional_degraded_v1_design.csv --genome GRCh38 --skip_preseq --max_cpus 16 -profile docker -w work                         \r\n",
      "2020-08-04 23:21:36\t16h 4m 40s    \texotic_bartik         \tERR   \t21be314954 \t9d191f62-68e0-4e0b-8c13-2e7c1b32ca2f\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --broad_peak --input ../AMLproject/nextflow/additional_degraded_v1_design.csv --genome GRCh38 --skip_preseq --max_cpus 16 -profile docker -w work -resume infallible_woese\r\n"
     ]
    }
   ],
   "source": [
    "! cd ../../data/ && sudo ../nextflow log ## to get access to the previous runs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "N E X T F L O W  ~  version 19.10.0\n",
      "Launching `nf-core/chipseq` [wise_legentil] - revision: 21be314954 [master]\n",
      "NOTE: Your local project version looks outdated - a different revision is available in the remote repository [0f487ed76d]\n",
      "-\u001b[2m--------------------------------------------------\u001b[0m-\n",
      "                                        \u001b[0;32m,--.\u001b[0;30m/\u001b[0;32m,-.\u001b[0m\n",
      "\u001b[0;34m        ___     __   __   __   ___     \u001b[0;32m/,-._.--~'\u001b[0m\n",
      "\u001b[0;34m  |\\ | |__  __ /  ` /  \\ |__) |__         \u001b[0;33m}  {\u001b[0m\n",
      "\u001b[0;34m  | \\| |       \\__, \\__/ |  \\ |___     \u001b[0;32m\\`-._,-`-,\u001b[0m\n",
      "                                        \u001b[0;32m`._,._,'\u001b[0m\n",
      "\u001b[0;35m  nf-core/chipseq v1.1.0\u001b[0m\n",
      "-\u001b[2m--------------------------------------------------\u001b[0m-\n",
      "Run Name            : wise_legentil\n",
      "Data Type           : Paired-End\n",
      "Design File         : ../AMLproject/nextflow/additional_degraded_v1_design.csv\n",
      "Genome              : GRCh38\n",
      "Fasta File          : s3://ngi-igenomes/igenomes//Homo_sapiens/NCBI/GRCh38/Sequence/WholeGenomeFasta/genome.fa\n",
      "GTF File            : s3://ngi-igenomes/igenomes//Homo_sapiens/NCBI/GRCh38/Annotation/Genes/genes.gtf\n",
      "Gene BED File       : s3://ngi-igenomes/igenomes//Homo_sapiens/NCBI/GRCh38/Annotation/Genes/genes.bed\n",
      "BWA Index           : s3://ngi-igenomes/igenomes//Homo_sapiens/NCBI/GRCh38/Sequence/BWAIndex/genome.fa\n",
      "Blacklist BED       : /root/.nextflow/assets/nf-core/chipseq/assets/blacklists/hg38-blacklist.bed\n",
      "MACS2 Genome Size   : 2.7e9\n",
      "Min Consensus Reps  : 1\n",
      "MACS2 Narrow Peaks  : No\n",
      "MACS2 Broad Cutoff  : 0.1\n",
      "Trim R1             : 0 bp\n",
      "Trim R2             : 0 bp\n",
      "Trim 3' R1          : 0 bp\n",
      "Trim 3' R2          : 0 bp\n",
      "NextSeq Trim        : 0 bp\n",
      "Sequencing Center   : DFCI\n",
      "Fingerprint Bins    : 500000\n",
      "Save Genome Index   : No\n",
      "Skip Preseq         : Yes\n",
      "Max Resources       : 128 GB memory, 16 cpus, 10d time per job\n",
      "Container           : docker - nfcore/chipseq:1.1.0\n",
      "Output Dir          : ./results\n",
      "Launch Dir          : /home/jeremie/data\n",
      "Working Dir         : /home/jeremie/data/work\n",
      "Script Dir          : /root/.nextflow/assets/nf-core/chipseq\n",
      "User                : root\n",
      "Config Profile      : docker\n",
      "E-mail Address      : jkobject@gmail.com\n",
      "E-mail on failure   : false\n",
      "MultiQC Max Size    : 25 MB\n",
      "-\u001b[2m--------------------------------------------------\u001b[0m-\n",
      "[-        ] process > CheckDesign      -\u001b[K\n",
      "[-        ] process > MakeTSSBED       -\u001b[K\n",
      "[-        ] process > MakeGenomeFilter -\u001b[K\n",
      "[-        ] process > FastQC           -\u001b[K\n",
      "[-        ] process > TrimGalore       -\u001b[K\n",
      "[-        ] process > BWAMem           -\u001b[K\n",
      "\u001b[7A\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[-        ] process > MakeTSSBED                     -\u001b[K\n",
      "[-        ] process > MakeGenomeFilter               -\u001b[K\n",
      "[-        ] process > FastQC                         -\u001b[K\n",
      "[-        ] process > TrimGalore                     -\u001b[K\n",
      "[-        ] process > BWAMem                         -\u001b[K\n",
      "[-        ] process > SortBAM                        -\u001b[K\n",
      "[-        ] process > MergeBAM                       -\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "\u001b[20A\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[-        ] process > MakeGenomeFilter               -\u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[ee/468381] process > TrimGalore (MV411_RNP_RUNX1... [100%] 7 of 7, cached: 7\u001b[K\n",
      "[-        ] process > BWAMem                         -\u001b[K\n",
      "[-        ] process > SortBAM                        -\u001b[K\n",
      "[-        ] process > MergeBAM                       -\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[-        ] process > get_software_versions          -\u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "\u001b[26A\n",
      "executor >  local (1)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[-        ] process > MakeGenomeFilter               -\u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[-        ] process > BWAMem                         -\u001b[K\n",
      "[-        ] process > SortBAM                        -\u001b[K\n",
      "[-        ] process > MergeBAM                       -\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (1)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[-        ] process > MakeGenomeFilter               -\u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[8f/3989b4] process > BWAMem (MV411_RNP_AAVS1-H3K... [100%] 1 of 1, cached: 1\u001b[K\n",
      "[-        ] process > SortBAM                        -\u001b[K\n",
      "[-        ] process > MergeBAM                       -\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (1)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[-        ] process > MergeBAM                       -\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (3)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/8c0a93] process > MergeBAM (MV411_RNP_RUNX2-H... [ 85%] 11 of 13, cached: 11\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (3)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/8c0a93] process > MergeBAM (MV411_RNP_RUNX2-H... [ 85%] 11 of 13, cached: 11\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (4)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/8c0a93] process > MergeBAM (MV411_RNP_RUNX2-H... [ 86%] 12 of 14, cached: 11\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (5)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[c6/4b33eb] process > MergeBAM (MV411_RNP_MEF2D-H... [ 87%] 13 of 15, cached: 11\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (6)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[fc/cf6140] process > MergeBAM (MV411_RNP_SPI1-H3... [ 88%] 14 of 16, cached: 11\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (7)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[c6/4b33eb] process > MergeBAM (MV411_RNP_MEF2D-H... [ 88%] 15 of 17, cached: 11\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (8)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[12/a07a72] process > MergeBAM (MV411_RNP_MYB-H3K... [ 94%] 16 of 17, cached: 11\u001b[K\n",
      "[84/574208] process > MergeBAMFilter (MV411_RNP_S... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (9)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[66/a24586] process > MergeBAMFilter (MV411_RNP_R... [  0%] 0 of 2\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (10)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/83ab58] process > MergeBAMFilter (MV411_RNP_M... [ 33%] 1 of 3\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (11)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[84/574208] process > MergeBAMFilter (MV411_RNP_S... [ 50%] 2 of 4\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (12)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/83ab58] process > MergeBAMFilter (MV411_RNP_M... [ 60%] 3 of 5\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (13)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[60/4767d1] process > MergeBAMFilter (MV411_RNP_R... [ 67%] 4 of 6\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (14)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[62/3caa87] process > MergeBAMFilter (MV411_RNP_I... [ 57%] 4 of 7\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (14)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[60/4767d1] process > MergeBAMFilter (MV411_RNP_R... [ 71%] 5 of 7\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (15)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[e1/d4ac46] process > MergeBAMFilter (MV411_RNP_A... [ 63%] 5 of 8\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (15)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[9f/a848c2] process > MergeBAMFilter (INPUT_R1)      [ 75%] 6 of 8\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (16)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[ff/e09463] process > MergeBAMFilter (MV411_RNP_M... [ 78%] 7 of 9\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (17)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[77/a464ff] process > MergeBAMFilter (MV411_RNP_I... [ 80%] 8 of 10\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (18)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[ff/e09463] process > MergeBAMFilter (MV411_RNP_M... [ 82%] 9 of 11\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (18)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[77/a464ff] process > MergeBAMFilter (MV411_RNP_I... [ 91%] 10 of 11\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (19)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[e2/12ed6b] process > MergeBAMFilter (MV411_RNP_R... [ 83%] 10 of 12\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (20)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[07/b34e79] process > MergeBAMFilter (MV411_RNP_R... [ 85%] 11 of 13\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (21)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[9a/a02fdf] process > MergeBAMFilter (MV411_RNP_S... [ 86%] 12 of 14\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (22)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[65/4e0731] process > MergeBAMFilter (MV411_RNP_M... [ 87%] 13 of 15\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (23)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[e0/a67acd] process > MergeBAMFilter (MV411_RNP_M... [ 88%] 14 of 16\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (24)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [ 88%] 15 of 17\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (25)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[e0/a67acd] process > MergeBAMFilter (MV411_RNP_M... [ 94%] 16 of 17\u001b[K\n",
      "[21/51df00] process > MergeBAMRemoveOrphan (MV411... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (26)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[e0/a67acd] process > MergeBAMFilter (MV411_RNP_M... [ 94%] 16 of 17\u001b[K\n",
      "[21/51df00] process > MergeBAMRemoveOrphan (MV411... [ 50%] 1 of 2\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (27)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[30/1ef981] process > MergeBAMRemoveOrphan (MV411... [ 33%] 1 of 3\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (28)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[57/199f77] process > MergeBAMRemoveOrphan (MV411... [ 50%] 2 of 4\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (29)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[30/1ef981] process > MergeBAMRemoveOrphan (MV411... [ 60%] 3 of 5\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (30)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[bf/2620e0] process > MergeBAMRemoveOrphan (MV411... [ 67%] 4 of 6\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (31)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[c7/9d9b0a] process > MergeBAMRemoveOrphan (MV411... [ 71%] 5 of 7\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (32)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[70/ef513d] process > MergeBAMRemoveOrphan (INPUT... [ 75%] 6 of 8\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (33)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[63/892aee] process > MergeBAMRemoveOrphan (MV411... [ 78%] 7 of 9\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (34)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[b3/9db05d] process > MergeBAMRemoveOrphan (MV411... [ 80%] 8 of 10\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (35)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[9f/c2beee] process > MergeBAMRemoveOrphan (MV411... [ 82%] 9 of 11\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (36)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[22/110bf7] process > MergeBAMRemoveOrphan (MV411... [ 83%] 10 of 12\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (37)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[1b/49d44f] process > MergeBAMRemoveOrphan (MV411... [ 85%] 11 of 13\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (38)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[ba/00ce48] process > MergeBAMRemoveOrphan (MV411... [ 79%] 11 of 14\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (38)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[08/ccd392] process > MergeBAMRemoveOrphan (MV411... [ 86%] 12 of 14\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (39)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[9a/fa4609] process > MergeBAMRemoveOrphan (MV411... [ 87%] 13 of 15\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (40)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[04/2a4108] process > MergeBAMRemoveOrphan (MV411... [ 81%] 13 of 16\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (40)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[ba/00ce48] process > MergeBAMRemoveOrphan (MV411... [ 88%] 14 of 16\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (41)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[c0/52c2db] process > MergeBAMRemoveOrphan (MV411... [ 94%] 15 of 16\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[68/d5d1f4] process > PhantomPeakQualTools (MV411... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (42)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[04/2a4108] process > MergeBAMRemoveOrphan (MV411... [100%] 16 of 16\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[41/1dc1b3] process > BigWig (MV411_RNP_RUNX1_RUN... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[68/d5d1f4] process > PhantomPeakQualTools (MV411... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (43)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[04/2a4108] process > MergeBAMRemoveOrphan (MV411... [100%] 16 of 16\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[18/e3205c] process > CollectMultipleMetrics (MV4... [  0%] 0 of 1\u001b[K\n",
      "[41/1dc1b3] process > BigWig (MV411_RNP_RUNX1_RUN... [100%] 1 of 1\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[68/d5d1f4] process > PhantomPeakQualTools (MV411... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (44)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [ 94%] 16 of 17\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[18/e3205c] process > CollectMultipleMetrics (MV4... [  0%] 0 of 1\u001b[K\n",
      "[41/1dc1b3] process > BigWig (MV411_RNP_RUNX1_RUN... [100%] 1 of 1\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[68/d5d1f4] process > PhantomPeakQualTools (MV411... [100%] 1 of 1\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (45)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [ 94%] 16 of 17\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[18/e3205c] process > CollectMultipleMetrics (MV4... [100%] 1 of 1\u001b[K\n",
      "[fa/7ca520] process > BigWig (MV411_RNP_SPI1-H3K2... [ 50%] 1 of 2\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[68/d5d1f4] process > PhantomPeakQualTools (MV411... [100%] 1 of 1\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (46)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [ 94%] 16 of 17\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[18/e3205c] process > CollectMultipleMetrics (MV4... [100%] 1 of 1\u001b[K\n",
      "[fa/7ca520] process > BigWig (MV411_RNP_SPI1-H3K2... [100%] 2 of 2\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[c1/b1a581] process > PhantomPeakQualTools (MV411... [ 50%] 1 of 2\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (47)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[ea/949a3e] process > CollectMultipleMetrics (MV4... [ 50%] 1 of 2\u001b[K\n",
      "[fa/7ca520] process > BigWig (MV411_RNP_SPI1-H3K2... [100%] 2 of 2\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[c1/b1a581] process > PhantomPeakQualTools (MV411... [ 50%] 1 of 2\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (48)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[ea/949a3e] process > CollectMultipleMetrics (MV4... [100%] 2 of 2\u001b[K\n",
      "[55/063141] process > BigWig (MV411_RNP_MEF2D-H3K... [ 67%] 2 of 3\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[c1/b1a581] process > PhantomPeakQualTools (MV411... [ 50%] 1 of 2\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (49)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[ea/949a3e] process > CollectMultipleMetrics (MV4... [100%] 2 of 2\u001b[K\n",
      "[55/063141] process > BigWig (MV411_RNP_MEF2D-H3K... [100%] 3 of 3\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[9d/c498be] process > PhantomPeakQualTools (MV411... [ 33%] 1 of 3\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (50)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[96/83ce50] process > CollectMultipleMetrics (MV4... [ 67%] 2 of 3\u001b[K\n",
      "[55/063141] process > BigWig (MV411_RNP_MEF2D-H3K... [100%] 3 of 3\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[c1/b1a581] process > PhantomPeakQualTools (MV411... [ 67%] 2 of 3\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (51)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[96/83ce50] process > CollectMultipleMetrics (MV4... [100%] 3 of 3\u001b[K\n",
      "[55/063141] process > BigWig (MV411_RNP_MEF2D-H3K... [100%] 3 of 3\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[65/278ab6] process > PhantomPeakQualTools (MV411... [ 50%] 2 of 4\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (52)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[96/83ce50] process > CollectMultipleMetrics (MV4... [100%] 3 of 3\u001b[K\n",
      "[c4/0c2b62] process > BigWig (MV411_RNP_AAVS1-H3K... [ 75%] 3 of 4\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[9d/c498be] process > PhantomPeakQualTools (MV411... [ 75%] 3 of 4\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (53)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[24/913f78] process > CollectMultipleMetrics (MV4... [ 75%] 3 of 4\u001b[K\n",
      "[c4/0c2b62] process > BigWig (MV411_RNP_AAVS1-H3K... [100%] 4 of 4\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[9d/c498be] process > PhantomPeakQualTools (MV411... [ 75%] 3 of 4\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (54)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[24/913f78] process > CollectMultipleMetrics (MV4... [ 75%] 3 of 4\u001b[K\n",
      "[c4/0c2b62] process > BigWig (MV411_RNP_AAVS1-H3K... [100%] 4 of 4\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[65/278ab6] process > PhantomPeakQualTools (MV411... [ 80%] 4 of 5\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (55)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[24/913f78] process > CollectMultipleMetrics (MV4... [100%] 4 of 4\u001b[K\n",
      "[26/8106f4] process > BigWig (MV411_RNP_RUNX1-H3K... [ 80%] 4 of 5\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[65/278ab6] process > PhantomPeakQualTools (MV411... [ 80%] 4 of 5\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (56)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[36/03ab9d] process > CollectMultipleMetrics (MV4... [ 80%] 4 of 5\u001b[K\n",
      "[26/8106f4] process > BigWig (MV411_RNP_RUNX1-H3K... [100%] 5 of 5\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[65/278ab6] process > PhantomPeakQualTools (MV411... [ 80%] 4 of 5\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (57)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[36/03ab9d] process > CollectMultipleMetrics (MV4... [100%] 5 of 5\u001b[K\n",
      "[26/8106f4] process > BigWig (MV411_RNP_RUNX1-H3K... [100%] 5 of 5\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[41/509b9d] process > PhantomPeakQualTools (INPUT... [ 67%] 4 of 6\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (58)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[36/03ab9d] process > CollectMultipleMetrics (MV4... [100%] 5 of 5\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[b6/1c51c9] process > PhantomPeakQualTools (MV411... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (59)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[36/03ab9d] process > CollectMultipleMetrics (MV4... [100%] 5 of 5\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[b6/1c51c9] process > PhantomPeakQualTools (MV411... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[df/af3c12] process > MACSCallPeak (MV411_RNP_AAV... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (60)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[36/03ab9d] process > CollectMultipleMetrics (MV4... [100%] 5 of 5\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[b6/1c51c9] process > PhantomPeakQualTools (MV411... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[df/af3c12] process > MACSCallPeak (MV411_RNP_AAV... [ 50%] 1 of 2\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (60)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[36/03ab9d] process > CollectMultipleMetrics (MV4... [100%] 5 of 5\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[b6/1c51c9] process > PhantomPeakQualTools (MV411... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[df/af3c12] process > MACSCallPeak (MV411_RNP_AAV... [ 50%] 1 of 2\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "Staging foreign file: s3://ngi-igenomes/igenomes/Homo_sapiens/NCBI/GRCh38/Annotation/Genes/genes.gtf\u001b[K\n",
      "\u001b[29A\n",
      "executor >  local (61)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[36/03ab9d] process > CollectMultipleMetrics (MV4... [100%] 5 of 5\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[41/509b9d] process > PhantomPeakQualTools (INPUT... [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[ec/449ea1] process > MACSCallPeak (MV411_RNP_RUN... [ 33%] 1 of 3\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[K\n",
      "\u001b[29A\n",
      "executor >  local (62)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[36/03ab9d] process > CollectMultipleMetrics (MV4... [100%] 5 of 5\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[41/509b9d] process > PhantomPeakQualTools (INPUT... [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[ca/eb457d] process > MACSCallPeak (MV411_RNP_MEF... [ 50%] 2 of 4\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (63)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[36/03ab9d] process > CollectMultipleMetrics (MV4... [100%] 5 of 5\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[41/509b9d] process > PhantomPeakQualTools (INPUT... [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[ec/449ea1] process > MACSCallPeak (MV411_RNP_RUN... [ 60%] 3 of 5\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (64)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[6f/0c32f5] process > CollectMultipleMetrics (INP... [ 83%] 5 of 6\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[41/509b9d] process > PhantomPeakQualTools (INPUT... [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[99/4577c5] process > MACSCallPeak (MV411_RNP_RUN... [ 80%] 4 of 5\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (65)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[6f/0c32f5] process > CollectMultipleMetrics (INP... [100%] 6 of 6\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[8c/6222bd] process > PhantomPeakQualTools (MV411... [ 86%] 6 of 7\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[99/4577c5] process > MACSCallPeak (MV411_RNP_RUN... [ 80%] 4 of 5\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (66)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[6f/0c32f5] process > CollectMultipleMetrics (INP... [100%] 6 of 6\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[8c/6222bd] process > PhantomPeakQualTools (MV411... [ 86%] 6 of 7\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[74/d6d890] process > MACSCallPeak (MV411_RNP_SPI... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (67)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[6f/0c32f5] process > CollectMultipleMetrics (INP... [100%] 6 of 6\u001b[K\n",
      "[7f/a5a2c1] process > BigWig (MV411_RNP_AAVS1-H3K... [ 86%] 6 of 7\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[8c/6222bd] process > PhantomPeakQualTools (MV411... [100%] 7 of 7\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[74/d6d890] process > MACSCallPeak (MV411_RNP_SPI... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (68)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[77/85e402] process > CollectMultipleMetrics (MV4... [ 86%] 6 of 7\u001b[K\n",
      "[7f/a5a2c1] process > BigWig (MV411_RNP_AAVS1-H3K... [100%] 7 of 7\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[8c/6222bd] process > PhantomPeakQualTools (MV411... [100%] 7 of 7\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[74/d6d890] process > MACSCallPeak (MV411_RNP_SPI... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (69)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[77/85e402] process > CollectMultipleMetrics (MV4... [100%] 7 of 7\u001b[K\n",
      "[83/f433d6] process > BigWig (MV411_RNP_MYB-H3K27... [ 88%] 7 of 8\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[8c/6222bd] process > PhantomPeakQualTools (MV411... [100%] 7 of 7\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[74/d6d890] process > MACSCallPeak (MV411_RNP_SPI... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (70)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[77/85e402] process > CollectMultipleMetrics (MV4... [100%] 7 of 7\u001b[K\n",
      "[83/f433d6] process > BigWig (MV411_RNP_MYB-H3K27... [100%] 8 of 8\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[83/e5aeda] process > PhantomPeakQualTools (MV411... [ 88%] 7 of 8\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[74/d6d890] process > MACSCallPeak (MV411_RNP_SPI... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (71)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[77/85e402] process > CollectMultipleMetrics (MV4... [100%] 7 of 7\u001b[K\n",
      "[83/f433d6] process > BigWig (MV411_RNP_MYB-H3K27... [100%] 8 of 8\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[83/e5aeda] process > PhantomPeakQualTools (MV411... [ 88%] 7 of 8\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[c5/1a276a] process > MACSCallPeak (MV411_RNP_MYB... [ 71%] 5 of 7\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (71)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[77/85e402] process > CollectMultipleMetrics (MV4... [100%] 7 of 7\u001b[K\n",
      "[83/f433d6] process > BigWig (MV411_RNP_MYB-H3K27... [100%] 8 of 8\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[83/e5aeda] process > PhantomPeakQualTools (MV411... [ 88%] 7 of 8\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[aa/9e4d91] process > MACSCallPeak (MV411_RNP_AAV... [ 86%] 6 of 7\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (72)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[f7/870fa3] process > CollectMultipleMetrics (MV4... [ 88%] 7 of 8\u001b[K\n",
      "[83/f433d6] process > BigWig (MV411_RNP_MYB-H3K27... [100%] 8 of 8\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[83/e5aeda] process > PhantomPeakQualTools (MV411... [100%] 8 of 8\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[aa/9e4d91] process > MACSCallPeak (MV411_RNP_AAV... [ 86%] 6 of 7\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (73)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[f7/870fa3] process > CollectMultipleMetrics (MV4... [100%] 8 of 8\u001b[K\n",
      "[83/f433d6] process > BigWig (MV411_RNP_MYB-H3K27... [100%] 8 of 8\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[bc/873392] process > PhantomPeakQualTools (MV411... [ 89%] 8 of 9\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[aa/9e4d91] process > MACSCallPeak (MV411_RNP_AAV... [ 86%] 6 of 7\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (74)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[f7/870fa3] process > CollectMultipleMetrics (MV4... [100%] 8 of 8\u001b[K\n",
      "[be/8c5392] process > BigWig (MV411_RNP_IRF8-H3K2... [ 89%] 8 of 9\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[bc/873392] process > PhantomPeakQualTools (MV411... [ 89%] 8 of 9\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[c5/1a276a] process > MACSCallPeak (MV411_RNP_MYB... [100%] 7 of 7\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (75)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[f7/870fa3] process > CollectMultipleMetrics (MV4... [100%] 8 of 8\u001b[K\n",
      "[be/8c5392] process > BigWig (MV411_RNP_IRF8-H3K2... [ 89%] 8 of 9\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[bc/873392] process > PhantomPeakQualTools (MV411... [100%] 9 of 9\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[47/040371] process > MACSCallPeak (MV411_RNP_IRF... [ 88%] 7 of 8\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (76)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[eb/85c928] process > CollectMultipleMetrics (MV4... [ 89%] 8 of 9\u001b[K\n",
      "[be/8c5392] process > BigWig (MV411_RNP_IRF8-H3K2... [100%] 9 of 9\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[bc/873392] process > PhantomPeakQualTools (MV411... [100%] 9 of 9\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[47/040371] process > MACSCallPeak (MV411_RNP_IRF... [ 88%] 7 of 8\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (77)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[eb/85c928] process > CollectMultipleMetrics (MV4... [100%] 9 of 9\u001b[K\n",
      "[be/8c5392] process > BigWig (MV411_RNP_IRF8-H3K2... [100%] 9 of 9\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/001577] process > PhantomPeakQualTools (MV411... [ 90%] 9 of 10\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[47/040371] process > MACSCallPeak (MV411_RNP_IRF... [ 88%] 7 of 8\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (78)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[eb/85c928] process > CollectMultipleMetrics (MV4... [100%] 9 of 9\u001b[K\n",
      "[4e/c46492] process > BigWig (MV411_RNP_RUNX1_RUN... [ 90%] 9 of 10\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/001577] process > PhantomPeakQualTools (MV411... [100%] 10 of 10\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[47/040371] process > MACSCallPeak (MV411_RNP_IRF... [ 88%] 7 of 8\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (79)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[eb/85c928] process > CollectMultipleMetrics (MV4... [100%] 9 of 9\u001b[K\n",
      "[4e/c46492] process > BigWig (MV411_RNP_RUNX1_RUN... [100%] 10 of 10\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/001577] process > PhantomPeakQualTools (MV411... [100%] 10 of 10\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[3f/dba7c2] process > MACSCallPeak (MV411_RNP_RUN... [ 78%] 7 of 9\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (80)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[56/e75566] process > CollectMultipleMetrics (MV4... [ 90%] 9 of 10\u001b[K\n",
      "[4e/c46492] process > BigWig (MV411_RNP_RUNX1_RUN... [100%] 10 of 10\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/001577] process > PhantomPeakQualTools (MV411... [100%] 10 of 10\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[47/040371] process > MACSCallPeak (MV411_RNP_IRF... [ 89%] 8 of 9\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (81)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[56/e75566] process > CollectMultipleMetrics (MV4... [100%] 10 of 10\u001b[K\n",
      "[60/3850df] process > BigWig (MV411_RNP_IRF8-H3K2... [ 91%] 10 of 11\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/001577] process > PhantomPeakQualTools (MV411... [100%] 10 of 10\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[47/040371] process > MACSCallPeak (MV411_RNP_IRF... [ 89%] 8 of 9\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (82)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[56/e75566] process > CollectMultipleMetrics (MV4... [100%] 10 of 10\u001b[K\n",
      "[60/3850df] process > BigWig (MV411_RNP_IRF8-H3K2... [100%] 11 of 11\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[1c/07bf5e] process > PhantomPeakQualTools (MV411... [ 91%] 10 of 11\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[47/040371] process > MACSCallPeak (MV411_RNP_IRF... [ 89%] 8 of 9\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (83)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[56/e75566] process > CollectMultipleMetrics (MV4... [100%] 10 of 10\u001b[K\n",
      "[60/3850df] process > BigWig (MV411_RNP_IRF8-H3K2... [100%] 11 of 11\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[1c/07bf5e] process > PhantomPeakQualTools (MV411... [ 91%] 10 of 11\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[3f/dba7c2] process > MACSCallPeak (MV411_RNP_RUN... [ 90%] 9 of 10\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (84)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[af/724a3d] process > CollectMultipleMetrics (MV4... [ 91%] 10 of 11\u001b[K\n",
      "[60/3850df] process > BigWig (MV411_RNP_IRF8-H3K2... [100%] 11 of 11\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[1c/07bf5e] process > PhantomPeakQualTools (MV411... [100%] 11 of 11\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[3f/dba7c2] process > MACSCallPeak (MV411_RNP_RUN... [ 90%] 9 of 10\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (85)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[af/724a3d] process > CollectMultipleMetrics (MV4... [100%] 11 of 11\u001b[K\n",
      "[f7/5fa48c] process > BigWig (MV411_RNP_RUNX2-H3K... [ 92%] 11 of 12\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[1c/07bf5e] process > PhantomPeakQualTools (MV411... [100%] 11 of 11\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[3f/dba7c2] process > MACSCallPeak (MV411_RNP_RUN... [ 90%] 9 of 10\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (86)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[af/724a3d] process > CollectMultipleMetrics (MV4... [100%] 11 of 11\u001b[K\n",
      "[f7/5fa48c] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 12 of 12\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[21/517648] process > PhantomPeakQualTools (MV411... [ 92%] 11 of 12\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[3f/dba7c2] process > MACSCallPeak (MV411_RNP_RUN... [ 90%] 9 of 10\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (87)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[af/724a3d] process > CollectMultipleMetrics (MV4... [100%] 11 of 11\u001b[K\n",
      "[f7/5fa48c] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 12 of 12\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[21/517648] process > PhantomPeakQualTools (MV411... [ 92%] 11 of 12\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[51/147018] process > MACSCallPeak (MV411_RNP_IRF... [ 91%] 10 of 11\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (88)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[be/e4f2a1] process > CollectMultipleMetrics (MV4... [ 92%] 11 of 12\u001b[K\n",
      "[f7/5fa48c] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 12 of 12\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[21/517648] process > PhantomPeakQualTools (MV411... [100%] 12 of 12\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[51/147018] process > MACSCallPeak (MV411_RNP_IRF... [ 91%] 10 of 11\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (89)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[be/e4f2a1] process > CollectMultipleMetrics (MV4... [100%] 12 of 12\u001b[K\n",
      "[eb/3a1856] process > BigWig (MV411_RNP_RUNX1-H3K... [ 92%] 12 of 13\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[21/517648] process > PhantomPeakQualTools (MV411... [100%] 12 of 12\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[51/147018] process > MACSCallPeak (MV411_RNP_IRF... [ 91%] 10 of 11\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (90)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[be/e4f2a1] process > CollectMultipleMetrics (MV4... [100%] 12 of 12\u001b[K\n",
      "[eb/3a1856] process > BigWig (MV411_RNP_RUNX1-H3K... [100%] 13 of 13\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[21/517648] process > PhantomPeakQualTools (MV411... [100%] 12 of 12\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[d8/0be403] process > MACSCallPeak (MV411_RNP_RUN... [ 83%] 10 of 12\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (91)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[be/e4f2a1] process > CollectMultipleMetrics (MV4... [100%] 12 of 12\u001b[K\n",
      "[eb/3a1856] process > BigWig (MV411_RNP_RUNX1-H3K... [100%] 13 of 13\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[ce/6b94f7] process > PhantomPeakQualTools (MV411... [ 92%] 12 of 13\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[99/a486f8] process > MACSCallPeak (MV411_RNP_RUN... [ 92%] 11 of 12\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (92)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[58/9c859d] process > CollectMultipleMetrics (MV4... [ 92%] 12 of 13\u001b[K\n",
      "[eb/3a1856] process > BigWig (MV411_RNP_RUNX1-H3K... [100%] 13 of 13\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[ce/6b94f7] process > PhantomPeakQualTools (MV411... [ 92%] 12 of 13\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[d8/0be403] process > MACSCallPeak (MV411_RNP_RUN... [100%] 12 of 12\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (93)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[58/9c859d] process > CollectMultipleMetrics (MV4... [100%] 13 of 13\u001b[K\n",
      "[eb/3a1856] process > BigWig (MV411_RNP_RUNX1-H3K... [100%] 13 of 13\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[1b/b9c016] process > PhantomPeakQualTools (MV411... [ 86%] 12 of 14\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[d8/0be403] process > MACSCallPeak (MV411_RNP_RUN... [100%] 12 of 12\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (94)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[58/9c859d] process > CollectMultipleMetrics (MV4... [100%] 13 of 13\u001b[K\n",
      "[eb/3a1856] process > BigWig (MV411_RNP_RUNX1-H3K... [100%] 13 of 13\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[ce/6b94f7] process > PhantomPeakQualTools (MV411... [ 93%] 13 of 14\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[46/17f258] process > MACSCallPeak (MV411_RNP_SPI... [ 92%] 12 of 13\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (95)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[58/9c859d] process > CollectMultipleMetrics (MV4... [100%] 13 of 13\u001b[K\n",
      "[e7/594109] process > BigWig (MV411_RNP_SPI1-H3K2... [ 93%] 13 of 14\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[1b/b9c016] process > PhantomPeakQualTools (MV411... [100%] 14 of 14\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[46/17f258] process > MACSCallPeak (MV411_RNP_SPI... [ 92%] 12 of 13\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (96)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[39/7b314d] process > CollectMultipleMetrics (MV4... [ 93%] 13 of 14\u001b[K\n",
      "[e7/594109] process > BigWig (MV411_RNP_SPI1-H3K2... [100%] 14 of 14\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[1b/b9c016] process > PhantomPeakQualTools (MV411... [100%] 14 of 14\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[46/17f258] process > MACSCallPeak (MV411_RNP_SPI... [ 92%] 12 of 13\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (97)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[39/7b314d] process > CollectMultipleMetrics (MV4... [100%] 14 of 14\u001b[K\n",
      "[e7/594109] process > BigWig (MV411_RNP_SPI1-H3K2... [100%] 14 of 14\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[1b/b9c016] process > PhantomPeakQualTools (MV411... [100%] 14 of 14\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[fa/6c4132] process > MACSCallPeak (MV411_RNP_MEF... [ 86%] 12 of 14\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (98)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[39/7b314d] process > CollectMultipleMetrics (MV4... [100%] 14 of 14\u001b[K\n",
      "[e1/c25e9c] process > BigWig (MV411_RNP_MEF2D-H3K... [ 93%] 14 of 15\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[1b/b9c016] process > PhantomPeakQualTools (MV411... [100%] 14 of 14\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[46/17f258] process > MACSCallPeak (MV411_RNP_SPI... [ 93%] 13 of 14\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (99)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[39/7b314d] process > CollectMultipleMetrics (MV4... [100%] 14 of 14\u001b[K\n",
      "[e1/c25e9c] process > BigWig (MV411_RNP_MEF2D-H3K... [100%] 15 of 15\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[d7/e2fcc8] process > PhantomPeakQualTools (MV411... [ 93%] 14 of 15\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[46/17f258] process > MACSCallPeak (MV411_RNP_SPI... [ 93%] 13 of 14\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (100)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[49/670ca2] process > CollectMultipleMetrics (MV4... [ 93%] 14 of 15\u001b[K\n",
      "[e1/c25e9c] process > BigWig (MV411_RNP_MEF2D-H3K... [100%] 15 of 15\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[d7/e2fcc8] process > PhantomPeakQualTools (MV411... [100%] 15 of 15\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[46/17f258] process > MACSCallPeak (MV411_RNP_SPI... [ 93%] 13 of 14\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (101)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[49/670ca2] process > CollectMultipleMetrics (MV4... [ 93%] 14 of 15\u001b[K\n",
      "[e1/c25e9c] process > BigWig (MV411_RNP_MEF2D-H3K... [100%] 15 of 15\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/292df8] process > PhantomPeakQualTools (MV411... [ 94%] 15 of 16\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[fa/6c4132] process > MACSCallPeak (MV411_RNP_MEF... [100%] 14 of 14\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (102)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[49/670ca2] process > CollectMultipleMetrics (MV4... [100%] 15 of 15\u001b[K\n",
      "[9c/f2630b] process > BigWig (MV411_RNP_MYB-H3K27... [ 94%] 15 of 16\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/292df8] process > PhantomPeakQualTools (MV411... [ 94%] 15 of 16\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[fa/6c4132] process > MACSCallPeak (MV411_RNP_MEF... [100%] 14 of 14\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (103)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[49/670ca2] process > CollectMultipleMetrics (MV4... [100%] 15 of 15\u001b[K\n",
      "[9c/f2630b] process > BigWig (MV411_RNP_MYB-H3K27... [100%] 16 of 16\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/292df8] process > PhantomPeakQualTools (MV411... [ 94%] 15 of 16\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 93%] 14 of 15\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (104)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[85/5ed89a] process > CollectMultipleMetrics (MV4... [ 94%] 15 of 16\u001b[K\n",
      "[9c/f2630b] process > BigWig (MV411_RNP_MYB-H3K27... [100%] 16 of 16\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/292df8] process > PhantomPeakQualTools (MV411... [100%] 16 of 16\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 93%] 14 of 15\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (105)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[85/5ed89a] process > CollectMultipleMetrics (MV4... [100%] 16 of 16\u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [ 94%] 16 of 17\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/292df8] process > PhantomPeakQualTools (MV411... [100%] 16 of 16\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 93%] 14 of 15\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (106)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[85/5ed89a] process > CollectMultipleMetrics (MV4... [100%] 16 of 16\u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [ 94%] 16 of 17\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 93%] 14 of 15\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (107)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[85/5ed89a] process > CollectMultipleMetrics (MV4... [100%] 16 of 16\u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [ 94%] 16 of 17\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 94%] 15 of 16\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (108)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [ 94%] 16 of 17\u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 94%] 15 of 16\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (109)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 94%] 15 of 16\u001b[K\n",
      "[92/f4e285] process > AnnotatePeaks (MV411_RNP_AA... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (110)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 94%] 15 of 16\u001b[K\n",
      "[3d/7fd60b] process > AnnotatePeaks (MV411_RNP_ME... [ 50%] 1 of 2\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (111)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 94%] 15 of 16\u001b[K\n",
      "[aa/43437e] process > AnnotatePeaks (MV411_RNP_RU... [ 67%] 2 of 3\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (112)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 94%] 15 of 16\u001b[K\n",
      "[a0/3b8b5a] process > AnnotatePeaks (MV411_RNP_RU... [ 75%] 3 of 4\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (113)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 94%] 15 of 16\u001b[K\n",
      "[d9/83574c] process > AnnotatePeaks (MV411_RNP_SP... [ 80%] 4 of 5\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (114)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 94%] 15 of 16\u001b[K\n",
      "[d9/83574c] process > AnnotatePeaks (MV411_RNP_SP... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (115)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[d5/3a754a] process > AnnotatePeaks (MV411_RNP_MY... [ 71%] 5 of 7\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (117)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[46/342adc] process > AnnotatePeaks (MV411_RNP_IR... [ 75%] 6 of 8\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (118)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[48/756ae7] process > AnnotatePeaks (MV411_RNP_RU... [ 78%] 7 of 9\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (119)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[46/342adc] process > AnnotatePeaks (MV411_RNP_IR... [ 80%] 8 of 10\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (120)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[54/d0a95c] process > AnnotatePeaks (MV411_RNP_RU... [ 82%] 9 of 11\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (121)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[08/d35aca] process > AnnotatePeaks (MV411_RNP_RU... [ 83%] 10 of 12\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (122)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[c2/ac16d8] process > AnnotatePeaks (MV411_RNP_SP... [ 85%] 11 of 13\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (123)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[82/d0e0fc] process > AnnotatePeaks (MV411_RNP_ME... [ 86%] 12 of 14\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (124)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[0e/c24b0a] process > AnnotatePeaks (MV411_RNP_MY... [ 87%] 13 of 15\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (125)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [ 88%] 14 of 16\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (125)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[0e/c24b0a] process > AnnotatePeaks (MV411_RNP_MY... [ 94%] 15 of 16\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (126)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[bc/e561cf] process > PlotFingerprint (MV411_RNP_... [  0%] 0 of 1\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (127)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[bc/e561cf] process > PlotFingerprint (MV411_RNP_... [ 50%] 1 of 2\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (128)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[3f/9aec63] process > PlotFingerprint (MV411_RNP_... [ 67%] 2 of 3\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (129)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[8c/531d81] process > PlotFingerprint (MV411_RNP_... [ 75%] 3 of 4\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (129)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[8c/531d81] process > PlotFingerprint (MV411_RNP_... [ 75%] 3 of 4\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (130)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[20/afbec0] process > PlotFingerprint (MV411_RNP_... [ 80%] 4 of 5\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (131)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[20/afbec0] process > PlotFingerprint (MV411_RNP_... [ 83%] 5 of 6\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (132)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[5e/ac3cd1] process > PlotFingerprint (MV411_RNP_... [ 86%] 6 of 7\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (133)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[8f/17b4a2] process > PlotFingerprint (MV411_RNP_... [ 88%] 7 of 8\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (134)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[48/6bdc90] process > PlotFingerprint (MV411_RNP_... [ 89%] 8 of 9\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (135)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[48/6bdc90] process > PlotFingerprint (MV411_RNP_... [ 90%] 9 of 10\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (136)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[2b/144a84] process > PlotFingerprint (MV411_RNP_... [ 91%] 10 of 11\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (137)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[5c/b9836e] process > PlotFingerprint (MV411_RNP_... [ 92%] 11 of 12\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (138)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[72/264c6a] process > PlotFingerprint (MV411_RNP_... [ 92%] 12 of 13\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (139)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[8e/ba2fa1] process > PlotFingerprint (MV411_RNP_... [ 93%] 13 of 14\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (140)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[11/798ccb] process > PlotFingerprint (MV411_RNP_... [ 93%] 14 of 15\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (141)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[24/32de0c] process > PlotProfile (MV411_RNP_RUNX... [  0%] 0 of 1\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[11/798ccb] process > PlotFingerprint (MV411_RNP_... [100%] 15 of 15\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (142)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[24/32de0c] process > PlotProfile (MV411_RNP_RUNX... [ 50%] 1 of 2\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[11/798ccb] process > PlotFingerprint (MV411_RNP_... [100%] 15 of 15\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (143)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[e1/6ab2d8] process > PlotProfile (MV411_RNP_SPI1... [100%] 2 of 2\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [ 94%] 15 of 16\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (144)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[c5/fb716e] process > PlotProfile (MV411_RNP_MEF2... [ 67%] 2 of 3\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (145)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[da/b297d7] process > PlotProfile (MV411_RNP_AAVS... [ 75%] 3 of 4\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (146)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[da/b297d7] process > PlotProfile (MV411_RNP_AAVS... [ 80%] 4 of 5\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (147)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[e2/e0d2e6] process > PlotProfile (MV411_RNP_RUNX... [ 83%] 5 of 6\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (148)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[2c/37fe88] process > PlotProfile (MV411_RNP_AAVS... [ 86%] 6 of 7\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (149)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[2c/37fe88] process > PlotProfile (MV411_RNP_AAVS... [ 88%] 7 of 8\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (150)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[b8/ec5d9c] process > PlotProfile (MV411_RNP_MYB-... [ 89%] 8 of 9\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (151)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[de/2f9acd] process > PlotProfile (MV411_RNP_IRF8... [ 90%] 9 of 10\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (152)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[43/0ba3d9] process > PlotProfile (MV411_RNP_IRF8... [ 91%] 10 of 11\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (153)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[43/0ba3d9] process > PlotProfile (MV411_RNP_IRF8... [ 92%] 11 of 12\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (154)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[78/ac55d3] process > PlotProfile (MV411_RNP_RUNX... [ 92%] 12 of 13\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (155)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[05/0cacbc] process > PlotProfile (MV411_RNP_SPI1... [ 93%] 13 of 14\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (156)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[05/0cacbc] process > PlotProfile (MV411_RNP_SPI1... [ 93%] 14 of 15\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (157)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[28/273a25] process > PlotProfile (MV411_RNP_MEF2... [ 94%] 15 of 16\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (158)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[c2/bcd9cc] process > PlotProfile (MV411_RNP_MYB-... [ 94%] 16 of 17\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (160)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [  0%] 0 of 1\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (161)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [100%] 1 of 1 \u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[5e/b631fb] process > ConsensusPeakSetAnnotate (H... [  0%] 0 of 1\u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (161)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [100%] 1 of 1 \u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[5e/b631fb] process > ConsensusPeakSetAnnotate (H... [100%] 1 of 1 \u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (161)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [100%] 1 of 1 \u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[5e/b631fb] process > ConsensusPeakSetAnnotate (H... [100%] 1 of 1 \u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (162)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [100%] 1 of 1 \u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[5e/b631fb] process > ConsensusPeakSetAnnotate (H... [100%] 1 of 1 \u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[2b/af351b] process > MultiQC                        [  0%] 0 of 1\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (163)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [100%] 1 of 1 \u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[5e/b631fb] process > ConsensusPeakSetAnnotate (H... [100%] 1 of 1 \u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [100%] 1 of 1 \u001b[K\n",
      "[d4/87dc9d] process > IGV                            [  0%] 0 of 1\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[2b/af351b] process > MultiQC                        [  0%] 0 of 1\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (163)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [100%] 1 of 1 \u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[5e/b631fb] process > ConsensusPeakSetAnnotate (H... [100%] 1 of 1 \u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [100%] 1 of 1 \u001b[K\n",
      "[d4/87dc9d] process > IGV                            [100%] 1 of 1 \u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[2b/af351b] process > MultiQC                        [  0%] 0 of 1\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (163)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [100%] 1 of 1 \u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[5e/b631fb] process > ConsensusPeakSetAnnotate (H... [100%] 1 of 1 \u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [100%] 1 of 1 \u001b[K\n",
      "[d4/87dc9d] process > IGV                            [100%] 1 of 1 \u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[2b/af351b] process > MultiQC                        [100%] 1 of 1 \u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (163)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [100%] 1 of 1 \u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[5e/b631fb] process > ConsensusPeakSetAnnotate (H... [100%] 1 of 1 \u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [100%] 1 of 1 \u001b[K\n",
      "[d4/87dc9d] process > IGV                            [100%] 1 of 1 \u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[2b/af351b] process > MultiQC                        [100%] 1 of 1 \u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[31mFailed to invoke `workflow.onComplete` event handler\u001b[K\n",
      "\u001b[K\n",
      " -- Check script '/root/.nextflow/assets/nf-core/chipseq/main.nf' at line: 1610 or see '.nextflow.log' file for more details\u001b[39m\u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[31A\n",
      "executor >  local (163)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [100%] 1 of 1 \u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[5e/b631fb] process > ConsensusPeakSetAnnotate (H... [100%] 1 of 1 \u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [100%] 1 of 1 \u001b[K\n",
      "[d4/87dc9d] process > IGV                            [100%] 1 of 1 \u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[2b/af351b] process > MultiQC                        [100%] 1 of 1 \u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[31mFailed to invoke `workflow.onComplete` event handler\u001b[K\n",
      "\u001b[K\n",
      " -- Check script '/root/.nextflow/assets/nf-core/chipseq/main.nf' at line: 1610 or see '.nextflow.log' file for more details\u001b[39m\u001b[K\n",
      "\u001b[32;1mCompleted at: 06-Aug-2020 19:20:40\n",
      "Duration    : 1d 3h 52m 44s\n",
      "CPU hours   : 538.2 (38.8% cached)\n",
      "Succeeded   : 163\n",
      "Cached      : 83\n",
      "\u001b[22;39m\u001b[K\n",
      "\n"
     ]
    }
   ],
   "source": [
    "#process chips\n",
    "! cd ../../data/ && sudo ../nextflow run nf-core/chipseq --paired_end --seq_center 'DFCI' --email 'jkobject@gmail.com' --input ../AMLproject/nextflow/$project_design.csv --genome GRCh38 --skip_preseq --max_cpus 16 -profile docker -w work -resume exotic_bartik"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "^C\r\n"
     ]
    }
   ],
   "source": [
    "! sudo mv ../../data/results/* ../../data/$project/ && sudo rm -r ..data/work"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[0m\u001b[01;32m../../TrimGalore-0.6.5/trim_galore\u001b[0m*\r\n"
     ]
    }
   ],
   "source": [
    "ls ../../TrimGalore-0.6.5/trim_galore"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "using all files from folder\n",
      "need to be name_*1, name_*2\n",
      "[('mp881-MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D-r3_R1_001.fastq.gz', 'mp881-MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D-r3_R2_001.fastq.gz'), ('mp882-MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D-r1_R1_001.fastq.gz', 'mp882-MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D-r1_R2_001.fastq.gz'), ('mp883-MV411_MEF2D_NT_SC_63_DMSO-IFR8-r1_R1_001.fastq.gz', 'mp883-MV411_MEF2D_NT_SC_63_DMSO-IFR8-r1_R2_001.fastq.gz'), ('mp884-MV411_MEF2D_NT_SC_63_VHL-IFR8-r1_R1_001.fastq.gz', 'mp884-MV411_MEF2D_NT_SC_63_VHL-IFR8-r1_R2_001.fastq.gz'), ('mp885-MV411_MEF2D_NT_SC_63_DMSO-MED1-r1_R1_001.fastq.gz', 'mp885-MV411_MEF2D_NT_SC_63_DMSO-MED1-r1_R2_001.fastq.gz'), ('mp886-MV411_MEF2D_NT_SC_63_VHL-MED1-r1_R1_001.fastq.gz', 'mp886-MV411_MEF2D_NT_SC_63_VHL-MED1-r1_R2_001.fastq.gz'), ('mp887-MV411_MEF2D_NT_SC_63_DMSO-MEF2C-r1_R1_001.fastq.gz', 'mp887-MV411_MEF2D_NT_SC_63_DMSO-MEF2C-r1_R2_001.fastq.gz'), ('mp888-MV411_MEF2D_NT_SC_63_VHL-MEF2C-r1_R1_001.fastq.gz', 'mp888-MV411_MEF2D_NT_SC_63_VHL-MEF2C-r1_R2_001.fastq.gz'), ('mp889-MV411_MEF2D_NT_SC_63_DMSO-MYC-r1_R1_001.fastq.gz', 'mp889-MV411_MEF2D_NT_SC_63_DMSO-MYC-r1_R2_001.fastq.gz'), ('mp890-MV411_MEF2D_NT_SC_63_VHL-MYC-r1_R1_001.fastq.gz', 'mp890-MV411_MEF2D_NT_SC_63_VHL-MYC-r1_R2_001.fastq.gz'), ('mp891-MV411_MEF2D_NT_SC_63_DMSO-POLII-r1_R1_001.fastq.gz', 'mp891-MV411_MEF2D_NT_SC_63_DMSO-POLII-r1_R2_001.fastq.gz'), ('mp892-MV411_MEF2D_NT_SC_63_VHL-POLII-r1_R1_001.fastq.gz', 'mp892-MV411_MEF2D_NT_SC_63_VHL-POLII-r1_R2_001.fastq.gz')]\n",
      "\n",
      "\n",
      "trimming\n",
      "\n",
      "\n",
      "['../../TrimGalore-0.6.5/trim_galore --paired --fastqc --gzip ../../data/chipseq_MEF2D_degraded/fastqs/mp881-MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D-r3_R1_001.fastq.gz ../../data/chipseq_MEF2D_degraded/fastqs/mp881-MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D-r3_R2_001.fastq.gz -o ../../data/chipseq_MEF2D_degraded/', '../../TrimGalore-0.6.5/trim_galore --paired --fastqc --gzip ../../data/chipseq_MEF2D_degraded/fastqs/mp882-MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D-r1_R1_001.fastq.gz ../../data/chipseq_MEF2D_degraded/fastqs/mp882-MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D-r1_R2_001.fastq.gz -o ../../data/chipseq_MEF2D_degraded/', '../../TrimGalore-0.6.5/trim_galore --paired --fastqc --gzip ../../data/chipseq_MEF2D_degraded/fastqs/mp883-MV411_MEF2D_NT_SC_63_DMSO-IFR8-r1_R1_001.fastq.gz ../../data/chipseq_MEF2D_degraded/fastqs/mp883-MV411_MEF2D_NT_SC_63_DMSO-IFR8-r1_R2_001.fastq.gz -o ../../data/chipseq_MEF2D_degraded/', '../../TrimGalore-0.6.5/trim_galore --paired --fastqc --gzip ../../data/chipseq_MEF2D_degraded/fastqs/mp884-MV411_MEF2D_NT_SC_63_VHL-IFR8-r1_R1_001.fastq.gz ../../data/chipseq_MEF2D_degraded/fastqs/mp884-MV411_MEF2D_NT_SC_63_VHL-IFR8-r1_R2_001.fastq.gz -o ../../data/chipseq_MEF2D_degraded/', '../../TrimGalore-0.6.5/trim_galore --paired --fastqc --gzip ../../data/chipseq_MEF2D_degraded/fastqs/mp885-MV411_MEF2D_NT_SC_63_DMSO-MED1-r1_R1_001.fastq.gz ../../data/chipseq_MEF2D_degraded/fastqs/mp885-MV411_MEF2D_NT_SC_63_DMSO-MED1-r1_R2_001.fastq.gz -o ../../data/chipseq_MEF2D_degraded/', '../../TrimGalore-0.6.5/trim_galore --paired --fastqc --gzip ../../data/chipseq_MEF2D_degraded/fastqs/mp886-MV411_MEF2D_NT_SC_63_VHL-MED1-r1_R1_001.fastq.gz ../../data/chipseq_MEF2D_degraded/fastqs/mp886-MV411_MEF2D_NT_SC_63_VHL-MED1-r1_R2_001.fastq.gz -o ../../data/chipseq_MEF2D_degraded/', '../../TrimGalore-0.6.5/trim_galore --paired --fastqc --gzip ../../data/chipseq_MEF2D_degraded/fastqs/mp887-MV411_MEF2D_NT_SC_63_DMSO-MEF2C-r1_R1_001.fastq.gz ../../data/chipseq_MEF2D_degraded/fastqs/mp887-MV411_MEF2D_NT_SC_63_DMSO-MEF2C-r1_R2_001.fastq.gz -o ../../data/chipseq_MEF2D_degraded/', '../../TrimGalore-0.6.5/trim_galore --paired --fastqc --gzip ../../data/chipseq_MEF2D_degraded/fastqs/mp888-MV411_MEF2D_NT_SC_63_VHL-MEF2C-r1_R1_001.fastq.gz ../../data/chipseq_MEF2D_degraded/fastqs/mp888-MV411_MEF2D_NT_SC_63_VHL-MEF2C-r1_R2_001.fastq.gz -o ../../data/chipseq_MEF2D_degraded/', '../../TrimGalore-0.6.5/trim_galore --paired --fastqc --gzip ../../data/chipseq_MEF2D_degraded/fastqs/mp889-MV411_MEF2D_NT_SC_63_DMSO-MYC-r1_R1_001.fastq.gz ../../data/chipseq_MEF2D_degraded/fastqs/mp889-MV411_MEF2D_NT_SC_63_DMSO-MYC-r1_R2_001.fastq.gz -o ../../data/chipseq_MEF2D_degraded/', '../../TrimGalore-0.6.5/trim_galore --paired --fastqc --gzip ../../data/chipseq_MEF2D_degraded/fastqs/mp890-MV411_MEF2D_NT_SC_63_VHL-MYC-r1_R1_001.fastq.gz ../../data/chipseq_MEF2D_degraded/fastqs/mp890-MV411_MEF2D_NT_SC_63_VHL-MYC-r1_R2_001.fastq.gz -o ../../data/chipseq_MEF2D_degraded/', '../../TrimGalore-0.6.5/trim_galore --paired --fastqc --gzip ../../data/chipseq_MEF2D_degraded/fastqs/mp891-MV411_MEF2D_NT_SC_63_DMSO-POLII-r1_R1_001.fastq.gz ../../data/chipseq_MEF2D_degraded/fastqs/mp891-MV411_MEF2D_NT_SC_63_DMSO-POLII-r1_R2_001.fastq.gz -o ../../data/chipseq_MEF2D_degraded/', '../../TrimGalore-0.6.5/trim_galore --paired --fastqc --gzip ../../data/chipseq_MEF2D_degraded/fastqs/mp892-MV411_MEF2D_NT_SC_63_VHL-POLII-r1_R1_001.fastq.gz ../../data/chipseq_MEF2D_degraded/fastqs/mp892-MV411_MEF2D_NT_SC_63_VHL-POLII-r1_R2_001.fastq.gz -o ../../data/chipseq_MEF2D_degraded/']\n",
      "\n",
      "\n",
      "mapping\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "# get scaling values\n",
    "norm, mapped = h.getSpikeInControlScales(refgenome=\"../../data/ref/reference_droso.fna\",\n",
    "fastQfolder='../../data/'+project+'/fastqs/',\n",
    "pairedEnd=True, cores=12,\n",
    "tofilter=True,\n",
    "totrim=True,\n",
    "tomap=True,\n",
    "results=\"../../data/\"+project+\"/\",\n",
    "pathtotrim_galore=\"../../TrimGalore-0.6.5/trim_galore\")\n",
    "norm, mapped"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'mp881-MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D-r3_R1_001_val_1': 3913318,\n",
       " 'mp882-MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D-r1_R1_001_val_1': 3675100,\n",
       " 'mp883-MV411_MEF2D_NT_SC_63_DMSO-IFR8-r1_R1_001_val_1': 3909333,\n",
       " 'mp884-MV411_MEF2D_NT_SC_63_VHL-IFR8-r1_R1_001_val_1': 5479191,\n",
       " 'mp885-MV411_MEF2D_NT_SC_63_DMSO-MED1-r1_R1_001_val_1': 5217619,\n",
       " 'mp886-MV411_MEF2D_NT_SC_63_VHL-MED1-r1_R1_001_val_1': 5460060,\n",
       " 'mp887-MV411_MEF2D_NT_SC_63_DMSO-MEF2C-r1_R1_001_val_1': 4117350,\n",
       " 'mp888-MV411_MEF2D_NT_SC_63_VHL-MEF2C-r1_R1_001_val_1': 3388517,\n",
       " 'mp889-MV411_MEF2D_NT_SC_63_DMSO-MYC-r1_R1_001_val_1': 6008413,\n",
       " 'mp890-MV411_MEF2D_NT_SC_63_VHL-MYC-r1_R1_001_val_1': 5878757,\n",
       " 'mp891-MV411_MEF2D_NT_SC_63_DMSO-POLII-r1_R1_001_val_1': 3529483,\n",
       " 'mp892-MV411_MEF2D_NT_SC_63_VHL-POLII-r1_R1_001_val_1': 9964979}"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "mapped"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "computing scales from the excel sheet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 142,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [1.0,\n",
    "1.0,\n",
    "0.9644136372,\n",
    "0.9068238696,\n",
    "1.0,\n",
    "1.0,\n",
    "0.3954022974,\n",
    "0.8368988329,\n",
    "1.0,\n",
    "1.0,\n",
    "0.5551449949,\n",
    "0.844680746]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-MED1_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-MYC_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-POLII_R1.mLb.clN.sorted.bam']"
      ]
     },
     "execution_count": 36,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bams = ! ls ../../data/$project/bwa/mergedLibrary/*.bam\n",
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1.mLb.clN.sorted.bam'"
      ]
     },
     "execution_count": 37,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bams[6]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### on scalled data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mkdir ../../data/$project/diffPeaks/ && ! mkdir ../../data/$project/diffData/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mkdir ../../data/$project/droso_aligned"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mv ../../data/$project/mp* ../../data/$project/droso_aligned"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [],
   "source": [
    "wigs = ! ls ../../data/$project/bwa/mergedLibrary/bigwig/*.bigWig"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 131,
   "metadata": {},
   "outputs": [],
   "source": [
    "initscales = ! cat ../../data/$project/bwa/mergedLibrary/bigwig/scale/*.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 135,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/scale/INPUT_R1.mLb.clN.scale_factor.txt\r\n",
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/scale/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1.mLb.clN.scale_factor.txt\r\n",
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/scale/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1.mLb.clN.scale_factor.txt\r\n",
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/scale/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1.mLb.clN.scale_factor.txt\r\n",
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/scale/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1.mLb.clN.scale_factor.txt\r\n",
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/scale/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1.mLb.clN.scale_factor.txt\r\n",
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/scale/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1.mLb.clN.scale_factor.txt\r\n",
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/scale/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1.mLb.clN.scale_factor.txt\r\n",
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/scale/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1.mLb.clN.scale_factor.txt\r\n",
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/scale/MV411_MEF2D_NT_SC_63_VHL-MED1_R1.mLb.clN.scale_factor.txt\r\n",
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/scale/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1.mLb.clN.scale_factor.txt\r\n",
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/scale/MV411_MEF2D_NT_SC_63_VHL-MYC_R1.mLb.clN.scale_factor.txt\r\n",
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/scale/MV411_MEF2D_NT_SC_63_VHL-POLII_R1.mLb.clN.scale_factor.txt\r\n"
     ]
    }
   ],
   "source": [
    "! ls ../../data/$project/bwa/mergedLibrary/bigwig/scale/*.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 143,
   "metadata": {},
   "outputs": [],
   "source": [
    "rescales = [val*float(initscales[1+i]) for i, val in enumerate(scales)]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 144,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[0.00547739,\n",
       " 0.011174,\n",
       " 0.008672451055975512,\n",
       " 0.004635565734292153,\n",
       " 0.00588873,\n",
       " 0.00439044,\n",
       " 0.00738607537520226,\n",
       " 0.00599152612449768,\n",
       " 0.00599184,\n",
       " 0.0111614,\n",
       " 0.004559755584460487,\n",
       " 0.00375175089504852]"
      ]
     },
     "execution_count": 144,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "rescales"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 145,
   "metadata": {},
   "outputs": [],
   "source": [
    "chip.bigWigFrom(bams[1:], \n",
    "                genome='GRCh38',scaling=rescales,\n",
    "               numthreads=12)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 146,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mkdir: cannot create directory ../../data/chipseq_MEF2D_degraded/recalib_bigwig/: File exists\r\n"
     ]
    }
   ],
   "source": [
    "!mkdir ../../data/$project/recalib_bigwig/ && mv bigwig/* ../../data/$project/recalib_bigwig/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 155,
   "metadata": {},
   "outputs": [],
   "source": [
    "mv bigwig/* ../../data/$project/recalib_bigwig/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 156,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying file://../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1.bw [Content-Type=application/octet-stream]...\n",
      "==> NOTE: You are uploading one or more large file(s), which would run\n",
      "significantly faster if you enable parallel composite uploads. This\n",
      "feature can be enabled by editing the\n",
      "\"parallel_composite_upload_threshold\" value in your .boto\n",
      "configuration file. However, note that if you do this large files will\n",
      "be uploaded as `composite objects\n",
      "<https://cloud.google.com/storage/docs/composite-objects>`_,which\n",
      "means that any user who downloads such objects will need to have a\n",
      "compiled crcmod installed (see \"gsutil help crcmod\"). This is because\n",
      "without a compiled crcmod, computing checksums on composite objects is\n",
      "so slow that gsutil disables downloads of composite objects.\n",
      "\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1.bw [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1.bw [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_VHL-MYC_R1.bw [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1.bw [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1.bw [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1.bw [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1.bw [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1.bw [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1.bw [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_VHL-MED1_R1.bw [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_VHL-POLII_R1.bw [Content-Type=application/octet-stream]...\n",
      "/ [12/12 files][  2.8 GiB/  2.8 GiB] 100% Done                                  \n",
      "Operation completed over 12 objects/2.8 GiB.                                     \n"
     ]
    }
   ],
   "source": [
    "#! gsutil -m cp -r ../../data/$project/droso_aligned gs://amlproject/Chip/$project/\n",
    "! gsutil -m cp -r ../../data/$project/recalib_bigwig gs://amlproject/Chip/$project/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {},
   "outputs": [],
   "source": [
    "rm -r ../../data/$project/droso_aligned/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 201,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1.mLb.clN.sorted.bam ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1.mLb.clN.sorted.bam and ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "230\n",
      "computing the scaling values\n",
      "b'INFO  @ Mon, 02 Nov 2020 20:57:45: \\n# Command line: callpeak -B -t ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1.mLb.clN.sorted.bam -c ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 230 -n MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1 --outdir ../../data/chipseq_MEF2D_degraded/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Mon, 02 Nov 2020 20:57:45: #1 read fragment files... \\nINFO  @ Mon, 02 Nov 2020 20:57:45: #1 read treatment fragments... \\nINFO  @ Mon, 02 Nov 2020 20:57:50:  1000000 \\nINFO  @ Mon, 02 Nov 2020 20:57:54:  2000000 \\nINFO  @ Mon, 02 Nov 2020 20:57:59:  3000000 \\nINFO  @ Mon, 02 Nov 2020 20:58:04:  4000000 \\nINFO  @ Mon, 02 Nov 2020 20:58:08:  5000000 \\nINFO  @ Mon, 02 Nov 2020 20:58:13:  6000000 \\nINFO  @ Mon, 02 Nov 2020 20:58:17:  7000000 \\nINFO  @ Mon, 02 Nov 2020 20:58:22:  8000000 \\nINFO  @ Mon, 02 Nov 2020 20:58:27:  9000000 \\nINFO  @ Mon, 02 Nov 2020 20:58:32:  10000000 \\nINFO  @ Mon, 02 Nov 2020 20:58:36:  11000000 \\nINFO  @ Mon, 02 Nov 2020 20:58:41:  12000000 \\nINFO  @ Mon, 02 Nov 2020 20:58:46:  13000000 \\nINFO  @ Mon, 02 Nov 2020 20:58:50:  14000000 \\nINFO  @ Mon, 02 Nov 2020 20:58:55:  15000000 \\nINFO  @ Mon, 02 Nov 2020 20:58:59:  16000000 \\nINFO  @ Mon, 02 Nov 2020 20:59:04:  17000000 \\nINFO  @ Mon, 02 Nov 2020 20:59:09:  18000000 \\nINFO  @ Mon, 02 Nov 2020 20:59:13:  19000000 \\nINFO  @ Mon, 02 Nov 2020 20:59:18:  20000000 \\nINFO  @ Mon, 02 Nov 2020 20:59:22:  21000000 \\nINFO  @ Mon, 02 Nov 2020 20:59:27:  22000000 \\nINFO  @ Mon, 02 Nov 2020 20:59:31:  23000000 \\nINFO  @ Mon, 02 Nov 2020 20:59:36:  24000000 \\nINFO  @ Mon, 02 Nov 2020 20:59:41:  25000000 \\nINFO  @ Mon, 02 Nov 2020 20:59:45:  26000000 \\nINFO  @ Mon, 02 Nov 2020 20:59:50:  27000000 \\nINFO  @ Mon, 02 Nov 2020 20:59:55:  28000000 \\nINFO  @ Mon, 02 Nov 2020 20:59:59:  29000000 \\nINFO  @ Mon, 02 Nov 2020 21:00:04:  30000000 \\nINFO  @ Mon, 02 Nov 2020 21:00:08:  31000000 \\nINFO  @ Mon, 02 Nov 2020 21:00:13:  32000000 \\nINFO  @ Mon, 02 Nov 2020 21:00:17:  33000000 \\nINFO  @ Mon, 02 Nov 2020 21:00:22:  34000000 \\nINFO  @ Mon, 02 Nov 2020 21:00:26:  35000000 \\nINFO  @ Mon, 02 Nov 2020 21:00:31:  36000000 \\nINFO  @ Mon, 02 Nov 2020 21:00:35:  37000000 \\nINFO  @ Mon, 02 Nov 2020 21:00:40:  38000000 \\nINFO  @ Mon, 02 Nov 2020 21:00:45:  39000000 \\nINFO  @ Mon, 02 Nov 2020 21:00:50:  40000000 \\nINFO  @ Mon, 02 Nov 2020 21:00:55:  41000000 \\nINFO  @ Mon, 02 Nov 2020 21:01:00:  42000000 \\nINFO  @ Mon, 02 Nov 2020 21:01:05:  43000000 \\nINFO  @ Mon, 02 Nov 2020 21:01:09:  44000000 \\nINFO  @ Mon, 02 Nov 2020 21:01:14:  45000000 \\nINFO  @ Mon, 02 Nov 2020 21:01:18:  46000000 \\nINFO  @ Mon, 02 Nov 2020 21:01:23:  47000000 \\nINFO  @ Mon, 02 Nov 2020 21:01:27:  48000000 \\nINFO  @ Mon, 02 Nov 2020 21:01:33:  49000000 \\nINFO  @ Mon, 02 Nov 2020 21:01:37:  50000000 \\nINFO  @ Mon, 02 Nov 2020 21:01:42:  51000000 \\nINFO  @ Mon, 02 Nov 2020 21:01:47:  52000000 \\nINFO  @ Mon, 02 Nov 2020 21:01:51:  53000000 \\nINFO  @ Mon, 02 Nov 2020 21:01:56:  54000000 \\nINFO  @ Mon, 02 Nov 2020 21:02:00:  55000000 \\nINFO  @ Mon, 02 Nov 2020 21:02:05:  56000000 \\nINFO  @ Mon, 02 Nov 2020 21:02:10:  57000000 \\nINFO  @ Mon, 02 Nov 2020 21:02:14:  58000000 \\nINFO  @ Mon, 02 Nov 2020 21:02:21:  59000000 \\nINFO  @ Mon, 02 Nov 2020 21:02:26:  60000000 \\nINFO  @ Mon, 02 Nov 2020 21:02:31:  61000000 \\nINFO  @ Mon, 02 Nov 2020 21:02:35:  62000000 \\nINFO  @ Mon, 02 Nov 2020 21:02:40:  63000000 \\nINFO  @ Mon, 02 Nov 2020 21:02:44:  64000000 \\nINFO  @ Mon, 02 Nov 2020 21:02:49:  65000000 \\nINFO  @ Mon, 02 Nov 2020 21:02:53:  66000000 \\nINFO  @ Mon, 02 Nov 2020 21:02:58:  67000000 \\nINFO  @ Mon, 02 Nov 2020 21:03:03:  68000000 \\nINFO  @ Mon, 02 Nov 2020 21:03:07:  69000000 \\nINFO  @ Mon, 02 Nov 2020 21:03:12:  70000000 \\nINFO  @ Mon, 02 Nov 2020 21:03:16:  71000000 \\nINFO  @ Mon, 02 Nov 2020 21:03:21:  72000000 \\nINFO  @ Mon, 02 Nov 2020 21:03:25:  73000000 \\nINFO  @ Mon, 02 Nov 2020 21:03:30:  74000000 \\nINFO  @ Mon, 02 Nov 2020 21:03:34:  75000000 \\nINFO  @ Mon, 02 Nov 2020 21:03:39:  76000000 \\nINFO  @ Mon, 02 Nov 2020 21:03:44:  77000000 \\nINFO  @ Mon, 02 Nov 2020 21:03:48:  78000000 \\nINFO  @ Mon, 02 Nov 2020 21:03:53:  79000000 \\nINFO  @ Mon, 02 Nov 2020 21:03:57:  80000000 \\nINFO  @ Mon, 02 Nov 2020 21:04:02:  81000000 \\nINFO  @ Mon, 02 Nov 2020 21:04:06:  82000000 \\nINFO  @ Mon, 02 Nov 2020 21:04:11:  83000000 \\nINFO  @ Mon, 02 Nov 2020 21:04:15:  84000000 \\nINFO  @ Mon, 02 Nov 2020 21:04:20:  85000000 \\nINFO  @ Mon, 02 Nov 2020 21:04:24:  86000000 \\nINFO  @ Mon, 02 Nov 2020 21:04:29:  87000000 \\nINFO  @ Mon, 02 Nov 2020 21:04:33:  88000000 \\nINFO  @ Mon, 02 Nov 2020 21:04:38:  89000000 \\nINFO  @ Mon, 02 Nov 2020 21:04:43:  90000000 \\nINFO  @ Mon, 02 Nov 2020 21:04:46: 90585277 fragments have been read. \\nINFO  @ Mon, 02 Nov 2020 21:05:49: #1.2 read input fragments... \\nINFO  @ Mon, 02 Nov 2020 21:05:53:  1000000 \\nINFO  @ Mon, 02 Nov 2020 21:05:57:  2000000 \\nINFO  @ Mon, 02 Nov 2020 21:06:02:  3000000 \\nINFO  @ Mon, 02 Nov 2020 21:06:06:  4000000 \\nINFO  @ Mon, 02 Nov 2020 21:06:15:  5000000 \\nINFO  @ Mon, 02 Nov 2020 21:06:20:  6000000 \\nINFO  @ Mon, 02 Nov 2020 21:06:25:  7000000 \\nINFO  @ Mon, 02 Nov 2020 21:06:31:  8000000 \\nINFO  @ Mon, 02 Nov 2020 21:06:36:  9000000 \\nINFO  @ Mon, 02 Nov 2020 21:06:40:  10000000 \\nINFO  @ Mon, 02 Nov 2020 21:06:49:  11000000 \\nINFO  @ Mon, 02 Nov 2020 21:06:53:  12000000 \\nINFO  @ Mon, 02 Nov 2020 21:06:56:  13000000 \\nINFO  @ Mon, 02 Nov 2020 21:07:01:  14000000 \\nINFO  @ Mon, 02 Nov 2020 21:07:05:  15000000 \\nINFO  @ Mon, 02 Nov 2020 21:07:09:  16000000 \\nINFO  @ Mon, 02 Nov 2020 21:07:13:  17000000 \\nINFO  @ Mon, 02 Nov 2020 21:07:23:  18000000 \\nINFO  @ Mon, 02 Nov 2020 21:07:28:  19000000 \\nINFO  @ Mon, 02 Nov 2020 21:07:32:  20000000 \\nINFO  @ Mon, 02 Nov 2020 21:07:36:  21000000 \\nINFO  @ Mon, 02 Nov 2020 21:07:40:  22000000 \\nINFO  @ Mon, 02 Nov 2020 21:07:44:  23000000 \\nINFO  @ Mon, 02 Nov 2020 21:07:48:  24000000 \\nINFO  @ Mon, 02 Nov 2020 21:07:52:  25000000 \\nINFO  @ Mon, 02 Nov 2020 21:07:56:  26000000 \\nINFO  @ Mon, 02 Nov 2020 21:08:06:  27000000 \\nINFO  @ Mon, 02 Nov 2020 21:08:09:  28000000 \\nINFO  @ Mon, 02 Nov 2020 21:08:13:  29000000 \\nINFO  @ Mon, 02 Nov 2020 21:08:17:  30000000 \\nINFO  @ Mon, 02 Nov 2020 21:08:21:  31000000 \\nINFO  @ Mon, 02 Nov 2020 21:08:26:  32000000 \\nINFO  @ Mon, 02 Nov 2020 21:08:30:  33000000 \\nINFO  @ Mon, 02 Nov 2020 21:08:33:  34000000 \\nINFO  @ Mon, 02 Nov 2020 21:08:43:  35000000 \\nINFO  @ Mon, 02 Nov 2020 21:08:46:  36000000 \\nINFO  @ Mon, 02 Nov 2020 21:08:50:  37000000 \\nINFO  @ Mon, 02 Nov 2020 21:08:54:  38000000 \\nINFO  @ Mon, 02 Nov 2020 21:08:58:  39000000 \\nINFO  @ Mon, 02 Nov 2020 21:09:03:  40000000 \\nINFO  @ Mon, 02 Nov 2020 21:09:07:  41000000 \\nINFO  @ Mon, 02 Nov 2020 21:09:16:  42000000 \\nINFO  @ Mon, 02 Nov 2020 21:09:20:  43000000 \\nINFO  @ Mon, 02 Nov 2020 21:09:24:  44000000 \\nINFO  @ Mon, 02 Nov 2020 21:09:28:  45000000 \\nINFO  @ Mon, 02 Nov 2020 21:09:32:  46000000 \\nINFO  @ Mon, 02 Nov 2020 21:09:37:  47000000 \\nINFO  @ Mon, 02 Nov 2020 21:09:41:  48000000 \\nINFO  @ Mon, 02 Nov 2020 21:09:51:  49000000 \\nINFO  @ Mon, 02 Nov 2020 21:09:55:  50000000 \\nINFO  @ Mon, 02 Nov 2020 21:09:58:  51000000 \\nINFO  @ Mon, 02 Nov 2020 21:10:03:  52000000 \\nINFO  @ Mon, 02 Nov 2020 21:10:07:  53000000 \\nINFO  @ Mon, 02 Nov 2020 21:10:11:  54000000 \\nINFO  @ Mon, 02 Nov 2020 21:10:16:  55000000 \\nINFO  @ Mon, 02 Nov 2020 21:10:25:  56000000 \\nINFO  @ Mon, 02 Nov 2020 21:10:30:  57000000 \\nINFO  @ Mon, 02 Nov 2020 21:10:34:  58000000 \\nINFO  @ Mon, 02 Nov 2020 21:10:38:  59000000 \\nINFO  @ Mon, 02 Nov 2020 21:10:42:  60000000 \\nINFO  @ Mon, 02 Nov 2020 21:10:46:  61000000 \\nINFO  @ Mon, 02 Nov 2020 21:10:50:  62000000 \\nINFO  @ Mon, 02 Nov 2020 21:10:59:  63000000 \\nINFO  @ Mon, 02 Nov 2020 21:11:03:  64000000 \\nINFO  @ Mon, 02 Nov 2020 21:11:06:  65000000 \\nINFO  @ Mon, 02 Nov 2020 21:11:11:  66000000 \\nINFO  @ Mon, 02 Nov 2020 21:11:14: 66660428 fragments have been read. \\nINFO  @ Mon, 02 Nov 2020 21:11:58: #1 mean fragment size is determined as 269.3 bp from treatment \\nINFO  @ Mon, 02 Nov 2020 21:11:58: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Mon, 02 Nov 2020 21:11:58: #1 fragment size = 269.3 \\nINFO  @ Mon, 02 Nov 2020 21:11:58: #1  total fragments in treatment: 90585277 \\nINFO  @ Mon, 02 Nov 2020 21:11:58: #1 user defined the maximum fragments... \\nINFO  @ Mon, 02 Nov 2020 21:11:58: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 02 Nov 2020 21:14:25: #1  fragments after filtering in treatment: 90561281 \\nINFO  @ Mon, 02 Nov 2020 21:14:25: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 02 Nov 2020 21:14:25: #1  total fragments in control: 66660428 \\nINFO  @ Mon, 02 Nov 2020 21:14:25: #1 user defined the maximum fragments... \\nINFO  @ Mon, 02 Nov 2020 21:14:25: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 02 Nov 2020 21:16:12: #1  fragments after filtering in control: 66632618 \\nINFO  @ Mon, 02 Nov 2020 21:16:12: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 02 Nov 2020 21:16:12: #1 finished! \\nINFO  @ Mon, 02 Nov 2020 21:16:12: #2 Build Peak Model... \\nINFO  @ Mon, 02 Nov 2020 21:16:12: #2 Skipped... \\nINFO  @ Mon, 02 Nov 2020 21:16:12: #3 Call peaks... \\nINFO  @ Mon, 02 Nov 2020 21:16:12: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 02 Nov 2020 21:27:33: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 02 Nov 2020 21:27:33: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_treat_pileup.bdg \\nINFO  @ Mon, 02 Nov 2020 21:27:33: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_control_lambda.bdg \\nINFO  @ Mon, 02 Nov 2020 21:27:33: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 02 Nov 2020 21:27:33: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 02 Nov 2020 21:36:43: #4 Write output xls file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_peaks.xls \\nINFO  @ Mon, 02 Nov 2020 21:36:43: #4 Write peak in narrowPeak format file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_peaks.narrowPeak \\nINFO  @ Mon, 02 Nov 2020 21:36:43: #4 Write summits bed file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_summits.bed \\nINFO  @ Mon, 02 Nov 2020 21:36:43: Done! \\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'INFO  @ Mon, 02 Nov 2020 21:36:44: \\n# Command line: callpeak -B -t ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1.mLb.clN.sorted.bam -c ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 230 -n MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1 --outdir ../../data/chipseq_MEF2D_degraded/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Mon, 02 Nov 2020 21:36:44: #1 read fragment files... \\nINFO  @ Mon, 02 Nov 2020 21:36:44: #1 read treatment fragments... \\nINFO  @ Mon, 02 Nov 2020 21:36:48:  1000000 \\nINFO  @ Mon, 02 Nov 2020 21:36:59:  2000000 \\nINFO  @ Mon, 02 Nov 2020 21:37:04:  3000000 \\nINFO  @ Mon, 02 Nov 2020 21:37:09:  4000000 \\nINFO  @ Mon, 02 Nov 2020 21:37:15:  5000000 \\nINFO  @ Mon, 02 Nov 2020 21:37:20:  6000000 \\nINFO  @ Mon, 02 Nov 2020 21:37:29:  7000000 \\nINFO  @ Mon, 02 Nov 2020 21:37:34:  8000000 \\nINFO  @ Mon, 02 Nov 2020 21:37:40:  9000000 \\nINFO  @ Mon, 02 Nov 2020 21:37:45:  10000000 \\nINFO  @ Mon, 02 Nov 2020 21:37:50:  11000000 \\nINFO  @ Mon, 02 Nov 2020 21:38:01:  12000000 \\nINFO  @ Mon, 02 Nov 2020 21:38:06:  13000000 \\nINFO  @ Mon, 02 Nov 2020 21:38:11:  14000000 \\nINFO  @ Mon, 02 Nov 2020 21:38:17:  15000000 \\nINFO  @ Mon, 02 Nov 2020 21:38:22:  16000000 \\nINFO  @ Mon, 02 Nov 2020 21:38:26:  17000000 \\nINFO  @ Mon, 02 Nov 2020 21:38:39:  18000000 \\nINFO  @ Mon, 02 Nov 2020 21:38:44:  19000000 \\nINFO  @ Mon, 02 Nov 2020 21:38:49:  20000000 \\nINFO  @ Mon, 02 Nov 2020 21:38:54:  21000000 \\nINFO  @ Mon, 02 Nov 2020 21:39:00:  22000000 \\nINFO  @ Mon, 02 Nov 2020 21:39:06:  23000000 \\nINFO  @ Mon, 02 Nov 2020 21:39:15:  24000000 \\nINFO  @ Mon, 02 Nov 2020 21:39:21:  25000000 \\nINFO  @ Mon, 02 Nov 2020 21:39:26:  26000000 \\nINFO  @ Mon, 02 Nov 2020 21:39:29: 26542203 fragments have been read. \\nINFO  @ Mon, 02 Nov 2020 21:39:44: #1.2 read input fragments... \\nINFO  @ Mon, 02 Nov 2020 21:39:52:  1000000 \\nINFO  @ Mon, 02 Nov 2020 21:39:56:  2000000 \\nINFO  @ Mon, 02 Nov 2020 21:40:00:  3000000 \\nINFO  @ Mon, 02 Nov 2020 21:40:03:  4000000 \\nINFO  @ Mon, 02 Nov 2020 21:40:07:  5000000 \\nINFO  @ Mon, 02 Nov 2020 21:40:11:  6000000 \\nINFO  @ Mon, 02 Nov 2020 21:40:15:  7000000 \\nINFO  @ Mon, 02 Nov 2020 21:40:20:  8000000 \\nINFO  @ Mon, 02 Nov 2020 21:40:24:  9000000 \\nINFO  @ Mon, 02 Nov 2020 21:40:28:  10000000 \\nINFO  @ Mon, 02 Nov 2020 21:40:32:  11000000 \\nINFO  @ Mon, 02 Nov 2020 21:40:36:  12000000 \\nINFO  @ Mon, 02 Nov 2020 21:40:39:  13000000 \\nINFO  @ Mon, 02 Nov 2020 21:40:43:  14000000 \\nINFO  @ Mon, 02 Nov 2020 21:40:47:  15000000 \\nINFO  @ Mon, 02 Nov 2020 21:40:51:  16000000 \\nINFO  @ Mon, 02 Nov 2020 21:40:55:  17000000 \\nINFO  @ Mon, 02 Nov 2020 21:40:58:  18000000 \\nINFO  @ Mon, 02 Nov 2020 21:41:02:  19000000 \\nINFO  @ Mon, 02 Nov 2020 21:41:06:  20000000 \\nINFO  @ Mon, 02 Nov 2020 21:41:10:  21000000 \\nINFO  @ Mon, 02 Nov 2020 21:41:14:  22000000 \\nINFO  @ Mon, 02 Nov 2020 21:41:18:  23000000 \\nINFO  @ Mon, 02 Nov 2020 21:41:22:  24000000 \\nINFO  @ Mon, 02 Nov 2020 21:41:26:  25000000 \\nINFO  @ Mon, 02 Nov 2020 21:41:30:  26000000 \\nINFO  @ Mon, 02 Nov 2020 21:41:34:  27000000 \\nINFO  @ Mon, 02 Nov 2020 21:41:38:  28000000 \\nINFO  @ Mon, 02 Nov 2020 21:41:42:  29000000 \\nINFO  @ Mon, 02 Nov 2020 21:41:45:  30000000 \\nINFO  @ Mon, 02 Nov 2020 21:41:49:  31000000 \\nINFO  @ Mon, 02 Nov 2020 21:41:53:  32000000 \\nINFO  @ Mon, 02 Nov 2020 21:41:57:  33000000 \\nINFO  @ Mon, 02 Nov 2020 21:42:01:  34000000 \\nINFO  @ Mon, 02 Nov 2020 21:42:05:  35000000 \\nINFO  @ Mon, 02 Nov 2020 21:42:09:  36000000 \\nINFO  @ Mon, 02 Nov 2020 21:42:13:  37000000 \\nINFO  @ Mon, 02 Nov 2020 21:42:17:  38000000 \\nINFO  @ Mon, 02 Nov 2020 21:42:20:  39000000 \\nINFO  @ Mon, 02 Nov 2020 21:42:25:  40000000 \\nINFO  @ Mon, 02 Nov 2020 21:42:29:  41000000 \\nINFO  @ Mon, 02 Nov 2020 21:42:32:  42000000 \\nINFO  @ Mon, 02 Nov 2020 21:42:36:  43000000 \\nINFO  @ Mon, 02 Nov 2020 21:42:40:  44000000 \\nINFO  @ Mon, 02 Nov 2020 21:42:44:  45000000 \\nINFO  @ Mon, 02 Nov 2020 21:42:48:  46000000 \\nINFO  @ Mon, 02 Nov 2020 21:42:52:  47000000 \\nINFO  @ Mon, 02 Nov 2020 21:42:55:  48000000 \\nINFO  @ Mon, 02 Nov 2020 21:42:59:  49000000 \\nINFO  @ Mon, 02 Nov 2020 21:43:03:  50000000 \\nINFO  @ Mon, 02 Nov 2020 21:43:06:  51000000 \\nINFO  @ Mon, 02 Nov 2020 21:43:10:  52000000 \\nINFO  @ Mon, 02 Nov 2020 21:43:14:  53000000 \\nINFO  @ Mon, 02 Nov 2020 21:43:19:  54000000 \\nINFO  @ Mon, 02 Nov 2020 21:43:23:  55000000 \\nINFO  @ Mon, 02 Nov 2020 21:43:26:  56000000 \\nINFO  @ Mon, 02 Nov 2020 21:43:30:  57000000 \\nINFO  @ Mon, 02 Nov 2020 21:43:34:  58000000 \\nINFO  @ Mon, 02 Nov 2020 21:43:38:  59000000 \\nINFO  @ Mon, 02 Nov 2020 21:43:41:  60000000 \\nINFO  @ Mon, 02 Nov 2020 21:43:45:  61000000 \\nINFO  @ Mon, 02 Nov 2020 21:43:49:  62000000 \\nINFO  @ Mon, 02 Nov 2020 21:43:53:  63000000 \\nINFO  @ Mon, 02 Nov 2020 21:43:57:  64000000 \\nINFO  @ Mon, 02 Nov 2020 21:44:02:  65000000 \\nINFO  @ Mon, 02 Nov 2020 21:44:05:  66000000 \\nINFO  @ Mon, 02 Nov 2020 21:44:08: 66660428 fragments have been read. \\nINFO  @ Mon, 02 Nov 2020 21:44:54: #1 mean fragment size is determined as 250.9 bp from treatment \\nINFO  @ Mon, 02 Nov 2020 21:44:54: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Mon, 02 Nov 2020 21:44:54: #1 fragment size = 250.9 \\nINFO  @ Mon, 02 Nov 2020 21:44:54: #1  total fragments in treatment: 26542203 \\nINFO  @ Mon, 02 Nov 2020 21:44:54: #1 user defined the maximum fragments... \\nINFO  @ Mon, 02 Nov 2020 21:44:54: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 02 Nov 2020 21:45:38: #1  fragments after filtering in treatment: 26510732 \\nINFO  @ Mon, 02 Nov 2020 21:45:38: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 02 Nov 2020 21:45:38: #1  total fragments in control: 66660428 \\nINFO  @ Mon, 02 Nov 2020 21:45:38: #1 user defined the maximum fragments... \\nINFO  @ Mon, 02 Nov 2020 21:45:38: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 02 Nov 2020 21:47:28: #1  fragments after filtering in control: 66632618 \\nINFO  @ Mon, 02 Nov 2020 21:47:28: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 02 Nov 2020 21:47:28: #1 finished! \\nINFO  @ Mon, 02 Nov 2020 21:47:28: #2 Build Peak Model... \\nINFO  @ Mon, 02 Nov 2020 21:47:28: #2 Skipped... \\nINFO  @ Mon, 02 Nov 2020 21:47:28: #3 Call peaks... \\nINFO  @ Mon, 02 Nov 2020 21:47:28: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 02 Nov 2020 21:57:22: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 02 Nov 2020 21:57:22: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_treat_pileup.bdg \\nINFO  @ Mon, 02 Nov 2020 21:57:22: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_control_lambda.bdg \\nINFO  @ Mon, 02 Nov 2020 21:57:22: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 02 Nov 2020 21:57:22: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 02 Nov 2020 22:03:53: #4 Write output xls file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_peaks.xls \\nINFO  @ Mon, 02 Nov 2020 22:03:53: #4 Write peak in narrowPeak format file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_peaks.narrowPeak \\nINFO  @ Mon, 02 Nov 2020 22:03:53: #4 Write summits bed file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_summits.bed \\nINFO  @ Mon, 02 Nov 2020 22:03:53: Done! \\n'\n",
      "66632618 67047491\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_treat_pileup.bdg --c1 ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_control_lambda.bdg --t2 ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_treat_pileup.bdg --c2 ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_control_lambda.bdg --d1 66632618 --d2 67047491 -g 60 -l 230 --o-prefix MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_treat_pileup --outdir ../../data/chipseq_MEF2D_degraded/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Mon, 02 Nov 2020 22:03:53: Read and build treatment 1 bedGraph... \\nINFO  @ Mon, 02 Nov 2020 22:07:28: Read and build control 1 bedGraph... \\nINFO  @ Mon, 02 Nov 2020 22:12:02: Read and build treatment 2 bedGraph... \\nINFO  @ Mon, 02 Nov 2020 22:12:53: Read and build control 2 bedGraph... \\nINFO  @ Mon, 02 Nov 2020 22:27:56: Write peaks... \\nINFO  @ Mon, 02 Nov 2020 22:27:56: Done \\n')\n",
      "\n",
      "\n",
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1.mLb.clN.sorted.bam ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1.mLb.clN.sorted.bam and ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "208\n",
      "computing the scaling values\n",
      "b'INFO  @ Mon, 02 Nov 2020 22:31:57: \\n# Command line: callpeak -B -t ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1.mLb.clN.sorted.bam -c ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 208 -n MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1 --outdir ../../data/chipseq_MEF2D_degraded/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Mon, 02 Nov 2020 22:31:57: #1 read fragment files... \\nINFO  @ Mon, 02 Nov 2020 22:31:57: #1 read treatment fragments... \\nINFO  @ Mon, 02 Nov 2020 22:32:02:  1000000 \\nINFO  @ Mon, 02 Nov 2020 22:32:07:  2000000 \\nINFO  @ Mon, 02 Nov 2020 22:32:11:  3000000 \\nINFO  @ Mon, 02 Nov 2020 22:32:16:  4000000 \\nINFO  @ Mon, 02 Nov 2020 22:32:21:  5000000 \\nINFO  @ Mon, 02 Nov 2020 22:32:26:  6000000 \\nINFO  @ Mon, 02 Nov 2020 22:32:31:  7000000 \\nINFO  @ Mon, 02 Nov 2020 22:32:36:  8000000 \\nINFO  @ Mon, 02 Nov 2020 22:32:40:  9000000 \\nINFO  @ Mon, 02 Nov 2020 22:32:45:  10000000 \\nINFO  @ Mon, 02 Nov 2020 22:32:49:  11000000 \\nINFO  @ Mon, 02 Nov 2020 22:32:54:  12000000 \\nINFO  @ Mon, 02 Nov 2020 22:32:59:  13000000 \\nINFO  @ Mon, 02 Nov 2020 22:33:03:  14000000 \\nINFO  @ Mon, 02 Nov 2020 22:33:08:  15000000 \\nINFO  @ Mon, 02 Nov 2020 22:33:13:  16000000 \\nINFO  @ Mon, 02 Nov 2020 22:33:17:  17000000 \\nINFO  @ Mon, 02 Nov 2020 22:33:22:  18000000 \\nINFO  @ Mon, 02 Nov 2020 22:33:27:  19000000 \\nINFO  @ Mon, 02 Nov 2020 22:33:31:  20000000 \\nINFO  @ Mon, 02 Nov 2020 22:33:36:  21000000 \\nINFO  @ Mon, 02 Nov 2020 22:33:41:  22000000 \\nINFO  @ Mon, 02 Nov 2020 22:33:45:  23000000 \\nINFO  @ Mon, 02 Nov 2020 22:33:50:  24000000 \\nINFO  @ Mon, 02 Nov 2020 22:33:54:  25000000 \\nINFO  @ Mon, 02 Nov 2020 22:33:59:  26000000 \\nINFO  @ Mon, 02 Nov 2020 22:34:04:  27000000 \\nINFO  @ Mon, 02 Nov 2020 22:34:08:  28000000 \\nINFO  @ Mon, 02 Nov 2020 22:34:13:  29000000 \\nINFO  @ Mon, 02 Nov 2020 22:34:18:  30000000 \\nINFO  @ Mon, 02 Nov 2020 22:34:22:  31000000 \\nINFO  @ Mon, 02 Nov 2020 22:34:27:  32000000 \\nINFO  @ Mon, 02 Nov 2020 22:34:31:  33000000 \\nINFO  @ Mon, 02 Nov 2020 22:34:36:  34000000 \\nINFO  @ Mon, 02 Nov 2020 22:34:41:  35000000 \\nINFO  @ Mon, 02 Nov 2020 22:34:45:  36000000 \\nINFO  @ Mon, 02 Nov 2020 22:34:50:  37000000 \\nINFO  @ Mon, 02 Nov 2020 22:34:54:  38000000 \\nINFO  @ Mon, 02 Nov 2020 22:34:59:  39000000 \\nINFO  @ Mon, 02 Nov 2020 22:35:04:  40000000 \\nINFO  @ Mon, 02 Nov 2020 22:35:08:  41000000 \\nINFO  @ Mon, 02 Nov 2020 22:35:13:  42000000 \\nINFO  @ Mon, 02 Nov 2020 22:35:18:  43000000 \\nINFO  @ Mon, 02 Nov 2020 22:35:22:  44000000 \\nINFO  @ Mon, 02 Nov 2020 22:35:25: 44433177 fragments have been read. \\nINFO  @ Mon, 02 Nov 2020 22:35:53: #1.2 read input fragments... \\nINFO  @ Mon, 02 Nov 2020 22:35:57:  1000000 \\nINFO  @ Mon, 02 Nov 2020 22:36:01:  2000000 \\nINFO  @ Mon, 02 Nov 2020 22:36:05:  3000000 \\nINFO  @ Mon, 02 Nov 2020 22:36:09:  4000000 \\nINFO  @ Mon, 02 Nov 2020 22:36:17:  5000000 \\nINFO  @ Mon, 02 Nov 2020 22:36:21:  6000000 \\nINFO  @ Mon, 02 Nov 2020 22:36:25:  7000000 \\nINFO  @ Mon, 02 Nov 2020 22:36:29:  8000000 \\nINFO  @ Mon, 02 Nov 2020 22:36:34:  9000000 \\nINFO  @ Mon, 02 Nov 2020 22:36:38:  10000000 \\nINFO  @ Mon, 02 Nov 2020 22:36:42:  11000000 \\nINFO  @ Mon, 02 Nov 2020 22:36:49:  12000000 \\nINFO  @ Mon, 02 Nov 2020 22:36:53:  13000000 \\nINFO  @ Mon, 02 Nov 2020 22:36:57:  14000000 \\nINFO  @ Mon, 02 Nov 2020 22:37:01:  15000000 \\nINFO  @ Mon, 02 Nov 2020 22:37:05:  16000000 \\nINFO  @ Mon, 02 Nov 2020 22:37:09:  17000000 \\nINFO  @ Mon, 02 Nov 2020 22:37:13:  18000000 \\nINFO  @ Mon, 02 Nov 2020 22:37:17:  19000000 \\nINFO  @ Mon, 02 Nov 2020 22:37:25:  20000000 \\nINFO  @ Mon, 02 Nov 2020 22:37:29:  21000000 \\nINFO  @ Mon, 02 Nov 2020 22:37:33:  22000000 \\nINFO  @ Mon, 02 Nov 2020 22:37:36:  23000000 \\nINFO  @ Mon, 02 Nov 2020 22:37:40:  24000000 \\nINFO  @ Mon, 02 Nov 2020 22:37:44:  25000000 \\nINFO  @ Mon, 02 Nov 2020 22:37:48:  26000000 \\nINFO  @ Mon, 02 Nov 2020 22:37:52:  27000000 \\nINFO  @ Mon, 02 Nov 2020 22:37:59:  28000000 \\nINFO  @ Mon, 02 Nov 2020 22:38:02:  29000000 \\nINFO  @ Mon, 02 Nov 2020 22:38:06:  30000000 \\nINFO  @ Mon, 02 Nov 2020 22:38:10:  31000000 \\nINFO  @ Mon, 02 Nov 2020 22:38:14:  32000000 \\nINFO  @ Mon, 02 Nov 2020 22:38:18:  33000000 \\nINFO  @ Mon, 02 Nov 2020 22:38:26:  34000000 \\nINFO  @ Mon, 02 Nov 2020 22:38:30:  35000000 \\nINFO  @ Mon, 02 Nov 2020 22:38:34:  36000000 \\nINFO  @ Mon, 02 Nov 2020 22:38:38:  37000000 \\nINFO  @ Mon, 02 Nov 2020 22:38:41:  38000000 \\nINFO  @ Mon, 02 Nov 2020 22:38:45:  39000000 \\nINFO  @ Mon, 02 Nov 2020 22:38:49:  40000000 \\nINFO  @ Mon, 02 Nov 2020 22:38:54:  41000000 \\nINFO  @ Mon, 02 Nov 2020 22:39:01:  42000000 \\nINFO  @ Mon, 02 Nov 2020 22:39:05:  43000000 \\nINFO  @ Mon, 02 Nov 2020 22:39:09:  44000000 \\nINFO  @ Mon, 02 Nov 2020 22:39:14:  45000000 \\nINFO  @ Mon, 02 Nov 2020 22:39:18:  46000000 \\nINFO  @ Mon, 02 Nov 2020 22:39:22:  47000000 \\nINFO  @ Mon, 02 Nov 2020 22:39:25:  48000000 \\nINFO  @ Mon, 02 Nov 2020 22:39:29:  49000000 \\nINFO  @ Mon, 02 Nov 2020 22:39:37:  50000000 \\nINFO  @ Mon, 02 Nov 2020 22:39:41:  51000000 \\nINFO  @ Mon, 02 Nov 2020 22:39:45:  52000000 \\nINFO  @ Mon, 02 Nov 2020 22:39:49:  53000000 \\nINFO  @ Mon, 02 Nov 2020 22:39:53:  54000000 \\nINFO  @ Mon, 02 Nov 2020 22:39:57:  55000000 \\nINFO  @ Mon, 02 Nov 2020 22:40:01:  56000000 \\nINFO  @ Mon, 02 Nov 2020 22:40:09:  57000000 \\nINFO  @ Mon, 02 Nov 2020 22:40:13:  58000000 \\nINFO  @ Mon, 02 Nov 2020 22:40:17:  59000000 \\nINFO  @ Mon, 02 Nov 2020 22:40:21:  60000000 \\nINFO  @ Mon, 02 Nov 2020 22:40:25:  61000000 \\nINFO  @ Mon, 02 Nov 2020 22:40:29:  62000000 \\nINFO  @ Mon, 02 Nov 2020 22:40:32:  63000000 \\nINFO  @ Mon, 02 Nov 2020 22:40:37:  64000000 \\nINFO  @ Mon, 02 Nov 2020 22:40:45:  65000000 \\nINFO  @ Mon, 02 Nov 2020 22:40:49:  66000000 \\nINFO  @ Mon, 02 Nov 2020 22:40:52: 66660428 fragments have been read. \\nINFO  @ Mon, 02 Nov 2020 22:41:37: #1 mean fragment size is determined as 274.4 bp from treatment \\nINFO  @ Mon, 02 Nov 2020 22:41:37: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Mon, 02 Nov 2020 22:41:37: #1 fragment size = 274.4 \\nINFO  @ Mon, 02 Nov 2020 22:41:37: #1  total fragments in treatment: 44433177 \\nINFO  @ Mon, 02 Nov 2020 22:41:37: #1 user defined the maximum fragments... \\nINFO  @ Mon, 02 Nov 2020 22:41:37: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 02 Nov 2020 22:42:55: #1  fragments after filtering in treatment: 44411793 \\nINFO  @ Mon, 02 Nov 2020 22:42:55: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 02 Nov 2020 22:42:55: #1  total fragments in control: 66660428 \\nINFO  @ Mon, 02 Nov 2020 22:42:55: #1 user defined the maximum fragments... \\nINFO  @ Mon, 02 Nov 2020 22:42:55: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 02 Nov 2020 22:44:54: #1  fragments after filtering in control: 66632618 \\nINFO  @ Mon, 02 Nov 2020 22:44:54: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 02 Nov 2020 22:44:54: #1 finished! \\nINFO  @ Mon, 02 Nov 2020 22:44:54: #2 Build Peak Model... \\nINFO  @ Mon, 02 Nov 2020 22:44:54: #2 Skipped... \\nINFO  @ Mon, 02 Nov 2020 22:44:54: #3 Call peaks... \\nINFO  @ Mon, 02 Nov 2020 22:44:54: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 02 Nov 2020 22:55:38: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 02 Nov 2020 22:55:38: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_treat_pileup.bdg \\nINFO  @ Mon, 02 Nov 2020 22:55:38: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_control_lambda.bdg \\nINFO  @ Mon, 02 Nov 2020 22:55:38: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 02 Nov 2020 22:55:38: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 02 Nov 2020 23:02:51: #4 Write output xls file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_peaks.xls \\nINFO  @ Mon, 02 Nov 2020 23:02:51: #4 Write peak in narrowPeak format file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_peaks.narrowPeak \\nINFO  @ Mon, 02 Nov 2020 23:02:51: #4 Write summits bed file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_summits.bed \\nINFO  @ Mon, 02 Nov 2020 23:02:51: Done! \\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'INFO  @ Mon, 02 Nov 2020 23:02:52: \\n# Command line: callpeak -B -t ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1.mLb.clN.sorted.bam -c ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 208 -n MV411_MEF2D_NT_SC_63_VHL-IFR8_R1 --outdir ../../data/chipseq_MEF2D_degraded/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_MEF2D_NT_SC_63_VHL-IFR8_R1\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Mon, 02 Nov 2020 23:02:52: #1 read fragment files... \\nINFO  @ Mon, 02 Nov 2020 23:02:52: #1 read treatment fragments... \\nINFO  @ Mon, 02 Nov 2020 23:03:00:  1000000 \\nINFO  @ Mon, 02 Nov 2020 23:03:05:  2000000 \\nINFO  @ Mon, 02 Nov 2020 23:03:15:  3000000 \\nINFO  @ Mon, 02 Nov 2020 23:03:19:  4000000 \\nINFO  @ Mon, 02 Nov 2020 23:03:24:  5000000 \\nINFO  @ Mon, 02 Nov 2020 23:03:31:  6000000 \\nINFO  @ Mon, 02 Nov 2020 23:03:36:  7000000 \\nINFO  @ Mon, 02 Nov 2020 23:03:41:  8000000 \\nINFO  @ Mon, 02 Nov 2020 23:03:51:  9000000 \\nINFO  @ Mon, 02 Nov 2020 23:03:55:  10000000 \\nINFO  @ Mon, 02 Nov 2020 23:03:59:  11000000 \\nINFO  @ Mon, 02 Nov 2020 23:04:06:  12000000 \\nINFO  @ Mon, 02 Nov 2020 23:04:11:  13000000 \\nINFO  @ Mon, 02 Nov 2020 23:04:16:  14000000 \\nINFO  @ Mon, 02 Nov 2020 23:04:20:  15000000 \\nINFO  @ Mon, 02 Nov 2020 23:04:30:  16000000 \\nINFO  @ Mon, 02 Nov 2020 23:04:37:  17000000 \\nINFO  @ Mon, 02 Nov 2020 23:04:42:  18000000 \\nINFO  @ Mon, 02 Nov 2020 23:04:47:  19000000 \\nINFO  @ Mon, 02 Nov 2020 23:04:51:  20000000 \\nINFO  @ Mon, 02 Nov 2020 23:04:56:  21000000 \\nINFO  @ Mon, 02 Nov 2020 23:05:00:  22000000 \\nINFO  @ Mon, 02 Nov 2020 23:05:13:  23000000 \\nINFO  @ Mon, 02 Nov 2020 23:05:18:  24000000 \\nINFO  @ Mon, 02 Nov 2020 23:05:23:  25000000 \\nINFO  @ Mon, 02 Nov 2020 23:05:27:  26000000 \\nINFO  @ Mon, 02 Nov 2020 23:05:32:  27000000 \\nINFO  @ Mon, 02 Nov 2020 23:05:36:  28000000 \\nINFO  @ Mon, 02 Nov 2020 23:05:41:  29000000 \\nINFO  @ Mon, 02 Nov 2020 23:05:55:  30000000 \\nINFO  @ Mon, 02 Nov 2020 23:06:00:  31000000 \\nINFO  @ Mon, 02 Nov 2020 23:06:05:  32000000 \\nINFO  @ Mon, 02 Nov 2020 23:06:10:  33000000 \\nINFO  @ Mon, 02 Nov 2020 23:06:14:  34000000 \\nINFO  @ Mon, 02 Nov 2020 23:06:19:  35000000 \\nINFO  @ Mon, 02 Nov 2020 23:06:29:  36000000 \\nINFO  @ Mon, 02 Nov 2020 23:06:37:  37000000 \\nINFO  @ Mon, 02 Nov 2020 23:06:42:  38000000 \\nINFO  @ Mon, 02 Nov 2020 23:06:46:  39000000 \\nINFO  @ Mon, 02 Nov 2020 23:06:51:  40000000 \\nINFO  @ Mon, 02 Nov 2020 23:06:55:  41000000 \\nINFO  @ Mon, 02 Nov 2020 23:07:00:  42000000 \\nINFO  @ Mon, 02 Nov 2020 23:07:04:  43000000 \\nINFO  @ Mon, 02 Nov 2020 23:07:15:  44000000 \\nINFO  @ Mon, 02 Nov 2020 23:07:20:  45000000 \\nINFO  @ Mon, 02 Nov 2020 23:07:24:  46000000 \\nINFO  @ Mon, 02 Nov 2020 23:07:28:  47000000 \\nINFO  @ Mon, 02 Nov 2020 23:07:33:  48000000 \\nINFO  @ Mon, 02 Nov 2020 23:07:38:  49000000 \\nINFO  @ Mon, 02 Nov 2020 23:07:43:  50000000 \\nINFO  @ Mon, 02 Nov 2020 23:07:55:  51000000 \\nINFO  @ Mon, 02 Nov 2020 23:08:00:  52000000 \\nINFO  @ Mon, 02 Nov 2020 23:08:05:  53000000 \\nINFO  @ Mon, 02 Nov 2020 23:08:09:  54000000 \\nINFO  @ Mon, 02 Nov 2020 23:08:14:  55000000 \\nINFO  @ Mon, 02 Nov 2020 23:08:19:  56000000 \\nINFO  @ Mon, 02 Nov 2020 23:08:23:  57000000 \\nINFO  @ Mon, 02 Nov 2020 23:08:32:  58000000 \\nINFO  @ Mon, 02 Nov 2020 23:08:37:  59000000 \\nINFO  @ Mon, 02 Nov 2020 23:08:42:  60000000 \\nINFO  @ Mon, 02 Nov 2020 23:08:47:  61000000 \\nINFO  @ Mon, 02 Nov 2020 23:08:51:  62000000 \\nINFO  @ Mon, 02 Nov 2020 23:08:55:  63000000 \\nINFO  @ Mon, 02 Nov 2020 23:09:04:  64000000 \\nINFO  @ Mon, 02 Nov 2020 23:09:09:  65000000 \\nINFO  @ Mon, 02 Nov 2020 23:09:13:  66000000 \\nINFO  @ Mon, 02 Nov 2020 23:09:18:  67000000 \\nINFO  @ Mon, 02 Nov 2020 23:09:23:  68000000 \\nINFO  @ Mon, 02 Nov 2020 23:09:27:  69000000 \\nINFO  @ Mon, 02 Nov 2020 23:09:29: 69337135 fragments have been read. \\nINFO  @ Mon, 02 Nov 2020 23:10:17: #1.2 read input fragments... \\nINFO  @ Mon, 02 Nov 2020 23:10:21:  1000000 \\nINFO  @ Mon, 02 Nov 2020 23:10:24:  2000000 \\nINFO  @ Mon, 02 Nov 2020 23:10:28:  3000000 \\nINFO  @ Mon, 02 Nov 2020 23:10:32:  4000000 \\nINFO  @ Mon, 02 Nov 2020 23:10:36:  5000000 \\nINFO  @ Mon, 02 Nov 2020 23:10:40:  6000000 \\nINFO  @ Mon, 02 Nov 2020 23:10:43:  7000000 \\nINFO  @ Mon, 02 Nov 2020 23:10:51:  8000000 \\nINFO  @ Mon, 02 Nov 2020 23:10:55:  9000000 \\nINFO  @ Mon, 02 Nov 2020 23:10:59:  10000000 \\nINFO  @ Mon, 02 Nov 2020 23:11:02:  11000000 \\nINFO  @ Mon, 02 Nov 2020 23:11:06:  12000000 \\nINFO  @ Mon, 02 Nov 2020 23:11:10:  13000000 \\nINFO  @ Mon, 02 Nov 2020 23:11:14:  14000000 \\nINFO  @ Mon, 02 Nov 2020 23:11:17:  15000000 \\nINFO  @ Mon, 02 Nov 2020 23:11:25:  16000000 \\nINFO  @ Mon, 02 Nov 2020 23:11:29:  17000000 \\nINFO  @ Mon, 02 Nov 2020 23:11:33:  18000000 \\nINFO  @ Mon, 02 Nov 2020 23:11:36:  19000000 \\nINFO  @ Mon, 02 Nov 2020 23:11:41:  20000000 \\nINFO  @ Mon, 02 Nov 2020 23:11:45:  21000000 \\nINFO  @ Mon, 02 Nov 2020 23:11:49:  22000000 \\nINFO  @ Mon, 02 Nov 2020 23:11:57:  23000000 \\nINFO  @ Mon, 02 Nov 2020 23:12:01:  24000000 \\nINFO  @ Mon, 02 Nov 2020 23:12:04:  25000000 \\nINFO  @ Mon, 02 Nov 2020 23:12:08:  26000000 \\nINFO  @ Mon, 02 Nov 2020 23:12:12:  27000000 \\nINFO  @ Mon, 02 Nov 2020 23:12:16:  28000000 \\nINFO  @ Mon, 02 Nov 2020 23:12:20:  29000000 \\nINFO  @ Mon, 02 Nov 2020 23:12:24:  30000000 \\nINFO  @ Mon, 02 Nov 2020 23:12:32:  31000000 \\nINFO  @ Mon, 02 Nov 2020 23:12:35:  32000000 \\nINFO  @ Mon, 02 Nov 2020 23:12:39:  33000000 \\nINFO  @ Mon, 02 Nov 2020 23:12:43:  34000000 \\nINFO  @ Mon, 02 Nov 2020 23:12:47:  35000000 \\nINFO  @ Mon, 02 Nov 2020 23:12:51:  36000000 \\nINFO  @ Mon, 02 Nov 2020 23:12:55:  37000000 \\nINFO  @ Mon, 02 Nov 2020 23:12:59:  38000000 \\nINFO  @ Mon, 02 Nov 2020 23:13:06:  39000000 \\nINFO  @ Mon, 02 Nov 2020 23:13:10:  40000000 \\nINFO  @ Mon, 02 Nov 2020 23:13:14:  41000000 \\nINFO  @ Mon, 02 Nov 2020 23:13:18:  42000000 \\nINFO  @ Mon, 02 Nov 2020 23:13:23:  43000000 \\nINFO  @ Mon, 02 Nov 2020 23:13:27:  44000000 \\nINFO  @ Mon, 02 Nov 2020 23:13:30:  45000000 \\nINFO  @ Mon, 02 Nov 2020 23:13:34:  46000000 \\nINFO  @ Mon, 02 Nov 2020 23:13:42:  47000000 \\nINFO  @ Mon, 02 Nov 2020 23:13:46:  48000000 \\nINFO  @ Mon, 02 Nov 2020 23:13:50:  49000000 \\nINFO  @ Mon, 02 Nov 2020 23:13:54:  50000000 \\nINFO  @ Mon, 02 Nov 2020 23:13:58:  51000000 \\nINFO  @ Mon, 02 Nov 2020 23:14:02:  52000000 \\nINFO  @ Mon, 02 Nov 2020 23:14:06:  53000000 \\nINFO  @ Mon, 02 Nov 2020 23:14:11:  54000000 \\nINFO  @ Mon, 02 Nov 2020 23:14:17:  55000000 \\nINFO  @ Mon, 02 Nov 2020 23:14:21:  56000000 \\nINFO  @ Mon, 02 Nov 2020 23:14:25:  57000000 \\nINFO  @ Mon, 02 Nov 2020 23:14:30:  58000000 \\nINFO  @ Mon, 02 Nov 2020 23:14:33:  59000000 \\nINFO  @ Mon, 02 Nov 2020 23:14:37:  60000000 \\nINFO  @ Mon, 02 Nov 2020 23:14:42:  61000000 \\nINFO  @ Mon, 02 Nov 2020 23:14:50:  62000000 \\nINFO  @ Mon, 02 Nov 2020 23:14:53:  63000000 \\nINFO  @ Mon, 02 Nov 2020 23:14:57:  64000000 \\nINFO  @ Mon, 02 Nov 2020 23:15:01:  65000000 \\nINFO  @ Mon, 02 Nov 2020 23:15:05:  66000000 \\nINFO  @ Mon, 02 Nov 2020 23:15:08: 66660428 fragments have been read. \\nINFO  @ Mon, 02 Nov 2020 23:15:53: #1 mean fragment size is determined as 258.9 bp from treatment \\nINFO  @ Mon, 02 Nov 2020 23:15:53: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Mon, 02 Nov 2020 23:15:53: #1 fragment size = 258.9 \\nINFO  @ Mon, 02 Nov 2020 23:15:53: #1  total fragments in treatment: 69337135 \\nINFO  @ Mon, 02 Nov 2020 23:15:53: #1 user defined the maximum fragments... \\nINFO  @ Mon, 02 Nov 2020 23:15:53: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 02 Nov 2020 23:17:50: #1  fragments after filtering in treatment: 69298067 \\nINFO  @ Mon, 02 Nov 2020 23:17:50: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 02 Nov 2020 23:17:50: #1  total fragments in control: 66660428 \\nINFO  @ Mon, 02 Nov 2020 23:17:50: #1 user defined the maximum fragments... \\nINFO  @ Mon, 02 Nov 2020 23:17:50: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Mon, 02 Nov 2020 23:19:40: #1  fragments after filtering in control: 66632618 \\nINFO  @ Mon, 02 Nov 2020 23:19:40: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 02 Nov 2020 23:19:40: #1 finished! \\nINFO  @ Mon, 02 Nov 2020 23:19:40: #2 Build Peak Model... \\nINFO  @ Mon, 02 Nov 2020 23:19:40: #2 Skipped... \\nINFO  @ Mon, 02 Nov 2020 23:19:40: #3 Call peaks... \\nINFO  @ Mon, 02 Nov 2020 23:19:40: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 02 Nov 2020 23:31:14: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 02 Nov 2020 23:31:14: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_treat_pileup.bdg \\nINFO  @ Mon, 02 Nov 2020 23:31:14: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_control_lambda.bdg \\nINFO  @ Mon, 02 Nov 2020 23:31:14: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 02 Nov 2020 23:31:14: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 02 Nov 2020 23:40:57: #4 Write output xls file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_peaks.xls \\nINFO  @ Mon, 02 Nov 2020 23:40:58: #4 Write peak in narrowPeak format file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_peaks.narrowPeak \\nINFO  @ Mon, 02 Nov 2020 23:40:58: #4 Write summits bed file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_summits.bed \\nINFO  @ Mon, 02 Nov 2020 23:40:58: Done! \\n'\n",
      "44411793 79618485\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_treat_pileup.bdg --c1 ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_control_lambda.bdg --t2 ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_treat_pileup.bdg --c2 ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_control_lambda.bdg --d1 44411793 --d2 79618485 -g 60 -l 208 --o-prefix MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_treat_pileup --outdir ../../data/chipseq_MEF2D_degraded/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Mon, 02 Nov 2020 23:40:59: Read and build treatment 1 bedGraph... \\nINFO  @ Mon, 02 Nov 2020 23:42:35: Read and build control 1 bedGraph... \\nINFO  @ Mon, 02 Nov 2020 23:46:44: Read and build treatment 2 bedGraph... \\nINFO  @ Mon, 02 Nov 2020 23:48:42: Read and build control 2 bedGraph... \\nINFO  @ Tue, 03 Nov 2020 00:04:19: Write peaks... \\nINFO  @ Tue, 03 Nov 2020 00:04:19: Done \\n')\n",
      "\n",
      "\n",
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1.mLb.clN.sorted.bam ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-MED1_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1.mLb.clN.sorted.bam and ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-MED1_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "220\n",
      "computing the scaling values\n",
      "b'INFO  @ Tue, 03 Nov 2020 00:09:36: \\n# Command line: callpeak -B -t ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1.mLb.clN.sorted.bam -c ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 220 -n MV411_MEF2D_NT_SC_63_DMSO-MED1_R1 --outdir ../../data/chipseq_MEF2D_degraded/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_MEF2D_NT_SC_63_DMSO-MED1_R1\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Tue, 03 Nov 2020 00:09:36: #1 read fragment files... \\nINFO  @ Tue, 03 Nov 2020 00:09:36: #1 read treatment fragments... \\nINFO  @ Tue, 03 Nov 2020 00:09:41:  1000000 \\nINFO  @ Tue, 03 Nov 2020 00:09:45:  2000000 \\nINFO  @ Tue, 03 Nov 2020 00:09:50:  3000000 \\nINFO  @ Tue, 03 Nov 2020 00:09:55:  4000000 \\nINFO  @ Tue, 03 Nov 2020 00:09:59:  5000000 \\nINFO  @ Tue, 03 Nov 2020 00:10:04:  6000000 \\nINFO  @ Tue, 03 Nov 2020 00:10:08:  7000000 \\nINFO  @ Tue, 03 Nov 2020 00:10:13:  8000000 \\nINFO  @ Tue, 03 Nov 2020 00:10:17:  9000000 \\nINFO  @ Tue, 03 Nov 2020 00:10:22:  10000000 \\nINFO  @ Tue, 03 Nov 2020 00:10:27:  11000000 \\nINFO  @ Tue, 03 Nov 2020 00:10:31:  12000000 \\nINFO  @ Tue, 03 Nov 2020 00:10:36:  13000000 \\nINFO  @ Tue, 03 Nov 2020 00:10:40:  14000000 \\nINFO  @ Tue, 03 Nov 2020 00:10:45:  15000000 \\nINFO  @ Tue, 03 Nov 2020 00:10:49:  16000000 \\nINFO  @ Tue, 03 Nov 2020 00:10:54:  17000000 \\nINFO  @ Tue, 03 Nov 2020 00:10:58:  18000000 \\nINFO  @ Tue, 03 Nov 2020 00:11:03:  19000000 \\nINFO  @ Tue, 03 Nov 2020 00:11:08:  20000000 \\nINFO  @ Tue, 03 Nov 2020 00:11:12:  21000000 \\nINFO  @ Tue, 03 Nov 2020 00:11:17:  22000000 \\nINFO  @ Tue, 03 Nov 2020 00:11:22:  23000000 \\nINFO  @ Tue, 03 Nov 2020 00:11:27:  24000000 \\nINFO  @ Tue, 03 Nov 2020 00:11:31:  25000000 \\nINFO  @ Tue, 03 Nov 2020 00:11:36:  26000000 \\nINFO  @ Tue, 03 Nov 2020 00:11:40:  27000000 \\nINFO  @ Tue, 03 Nov 2020 00:11:45:  28000000 \\nINFO  @ Tue, 03 Nov 2020 00:11:49:  29000000 \\nINFO  @ Tue, 03 Nov 2020 00:11:53:  30000000 \\nINFO  @ Tue, 03 Nov 2020 00:11:58:  31000000 \\nINFO  @ Tue, 03 Nov 2020 00:12:02:  32000000 \\nINFO  @ Tue, 03 Nov 2020 00:12:07:  33000000 \\nINFO  @ Tue, 03 Nov 2020 00:12:11:  34000000 \\nINFO  @ Tue, 03 Nov 2020 00:12:16:  35000000 \\nINFO  @ Tue, 03 Nov 2020 00:12:20:  36000000 \\nINFO  @ Tue, 03 Nov 2020 00:12:25:  37000000 \\nINFO  @ Tue, 03 Nov 2020 00:12:30:  38000000 \\nINFO  @ Tue, 03 Nov 2020 00:12:35:  39000000 \\nINFO  @ Tue, 03 Nov 2020 00:12:39:  40000000 \\nINFO  @ Tue, 03 Nov 2020 00:12:44:  41000000 \\nINFO  @ Tue, 03 Nov 2020 00:12:49:  42000000 \\nINFO  @ Tue, 03 Nov 2020 00:12:53:  43000000 \\nINFO  @ Tue, 03 Nov 2020 00:12:58:  44000000 \\nINFO  @ Tue, 03 Nov 2020 00:13:02:  45000000 \\nINFO  @ Tue, 03 Nov 2020 00:13:07:  46000000 \\nINFO  @ Tue, 03 Nov 2020 00:13:11:  47000000 \\nINFO  @ Tue, 03 Nov 2020 00:13:16:  48000000 \\nINFO  @ Tue, 03 Nov 2020 00:13:21:  49000000 \\nINFO  @ Tue, 03 Nov 2020 00:13:26:  50000000 \\nINFO  @ Tue, 03 Nov 2020 00:13:30:  51000000 \\nINFO  @ Tue, 03 Nov 2020 00:13:35:  52000000 \\nINFO  @ Tue, 03 Nov 2020 00:13:39:  53000000 \\nINFO  @ Tue, 03 Nov 2020 00:13:44:  54000000 \\nINFO  @ Tue, 03 Nov 2020 00:13:49:  55000000 \\nINFO  @ Tue, 03 Nov 2020 00:13:50: 55243421 fragments have been read. \\nINFO  @ Tue, 03 Nov 2020 00:14:26: #1.2 read input fragments... \\nINFO  @ Tue, 03 Nov 2020 00:14:30:  1000000 \\nINFO  @ Tue, 03 Nov 2020 00:14:34:  2000000 \\nINFO  @ Tue, 03 Nov 2020 00:14:41:  3000000 \\nINFO  @ Tue, 03 Nov 2020 00:14:45:  4000000 \\nINFO  @ Tue, 03 Nov 2020 00:14:48:  5000000 \\nINFO  @ Tue, 03 Nov 2020 00:14:57:  6000000 \\nINFO  @ Tue, 03 Nov 2020 00:15:01:  7000000 \\nINFO  @ Tue, 03 Nov 2020 00:15:05:  8000000 \\nINFO  @ Tue, 03 Nov 2020 00:15:09:  9000000 \\nINFO  @ Tue, 03 Nov 2020 00:15:15:  10000000 \\nINFO  @ Tue, 03 Nov 2020 00:15:19:  11000000 \\nINFO  @ Tue, 03 Nov 2020 00:15:22:  12000000 \\nINFO  @ Tue, 03 Nov 2020 00:15:29:  13000000 \\nINFO  @ Tue, 03 Nov 2020 00:15:36:  14000000 \\nINFO  @ Tue, 03 Nov 2020 00:15:41:  15000000 \\nINFO  @ Tue, 03 Nov 2020 00:15:45:  16000000 \\nINFO  @ Tue, 03 Nov 2020 00:15:49:  17000000 \\nINFO  @ Tue, 03 Nov 2020 00:15:56:  18000000 \\nINFO  @ Tue, 03 Nov 2020 00:16:00:  19000000 \\nINFO  @ Tue, 03 Nov 2020 00:16:04:  20000000 \\nINFO  @ Tue, 03 Nov 2020 00:16:12:  21000000 \\nINFO  @ Tue, 03 Nov 2020 00:16:16:  22000000 \\nINFO  @ Tue, 03 Nov 2020 00:16:20:  23000000 \\nINFO  @ Tue, 03 Nov 2020 00:16:25:  24000000 \\nINFO  @ Tue, 03 Nov 2020 00:16:29:  25000000 \\nINFO  @ Tue, 03 Nov 2020 00:16:33:  26000000 \\nINFO  @ Tue, 03 Nov 2020 00:16:37:  27000000 \\nINFO  @ Tue, 03 Nov 2020 00:16:41:  28000000 \\nINFO  @ Tue, 03 Nov 2020 00:16:51:  29000000 \\nINFO  @ Tue, 03 Nov 2020 00:16:54:  30000000 \\nINFO  @ Tue, 03 Nov 2020 00:16:58:  31000000 \\nINFO  @ Tue, 03 Nov 2020 00:17:02:  32000000 \\nINFO  @ Tue, 03 Nov 2020 00:17:06:  33000000 \\nINFO  @ Tue, 03 Nov 2020 00:17:10:  34000000 \\nINFO  @ Tue, 03 Nov 2020 00:17:15:  35000000 \\nINFO  @ Tue, 03 Nov 2020 00:17:24:  36000000 \\nINFO  @ Tue, 03 Nov 2020 00:17:27:  37000000 \\nINFO  @ Tue, 03 Nov 2020 00:17:31:  38000000 \\nINFO  @ Tue, 03 Nov 2020 00:17:38:  39000000 \\nINFO  @ Tue, 03 Nov 2020 00:17:42:  40000000 \\nINFO  @ Tue, 03 Nov 2020 00:17:46:  41000000 \\nINFO  @ Tue, 03 Nov 2020 00:17:50:  42000000 \\nINFO  @ Tue, 03 Nov 2020 00:17:59:  43000000 \\nINFO  @ Tue, 03 Nov 2020 00:18:03:  44000000 \\nINFO  @ Tue, 03 Nov 2020 00:18:07:  45000000 \\nINFO  @ Tue, 03 Nov 2020 00:18:14:  46000000 \\nINFO  @ Tue, 03 Nov 2020 00:18:19:  47000000 \\nINFO  @ Tue, 03 Nov 2020 00:18:22:  48000000 \\nINFO  @ Tue, 03 Nov 2020 00:18:26:  49000000 \\nINFO  @ Tue, 03 Nov 2020 00:18:30:  50000000 \\nINFO  @ Tue, 03 Nov 2020 00:18:40:  51000000 \\nINFO  @ Tue, 03 Nov 2020 00:18:44:  52000000 \\nINFO  @ Tue, 03 Nov 2020 00:18:50:  53000000 \\nINFO  @ Tue, 03 Nov 2020 00:18:54:  54000000 \\nINFO  @ Tue, 03 Nov 2020 00:18:58:  55000000 \\nINFO  @ Tue, 03 Nov 2020 00:19:02:  56000000 \\nINFO  @ Tue, 03 Nov 2020 00:19:06:  57000000 \\nINFO  @ Tue, 03 Nov 2020 00:19:11:  58000000 \\nINFO  @ Tue, 03 Nov 2020 00:19:15:  59000000 \\nINFO  @ Tue, 03 Nov 2020 00:19:27:  60000000 \\nINFO  @ Tue, 03 Nov 2020 00:19:31:  61000000 \\nINFO  @ Tue, 03 Nov 2020 00:19:36:  62000000 \\nINFO  @ Tue, 03 Nov 2020 00:19:41:  63000000 \\nINFO  @ Tue, 03 Nov 2020 00:19:45:  64000000 \\nINFO  @ Tue, 03 Nov 2020 00:19:49:  65000000 \\nINFO  @ Tue, 03 Nov 2020 00:19:54:  66000000 \\nINFO  @ Tue, 03 Nov 2020 00:19:57: 66660428 fragments have been read. \\nINFO  @ Tue, 03 Nov 2020 00:20:42: #1 mean fragment size is determined as 263.2 bp from treatment \\nINFO  @ Tue, 03 Nov 2020 00:20:42: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Tue, 03 Nov 2020 00:20:42: #1 fragment size = 263.2 \\nINFO  @ Tue, 03 Nov 2020 00:20:42: #1  total fragments in treatment: 55243421 \\nINFO  @ Tue, 03 Nov 2020 00:20:42: #1 user defined the maximum fragments... \\nINFO  @ Tue, 03 Nov 2020 00:20:42: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Tue, 03 Nov 2020 00:22:18: #1  fragments after filtering in treatment: 55210307 \\nINFO  @ Tue, 03 Nov 2020 00:22:18: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Tue, 03 Nov 2020 00:22:18: #1  total fragments in control: 66660428 \\nINFO  @ Tue, 03 Nov 2020 00:22:18: #1 user defined the maximum fragments... \\nINFO  @ Tue, 03 Nov 2020 00:22:18: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Tue, 03 Nov 2020 00:24:14: #1  fragments after filtering in control: 66632618 \\nINFO  @ Tue, 03 Nov 2020 00:24:14: #1  Redundant rate of control: 0.00 \\nINFO  @ Tue, 03 Nov 2020 00:24:14: #1 finished! \\nINFO  @ Tue, 03 Nov 2020 00:24:14: #2 Build Peak Model... \\nINFO  @ Tue, 03 Nov 2020 00:24:14: #2 Skipped... \\nINFO  @ Tue, 03 Nov 2020 00:24:14: #3 Call peaks... \\nINFO  @ Tue, 03 Nov 2020 00:24:14: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 03 Nov 2020 00:35:15: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 03 Nov 2020 00:35:15: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_treat_pileup.bdg \\nINFO  @ Tue, 03 Nov 2020 00:35:15: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_control_lambda.bdg \\nINFO  @ Tue, 03 Nov 2020 00:35:15: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Tue, 03 Nov 2020 00:35:15: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 03 Nov 2020 00:42:46: #4 Write output xls file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_peaks.xls \\nINFO  @ Tue, 03 Nov 2020 00:42:46: #4 Write peak in narrowPeak format file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_peaks.narrowPeak \\nINFO  @ Tue, 03 Nov 2020 00:42:46: #4 Write summits bed file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_summits.bed \\nINFO  @ Tue, 03 Nov 2020 00:42:46: Done! \\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'INFO  @ Tue, 03 Nov 2020 00:42:46: \\n# Command line: callpeak -B -t ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-MED1_R1.mLb.clN.sorted.bam -c ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 220 -n MV411_MEF2D_NT_SC_63_VHL-MED1_R1 --outdir ../../data/chipseq_MEF2D_degraded/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_MEF2D_NT_SC_63_VHL-MED1_R1\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-MED1_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Tue, 03 Nov 2020 00:42:46: #1 read fragment files... \\nINFO  @ Tue, 03 Nov 2020 00:42:46: #1 read treatment fragments... \\nINFO  @ Tue, 03 Nov 2020 00:42:51:  1000000 \\nINFO  @ Tue, 03 Nov 2020 00:42:57:  2000000 \\nINFO  @ Tue, 03 Nov 2020 00:43:01:  3000000 \\nINFO  @ Tue, 03 Nov 2020 00:43:06:  4000000 \\nINFO  @ Tue, 03 Nov 2020 00:43:11:  5000000 \\nINFO  @ Tue, 03 Nov 2020 00:43:15:  6000000 \\nINFO  @ Tue, 03 Nov 2020 00:43:21:  7000000 \\nINFO  @ Tue, 03 Nov 2020 00:43:25:  8000000 \\nINFO  @ Tue, 03 Nov 2020 00:43:30:  9000000 \\nINFO  @ Tue, 03 Nov 2020 00:43:34:  10000000 \\nINFO  @ Tue, 03 Nov 2020 00:43:39:  11000000 \\nINFO  @ Tue, 03 Nov 2020 00:43:43:  12000000 \\nINFO  @ Tue, 03 Nov 2020 00:43:48:  13000000 \\nINFO  @ Tue, 03 Nov 2020 00:43:52:  14000000 \\nINFO  @ Tue, 03 Nov 2020 00:43:58:  15000000 \\nINFO  @ Tue, 03 Nov 2020 00:44:04:  16000000 \\nINFO  @ Tue, 03 Nov 2020 00:44:09:  17000000 \\nINFO  @ Tue, 03 Nov 2020 00:44:13:  18000000 \\nINFO  @ Tue, 03 Nov 2020 00:44:17:  19000000 \\nINFO  @ Tue, 03 Nov 2020 00:44:22:  20000000 \\nINFO  @ Tue, 03 Nov 2020 00:44:26:  21000000 \\nINFO  @ Tue, 03 Nov 2020 00:44:31:  22000000 \\nINFO  @ Tue, 03 Nov 2020 00:44:38:  23000000 \\nINFO  @ Tue, 03 Nov 2020 00:44:43:  24000000 \\nINFO  @ Tue, 03 Nov 2020 00:44:47:  25000000 \\nINFO  @ Tue, 03 Nov 2020 00:44:52:  26000000 \\nINFO  @ Tue, 03 Nov 2020 00:44:56:  27000000 \\nINFO  @ Tue, 03 Nov 2020 00:45:01:  28000000 \\nINFO  @ Tue, 03 Nov 2020 00:45:05:  29000000 \\nINFO  @ Tue, 03 Nov 2020 00:45:10:  30000000 \\nINFO  @ Tue, 03 Nov 2020 00:45:15:  31000000 \\nINFO  @ Tue, 03 Nov 2020 00:45:22:  32000000 \\nINFO  @ Tue, 03 Nov 2020 00:45:27:  33000000 \\nINFO  @ Tue, 03 Nov 2020 00:45:31:  34000000 \\nINFO  @ Tue, 03 Nov 2020 00:45:35:  35000000 \\nINFO  @ Tue, 03 Nov 2020 00:45:40:  36000000 \\nINFO  @ Tue, 03 Nov 2020 00:45:45:  37000000 \\nINFO  @ Tue, 03 Nov 2020 00:45:50:  38000000 \\nINFO  @ Tue, 03 Nov 2020 00:45:56:  39000000 \\nINFO  @ Tue, 03 Nov 2020 00:46:01:  40000000 \\nINFO  @ Tue, 03 Nov 2020 00:46:06:  41000000 \\nINFO  @ Tue, 03 Nov 2020 00:46:10:  42000000 \\nINFO  @ Tue, 03 Nov 2020 00:46:14:  43000000 \\nINFO  @ Tue, 03 Nov 2020 00:46:19:  44000000 \\nINFO  @ Tue, 03 Nov 2020 00:46:23:  45000000 \\nINFO  @ Tue, 03 Nov 2020 00:46:27:  46000000 \\nINFO  @ Tue, 03 Nov 2020 00:46:35:  47000000 \\nINFO  @ Tue, 03 Nov 2020 00:46:39:  48000000 \\nINFO  @ Tue, 03 Nov 2020 00:46:44:  49000000 \\nINFO  @ Tue, 03 Nov 2020 00:46:48:  50000000 \\nINFO  @ Tue, 03 Nov 2020 00:46:53:  51000000 \\nINFO  @ Tue, 03 Nov 2020 00:46:57:  52000000 \\nINFO  @ Tue, 03 Nov 2020 00:47:02:  53000000 \\nINFO  @ Tue, 03 Nov 2020 00:47:06:  54000000 \\nINFO  @ Tue, 03 Nov 2020 00:47:14:  55000000 \\nINFO  @ Tue, 03 Nov 2020 00:47:19:  56000000 \\nINFO  @ Tue, 03 Nov 2020 00:47:23:  57000000 \\nINFO  @ Tue, 03 Nov 2020 00:47:28:  58000000 \\nINFO  @ Tue, 03 Nov 2020 00:47:32:  59000000 \\nINFO  @ Tue, 03 Nov 2020 00:47:37:  60000000 \\nINFO  @ Tue, 03 Nov 2020 00:47:41:  61000000 \\nINFO  @ Tue, 03 Nov 2020 00:47:45:  62000000 \\nINFO  @ Tue, 03 Nov 2020 00:47:53:  63000000 \\nINFO  @ Tue, 03 Nov 2020 00:47:58:  64000000 \\nINFO  @ Tue, 03 Nov 2020 00:48:02:  65000000 \\nINFO  @ Tue, 03 Nov 2020 00:48:07:  66000000 \\nINFO  @ Tue, 03 Nov 2020 00:48:12:  67000000 \\nINFO  @ Tue, 03 Nov 2020 00:48:16:  68000000 \\nINFO  @ Tue, 03 Nov 2020 00:48:21:  69000000 \\nINFO  @ Tue, 03 Nov 2020 00:48:26:  70000000 \\nINFO  @ Tue, 03 Nov 2020 00:48:33:  71000000 \\nINFO  @ Tue, 03 Nov 2020 00:48:38:  72000000 \\nINFO  @ Tue, 03 Nov 2020 00:48:42:  73000000 \\nINFO  @ Tue, 03 Nov 2020 00:48:47:  74000000 \\nINFO  @ Tue, 03 Nov 2020 00:48:51:  75000000 \\nINFO  @ Tue, 03 Nov 2020 00:48:56:  76000000 \\nINFO  @ Tue, 03 Nov 2020 00:49:01:  77000000 \\nINFO  @ Tue, 03 Nov 2020 00:49:06:  78000000 \\nINFO  @ Tue, 03 Nov 2020 00:49:10:  79000000 \\nINFO  @ Tue, 03 Nov 2020 00:49:18:  80000000 \\nINFO  @ Tue, 03 Nov 2020 00:49:23:  81000000 \\nINFO  @ Tue, 03 Nov 2020 00:49:28:  82000000 \\nINFO  @ Tue, 03 Nov 2020 00:49:33:  83000000 \\nINFO  @ Tue, 03 Nov 2020 00:49:33: 83009589 fragments have been read. \\nINFO  @ Tue, 03 Nov 2020 00:50:29: #1.2 read input fragments... \\nINFO  @ Tue, 03 Nov 2020 00:50:33:  1000000 \\nINFO  @ Tue, 03 Nov 2020 00:50:37:  2000000 \\nINFO  @ Tue, 03 Nov 2020 00:50:41:  3000000 \\nINFO  @ Tue, 03 Nov 2020 00:50:45:  4000000 \\nINFO  @ Tue, 03 Nov 2020 00:50:49:  5000000 \\nINFO  @ Tue, 03 Nov 2020 00:50:52:  6000000 \\nINFO  @ Tue, 03 Nov 2020 00:50:56:  7000000 \\nINFO  @ Tue, 03 Nov 2020 00:51:00:  8000000 \\nINFO  @ Tue, 03 Nov 2020 00:51:04:  9000000 \\nINFO  @ Tue, 03 Nov 2020 00:51:08:  10000000 \\nINFO  @ Tue, 03 Nov 2020 00:51:11:  11000000 \\nINFO  @ Tue, 03 Nov 2020 00:51:15:  12000000 \\nINFO  @ Tue, 03 Nov 2020 00:51:19:  13000000 \\nINFO  @ Tue, 03 Nov 2020 00:51:23:  14000000 \\nINFO  @ Tue, 03 Nov 2020 00:51:27:  15000000 \\nINFO  @ Tue, 03 Nov 2020 00:51:30:  16000000 \\nINFO  @ Tue, 03 Nov 2020 00:51:34:  17000000 \\nINFO  @ Tue, 03 Nov 2020 00:51:38:  18000000 \\nINFO  @ Tue, 03 Nov 2020 00:51:42:  19000000 \\nINFO  @ Tue, 03 Nov 2020 00:51:46:  20000000 \\nINFO  @ Tue, 03 Nov 2020 00:51:49:  21000000 \\nINFO  @ Tue, 03 Nov 2020 00:51:53:  22000000 \\nINFO  @ Tue, 03 Nov 2020 00:51:57:  23000000 \\nINFO  @ Tue, 03 Nov 2020 00:52:00:  24000000 \\nINFO  @ Tue, 03 Nov 2020 00:52:04:  25000000 \\nINFO  @ Tue, 03 Nov 2020 00:52:08:  26000000 \\nINFO  @ Tue, 03 Nov 2020 00:52:12:  27000000 \\nINFO  @ Tue, 03 Nov 2020 00:52:15:  28000000 \\nINFO  @ Tue, 03 Nov 2020 00:52:19:  29000000 \\nINFO  @ Tue, 03 Nov 2020 00:52:23:  30000000 \\nINFO  @ Tue, 03 Nov 2020 00:52:27:  31000000 \\nINFO  @ Tue, 03 Nov 2020 00:52:31:  32000000 \\nINFO  @ Tue, 03 Nov 2020 00:52:35:  33000000 \\nINFO  @ Tue, 03 Nov 2020 00:52:38:  34000000 \\nINFO  @ Tue, 03 Nov 2020 00:52:42:  35000000 \\nINFO  @ Tue, 03 Nov 2020 00:52:46:  36000000 \\nINFO  @ Tue, 03 Nov 2020 00:52:50:  37000000 \\nINFO  @ Tue, 03 Nov 2020 00:52:53:  38000000 \\nINFO  @ Tue, 03 Nov 2020 00:52:57:  39000000 \\nINFO  @ Tue, 03 Nov 2020 00:53:01:  40000000 \\nINFO  @ Tue, 03 Nov 2020 00:53:05:  41000000 \\nINFO  @ Tue, 03 Nov 2020 00:53:09:  42000000 \\nINFO  @ Tue, 03 Nov 2020 00:53:13:  43000000 \\nINFO  @ Tue, 03 Nov 2020 00:53:17:  44000000 \\nINFO  @ Tue, 03 Nov 2020 00:53:22:  45000000 \\nINFO  @ Tue, 03 Nov 2020 00:53:25:  46000000 \\nINFO  @ Tue, 03 Nov 2020 00:53:29:  47000000 \\nINFO  @ Tue, 03 Nov 2020 00:53:33:  48000000 \\nINFO  @ Tue, 03 Nov 2020 00:53:37:  49000000 \\nINFO  @ Tue, 03 Nov 2020 00:53:41:  50000000 \\nINFO  @ Tue, 03 Nov 2020 00:53:45:  51000000 \\nINFO  @ Tue, 03 Nov 2020 00:53:48:  52000000 \\nINFO  @ Tue, 03 Nov 2020 00:53:52:  53000000 \\nINFO  @ Tue, 03 Nov 2020 00:53:56:  54000000 \\nINFO  @ Tue, 03 Nov 2020 00:54:00:  55000000 \\nINFO  @ Tue, 03 Nov 2020 00:54:04:  56000000 \\nINFO  @ Tue, 03 Nov 2020 00:54:07:  57000000 \\nINFO  @ Tue, 03 Nov 2020 00:54:11:  58000000 \\nINFO  @ Tue, 03 Nov 2020 00:54:15:  59000000 \\nINFO  @ Tue, 03 Nov 2020 00:54:19:  60000000 \\nINFO  @ Tue, 03 Nov 2020 00:54:23:  61000000 \\nINFO  @ Tue, 03 Nov 2020 00:54:26:  62000000 \\nINFO  @ Tue, 03 Nov 2020 00:54:30:  63000000 \\nINFO  @ Tue, 03 Nov 2020 00:54:34:  64000000 \\nINFO  @ Tue, 03 Nov 2020 00:54:38:  65000000 \\nINFO  @ Tue, 03 Nov 2020 00:54:42:  66000000 \\nINFO  @ Tue, 03 Nov 2020 00:54:44: 66660428 fragments have been read. \\nINFO  @ Tue, 03 Nov 2020 00:55:29: #1 mean fragment size is determined as 272.5 bp from treatment \\nINFO  @ Tue, 03 Nov 2020 00:55:29: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Tue, 03 Nov 2020 00:55:29: #1 fragment size = 272.5 \\nINFO  @ Tue, 03 Nov 2020 00:55:29: #1  total fragments in treatment: 83009589 \\nINFO  @ Tue, 03 Nov 2020 00:55:29: #1 user defined the maximum fragments... \\nINFO  @ Tue, 03 Nov 2020 00:55:29: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Tue, 03 Nov 2020 00:57:41: #1  fragments after filtering in treatment: 82971928 \\nINFO  @ Tue, 03 Nov 2020 00:57:41: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Tue, 03 Nov 2020 00:57:41: #1  total fragments in control: 66660428 \\nINFO  @ Tue, 03 Nov 2020 00:57:41: #1 user defined the maximum fragments... \\nINFO  @ Tue, 03 Nov 2020 00:57:41: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Tue, 03 Nov 2020 00:59:27: #1  fragments after filtering in control: 66632618 \\nINFO  @ Tue, 03 Nov 2020 00:59:27: #1  Redundant rate of control: 0.00 \\nINFO  @ Tue, 03 Nov 2020 00:59:27: #1 finished! \\nINFO  @ Tue, 03 Nov 2020 00:59:27: #2 Build Peak Model... \\nINFO  @ Tue, 03 Nov 2020 00:59:27: #2 Skipped... \\nINFO  @ Tue, 03 Nov 2020 00:59:27: #3 Call peaks... \\nINFO  @ Tue, 03 Nov 2020 00:59:27: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 03 Nov 2020 01:10:23: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 03 Nov 2020 01:10:23: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_MEF2D_NT_SC_63_VHL-MED1_R1_treat_pileup.bdg \\nINFO  @ Tue, 03 Nov 2020 01:10:23: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_MEF2D_NT_SC_63_VHL-MED1_R1_control_lambda.bdg \\nINFO  @ Tue, 03 Nov 2020 01:10:23: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Tue, 03 Nov 2020 01:10:23: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 03 Nov 2020 01:18:54: #4 Write output xls file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-MED1_R1_peaks.xls \\nINFO  @ Tue, 03 Nov 2020 01:18:54: #4 Write peak in narrowPeak format file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-MED1_R1_peaks.narrowPeak \\nINFO  @ Tue, 03 Nov 2020 01:18:54: #4 Write summits bed file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-MED1_R1_summits.bed \\nINFO  @ Tue, 03 Nov 2020 01:18:54: Done! \\n'\n",
      "57247538 66632618\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_treat_pileup.bdg --c1 ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_control_lambda.bdg --t2 ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-MED1_R1_treat_pileup.bdg --c2 ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-MED1_R1_control_lambda.bdg --d1 57247538 --d2 66632618 -g 60 -l 220 --o-prefix MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MED1_R1_treat_pileup --outdir ../../data/chipseq_MEF2D_degraded/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Tue, 03 Nov 2020 01:18:55: Read and build treatment 1 bedGraph... \\nINFO  @ Tue, 03 Nov 2020 01:20:39: Read and build control 1 bedGraph... \\nINFO  @ Tue, 03 Nov 2020 01:24:18: Read and build treatment 2 bedGraph... \\nINFO  @ Tue, 03 Nov 2020 01:26:47: Read and build control 2 bedGraph... \\nINFO  @ Tue, 03 Nov 2020 01:42:17: Write peaks... \\nINFO  @ Tue, 03 Nov 2020 01:42:17: Done \\n')\n",
      "\n",
      "\n",
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1.mLb.clN.sorted.bam ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1.mLb.clN.sorted.bam and ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "229\n",
      "computing the scaling values\n",
      "b'INFO  @ Tue, 03 Nov 2020 01:49:39: \\n# Command line: callpeak -B -t ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1.mLb.clN.sorted.bam -c ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 229 -n MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1 --outdir ../../data/chipseq_MEF2D_degraded/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Tue, 03 Nov 2020 01:49:39: #1 read fragment files... \\nINFO  @ Tue, 03 Nov 2020 01:49:39: #1 read treatment fragments... \\nINFO  @ Tue, 03 Nov 2020 01:49:45:  1000000 \\nINFO  @ Tue, 03 Nov 2020 01:49:49:  2000000 \\nINFO  @ Tue, 03 Nov 2020 01:49:54:  3000000 \\nINFO  @ Tue, 03 Nov 2020 01:49:58:  4000000 \\nINFO  @ Tue, 03 Nov 2020 01:50:03:  5000000 \\nINFO  @ Tue, 03 Nov 2020 01:50:07:  6000000 \\nINFO  @ Tue, 03 Nov 2020 01:50:11:  7000000 \\nINFO  @ Tue, 03 Nov 2020 01:50:16:  8000000 \\nINFO  @ Tue, 03 Nov 2020 01:50:20:  9000000 \\nINFO  @ Tue, 03 Nov 2020 01:50:25:  10000000 \\nINFO  @ Tue, 03 Nov 2020 01:50:29:  11000000 \\nINFO  @ Tue, 03 Nov 2020 01:50:33:  12000000 \\nINFO  @ Tue, 03 Nov 2020 01:50:38:  13000000 \\nINFO  @ Tue, 03 Nov 2020 01:50:42:  14000000 \\nINFO  @ Tue, 03 Nov 2020 01:50:46:  15000000 \\nINFO  @ Tue, 03 Nov 2020 01:50:51:  16000000 \\nINFO  @ Tue, 03 Nov 2020 01:50:55:  17000000 \\nINFO  @ Tue, 03 Nov 2020 01:51:00:  18000000 \\nINFO  @ Tue, 03 Nov 2020 01:51:04:  19000000 \\nINFO  @ Tue, 03 Nov 2020 01:51:08:  20000000 \\nINFO  @ Tue, 03 Nov 2020 01:51:12:  21000000 \\nINFO  @ Tue, 03 Nov 2020 01:51:17:  22000000 \\nINFO  @ Tue, 03 Nov 2020 01:51:21:  23000000 \\nINFO  @ Tue, 03 Nov 2020 01:51:25:  24000000 \\nINFO  @ Tue, 03 Nov 2020 01:51:30:  25000000 \\nINFO  @ Tue, 03 Nov 2020 01:51:34:  26000000 \\nINFO  @ Tue, 03 Nov 2020 01:51:38:  27000000 \\nINFO  @ Tue, 03 Nov 2020 01:51:43:  28000000 \\nINFO  @ Tue, 03 Nov 2020 01:51:47:  29000000 \\nINFO  @ Tue, 03 Nov 2020 01:51:51:  30000000 \\nINFO  @ Tue, 03 Nov 2020 01:51:55:  31000000 \\nINFO  @ Tue, 03 Nov 2020 01:52:00:  32000000 \\nINFO  @ Tue, 03 Nov 2020 01:52:04:  33000000 \\nINFO  @ Tue, 03 Nov 2020 01:52:09:  34000000 \\nINFO  @ Tue, 03 Nov 2020 01:52:13:  35000000 \\nINFO  @ Tue, 03 Nov 2020 01:52:17:  36000000 \\nINFO  @ Tue, 03 Nov 2020 01:52:22:  37000000 \\nINFO  @ Tue, 03 Nov 2020 01:52:26:  38000000 \\nINFO  @ Tue, 03 Nov 2020 01:52:30:  39000000 \\nINFO  @ Tue, 03 Nov 2020 01:52:35:  40000000 \\nINFO  @ Tue, 03 Nov 2020 01:52:39:  41000000 \\nINFO  @ Tue, 03 Nov 2020 01:52:43:  42000000 \\nINFO  @ Tue, 03 Nov 2020 01:52:47:  43000000 \\nINFO  @ Tue, 03 Nov 2020 01:52:52:  44000000 \\nINFO  @ Tue, 03 Nov 2020 01:52:56:  45000000 \\nINFO  @ Tue, 03 Nov 2020 01:53:01:  46000000 \\nINFO  @ Tue, 03 Nov 2020 01:53:05:  47000000 \\nINFO  @ Tue, 03 Nov 2020 01:53:10:  48000000 \\nINFO  @ Tue, 03 Nov 2020 01:53:14:  49000000 \\nINFO  @ Tue, 03 Nov 2020 01:53:18:  50000000 \\nINFO  @ Tue, 03 Nov 2020 01:53:23:  51000000 \\nINFO  @ Tue, 03 Nov 2020 01:53:27:  52000000 \\nINFO  @ Tue, 03 Nov 2020 01:53:31:  53000000 \\nINFO  @ Tue, 03 Nov 2020 01:53:35:  54000000 \\nINFO  @ Tue, 03 Nov 2020 01:53:40:  55000000 \\nINFO  @ Tue, 03 Nov 2020 01:53:44:  56000000 \\nINFO  @ Tue, 03 Nov 2020 01:53:48:  57000000 \\nINFO  @ Tue, 03 Nov 2020 01:53:53:  58000000 \\nINFO  @ Tue, 03 Nov 2020 01:53:57:  59000000 \\nINFO  @ Tue, 03 Nov 2020 01:54:02:  60000000 \\nINFO  @ Tue, 03 Nov 2020 01:54:06:  61000000 \\nINFO  @ Tue, 03 Nov 2020 01:54:11:  62000000 \\nINFO  @ Tue, 03 Nov 2020 01:54:15:  63000000 \\nINFO  @ Tue, 03 Nov 2020 01:54:19:  64000000 \\nINFO  @ Tue, 03 Nov 2020 01:54:24:  65000000 \\nINFO  @ Tue, 03 Nov 2020 01:54:28:  66000000 \\nINFO  @ Tue, 03 Nov 2020 01:54:33:  67000000 \\nINFO  @ Tue, 03 Nov 2020 01:54:37:  68000000 \\nINFO  @ Tue, 03 Nov 2020 01:54:41:  69000000 \\nINFO  @ Tue, 03 Nov 2020 01:54:46:  70000000 \\nINFO  @ Tue, 03 Nov 2020 01:54:50:  71000000 \\nINFO  @ Tue, 03 Nov 2020 01:54:54:  72000000 \\nINFO  @ Tue, 03 Nov 2020 01:54:59:  73000000 \\nINFO  @ Tue, 03 Nov 2020 01:55:03:  74000000 \\nINFO  @ Tue, 03 Nov 2020 01:55:07:  75000000 \\nINFO  @ Tue, 03 Nov 2020 01:55:12:  76000000 \\nINFO  @ Tue, 03 Nov 2020 01:55:16:  77000000 \\nINFO  @ Tue, 03 Nov 2020 01:55:20:  78000000 \\nINFO  @ Tue, 03 Nov 2020 01:55:25:  79000000 \\nINFO  @ Tue, 03 Nov 2020 01:55:29:  80000000 \\nINFO  @ Tue, 03 Nov 2020 01:55:34:  81000000 \\nINFO  @ Tue, 03 Nov 2020 01:55:38:  82000000 \\nINFO  @ Tue, 03 Nov 2020 01:55:43:  83000000 \\nINFO  @ Tue, 03 Nov 2020 01:55:47:  84000000 \\nINFO  @ Tue, 03 Nov 2020 01:55:52:  85000000 \\nINFO  @ Tue, 03 Nov 2020 01:55:56:  86000000 \\nINFO  @ Tue, 03 Nov 2020 01:56:00:  87000000 \\nINFO  @ Tue, 03 Nov 2020 01:56:05:  88000000 \\nINFO  @ Tue, 03 Nov 2020 01:56:10:  89000000 \\nINFO  @ Tue, 03 Nov 2020 01:56:14:  90000000 \\nINFO  @ Tue, 03 Nov 2020 01:56:19:  91000000 \\nINFO  @ Tue, 03 Nov 2020 01:56:23:  92000000 \\nINFO  @ Tue, 03 Nov 2020 01:56:27:  93000000 \\nINFO  @ Tue, 03 Nov 2020 01:56:32:  94000000 \\nINFO  @ Tue, 03 Nov 2020 01:56:36:  95000000 \\nINFO  @ Tue, 03 Nov 2020 01:56:41:  96000000 \\nINFO  @ Tue, 03 Nov 2020 01:56:45:  97000000 \\nINFO  @ Tue, 03 Nov 2020 01:56:48: 97438650 fragments have been read. \\nINFO  @ Tue, 03 Nov 2020 01:57:55: #1.2 read input fragments... \\nINFO  @ Tue, 03 Nov 2020 01:58:00:  1000000 \\nINFO  @ Tue, 03 Nov 2020 01:58:04:  2000000 \\nINFO  @ Tue, 03 Nov 2020 01:58:08:  3000000 \\nINFO  @ Tue, 03 Nov 2020 01:58:12:  4000000 \\nINFO  @ Tue, 03 Nov 2020 01:58:15:  5000000 \\nINFO  @ Tue, 03 Nov 2020 01:58:19:  6000000 \\nINFO  @ Tue, 03 Nov 2020 01:58:23:  7000000 \\nINFO  @ Tue, 03 Nov 2020 01:58:27:  8000000 \\nINFO  @ Tue, 03 Nov 2020 01:58:31:  9000000 \\nINFO  @ Tue, 03 Nov 2020 01:58:34:  10000000 \\nINFO  @ Tue, 03 Nov 2020 01:58:39:  11000000 \\nINFO  @ Tue, 03 Nov 2020 01:58:43:  12000000 \\nINFO  @ Tue, 03 Nov 2020 01:58:46:  13000000 \\nINFO  @ Tue, 03 Nov 2020 01:58:50:  14000000 \\nINFO  @ Tue, 03 Nov 2020 01:58:54:  15000000 \\nINFO  @ Tue, 03 Nov 2020 01:58:57:  16000000 \\nINFO  @ Tue, 03 Nov 2020 01:59:01:  17000000 \\nINFO  @ Tue, 03 Nov 2020 01:59:05:  18000000 \\nINFO  @ Tue, 03 Nov 2020 01:59:09:  19000000 \\nINFO  @ Tue, 03 Nov 2020 01:59:12:  20000000 \\nINFO  @ Tue, 03 Nov 2020 01:59:16:  21000000 \\nINFO  @ Tue, 03 Nov 2020 01:59:20:  22000000 \\nINFO  @ Tue, 03 Nov 2020 01:59:24:  23000000 \\nINFO  @ Tue, 03 Nov 2020 01:59:27:  24000000 \\nINFO  @ Tue, 03 Nov 2020 01:59:31:  25000000 \\nINFO  @ Tue, 03 Nov 2020 01:59:35:  26000000 \\nINFO  @ Tue, 03 Nov 2020 01:59:38:  27000000 \\nINFO  @ Tue, 03 Nov 2020 01:59:42:  28000000 \\nINFO  @ Tue, 03 Nov 2020 01:59:46:  29000000 \\nINFO  @ Tue, 03 Nov 2020 01:59:49:  30000000 \\nINFO  @ Tue, 03 Nov 2020 01:59:53:  31000000 \\nINFO  @ Tue, 03 Nov 2020 01:59:57:  32000000 \\nINFO  @ Tue, 03 Nov 2020 02:00:01:  33000000 \\nINFO  @ Tue, 03 Nov 2020 02:00:05:  34000000 \\nINFO  @ Tue, 03 Nov 2020 02:00:08:  35000000 \\nINFO  @ Tue, 03 Nov 2020 02:00:12:  36000000 \\nINFO  @ Tue, 03 Nov 2020 02:00:16:  37000000 \\nINFO  @ Tue, 03 Nov 2020 02:00:20:  38000000 \\nINFO  @ Tue, 03 Nov 2020 02:00:23:  39000000 \\nINFO  @ Tue, 03 Nov 2020 02:00:27:  40000000 \\nINFO  @ Tue, 03 Nov 2020 02:00:31:  41000000 \\nINFO  @ Tue, 03 Nov 2020 02:00:35:  42000000 \\nINFO  @ Tue, 03 Nov 2020 02:00:39:  43000000 \\nINFO  @ Tue, 03 Nov 2020 02:00:43:  44000000 \\nINFO  @ Tue, 03 Nov 2020 02:00:47:  45000000 \\nINFO  @ Tue, 03 Nov 2020 02:00:50:  46000000 \\nINFO  @ Tue, 03 Nov 2020 02:00:54:  47000000 \\nINFO  @ Tue, 03 Nov 2020 02:00:58:  48000000 \\nINFO  @ Tue, 03 Nov 2020 02:01:02:  49000000 \\nINFO  @ Tue, 03 Nov 2020 02:01:06:  50000000 \\nINFO  @ Tue, 03 Nov 2020 02:01:09:  51000000 \\nINFO  @ Tue, 03 Nov 2020 02:01:13:  52000000 \\nINFO  @ Tue, 03 Nov 2020 02:01:17:  53000000 \\nINFO  @ Tue, 03 Nov 2020 02:01:21:  54000000 \\nINFO  @ Tue, 03 Nov 2020 02:01:25:  55000000 \\nINFO  @ Tue, 03 Nov 2020 02:01:28:  56000000 \\nINFO  @ Tue, 03 Nov 2020 02:01:32:  57000000 \\nINFO  @ Tue, 03 Nov 2020 02:01:36:  58000000 \\nINFO  @ Tue, 03 Nov 2020 02:01:40:  59000000 \\nINFO  @ Tue, 03 Nov 2020 02:01:44:  60000000 \\nINFO  @ Tue, 03 Nov 2020 02:01:48:  61000000 \\nINFO  @ Tue, 03 Nov 2020 02:01:51:  62000000 \\nINFO  @ Tue, 03 Nov 2020 02:01:55:  63000000 \\nINFO  @ Tue, 03 Nov 2020 02:01:59:  64000000 \\nINFO  @ Tue, 03 Nov 2020 02:02:03:  65000000 \\nINFO  @ Tue, 03 Nov 2020 02:02:07:  66000000 \\nINFO  @ Tue, 03 Nov 2020 02:02:09: 66660428 fragments have been read. \\nINFO  @ Tue, 03 Nov 2020 02:02:54: #1 mean fragment size is determined as 290.0 bp from treatment \\nINFO  @ Tue, 03 Nov 2020 02:02:54: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Tue, 03 Nov 2020 02:02:54: #1 fragment size = 290.0 \\nINFO  @ Tue, 03 Nov 2020 02:02:54: #1  total fragments in treatment: 97438650 \\nINFO  @ Tue, 03 Nov 2020 02:02:54: #1 user defined the maximum fragments... \\nINFO  @ Tue, 03 Nov 2020 02:02:54: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Tue, 03 Nov 2020 02:05:33: #1  fragments after filtering in treatment: 97411926 \\nINFO  @ Tue, 03 Nov 2020 02:05:33: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Tue, 03 Nov 2020 02:05:33: #1  total fragments in control: 66660428 \\nINFO  @ Tue, 03 Nov 2020 02:05:33: #1 user defined the maximum fragments... \\nINFO  @ Tue, 03 Nov 2020 02:05:33: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Tue, 03 Nov 2020 02:07:23: #1  fragments after filtering in control: 66632618 \\nINFO  @ Tue, 03 Nov 2020 02:07:23: #1  Redundant rate of control: 0.00 \\nINFO  @ Tue, 03 Nov 2020 02:07:23: #1 finished! \\nINFO  @ Tue, 03 Nov 2020 02:07:23: #2 Build Peak Model... \\nINFO  @ Tue, 03 Nov 2020 02:07:23: #2 Skipped... \\nINFO  @ Tue, 03 Nov 2020 02:07:23: #3 Call peaks... \\nINFO  @ Tue, 03 Nov 2020 02:07:23: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 03 Nov 2020 02:18:43: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 03 Nov 2020 02:18:43: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_treat_pileup.bdg \\nINFO  @ Tue, 03 Nov 2020 02:18:43: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_control_lambda.bdg \\nINFO  @ Tue, 03 Nov 2020 02:18:43: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Tue, 03 Nov 2020 02:18:43: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 03 Nov 2020 02:28:00: #4 Write output xls file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_peaks.xls \\nINFO  @ Tue, 03 Nov 2020 02:28:00: #4 Write peak in narrowPeak format file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_peaks.narrowPeak \\nINFO  @ Tue, 03 Nov 2020 02:28:00: #4 Write summits bed file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_summits.bed \\nINFO  @ Tue, 03 Nov 2020 02:28:00: Done! \\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'INFO  @ Tue, 03 Nov 2020 02:28:00: \\n# Command line: callpeak -B -t ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1.mLb.clN.sorted.bam -c ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 229 -n MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1 --outdir ../../data/chipseq_MEF2D_degraded/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Tue, 03 Nov 2020 02:28:00: #1 read fragment files... \\nINFO  @ Tue, 03 Nov 2020 02:28:00: #1 read treatment fragments... \\nINFO  @ Tue, 03 Nov 2020 02:28:05:  1000000 \\nINFO  @ Tue, 03 Nov 2020 02:28:10:  2000000 \\nINFO  @ Tue, 03 Nov 2020 02:28:15:  3000000 \\nINFO  @ Tue, 03 Nov 2020 02:28:20:  4000000 \\nINFO  @ Tue, 03 Nov 2020 02:28:24:  5000000 \\nINFO  @ Tue, 03 Nov 2020 02:28:29:  6000000 \\nINFO  @ Tue, 03 Nov 2020 02:28:34:  7000000 \\nINFO  @ Tue, 03 Nov 2020 02:28:39:  8000000 \\nINFO  @ Tue, 03 Nov 2020 02:28:44:  9000000 \\nINFO  @ Tue, 03 Nov 2020 02:28:48:  10000000 \\nINFO  @ Tue, 03 Nov 2020 02:28:53:  11000000 \\nINFO  @ Tue, 03 Nov 2020 02:28:58:  12000000 \\nINFO  @ Tue, 03 Nov 2020 02:29:03:  13000000 \\nINFO  @ Tue, 03 Nov 2020 02:29:07:  14000000 \\nINFO  @ Tue, 03 Nov 2020 02:29:12:  15000000 \\nINFO  @ Tue, 03 Nov 2020 02:29:17:  16000000 \\nINFO  @ Tue, 03 Nov 2020 02:29:22:  17000000 \\nINFO  @ Tue, 03 Nov 2020 02:29:27:  18000000 \\nINFO  @ Tue, 03 Nov 2020 02:29:32:  19000000 \\nINFO  @ Tue, 03 Nov 2020 02:29:36:  20000000 \\nINFO  @ Tue, 03 Nov 2020 02:29:41:  21000000 \\nINFO  @ Tue, 03 Nov 2020 02:29:46:  22000000 \\nINFO  @ Tue, 03 Nov 2020 02:29:51:  23000000 \\nINFO  @ Tue, 03 Nov 2020 02:29:55:  24000000 \\nINFO  @ Tue, 03 Nov 2020 02:30:00:  25000000 \\nINFO  @ Tue, 03 Nov 2020 02:30:04:  26000000 \\nINFO  @ Tue, 03 Nov 2020 02:30:09:  27000000 \\nINFO  @ Tue, 03 Nov 2020 02:30:14:  28000000 \\nINFO  @ Tue, 03 Nov 2020 02:30:19:  29000000 \\nINFO  @ Tue, 03 Nov 2020 02:30:24:  30000000 \\nINFO  @ Tue, 03 Nov 2020 02:30:29:  31000000 \\nINFO  @ Tue, 03 Nov 2020 02:30:34:  32000000 \\nINFO  @ Tue, 03 Nov 2020 02:30:39:  33000000 \\nINFO  @ Tue, 03 Nov 2020 02:30:43:  34000000 \\nINFO  @ Tue, 03 Nov 2020 02:30:48:  35000000 \\nINFO  @ Tue, 03 Nov 2020 02:30:53:  36000000 \\nINFO  @ Tue, 03 Nov 2020 02:30:58:  37000000 \\nINFO  @ Tue, 03 Nov 2020 02:31:03:  38000000 \\nINFO  @ Tue, 03 Nov 2020 02:31:07:  39000000 \\nINFO  @ Tue, 03 Nov 2020 02:31:12:  40000000 \\nINFO  @ Tue, 03 Nov 2020 02:31:17:  41000000 \\nINFO  @ Tue, 03 Nov 2020 02:31:22:  42000000 \\nINFO  @ Tue, 03 Nov 2020 02:31:27:  43000000 \\nINFO  @ Tue, 03 Nov 2020 02:31:31:  44000000 \\nINFO  @ Tue, 03 Nov 2020 02:31:34: 44515175 fragments have been read. \\nINFO  @ Tue, 03 Nov 2020 02:32:02: #1.2 read input fragments... \\nINFO  @ Tue, 03 Nov 2020 02:32:06:  1000000 \\nINFO  @ Tue, 03 Nov 2020 02:32:10:  2000000 \\nINFO  @ Tue, 03 Nov 2020 02:32:14:  3000000 \\nINFO  @ Tue, 03 Nov 2020 02:32:17:  4000000 \\nINFO  @ Tue, 03 Nov 2020 02:32:21:  5000000 \\nINFO  @ Tue, 03 Nov 2020 02:32:25:  6000000 \\nINFO  @ Tue, 03 Nov 2020 02:32:29:  7000000 \\nINFO  @ Tue, 03 Nov 2020 02:32:32:  8000000 \\nINFO  @ Tue, 03 Nov 2020 02:32:36:  9000000 \\nINFO  @ Tue, 03 Nov 2020 02:32:40:  10000000 \\nINFO  @ Tue, 03 Nov 2020 02:32:44:  11000000 \\nINFO  @ Tue, 03 Nov 2020 02:32:48:  12000000 \\nINFO  @ Tue, 03 Nov 2020 02:32:51:  13000000 \\nINFO  @ Tue, 03 Nov 2020 02:32:55:  14000000 \\nINFO  @ Tue, 03 Nov 2020 02:32:59:  15000000 \\nINFO  @ Tue, 03 Nov 2020 02:33:03:  16000000 \\nINFO  @ Tue, 03 Nov 2020 02:33:07:  17000000 \\nINFO  @ Tue, 03 Nov 2020 02:33:10:  18000000 \\nINFO  @ Tue, 03 Nov 2020 02:33:14:  19000000 \\nINFO  @ Tue, 03 Nov 2020 02:33:18:  20000000 \\nINFO  @ Tue, 03 Nov 2020 02:33:22:  21000000 \\nINFO  @ Tue, 03 Nov 2020 02:33:26:  22000000 \\nINFO  @ Tue, 03 Nov 2020 02:33:29:  23000000 \\nINFO  @ Tue, 03 Nov 2020 02:33:33:  24000000 \\nINFO  @ Tue, 03 Nov 2020 02:33:37:  25000000 \\nINFO  @ Tue, 03 Nov 2020 02:33:41:  26000000 \\nINFO  @ Tue, 03 Nov 2020 02:33:44:  27000000 \\nINFO  @ Tue, 03 Nov 2020 02:33:48:  28000000 \\nINFO  @ Tue, 03 Nov 2020 02:33:52:  29000000 \\nINFO  @ Tue, 03 Nov 2020 02:33:56:  30000000 \\nINFO  @ Tue, 03 Nov 2020 02:34:00:  31000000 \\nINFO  @ Tue, 03 Nov 2020 02:34:03:  32000000 \\nINFO  @ Tue, 03 Nov 2020 02:34:07:  33000000 \\nINFO  @ Tue, 03 Nov 2020 02:34:11:  34000000 \\nINFO  @ Tue, 03 Nov 2020 02:34:15:  35000000 \\nINFO  @ Tue, 03 Nov 2020 02:34:18:  36000000 \\nINFO  @ Tue, 03 Nov 2020 02:34:22:  37000000 \\nINFO  @ Tue, 03 Nov 2020 02:34:26:  38000000 \\nINFO  @ Tue, 03 Nov 2020 02:34:30:  39000000 \\nINFO  @ Tue, 03 Nov 2020 02:34:33:  40000000 \\nINFO  @ Tue, 03 Nov 2020 02:34:37:  41000000 \\nINFO  @ Tue, 03 Nov 2020 02:34:41:  42000000 \\nINFO  @ Tue, 03 Nov 2020 02:34:45:  43000000 \\nINFO  @ Tue, 03 Nov 2020 02:34:49:  44000000 \\nINFO  @ Tue, 03 Nov 2020 02:34:53:  45000000 \\nINFO  @ Tue, 03 Nov 2020 02:34:56:  46000000 \\nINFO  @ Tue, 03 Nov 2020 02:35:00:  47000000 \\nINFO  @ Tue, 03 Nov 2020 02:35:04:  48000000 \\nINFO  @ Tue, 03 Nov 2020 02:35:08:  49000000 \\nINFO  @ Tue, 03 Nov 2020 02:35:12:  50000000 \\nINFO  @ Tue, 03 Nov 2020 02:35:15:  51000000 \\nINFO  @ Tue, 03 Nov 2020 02:35:19:  52000000 \\nINFO  @ Tue, 03 Nov 2020 02:35:23:  53000000 \\nINFO  @ Tue, 03 Nov 2020 02:35:27:  54000000 \\nINFO  @ Tue, 03 Nov 2020 02:35:31:  55000000 \\nINFO  @ Tue, 03 Nov 2020 02:35:34:  56000000 \\nINFO  @ Tue, 03 Nov 2020 02:35:38:  57000000 \\nINFO  @ Tue, 03 Nov 2020 02:35:42:  58000000 \\nINFO  @ Tue, 03 Nov 2020 02:35:46:  59000000 \\nINFO  @ Tue, 03 Nov 2020 02:35:50:  60000000 \\nINFO  @ Tue, 03 Nov 2020 02:35:54:  61000000 \\nINFO  @ Tue, 03 Nov 2020 02:35:58:  62000000 \\nINFO  @ Tue, 03 Nov 2020 02:36:01:  63000000 \\nINFO  @ Tue, 03 Nov 2020 02:36:05:  64000000 \\nINFO  @ Tue, 03 Nov 2020 02:36:09:  65000000 \\nINFO  @ Tue, 03 Nov 2020 02:36:13:  66000000 \\nINFO  @ Tue, 03 Nov 2020 02:36:16: 66660428 fragments have been read. \\nINFO  @ Tue, 03 Nov 2020 02:37:00: #1 mean fragment size is determined as 273.4 bp from treatment \\nINFO  @ Tue, 03 Nov 2020 02:37:00: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Tue, 03 Nov 2020 02:37:00: #1 fragment size = 273.4 \\nINFO  @ Tue, 03 Nov 2020 02:37:00: #1  total fragments in treatment: 44515175 \\nINFO  @ Tue, 03 Nov 2020 02:37:00: #1 user defined the maximum fragments... \\nINFO  @ Tue, 03 Nov 2020 02:37:00: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Tue, 03 Nov 2020 02:38:15: #1  fragments after filtering in treatment: 44499637 \\nINFO  @ Tue, 03 Nov 2020 02:38:15: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Tue, 03 Nov 2020 02:38:15: #1  total fragments in control: 66660428 \\nINFO  @ Tue, 03 Nov 2020 02:38:15: #1 user defined the maximum fragments... \\nINFO  @ Tue, 03 Nov 2020 02:38:15: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Tue, 03 Nov 2020 02:40:06: #1  fragments after filtering in control: 66632618 \\nINFO  @ Tue, 03 Nov 2020 02:40:06: #1  Redundant rate of control: 0.00 \\nINFO  @ Tue, 03 Nov 2020 02:40:06: #1 finished! \\nINFO  @ Tue, 03 Nov 2020 02:40:06: #2 Build Peak Model... \\nINFO  @ Tue, 03 Nov 2020 02:40:06: #2 Skipped... \\nINFO  @ Tue, 03 Nov 2020 02:40:06: #3 Call peaks... \\nINFO  @ Tue, 03 Nov 2020 02:40:06: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 03 Nov 2020 02:50:16: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 03 Nov 2020 02:50:16: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_treat_pileup.bdg \\nINFO  @ Tue, 03 Nov 2020 02:50:16: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_control_lambda.bdg \\nINFO  @ Tue, 03 Nov 2020 02:50:16: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Tue, 03 Nov 2020 02:50:16: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 03 Nov 2020 02:57:19: #4 Write output xls file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_peaks.xls \\nINFO  @ Tue, 03 Nov 2020 02:57:19: #4 Write peak in narrowPeak format file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_peaks.narrowPeak \\nINFO  @ Tue, 03 Nov 2020 02:57:19: #4 Write summits bed file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_summits.bed \\nINFO  @ Tue, 03 Nov 2020 02:57:19: Done! \\n'\n",
      "73479117 44499637\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_treat_pileup.bdg --c1 ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_control_lambda.bdg --t2 ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_treat_pileup.bdg --c2 ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_control_lambda.bdg --d1 73479117 --d2 44499637 -g 60 -l 229 --o-prefix MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_treat_pileup --outdir ../../data/chipseq_MEF2D_degraded/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Tue, 03 Nov 2020 02:57:19: Read and build treatment 1 bedGraph... \\nINFO  @ Tue, 03 Nov 2020 03:00:16: Read and build control 1 bedGraph... \\nINFO  @ Tue, 03 Nov 2020 03:03:40: Read and build treatment 2 bedGraph... \\nINFO  @ Tue, 03 Nov 2020 03:04:59: Read and build control 2 bedGraph... \\nINFO  @ Tue, 03 Nov 2020 03:20:21: Write peaks... \\nINFO  @ Tue, 03 Nov 2020 03:20:21: Done \\n')\n",
      "\n",
      "\n",
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1.mLb.clN.sorted.bam ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-POLII_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1.mLb.clN.sorted.bam and ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-POLII_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "250\n",
      "computing the scaling values\n",
      "b'INFO  @ Tue, 03 Nov 2020 03:28:49: \\n# Command line: callpeak -B -t ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1.mLb.clN.sorted.bam -c ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 250 -n MV411_MEF2D_NT_SC_63_DMSO-POLII_R1 --outdir ../../data/chipseq_MEF2D_degraded/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_MEF2D_NT_SC_63_DMSO-POLII_R1\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Tue, 03 Nov 2020 03:28:49: #1 read fragment files... \\nINFO  @ Tue, 03 Nov 2020 03:28:49: #1 read treatment fragments... \\nINFO  @ Tue, 03 Nov 2020 03:28:53:  1000000 \\nINFO  @ Tue, 03 Nov 2020 03:28:57:  2000000 \\nINFO  @ Tue, 03 Nov 2020 03:29:02:  3000000 \\nINFO  @ Tue, 03 Nov 2020 03:29:06:  4000000 \\nINFO  @ Tue, 03 Nov 2020 03:29:10:  5000000 \\nINFO  @ Tue, 03 Nov 2020 03:29:15:  6000000 \\nINFO  @ Tue, 03 Nov 2020 03:29:19:  7000000 \\nINFO  @ Tue, 03 Nov 2020 03:29:23:  8000000 \\nINFO  @ Tue, 03 Nov 2020 03:29:28:  9000000 \\nINFO  @ Tue, 03 Nov 2020 03:29:32:  10000000 \\nINFO  @ Tue, 03 Nov 2020 03:29:36:  11000000 \\nINFO  @ Tue, 03 Nov 2020 03:29:40:  12000000 \\nINFO  @ Tue, 03 Nov 2020 03:29:45:  13000000 \\nINFO  @ Tue, 03 Nov 2020 03:29:49:  14000000 \\nINFO  @ Tue, 03 Nov 2020 03:29:53:  15000000 \\nINFO  @ Tue, 03 Nov 2020 03:29:58:  16000000 \\nINFO  @ Tue, 03 Nov 2020 03:30:03:  17000000 \\nINFO  @ Tue, 03 Nov 2020 03:30:07:  18000000 \\nINFO  @ Tue, 03 Nov 2020 03:30:11:  19000000 \\nINFO  @ Tue, 03 Nov 2020 03:30:16:  20000000 \\nINFO  @ Tue, 03 Nov 2020 03:30:20:  21000000 \\nINFO  @ Tue, 03 Nov 2020 03:30:24:  22000000 \\nINFO  @ Tue, 03 Nov 2020 03:30:29:  23000000 \\nINFO  @ Tue, 03 Nov 2020 03:30:33:  24000000 \\nINFO  @ Tue, 03 Nov 2020 03:30:37:  25000000 \\nINFO  @ Tue, 03 Nov 2020 03:30:42:  26000000 \\nINFO  @ Tue, 03 Nov 2020 03:30:46:  27000000 \\nINFO  @ Tue, 03 Nov 2020 03:30:50:  28000000 \\nINFO  @ Tue, 03 Nov 2020 03:30:54:  29000000 \\nINFO  @ Tue, 03 Nov 2020 03:30:59:  30000000 \\nINFO  @ Tue, 03 Nov 2020 03:31:03:  31000000 \\nINFO  @ Tue, 03 Nov 2020 03:31:08:  32000000 \\nINFO  @ Tue, 03 Nov 2020 03:31:12:  33000000 \\nINFO  @ Tue, 03 Nov 2020 03:31:16:  34000000 \\nINFO  @ Tue, 03 Nov 2020 03:31:21:  35000000 \\nINFO  @ Tue, 03 Nov 2020 03:31:25:  36000000 \\nINFO  @ Tue, 03 Nov 2020 03:31:29:  37000000 \\nINFO  @ Tue, 03 Nov 2020 03:31:34:  38000000 \\nINFO  @ Tue, 03 Nov 2020 03:31:38:  39000000 \\nINFO  @ Tue, 03 Nov 2020 03:31:42:  40000000 \\nINFO  @ Tue, 03 Nov 2020 03:31:46:  41000000 \\nINFO  @ Tue, 03 Nov 2020 03:31:50:  42000000 \\nINFO  @ Tue, 03 Nov 2020 03:31:55:  43000000 \\nINFO  @ Tue, 03 Nov 2020 03:31:59:  44000000 \\nINFO  @ Tue, 03 Nov 2020 03:32:03:  45000000 \\nINFO  @ Tue, 03 Nov 2020 03:32:08:  46000000 \\nINFO  @ Tue, 03 Nov 2020 03:32:12:  47000000 \\nINFO  @ Tue, 03 Nov 2020 03:32:16:  48000000 \\nINFO  @ Tue, 03 Nov 2020 03:32:21:  49000000 \\nINFO  @ Tue, 03 Nov 2020 03:32:25:  50000000 \\nINFO  @ Tue, 03 Nov 2020 03:32:29:  51000000 \\nINFO  @ Tue, 03 Nov 2020 03:32:34:  52000000 \\nINFO  @ Tue, 03 Nov 2020 03:32:38:  53000000 \\nINFO  @ Tue, 03 Nov 2020 03:32:42:  54000000 \\nINFO  @ Tue, 03 Nov 2020 03:32:47:  55000000 \\nINFO  @ Tue, 03 Nov 2020 03:32:51:  56000000 \\nINFO  @ Tue, 03 Nov 2020 03:32:55:  57000000 \\nINFO  @ Tue, 03 Nov 2020 03:32:59:  58000000 \\nINFO  @ Tue, 03 Nov 2020 03:33:04:  59000000 \\nINFO  @ Tue, 03 Nov 2020 03:33:08:  60000000 \\nINFO  @ Tue, 03 Nov 2020 03:33:12:  61000000 \\nINFO  @ Tue, 03 Nov 2020 03:33:17:  62000000 \\nINFO  @ Tue, 03 Nov 2020 03:33:21:  63000000 \\nINFO  @ Tue, 03 Nov 2020 03:33:25:  64000000 \\nINFO  @ Tue, 03 Nov 2020 03:33:30:  65000000 \\nINFO  @ Tue, 03 Nov 2020 03:33:34:  66000000 \\nINFO  @ Tue, 03 Nov 2020 03:33:38:  67000000 \\nINFO  @ Tue, 03 Nov 2020 03:33:42:  68000000 \\nINFO  @ Tue, 03 Nov 2020 03:33:47:  69000000 \\nINFO  @ Tue, 03 Nov 2020 03:33:51:  70000000 \\nINFO  @ Tue, 03 Nov 2020 03:33:55:  71000000 \\nINFO  @ Tue, 03 Nov 2020 03:33:59:  72000000 \\nINFO  @ Tue, 03 Nov 2020 03:34:04:  73000000 \\nINFO  @ Tue, 03 Nov 2020 03:34:08:  74000000 \\nINFO  @ Tue, 03 Nov 2020 03:34:14:  75000000 \\nINFO  @ Tue, 03 Nov 2020 03:34:18:  76000000 \\nINFO  @ Tue, 03 Nov 2020 03:34:22:  77000000 \\nINFO  @ Tue, 03 Nov 2020 03:34:26:  78000000 \\nINFO  @ Tue, 03 Nov 2020 03:34:30:  79000000 \\nINFO  @ Tue, 03 Nov 2020 03:34:35:  80000000 \\nINFO  @ Tue, 03 Nov 2020 03:34:39:  81000000 \\nINFO  @ Tue, 03 Nov 2020 03:34:44:  82000000 \\nINFO  @ Tue, 03 Nov 2020 03:34:48:  83000000 \\nINFO  @ Tue, 03 Nov 2020 03:34:52:  84000000 \\nINFO  @ Tue, 03 Nov 2020 03:34:56:  85000000 \\nINFO  @ Tue, 03 Nov 2020 03:35:01:  86000000 \\nINFO  @ Tue, 03 Nov 2020 03:35:05:  87000000 \\nINFO  @ Tue, 03 Nov 2020 03:35:09:  88000000 \\nINFO  @ Tue, 03 Nov 2020 03:35:14:  89000000 \\nINFO  @ Tue, 03 Nov 2020 03:35:18:  90000000 \\nINFO  @ Tue, 03 Nov 2020 03:35:23:  91000000 \\nINFO  @ Tue, 03 Nov 2020 03:35:27:  92000000 \\nINFO  @ Tue, 03 Nov 2020 03:35:31:  93000000 \\nINFO  @ Tue, 03 Nov 2020 03:35:36:  94000000 \\nINFO  @ Tue, 03 Nov 2020 03:35:40:  95000000 \\nINFO  @ Tue, 03 Nov 2020 03:35:44:  96000000 \\nINFO  @ Tue, 03 Nov 2020 03:35:48:  97000000 \\nINFO  @ Tue, 03 Nov 2020 03:35:53:  98000000 \\nINFO  @ Tue, 03 Nov 2020 03:35:57:  99000000 \\nINFO  @ Tue, 03 Nov 2020 03:36:01:  100000000 \\nINFO  @ Tue, 03 Nov 2020 03:36:05:  101000000 \\nINFO  @ Tue, 03 Nov 2020 03:36:09:  102000000 \\nINFO  @ Tue, 03 Nov 2020 03:36:13:  103000000 \\nINFO  @ Tue, 03 Nov 2020 03:36:17:  104000000 \\nINFO  @ Tue, 03 Nov 2020 03:36:22:  105000000 \\nINFO  @ Tue, 03 Nov 2020 03:36:26:  106000000 \\nINFO  @ Tue, 03 Nov 2020 03:36:30:  107000000 \\nINFO  @ Tue, 03 Nov 2020 03:36:34:  108000000 \\nINFO  @ Tue, 03 Nov 2020 03:36:39:  109000000 \\nINFO  @ Tue, 03 Nov 2020 03:36:43:  110000000 \\nINFO  @ Tue, 03 Nov 2020 03:36:47:  111000000 \\nINFO  @ Tue, 03 Nov 2020 03:36:52:  112000000 \\nINFO  @ Tue, 03 Nov 2020 03:36:56:  113000000 \\nINFO  @ Tue, 03 Nov 2020 03:36:57: 113234286 fragments have been read. \\nINFO  @ Tue, 03 Nov 2020 03:38:15: #1.2 read input fragments... \\nINFO  @ Tue, 03 Nov 2020 03:38:19:  1000000 \\nINFO  @ Tue, 03 Nov 2020 03:38:23:  2000000 \\nINFO  @ Tue, 03 Nov 2020 03:38:27:  3000000 \\nINFO  @ Tue, 03 Nov 2020 03:38:30:  4000000 \\nINFO  @ Tue, 03 Nov 2020 03:38:34:  5000000 \\nINFO  @ Tue, 03 Nov 2020 03:38:38:  6000000 \\nINFO  @ Tue, 03 Nov 2020 03:38:41:  7000000 \\nINFO  @ Tue, 03 Nov 2020 03:38:45:  8000000 \\nINFO  @ Tue, 03 Nov 2020 03:38:49:  9000000 \\nINFO  @ Tue, 03 Nov 2020 03:38:53:  10000000 \\nINFO  @ Tue, 03 Nov 2020 03:38:56:  11000000 \\nINFO  @ Tue, 03 Nov 2020 03:39:00:  12000000 \\nINFO  @ Tue, 03 Nov 2020 03:39:04:  13000000 \\nINFO  @ Tue, 03 Nov 2020 03:39:07:  14000000 \\nINFO  @ Tue, 03 Nov 2020 03:39:11:  15000000 \\nINFO  @ Tue, 03 Nov 2020 03:39:15:  16000000 \\nINFO  @ Tue, 03 Nov 2020 03:39:18:  17000000 \\nINFO  @ Tue, 03 Nov 2020 03:39:22:  18000000 \\nINFO  @ Tue, 03 Nov 2020 03:39:26:  19000000 \\nINFO  @ Tue, 03 Nov 2020 03:39:30:  20000000 \\nINFO  @ Tue, 03 Nov 2020 03:39:33:  21000000 \\nINFO  @ Tue, 03 Nov 2020 03:39:37:  22000000 \\nINFO  @ Tue, 03 Nov 2020 03:39:41:  23000000 \\nINFO  @ Tue, 03 Nov 2020 03:39:44:  24000000 \\nINFO  @ Tue, 03 Nov 2020 03:39:48:  25000000 \\nINFO  @ Tue, 03 Nov 2020 03:39:52:  26000000 \\nINFO  @ Tue, 03 Nov 2020 03:39:55:  27000000 \\nINFO  @ Tue, 03 Nov 2020 03:39:59:  28000000 \\nINFO  @ Tue, 03 Nov 2020 03:40:03:  29000000 \\nINFO  @ Tue, 03 Nov 2020 03:40:07:  30000000 \\nINFO  @ Tue, 03 Nov 2020 03:40:10:  31000000 \\nINFO  @ Tue, 03 Nov 2020 03:40:14:  32000000 \\nINFO  @ Tue, 03 Nov 2020 03:40:18:  33000000 \\nINFO  @ Tue, 03 Nov 2020 03:40:21:  34000000 \\nINFO  @ Tue, 03 Nov 2020 03:40:25:  35000000 \\nINFO  @ Tue, 03 Nov 2020 03:40:29:  36000000 \\nINFO  @ Tue, 03 Nov 2020 03:40:32:  37000000 \\nINFO  @ Tue, 03 Nov 2020 03:40:36:  38000000 \\nINFO  @ Tue, 03 Nov 2020 03:40:40:  39000000 \\nINFO  @ Tue, 03 Nov 2020 03:40:44:  40000000 \\nINFO  @ Tue, 03 Nov 2020 03:40:47:  41000000 \\nINFO  @ Tue, 03 Nov 2020 03:40:51:  42000000 \\nINFO  @ Tue, 03 Nov 2020 03:40:55:  43000000 \\nINFO  @ Tue, 03 Nov 2020 03:40:59:  44000000 \\nINFO  @ Tue, 03 Nov 2020 03:41:02:  45000000 \\nINFO  @ Tue, 03 Nov 2020 03:41:06:  46000000 \\nINFO  @ Tue, 03 Nov 2020 03:41:10:  47000000 \\nINFO  @ Tue, 03 Nov 2020 03:41:14:  48000000 \\nINFO  @ Tue, 03 Nov 2020 03:41:17:  49000000 \\nINFO  @ Tue, 03 Nov 2020 03:41:21:  50000000 \\nINFO  @ Tue, 03 Nov 2020 03:41:25:  51000000 \\nINFO  @ Tue, 03 Nov 2020 03:41:28:  52000000 \\nINFO  @ Tue, 03 Nov 2020 03:41:32:  53000000 \\nINFO  @ Tue, 03 Nov 2020 03:41:36:  54000000 \\nINFO  @ Tue, 03 Nov 2020 03:41:40:  55000000 \\nINFO  @ Tue, 03 Nov 2020 03:41:43:  56000000 \\nINFO  @ Tue, 03 Nov 2020 03:41:47:  57000000 \\nINFO  @ Tue, 03 Nov 2020 03:41:51:  58000000 \\nINFO  @ Tue, 03 Nov 2020 03:41:55:  59000000 \\nINFO  @ Tue, 03 Nov 2020 03:41:58:  60000000 \\nINFO  @ Tue, 03 Nov 2020 03:42:02:  61000000 \\nINFO  @ Tue, 03 Nov 2020 03:42:06:  62000000 \\nINFO  @ Tue, 03 Nov 2020 03:42:10:  63000000 \\nINFO  @ Tue, 03 Nov 2020 03:42:15:  64000000 \\nINFO  @ Tue, 03 Nov 2020 03:42:20:  65000000 \\nINFO  @ Tue, 03 Nov 2020 03:42:23:  66000000 \\nINFO  @ Tue, 03 Nov 2020 03:42:26: 66660428 fragments have been read. \\nINFO  @ Tue, 03 Nov 2020 03:43:10: #1 mean fragment size is determined as 273.6 bp from treatment \\nINFO  @ Tue, 03 Nov 2020 03:43:10: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Tue, 03 Nov 2020 03:43:10: #1 fragment size = 273.6 \\nINFO  @ Tue, 03 Nov 2020 03:43:10: #1  total fragments in treatment: 113234286 \\nINFO  @ Tue, 03 Nov 2020 03:43:10: #1 user defined the maximum fragments... \\nINFO  @ Tue, 03 Nov 2020 03:43:10: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Tue, 03 Nov 2020 03:46:14: #1  fragments after filtering in treatment: 113199348 \\nINFO  @ Tue, 03 Nov 2020 03:46:14: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Tue, 03 Nov 2020 03:46:14: #1  total fragments in control: 66660428 \\nINFO  @ Tue, 03 Nov 2020 03:46:14: #1 user defined the maximum fragments... \\nINFO  @ Tue, 03 Nov 2020 03:46:14: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Tue, 03 Nov 2020 03:48:02: #1  fragments after filtering in control: 66632618 \\nINFO  @ Tue, 03 Nov 2020 03:48:02: #1  Redundant rate of control: 0.00 \\nINFO  @ Tue, 03 Nov 2020 03:48:02: #1 finished! \\nINFO  @ Tue, 03 Nov 2020 03:48:02: #2 Build Peak Model... \\nINFO  @ Tue, 03 Nov 2020 03:48:02: #2 Skipped... \\nINFO  @ Tue, 03 Nov 2020 03:48:02: #3 Call peaks... \\nINFO  @ Tue, 03 Nov 2020 03:48:02: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 03 Nov 2020 04:00:06: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 03 Nov 2020 04:00:06: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_treat_pileup.bdg \\nINFO  @ Tue, 03 Nov 2020 04:00:06: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_control_lambda.bdg \\nINFO  @ Tue, 03 Nov 2020 04:00:06: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Tue, 03 Nov 2020 04:00:06: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 03 Nov 2020 04:10:22: #4 Write output xls file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_peaks.xls \\nINFO  @ Tue, 03 Nov 2020 04:10:23: #4 Write peak in narrowPeak format file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_peaks.narrowPeak \\nINFO  @ Tue, 03 Nov 2020 04:10:23: #4 Write summits bed file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_summits.bed \\nINFO  @ Tue, 03 Nov 2020 04:10:23: Done! \\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'INFO  @ Tue, 03 Nov 2020 04:10:24: \\n# Command line: callpeak -B -t ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-POLII_R1.mLb.clN.sorted.bam -c ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 250 -n MV411_MEF2D_NT_SC_63_VHL-POLII_R1 --outdir ../../data/chipseq_MEF2D_degraded/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_MEF2D_NT_SC_63_VHL-POLII_R1\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-POLII_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Tue, 03 Nov 2020 04:10:24: #1 read fragment files... \\nINFO  @ Tue, 03 Nov 2020 04:10:24: #1 read treatment fragments... \\nINFO  @ Tue, 03 Nov 2020 04:10:28:  1000000 \\nINFO  @ Tue, 03 Nov 2020 04:10:32:  2000000 \\nINFO  @ Tue, 03 Nov 2020 04:10:36:  3000000 \\nINFO  @ Tue, 03 Nov 2020 04:10:41:  4000000 \\nINFO  @ Tue, 03 Nov 2020 04:10:45:  5000000 \\nINFO  @ Tue, 03 Nov 2020 04:10:49:  6000000 \\nINFO  @ Tue, 03 Nov 2020 04:10:54:  7000000 \\nINFO  @ Tue, 03 Nov 2020 04:10:58:  8000000 \\nINFO  @ Tue, 03 Nov 2020 04:11:02:  9000000 \\nINFO  @ Tue, 03 Nov 2020 04:11:06:  10000000 \\nINFO  @ Tue, 03 Nov 2020 04:11:11:  11000000 \\nINFO  @ Tue, 03 Nov 2020 04:11:15:  12000000 \\nINFO  @ Tue, 03 Nov 2020 04:11:20:  13000000 \\nINFO  @ Tue, 03 Nov 2020 04:11:24:  14000000 \\nINFO  @ Tue, 03 Nov 2020 04:11:28:  15000000 \\nINFO  @ Tue, 03 Nov 2020 04:11:33:  16000000 \\nINFO  @ Tue, 03 Nov 2020 04:11:37:  17000000 \\nINFO  @ Tue, 03 Nov 2020 04:11:41:  18000000 \\nINFO  @ Tue, 03 Nov 2020 04:11:45:  19000000 \\nINFO  @ Tue, 03 Nov 2020 04:11:49:  20000000 \\nINFO  @ Tue, 03 Nov 2020 04:11:54:  21000000 \\nINFO  @ Tue, 03 Nov 2020 04:11:58:  22000000 \\nINFO  @ Tue, 03 Nov 2020 04:12:02:  23000000 \\nINFO  @ Tue, 03 Nov 2020 04:12:07:  24000000 \\nINFO  @ Tue, 03 Nov 2020 04:12:11:  25000000 \\nINFO  @ Tue, 03 Nov 2020 04:12:15:  26000000 \\nINFO  @ Tue, 03 Nov 2020 04:12:19:  27000000 \\nINFO  @ Tue, 03 Nov 2020 04:12:24:  28000000 \\nINFO  @ Tue, 03 Nov 2020 04:12:28:  29000000 \\nINFO  @ Tue, 03 Nov 2020 04:12:32:  30000000 \\nINFO  @ Tue, 03 Nov 2020 04:12:37:  31000000 \\nINFO  @ Tue, 03 Nov 2020 04:12:41:  32000000 \\nINFO  @ Tue, 03 Nov 2020 04:12:46:  33000000 \\nINFO  @ Tue, 03 Nov 2020 04:12:50:  34000000 \\nINFO  @ Tue, 03 Nov 2020 04:12:55:  35000000 \\nINFO  @ Tue, 03 Nov 2020 04:12:59:  36000000 \\nINFO  @ Tue, 03 Nov 2020 04:13:04:  37000000 \\nINFO  @ Tue, 03 Nov 2020 04:13:08:  38000000 \\nINFO  @ Tue, 03 Nov 2020 04:13:12:  39000000 \\nINFO  @ Tue, 03 Nov 2020 04:13:16:  40000000 \\nINFO  @ Tue, 03 Nov 2020 04:13:20:  41000000 \\nINFO  @ Tue, 03 Nov 2020 04:13:25:  42000000 \\nINFO  @ Tue, 03 Nov 2020 04:13:29:  43000000 \\nINFO  @ Tue, 03 Nov 2020 04:13:34:  44000000 \\nINFO  @ Tue, 03 Nov 2020 04:13:38:  45000000 \\nINFO  @ Tue, 03 Nov 2020 04:13:42:  46000000 \\nINFO  @ Tue, 03 Nov 2020 04:13:47:  47000000 \\nINFO  @ Tue, 03 Nov 2020 04:13:51:  48000000 \\nINFO  @ Tue, 03 Nov 2020 04:13:56:  49000000 \\nINFO  @ Tue, 03 Nov 2020 04:14:00:  50000000 \\nINFO  @ Tue, 03 Nov 2020 04:14:04:  51000000 \\nINFO  @ Tue, 03 Nov 2020 04:14:09:  52000000 \\nINFO  @ Tue, 03 Nov 2020 04:14:13:  53000000 \\nINFO  @ Tue, 03 Nov 2020 04:14:18:  54000000 \\nINFO  @ Tue, 03 Nov 2020 04:14:22:  55000000 \\nINFO  @ Tue, 03 Nov 2020 04:14:26:  56000000 \\nINFO  @ Tue, 03 Nov 2020 04:14:30:  57000000 \\nINFO  @ Tue, 03 Nov 2020 04:14:35:  58000000 \\nINFO  @ Tue, 03 Nov 2020 04:14:39:  59000000 \\nINFO  @ Tue, 03 Nov 2020 04:14:44:  60000000 \\nINFO  @ Tue, 03 Nov 2020 04:14:48:  61000000 \\nINFO  @ Tue, 03 Nov 2020 04:14:52:  62000000 \\nINFO  @ Tue, 03 Nov 2020 04:14:57:  63000000 \\nINFO  @ Tue, 03 Nov 2020 04:15:01:  64000000 \\nINFO  @ Tue, 03 Nov 2020 04:15:06:  65000000 \\nINFO  @ Tue, 03 Nov 2020 04:15:10:  66000000 \\nINFO  @ Tue, 03 Nov 2020 04:15:14:  67000000 \\nINFO  @ Tue, 03 Nov 2020 04:15:18:  68000000 \\nINFO  @ Tue, 03 Nov 2020 04:15:23:  69000000 \\nINFO  @ Tue, 03 Nov 2020 04:15:27:  70000000 \\nINFO  @ Tue, 03 Nov 2020 04:15:31:  71000000 \\nINFO  @ Tue, 03 Nov 2020 04:15:36:  72000000 \\nINFO  @ Tue, 03 Nov 2020 04:15:40:  73000000 \\nINFO  @ Tue, 03 Nov 2020 04:15:45:  74000000 \\nINFO  @ Tue, 03 Nov 2020 04:15:49:  75000000 \\nINFO  @ Tue, 03 Nov 2020 04:15:53:  76000000 \\nINFO  @ Tue, 03 Nov 2020 04:15:58:  77000000 \\nINFO  @ Tue, 03 Nov 2020 04:16:03:  78000000 \\nINFO  @ Tue, 03 Nov 2020 04:16:07:  79000000 \\nINFO  @ Tue, 03 Nov 2020 04:16:12:  80000000 \\nINFO  @ Tue, 03 Nov 2020 04:16:16:  81000000 \\nINFO  @ Tue, 03 Nov 2020 04:16:21:  82000000 \\nINFO  @ Tue, 03 Nov 2020 04:16:25:  83000000 \\nINFO  @ Tue, 03 Nov 2020 04:16:29:  84000000 \\nINFO  @ Tue, 03 Nov 2020 04:16:34:  85000000 \\nINFO  @ Tue, 03 Nov 2020 04:16:38:  86000000 \\nINFO  @ Tue, 03 Nov 2020 04:16:42:  87000000 \\nINFO  @ Tue, 03 Nov 2020 04:16:47:  88000000 \\nINFO  @ Tue, 03 Nov 2020 04:16:51:  89000000 \\nINFO  @ Tue, 03 Nov 2020 04:16:55:  90000000 \\nINFO  @ Tue, 03 Nov 2020 04:16:59:  91000000 \\nINFO  @ Tue, 03 Nov 2020 04:17:03:  92000000 \\nINFO  @ Tue, 03 Nov 2020 04:17:08:  93000000 \\nINFO  @ Tue, 03 Nov 2020 04:17:12:  94000000 \\nINFO  @ Tue, 03 Nov 2020 04:17:16:  95000000 \\nINFO  @ Tue, 03 Nov 2020 04:17:21:  96000000 \\nINFO  @ Tue, 03 Nov 2020 04:17:25:  97000000 \\nINFO  @ Tue, 03 Nov 2020 04:17:29:  98000000 \\nINFO  @ Tue, 03 Nov 2020 04:17:33:  99000000 \\nINFO  @ Tue, 03 Nov 2020 04:17:37:  100000000 \\nINFO  @ Tue, 03 Nov 2020 04:17:42:  101000000 \\nINFO  @ Tue, 03 Nov 2020 04:17:46:  102000000 \\nINFO  @ Tue, 03 Nov 2020 04:17:50:  103000000 \\nINFO  @ Tue, 03 Nov 2020 04:17:54:  104000000 \\nINFO  @ Tue, 03 Nov 2020 04:17:59:  105000000 \\nINFO  @ Tue, 03 Nov 2020 04:18:03:  106000000 \\nINFO  @ Tue, 03 Nov 2020 04:18:08:  107000000 \\nINFO  @ Tue, 03 Nov 2020 04:18:12:  108000000 \\nINFO  @ Tue, 03 Nov 2020 04:18:16:  109000000 \\nINFO  @ Tue, 03 Nov 2020 04:18:21:  110000000 \\nINFO  @ Tue, 03 Nov 2020 04:18:25:  111000000 \\nINFO  @ Tue, 03 Nov 2020 04:18:30:  112000000 \\nINFO  @ Tue, 03 Nov 2020 04:18:30: 112023126 fragments have been read. \\nINFO  @ Tue, 03 Nov 2020 04:19:48: #1.2 read input fragments... \\nINFO  @ Tue, 03 Nov 2020 04:19:51:  1000000 \\nINFO  @ Tue, 03 Nov 2020 04:19:55:  2000000 \\nINFO  @ Tue, 03 Nov 2020 04:19:59:  3000000 \\nINFO  @ Tue, 03 Nov 2020 04:20:02:  4000000 \\nINFO  @ Tue, 03 Nov 2020 04:20:06:  5000000 \\nINFO  @ Tue, 03 Nov 2020 04:20:10:  6000000 \\nINFO  @ Tue, 03 Nov 2020 04:20:13:  7000000 \\nINFO  @ Tue, 03 Nov 2020 04:20:17:  8000000 \\nINFO  @ Tue, 03 Nov 2020 04:20:21:  9000000 \\nINFO  @ Tue, 03 Nov 2020 04:20:24:  10000000 \\nINFO  @ Tue, 03 Nov 2020 04:20:28:  11000000 \\nINFO  @ Tue, 03 Nov 2020 04:20:32:  12000000 \\nINFO  @ Tue, 03 Nov 2020 04:20:35:  13000000 \\nINFO  @ Tue, 03 Nov 2020 04:20:39:  14000000 \\nINFO  @ Tue, 03 Nov 2020 04:20:43:  15000000 \\nINFO  @ Tue, 03 Nov 2020 04:20:46:  16000000 \\nINFO  @ Tue, 03 Nov 2020 04:20:50:  17000000 \\nINFO  @ Tue, 03 Nov 2020 04:20:54:  18000000 \\nINFO  @ Tue, 03 Nov 2020 04:20:57:  19000000 \\nINFO  @ Tue, 03 Nov 2020 04:21:01:  20000000 \\nINFO  @ Tue, 03 Nov 2020 04:21:05:  21000000 \\nINFO  @ Tue, 03 Nov 2020 04:21:08:  22000000 \\nINFO  @ Tue, 03 Nov 2020 04:21:12:  23000000 \\nINFO  @ Tue, 03 Nov 2020 04:21:16:  24000000 \\nINFO  @ Tue, 03 Nov 2020 04:21:19:  25000000 \\nINFO  @ Tue, 03 Nov 2020 04:21:23:  26000000 \\nINFO  @ Tue, 03 Nov 2020 04:21:27:  27000000 \\nINFO  @ Tue, 03 Nov 2020 04:21:30:  28000000 \\nINFO  @ Tue, 03 Nov 2020 04:21:34:  29000000 \\nINFO  @ Tue, 03 Nov 2020 04:21:38:  30000000 \\nINFO  @ Tue, 03 Nov 2020 04:21:41:  31000000 \\nINFO  @ Tue, 03 Nov 2020 04:21:45:  32000000 \\nINFO  @ Tue, 03 Nov 2020 04:21:49:  33000000 \\nINFO  @ Tue, 03 Nov 2020 04:21:52:  34000000 \\nINFO  @ Tue, 03 Nov 2020 04:21:56:  35000000 \\nINFO  @ Tue, 03 Nov 2020 04:21:59:  36000000 \\nINFO  @ Tue, 03 Nov 2020 04:22:03:  37000000 \\nINFO  @ Tue, 03 Nov 2020 04:22:07:  38000000 \\nINFO  @ Tue, 03 Nov 2020 04:22:10:  39000000 \\nINFO  @ Tue, 03 Nov 2020 04:22:14:  40000000 \\nINFO  @ Tue, 03 Nov 2020 04:22:18:  41000000 \\nINFO  @ Tue, 03 Nov 2020 04:22:23:  42000000 \\nINFO  @ Tue, 03 Nov 2020 04:22:26:  43000000 \\nINFO  @ Tue, 03 Nov 2020 04:22:30:  44000000 \\nINFO  @ Tue, 03 Nov 2020 04:22:34:  45000000 \\nINFO  @ Tue, 03 Nov 2020 04:22:37:  46000000 \\nINFO  @ Tue, 03 Nov 2020 04:22:41:  47000000 \\nINFO  @ Tue, 03 Nov 2020 04:22:45:  48000000 \\nINFO  @ Tue, 03 Nov 2020 04:22:48:  49000000 \\nINFO  @ Tue, 03 Nov 2020 04:22:52:  50000000 \\nINFO  @ Tue, 03 Nov 2020 04:22:56:  51000000 \\nINFO  @ Tue, 03 Nov 2020 04:22:59:  52000000 \\nINFO  @ Tue, 03 Nov 2020 04:23:03:  53000000 \\nINFO  @ Tue, 03 Nov 2020 04:23:07:  54000000 \\nINFO  @ Tue, 03 Nov 2020 04:23:11:  55000000 \\nINFO  @ Tue, 03 Nov 2020 04:23:14:  56000000 \\nINFO  @ Tue, 03 Nov 2020 04:23:18:  57000000 \\nINFO  @ Tue, 03 Nov 2020 04:23:22:  58000000 \\nINFO  @ Tue, 03 Nov 2020 04:23:26:  59000000 \\nINFO  @ Tue, 03 Nov 2020 04:23:29:  60000000 \\nINFO  @ Tue, 03 Nov 2020 04:23:33:  61000000 \\nINFO  @ Tue, 03 Nov 2020 04:23:37:  62000000 \\nINFO  @ Tue, 03 Nov 2020 04:23:40:  63000000 \\nINFO  @ Tue, 03 Nov 2020 04:23:44:  64000000 \\nINFO  @ Tue, 03 Nov 2020 04:23:48:  65000000 \\nINFO  @ Tue, 03 Nov 2020 04:23:52:  66000000 \\nINFO  @ Tue, 03 Nov 2020 04:23:54: 66660428 fragments have been read. \\nINFO  @ Tue, 03 Nov 2020 04:24:38: #1 mean fragment size is determined as 266.2 bp from treatment \\nINFO  @ Tue, 03 Nov 2020 04:24:38: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Tue, 03 Nov 2020 04:24:38: #1 fragment size = 266.2 \\nINFO  @ Tue, 03 Nov 2020 04:24:38: #1  total fragments in treatment: 112023126 \\nINFO  @ Tue, 03 Nov 2020 04:24:38: #1 user defined the maximum fragments... \\nINFO  @ Tue, 03 Nov 2020 04:24:38: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Tue, 03 Nov 2020 04:27:44: #1  fragments after filtering in treatment: 111988498 \\nINFO  @ Tue, 03 Nov 2020 04:27:44: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Tue, 03 Nov 2020 04:27:44: #1  total fragments in control: 66660428 \\nINFO  @ Tue, 03 Nov 2020 04:27:44: #1 user defined the maximum fragments... \\nINFO  @ Tue, 03 Nov 2020 04:27:44: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Tue, 03 Nov 2020 04:29:34: #1  fragments after filtering in control: 66632618 \\nINFO  @ Tue, 03 Nov 2020 04:29:34: #1  Redundant rate of control: 0.00 \\nINFO  @ Tue, 03 Nov 2020 04:29:34: #1 finished! \\nINFO  @ Tue, 03 Nov 2020 04:29:34: #2 Build Peak Model... \\nINFO  @ Tue, 03 Nov 2020 04:29:34: #2 Skipped... \\nINFO  @ Tue, 03 Nov 2020 04:29:34: #3 Call peaks... \\nINFO  @ Tue, 03 Nov 2020 04:29:34: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 03 Nov 2020 04:41:49: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 03 Nov 2020 04:41:49: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_MEF2D_NT_SC_63_VHL-POLII_R1_treat_pileup.bdg \\nINFO  @ Tue, 03 Nov 2020 04:41:49: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_MEF2D_NT_SC_63_VHL-POLII_R1_control_lambda.bdg \\nINFO  @ Tue, 03 Nov 2020 04:41:49: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Tue, 03 Nov 2020 04:41:49: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 03 Nov 2020 04:52:31: #4 Write output xls file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-POLII_R1_peaks.xls \\nINFO  @ Tue, 03 Nov 2020 04:52:31: #4 Write peak in narrowPeak format file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-POLII_R1_peaks.narrowPeak \\nINFO  @ Tue, 03 Nov 2020 04:52:32: #4 Write summits bed file... ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-POLII_R1_summits.bed \\nINFO  @ Tue, 03 Nov 2020 04:52:32: Done! \\n'\n",
      "66632618 78884973\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_treat_pileup.bdg --c1 ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_control_lambda.bdg --t2 ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-POLII_R1_treat_pileup.bdg --c2 ../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-POLII_R1_control_lambda.bdg --d1 66632618 --d2 78884973 -g 60 -l 250 --o-prefix MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-POLII_R1_treat_pileup --outdir ../../data/chipseq_MEF2D_degraded/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Tue, 03 Nov 2020 04:52:32: Read and build treatment 1 bedGraph... \\nINFO  @ Tue, 03 Nov 2020 04:55:34: Read and build control 1 bedGraph... \\nINFO  @ Tue, 03 Nov 2020 04:59:16: Read and build treatment 2 bedGraph... \\nINFO  @ Tue, 03 Nov 2020 05:02:05: Read and build control 2 bedGraph... \\nINFO  @ Tue, 03 Nov 2020 05:22:27: Write peaks... \\nINFO  @ Tue, 03 Nov 2020 05:22:28: Done \\n')\n"
     ]
    }
   ],
   "source": [
    "# diffPeak on scaled data \n",
    "for i in range(int((len(bams)-1)/2)):\n",
    "    if i==4:\n",
    "        continue\n",
    "    name1 = bams[i+1]\n",
    "    name2 = bams[i+7]\n",
    "    print('\\n')\n",
    "    print(name1,name2)\n",
    "    print(chip.fullDiffPeak(name1, name2, control1=bams[0], res_directory=\"../../data/\"+project+\"/diffPeaks/\", scaling=[scales[i],scales[i+6]], directory=\"../../data/\"+project+'/diffData/'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 202,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['728 ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_treat_pileup_c3.0_common.bed',\n",
       " '10695 ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '245 ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_treat_pileup_c3.0_cond2.bed',\n",
       " '17543 ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_treat_pileup_c3.0_common.bed',\n",
       " '253 ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '16862 ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_treat_pileup_c3.0_cond2.bed',\n",
       " '5711 ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MED1_R1_treat_pileup_c3.0_common.bed',\n",
       " '11 ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MED1_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '684 ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MED1_R1_treat_pileup_c3.0_cond2.bed',\n",
       " '681 ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_treat_pileup_c3.0_common.bed',\n",
       " '990 ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '7437 ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_treat_pileup_c3.0_cond2.bed',\n",
       " '11800 ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MYC_R1_treat_pileup_c3.0_common.bed',\n",
       " '48949 ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MYC_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '612 ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MYC_R1_treat_pileup_c3.0_cond2.bed',\n",
       " '85366 ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-POLII_R1_treat_pileup_c3.0_common.bed',\n",
       " '12879 ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-POLII_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '391 ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-POLII_R1_treat_pileup_c3.0_cond2.bed',\n",
       " '']"
      ]
     },
     "execution_count": 202,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "os.popen('for i in $(ls ../../data/'+project+'/diffPeaks/*.bed); \\\n",
    "            do echo $(wc -l $i); \\\n",
    "            done').read().split('\\n')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 203,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1.bw',\n",
       " '../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1.bw',\n",
       " '../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1.bw',\n",
       " '../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1.bw',\n",
       " '../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1.bw',\n",
       " '../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1.bw',\n",
       " '../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1.bw',\n",
       " '../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1.bw',\n",
       " '../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_VHL-MED1_R1.bw',\n",
       " '../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1.bw',\n",
       " '../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_VHL-MYC_R1.bw',\n",
       " '../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_VHL-POLII_R1.bw']"
      ]
     },
     "execution_count": 203,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bw = ! ls ../../data/$project/recalib_bigwig/*\n",
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 151,
   "metadata": {},
   "outputs": [],
   "source": [
    "!mkdir ../results/$project/\n",
    "!mkdir ../results/$project/plots\n",
    "!mkdir ../results/$project/plots/heatmaps/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# GENOME WIDE comparison"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 157,
   "metadata": {},
   "outputs": [],
   "source": [
    "peaks = ! ls ../../data/$project/bwa/mergedLibrary/macs/broadPeak/*.broadPeak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 171,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"DMSO_FLAG_MEF2D\",\n",
    "\"DMSO_IFR8\",\n",
    "\"DMSO_MED1\",\n",
    "\"DMSO_MEF2C\",\n",
    "\"DMSO_MYC\",\n",
    "\"DMSO_POLII\",\n",
    "\"VHL_FLAG_MEF2D\",\n",
    "\"VHL_IFR8\",\n",
    "\"VHL_MED1\",\n",
    "\"VHL_MEF2C\",\n",
    "\"VHL_MYC\",\n",
    "\"VHL_POLII\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 162,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mkdir ../../data/$project/peakplot/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 207,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FLAG_MEF2D\n"
     ]
    },
    {
     "ename": "KeyboardInterrupt",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m-------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m                       Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-207-afc83ccb6aa2>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m      4\u001b[0m     \u001b[0mname\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mnames\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mi\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      5\u001b[0m     \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mname\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 6\u001b[0;31m     \u001b[0mchip\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mgetPeaksAt\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mpeaks\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mi\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mbigwigs\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mval\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mbigwignames\u001b[0m\u001b[0;34m=\u001b[0m \u001b[0mname\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mpeaknames\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m'Macs2_Peaks'\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mwindow\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;36m3000\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mfolder\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m\"\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mtitle\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mname\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mnumthreads\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;36m8\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrefpoint\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m\"center\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mname\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m'../../data/'\u001b[0m\u001b[0;34m+\u001b[0m\u001b[0mproject\u001b[0m\u001b[0;34m+\u001b[0m\u001b[0;34m'/peakplot/'\u001b[0m\u001b[0;34m+\u001b[0m\u001b[0mname\u001b[0m\u001b[0;34m+\u001b[0m\u001b[0;34m'_mat.pdf'\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mwithDeeptools\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mTrue\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mtorecompute\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mTrue\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mvmax\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;36m2.5\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mlegendLoc\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m\"lower-left\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;32m~/JKBio/epigenetics/ChIP_helper.py\u001b[0m in \u001b[0;36mgetPeaksAt\u001b[0;34m(peaks, bigwigs, folder, bigwignames, peaknames, window, title, numpeaks, numthreads, width, length, torecompute, name, refpoint, scale, sort, withDeeptools, onlyProfile, cluster, vmax, vmin, overlap, legendLoc)\u001b[0m\n\u001b[1;32m    372\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mtitle\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    373\u001b[0m             \u001b[0mcmd\u001b[0m \u001b[0;34m+=\u001b[0m \u001b[0;34m\" --plotTitle \"\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mtitle\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 374\u001b[0;31m         \u001b[0mdata\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0msubprocess\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrun\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mcmd\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mshell\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mTrue\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcapture_output\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mTrue\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    375\u001b[0m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    376\u001b[0m     \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/lib/python3.8/subprocess.py\u001b[0m in \u001b[0;36mrun\u001b[0;34m(input, capture_output, timeout, check, *popenargs, **kwargs)\u001b[0m\n\u001b[1;32m    489\u001b[0m     \u001b[0;32mwith\u001b[0m \u001b[0mPopen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0mpopenargs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;32mas\u001b[0m \u001b[0mprocess\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    490\u001b[0m         \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 491\u001b[0;31m             \u001b[0mstdout\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mstderr\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mprocess\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mcommunicate\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0minput\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mtimeout\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mtimeout\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    492\u001b[0m         \u001b[0;32mexcept\u001b[0m \u001b[0mTimeoutExpired\u001b[0m \u001b[0;32mas\u001b[0m \u001b[0mexc\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    493\u001b[0m             \u001b[0mprocess\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mkill\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/lib/python3.8/subprocess.py\u001b[0m in \u001b[0;36mcommunicate\u001b[0;34m(self, input, timeout)\u001b[0m\n\u001b[1;32m   1022\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1023\u001b[0m             \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1024\u001b[0;31m                 \u001b[0mstdout\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mstderr\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_communicate\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0minput\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mendtime\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mtimeout\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1025\u001b[0m             \u001b[0;32mexcept\u001b[0m \u001b[0mKeyboardInterrupt\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1026\u001b[0m                 \u001b[0;31m# https://bugs.python.org/issue25942\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/lib/python3.8/subprocess.py\u001b[0m in \u001b[0;36m_communicate\u001b[0;34m(self, input, endtime, orig_timeout)\u001b[0m\n\u001b[1;32m   1864\u001b[0m                             'failed to raise TimeoutExpired.')\n\u001b[1;32m   1865\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1866\u001b[0;31m                     \u001b[0mready\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mselector\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mselect\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mtimeout\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1867\u001b[0m                     \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_check_timeout\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mendtime\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0morig_timeout\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mstdout\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mstderr\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1868\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/lib/python3.8/selectors.py\u001b[0m in \u001b[0;36mselect\u001b[0;34m(self, timeout)\u001b[0m\n\u001b[1;32m    413\u001b[0m         \u001b[0mready\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    414\u001b[0m         \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 415\u001b[0;31m             \u001b[0mfd_event_list\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_selector\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mpoll\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mtimeout\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    416\u001b[0m         \u001b[0;32mexcept\u001b[0m \u001b[0mInterruptedError\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    417\u001b[0m             \u001b[0;32mreturn\u001b[0m \u001b[0mready\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
     ]
    }
   ],
   "source": [
    "for i, val in enumerate(bw):\n",
    "    if i <0:\n",
    "        continue\n",
    "    name = names[i]\n",
    "    print(name)\n",
    "    chip.getPeaksAt(peaks[i], bigwigs = val, bigwignames= name, peaknames=['Macs2_Peaks'], window=3000, folder=\"\", title=name, numthreads=8, refpoint=\"center\", name='../../data/'+project+'/peakplot/'+name+'_mat.pdf', withDeeptools=True, torecompute=True, vmax=2.5, legendLoc=\"lower-left\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 179,
   "metadata": {},
   "outputs": [],
   "source": [
    "rm ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-POLII_R1_tre*"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 204,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MED1_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MYC_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-POLII_R1_treat_pileup_c3.0_cond1.bed']"
      ]
     },
     "execution_count": 204,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "cond1peak = ! ls ../../data/$project/diffPeaks/*cond1.bed\n",
    "cond2peak = ! ls ../../data/$project/diffPeaks/*cond2.bed\n",
    "commonpeak = ! ls ../../data/$project/diffPeaks/*common.bed\n",
    "cond1peak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 205,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"FLAG_MEF2D\",\n",
    "\"IFR8\",\n",
    "\"MED1\",\n",
    "\"MEF2C\",\n",
    "\"MYC\",\n",
    "\"POLII\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 209,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1.bw ../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1.bw ../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1.bw  --referencePoint center --regionsFileName ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_treat_pileup_c3.0_cond1.bed ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_treat_pileup_c3.0_common.bed ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_treat_pileup_c3.0_cond2.bed --missingDataAsZero --outFileName ../../data/chipseq_MEF2D_degraded/diffPeaks/FLAG_MEF2D_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/chipseq_MEF2D_degraded/diffPeaks/FLAG_MEF2D_mat.gz --outFileName ../../data/chipseq_MEF2D_degraded/diffPeaks/FLAG_MEF2D_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks --plotTitle FLAG_MEF2D', returncode=0, stdout=b'', stderr=b'')\n",
      "../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1.bw ../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1.bw ../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1.bw  --referencePoint center --regionsFileName ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_treat_pileup_c3.0_cond1.bed ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_treat_pileup_c3.0_common.bed ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_treat_pileup_c3.0_cond2.bed --missingDataAsZero --outFileName ../../data/chipseq_MEF2D_degraded/diffPeaks/IFR8_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/chipseq_MEF2D_degraded/diffPeaks/IFR8_mat.gz --outFileName ../../data/chipseq_MEF2D_degraded/diffPeaks/IFR8_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks --plotTitle IFR8', returncode=0, stdout=b'', stderr=b'')\n",
      "../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1.bw ../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_VHL-MED1_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1.bw ../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_VHL-MED1_R1.bw  --referencePoint center --regionsFileName ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MED1_R1_treat_pileup_c3.0_common.bed ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MED1_R1_treat_pileup_c3.0_cond2.bed --missingDataAsZero --outFileName ../../data/chipseq_MEF2D_degraded/diffPeaks/MED1_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/chipseq_MEF2D_degraded/diffPeaks/MED1_mat.gz --outFileName ../../data/chipseq_MEF2D_degraded/diffPeaks/MED1_mat.pdf --refPointLabel center --regionsLabel common VHL_peaks --plotTitle MED1', returncode=0, stdout=b'', stderr=b'')\n",
      "../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1.bw ../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1.bw ../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1.bw  --referencePoint center --regionsFileName ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_treat_pileup_c3.0_cond1.bed ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_treat_pileup_c3.0_common.bed ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_treat_pileup_c3.0_cond2.bed --missingDataAsZero --outFileName ../../data/chipseq_MEF2D_degraded/diffPeaks/MEF2C_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/chipseq_MEF2D_degraded/diffPeaks/MEF2C_mat.gz --outFileName ../../data/chipseq_MEF2D_degraded/diffPeaks/MEF2C_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks --plotTitle MEF2C', returncode=0, stdout=b'', stderr=b'')\n",
      "../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1.bw ../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_VHL-MYC_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1.bw ../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_VHL-MYC_R1.bw  --referencePoint center --regionsFileName ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MYC_R1_treat_pileup_c3.0_cond1.bed ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MYC_R1_treat_pileup_c3.0_common.bed ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MYC_R1_treat_pileup_c3.0_cond2.bed --missingDataAsZero --outFileName ../../data/chipseq_MEF2D_degraded/diffPeaks/MYC_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/chipseq_MEF2D_degraded/diffPeaks/MYC_mat.gz --outFileName ../../data/chipseq_MEF2D_degraded/diffPeaks/MYC_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks --plotTitle MYC', returncode=0, stdout=b'', stderr=b'')\n",
      "../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1.bw ../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_VHL-POLII_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1.bw ../../data/chipseq_MEF2D_degraded/recalib_bigwig/MV411_MEF2D_NT_SC_63_VHL-POLII_R1.bw  --referencePoint center --regionsFileName ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-POLII_R1_treat_pileup_c3.0_cond1.bed ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-POLII_R1_treat_pileup_c3.0_common.bed ../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-POLII_R1_treat_pileup_c3.0_cond2.bed --missingDataAsZero --outFileName ../../data/chipseq_MEF2D_degraded/diffPeaks/POLII_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/chipseq_MEF2D_degraded/diffPeaks/POLII_mat.gz --outFileName ../../data/chipseq_MEF2D_degraded/diffPeaks/POLII_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks --plotTitle POLII', returncode=0, stdout=b'', stderr=b\"WARNING: Group 'MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-POLII_R1_treat_pileup_c3.0_cond2.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n"
     ]
    }
   ],
   "source": [
    "for i in range(int(len(bw)/2)):\n",
    "    if i <0:\n",
    "        continue\n",
    "    cl = len(bw)\n",
    "    name1 = bw[i]\n",
    "    name2 = bw[i+6]\n",
    "    peak = [cond1peak[i], commonpeak[i], cond2peak[i]]\n",
    "   # for val in peak:\n",
    "      #  chip.dropWeirdChromosomes(val)\n",
    "    name = names[i]\n",
    "    print(name1,name2)\n",
    "    chip.getPeaksAt(peak, [name1, name2], bigwignames=['DMSO', 'VHL'], peaknames=['DMSO_peaks', 'common', 'VHL_peaks'], window=3000, folder=\"\", title=name, numthreads=7, refpoint=\"center\", name='../../data/'+project+'/diffPeaks/'+name+'_mat.pdf', withDeeptools=True, torecompute=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 210,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "#h.createFoldersFor('../results/'+project+'/plots/scaled/diffPeaks/')\n",
    "! cp ../../data/$project/diffPeaks/*.pdf ../results/$project/plots/scaled/diffPeaks/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 211,
   "metadata": {},
   "outputs": [],
   "source": [
    "!cp ../../data/$project/peakplot/*_mat.pdf ../results/$project/plots/scaled/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### looking at TSS"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 301,
   "metadata": {},
   "outputs": [],
   "source": [
    "server = BiomartServer( \"http://www.ensembl.org/biomart\" )\n",
    "ensembl = server.datasets['hsapiens_gene_ensembl']\n",
    "ensembltss = pd.read_csv(io.StringIO(ensmbl.search({\n",
    "  'attributes': ['ensembl_gene_id','gene_biotype', \"transcription_start_site\", \"3_utr_start\",\"start_position\",\"external_gene_name\"]\n",
    "}, header=1).content.decode()), sep='\\t')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 298,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'feature_page': 'Features': (attributes: {'ensembl_gene_id': 'Gene stable ID' (default: True), 'ensembl_gene_id_version': 'Gene stable ID version' (default: True), 'ensembl_transcript_id': 'Transcript stable ID' (default: True), 'ensembl_transcript_id_version': 'Transcript stable ID version' (default: True), 'ensembl_peptide_id': 'Protein stable ID' (default: False), 'ensembl_peptide_id_version': 'Protein stable ID version' (default: False), 'ensembl_exon_id': 'Exon stable ID' (default: False), 'description': 'Gene description' (default: False), 'chromosome_name': 'Chromosome/scaffold name' (default: False), 'start_position': 'Gene start (bp)' (default: False), 'end_position': 'Gene end (bp)' (default: False), 'strand': 'Strand' (default: False), 'band': 'Karyotype band' (default: False), 'transcript_start': 'Transcript start (bp)' (default: False), 'transcript_end': 'Transcript end (bp)' (default: False), 'transcription_start_site': 'Transcription start site (TSS)' (default: False), 'transcript_length': 'Transcript length (including UTRs and CDS)' (default: False), 'transcript_tsl': 'Transcript support level (TSL)' (default: False), 'transcript_gencode_basic': 'GENCODE basic annotation' (default: False), 'transcript_appris': 'APPRIS annotation' (default: False), 'transcript_mane_select': 'RefSeq match transcript' (default: False), 'external_gene_name': 'Gene name' (default: False), 'external_gene_source': 'Source of gene name' (default: False), 'external_transcript_name': 'Transcript name' (default: False), 'external_transcript_source_name': 'Source of transcript name' (default: False), 'transcript_count': 'Transcript count' (default: False), 'percentage_gene_gc_content': 'Gene % GC content' (default: False), 'gene_biotype': 'Gene type' (default: False), 'transcript_biotype': 'Transcript type' (default: False), 'source': 'Source (gene)' (default: False), 'transcript_source': 'Source (transcript)' (default: False), 'version': 'Version (gene)' (default: False), 'transcript_version': 'Version (transcript)' (default: False), 'peptide_version': 'Version (protein)' (default: False), 'external_synonym': 'Gene Synonym' (default: False), 'phenotype_description': 'Phenotype description' (default: False), 'source_name': 'Source name' (default: False), 'study_external_id': 'Study external reference' (default: False), 'strain_name': 'Strain name' (default: False), 'strain_gender': 'Strain gender' (default: False), 'p_value': 'P value' (default: False), 'go_id': 'GO term accession' (default: False), 'name_1006': 'GO term name' (default: False), 'definition_1006': 'GO term definition' (default: False), 'go_linkage_type': 'GO term evidence code' (default: False), 'namespace_1003': 'GO domain' (default: False), 'goslim_goa_accession': 'GOSlim GOA Accession(s)' (default: False), 'goslim_goa_description': 'GOSlim GOA Description' (default: False), 'ccds': 'CCDS ID' (default: False), 'chembl': 'ChEMBL ID' (default: False), 'clone_based_ensembl_gene': 'Clone-based (Ensembl) gene ID' (default: False), 'clone_based_ensembl_transcript': 'Clone-based (Ensembl) transcript ID' (default: False), 'dbass3_name': 'DataBase of Aberrant 3' Splice Sites name' (default: False), 'dbass3_id': 'DataBase of Aberrant 3' Splice Sites ID' (default: False), 'dbass5_name': 'DataBase of Aberrant 5' Splice Sites name' (default: False), 'dbass5_id': 'DataBase of Aberrant 5' Splice Sites ID' (default: False), 'ens_hs_transcript': 'Ensembl Human Transcript ID' (default: False), 'ens_hs_translation': 'Ensembl Human Translation ID' (default: False), 'entrezgene_trans_name': 'EntrezGene transcript name ID' (default: False), 'embl': 'European Nucleotide Archive ID' (default: False), 'arrayexpress': 'Expression Atlas ID' (default: False), 'genedb': 'GeneDB ID' (default: False), 'hgnc_id': 'HGNC ID' (default: False), 'hgnc_symbol': 'HGNC symbol' (default: False), 'hpa_accession': 'Human Protein Atlas accession' (default: False), 'hpa_id': 'Human Protein Atlas ID' (default: False), 'protein_id': 'INSDC protein ID' (default: False), 'ens_lrg_gene': 'LRG display in Ensembl gene ID' (default: False), 'ens_lrg_transcript': 'LRG display in Ensembl transcript ID' (default: False), 'merops': 'MEROPS - the Peptidase Database ID' (default: False), 'mim_gene_description': 'MIM gene description' (default: False), 'mim_gene_accession': 'MIM gene accession' (default: False), 'mim_morbid_description': 'MIM morbid description' (default: False), 'mim_morbid_accession': 'MIM morbid accession' (default: False), 'mirbase_accession': 'miRBase accession' (default: False), 'mirbase_id': 'miRBase ID' (default: False), 'mirbase_trans_name': 'miRBase transcript name ID' (default: False), 'entrezgene_description': 'NCBI gene (formerly Entrezgene) description' (default: False), 'entrezgene_accession': 'NCBI gene (formerly Entrezgene) accession' (default: False), 'entrezgene_id': 'NCBI gene (formerly Entrezgene) ID' (default: False), 'pdb': 'PDB ID' (default: False), 'reactome': 'Reactome ID' (default: False), 'reactome_gene': 'Reactome gene ID' (default: False), 'reactome_transcript': 'Reactome transcript ID' (default: False), 'refseq_mrna': 'RefSeq mRNA ID' (default: False), 'refseq_mrna_predicted': 'RefSeq mRNA predicted ID' (default: False), 'refseq_ncrna': 'RefSeq ncRNA ID' (default: False), 'refseq_ncrna_predicted': 'RefSeq ncRNA predicted ID' (default: False), 'refseq_peptide': 'RefSeq peptide ID' (default: False), 'refseq_peptide_predicted': 'RefSeq peptide predicted ID' (default: False), 'rfam': 'RFAM ID' (default: False), 'rfam_trans_name': 'RFAM transcript name ID' (default: False), 'rnacentral': 'RNAcentral ID' (default: False), 'hgnc_trans_name': 'Transcript name ID' (default: False), 'ucsc': 'UCSC Stable ID' (default: False), 'uniparc': 'UniParc ID' (default: False), 'uniprot_gn_symbol': 'UniProtKB Gene Name symbol' (default: False), 'uniprot_gn_id': 'UniProtKB Gene Name ID' (default: False), 'uniprotswissprot': 'UniProtKB/Swiss-Prot ID' (default: False), 'uniprotsptrembl': 'UniProtKB/TrEMBL ID' (default: False), 'wikigene_description': 'WikiGene description' (default: False), 'wikigene_name': 'WikiGene name' (default: False), 'wikigene_id': 'WikiGene ID' (default: False), 'affy_hc_g110': 'AFFY HC G110 probe' (default: False), 'affy_hg_focus': 'AFFY HG Focus probe' (default: False), 'affy_hg_u133a': 'AFFY HG U133A probe' (default: False), 'affy_hg_u133a_2': 'AFFY HG U133A 2 probe' (default: False), 'affy_hg_u133b': 'AFFY HG U133B probe' (default: False), 'affy_hg_u133_plus_2': 'AFFY HG U133 Plus 2 probe' (default: False), 'affy_hg_u95a': 'AFFY HG U95A probe' (default: False), 'affy_hg_u95av2': 'AFFY HG U95Av2 probe' (default: False), 'affy_hg_u95b': 'AFFY HG U95B probe' (default: False), 'affy_hg_u95c': 'AFFY HG U95C probe' (default: False), 'affy_hg_u95d': 'AFFY HG U95D probe' (default: False), 'affy_hg_u95e': 'AFFY HG U95E probe' (default: False), 'affy_hta_2_0': 'AFFY HTA 2 0 probe' (default: False), 'affy_huex_1_0_st_v2': 'AFFY HuEx 1 0 st v2 probe' (default: False), 'affy_hugenefl': 'AFFY HuGeneFL probe' (default: False), 'affy_hugene_1_0_st_v1': 'AFFY HuGene 1 0 st v1 probe' (default: False), 'affy_hugene_2_0_st_v1': 'AFFY HuGene 2 0 st v1 probe' (default: False), 'affy_primeview': 'AFFY PrimeView probe' (default: False), 'affy_u133_x3p': 'AFFY U133 X3P probe' (default: False), 'agilent_cgh_44b': 'AGILENT CGH 44b probe' (default: False), 'agilent_gpl6848': 'AGILENT GPL6848 probe' (default: False), 'agilent_sureprint_g3_ge_8x60k': 'AGILENT SurePrint G3 GE 8x60k probe' (default: False), 'agilent_sureprint_g3_ge_8x60k_v2': 'AGILENT SurePrint G3 GE 8x60k v2 probe' (default: False), 'agilent_wholegenome': 'AGILENT WholeGenome probe' (default: False), 'agilent_wholegenome_4x44k_v1': 'AGILENT WholeGenome 4x44k v1 probe' (default: False), 'agilent_wholegenome_4x44k_v2': 'AGILENT WholeGenome 4x44k v2 probe' (default: False), 'codelink_codelink': 'CODELINK CODELINK probe' (default: False), 'illumina_humanht_12_v3': 'ILLUMINA HumanHT 12 V3 probe' (default: False), 'illumina_humanht_12_v4': 'ILLUMINA HumanHT 12 V4 probe' (default: False), 'illumina_humanref_8_v3': 'ILLUMINA HumanRef 8 V3 probe' (default: False), 'illumina_humanwg_6_v1': 'ILLUMINA HumanWG 6 V1 probe' (default: False), 'illumina_humanwg_6_v2': 'ILLUMINA HumanWG 6 V2 probe' (default: False), 'illumina_humanwg_6_v3': 'ILLUMINA HumanWG 6 V3 probe' (default: False), 'phalanx_onearray': 'PHALANX OneArray probe' (default: False), 'family': 'Ensembl Protein Family ID(s)' (default: False), 'family_description': 'Ensembl Family Description' (default: False), 'cdd': 'CDD ID' (default: False), 'cdd_start': 'CDD start' (default: False), 'cdd_end': 'CDD end' (default: False), 'gene3d': 'Gene3D ID' (default: False), 'gene3d_start': 'Gene3D start' (default: False), 'gene3d_end': 'Gene3D end' (default: False), 'hamap': 'HAMAP ID' (default: False), 'hamap_start': 'HAMAP start' (default: False), 'hamap_end': 'HAMAP end' (default: False), 'hmmpanther': 'PANTHER ID' (default: False), 'hmmpanther_start': 'PANTHER start' (default: False), 'hmmpanther_end': 'PANTHER end' (default: False), 'pfam': 'Pfam ID' (default: False), 'pfam_start': 'Pfam start' (default: False), 'pfam_end': 'Pfam end' (default: False), 'pirsf': 'PIRSF ID' (default: False), 'pirsf_start': 'PIRSF start' (default: False), 'pirsf_end': 'PIRSF end' (default: False), 'prints': 'Prints ID' (default: False), 'prints_start': 'Prints start' (default: False), 'prints_end': 'Prints end' (default: False), 'scanprosite': 'PROSITE patterns ID' (default: False), 'scanprosite_start': 'PROSITE patterns start' (default: False), 'scanprosite_end': 'PROSITE patterns end' (default: False), 'pfscan': 'PROSITE profiles ID' (default: False), 'pfscan_start': 'PROSITE profiles start' (default: False), 'pfscan_end': 'PROSITE profiles end' (default: False), 'sfld': 'SFLD ID' (default: False), 'sfld_start': 'SFLD start' (default: False), 'sfld_end': 'SFLD end' (default: False), 'smart': 'SMART ID' (default: False), 'smart_start': 'SMART start' (default: False), 'smart_end': 'SMART end' (default: False), 'superfamily': 'Superfamily ID' (default: False), 'superfamily_start': 'Superfamily start' (default: False), 'superfamily_end': 'Superfamily end' (default: False), 'tigrfam': 'TIGRFAM ID' (default: False), 'tigrfam_start': 'TIGRFAM start' (default: False), 'tigrfam_end': 'TIGRFAM end' (default: False), 'interpro': 'Interpro ID' (default: False), 'interpro_short_description': 'Interpro Short Description' (default: False), 'interpro_description': 'Interpro Description' (default: False), 'interpro_start': 'Interpro start' (default: False), 'interpro_end': 'Interpro end' (default: False), 'mobidblite': 'MobiDB lite' (default: False), 'mobidblite_start': 'MobiDB lite start' (default: False), 'mobidblite_end': 'MobiDB lite end' (default: False), 'ncoils': 'Coiled-coils (Ncoils)' (default: False), 'ncoils_start': 'Coiled-coils (Ncoils) start' (default: False), 'ncoils_end': 'Coiled-coils (Ncoils) end' (default: False), 'seg': 'Low complexity (Seg)' (default: False), 'seg_start': 'Low complexity (Seg) start' (default: False), 'seg_end': 'Low complexity (Seg) end' (default: False), 'sifts_import': 'PDB-ENSP mappings' (default: False), 'sifts_import_start': 'PDB-ENSP mappings start' (default: False), 'sifts_import_end': 'PDB-ENSP mappings end' (default: False), 'signalp': 'Cleavage site (Signalp)' (default: False), 'signalp_start': 'Cleavage site (Signalp) start' (default: False), 'signalp_end': 'Cleavage site (Signalp) end' (default: False), 'tmhmm': 'Transmembrane helices' (default: False), 'tmhmm_start': 'Transmembrane helices start' (default: False), 'tmhmm_end': 'Transmembrane helices end' (default: False)}, defaults: ['ensembl_gene_id', 'ensembl_gene_id_version', 'ensembl_transcript_id', 'ensembl_transcript_id_version']),\n",
      " 'homologs': 'Homologues (Max select 6 orthologues)': (attributes: {'ensembl_gene_id': 'Gene stable ID' (default: False), 'ensembl_gene_id_version': 'Gene stable ID version' (default: False), 'version': 'Version (gene)' (default: False), 'ensembl_transcript_id': 'Transcript stable ID' (default: False), 'ensembl_transcript_id_version': 'Transcript stable ID version' (default: False), 'transcript_version': 'Version (transcript)' (default: False), 'ensembl_peptide_id': 'Protein stable ID' (default: False), 'ensembl_peptide_id_version': 'Protein stable ID version' (default: False), 'peptide_version': 'Version (protein)' (default: False), 'chromosome_name': 'Chromosome/scaffold name' (default: False), 'start_position': 'Gene start (bp)' (default: False), 'end_position': 'Gene end (bp)' (default: False), 'strand': 'Strand' (default: False), 'band': 'Karyotype band' (default: False), 'external_gene_name': 'Gene name' (default: False), 'external_gene_source': 'Source of gene name' (default: False), 'transcript_count': 'Transcript count' (default: False), 'percentage_gene_gc_content': 'Gene % GC content' (default: False), 'description': 'Gene description' (default: False), 'cabingdonii_homolog_ensembl_gene': 'Abingdon island giant tortoise gene stable ID' (default: False), 'cabingdonii_homolog_associated_gene_name': 'Abingdon island giant tortoise gene name' (default: False), 'cabingdonii_homolog_ensembl_peptide': 'Abingdon island giant tortoise protein or transcript stable ID' (default: False), 'cabingdonii_homolog_chromosome': 'Abingdon island giant tortoise chromosome/scaffold name' (default: False), 'cabingdonii_homolog_chrom_start': 'Abingdon island giant tortoise chromosome/scaffold start (bp)' (default: False), 'cabingdonii_homolog_chrom_end': 'Abingdon island giant tortoise chromosome/scaffold end (bp)' (default: False), 'cabingdonii_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'cabingdonii_homolog_subtype': 'Last common ancestor with Abingdon island giant tortoise' (default: False), 'cabingdonii_homolog_orthology_type': 'Abingdon island giant tortoise homology type' (default: False), 'cabingdonii_homolog_perc_id': '%id. target Abingdon island giant tortoise gene identical to query gene' (default: False), 'cabingdonii_homolog_perc_id_r1': '%id. query gene identical to target Abingdon island giant tortoise gene' (default: False), 'cabingdonii_homolog_goc_score': 'Abingdon island giant tortoise Gene-order conservation score' (default: False), 'cabingdonii_homolog_wga_coverage': 'Abingdon island giant tortoise Whole-genome alignment coverage' (default: False), 'cabingdonii_homolog_orthology_confidence': 'Abingdon island giant tortoise orthology confidence [0 low, 1 high]' (default: False), 'gagassizii_homolog_ensembl_gene': 'Agassiz's desert tortoise gene stable ID' (default: False), 'gagassizii_homolog_associated_gene_name': 'Agassiz's desert tortoise gene name' (default: False), 'gagassizii_homolog_ensembl_peptide': 'Agassiz's desert tortoise protein or transcript stable ID' (default: False), 'gagassizii_homolog_chromosome': 'Agassiz's desert tortoise chromosome/scaffold name' (default: False), 'gagassizii_homolog_chrom_start': 'Agassiz's desert tortoise chromosome/scaffold start (bp)' (default: False), 'gagassizii_homolog_chrom_end': 'Agassiz's desert tortoise chromosome/scaffold end (bp)' (default: False), 'gagassizii_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'gagassizii_homolog_subtype': 'Last common ancestor with Agassiz's desert tortoise' (default: False), 'gagassizii_homolog_orthology_type': 'Agassiz's desert tortoise homology type' (default: False), 'gagassizii_homolog_perc_id': '%id. target Agassiz's desert tortoise gene identical to query gene' (default: False), 'gagassizii_homolog_perc_id_r1': '%id. query gene identical to target Agassiz's desert tortoise gene' (default: False), 'gagassizii_homolog_goc_score': 'Agassiz's desert tortoise Gene-order conservation score' (default: False), 'gagassizii_homolog_wga_coverage': 'Agassiz's desert tortoise Whole-genome alignment coverage' (default: False), 'gagassizii_homolog_orthology_confidence': 'Agassiz's desert tortoise orthology confidence [0 low, 1 high]' (default: False), 'mspretus_homolog_ensembl_gene': 'Algerian mouse gene stable ID' (default: False), 'mspretus_homolog_associated_gene_name': 'Algerian mouse gene name' (default: False), 'mspretus_homolog_ensembl_peptide': 'Algerian mouse protein or transcript stable ID' (default: False), 'mspretus_homolog_chromosome': 'Algerian mouse chromosome/scaffold name' (default: False), 'mspretus_homolog_chrom_start': 'Algerian mouse chromosome/scaffold start (bp)' (default: False), 'mspretus_homolog_chrom_end': 'Algerian mouse chromosome/scaffold end (bp)' (default: False), 'mspretus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'mspretus_homolog_subtype': 'Last common ancestor with Algerian mouse' (default: False), 'mspretus_homolog_orthology_type': 'Algerian mouse homology type' (default: False), 'mspretus_homolog_perc_id': '%id. target Algerian mouse gene identical to query gene' (default: False), 'mspretus_homolog_perc_id_r1': '%id. query gene identical to target Algerian mouse gene' (default: False), 'mspretus_homolog_goc_score': 'Algerian mouse Gene-order conservation score' (default: False), 'mspretus_homolog_wga_coverage': 'Algerian mouse Whole-genome alignment coverage' (default: False), 'mspretus_homolog_orthology_confidence': 'Algerian mouse orthology confidence [0 low, 1 high]' (default: False), 'vpacos_homolog_ensembl_gene': 'Alpaca gene stable ID' (default: False), 'vpacos_homolog_associated_gene_name': 'Alpaca gene name' (default: False), 'vpacos_homolog_ensembl_peptide': 'Alpaca protein or transcript stable ID' (default: False), 'vpacos_homolog_chromosome': 'Alpaca chromosome/scaffold name' (default: False), 'vpacos_homolog_chrom_start': 'Alpaca chromosome/scaffold start (bp)' (default: False), 'vpacos_homolog_chrom_end': 'Alpaca chromosome/scaffold end (bp)' (default: False), 'vpacos_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'vpacos_homolog_subtype': 'Last common ancestor with Alpaca' (default: False), 'vpacos_homolog_orthology_type': 'Alpaca homology type' (default: False), 'vpacos_homolog_perc_id': '%id. target Alpaca gene identical to query gene' (default: False), 'vpacos_homolog_perc_id_r1': '%id. query gene identical to target Alpaca gene' (default: False), 'vpacos_homolog_goc_score': 'Alpaca Gene-order conservation score' (default: False), 'vpacos_homolog_wga_coverage': 'Alpaca Whole-genome alignment coverage' (default: False), 'vpacos_homolog_orthology_confidence': 'Alpaca orthology confidence [0 low, 1 high]' (default: False), 'mmmarmota_homolog_ensembl_gene': 'Alpine marmot gene stable ID' (default: False), 'mmmarmota_homolog_associated_gene_name': 'Alpine marmot gene name' (default: False), 'mmmarmota_homolog_ensembl_peptide': 'Alpine marmot protein or transcript stable ID' (default: False), 'mmmarmota_homolog_chromosome': 'Alpine marmot chromosome/scaffold name' (default: False), 'mmmarmota_homolog_chrom_start': 'Alpine marmot chromosome/scaffold start (bp)' (default: False), 'mmmarmota_homolog_chrom_end': 'Alpine marmot chromosome/scaffold end (bp)' (default: False), 'mmmarmota_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'mmmarmota_homolog_subtype': 'Last common ancestor with Alpine marmot' (default: False), 'mmmarmota_homolog_orthology_type': 'Alpine marmot homology type' (default: False), 'mmmarmota_homolog_perc_id': '%id. target Alpine marmot gene identical to query gene' (default: False), 'mmmarmota_homolog_perc_id_r1': '%id. query gene identical to target Alpine marmot gene' (default: False), 'mmmarmota_homolog_goc_score': 'Alpine marmot Gene-order conservation score' (default: False), 'mmmarmota_homolog_wga_coverage': 'Alpine marmot Whole-genome alignment coverage' (default: False), 'mmmarmota_homolog_orthology_confidence': 'Alpine marmot orthology confidence [0 low, 1 high]' (default: False), 'pformosa_homolog_ensembl_gene': 'Amazon molly gene stable ID' (default: False), 'pformosa_homolog_associated_gene_name': 'Amazon molly gene name' (default: False), 'pformosa_homolog_ensembl_peptide': 'Amazon molly protein or transcript stable ID' (default: False), 'pformosa_homolog_chromosome': 'Amazon molly chromosome/scaffold name' (default: False), 'pformosa_homolog_chrom_start': 'Amazon molly chromosome/scaffold start (bp)' (default: False), 'pformosa_homolog_chrom_end': 'Amazon molly chromosome/scaffold end (bp)' (default: False), 'pformosa_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'pformosa_homolog_subtype': 'Last common ancestor with Amazon molly' (default: False), 'pformosa_homolog_orthology_type': 'Amazon molly homology type' (default: False), 'pformosa_homolog_perc_id': '%id. target Amazon molly gene identical to query gene' (default: False), 'pformosa_homolog_perc_id_r1': '%id. query gene identical to target Amazon molly gene' (default: False), 'pformosa_homolog_goc_score': 'Amazon molly Gene-order conservation score' (default: False), 'pformosa_homolog_wga_coverage': 'Amazon molly Whole-genome alignment coverage' (default: False), 'pformosa_homolog_orthology_confidence': 'Amazon molly orthology confidence [0 low, 1 high]' (default: False), 'ccanadensis_homolog_ensembl_gene': 'American beaver gene stable ID' (default: False), 'ccanadensis_homolog_associated_gene_name': 'American beaver gene name' (default: False), 'ccanadensis_homolog_ensembl_peptide': 'American beaver protein or transcript stable ID' (default: False), 'ccanadensis_homolog_chromosome': 'American beaver chromosome/scaffold name' (default: False), 'ccanadensis_homolog_chrom_start': 'American beaver chromosome/scaffold start (bp)' (default: False), 'ccanadensis_homolog_chrom_end': 'American beaver chromosome/scaffold end (bp)' (default: False), 'ccanadensis_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'ccanadensis_homolog_subtype': 'Last common ancestor with American beaver' (default: False), 'ccanadensis_homolog_orthology_type': 'American beaver homology type' (default: False), 'ccanadensis_homolog_perc_id': '%id. target American beaver gene identical to query gene' (default: False), 'ccanadensis_homolog_perc_id_r1': '%id. query gene identical to target American beaver gene' (default: False), 'ccanadensis_homolog_goc_score': 'American beaver Gene-order conservation score' (default: False), 'ccanadensis_homolog_wga_coverage': 'American beaver Whole-genome alignment coverage' (default: False), 'ccanadensis_homolog_orthology_confidence': 'American beaver orthology confidence [0 low, 1 high]' (default: False), 'bbbison_homolog_ensembl_gene': 'American bison gene stable ID' (default: False), 'bbbison_homolog_associated_gene_name': 'American bison gene name' (default: False), 'bbbison_homolog_ensembl_peptide': 'American bison protein or transcript stable ID' (default: False), 'bbbison_homolog_chromosome': 'American bison chromosome/scaffold name' (default: False), 'bbbison_homolog_chrom_start': 'American bison chromosome/scaffold start (bp)' (default: False), 'bbbison_homolog_chrom_end': 'American bison chromosome/scaffold end (bp)' (default: False), 'bbbison_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'bbbison_homolog_subtype': 'Last common ancestor with American bison' (default: False), 'bbbison_homolog_orthology_type': 'American bison homology type' (default: False), 'bbbison_homolog_perc_id': '%id. target American bison gene identical to query gene' (default: False), 'bbbison_homolog_perc_id_r1': '%id. query gene identical to target American bison gene' (default: False), 'bbbison_homolog_goc_score': 'American bison Gene-order conservation score' (default: False), 'bbbison_homolog_wga_coverage': 'American bison Whole-genome alignment coverage' (default: False), 'bbbison_homolog_orthology_confidence': 'American bison orthology confidence [0 low, 1 high]' (default: False), 'uamericanus_homolog_ensembl_gene': 'American black bear gene stable ID' (default: False), 'uamericanus_homolog_associated_gene_name': 'American black bear gene name' (default: False), 'uamericanus_homolog_ensembl_peptide': 'American black bear protein or transcript stable ID' (default: False), 'uamericanus_homolog_chromosome': 'American black bear chromosome/scaffold name' (default: False), 'uamericanus_homolog_chrom_start': 'American black bear chromosome/scaffold start (bp)' (default: False), 'uamericanus_homolog_chrom_end': 'American black bear chromosome/scaffold end (bp)' (default: False), 'uamericanus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'uamericanus_homolog_subtype': 'Last common ancestor with American black bear' (default: False), 'uamericanus_homolog_orthology_type': 'American black bear homology type' (default: False), 'uamericanus_homolog_perc_id': '%id. target American black bear gene identical to query gene' (default: False), 'uamericanus_homolog_perc_id_r1': '%id. query gene identical to target American black bear gene' (default: False), 'uamericanus_homolog_goc_score': 'American black bear Gene-order conservation score' (default: False), 'uamericanus_homolog_wga_coverage': 'American black bear Whole-genome alignment coverage' (default: False), 'uamericanus_homolog_orthology_confidence': 'American black bear orthology confidence [0 low, 1 high]' (default: False), 'nvison_homolog_ensembl_gene': 'American mink gene stable ID' (default: False), 'nvison_homolog_associated_gene_name': 'American mink gene name' (default: False), 'nvison_homolog_ensembl_peptide': 'American mink protein or transcript stable ID' (default: False), 'nvison_homolog_chromosome': 'American mink chromosome/scaffold name' (default: False), 'nvison_homolog_chrom_start': 'American mink chromosome/scaffold start (bp)' (default: False), 'nvison_homolog_chrom_end': 'American mink chromosome/scaffold end (bp)' (default: False), 'nvison_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'nvison_homolog_subtype': 'Last common ancestor with American mink' (default: False), 'nvison_homolog_orthology_type': 'American mink homology type' (default: False), 'nvison_homolog_perc_id': '%id. target American mink gene identical to query gene' (default: False), 'nvison_homolog_perc_id_r1': '%id. query gene identical to target American mink gene' (default: False), 'nvison_homolog_goc_score': 'American mink Gene-order conservation score' (default: False), 'nvison_homolog_wga_coverage': 'American mink Whole-genome alignment coverage' (default: False), 'nvison_homolog_orthology_confidence': 'American mink orthology confidence [0 low, 1 high]' (default: False), 'capalliatus_homolog_ensembl_gene': 'Angola colobus gene stable ID' (default: False), 'capalliatus_homolog_associated_gene_name': 'Angola colobus gene name' (default: False), 'capalliatus_homolog_ensembl_peptide': 'Angola colobus protein or transcript stable ID' (default: False), 'capalliatus_homolog_chromosome': 'Angola colobus chromosome/scaffold name' (default: False), 'capalliatus_homolog_chrom_start': 'Angola colobus chromosome/scaffold start (bp)' (default: False), 'capalliatus_homolog_chrom_end': 'Angola colobus chromosome/scaffold end (bp)' (default: False), 'capalliatus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'capalliatus_homolog_subtype': 'Last common ancestor with Angola colobus' (default: False), 'capalliatus_homolog_orthology_type': 'Angola colobus homology type' (default: False), 'capalliatus_homolog_perc_id': '%id. target Angola colobus gene identical to query gene' (default: False), 'capalliatus_homolog_perc_id_r1': '%id. query gene identical to target Angola colobus gene' (default: False), 'capalliatus_homolog_goc_score': 'Angola colobus Gene-order conservation score' (default: False), 'capalliatus_homolog_wga_coverage': 'Angola colobus Whole-genome alignment coverage' (default: False), 'capalliatus_homolog_orthology_confidence': 'Angola colobus orthology confidence [0 low, 1 high]' (default: False), 'acarolinensis_homolog_ensembl_gene': 'Anole lizard gene stable ID' (default: False), 'acarolinensis_homolog_associated_gene_name': 'Anole lizard gene name' (default: False), 'acarolinensis_homolog_ensembl_peptide': 'Anole lizard protein or transcript stable ID' (default: False), 'acarolinensis_homolog_chromosome': 'Anole lizard chromosome/scaffold name' (default: False), 'acarolinensis_homolog_chrom_start': 'Anole lizard chromosome/scaffold start (bp)' (default: False), 'acarolinensis_homolog_chrom_end': 'Anole lizard chromosome/scaffold end (bp)' (default: False), 'acarolinensis_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'acarolinensis_homolog_subtype': 'Last common ancestor with Anole lizard' (default: False), 'acarolinensis_homolog_orthology_type': 'Anole lizard homology type' (default: False), 'acarolinensis_homolog_perc_id': '%id. target Anole lizard gene identical to query gene' (default: False), 'acarolinensis_homolog_perc_id_r1': '%id. query gene identical to target Anole lizard gene' (default: False), 'acarolinensis_homolog_goc_score': 'Anole lizard Gene-order conservation score' (default: False), 'acarolinensis_homolog_wga_coverage': 'Anole lizard Whole-genome alignment coverage' (default: False), 'acarolinensis_homolog_orthology_confidence': 'Anole lizard orthology confidence [0 low, 1 high]' (default: False), 'cdromedarius_homolog_ensembl_gene': 'Arabian camel gene stable ID' (default: False), 'cdromedarius_homolog_associated_gene_name': 'Arabian camel gene name' (default: False), 'cdromedarius_homolog_ensembl_peptide': 'Arabian camel protein or transcript stable ID' (default: False), 'cdromedarius_homolog_chromosome': 'Arabian camel chromosome/scaffold name' (default: False), 'cdromedarius_homolog_chrom_start': 'Arabian camel chromosome/scaffold start (bp)' (default: False), 'cdromedarius_homolog_chrom_end': 'Arabian camel chromosome/scaffold end (bp)' (default: False), 'cdromedarius_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'cdromedarius_homolog_subtype': 'Last common ancestor with Arabian camel' (default: False), 'cdromedarius_homolog_orthology_type': 'Arabian camel homology type' (default: False), 'cdromedarius_homolog_perc_id': '%id. target Arabian camel gene identical to query gene' (default: False), 'cdromedarius_homolog_perc_id_r1': '%id. query gene identical to target Arabian camel gene' (default: False), 'cdromedarius_homolog_goc_score': 'Arabian camel Gene-order conservation score' (default: False), 'cdromedarius_homolog_wga_coverage': 'Arabian camel Whole-genome alignment coverage' (default: False), 'cdromedarius_homolog_orthology_confidence': 'Arabian camel orthology confidence [0 low, 1 high]' (default: False), 'smerianae_homolog_ensembl_gene': 'Argentine black and white tegu gene stable ID' (default: False), 'smerianae_homolog_associated_gene_name': 'Argentine black and white tegu gene name' (default: False), 'smerianae_homolog_ensembl_peptide': 'Argentine black and white tegu protein or transcript stable ID' (default: False), 'smerianae_homolog_chromosome': 'Argentine black and white tegu chromosome/scaffold name' (default: False), 'smerianae_homolog_chrom_start': 'Argentine black and white tegu chromosome/scaffold start (bp)' (default: False), 'smerianae_homolog_chrom_end': 'Argentine black and white tegu chromosome/scaffold end (bp)' (default: False), 'smerianae_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'smerianae_homolog_subtype': 'Last common ancestor with Argentine black and white tegu' (default: False), 'smerianae_homolog_orthology_type': 'Argentine black and white tegu homology type' (default: False), 'smerianae_homolog_perc_id': '%id. target Argentine black and white tegu gene identical to query gene' (default: False), 'smerianae_homolog_perc_id_r1': '%id. query gene identical to target Argentine black and white tegu gene' (default: False), 'smerianae_homolog_goc_score': 'Argentine black and white tegu Gene-order conservation score' (default: False), 'smerianae_homolog_wga_coverage': 'Argentine black and white tegu Whole-genome alignment coverage' (default: False), 'smerianae_homolog_orthology_confidence': 'Argentine black and white tegu orthology confidence [0 low, 1 high]' (default: False), 'dnovemcinctus_homolog_ensembl_gene': 'Armadillo gene stable ID' (default: False), 'dnovemcinctus_homolog_associated_gene_name': 'Armadillo gene name' (default: False), 'dnovemcinctus_homolog_ensembl_peptide': 'Armadillo protein or transcript stable ID' (default: False), 'dnovemcinctus_homolog_chromosome': 'Armadillo chromosome/scaffold name' (default: False), 'dnovemcinctus_homolog_chrom_start': 'Armadillo chromosome/scaffold start (bp)' (default: False), 'dnovemcinctus_homolog_chrom_end': 'Armadillo chromosome/scaffold end (bp)' (default: False), 'dnovemcinctus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'dnovemcinctus_homolog_subtype': 'Last common ancestor with Armadillo' (default: False), 'dnovemcinctus_homolog_orthology_type': 'Armadillo homology type' (default: False), 'dnovemcinctus_homolog_perc_id': '%id. target Armadillo gene identical to query gene' (default: False), 'dnovemcinctus_homolog_perc_id_r1': '%id. query gene identical to target Armadillo gene' (default: False), 'dnovemcinctus_homolog_goc_score': 'Armadillo Gene-order conservation score' (default: False), 'dnovemcinctus_homolog_wga_coverage': 'Armadillo Whole-genome alignment coverage' (default: False), 'dnovemcinctus_homolog_orthology_confidence': 'Armadillo orthology confidence [0 low, 1 high]' (default: False), 'sformosus_homolog_ensembl_gene': 'Asian bonytongue gene stable ID' (default: False), 'sformosus_homolog_associated_gene_name': 'Asian bonytongue gene name' (default: False), 'sformosus_homolog_ensembl_peptide': 'Asian bonytongue protein or transcript stable ID' (default: False), 'sformosus_homolog_chromosome': 'Asian bonytongue chromosome/scaffold name' (default: False), 'sformosus_homolog_chrom_start': 'Asian bonytongue chromosome/scaffold start (bp)' (default: False), 'sformosus_homolog_chrom_end': 'Asian bonytongue chromosome/scaffold end (bp)' (default: False), 'sformosus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'sformosus_homolog_subtype': 'Last common ancestor with Asian bonytongue' (default: False), 'sformosus_homolog_orthology_type': 'Asian bonytongue homology type' (default: False), 'sformosus_homolog_perc_id': '%id. target Asian bonytongue gene identical to query gene' (default: False), 'sformosus_homolog_perc_id_r1': '%id. query gene identical to target Asian bonytongue gene' (default: False), 'sformosus_homolog_goc_score': 'Asian bonytongue Gene-order conservation score' (default: False), 'sformosus_homolog_wga_coverage': 'Asian bonytongue Whole-genome alignment coverage' (default: False), 'sformosus_homolog_orthology_confidence': 'Asian bonytongue orthology confidence [0 low, 1 high]' (default: False), 'charengus_homolog_ensembl_gene': 'Atlantic herring gene stable ID' (default: False), 'charengus_homolog_associated_gene_name': 'Atlantic herring gene name' (default: False), 'charengus_homolog_ensembl_peptide': 'Atlantic herring protein or transcript stable ID' (default: False), 'charengus_homolog_chromosome': 'Atlantic herring chromosome/scaffold name' (default: False), 'charengus_homolog_chrom_start': 'Atlantic herring chromosome/scaffold start (bp)' (default: False), 'charengus_homolog_chrom_end': 'Atlantic herring chromosome/scaffold end (bp)' (default: False), 'charengus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'charengus_homolog_subtype': 'Last common ancestor with Atlantic herring' (default: False), 'charengus_homolog_orthology_type': 'Atlantic herring homology type' (default: False), 'charengus_homolog_perc_id': '%id. target Atlantic herring gene identical to query gene' (default: False), 'charengus_homolog_perc_id_r1': '%id. query gene identical to target Atlantic herring gene' (default: False), 'charengus_homolog_goc_score': 'Atlantic herring Gene-order conservation score' (default: False), 'charengus_homolog_wga_coverage': 'Atlantic herring Whole-genome alignment coverage' (default: False), 'charengus_homolog_orthology_confidence': 'Atlantic herring orthology confidence [0 low, 1 high]' (default: False), 'ssalar_homolog_ensembl_gene': 'Atlantic salmon gene stable ID' (default: False), 'ssalar_homolog_associated_gene_name': 'Atlantic salmon gene name' (default: False), 'ssalar_homolog_ensembl_peptide': 'Atlantic salmon protein or transcript stable ID' (default: False), 'ssalar_homolog_chromosome': 'Atlantic salmon chromosome/scaffold name' (default: False), 'ssalar_homolog_chrom_start': 'Atlantic salmon chromosome/scaffold start (bp)' (default: False), 'ssalar_homolog_chrom_end': 'Atlantic salmon chromosome/scaffold end (bp)' (default: False), 'ssalar_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'ssalar_homolog_subtype': 'Last common ancestor with Atlantic salmon' (default: False), 'ssalar_homolog_orthology_type': 'Atlantic salmon homology type' (default: False), 'ssalar_homolog_perc_id': '%id. target Atlantic salmon gene identical to query gene' (default: False), 'ssalar_homolog_perc_id_r1': '%id. query gene identical to target Atlantic salmon gene' (default: False), 'ssalar_homolog_goc_score': 'Atlantic salmon Gene-order conservation score' (default: False), 'ssalar_homolog_wga_coverage': 'Atlantic salmon Whole-genome alignment coverage' (default: False), 'ssalar_homolog_orthology_confidence': 'Atlantic salmon orthology confidence [0 low, 1 high]' (default: False), 'cporosus_homolog_ensembl_gene': 'Australian saltwater crocodile gene stable ID' (default: False), 'cporosus_homolog_associated_gene_name': 'Australian saltwater crocodile gene name' (default: False), 'cporosus_homolog_ensembl_peptide': 'Australian saltwater crocodile protein or transcript stable ID' (default: False), 'cporosus_homolog_chromosome': 'Australian saltwater crocodile chromosome/scaffold name' (default: False), 'cporosus_homolog_chrom_start': 'Australian saltwater crocodile chromosome/scaffold start (bp)' (default: False), 'cporosus_homolog_chrom_end': 'Australian saltwater crocodile chromosome/scaffold end (bp)' (default: False), 'cporosus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'cporosus_homolog_subtype': 'Last common ancestor with Australian saltwater crocodile' (default: False), 'cporosus_homolog_orthology_type': 'Australian saltwater crocodile homology type' (default: False), 'cporosus_homolog_perc_id': '%id. target Australian saltwater crocodile gene identical to query gene' (default: False), 'cporosus_homolog_perc_id_r1': '%id. query gene identical to target Australian saltwater crocodile gene' (default: False), 'cporosus_homolog_goc_score': 'Australian saltwater crocodile Gene-order conservation score' (default: False), 'cporosus_homolog_wga_coverage': 'Australian saltwater crocodile Whole-genome alignment coverage' (default: False), 'cporosus_homolog_orthology_confidence': 'Australian saltwater crocodile orthology confidence [0 low, 1 high]' (default: False), 'lbergylta_homolog_ensembl_gene': 'Ballan wrasse gene stable ID' (default: False), 'lbergylta_homolog_associated_gene_name': 'Ballan wrasse gene name' (default: False), 'lbergylta_homolog_ensembl_peptide': 'Ballan wrasse protein or transcript stable ID' (default: False), 'lbergylta_homolog_chromosome': 'Ballan wrasse chromosome/scaffold name' (default: False), 'lbergylta_homolog_chrom_start': 'Ballan wrasse chromosome/scaffold start (bp)' (default: False), 'lbergylta_homolog_chrom_end': 'Ballan wrasse chromosome/scaffold end (bp)' (default: False), 'lbergylta_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'lbergylta_homolog_subtype': 'Last common ancestor with Ballan wrasse' (default: False), 'lbergylta_homolog_orthology_type': 'Ballan wrasse homology type' (default: False), 'lbergylta_homolog_perc_id': '%id. target Ballan wrasse gene identical to query gene' (default: False), 'lbergylta_homolog_perc_id_r1': '%id. query gene identical to target Ballan wrasse gene' (default: False), 'lbergylta_homolog_goc_score': 'Ballan wrasse Gene-order conservation score' (default: False), 'lbergylta_homolog_wga_coverage': 'Ballan wrasse Whole-genome alignment coverage' (default: False), 'lbergylta_homolog_orthology_confidence': 'Ballan wrasse orthology confidence [0 low, 1 high]' (default: False), 'lcalcarifer_homolog_ensembl_gene': 'Barramundi perch gene stable ID' (default: False), 'lcalcarifer_homolog_associated_gene_name': 'Barramundi perch gene name' (default: False), 'lcalcarifer_homolog_ensembl_peptide': 'Barramundi perch protein or transcript stable ID' (default: False), 'lcalcarifer_homolog_chromosome': 'Barramundi perch chromosome/scaffold name' (default: False), 'lcalcarifer_homolog_chrom_start': 'Barramundi perch chromosome/scaffold start (bp)' (default: False), 'lcalcarifer_homolog_chrom_end': 'Barramundi perch chromosome/scaffold end (bp)' (default: False), 'lcalcarifer_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'lcalcarifer_homolog_subtype': 'Last common ancestor with Barramundi perch' (default: False), 'lcalcarifer_homolog_orthology_type': 'Barramundi perch homology type' (default: False), 'lcalcarifer_homolog_perc_id': '%id. target Barramundi perch gene identical to query gene' (default: False), 'lcalcarifer_homolog_perc_id_r1': '%id. query gene identical to target Barramundi perch gene' (default: False), 'lcalcarifer_homolog_goc_score': 'Barramundi perch Gene-order conservation score' (default: False), 'lcalcarifer_homolog_wga_coverage': 'Barramundi perch Whole-genome alignment coverage' (default: False), 'lcalcarifer_homolog_orthology_confidence': 'Barramundi perch orthology confidence [0 low, 1 high]' (default: False), 'lsdomestica_homolog_ensembl_gene': 'Bengalese finch gene stable ID' (default: False), 'lsdomestica_homolog_associated_gene_name': 'Bengalese finch gene name' (default: False), 'lsdomestica_homolog_ensembl_peptide': 'Bengalese finch protein or transcript stable ID' (default: False), 'lsdomestica_homolog_chromosome': 'Bengalese finch chromosome/scaffold name' (default: False), 'lsdomestica_homolog_chrom_start': 'Bengalese finch chromosome/scaffold start (bp)' (default: False), 'lsdomestica_homolog_chrom_end': 'Bengalese finch chromosome/scaffold end (bp)' (default: False), 'lsdomestica_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'lsdomestica_homolog_subtype': 'Last common ancestor with Bengalese finch' (default: False), 'lsdomestica_homolog_orthology_type': 'Bengalese finch homology type' (default: False), 'lsdomestica_homolog_perc_id': '%id. target Bengalese finch gene identical to query gene' (default: False), 'lsdomestica_homolog_perc_id_r1': '%id. query gene identical to target Bengalese finch gene' (default: False), 'lsdomestica_homolog_goc_score': 'Bengalese finch Gene-order conservation score' (default: False), 'lsdomestica_homolog_wga_coverage': 'Bengalese finch Whole-genome alignment coverage' (default: False), 'lsdomestica_homolog_orthology_confidence': 'Bengalese finch orthology confidence [0 low, 1 high]' (default: False), 'rbieti_homolog_ensembl_gene': 'Black snub-nosed monkey gene stable ID' (default: False), 'rbieti_homolog_associated_gene_name': 'Black snub-nosed monkey gene name' (default: False), 'rbieti_homolog_ensembl_peptide': 'Black snub-nosed monkey protein or transcript stable ID' (default: False), 'rbieti_homolog_chromosome': 'Black snub-nosed monkey chromosome/scaffold name' (default: False), 'rbieti_homolog_chrom_start': 'Black snub-nosed monkey chromosome/scaffold start (bp)' (default: False), 'rbieti_homolog_chrom_end': 'Black snub-nosed monkey chromosome/scaffold end (bp)' (default: False), 'rbieti_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'rbieti_homolog_subtype': 'Last common ancestor with Black snub-nosed monkey' (default: False), 'rbieti_homolog_orthology_type': 'Black snub-nosed monkey homology type' (default: False), 'rbieti_homolog_perc_id': '%id. target Black snub-nosed monkey gene identical to query gene' (default: False), 'rbieti_homolog_perc_id_r1': '%id. query gene identical to target Black snub-nosed monkey gene' (default: False), 'rbieti_homolog_goc_score': 'Black snub-nosed monkey Gene-order conservation score' (default: False), 'rbieti_homolog_wga_coverage': 'Black snub-nosed monkey Whole-genome alignment coverage' (default: False), 'rbieti_homolog_orthology_confidence': 'Black snub-nosed monkey orthology confidence [0 low, 1 high]' (default: False), 'oaureus_homolog_ensembl_gene': 'Blue tilapia gene stable ID' (default: False), 'oaureus_homolog_associated_gene_name': 'Blue tilapia gene name' (default: False), 'oaureus_homolog_ensembl_peptide': 'Blue tilapia protein or transcript stable ID' (default: False), 'oaureus_homolog_chromosome': 'Blue tilapia chromosome/scaffold name' (default: False), 'oaureus_homolog_chrom_start': 'Blue tilapia chromosome/scaffold start (bp)' (default: False), 'oaureus_homolog_chrom_end': 'Blue tilapia chromosome/scaffold end (bp)' (default: False), 'oaureus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'oaureus_homolog_subtype': 'Last common ancestor with Blue tilapia' (default: False), 'oaureus_homolog_orthology_type': 'Blue tilapia homology type' (default: False), 'oaureus_homolog_perc_id': '%id. target Blue tilapia gene identical to query gene' (default: False), 'oaureus_homolog_perc_id_r1': '%id. query gene identical to target Blue tilapia gene' (default: False), 'oaureus_homolog_goc_score': 'Blue tilapia Gene-order conservation score' (default: False), 'oaureus_homolog_wga_coverage': 'Blue tilapia Whole-genome alignment coverage' (default: False), 'oaureus_homolog_orthology_confidence': 'Blue tilapia orthology confidence [0 low, 1 high]' (default: False), 'gwilldenowi_homolog_ensembl_gene': 'Blunt-snouted clingfish gene stable ID' (default: False), 'gwilldenowi_homolog_associated_gene_name': 'Blunt-snouted clingfish gene name' (default: False), 'gwilldenowi_homolog_ensembl_peptide': 'Blunt-snouted clingfish protein or transcript stable ID' (default: False), 'gwilldenowi_homolog_chromosome': 'Blunt-snouted clingfish chromosome/scaffold name' (default: False), 'gwilldenowi_homolog_chrom_start': 'Blunt-snouted clingfish chromosome/scaffold start (bp)' (default: False), 'gwilldenowi_homolog_chrom_end': 'Blunt-snouted clingfish chromosome/scaffold end (bp)' (default: False), 'gwilldenowi_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'gwilldenowi_homolog_subtype': 'Last common ancestor with Blunt-snouted clingfish' (default: False), 'gwilldenowi_homolog_orthology_type': 'Blunt-snouted clingfish homology type' (default: False), 'gwilldenowi_homolog_perc_id': '%id. target Blunt-snouted clingfish gene identical to query gene' (default: False), 'gwilldenowi_homolog_perc_id_r1': '%id. query gene identical to target Blunt-snouted clingfish gene' (default: False), 'gwilldenowi_homolog_goc_score': 'Blunt-snouted clingfish Gene-order conservation score' (default: False), 'gwilldenowi_homolog_wga_coverage': 'Blunt-snouted clingfish Whole-genome alignment coverage' (default: False), 'gwilldenowi_homolog_orthology_confidence': 'Blunt-snouted clingfish orthology confidence [0 low, 1 high]' (default: False), 'sbboliviensis_homolog_ensembl_gene': 'Bolivian squirrel monkey gene stable ID' (default: False), 'sbboliviensis_homolog_associated_gene_name': 'Bolivian squirrel monkey gene name' (default: False), 'sbboliviensis_homolog_ensembl_peptide': 'Bolivian squirrel monkey protein or transcript stable ID' (default: False), 'sbboliviensis_homolog_chromosome': 'Bolivian squirrel monkey chromosome/scaffold name' (default: False), 'sbboliviensis_homolog_chrom_start': 'Bolivian squirrel monkey chromosome/scaffold start (bp)' (default: False), 'sbboliviensis_homolog_chrom_end': 'Bolivian squirrel monkey chromosome/scaffold end (bp)' (default: False), 'sbboliviensis_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'sbboliviensis_homolog_subtype': 'Last common ancestor with Bolivian squirrel monkey' (default: False), 'sbboliviensis_homolog_orthology_type': 'Bolivian squirrel monkey homology type' (default: False), 'sbboliviensis_homolog_perc_id': '%id. target Bolivian squirrel monkey gene identical to query gene' (default: False), 'sbboliviensis_homolog_perc_id_r1': '%id. query gene identical to target Bolivian squirrel monkey gene' (default: False), 'sbboliviensis_homolog_goc_score': 'Bolivian squirrel monkey Gene-order conservation score' (default: False), 'sbboliviensis_homolog_wga_coverage': 'Bolivian squirrel monkey Whole-genome alignment coverage' (default: False), 'sbboliviensis_homolog_orthology_confidence': 'Bolivian squirrel monkey orthology confidence [0 low, 1 high]' (default: False), 'ppaniscus_homolog_ensembl_gene': 'Bonobo gene stable ID' (default: False), 'ppaniscus_homolog_associated_gene_name': 'Bonobo gene name' (default: False), 'ppaniscus_homolog_ensembl_peptide': 'Bonobo protein or transcript stable ID' (default: False), 'ppaniscus_homolog_chromosome': 'Bonobo chromosome/scaffold name' (default: False), 'ppaniscus_homolog_chrom_start': 'Bonobo chromosome/scaffold start (bp)' (default: False), 'ppaniscus_homolog_chrom_end': 'Bonobo chromosome/scaffold end (bp)' (default: False), 'ppaniscus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'ppaniscus_homolog_subtype': 'Last common ancestor with Bonobo' (default: False), 'ppaniscus_homolog_orthology_type': 'Bonobo homology type' (default: False), 'ppaniscus_homolog_perc_id': '%id. target Bonobo gene identical to query gene' (default: False), 'ppaniscus_homolog_perc_id_r1': '%id. query gene identical to target Bonobo gene' (default: False), 'ppaniscus_homolog_goc_score': 'Bonobo Gene-order conservation score' (default: False), 'ppaniscus_homolog_wga_coverage': 'Bonobo Whole-genome alignment coverage' (default: False), 'ppaniscus_homolog_orthology_confidence': 'Bonobo orthology confidence [0 low, 1 high]' (default: False), 'caperea_homolog_ensembl_gene': 'Brazilian guinea pig gene stable ID' (default: False), 'caperea_homolog_associated_gene_name': 'Brazilian guinea pig gene name' (default: False), 'caperea_homolog_ensembl_peptide': 'Brazilian guinea pig protein or transcript stable ID' (default: False), 'caperea_homolog_chromosome': 'Brazilian guinea pig chromosome/scaffold name' (default: False), 'caperea_homolog_chrom_start': 'Brazilian guinea pig chromosome/scaffold start (bp)' (default: False), 'caperea_homolog_chrom_end': 'Brazilian guinea pig chromosome/scaffold end (bp)' (default: False), 'caperea_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'caperea_homolog_subtype': 'Last common ancestor with Brazilian guinea pig' (default: False), 'caperea_homolog_orthology_type': 'Brazilian guinea pig homology type' (default: False), 'caperea_homolog_perc_id': '%id. target Brazilian guinea pig gene identical to query gene' (default: False), 'caperea_homolog_perc_id_r1': '%id. query gene identical to target Brazilian guinea pig gene' (default: False), 'caperea_homolog_goc_score': 'Brazilian guinea pig Gene-order conservation score' (default: False), 'caperea_homolog_wga_coverage': 'Brazilian guinea pig Whole-genome alignment coverage' (default: False), 'caperea_homolog_orthology_confidence': 'Brazilian guinea pig orthology confidence [0 low, 1 high]' (default: False), 'strutta_homolog_ensembl_gene': 'Brown trout gene stable ID' (default: False), 'strutta_homolog_associated_gene_name': 'Brown trout gene name' (default: False), 'strutta_homolog_ensembl_peptide': 'Brown trout protein or transcript stable ID' (default: False), 'strutta_homolog_chromosome': 'Brown trout chromosome/scaffold name' (default: False), 'strutta_homolog_chrom_start': 'Brown trout chromosome/scaffold start (bp)' (default: False), 'strutta_homolog_chrom_end': 'Brown trout chromosome/scaffold end (bp)' (default: False), 'strutta_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'strutta_homolog_subtype': 'Last common ancestor with Brown trout' (default: False), 'strutta_homolog_orthology_type': 'Brown trout homology type' (default: False), 'strutta_homolog_perc_id': '%id. target Brown trout gene identical to query gene' (default: False), 'strutta_homolog_perc_id_r1': '%id. query gene identical to target Brown trout gene' (default: False), 'strutta_homolog_goc_score': 'Brown trout Gene-order conservation score' (default: False), 'strutta_homolog_wga_coverage': 'Brown trout Whole-genome alignment coverage' (default: False), 'strutta_homolog_orthology_confidence': 'Brown trout orthology confidence [0 low, 1 high]' (default: False), 'mundulatus_homolog_ensembl_gene': 'Budgerigar gene stable ID' (default: False), 'mundulatus_homolog_associated_gene_name': 'Budgerigar gene name' (default: False), 'mundulatus_homolog_ensembl_peptide': 'Budgerigar protein or transcript stable ID' (default: False), 'mundulatus_homolog_chromosome': 'Budgerigar chromosome/scaffold name' (default: False), 'mundulatus_homolog_chrom_start': 'Budgerigar chromosome/scaffold start (bp)' (default: False), 'mundulatus_homolog_chrom_end': 'Budgerigar chromosome/scaffold end (bp)' (default: False), 'mundulatus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'mundulatus_homolog_subtype': 'Last common ancestor with Budgerigar' (default: False), 'mundulatus_homolog_orthology_type': 'Budgerigar homology type' (default: False), 'mundulatus_homolog_perc_id': '%id. target Budgerigar gene identical to query gene' (default: False), 'mundulatus_homolog_perc_id_r1': '%id. query gene identical to target Budgerigar gene' (default: False), 'mundulatus_homolog_goc_score': 'Budgerigar Gene-order conservation score' (default: False), 'mundulatus_homolog_wga_coverage': 'Budgerigar Whole-genome alignment coverage' (default: False), 'mundulatus_homolog_orthology_confidence': 'Budgerigar orthology confidence [0 low, 1 high]' (default: False), 'hburtoni_homolog_ensembl_gene': 'Burton's mouthbrooder gene stable ID' (default: False), 'hburtoni_homolog_associated_gene_name': 'Burton's mouthbrooder gene name' (default: False), 'hburtoni_homolog_ensembl_peptide': 'Burton's mouthbrooder protein or transcript stable ID' (default: False), 'hburtoni_homolog_chromosome': 'Burton's mouthbrooder chromosome/scaffold name' (default: False), 'hburtoni_homolog_chrom_start': 'Burton's mouthbrooder chromosome/scaffold start (bp)' (default: False), 'hburtoni_homolog_chrom_end': 'Burton's mouthbrooder chromosome/scaffold end (bp)' (default: False), 'hburtoni_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'hburtoni_homolog_subtype': 'Last common ancestor with Burton's mouthbrooder' (default: False), 'hburtoni_homolog_orthology_type': 'Burton's mouthbrooder homology type' (default: False), 'hburtoni_homolog_perc_id': '%id. target Burton's mouthbrooder gene identical to query gene' (default: False), 'hburtoni_homolog_perc_id_r1': '%id. query gene identical to target Burton's mouthbrooder gene' (default: False), 'hburtoni_homolog_goc_score': 'Burton's mouthbrooder Gene-order conservation score' (default: False), 'hburtoni_homolog_wga_coverage': 'Burton's mouthbrooder Whole-genome alignment coverage' (default: False), 'hburtoni_homolog_orthology_confidence': 'Burton's mouthbrooder orthology confidence [0 low, 1 high]' (default: False), 'ogarnettii_homolog_ensembl_gene': 'Bushbaby gene stable ID' (default: False), 'ogarnettii_homolog_associated_gene_name': 'Bushbaby gene name' (default: False), 'ogarnettii_homolog_ensembl_peptide': 'Bushbaby protein or transcript stable ID' (default: False), 'ogarnettii_homolog_chromosome': 'Bushbaby chromosome/scaffold name' (default: False), 'ogarnettii_homolog_chrom_start': 'Bushbaby chromosome/scaffold start (bp)' (default: False), 'ogarnettii_homolog_chrom_end': 'Bushbaby chromosome/scaffold end (bp)' (default: False), 'ogarnettii_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'ogarnettii_homolog_subtype': 'Last common ancestor with Bushbaby' (default: False), 'ogarnettii_homolog_orthology_type': 'Bushbaby homology type' (default: False), 'ogarnettii_homolog_perc_id': '%id. target Bushbaby gene identical to query gene' (default: False), 'ogarnettii_homolog_perc_id_r1': '%id. query gene identical to target Bushbaby gene' (default: False), 'ogarnettii_homolog_goc_score': 'Bushbaby Gene-order conservation score' (default: False), 'ogarnettii_homolog_wga_coverage': 'Bushbaby Whole-genome alignment coverage' (default: False), 'ogarnettii_homolog_orthology_confidence': 'Bushbaby orthology confidence [0 low, 1 high]' (default: False), 'cintestinalis_homolog_ensembl_gene': 'C.intestinalis gene stable ID' (default: False), 'cintestinalis_homolog_associated_gene_name': 'C.intestinalis gene name' (default: False), 'cintestinalis_homolog_ensembl_peptide': 'C.intestinalis protein or transcript stable ID' (default: False), 'cintestinalis_homolog_chromosome': 'C.intestinalis chromosome/scaffold name' (default: False), 'cintestinalis_homolog_chrom_start': 'C.intestinalis chromosome/scaffold start (bp)' (default: False), 'cintestinalis_homolog_chrom_end': 'C.intestinalis chromosome/scaffold end (bp)' (default: False), 'cintestinalis_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'cintestinalis_homolog_subtype': 'Last common ancestor with C.intestinalis' (default: False), 'cintestinalis_homolog_orthology_type': 'C.intestinalis homology type' (default: False), 'cintestinalis_homolog_perc_id': '%id. target C.intestinalis gene identical to query gene' (default: False), 'cintestinalis_homolog_perc_id_r1': '%id. query gene identical to target C.intestinalis gene' (default: False), 'cintestinalis_homolog_wga_coverage': 'C.intestinalis Whole-genome alignment coverage' (default: False), 'cintestinalis_homolog_orthology_confidence': 'C.intestinalis orthology confidence [0 low, 1 high]' (default: False), 'csavignyi_homolog_ensembl_gene': 'C.savignyi gene stable ID' (default: False), 'csavignyi_homolog_associated_gene_name': 'C.savignyi gene name' (default: False), 'csavignyi_homolog_ensembl_peptide': 'C.savignyi protein or transcript stable ID' (default: False), 'csavignyi_homolog_chromosome': 'C.savignyi chromosome/scaffold name' (default: False), 'csavignyi_homolog_chrom_start': 'C.savignyi chromosome/scaffold start (bp)' (default: False), 'csavignyi_homolog_chrom_end': 'C.savignyi chromosome/scaffold end (bp)' (default: False), 'csavignyi_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'csavignyi_homolog_subtype': 'Last common ancestor with C.savignyi' (default: False), 'csavignyi_homolog_orthology_type': 'C.savignyi homology type' (default: False), 'csavignyi_homolog_perc_id': '%id. target C.savignyi gene identical to query gene' (default: False), 'csavignyi_homolog_perc_id_r1': '%id. query gene identical to target C.savignyi gene' (default: False), 'csavignyi_homolog_wga_coverage': 'C.savignyi Whole-genome alignment coverage' (default: False), 'csavignyi_homolog_orthology_confidence': 'C.savignyi orthology confidence [0 low, 1 high]' (default: False), 'celegans_homolog_ensembl_gene': 'Caenorhabditis elegans gene stable ID' (default: False), 'celegans_homolog_associated_gene_name': 'Caenorhabditis elegans gene name' (default: False), 'celegans_homolog_ensembl_peptide': 'Caenorhabditis elegans protein or transcript stable ID' (default: False), 'celegans_homolog_chromosome': 'Caenorhabditis elegans chromosome/scaffold name' (default: False), 'celegans_homolog_chrom_start': 'Caenorhabditis elegans chromosome/scaffold start (bp)' (default: False), 'celegans_homolog_chrom_end': 'Caenorhabditis elegans chromosome/scaffold end (bp)' (default: False), 'celegans_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'celegans_homolog_subtype': 'Last common ancestor with Caenorhabditis elegans' (default: False), 'celegans_homolog_orthology_type': 'Caenorhabditis elegans homology type' (default: False), 'celegans_homolog_perc_id': '%id. target Caenorhabditis elegans gene identical to query gene' (default: False), 'celegans_homolog_perc_id_r1': '%id. query gene identical to target Caenorhabditis elegans gene' (default: False), 'celegans_homolog_orthology_confidence': 'Caenorhabditis elegans orthology confidence [0 low, 1 high]' (default: False), 'lcanadensis_homolog_ensembl_gene': 'Canada lynx gene stable ID' (default: False), 'lcanadensis_homolog_associated_gene_name': 'Canada lynx gene name' (default: False), 'lcanadensis_homolog_ensembl_peptide': 'Canada lynx protein or transcript stable ID' (default: False), 'lcanadensis_homolog_chromosome': 'Canada lynx chromosome/scaffold name' (default: False), 'lcanadensis_homolog_chrom_start': 'Canada lynx chromosome/scaffold start (bp)' (default: False), 'lcanadensis_homolog_chrom_end': 'Canada lynx chromosome/scaffold end (bp)' (default: False), 'lcanadensis_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'lcanadensis_homolog_subtype': 'Last common ancestor with Canada lynx' (default: False), 'lcanadensis_homolog_orthology_type': 'Canada lynx homology type' (default: False), 'lcanadensis_homolog_perc_id': '%id. target Canada lynx gene identical to query gene' (default: False), 'lcanadensis_homolog_perc_id_r1': '%id. query gene identical to target Canada lynx gene' (default: False), 'lcanadensis_homolog_goc_score': 'Canada lynx Gene-order conservation score' (default: False), 'lcanadensis_homolog_wga_coverage': 'Canada lynx Whole-genome alignment coverage' (default: False), 'lcanadensis_homolog_orthology_confidence': 'Canada lynx orthology confidence [0 low, 1 high]' (default: False), 'ccapucinus_homolog_ensembl_gene': 'Capuchin gene stable ID' (default: False), 'ccapucinus_homolog_associated_gene_name': 'Capuchin gene name' (default: False), 'ccapucinus_homolog_ensembl_peptide': 'Capuchin protein or transcript stable ID' (default: False), 'ccapucinus_homolog_chromosome': 'Capuchin chromosome/scaffold name' (default: False), 'ccapucinus_homolog_chrom_start': 'Capuchin chromosome/scaffold start (bp)' (default: False), 'ccapucinus_homolog_chrom_end': 'Capuchin chromosome/scaffold end (bp)' (default: False), 'ccapucinus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'ccapucinus_homolog_subtype': 'Last common ancestor with Capuchin' (default: False), 'ccapucinus_homolog_orthology_type': 'Capuchin homology type' (default: False), 'ccapucinus_homolog_perc_id': '%id. target Capuchin gene identical to query gene' (default: False), 'ccapucinus_homolog_perc_id_r1': '%id. query gene identical to target Capuchin gene' (default: False), 'ccapucinus_homolog_goc_score': 'Capuchin Gene-order conservation score' (default: False), 'ccapucinus_homolog_wga_coverage': 'Capuchin Whole-genome alignment coverage' (default: False), 'ccapucinus_homolog_orthology_confidence': 'Capuchin orthology confidence [0 low, 1 high]' (default: False), 'fcatus_homolog_ensembl_gene': 'Cat gene stable ID' (default: False), 'fcatus_homolog_associated_gene_name': 'Cat gene name' (default: False), 'fcatus_homolog_ensembl_peptide': 'Cat protein or transcript stable ID' (default: False), 'fcatus_homolog_chromosome': 'Cat chromosome/scaffold name' (default: False), 'fcatus_homolog_chrom_start': 'Cat chromosome/scaffold start (bp)' (default: False), 'fcatus_homolog_chrom_end': 'Cat chromosome/scaffold end (bp)' (default: False), 'fcatus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'fcatus_homolog_subtype': 'Last common ancestor with Cat' (default: False), 'fcatus_homolog_orthology_type': 'Cat homology type' (default: False), 'fcatus_homolog_perc_id': '%id. target Cat gene identical to query gene' (default: False), 'fcatus_homolog_perc_id_r1': '%id. query gene identical to target Cat gene' (default: False), 'fcatus_homolog_goc_score': 'Cat Gene-order conservation score' (default: False), 'fcatus_homolog_wga_coverage': 'Cat Whole-genome alignment coverage' (default: False), 'fcatus_homolog_orthology_confidence': 'Cat orthology confidence [0 low, 1 high]' (default: False), 'pvitticeps_homolog_ensembl_gene': 'Central bearded dragon gene stable ID' (default: False), 'pvitticeps_homolog_associated_gene_name': 'Central bearded dragon gene name' (default: False), 'pvitticeps_homolog_ensembl_peptide': 'Central bearded dragon protein or transcript stable ID' (default: False), 'pvitticeps_homolog_chromosome': 'Central bearded dragon chromosome/scaffold name' (default: False), 'pvitticeps_homolog_chrom_start': 'Central bearded dragon chromosome/scaffold start (bp)' (default: False), 'pvitticeps_homolog_chrom_end': 'Central bearded dragon chromosome/scaffold end (bp)' (default: False), 'pvitticeps_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'pvitticeps_homolog_subtype': 'Last common ancestor with Central bearded dragon' (default: False), 'pvitticeps_homolog_orthology_type': 'Central bearded dragon homology type' (default: False), 'pvitticeps_homolog_perc_id': '%id. target Central bearded dragon gene identical to query gene' (default: False), 'pvitticeps_homolog_perc_id_r1': '%id. query gene identical to target Central bearded dragon gene' (default: False), 'pvitticeps_homolog_goc_score': 'Central bearded dragon Gene-order conservation score' (default: False), 'pvitticeps_homolog_wga_coverage': 'Central bearded dragon Whole-genome alignment coverage' (default: False), 'pvitticeps_homolog_orthology_confidence': 'Central bearded dragon orthology confidence [0 low, 1 high]' (default: False), 'cgobio_homolog_ensembl_gene': 'Channel bull blenny gene stable ID' (default: False), 'cgobio_homolog_associated_gene_name': 'Channel bull blenny gene name' (default: False), 'cgobio_homolog_ensembl_peptide': 'Channel bull blenny protein or transcript stable ID' (default: False), 'cgobio_homolog_chromosome': 'Channel bull blenny chromosome/scaffold name' (default: False), 'cgobio_homolog_chrom_start': 'Channel bull blenny chromosome/scaffold start (bp)' (default: False), 'cgobio_homolog_chrom_end': 'Channel bull blenny chromosome/scaffold end (bp)' (default: False), 'cgobio_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'cgobio_homolog_subtype': 'Last common ancestor with Channel bull blenny' (default: False), 'cgobio_homolog_orthology_type': 'Channel bull blenny homology type' (default: False), 'cgobio_homolog_perc_id': '%id. target Channel bull blenny gene identical to query gene' (default: False), 'cgobio_homolog_perc_id_r1': '%id. query gene identical to target Channel bull blenny gene' (default: False), 'cgobio_homolog_goc_score': 'Channel bull blenny Gene-order conservation score' (default: False), 'cgobio_homolog_wga_coverage': 'Channel bull blenny Whole-genome alignment coverage' (default: False), 'cgobio_homolog_orthology_confidence': 'Channel bull blenny orthology confidence [0 low, 1 high]' (default: False), 'ipunctatus_homolog_ensembl_gene': 'Channel catfish gene stable ID' (default: False), 'ipunctatus_homolog_associated_gene_name': 'Channel catfish gene name' (default: False), 'ipunctatus_homolog_ensembl_peptide': 'Channel catfish protein or transcript stable ID' (default: False), 'ipunctatus_homolog_chromosome': 'Channel catfish chromosome/scaffold name' (default: False), 'ipunctatus_homolog_chrom_start': 'Channel catfish chromosome/scaffold start (bp)' (default: False), 'ipunctatus_homolog_chrom_end': 'Channel catfish chromosome/scaffold end (bp)' (default: False), 'ipunctatus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'ipunctatus_homolog_subtype': 'Last common ancestor with Channel catfish' (default: False), 'ipunctatus_homolog_orthology_type': 'Channel catfish homology type' (default: False), 'ipunctatus_homolog_perc_id': '%id. target Channel catfish gene identical to query gene' (default: False), 'ipunctatus_homolog_perc_id_r1': '%id. query gene identical to target Channel catfish gene' (default: False), 'ipunctatus_homolog_goc_score': 'Channel catfish Gene-order conservation score' (default: False), 'ipunctatus_homolog_wga_coverage': 'Channel catfish Whole-genome alignment coverage' (default: False), 'ipunctatus_homolog_orthology_confidence': 'Channel catfish orthology confidence [0 low, 1 high]' (default: False), 'ggallus_homolog_ensembl_gene': 'Chicken gene stable ID' (default: False), 'ggallus_homolog_associated_gene_name': 'Chicken gene name' (default: False), 'ggallus_homolog_ensembl_peptide': 'Chicken protein or transcript stable ID' (default: False), 'ggallus_homolog_chromosome': 'Chicken chromosome/scaffold name' (default: False), 'ggallus_homolog_chrom_start': 'Chicken chromosome/scaffold start (bp)' (default: False), 'ggallus_homolog_chrom_end': 'Chicken chromosome/scaffold end (bp)' (default: False), 'ggallus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'ggallus_homolog_subtype': 'Last common ancestor with Chicken' (default: False), 'ggallus_homolog_orthology_type': 'Chicken homology type' (default: False), 'ggallus_homolog_perc_id': '%id. target Chicken gene identical to query gene' (default: False), 'ggallus_homolog_perc_id_r1': '%id. query gene identical to target Chicken gene' (default: False), 'ggallus_homolog_goc_score': 'Chicken Gene-order conservation score' (default: False), 'ggallus_homolog_wga_coverage': 'Chicken Whole-genome alignment coverage' (default: False), 'ggallus_homolog_orthology_confidence': 'Chicken orthology confidence [0 low, 1 high]' (default: False), 'ptroglodytes_homolog_ensembl_gene': 'Chimpanzee gene stable ID' (default: False), 'ptroglodytes_homolog_associated_gene_name': 'Chimpanzee gene name' (default: False), 'ptroglodytes_homolog_ensembl_peptide': 'Chimpanzee protein or transcript stable ID' (default: False), 'ptroglodytes_homolog_chromosome': 'Chimpanzee chromosome/scaffold name' (default: False), 'ptroglodytes_homolog_chrom_start': 'Chimpanzee chromosome/scaffold start (bp)' (default: False), 'ptroglodytes_homolog_chrom_end': 'Chimpanzee chromosome/scaffold end (bp)' (default: False), 'ptroglodytes_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'ptroglodytes_homolog_subtype': 'Last common ancestor with Chimpanzee' (default: False), 'ptroglodytes_homolog_orthology_type': 'Chimpanzee homology type' (default: False), 'ptroglodytes_homolog_perc_id': '%id. target Chimpanzee gene identical to query gene' (default: False), 'ptroglodytes_homolog_perc_id_r1': '%id. query gene identical to target Chimpanzee gene' (default: False), 'ptroglodytes_homolog_goc_score': 'Chimpanzee Gene-order conservation score' (default: False), 'ptroglodytes_homolog_wga_coverage': 'Chimpanzee Whole-genome alignment coverage' (default: False), 'ptroglodytes_homolog_orthology_confidence': 'Chimpanzee orthology confidence [0 low, 1 high]' (default: False), 'cgchok1gshd_homolog_ensembl_gene': 'Chinese hamster CHOK1GS gene stable ID' (default: False), 'cgchok1gshd_homolog_associated_gene_name': 'Chinese hamster CHOK1GS gene name' (default: False), 'cgchok1gshd_homolog_ensembl_peptide': 'Chinese hamster CHOK1GS protein or transcript stable ID' (default: False), 'cgchok1gshd_homolog_chromosome': 'Chinese hamster CHOK1GS chromosome/scaffold name' (default: False), 'cgchok1gshd_homolog_chrom_start': 'Chinese hamster CHOK1GS chromosome/scaffold start (bp)' (default: False), 'cgchok1gshd_homolog_chrom_end': 'Chinese hamster CHOK1GS chromosome/scaffold end (bp)' (default: False), 'cgchok1gshd_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'cgchok1gshd_homolog_subtype': 'Last common ancestor with Chinese hamster CHOK1GS' (default: False), 'cgchok1gshd_homolog_orthology_type': 'Chinese hamster CHOK1GS homology type' (default: False), 'cgchok1gshd_homolog_perc_id': '%id. target Chinese hamster CHOK1GS gene identical to query gene' (default: False), 'cgchok1gshd_homolog_perc_id_r1': '%id. query gene identical to target Chinese hamster CHOK1GS gene' (default: False), 'cgchok1gshd_homolog_goc_score': 'Chinese hamster CHOK1GS Gene-order conservation score' (default: False), 'cgchok1gshd_homolog_wga_coverage': 'Chinese hamster CHOK1GS Whole-genome alignment coverage' (default: False), 'cgchok1gshd_homolog_orthology_confidence': 'Chinese hamster CHOK1GS orthology confidence [0 low, 1 high]' (default: False), 'cgcrigri_homolog_ensembl_gene': 'Chinese hamster CriGri gene stable ID' (default: False), 'cgcrigri_homolog_associated_gene_name': 'Chinese hamster CriGri gene name' (default: False), 'cgcrigri_homolog_ensembl_peptide': 'Chinese hamster CriGri protein or transcript stable ID' (default: False), 'cgcrigri_homolog_chromosome': 'Chinese hamster CriGri chromosome/scaffold name' (default: False), 'cgcrigri_homolog_chrom_start': 'Chinese hamster CriGri chromosome/scaffold start (bp)' (default: False), 'cgcrigri_homolog_chrom_end': 'Chinese hamster CriGri chromosome/scaffold end (bp)' (default: False), 'cgcrigri_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'cgcrigri_homolog_subtype': 'Last common ancestor with Chinese hamster CriGri' (default: False), 'cgcrigri_homolog_orthology_type': 'Chinese hamster CriGri homology type' (default: False), 'cgcrigri_homolog_perc_id': '%id. target Chinese hamster CriGri gene identical to query gene' (default: False), 'cgcrigri_homolog_perc_id_r1': '%id. query gene identical to target Chinese hamster CriGri gene' (default: False), 'cgcrigri_homolog_goc_score': 'Chinese hamster CriGri Gene-order conservation score' (default: False), 'cgcrigri_homolog_wga_coverage': 'Chinese hamster CriGri Whole-genome alignment coverage' (default: False), 'cgcrigri_homolog_orthology_confidence': 'Chinese hamster CriGri orthology confidence [0 low, 1 high]' (default: False), 'cgpicr_homolog_ensembl_gene': 'Chinese hamster PICR gene stable ID' (default: False), 'cgpicr_homolog_associated_gene_name': 'Chinese hamster PICR gene name' (default: False), 'cgpicr_homolog_ensembl_peptide': 'Chinese hamster PICR protein or transcript stable ID' (default: False), 'cgpicr_homolog_chromosome': 'Chinese hamster PICR chromosome/scaffold name' (default: False), 'cgpicr_homolog_chrom_start': 'Chinese hamster PICR chromosome/scaffold start (bp)' (default: False), 'cgpicr_homolog_chrom_end': 'Chinese hamster PICR chromosome/scaffold end (bp)' (default: False), 'cgpicr_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'cgpicr_homolog_subtype': 'Last common ancestor with Chinese hamster PICR' (default: False), 'cgpicr_homolog_orthology_type': 'Chinese hamster PICR homology type' (default: False), 'cgpicr_homolog_perc_id': '%id. target Chinese hamster PICR gene identical to query gene' (default: False), 'cgpicr_homolog_perc_id_r1': '%id. query gene identical to target Chinese hamster PICR gene' (default: False), 'cgpicr_homolog_goc_score': 'Chinese hamster PICR Gene-order conservation score' (default: False), 'cgpicr_homolog_wga_coverage': 'Chinese hamster PICR Whole-genome alignment coverage' (default: False), 'cgpicr_homolog_orthology_confidence': 'Chinese hamster PICR orthology confidence [0 low, 1 high]' (default: False), 'psinensis_homolog_ensembl_gene': 'Chinese softshell turtle gene stable ID' (default: False), 'psinensis_homolog_associated_gene_name': 'Chinese softshell turtle gene name' (default: False), 'psinensis_homolog_ensembl_peptide': 'Chinese softshell turtle protein or transcript stable ID' (default: False), 'psinensis_homolog_chromosome': 'Chinese softshell turtle chromosome/scaffold name' (default: False), 'psinensis_homolog_chrom_start': 'Chinese softshell turtle chromosome/scaffold start (bp)' (default: False), 'psinensis_homolog_chrom_end': 'Chinese softshell turtle chromosome/scaffold end (bp)' (default: False), 'psinensis_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'psinensis_homolog_subtype': 'Last common ancestor with Chinese softshell turtle' (default: False), 'psinensis_homolog_orthology_type': 'Chinese softshell turtle homology type' (default: False), 'psinensis_homolog_perc_id': '%id. target Chinese softshell turtle gene identical to query gene' (default: False), 'psinensis_homolog_perc_id_r1': '%id. query gene identical to target Chinese softshell turtle gene' (default: False), 'psinensis_homolog_goc_score': 'Chinese softshell turtle Gene-order conservation score' (default: False), 'psinensis_homolog_wga_coverage': 'Chinese softshell turtle Whole-genome alignment coverage' (default: False), 'psinensis_homolog_orthology_confidence': 'Chinese softshell turtle orthology confidence [0 low, 1 high]' (default: False), 'atestudineus_homolog_ensembl_gene': 'Climbing perch gene stable ID' (default: False), 'atestudineus_homolog_associated_gene_name': 'Climbing perch gene name' (default: False), 'atestudineus_homolog_ensembl_peptide': 'Climbing perch protein or transcript stable ID' (default: False), 'atestudineus_homolog_chromosome': 'Climbing perch chromosome/scaffold name' (default: False), 'atestudineus_homolog_chrom_start': 'Climbing perch chromosome/scaffold start (bp)' (default: False), 'atestudineus_homolog_chrom_end': 'Climbing perch chromosome/scaffold end (bp)' (default: False), 'atestudineus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'atestudineus_homolog_subtype': 'Last common ancestor with Climbing perch' (default: False), 'atestudineus_homolog_orthology_type': 'Climbing perch homology type' (default: False), 'atestudineus_homolog_perc_id': '%id. target Climbing perch gene identical to query gene' (default: False), 'atestudineus_homolog_perc_id_r1': '%id. query gene identical to target Climbing perch gene' (default: False), 'atestudineus_homolog_goc_score': 'Climbing perch Gene-order conservation score' (default: False), 'atestudineus_homolog_wga_coverage': 'Climbing perch Whole-genome alignment coverage' (default: False), 'atestudineus_homolog_orthology_confidence': 'Climbing perch orthology confidence [0 low, 1 high]' (default: False), 'gmorhua_homolog_ensembl_gene': 'Cod gene stable ID' (default: False), 'gmorhua_homolog_associated_gene_name': 'Cod gene name' (default: False), 'gmorhua_homolog_ensembl_peptide': 'Cod protein or transcript stable ID' (default: False), 'gmorhua_homolog_chromosome': 'Cod chromosome/scaffold name' (default: False), 'gmorhua_homolog_chrom_start': 'Cod chromosome/scaffold start (bp)' (default: False), 'gmorhua_homolog_chrom_end': 'Cod chromosome/scaffold end (bp)' (default: False), 'gmorhua_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'gmorhua_homolog_subtype': 'Last common ancestor with Cod' (default: False), 'gmorhua_homolog_orthology_type': 'Cod homology type' (default: False), 'gmorhua_homolog_perc_id': '%id. target Cod gene identical to query gene' (default: False), 'gmorhua_homolog_perc_id_r1': '%id. query gene identical to target Cod gene' (default: False), 'gmorhua_homolog_goc_score': 'Cod Gene-order conservation score' (default: False), 'gmorhua_homolog_wga_coverage': 'Cod Whole-genome alignment coverage' (default: False), 'gmorhua_homolog_orthology_confidence': 'Cod orthology confidence [0 low, 1 high]' (default: False), 'lchalumnae_homolog_ensembl_gene': 'Coelacanth gene stable ID' (default: False), 'lchalumnae_homolog_associated_gene_name': 'Coelacanth gene name' (default: False), 'lchalumnae_homolog_ensembl_peptide': 'Coelacanth protein or transcript stable ID' (default: False), 'lchalumnae_homolog_chromosome': 'Coelacanth chromosome/scaffold name' (default: False), 'lchalumnae_homolog_chrom_start': 'Coelacanth chromosome/scaffold start (bp)' (default: False), 'lchalumnae_homolog_chrom_end': 'Coelacanth chromosome/scaffold end (bp)' (default: False), 'lchalumnae_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'lchalumnae_homolog_subtype': 'Last common ancestor with Coelacanth' (default: False), 'lchalumnae_homolog_orthology_type': 'Coelacanth homology type' (default: False), 'lchalumnae_homolog_perc_id': '%id. target Coelacanth gene identical to query gene' (default: False), 'lchalumnae_homolog_perc_id_r1': '%id. query gene identical to target Coelacanth gene' (default: False), 'lchalumnae_homolog_goc_score': 'Coelacanth Gene-order conservation score' (default: False), 'lchalumnae_homolog_wga_coverage': 'Coelacanth Whole-genome alignment coverage' (default: False), 'lchalumnae_homolog_orthology_confidence': 'Coelacanth orthology confidence [0 low, 1 high]' (default: False), 'scanaria_homolog_ensembl_gene': 'Common canary gene stable ID' (default: False), 'scanaria_homolog_associated_gene_name': 'Common canary gene name' (default: False), 'scanaria_homolog_ensembl_peptide': 'Common canary protein or transcript stable ID' (default: False), 'scanaria_homolog_chromosome': 'Common canary chromosome/scaffold name' (default: False), 'scanaria_homolog_chrom_start': 'Common canary chromosome/scaffold start (bp)' (default: False), 'scanaria_homolog_chrom_end': 'Common canary chromosome/scaffold end (bp)' (default: False), 'scanaria_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'scanaria_homolog_subtype': 'Last common ancestor with Common canary' (default: False), 'scanaria_homolog_orthology_type': 'Common canary homology type' (default: False), 'scanaria_homolog_perc_id': '%id. target Common canary gene identical to query gene' (default: False), 'scanaria_homolog_perc_id_r1': '%id. query gene identical to target Common canary gene' (default: False), 'scanaria_homolog_goc_score': 'Common canary Gene-order conservation score' (default: False), 'scanaria_homolog_wga_coverage': 'Common canary Whole-genome alignment coverage' (default: False), 'scanaria_homolog_orthology_confidence': 'Common canary orthology confidence [0 low, 1 high]' (default: False), 'ccarpio_homolog_ensembl_gene': 'Common carp gene stable ID' (default: False), 'ccarpio_homolog_associated_gene_name': 'Common carp gene name' (default: False), 'ccarpio_homolog_ensembl_peptide': 'Common carp protein or transcript stable ID' (default: False), 'ccarpio_homolog_chromosome': 'Common carp chromosome/scaffold name' (default: False), 'ccarpio_homolog_chrom_start': 'Common carp chromosome/scaffold start (bp)' (default: False), 'ccarpio_homolog_chrom_end': 'Common carp chromosome/scaffold end (bp)' (default: False), 'ccarpio_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'ccarpio_homolog_subtype': 'Last common ancestor with Common carp' (default: False), 'ccarpio_homolog_orthology_type': 'Common carp homology type' (default: False), 'ccarpio_homolog_perc_id': '%id. target Common carp gene identical to query gene' (default: False), 'ccarpio_homolog_perc_id_r1': '%id. query gene identical to target Common carp gene' (default: False), 'ccarpio_homolog_goc_score': 'Common carp Gene-order conservation score' (default: False), 'ccarpio_homolog_wga_coverage': 'Common carp Whole-genome alignment coverage' (default: False), 'ccarpio_homolog_orthology_confidence': 'Common carp orthology confidence [0 low, 1 high]' (default: False), 'vursinus_homolog_ensembl_gene': 'Common wombat gene stable ID' (default: False), 'vursinus_homolog_associated_gene_name': 'Common wombat gene name' (default: False), 'vursinus_homolog_ensembl_peptide': 'Common wombat protein or transcript stable ID' (default: False), 'vursinus_homolog_chromosome': 'Common wombat chromosome/scaffold name' (default: False), 'vursinus_homolog_chrom_start': 'Common wombat chromosome/scaffold start (bp)' (default: False), 'vursinus_homolog_chrom_end': 'Common wombat chromosome/scaffold end (bp)' (default: False), 'vursinus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'vursinus_homolog_subtype': 'Last common ancestor with Common wombat' (default: False), 'vursinus_homolog_orthology_type': 'Common wombat homology type' (default: False), 'vursinus_homolog_perc_id': '%id. target Common wombat gene identical to query gene' (default: False), 'vursinus_homolog_perc_id_r1': '%id. query gene identical to target Common wombat gene' (default: False), 'vursinus_homolog_goc_score': 'Common wombat Gene-order conservation score' (default: False), 'vursinus_homolog_wga_coverage': 'Common wombat Whole-genome alignment coverage' (default: False), 'vursinus_homolog_orthology_confidence': 'Common wombat orthology confidence [0 low, 1 high]' (default: False), 'pcoquereli_homolog_ensembl_gene': 'Coquerel's sifaka gene stable ID' (default: False), 'pcoquereli_homolog_associated_gene_name': 'Coquerel's sifaka gene name' (default: False), 'pcoquereli_homolog_ensembl_peptide': 'Coquerel's sifaka protein or transcript stable ID' (default: False), 'pcoquereli_homolog_chromosome': 'Coquerel's sifaka chromosome/scaffold name' (default: False), 'pcoquereli_homolog_chrom_start': 'Coquerel's sifaka chromosome/scaffold start (bp)' (default: False), 'pcoquereli_homolog_chrom_end': 'Coquerel's sifaka chromosome/scaffold end (bp)' (default: False), 'pcoquereli_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'pcoquereli_homolog_subtype': 'Last common ancestor with Coquerel's sifaka' (default: False), 'pcoquereli_homolog_orthology_type': 'Coquerel's sifaka homology type' (default: False), 'pcoquereli_homolog_perc_id': '%id. target Coquerel's sifaka gene identical to query gene' (default: False), 'pcoquereli_homolog_perc_id_r1': '%id. query gene identical to target Coquerel's sifaka gene' (default: False), 'pcoquereli_homolog_goc_score': 'Coquerel's sifaka Gene-order conservation score' (default: False), 'pcoquereli_homolog_wga_coverage': 'Coquerel's sifaka Whole-genome alignment coverage' (default: False), 'pcoquereli_homolog_orthology_confidence': 'Coquerel's sifaka orthology confidence [0 low, 1 high]' (default: False), 'btaurus_homolog_ensembl_gene': 'Cow gene stable ID' (default: False), 'btaurus_homolog_associated_gene_name': 'Cow gene name' (default: False), 'btaurus_homolog_ensembl_peptide': 'Cow protein or transcript stable ID' (default: False), 'btaurus_homolog_chromosome': 'Cow chromosome/scaffold name' (default: False), 'btaurus_homolog_chrom_start': 'Cow chromosome/scaffold start (bp)' (default: False), 'btaurus_homolog_chrom_end': 'Cow chromosome/scaffold end (bp)' (default: False), 'btaurus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'btaurus_homolog_subtype': 'Last common ancestor with Cow' (default: False), 'btaurus_homolog_orthology_type': 'Cow homology type' (default: False), 'btaurus_homolog_perc_id': '%id. target Cow gene identical to query gene' (default: False), 'btaurus_homolog_perc_id_r1': '%id. query gene identical to target Cow gene' (default: False), 'btaurus_homolog_goc_score': 'Cow Gene-order conservation score' (default: False), 'btaurus_homolog_wga_coverage': 'Cow Whole-genome alignment coverage' (default: False), 'btaurus_homolog_orthology_confidence': 'Cow orthology confidence [0 low, 1 high]' (default: False), 'mfascicularis_homolog_ensembl_gene': 'Crab-eating macaque gene stable ID' (default: False), 'mfascicularis_homolog_associated_gene_name': 'Crab-eating macaque gene name' (default: False), 'mfascicularis_homolog_ensembl_peptide': 'Crab-eating macaque protein or transcript stable ID' (default: False), 'mfascicularis_homolog_chromosome': 'Crab-eating macaque chromosome/scaffold name' (default: False), 'mfascicularis_homolog_chrom_start': 'Crab-eating macaque chromosome/scaffold start (bp)' (default: False), 'mfascicularis_homolog_chrom_end': 'Crab-eating macaque chromosome/scaffold end (bp)' (default: False), 'mfascicularis_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'mfascicularis_homolog_subtype': 'Last common ancestor with Crab-eating macaque' (default: False), 'mfascicularis_homolog_orthology_type': 'Crab-eating macaque homology type' (default: False), 'mfascicularis_homolog_perc_id': '%id. target Crab-eating macaque gene identical to query gene' (default: False), 'mfascicularis_homolog_perc_id_r1': '%id. query gene identical to target Crab-eating macaque gene' (default: False), 'mfascicularis_homolog_goc_score': 'Crab-eating macaque Gene-order conservation score' (default: False), 'mfascicularis_homolog_wga_coverage': 'Crab-eating macaque Whole-genome alignment coverage' (default: False), 'mfascicularis_homolog_orthology_confidence': 'Crab-eating macaque orthology confidence [0 low, 1 high]' (default: False), 'fdamarensis_homolog_ensembl_gene': 'Damara mole rat gene stable ID' (default: False), 'fdamarensis_homolog_associated_gene_name': 'Damara mole rat gene name' (default: False), 'fdamarensis_homolog_ensembl_peptide': 'Damara mole rat protein or transcript stable ID' (default: False), 'fdamarensis_homolog_chromosome': 'Damara mole rat chromosome/scaffold name' (default: False), 'fdamarensis_homolog_chrom_start': 'Damara mole rat chromosome/scaffold start (bp)' (default: False), 'fdamarensis_homolog_chrom_end': 'Damara mole rat chromosome/scaffold end (bp)' (default: False), 'fdamarensis_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'fdamarensis_homolog_subtype': 'Last common ancestor with Damara mole rat' (default: False), 'fdamarensis_homolog_orthology_type': 'Damara mole rat homology type' (default: False), 'fdamarensis_homolog_perc_id': '%id. target Damara mole rat gene identical to query gene' (default: False), 'fdamarensis_homolog_perc_id_r1': '%id. query gene identical to target Damara mole rat gene' (default: False), 'fdamarensis_homolog_goc_score': 'Damara mole rat Gene-order conservation score' (default: False), 'fdamarensis_homolog_wga_coverage': 'Damara mole rat Whole-genome alignment coverage' (default: False), 'fdamarensis_homolog_orthology_confidence': 'Damara mole rat orthology confidence [0 low, 1 high]' (default: False), 'jhyemalis_homolog_ensembl_gene': 'Dark-eyed junco gene stable ID' (default: False), 'jhyemalis_homolog_associated_gene_name': 'Dark-eyed junco gene name' (default: False), 'jhyemalis_homolog_ensembl_peptide': 'Dark-eyed junco protein or transcript stable ID' (default: False), 'jhyemalis_homolog_chromosome': 'Dark-eyed junco chromosome/scaffold name' (default: False), 'jhyemalis_homolog_chrom_start': 'Dark-eyed junco chromosome/scaffold start (bp)' (default: False), 'jhyemalis_homolog_chrom_end': 'Dark-eyed junco chromosome/scaffold end (bp)' (default: False), 'jhyemalis_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'jhyemalis_homolog_subtype': 'Last common ancestor with Dark-eyed junco' (default: False), 'jhyemalis_homolog_orthology_type': 'Dark-eyed junco homology type' (default: False), 'jhyemalis_homolog_perc_id': '%id. target Dark-eyed junco gene identical to query gene' (default: False), 'jhyemalis_homolog_perc_id_r1': '%id. query gene identical to target Dark-eyed junco gene' (default: False), 'jhyemalis_homolog_goc_score': 'Dark-eyed junco Gene-order conservation score' (default: False), 'jhyemalis_homolog_wga_coverage': 'Dark-eyed junco Whole-genome alignment coverage' (default: False), 'jhyemalis_homolog_orthology_confidence': 'Dark-eyed junco orthology confidence [0 low, 1 high]' (default: False), 'odegus_homolog_ensembl_gene': 'Degu gene stable ID' (default: False), 'odegus_homolog_associated_gene_name': 'Degu gene name' (default: False), 'odegus_homolog_ensembl_peptide': 'Degu protein or transcript stable ID' (default: False), 'odegus_homolog_chromosome': 'Degu chromosome/scaffold name' (default: False), 'odegus_homolog_chrom_start': 'Degu chromosome/scaffold start (bp)' (default: False), 'odegus_homolog_chrom_end': 'Degu chromosome/scaffold end (bp)' (default: False), 'odegus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'odegus_homolog_subtype': 'Last common ancestor with Degu' (default: False), 'odegus_homolog_orthology_type': 'Degu homology type' (default: False), 'odegus_homolog_perc_id': '%id. target Degu gene identical to query gene' (default: False), 'odegus_homolog_perc_id_r1': '%id. query gene identical to target Degu gene' (default: False), 'odegus_homolog_goc_score': 'Degu Gene-order conservation score' (default: False), 'odegus_homolog_wga_coverage': 'Degu Whole-genome alignment coverage' (default: False), 'odegus_homolog_orthology_confidence': 'Degu orthology confidence [0 low, 1 high]' (default: False), 'dclupeoides_homolog_ensembl_gene': 'Denticle herring gene stable ID' (default: False), 'dclupeoides_homolog_associated_gene_name': 'Denticle herring gene name' (default: False), 'dclupeoides_homolog_ensembl_peptide': 'Denticle herring protein or transcript stable ID' (default: False), 'dclupeoides_homolog_chromosome': 'Denticle herring chromosome/scaffold name' (default: False), 'dclupeoides_homolog_chrom_start': 'Denticle herring chromosome/scaffold start (bp)' (default: False), 'dclupeoides_homolog_chrom_end': 'Denticle herring chromosome/scaffold end (bp)' (default: False), 'dclupeoides_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'dclupeoides_homolog_subtype': 'Last common ancestor with Denticle herring' (default: False), 'dclupeoides_homolog_orthology_type': 'Denticle herring homology type' (default: False), 'dclupeoides_homolog_perc_id': '%id. target Denticle herring gene identical to query gene' (default: False), 'dclupeoides_homolog_perc_id_r1': '%id. query gene identical to target Denticle herring gene' (default: False), 'dclupeoides_homolog_goc_score': 'Denticle herring Gene-order conservation score' (default: False), 'dclupeoides_homolog_wga_coverage': 'Denticle herring Whole-genome alignment coverage' (default: False), 'dclupeoides_homolog_orthology_confidence': 'Denticle herring orthology confidence [0 low, 1 high]' (default: False), 'cldingo_homolog_ensembl_gene': 'Dingo gene stable ID' (default: False), 'cldingo_homolog_associated_gene_name': 'Dingo gene name' (default: False), 'cldingo_homolog_ensembl_peptide': 'Dingo protein or transcript stable ID' (default: False), 'cldingo_homolog_chromosome': 'Dingo chromosome/scaffold name' (default: False), 'cldingo_homolog_chrom_start': 'Dingo chromosome/scaffold start (bp)' (default: False), 'cldingo_homolog_chrom_end': 'Dingo chromosome/scaffold end (bp)' (default: False), 'cldingo_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'cldingo_homolog_subtype': 'Last common ancestor with Dingo' (default: False), 'cldingo_homolog_orthology_type': 'Dingo homology type' (default: False), 'cldingo_homolog_perc_id': '%id. target Dingo gene identical to query gene' (default: False), 'cldingo_homolog_perc_id_r1': '%id. query gene identical to target Dingo gene' (default: False), 'cldingo_homolog_goc_score': 'Dingo Gene-order conservation score' (default: False), 'cldingo_homolog_wga_coverage': 'Dingo Whole-genome alignment coverage' (default: False), 'cldingo_homolog_orthology_confidence': 'Dingo orthology confidence [0 low, 1 high]' (default: False), 'clfamiliaris_homolog_ensembl_gene': 'Dog gene stable ID' (default: False), 'clfamiliaris_homolog_associated_gene_name': 'Dog gene name' (default: False), 'clfamiliaris_homolog_ensembl_peptide': 'Dog protein or transcript stable ID' (default: False), 'clfamiliaris_homolog_chromosome': 'Dog chromosome/scaffold name' (default: False), 'clfamiliaris_homolog_chrom_start': 'Dog chromosome/scaffold start (bp)' (default: False), 'clfamiliaris_homolog_chrom_end': 'Dog chromosome/scaffold end (bp)' (default: False), 'clfamiliaris_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'clfamiliaris_homolog_subtype': 'Last common ancestor with Dog' (default: False), 'clfamiliaris_homolog_orthology_type': 'Dog homology type' (default: False), 'clfamiliaris_homolog_perc_id': '%id. target Dog gene identical to query gene' (default: False), 'clfamiliaris_homolog_perc_id_r1': '%id. query gene identical to target Dog gene' (default: False), 'clfamiliaris_homolog_goc_score': 'Dog Gene-order conservation score' (default: False), 'clfamiliaris_homolog_wga_coverage': 'Dog Whole-genome alignment coverage' (default: False), 'clfamiliaris_homolog_orthology_confidence': 'Dog orthology confidence [0 low, 1 high]' (default: False), 'ttruncatus_homolog_ensembl_gene': 'Dolphin gene stable ID' (default: False), 'ttruncatus_homolog_associated_gene_name': 'Dolphin gene name' (default: False), 'ttruncatus_homolog_ensembl_peptide': 'Dolphin protein or transcript stable ID' (default: False), 'ttruncatus_homolog_chromosome': 'Dolphin chromosome/scaffold name' (default: False), 'ttruncatus_homolog_chrom_start': 'Dolphin chromosome/scaffold start (bp)' (default: False), 'ttruncatus_homolog_chrom_end': 'Dolphin chromosome/scaffold end (bp)' (default: False), 'ttruncatus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'ttruncatus_homolog_subtype': 'Last common ancestor with Dolphin' (default: False), 'ttruncatus_homolog_orthology_type': 'Dolphin homology type' (default: False), 'ttruncatus_homolog_perc_id': '%id. target Dolphin gene identical to query gene' (default: False), 'ttruncatus_homolog_perc_id_r1': '%id. query gene identical to target Dolphin gene' (default: False), 'ttruncatus_homolog_goc_score': 'Dolphin Gene-order conservation score' (default: False), 'ttruncatus_homolog_wga_coverage': 'Dolphin Whole-genome alignment coverage' (default: False), 'ttruncatus_homolog_orthology_confidence': 'Dolphin orthology confidence [0 low, 1 high]' (default: False), 'bgrunniens_homolog_ensembl_gene': 'Domestic yak gene stable ID' (default: False), 'bgrunniens_homolog_associated_gene_name': 'Domestic yak gene name' (default: False), 'bgrunniens_homolog_ensembl_peptide': 'Domestic yak protein or transcript stable ID' (default: False), 'bgrunniens_homolog_chromosome': 'Domestic yak chromosome/scaffold name' (default: False), 'bgrunniens_homolog_chrom_start': 'Domestic yak chromosome/scaffold start (bp)' (default: False), 'bgrunniens_homolog_chrom_end': 'Domestic yak chromosome/scaffold end (bp)' (default: False), 'bgrunniens_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'bgrunniens_homolog_subtype': 'Last common ancestor with Domestic yak' (default: False), 'bgrunniens_homolog_orthology_type': 'Domestic yak homology type' (default: False), 'bgrunniens_homolog_perc_id': '%id. target Domestic yak gene identical to query gene' (default: False), 'bgrunniens_homolog_perc_id_r1': '%id. query gene identical to target Domestic yak gene' (default: False), 'bgrunniens_homolog_goc_score': 'Domestic yak Gene-order conservation score' (default: False), 'bgrunniens_homolog_wga_coverage': 'Domestic yak Whole-genome alignment coverage' (default: False), 'bgrunniens_homolog_orthology_confidence': 'Domestic yak orthology confidence [0 low, 1 high]' (default: False), 'eaasinus_homolog_ensembl_gene': 'Donkey gene stable ID' (default: False), 'eaasinus_homolog_associated_gene_name': 'Donkey gene name' (default: False), 'eaasinus_homolog_ensembl_peptide': 'Donkey protein or transcript stable ID' (default: False), 'eaasinus_homolog_chromosome': 'Donkey chromosome/scaffold name' (default: False), 'eaasinus_homolog_chrom_start': 'Donkey chromosome/scaffold start (bp)' (default: False), 'eaasinus_homolog_chrom_end': 'Donkey chromosome/scaffold end (bp)' (default: False), 'eaasinus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'eaasinus_homolog_subtype': 'Last common ancestor with Donkey' (default: False), 'eaasinus_homolog_orthology_type': 'Donkey homology type' (default: False), 'eaasinus_homolog_perc_id': '%id. target Donkey gene identical to query gene' (default: False), 'eaasinus_homolog_perc_id_r1': '%id. query gene identical to target Donkey gene' (default: False), 'eaasinus_homolog_goc_score': 'Donkey Gene-order conservation score' (default: False), 'eaasinus_homolog_wga_coverage': 'Donkey Whole-genome alignment coverage' (default: False), 'eaasinus_homolog_orthology_confidence': 'Donkey orthology confidence [0 low, 1 high]' (default: False), 'mleucophaeus_homolog_ensembl_gene': 'Drill gene stable ID' (default: False), 'mleucophaeus_homolog_associated_gene_name': 'Drill gene name' (default: False), 'mleucophaeus_homolog_ensembl_peptide': 'Drill protein or transcript stable ID' (default: False), 'mleucophaeus_homolog_chromosome': 'Drill chromosome/scaffold name' (default: False), 'mleucophaeus_homolog_chrom_start': 'Drill chromosome/scaffold start (bp)' (default: False), 'mleucophaeus_homolog_chrom_end': 'Drill chromosome/scaffold end (bp)' (default: False), 'mleucophaeus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'mleucophaeus_homolog_subtype': 'Last common ancestor with Drill' (default: False), 'mleucophaeus_homolog_orthology_type': 'Drill homology type' (default: False), 'mleucophaeus_homolog_perc_id': '%id. target Drill gene identical to query gene' (default: False), 'mleucophaeus_homolog_perc_id_r1': '%id. query gene identical to target Drill gene' (default: False), 'mleucophaeus_homolog_goc_score': 'Drill Gene-order conservation score' (default: False), 'mleucophaeus_homolog_wga_coverage': 'Drill Whole-genome alignment coverage' (default: False), 'mleucophaeus_homolog_orthology_confidence': 'Drill orthology confidence [0 low, 1 high]' (default: False), 'dmelanogaster_homolog_ensembl_gene': 'Drosophila melanogaster gene stable ID' (default: False), 'dmelanogaster_homolog_associated_gene_name': 'Drosophila melanogaster gene name' (default: False), 'dmelanogaster_homolog_ensembl_peptide': 'Drosophila melanogaster protein or transcript stable ID' (default: False), 'dmelanogaster_homolog_chromosome': 'Drosophila melanogaster chromosome/scaffold name' (default: False), 'dmelanogaster_homolog_chrom_start': 'Drosophila melanogaster chromosome/scaffold start (bp)' (default: False), 'dmelanogaster_homolog_chrom_end': 'Drosophila melanogaster chromosome/scaffold end (bp)' (default: False), 'dmelanogaster_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'dmelanogaster_homolog_subtype': 'Last common ancestor with Drosophila melanogaster' (default: False), 'dmelanogaster_homolog_orthology_type': 'Drosophila melanogaster homology type' (default: False), 'dmelanogaster_homolog_perc_id': '%id. target Drosophila melanogaster gene identical to query gene' (default: False), 'dmelanogaster_homolog_perc_id_r1': '%id. query gene identical to target Drosophila melanogaster gene' (default: False), 'dmelanogaster_homolog_orthology_confidence': 'Drosophila melanogaster orthology confidence [0 low, 1 high]' (default: False), 'applatyrhynchos_homolog_ensembl_gene': 'Duck gene stable ID' (default: False), 'applatyrhynchos_homolog_associated_gene_name': 'Duck gene name' (default: False), 'applatyrhynchos_homolog_ensembl_peptide': 'Duck protein or transcript stable ID' (default: False), 'applatyrhynchos_homolog_chromosome': 'Duck chromosome/scaffold name' (default: False), 'applatyrhynchos_homolog_chrom_start': 'Duck chromosome/scaffold start (bp)' (default: False), 'applatyrhynchos_homolog_chrom_end': 'Duck chromosome/scaffold end (bp)' (default: False), 'applatyrhynchos_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'applatyrhynchos_homolog_subtype': 'Last common ancestor with Duck' (default: False), 'applatyrhynchos_homolog_orthology_type': 'Duck homology type' (default: False), 'applatyrhynchos_homolog_perc_id': '%id. target Duck gene identical to query gene' (default: False), 'applatyrhynchos_homolog_perc_id_r1': '%id. query gene identical to target Duck gene' (default: False), 'applatyrhynchos_homolog_goc_score': 'Duck Gene-order conservation score' (default: False), 'applatyrhynchos_homolog_wga_coverage': 'Duck Whole-genome alignment coverage' (default: False), 'applatyrhynchos_homolog_orthology_confidence': 'Duck orthology confidence [0 low, 1 high]' (default: False), 'acalliptera_homolog_ensembl_gene': 'Eastern happy gene stable ID' (default: False), 'acalliptera_homolog_associated_gene_name': 'Eastern happy gene name' (default: False), 'acalliptera_homolog_ensembl_peptide': 'Eastern happy protein or transcript stable ID' (default: False), 'acalliptera_homolog_chromosome': 'Eastern happy chromosome/scaffold name' (default: False), 'acalliptera_homolog_chrom_start': 'Eastern happy chromosome/scaffold start (bp)' (default: False), 'acalliptera_homolog_chrom_end': 'Eastern happy chromosome/scaffold end (bp)' (default: False), 'acalliptera_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'acalliptera_homolog_subtype': 'Last common ancestor with Eastern happy' (default: False), 'acalliptera_homolog_orthology_type': 'Eastern happy homology type' (default: False), 'acalliptera_homolog_perc_id': '%id. target Eastern happy gene identical to query gene' (default: False), 'acalliptera_homolog_perc_id_r1': '%id. query gene identical to target Eastern happy gene' (default: False), 'acalliptera_homolog_goc_score': 'Eastern happy Gene-order conservation score' (default: False), 'acalliptera_homolog_wga_coverage': 'Eastern happy Whole-genome alignment coverage' (default: False), 'acalliptera_homolog_orthology_confidence': 'Eastern happy orthology confidence [0 low, 1 high]' (default: False), 'eelectricus_homolog_ensembl_gene': 'Electric eel gene stable ID' (default: False), 'eelectricus_homolog_associated_gene_name': 'Electric eel gene name' (default: False), 'eelectricus_homolog_ensembl_peptide': 'Electric eel protein or transcript stable ID' (default: False), 'eelectricus_homolog_chromosome': 'Electric eel chromosome/scaffold name' (default: False), 'eelectricus_homolog_chrom_start': 'Electric eel chromosome/scaffold start (bp)' (default: False), 'eelectricus_homolog_chrom_end': 'Electric eel chromosome/scaffold end (bp)' (default: False), 'eelectricus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'eelectricus_homolog_subtype': 'Last common ancestor with Electric eel' (default: False), 'eelectricus_homolog_orthology_type': 'Electric eel homology type' (default: False), 'eelectricus_homolog_perc_id': '%id. target Electric eel gene identical to query gene' (default: False), 'eelectricus_homolog_perc_id_r1': '%id. query gene identical to target Electric eel gene' (default: False), 'eelectricus_homolog_goc_score': 'Electric eel Gene-order conservation score' (default: False), 'eelectricus_homolog_wga_coverage': 'Electric eel Whole-genome alignment coverage' (default: False), 'eelectricus_homolog_orthology_confidence': 'Electric eel orthology confidence [0 low, 1 high]' (default: False), 'lafricana_homolog_ensembl_gene': 'Elephant gene stable ID' (default: False), 'lafricana_homolog_associated_gene_name': 'Elephant gene name' (default: False), 'lafricana_homolog_ensembl_peptide': 'Elephant protein or transcript stable ID' (default: False), 'lafricana_homolog_chromosome': 'Elephant chromosome/scaffold name' (default: False), 'lafricana_homolog_chrom_start': 'Elephant chromosome/scaffold start (bp)' (default: False), 'lafricana_homolog_chrom_end': 'Elephant chromosome/scaffold end (bp)' (default: False), 'lafricana_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'lafricana_homolog_subtype': 'Last common ancestor with Elephant' (default: False), 'lafricana_homolog_orthology_type': 'Elephant homology type' (default: False), 'lafricana_homolog_perc_id': '%id. target Elephant gene identical to query gene' (default: False), 'lafricana_homolog_perc_id_r1': '%id. query gene identical to target Elephant gene' (default: False), 'lafricana_homolog_goc_score': 'Elephant Gene-order conservation score' (default: False), 'lafricana_homolog_wga_coverage': 'Elephant Whole-genome alignment coverage' (default: False), 'lafricana_homolog_orthology_confidence': 'Elephant orthology confidence [0 low, 1 high]' (default: False), 'dnovaehollandiae_homolog_ensembl_gene': 'Emu gene stable ID' (default: False), 'dnovaehollandiae_homolog_associated_gene_name': 'Emu gene name' (default: False), 'dnovaehollandiae_homolog_ensembl_peptide': 'Emu protein or transcript stable ID' (default: False), 'dnovaehollandiae_homolog_chromosome': 'Emu chromosome/scaffold name' (default: False), 'dnovaehollandiae_homolog_chrom_start': 'Emu chromosome/scaffold start (bp)' (default: False), 'dnovaehollandiae_homolog_chrom_end': 'Emu chromosome/scaffold end (bp)' (default: False), 'dnovaehollandiae_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'dnovaehollandiae_homolog_subtype': 'Last common ancestor with Emu' (default: False), 'dnovaehollandiae_homolog_orthology_type': 'Emu homology type' (default: False), 'dnovaehollandiae_homolog_perc_id': '%id. target Emu gene identical to query gene' (default: False), 'dnovaehollandiae_homolog_perc_id_r1': '%id. query gene identical to target Emu gene' (default: False), 'dnovaehollandiae_homolog_goc_score': 'Emu Gene-order conservation score' (default: False), 'dnovaehollandiae_homolog_wga_coverage': 'Emu Whole-genome alignment coverage' (default: False), 'dnovaehollandiae_homolog_orthology_confidence': 'Emu orthology confidence [0 low, 1 high]' (default: False), 'dlabrax_homolog_ensembl_gene': 'European seabass gene stable ID' (default: False), 'dlabrax_homolog_associated_gene_name': 'European seabass gene name' (default: False), 'dlabrax_homolog_ensembl_peptide': 'European seabass protein or transcript stable ID' (default: False), 'dlabrax_homolog_chromosome': 'European seabass chromosome/scaffold name' (default: False), 'dlabrax_homolog_chrom_start': 'European seabass chromosome/scaffold start (bp)' (default: False), 'dlabrax_homolog_chrom_end': 'European seabass chromosome/scaffold end (bp)' (default: False), 'dlabrax_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'dlabrax_homolog_subtype': 'Last common ancestor with European seabass' (default: False), 'dlabrax_homolog_orthology_type': 'European seabass homology type' (default: False), 'dlabrax_homolog_perc_id': '%id. target European seabass gene identical to query gene' (default: False), 'dlabrax_homolog_perc_id_r1': '%id. query gene identical to target European seabass gene' (default: False), 'dlabrax_homolog_goc_score': 'European seabass Gene-order conservation score' (default: False), 'dlabrax_homolog_wga_coverage': 'European seabass Whole-genome alignment coverage' (default: False), 'dlabrax_homolog_orthology_confidence': 'European seabass orthology confidence [0 low, 1 high]' (default: False), 'mpfuro_homolog_ensembl_gene': 'Ferret gene stable ID' (default: False), 'mpfuro_homolog_associated_gene_name': 'Ferret gene name' (default: False), 'mpfuro_homolog_ensembl_peptide': 'Ferret protein or transcript stable ID' (default: False), 'mpfuro_homolog_chromosome': 'Ferret chromosome/scaffold name' (default: False), 'mpfuro_homolog_chrom_start': 'Ferret chromosome/scaffold start (bp)' (default: False), 'mpfuro_homolog_chrom_end': 'Ferret chromosome/scaffold end (bp)' (default: False), 'mpfuro_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'mpfuro_homolog_subtype': 'Last common ancestor with Ferret' (default: False), 'mpfuro_homolog_orthology_type': 'Ferret homology type' (default: False), 'mpfuro_homolog_perc_id': '%id. target Ferret gene identical to query gene' (default: False), 'mpfuro_homolog_perc_id_r1': '%id. query gene identical to target Ferret gene' (default: False), 'mpfuro_homolog_goc_score': 'Ferret Gene-order conservation score' (default: False), 'mpfuro_homolog_wga_coverage': 'Ferret Whole-genome alignment coverage' (default: False), 'mpfuro_homolog_orthology_confidence': 'Ferret orthology confidence [0 low, 1 high]' (default: False), 'falbicollis_homolog_ensembl_gene': 'Flycatcher gene stable ID' (default: False), 'falbicollis_homolog_associated_gene_name': 'Flycatcher gene name' (default: False), 'falbicollis_homolog_ensembl_peptide': 'Flycatcher protein or transcript stable ID' (default: False), 'falbicollis_homolog_chromosome': 'Flycatcher chromosome/scaffold name' (default: False), 'falbicollis_homolog_chrom_start': 'Flycatcher chromosome/scaffold start (bp)' (default: False), 'falbicollis_homolog_chrom_end': 'Flycatcher chromosome/scaffold end (bp)' (default: False), 'falbicollis_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'falbicollis_homolog_subtype': 'Last common ancestor with Flycatcher' (default: False), 'falbicollis_homolog_orthology_type': 'Flycatcher homology type' (default: False), 'falbicollis_homolog_perc_id': '%id. target Flycatcher gene identical to query gene' (default: False), 'falbicollis_homolog_perc_id_r1': '%id. query gene identical to target Flycatcher gene' (default: False), 'falbicollis_homolog_goc_score': 'Flycatcher Gene-order conservation score' (default: False), 'falbicollis_homolog_wga_coverage': 'Flycatcher Whole-genome alignment coverage' (default: False), 'falbicollis_homolog_orthology_confidence': 'Flycatcher orthology confidence [0 low, 1 high]' (default: False), 'trubripes_homolog_ensembl_gene': 'Fugu gene stable ID' (default: False), 'trubripes_homolog_associated_gene_name': 'Fugu gene name' (default: False), 'trubripes_homolog_ensembl_peptide': 'Fugu protein or transcript stable ID' (default: False), 'trubripes_homolog_chromosome': 'Fugu chromosome/scaffold name' (default: False), 'trubripes_homolog_chrom_start': 'Fugu chromosome/scaffold start (bp)' (default: False), 'trubripes_homolog_chrom_end': 'Fugu chromosome/scaffold end (bp)' (default: False), 'trubripes_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'trubripes_homolog_subtype': 'Last common ancestor with Fugu' (default: False), 'trubripes_homolog_orthology_type': 'Fugu homology type' (default: False), 'trubripes_homolog_perc_id': '%id. target Fugu gene identical to query gene' (default: False), 'trubripes_homolog_perc_id_r1': '%id. query gene identical to target Fugu gene' (default: False), 'trubripes_homolog_goc_score': 'Fugu Gene-order conservation score' (default: False), 'trubripes_homolog_wga_coverage': 'Fugu Whole-genome alignment coverage' (default: False), 'trubripes_homolog_orthology_confidence': 'Fugu orthology confidence [0 low, 1 high]' (default: False), 'tgelada_homolog_ensembl_gene': 'Gelada gene stable ID' (default: False), 'tgelada_homolog_associated_gene_name': 'Gelada gene name' (default: False), 'tgelada_homolog_ensembl_peptide': 'Gelada protein or transcript stable ID' (default: False), 'tgelada_homolog_chromosome': 'Gelada chromosome/scaffold name' (default: False), 'tgelada_homolog_chrom_start': 'Gelada chromosome/scaffold start (bp)' (default: False), 'tgelada_homolog_chrom_end': 'Gelada chromosome/scaffold end (bp)' (default: False), 'tgelada_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'tgelada_homolog_subtype': 'Last common ancestor with Gelada' (default: False), 'tgelada_homolog_orthology_type': 'Gelada homology type' (default: False), 'tgelada_homolog_perc_id': '%id. target Gelada gene identical to query gene' (default: False), 'tgelada_homolog_perc_id_r1': '%id. query gene identical to target Gelada gene' (default: False), 'tgelada_homolog_goc_score': 'Gelada Gene-order conservation score' (default: False), 'tgelada_homolog_wga_coverage': 'Gelada Whole-genome alignment coverage' (default: False), 'tgelada_homolog_orthology_confidence': 'Gelada orthology confidence [0 low, 1 high]' (default: False), 'nleucogenys_homolog_ensembl_gene': 'Gibbon gene stable ID' (default: False), 'nleucogenys_homolog_associated_gene_name': 'Gibbon gene name' (default: False), 'nleucogenys_homolog_ensembl_peptide': 'Gibbon protein or transcript stable ID' (default: False), 'nleucogenys_homolog_chromosome': 'Gibbon chromosome/scaffold name' (default: False), 'nleucogenys_homolog_chrom_start': 'Gibbon chromosome/scaffold start (bp)' (default: False), 'nleucogenys_homolog_chrom_end': 'Gibbon chromosome/scaffold end (bp)' (default: False), 'nleucogenys_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'nleucogenys_homolog_subtype': 'Last common ancestor with Gibbon' (default: False), 'nleucogenys_homolog_orthology_type': 'Gibbon homology type' (default: False), 'nleucogenys_homolog_perc_id': '%id. target Gibbon gene identical to query gene' (default: False), 'nleucogenys_homolog_perc_id_r1': '%id. query gene identical to target Gibbon gene' (default: False), 'nleucogenys_homolog_goc_score': 'Gibbon Gene-order conservation score' (default: False), 'nleucogenys_homolog_wga_coverage': 'Gibbon Whole-genome alignment coverage' (default: False), 'nleucogenys_homolog_orthology_confidence': 'Gibbon orthology confidence [0 low, 1 high]' (default: False), 'saurata_homolog_ensembl_gene': 'Gilthead seabream gene stable ID' (default: False), 'saurata_homolog_associated_gene_name': 'Gilthead seabream gene name' (default: False), 'saurata_homolog_ensembl_peptide': 'Gilthead seabream protein or transcript stable ID' (default: False), 'saurata_homolog_chromosome': 'Gilthead seabream chromosome/scaffold name' (default: False), 'saurata_homolog_chrom_start': 'Gilthead seabream chromosome/scaffold start (bp)' (default: False), 'saurata_homolog_chrom_end': 'Gilthead seabream chromosome/scaffold end (bp)' (default: False), 'saurata_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'saurata_homolog_subtype': 'Last common ancestor with Gilthead seabream' (default: False), 'saurata_homolog_orthology_type': 'Gilthead seabream homology type' (default: False), 'saurata_homolog_perc_id': '%id. target Gilthead seabream gene identical to query gene' (default: False), 'saurata_homolog_perc_id_r1': '%id. query gene identical to target Gilthead seabream gene' (default: False), 'saurata_homolog_goc_score': 'Gilthead seabream Gene-order conservation score' (default: False), 'saurata_homolog_wga_coverage': 'Gilthead seabream Whole-genome alignment coverage' (default: False), 'saurata_homolog_orthology_confidence': 'Gilthead seabream orthology confidence [0 low, 1 high]' (default: False), 'chircus_homolog_ensembl_gene': 'Goat gene stable ID' (default: False), 'chircus_homolog_associated_gene_name': 'Goat gene name' (default: False), 'chircus_homolog_ensembl_peptide': 'Goat protein or transcript stable ID' (default: False), 'chircus_homolog_chromosome': 'Goat chromosome/scaffold name' (default: False), 'chircus_homolog_chrom_start': 'Goat chromosome/scaffold start (bp)' (default: False), 'chircus_homolog_chrom_end': 'Goat chromosome/scaffold end (bp)' (default: False), 'chircus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'chircus_homolog_subtype': 'Last common ancestor with Goat' (default: False), 'chircus_homolog_orthology_type': 'Goat homology type' (default: False), 'chircus_homolog_perc_id': '%id. target Goat gene identical to query gene' (default: False), 'chircus_homolog_perc_id_r1': '%id. query gene identical to target Goat gene' (default: False), 'chircus_homolog_goc_score': 'Goat Gene-order conservation score' (default: False), 'chircus_homolog_wga_coverage': 'Goat Whole-genome alignment coverage' (default: False), 'chircus_homolog_orthology_confidence': 'Goat orthology confidence [0 low, 1 high]' (default: False), 'mauratus_homolog_ensembl_gene': 'Golden Hamster gene stable ID' (default: False), 'mauratus_homolog_associated_gene_name': 'Golden Hamster gene name' (default: False), 'mauratus_homolog_ensembl_peptide': 'Golden Hamster protein or transcript stable ID' (default: False), 'mauratus_homolog_chromosome': 'Golden Hamster chromosome/scaffold name' (default: False), 'mauratus_homolog_chrom_start': 'Golden Hamster chromosome/scaffold start (bp)' (default: False), 'mauratus_homolog_chrom_end': 'Golden Hamster chromosome/scaffold end (bp)' (default: False), 'mauratus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'mauratus_homolog_subtype': 'Last common ancestor with Golden Hamster' (default: False), 'mauratus_homolog_orthology_type': 'Golden Hamster homology type' (default: False), 'mauratus_homolog_perc_id': '%id. target Golden Hamster gene identical to query gene' (default: False), 'mauratus_homolog_perc_id_r1': '%id. query gene identical to target Golden Hamster gene' (default: False), 'mauratus_homolog_goc_score': 'Golden Hamster Gene-order conservation score' (default: False), 'mauratus_homolog_wga_coverage': 'Golden Hamster Whole-genome alignment coverage' (default: False), 'mauratus_homolog_orthology_confidence': 'Golden Hamster orthology confidence [0 low, 1 high]' (default: False), 'acchrysaetos_homolog_ensembl_gene': 'Golden eagle gene stable ID' (default: False), 'acchrysaetos_homolog_associated_gene_name': 'Golden eagle gene name' (default: False), 'acchrysaetos_homolog_ensembl_peptide': 'Golden eagle protein or transcript stable ID' (default: False), 'acchrysaetos_homolog_chromosome': 'Golden eagle chromosome/scaffold name' (default: False), 'acchrysaetos_homolog_chrom_start': 'Golden eagle chromosome/scaffold start (bp)' (default: False), 'acchrysaetos_homolog_chrom_end': 'Golden eagle chromosome/scaffold end (bp)' (default: False), 'acchrysaetos_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'acchrysaetos_homolog_subtype': 'Last common ancestor with Golden eagle' (default: False), 'acchrysaetos_homolog_orthology_type': 'Golden eagle homology type' (default: False), 'acchrysaetos_homolog_perc_id': '%id. target Golden eagle gene identical to query gene' (default: False), 'acchrysaetos_homolog_perc_id_r1': '%id. query gene identical to target Golden eagle gene' (default: False), 'acchrysaetos_homolog_goc_score': 'Golden eagle Gene-order conservation score' (default: False), 'acchrysaetos_homolog_wga_coverage': 'Golden eagle Whole-genome alignment coverage' (default: False), 'acchrysaetos_homolog_orthology_confidence': 'Golden eagle orthology confidence [0 low, 1 high]' (default: False), 'rroxellana_homolog_ensembl_gene': 'Golden snub-nosed monkey gene stable ID' (default: False), 'rroxellana_homolog_associated_gene_name': 'Golden snub-nosed monkey gene name' (default: False), 'rroxellana_homolog_ensembl_peptide': 'Golden snub-nosed monkey protein or transcript stable ID' (default: False), 'rroxellana_homolog_chromosome': 'Golden snub-nosed monkey chromosome/scaffold name' (default: False), 'rroxellana_homolog_chrom_start': 'Golden snub-nosed monkey chromosome/scaffold start (bp)' (default: False), 'rroxellana_homolog_chrom_end': 'Golden snub-nosed monkey chromosome/scaffold end (bp)' (default: False), 'rroxellana_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'rroxellana_homolog_subtype': 'Last common ancestor with Golden snub-nosed monkey' (default: False), 'rroxellana_homolog_orthology_type': 'Golden snub-nosed monkey homology type' (default: False), 'rroxellana_homolog_perc_id': '%id. target Golden snub-nosed monkey gene identical to query gene' (default: False), 'rroxellana_homolog_perc_id_r1': '%id. query gene identical to target Golden snub-nosed monkey gene' (default: False), 'rroxellana_homolog_goc_score': 'Golden snub-nosed monkey Gene-order conservation score' (default: False), 'rroxellana_homolog_wga_coverage': 'Golden snub-nosed monkey Whole-genome alignment coverage' (default: False), 'rroxellana_homolog_orthology_confidence': 'Golden snub-nosed monkey orthology confidence [0 low, 1 high]' (default: False), 'mvitellinus_homolog_ensembl_gene': 'Golden-collared manakin gene stable ID' (default: False), 'mvitellinus_homolog_associated_gene_name': 'Golden-collared manakin gene name' (default: False), 'mvitellinus_homolog_ensembl_peptide': 'Golden-collared manakin protein or transcript stable ID' (default: False), 'mvitellinus_homolog_chromosome': 'Golden-collared manakin chromosome/scaffold name' (default: False), 'mvitellinus_homolog_chrom_start': 'Golden-collared manakin chromosome/scaffold start (bp)' (default: False), 'mvitellinus_homolog_chrom_end': 'Golden-collared manakin chromosome/scaffold end (bp)' (default: False), 'mvitellinus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'mvitellinus_homolog_subtype': 'Last common ancestor with Golden-collared manakin' (default: False), 'mvitellinus_homolog_orthology_type': 'Golden-collared manakin homology type' (default: False), 'mvitellinus_homolog_perc_id': '%id. target Golden-collared manakin gene identical to query gene' (default: False), 'mvitellinus_homolog_perc_id_r1': '%id. query gene identical to target Golden-collared manakin gene' (default: False), 'mvitellinus_homolog_goc_score': 'Golden-collared manakin Gene-order conservation score' (default: False), 'mvitellinus_homolog_wga_coverage': 'Golden-collared manakin Whole-genome alignment coverage' (default: False), 'mvitellinus_homolog_orthology_confidence': 'Golden-collared manakin orthology confidence [0 low, 1 high]' (default: False), 'ggorilla_homolog_ensembl_gene': 'Gorilla gene stable ID' (default: False), 'ggorilla_homolog_associated_gene_name': 'Gorilla gene name' (default: False), 'ggorilla_homolog_ensembl_peptide': 'Gorilla protein or transcript stable ID' (default: False), 'ggorilla_homolog_chromosome': 'Gorilla chromosome/scaffold name' (default: False), 'ggorilla_homolog_chrom_start': 'Gorilla chromosome/scaffold start (bp)' (default: False), 'ggorilla_homolog_chrom_end': 'Gorilla chromosome/scaffold end (bp)' (default: False), 'ggorilla_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'ggorilla_homolog_subtype': 'Last common ancestor with Gorilla' (default: False), 'ggorilla_homolog_orthology_type': 'Gorilla homology type' (default: False), 'ggorilla_homolog_perc_id': '%id. target Gorilla gene identical to query gene' (default: False), 'ggorilla_homolog_perc_id_r1': '%id. query gene identical to target Gorilla gene' (default: False), 'ggorilla_homolog_goc_score': 'Gorilla Gene-order conservation score' (default: False), 'ggorilla_homolog_wga_coverage': 'Gorilla Whole-genome alignment coverage' (default: False), 'ggorilla_homolog_orthology_confidence': 'Gorilla orthology confidence [0 low, 1 high]' (default: False), 'sdumerili_homolog_ensembl_gene': 'Greater amberjack gene stable ID' (default: False), 'sdumerili_homolog_associated_gene_name': 'Greater amberjack gene name' (default: False), 'sdumerili_homolog_ensembl_peptide': 'Greater amberjack protein or transcript stable ID' (default: False), 'sdumerili_homolog_chromosome': 'Greater amberjack chromosome/scaffold name' (default: False), 'sdumerili_homolog_chrom_start': 'Greater amberjack chromosome/scaffold start (bp)' (default: False), 'sdumerili_homolog_chrom_end': 'Greater amberjack chromosome/scaffold end (bp)' (default: False), 'sdumerili_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'sdumerili_homolog_subtype': 'Last common ancestor with Greater amberjack' (default: False), 'sdumerili_homolog_orthology_type': 'Greater amberjack homology type' (default: False), 'sdumerili_homolog_perc_id': '%id. target Greater amberjack gene identical to query gene' (default: False), 'sdumerili_homolog_perc_id_r1': '%id. query gene identical to target Greater amberjack gene' (default: False), 'sdumerili_homolog_goc_score': 'Greater amberjack Gene-order conservation score' (default: False), 'sdumerili_homolog_wga_coverage': 'Greater amberjack Whole-genome alignment coverage' (default: False), 'sdumerili_homolog_orthology_confidence': 'Greater amberjack orthology confidence [0 low, 1 high]' (default: False), 'psimus_homolog_ensembl_gene': 'Greater bamboo lemur gene stable ID' (default: False), 'psimus_homolog_associated_gene_name': 'Greater bamboo lemur gene name' (default: False), 'psimus_homolog_ensembl_peptide': 'Greater bamboo lemur protein or transcript stable ID' (default: False), 'psimus_homolog_chromosome': 'Greater bamboo lemur chromosome/scaffold name' (default: False), 'psimus_homolog_chrom_start': 'Greater bamboo lemur chromosome/scaffold start (bp)' (default: False), 'psimus_homolog_chrom_end': 'Greater bamboo lemur chromosome/scaffold end (bp)' (default: False), 'psimus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'psimus_homolog_subtype': 'Last common ancestor with Greater bamboo lemur' (default: False), 'psimus_homolog_orthology_type': 'Greater bamboo lemur homology type' (default: False), 'psimus_homolog_perc_id': '%id. target Greater bamboo lemur gene identical to query gene' (default: False), 'psimus_homolog_perc_id_r1': '%id. query gene identical to target Greater bamboo lemur gene' (default: False), 'psimus_homolog_goc_score': 'Greater bamboo lemur Gene-order conservation score' (default: False), 'psimus_homolog_wga_coverage': 'Greater bamboo lemur Whole-genome alignment coverage' (default: False), 'psimus_homolog_orthology_confidence': 'Greater bamboo lemur orthology confidence [0 low, 1 high]' (default: False), 'rferrumequinum_homolog_ensembl_gene': 'Greater horseshoe bat gene stable ID' (default: False), 'rferrumequinum_homolog_associated_gene_name': 'Greater horseshoe bat gene name' (default: False), 'rferrumequinum_homolog_ensembl_peptide': 'Greater horseshoe bat protein or transcript stable ID' (default: False), 'rferrumequinum_homolog_chromosome': 'Greater horseshoe bat chromosome/scaffold name' (default: False), 'rferrumequinum_homolog_chrom_start': 'Greater horseshoe bat chromosome/scaffold start (bp)' (default: False), 'rferrumequinum_homolog_chrom_end': 'Greater horseshoe bat chromosome/scaffold end (bp)' (default: False), 'rferrumequinum_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'rferrumequinum_homolog_subtype': 'Last common ancestor with Greater horseshoe bat' (default: False), 'rferrumequinum_homolog_orthology_type': 'Greater horseshoe bat homology type' (default: False), 'rferrumequinum_homolog_perc_id': '%id. target Greater horseshoe bat gene identical to query gene' (default: False), 'rferrumequinum_homolog_perc_id_r1': '%id. query gene identical to target Greater horseshoe bat gene' (default: False), 'rferrumequinum_homolog_goc_score': 'Greater horseshoe bat Gene-order conservation score' (default: False), 'rferrumequinum_homolog_wga_coverage': 'Greater horseshoe bat Whole-genome alignment coverage' (default: False), 'rferrumequinum_homolog_orthology_confidence': 'Greater horseshoe bat orthology confidence [0 low, 1 high]' (default: False), 'cporcellus_homolog_ensembl_gene': 'Guinea Pig gene stable ID' (default: False), 'cporcellus_homolog_associated_gene_name': 'Guinea Pig gene name' (default: False), 'cporcellus_homolog_ensembl_peptide': 'Guinea Pig protein or transcript stable ID' (default: False), 'cporcellus_homolog_chromosome': 'Guinea Pig chromosome/scaffold name' (default: False), 'cporcellus_homolog_chrom_start': 'Guinea Pig chromosome/scaffold start (bp)' (default: False), 'cporcellus_homolog_chrom_end': 'Guinea Pig chromosome/scaffold end (bp)' (default: False), 'cporcellus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'cporcellus_homolog_subtype': 'Last common ancestor with Guinea Pig' (default: False), 'cporcellus_homolog_orthology_type': 'Guinea Pig homology type' (default: False), 'cporcellus_homolog_perc_id': '%id. target Guinea Pig gene identical to query gene' (default: False), 'cporcellus_homolog_perc_id_r1': '%id. query gene identical to target Guinea Pig gene' (default: False), 'cporcellus_homolog_goc_score': 'Guinea Pig Gene-order conservation score' (default: False), 'cporcellus_homolog_wga_coverage': 'Guinea Pig Whole-genome alignment coverage' (default: False), 'cporcellus_homolog_orthology_confidence': 'Guinea Pig orthology confidence [0 low, 1 high]' (default: False), 'preticulata_homolog_ensembl_gene': 'Guppy gene stable ID' (default: False), 'preticulata_homolog_associated_gene_name': 'Guppy gene name' (default: False), 'preticulata_homolog_ensembl_peptide': 'Guppy protein or transcript stable ID' (default: False), 'preticulata_homolog_chromosome': 'Guppy chromosome/scaffold name' (default: False), 'preticulata_homolog_chrom_start': 'Guppy chromosome/scaffold start (bp)' (default: False), 'preticulata_homolog_chrom_end': 'Guppy chromosome/scaffold end (bp)' (default: False), 'preticulata_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'preticulata_homolog_subtype': 'Last common ancestor with Guppy' (default: False), 'preticulata_homolog_orthology_type': 'Guppy homology type' (default: False), 'preticulata_homolog_perc_id': '%id. target Guppy gene identical to query gene' (default: False), 'preticulata_homolog_perc_id_r1': '%id. query gene identical to target Guppy gene' (default: False), 'preticulata_homolog_goc_score': 'Guppy Gene-order conservation score' (default: False), 'preticulata_homolog_wga_coverage': 'Guppy Whole-genome alignment coverage' (default: False), 'preticulata_homolog_orthology_confidence': 'Guppy orthology confidence [0 low, 1 high]' (default: False), 'eburgeri_homolog_ensembl_gene': 'Hagfish gene stable ID' (default: False), 'eburgeri_homolog_associated_gene_name': 'Hagfish gene name' (default: False), 'eburgeri_homolog_ensembl_peptide': 'Hagfish protein or transcript stable ID' (default: False), 'eburgeri_homolog_chromosome': 'Hagfish chromosome/scaffold name' (default: False), 'eburgeri_homolog_chrom_start': 'Hagfish chromosome/scaffold start (bp)' (default: False), 'eburgeri_homolog_chrom_end': 'Hagfish chromosome/scaffold end (bp)' (default: False), 'eburgeri_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'eburgeri_homolog_subtype': 'Last common ancestor with Hagfish' (default: False), 'eburgeri_homolog_orthology_type': 'Hagfish homology type' (default: False), 'eburgeri_homolog_perc_id': '%id. target Hagfish gene identical to query gene' (default: False), 'eburgeri_homolog_perc_id_r1': '%id. query gene identical to target Hagfish gene' (default: False), 'eburgeri_homolog_wga_coverage': 'Hagfish Whole-genome alignment coverage' (default: False), 'eburgeri_homolog_orthology_confidence': 'Hagfish orthology confidence [0 low, 1 high]' (default: False), 'eeuropaeus_homolog_ensembl_gene': 'Hedgehog gene stable ID' (default: False), 'eeuropaeus_homolog_associated_gene_name': 'Hedgehog gene name' (default: False), 'eeuropaeus_homolog_ensembl_peptide': 'Hedgehog protein or transcript stable ID' (default: False), 'eeuropaeus_homolog_chromosome': 'Hedgehog chromosome/scaffold name' (default: False), 'eeuropaeus_homolog_chrom_start': 'Hedgehog chromosome/scaffold start (bp)' (default: False), 'eeuropaeus_homolog_chrom_end': 'Hedgehog chromosome/scaffold end (bp)' (default: False), 'eeuropaeus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'eeuropaeus_homolog_subtype': 'Last common ancestor with Hedgehog' (default: False), 'eeuropaeus_homolog_orthology_type': 'Hedgehog homology type' (default: False), 'eeuropaeus_homolog_perc_id': '%id. target Hedgehog gene identical to query gene' (default: False), 'eeuropaeus_homolog_perc_id_r1': '%id. query gene identical to target Hedgehog gene' (default: False), 'eeuropaeus_homolog_goc_score': 'Hedgehog Gene-order conservation score' (default: False), 'eeuropaeus_homolog_wga_coverage': 'Hedgehog Whole-genome alignment coverage' (default: False), 'eeuropaeus_homolog_orthology_confidence': 'Hedgehog orthology confidence [0 low, 1 high]' (default: False), 'ecaballus_homolog_ensembl_gene': 'Horse gene stable ID' (default: False), 'ecaballus_homolog_associated_gene_name': 'Horse gene name' (default: False), 'ecaballus_homolog_ensembl_peptide': 'Horse protein or transcript stable ID' (default: False), 'ecaballus_homolog_chromosome': 'Horse chromosome/scaffold name' (default: False), 'ecaballus_homolog_chrom_start': 'Horse chromosome/scaffold start (bp)' (default: False), 'ecaballus_homolog_chrom_end': 'Horse chromosome/scaffold end (bp)' (default: False), 'ecaballus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'ecaballus_homolog_subtype': 'Last common ancestor with Horse' (default: False), 'ecaballus_homolog_orthology_type': 'Horse homology type' (default: False), 'ecaballus_homolog_perc_id': '%id. target Horse gene identical to query gene' (default: False), 'ecaballus_homolog_perc_id_r1': '%id. query gene identical to target Horse gene' (default: False), 'ecaballus_homolog_goc_score': 'Horse Gene-order conservation score' (default: False), 'ecaballus_homolog_wga_coverage': 'Horse Whole-genome alignment coverage' (default: False), 'ecaballus_homolog_orthology_confidence': 'Horse orthology confidence [0 low, 1 high]' (default: False), 'hhucho_homolog_ensembl_gene': 'Huchen gene stable ID' (default: False), 'hhucho_homolog_associated_gene_name': 'Huchen gene name' (default: False), 'hhucho_homolog_ensembl_peptide': 'Huchen protein or transcript stable ID' (default: False), 'hhucho_homolog_chromosome': 'Huchen chromosome/scaffold name' (default: False), 'hhucho_homolog_chrom_start': 'Huchen chromosome/scaffold start (bp)' (default: False), 'hhucho_homolog_chrom_end': 'Huchen chromosome/scaffold end (bp)' (default: False), 'hhucho_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'hhucho_homolog_subtype': 'Last common ancestor with Huchen' (default: False), 'hhucho_homolog_orthology_type': 'Huchen homology type' (default: False), 'hhucho_homolog_perc_id': '%id. target Huchen gene identical to query gene' (default: False), 'hhucho_homolog_perc_id_r1': '%id. query gene identical to target Huchen gene' (default: False), 'hhucho_homolog_goc_score': 'Huchen Gene-order conservation score' (default: False), 'hhucho_homolog_wga_coverage': 'Huchen Whole-genome alignment coverage' (default: False), 'hhucho_homolog_orthology_confidence': 'Huchen orthology confidence [0 low, 1 high]' (default: False), 'bihybrid_homolog_ensembl_gene': 'Hybrid - Bos Indicus gene stable ID' (default: False), 'bihybrid_homolog_associated_gene_name': 'Hybrid - Bos Indicus gene name' (default: False), 'bihybrid_homolog_ensembl_peptide': 'Hybrid - Bos Indicus protein or transcript stable ID' (default: False), 'bihybrid_homolog_chromosome': 'Hybrid - Bos Indicus chromosome/scaffold name' (default: False), 'bihybrid_homolog_chrom_start': 'Hybrid - Bos Indicus chromosome/scaffold start (bp)' (default: False), 'bihybrid_homolog_chrom_end': 'Hybrid - Bos Indicus chromosome/scaffold end (bp)' (default: False), 'bihybrid_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'bihybrid_homolog_subtype': 'Last common ancestor with Hybrid - Bos Indicus' (default: False), 'bihybrid_homolog_orthology_type': 'Hybrid - Bos Indicus homology type' (default: False), 'bihybrid_homolog_perc_id': '%id. target Hybrid - Bos Indicus gene identical to query gene' (default: False), 'bihybrid_homolog_perc_id_r1': '%id. query gene identical to target Hybrid - Bos Indicus gene' (default: False), 'bihybrid_homolog_goc_score': 'Hybrid - Bos Indicus Gene-order conservation score' (default: False), 'bihybrid_homolog_wga_coverage': 'Hybrid - Bos Indicus Whole-genome alignment coverage' (default: False), 'bihybrid_homolog_orthology_confidence': 'Hybrid - Bos Indicus orthology confidence [0 low, 1 high]' (default: False), 'bthybrid_homolog_ensembl_gene': 'Hybrid - Bos Taurus gene stable ID' (default: False), 'bthybrid_homolog_associated_gene_name': 'Hybrid - Bos Taurus gene name' (default: False), 'bthybrid_homolog_ensembl_peptide': 'Hybrid - Bos Taurus protein or transcript stable ID' (default: False), 'bthybrid_homolog_chromosome': 'Hybrid - Bos Taurus chromosome/scaffold name' (default: False), 'bthybrid_homolog_chrom_start': 'Hybrid - Bos Taurus chromosome/scaffold start (bp)' (default: False), 'bthybrid_homolog_chrom_end': 'Hybrid - Bos Taurus chromosome/scaffold end (bp)' (default: False), 'bthybrid_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'bthybrid_homolog_subtype': 'Last common ancestor with Hybrid - Bos Taurus' (default: False), 'bthybrid_homolog_orthology_type': 'Hybrid - Bos Taurus homology type' (default: False), 'bthybrid_homolog_perc_id': '%id. target Hybrid - Bos Taurus gene identical to query gene' (default: False), 'bthybrid_homolog_perc_id_r1': '%id. query gene identical to target Hybrid - Bos Taurus gene' (default: False), 'bthybrid_homolog_goc_score': 'Hybrid - Bos Taurus Gene-order conservation score' (default: False), 'bthybrid_homolog_wga_coverage': 'Hybrid - Bos Taurus Whole-genome alignment coverage' (default: False), 'bthybrid_homolog_orthology_confidence': 'Hybrid - Bos Taurus orthology confidence [0 low, 1 high]' (default: False), 'pcapensis_homolog_ensembl_gene': 'Hyrax gene stable ID' (default: False), 'pcapensis_homolog_associated_gene_name': 'Hyrax gene name' (default: False), 'pcapensis_homolog_ensembl_peptide': 'Hyrax protein or transcript stable ID' (default: False), 'pcapensis_homolog_chromosome': 'Hyrax chromosome/scaffold name' (default: False), 'pcapensis_homolog_chrom_start': 'Hyrax chromosome/scaffold start (bp)' (default: False), 'pcapensis_homolog_chrom_end': 'Hyrax chromosome/scaffold end (bp)' (default: False), 'pcapensis_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'pcapensis_homolog_subtype': 'Last common ancestor with Hyrax' (default: False), 'pcapensis_homolog_orthology_type': 'Hyrax homology type' (default: False), 'pcapensis_homolog_perc_id': '%id. target Hyrax gene identical to query gene' (default: False), 'pcapensis_homolog_perc_id_r1': '%id. query gene identical to target Hyrax gene' (default: False), 'pcapensis_homolog_goc_score': 'Hyrax Gene-order conservation score' (default: False), 'pcapensis_homolog_wga_coverage': 'Hyrax Whole-genome alignment coverage' (default: False), 'pcapensis_homolog_orthology_confidence': 'Hyrax orthology confidence [0 low, 1 high]' (default: False), 'pranga_homolog_ensembl_gene': 'Indian glassy fish gene stable ID' (default: False), 'pranga_homolog_associated_gene_name': 'Indian glassy fish gene name' (default: False), 'pranga_homolog_ensembl_peptide': 'Indian glassy fish protein or transcript stable ID' (default: False), 'pranga_homolog_chromosome': 'Indian glassy fish chromosome/scaffold name' (default: False), 'pranga_homolog_chrom_start': 'Indian glassy fish chromosome/scaffold start (bp)' (default: False), 'pranga_homolog_chrom_end': 'Indian glassy fish chromosome/scaffold end (bp)' (default: False), 'pranga_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'pranga_homolog_subtype': 'Last common ancestor with Indian glassy fish' (default: False), 'pranga_homolog_orthology_type': 'Indian glassy fish homology type' (default: False), 'pranga_homolog_perc_id': '%id. target Indian glassy fish gene identical to query gene' (default: False), 'pranga_homolog_perc_id_r1': '%id. query gene identical to target Indian glassy fish gene' (default: False), 'pranga_homolog_goc_score': 'Indian glassy fish Gene-order conservation score' (default: False), 'pranga_homolog_wga_coverage': 'Indian glassy fish Whole-genome alignment coverage' (default: False), 'pranga_homolog_orthology_confidence': 'Indian glassy fish orthology confidence [0 low, 1 high]' (default: False), 'omelastigma_homolog_ensembl_gene': 'Indian medaka gene stable ID' (default: False), 'omelastigma_homolog_associated_gene_name': 'Indian medaka gene name' (default: False), 'omelastigma_homolog_ensembl_peptide': 'Indian medaka protein or transcript stable ID' (default: False), 'omelastigma_homolog_chromosome': 'Indian medaka chromosome/scaffold name' (default: False), 'omelastigma_homolog_chrom_start': 'Indian medaka chromosome/scaffold start (bp)' (default: False), 'omelastigma_homolog_chrom_end': 'Indian medaka chromosome/scaffold end (bp)' (default: False), 'omelastigma_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'omelastigma_homolog_subtype': 'Last common ancestor with Indian medaka' (default: False), 'omelastigma_homolog_orthology_type': 'Indian medaka homology type' (default: False), 'omelastigma_homolog_perc_id': '%id. target Indian medaka gene identical to query gene' (default: False), 'omelastigma_homolog_perc_id_r1': '%id. query gene identical to target Indian medaka gene' (default: False), 'omelastigma_homolog_goc_score': 'Indian medaka Gene-order conservation score' (default: False), 'omelastigma_homolog_wga_coverage': 'Indian medaka Whole-genome alignment coverage' (default: False), 'omelastigma_homolog_orthology_confidence': 'Indian medaka orthology confidence [0 low, 1 high]' (default: False), 'olhni_homolog_ensembl_gene': 'Japanese medaka HNI gene stable ID' (default: False), 'olhni_homolog_associated_gene_name': 'Japanese medaka HNI gene name' (default: False), 'olhni_homolog_ensembl_peptide': 'Japanese medaka HNI protein or transcript stable ID' (default: False), 'olhni_homolog_chromosome': 'Japanese medaka HNI chromosome/scaffold name' (default: False), 'olhni_homolog_chrom_start': 'Japanese medaka HNI chromosome/scaffold start (bp)' (default: False), 'olhni_homolog_chrom_end': 'Japanese medaka HNI chromosome/scaffold end (bp)' (default: False), 'olhni_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'olhni_homolog_subtype': 'Last common ancestor with Japanese medaka HNI' (default: False), 'olhni_homolog_orthology_type': 'Japanese medaka HNI homology type' (default: False), 'olhni_homolog_perc_id': '%id. target Japanese medaka HNI gene identical to query gene' (default: False), 'olhni_homolog_perc_id_r1': '%id. query gene identical to target Japanese medaka HNI gene' (default: False), 'olhni_homolog_goc_score': 'Japanese medaka HNI Gene-order conservation score' (default: False), 'olhni_homolog_wga_coverage': 'Japanese medaka HNI Whole-genome alignment coverage' (default: False), 'olhni_homolog_orthology_confidence': 'Japanese medaka HNI orthology confidence [0 low, 1 high]' (default: False), 'olhsok_homolog_ensembl_gene': 'Japanese medaka HSOK gene stable ID' (default: False), 'olhsok_homolog_associated_gene_name': 'Japanese medaka HSOK gene name' (default: False), 'olhsok_homolog_ensembl_peptide': 'Japanese medaka HSOK protein or transcript stable ID' (default: False), 'olhsok_homolog_chromosome': 'Japanese medaka HSOK chromosome/scaffold name' (default: False), 'olhsok_homolog_chrom_start': 'Japanese medaka HSOK chromosome/scaffold start (bp)' (default: False), 'olhsok_homolog_chrom_end': 'Japanese medaka HSOK chromosome/scaffold end (bp)' (default: False), 'olhsok_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'olhsok_homolog_subtype': 'Last common ancestor with Japanese medaka HSOK' (default: False), 'olhsok_homolog_orthology_type': 'Japanese medaka HSOK homology type' (default: False), 'olhsok_homolog_perc_id': '%id. target Japanese medaka HSOK gene identical to query gene' (default: False), 'olhsok_homolog_perc_id_r1': '%id. query gene identical to target Japanese medaka HSOK gene' (default: False), 'olhsok_homolog_goc_score': 'Japanese medaka HSOK Gene-order conservation score' (default: False), 'olhsok_homolog_wga_coverage': 'Japanese medaka HSOK Whole-genome alignment coverage' (default: False), 'olhsok_homolog_orthology_confidence': 'Japanese medaka HSOK orthology confidence [0 low, 1 high]' (default: False), 'olatipes_homolog_ensembl_gene': 'Japanese medaka HdrR gene stable ID' (default: False), 'olatipes_homolog_associated_gene_name': 'Japanese medaka HdrR gene name' (default: False), 'olatipes_homolog_ensembl_peptide': 'Japanese medaka HdrR protein or transcript stable ID' (default: False), 'olatipes_homolog_chromosome': 'Japanese medaka HdrR chromosome/scaffold name' (default: False), 'olatipes_homolog_chrom_start': 'Japanese medaka HdrR chromosome/scaffold start (bp)' (default: False), 'olatipes_homolog_chrom_end': 'Japanese medaka HdrR chromosome/scaffold end (bp)' (default: False), 'olatipes_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'olatipes_homolog_subtype': 'Last common ancestor with Japanese medaka HdrR' (default: False), 'olatipes_homolog_orthology_type': 'Japanese medaka HdrR homology type' (default: False), 'olatipes_homolog_perc_id': '%id. target Japanese medaka HdrR gene identical to query gene' (default: False), 'olatipes_homolog_perc_id_r1': '%id. query gene identical to target Japanese medaka HdrR gene' (default: False), 'olatipes_homolog_goc_score': 'Japanese medaka HdrR Gene-order conservation score' (default: False), 'olatipes_homolog_wga_coverage': 'Japanese medaka HdrR Whole-genome alignment coverage' (default: False), 'olatipes_homolog_orthology_confidence': 'Japanese medaka HdrR orthology confidence [0 low, 1 high]' (default: False), 'cjaponica_homolog_ensembl_gene': 'Japanese quail gene stable ID' (default: False), 'cjaponica_homolog_associated_gene_name': 'Japanese quail gene name' (default: False), 'cjaponica_homolog_ensembl_peptide': 'Japanese quail protein or transcript stable ID' (default: False), 'cjaponica_homolog_chromosome': 'Japanese quail chromosome/scaffold name' (default: False), 'cjaponica_homolog_chrom_start': 'Japanese quail chromosome/scaffold start (bp)' (default: False), 'cjaponica_homolog_chrom_end': 'Japanese quail chromosome/scaffold end (bp)' (default: False), 'cjaponica_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'cjaponica_homolog_subtype': 'Last common ancestor with Japanese quail' (default: False), 'cjaponica_homolog_orthology_type': 'Japanese quail homology type' (default: False), 'cjaponica_homolog_perc_id': '%id. target Japanese quail gene identical to query gene' (default: False), 'cjaponica_homolog_perc_id_r1': '%id. query gene identical to target Japanese quail gene' (default: False), 'cjaponica_homolog_goc_score': 'Japanese quail Gene-order conservation score' (default: False), 'cjaponica_homolog_wga_coverage': 'Japanese quail Whole-genome alignment coverage' (default: False), 'cjaponica_homolog_orthology_confidence': 'Japanese quail orthology confidence [0 low, 1 high]' (default: False), 'sfasciatus_homolog_ensembl_gene': 'Jewelled blenny gene stable ID' (default: False), 'sfasciatus_homolog_associated_gene_name': 'Jewelled blenny gene name' (default: False), 'sfasciatus_homolog_ensembl_peptide': 'Jewelled blenny protein or transcript stable ID' (default: False), 'sfasciatus_homolog_chromosome': 'Jewelled blenny chromosome/scaffold name' (default: False), 'sfasciatus_homolog_chrom_start': 'Jewelled blenny chromosome/scaffold start (bp)' (default: False), 'sfasciatus_homolog_chrom_end': 'Jewelled blenny chromosome/scaffold end (bp)' (default: False), 'sfasciatus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'sfasciatus_homolog_subtype': 'Last common ancestor with Jewelled blenny' (default: False), 'sfasciatus_homolog_orthology_type': 'Jewelled blenny homology type' (default: False), 'sfasciatus_homolog_perc_id': '%id. target Jewelled blenny gene identical to query gene' (default: False), 'sfasciatus_homolog_perc_id_r1': '%id. query gene identical to target Jewelled blenny gene' (default: False), 'sfasciatus_homolog_goc_score': 'Jewelled blenny Gene-order conservation score' (default: False), 'sfasciatus_homolog_wga_coverage': 'Jewelled blenny Whole-genome alignment coverage' (default: False), 'sfasciatus_homolog_orthology_confidence': 'Jewelled blenny orthology confidence [0 low, 1 high]' (default: False), 'shabroptila_homolog_ensembl_gene': 'Kakapo gene stable ID' (default: False), 'shabroptila_homolog_associated_gene_name': 'Kakapo gene name' (default: False), 'shabroptila_homolog_ensembl_peptide': 'Kakapo protein or transcript stable ID' (default: False), 'shabroptila_homolog_chromosome': 'Kakapo chromosome/scaffold name' (default: False), 'shabroptila_homolog_chrom_start': 'Kakapo chromosome/scaffold start (bp)' (default: False), 'shabroptila_homolog_chrom_end': 'Kakapo chromosome/scaffold end (bp)' (default: False), 'shabroptila_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'shabroptila_homolog_subtype': 'Last common ancestor with Kakapo' (default: False), 'shabroptila_homolog_orthology_type': 'Kakapo homology type' (default: False), 'shabroptila_homolog_perc_id': '%id. target Kakapo gene identical to query gene' (default: False), 'shabroptila_homolog_perc_id_r1': '%id. query gene identical to target Kakapo gene' (default: False), 'shabroptila_homolog_goc_score': 'Kakapo Gene-order conservation score' (default: False), 'shabroptila_homolog_wga_coverage': 'Kakapo Whole-genome alignment coverage' (default: False), 'shabroptila_homolog_orthology_confidence': 'Kakapo orthology confidence [0 low, 1 high]' (default: False), 'dordii_homolog_ensembl_gene': 'Kangaroo rat gene stable ID' (default: False), 'dordii_homolog_associated_gene_name': 'Kangaroo rat gene name' (default: False), 'dordii_homolog_ensembl_peptide': 'Kangaroo rat protein or transcript stable ID' (default: False), 'dordii_homolog_chromosome': 'Kangaroo rat chromosome/scaffold name' (default: False), 'dordii_homolog_chrom_start': 'Kangaroo rat chromosome/scaffold start (bp)' (default: False), 'dordii_homolog_chrom_end': 'Kangaroo rat chromosome/scaffold end (bp)' (default: False), 'dordii_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'dordii_homolog_subtype': 'Last common ancestor with Kangaroo rat' (default: False), 'dordii_homolog_orthology_type': 'Kangaroo rat homology type' (default: False), 'dordii_homolog_perc_id': '%id. target Kangaroo rat gene identical to query gene' (default: False), 'dordii_homolog_perc_id_r1': '%id. query gene identical to target Kangaroo rat gene' (default: False), 'dordii_homolog_goc_score': 'Kangaroo rat Gene-order conservation score' (default: False), 'dordii_homolog_wga_coverage': 'Kangaroo rat Whole-genome alignment coverage' (default: False), 'dordii_homolog_orthology_confidence': 'Kangaroo rat orthology confidence [0 low, 1 high]' (default: False), 'pcinereus_homolog_ensembl_gene': 'Koala gene stable ID' (default: False), 'pcinereus_homolog_associated_gene_name': 'Koala gene name' (default: False), 'pcinereus_homolog_ensembl_peptide': 'Koala protein or transcript stable ID' (default: False), 'pcinereus_homolog_chromosome': 'Koala chromosome/scaffold name' (default: False), 'pcinereus_homolog_chrom_start': 'Koala chromosome/scaffold start (bp)' (default: False), 'pcinereus_homolog_chrom_end': 'Koala chromosome/scaffold end (bp)' (default: False), 'pcinereus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'pcinereus_homolog_subtype': 'Last common ancestor with Koala' (default: False), 'pcinereus_homolog_orthology_type': 'Koala homology type' (default: False), 'pcinereus_homolog_perc_id': '%id. target Koala gene identical to query gene' (default: False), 'pcinereus_homolog_perc_id_r1': '%id. query gene identical to target Koala gene' (default: False), 'pcinereus_homolog_goc_score': 'Koala Gene-order conservation score' (default: False), 'pcinereus_homolog_wga_coverage': 'Koala Whole-genome alignment coverage' (default: False), 'pcinereus_homolog_orthology_confidence': 'Koala orthology confidence [0 low, 1 high]' (default: False), 'pmarinus_homolog_ensembl_gene': 'Lamprey gene stable ID' (default: False), 'pmarinus_homolog_associated_gene_name': 'Lamprey gene name' (default: False), 'pmarinus_homolog_ensembl_peptide': 'Lamprey protein or transcript stable ID' (default: False), 'pmarinus_homolog_chromosome': 'Lamprey chromosome/scaffold name' (default: False), 'pmarinus_homolog_chrom_start': 'Lamprey chromosome/scaffold start (bp)' (default: False), 'pmarinus_homolog_chrom_end': 'Lamprey chromosome/scaffold end (bp)' (default: False), 'pmarinus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'pmarinus_homolog_subtype': 'Last common ancestor with Lamprey' (default: False), 'pmarinus_homolog_orthology_type': 'Lamprey homology type' (default: False), 'pmarinus_homolog_perc_id': '%id. target Lamprey gene identical to query gene' (default: False), 'pmarinus_homolog_perc_id_r1': '%id. query gene identical to target Lamprey gene' (default: False), 'pmarinus_homolog_wga_coverage': 'Lamprey Whole-genome alignment coverage' (default: False), 'pmarinus_homolog_orthology_confidence': 'Lamprey orthology confidence [0 low, 1 high]' (default: False), 'lcrocea_homolog_ensembl_gene': 'Large yellow croaker gene stable ID' (default: False), 'lcrocea_homolog_associated_gene_name': 'Large yellow croaker gene name' (default: False), 'lcrocea_homolog_ensembl_peptide': 'Large yellow croaker protein or transcript stable ID' (default: False), 'lcrocea_homolog_chromosome': 'Large yellow croaker chromosome/scaffold name' (default: False), 'lcrocea_homolog_chrom_start': 'Large yellow croaker chromosome/scaffold start (bp)' (default: False), 'lcrocea_homolog_chrom_end': 'Large yellow croaker chromosome/scaffold end (bp)' (default: False), 'lcrocea_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'lcrocea_homolog_subtype': 'Last common ancestor with Large yellow croaker' (default: False), 'lcrocea_homolog_orthology_type': 'Large yellow croaker homology type' (default: False), 'lcrocea_homolog_perc_id': '%id. target Large yellow croaker gene identical to query gene' (default: False), 'lcrocea_homolog_perc_id_r1': '%id. query gene identical to target Large yellow croaker gene' (default: False), 'lcrocea_homolog_goc_score': 'Large yellow croaker Gene-order conservation score' (default: False), 'lcrocea_homolog_wga_coverage': 'Large yellow croaker Whole-genome alignment coverage' (default: False), 'lcrocea_homolog_orthology_confidence': 'Large yellow croaker orthology confidence [0 low, 1 high]' (default: False), 'ppardus_homolog_ensembl_gene': 'Leopard gene stable ID' (default: False), 'ppardus_homolog_associated_gene_name': 'Leopard gene name' (default: False), 'ppardus_homolog_ensembl_peptide': 'Leopard protein or transcript stable ID' (default: False), 'ppardus_homolog_chromosome': 'Leopard chromosome/scaffold name' (default: False), 'ppardus_homolog_chrom_start': 'Leopard chromosome/scaffold start (bp)' (default: False), 'ppardus_homolog_chrom_end': 'Leopard chromosome/scaffold end (bp)' (default: False), 'ppardus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'ppardus_homolog_subtype': 'Last common ancestor with Leopard' (default: False), 'ppardus_homolog_orthology_type': 'Leopard homology type' (default: False), 'ppardus_homolog_perc_id': '%id. target Leopard gene identical to query gene' (default: False), 'ppardus_homolog_perc_id_r1': '%id. query gene identical to target Leopard gene' (default: False), 'ppardus_homolog_goc_score': 'Leopard Gene-order conservation score' (default: False), 'ppardus_homolog_wga_coverage': 'Leopard Whole-genome alignment coverage' (default: False), 'ppardus_homolog_orthology_confidence': 'Leopard orthology confidence [0 low, 1 high]' (default: False), 'jjaculus_homolog_ensembl_gene': 'Lesser Egyptian jerboa gene stable ID' (default: False), 'jjaculus_homolog_associated_gene_name': 'Lesser Egyptian jerboa gene name' (default: False), 'jjaculus_homolog_ensembl_peptide': 'Lesser Egyptian jerboa protein or transcript stable ID' (default: False), 'jjaculus_homolog_chromosome': 'Lesser Egyptian jerboa chromosome/scaffold name' (default: False), 'jjaculus_homolog_chrom_start': 'Lesser Egyptian jerboa chromosome/scaffold start (bp)' (default: False), 'jjaculus_homolog_chrom_end': 'Lesser Egyptian jerboa chromosome/scaffold end (bp)' (default: False), 'jjaculus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'jjaculus_homolog_subtype': 'Last common ancestor with Lesser Egyptian jerboa' (default: False), 'jjaculus_homolog_orthology_type': 'Lesser Egyptian jerboa homology type' (default: False), 'jjaculus_homolog_perc_id': '%id. target Lesser Egyptian jerboa gene identical to query gene' (default: False), 'jjaculus_homolog_perc_id_r1': '%id. query gene identical to target Lesser Egyptian jerboa gene' (default: False), 'jjaculus_homolog_goc_score': 'Lesser Egyptian jerboa Gene-order conservation score' (default: False), 'jjaculus_homolog_wga_coverage': 'Lesser Egyptian jerboa Whole-genome alignment coverage' (default: False), 'jjaculus_homolog_orthology_confidence': 'Lesser Egyptian jerboa orthology confidence [0 low, 1 high]' (default: False), 'etelfairi_homolog_ensembl_gene': 'Lesser hedgehog tenrec gene stable ID' (default: False), 'etelfairi_homolog_associated_gene_name': 'Lesser hedgehog tenrec gene name' (default: False), 'etelfairi_homolog_ensembl_peptide': 'Lesser hedgehog tenrec protein or transcript stable ID' (default: False), 'etelfairi_homolog_chromosome': 'Lesser hedgehog tenrec chromosome/scaffold name' (default: False), 'etelfairi_homolog_chrom_start': 'Lesser hedgehog tenrec chromosome/scaffold start (bp)' (default: False), 'etelfairi_homolog_chrom_end': 'Lesser hedgehog tenrec chromosome/scaffold end (bp)' (default: False), 'etelfairi_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'etelfairi_homolog_subtype': 'Last common ancestor with Lesser hedgehog tenrec' (default: False), 'etelfairi_homolog_orthology_type': 'Lesser hedgehog tenrec homology type' (default: False), 'etelfairi_homolog_perc_id': '%id. target Lesser hedgehog tenrec gene identical to query gene' (default: False), 'etelfairi_homolog_perc_id_r1': '%id. query gene identical to target Lesser hedgehog tenrec gene' (default: False), 'etelfairi_homolog_goc_score': 'Lesser hedgehog tenrec Gene-order conservation score' (default: False), 'etelfairi_homolog_wga_coverage': 'Lesser hedgehog tenrec Whole-genome alignment coverage' (default: False), 'etelfairi_homolog_orthology_confidence': 'Lesser hedgehog tenrec orthology confidence [0 low, 1 high]' (default: False), 'enaucrates_homolog_ensembl_gene': 'Live sharksucker gene stable ID' (default: False), 'enaucrates_homolog_associated_gene_name': 'Live sharksucker gene name' (default: False), 'enaucrates_homolog_ensembl_peptide': 'Live sharksucker protein or transcript stable ID' (default: False), 'enaucrates_homolog_chromosome': 'Live sharksucker chromosome/scaffold name' (default: False), 'enaucrates_homolog_chrom_start': 'Live sharksucker chromosome/scaffold start (bp)' (default: False), 'enaucrates_homolog_chrom_end': 'Live sharksucker chromosome/scaffold end (bp)' (default: False), 'enaucrates_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'enaucrates_homolog_subtype': 'Last common ancestor with Live sharksucker' (default: False), 'enaucrates_homolog_orthology_type': 'Live sharksucker homology type' (default: False), 'enaucrates_homolog_perc_id': '%id. target Live sharksucker gene identical to query gene' (default: False), 'enaucrates_homolog_perc_id_r1': '%id. query gene identical to target Live sharksucker gene' (default: False), 'enaucrates_homolog_goc_score': 'Live sharksucker Gene-order conservation score' (default: False), 'enaucrates_homolog_wga_coverage': 'Live sharksucker Whole-genome alignment coverage' (default: False), 'enaucrates_homolog_orthology_confidence': 'Live sharksucker orthology confidence [0 low, 1 high]' (default: False), 'clanigera_homolog_ensembl_gene': 'Long-tailed chinchilla gene stable ID' (default: False), 'clanigera_homolog_associated_gene_name': 'Long-tailed chinchilla gene name' (default: False), 'clanigera_homolog_ensembl_peptide': 'Long-tailed chinchilla protein or transcript stable ID' (default: False), 'clanigera_homolog_chromosome': 'Long-tailed chinchilla chromosome/scaffold name' (default: False), 'clanigera_homolog_chrom_start': 'Long-tailed chinchilla chromosome/scaffold start (bp)' (default: False), 'clanigera_homolog_chrom_end': 'Long-tailed chinchilla chromosome/scaffold end (bp)' (default: False), 'clanigera_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'clanigera_homolog_subtype': 'Last common ancestor with Long-tailed chinchilla' (default: False), 'clanigera_homolog_orthology_type': 'Long-tailed chinchilla homology type' (default: False), 'clanigera_homolog_perc_id': '%id. target Long-tailed chinchilla gene identical to query gene' (default: False), 'clanigera_homolog_perc_id_r1': '%id. query gene identical to target Long-tailed chinchilla gene' (default: False), 'clanigera_homolog_goc_score': 'Long-tailed chinchilla Gene-order conservation score' (default: False), 'clanigera_homolog_wga_coverage': 'Long-tailed chinchilla Whole-genome alignment coverage' (default: False), 'clanigera_homolog_orthology_confidence': 'Long-tailed chinchilla orthology confidence [0 low, 1 high]' (default: False), 'nbrichardi_homolog_ensembl_gene': 'Lyretail cichlid gene stable ID' (default: False), 'nbrichardi_homolog_associated_gene_name': 'Lyretail cichlid gene name' (default: False), 'nbrichardi_homolog_ensembl_peptide': 'Lyretail cichlid protein or transcript stable ID' (default: False), 'nbrichardi_homolog_chromosome': 'Lyretail cichlid chromosome/scaffold name' (default: False), 'nbrichardi_homolog_chrom_start': 'Lyretail cichlid chromosome/scaffold start (bp)' (default: False), 'nbrichardi_homolog_chrom_end': 'Lyretail cichlid chromosome/scaffold end (bp)' (default: False), 'nbrichardi_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'nbrichardi_homolog_subtype': 'Last common ancestor with Lyretail cichlid' (default: False), 'nbrichardi_homolog_orthology_type': 'Lyretail cichlid homology type' (default: False), 'nbrichardi_homolog_perc_id': '%id. target Lyretail cichlid gene identical to query gene' (default: False), 'nbrichardi_homolog_perc_id_r1': '%id. query gene identical to target Lyretail cichlid gene' (default: False), 'nbrichardi_homolog_goc_score': 'Lyretail cichlid Gene-order conservation score' (default: False), 'nbrichardi_homolog_wga_coverage': 'Lyretail cichlid Whole-genome alignment coverage' (default: False), 'nbrichardi_homolog_orthology_confidence': 'Lyretail cichlid orthology confidence [0 low, 1 high]' (default: False), 'anancymaae_homolog_ensembl_gene': 'Ma's night monkey gene stable ID' (default: False), 'anancymaae_homolog_associated_gene_name': 'Ma's night monkey gene name' (default: False), 'anancymaae_homolog_ensembl_peptide': 'Ma's night monkey protein or transcript stable ID' (default: False), 'anancymaae_homolog_chromosome': 'Ma's night monkey chromosome/scaffold name' (default: False), 'anancymaae_homolog_chrom_start': 'Ma's night monkey chromosome/scaffold start (bp)' (default: False), 'anancymaae_homolog_chrom_end': 'Ma's night monkey chromosome/scaffold end (bp)' (default: False), 'anancymaae_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'anancymaae_homolog_subtype': 'Last common ancestor with Ma's night monkey' (default: False), 'anancymaae_homolog_orthology_type': 'Ma's night monkey homology type' (default: False), 'anancymaae_homolog_perc_id': '%id. target Ma's night monkey gene identical to query gene' (default: False), 'anancymaae_homolog_perc_id_r1': '%id. query gene identical to target Ma's night monkey gene' (default: False), 'anancymaae_homolog_goc_score': 'Ma's night monkey Gene-order conservation score' (default: False), 'anancymaae_homolog_wga_coverage': 'Ma's night monkey Whole-genome alignment coverage' (default: False), 'anancymaae_homolog_orthology_confidence': 'Ma's night monkey orthology confidence [0 low, 1 high]' (default: False), 'mmulatta_homolog_ensembl_gene': 'Macaque gene stable ID' (default: False), 'mmulatta_homolog_associated_gene_name': 'Macaque gene name' (default: False), 'mmulatta_homolog_ensembl_peptide': 'Macaque protein or transcript stable ID' (default: False), 'mmulatta_homolog_chromosome': 'Macaque chromosome/scaffold name' (default: False), 'mmulatta_homolog_chrom_start': 'Macaque chromosome/scaffold start (bp)' (default: False), 'mmulatta_homolog_chrom_end': 'Macaque chromosome/scaffold end (bp)' (default: False), 'mmulatta_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'mmulatta_homolog_subtype': 'Last common ancestor with Macaque' (default: False), 'mmulatta_homolog_orthology_type': 'Macaque homology type' (default: False), 'mmulatta_homolog_perc_id': '%id. target Macaque gene identical to query gene' (default: False), 'mmulatta_homolog_perc_id_r1': '%id. query gene identical to target Macaque gene' (default: False), 'mmulatta_homolog_goc_score': 'Macaque Gene-order conservation score' (default: False), 'mmulatta_homolog_wga_coverage': 'Macaque Whole-genome alignment coverage' (default: False), 'mmulatta_homolog_orthology_confidence': 'Macaque orthology confidence [0 low, 1 high]' (default: False), 'pnyererei_homolog_ensembl_gene': 'Makobe Island cichlid gene stable ID' (default: False), 'pnyererei_homolog_associated_gene_name': 'Makobe Island cichlid gene name' (default: False), 'pnyererei_homolog_ensembl_peptide': 'Makobe Island cichlid protein or transcript stable ID' (default: False), 'pnyererei_homolog_chromosome': 'Makobe Island cichlid chromosome/scaffold name' (default: False), 'pnyererei_homolog_chrom_start': 'Makobe Island cichlid chromosome/scaffold start (bp)' (default: False), 'pnyererei_homolog_chrom_end': 'Makobe Island cichlid chromosome/scaffold end (bp)' (default: False), 'pnyererei_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'pnyererei_homolog_subtype': 'Last common ancestor with Makobe Island cichlid' (default: False), 'pnyererei_homolog_orthology_type': 'Makobe Island cichlid homology type' (default: False), 'pnyererei_homolog_perc_id': '%id. target Makobe Island cichlid gene identical to query gene' (default: False), 'pnyererei_homolog_perc_id_r1': '%id. query gene identical to target Makobe Island cichlid gene' (default: False), 'pnyererei_homolog_goc_score': 'Makobe Island cichlid Gene-order conservation score' (default: False), 'pnyererei_homolog_wga_coverage': 'Makobe Island cichlid Whole-genome alignment coverage' (default: False), 'pnyererei_homolog_orthology_confidence': 'Makobe Island cichlid orthology confidence [0 low, 1 high]' (default: False), 'aplatyrhynchos_homolog_ensembl_gene': 'Mallard gene stable ID' (default: False), 'aplatyrhynchos_homolog_associated_gene_name': 'Mallard gene name' (default: False), 'aplatyrhynchos_homolog_ensembl_peptide': 'Mallard protein or transcript stable ID' (default: False), 'aplatyrhynchos_homolog_chromosome': 'Mallard chromosome/scaffold name' (default: False), 'aplatyrhynchos_homolog_chrom_start': 'Mallard chromosome/scaffold start (bp)' (default: False), 'aplatyrhynchos_homolog_chrom_end': 'Mallard chromosome/scaffold end (bp)' (default: False), 'aplatyrhynchos_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'aplatyrhynchos_homolog_subtype': 'Last common ancestor with Mallard' (default: False), 'aplatyrhynchos_homolog_orthology_type': 'Mallard homology type' (default: False), 'aplatyrhynchos_homolog_perc_id': '%id. target Mallard gene identical to query gene' (default: False), 'aplatyrhynchos_homolog_perc_id_r1': '%id. query gene identical to target Mallard gene' (default: False), 'aplatyrhynchos_homolog_goc_score': 'Mallard Gene-order conservation score' (default: False), 'aplatyrhynchos_homolog_wga_coverage': 'Mallard Whole-genome alignment coverage' (default: False), 'aplatyrhynchos_homolog_orthology_confidence': 'Mallard orthology confidence [0 low, 1 high]' (default: False), 'cjacchus_homolog_ensembl_gene': 'Marmoset gene stable ID' (default: False), 'cjacchus_homolog_associated_gene_name': 'Marmoset gene name' (default: False), 'cjacchus_homolog_ensembl_peptide': 'Marmoset protein or transcript stable ID' (default: False), 'cjacchus_homolog_chromosome': 'Marmoset chromosome/scaffold name' (default: False), 'cjacchus_homolog_chrom_start': 'Marmoset chromosome/scaffold start (bp)' (default: False), 'cjacchus_homolog_chrom_end': 'Marmoset chromosome/scaffold end (bp)' (default: False), 'cjacchus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'cjacchus_homolog_subtype': 'Last common ancestor with Marmoset' (default: False), 'cjacchus_homolog_orthology_type': 'Marmoset homology type' (default: False), 'cjacchus_homolog_perc_id': '%id. target Marmoset gene identical to query gene' (default: False), 'cjacchus_homolog_perc_id_r1': '%id. query gene identical to target Marmoset gene' (default: False), 'cjacchus_homolog_goc_score': 'Marmoset Gene-order conservation score' (default: False), 'cjacchus_homolog_wga_coverage': 'Marmoset Whole-genome alignment coverage' (default: False), 'cjacchus_homolog_orthology_confidence': 'Marmoset orthology confidence [0 low, 1 high]' (default: False), 'pvampyrus_homolog_ensembl_gene': 'Megabat gene stable ID' (default: False), 'pvampyrus_homolog_associated_gene_name': 'Megabat gene name' (default: False), 'pvampyrus_homolog_ensembl_peptide': 'Megabat protein or transcript stable ID' (default: False), 'pvampyrus_homolog_chromosome': 'Megabat chromosome/scaffold name' (default: False), 'pvampyrus_homolog_chrom_start': 'Megabat chromosome/scaffold start (bp)' (default: False), 'pvampyrus_homolog_chrom_end': 'Megabat chromosome/scaffold end (bp)' (default: False), 'pvampyrus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'pvampyrus_homolog_subtype': 'Last common ancestor with Megabat' (default: False), 'pvampyrus_homolog_orthology_type': 'Megabat homology type' (default: False), 'pvampyrus_homolog_perc_id': '%id. target Megabat gene identical to query gene' (default: False), 'pvampyrus_homolog_perc_id_r1': '%id. query gene identical to target Megabat gene' (default: False), 'pvampyrus_homolog_goc_score': 'Megabat Gene-order conservation score' (default: False), 'pvampyrus_homolog_wga_coverage': 'Megabat Whole-genome alignment coverage' (default: False), 'pvampyrus_homolog_orthology_confidence': 'Megabat orthology confidence [0 low, 1 high]' (default: False), 'amexicanus_homolog_ensembl_gene': 'Mexican tetra gene stable ID' (default: False), 'amexicanus_homolog_associated_gene_name': 'Mexican tetra gene name' (default: False), 'amexicanus_homolog_ensembl_peptide': 'Mexican tetra protein or transcript stable ID' (default: False), 'amexicanus_homolog_chromosome': 'Mexican tetra chromosome/scaffold name' (default: False), 'amexicanus_homolog_chrom_start': 'Mexican tetra chromosome/scaffold start (bp)' (default: False), 'amexicanus_homolog_chrom_end': 'Mexican tetra chromosome/scaffold end (bp)' (default: False), 'amexicanus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'amexicanus_homolog_subtype': 'Last common ancestor with Mexican tetra' (default: False), 'amexicanus_homolog_orthology_type': 'Mexican tetra homology type' (default: False), 'amexicanus_homolog_perc_id': '%id. target Mexican tetra gene identical to query gene' (default: False), 'amexicanus_homolog_perc_id_r1': '%id. query gene identical to target Mexican tetra gene' (default: False), 'amexicanus_homolog_goc_score': 'Mexican tetra Gene-order conservation score' (default: False), 'amexicanus_homolog_wga_coverage': 'Mexican tetra Whole-genome alignment coverage' (default: False), 'amexicanus_homolog_orthology_confidence': 'Mexican tetra orthology confidence [0 low, 1 high]' (default: False), 'mlucifugus_homolog_ensembl_gene': 'Microbat gene stable ID' (default: False), 'mlucifugus_homolog_associated_gene_name': 'Microbat gene name' (default: False), 'mlucifugus_homolog_ensembl_peptide': 'Microbat protein or transcript stable ID' (default: False), 'mlucifugus_homolog_chromosome': 'Microbat chromosome/scaffold name' (default: False), 'mlucifugus_homolog_chrom_start': 'Microbat chromosome/scaffold start (bp)' (default: False), 'mlucifugus_homolog_chrom_end': 'Microbat chromosome/scaffold end (bp)' (default: False), 'mlucifugus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'mlucifugus_homolog_subtype': 'Last common ancestor with Microbat' (default: False), 'mlucifugus_homolog_orthology_type': 'Microbat homology type' (default: False), 'mlucifugus_homolog_perc_id': '%id. target Microbat gene identical to query gene' (default: False), 'mlucifugus_homolog_perc_id_r1': '%id. query gene identical to target Microbat gene' (default: False), 'mlucifugus_homolog_goc_score': 'Microbat Gene-order conservation score' (default: False), 'mlucifugus_homolog_wga_coverage': 'Microbat Whole-genome alignment coverage' (default: False), 'mlucifugus_homolog_orthology_confidence': 'Microbat orthology confidence [0 low, 1 high]' (default: False), 'acitrinellus_homolog_ensembl_gene': 'Midas cichlid gene stable ID' (default: False), 'acitrinellus_homolog_associated_gene_name': 'Midas cichlid gene name' (default: False), 'acitrinellus_homolog_ensembl_peptide': 'Midas cichlid protein or transcript stable ID' (default: False), 'acitrinellus_homolog_chromosome': 'Midas cichlid chromosome/scaffold name' (default: False), 'acitrinellus_homolog_chrom_start': 'Midas cichlid chromosome/scaffold start (bp)' (default: False), 'acitrinellus_homolog_chrom_end': 'Midas cichlid chromosome/scaffold end (bp)' (default: False), 'acitrinellus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'acitrinellus_homolog_subtype': 'Last common ancestor with Midas cichlid' (default: False), 'acitrinellus_homolog_orthology_type': 'Midas cichlid homology type' (default: False), 'acitrinellus_homolog_perc_id': '%id. target Midas cichlid gene identical to query gene' (default: False), 'acitrinellus_homolog_perc_id_r1': '%id. query gene identical to target Midas cichlid gene' (default: False), 'acitrinellus_homolog_goc_score': 'Midas cichlid Gene-order conservation score' (default: False), 'acitrinellus_homolog_wga_coverage': 'Midas cichlid Whole-genome alignment coverage' (default: False), 'acitrinellus_homolog_orthology_confidence': 'Midas cichlid orthology confidence [0 low, 1 high]' (default: False), 'munguiculatus_homolog_ensembl_gene': 'Mongolian gerbil gene stable ID' (default: False), 'munguiculatus_homolog_associated_gene_name': 'Mongolian gerbil gene name' (default: False), 'munguiculatus_homolog_ensembl_peptide': 'Mongolian gerbil protein or transcript stable ID' (default: False), 'munguiculatus_homolog_chromosome': 'Mongolian gerbil chromosome/scaffold name' (default: False), 'munguiculatus_homolog_chrom_start': 'Mongolian gerbil chromosome/scaffold start (bp)' (default: False), 'munguiculatus_homolog_chrom_end': 'Mongolian gerbil chromosome/scaffold end (bp)' (default: False), 'munguiculatus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'munguiculatus_homolog_subtype': 'Last common ancestor with Mongolian gerbil' (default: False), 'munguiculatus_homolog_orthology_type': 'Mongolian gerbil homology type' (default: False), 'munguiculatus_homolog_perc_id': '%id. target Mongolian gerbil gene identical to query gene' (default: False), 'munguiculatus_homolog_perc_id_r1': '%id. query gene identical to target Mongolian gerbil gene' (default: False), 'munguiculatus_homolog_goc_score': 'Mongolian gerbil Gene-order conservation score' (default: False), 'munguiculatus_homolog_wga_coverage': 'Mongolian gerbil Whole-genome alignment coverage' (default: False), 'munguiculatus_homolog_orthology_confidence': 'Mongolian gerbil orthology confidence [0 low, 1 high]' (default: False), 'mmusculus_homolog_ensembl_gene': 'Mouse gene stable ID' (default: False), 'mmusculus_homolog_associated_gene_name': 'Mouse gene name' (default: False), 'mmusculus_homolog_ensembl_peptide': 'Mouse protein or transcript stable ID' (default: False), 'mmusculus_homolog_chromosome': 'Mouse chromosome/scaffold name' (default: False), 'mmusculus_homolog_chrom_start': 'Mouse chromosome/scaffold start (bp)' (default: False), 'mmusculus_homolog_chrom_end': 'Mouse chromosome/scaffold end (bp)' (default: False), 'mmusculus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'mmusculus_homolog_subtype': 'Last common ancestor with Mouse' (default: False), 'mmusculus_homolog_orthology_type': 'Mouse homology type' (default: False), 'mmusculus_homolog_perc_id': '%id. target Mouse gene identical to query gene' (default: False), 'mmusculus_homolog_perc_id_r1': '%id. query gene identical to target Mouse gene' (default: False), 'mmusculus_homolog_goc_score': 'Mouse Gene-order conservation score' (default: False), 'mmusculus_homolog_wga_coverage': 'Mouse Whole-genome alignment coverage' (default: False), 'mmusculus_homolog_orthology_confidence': 'Mouse orthology confidence [0 low, 1 high]' (default: False), 'mmurinus_homolog_ensembl_gene': 'Mouse Lemur gene stable ID' (default: False), 'mmurinus_homolog_associated_gene_name': 'Mouse Lemur gene name' (default: False), 'mmurinus_homolog_ensembl_peptide': 'Mouse Lemur protein or transcript stable ID' (default: False), 'mmurinus_homolog_chromosome': 'Mouse Lemur chromosome/scaffold name' (default: False), 'mmurinus_homolog_chrom_start': 'Mouse Lemur chromosome/scaffold start (bp)' (default: False), 'mmurinus_homolog_chrom_end': 'Mouse Lemur chromosome/scaffold end (bp)' (default: False), 'mmurinus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'mmurinus_homolog_subtype': 'Last common ancestor with Mouse Lemur' (default: False), 'mmurinus_homolog_orthology_type': 'Mouse Lemur homology type' (default: False), 'mmurinus_homolog_perc_id': '%id. target Mouse Lemur gene identical to query gene' (default: False), 'mmurinus_homolog_perc_id_r1': '%id. query gene identical to target Mouse Lemur gene' (default: False), 'mmurinus_homolog_goc_score': 'Mouse Lemur Gene-order conservation score' (default: False), 'mmurinus_homolog_wga_coverage': 'Mouse Lemur Whole-genome alignment coverage' (default: False), 'mmurinus_homolog_orthology_confidence': 'Mouse Lemur orthology confidence [0 low, 1 high]' (default: False), 'fheteroclitus_homolog_ensembl_gene': 'Mummichog gene stable ID' (default: False), 'fheteroclitus_homolog_associated_gene_name': 'Mummichog gene name' (default: False), 'fheteroclitus_homolog_ensembl_peptide': 'Mummichog protein or transcript stable ID' (default: False), 'fheteroclitus_homolog_chromosome': 'Mummichog chromosome/scaffold name' (default: False), 'fheteroclitus_homolog_chrom_start': 'Mummichog chromosome/scaffold start (bp)' (default: False), 'fheteroclitus_homolog_chrom_end': 'Mummichog chromosome/scaffold end (bp)' (default: False), 'fheteroclitus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'fheteroclitus_homolog_subtype': 'Last common ancestor with Mummichog' (default: False), 'fheteroclitus_homolog_orthology_type': 'Mummichog homology type' (default: False), 'fheteroclitus_homolog_perc_id': '%id. target Mummichog gene identical to query gene' (default: False), 'fheteroclitus_homolog_perc_id_r1': '%id. query gene identical to target Mummichog gene' (default: False), 'fheteroclitus_homolog_goc_score': 'Mummichog Gene-order conservation score' (default: False), 'fheteroclitus_homolog_wga_coverage': 'Mummichog Whole-genome alignment coverage' (default: False), 'fheteroclitus_homolog_orthology_confidence': 'Mummichog orthology confidence [0 low, 1 high]' (default: False), 'hgfemale_homolog_ensembl_gene': 'Naked mole-rat female gene stable ID' (default: False), 'hgfemale_homolog_associated_gene_name': 'Naked mole-rat female gene name' (default: False), 'hgfemale_homolog_ensembl_peptide': 'Naked mole-rat female protein or transcript stable ID' (default: False), 'hgfemale_homolog_chromosome': 'Naked mole-rat female chromosome/scaffold name' (default: False), 'hgfemale_homolog_chrom_start': 'Naked mole-rat female chromosome/scaffold start (bp)' (default: False), 'hgfemale_homolog_chrom_end': 'Naked mole-rat female chromosome/scaffold end (bp)' (default: False), 'hgfemale_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'hgfemale_homolog_subtype': 'Last common ancestor with Naked mole-rat female' (default: False), 'hgfemale_homolog_orthology_type': 'Naked mole-rat female homology type' (default: False), 'hgfemale_homolog_perc_id': '%id. target Naked mole-rat female gene identical to query gene' (default: False), 'hgfemale_homolog_perc_id_r1': '%id. query gene identical to target Naked mole-rat female gene' (default: False), 'hgfemale_homolog_goc_score': 'Naked mole-rat female Gene-order conservation score' (default: False), 'hgfemale_homolog_wga_coverage': 'Naked mole-rat female Whole-genome alignment coverage' (default: False), 'hgfemale_homolog_orthology_confidence': 'Naked mole-rat female orthology confidence [0 low, 1 high]' (default: False), 'hgmale_homolog_ensembl_gene': 'Naked mole-rat male gene stable ID' (default: False), 'hgmale_homolog_associated_gene_name': 'Naked mole-rat male gene name' (default: False), 'hgmale_homolog_ensembl_peptide': 'Naked mole-rat male protein or transcript stable ID' (default: False), 'hgmale_homolog_chromosome': 'Naked mole-rat male chromosome/scaffold name' (default: False), 'hgmale_homolog_chrom_start': 'Naked mole-rat male chromosome/scaffold start (bp)' (default: False), 'hgmale_homolog_chrom_end': 'Naked mole-rat male chromosome/scaffold end (bp)' (default: False), 'hgmale_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'hgmale_homolog_subtype': 'Last common ancestor with Naked mole-rat male' (default: False), 'hgmale_homolog_orthology_type': 'Naked mole-rat male homology type' (default: False), 'hgmale_homolog_perc_id': '%id. target Naked mole-rat male gene identical to query gene' (default: False), 'hgmale_homolog_perc_id_r1': '%id. query gene identical to target Naked mole-rat male gene' (default: False), 'hgmale_homolog_goc_score': 'Naked mole-rat male Gene-order conservation score' (default: False), 'hgmale_homolog_wga_coverage': 'Naked mole-rat male Whole-genome alignment coverage' (default: False), 'hgmale_homolog_orthology_confidence': 'Naked mole-rat male orthology confidence [0 low, 1 high]' (default: False), 'oniloticus_homolog_ensembl_gene': 'Nile tilapia gene stable ID' (default: False), 'oniloticus_homolog_associated_gene_name': 'Nile tilapia gene name' (default: False), 'oniloticus_homolog_ensembl_peptide': 'Nile tilapia protein or transcript stable ID' (default: False), 'oniloticus_homolog_chromosome': 'Nile tilapia chromosome/scaffold name' (default: False), 'oniloticus_homolog_chrom_start': 'Nile tilapia chromosome/scaffold start (bp)' (default: False), 'oniloticus_homolog_chrom_end': 'Nile tilapia chromosome/scaffold end (bp)' (default: False), 'oniloticus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'oniloticus_homolog_subtype': 'Last common ancestor with Nile tilapia' (default: False), 'oniloticus_homolog_orthology_type': 'Nile tilapia homology type' (default: False), 'oniloticus_homolog_perc_id': '%id. target Nile tilapia gene identical to query gene' (default: False), 'oniloticus_homolog_perc_id_r1': '%id. query gene identical to target Nile tilapia gene' (default: False), 'oniloticus_homolog_goc_score': 'Nile tilapia Gene-order conservation score' (default: False), 'oniloticus_homolog_wga_coverage': 'Nile tilapia Whole-genome alignment coverage' (default: False), 'oniloticus_homolog_orthology_confidence': 'Nile tilapia orthology confidence [0 low, 1 high]' (default: False), 'pmbairdii_homolog_ensembl_gene': 'Northern American deer mouse gene stable ID' (default: False), 'pmbairdii_homolog_associated_gene_name': 'Northern American deer mouse gene name' (default: False), 'pmbairdii_homolog_ensembl_peptide': 'Northern American deer mouse protein or transcript stable ID' (default: False), 'pmbairdii_homolog_chromosome': 'Northern American deer mouse chromosome/scaffold name' (default: False), 'pmbairdii_homolog_chrom_start': 'Northern American deer mouse chromosome/scaffold start (bp)' (default: False), 'pmbairdii_homolog_chrom_end': 'Northern American deer mouse chromosome/scaffold end (bp)' (default: False), 'pmbairdii_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'pmbairdii_homolog_subtype': 'Last common ancestor with Northern American deer mouse' (default: False), 'pmbairdii_homolog_orthology_type': 'Northern American deer mouse homology type' (default: False), 'pmbairdii_homolog_perc_id': '%id. target Northern American deer mouse gene identical to query gene' (default: False), 'pmbairdii_homolog_perc_id_r1': '%id. query gene identical to target Northern American deer mouse gene' (default: False), 'pmbairdii_homolog_goc_score': 'Northern American deer mouse Gene-order conservation score' (default: False), 'pmbairdii_homolog_wga_coverage': 'Northern American deer mouse Whole-genome alignment coverage' (default: False), 'pmbairdii_homolog_orthology_confidence': 'Northern American deer mouse orthology confidence [0 low, 1 high]' (default: False), 'elucius_homolog_ensembl_gene': 'Northern pike gene stable ID' (default: False), 'elucius_homolog_associated_gene_name': 'Northern pike gene name' (default: False), 'elucius_homolog_ensembl_peptide': 'Northern pike protein or transcript stable ID' (default: False), 'elucius_homolog_chromosome': 'Northern pike chromosome/scaffold name' (default: False), 'elucius_homolog_chrom_start': 'Northern pike chromosome/scaffold start (bp)' (default: False), 'elucius_homolog_chrom_end': 'Northern pike chromosome/scaffold end (bp)' (default: False), 'elucius_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'elucius_homolog_subtype': 'Last common ancestor with Northern pike' (default: False), 'elucius_homolog_orthology_type': 'Northern pike homology type' (default: False), 'elucius_homolog_perc_id': '%id. target Northern pike gene identical to query gene' (default: False), 'elucius_homolog_perc_id_r1': '%id. query gene identical to target Northern pike gene' (default: False), 'elucius_homolog_goc_score': 'Northern pike Gene-order conservation score' (default: False), 'elucius_homolog_wga_coverage': 'Northern pike Whole-genome alignment coverage' (default: False), 'elucius_homolog_orthology_confidence': 'Northern pike orthology confidence [0 low, 1 high]' (default: False), 'panubis_homolog_ensembl_gene': 'Olive baboon gene stable ID' (default: False), 'panubis_homolog_associated_gene_name': 'Olive baboon gene name' (default: False), 'panubis_homolog_ensembl_peptide': 'Olive baboon protein or transcript stable ID' (default: False), 'panubis_homolog_chromosome': 'Olive baboon chromosome/scaffold name' (default: False), 'panubis_homolog_chrom_start': 'Olive baboon chromosome/scaffold start (bp)' (default: False), 'panubis_homolog_chrom_end': 'Olive baboon chromosome/scaffold end (bp)' (default: False), 'panubis_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'panubis_homolog_subtype': 'Last common ancestor with Olive baboon' (default: False), 'panubis_homolog_orthology_type': 'Olive baboon homology type' (default: False), 'panubis_homolog_perc_id': '%id. target Olive baboon gene identical to query gene' (default: False), 'panubis_homolog_perc_id_r1': '%id. query gene identical to target Olive baboon gene' (default: False), 'panubis_homolog_goc_score': 'Olive baboon Gene-order conservation score' (default: False), 'panubis_homolog_wga_coverage': 'Olive baboon Whole-genome alignment coverage' (default: False), 'panubis_homolog_orthology_confidence': 'Olive baboon orthology confidence [0 low, 1 high]' (default: False), 'mdomestica_homolog_ensembl_gene': 'Opossum gene stable ID' (default: False), 'mdomestica_homolog_associated_gene_name': 'Opossum gene name' (default: False), 'mdomestica_homolog_ensembl_peptide': 'Opossum protein or transcript stable ID' (default: False), 'mdomestica_homolog_chromosome': 'Opossum chromosome/scaffold name' (default: False), 'mdomestica_homolog_chrom_start': 'Opossum chromosome/scaffold start (bp)' (default: False), 'mdomestica_homolog_chrom_end': 'Opossum chromosome/scaffold end (bp)' (default: False), 'mdomestica_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'mdomestica_homolog_subtype': 'Last common ancestor with Opossum' (default: False), 'mdomestica_homolog_orthology_type': 'Opossum homology type' (default: False), 'mdomestica_homolog_perc_id': '%id. target Opossum gene identical to query gene' (default: False), 'mdomestica_homolog_perc_id_r1': '%id. query gene identical to target Opossum gene' (default: False), 'mdomestica_homolog_goc_score': 'Opossum Gene-order conservation score' (default: False), 'mdomestica_homolog_wga_coverage': 'Opossum Whole-genome alignment coverage' (default: False), 'mdomestica_homolog_orthology_confidence': 'Opossum orthology confidence [0 low, 1 high]' (default: False), 'pabelii_homolog_ensembl_gene': 'Orangutan gene stable ID' (default: False), 'pabelii_homolog_associated_gene_name': 'Orangutan gene name' (default: False), 'pabelii_homolog_ensembl_peptide': 'Orangutan protein or transcript stable ID' (default: False), 'pabelii_homolog_chromosome': 'Orangutan chromosome/scaffold name' (default: False), 'pabelii_homolog_chrom_start': 'Orangutan chromosome/scaffold start (bp)' (default: False), 'pabelii_homolog_chrom_end': 'Orangutan chromosome/scaffold end (bp)' (default: False), 'pabelii_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'pabelii_homolog_subtype': 'Last common ancestor with Orangutan' (default: False), 'pabelii_homolog_orthology_type': 'Orangutan homology type' (default: False), 'pabelii_homolog_perc_id': '%id. target Orangutan gene identical to query gene' (default: False), 'pabelii_homolog_perc_id_r1': '%id. query gene identical to target Orangutan gene' (default: False), 'pabelii_homolog_goc_score': 'Orangutan Gene-order conservation score' (default: False), 'pabelii_homolog_wga_coverage': 'Orangutan Whole-genome alignment coverage' (default: False), 'pabelii_homolog_orthology_confidence': 'Orangutan orthology confidence [0 low, 1 high]' (default: False), 'sorbicularis_homolog_ensembl_gene': 'Orbiculate cardinalfish gene stable ID' (default: False), 'sorbicularis_homolog_associated_gene_name': 'Orbiculate cardinalfish gene name' (default: False), 'sorbicularis_homolog_ensembl_peptide': 'Orbiculate cardinalfish protein or transcript stable ID' (default: False), 'sorbicularis_homolog_chromosome': 'Orbiculate cardinalfish chromosome/scaffold name' (default: False), 'sorbicularis_homolog_chrom_start': 'Orbiculate cardinalfish chromosome/scaffold start (bp)' (default: False), 'sorbicularis_homolog_chrom_end': 'Orbiculate cardinalfish chromosome/scaffold end (bp)' (default: False), 'sorbicularis_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'sorbicularis_homolog_subtype': 'Last common ancestor with Orbiculate cardinalfish' (default: False), 'sorbicularis_homolog_orthology_type': 'Orbiculate cardinalfish homology type' (default: False), 'sorbicularis_homolog_perc_id': '%id. target Orbiculate cardinalfish gene identical to query gene' (default: False), 'sorbicularis_homolog_perc_id_r1': '%id. query gene identical to target Orbiculate cardinalfish gene' (default: False), 'sorbicularis_homolog_goc_score': 'Orbiculate cardinalfish Gene-order conservation score' (default: False), 'sorbicularis_homolog_wga_coverage': 'Orbiculate cardinalfish Whole-genome alignment coverage' (default: False), 'sorbicularis_homolog_orthology_confidence': 'Orbiculate cardinalfish orthology confidence [0 low, 1 high]' (default: False), 'ampachon_homolog_ensembl_gene': 'Pachon cavefish gene stable ID' (default: False), 'ampachon_homolog_associated_gene_name': 'Pachon cavefish gene name' (default: False), 'ampachon_homolog_ensembl_peptide': 'Pachon cavefish protein or transcript stable ID' (default: False), 'ampachon_homolog_chromosome': 'Pachon cavefish chromosome/scaffold name' (default: False), 'ampachon_homolog_chrom_start': 'Pachon cavefish chromosome/scaffold start (bp)' (default: False), 'ampachon_homolog_chrom_end': 'Pachon cavefish chromosome/scaffold end (bp)' (default: False), 'ampachon_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'ampachon_homolog_subtype': 'Last common ancestor with Pachon cavefish' (default: False), 'ampachon_homolog_orthology_type': 'Pachon cavefish homology type' (default: False), 'ampachon_homolog_perc_id': '%id. target Pachon cavefish gene identical to query gene' (default: False), 'ampachon_homolog_perc_id_r1': '%id. query gene identical to target Pachon cavefish gene' (default: False), 'ampachon_homolog_goc_score': 'Pachon cavefish Gene-order conservation score' (default: False), 'ampachon_homolog_wga_coverage': 'Pachon cavefish Whole-genome alignment coverage' (default: False), 'ampachon_homolog_orthology_confidence': 'Pachon cavefish orthology confidence [0 low, 1 high]' (default: False), 'cpbellii_homolog_ensembl_gene': 'Painted turtle gene stable ID' (default: False), 'cpbellii_homolog_associated_gene_name': 'Painted turtle gene name' (default: False), 'cpbellii_homolog_ensembl_peptide': 'Painted turtle protein or transcript stable ID' (default: False), 'cpbellii_homolog_chromosome': 'Painted turtle chromosome/scaffold name' (default: False), 'cpbellii_homolog_chrom_start': 'Painted turtle chromosome/scaffold start (bp)' (default: False), 'cpbellii_homolog_chrom_end': 'Painted turtle chromosome/scaffold end (bp)' (default: False), 'cpbellii_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'cpbellii_homolog_subtype': 'Last common ancestor with Painted turtle' (default: False), 'cpbellii_homolog_orthology_type': 'Painted turtle homology type' (default: False), 'cpbellii_homolog_perc_id': '%id. target Painted turtle gene identical to query gene' (default: False), 'cpbellii_homolog_perc_id_r1': '%id. query gene identical to target Painted turtle gene' (default: False), 'cpbellii_homolog_goc_score': 'Painted turtle Gene-order conservation score' (default: False), 'cpbellii_homolog_wga_coverage': 'Painted turtle Whole-genome alignment coverage' (default: False), 'cpbellii_homolog_orthology_confidence': 'Painted turtle orthology confidence [0 low, 1 high]' (default: False), 'amelanoleuca_homolog_ensembl_gene': 'Panda gene stable ID' (default: False), 'amelanoleuca_homolog_associated_gene_name': 'Panda gene name' (default: False), 'amelanoleuca_homolog_ensembl_peptide': 'Panda protein or transcript stable ID' (default: False), 'amelanoleuca_homolog_chromosome': 'Panda chromosome/scaffold name' (default: False), 'amelanoleuca_homolog_chrom_start': 'Panda chromosome/scaffold start (bp)' (default: False), 'amelanoleuca_homolog_chrom_end': 'Panda chromosome/scaffold end (bp)' (default: False), 'amelanoleuca_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'amelanoleuca_homolog_subtype': 'Last common ancestor with Panda' (default: False), 'amelanoleuca_homolog_orthology_type': 'Panda homology type' (default: False), 'amelanoleuca_homolog_perc_id': '%id. target Panda gene identical to query gene' (default: False), 'amelanoleuca_homolog_perc_id_r1': '%id. query gene identical to target Panda gene' (default: False), 'amelanoleuca_homolog_goc_score': 'Panda Gene-order conservation score' (default: False), 'amelanoleuca_homolog_wga_coverage': 'Panda Whole-genome alignment coverage' (default: False), 'amelanoleuca_homolog_orthology_confidence': 'Panda orthology confidence [0 low, 1 high]' (default: False), 'sscrofa_homolog_ensembl_gene': 'Pig gene stable ID' (default: False), 'sscrofa_homolog_associated_gene_name': 'Pig gene name' (default: False), 'sscrofa_homolog_ensembl_peptide': 'Pig protein or transcript stable ID' (default: False), 'sscrofa_homolog_chromosome': 'Pig chromosome/scaffold name' (default: False), 'sscrofa_homolog_chrom_start': 'Pig chromosome/scaffold start (bp)' (default: False), 'sscrofa_homolog_chrom_end': 'Pig chromosome/scaffold end (bp)' (default: False), 'sscrofa_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'sscrofa_homolog_subtype': 'Last common ancestor with Pig' (default: False), 'sscrofa_homolog_orthology_type': 'Pig homology type' (default: False), 'sscrofa_homolog_perc_id': '%id. target Pig gene identical to query gene' (default: False), 'sscrofa_homolog_perc_id_r1': '%id. query gene identical to target Pig gene' (default: False), 'sscrofa_homolog_goc_score': 'Pig Gene-order conservation score' (default: False), 'sscrofa_homolog_wga_coverage': 'Pig Whole-genome alignment coverage' (default: False), 'sscrofa_homolog_orthology_confidence': 'Pig orthology confidence [0 low, 1 high]' (default: False), 'mnemestrina_homolog_ensembl_gene': 'Pig-tailed macaque gene stable ID' (default: False), 'mnemestrina_homolog_associated_gene_name': 'Pig-tailed macaque gene name' (default: False), 'mnemestrina_homolog_ensembl_peptide': 'Pig-tailed macaque protein or transcript stable ID' (default: False), 'mnemestrina_homolog_chromosome': 'Pig-tailed macaque chromosome/scaffold name' (default: False), 'mnemestrina_homolog_chrom_start': 'Pig-tailed macaque chromosome/scaffold start (bp)' (default: False), 'mnemestrina_homolog_chrom_end': 'Pig-tailed macaque chromosome/scaffold end (bp)' (default: False), 'mnemestrina_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'mnemestrina_homolog_subtype': 'Last common ancestor with Pig-tailed macaque' (default: False), 'mnemestrina_homolog_orthology_type': 'Pig-tailed macaque homology type' (default: False), 'mnemestrina_homolog_perc_id': '%id. target Pig-tailed macaque gene identical to query gene' (default: False), 'mnemestrina_homolog_perc_id_r1': '%id. query gene identical to target Pig-tailed macaque gene' (default: False), 'mnemestrina_homolog_goc_score': 'Pig-tailed macaque Gene-order conservation score' (default: False), 'mnemestrina_homolog_wga_coverage': 'Pig-tailed macaque Whole-genome alignment coverage' (default: False), 'mnemestrina_homolog_orthology_confidence': 'Pig-tailed macaque orthology confidence [0 low, 1 high]' (default: False), 'oprinceps_homolog_ensembl_gene': 'Pika gene stable ID' (default: False), 'oprinceps_homolog_associated_gene_name': 'Pika gene name' (default: False), 'oprinceps_homolog_ensembl_peptide': 'Pika protein or transcript stable ID' (default: False), 'oprinceps_homolog_chromosome': 'Pika chromosome/scaffold name' (default: False), 'oprinceps_homolog_chrom_start': 'Pika chromosome/scaffold start (bp)' (default: False), 'oprinceps_homolog_chrom_end': 'Pika chromosome/scaffold end (bp)' (default: False), 'oprinceps_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'oprinceps_homolog_subtype': 'Last common ancestor with Pika' (default: False), 'oprinceps_homolog_orthology_type': 'Pika homology type' (default: False), 'oprinceps_homolog_perc_id': '%id. target Pika gene identical to query gene' (default: False), 'oprinceps_homolog_perc_id_r1': '%id. query gene identical to target Pika gene' (default: False), 'oprinceps_homolog_goc_score': 'Pika Gene-order conservation score' (default: False), 'oprinceps_homolog_wga_coverage': 'Pika Whole-genome alignment coverage' (default: False), 'oprinceps_homolog_orthology_confidence': 'Pika orthology confidence [0 low, 1 high]' (default: False), 'mmurdjan_homolog_ensembl_gene': 'Pinecone soldierfish gene stable ID' (default: False), 'mmurdjan_homolog_associated_gene_name': 'Pinecone soldierfish gene name' (default: False), 'mmurdjan_homolog_ensembl_peptide': 'Pinecone soldierfish protein or transcript stable ID' (default: False), 'mmurdjan_homolog_chromosome': 'Pinecone soldierfish chromosome/scaffold name' (default: False), 'mmurdjan_homolog_chrom_start': 'Pinecone soldierfish chromosome/scaffold start (bp)' (default: False), 'mmurdjan_homolog_chrom_end': 'Pinecone soldierfish chromosome/scaffold end (bp)' (default: False), 'mmurdjan_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'mmurdjan_homolog_subtype': 'Last common ancestor with Pinecone soldierfish' (default: False), 'mmurdjan_homolog_orthology_type': 'Pinecone soldierfish homology type' (default: False), 'mmurdjan_homolog_perc_id': '%id. target Pinecone soldierfish gene identical to query gene' (default: False), 'mmurdjan_homolog_perc_id_r1': '%id. query gene identical to target Pinecone soldierfish gene' (default: False), 'mmurdjan_homolog_goc_score': 'Pinecone soldierfish Gene-order conservation score' (default: False), 'mmurdjan_homolog_wga_coverage': 'Pinecone soldierfish Whole-genome alignment coverage' (default: False), 'mmurdjan_homolog_orthology_confidence': 'Pinecone soldierfish orthology confidence [0 low, 1 high]' (default: False), 'oanatinus_homolog_ensembl_gene': 'Platypus gene stable ID' (default: False), 'oanatinus_homolog_associated_gene_name': 'Platypus gene name' (default: False), 'oanatinus_homolog_ensembl_peptide': 'Platypus protein or transcript stable ID' (default: False), 'oanatinus_homolog_chromosome': 'Platypus chromosome/scaffold name' (default: False), 'oanatinus_homolog_chrom_start': 'Platypus chromosome/scaffold start (bp)' (default: False), 'oanatinus_homolog_chrom_end': 'Platypus chromosome/scaffold end (bp)' (default: False), 'oanatinus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'oanatinus_homolog_subtype': 'Last common ancestor with Platypus' (default: False), 'oanatinus_homolog_orthology_type': 'Platypus homology type' (default: False), 'oanatinus_homolog_perc_id': '%id. target Platypus gene identical to query gene' (default: False), 'oanatinus_homolog_perc_id_r1': '%id. query gene identical to target Platypus gene' (default: False), 'oanatinus_homolog_goc_score': 'Platypus Gene-order conservation score' (default: False), 'oanatinus_homolog_wga_coverage': 'Platypus Whole-genome alignment coverage' (default: False), 'oanatinus_homolog_orthology_confidence': 'Platypus orthology confidence [0 low, 1 high]' (default: False), 'umaritimus_homolog_ensembl_gene': 'Polar bear gene stable ID' (default: False), 'umaritimus_homolog_associated_gene_name': 'Polar bear gene name' (default: False), 'umaritimus_homolog_ensembl_peptide': 'Polar bear protein or transcript stable ID' (default: False), 'umaritimus_homolog_chromosome': 'Polar bear chromosome/scaffold name' (default: False), 'umaritimus_homolog_chrom_start': 'Polar bear chromosome/scaffold start (bp)' (default: False), 'umaritimus_homolog_chrom_end': 'Polar bear chromosome/scaffold end (bp)' (default: False), 'umaritimus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'umaritimus_homolog_subtype': 'Last common ancestor with Polar bear' (default: False), 'umaritimus_homolog_orthology_type': 'Polar bear homology type' (default: False), 'umaritimus_homolog_perc_id': '%id. target Polar bear gene identical to query gene' (default: False), 'umaritimus_homolog_perc_id_r1': '%id. query gene identical to target Polar bear gene' (default: False), 'umaritimus_homolog_goc_score': 'Polar bear Gene-order conservation score' (default: False), 'umaritimus_homolog_wga_coverage': 'Polar bear Whole-genome alignment coverage' (default: False), 'umaritimus_homolog_orthology_confidence': 'Polar bear orthology confidence [0 low, 1 high]' (default: False), 'mochrogaster_homolog_ensembl_gene': 'Prairie vole gene stable ID' (default: False), 'mochrogaster_homolog_associated_gene_name': 'Prairie vole gene name' (default: False), 'mochrogaster_homolog_ensembl_peptide': 'Prairie vole protein or transcript stable ID' (default: False), 'mochrogaster_homolog_chromosome': 'Prairie vole chromosome/scaffold name' (default: False), 'mochrogaster_homolog_chrom_start': 'Prairie vole chromosome/scaffold start (bp)' (default: False), 'mochrogaster_homolog_chrom_end': 'Prairie vole chromosome/scaffold end (bp)' (default: False), 'mochrogaster_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'mochrogaster_homolog_subtype': 'Last common ancestor with Prairie vole' (default: False), 'mochrogaster_homolog_orthology_type': 'Prairie vole homology type' (default: False), 'mochrogaster_homolog_perc_id': '%id. target Prairie vole gene identical to query gene' (default: False), 'mochrogaster_homolog_perc_id_r1': '%id. query gene identical to target Prairie vole gene' (default: False), 'mochrogaster_homolog_goc_score': 'Prairie vole Gene-order conservation score' (default: False), 'mochrogaster_homolog_wga_coverage': 'Prairie vole Whole-genome alignment coverage' (default: False), 'mochrogaster_homolog_orthology_confidence': 'Prairie vole orthology confidence [0 low, 1 high]' (default: False), 'ocuniculus_homolog_ensembl_gene': 'Rabbit gene stable ID' (default: False), 'ocuniculus_homolog_associated_gene_name': 'Rabbit gene name' (default: False), 'ocuniculus_homolog_ensembl_peptide': 'Rabbit protein or transcript stable ID' (default: False), 'ocuniculus_homolog_chromosome': 'Rabbit chromosome/scaffold name' (default: False), 'ocuniculus_homolog_chrom_start': 'Rabbit chromosome/scaffold start (bp)' (default: False), 'ocuniculus_homolog_chrom_end': 'Rabbit chromosome/scaffold end (bp)' (default: False), 'ocuniculus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'ocuniculus_homolog_subtype': 'Last common ancestor with Rabbit' (default: False), 'ocuniculus_homolog_orthology_type': 'Rabbit homology type' (default: False), 'ocuniculus_homolog_perc_id': '%id. target Rabbit gene identical to query gene' (default: False), 'ocuniculus_homolog_perc_id_r1': '%id. query gene identical to target Rabbit gene' (default: False), 'ocuniculus_homolog_goc_score': 'Rabbit Gene-order conservation score' (default: False), 'ocuniculus_homolog_wga_coverage': 'Rabbit Whole-genome alignment coverage' (default: False), 'ocuniculus_homolog_orthology_confidence': 'Rabbit orthology confidence [0 low, 1 high]' (default: False), 'omykiss_homolog_ensembl_gene': 'Rainbow trout gene stable ID' (default: False), 'omykiss_homolog_associated_gene_name': 'Rainbow trout gene name' (default: False), 'omykiss_homolog_ensembl_peptide': 'Rainbow trout protein or transcript stable ID' (default: False), 'omykiss_homolog_chromosome': 'Rainbow trout chromosome/scaffold name' (default: False), 'omykiss_homolog_chrom_start': 'Rainbow trout chromosome/scaffold start (bp)' (default: False), 'omykiss_homolog_chrom_end': 'Rainbow trout chromosome/scaffold end (bp)' (default: False), 'omykiss_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'omykiss_homolog_subtype': 'Last common ancestor with Rainbow trout' (default: False), 'omykiss_homolog_orthology_type': 'Rainbow trout homology type' (default: False), 'omykiss_homolog_perc_id': '%id. target Rainbow trout gene identical to query gene' (default: False), 'omykiss_homolog_perc_id_r1': '%id. query gene identical to target Rainbow trout gene' (default: False), 'omykiss_homolog_goc_score': 'Rainbow trout Gene-order conservation score' (default: False), 'omykiss_homolog_wga_coverage': 'Rainbow trout Whole-genome alignment coverage' (default: False), 'omykiss_homolog_orthology_confidence': 'Rainbow trout orthology confidence [0 low, 1 high]' (default: False), 'rnorvegicus_homolog_ensembl_gene': 'Rat gene stable ID' (default: False), 'rnorvegicus_homolog_associated_gene_name': 'Rat gene name' (default: False), 'rnorvegicus_homolog_ensembl_peptide': 'Rat protein or transcript stable ID' (default: False), 'rnorvegicus_homolog_chromosome': 'Rat chromosome/scaffold name' (default: False), 'rnorvegicus_homolog_chrom_start': 'Rat chromosome/scaffold start (bp)' (default: False), 'rnorvegicus_homolog_chrom_end': 'Rat chromosome/scaffold end (bp)' (default: False), 'rnorvegicus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'rnorvegicus_homolog_subtype': 'Last common ancestor with Rat' (default: False), 'rnorvegicus_homolog_orthology_type': 'Rat homology type' (default: False), 'rnorvegicus_homolog_perc_id': '%id. target Rat gene identical to query gene' (default: False), 'rnorvegicus_homolog_perc_id_r1': '%id. query gene identical to target Rat gene' (default: False), 'rnorvegicus_homolog_goc_score': 'Rat Gene-order conservation score' (default: False), 'rnorvegicus_homolog_wga_coverage': 'Rat Whole-genome alignment coverage' (default: False), 'rnorvegicus_homolog_orthology_confidence': 'Rat orthology confidence [0 low, 1 high]' (default: False), 'vvulpes_homolog_ensembl_gene': 'Red fox gene stable ID' (default: False), 'vvulpes_homolog_associated_gene_name': 'Red fox gene name' (default: False), 'vvulpes_homolog_ensembl_peptide': 'Red fox protein or transcript stable ID' (default: False), 'vvulpes_homolog_chromosome': 'Red fox chromosome/scaffold name' (default: False), 'vvulpes_homolog_chrom_start': 'Red fox chromosome/scaffold start (bp)' (default: False), 'vvulpes_homolog_chrom_end': 'Red fox chromosome/scaffold end (bp)' (default: False), 'vvulpes_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'vvulpes_homolog_subtype': 'Last common ancestor with Red fox' (default: False), 'vvulpes_homolog_orthology_type': 'Red fox homology type' (default: False), 'vvulpes_homolog_perc_id': '%id. target Red fox gene identical to query gene' (default: False), 'vvulpes_homolog_perc_id_r1': '%id. query gene identical to target Red fox gene' (default: False), 'vvulpes_homolog_goc_score': 'Red fox Gene-order conservation score' (default: False), 'vvulpes_homolog_wga_coverage': 'Red fox Whole-genome alignment coverage' (default: False), 'vvulpes_homolog_orthology_confidence': 'Red fox orthology confidence [0 low, 1 high]' (default: False), 'pnattereri_homolog_ensembl_gene': 'Red-bellied piranha gene stable ID' (default: False), 'pnattereri_homolog_associated_gene_name': 'Red-bellied piranha gene name' (default: False), 'pnattereri_homolog_ensembl_peptide': 'Red-bellied piranha protein or transcript stable ID' (default: False), 'pnattereri_homolog_chromosome': 'Red-bellied piranha chromosome/scaffold name' (default: False), 'pnattereri_homolog_chrom_start': 'Red-bellied piranha chromosome/scaffold start (bp)' (default: False), 'pnattereri_homolog_chrom_end': 'Red-bellied piranha chromosome/scaffold end (bp)' (default: False), 'pnattereri_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'pnattereri_homolog_subtype': 'Last common ancestor with Red-bellied piranha' (default: False), 'pnattereri_homolog_orthology_type': 'Red-bellied piranha homology type' (default: False), 'pnattereri_homolog_perc_id': '%id. target Red-bellied piranha gene identical to query gene' (default: False), 'pnattereri_homolog_perc_id_r1': '%id. query gene identical to target Red-bellied piranha gene' (default: False), 'pnattereri_homolog_goc_score': 'Red-bellied piranha Gene-order conservation score' (default: False), 'pnattereri_homolog_wga_coverage': 'Red-bellied piranha Whole-genome alignment coverage' (default: False), 'pnattereri_homolog_orthology_confidence': 'Red-bellied piranha orthology confidence [0 low, 1 high]' (default: False), 'ecalabaricus_homolog_ensembl_gene': 'Reedfish gene stable ID' (default: False), 'ecalabaricus_homolog_associated_gene_name': 'Reedfish gene name' (default: False), 'ecalabaricus_homolog_ensembl_peptide': 'Reedfish protein or transcript stable ID' (default: False), 'ecalabaricus_homolog_chromosome': 'Reedfish chromosome/scaffold name' (default: False), 'ecalabaricus_homolog_chrom_start': 'Reedfish chromosome/scaffold start (bp)' (default: False), 'ecalabaricus_homolog_chrom_end': 'Reedfish chromosome/scaffold end (bp)' (default: False), 'ecalabaricus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'ecalabaricus_homolog_subtype': 'Last common ancestor with Reedfish' (default: False), 'ecalabaricus_homolog_orthology_type': 'Reedfish homology type' (default: False), 'ecalabaricus_homolog_perc_id': '%id. target Reedfish gene identical to query gene' (default: False), 'ecalabaricus_homolog_perc_id_r1': '%id. query gene identical to target Reedfish gene' (default: False), 'ecalabaricus_homolog_goc_score': 'Reedfish Gene-order conservation score' (default: False), 'ecalabaricus_homolog_wga_coverage': 'Reedfish Whole-genome alignment coverage' (default: False), 'ecalabaricus_homolog_orthology_confidence': 'Reedfish orthology confidence [0 low, 1 high]' (default: False), 'mcaroli_homolog_ensembl_gene': 'Ryukyu mouse gene stable ID' (default: False), 'mcaroli_homolog_associated_gene_name': 'Ryukyu mouse gene name' (default: False), 'mcaroli_homolog_ensembl_peptide': 'Ryukyu mouse protein or transcript stable ID' (default: False), 'mcaroli_homolog_chromosome': 'Ryukyu mouse chromosome/scaffold name' (default: False), 'mcaroli_homolog_chrom_start': 'Ryukyu mouse chromosome/scaffold start (bp)' (default: False), 'mcaroli_homolog_chrom_end': 'Ryukyu mouse chromosome/scaffold end (bp)' (default: False), 'mcaroli_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'mcaroli_homolog_subtype': 'Last common ancestor with Ryukyu mouse' (default: False), 'mcaroli_homolog_orthology_type': 'Ryukyu mouse homology type' (default: False), 'mcaroli_homolog_perc_id': '%id. target Ryukyu mouse gene identical to query gene' (default: False), 'mcaroli_homolog_perc_id_r1': '%id. query gene identical to target Ryukyu mouse gene' (default: False), 'mcaroli_homolog_goc_score': 'Ryukyu mouse Gene-order conservation score' (default: False), 'mcaroli_homolog_wga_coverage': 'Ryukyu mouse Whole-genome alignment coverage' (default: False), 'mcaroli_homolog_orthology_confidence': 'Ryukyu mouse orthology confidence [0 low, 1 high]' (default: False), 'scerevisiae_homolog_ensembl_gene': 'Saccharomyces cerevisiae gene stable ID' (default: False), 'scerevisiae_homolog_associated_gene_name': 'Saccharomyces cerevisiae gene name' (default: False), 'scerevisiae_homolog_ensembl_peptide': 'Saccharomyces cerevisiae protein or transcript stable ID' (default: False), 'scerevisiae_homolog_chromosome': 'Saccharomyces cerevisiae chromosome/scaffold name' (default: False), 'scerevisiae_homolog_chrom_start': 'Saccharomyces cerevisiae chromosome/scaffold start (bp)' (default: False), 'scerevisiae_homolog_chrom_end': 'Saccharomyces cerevisiae chromosome/scaffold end (bp)' (default: False), 'scerevisiae_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'scerevisiae_homolog_subtype': 'Last common ancestor with Saccharomyces cerevisiae' (default: False), 'scerevisiae_homolog_orthology_type': 'Saccharomyces cerevisiae homology type' (default: False), 'scerevisiae_homolog_perc_id': '%id. target Saccharomyces cerevisiae gene identical to query gene' (default: False), 'scerevisiae_homolog_perc_id_r1': '%id. query gene identical to target Saccharomyces cerevisiae gene' (default: False), 'scerevisiae_homolog_orthology_confidence': 'Saccharomyces cerevisiae orthology confidence [0 low, 1 high]' (default: False), 'platipinna_homolog_ensembl_gene': 'Sailfin molly gene stable ID' (default: False), 'platipinna_homolog_associated_gene_name': 'Sailfin molly gene name' (default: False), 'platipinna_homolog_ensembl_peptide': 'Sailfin molly protein or transcript stable ID' (default: False), 'platipinna_homolog_chromosome': 'Sailfin molly chromosome/scaffold name' (default: False), 'platipinna_homolog_chrom_start': 'Sailfin molly chromosome/scaffold start (bp)' (default: False), 'platipinna_homolog_chrom_end': 'Sailfin molly chromosome/scaffold end (bp)' (default: False), 'platipinna_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'platipinna_homolog_subtype': 'Last common ancestor with Sailfin molly' (default: False), 'platipinna_homolog_orthology_type': 'Sailfin molly homology type' (default: False), 'platipinna_homolog_perc_id': '%id. target Sailfin molly gene identical to query gene' (default: False), 'platipinna_homolog_perc_id_r1': '%id. query gene identical to target Sailfin molly gene' (default: False), 'platipinna_homolog_goc_score': 'Sailfin molly Gene-order conservation score' (default: False), 'platipinna_homolog_wga_coverage': 'Sailfin molly Whole-genome alignment coverage' (default: False), 'platipinna_homolog_orthology_confidence': 'Sailfin molly orthology confidence [0 low, 1 high]' (default: False), 'oaries_homolog_ensembl_gene': 'Sheep (texel) gene stable ID' (default: False), 'oaries_homolog_associated_gene_name': 'Sheep (texel) gene name' (default: False), 'oaries_homolog_ensembl_peptide': 'Sheep (texel) protein or transcript stable ID' (default: False), 'oaries_homolog_chromosome': 'Sheep (texel) chromosome/scaffold name' (default: False), 'oaries_homolog_chrom_start': 'Sheep (texel) chromosome/scaffold start (bp)' (default: False), 'oaries_homolog_chrom_end': 'Sheep (texel) chromosome/scaffold end (bp)' (default: False), 'oaries_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'oaries_homolog_subtype': 'Last common ancestor with Sheep (texel)' (default: False), 'oaries_homolog_orthology_type': 'Sheep (texel) homology type' (default: False), 'oaries_homolog_perc_id': '%id. target Sheep (texel) gene identical to query gene' (default: False), 'oaries_homolog_perc_id_r1': '%id. query gene identical to target Sheep (texel) gene' (default: False), 'oaries_homolog_goc_score': 'Sheep (texel) Gene-order conservation score' (default: False), 'oaries_homolog_wga_coverage': 'Sheep (texel) Whole-genome alignment coverage' (default: False), 'oaries_homolog_orthology_confidence': 'Sheep (texel) orthology confidence [0 low, 1 high]' (default: False), 'cvariegatus_homolog_ensembl_gene': 'Sheepshead minnow gene stable ID' (default: False), 'cvariegatus_homolog_associated_gene_name': 'Sheepshead minnow gene name' (default: False), 'cvariegatus_homolog_ensembl_peptide': 'Sheepshead minnow protein or transcript stable ID' (default: False), 'cvariegatus_homolog_chromosome': 'Sheepshead minnow chromosome/scaffold name' (default: False), 'cvariegatus_homolog_chrom_start': 'Sheepshead minnow chromosome/scaffold start (bp)' (default: False), 'cvariegatus_homolog_chrom_end': 'Sheepshead minnow chromosome/scaffold end (bp)' (default: False), 'cvariegatus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'cvariegatus_homolog_subtype': 'Last common ancestor with Sheepshead minnow' (default: False), 'cvariegatus_homolog_orthology_type': 'Sheepshead minnow homology type' (default: False), 'cvariegatus_homolog_perc_id': '%id. target Sheepshead minnow gene identical to query gene' (default: False), 'cvariegatus_homolog_perc_id_r1': '%id. query gene identical to target Sheepshead minnow gene' (default: False), 'cvariegatus_homolog_goc_score': 'Sheepshead minnow Gene-order conservation score' (default: False), 'cvariegatus_homolog_wga_coverage': 'Sheepshead minnow Whole-genome alignment coverage' (default: False), 'cvariegatus_homolog_orthology_confidence': 'Sheepshead minnow orthology confidence [0 low, 1 high]' (default: False), 'pmexicana_homolog_ensembl_gene': 'Shortfin molly gene stable ID' (default: False), 'pmexicana_homolog_associated_gene_name': 'Shortfin molly gene name' (default: False), 'pmexicana_homolog_ensembl_peptide': 'Shortfin molly protein or transcript stable ID' (default: False), 'pmexicana_homolog_chromosome': 'Shortfin molly chromosome/scaffold name' (default: False), 'pmexicana_homolog_chrom_start': 'Shortfin molly chromosome/scaffold start (bp)' (default: False), 'pmexicana_homolog_chrom_end': 'Shortfin molly chromosome/scaffold end (bp)' (default: False), 'pmexicana_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'pmexicana_homolog_subtype': 'Last common ancestor with Shortfin molly' (default: False), 'pmexicana_homolog_orthology_type': 'Shortfin molly homology type' (default: False), 'pmexicana_homolog_perc_id': '%id. target Shortfin molly gene identical to query gene' (default: False), 'pmexicana_homolog_perc_id_r1': '%id. query gene identical to target Shortfin molly gene' (default: False), 'pmexicana_homolog_goc_score': 'Shortfin molly Gene-order conservation score' (default: False), 'pmexicana_homolog_wga_coverage': 'Shortfin molly Whole-genome alignment coverage' (default: False), 'pmexicana_homolog_orthology_confidence': 'Shortfin molly orthology confidence [0 low, 1 high]' (default: False), 'saraneus_homolog_ensembl_gene': 'Shrew gene stable ID' (default: False), 'saraneus_homolog_associated_gene_name': 'Shrew gene name' (default: False), 'saraneus_homolog_ensembl_peptide': 'Shrew protein or transcript stable ID' (default: False), 'saraneus_homolog_chromosome': 'Shrew chromosome/scaffold name' (default: False), 'saraneus_homolog_chrom_start': 'Shrew chromosome/scaffold start (bp)' (default: False), 'saraneus_homolog_chrom_end': 'Shrew chromosome/scaffold end (bp)' (default: False), 'saraneus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'saraneus_homolog_subtype': 'Last common ancestor with Shrew' (default: False), 'saraneus_homolog_orthology_type': 'Shrew homology type' (default: False), 'saraneus_homolog_perc_id': '%id. target Shrew gene identical to query gene' (default: False), 'saraneus_homolog_perc_id_r1': '%id. query gene identical to target Shrew gene' (default: False), 'saraneus_homolog_goc_score': 'Shrew Gene-order conservation score' (default: False), 'saraneus_homolog_wga_coverage': 'Shrew Whole-genome alignment coverage' (default: False), 'saraneus_homolog_orthology_confidence': 'Shrew orthology confidence [0 low, 1 high]' (default: False), 'mpahari_homolog_ensembl_gene': 'Shrew mouse gene stable ID' (default: False), 'mpahari_homolog_associated_gene_name': 'Shrew mouse gene name' (default: False), 'mpahari_homolog_ensembl_peptide': 'Shrew mouse protein or transcript stable ID' (default: False), 'mpahari_homolog_chromosome': 'Shrew mouse chromosome/scaffold name' (default: False), 'mpahari_homolog_chrom_start': 'Shrew mouse chromosome/scaffold start (bp)' (default: False), 'mpahari_homolog_chrom_end': 'Shrew mouse chromosome/scaffold end (bp)' (default: False), 'mpahari_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'mpahari_homolog_subtype': 'Last common ancestor with Shrew mouse' (default: False), 'mpahari_homolog_orthology_type': 'Shrew mouse homology type' (default: False), 'mpahari_homolog_perc_id': '%id. target Shrew mouse gene identical to query gene' (default: False), 'mpahari_homolog_perc_id_r1': '%id. query gene identical to target Shrew mouse gene' (default: False), 'mpahari_homolog_goc_score': 'Shrew mouse Gene-order conservation score' (default: False), 'mpahari_homolog_wga_coverage': 'Shrew mouse Whole-genome alignment coverage' (default: False), 'mpahari_homolog_orthology_confidence': 'Shrew mouse orthology confidence [0 low, 1 high]' (default: False), 'bsplendens_homolog_ensembl_gene': 'Siamese fighting fish gene stable ID' (default: False), 'bsplendens_homolog_associated_gene_name': 'Siamese fighting fish gene name' (default: False), 'bsplendens_homolog_ensembl_peptide': 'Siamese fighting fish protein or transcript stable ID' (default: False), 'bsplendens_homolog_chromosome': 'Siamese fighting fish chromosome/scaffold name' (default: False), 'bsplendens_homolog_chrom_start': 'Siamese fighting fish chromosome/scaffold start (bp)' (default: False), 'bsplendens_homolog_chrom_end': 'Siamese fighting fish chromosome/scaffold end (bp)' (default: False), 'bsplendens_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'bsplendens_homolog_subtype': 'Last common ancestor with Siamese fighting fish' (default: False), 'bsplendens_homolog_orthology_type': 'Siamese fighting fish homology type' (default: False), 'bsplendens_homolog_perc_id': '%id. target Siamese fighting fish gene identical to query gene' (default: False), 'bsplendens_homolog_perc_id_r1': '%id. query gene identical to target Siamese fighting fish gene' (default: False), 'bsplendens_homolog_goc_score': 'Siamese fighting fish Gene-order conservation score' (default: False), 'bsplendens_homolog_wga_coverage': 'Siamese fighting fish Whole-genome alignment coverage' (default: False), 'bsplendens_homolog_orthology_confidence': 'Siamese fighting fish orthology confidence [0 low, 1 high]' (default: False), 'choffmanni_homolog_ensembl_gene': 'Sloth gene stable ID' (default: False), 'choffmanni_homolog_associated_gene_name': 'Sloth gene name' (default: False), 'choffmanni_homolog_ensembl_peptide': 'Sloth protein or transcript stable ID' (default: False), 'choffmanni_homolog_chromosome': 'Sloth chromosome/scaffold name' (default: False), 'choffmanni_homolog_chrom_start': 'Sloth chromosome/scaffold start (bp)' (default: False), 'choffmanni_homolog_chrom_end': 'Sloth chromosome/scaffold end (bp)' (default: False), 'choffmanni_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'choffmanni_homolog_subtype': 'Last common ancestor with Sloth' (default: False), 'choffmanni_homolog_orthology_type': 'Sloth homology type' (default: False), 'choffmanni_homolog_perc_id': '%id. target Sloth gene identical to query gene' (default: False), 'choffmanni_homolog_perc_id_r1': '%id. query gene identical to target Sloth gene' (default: False), 'choffmanni_homolog_goc_score': 'Sloth Gene-order conservation score' (default: False), 'choffmanni_homolog_wga_coverage': 'Sloth Whole-genome alignment coverage' (default: False), 'choffmanni_homolog_orthology_confidence': 'Sloth orthology confidence [0 low, 1 high]' (default: False), 'catys_homolog_ensembl_gene': 'Sooty mangabey gene stable ID' (default: False), 'catys_homolog_associated_gene_name': 'Sooty mangabey gene name' (default: False), 'catys_homolog_ensembl_peptide': 'Sooty mangabey protein or transcript stable ID' (default: False), 'catys_homolog_chromosome': 'Sooty mangabey chromosome/scaffold name' (default: False), 'catys_homolog_chrom_start': 'Sooty mangabey chromosome/scaffold start (bp)' (default: False), 'catys_homolog_chrom_end': 'Sooty mangabey chromosome/scaffold end (bp)' (default: False), 'catys_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'catys_homolog_subtype': 'Last common ancestor with Sooty mangabey' (default: False), 'catys_homolog_orthology_type': 'Sooty mangabey homology type' (default: False), 'catys_homolog_perc_id': '%id. target Sooty mangabey gene identical to query gene' (default: False), 'catys_homolog_perc_id_r1': '%id. query gene identical to target Sooty mangabey gene' (default: False), 'catys_homolog_goc_score': 'Sooty mangabey Gene-order conservation score' (default: False), 'catys_homolog_wga_coverage': 'Sooty mangabey Whole-genome alignment coverage' (default: False), 'catys_homolog_orthology_confidence': 'Sooty mangabey orthology confidence [0 low, 1 high]' (default: False), 'loculatus_homolog_ensembl_gene': 'Spotted gar gene stable ID' (default: False), 'loculatus_homolog_associated_gene_name': 'Spotted gar gene name' (default: False), 'loculatus_homolog_ensembl_peptide': 'Spotted gar protein or transcript stable ID' (default: False), 'loculatus_homolog_chromosome': 'Spotted gar chromosome/scaffold name' (default: False), 'loculatus_homolog_chrom_start': 'Spotted gar chromosome/scaffold start (bp)' (default: False), 'loculatus_homolog_chrom_end': 'Spotted gar chromosome/scaffold end (bp)' (default: False), 'loculatus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'loculatus_homolog_subtype': 'Last common ancestor with Spotted gar' (default: False), 'loculatus_homolog_orthology_type': 'Spotted gar homology type' (default: False), 'loculatus_homolog_perc_id': '%id. target Spotted gar gene identical to query gene' (default: False), 'loculatus_homolog_perc_id_r1': '%id. query gene identical to target Spotted gar gene' (default: False), 'loculatus_homolog_goc_score': 'Spotted gar Gene-order conservation score' (default: False), 'loculatus_homolog_wga_coverage': 'Spotted gar Whole-genome alignment coverage' (default: False), 'loculatus_homolog_orthology_confidence': 'Spotted gar orthology confidence [0 low, 1 high]' (default: False), 'itridecemlineatus_homolog_ensembl_gene': 'Squirrel gene stable ID' (default: False), 'itridecemlineatus_homolog_associated_gene_name': 'Squirrel gene name' (default: False), 'itridecemlineatus_homolog_ensembl_peptide': 'Squirrel protein or transcript stable ID' (default: False), 'itridecemlineatus_homolog_chromosome': 'Squirrel chromosome/scaffold name' (default: False), 'itridecemlineatus_homolog_chrom_start': 'Squirrel chromosome/scaffold start (bp)' (default: False), 'itridecemlineatus_homolog_chrom_end': 'Squirrel chromosome/scaffold end (bp)' (default: False), 'itridecemlineatus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'itridecemlineatus_homolog_subtype': 'Last common ancestor with Squirrel' (default: False), 'itridecemlineatus_homolog_orthology_type': 'Squirrel homology type' (default: False), 'itridecemlineatus_homolog_perc_id': '%id. target Squirrel gene identical to query gene' (default: False), 'itridecemlineatus_homolog_perc_id_r1': '%id. query gene identical to target Squirrel gene' (default: False), 'itridecemlineatus_homolog_goc_score': 'Squirrel Gene-order conservation score' (default: False), 'itridecemlineatus_homolog_wga_coverage': 'Squirrel Whole-genome alignment coverage' (default: False), 'itridecemlineatus_homolog_orthology_confidence': 'Squirrel orthology confidence [0 low, 1 high]' (default: False), 'mspicilegus_homolog_ensembl_gene': 'Steppe mouse gene stable ID' (default: False), 'mspicilegus_homolog_associated_gene_name': 'Steppe mouse gene name' (default: False), 'mspicilegus_homolog_ensembl_peptide': 'Steppe mouse protein or transcript stable ID' (default: False), 'mspicilegus_homolog_chromosome': 'Steppe mouse chromosome/scaffold name' (default: False), 'mspicilegus_homolog_chrom_start': 'Steppe mouse chromosome/scaffold start (bp)' (default: False), 'mspicilegus_homolog_chrom_end': 'Steppe mouse chromosome/scaffold end (bp)' (default: False), 'mspicilegus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'mspicilegus_homolog_subtype': 'Last common ancestor with Steppe mouse' (default: False), 'mspicilegus_homolog_orthology_type': 'Steppe mouse homology type' (default: False), 'mspicilegus_homolog_perc_id': '%id. target Steppe mouse gene identical to query gene' (default: False), 'mspicilegus_homolog_perc_id_r1': '%id. query gene identical to target Steppe mouse gene' (default: False), 'mspicilegus_homolog_goc_score': 'Steppe mouse Gene-order conservation score' (default: False), 'mspicilegus_homolog_wga_coverage': 'Steppe mouse Whole-genome alignment coverage' (default: False), 'mspicilegus_homolog_orthology_confidence': 'Steppe mouse orthology confidence [0 low, 1 high]' (default: False), 'gaculeatus_homolog_ensembl_gene': 'Stickleback gene stable ID' (default: False), 'gaculeatus_homolog_associated_gene_name': 'Stickleback gene name' (default: False), 'gaculeatus_homolog_ensembl_peptide': 'Stickleback protein or transcript stable ID' (default: False), 'gaculeatus_homolog_chromosome': 'Stickleback chromosome/scaffold name' (default: False), 'gaculeatus_homolog_chrom_start': 'Stickleback chromosome/scaffold start (bp)' (default: False), 'gaculeatus_homolog_chrom_end': 'Stickleback chromosome/scaffold end (bp)' (default: False), 'gaculeatus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'gaculeatus_homolog_subtype': 'Last common ancestor with Stickleback' (default: False), 'gaculeatus_homolog_orthology_type': 'Stickleback homology type' (default: False), 'gaculeatus_homolog_perc_id': '%id. target Stickleback gene identical to query gene' (default: False), 'gaculeatus_homolog_perc_id_r1': '%id. query gene identical to target Stickleback gene' (default: False), 'gaculeatus_homolog_goc_score': 'Stickleback Gene-order conservation score' (default: False), 'gaculeatus_homolog_wga_coverage': 'Stickleback Whole-genome alignment coverage' (default: False), 'gaculeatus_homolog_orthology_confidence': 'Stickleback orthology confidence [0 low, 1 high]' (default: False), 'malbus_homolog_ensembl_gene': 'Swamp eel gene stable ID' (default: False), 'malbus_homolog_associated_gene_name': 'Swamp eel gene name' (default: False), 'malbus_homolog_ensembl_peptide': 'Swamp eel protein or transcript stable ID' (default: False), 'malbus_homolog_chromosome': 'Swamp eel chromosome/scaffold name' (default: False), 'malbus_homolog_chrom_start': 'Swamp eel chromosome/scaffold start (bp)' (default: False), 'malbus_homolog_chrom_end': 'Swamp eel chromosome/scaffold end (bp)' (default: False), 'malbus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'malbus_homolog_subtype': 'Last common ancestor with Swamp eel' (default: False), 'malbus_homolog_orthology_type': 'Swamp eel homology type' (default: False), 'malbus_homolog_perc_id': '%id. target Swamp eel gene identical to query gene' (default: False), 'malbus_homolog_perc_id_r1': '%id. query gene identical to target Swamp eel gene' (default: False), 'malbus_homolog_goc_score': 'Swamp eel Gene-order conservation score' (default: False), 'malbus_homolog_wga_coverage': 'Swamp eel Whole-genome alignment coverage' (default: False), 'malbus_homolog_orthology_confidence': 'Swamp eel orthology confidence [0 low, 1 high]' (default: False), 'csyrichta_homolog_ensembl_gene': 'Tarsier gene stable ID' (default: False), 'csyrichta_homolog_associated_gene_name': 'Tarsier gene name' (default: False), 'csyrichta_homolog_ensembl_peptide': 'Tarsier protein or transcript stable ID' (default: False), 'csyrichta_homolog_chromosome': 'Tarsier chromosome/scaffold name' (default: False), 'csyrichta_homolog_chrom_start': 'Tarsier chromosome/scaffold start (bp)' (default: False), 'csyrichta_homolog_chrom_end': 'Tarsier chromosome/scaffold end (bp)' (default: False), 'csyrichta_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'csyrichta_homolog_subtype': 'Last common ancestor with Tarsier' (default: False), 'csyrichta_homolog_orthology_type': 'Tarsier homology type' (default: False), 'csyrichta_homolog_perc_id': '%id. target Tarsier gene identical to query gene' (default: False), 'csyrichta_homolog_perc_id_r1': '%id. query gene identical to target Tarsier gene' (default: False), 'csyrichta_homolog_goc_score': 'Tarsier Gene-order conservation score' (default: False), 'csyrichta_homolog_wga_coverage': 'Tarsier Whole-genome alignment coverage' (default: False), 'csyrichta_homolog_orthology_confidence': 'Tarsier orthology confidence [0 low, 1 high]' (default: False), 'sharrisii_homolog_ensembl_gene': 'Tasmanian devil gene stable ID' (default: False), 'sharrisii_homolog_associated_gene_name': 'Tasmanian devil gene name' (default: False), 'sharrisii_homolog_ensembl_peptide': 'Tasmanian devil protein or transcript stable ID' (default: False), 'sharrisii_homolog_chromosome': 'Tasmanian devil chromosome/scaffold name' (default: False), 'sharrisii_homolog_chrom_start': 'Tasmanian devil chromosome/scaffold start (bp)' (default: False), 'sharrisii_homolog_chrom_end': 'Tasmanian devil chromosome/scaffold end (bp)' (default: False), 'sharrisii_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'sharrisii_homolog_subtype': 'Last common ancestor with Tasmanian devil' (default: False), 'sharrisii_homolog_orthology_type': 'Tasmanian devil homology type' (default: False), 'sharrisii_homolog_perc_id': '%id. target Tasmanian devil gene identical to query gene' (default: False), 'sharrisii_homolog_perc_id_r1': '%id. query gene identical to target Tasmanian devil gene' (default: False), 'sharrisii_homolog_goc_score': 'Tasmanian devil Gene-order conservation score' (default: False), 'sharrisii_homolog_wga_coverage': 'Tasmanian devil Whole-genome alignment coverage' (default: False), 'sharrisii_homolog_orthology_confidence': 'Tasmanian devil orthology confidence [0 low, 1 high]' (default: False), 'tnigroviridis_homolog_ensembl_gene': 'Tetraodon gene stable ID' (default: False), 'tnigroviridis_homolog_associated_gene_name': 'Tetraodon gene name' (default: False), 'tnigroviridis_homolog_ensembl_peptide': 'Tetraodon protein or transcript stable ID' (default: False), 'tnigroviridis_homolog_chromosome': 'Tetraodon chromosome/scaffold name' (default: False), 'tnigroviridis_homolog_chrom_start': 'Tetraodon chromosome/scaffold start (bp)' (default: False), 'tnigroviridis_homolog_chrom_end': 'Tetraodon chromosome/scaffold end (bp)' (default: False), 'tnigroviridis_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'tnigroviridis_homolog_subtype': 'Last common ancestor with Tetraodon' (default: False), 'tnigroviridis_homolog_orthology_type': 'Tetraodon homology type' (default: False), 'tnigroviridis_homolog_perc_id': '%id. target Tetraodon gene identical to query gene' (default: False), 'tnigroviridis_homolog_perc_id_r1': '%id. query gene identical to target Tetraodon gene' (default: False), 'tnigroviridis_homolog_goc_score': 'Tetraodon Gene-order conservation score' (default: False), 'tnigroviridis_homolog_wga_coverage': 'Tetraodon Whole-genome alignment coverage' (default: False), 'tnigroviridis_homolog_orthology_confidence': 'Tetraodon orthology confidence [0 low, 1 high]' (default: False), 'ptaltaica_homolog_ensembl_gene': 'Tiger gene stable ID' (default: False), 'ptaltaica_homolog_associated_gene_name': 'Tiger gene name' (default: False), 'ptaltaica_homolog_ensembl_peptide': 'Tiger protein or transcript stable ID' (default: False), 'ptaltaica_homolog_chromosome': 'Tiger chromosome/scaffold name' (default: False), 'ptaltaica_homolog_chrom_start': 'Tiger chromosome/scaffold start (bp)' (default: False), 'ptaltaica_homolog_chrom_end': 'Tiger chromosome/scaffold end (bp)' (default: False), 'ptaltaica_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'ptaltaica_homolog_subtype': 'Last common ancestor with Tiger' (default: False), 'ptaltaica_homolog_orthology_type': 'Tiger homology type' (default: False), 'ptaltaica_homolog_perc_id': '%id. target Tiger gene identical to query gene' (default: False), 'ptaltaica_homolog_perc_id_r1': '%id. query gene identical to target Tiger gene' (default: False), 'ptaltaica_homolog_goc_score': 'Tiger Gene-order conservation score' (default: False), 'ptaltaica_homolog_wga_coverage': 'Tiger Whole-genome alignment coverage' (default: False), 'ptaltaica_homolog_orthology_confidence': 'Tiger orthology confidence [0 low, 1 high]' (default: False), 'hcomes_homolog_ensembl_gene': 'Tiger tail seahorse gene stable ID' (default: False), 'hcomes_homolog_associated_gene_name': 'Tiger tail seahorse gene name' (default: False), 'hcomes_homolog_ensembl_peptide': 'Tiger tail seahorse protein or transcript stable ID' (default: False), 'hcomes_homolog_chromosome': 'Tiger tail seahorse chromosome/scaffold name' (default: False), 'hcomes_homolog_chrom_start': 'Tiger tail seahorse chromosome/scaffold start (bp)' (default: False), 'hcomes_homolog_chrom_end': 'Tiger tail seahorse chromosome/scaffold end (bp)' (default: False), 'hcomes_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'hcomes_homolog_subtype': 'Last common ancestor with Tiger tail seahorse' (default: False), 'hcomes_homolog_orthology_type': 'Tiger tail seahorse homology type' (default: False), 'hcomes_homolog_perc_id': '%id. target Tiger tail seahorse gene identical to query gene' (default: False), 'hcomes_homolog_perc_id_r1': '%id. query gene identical to target Tiger tail seahorse gene' (default: False), 'hcomes_homolog_goc_score': 'Tiger tail seahorse Gene-order conservation score' (default: False), 'hcomes_homolog_wga_coverage': 'Tiger tail seahorse Whole-genome alignment coverage' (default: False), 'hcomes_homolog_orthology_confidence': 'Tiger tail seahorse orthology confidence [0 low, 1 high]' (default: False), 'csemilaevis_homolog_ensembl_gene': 'Tongue sole gene stable ID' (default: False), 'csemilaevis_homolog_associated_gene_name': 'Tongue sole gene name' (default: False), 'csemilaevis_homolog_ensembl_peptide': 'Tongue sole protein or transcript stable ID' (default: False), 'csemilaevis_homolog_chromosome': 'Tongue sole chromosome/scaffold name' (default: False), 'csemilaevis_homolog_chrom_start': 'Tongue sole chromosome/scaffold start (bp)' (default: False), 'csemilaevis_homolog_chrom_end': 'Tongue sole chromosome/scaffold end (bp)' (default: False), 'csemilaevis_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'csemilaevis_homolog_subtype': 'Last common ancestor with Tongue sole' (default: False), 'csemilaevis_homolog_orthology_type': 'Tongue sole homology type' (default: False), 'csemilaevis_homolog_perc_id': '%id. target Tongue sole gene identical to query gene' (default: False), 'csemilaevis_homolog_perc_id_r1': '%id. query gene identical to target Tongue sole gene' (default: False), 'csemilaevis_homolog_goc_score': 'Tongue sole Gene-order conservation score' (default: False), 'csemilaevis_homolog_wga_coverage': 'Tongue sole Whole-genome alignment coverage' (default: False), 'csemilaevis_homolog_orthology_confidence': 'Tongue sole orthology confidence [0 low, 1 high]' (default: False), 'tbelangeri_homolog_ensembl_gene': 'Tree Shrew gene stable ID' (default: False), 'tbelangeri_homolog_associated_gene_name': 'Tree Shrew gene name' (default: False), 'tbelangeri_homolog_ensembl_peptide': 'Tree Shrew protein or transcript stable ID' (default: False), 'tbelangeri_homolog_chromosome': 'Tree Shrew chromosome/scaffold name' (default: False), 'tbelangeri_homolog_chrom_start': 'Tree Shrew chromosome/scaffold start (bp)' (default: False), 'tbelangeri_homolog_chrom_end': 'Tree Shrew chromosome/scaffold end (bp)' (default: False), 'tbelangeri_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'tbelangeri_homolog_subtype': 'Last common ancestor with Tree Shrew' (default: False), 'tbelangeri_homolog_orthology_type': 'Tree Shrew homology type' (default: False), 'tbelangeri_homolog_perc_id': '%id. target Tree Shrew gene identical to query gene' (default: False), 'tbelangeri_homolog_perc_id_r1': '%id. query gene identical to target Tree Shrew gene' (default: False), 'tbelangeri_homolog_goc_score': 'Tree Shrew Gene-order conservation score' (default: False), 'tbelangeri_homolog_wga_coverage': 'Tree Shrew Whole-genome alignment coverage' (default: False), 'tbelangeri_homolog_orthology_confidence': 'Tree Shrew orthology confidence [0 low, 1 high]' (default: False), 'xtropicalis_homolog_ensembl_gene': 'Tropical clawed frog gene stable ID' (default: False), 'xtropicalis_homolog_associated_gene_name': 'Tropical clawed frog gene name' (default: False), 'xtropicalis_homolog_ensembl_peptide': 'Tropical clawed frog protein or transcript stable ID' (default: False), 'xtropicalis_homolog_chromosome': 'Tropical clawed frog chromosome/scaffold name' (default: False), 'xtropicalis_homolog_chrom_start': 'Tropical clawed frog chromosome/scaffold start (bp)' (default: False), 'xtropicalis_homolog_chrom_end': 'Tropical clawed frog chromosome/scaffold end (bp)' (default: False), 'xtropicalis_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'xtropicalis_homolog_subtype': 'Last common ancestor with Tropical clawed frog' (default: False), 'xtropicalis_homolog_orthology_type': 'Tropical clawed frog homology type' (default: False), 'xtropicalis_homolog_perc_id': '%id. target Tropical clawed frog gene identical to query gene' (default: False), 'xtropicalis_homolog_perc_id_r1': '%id. query gene identical to target Tropical clawed frog gene' (default: False), 'xtropicalis_homolog_goc_score': 'Tropical clawed frog Gene-order conservation score' (default: False), 'xtropicalis_homolog_wga_coverage': 'Tropical clawed frog Whole-genome alignment coverage' (default: False), 'xtropicalis_homolog_orthology_confidence': 'Tropical clawed frog orthology confidence [0 low, 1 high]' (default: False), 'spunctatus_homolog_ensembl_gene': 'Tuatara gene stable ID' (default: False), 'spunctatus_homolog_associated_gene_name': 'Tuatara gene name' (default: False), 'spunctatus_homolog_ensembl_peptide': 'Tuatara protein or transcript stable ID' (default: False), 'spunctatus_homolog_chromosome': 'Tuatara chromosome/scaffold name' (default: False), 'spunctatus_homolog_chrom_start': 'Tuatara chromosome/scaffold start (bp)' (default: False), 'spunctatus_homolog_chrom_end': 'Tuatara chromosome/scaffold end (bp)' (default: False), 'spunctatus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'spunctatus_homolog_subtype': 'Last common ancestor with Tuatara' (default: False), 'spunctatus_homolog_orthology_type': 'Tuatara homology type' (default: False), 'spunctatus_homolog_perc_id': '%id. target Tuatara gene identical to query gene' (default: False), 'spunctatus_homolog_perc_id_r1': '%id. query gene identical to target Tuatara gene' (default: False), 'spunctatus_homolog_goc_score': 'Tuatara Gene-order conservation score' (default: False), 'spunctatus_homolog_wga_coverage': 'Tuatara Whole-genome alignment coverage' (default: False), 'spunctatus_homolog_orthology_confidence': 'Tuatara orthology confidence [0 low, 1 high]' (default: False), 'smaximus_homolog_ensembl_gene': 'Turbot gene stable ID' (default: False), 'smaximus_homolog_associated_gene_name': 'Turbot gene name' (default: False), 'smaximus_homolog_ensembl_peptide': 'Turbot protein or transcript stable ID' (default: False), 'smaximus_homolog_chromosome': 'Turbot chromosome/scaffold name' (default: False), 'smaximus_homolog_chrom_start': 'Turbot chromosome/scaffold start (bp)' (default: False), 'smaximus_homolog_chrom_end': 'Turbot chromosome/scaffold end (bp)' (default: False), 'smaximus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'smaximus_homolog_subtype': 'Last common ancestor with Turbot' (default: False), 'smaximus_homolog_orthology_type': 'Turbot homology type' (default: False), 'smaximus_homolog_perc_id': '%id. target Turbot gene identical to query gene' (default: False), 'smaximus_homolog_perc_id_r1': '%id. query gene identical to target Turbot gene' (default: False), 'smaximus_homolog_goc_score': 'Turbot Gene-order conservation score' (default: False), 'smaximus_homolog_wga_coverage': 'Turbot Whole-genome alignment coverage' (default: False), 'smaximus_homolog_orthology_confidence': 'Turbot orthology confidence [0 low, 1 high]' (default: False), 'mgallopavo_homolog_ensembl_gene': 'Turkey gene stable ID' (default: False), 'mgallopavo_homolog_associated_gene_name': 'Turkey gene name' (default: False), 'mgallopavo_homolog_ensembl_peptide': 'Turkey protein or transcript stable ID' (default: False), 'mgallopavo_homolog_chromosome': 'Turkey chromosome/scaffold name' (default: False), 'mgallopavo_homolog_chrom_start': 'Turkey chromosome/scaffold start (bp)' (default: False), 'mgallopavo_homolog_chrom_end': 'Turkey chromosome/scaffold end (bp)' (default: False), 'mgallopavo_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'mgallopavo_homolog_subtype': 'Last common ancestor with Turkey' (default: False), 'mgallopavo_homolog_orthology_type': 'Turkey homology type' (default: False), 'mgallopavo_homolog_perc_id': '%id. target Turkey gene identical to query gene' (default: False), 'mgallopavo_homolog_perc_id_r1': '%id. query gene identical to target Turkey gene' (default: False), 'mgallopavo_homolog_goc_score': 'Turkey Gene-order conservation score' (default: False), 'mgallopavo_homolog_wga_coverage': 'Turkey Whole-genome alignment coverage' (default: False), 'mgallopavo_homolog_orthology_confidence': 'Turkey orthology confidence [0 low, 1 high]' (default: False), 'ptephrosceles_homolog_ensembl_gene': 'Ugandan red Colobus gene stable ID' (default: False), 'ptephrosceles_homolog_associated_gene_name': 'Ugandan red Colobus gene name' (default: False), 'ptephrosceles_homolog_ensembl_peptide': 'Ugandan red Colobus protein or transcript stable ID' (default: False), 'ptephrosceles_homolog_chromosome': 'Ugandan red Colobus chromosome/scaffold name' (default: False), 'ptephrosceles_homolog_chrom_start': 'Ugandan red Colobus chromosome/scaffold start (bp)' (default: False), 'ptephrosceles_homolog_chrom_end': 'Ugandan red Colobus chromosome/scaffold end (bp)' (default: False), 'ptephrosceles_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'ptephrosceles_homolog_subtype': 'Last common ancestor with Ugandan red Colobus' (default: False), 'ptephrosceles_homolog_orthology_type': 'Ugandan red Colobus homology type' (default: False), 'ptephrosceles_homolog_perc_id': '%id. target Ugandan red Colobus gene identical to query gene' (default: False), 'ptephrosceles_homolog_perc_id_r1': '%id. query gene identical to target Ugandan red Colobus gene' (default: False), 'ptephrosceles_homolog_goc_score': 'Ugandan red Colobus Gene-order conservation score' (default: False), 'ptephrosceles_homolog_wga_coverage': 'Ugandan red Colobus Whole-genome alignment coverage' (default: False), 'ptephrosceles_homolog_orthology_confidence': 'Ugandan red Colobus orthology confidence [0 low, 1 high]' (default: False), 'ngalili_homolog_ensembl_gene': 'Upper Galilee mountains blind mole rat gene stable ID' (default: False), 'ngalili_homolog_associated_gene_name': 'Upper Galilee mountains blind mole rat gene name' (default: False), 'ngalili_homolog_ensembl_peptide': 'Upper Galilee mountains blind mole rat protein or transcript stable ID' (default: False), 'ngalili_homolog_chromosome': 'Upper Galilee mountains blind mole rat chromosome/scaffold name' (default: False), 'ngalili_homolog_chrom_start': 'Upper Galilee mountains blind mole rat chromosome/scaffold start (bp)' (default: False), 'ngalili_homolog_chrom_end': 'Upper Galilee mountains blind mole rat chromosome/scaffold end (bp)' (default: False), 'ngalili_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'ngalili_homolog_subtype': 'Last common ancestor with Upper Galilee mountains blind mole rat' (default: False), 'ngalili_homolog_orthology_type': 'Upper Galilee mountains blind mole rat homology type' (default: False), 'ngalili_homolog_perc_id': '%id. target Upper Galilee mountains blind mole rat gene identical to query gene' (default: False), 'ngalili_homolog_perc_id_r1': '%id. query gene identical to target Upper Galilee mountains blind mole rat gene' (default: False), 'ngalili_homolog_goc_score': 'Upper Galilee mountains blind mole rat Gene-order conservation score' (default: False), 'ngalili_homolog_wga_coverage': 'Upper Galilee mountains blind mole rat Whole-genome alignment coverage' (default: False), 'ngalili_homolog_orthology_confidence': 'Upper Galilee mountains blind mole rat orthology confidence [0 low, 1 high]' (default: False), 'csabaeus_homolog_ensembl_gene': 'Vervet-AGM gene stable ID' (default: False), 'csabaeus_homolog_associated_gene_name': 'Vervet-AGM gene name' (default: False), 'csabaeus_homolog_ensembl_peptide': 'Vervet-AGM protein or transcript stable ID' (default: False), 'csabaeus_homolog_chromosome': 'Vervet-AGM chromosome/scaffold name' (default: False), 'csabaeus_homolog_chrom_start': 'Vervet-AGM chromosome/scaffold start (bp)' (default: False), 'csabaeus_homolog_chrom_end': 'Vervet-AGM chromosome/scaffold end (bp)' (default: False), 'csabaeus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'csabaeus_homolog_subtype': 'Last common ancestor with Vervet-AGM' (default: False), 'csabaeus_homolog_orthology_type': 'Vervet-AGM homology type' (default: False), 'csabaeus_homolog_perc_id': '%id. target Vervet-AGM gene identical to query gene' (default: False), 'csabaeus_homolog_perc_id_r1': '%id. query gene identical to target Vervet-AGM gene' (default: False), 'csabaeus_homolog_goc_score': 'Vervet-AGM Gene-order conservation score' (default: False), 'csabaeus_homolog_wga_coverage': 'Vervet-AGM Whole-genome alignment coverage' (default: False), 'csabaeus_homolog_orthology_confidence': 'Vervet-AGM orthology confidence [0 low, 1 high]' (default: False), 'neugenii_homolog_ensembl_gene': 'Wallaby gene stable ID' (default: False), 'neugenii_homolog_associated_gene_name': 'Wallaby gene name' (default: False), 'neugenii_homolog_ensembl_peptide': 'Wallaby protein or transcript stable ID' (default: False), 'neugenii_homolog_chromosome': 'Wallaby chromosome/scaffold name' (default: False), 'neugenii_homolog_chrom_start': 'Wallaby chromosome/scaffold start (bp)' (default: False), 'neugenii_homolog_chrom_end': 'Wallaby chromosome/scaffold end (bp)' (default: False), 'neugenii_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'neugenii_homolog_subtype': 'Last common ancestor with Wallaby' (default: False), 'neugenii_homolog_orthology_type': 'Wallaby homology type' (default: False), 'neugenii_homolog_perc_id': '%id. target Wallaby gene identical to query gene' (default: False), 'neugenii_homolog_perc_id_r1': '%id. query gene identical to target Wallaby gene' (default: False), 'neugenii_homolog_goc_score': 'Wallaby Gene-order conservation score' (default: False), 'neugenii_homolog_wga_coverage': 'Wallaby Whole-genome alignment coverage' (default: False), 'neugenii_homolog_orthology_confidence': 'Wallaby orthology confidence [0 low, 1 high]' (default: False), 'bmutus_homolog_ensembl_gene': 'Wild yak gene stable ID' (default: False), 'bmutus_homolog_associated_gene_name': 'Wild yak gene name' (default: False), 'bmutus_homolog_ensembl_peptide': 'Wild yak protein or transcript stable ID' (default: False), 'bmutus_homolog_chromosome': 'Wild yak chromosome/scaffold name' (default: False), 'bmutus_homolog_chrom_start': 'Wild yak chromosome/scaffold start (bp)' (default: False), 'bmutus_homolog_chrom_end': 'Wild yak chromosome/scaffold end (bp)' (default: False), 'bmutus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'bmutus_homolog_subtype': 'Last common ancestor with Wild yak' (default: False), 'bmutus_homolog_orthology_type': 'Wild yak homology type' (default: False), 'bmutus_homolog_perc_id': '%id. target Wild yak gene identical to query gene' (default: False), 'bmutus_homolog_perc_id_r1': '%id. query gene identical to target Wild yak gene' (default: False), 'bmutus_homolog_goc_score': 'Wild yak Gene-order conservation score' (default: False), 'bmutus_homolog_wga_coverage': 'Wild yak Whole-genome alignment coverage' (default: False), 'bmutus_homolog_orthology_confidence': 'Wild yak orthology confidence [0 low, 1 high]' (default: False), 'sldorsalis_homolog_ensembl_gene': 'Yellowtail amberjack gene stable ID' (default: False), 'sldorsalis_homolog_associated_gene_name': 'Yellowtail amberjack gene name' (default: False), 'sldorsalis_homolog_ensembl_peptide': 'Yellowtail amberjack protein or transcript stable ID' (default: False), 'sldorsalis_homolog_chromosome': 'Yellowtail amberjack chromosome/scaffold name' (default: False), 'sldorsalis_homolog_chrom_start': 'Yellowtail amberjack chromosome/scaffold start (bp)' (default: False), 'sldorsalis_homolog_chrom_end': 'Yellowtail amberjack chromosome/scaffold end (bp)' (default: False), 'sldorsalis_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'sldorsalis_homolog_subtype': 'Last common ancestor with Yellowtail amberjack' (default: False), 'sldorsalis_homolog_orthology_type': 'Yellowtail amberjack homology type' (default: False), 'sldorsalis_homolog_perc_id': '%id. target Yellowtail amberjack gene identical to query gene' (default: False), 'sldorsalis_homolog_perc_id_r1': '%id. query gene identical to target Yellowtail amberjack gene' (default: False), 'sldorsalis_homolog_goc_score': 'Yellowtail amberjack Gene-order conservation score' (default: False), 'sldorsalis_homolog_wga_coverage': 'Yellowtail amberjack Whole-genome alignment coverage' (default: False), 'sldorsalis_homolog_orthology_confidence': 'Yellowtail amberjack orthology confidence [0 low, 1 high]' (default: False), 'tguttata_homolog_ensembl_gene': 'Zebra finch gene stable ID' (default: False), 'tguttata_homolog_associated_gene_name': 'Zebra finch gene name' (default: False), 'tguttata_homolog_ensembl_peptide': 'Zebra finch protein or transcript stable ID' (default: False), 'tguttata_homolog_chromosome': 'Zebra finch chromosome/scaffold name' (default: False), 'tguttata_homolog_chrom_start': 'Zebra finch chromosome/scaffold start (bp)' (default: False), 'tguttata_homolog_chrom_end': 'Zebra finch chromosome/scaffold end (bp)' (default: False), 'tguttata_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'tguttata_homolog_subtype': 'Last common ancestor with Zebra finch' (default: False), 'tguttata_homolog_orthology_type': 'Zebra finch homology type' (default: False), 'tguttata_homolog_perc_id': '%id. target Zebra finch gene identical to query gene' (default: False), 'tguttata_homolog_perc_id_r1': '%id. query gene identical to target Zebra finch gene' (default: False), 'tguttata_homolog_goc_score': 'Zebra finch Gene-order conservation score' (default: False), 'tguttata_homolog_wga_coverage': 'Zebra finch Whole-genome alignment coverage' (default: False), 'tguttata_homolog_orthology_confidence': 'Zebra finch orthology confidence [0 low, 1 high]' (default: False), 'mzebra_homolog_ensembl_gene': 'Zebra mbuna gene stable ID' (default: False), 'mzebra_homolog_associated_gene_name': 'Zebra mbuna gene name' (default: False), 'mzebra_homolog_ensembl_peptide': 'Zebra mbuna protein or transcript stable ID' (default: False), 'mzebra_homolog_chromosome': 'Zebra mbuna chromosome/scaffold name' (default: False), 'mzebra_homolog_chrom_start': 'Zebra mbuna chromosome/scaffold start (bp)' (default: False), 'mzebra_homolog_chrom_end': 'Zebra mbuna chromosome/scaffold end (bp)' (default: False), 'mzebra_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'mzebra_homolog_subtype': 'Last common ancestor with Zebra mbuna' (default: False), 'mzebra_homolog_orthology_type': 'Zebra mbuna homology type' (default: False), 'mzebra_homolog_perc_id': '%id. target Zebra mbuna gene identical to query gene' (default: False), 'mzebra_homolog_perc_id_r1': '%id. query gene identical to target Zebra mbuna gene' (default: False), 'mzebra_homolog_goc_score': 'Zebra mbuna Gene-order conservation score' (default: False), 'mzebra_homolog_wga_coverage': 'Zebra mbuna Whole-genome alignment coverage' (default: False), 'mzebra_homolog_orthology_confidence': 'Zebra mbuna orthology confidence [0 low, 1 high]' (default: False), 'drerio_homolog_ensembl_gene': 'Zebrafish gene stable ID' (default: False), 'drerio_homolog_associated_gene_name': 'Zebrafish gene name' (default: False), 'drerio_homolog_ensembl_peptide': 'Zebrafish protein or transcript stable ID' (default: False), 'drerio_homolog_chromosome': 'Zebrafish chromosome/scaffold name' (default: False), 'drerio_homolog_chrom_start': 'Zebrafish chromosome/scaffold start (bp)' (default: False), 'drerio_homolog_chrom_end': 'Zebrafish chromosome/scaffold end (bp)' (default: False), 'drerio_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'drerio_homolog_subtype': 'Last common ancestor with Zebrafish' (default: False), 'drerio_homolog_orthology_type': 'Zebrafish homology type' (default: False), 'drerio_homolog_perc_id': '%id. target Zebrafish gene identical to query gene' (default: False), 'drerio_homolog_perc_id_r1': '%id. query gene identical to target Zebrafish gene' (default: False), 'drerio_homolog_goc_score': 'Zebrafish Gene-order conservation score' (default: False), 'drerio_homolog_wga_coverage': 'Zebrafish Whole-genome alignment coverage' (default: False), 'drerio_homolog_orthology_confidence': 'Zebrafish orthology confidence [0 low, 1 high]' (default: False), 'marmatus_homolog_ensembl_gene': 'Zig-zag eel gene stable ID' (default: False), 'marmatus_homolog_associated_gene_name': 'Zig-zag eel gene name' (default: False), 'marmatus_homolog_ensembl_peptide': 'Zig-zag eel protein or transcript stable ID' (default: False), 'marmatus_homolog_chromosome': 'Zig-zag eel chromosome/scaffold name' (default: False), 'marmatus_homolog_chrom_start': 'Zig-zag eel chromosome/scaffold start (bp)' (default: False), 'marmatus_homolog_chrom_end': 'Zig-zag eel chromosome/scaffold end (bp)' (default: False), 'marmatus_homolog_canonical_transcript_protein': 'Query protein or transcript ID' (default: False), 'marmatus_homolog_subtype': 'Last common ancestor with Zig-zag eel' (default: False), 'marmatus_homolog_orthology_type': 'Zig-zag eel homology type' (default: False), 'marmatus_homolog_perc_id': '%id. target Zig-zag eel gene identical to query gene' (default: False), 'marmatus_homolog_perc_id_r1': '%id. query gene identical to target Zig-zag eel gene' (default: False), 'marmatus_homolog_goc_score': 'Zig-zag eel Gene-order conservation score' (default: False), 'marmatus_homolog_wga_coverage': 'Zig-zag eel Whole-genome alignment coverage' (default: False), 'marmatus_homolog_orthology_confidence': 'Zig-zag eel orthology confidence [0 low, 1 high]' (default: False), 'hsapiens_paralog_ensembl_gene': 'Human paralogue gene stable ID' (default: False), 'hsapiens_paralog_associated_gene_name': 'Human paralogue associated gene name' (default: False), 'hsapiens_paralog_ensembl_peptide': 'Human paralogue protein or transcript ID' (default: False), 'hsapiens_paralog_chromosome': 'Human paralogue chromosome/scaffold name' (default: False), 'hsapiens_paralog_chrom_start': 'Human paralogue chromosome/scaffold start (bp)' (default: False), 'hsapiens_paralog_chrom_end': 'Human paralogue chromosome/scaffold end (bp)' (default: False), 'hsapiens_paralog_canonical_transcript_protein': 'Paralogue query protein or transcript ID' (default: False), 'hsapiens_paralog_subtype': 'Paralogue last common ancestor with Human' (default: False), 'hsapiens_paralog_orthology_type': 'Human paralogue homology type' (default: False), 'hsapiens_paralog_perc_id': 'Paralogue %id. target Human gene identical to query gene' (default: False), 'hsapiens_paralog_perc_id_r1': 'Paralogue %id. query gene identical to target Human gene' (default: False)}, defaults: []),\n",
      " 'link_attributes': 'ATTRIBUTES FOR LINKS': (attributes: {}, defaults: []),\n",
      " 'sequences': 'Sequences': (attributes: {'transcript_exon_intron': 'Unspliced (Transcript)' (default: False), 'gene_exon_intron': 'Unspliced (Gene)' (default: False), 'transcript_flank': 'Flank (Transcript)' (default: False), 'gene_flank': 'Flank (Gene)' (default: False), 'coding_transcript_flank': 'Flank-coding region (Transcript)' (default: False), 'coding_gene_flank': 'Flank-coding region (Gene)' (default: False), '5utr': '5' UTR' (default: False), '3utr': '3' UTR' (default: False), 'gene_exon': 'Exon sequences' (default: False), 'cdna': 'cDNA sequences' (default: False), 'coding': 'Coding sequence' (default: False), 'peptide': 'Peptide' (default: False), 'upstream_flank': 'upstream_flank' (default: False), 'downstream_flank': 'downstream_flank' (default: False), 'ensembl_gene_id': 'Gene stable ID' (default: False), 'ensembl_gene_id_version': 'Gene stable ID version' (default: False), 'description': 'Gene description' (default: False), 'external_gene_name': 'Gene name' (default: False), 'external_gene_source': 'Source of gene name' (default: False), 'chromosome_name': 'Chromosome/scaffold name' (default: False), 'start_position': 'Gene start (bp)' (default: False), 'end_position': 'Gene end (bp)' (default: False), 'gene_biotype': 'Gene type' (default: False), 'version': 'Version (gene)' (default: False), 'family': 'Ensembl Protein Family ID(s)' (default: False), 'uniparc': 'UniParc ID' (default: False), 'uniprotswissprot': 'UniProtKB/Swiss-Prot ID' (default: False), 'uniprotsptrembl': 'UniProtKB/TrEMBL ID' (default: False), 'cdna_coding_start': 'cDNA coding start' (default: False), 'cdna_coding_end': 'cDNA coding end' (default: False), '5_utr_start': '5' UTR start' (default: False), '5_utr_end': '5' UTR end' (default: False), '3_utr_start': '3' UTR start' (default: False), '3_utr_end': '3' UTR end' (default: False), 'ensembl_transcript_id': 'Transcript stable ID' (default: False), 'ensembl_transcript_id_version': 'Transcript stable ID version' (default: False), 'ensembl_peptide_id': 'Protein stable ID' (default: False), 'ensembl_peptide_id_version': 'Protein stable ID version' (default: False), 'transcript_biotype': 'Transcript type' (default: False), 'transcript_version': 'Version (transcript)' (default: False), 'peptide_version': 'Version (protein)' (default: False), 'strand': 'Strand' (default: False), 'transcript_start': 'Transcript start (bp)' (default: False), 'transcript_end': 'Transcript end (bp)' (default: False), 'transcription_start_site': 'Transcription start site (TSS)' (default: False), 'transcript_length': 'Transcript length (including UTRs and CDS)' (default: False), 'cds_length': 'CDS Length' (default: False), 'cds_start': 'CDS start' (default: False), 'cds_end': 'CDS end' (default: False), 'ensembl_exon_id': 'Exon stable ID' (default: False), 'exon_chrom_start': 'Exon region start (bp)' (default: False), 'exon_chrom_end': 'Exon region end (bp)' (default: False), 'rank': 'Exon rank in transcript' (default: False), 'phase': 'Start phase' (default: False), 'end_phase': 'End phase' (default: False), 'genomic_coding_start': 'Genomic coding start' (default: False), 'genomic_coding_end': 'Genomic coding end' (default: False), 'is_constitutive': 'Constitutive exon' (default: False)}, defaults: []),\n",
      " 'snp': 'Variant (Germline)': (attributes: {'ensembl_gene_id': 'Gene stable ID' (default: False), 'ensembl_gene_id_version': 'Gene stable ID version' (default: False), 'version': 'Version (gene)' (default: False), 'ensembl_transcript_id': 'Transcript stable ID' (default: False), 'ensembl_transcript_id_version': 'Transcript stable ID version' (default: False), 'transcript_version': 'Version (transcript)' (default: False), 'ensembl_peptide_id': 'Protein stable ID' (default: False), 'ensembl_peptide_id_version': 'Protein stable ID version' (default: False), 'peptide_version': 'Version (protein)' (default: False), 'chromosome_name': 'Chromosome/scaffold name' (default: False), 'start_position': 'Gene start (bp)' (default: False), 'end_position': 'Gene end (bp)' (default: False), 'strand': 'Strand' (default: False), 'band': 'Karyotype band' (default: False), 'external_gene_name': 'Gene name' (default: False), 'external_gene_source': 'Source of gene name' (default: False), 'transcript_count': 'Transcript count' (default: False), 'percentage_gene_gc_content': 'Gene % GC content' (default: False), 'description': 'Gene description' (default: False), 'variation_name': 'Variant name' (default: True), 'germ_line_variation_source': 'Variant source' (default: False), 'source_description': 'Variant source description' (default: False), 'allele': 'Variant alleles' (default: False), 'validated': 'Variant supporting evidence' (default: False), 'mapweight': 'Mapweight' (default: False), 'minor_allele': 'Minor allele' (default: False), 'minor_allele_freq': 'Minor allele frequency' (default: False), 'minor_allele_count': 'Minor allele count' (default: False), 'clinical_significance': 'Clinical significance' (default: False), 'transcript_location': 'Transcript location (bp)' (default: False), 'snp_chromosome_strand': 'Variant chromosome Strand' (default: False), 'peptide_location': 'Protein location (aa)' (default: False), 'chromosome_start': 'chromosome/scaffold position start (bp)' (default: False), 'chromosome_end': 'Chromosome/scaffold position end (bp)' (default: False), 'polyphen_prediction_2076': 'PolyPhen prediction' (default: False), 'polyphen_score_2076': 'PolyPhen score' (default: False), 'sift_prediction_2076': 'SIFT prediction' (default: False), 'sift_score_2076': 'SIFT score' (default: False), 'distance_to_transcript_2076': 'Distance to transcript' (default: False), 'cds_start_2076': 'CDS start' (default: False), 'cds_end_2076': 'CDS end' (default: False), 'peptide_shift': 'Protein allele' (default: False), 'synonymous_status': 'Variant consequence' (default: False), 'allele_string_2076': 'Consequence specific allele' (default: False)}, defaults: ['variation_name']),\n",
      " 'snp_somatic': 'Variant (Somatic)': (attributes: {'ensembl_gene_id': 'Gene stable ID' (default: False), 'ensembl_gene_id_version': 'Gene stable ID version' (default: False), 'version': 'Version (gene)' (default: False), 'ensembl_transcript_id': 'Transcript stable ID' (default: False), 'ensembl_transcript_id_version': 'Transcript stable ID version' (default: False), 'transcript_version': 'Version (transcript)' (default: False), 'ensembl_peptide_id': 'Protein stable ID' (default: False), 'ensembl_peptide_id_version': 'Protein stable ID version' (default: False), 'peptide_version': 'Version (protein)' (default: False), 'chromosome_name': 'Chromosome/scaffold name' (default: False), 'start_position': 'Gene start (bp)' (default: False), 'end_position': 'Gene end (bp)' (default: False), 'strand': 'Strand' (default: False), 'band': 'Karyotype band' (default: False), 'external_gene_name': 'Gene name' (default: False), 'external_gene_source': 'Source of gene name' (default: False), 'transcript_count': 'Transcript count' (default: False), 'percentage_gene_gc_content': 'Gene % GC content' (default: False), 'description': 'Gene description' (default: False), 'somatic_variation_name': 'Variant name' (default: True), 'somatic_source_name': 'Variant source' (default: False), 'somatic_source_description': 'Variant source description' (default: False), 'somatic_allele': 'Variant alleles' (default: False), 'somatic_validated': 'Variant supporting evidence' (default: False), 'somatic_mapweight': 'Mapweight' (default: False), 'somatic_transcript_location': 'Transcript location (bp)' (default: False), 'somatic_snp_chromosome_strand': 'Variant chromosome/scaffold strand' (default: False), 'somatic_peptide_location': 'Protein location (aa)' (default: False), 'somatic_chromosome_start': 'Chromosome/scaffold position start (bp)' (default: False), 'somatic_chromosome_end': 'Chromosome/scaffold position end (bp)' (default: False), 'mart_transcript_variation_som__dm_distance_to_transcript_2076': 'Distance to transcript' (default: False), 'somatic_cds_start_2076': 'CDS start' (default: False), 'somatic_cds_end_2076': 'CDS end' (default: False), 'somatic_synonymous_status': 'Variant consequence' (default: False), 'mart_transcript_variation_som__dm_allele_string_2076': 'Consequence specific allele' (default: False)}, defaults: ['somatic_variation_name']),\n",
      " 'structure': 'Structures': (attributes: {'ensembl_gene_id': 'Gene stable ID' (default: False), 'ensembl_gene_id_version': 'Gene stable ID version' (default: False), 'version': 'Version (gene)' (default: False), 'ensembl_transcript_id': 'Transcript stable ID' (default: False), 'ensembl_transcript_id_version': 'Transcript stable ID version' (default: False), 'transcript_version': 'Version (transcript)' (default: False), 'ensembl_peptide_id': 'Protein stable ID' (default: False), 'ensembl_peptide_id_version': 'Protein stable ID version' (default: False), 'peptide_version': 'Version (protein)' (default: False), 'chromosome_name': 'Chromosome/scaffold name' (default: False), 'start_position': 'Gene start (bp)' (default: False), 'end_position': 'Gene end (bp)' (default: False), 'transcript_start': 'Transcript start (bp)' (default: False), 'transcript_end': 'Transcript end (bp)' (default: False), 'transcription_start_site': 'Transcription start site (TSS)' (default: False), 'transcript_length': 'Transcript length (including UTRs and CDS)' (default: False), 'strand': 'Strand' (default: False), 'external_gene_name': 'Gene name' (default: False), 'external_gene_source': 'Source of gene name' (default: False), '5_utr_start': '5' UTR start' (default: False), '5_utr_end': '5' UTR end' (default: False), '3_utr_start': '3' UTR start' (default: False), '3_utr_end': '3' UTR end' (default: False), 'cds_length': 'CDS Length' (default: False), 'transcript_count': 'Transcript count' (default: False), 'description': 'Gene description' (default: False), 'gene_biotype': 'Gene type' (default: False), 'exon_chrom_start': 'Exon region start (bp)' (default: False), 'exon_chrom_end': 'Exon region end (bp)' (default: False), 'is_constitutive': 'Constitutive exon' (default: False), 'rank': 'Exon rank in transcript' (default: False), 'phase': 'Start phase' (default: False), 'end_phase': 'End phase' (default: False), 'cdna_coding_start': 'cDNA coding start' (default: False), 'cdna_coding_end': 'cDNA coding end' (default: False), 'genomic_coding_start': 'Genomic coding start' (default: False), 'genomic_coding_end': 'Genomic coding end' (default: False), 'ensembl_exon_id': 'Exon stable ID' (default: False), 'cds_start': 'CDS start' (default: False), 'cds_end': 'CDS end' (default: False)}, defaults: [])}\n"
     ]
    }
   ],
   "source": [
    "ensmbl.show_attributes_by_page()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 294,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'# Filtered by: padj <=0.05 in SLAM-seq at 2 or 24 hours, basemean>10, log2FC SLAM at 2 hours <-0.2'"
      ]
     },
     "execution_count": 294,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "a = h.fileToList('../results/slamseqMax/MEF2Dtargets.txt')\n",
    "a.pop(0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 302,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Gene stable ID</th>\n",
       "      <th>Gene type</th>\n",
       "      <th>Transcription start site (TSS)</th>\n",
       "      <th>3' UTR start</th>\n",
       "      <th>Gene start (bp)</th>\n",
       "      <th>Gene name</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>ENSG00000210049</td>\n",
       "      <td>Mt_tRNA</td>\n",
       "      <td>577</td>\n",
       "      <td>NaN</td>\n",
       "      <td>577</td>\n",
       "      <td>MT-TF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>ENSG00000211459</td>\n",
       "      <td>Mt_rRNA</td>\n",
       "      <td>648</td>\n",
       "      <td>NaN</td>\n",
       "      <td>648</td>\n",
       "      <td>MT-RNR1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>ENSG00000210077</td>\n",
       "      <td>Mt_tRNA</td>\n",
       "      <td>1602</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1602</td>\n",
       "      <td>MT-TV</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>ENSG00000210082</td>\n",
       "      <td>Mt_rRNA</td>\n",
       "      <td>1671</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1671</td>\n",
       "      <td>MT-RNR2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>ENSG00000209082</td>\n",
       "      <td>Mt_tRNA</td>\n",
       "      <td>3230</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3230</td>\n",
       "      <td>MT-TL1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>383612</th>\n",
       "      <td>ENSG00000285114</td>\n",
       "      <td>protein_coding</td>\n",
       "      <td>129795674</td>\n",
       "      <td>NaN</td>\n",
       "      <td>129757225</td>\n",
       "      <td>GSDMC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>383613</th>\n",
       "      <td>ENSG00000285114</td>\n",
       "      <td>protein_coding</td>\n",
       "      <td>129795674</td>\n",
       "      <td>129757225.0</td>\n",
       "      <td>129757225</td>\n",
       "      <td>GSDMC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>383614</th>\n",
       "      <td>ENSG00000285114</td>\n",
       "      <td>protein_coding</td>\n",
       "      <td>129786637</td>\n",
       "      <td>NaN</td>\n",
       "      <td>129757225</td>\n",
       "      <td>GSDMC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>383615</th>\n",
       "      <td>ENSG00000285114</td>\n",
       "      <td>protein_coding</td>\n",
       "      <td>129786637</td>\n",
       "      <td>129757226.0</td>\n",
       "      <td>129757225</td>\n",
       "      <td>GSDMC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>383616</th>\n",
       "      <td>ENSG00000285114</td>\n",
       "      <td>protein_coding</td>\n",
       "      <td>129761786</td>\n",
       "      <td>NaN</td>\n",
       "      <td>129757225</td>\n",
       "      <td>GSDMC</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>383617 rows  6 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "         Gene stable ID       Gene type  Transcription start site (TSS)  \\\n",
       "0       ENSG00000210049         Mt_tRNA                             577   \n",
       "1       ENSG00000211459         Mt_rRNA                             648   \n",
       "2       ENSG00000210077         Mt_tRNA                            1602   \n",
       "3       ENSG00000210082         Mt_rRNA                            1671   \n",
       "4       ENSG00000209082         Mt_tRNA                            3230   \n",
       "...                 ...             ...                             ...   \n",
       "383612  ENSG00000285114  protein_coding                       129795674   \n",
       "383613  ENSG00000285114  protein_coding                       129795674   \n",
       "383614  ENSG00000285114  protein_coding                       129786637   \n",
       "383615  ENSG00000285114  protein_coding                       129786637   \n",
       "383616  ENSG00000285114  protein_coding                       129761786   \n",
       "\n",
       "        3' UTR start  Gene start (bp) Gene name  \n",
       "0                NaN              577     MT-TF  \n",
       "1                NaN              648   MT-RNR1  \n",
       "2                NaN             1602     MT-TV  \n",
       "3                NaN             1671   MT-RNR2  \n",
       "4                NaN             3230    MT-TL1  \n",
       "...              ...              ...       ...  \n",
       "383612           NaN        129757225     GSDMC  \n",
       "383613   129757225.0        129757225     GSDMC  \n",
       "383614           NaN        129757225     GSDMC  \n",
       "383615   129757226.0        129757225     GSDMC  \n",
       "383616           NaN        129757225     GSDMC  \n",
       "\n",
       "[383617 rows x 6 columns]"
      ]
     },
     "execution_count": 302,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ensembltss"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### on unscalled data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bams = !ls ../../data/$project/bwa/mergedLibrary/mp*.bam\n",
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 193,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mkdir ../../data/$project/diffPeaks_unscaled"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 191,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-MED1_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-MYC_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-POLII_R1.mLb.clN.sorted.bam']"
      ]
     },
     "execution_count": 191,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 213,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1.mLb.clN.sorted.bam ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1.mLb.clN.sorted.bam and ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "230\n",
      "computing the scaling values\n",
      "b'INFO  @ Tue, 03 Nov 2020 14:54:27: \\n# Command line: callpeak -B -t ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1.mLb.clN.sorted.bam -c ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 230 -n MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1 --outdir ../../data/chipseq_MEF2D_degraded/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 03 Nov 2020 14:54:27: #1 read tag files... \\nINFO  @ Tue, 03 Nov 2020 14:54:27: #1 read treatment tags... \\nINFO  @ Tue, 03 Nov 2020 14:54:31:  1000000 \\nINFO  @ Tue, 03 Nov 2020 14:54:36:  2000000 \\nINFO  @ Tue, 03 Nov 2020 14:54:40:  3000000 \\nINFO  @ Tue, 03 Nov 2020 14:54:45:  4000000 \\nINFO  @ Tue, 03 Nov 2020 14:54:49:  5000000 \\nINFO  @ Tue, 03 Nov 2020 14:54:54:  6000000 \\nINFO  @ Tue, 03 Nov 2020 14:54:59:  7000000 \\nINFO  @ Tue, 03 Nov 2020 14:55:03:  8000000 \\nINFO  @ Tue, 03 Nov 2020 14:55:08:  9000000 \\nINFO  @ Tue, 03 Nov 2020 14:55:12:  10000000 \\nINFO  @ Tue, 03 Nov 2020 14:55:17:  11000000 \\nINFO  @ Tue, 03 Nov 2020 14:55:21:  12000000 \\nINFO  @ Tue, 03 Nov 2020 14:55:26:  13000000 \\nINFO  @ Tue, 03 Nov 2020 14:55:30:  14000000 \\nINFO  @ Tue, 03 Nov 2020 14:55:35:  15000000 \\nINFO  @ Tue, 03 Nov 2020 14:55:39:  16000000 \\nINFO  @ Tue, 03 Nov 2020 14:55:44:  17000000 \\nINFO  @ Tue, 03 Nov 2020 14:55:48:  18000000 \\nINFO  @ Tue, 03 Nov 2020 14:55:53:  19000000 \\nINFO  @ Tue, 03 Nov 2020 14:55:57:  20000000 \\nINFO  @ Tue, 03 Nov 2020 14:56:01:  21000000 \\nINFO  @ Tue, 03 Nov 2020 14:56:06:  22000000 \\nINFO  @ Tue, 03 Nov 2020 14:56:10:  23000000 \\nINFO  @ Tue, 03 Nov 2020 14:56:15:  24000000 \\nINFO  @ Tue, 03 Nov 2020 14:56:19:  25000000 \\nINFO  @ Tue, 03 Nov 2020 14:56:24:  26000000 \\nINFO  @ Tue, 03 Nov 2020 14:56:28:  27000000 \\nINFO  @ Tue, 03 Nov 2020 14:56:32:  28000000 \\nINFO  @ Tue, 03 Nov 2020 14:56:37:  29000000 \\nINFO  @ Tue, 03 Nov 2020 14:56:41:  30000000 \\nINFO  @ Tue, 03 Nov 2020 14:56:46:  31000000 \\nINFO  @ Tue, 03 Nov 2020 14:56:50:  32000000 \\nINFO  @ Tue, 03 Nov 2020 14:56:55:  33000000 \\nINFO  @ Tue, 03 Nov 2020 14:56:59:  34000000 \\nINFO  @ Tue, 03 Nov 2020 14:57:04:  35000000 \\nINFO  @ Tue, 03 Nov 2020 14:57:08:  36000000 \\nINFO  @ Tue, 03 Nov 2020 14:57:13:  37000000 \\nINFO  @ Tue, 03 Nov 2020 14:57:17:  38000000 \\nINFO  @ Tue, 03 Nov 2020 14:57:22:  39000000 \\nINFO  @ Tue, 03 Nov 2020 14:57:26:  40000000 \\nINFO  @ Tue, 03 Nov 2020 14:57:30:  41000000 \\nINFO  @ Tue, 03 Nov 2020 14:57:35:  42000000 \\nINFO  @ Tue, 03 Nov 2020 14:57:39:  43000000 \\nINFO  @ Tue, 03 Nov 2020 14:57:44:  44000000 \\nINFO  @ Tue, 03 Nov 2020 14:57:48:  45000000 \\nINFO  @ Tue, 03 Nov 2020 14:57:52:  46000000 \\nINFO  @ Tue, 03 Nov 2020 14:57:57:  47000000 \\nINFO  @ Tue, 03 Nov 2020 14:58:01:  48000000 \\nINFO  @ Tue, 03 Nov 2020 14:58:05:  49000000 \\nINFO  @ Tue, 03 Nov 2020 14:58:10:  50000000 \\nINFO  @ Tue, 03 Nov 2020 14:58:14:  51000000 \\nINFO  @ Tue, 03 Nov 2020 14:58:19:  52000000 \\nINFO  @ Tue, 03 Nov 2020 14:58:23:  53000000 \\nINFO  @ Tue, 03 Nov 2020 14:58:28:  54000000 \\nINFO  @ Tue, 03 Nov 2020 14:58:32:  55000000 \\nINFO  @ Tue, 03 Nov 2020 14:58:37:  56000000 \\nINFO  @ Tue, 03 Nov 2020 14:58:41:  57000000 \\nINFO  @ Tue, 03 Nov 2020 14:58:46:  58000000 \\nINFO  @ Tue, 03 Nov 2020 14:58:50:  59000000 \\nINFO  @ Tue, 03 Nov 2020 14:58:54:  60000000 \\nINFO  @ Tue, 03 Nov 2020 14:58:59:  61000000 \\nINFO  @ Tue, 03 Nov 2020 14:59:03:  62000000 \\nINFO  @ Tue, 03 Nov 2020 14:59:08:  63000000 \\nINFO  @ Tue, 03 Nov 2020 14:59:12:  64000000 \\nINFO  @ Tue, 03 Nov 2020 14:59:17:  65000000 \\nINFO  @ Tue, 03 Nov 2020 14:59:21:  66000000 \\nINFO  @ Tue, 03 Nov 2020 14:59:25:  67000000 \\nINFO  @ Tue, 03 Nov 2020 14:59:30:  68000000 \\nINFO  @ Tue, 03 Nov 2020 14:59:35:  69000000 \\nINFO  @ Tue, 03 Nov 2020 14:59:39:  70000000 \\nINFO  @ Tue, 03 Nov 2020 14:59:43:  71000000 \\nINFO  @ Tue, 03 Nov 2020 14:59:48:  72000000 \\nINFO  @ Tue, 03 Nov 2020 14:59:52:  73000000 \\nINFO  @ Tue, 03 Nov 2020 14:59:57:  74000000 \\nINFO  @ Tue, 03 Nov 2020 15:00:01:  75000000 \\nINFO  @ Tue, 03 Nov 2020 15:00:06:  76000000 \\nINFO  @ Tue, 03 Nov 2020 15:00:10:  77000000 \\nINFO  @ Tue, 03 Nov 2020 15:00:14:  78000000 \\nINFO  @ Tue, 03 Nov 2020 15:00:19:  79000000 \\nINFO  @ Tue, 03 Nov 2020 15:00:23:  80000000 \\nINFO  @ Tue, 03 Nov 2020 15:00:28:  81000000 \\nINFO  @ Tue, 03 Nov 2020 15:00:32:  82000000 \\nINFO  @ Tue, 03 Nov 2020 15:00:36:  83000000 \\nINFO  @ Tue, 03 Nov 2020 15:00:41:  84000000 \\nINFO  @ Tue, 03 Nov 2020 15:00:45:  85000000 \\nINFO  @ Tue, 03 Nov 2020 15:00:50:  86000000 \\nINFO  @ Tue, 03 Nov 2020 15:00:55:  87000000 \\nINFO  @ Tue, 03 Nov 2020 15:00:59:  88000000 \\nINFO  @ Tue, 03 Nov 2020 15:01:04:  89000000 \\nINFO  @ Tue, 03 Nov 2020 15:01:09:  90000000 \\nINFO  @ Tue, 03 Nov 2020 15:01:12: 90585277 reads have been read. \\nINFO  @ Tue, 03 Nov 2020 15:01:13: #1.2 read input tags... \\nINFO  @ Tue, 03 Nov 2020 15:01:17:  1000000 \\nINFO  @ Tue, 03 Nov 2020 15:01:21:  2000000 \\nINFO  @ Tue, 03 Nov 2020 15:01:25:  3000000 \\nINFO  @ Tue, 03 Nov 2020 15:01:29:  4000000 \\nINFO  @ Tue, 03 Nov 2020 15:01:33:  5000000 \\nINFO  @ Tue, 03 Nov 2020 15:01:36:  6000000 \\nINFO  @ Tue, 03 Nov 2020 15:01:40:  7000000 \\nINFO  @ Tue, 03 Nov 2020 15:01:44:  8000000 \\nINFO  @ Tue, 03 Nov 2020 15:01:48:  9000000 \\nINFO  @ Tue, 03 Nov 2020 15:01:52:  10000000 \\nINFO  @ Tue, 03 Nov 2020 15:01:56:  11000000 \\nINFO  @ Tue, 03 Nov 2020 15:02:00:  12000000 \\nINFO  @ Tue, 03 Nov 2020 15:02:04:  13000000 \\nINFO  @ Tue, 03 Nov 2020 15:02:08:  14000000 \\nINFO  @ Tue, 03 Nov 2020 15:02:11:  15000000 \\nINFO  @ Tue, 03 Nov 2020 15:02:15:  16000000 \\nINFO  @ Tue, 03 Nov 2020 15:02:19:  17000000 \\nINFO  @ Tue, 03 Nov 2020 15:02:22:  18000000 \\nINFO  @ Tue, 03 Nov 2020 15:02:26:  19000000 \\nINFO  @ Tue, 03 Nov 2020 15:02:30:  20000000 \\nINFO  @ Tue, 03 Nov 2020 15:02:34:  21000000 \\nINFO  @ Tue, 03 Nov 2020 15:02:38:  22000000 \\nINFO  @ Tue, 03 Nov 2020 15:02:42:  23000000 \\nINFO  @ Tue, 03 Nov 2020 15:02:46:  24000000 \\nINFO  @ Tue, 03 Nov 2020 15:02:49:  25000000 \\nINFO  @ Tue, 03 Nov 2020 15:02:53:  26000000 \\nINFO  @ Tue, 03 Nov 2020 15:02:57:  27000000 \\nINFO  @ Tue, 03 Nov 2020 15:03:01:  28000000 \\nINFO  @ Tue, 03 Nov 2020 15:03:04:  29000000 \\nINFO  @ Tue, 03 Nov 2020 15:03:08:  30000000 \\nINFO  @ Tue, 03 Nov 2020 15:03:12:  31000000 \\nINFO  @ Tue, 03 Nov 2020 15:03:17:  32000000 \\nINFO  @ Tue, 03 Nov 2020 15:03:21:  33000000 \\nINFO  @ Tue, 03 Nov 2020 15:03:25:  34000000 \\nINFO  @ Tue, 03 Nov 2020 15:03:29:  35000000 \\nINFO  @ Tue, 03 Nov 2020 15:03:33:  36000000 \\nINFO  @ Tue, 03 Nov 2020 15:03:37:  37000000 \\nINFO  @ Tue, 03 Nov 2020 15:03:41:  38000000 \\nINFO  @ Tue, 03 Nov 2020 15:03:45:  39000000 \\nINFO  @ Tue, 03 Nov 2020 15:03:49:  40000000 \\nINFO  @ Tue, 03 Nov 2020 15:03:53:  41000000 \\nINFO  @ Tue, 03 Nov 2020 15:03:57:  42000000 \\nINFO  @ Tue, 03 Nov 2020 15:04:01:  43000000 \\nINFO  @ Tue, 03 Nov 2020 15:04:06:  44000000 \\nINFO  @ Tue, 03 Nov 2020 15:04:10:  45000000 \\nINFO  @ Tue, 03 Nov 2020 15:04:14:  46000000 \\nINFO  @ Tue, 03 Nov 2020 15:04:18:  47000000 \\nINFO  @ Tue, 03 Nov 2020 15:04:22:  48000000 \\nINFO  @ Tue, 03 Nov 2020 15:04:26:  49000000 \\nINFO  @ Tue, 03 Nov 2020 15:04:31:  50000000 \\nINFO  @ Tue, 03 Nov 2020 15:04:35:  51000000 \\nINFO  @ Tue, 03 Nov 2020 15:04:40:  52000000 \\nINFO  @ Tue, 03 Nov 2020 15:04:44:  53000000 \\nINFO  @ Tue, 03 Nov 2020 15:04:48:  54000000 \\nINFO  @ Tue, 03 Nov 2020 15:04:53:  55000000 \\nINFO  @ Tue, 03 Nov 2020 15:04:57:  56000000 \\nINFO  @ Tue, 03 Nov 2020 15:05:02:  57000000 \\nINFO  @ Tue, 03 Nov 2020 15:05:06:  58000000 \\nINFO  @ Tue, 03 Nov 2020 15:05:10:  59000000 \\nINFO  @ Tue, 03 Nov 2020 15:05:14:  60000000 \\nINFO  @ Tue, 03 Nov 2020 15:05:19:  61000000 \\nINFO  @ Tue, 03 Nov 2020 15:05:23:  62000000 \\nINFO  @ Tue, 03 Nov 2020 15:05:27:  63000000 \\nINFO  @ Tue, 03 Nov 2020 15:05:30:  64000000 \\nINFO  @ Tue, 03 Nov 2020 15:05:34:  65000000 \\nINFO  @ Tue, 03 Nov 2020 15:05:39:  66000000 \\nINFO  @ Tue, 03 Nov 2020 15:05:42: 66660428 reads have been read. \\nINFO  @ Tue, 03 Nov 2020 15:05:43: #1 tag size is determined as 99 bps \\nINFO  @ Tue, 03 Nov 2020 15:05:43: #1 tag size = 99.0 \\nINFO  @ Tue, 03 Nov 2020 15:05:43: #1  total tags in treatment: 90585277 \\nINFO  @ Tue, 03 Nov 2020 15:05:43: #1 user defined the maximum tags... \\nINFO  @ Tue, 03 Nov 2020 15:05:43: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 03 Nov 2020 15:05:43: #1  tags after filtering in treatment: 89139933 \\nINFO  @ Tue, 03 Nov 2020 15:05:43: #1  Redundant rate of treatment: 0.02 \\nINFO  @ Tue, 03 Nov 2020 15:05:43: #1  total tags in control: 66660428 \\nINFO  @ Tue, 03 Nov 2020 15:05:43: #1 user defined the maximum tags... \\nINFO  @ Tue, 03 Nov 2020 15:05:43: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 03 Nov 2020 15:05:44: #1  tags after filtering in control: 65724989 \\nINFO  @ Tue, 03 Nov 2020 15:05:44: #1  Redundant rate of control: 0.01 \\nINFO  @ Tue, 03 Nov 2020 15:05:44: #1 finished! \\nINFO  @ Tue, 03 Nov 2020 15:05:44: #2 Build Peak Model... \\nINFO  @ Tue, 03 Nov 2020 15:05:44: #2 Skipped... \\nINFO  @ Tue, 03 Nov 2020 15:05:44: #2 Use 230 as fragment length \\nINFO  @ Tue, 03 Nov 2020 15:05:44: #3 Call peaks... \\nINFO  @ Tue, 03 Nov 2020 15:05:44: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 03 Nov 2020 15:12:27: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 03 Nov 2020 15:12:27: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_treat_pileup.bdg \\nINFO  @ Tue, 03 Nov 2020 15:12:27: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_control_lambda.bdg \\nINFO  @ Tue, 03 Nov 2020 15:12:27: #3   Pileup will be based on sequencing depth in control. \\nINFO  @ Tue, 03 Nov 2020 15:12:27: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 03 Nov 2020 15:18:36: #4 Write output xls file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_peaks.xls \\nINFO  @ Tue, 03 Nov 2020 15:18:36: #4 Write peak in narrowPeak format file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_peaks.narrowPeak \\nINFO  @ Tue, 03 Nov 2020 15:18:36: #4 Write summits bed file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_summits.bed \\nINFO  @ Tue, 03 Nov 2020 15:18:36: Done! \\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'INFO  @ Tue, 03 Nov 2020 15:18:37: \\n# Command line: callpeak -B -t ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1.mLb.clN.sorted.bam -c ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 230 -n MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1 --outdir ../../data/chipseq_MEF2D_degraded/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 03 Nov 2020 15:18:37: #1 read tag files... \\nINFO  @ Tue, 03 Nov 2020 15:18:37: #1 read treatment tags... \\nINFO  @ Tue, 03 Nov 2020 15:18:42:  1000000 \\nINFO  @ Tue, 03 Nov 2020 15:18:47:  2000000 \\nINFO  @ Tue, 03 Nov 2020 15:18:52:  3000000 \\nINFO  @ Tue, 03 Nov 2020 15:18:56:  4000000 \\nINFO  @ Tue, 03 Nov 2020 15:19:01:  5000000 \\nINFO  @ Tue, 03 Nov 2020 15:19:06:  6000000 \\nINFO  @ Tue, 03 Nov 2020 15:19:11:  7000000 \\nINFO  @ Tue, 03 Nov 2020 15:19:16:  8000000 \\nINFO  @ Tue, 03 Nov 2020 15:19:21:  9000000 \\nINFO  @ Tue, 03 Nov 2020 15:19:26:  10000000 \\nINFO  @ Tue, 03 Nov 2020 15:19:30:  11000000 \\nINFO  @ Tue, 03 Nov 2020 15:19:35:  12000000 \\nINFO  @ Tue, 03 Nov 2020 15:19:40:  13000000 \\nINFO  @ Tue, 03 Nov 2020 15:19:45:  14000000 \\nINFO  @ Tue, 03 Nov 2020 15:19:49:  15000000 \\nINFO  @ Tue, 03 Nov 2020 15:19:54:  16000000 \\nINFO  @ Tue, 03 Nov 2020 15:19:59:  17000000 \\nINFO  @ Tue, 03 Nov 2020 15:20:04:  18000000 \\nINFO  @ Tue, 03 Nov 2020 15:20:09:  19000000 \\nINFO  @ Tue, 03 Nov 2020 15:20:14:  20000000 \\nINFO  @ Tue, 03 Nov 2020 15:20:18:  21000000 \\nINFO  @ Tue, 03 Nov 2020 15:20:23:  22000000 \\nINFO  @ Tue, 03 Nov 2020 15:20:28:  23000000 \\nINFO  @ Tue, 03 Nov 2020 15:20:33:  24000000 \\nINFO  @ Tue, 03 Nov 2020 15:20:38:  25000000 \\nINFO  @ Tue, 03 Nov 2020 15:20:42:  26000000 \\nINFO  @ Tue, 03 Nov 2020 15:20:45: 26542203 reads have been read. \\nINFO  @ Tue, 03 Nov 2020 15:20:45: #1.2 read input tags... \\nINFO  @ Tue, 03 Nov 2020 15:20:49:  1000000 \\nINFO  @ Tue, 03 Nov 2020 15:20:53:  2000000 \\nINFO  @ Tue, 03 Nov 2020 15:20:57:  3000000 \\nINFO  @ Tue, 03 Nov 2020 15:21:00:  4000000 \\nINFO  @ Tue, 03 Nov 2020 15:21:04:  5000000 \\nINFO  @ Tue, 03 Nov 2020 15:21:08:  6000000 \\nINFO  @ Tue, 03 Nov 2020 15:21:11:  7000000 \\nINFO  @ Tue, 03 Nov 2020 15:21:15:  8000000 \\nINFO  @ Tue, 03 Nov 2020 15:21:19:  9000000 \\nINFO  @ Tue, 03 Nov 2020 15:21:22:  10000000 \\nINFO  @ Tue, 03 Nov 2020 15:21:26:  11000000 \\nINFO  @ Tue, 03 Nov 2020 15:21:30:  12000000 \\nINFO  @ Tue, 03 Nov 2020 15:21:34:  13000000 \\nINFO  @ Tue, 03 Nov 2020 15:21:37:  14000000 \\nINFO  @ Tue, 03 Nov 2020 15:21:41:  15000000 \\nINFO  @ Tue, 03 Nov 2020 15:21:45:  16000000 \\nINFO  @ Tue, 03 Nov 2020 15:21:48:  17000000 \\nINFO  @ Tue, 03 Nov 2020 15:21:52:  18000000 \\nINFO  @ Tue, 03 Nov 2020 15:21:56:  19000000 \\nINFO  @ Tue, 03 Nov 2020 15:22:00:  20000000 \\nINFO  @ Tue, 03 Nov 2020 15:22:03:  21000000 \\nINFO  @ Tue, 03 Nov 2020 15:22:07:  22000000 \\nINFO  @ Tue, 03 Nov 2020 15:22:11:  23000000 \\nINFO  @ Tue, 03 Nov 2020 15:22:15:  24000000 \\nINFO  @ Tue, 03 Nov 2020 15:22:19:  25000000 \\nINFO  @ Tue, 03 Nov 2020 15:22:22:  26000000 \\nINFO  @ Tue, 03 Nov 2020 15:22:26:  27000000 \\nINFO  @ Tue, 03 Nov 2020 15:22:30:  28000000 \\nINFO  @ Tue, 03 Nov 2020 15:22:33:  29000000 \\nINFO  @ Tue, 03 Nov 2020 15:22:37:  30000000 \\nINFO  @ Tue, 03 Nov 2020 15:22:41:  31000000 \\nINFO  @ Tue, 03 Nov 2020 15:22:44:  32000000 \\nINFO  @ Tue, 03 Nov 2020 15:22:48:  33000000 \\nINFO  @ Tue, 03 Nov 2020 15:22:52:  34000000 \\nINFO  @ Tue, 03 Nov 2020 15:22:56:  35000000 \\nINFO  @ Tue, 03 Nov 2020 15:22:59:  36000000 \\nINFO  @ Tue, 03 Nov 2020 15:23:03:  37000000 \\nINFO  @ Tue, 03 Nov 2020 15:23:06:  38000000 \\nINFO  @ Tue, 03 Nov 2020 15:23:10:  39000000 \\nINFO  @ Tue, 03 Nov 2020 15:23:14:  40000000 \\nINFO  @ Tue, 03 Nov 2020 15:23:17:  41000000 \\nINFO  @ Tue, 03 Nov 2020 15:23:21:  42000000 \\nINFO  @ Tue, 03 Nov 2020 15:23:25:  43000000 \\nINFO  @ Tue, 03 Nov 2020 15:23:28:  44000000 \\nINFO  @ Tue, 03 Nov 2020 15:23:32:  45000000 \\nINFO  @ Tue, 03 Nov 2020 15:23:36:  46000000 \\nINFO  @ Tue, 03 Nov 2020 15:23:39:  47000000 \\nINFO  @ Tue, 03 Nov 2020 15:23:43:  48000000 \\nINFO  @ Tue, 03 Nov 2020 15:23:47:  49000000 \\nINFO  @ Tue, 03 Nov 2020 15:23:50:  50000000 \\nINFO  @ Tue, 03 Nov 2020 15:23:54:  51000000 \\nINFO  @ Tue, 03 Nov 2020 15:23:58:  52000000 \\nINFO  @ Tue, 03 Nov 2020 15:24:01:  53000000 \\nINFO  @ Tue, 03 Nov 2020 15:24:05:  54000000 \\nINFO  @ Tue, 03 Nov 2020 15:24:09:  55000000 \\nINFO  @ Tue, 03 Nov 2020 15:24:13:  56000000 \\nINFO  @ Tue, 03 Nov 2020 15:24:16:  57000000 \\nINFO  @ Tue, 03 Nov 2020 15:24:20:  58000000 \\nINFO  @ Tue, 03 Nov 2020 15:24:24:  59000000 \\nINFO  @ Tue, 03 Nov 2020 15:24:28:  60000000 \\nINFO  @ Tue, 03 Nov 2020 15:24:31:  61000000 \\nINFO  @ Tue, 03 Nov 2020 15:24:35:  62000000 \\nINFO  @ Tue, 03 Nov 2020 15:24:39:  63000000 \\nINFO  @ Tue, 03 Nov 2020 15:24:42:  64000000 \\nINFO  @ Tue, 03 Nov 2020 15:24:46:  65000000 \\nINFO  @ Tue, 03 Nov 2020 15:24:50:  66000000 \\nINFO  @ Tue, 03 Nov 2020 15:24:53: 66660428 reads have been read. \\nINFO  @ Tue, 03 Nov 2020 15:24:53: #1 tag size is determined as 97 bps \\nINFO  @ Tue, 03 Nov 2020 15:24:53: #1 tag size = 97.0 \\nINFO  @ Tue, 03 Nov 2020 15:24:53: #1  total tags in treatment: 26542203 \\nINFO  @ Tue, 03 Nov 2020 15:24:53: #1 user defined the maximum tags... \\nINFO  @ Tue, 03 Nov 2020 15:24:53: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 03 Nov 2020 15:24:53: #1  tags after filtering in treatment: 26154750 \\nINFO  @ Tue, 03 Nov 2020 15:24:53: #1  Redundant rate of treatment: 0.01 \\nINFO  @ Tue, 03 Nov 2020 15:24:53: #1  total tags in control: 66660428 \\nINFO  @ Tue, 03 Nov 2020 15:24:53: #1 user defined the maximum tags... \\nINFO  @ Tue, 03 Nov 2020 15:24:53: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 03 Nov 2020 15:24:54: #1  tags after filtering in control: 65724989 \\nINFO  @ Tue, 03 Nov 2020 15:24:54: #1  Redundant rate of control: 0.01 \\nINFO  @ Tue, 03 Nov 2020 15:24:54: #1 finished! \\nINFO  @ Tue, 03 Nov 2020 15:24:54: #2 Build Peak Model... \\nINFO  @ Tue, 03 Nov 2020 15:24:54: #2 Skipped... \\nINFO  @ Tue, 03 Nov 2020 15:24:54: #2 Use 230 as fragment length \\nINFO  @ Tue, 03 Nov 2020 15:24:54: #3 Call peaks... \\nINFO  @ Tue, 03 Nov 2020 15:24:54: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 03 Nov 2020 15:29:42: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 03 Nov 2020 15:29:42: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_treat_pileup.bdg \\nINFO  @ Tue, 03 Nov 2020 15:29:42: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_control_lambda.bdg \\nINFO  @ Tue, 03 Nov 2020 15:29:42: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Tue, 03 Nov 2020 15:29:42: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 03 Nov 2020 15:33:24: #4 Write output xls file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_peaks.xls \\nINFO  @ Tue, 03 Nov 2020 15:33:24: #4 Write peak in narrowPeak format file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_peaks.narrowPeak \\nINFO  @ Tue, 03 Nov 2020 15:33:24: #4 Write summits bed file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_summits.bed \\nINFO  @ Tue, 03 Nov 2020 15:33:24: Done! \\n'\n",
      "65724989 26154750\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1.mLb.clN.sorted.bam ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1.mLb.clN.sorted.bam and ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "208\n",
      "computing the scaling values\n",
      "b'INFO  @ Tue, 03 Nov 2020 15:54:07: \\n# Command line: callpeak -B -t ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1.mLb.clN.sorted.bam -c ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 208 -n MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1 --outdir ../../data/chipseq_MEF2D_degraded/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 03 Nov 2020 15:54:07: #1 read tag files... \\nINFO  @ Tue, 03 Nov 2020 15:54:07: #1 read treatment tags... \\nINFO  @ Tue, 03 Nov 2020 15:54:12:  1000000 \\nINFO  @ Tue, 03 Nov 2020 15:54:16:  2000000 \\nINFO  @ Tue, 03 Nov 2020 15:54:21:  3000000 \\nINFO  @ Tue, 03 Nov 2020 15:54:26:  4000000 \\nINFO  @ Tue, 03 Nov 2020 15:54:31:  5000000 \\nINFO  @ Tue, 03 Nov 2020 15:54:35:  6000000 \\nINFO  @ Tue, 03 Nov 2020 15:54:40:  7000000 \\nINFO  @ Tue, 03 Nov 2020 15:54:45:  8000000 \\nINFO  @ Tue, 03 Nov 2020 15:54:50:  9000000 \\nINFO  @ Tue, 03 Nov 2020 15:54:54:  10000000 \\nINFO  @ Tue, 03 Nov 2020 15:54:59:  11000000 \\nINFO  @ Tue, 03 Nov 2020 15:55:04:  12000000 \\nINFO  @ Tue, 03 Nov 2020 15:55:08:  13000000 \\nINFO  @ Tue, 03 Nov 2020 15:55:13:  14000000 \\nINFO  @ Tue, 03 Nov 2020 15:55:18:  15000000 \\nINFO  @ Tue, 03 Nov 2020 15:55:23:  16000000 \\nINFO  @ Tue, 03 Nov 2020 15:55:28:  17000000 \\nINFO  @ Tue, 03 Nov 2020 15:55:32:  18000000 \\nINFO  @ Tue, 03 Nov 2020 15:55:37:  19000000 \\nINFO  @ Tue, 03 Nov 2020 15:55:42:  20000000 \\nINFO  @ Tue, 03 Nov 2020 15:55:46:  21000000 \\nINFO  @ Tue, 03 Nov 2020 15:55:51:  22000000 \\nINFO  @ Tue, 03 Nov 2020 15:55:56:  23000000 \\nINFO  @ Tue, 03 Nov 2020 15:56:00:  24000000 \\nINFO  @ Tue, 03 Nov 2020 15:56:05:  25000000 \\nINFO  @ Tue, 03 Nov 2020 15:56:09:  26000000 \\nINFO  @ Tue, 03 Nov 2020 15:56:14:  27000000 \\nINFO  @ Tue, 03 Nov 2020 15:56:18:  28000000 \\nINFO  @ Tue, 03 Nov 2020 15:56:23:  29000000 \\nINFO  @ Tue, 03 Nov 2020 15:56:27:  30000000 \\nINFO  @ Tue, 03 Nov 2020 15:56:32:  31000000 \\nINFO  @ Tue, 03 Nov 2020 15:56:36:  32000000 \\nINFO  @ Tue, 03 Nov 2020 15:56:40:  33000000 \\nINFO  @ Tue, 03 Nov 2020 15:56:45:  34000000 \\nINFO  @ Tue, 03 Nov 2020 15:56:49:  35000000 \\nINFO  @ Tue, 03 Nov 2020 15:56:54:  36000000 \\nINFO  @ Tue, 03 Nov 2020 15:56:58:  37000000 \\nINFO  @ Tue, 03 Nov 2020 15:57:03:  38000000 \\nINFO  @ Tue, 03 Nov 2020 15:57:07:  39000000 \\nINFO  @ Tue, 03 Nov 2020 15:57:12:  40000000 \\nINFO  @ Tue, 03 Nov 2020 15:57:16:  41000000 \\nINFO  @ Tue, 03 Nov 2020 15:57:21:  42000000 \\nINFO  @ Tue, 03 Nov 2020 15:57:26:  43000000 \\nINFO  @ Tue, 03 Nov 2020 15:57:31:  44000000 \\nINFO  @ Tue, 03 Nov 2020 15:57:33: 44433177 reads have been read. \\nINFO  @ Tue, 03 Nov 2020 15:57:33: #1.2 read input tags... \\nINFO  @ Tue, 03 Nov 2020 15:57:38:  1000000 \\nINFO  @ Tue, 03 Nov 2020 15:57:42:  2000000 \\nINFO  @ Tue, 03 Nov 2020 15:57:47:  3000000 \\nINFO  @ Tue, 03 Nov 2020 15:57:52:  4000000 \\nINFO  @ Tue, 03 Nov 2020 15:57:55:  5000000 \\nINFO  @ Tue, 03 Nov 2020 15:57:59:  6000000 \\nINFO  @ Tue, 03 Nov 2020 15:58:03:  7000000 \\nINFO  @ Tue, 03 Nov 2020 15:58:07:  8000000 \\nINFO  @ Tue, 03 Nov 2020 15:58:11:  9000000 \\nINFO  @ Tue, 03 Nov 2020 15:58:14:  10000000 \\nINFO  @ Tue, 03 Nov 2020 15:58:18:  11000000 \\nINFO  @ Tue, 03 Nov 2020 15:58:22:  12000000 \\nINFO  @ Tue, 03 Nov 2020 15:58:26:  13000000 \\nINFO  @ Tue, 03 Nov 2020 15:58:30:  14000000 \\nINFO  @ Tue, 03 Nov 2020 15:58:34:  15000000 \\nINFO  @ Tue, 03 Nov 2020 15:58:38:  16000000 \\nINFO  @ Tue, 03 Nov 2020 15:58:42:  17000000 \\nINFO  @ Tue, 03 Nov 2020 15:58:46:  18000000 \\nINFO  @ Tue, 03 Nov 2020 15:58:49:  19000000 \\nINFO  @ Tue, 03 Nov 2020 15:58:53:  20000000 \\nINFO  @ Tue, 03 Nov 2020 15:58:57:  21000000 \\nINFO  @ Tue, 03 Nov 2020 15:59:01:  22000000 \\nINFO  @ Tue, 03 Nov 2020 15:59:05:  23000000 \\nINFO  @ Tue, 03 Nov 2020 15:59:09:  24000000 \\nINFO  @ Tue, 03 Nov 2020 15:59:13:  25000000 \\nINFO  @ Tue, 03 Nov 2020 15:59:17:  26000000 \\nINFO  @ Tue, 03 Nov 2020 15:59:21:  27000000 \\nINFO  @ Tue, 03 Nov 2020 15:59:25:  28000000 \\nINFO  @ Tue, 03 Nov 2020 15:59:28:  29000000 \\nINFO  @ Tue, 03 Nov 2020 15:59:32:  30000000 \\nINFO  @ Tue, 03 Nov 2020 15:59:36:  31000000 \\nINFO  @ Tue, 03 Nov 2020 15:59:40:  32000000 \\nINFO  @ Tue, 03 Nov 2020 15:59:44:  33000000 \\nINFO  @ Tue, 03 Nov 2020 15:59:48:  34000000 \\nINFO  @ Tue, 03 Nov 2020 15:59:51:  35000000 \\nINFO  @ Tue, 03 Nov 2020 15:59:55:  36000000 \\nINFO  @ Tue, 03 Nov 2020 15:59:59:  37000000 \\nINFO  @ Tue, 03 Nov 2020 16:00:02:  38000000 \\nINFO  @ Tue, 03 Nov 2020 16:00:06:  39000000 \\nINFO  @ Tue, 03 Nov 2020 16:00:10:  40000000 \\nINFO  @ Tue, 03 Nov 2020 16:00:14:  41000000 \\nINFO  @ Tue, 03 Nov 2020 16:00:18:  42000000 \\nINFO  @ Tue, 03 Nov 2020 16:00:22:  43000000 \\nINFO  @ Tue, 03 Nov 2020 16:00:26:  44000000 \\nINFO  @ Tue, 03 Nov 2020 16:00:29:  45000000 \\nINFO  @ Tue, 03 Nov 2020 16:00:33:  46000000 \\nINFO  @ Tue, 03 Nov 2020 16:00:37:  47000000 \\nINFO  @ Tue, 03 Nov 2020 16:00:41:  48000000 \\nINFO  @ Tue, 03 Nov 2020 16:00:45:  49000000 \\nINFO  @ Tue, 03 Nov 2020 16:00:48:  50000000 \\nINFO  @ Tue, 03 Nov 2020 16:00:52:  51000000 \\nINFO  @ Tue, 03 Nov 2020 16:00:56:  52000000 \\nINFO  @ Tue, 03 Nov 2020 16:01:00:  53000000 \\nINFO  @ Tue, 03 Nov 2020 16:01:04:  54000000 \\nINFO  @ Tue, 03 Nov 2020 16:01:07:  55000000 \\nINFO  @ Tue, 03 Nov 2020 16:01:11:  56000000 \\nINFO  @ Tue, 03 Nov 2020 16:01:15:  57000000 \\nINFO  @ Tue, 03 Nov 2020 16:01:19:  58000000 \\nINFO  @ Tue, 03 Nov 2020 16:01:23:  59000000 \\nINFO  @ Tue, 03 Nov 2020 16:01:26:  60000000 \\nINFO  @ Tue, 03 Nov 2020 16:01:30:  61000000 \\nINFO  @ Tue, 03 Nov 2020 16:01:34:  62000000 \\nINFO  @ Tue, 03 Nov 2020 16:01:38:  63000000 \\nINFO  @ Tue, 03 Nov 2020 16:01:42:  64000000 \\nINFO  @ Tue, 03 Nov 2020 16:01:46:  65000000 \\nINFO  @ Tue, 03 Nov 2020 16:01:50:  66000000 \\nINFO  @ Tue, 03 Nov 2020 16:01:53: 66660428 reads have been read. \\nINFO  @ Tue, 03 Nov 2020 16:01:53: #1 tag size is determined as 95 bps \\nINFO  @ Tue, 03 Nov 2020 16:01:53: #1 tag size = 95.0 \\nINFO  @ Tue, 03 Nov 2020 16:01:53: #1  total tags in treatment: 44433177 \\nINFO  @ Tue, 03 Nov 2020 16:01:53: #1 user defined the maximum tags... \\nINFO  @ Tue, 03 Nov 2020 16:01:53: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 03 Nov 2020 16:01:53: #1  tags after filtering in treatment: 43197324 \\nINFO  @ Tue, 03 Nov 2020 16:01:53: #1  Redundant rate of treatment: 0.03 \\nINFO  @ Tue, 03 Nov 2020 16:01:53: #1  total tags in control: 66660428 \\nINFO  @ Tue, 03 Nov 2020 16:01:53: #1 user defined the maximum tags... \\nINFO  @ Tue, 03 Nov 2020 16:01:53: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 03 Nov 2020 16:01:54: #1  tags after filtering in control: 65724989 \\nINFO  @ Tue, 03 Nov 2020 16:01:54: #1  Redundant rate of control: 0.01 \\nINFO  @ Tue, 03 Nov 2020 16:01:54: #1 finished! \\nINFO  @ Tue, 03 Nov 2020 16:01:54: #2 Build Peak Model... \\nINFO  @ Tue, 03 Nov 2020 16:01:54: #2 Skipped... \\nINFO  @ Tue, 03 Nov 2020 16:01:54: #2 Use 208 as fragment length \\nINFO  @ Tue, 03 Nov 2020 16:01:54: #3 Call peaks... \\nINFO  @ Tue, 03 Nov 2020 16:01:54: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 03 Nov 2020 16:07:30: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 03 Nov 2020 16:07:30: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_treat_pileup.bdg \\nINFO  @ Tue, 03 Nov 2020 16:07:30: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_control_lambda.bdg \\nINFO  @ Tue, 03 Nov 2020 16:07:30: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Tue, 03 Nov 2020 16:07:30: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 03 Nov 2020 16:11:59: #4 Write output xls file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_peaks.xls \\nINFO  @ Tue, 03 Nov 2020 16:11:59: #4 Write peak in narrowPeak format file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_peaks.narrowPeak \\nINFO  @ Tue, 03 Nov 2020 16:11:59: #4 Write summits bed file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_summits.bed \\nINFO  @ Tue, 03 Nov 2020 16:11:59: Done! \\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'INFO  @ Tue, 03 Nov 2020 16:12:00: \\n# Command line: callpeak -B -t ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1.mLb.clN.sorted.bam -c ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 208 -n MV411_MEF2D_NT_SC_63_VHL-IFR8_R1 --outdir ../../data/chipseq_MEF2D_degraded/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = MV411_MEF2D_NT_SC_63_VHL-IFR8_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 03 Nov 2020 16:12:00: #1 read tag files... \\nINFO  @ Tue, 03 Nov 2020 16:12:00: #1 read treatment tags... \\nINFO  @ Tue, 03 Nov 2020 16:12:04:  1000000 \\nINFO  @ Tue, 03 Nov 2020 16:12:09:  2000000 \\nINFO  @ Tue, 03 Nov 2020 16:12:14:  3000000 \\nINFO  @ Tue, 03 Nov 2020 16:12:19:  4000000 \\nINFO  @ Tue, 03 Nov 2020 16:12:23:  5000000 \\nINFO  @ Tue, 03 Nov 2020 16:12:27:  6000000 \\nINFO  @ Tue, 03 Nov 2020 16:12:32:  7000000 \\nINFO  @ Tue, 03 Nov 2020 16:12:36:  8000000 \\nINFO  @ Tue, 03 Nov 2020 16:12:41:  9000000 \\nINFO  @ Tue, 03 Nov 2020 16:12:45:  10000000 \\nINFO  @ Tue, 03 Nov 2020 16:12:50:  11000000 \\nINFO  @ Tue, 03 Nov 2020 16:12:54:  12000000 \\nINFO  @ Tue, 03 Nov 2020 16:12:58:  13000000 \\nINFO  @ Tue, 03 Nov 2020 16:13:02:  14000000 \\nINFO  @ Tue, 03 Nov 2020 16:13:06:  15000000 \\nINFO  @ Tue, 03 Nov 2020 16:13:11:  16000000 \\nINFO  @ Tue, 03 Nov 2020 16:13:15:  17000000 \\nINFO  @ Tue, 03 Nov 2020 16:13:20:  18000000 \\nINFO  @ Tue, 03 Nov 2020 16:13:24:  19000000 \\nINFO  @ Tue, 03 Nov 2020 16:13:29:  20000000 \\nINFO  @ Tue, 03 Nov 2020 16:13:33:  21000000 \\nINFO  @ Tue, 03 Nov 2020 16:13:37:  22000000 \\nINFO  @ Tue, 03 Nov 2020 16:13:42:  23000000 \\nINFO  @ Tue, 03 Nov 2020 16:13:46:  24000000 \\nINFO  @ Tue, 03 Nov 2020 16:13:51:  25000000 \\nINFO  @ Tue, 03 Nov 2020 16:13:55:  26000000 \\nINFO  @ Tue, 03 Nov 2020 16:13:59:  27000000 \\nINFO  @ Tue, 03 Nov 2020 16:14:04:  28000000 \\nINFO  @ Tue, 03 Nov 2020 16:14:08:  29000000 \\nINFO  @ Tue, 03 Nov 2020 16:14:13:  30000000 \\nINFO  @ Tue, 03 Nov 2020 16:14:17:  31000000 \\nINFO  @ Tue, 03 Nov 2020 16:14:22:  32000000 \\nINFO  @ Tue, 03 Nov 2020 16:14:26:  33000000 \\nINFO  @ Tue, 03 Nov 2020 16:14:31:  34000000 \\nINFO  @ Tue, 03 Nov 2020 16:14:35:  35000000 \\nINFO  @ Tue, 03 Nov 2020 16:14:39:  36000000 \\nINFO  @ Tue, 03 Nov 2020 16:14:44:  37000000 \\nINFO  @ Tue, 03 Nov 2020 16:14:49:  38000000 \\nINFO  @ Tue, 03 Nov 2020 16:14:53:  39000000 \\nINFO  @ Tue, 03 Nov 2020 16:14:58:  40000000 \\nINFO  @ Tue, 03 Nov 2020 16:15:02:  41000000 \\nINFO  @ Tue, 03 Nov 2020 16:15:06:  42000000 \\nINFO  @ Tue, 03 Nov 2020 16:15:11:  43000000 \\nINFO  @ Tue, 03 Nov 2020 16:15:15:  44000000 \\nINFO  @ Tue, 03 Nov 2020 16:15:19:  45000000 \\nINFO  @ Tue, 03 Nov 2020 16:15:24:  46000000 \\nINFO  @ Tue, 03 Nov 2020 16:15:28:  47000000 \\nINFO  @ Tue, 03 Nov 2020 16:15:33:  48000000 \\nINFO  @ Tue, 03 Nov 2020 16:15:37:  49000000 \\nINFO  @ Tue, 03 Nov 2020 16:15:41:  50000000 \\nINFO  @ Tue, 03 Nov 2020 16:15:45:  51000000 \\nINFO  @ Tue, 03 Nov 2020 16:15:50:  52000000 \\nINFO  @ Tue, 03 Nov 2020 16:15:54:  53000000 \\nINFO  @ Tue, 03 Nov 2020 16:15:59:  54000000 \\nINFO  @ Tue, 03 Nov 2020 16:16:03:  55000000 \\nINFO  @ Tue, 03 Nov 2020 16:16:08:  56000000 \\nINFO  @ Tue, 03 Nov 2020 16:16:13:  57000000 \\nINFO  @ Tue, 03 Nov 2020 16:16:17:  58000000 \\nINFO  @ Tue, 03 Nov 2020 16:16:22:  59000000 \\nINFO  @ Tue, 03 Nov 2020 16:16:26:  60000000 \\nINFO  @ Tue, 03 Nov 2020 16:16:31:  61000000 \\nINFO  @ Tue, 03 Nov 2020 16:16:36:  62000000 \\nINFO  @ Tue, 03 Nov 2020 16:16:41:  63000000 \\nINFO  @ Tue, 03 Nov 2020 16:16:45:  64000000 \\nINFO  @ Tue, 03 Nov 2020 16:16:50:  65000000 \\nINFO  @ Tue, 03 Nov 2020 16:16:54:  66000000 \\nINFO  @ Tue, 03 Nov 2020 16:16:59:  67000000 \\nINFO  @ Tue, 03 Nov 2020 16:17:04:  68000000 \\nINFO  @ Tue, 03 Nov 2020 16:17:09:  69000000 \\nINFO  @ Tue, 03 Nov 2020 16:17:11: 69337135 reads have been read. \\nINFO  @ Tue, 03 Nov 2020 16:17:12: #1.2 read input tags... \\nINFO  @ Tue, 03 Nov 2020 16:17:15:  1000000 \\nINFO  @ Tue, 03 Nov 2020 16:17:19:  2000000 \\nINFO  @ Tue, 03 Nov 2020 16:17:22:  3000000 \\nINFO  @ Tue, 03 Nov 2020 16:17:26:  4000000 \\nINFO  @ Tue, 03 Nov 2020 16:17:30:  5000000 \\nINFO  @ Tue, 03 Nov 2020 16:17:33:  6000000 \\nINFO  @ Tue, 03 Nov 2020 16:17:37:  7000000 \\nINFO  @ Tue, 03 Nov 2020 16:17:41:  8000000 \\nINFO  @ Tue, 03 Nov 2020 16:17:44:  9000000 \\nINFO  @ Tue, 03 Nov 2020 16:17:48:  10000000 \\nINFO  @ Tue, 03 Nov 2020 16:17:51:  11000000 \\nINFO  @ Tue, 03 Nov 2020 16:17:55:  12000000 \\nINFO  @ Tue, 03 Nov 2020 16:17:59:  13000000 \\nINFO  @ Tue, 03 Nov 2020 16:18:03:  14000000 \\nINFO  @ Tue, 03 Nov 2020 16:18:06:  15000000 \\nINFO  @ Tue, 03 Nov 2020 16:18:10:  16000000 \\nINFO  @ Tue, 03 Nov 2020 16:18:13:  17000000 \\nINFO  @ Tue, 03 Nov 2020 16:18:17:  18000000 \\nINFO  @ Tue, 03 Nov 2020 16:18:21:  19000000 \\nINFO  @ Tue, 03 Nov 2020 16:18:24:  20000000 \\nINFO  @ Tue, 03 Nov 2020 16:18:28:  21000000 \\nINFO  @ Tue, 03 Nov 2020 16:18:32:  22000000 \\nINFO  @ Tue, 03 Nov 2020 16:18:36:  23000000 \\nINFO  @ Tue, 03 Nov 2020 16:18:40:  24000000 \\nINFO  @ Tue, 03 Nov 2020 16:18:43:  25000000 \\nINFO  @ Tue, 03 Nov 2020 16:18:47:  26000000 \\nINFO  @ Tue, 03 Nov 2020 16:18:51:  27000000 \\nINFO  @ Tue, 03 Nov 2020 16:18:54:  28000000 \\nINFO  @ Tue, 03 Nov 2020 16:18:58:  29000000 \\nINFO  @ Tue, 03 Nov 2020 16:19:02:  30000000 \\nINFO  @ Tue, 03 Nov 2020 16:19:05:  31000000 \\nINFO  @ Tue, 03 Nov 2020 16:19:09:  32000000 \\nINFO  @ Tue, 03 Nov 2020 16:19:13:  33000000 \\nINFO  @ Tue, 03 Nov 2020 16:19:16:  34000000 \\nINFO  @ Tue, 03 Nov 2020 16:19:20:  35000000 \\nINFO  @ Tue, 03 Nov 2020 16:19:23:  36000000 \\nINFO  @ Tue, 03 Nov 2020 16:19:27:  37000000 \\nINFO  @ Tue, 03 Nov 2020 16:19:30:  38000000 \\nINFO  @ Tue, 03 Nov 2020 16:19:34:  39000000 \\nINFO  @ Tue, 03 Nov 2020 16:19:38:  40000000 \\nINFO  @ Tue, 03 Nov 2020 16:19:41:  41000000 \\nINFO  @ Tue, 03 Nov 2020 16:19:45:  42000000 \\nINFO  @ Tue, 03 Nov 2020 16:19:49:  43000000 \\nINFO  @ Tue, 03 Nov 2020 16:19:52:  44000000 \\nINFO  @ Tue, 03 Nov 2020 16:19:56:  45000000 \\nINFO  @ Tue, 03 Nov 2020 16:19:59:  46000000 \\nINFO  @ Tue, 03 Nov 2020 16:20:03:  47000000 \\nINFO  @ Tue, 03 Nov 2020 16:20:07:  48000000 \\nINFO  @ Tue, 03 Nov 2020 16:20:10:  49000000 \\nINFO  @ Tue, 03 Nov 2020 16:20:14:  50000000 \\nINFO  @ Tue, 03 Nov 2020 16:20:17:  51000000 \\nINFO  @ Tue, 03 Nov 2020 16:20:21:  52000000 \\nINFO  @ Tue, 03 Nov 2020 16:20:25:  53000000 \\nINFO  @ Tue, 03 Nov 2020 16:20:28:  54000000 \\nINFO  @ Tue, 03 Nov 2020 16:20:32:  55000000 \\nINFO  @ Tue, 03 Nov 2020 16:20:36:  56000000 \\nINFO  @ Tue, 03 Nov 2020 16:20:39:  57000000 \\nINFO  @ Tue, 03 Nov 2020 16:20:43:  58000000 \\nINFO  @ Tue, 03 Nov 2020 16:20:46:  59000000 \\nINFO  @ Tue, 03 Nov 2020 16:20:50:  60000000 \\nINFO  @ Tue, 03 Nov 2020 16:20:54:  61000000 \\nINFO  @ Tue, 03 Nov 2020 16:20:57:  62000000 \\nINFO  @ Tue, 03 Nov 2020 16:21:01:  63000000 \\nINFO  @ Tue, 03 Nov 2020 16:21:05:  64000000 \\nINFO  @ Tue, 03 Nov 2020 16:21:08:  65000000 \\nINFO  @ Tue, 03 Nov 2020 16:21:12:  66000000 \\nINFO  @ Tue, 03 Nov 2020 16:21:15: 66660428 reads have been read. \\nINFO  @ Tue, 03 Nov 2020 16:21:16: #1 tag size is determined as 96 bps \\nINFO  @ Tue, 03 Nov 2020 16:21:16: #1 tag size = 96.0 \\nINFO  @ Tue, 03 Nov 2020 16:21:16: #1  total tags in treatment: 69337135 \\nINFO  @ Tue, 03 Nov 2020 16:21:16: #1 user defined the maximum tags... \\nINFO  @ Tue, 03 Nov 2020 16:21:16: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 03 Nov 2020 16:21:16: #1  tags after filtering in treatment: 64183365 \\nINFO  @ Tue, 03 Nov 2020 16:21:16: #1  Redundant rate of treatment: 0.07 \\nINFO  @ Tue, 03 Nov 2020 16:21:16: #1  total tags in control: 66660428 \\nINFO  @ Tue, 03 Nov 2020 16:21:16: #1 user defined the maximum tags... \\nINFO  @ Tue, 03 Nov 2020 16:21:16: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 03 Nov 2020 16:21:16: #1  tags after filtering in control: 65724989 \\nINFO  @ Tue, 03 Nov 2020 16:21:16: #1  Redundant rate of control: 0.01 \\nINFO  @ Tue, 03 Nov 2020 16:21:16: #1 finished! \\nINFO  @ Tue, 03 Nov 2020 16:21:16: #2 Build Peak Model... \\nINFO  @ Tue, 03 Nov 2020 16:21:16: #2 Skipped... \\nINFO  @ Tue, 03 Nov 2020 16:21:16: #2 Use 208 as fragment length \\nINFO  @ Tue, 03 Nov 2020 16:21:16: #3 Call peaks... \\nINFO  @ Tue, 03 Nov 2020 16:21:16: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 03 Nov 2020 16:27:12: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 03 Nov 2020 16:27:12: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_treat_pileup.bdg \\nINFO  @ Tue, 03 Nov 2020 16:27:12: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_control_lambda.bdg \\nINFO  @ Tue, 03 Nov 2020 16:27:12: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Tue, 03 Nov 2020 16:27:12: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 03 Nov 2020 16:32:14: #4 Write output xls file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_peaks.xls \\nINFO  @ Tue, 03 Nov 2020 16:32:14: #4 Write peak in narrowPeak format file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_peaks.narrowPeak \\nINFO  @ Tue, 03 Nov 2020 16:32:14: #4 Write summits bed file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_summits.bed \\nINFO  @ Tue, 03 Nov 2020 16:32:15: Done! \\n'\n",
      "43197324 64183365\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1.mLb.clN.sorted.bam ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-MED1_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1.mLb.clN.sorted.bam and ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-MED1_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "220\n",
      "computing the scaling values\n",
      "b'INFO  @ Tue, 03 Nov 2020 16:51:34: \\n# Command line: callpeak -B -t ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1.mLb.clN.sorted.bam -c ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 220 -n MV411_MEF2D_NT_SC_63_DMSO-MED1_R1 --outdir ../../data/chipseq_MEF2D_degraded/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = MV411_MEF2D_NT_SC_63_DMSO-MED1_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 03 Nov 2020 16:51:34: #1 read tag files... \\nINFO  @ Tue, 03 Nov 2020 16:51:34: #1 read treatment tags... \\nINFO  @ Tue, 03 Nov 2020 16:51:38:  1000000 \\nINFO  @ Tue, 03 Nov 2020 16:51:43:  2000000 \\nINFO  @ Tue, 03 Nov 2020 16:51:47:  3000000 \\nINFO  @ Tue, 03 Nov 2020 16:51:52:  4000000 \\nINFO  @ Tue, 03 Nov 2020 16:51:56:  5000000 \\nINFO  @ Tue, 03 Nov 2020 16:52:01:  6000000 \\nINFO  @ Tue, 03 Nov 2020 16:52:05:  7000000 \\nINFO  @ Tue, 03 Nov 2020 16:52:10:  8000000 \\nINFO  @ Tue, 03 Nov 2020 16:52:14:  9000000 \\nINFO  @ Tue, 03 Nov 2020 16:52:19:  10000000 \\nINFO  @ Tue, 03 Nov 2020 16:52:23:  11000000 \\nINFO  @ Tue, 03 Nov 2020 16:52:27:  12000000 \\nINFO  @ Tue, 03 Nov 2020 16:52:32:  13000000 \\nINFO  @ Tue, 03 Nov 2020 16:52:36:  14000000 \\nINFO  @ Tue, 03 Nov 2020 16:52:41:  15000000 \\nINFO  @ Tue, 03 Nov 2020 16:52:45:  16000000 \\nINFO  @ Tue, 03 Nov 2020 16:52:49:  17000000 \\nINFO  @ Tue, 03 Nov 2020 16:52:54:  18000000 \\nINFO  @ Tue, 03 Nov 2020 16:52:59:  19000000 \\nINFO  @ Tue, 03 Nov 2020 16:53:03:  20000000 \\nINFO  @ Tue, 03 Nov 2020 16:53:08:  21000000 \\nINFO  @ Tue, 03 Nov 2020 16:53:12:  22000000 \\nINFO  @ Tue, 03 Nov 2020 16:53:16:  23000000 \\nINFO  @ Tue, 03 Nov 2020 16:53:21:  24000000 \\nINFO  @ Tue, 03 Nov 2020 16:53:25:  25000000 \\nINFO  @ Tue, 03 Nov 2020 16:53:29:  26000000 \\nINFO  @ Tue, 03 Nov 2020 16:53:34:  27000000 \\nINFO  @ Tue, 03 Nov 2020 16:53:38:  28000000 \\nINFO  @ Tue, 03 Nov 2020 16:53:42:  29000000 \\nINFO  @ Tue, 03 Nov 2020 16:53:47:  30000000 \\nINFO  @ Tue, 03 Nov 2020 16:53:51:  31000000 \\nINFO  @ Tue, 03 Nov 2020 16:53:55:  32000000 \\nINFO  @ Tue, 03 Nov 2020 16:54:00:  33000000 \\nINFO  @ Tue, 03 Nov 2020 16:54:04:  34000000 \\nINFO  @ Tue, 03 Nov 2020 16:54:09:  35000000 \\nINFO  @ Tue, 03 Nov 2020 16:54:13:  36000000 \\nINFO  @ Tue, 03 Nov 2020 16:54:17:  37000000 \\nINFO  @ Tue, 03 Nov 2020 16:54:22:  38000000 \\nINFO  @ Tue, 03 Nov 2020 16:54:26:  39000000 \\nINFO  @ Tue, 03 Nov 2020 16:54:30:  40000000 \\nINFO  @ Tue, 03 Nov 2020 16:54:35:  41000000 \\nINFO  @ Tue, 03 Nov 2020 16:54:39:  42000000 \\nINFO  @ Tue, 03 Nov 2020 16:54:43:  43000000 \\nINFO  @ Tue, 03 Nov 2020 16:54:48:  44000000 \\nINFO  @ Tue, 03 Nov 2020 16:54:52:  45000000 \\nINFO  @ Tue, 03 Nov 2020 16:54:57:  46000000 \\nINFO  @ Tue, 03 Nov 2020 16:55:01:  47000000 \\nINFO  @ Tue, 03 Nov 2020 16:55:05:  48000000 \\nINFO  @ Tue, 03 Nov 2020 16:55:10:  49000000 \\nINFO  @ Tue, 03 Nov 2020 16:55:14:  50000000 \\nINFO  @ Tue, 03 Nov 2020 16:55:18:  51000000 \\nINFO  @ Tue, 03 Nov 2020 16:55:23:  52000000 \\nINFO  @ Tue, 03 Nov 2020 16:55:27:  53000000 \\nINFO  @ Tue, 03 Nov 2020 16:55:32:  54000000 \\nINFO  @ Tue, 03 Nov 2020 16:55:36:  55000000 \\nINFO  @ Tue, 03 Nov 2020 16:55:37: 55243421 reads have been read. \\nINFO  @ Tue, 03 Nov 2020 16:55:38: #1.2 read input tags... \\nINFO  @ Tue, 03 Nov 2020 16:55:42:  1000000 \\nINFO  @ Tue, 03 Nov 2020 16:55:45:  2000000 \\nINFO  @ Tue, 03 Nov 2020 16:55:49:  3000000 \\nINFO  @ Tue, 03 Nov 2020 16:55:53:  4000000 \\nINFO  @ Tue, 03 Nov 2020 16:55:56:  5000000 \\nINFO  @ Tue, 03 Nov 2020 16:56:00:  6000000 \\nINFO  @ Tue, 03 Nov 2020 16:56:04:  7000000 \\nINFO  @ Tue, 03 Nov 2020 16:56:09:  8000000 \\nINFO  @ Tue, 03 Nov 2020 16:56:13:  9000000 \\nINFO  @ Tue, 03 Nov 2020 16:56:16:  10000000 \\nINFO  @ Tue, 03 Nov 2020 16:56:20:  11000000 \\nINFO  @ Tue, 03 Nov 2020 16:56:23:  12000000 \\nINFO  @ Tue, 03 Nov 2020 16:56:27:  13000000 \\nINFO  @ Tue, 03 Nov 2020 16:56:31:  14000000 \\nINFO  @ Tue, 03 Nov 2020 16:56:34:  15000000 \\nINFO  @ Tue, 03 Nov 2020 16:56:38:  16000000 \\nINFO  @ Tue, 03 Nov 2020 16:56:41:  17000000 \\nINFO  @ Tue, 03 Nov 2020 16:56:45:  18000000 \\nINFO  @ Tue, 03 Nov 2020 16:56:49:  19000000 \\nINFO  @ Tue, 03 Nov 2020 16:56:53:  20000000 \\nINFO  @ Tue, 03 Nov 2020 16:56:56:  21000000 \\nINFO  @ Tue, 03 Nov 2020 16:57:00:  22000000 \\nINFO  @ Tue, 03 Nov 2020 16:57:03:  23000000 \\nINFO  @ Tue, 03 Nov 2020 16:57:07:  24000000 \\nINFO  @ Tue, 03 Nov 2020 16:57:11:  25000000 \\nINFO  @ Tue, 03 Nov 2020 16:57:14:  26000000 \\nINFO  @ Tue, 03 Nov 2020 16:57:18:  27000000 \\nINFO  @ Tue, 03 Nov 2020 16:57:22:  28000000 \\nINFO  @ Tue, 03 Nov 2020 16:57:25:  29000000 \\nINFO  @ Tue, 03 Nov 2020 16:57:29:  30000000 \\nINFO  @ Tue, 03 Nov 2020 16:57:32:  31000000 \\nINFO  @ Tue, 03 Nov 2020 16:57:36:  32000000 \\nINFO  @ Tue, 03 Nov 2020 16:57:40:  33000000 \\nINFO  @ Tue, 03 Nov 2020 16:57:43:  34000000 \\nINFO  @ Tue, 03 Nov 2020 16:57:47:  35000000 \\nINFO  @ Tue, 03 Nov 2020 16:57:50:  36000000 \\nINFO  @ Tue, 03 Nov 2020 16:57:54:  37000000 \\nINFO  @ Tue, 03 Nov 2020 16:57:57:  38000000 \\nINFO  @ Tue, 03 Nov 2020 16:58:01:  39000000 \\nINFO  @ Tue, 03 Nov 2020 16:58:05:  40000000 \\nINFO  @ Tue, 03 Nov 2020 16:58:08:  41000000 \\nINFO  @ Tue, 03 Nov 2020 16:58:12:  42000000 \\nINFO  @ Tue, 03 Nov 2020 16:58:16:  43000000 \\nINFO  @ Tue, 03 Nov 2020 16:58:20:  44000000 \\nINFO  @ Tue, 03 Nov 2020 16:58:23:  45000000 \\nINFO  @ Tue, 03 Nov 2020 16:58:27:  46000000 \\nINFO  @ Tue, 03 Nov 2020 16:58:31:  47000000 \\nINFO  @ Tue, 03 Nov 2020 16:58:34:  48000000 \\nINFO  @ Tue, 03 Nov 2020 16:58:38:  49000000 \\nINFO  @ Tue, 03 Nov 2020 16:58:42:  50000000 \\nINFO  @ Tue, 03 Nov 2020 16:58:45:  51000000 \\nINFO  @ Tue, 03 Nov 2020 16:58:49:  52000000 \\nINFO  @ Tue, 03 Nov 2020 16:58:53:  53000000 \\nINFO  @ Tue, 03 Nov 2020 16:58:56:  54000000 \\nINFO  @ Tue, 03 Nov 2020 16:59:00:  55000000 \\nINFO  @ Tue, 03 Nov 2020 16:59:04:  56000000 \\nINFO  @ Tue, 03 Nov 2020 16:59:07:  57000000 \\nINFO  @ Tue, 03 Nov 2020 16:59:11:  58000000 \\nINFO  @ Tue, 03 Nov 2020 16:59:15:  59000000 \\nINFO  @ Tue, 03 Nov 2020 16:59:18:  60000000 \\nINFO  @ Tue, 03 Nov 2020 16:59:22:  61000000 \\nINFO  @ Tue, 03 Nov 2020 16:59:26:  62000000 \\nINFO  @ Tue, 03 Nov 2020 16:59:30:  63000000 \\nINFO  @ Tue, 03 Nov 2020 16:59:33:  64000000 \\nINFO  @ Tue, 03 Nov 2020 16:59:37:  65000000 \\nINFO  @ Tue, 03 Nov 2020 16:59:41:  66000000 \\nINFO  @ Tue, 03 Nov 2020 16:59:43: 66660428 reads have been read. \\nINFO  @ Tue, 03 Nov 2020 16:59:44: #1 tag size is determined as 96 bps \\nINFO  @ Tue, 03 Nov 2020 16:59:44: #1 tag size = 96.0 \\nINFO  @ Tue, 03 Nov 2020 16:59:44: #1  total tags in treatment: 55243421 \\nINFO  @ Tue, 03 Nov 2020 16:59:44: #1 user defined the maximum tags... \\nINFO  @ Tue, 03 Nov 2020 16:59:44: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 03 Nov 2020 16:59:44: #1  tags after filtering in treatment: 54478100 \\nINFO  @ Tue, 03 Nov 2020 16:59:44: #1  Redundant rate of treatment: 0.01 \\nINFO  @ Tue, 03 Nov 2020 16:59:44: #1  total tags in control: 66660428 \\nINFO  @ Tue, 03 Nov 2020 16:59:44: #1 user defined the maximum tags... \\nINFO  @ Tue, 03 Nov 2020 16:59:44: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 03 Nov 2020 16:59:45: #1  tags after filtering in control: 65724989 \\nINFO  @ Tue, 03 Nov 2020 16:59:45: #1  Redundant rate of control: 0.01 \\nINFO  @ Tue, 03 Nov 2020 16:59:45: #1 finished! \\nINFO  @ Tue, 03 Nov 2020 16:59:45: #2 Build Peak Model... \\nINFO  @ Tue, 03 Nov 2020 16:59:45: #2 Skipped... \\nINFO  @ Tue, 03 Nov 2020 16:59:45: #2 Use 220 as fragment length \\nINFO  @ Tue, 03 Nov 2020 16:59:45: #3 Call peaks... \\nINFO  @ Tue, 03 Nov 2020 16:59:45: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 03 Nov 2020 17:05:13: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 03 Nov 2020 17:05:13: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_treat_pileup.bdg \\nINFO  @ Tue, 03 Nov 2020 17:05:13: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_control_lambda.bdg \\nINFO  @ Tue, 03 Nov 2020 17:05:13: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Tue, 03 Nov 2020 17:05:13: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 03 Nov 2020 17:09:56: #4 Write output xls file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_peaks.xls \\nINFO  @ Tue, 03 Nov 2020 17:09:56: #4 Write peak in narrowPeak format file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_peaks.narrowPeak \\nINFO  @ Tue, 03 Nov 2020 17:09:56: #4 Write summits bed file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_summits.bed \\nINFO  @ Tue, 03 Nov 2020 17:09:56: Done! \\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'INFO  @ Tue, 03 Nov 2020 17:09:57: \\n# Command line: callpeak -B -t ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-MED1_R1.mLb.clN.sorted.bam -c ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 220 -n MV411_MEF2D_NT_SC_63_VHL-MED1_R1 --outdir ../../data/chipseq_MEF2D_degraded/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = MV411_MEF2D_NT_SC_63_VHL-MED1_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-MED1_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 03 Nov 2020 17:09:57: #1 read tag files... \\nINFO  @ Tue, 03 Nov 2020 17:09:57: #1 read treatment tags... \\nINFO  @ Tue, 03 Nov 2020 17:10:01:  1000000 \\nINFO  @ Tue, 03 Nov 2020 17:10:06:  2000000 \\nINFO  @ Tue, 03 Nov 2020 17:10:10:  3000000 \\nINFO  @ Tue, 03 Nov 2020 17:10:15:  4000000 \\nINFO  @ Tue, 03 Nov 2020 17:10:19:  5000000 \\nINFO  @ Tue, 03 Nov 2020 17:10:24:  6000000 \\nINFO  @ Tue, 03 Nov 2020 17:10:29:  7000000 \\nINFO  @ Tue, 03 Nov 2020 17:10:33:  8000000 \\nINFO  @ Tue, 03 Nov 2020 17:10:38:  9000000 \\nINFO  @ Tue, 03 Nov 2020 17:10:42:  10000000 \\nINFO  @ Tue, 03 Nov 2020 17:10:47:  11000000 \\nINFO  @ Tue, 03 Nov 2020 17:10:51:  12000000 \\nINFO  @ Tue, 03 Nov 2020 17:10:55:  13000000 \\nINFO  @ Tue, 03 Nov 2020 17:11:00:  14000000 \\nINFO  @ Tue, 03 Nov 2020 17:11:04:  15000000 \\nINFO  @ Tue, 03 Nov 2020 17:11:09:  16000000 \\nINFO  @ Tue, 03 Nov 2020 17:11:13:  17000000 \\nINFO  @ Tue, 03 Nov 2020 17:11:17:  18000000 \\nINFO  @ Tue, 03 Nov 2020 17:11:22:  19000000 \\nINFO  @ Tue, 03 Nov 2020 17:11:26:  20000000 \\nINFO  @ Tue, 03 Nov 2020 17:11:31:  21000000 \\nINFO  @ Tue, 03 Nov 2020 17:11:35:  22000000 \\nINFO  @ Tue, 03 Nov 2020 17:11:40:  23000000 \\nINFO  @ Tue, 03 Nov 2020 17:11:44:  24000000 \\nINFO  @ Tue, 03 Nov 2020 17:11:49:  25000000 \\nINFO  @ Tue, 03 Nov 2020 17:11:53:  26000000 \\nINFO  @ Tue, 03 Nov 2020 17:11:58:  27000000 \\nINFO  @ Tue, 03 Nov 2020 17:12:02:  28000000 \\nINFO  @ Tue, 03 Nov 2020 17:12:06:  29000000 \\nINFO  @ Tue, 03 Nov 2020 17:12:11:  30000000 \\nINFO  @ Tue, 03 Nov 2020 17:12:16:  31000000 \\nINFO  @ Tue, 03 Nov 2020 17:12:20:  32000000 \\nINFO  @ Tue, 03 Nov 2020 17:12:24:  33000000 \\nINFO  @ Tue, 03 Nov 2020 17:12:29:  34000000 \\nINFO  @ Tue, 03 Nov 2020 17:12:33:  35000000 \\nINFO  @ Tue, 03 Nov 2020 17:12:38:  36000000 \\nINFO  @ Tue, 03 Nov 2020 17:12:42:  37000000 \\nINFO  @ Tue, 03 Nov 2020 17:12:47:  38000000 \\nINFO  @ Tue, 03 Nov 2020 17:12:51:  39000000 \\nINFO  @ Tue, 03 Nov 2020 17:12:55:  40000000 \\nINFO  @ Tue, 03 Nov 2020 17:13:00:  41000000 \\nINFO  @ Tue, 03 Nov 2020 17:13:04:  42000000 \\nINFO  @ Tue, 03 Nov 2020 17:13:09:  43000000 \\nINFO  @ Tue, 03 Nov 2020 17:13:13:  44000000 \\nINFO  @ Tue, 03 Nov 2020 17:13:17:  45000000 \\nINFO  @ Tue, 03 Nov 2020 17:13:22:  46000000 \\nINFO  @ Tue, 03 Nov 2020 17:13:26:  47000000 \\nINFO  @ Tue, 03 Nov 2020 17:13:31:  48000000 \\nINFO  @ Tue, 03 Nov 2020 17:13:36:  49000000 \\nINFO  @ Tue, 03 Nov 2020 17:13:41:  50000000 \\nINFO  @ Tue, 03 Nov 2020 17:13:46:  51000000 \\nINFO  @ Tue, 03 Nov 2020 17:13:50:  52000000 \\nINFO  @ Tue, 03 Nov 2020 17:13:55:  53000000 \\nINFO  @ Tue, 03 Nov 2020 17:13:59:  54000000 \\nINFO  @ Tue, 03 Nov 2020 17:14:04:  55000000 \\nINFO  @ Tue, 03 Nov 2020 17:14:09:  56000000 \\nINFO  @ Tue, 03 Nov 2020 17:14:13:  57000000 \\nINFO  @ Tue, 03 Nov 2020 17:14:18:  58000000 \\nINFO  @ Tue, 03 Nov 2020 17:14:22:  59000000 \\nINFO  @ Tue, 03 Nov 2020 17:14:27:  60000000 \\nINFO  @ Tue, 03 Nov 2020 17:14:31:  61000000 \\nINFO  @ Tue, 03 Nov 2020 17:14:36:  62000000 \\nINFO  @ Tue, 03 Nov 2020 17:14:40:  63000000 \\nINFO  @ Tue, 03 Nov 2020 17:14:45:  64000000 \\nINFO  @ Tue, 03 Nov 2020 17:14:50:  65000000 \\nINFO  @ Tue, 03 Nov 2020 17:14:54:  66000000 \\nINFO  @ Tue, 03 Nov 2020 17:14:59:  67000000 \\nINFO  @ Tue, 03 Nov 2020 17:15:03:  68000000 \\nINFO  @ Tue, 03 Nov 2020 17:15:08:  69000000 \\nINFO  @ Tue, 03 Nov 2020 17:15:13:  70000000 \\nINFO  @ Tue, 03 Nov 2020 17:15:17:  71000000 \\nINFO  @ Tue, 03 Nov 2020 17:15:22:  72000000 \\nINFO  @ Tue, 03 Nov 2020 17:15:27:  73000000 \\nINFO  @ Tue, 03 Nov 2020 17:15:31:  74000000 \\nINFO  @ Tue, 03 Nov 2020 17:15:36:  75000000 \\nINFO  @ Tue, 03 Nov 2020 17:15:40:  76000000 \\nINFO  @ Tue, 03 Nov 2020 17:15:45:  77000000 \\nINFO  @ Tue, 03 Nov 2020 17:15:50:  78000000 \\nINFO  @ Tue, 03 Nov 2020 17:15:54:  79000000 \\nINFO  @ Tue, 03 Nov 2020 17:15:59:  80000000 \\nINFO  @ Tue, 03 Nov 2020 17:16:04:  81000000 \\nINFO  @ Tue, 03 Nov 2020 17:16:09:  82000000 \\nINFO  @ Tue, 03 Nov 2020 17:16:14:  83000000 \\nINFO  @ Tue, 03 Nov 2020 17:16:14: 83009589 reads have been read. \\nINFO  @ Tue, 03 Nov 2020 17:16:15: #1.2 read input tags... \\nINFO  @ Tue, 03 Nov 2020 17:16:18:  1000000 \\nINFO  @ Tue, 03 Nov 2020 17:16:22:  2000000 \\nINFO  @ Tue, 03 Nov 2020 17:16:26:  3000000 \\nINFO  @ Tue, 03 Nov 2020 17:16:29:  4000000 \\nINFO  @ Tue, 03 Nov 2020 17:16:33:  5000000 \\nINFO  @ Tue, 03 Nov 2020 17:16:36:  6000000 \\nINFO  @ Tue, 03 Nov 2020 17:16:40:  7000000 \\nINFO  @ Tue, 03 Nov 2020 17:16:44:  8000000 \\nINFO  @ Tue, 03 Nov 2020 17:16:47:  9000000 \\nINFO  @ Tue, 03 Nov 2020 17:16:51:  10000000 \\nINFO  @ Tue, 03 Nov 2020 17:16:55:  11000000 \\nINFO  @ Tue, 03 Nov 2020 17:16:58:  12000000 \\nINFO  @ Tue, 03 Nov 2020 17:17:02:  13000000 \\nINFO  @ Tue, 03 Nov 2020 17:17:06:  14000000 \\nINFO  @ Tue, 03 Nov 2020 17:17:09:  15000000 \\nINFO  @ Tue, 03 Nov 2020 17:17:13:  16000000 \\nINFO  @ Tue, 03 Nov 2020 17:17:16:  17000000 \\nINFO  @ Tue, 03 Nov 2020 17:17:20:  18000000 \\nINFO  @ Tue, 03 Nov 2020 17:17:24:  19000000 \\nINFO  @ Tue, 03 Nov 2020 17:17:27:  20000000 \\nINFO  @ Tue, 03 Nov 2020 17:17:31:  21000000 \\nINFO  @ Tue, 03 Nov 2020 17:17:35:  22000000 \\nINFO  @ Tue, 03 Nov 2020 17:17:38:  23000000 \\nINFO  @ Tue, 03 Nov 2020 17:17:42:  24000000 \\nINFO  @ Tue, 03 Nov 2020 17:17:46:  25000000 \\nINFO  @ Tue, 03 Nov 2020 17:17:49:  26000000 \\nINFO  @ Tue, 03 Nov 2020 17:17:53:  27000000 \\nINFO  @ Tue, 03 Nov 2020 17:17:57:  28000000 \\nINFO  @ Tue, 03 Nov 2020 17:18:00:  29000000 \\nINFO  @ Tue, 03 Nov 2020 17:18:04:  30000000 \\nINFO  @ Tue, 03 Nov 2020 17:18:08:  31000000 \\nINFO  @ Tue, 03 Nov 2020 17:18:11:  32000000 \\nINFO  @ Tue, 03 Nov 2020 17:18:15:  33000000 \\nINFO  @ Tue, 03 Nov 2020 17:18:18:  34000000 \\nINFO  @ Tue, 03 Nov 2020 17:18:22:  35000000 \\nINFO  @ Tue, 03 Nov 2020 17:18:26:  36000000 \\nINFO  @ Tue, 03 Nov 2020 17:18:29:  37000000 \\nINFO  @ Tue, 03 Nov 2020 17:18:33:  38000000 \\nINFO  @ Tue, 03 Nov 2020 17:18:36:  39000000 \\nINFO  @ Tue, 03 Nov 2020 17:18:40:  40000000 \\nINFO  @ Tue, 03 Nov 2020 17:18:44:  41000000 \\nINFO  @ Tue, 03 Nov 2020 17:18:48:  42000000 \\nINFO  @ Tue, 03 Nov 2020 17:18:51:  43000000 \\nINFO  @ Tue, 03 Nov 2020 17:18:55:  44000000 \\nINFO  @ Tue, 03 Nov 2020 17:18:59:  45000000 \\nINFO  @ Tue, 03 Nov 2020 17:19:02:  46000000 \\nINFO  @ Tue, 03 Nov 2020 17:19:06:  47000000 \\nINFO  @ Tue, 03 Nov 2020 17:19:10:  48000000 \\nINFO  @ Tue, 03 Nov 2020 17:19:13:  49000000 \\nINFO  @ Tue, 03 Nov 2020 17:19:17:  50000000 \\nINFO  @ Tue, 03 Nov 2020 17:19:20:  51000000 \\nINFO  @ Tue, 03 Nov 2020 17:19:24:  52000000 \\nINFO  @ Tue, 03 Nov 2020 17:19:28:  53000000 \\nINFO  @ Tue, 03 Nov 2020 17:19:31:  54000000 \\nINFO  @ Tue, 03 Nov 2020 17:19:35:  55000000 \\nINFO  @ Tue, 03 Nov 2020 17:19:39:  56000000 \\nINFO  @ Tue, 03 Nov 2020 17:19:42:  57000000 \\nINFO  @ Tue, 03 Nov 2020 17:19:46:  58000000 \\nINFO  @ Tue, 03 Nov 2020 17:19:50:  59000000 \\nINFO  @ Tue, 03 Nov 2020 17:19:53:  60000000 \\nINFO  @ Tue, 03 Nov 2020 17:19:57:  61000000 \\nINFO  @ Tue, 03 Nov 2020 17:20:01:  62000000 \\nINFO  @ Tue, 03 Nov 2020 17:20:04:  63000000 \\nINFO  @ Tue, 03 Nov 2020 17:20:08:  64000000 \\nINFO  @ Tue, 03 Nov 2020 17:20:12:  65000000 \\nINFO  @ Tue, 03 Nov 2020 17:20:16:  66000000 \\nINFO  @ Tue, 03 Nov 2020 17:20:19: 66660428 reads have been read. \\nINFO  @ Tue, 03 Nov 2020 17:20:19: #1 tag size is determined as 98 bps \\nINFO  @ Tue, 03 Nov 2020 17:20:19: #1 tag size = 98.0 \\nINFO  @ Tue, 03 Nov 2020 17:20:19: #1  total tags in treatment: 83009589 \\nINFO  @ Tue, 03 Nov 2020 17:20:19: #1 user defined the maximum tags... \\nINFO  @ Tue, 03 Nov 2020 17:20:19: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 03 Nov 2020 17:20:20: #1  tags after filtering in treatment: 81620483 \\nINFO  @ Tue, 03 Nov 2020 17:20:20: #1  Redundant rate of treatment: 0.02 \\nINFO  @ Tue, 03 Nov 2020 17:20:20: #1  total tags in control: 66660428 \\nINFO  @ Tue, 03 Nov 2020 17:20:20: #1 user defined the maximum tags... \\nINFO  @ Tue, 03 Nov 2020 17:20:20: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 03 Nov 2020 17:20:20: #1  tags after filtering in control: 65724989 \\nINFO  @ Tue, 03 Nov 2020 17:20:20: #1  Redundant rate of control: 0.01 \\nINFO  @ Tue, 03 Nov 2020 17:20:20: #1 finished! \\nINFO  @ Tue, 03 Nov 2020 17:20:20: #2 Build Peak Model... \\nINFO  @ Tue, 03 Nov 2020 17:20:20: #2 Skipped... \\nINFO  @ Tue, 03 Nov 2020 17:20:20: #2 Use 220 as fragment length \\nINFO  @ Tue, 03 Nov 2020 17:20:20: #3 Call peaks... \\nINFO  @ Tue, 03 Nov 2020 17:20:20: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 03 Nov 2020 17:26:37: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 03 Nov 2020 17:26:37: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_MEF2D_NT_SC_63_VHL-MED1_R1_treat_pileup.bdg \\nINFO  @ Tue, 03 Nov 2020 17:26:37: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_MEF2D_NT_SC_63_VHL-MED1_R1_control_lambda.bdg \\nINFO  @ Tue, 03 Nov 2020 17:26:37: #3   Pileup will be based on sequencing depth in control. \\nINFO  @ Tue, 03 Nov 2020 17:26:37: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 03 Nov 2020 17:31:46: #4 Write output xls file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-MED1_R1_peaks.xls \\nINFO  @ Tue, 03 Nov 2020 17:31:46: #4 Write peak in narrowPeak format file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-MED1_R1_peaks.narrowPeak \\nINFO  @ Tue, 03 Nov 2020 17:31:46: #4 Write summits bed file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-MED1_R1_summits.bed \\nINFO  @ Tue, 03 Nov 2020 17:31:46: Done! \\n'\n",
      "54478100 65724989\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1.mLb.clN.sorted.bam ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1.mLb.clN.sorted.bam and ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "229\n",
      "computing the scaling values\n",
      "b'INFO  @ Tue, 03 Nov 2020 17:55:31: \\n# Command line: callpeak -B -t ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1.mLb.clN.sorted.bam -c ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 229 -n MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1 --outdir ../../data/chipseq_MEF2D_degraded/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 03 Nov 2020 17:55:31: #1 read tag files... \\nINFO  @ Tue, 03 Nov 2020 17:55:31: #1 read treatment tags... \\nINFO  @ Tue, 03 Nov 2020 17:55:36:  1000000 \\nINFO  @ Tue, 03 Nov 2020 17:55:41:  2000000 \\nINFO  @ Tue, 03 Nov 2020 17:55:45:  3000000 \\nINFO  @ Tue, 03 Nov 2020 17:55:50:  4000000 \\nINFO  @ Tue, 03 Nov 2020 17:55:54:  5000000 \\nINFO  @ Tue, 03 Nov 2020 17:55:58:  6000000 \\nINFO  @ Tue, 03 Nov 2020 17:56:03:  7000000 \\nINFO  @ Tue, 03 Nov 2020 17:56:07:  8000000 \\nINFO  @ Tue, 03 Nov 2020 17:56:12:  9000000 \\nINFO  @ Tue, 03 Nov 2020 17:56:16:  10000000 \\nINFO  @ Tue, 03 Nov 2020 17:56:20:  11000000 \\nINFO  @ Tue, 03 Nov 2020 17:56:25:  12000000 \\nINFO  @ Tue, 03 Nov 2020 17:56:29:  13000000 \\nINFO  @ Tue, 03 Nov 2020 17:56:33:  14000000 \\nINFO  @ Tue, 03 Nov 2020 17:56:37:  15000000 \\nINFO  @ Tue, 03 Nov 2020 17:56:42:  16000000 \\nINFO  @ Tue, 03 Nov 2020 17:56:46:  17000000 \\nINFO  @ Tue, 03 Nov 2020 17:56:50:  18000000 \\nINFO  @ Tue, 03 Nov 2020 17:56:55:  19000000 \\nINFO  @ Tue, 03 Nov 2020 17:56:59:  20000000 \\nINFO  @ Tue, 03 Nov 2020 17:57:03:  21000000 \\nINFO  @ Tue, 03 Nov 2020 17:57:07:  22000000 \\nINFO  @ Tue, 03 Nov 2020 17:57:12:  23000000 \\nINFO  @ Tue, 03 Nov 2020 17:57:16:  24000000 \\nINFO  @ Tue, 03 Nov 2020 17:57:20:  25000000 \\nINFO  @ Tue, 03 Nov 2020 17:57:24:  26000000 \\nINFO  @ Tue, 03 Nov 2020 17:57:29:  27000000 \\nINFO  @ Tue, 03 Nov 2020 17:57:33:  28000000 \\nINFO  @ Tue, 03 Nov 2020 17:57:37:  29000000 \\nINFO  @ Tue, 03 Nov 2020 17:57:41:  30000000 \\nINFO  @ Tue, 03 Nov 2020 17:57:46:  31000000 \\nINFO  @ Tue, 03 Nov 2020 17:57:50:  32000000 \\nINFO  @ Tue, 03 Nov 2020 17:57:54:  33000000 \\nINFO  @ Tue, 03 Nov 2020 17:57:58:  34000000 \\nINFO  @ Tue, 03 Nov 2020 17:58:03:  35000000 \\nINFO  @ Tue, 03 Nov 2020 17:58:07:  36000000 \\nINFO  @ Tue, 03 Nov 2020 17:58:11:  37000000 \\nINFO  @ Tue, 03 Nov 2020 17:58:16:  38000000 \\nINFO  @ Tue, 03 Nov 2020 17:58:20:  39000000 \\nINFO  @ Tue, 03 Nov 2020 17:58:24:  40000000 \\nINFO  @ Tue, 03 Nov 2020 17:58:28:  41000000 \\nINFO  @ Tue, 03 Nov 2020 17:58:33:  42000000 \\nINFO  @ Tue, 03 Nov 2020 17:58:37:  43000000 \\nINFO  @ Tue, 03 Nov 2020 17:58:41:  44000000 \\nINFO  @ Tue, 03 Nov 2020 17:58:46:  45000000 \\nINFO  @ Tue, 03 Nov 2020 17:58:50:  46000000 \\nINFO  @ Tue, 03 Nov 2020 17:58:54:  47000000 \\nINFO  @ Tue, 03 Nov 2020 17:58:59:  48000000 \\nINFO  @ Tue, 03 Nov 2020 17:59:03:  49000000 \\nINFO  @ Tue, 03 Nov 2020 17:59:07:  50000000 \\nINFO  @ Tue, 03 Nov 2020 17:59:12:  51000000 \\nINFO  @ Tue, 03 Nov 2020 17:59:16:  52000000 \\nINFO  @ Tue, 03 Nov 2020 17:59:21:  53000000 \\nINFO  @ Tue, 03 Nov 2020 17:59:25:  54000000 \\nINFO  @ Tue, 03 Nov 2020 17:59:29:  55000000 \\nINFO  @ Tue, 03 Nov 2020 17:59:34:  56000000 \\nINFO  @ Tue, 03 Nov 2020 17:59:38:  57000000 \\nINFO  @ Tue, 03 Nov 2020 17:59:42:  58000000 \\nINFO  @ Tue, 03 Nov 2020 17:59:47:  59000000 \\nINFO  @ Tue, 03 Nov 2020 17:59:51:  60000000 \\nINFO  @ Tue, 03 Nov 2020 17:59:56:  61000000 \\nINFO  @ Tue, 03 Nov 2020 18:00:00:  62000000 \\nINFO  @ Tue, 03 Nov 2020 18:00:04:  63000000 \\nINFO  @ Tue, 03 Nov 2020 18:00:09:  64000000 \\nINFO  @ Tue, 03 Nov 2020 18:00:13:  65000000 \\nINFO  @ Tue, 03 Nov 2020 18:00:17:  66000000 \\nINFO  @ Tue, 03 Nov 2020 18:00:22:  67000000 \\nINFO  @ Tue, 03 Nov 2020 18:00:26:  68000000 \\nINFO  @ Tue, 03 Nov 2020 18:00:30:  69000000 \\nINFO  @ Tue, 03 Nov 2020 18:00:35:  70000000 \\nINFO  @ Tue, 03 Nov 2020 18:00:39:  71000000 \\nINFO  @ Tue, 03 Nov 2020 18:00:43:  72000000 \\nINFO  @ Tue, 03 Nov 2020 18:00:48:  73000000 \\nINFO  @ Tue, 03 Nov 2020 18:00:52:  74000000 \\nINFO  @ Tue, 03 Nov 2020 18:00:56:  75000000 \\nINFO  @ Tue, 03 Nov 2020 18:01:01:  76000000 \\nINFO  @ Tue, 03 Nov 2020 18:01:05:  77000000 \\nINFO  @ Tue, 03 Nov 2020 18:01:09:  78000000 \\nINFO  @ Tue, 03 Nov 2020 18:01:14:  79000000 \\nINFO  @ Tue, 03 Nov 2020 18:01:18:  80000000 \\nINFO  @ Tue, 03 Nov 2020 18:01:22:  81000000 \\nINFO  @ Tue, 03 Nov 2020 18:01:27:  82000000 \\nINFO  @ Tue, 03 Nov 2020 18:01:31:  83000000 \\nINFO  @ Tue, 03 Nov 2020 18:01:36:  84000000 \\nINFO  @ Tue, 03 Nov 2020 18:01:40:  85000000 \\nINFO  @ Tue, 03 Nov 2020 18:01:44:  86000000 \\nINFO  @ Tue, 03 Nov 2020 18:01:49:  87000000 \\nINFO  @ Tue, 03 Nov 2020 18:01:53:  88000000 \\nINFO  @ Tue, 03 Nov 2020 18:01:57:  89000000 \\nINFO  @ Tue, 03 Nov 2020 18:02:02:  90000000 \\nINFO  @ Tue, 03 Nov 2020 18:02:06:  91000000 \\nINFO  @ Tue, 03 Nov 2020 18:02:11:  92000000 \\nINFO  @ Tue, 03 Nov 2020 18:02:15:  93000000 \\nINFO  @ Tue, 03 Nov 2020 18:02:19:  94000000 \\nINFO  @ Tue, 03 Nov 2020 18:02:24:  95000000 \\nINFO  @ Tue, 03 Nov 2020 18:02:28:  96000000 \\nINFO  @ Tue, 03 Nov 2020 18:02:33:  97000000 \\nINFO  @ Tue, 03 Nov 2020 18:02:35: 97438650 reads have been read. \\nINFO  @ Tue, 03 Nov 2020 18:02:36: #1.2 read input tags... \\nINFO  @ Tue, 03 Nov 2020 18:02:40:  1000000 \\nINFO  @ Tue, 03 Nov 2020 18:02:43:  2000000 \\nINFO  @ Tue, 03 Nov 2020 18:02:47:  3000000 \\nINFO  @ Tue, 03 Nov 2020 18:02:51:  4000000 \\nINFO  @ Tue, 03 Nov 2020 18:02:54:  5000000 \\nINFO  @ Tue, 03 Nov 2020 18:02:58:  6000000 \\nINFO  @ Tue, 03 Nov 2020 18:03:01:  7000000 \\nINFO  @ Tue, 03 Nov 2020 18:03:05:  8000000 \\nINFO  @ Tue, 03 Nov 2020 18:03:09:  9000000 \\nINFO  @ Tue, 03 Nov 2020 18:03:12:  10000000 \\nINFO  @ Tue, 03 Nov 2020 18:03:16:  11000000 \\nINFO  @ Tue, 03 Nov 2020 18:03:20:  12000000 \\nINFO  @ Tue, 03 Nov 2020 18:03:23:  13000000 \\nINFO  @ Tue, 03 Nov 2020 18:03:27:  14000000 \\nINFO  @ Tue, 03 Nov 2020 18:03:30:  15000000 \\nINFO  @ Tue, 03 Nov 2020 18:03:34:  16000000 \\nINFO  @ Tue, 03 Nov 2020 18:03:38:  17000000 \\nINFO  @ Tue, 03 Nov 2020 18:03:41:  18000000 \\nINFO  @ Tue, 03 Nov 2020 18:03:45:  19000000 \\nINFO  @ Tue, 03 Nov 2020 18:03:48:  20000000 \\nINFO  @ Tue, 03 Nov 2020 18:03:52:  21000000 \\nINFO  @ Tue, 03 Nov 2020 18:03:56:  22000000 \\nINFO  @ Tue, 03 Nov 2020 18:03:59:  23000000 \\nINFO  @ Tue, 03 Nov 2020 18:04:03:  24000000 \\nINFO  @ Tue, 03 Nov 2020 18:04:07:  25000000 \\nINFO  @ Tue, 03 Nov 2020 18:04:10:  26000000 \\nINFO  @ Tue, 03 Nov 2020 18:04:14:  27000000 \\nINFO  @ Tue, 03 Nov 2020 18:04:17:  28000000 \\nINFO  @ Tue, 03 Nov 2020 18:04:21:  29000000 \\nINFO  @ Tue, 03 Nov 2020 18:04:25:  30000000 \\nINFO  @ Tue, 03 Nov 2020 18:04:28:  31000000 \\nINFO  @ Tue, 03 Nov 2020 18:04:32:  32000000 \\nINFO  @ Tue, 03 Nov 2020 18:04:36:  33000000 \\nINFO  @ Tue, 03 Nov 2020 18:04:39:  34000000 \\nINFO  @ Tue, 03 Nov 2020 18:04:43:  35000000 \\nINFO  @ Tue, 03 Nov 2020 18:04:46:  36000000 \\nINFO  @ Tue, 03 Nov 2020 18:04:50:  37000000 \\nINFO  @ Tue, 03 Nov 2020 18:04:53:  38000000 \\nINFO  @ Tue, 03 Nov 2020 18:04:57:  39000000 \\nINFO  @ Tue, 03 Nov 2020 18:05:01:  40000000 \\nINFO  @ Tue, 03 Nov 2020 18:05:04:  41000000 \\nINFO  @ Tue, 03 Nov 2020 18:05:08:  42000000 \\nINFO  @ Tue, 03 Nov 2020 18:05:12:  43000000 \\nINFO  @ Tue, 03 Nov 2020 18:05:15:  44000000 \\nINFO  @ Tue, 03 Nov 2020 18:05:19:  45000000 \\nINFO  @ Tue, 03 Nov 2020 18:05:23:  46000000 \\nINFO  @ Tue, 03 Nov 2020 18:05:26:  47000000 \\nINFO  @ Tue, 03 Nov 2020 18:05:30:  48000000 \\nINFO  @ Tue, 03 Nov 2020 18:05:33:  49000000 \\nINFO  @ Tue, 03 Nov 2020 18:05:37:  50000000 \\nINFO  @ Tue, 03 Nov 2020 18:05:41:  51000000 \\nINFO  @ Tue, 03 Nov 2020 18:05:44:  52000000 \\nINFO  @ Tue, 03 Nov 2020 18:05:48:  53000000 \\nINFO  @ Tue, 03 Nov 2020 18:05:52:  54000000 \\nINFO  @ Tue, 03 Nov 2020 18:05:55:  55000000 \\nINFO  @ Tue, 03 Nov 2020 18:05:59:  56000000 \\nINFO  @ Tue, 03 Nov 2020 18:06:03:  57000000 \\nINFO  @ Tue, 03 Nov 2020 18:06:06:  58000000 \\nINFO  @ Tue, 03 Nov 2020 18:06:10:  59000000 \\nINFO  @ Tue, 03 Nov 2020 18:06:14:  60000000 \\nINFO  @ Tue, 03 Nov 2020 18:06:17:  61000000 \\nINFO  @ Tue, 03 Nov 2020 18:06:21:  62000000 \\nINFO  @ Tue, 03 Nov 2020 18:06:25:  63000000 \\nINFO  @ Tue, 03 Nov 2020 18:06:28:  64000000 \\nINFO  @ Tue, 03 Nov 2020 18:06:32:  65000000 \\nINFO  @ Tue, 03 Nov 2020 18:06:36:  66000000 \\nINFO  @ Tue, 03 Nov 2020 18:06:39: 66660428 reads have been read. \\nINFO  @ Tue, 03 Nov 2020 18:06:39: #1 tag size is determined as 90 bps \\nINFO  @ Tue, 03 Nov 2020 18:06:39: #1 tag size = 90.0 \\nINFO  @ Tue, 03 Nov 2020 18:06:39: #1  total tags in treatment: 97438650 \\nINFO  @ Tue, 03 Nov 2020 18:06:39: #1 user defined the maximum tags... \\nINFO  @ Tue, 03 Nov 2020 18:06:39: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 03 Nov 2020 18:06:40: #1  tags after filtering in treatment: 95671904 \\nINFO  @ Tue, 03 Nov 2020 18:06:40: #1  Redundant rate of treatment: 0.02 \\nINFO  @ Tue, 03 Nov 2020 18:06:40: #1  total tags in control: 66660428 \\nINFO  @ Tue, 03 Nov 2020 18:06:40: #1 user defined the maximum tags... \\nINFO  @ Tue, 03 Nov 2020 18:06:40: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 03 Nov 2020 18:06:40: #1  tags after filtering in control: 65724989 \\nINFO  @ Tue, 03 Nov 2020 18:06:40: #1  Redundant rate of control: 0.01 \\nINFO  @ Tue, 03 Nov 2020 18:06:40: #1 finished! \\nINFO  @ Tue, 03 Nov 2020 18:06:40: #2 Build Peak Model... \\nINFO  @ Tue, 03 Nov 2020 18:06:40: #2 Skipped... \\nINFO  @ Tue, 03 Nov 2020 18:06:40: #2 Use 229 as fragment length \\nINFO  @ Tue, 03 Nov 2020 18:06:40: #3 Call peaks... \\nINFO  @ Tue, 03 Nov 2020 18:06:40: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 03 Nov 2020 18:13:16: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 03 Nov 2020 18:13:16: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_treat_pileup.bdg \\nINFO  @ Tue, 03 Nov 2020 18:13:16: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_control_lambda.bdg \\nINFO  @ Tue, 03 Nov 2020 18:13:16: #3   Pileup will be based on sequencing depth in control. \\nINFO  @ Tue, 03 Nov 2020 18:13:16: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 03 Nov 2020 18:19:07: #4 Write output xls file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_peaks.xls \\nINFO  @ Tue, 03 Nov 2020 18:19:07: #4 Write peak in narrowPeak format file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_peaks.narrowPeak \\nINFO  @ Tue, 03 Nov 2020 18:19:07: #4 Write summits bed file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_summits.bed \\nINFO  @ Tue, 03 Nov 2020 18:19:07: Done! \\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'INFO  @ Tue, 03 Nov 2020 18:19:07: \\n# Command line: callpeak -B -t ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1.mLb.clN.sorted.bam -c ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 229 -n MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1 --outdir ../../data/chipseq_MEF2D_degraded/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 03 Nov 2020 18:19:07: #1 read tag files... \\nINFO  @ Tue, 03 Nov 2020 18:19:07: #1 read treatment tags... \\nINFO  @ Tue, 03 Nov 2020 18:19:12:  1000000 \\nINFO  @ Tue, 03 Nov 2020 18:19:18:  2000000 \\nINFO  @ Tue, 03 Nov 2020 18:19:23:  3000000 \\nINFO  @ Tue, 03 Nov 2020 18:19:27:  4000000 \\nINFO  @ Tue, 03 Nov 2020 18:19:32:  5000000 \\nINFO  @ Tue, 03 Nov 2020 18:19:37:  6000000 \\nINFO  @ Tue, 03 Nov 2020 18:19:42:  7000000 \\nINFO  @ Tue, 03 Nov 2020 18:19:46:  8000000 \\nINFO  @ Tue, 03 Nov 2020 18:19:51:  9000000 \\nINFO  @ Tue, 03 Nov 2020 18:19:55:  10000000 \\nINFO  @ Tue, 03 Nov 2020 18:20:00:  11000000 \\nINFO  @ Tue, 03 Nov 2020 18:20:05:  12000000 \\nINFO  @ Tue, 03 Nov 2020 18:20:09:  13000000 \\nINFO  @ Tue, 03 Nov 2020 18:20:14:  14000000 \\nINFO  @ Tue, 03 Nov 2020 18:20:18:  15000000 \\nINFO  @ Tue, 03 Nov 2020 18:20:23:  16000000 \\nINFO  @ Tue, 03 Nov 2020 18:20:28:  17000000 \\nINFO  @ Tue, 03 Nov 2020 18:20:32:  18000000 \\nINFO  @ Tue, 03 Nov 2020 18:20:37:  19000000 \\nINFO  @ Tue, 03 Nov 2020 18:20:41:  20000000 \\nINFO  @ Tue, 03 Nov 2020 18:20:46:  21000000 \\nINFO  @ Tue, 03 Nov 2020 18:20:51:  22000000 \\nINFO  @ Tue, 03 Nov 2020 18:20:55:  23000000 \\nINFO  @ Tue, 03 Nov 2020 18:21:00:  24000000 \\nINFO  @ Tue, 03 Nov 2020 18:21:04:  25000000 \\nINFO  @ Tue, 03 Nov 2020 18:21:09:  26000000 \\nINFO  @ Tue, 03 Nov 2020 18:21:14:  27000000 \\nINFO  @ Tue, 03 Nov 2020 18:21:18:  28000000 \\nINFO  @ Tue, 03 Nov 2020 18:21:23:  29000000 \\nINFO  @ Tue, 03 Nov 2020 18:21:27:  30000000 \\nINFO  @ Tue, 03 Nov 2020 18:21:32:  31000000 \\nINFO  @ Tue, 03 Nov 2020 18:21:37:  32000000 \\nINFO  @ Tue, 03 Nov 2020 18:21:41:  33000000 \\nINFO  @ Tue, 03 Nov 2020 18:21:46:  34000000 \\nINFO  @ Tue, 03 Nov 2020 18:21:51:  35000000 \\nINFO  @ Tue, 03 Nov 2020 18:21:55:  36000000 \\nINFO  @ Tue, 03 Nov 2020 18:22:00:  37000000 \\nINFO  @ Tue, 03 Nov 2020 18:22:04:  38000000 \\nINFO  @ Tue, 03 Nov 2020 18:22:09:  39000000 \\nINFO  @ Tue, 03 Nov 2020 18:22:14:  40000000 \\nINFO  @ Tue, 03 Nov 2020 18:22:18:  41000000 \\nINFO  @ Tue, 03 Nov 2020 18:22:23:  42000000 \\nINFO  @ Tue, 03 Nov 2020 18:22:27:  43000000 \\nINFO  @ Tue, 03 Nov 2020 18:22:32:  44000000 \\nINFO  @ Tue, 03 Nov 2020 18:22:35: 44515175 reads have been read. \\nINFO  @ Tue, 03 Nov 2020 18:22:35: #1.2 read input tags... \\nINFO  @ Tue, 03 Nov 2020 18:22:39:  1000000 \\nINFO  @ Tue, 03 Nov 2020 18:22:43:  2000000 \\nINFO  @ Tue, 03 Nov 2020 18:22:46:  3000000 \\nINFO  @ Tue, 03 Nov 2020 18:22:50:  4000000 \\nINFO  @ Tue, 03 Nov 2020 18:22:54:  5000000 \\nINFO  @ Tue, 03 Nov 2020 18:22:57:  6000000 \\nINFO  @ Tue, 03 Nov 2020 18:23:01:  7000000 \\nINFO  @ Tue, 03 Nov 2020 18:23:05:  8000000 \\nINFO  @ Tue, 03 Nov 2020 18:23:08:  9000000 \\nINFO  @ Tue, 03 Nov 2020 18:23:12:  10000000 \\nINFO  @ Tue, 03 Nov 2020 18:23:16:  11000000 \\nINFO  @ Tue, 03 Nov 2020 18:23:19:  12000000 \\nINFO  @ Tue, 03 Nov 2020 18:23:23:  13000000 \\nINFO  @ Tue, 03 Nov 2020 18:23:27:  14000000 \\nINFO  @ Tue, 03 Nov 2020 18:23:30:  15000000 \\nINFO  @ Tue, 03 Nov 2020 18:23:34:  16000000 \\nINFO  @ Tue, 03 Nov 2020 18:23:37:  17000000 \\nINFO  @ Tue, 03 Nov 2020 18:23:42:  18000000 \\nINFO  @ Tue, 03 Nov 2020 18:23:45:  19000000 \\nINFO  @ Tue, 03 Nov 2020 18:23:49:  20000000 \\nINFO  @ Tue, 03 Nov 2020 18:23:53:  21000000 \\nINFO  @ Tue, 03 Nov 2020 18:23:56:  22000000 \\nINFO  @ Tue, 03 Nov 2020 18:24:00:  23000000 \\nINFO  @ Tue, 03 Nov 2020 18:24:04:  24000000 \\nINFO  @ Tue, 03 Nov 2020 18:24:07:  25000000 \\nINFO  @ Tue, 03 Nov 2020 18:24:11:  26000000 \\nINFO  @ Tue, 03 Nov 2020 18:24:15:  27000000 \\nINFO  @ Tue, 03 Nov 2020 18:24:18:  28000000 \\nINFO  @ Tue, 03 Nov 2020 18:24:22:  29000000 \\nINFO  @ Tue, 03 Nov 2020 18:24:26:  30000000 \\nINFO  @ Tue, 03 Nov 2020 18:24:30:  31000000 \\nINFO  @ Tue, 03 Nov 2020 18:24:33:  32000000 \\nINFO  @ Tue, 03 Nov 2020 18:24:37:  33000000 \\nINFO  @ Tue, 03 Nov 2020 18:24:41:  34000000 \\nINFO  @ Tue, 03 Nov 2020 18:24:44:  35000000 \\nINFO  @ Tue, 03 Nov 2020 18:24:48:  36000000 \\nINFO  @ Tue, 03 Nov 2020 18:24:52:  37000000 \\nINFO  @ Tue, 03 Nov 2020 18:24:55:  38000000 \\nINFO  @ Tue, 03 Nov 2020 18:24:59:  39000000 \\nINFO  @ Tue, 03 Nov 2020 18:25:03:  40000000 \\nINFO  @ Tue, 03 Nov 2020 18:25:07:  41000000 \\nINFO  @ Tue, 03 Nov 2020 18:25:11:  42000000 \\nINFO  @ Tue, 03 Nov 2020 18:25:15:  43000000 \\nINFO  @ Tue, 03 Nov 2020 18:25:18:  44000000 \\nINFO  @ Tue, 03 Nov 2020 18:25:22:  45000000 \\nINFO  @ Tue, 03 Nov 2020 18:25:26:  46000000 \\nINFO  @ Tue, 03 Nov 2020 18:25:30:  47000000 \\nINFO  @ Tue, 03 Nov 2020 18:25:34:  48000000 \\nINFO  @ Tue, 03 Nov 2020 18:25:37:  49000000 \\nINFO  @ Tue, 03 Nov 2020 18:25:41:  50000000 \\nINFO  @ Tue, 03 Nov 2020 18:25:45:  51000000 \\nINFO  @ Tue, 03 Nov 2020 18:25:49:  52000000 \\nINFO  @ Tue, 03 Nov 2020 18:25:53:  53000000 \\nINFO  @ Tue, 03 Nov 2020 18:25:57:  54000000 \\nINFO  @ Tue, 03 Nov 2020 18:26:00:  55000000 \\nINFO  @ Tue, 03 Nov 2020 18:26:04:  56000000 \\nINFO  @ Tue, 03 Nov 2020 18:26:08:  57000000 \\nINFO  @ Tue, 03 Nov 2020 18:26:12:  58000000 \\nINFO  @ Tue, 03 Nov 2020 18:26:16:  59000000 \\nINFO  @ Tue, 03 Nov 2020 18:26:20:  60000000 \\nINFO  @ Tue, 03 Nov 2020 18:26:23:  61000000 \\nINFO  @ Tue, 03 Nov 2020 18:26:27:  62000000 \\nINFO  @ Tue, 03 Nov 2020 18:26:31:  63000000 \\nINFO  @ Tue, 03 Nov 2020 18:26:35:  64000000 \\nINFO  @ Tue, 03 Nov 2020 18:26:39:  65000000 \\nINFO  @ Tue, 03 Nov 2020 18:26:43:  66000000 \\nINFO  @ Tue, 03 Nov 2020 18:26:45: 66660428 reads have been read. \\nINFO  @ Tue, 03 Nov 2020 18:26:46: #1 tag size is determined as 99 bps \\nINFO  @ Tue, 03 Nov 2020 18:26:46: #1 tag size = 99.0 \\nINFO  @ Tue, 03 Nov 2020 18:26:46: #1  total tags in treatment: 44515175 \\nINFO  @ Tue, 03 Nov 2020 18:26:46: #1 user defined the maximum tags... \\nINFO  @ Tue, 03 Nov 2020 18:26:46: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 03 Nov 2020 18:26:46: #1  tags after filtering in treatment: 43855267 \\nINFO  @ Tue, 03 Nov 2020 18:26:46: #1  Redundant rate of treatment: 0.01 \\nINFO  @ Tue, 03 Nov 2020 18:26:46: #1  total tags in control: 66660428 \\nINFO  @ Tue, 03 Nov 2020 18:26:46: #1 user defined the maximum tags... \\nINFO  @ Tue, 03 Nov 2020 18:26:46: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 03 Nov 2020 18:26:46: #1  tags after filtering in control: 65724989 \\nINFO  @ Tue, 03 Nov 2020 18:26:46: #1  Redundant rate of control: 0.01 \\nINFO  @ Tue, 03 Nov 2020 18:26:46: #1 finished! \\nINFO  @ Tue, 03 Nov 2020 18:26:46: #2 Build Peak Model... \\nINFO  @ Tue, 03 Nov 2020 18:26:46: #2 Skipped... \\nINFO  @ Tue, 03 Nov 2020 18:26:46: #2 Use 229 as fragment length \\nINFO  @ Tue, 03 Nov 2020 18:26:46: #3 Call peaks... \\nINFO  @ Tue, 03 Nov 2020 18:26:46: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 03 Nov 2020 18:32:06: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 03 Nov 2020 18:32:06: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_treat_pileup.bdg \\nINFO  @ Tue, 03 Nov 2020 18:32:06: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_control_lambda.bdg \\nINFO  @ Tue, 03 Nov 2020 18:32:06: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Tue, 03 Nov 2020 18:32:06: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 03 Nov 2020 18:36:25: #4 Write output xls file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_peaks.xls \\nINFO  @ Tue, 03 Nov 2020 18:36:25: #4 Write peak in narrowPeak format file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_peaks.narrowPeak \\nINFO  @ Tue, 03 Nov 2020 18:36:25: #4 Write summits bed file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_summits.bed \\nINFO  @ Tue, 03 Nov 2020 18:36:25: Done! \\n'\n",
      "65724989 43855267\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1.mLb.clN.sorted.bam ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-MYC_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1.mLb.clN.sorted.bam and ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-MYC_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "216\n",
      "computing the scaling values\n",
      "b'INFO  @ Tue, 03 Nov 2020 19:00:21: \\n# Command line: callpeak -B -t ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1.mLb.clN.sorted.bam -c ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 216 -n MV411_MEF2D_NT_SC_63_DMSO-MYC_R1 --outdir ../../data/chipseq_MEF2D_degraded/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = MV411_MEF2D_NT_SC_63_DMSO-MYC_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 03 Nov 2020 19:00:21: #1 read tag files... \\nINFO  @ Tue, 03 Nov 2020 19:00:21: #1 read treatment tags... \\nINFO  @ Tue, 03 Nov 2020 19:00:25:  1000000 \\nINFO  @ Tue, 03 Nov 2020 19:00:29:  2000000 \\nINFO  @ Tue, 03 Nov 2020 19:00:34:  3000000 \\nINFO  @ Tue, 03 Nov 2020 19:00:38:  4000000 \\nINFO  @ Tue, 03 Nov 2020 19:00:42:  5000000 \\nINFO  @ Tue, 03 Nov 2020 19:00:47:  6000000 \\nINFO  @ Tue, 03 Nov 2020 19:00:51:  7000000 \\nINFO  @ Tue, 03 Nov 2020 19:00:55:  8000000 \\nINFO  @ Tue, 03 Nov 2020 19:01:00:  9000000 \\nINFO  @ Tue, 03 Nov 2020 19:01:05:  10000000 \\nINFO  @ Tue, 03 Nov 2020 19:01:09:  11000000 \\nINFO  @ Tue, 03 Nov 2020 19:01:13:  12000000 \\nINFO  @ Tue, 03 Nov 2020 19:01:18:  13000000 \\nINFO  @ Tue, 03 Nov 2020 19:01:22:  14000000 \\nINFO  @ Tue, 03 Nov 2020 19:01:26:  15000000 \\nINFO  @ Tue, 03 Nov 2020 19:01:30:  16000000 \\nINFO  @ Tue, 03 Nov 2020 19:01:35:  17000000 \\nINFO  @ Tue, 03 Nov 2020 19:01:39:  18000000 \\nINFO  @ Tue, 03 Nov 2020 19:01:43:  19000000 \\nINFO  @ Tue, 03 Nov 2020 19:01:47:  20000000 \\nINFO  @ Tue, 03 Nov 2020 19:01:52:  21000000 \\nINFO  @ Tue, 03 Nov 2020 19:01:56:  22000000 \\nINFO  @ Tue, 03 Nov 2020 19:02:01:  23000000 \\nINFO  @ Tue, 03 Nov 2020 19:02:05:  24000000 \\nINFO  @ Tue, 03 Nov 2020 19:02:10:  25000000 \\nINFO  @ Tue, 03 Nov 2020 19:02:15:  26000000 \\nINFO  @ Tue, 03 Nov 2020 19:02:19:  27000000 \\nINFO  @ Tue, 03 Nov 2020 19:02:24:  28000000 \\nINFO  @ Tue, 03 Nov 2020 19:02:28:  29000000 \\nINFO  @ Tue, 03 Nov 2020 19:02:33:  30000000 \\nINFO  @ Tue, 03 Nov 2020 19:02:38:  31000000 \\nINFO  @ Tue, 03 Nov 2020 19:02:42:  32000000 \\nINFO  @ Tue, 03 Nov 2020 19:02:47:  33000000 \\nINFO  @ Tue, 03 Nov 2020 19:02:51:  34000000 \\nINFO  @ Tue, 03 Nov 2020 19:02:55:  35000000 \\nINFO  @ Tue, 03 Nov 2020 19:03:00:  36000000 \\nINFO  @ Tue, 03 Nov 2020 19:03:04:  37000000 \\nINFO  @ Tue, 03 Nov 2020 19:03:09:  38000000 \\nINFO  @ Tue, 03 Nov 2020 19:03:13:  39000000 \\nINFO  @ Tue, 03 Nov 2020 19:03:18:  40000000 \\nINFO  @ Tue, 03 Nov 2020 19:03:22:  41000000 \\nINFO  @ Tue, 03 Nov 2020 19:03:27:  42000000 \\nINFO  @ Tue, 03 Nov 2020 19:03:31:  43000000 \\nINFO  @ Tue, 03 Nov 2020 19:03:36:  44000000 \\nINFO  @ Tue, 03 Nov 2020 19:03:41:  45000000 \\nINFO  @ Tue, 03 Nov 2020 19:03:46:  46000000 \\nINFO  @ Tue, 03 Nov 2020 19:03:50:  47000000 \\nINFO  @ Tue, 03 Nov 2020 19:03:54:  48000000 \\nINFO  @ Tue, 03 Nov 2020 19:03:59:  49000000 \\nINFO  @ Tue, 03 Nov 2020 19:04:03:  50000000 \\nINFO  @ Tue, 03 Nov 2020 19:04:08:  51000000 \\nINFO  @ Tue, 03 Nov 2020 19:04:13:  52000000 \\nINFO  @ Tue, 03 Nov 2020 19:04:18:  53000000 \\nINFO  @ Tue, 03 Nov 2020 19:04:23:  54000000 \\nINFO  @ Tue, 03 Nov 2020 19:04:27:  55000000 \\nINFO  @ Tue, 03 Nov 2020 19:04:32:  56000000 \\nINFO  @ Tue, 03 Nov 2020 19:04:36:  57000000 \\nINFO  @ Tue, 03 Nov 2020 19:04:40:  58000000 \\nINFO  @ Tue, 03 Nov 2020 19:04:45:  59000000 \\nINFO  @ Tue, 03 Nov 2020 19:04:49:  60000000 \\nINFO  @ Tue, 03 Nov 2020 19:04:53:  61000000 \\nINFO  @ Tue, 03 Nov 2020 19:04:57:  62000000 \\nINFO  @ Tue, 03 Nov 2020 19:05:02:  63000000 \\nINFO  @ Tue, 03 Nov 2020 19:05:06:  64000000 \\nINFO  @ Tue, 03 Nov 2020 19:05:11:  65000000 \\nINFO  @ Tue, 03 Nov 2020 19:05:15:  66000000 \\nINFO  @ Tue, 03 Nov 2020 19:05:19:  67000000 \\nINFO  @ Tue, 03 Nov 2020 19:05:23:  68000000 \\nINFO  @ Tue, 03 Nov 2020 19:05:28:  69000000 \\nINFO  @ Tue, 03 Nov 2020 19:05:32:  70000000 \\nINFO  @ Tue, 03 Nov 2020 19:05:36:  71000000 \\nINFO  @ Tue, 03 Nov 2020 19:05:41:  72000000 \\nINFO  @ Tue, 03 Nov 2020 19:05:45:  73000000 \\nINFO  @ Tue, 03 Nov 2020 19:05:49:  74000000 \\nINFO  @ Tue, 03 Nov 2020 19:05:53:  75000000 \\nINFO  @ Tue, 03 Nov 2020 19:05:57:  76000000 \\nINFO  @ Tue, 03 Nov 2020 19:06:02:  77000000 \\nINFO  @ Tue, 03 Nov 2020 19:06:06:  78000000 \\nINFO  @ Tue, 03 Nov 2020 19:06:10:  79000000 \\nINFO  @ Tue, 03 Nov 2020 19:06:15:  80000000 \\nINFO  @ Tue, 03 Nov 2020 19:06:19:  81000000 \\nINFO  @ Tue, 03 Nov 2020 19:06:23:  82000000 \\nINFO  @ Tue, 03 Nov 2020 19:06:28:  83000000 \\nINFO  @ Tue, 03 Nov 2020 19:06:32:  84000000 \\nINFO  @ Tue, 03 Nov 2020 19:06:35: 84356046 reads have been read. \\nINFO  @ Tue, 03 Nov 2020 19:06:35: #1.2 read input tags... \\nINFO  @ Tue, 03 Nov 2020 19:06:40:  1000000 \\nINFO  @ Tue, 03 Nov 2020 19:06:44:  2000000 \\nINFO  @ Tue, 03 Nov 2020 19:06:48:  3000000 \\nINFO  @ Tue, 03 Nov 2020 19:06:51:  4000000 \\nINFO  @ Tue, 03 Nov 2020 19:06:55:  5000000 \\nINFO  @ Tue, 03 Nov 2020 19:06:59:  6000000 \\nINFO  @ Tue, 03 Nov 2020 19:07:03:  7000000 \\nINFO  @ Tue, 03 Nov 2020 19:07:07:  8000000 \\nINFO  @ Tue, 03 Nov 2020 19:07:11:  9000000 \\nINFO  @ Tue, 03 Nov 2020 19:07:14:  10000000 \\nINFO  @ Tue, 03 Nov 2020 19:07:18:  11000000 \\nINFO  @ Tue, 03 Nov 2020 19:07:22:  12000000 \\nINFO  @ Tue, 03 Nov 2020 19:07:26:  13000000 \\nINFO  @ Tue, 03 Nov 2020 19:07:30:  14000000 \\nINFO  @ Tue, 03 Nov 2020 19:07:33:  15000000 \\nINFO  @ Tue, 03 Nov 2020 19:07:37:  16000000 \\nINFO  @ Tue, 03 Nov 2020 19:07:41:  17000000 \\nINFO  @ Tue, 03 Nov 2020 19:07:45:  18000000 \\nINFO  @ Tue, 03 Nov 2020 19:07:49:  19000000 \\nINFO  @ Tue, 03 Nov 2020 19:07:52:  20000000 \\nINFO  @ Tue, 03 Nov 2020 19:07:56:  21000000 \\nINFO  @ Tue, 03 Nov 2020 19:08:00:  22000000 \\nINFO  @ Tue, 03 Nov 2020 19:08:03:  23000000 \\nINFO  @ Tue, 03 Nov 2020 19:08:07:  24000000 \\nINFO  @ Tue, 03 Nov 2020 19:08:11:  25000000 \\nINFO  @ Tue, 03 Nov 2020 19:08:16:  26000000 \\nINFO  @ Tue, 03 Nov 2020 19:08:19:  27000000 \\nINFO  @ Tue, 03 Nov 2020 19:08:23:  28000000 \\nINFO  @ Tue, 03 Nov 2020 19:08:27:  29000000 \\nINFO  @ Tue, 03 Nov 2020 19:08:31:  30000000 \\nINFO  @ Tue, 03 Nov 2020 19:08:34:  31000000 \\nINFO  @ Tue, 03 Nov 2020 19:08:38:  32000000 \\nINFO  @ Tue, 03 Nov 2020 19:08:42:  33000000 \\nINFO  @ Tue, 03 Nov 2020 19:08:46:  34000000 \\nINFO  @ Tue, 03 Nov 2020 19:08:49:  35000000 \\nINFO  @ Tue, 03 Nov 2020 19:08:53:  36000000 \\nINFO  @ Tue, 03 Nov 2020 19:08:56:  37000000 \\nINFO  @ Tue, 03 Nov 2020 19:09:00:  38000000 \\nINFO  @ Tue, 03 Nov 2020 19:09:04:  39000000 \\nINFO  @ Tue, 03 Nov 2020 19:09:08:  40000000 \\nINFO  @ Tue, 03 Nov 2020 19:09:12:  41000000 \\nINFO  @ Tue, 03 Nov 2020 19:09:16:  42000000 \\nINFO  @ Tue, 03 Nov 2020 19:09:19:  43000000 \\nINFO  @ Tue, 03 Nov 2020 19:09:23:  44000000 \\nINFO  @ Tue, 03 Nov 2020 19:09:27:  45000000 \\nINFO  @ Tue, 03 Nov 2020 19:09:30:  46000000 \\nINFO  @ Tue, 03 Nov 2020 19:09:34:  47000000 \\nINFO  @ Tue, 03 Nov 2020 19:09:38:  48000000 \\nINFO  @ Tue, 03 Nov 2020 19:09:41:  49000000 \\nINFO  @ Tue, 03 Nov 2020 19:09:45:  50000000 \\nINFO  @ Tue, 03 Nov 2020 19:09:49:  51000000 \\nINFO  @ Tue, 03 Nov 2020 19:09:52:  52000000 \\nINFO  @ Tue, 03 Nov 2020 19:09:56:  53000000 \\nINFO  @ Tue, 03 Nov 2020 19:10:00:  54000000 \\nINFO  @ Tue, 03 Nov 2020 19:10:03:  55000000 \\nINFO  @ Tue, 03 Nov 2020 19:10:07:  56000000 \\nINFO  @ Tue, 03 Nov 2020 19:10:11:  57000000 \\nINFO  @ Tue, 03 Nov 2020 19:10:14:  58000000 \\nINFO  @ Tue, 03 Nov 2020 19:10:18:  59000000 \\nINFO  @ Tue, 03 Nov 2020 19:10:22:  60000000 \\nINFO  @ Tue, 03 Nov 2020 19:10:25:  61000000 \\nINFO  @ Tue, 03 Nov 2020 19:10:29:  62000000 \\nINFO  @ Tue, 03 Nov 2020 19:10:33:  63000000 \\nINFO  @ Tue, 03 Nov 2020 19:10:37:  64000000 \\nINFO  @ Tue, 03 Nov 2020 19:10:40:  65000000 \\nINFO  @ Tue, 03 Nov 2020 19:10:44:  66000000 \\nINFO  @ Tue, 03 Nov 2020 19:10:47: 66660428 reads have been read. \\nINFO  @ Tue, 03 Nov 2020 19:10:48: #1 tag size is determined as 99 bps \\nINFO  @ Tue, 03 Nov 2020 19:10:48: #1 tag size = 99.0 \\nINFO  @ Tue, 03 Nov 2020 19:10:48: #1  total tags in treatment: 84356046 \\nINFO  @ Tue, 03 Nov 2020 19:10:48: #1 user defined the maximum tags... \\nINFO  @ Tue, 03 Nov 2020 19:10:48: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 03 Nov 2020 19:10:49: #1  tags after filtering in treatment: 80288739 \\nINFO  @ Tue, 03 Nov 2020 19:10:49: #1  Redundant rate of treatment: 0.05 \\nINFO  @ Tue, 03 Nov 2020 19:10:49: #1  total tags in control: 66660428 \\nINFO  @ Tue, 03 Nov 2020 19:10:49: #1 user defined the maximum tags... \\nINFO  @ Tue, 03 Nov 2020 19:10:49: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 03 Nov 2020 19:10:49: #1  tags after filtering in control: 65724989 \\nINFO  @ Tue, 03 Nov 2020 19:10:49: #1  Redundant rate of control: 0.01 \\nINFO  @ Tue, 03 Nov 2020 19:10:49: #1 finished! \\nINFO  @ Tue, 03 Nov 2020 19:10:49: #2 Build Peak Model... \\nINFO  @ Tue, 03 Nov 2020 19:10:49: #2 Skipped... \\nINFO  @ Tue, 03 Nov 2020 19:10:49: #2 Use 216 as fragment length \\nINFO  @ Tue, 03 Nov 2020 19:10:49: #3 Call peaks... \\nINFO  @ Tue, 03 Nov 2020 19:10:49: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 03 Nov 2020 19:17:10: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 03 Nov 2020 19:17:10: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_treat_pileup.bdg \\nINFO  @ Tue, 03 Nov 2020 19:17:10: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_control_lambda.bdg \\nINFO  @ Tue, 03 Nov 2020 19:17:10: #3   Pileup will be based on sequencing depth in control. \\nINFO  @ Tue, 03 Nov 2020 19:17:10: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 03 Nov 2020 19:22:55: #4 Write output xls file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_peaks.xls \\nINFO  @ Tue, 03 Nov 2020 19:22:55: #4 Write peak in narrowPeak format file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_peaks.narrowPeak \\nINFO  @ Tue, 03 Nov 2020 19:22:55: #4 Write summits bed file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_summits.bed \\nINFO  @ Tue, 03 Nov 2020 19:22:55: Done! \\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'INFO  @ Tue, 03 Nov 2020 19:22:55: \\n# Command line: callpeak -B -t ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-MYC_R1.mLb.clN.sorted.bam -c ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 216 -n MV411_MEF2D_NT_SC_63_VHL-MYC_R1 --outdir ../../data/chipseq_MEF2D_degraded/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = MV411_MEF2D_NT_SC_63_VHL-MYC_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-MYC_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 03 Nov 2020 19:22:55: #1 read tag files... \\nINFO  @ Tue, 03 Nov 2020 19:22:55: #1 read treatment tags... \\nINFO  @ Tue, 03 Nov 2020 19:23:00:  1000000 \\nINFO  @ Tue, 03 Nov 2020 19:23:04:  2000000 \\nINFO  @ Tue, 03 Nov 2020 19:23:11:  3000000 \\nINFO  @ Tue, 03 Nov 2020 19:23:16:  4000000 \\nINFO  @ Tue, 03 Nov 2020 19:23:23:  5000000 \\nINFO  @ Tue, 03 Nov 2020 19:23:27:  6000000 \\nINFO  @ Tue, 03 Nov 2020 19:23:31:  7000000 \\nINFO  @ Tue, 03 Nov 2020 19:23:36:  8000000 \\nINFO  @ Tue, 03 Nov 2020 19:23:40:  9000000 \\nINFO  @ Tue, 03 Nov 2020 19:23:45:  10000000 \\nINFO  @ Tue, 03 Nov 2020 19:23:49:  11000000 \\nINFO  @ Tue, 03 Nov 2020 19:23:56:  12000000 \\nINFO  @ Tue, 03 Nov 2020 19:24:00:  13000000 \\nINFO  @ Tue, 03 Nov 2020 19:24:04:  14000000 \\nINFO  @ Tue, 03 Nov 2020 19:24:09:  15000000 \\nINFO  @ Tue, 03 Nov 2020 19:24:13:  16000000 \\nINFO  @ Tue, 03 Nov 2020 19:24:18:  17000000 \\nINFO  @ Tue, 03 Nov 2020 19:24:22:  18000000 \\nINFO  @ Tue, 03 Nov 2020 19:24:26:  19000000 \\nINFO  @ Tue, 03 Nov 2020 19:24:34:  20000000 \\nINFO  @ Tue, 03 Nov 2020 19:24:38:  21000000 \\nINFO  @ Tue, 03 Nov 2020 19:24:43:  22000000 \\nINFO  @ Tue, 03 Nov 2020 19:24:47:  23000000 \\nINFO  @ Tue, 03 Nov 2020 19:24:51:  24000000 \\nINFO  @ Tue, 03 Nov 2020 19:24:56:  25000000 \\nINFO  @ Tue, 03 Nov 2020 19:25:02:  26000000 \\nINFO  @ Tue, 03 Nov 2020 19:25:06:  27000000 \\nINFO  @ Tue, 03 Nov 2020 19:25:11:  28000000 \\nINFO  @ Tue, 03 Nov 2020 19:25:15:  29000000 \\nINFO  @ Tue, 03 Nov 2020 19:25:19:  30000000 \\nINFO  @ Tue, 03 Nov 2020 19:25:24:  31000000 \\nINFO  @ Tue, 03 Nov 2020 19:25:29:  32000000 \\nINFO  @ Tue, 03 Nov 2020 19:25:33:  33000000 \\nINFO  @ Tue, 03 Nov 2020 19:25:38:  34000000 \\nINFO  @ Tue, 03 Nov 2020 19:25:42:  35000000 \\nINFO  @ Tue, 03 Nov 2020 19:25:46:  36000000 \\nINFO  @ Tue, 03 Nov 2020 19:25:51:  37000000 \\nINFO  @ Tue, 03 Nov 2020 19:25:55:  38000000 \\nINFO  @ Tue, 03 Nov 2020 19:25:59:  39000000 \\nINFO  @ Tue, 03 Nov 2020 19:26:04:  40000000 \\nINFO  @ Tue, 03 Nov 2020 19:26:11:  41000000 \\nINFO  @ Tue, 03 Nov 2020 19:26:15:  42000000 \\nINFO  @ Tue, 03 Nov 2020 19:26:20:  43000000 \\nINFO  @ Tue, 03 Nov 2020 19:26:24:  44000000 \\nINFO  @ Tue, 03 Nov 2020 19:26:28:  45000000 \\nINFO  @ Tue, 03 Nov 2020 19:26:33:  46000000 \\nINFO  @ Tue, 03 Nov 2020 19:26:37:  47000000 \\nINFO  @ Tue, 03 Nov 2020 19:26:41:  48000000 \\nINFO  @ Tue, 03 Nov 2020 19:26:48:  49000000 \\nINFO  @ Tue, 03 Nov 2020 19:26:52:  50000000 \\nINFO  @ Tue, 03 Nov 2020 19:26:56:  51000000 \\nINFO  @ Tue, 03 Nov 2020 19:27:01:  52000000 \\nINFO  @ Tue, 03 Nov 2020 19:27:05:  53000000 \\nINFO  @ Tue, 03 Nov 2020 19:27:09:  54000000 \\nINFO  @ Tue, 03 Nov 2020 19:27:13:  55000000 \\nINFO  @ Tue, 03 Nov 2020 19:27:18:  56000000 \\nINFO  @ Tue, 03 Nov 2020 19:27:22:  57000000 \\nINFO  @ Tue, 03 Nov 2020 19:27:30:  58000000 \\nINFO  @ Tue, 03 Nov 2020 19:27:34:  59000000 \\nINFO  @ Tue, 03 Nov 2020 19:27:39:  60000000 \\nINFO  @ Tue, 03 Nov 2020 19:27:41: 60348972 reads have been read. \\nINFO  @ Tue, 03 Nov 2020 19:27:41: #1.2 read input tags... \\nINFO  @ Tue, 03 Nov 2020 19:27:45:  1000000 \\nINFO  @ Tue, 03 Nov 2020 19:27:48:  2000000 \\nINFO  @ Tue, 03 Nov 2020 19:27:52:  3000000 \\nINFO  @ Tue, 03 Nov 2020 19:27:55:  4000000 \\nINFO  @ Tue, 03 Nov 2020 19:27:59:  5000000 \\nINFO  @ Tue, 03 Nov 2020 19:28:03:  6000000 \\nINFO  @ Tue, 03 Nov 2020 19:28:06:  7000000 \\nINFO  @ Tue, 03 Nov 2020 19:28:10:  8000000 \\nINFO  @ Tue, 03 Nov 2020 19:28:13:  9000000 \\nINFO  @ Tue, 03 Nov 2020 19:28:17:  10000000 \\nINFO  @ Tue, 03 Nov 2020 19:28:20:  11000000 \\nINFO  @ Tue, 03 Nov 2020 19:28:24:  12000000 \\nINFO  @ Tue, 03 Nov 2020 19:28:27:  13000000 \\nINFO  @ Tue, 03 Nov 2020 19:28:31:  14000000 \\nINFO  @ Tue, 03 Nov 2020 19:28:35:  15000000 \\nINFO  @ Tue, 03 Nov 2020 19:28:38:  16000000 \\nINFO  @ Tue, 03 Nov 2020 19:28:42:  17000000 \\nINFO  @ Tue, 03 Nov 2020 19:28:45:  18000000 \\nINFO  @ Tue, 03 Nov 2020 19:28:49:  19000000 \\nINFO  @ Tue, 03 Nov 2020 19:28:52:  20000000 \\nINFO  @ Tue, 03 Nov 2020 19:28:56:  21000000 \\nINFO  @ Tue, 03 Nov 2020 19:28:59:  22000000 \\nINFO  @ Tue, 03 Nov 2020 19:29:03:  23000000 \\nINFO  @ Tue, 03 Nov 2020 19:29:06:  24000000 \\nINFO  @ Tue, 03 Nov 2020 19:29:10:  25000000 \\nINFO  @ Tue, 03 Nov 2020 19:29:13:  26000000 \\nINFO  @ Tue, 03 Nov 2020 19:29:17:  27000000 \\nINFO  @ Tue, 03 Nov 2020 19:29:21:  28000000 \\nINFO  @ Tue, 03 Nov 2020 19:29:24:  29000000 \\nINFO  @ Tue, 03 Nov 2020 19:29:28:  30000000 \\nINFO  @ Tue, 03 Nov 2020 19:29:31:  31000000 \\nINFO  @ Tue, 03 Nov 2020 19:29:35:  32000000 \\nINFO  @ Tue, 03 Nov 2020 19:29:38:  33000000 \\nINFO  @ Tue, 03 Nov 2020 19:29:42:  34000000 \\nINFO  @ Tue, 03 Nov 2020 19:29:45:  35000000 \\nINFO  @ Tue, 03 Nov 2020 19:29:49:  36000000 \\nINFO  @ Tue, 03 Nov 2020 19:29:52:  37000000 \\nINFO  @ Tue, 03 Nov 2020 19:29:56:  38000000 \\nINFO  @ Tue, 03 Nov 2020 19:29:59:  39000000 \\nINFO  @ Tue, 03 Nov 2020 19:30:03:  40000000 \\nINFO  @ Tue, 03 Nov 2020 19:30:06:  41000000 \\nINFO  @ Tue, 03 Nov 2020 19:30:10:  42000000 \\nINFO  @ Tue, 03 Nov 2020 19:30:14:  43000000 \\nINFO  @ Tue, 03 Nov 2020 19:30:17:  44000000 \\nINFO  @ Tue, 03 Nov 2020 19:30:21:  45000000 \\nINFO  @ Tue, 03 Nov 2020 19:30:24:  46000000 \\nINFO  @ Tue, 03 Nov 2020 19:30:28:  47000000 \\nINFO  @ Tue, 03 Nov 2020 19:30:31:  48000000 \\nINFO  @ Tue, 03 Nov 2020 19:30:35:  49000000 \\nINFO  @ Tue, 03 Nov 2020 19:30:39:  50000000 \\nINFO  @ Tue, 03 Nov 2020 19:30:42:  51000000 \\nINFO  @ Tue, 03 Nov 2020 19:30:46:  52000000 \\nINFO  @ Tue, 03 Nov 2020 19:30:50:  53000000 \\nINFO  @ Tue, 03 Nov 2020 19:30:53:  54000000 \\nINFO  @ Tue, 03 Nov 2020 19:30:58:  55000000 \\nINFO  @ Tue, 03 Nov 2020 19:31:01:  56000000 \\nINFO  @ Tue, 03 Nov 2020 19:31:05:  57000000 \\nINFO  @ Tue, 03 Nov 2020 19:31:09:  58000000 \\nINFO  @ Tue, 03 Nov 2020 19:31:12:  59000000 \\nINFO  @ Tue, 03 Nov 2020 19:31:16:  60000000 \\nINFO  @ Tue, 03 Nov 2020 19:31:19:  61000000 \\nINFO  @ Tue, 03 Nov 2020 19:31:23:  62000000 \\nINFO  @ Tue, 03 Nov 2020 19:31:27:  63000000 \\nINFO  @ Tue, 03 Nov 2020 19:31:31:  64000000 \\nINFO  @ Tue, 03 Nov 2020 19:31:35:  65000000 \\nINFO  @ Tue, 03 Nov 2020 19:31:39:  66000000 \\nINFO  @ Tue, 03 Nov 2020 19:31:41: 66660428 reads have been read. \\nINFO  @ Tue, 03 Nov 2020 19:31:42: #1 tag size is determined as 99 bps \\nINFO  @ Tue, 03 Nov 2020 19:31:42: #1 tag size = 99.0 \\nINFO  @ Tue, 03 Nov 2020 19:31:42: #1  total tags in treatment: 60348972 \\nINFO  @ Tue, 03 Nov 2020 19:31:42: #1 user defined the maximum tags... \\nINFO  @ Tue, 03 Nov 2020 19:31:42: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 03 Nov 2020 19:31:42: #1  tags after filtering in treatment: 57565992 \\nINFO  @ Tue, 03 Nov 2020 19:31:42: #1  Redundant rate of treatment: 0.05 \\nINFO  @ Tue, 03 Nov 2020 19:31:42: #1  total tags in control: 66660428 \\nINFO  @ Tue, 03 Nov 2020 19:31:42: #1 user defined the maximum tags... \\nINFO  @ Tue, 03 Nov 2020 19:31:42: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 03 Nov 2020 19:31:42: #1  tags after filtering in control: 65724989 \\nINFO  @ Tue, 03 Nov 2020 19:31:42: #1  Redundant rate of control: 0.01 \\nINFO  @ Tue, 03 Nov 2020 19:31:42: #1 finished! \\nINFO  @ Tue, 03 Nov 2020 19:31:42: #2 Build Peak Model... \\nINFO  @ Tue, 03 Nov 2020 19:31:42: #2 Skipped... \\nINFO  @ Tue, 03 Nov 2020 19:31:42: #2 Use 216 as fragment length \\nINFO  @ Tue, 03 Nov 2020 19:31:42: #3 Call peaks... \\nINFO  @ Tue, 03 Nov 2020 19:31:42: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 03 Nov 2020 19:37:15: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 03 Nov 2020 19:37:15: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_MEF2D_NT_SC_63_VHL-MYC_R1_treat_pileup.bdg \\nINFO  @ Tue, 03 Nov 2020 19:37:15: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_MEF2D_NT_SC_63_VHL-MYC_R1_control_lambda.bdg \\nINFO  @ Tue, 03 Nov 2020 19:37:15: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Tue, 03 Nov 2020 19:37:15: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 03 Nov 2020 19:41:56: #4 Write output xls file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-MYC_R1_peaks.xls \\nINFO  @ Tue, 03 Nov 2020 19:41:56: #4 Write peak in narrowPeak format file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-MYC_R1_peaks.narrowPeak \\nINFO  @ Tue, 03 Nov 2020 19:41:56: #4 Write summits bed file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-MYC_R1_summits.bed \\nINFO  @ Tue, 03 Nov 2020 19:41:56: Done! \\n'\n",
      "65724989 57565992\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1.mLb.clN.sorted.bam ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-POLII_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1.mLb.clN.sorted.bam and ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-POLII_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "250\n",
      "computing the scaling values\n",
      "b'INFO  @ Tue, 03 Nov 2020 20:09:13: \\n# Command line: callpeak -B -t ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1.mLb.clN.sorted.bam -c ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 250 -n MV411_MEF2D_NT_SC_63_DMSO-POLII_R1 --outdir ../../data/chipseq_MEF2D_degraded/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = MV411_MEF2D_NT_SC_63_DMSO-POLII_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 03 Nov 2020 20:09:13: #1 read tag files... \\nINFO  @ Tue, 03 Nov 2020 20:09:13: #1 read treatment tags... \\nINFO  @ Tue, 03 Nov 2020 20:09:17:  1000000 \\nINFO  @ Tue, 03 Nov 2020 20:09:22:  2000000 \\nINFO  @ Tue, 03 Nov 2020 20:09:26:  3000000 \\nINFO  @ Tue, 03 Nov 2020 20:09:30:  4000000 \\nINFO  @ Tue, 03 Nov 2020 20:09:34:  5000000 \\nINFO  @ Tue, 03 Nov 2020 20:09:39:  6000000 \\nINFO  @ Tue, 03 Nov 2020 20:09:43:  7000000 \\nINFO  @ Tue, 03 Nov 2020 20:09:47:  8000000 \\nINFO  @ Tue, 03 Nov 2020 20:09:51:  9000000 \\nINFO  @ Tue, 03 Nov 2020 20:09:55:  10000000 \\nINFO  @ Tue, 03 Nov 2020 20:10:00:  11000000 \\nINFO  @ Tue, 03 Nov 2020 20:10:04:  12000000 \\nINFO  @ Tue, 03 Nov 2020 20:10:08:  13000000 \\nINFO  @ Tue, 03 Nov 2020 20:10:12:  14000000 \\nINFO  @ Tue, 03 Nov 2020 20:10:17:  15000000 \\nINFO  @ Tue, 03 Nov 2020 20:10:21:  16000000 \\nINFO  @ Tue, 03 Nov 2020 20:10:25:  17000000 \\nINFO  @ Tue, 03 Nov 2020 20:10:30:  18000000 \\nINFO  @ Tue, 03 Nov 2020 20:10:34:  19000000 \\nINFO  @ Tue, 03 Nov 2020 20:10:38:  20000000 \\nINFO  @ Tue, 03 Nov 2020 20:10:42:  21000000 \\nINFO  @ Tue, 03 Nov 2020 20:10:46:  22000000 \\nINFO  @ Tue, 03 Nov 2020 20:10:51:  23000000 \\nINFO  @ Tue, 03 Nov 2020 20:10:55:  24000000 \\nINFO  @ Tue, 03 Nov 2020 20:10:59:  25000000 \\nINFO  @ Tue, 03 Nov 2020 20:11:03:  26000000 \\nINFO  @ Tue, 03 Nov 2020 20:11:07:  27000000 \\nINFO  @ Tue, 03 Nov 2020 20:11:12:  28000000 \\nINFO  @ Tue, 03 Nov 2020 20:11:16:  29000000 \\nINFO  @ Tue, 03 Nov 2020 20:11:20:  30000000 \\nINFO  @ Tue, 03 Nov 2020 20:11:25:  31000000 \\nINFO  @ Tue, 03 Nov 2020 20:11:29:  32000000 \\nINFO  @ Tue, 03 Nov 2020 20:11:33:  33000000 \\nINFO  @ Tue, 03 Nov 2020 20:11:38:  34000000 \\nINFO  @ Tue, 03 Nov 2020 20:11:42:  35000000 \\nINFO  @ Tue, 03 Nov 2020 20:11:46:  36000000 \\nINFO  @ Tue, 03 Nov 2020 20:11:51:  37000000 \\nINFO  @ Tue, 03 Nov 2020 20:11:55:  38000000 \\nINFO  @ Tue, 03 Nov 2020 20:11:59:  39000000 \\nINFO  @ Tue, 03 Nov 2020 20:12:03:  40000000 \\nINFO  @ Tue, 03 Nov 2020 20:12:07:  41000000 \\nINFO  @ Tue, 03 Nov 2020 20:12:11:  42000000 \\nINFO  @ Tue, 03 Nov 2020 20:12:16:  43000000 \\nINFO  @ Tue, 03 Nov 2020 20:12:20:  44000000 \\nINFO  @ Tue, 03 Nov 2020 20:12:25:  45000000 \\nINFO  @ Tue, 03 Nov 2020 20:12:29:  46000000 \\nINFO  @ Tue, 03 Nov 2020 20:12:33:  47000000 \\nINFO  @ Tue, 03 Nov 2020 20:12:37:  48000000 \\nINFO  @ Tue, 03 Nov 2020 20:12:42:  49000000 \\nINFO  @ Tue, 03 Nov 2020 20:12:46:  50000000 \\nINFO  @ Tue, 03 Nov 2020 20:12:51:  51000000 \\nINFO  @ Tue, 03 Nov 2020 20:12:55:  52000000 \\nINFO  @ Tue, 03 Nov 2020 20:13:00:  53000000 \\nINFO  @ Tue, 03 Nov 2020 20:13:04:  54000000 \\nINFO  @ Tue, 03 Nov 2020 20:13:08:  55000000 \\nINFO  @ Tue, 03 Nov 2020 20:13:13:  56000000 \\nINFO  @ Tue, 03 Nov 2020 20:13:18:  57000000 \\nINFO  @ Tue, 03 Nov 2020 20:13:22:  58000000 \\nINFO  @ Tue, 03 Nov 2020 20:13:26:  59000000 \\nINFO  @ Tue, 03 Nov 2020 20:13:31:  60000000 \\nINFO  @ Tue, 03 Nov 2020 20:13:35:  61000000 \\nINFO  @ Tue, 03 Nov 2020 20:13:40:  62000000 \\nINFO  @ Tue, 03 Nov 2020 20:13:44:  63000000 \\nINFO  @ Tue, 03 Nov 2020 20:13:48:  64000000 \\nINFO  @ Tue, 03 Nov 2020 20:13:53:  65000000 \\nINFO  @ Tue, 03 Nov 2020 20:13:57:  66000000 \\nINFO  @ Tue, 03 Nov 2020 20:14:01:  67000000 \\nINFO  @ Tue, 03 Nov 2020 20:14:05:  68000000 \\nINFO  @ Tue, 03 Nov 2020 20:14:09:  69000000 \\nINFO  @ Tue, 03 Nov 2020 20:14:14:  70000000 \\nINFO  @ Tue, 03 Nov 2020 20:14:18:  71000000 \\nINFO  @ Tue, 03 Nov 2020 20:14:22:  72000000 \\nINFO  @ Tue, 03 Nov 2020 20:14:26:  73000000 \\nINFO  @ Tue, 03 Nov 2020 20:14:30:  74000000 \\nINFO  @ Tue, 03 Nov 2020 20:14:35:  75000000 \\nINFO  @ Tue, 03 Nov 2020 20:14:39:  76000000 \\nINFO  @ Tue, 03 Nov 2020 20:14:43:  77000000 \\nINFO  @ Tue, 03 Nov 2020 20:14:47:  78000000 \\nINFO  @ Tue, 03 Nov 2020 20:14:52:  79000000 \\nINFO  @ Tue, 03 Nov 2020 20:14:56:  80000000 \\nINFO  @ Tue, 03 Nov 2020 20:15:00:  81000000 \\nINFO  @ Tue, 03 Nov 2020 20:15:04:  82000000 \\nINFO  @ Tue, 03 Nov 2020 20:15:09:  83000000 \\nINFO  @ Tue, 03 Nov 2020 20:15:13:  84000000 \\nINFO  @ Tue, 03 Nov 2020 20:15:17:  85000000 \\nINFO  @ Tue, 03 Nov 2020 20:15:22:  86000000 \\nINFO  @ Tue, 03 Nov 2020 20:15:26:  87000000 \\nINFO  @ Tue, 03 Nov 2020 20:15:30:  88000000 \\nINFO  @ Tue, 03 Nov 2020 20:15:34:  89000000 \\nINFO  @ Tue, 03 Nov 2020 20:15:38:  90000000 \\nINFO  @ Tue, 03 Nov 2020 20:15:42:  91000000 \\nINFO  @ Tue, 03 Nov 2020 20:15:47:  92000000 \\nINFO  @ Tue, 03 Nov 2020 20:15:51:  93000000 \\nINFO  @ Tue, 03 Nov 2020 20:15:55:  94000000 \\nINFO  @ Tue, 03 Nov 2020 20:15:59:  95000000 \\nINFO  @ Tue, 03 Nov 2020 20:16:03:  96000000 \\nINFO  @ Tue, 03 Nov 2020 20:16:08:  97000000 \\nINFO  @ Tue, 03 Nov 2020 20:16:12:  98000000 \\nINFO  @ Tue, 03 Nov 2020 20:16:16:  99000000 \\nINFO  @ Tue, 03 Nov 2020 20:16:20:  100000000 \\nINFO  @ Tue, 03 Nov 2020 20:16:24:  101000000 \\nINFO  @ Tue, 03 Nov 2020 20:16:28:  102000000 \\nINFO  @ Tue, 03 Nov 2020 20:16:32:  103000000 \\nINFO  @ Tue, 03 Nov 2020 20:16:37:  104000000 \\nINFO  @ Tue, 03 Nov 2020 20:16:41:  105000000 \\nINFO  @ Tue, 03 Nov 2020 20:16:45:  106000000 \\nINFO  @ Tue, 03 Nov 2020 20:16:49:  107000000 \\nINFO  @ Tue, 03 Nov 2020 20:16:54:  108000000 \\nINFO  @ Tue, 03 Nov 2020 20:16:58:  109000000 \\nINFO  @ Tue, 03 Nov 2020 20:17:02:  110000000 \\nINFO  @ Tue, 03 Nov 2020 20:17:07:  111000000 \\nINFO  @ Tue, 03 Nov 2020 20:17:11:  112000000 \\nINFO  @ Tue, 03 Nov 2020 20:17:16:  113000000 \\nINFO  @ Tue, 03 Nov 2020 20:17:17: 113234286 reads have been read. \\nINFO  @ Tue, 03 Nov 2020 20:17:18: #1.2 read input tags... \\nINFO  @ Tue, 03 Nov 2020 20:17:22:  1000000 \\nINFO  @ Tue, 03 Nov 2020 20:17:26:  2000000 \\nINFO  @ Tue, 03 Nov 2020 20:17:30:  3000000 \\nINFO  @ Tue, 03 Nov 2020 20:17:34:  4000000 \\nINFO  @ Tue, 03 Nov 2020 20:17:37:  5000000 \\nINFO  @ Tue, 03 Nov 2020 20:17:41:  6000000 \\nINFO  @ Tue, 03 Nov 2020 20:17:45:  7000000 \\nINFO  @ Tue, 03 Nov 2020 20:17:49:  8000000 \\nINFO  @ Tue, 03 Nov 2020 20:17:53:  9000000 \\nINFO  @ Tue, 03 Nov 2020 20:17:56:  10000000 \\nINFO  @ Tue, 03 Nov 2020 20:18:00:  11000000 \\nINFO  @ Tue, 03 Nov 2020 20:18:05:  12000000 \\nINFO  @ Tue, 03 Nov 2020 20:18:09:  13000000 \\nINFO  @ Tue, 03 Nov 2020 20:18:13:  14000000 \\nINFO  @ Tue, 03 Nov 2020 20:18:16:  15000000 \\nINFO  @ Tue, 03 Nov 2020 20:18:20:  16000000 \\nINFO  @ Tue, 03 Nov 2020 20:18:24:  17000000 \\nINFO  @ Tue, 03 Nov 2020 20:18:28:  18000000 \\nINFO  @ Tue, 03 Nov 2020 20:18:31:  19000000 \\nINFO  @ Tue, 03 Nov 2020 20:18:35:  20000000 \\nINFO  @ Tue, 03 Nov 2020 20:18:39:  21000000 \\nINFO  @ Tue, 03 Nov 2020 20:18:43:  22000000 \\nINFO  @ Tue, 03 Nov 2020 20:18:46:  23000000 \\nINFO  @ Tue, 03 Nov 2020 20:18:50:  24000000 \\nINFO  @ Tue, 03 Nov 2020 20:18:54:  25000000 \\nINFO  @ Tue, 03 Nov 2020 20:18:57:  26000000 \\nINFO  @ Tue, 03 Nov 2020 20:19:01:  27000000 \\nINFO  @ Tue, 03 Nov 2020 20:19:05:  28000000 \\nINFO  @ Tue, 03 Nov 2020 20:19:08:  29000000 \\nINFO  @ Tue, 03 Nov 2020 20:19:12:  30000000 \\nINFO  @ Tue, 03 Nov 2020 20:19:16:  31000000 \\nINFO  @ Tue, 03 Nov 2020 20:19:19:  32000000 \\nINFO  @ Tue, 03 Nov 2020 20:19:23:  33000000 \\nINFO  @ Tue, 03 Nov 2020 20:19:27:  34000000 \\nINFO  @ Tue, 03 Nov 2020 20:19:30:  35000000 \\nINFO  @ Tue, 03 Nov 2020 20:19:34:  36000000 \\nINFO  @ Tue, 03 Nov 2020 20:19:38:  37000000 \\nINFO  @ Tue, 03 Nov 2020 20:19:41:  38000000 \\nINFO  @ Tue, 03 Nov 2020 20:19:46:  39000000 \\nINFO  @ Tue, 03 Nov 2020 20:19:50:  40000000 \\nINFO  @ Tue, 03 Nov 2020 20:19:53:  41000000 \\nINFO  @ Tue, 03 Nov 2020 20:19:57:  42000000 \\nINFO  @ Tue, 03 Nov 2020 20:20:01:  43000000 \\nINFO  @ Tue, 03 Nov 2020 20:20:04:  44000000 \\nINFO  @ Tue, 03 Nov 2020 20:20:08:  45000000 \\nINFO  @ Tue, 03 Nov 2020 20:20:12:  46000000 \\nINFO  @ Tue, 03 Nov 2020 20:20:15:  47000000 \\nINFO  @ Tue, 03 Nov 2020 20:20:19:  48000000 \\nINFO  @ Tue, 03 Nov 2020 20:20:23:  49000000 \\nINFO  @ Tue, 03 Nov 2020 20:20:26:  50000000 \\nINFO  @ Tue, 03 Nov 2020 20:20:30:  51000000 \\nINFO  @ Tue, 03 Nov 2020 20:20:33:  52000000 \\nINFO  @ Tue, 03 Nov 2020 20:20:37:  53000000 \\nINFO  @ Tue, 03 Nov 2020 20:20:41:  54000000 \\nINFO  @ Tue, 03 Nov 2020 20:20:44:  55000000 \\nINFO  @ Tue, 03 Nov 2020 20:20:48:  56000000 \\nINFO  @ Tue, 03 Nov 2020 20:20:52:  57000000 \\nINFO  @ Tue, 03 Nov 2020 20:20:55:  58000000 \\nINFO  @ Tue, 03 Nov 2020 20:20:59:  59000000 \\nINFO  @ Tue, 03 Nov 2020 20:21:02:  60000000 \\nINFO  @ Tue, 03 Nov 2020 20:21:06:  61000000 \\nINFO  @ Tue, 03 Nov 2020 20:21:10:  62000000 \\nINFO  @ Tue, 03 Nov 2020 20:21:14:  63000000 \\nINFO  @ Tue, 03 Nov 2020 20:21:17:  64000000 \\nINFO  @ Tue, 03 Nov 2020 20:21:21:  65000000 \\nINFO  @ Tue, 03 Nov 2020 20:21:25:  66000000 \\nINFO  @ Tue, 03 Nov 2020 20:21:27: 66660428 reads have been read. \\nINFO  @ Tue, 03 Nov 2020 20:21:28: #1 tag size is determined as 96 bps \\nINFO  @ Tue, 03 Nov 2020 20:21:28: #1 tag size = 96.0 \\nINFO  @ Tue, 03 Nov 2020 20:21:28: #1  total tags in treatment: 113234286 \\nINFO  @ Tue, 03 Nov 2020 20:21:28: #1 user defined the maximum tags... \\nINFO  @ Tue, 03 Nov 2020 20:21:28: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 03 Nov 2020 20:21:28: #1  tags after filtering in treatment: 103178779 \\nINFO  @ Tue, 03 Nov 2020 20:21:28: #1  Redundant rate of treatment: 0.09 \\nINFO  @ Tue, 03 Nov 2020 20:21:28: #1  total tags in control: 66660428 \\nINFO  @ Tue, 03 Nov 2020 20:21:28: #1 user defined the maximum tags... \\nINFO  @ Tue, 03 Nov 2020 20:21:28: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 03 Nov 2020 20:21:29: #1  tags after filtering in control: 65724989 \\nINFO  @ Tue, 03 Nov 2020 20:21:29: #1  Redundant rate of control: 0.01 \\nINFO  @ Tue, 03 Nov 2020 20:21:29: #1 finished! \\nINFO  @ Tue, 03 Nov 2020 20:21:29: #2 Build Peak Model... \\nINFO  @ Tue, 03 Nov 2020 20:21:29: #2 Skipped... \\nINFO  @ Tue, 03 Nov 2020 20:21:29: #2 Use 250 as fragment length \\nINFO  @ Tue, 03 Nov 2020 20:21:29: #3 Call peaks... \\nINFO  @ Tue, 03 Nov 2020 20:21:29: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 03 Nov 2020 20:28:07: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 03 Nov 2020 20:28:07: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_treat_pileup.bdg \\nINFO  @ Tue, 03 Nov 2020 20:28:07: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_control_lambda.bdg \\nINFO  @ Tue, 03 Nov 2020 20:28:07: #3   Pileup will be based on sequencing depth in control. \\nINFO  @ Tue, 03 Nov 2020 20:28:07: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 03 Nov 2020 20:34:11: #4 Write output xls file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_peaks.xls \\nINFO  @ Tue, 03 Nov 2020 20:34:12: #4 Write peak in narrowPeak format file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_peaks.narrowPeak \\nINFO  @ Tue, 03 Nov 2020 20:34:12: #4 Write summits bed file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_summits.bed \\nINFO  @ Tue, 03 Nov 2020 20:34:12: Done! \\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'INFO  @ Tue, 03 Nov 2020 20:34:13: \\n# Command line: callpeak -B -t ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-POLII_R1.mLb.clN.sorted.bam -c ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 250 -n MV411_MEF2D_NT_SC_63_VHL-POLII_R1 --outdir ../../data/chipseq_MEF2D_degraded/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = MV411_MEF2D_NT_SC_63_VHL-POLII_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/MV411_MEF2D_NT_SC_63_VHL-POLII_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 03 Nov 2020 20:34:13: #1 read tag files... \\nINFO  @ Tue, 03 Nov 2020 20:34:13: #1 read treatment tags... \\nINFO  @ Tue, 03 Nov 2020 20:34:17:  1000000 \\nINFO  @ Tue, 03 Nov 2020 20:34:21:  2000000 \\nINFO  @ Tue, 03 Nov 2020 20:34:26:  3000000 \\nINFO  @ Tue, 03 Nov 2020 20:34:30:  4000000 \\nINFO  @ Tue, 03 Nov 2020 20:34:36:  5000000 \\nINFO  @ Tue, 03 Nov 2020 20:34:40:  6000000 \\nINFO  @ Tue, 03 Nov 2020 20:34:44:  7000000 \\nINFO  @ Tue, 03 Nov 2020 20:34:51:  8000000 \\nINFO  @ Tue, 03 Nov 2020 20:34:56:  9000000 \\nINFO  @ Tue, 03 Nov 2020 20:35:00:  10000000 \\nINFO  @ Tue, 03 Nov 2020 20:35:04:  11000000 \\nINFO  @ Tue, 03 Nov 2020 20:35:09:  12000000 \\nINFO  @ Tue, 03 Nov 2020 20:35:13:  13000000 \\nINFO  @ Tue, 03 Nov 2020 20:35:18:  14000000 \\nINFO  @ Tue, 03 Nov 2020 20:35:25:  15000000 \\nINFO  @ Tue, 03 Nov 2020 20:35:29:  16000000 \\nINFO  @ Tue, 03 Nov 2020 20:35:33:  17000000 \\nINFO  @ Tue, 03 Nov 2020 20:35:38:  18000000 \\nINFO  @ Tue, 03 Nov 2020 20:35:42:  19000000 \\nINFO  @ Tue, 03 Nov 2020 20:35:47:  20000000 \\nINFO  @ Tue, 03 Nov 2020 20:35:51:  21000000 \\nINFO  @ Tue, 03 Nov 2020 20:35:58:  22000000 \\nINFO  @ Tue, 03 Nov 2020 20:36:02:  23000000 \\nINFO  @ Tue, 03 Nov 2020 20:36:06:  24000000 \\nINFO  @ Tue, 03 Nov 2020 20:36:11:  25000000 \\nINFO  @ Tue, 03 Nov 2020 20:36:15:  26000000 \\nINFO  @ Tue, 03 Nov 2020 20:36:19:  27000000 \\nINFO  @ Tue, 03 Nov 2020 20:36:24:  28000000 \\nINFO  @ Tue, 03 Nov 2020 20:36:29:  29000000 \\nINFO  @ Tue, 03 Nov 2020 20:36:36:  30000000 \\nINFO  @ Tue, 03 Nov 2020 20:36:41:  31000000 \\nINFO  @ Tue, 03 Nov 2020 20:36:45:  32000000 \\nINFO  @ Tue, 03 Nov 2020 20:36:50:  33000000 \\nINFO  @ Tue, 03 Nov 2020 20:36:55:  34000000 \\nINFO  @ Tue, 03 Nov 2020 20:37:00:  35000000 \\nINFO  @ Tue, 03 Nov 2020 20:37:04:  36000000 \\nINFO  @ Tue, 03 Nov 2020 20:37:08:  37000000 \\nINFO  @ Tue, 03 Nov 2020 20:37:15:  38000000 \\nINFO  @ Tue, 03 Nov 2020 20:37:20:  39000000 \\nINFO  @ Tue, 03 Nov 2020 20:37:24:  40000000 \\nINFO  @ Tue, 03 Nov 2020 20:37:28:  41000000 \\nINFO  @ Tue, 03 Nov 2020 20:37:33:  42000000 \\nINFO  @ Tue, 03 Nov 2020 20:37:37:  43000000 \\nINFO  @ Tue, 03 Nov 2020 20:37:42:  44000000 \\nINFO  @ Tue, 03 Nov 2020 20:37:50:  45000000 \\nINFO  @ Tue, 03 Nov 2020 20:37:55:  46000000 \\nINFO  @ Tue, 03 Nov 2020 20:37:59:  47000000 \\nINFO  @ Tue, 03 Nov 2020 20:38:03:  48000000 \\nINFO  @ Tue, 03 Nov 2020 20:38:08:  49000000 \\nINFO  @ Tue, 03 Nov 2020 20:38:12:  50000000 \\nINFO  @ Tue, 03 Nov 2020 20:38:17:  51000000 \\nINFO  @ Tue, 03 Nov 2020 20:38:21:  52000000 \\nINFO  @ Tue, 03 Nov 2020 20:38:26:  53000000 \\nINFO  @ Tue, 03 Nov 2020 20:38:31:  54000000 \\nINFO  @ Tue, 03 Nov 2020 20:38:38:  55000000 \\nINFO  @ Tue, 03 Nov 2020 20:38:42:  56000000 \\nINFO  @ Tue, 03 Nov 2020 20:38:46:  57000000 \\nINFO  @ Tue, 03 Nov 2020 20:38:51:  58000000 \\nINFO  @ Tue, 03 Nov 2020 20:38:55:  59000000 \\nINFO  @ Tue, 03 Nov 2020 20:39:00:  60000000 \\nINFO  @ Tue, 03 Nov 2020 20:39:04:  61000000 \\nINFO  @ Tue, 03 Nov 2020 20:39:09:  62000000 \\nINFO  @ Tue, 03 Nov 2020 20:39:14:  63000000 \\nINFO  @ Tue, 03 Nov 2020 20:39:18:  64000000 \\nINFO  @ Tue, 03 Nov 2020 20:39:26:  65000000 \\nINFO  @ Tue, 03 Nov 2020 20:39:30:  66000000 \\nINFO  @ Tue, 03 Nov 2020 20:39:35:  67000000 \\nINFO  @ Tue, 03 Nov 2020 20:39:39:  68000000 \\nINFO  @ Tue, 03 Nov 2020 20:39:43:  69000000 \\nINFO  @ Tue, 03 Nov 2020 20:39:48:  70000000 \\nINFO  @ Tue, 03 Nov 2020 20:39:53:  71000000 \\nINFO  @ Tue, 03 Nov 2020 20:39:57:  72000000 \\nINFO  @ Tue, 03 Nov 2020 20:40:02:  73000000 \\nINFO  @ Tue, 03 Nov 2020 20:40:06:  74000000 \\nINFO  @ Tue, 03 Nov 2020 20:40:10:  75000000 \\nINFO  @ Tue, 03 Nov 2020 20:40:15:  76000000 \\nINFO  @ Tue, 03 Nov 2020 20:40:19:  77000000 \\nINFO  @ Tue, 03 Nov 2020 20:40:24:  78000000 \\nINFO  @ Tue, 03 Nov 2020 20:40:28:  79000000 \\nINFO  @ Tue, 03 Nov 2020 20:40:33:  80000000 \\nINFO  @ Tue, 03 Nov 2020 20:40:37:  81000000 \\nINFO  @ Tue, 03 Nov 2020 20:40:41:  82000000 \\nINFO  @ Tue, 03 Nov 2020 20:40:46:  83000000 \\nINFO  @ Tue, 03 Nov 2020 20:40:50:  84000000 \\nINFO  @ Tue, 03 Nov 2020 20:40:54:  85000000 \\nINFO  @ Tue, 03 Nov 2020 20:40:59:  86000000 \\nINFO  @ Tue, 03 Nov 2020 20:41:03:  87000000 \\nINFO  @ Tue, 03 Nov 2020 20:41:08:  88000000 \\nINFO  @ Tue, 03 Nov 2020 20:41:12:  89000000 \\nINFO  @ Tue, 03 Nov 2020 20:41:17:  90000000 \\nINFO  @ Tue, 03 Nov 2020 20:41:22:  91000000 \\nINFO  @ Tue, 03 Nov 2020 20:41:26:  92000000 \\nINFO  @ Tue, 03 Nov 2020 20:41:30:  93000000 \\nINFO  @ Tue, 03 Nov 2020 20:41:35:  94000000 \\nINFO  @ Tue, 03 Nov 2020 20:41:39:  95000000 \\nINFO  @ Tue, 03 Nov 2020 20:41:44:  96000000 \\nINFO  @ Tue, 03 Nov 2020 20:41:49:  97000000 \\nINFO  @ Tue, 03 Nov 2020 20:41:53:  98000000 \\nINFO  @ Tue, 03 Nov 2020 20:41:57:  99000000 \\nINFO  @ Tue, 03 Nov 2020 20:42:02:  100000000 \\nINFO  @ Tue, 03 Nov 2020 20:42:07:  101000000 \\nINFO  @ Tue, 03 Nov 2020 20:42:11:  102000000 \\nINFO  @ Tue, 03 Nov 2020 20:42:16:  103000000 \\nINFO  @ Tue, 03 Nov 2020 20:42:20:  104000000 \\nINFO  @ Tue, 03 Nov 2020 20:42:25:  105000000 \\nINFO  @ Tue, 03 Nov 2020 20:42:30:  106000000 \\nINFO  @ Tue, 03 Nov 2020 20:42:34:  107000000 \\nINFO  @ Tue, 03 Nov 2020 20:42:39:  108000000 \\nINFO  @ Tue, 03 Nov 2020 20:42:43:  109000000 \\nINFO  @ Tue, 03 Nov 2020 20:42:48:  110000000 \\nINFO  @ Tue, 03 Nov 2020 20:42:53:  111000000 \\nINFO  @ Tue, 03 Nov 2020 20:42:57:  112000000 \\nINFO  @ Tue, 03 Nov 2020 20:42:57: 112023126 reads have been read. \\nINFO  @ Tue, 03 Nov 2020 20:42:59: #1.2 read input tags... \\nINFO  @ Tue, 03 Nov 2020 20:43:03:  1000000 \\nINFO  @ Tue, 03 Nov 2020 20:43:06:  2000000 \\nINFO  @ Tue, 03 Nov 2020 20:43:10:  3000000 \\nINFO  @ Tue, 03 Nov 2020 20:43:13:  4000000 \\nINFO  @ Tue, 03 Nov 2020 20:43:17:  5000000 \\nINFO  @ Tue, 03 Nov 2020 20:43:21:  6000000 \\nINFO  @ Tue, 03 Nov 2020 20:43:24:  7000000 \\nINFO  @ Tue, 03 Nov 2020 20:43:28:  8000000 \\nINFO  @ Tue, 03 Nov 2020 20:43:32:  9000000 \\nINFO  @ Tue, 03 Nov 2020 20:43:35:  10000000 \\nINFO  @ Tue, 03 Nov 2020 20:43:39:  11000000 \\nINFO  @ Tue, 03 Nov 2020 20:43:43:  12000000 \\nINFO  @ Tue, 03 Nov 2020 20:43:46:  13000000 \\nINFO  @ Tue, 03 Nov 2020 20:43:50:  14000000 \\nINFO  @ Tue, 03 Nov 2020 20:43:53:  15000000 \\nINFO  @ Tue, 03 Nov 2020 20:43:57:  16000000 \\nINFO  @ Tue, 03 Nov 2020 20:44:01:  17000000 \\nINFO  @ Tue, 03 Nov 2020 20:44:05:  18000000 \\nINFO  @ Tue, 03 Nov 2020 20:44:08:  19000000 \\nINFO  @ Tue, 03 Nov 2020 20:44:12:  20000000 \\nINFO  @ Tue, 03 Nov 2020 20:44:16:  21000000 \\nINFO  @ Tue, 03 Nov 2020 20:44:19:  22000000 \\nINFO  @ Tue, 03 Nov 2020 20:44:23:  23000000 \\nINFO  @ Tue, 03 Nov 2020 20:44:26:  24000000 \\nINFO  @ Tue, 03 Nov 2020 20:44:30:  25000000 \\nINFO  @ Tue, 03 Nov 2020 20:44:34:  26000000 \\nINFO  @ Tue, 03 Nov 2020 20:44:38:  27000000 \\nINFO  @ Tue, 03 Nov 2020 20:44:41:  28000000 \\nINFO  @ Tue, 03 Nov 2020 20:44:45:  29000000 \\nINFO  @ Tue, 03 Nov 2020 20:44:49:  30000000 \\nINFO  @ Tue, 03 Nov 2020 20:44:52:  31000000 \\nINFO  @ Tue, 03 Nov 2020 20:44:56:  32000000 \\nINFO  @ Tue, 03 Nov 2020 20:45:00:  33000000 \\nINFO  @ Tue, 03 Nov 2020 20:45:04:  34000000 \\nINFO  @ Tue, 03 Nov 2020 20:45:07:  35000000 \\nINFO  @ Tue, 03 Nov 2020 20:45:11:  36000000 \\nINFO  @ Tue, 03 Nov 2020 20:45:15:  37000000 \\nINFO  @ Tue, 03 Nov 2020 20:45:19:  38000000 \\nINFO  @ Tue, 03 Nov 2020 20:45:23:  39000000 \\nINFO  @ Tue, 03 Nov 2020 20:45:27:  40000000 \\nINFO  @ Tue, 03 Nov 2020 20:45:31:  41000000 \\nINFO  @ Tue, 03 Nov 2020 20:45:34:  42000000 \\nINFO  @ Tue, 03 Nov 2020 20:45:38:  43000000 \\nINFO  @ Tue, 03 Nov 2020 20:45:42:  44000000 \\nINFO  @ Tue, 03 Nov 2020 20:45:46:  45000000 \\nINFO  @ Tue, 03 Nov 2020 20:45:50:  46000000 \\nINFO  @ Tue, 03 Nov 2020 20:45:53:  47000000 \\nINFO  @ Tue, 03 Nov 2020 20:45:57:  48000000 \\nINFO  @ Tue, 03 Nov 2020 20:46:01:  49000000 \\nINFO  @ Tue, 03 Nov 2020 20:46:05:  50000000 \\nINFO  @ Tue, 03 Nov 2020 20:46:09:  51000000 \\nINFO  @ Tue, 03 Nov 2020 20:46:12:  52000000 \\nINFO  @ Tue, 03 Nov 2020 20:46:16:  53000000 \\nINFO  @ Tue, 03 Nov 2020 20:46:20:  54000000 \\nINFO  @ Tue, 03 Nov 2020 20:46:24:  55000000 \\nINFO  @ Tue, 03 Nov 2020 20:46:28:  56000000 \\nINFO  @ Tue, 03 Nov 2020 20:46:32:  57000000 \\nINFO  @ Tue, 03 Nov 2020 20:46:35:  58000000 \\nINFO  @ Tue, 03 Nov 2020 20:46:39:  59000000 \\nINFO  @ Tue, 03 Nov 2020 20:46:43:  60000000 \\nINFO  @ Tue, 03 Nov 2020 20:46:47:  61000000 \\nINFO  @ Tue, 03 Nov 2020 20:46:51:  62000000 \\nINFO  @ Tue, 03 Nov 2020 20:46:54:  63000000 \\nINFO  @ Tue, 03 Nov 2020 20:46:58:  64000000 \\nINFO  @ Tue, 03 Nov 2020 20:47:02:  65000000 \\nINFO  @ Tue, 03 Nov 2020 20:47:06:  66000000 \\nINFO  @ Tue, 03 Nov 2020 20:47:09: 66660428 reads have been read. \\nINFO  @ Tue, 03 Nov 2020 20:47:09: #1 tag size is determined as 98 bps \\nINFO  @ Tue, 03 Nov 2020 20:47:09: #1 tag size = 98.0 \\nINFO  @ Tue, 03 Nov 2020 20:47:09: #1  total tags in treatment: 112023126 \\nINFO  @ Tue, 03 Nov 2020 20:47:09: #1 user defined the maximum tags... \\nINFO  @ Tue, 03 Nov 2020 20:47:09: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 03 Nov 2020 20:47:10: #1  tags after filtering in treatment: 100988439 \\nINFO  @ Tue, 03 Nov 2020 20:47:10: #1  Redundant rate of treatment: 0.10 \\nINFO  @ Tue, 03 Nov 2020 20:47:10: #1  total tags in control: 66660428 \\nINFO  @ Tue, 03 Nov 2020 20:47:10: #1 user defined the maximum tags... \\nINFO  @ Tue, 03 Nov 2020 20:47:10: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 03 Nov 2020 20:47:10: #1  tags after filtering in control: 65724989 \\nINFO  @ Tue, 03 Nov 2020 20:47:10: #1  Redundant rate of control: 0.01 \\nINFO  @ Tue, 03 Nov 2020 20:47:10: #1 finished! \\nINFO  @ Tue, 03 Nov 2020 20:47:10: #2 Build Peak Model... \\nINFO  @ Tue, 03 Nov 2020 20:47:10: #2 Skipped... \\nINFO  @ Tue, 03 Nov 2020 20:47:10: #2 Use 250 as fragment length \\nINFO  @ Tue, 03 Nov 2020 20:47:10: #3 Call peaks... \\nINFO  @ Tue, 03 Nov 2020 20:47:10: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 03 Nov 2020 20:53:32: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 03 Nov 2020 20:53:32: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_MEF2D_NT_SC_63_VHL-POLII_R1_treat_pileup.bdg \\nINFO  @ Tue, 03 Nov 2020 20:53:32: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_MEF2D_NT_SC_63_VHL-POLII_R1_control_lambda.bdg \\nINFO  @ Tue, 03 Nov 2020 20:53:32: #3   Pileup will be based on sequencing depth in control. \\nINFO  @ Tue, 03 Nov 2020 20:53:32: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 03 Nov 2020 20:59:34: #4 Write output xls file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-POLII_R1_peaks.xls \\nINFO  @ Tue, 03 Nov 2020 20:59:35: #4 Write peak in narrowPeak format file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-POLII_R1_peaks.narrowPeak \\nINFO  @ Tue, 03 Nov 2020 20:59:35: #4 Write summits bed file... ../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-POLII_R1_summits.bed \\nINFO  @ Tue, 03 Nov 2020 20:59:35: Done! \\n'\n",
      "65724989 65724989\n",
      "doing differential peak binding\n"
     ]
    }
   ],
   "source": [
    "#on unscalled data \n",
    "for i in range(int(len(bams[1:])/2)):\n",
    "    if i < 0:\n",
    "        continue\n",
    "    name1 = bams[1+i]\n",
    "    name2 = bams[7+i]\n",
    "    print(name1,name2)\n",
    "    chip.fullDiffPeak(name1,name2, control1='../../data/'+project+'/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam', directory = \"../../data/\"+project+\"/diffData_unscaled/\", res_directory = \"../../data/\"+project+\"/diffPeaks_unscaled/\",pairedend=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 214,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bigWig',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1.mLb.clN.bigWig',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1.mLb.clN.bigWig',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1.mLb.clN.bigWig',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1.mLb.clN.bigWig',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1.mLb.clN.bigWig',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1.mLb.clN.bigWig',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1.mLb.clN.bigWig',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1.mLb.clN.bigWig',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_VHL-MED1_R1.mLb.clN.bigWig',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1.mLb.clN.bigWig',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_VHL-MYC_R1.mLb.clN.bigWig',\n",
       " '../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_VHL-POLII_R1.mLb.clN.bigWig']"
      ]
     },
     "execution_count": 214,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bw = ! ls ../../data/$project/bwa/mergedLibrary/bigwig/*.bigWig\n",
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 215,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_treat_pileup_c3.0_common.bed',\n",
       " '../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_treat_pileup_c3.0_common.bed',\n",
       " '../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MED1_R1_treat_pileup_c3.0_common.bed',\n",
       " '../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_treat_pileup_c3.0_common.bed',\n",
       " '../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MYC_R1_treat_pileup_c3.0_common.bed',\n",
       " '../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-POLII_R1_treat_pileup_c3.0_common.bed']"
      ]
     },
     "execution_count": 215,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "cond1peak = ! ls ../../data/$project/diffPeaks_unscaled/*cond1.bed\n",
    "cond2peak = ! ls ../../data/$project/diffPeaks_unscaled/*cond2.bed\n",
    "commonpeak = ! ls ../../data/$project/diffPeaks_unscaled/*common.bed\n",
    "commonpeak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 216,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"FLAG_MEF2D\",\n",
    "\"IFR8\",\n",
    "\"MED1\",\n",
    "\"MEF2C\",\n",
    "\"MYC\",\n",
    "\"POLII\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 217,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bigWig ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1.mLb.clN.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bigWig ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_treat_pileup_c3.0_common.bed ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_treat_pileup_c3.0_cond2.bed --missingDataAsZero --outFileName ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/FLAG_MEF2D_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/FLAG_MEF2D_mat.gz --outFileName ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/FLAG_MEF2D_mat.pdf --refPointLabel center --regionsLabel common VHL_peaks --plotTitle FLAG_MEF2D', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270340v1\\t      1428\\nchrUn_KI270371v1\\t      2805\\n         chrEBV\\t    171823\\nchrUn_KI270372v1\\t      1650\\nchrUn_KI270752v1\\t     27745\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270383v1\\t      1750\\nchrUn_KI270394v1\\t       970\\nchrUn_KI270388v1\\t      1216\\nchrUn_KI270373v1\\t      1451\\n')\n",
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1.mLb.clN.bigWig ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1.mLb.clN.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1.mLb.clN.bigWig ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_treat_pileup_c3.0_cond1.bed ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_treat_pileup_c3.0_common.bed ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_treat_pileup_c3.0_cond2.bed --missingDataAsZero --outFileName ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/IFR8_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/IFR8_mat.gz --outFileName ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/IFR8_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks --plotTitle IFR8', returncode=0, stdout=b'', stderr=b\"\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270371v1\\t      2805\\nchrUn_KI270376v1\\t      1136\\nchrUn_KI270752v1\\t     27745\\nchrUn_KI270386v1\\t      1788\\n         chrEBV\\t    171823\\nchrUn_KI270388v1\\t      1216\\nchrUn_KI270312v1\\t       998\\nchrUn_KI270387v1\\t      1537\\nchrUn_KI270334v1\\t      1368\\nchrUn_KI270425v1\\t      1884\\nchrUn_KI270393v1\\t      1308\\nOne of the groups defined in the bed file is too small.\\nGroups that are too small can't be plotted. \\nWARNING: Group 'MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_treat_pileup_c3.0_cond1.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n",
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1.mLb.clN.bigWig ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1.mLb.clN.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1.mLb.clN.bigWig ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MED1_R1_treat_pileup_c3.0_cond1.bed ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MED1_R1_treat_pileup_c3.0_common.bed --missingDataAsZero --outFileName ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MED1_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MED1_mat.gz --outFileName ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MED1_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common --plotTitle MED1', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270376v1\\t      1136\\nchrUn_KI270371v1\\t      2805\\nchrUn_KI270335v1\\t      1048\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270312v1\\t       998\\n')\n",
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1.mLb.clN.bigWig ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_VHL-MED1_R1.mLb.clN.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1.mLb.clN.bigWig ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_VHL-MED1_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_treat_pileup_c3.0_common.bed ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_treat_pileup_c3.0_cond2.bed --missingDataAsZero --outFileName ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MEF2C_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MEF2C_mat.gz --outFileName ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MEF2C_mat.pdf --refPointLabel center --regionsLabel common VHL_peaks --plotTitle MEF2C', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270334v1\\t      1368\\nchrUn_KI270335v1\\t      1048\\nchrUn_KI270376v1\\t      1136\\nchrUn_KI270375v1\\t      2378\\n')\n",
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1.mLb.clN.bigWig ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1.mLb.clN.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1.mLb.clN.bigWig ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MYC_R1_treat_pileup_c3.0_common.bed ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MYC_R1_treat_pileup_c3.0_cond2.bed --missingDataAsZero --outFileName ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MYC_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MYC_mat.gz --outFileName ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MYC_mat.pdf --refPointLabel center --regionsLabel common VHL_peaks --plotTitle MYC', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270376v1\\t      1136\\nchrUn_KI270335v1\\t      1048\\nchrUn_KI270385v1\\t       990\\nchrUn_KI270388v1\\t      1216\\nchrUn_KI270375v1\\t      2378\\nchrUn_KI270379v1\\t      1045\\n')\n",
      "../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1.mLb.clN.bigWig ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_VHL-MYC_R1.mLb.clN.bigWig\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1.mLb.clN.bigWig ../../data/chipseq_MEF2D_degraded/bwa/mergedLibrary/bigwig/MV411_MEF2D_NT_SC_63_VHL-MYC_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-POLII_R1_treat_pileup_c3.0_cond1.bed ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-POLII_R1_treat_pileup_c3.0_common.bed ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-POLII_R1_treat_pileup_c3.0_cond2.bed --missingDataAsZero --outFileName ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/POLII_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/POLII_mat.gz --outFileName ../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/POLII_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks --plotTitle POLII', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270334v1\\t      1368\\nchrUn_KI270388v1\\t      1216\\nchrUn_KI270375v1\\t      2378\\nchrUn_KI270374v1\\t      2656\\n')\n"
     ]
    }
   ],
   "source": [
    "for i in range(int(len(bw)/2)):\n",
    "    name1 = bw[i]\n",
    "    name2 = bw[6+i]\n",
    "    peak = [cond1peak[i], commonpeak[i], cond2peak[i]]\n",
    "    #for val in peak:\n",
    "     #   chip.dropWeirdChromosomes(val)\n",
    "    name = names[i]\n",
    "    print(name1,name2)\n",
    "    chip.getPeaksAt(peak, [name1, name2], bigwignames=['DMSO', 'VHL'], peaknames=['DMSO_peaks', 'common', 'VHL_peaks'], window=3000, folder=\"\", title=name, numthreads=7, torecompute=True, refpoint='center', name='../../data/'+project+'/diffPeaks_unscaled/'+name+'_mat.pdf', withDeeptools=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 228,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying gs://amlproject/Chip/chipseq_MEF2D_degraded/bwa/mergedLibrary/deepTools/plotFingerprint/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1.plotFingerprint.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_MEF2D_degraded/bwa/mergedLibrary/deepTools/plotFingerprint/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1.plotFingerprint.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_MEF2D_degraded/bwa/mergedLibrary/deepTools/plotFingerprint/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1.plotFingerprint.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_MEF2D_degraded/bwa/mergedLibrary/deepTools/plotFingerprint/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1.plotFingerprint.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_MEF2D_degraded/bwa/mergedLibrary/deepTools/plotFingerprint/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1.plotFingerprint.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_MEF2D_degraded/bwa/mergedLibrary/deepTools/plotFingerprint/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D.plotFingerprint.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_MEF2D_degraded/bwa/mergedLibrary/deepTools/plotFingerprint/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1.plotFingerprint.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_MEF2D_degraded/bwa/mergedLibrary/deepTools/plotFingerprint/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1.plotFingerprint.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_MEF2D_degraded/bwa/mergedLibrary/deepTools/plotFingerprint/MV411_MEF2D_NT_SC_63_VHL-MED1_R1.plotFingerprint.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_MEF2D_degraded/bwa/mergedLibrary/deepTools/plotFingerprint/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1.plotFingerprint.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_MEF2D_degraded/bwa/mergedLibrary/deepTools/plotFingerprint/MV411_MEF2D_NT_SC_63_VHL-POLII_R1_R1.plotFingerprint.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_MEF2D_degraded/bwa/mergedLibrary/deepTools/plotFingerprint/MV411_MEF2D_NT_SC_63_VHL-MYC_R1.plotFingerprint.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_MEF2D_degraded/bwa/mergedLibrary/deepTools/plotProfile/INPUT_R1.plotProfile.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_MEF2D_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1.plotProfile.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_MEF2D_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1.plotProfile.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_MEF2D_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1.plotProfile.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_MEF2D_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1.plotProfile.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_MEF2D_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1.plotProfile.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_MEF2D_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1.plotProfile.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_MEF2D_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D.plotProfile.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_MEF2D_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1.plotProfile.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_MEF2D_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_MEF2D_NT_SC_63_VHL-MED1_R1.plotProfile.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_MEF2D_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1.plotProfile.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_MEF2D_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_MEF2D_NT_SC_63_VHL-MYC_R1.plotProfile.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_MEF2D_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_MEF2D_NT_SC_63_VHL-POLII_R1_R1.plotProfile.pdf...\n",
      "- [25/25 files][  9.2 MiB/  9.2 MiB] 100% Done                                  \n",
      "Operation completed over 25 objects/9.2 MiB.                                     \n"
     ]
    }
   ],
   "source": [
    "h.createFoldersFor('../results/'+project+'/plots/unscaled/diffPeaks/')\n",
    "! cp ../../data/$project/diffPeaks_unscaled/*.pdf ../results/$project/plots/unscaled/diffPeaks/\n",
    "\n",
    "! gsutil -m cp gs://amlproject/Chip/$project/bwa/mergedLibrary/deepTools/**.pdf ../results/$project/plots/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 236,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MED1_R1_treat_pileup_c3.0_common.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_treat_pileup_c3.0_cond2.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/FLAG_MEF2D_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MED1_R1_treat_pileup_c3.0_cond1.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/POLII_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_treat_pileup_c3.0_cond1.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MYC_R1_treat_pileup_c3.0_common.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MYC_R1_treat_pileup_c3.0_cond1.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/IFR8_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-POLII_R1_treat_pileup_c3.0_common.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-POLII_R1_treat_pileup_c3.0_cond1.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/MED1_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_treat_pileup_c3.0_cond2.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_treat_pileup_c3.0_common.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-POLII_R1_treat_pileup_c3.0_cond2.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/MEF2C_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MYC_R1_treat_pileup_c3.0_cond2.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/POLII_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/MEF2C_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/FLAG_MEF2D_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/MYC_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MED1_R1_treat_pileup_c3.0_cond2.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/MED1_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_treat_pileup_c3.0_cond2.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_treat_pileup_c3.0_common.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_treat_pileup_c3.0_common.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/IFR8_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/MYC_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_treat_pileup_c3.0_cond1.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_treat_pileup_c3.0_cond1.bed [Content-Type=application/octet-stream]...\n",
      "- [30/30 files][242.2 MiB/242.2 MiB] 100% Done                                  \n",
      "Operation completed over 30 objects/242.2 MiB.                                   \n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-POLII_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "==> NOTE: You are uploading one or more large file(s), which would run          \n",
      "significantly faster if you enable parallel composite uploads. This\n",
      "feature can be enabled by editing the\n",
      "\"parallel_composite_upload_threshold\" value in your .boto\n",
      "configuration file. However, note that if you do this large files will\n",
      "be uploaded as `composite objects\n",
      "<https://cloud.google.com/storage/docs/composite-objects>`_,which\n",
      "means that any user who downloads such objects will need to have a\n",
      "compiled crcmod installed (see \"gsutil help crcmod\"). This is because\n",
      "without a compiled crcmod, computing checksums on composite objects is\n",
      "so slow that gsutil disables downloads of composite objects.\n",
      "\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-POLII_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-MED1_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-MYC_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-MYC_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-MYC_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-MED1_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-MED1_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-POLII_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-POLII_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-MYC_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-MED1_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-MED1_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-POLII_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-MYC_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "- [60/60 files][118.9 GiB/118.9 GiB] 100% Done  98.1 MiB/s ETA 00:00:00         \n",
      "Operation completed over 60 objects/118.9 GiB.                                   \n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MED1_R1_treat_pileup_c3.0_common.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_treat_pileup_c3.0_cond2.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/FLAG_MEF2D_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_treat_pileup_c3.0_cond1.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MYC_R1_treat_pileup_c3.0_common.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MED1_R1_treat_pileup_c3.0_cond1.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/IFR8_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-POLII_R1_treat_pileup_c3.0_common.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MYC_R1_treat_pileup_c3.0_cond1.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/POLII_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-POLII_R1_treat_pileup_c3.0_cond1.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-POLII_R1_treat_pileup_c3.0_cond2.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MED1_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_treat_pileup_c3.0_cond2.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MEF2C_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/POLII_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MYC_R1_treat_pileup_c3.0_cond2.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_treat_pileup_c3.0_common.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/FLAG_MEF2D_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MEF2C_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MED1_mat.pdf [Content-Type=application/pdf]...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MYC_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_treat_pileup_c3.0_cond2.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MED1_R1_treat_pileup_c3.0_cond2.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_treat_pileup_c3.0_common.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_treat_pileup_c3.0_common.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/IFR8_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MYC_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_treat_pileup_c3.0_cond1.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffPeaks_unscaled/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_treat_pileup_vs_MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_treat_pileup_c3.0_cond1.bed [Content-Type=application/octet-stream]...\n",
      "- [30/30 files][259.6 MiB/259.6 MiB] 100% Done                                  \n",
      "Operation completed over 30 objects/259.6 MiB.                                   \n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-POLII_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "==> NOTE: You are uploading one or more large file(s), which would run          \n",
      "significantly faster if you enable parallel composite uploads. This\n",
      "feature can be enabled by editing the\n",
      "\"parallel_composite_upload_threshold\" value in your .boto\n",
      "configuration file. However, note that if you do this large files will\n",
      "be uploaded as `composite objects\n",
      "<https://cloud.google.com/storage/docs/composite-objects>`_,which\n",
      "means that any user who downloads such objects will need to have a\n",
      "compiled crcmod installed (see \"gsutil help crcmod\"). This is because\n",
      "without a compiled crcmod, computing checksums on composite objects is\n",
      "so slow that gsutil disables downloads of composite objects.\n",
      "\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-IFR8_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-POLII_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-MED1_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-MYC_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-MYC_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-MYC_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-MED1_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-MED1_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-POLII_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-MEF2C_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-POLII_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-IFR8_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-MYC_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MEF2C_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-MED1_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-POLII_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-MYC_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-MED1_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MYC_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-FLAG_MEF2D_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-MED1_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Catching up md5 for file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_DMSO-POLII_R1_control_lambda.bdg...\n",
      "Catching up md5 for file://../../data/chipseq_MEF2D_degraded/diffData_unscaled/MV411_MEF2D_NT_SC_63_VHL-FLAG_MEF2D_R1_control_lambda.bdg...\n",
      "| [60/60 files][ 86.8 GiB/ 86.8 GiB] 100% Done  99.5 MiB/s ETA 00:00:00         \n",
      "Operation completed over 60 objects/86.8 GiB.                                    \n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/VHL_MED1_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/VHL_MEF2C_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/DMSO_IFR8_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/DMSO_FLAG_MEF2D_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/POLII_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/VHL_MYC_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/VHL_IFR8_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/DMSO_MED1_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/IFR8_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/VHL_MEF2C_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/DMSO_FLAG_MEF2D_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/VHL_MED1_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/MED1_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/VHL_FLAG_MEF2D_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/MEF2C_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/DMSO_POLII_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/POLII_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/VHL_FLAG_MEF2D_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/DMSO_MEF2C_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/DMSO_IFR8_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/VHL_POLII_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/MED1_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/DMSO_MYC_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/VHL_MYC_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/VHL_POLII_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/MEF2C_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/DMSO_MYC_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/DMSO_MEF2C_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/FLAG_MEF2D_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/VHL_IFR8_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/MYC_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/DMSO_POLII_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/IFR8_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/MYC_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_MEF2D_degraded/peakplot/DMSO_MED1_mat.pdf [Content-Type=application/pdf]...\n",
      "/ [35/35 files][268.3 MiB/268.3 MiB] 100% Done                                  \n",
      "Operation completed over 35 objects/268.3 MiB.                                   \n"
     ]
    }
   ],
   "source": [
    "! gsutil -m cp -r ../../data/$project/diffPeaks gs://amlproject/Chip/$project/\n",
    "! gsutil -m cp -r ../../data/$project/diffData gs://amlproject/Chip/$project/\n",
    "! gsutil -m cp -r ../../data/$project/diffPeaks_unscaled gs://amlproject/Chip/$project/\n",
    "! gsutil -m cp -r ../../data/$project/diffData_unscaled gs://amlproject/Chip/$project/\n",
    "! gsutil -m cp -r ../../data/$project/peakplot gs://amlproject/Chip/$project/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## v7"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "project=\"chipseq_MEF2CMEF2D_degraded\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! gsutil ls gs://transfer-amlproject/200924_MP8230_fastq/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 271,
   "metadata": {},
   "outputs": [],
   "source": [
    "mkdir ../data/$project/ && mkdir ../data/$project/qc/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 272,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/multiqc_report.html...\n",
      "/ [1/1 files][  1.3 MiB/  1.3 MiB] 100% Done                                    \n",
      "Operation completed over 1 objects/1.3 MiB.                                      \n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP1_MP8230_S113_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP1_MP8230_S113_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP1_MP8230_S113_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP1_MP8230_S113_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP2_MP8230_S114_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP2_MP8230_S114_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP2_MP8230_S114_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP2_MP8230_S114_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP3_MP8230_S115_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP3_MP8230_S115_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP3_MP8230_S115_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP3_MP8230_S115_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP4_MP8230_S116_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP4_MP8230_S116_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP4_MP8230_S116_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP4_MP8230_S116_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP5_MP8230_S117_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP5_MP8230_S117_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP5_MP8230_S117_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP5_MP8230_S117_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP6_MP8230_S118_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP6_MP8230_S118_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP6_MP8230_S118_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP6_MP8230_S118_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP7_MP8230_S119_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP7_MP8230_S119_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP7_MP8230_S119_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP7_MP8230_S119_R2_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP8_MP8230_S120_R1_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP8_MP8230_S120_R1_001_fastqc.zip...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP8_MP8230_S120_R2_001_fastqc.html...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/Reports/20200924_MP8_MP8230_S120_R2_001_fastqc.zip...\n",
      "/ [32/32 files][ 16.9 MiB/ 16.9 MiB] 100% Done                                  \n",
      "Operation completed over 32 objects/16.9 MiB.                                    \n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/multiqc_data/multiqc.log...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/multiqc_data/multiqc_data.json...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/multiqc_data/multiqc_fastqc.txt...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/multiqc_data/multiqc_general_stats.txt...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/multiqc_data/multiqc_qualimap_bamqc_genome_results.txt...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/multiqc_data/multiqc_sources.txt...\n",
      "/ [6/6 files][966.4 KiB/966.4 KiB] 100% Done                                    \n",
      "Operation completed over 6 objects/966.4 KiB.                                    \n"
     ]
    }
   ],
   "source": [
    "!gsutil -m cp gs://transfer-amlproject/200924_MP8230_fastq/multiqc_report.html ../data/$project/qc/\n",
    "!gsutil -m cp -r gs://transfer-amlproject/200924_MP8230_fastq/Reports/ ../data/$project/qc/\n",
    "!gsutil -m cp -r gs://transfer-amlproject/200924_MP8230_fastq/multiqc_data/ ../data/$project/qc/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 273,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Omitting prefix \"gs://transfer-amlproject/200924_MP8230_fastq/Reports/\". (Did you mean to do cp -r?)\n",
      "Omitting prefix \"gs://transfer-amlproject/200924_MP8230_fastq/multiqc_data/\". (Did you mean to do cp -r?)\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/20200924_MP1_MP8230_S113_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/20200924_MP1_MP8230_S113_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/20200924_MP2_MP8230_S114_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/20200924_MP2_MP8230_S114_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/20200924_MP3_MP8230_S115_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/20200924_MP3_MP8230_S115_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/20200924_MP4_MP8230_S116_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/20200924_MP4_MP8230_S116_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/20200924_MP5_MP8230_S117_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/20200924_MP5_MP8230_S117_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/20200924_MP6_MP8230_S118_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/20200924_MP6_MP8230_S118_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/20200924_MP7_MP8230_S119_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/20200924_MP7_MP8230_S119_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/20200924_MP8_MP8230_S120_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/20200924_MP8_MP8230_S120_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/200924_MP8230_fastq/multiqc_report.html [Content-Type=text/html]...\n",
      "- [17/17 files][ 23.1 GiB/ 23.1 GiB] 100% Done                                  \n",
      "Operation completed over 17 objects/23.1 GiB.                                    \n"
     ]
    }
   ],
   "source": [
    "! gsutil -m cp gs://transfer-amlproject/200924_MP8230_fastq/*  gs://transfer-amlproject/$project/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 274,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying gs://transfer-amlproject/chipseq_MEFF2CMEF2D_degraded/20200924_MP1_MP8230_S113_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEFF2CMEF2D_degraded/20200924_MP1_MP8230_S113_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEFF2CMEF2D_degraded/20200924_MP2_MP8230_S114_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEFF2CMEF2D_degraded/20200924_MP2_MP8230_S114_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEFF2CMEF2D_degraded/20200924_MP3_MP8230_S115_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEFF2CMEF2D_degraded/20200924_MP3_MP8230_S115_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEFF2CMEF2D_degraded/20200924_MP4_MP8230_S116_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEFF2CMEF2D_degraded/20200924_MP4_MP8230_S116_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEFF2CMEF2D_degraded/20200924_MP5_MP8230_S117_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEFF2CMEF2D_degraded/20200924_MP5_MP8230_S117_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEFF2CMEF2D_degraded/20200924_MP6_MP8230_S118_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEFF2CMEF2D_degraded/20200924_MP6_MP8230_S118_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEFF2CMEF2D_degraded/20200924_MP7_MP8230_S119_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEFF2CMEF2D_degraded/20200924_MP7_MP8230_S119_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEFF2CMEF2D_degraded/20200924_MP8_MP8230_S120_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEFF2CMEF2D_degraded/20200924_MP8_MP8230_S120_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/chipseq_MEFF2CMEF2D_degraded/multiqc_report.html...\n",
      "- [17/17 files][ 23.1 GiB/ 23.1 GiB] 100% Done  86.4 MiB/s ETA 00:00:00         \n",
      "Operation completed over 17 objects/23.1 GiB.                                    \n"
     ]
    }
   ],
   "source": [
    "! mkdir ../../data/$project && mkdir ../../data/$project/fastqs && gsutil -m cp gs://transfer-amlproject/$project/* ../../data/$project/fastqs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 275,
   "metadata": {},
   "outputs": [],
   "source": [
    "! rm ../../data/$project/fastqs/multiqc_report.html"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 276,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['20200924_MP1_MP8230_S113_R1_001.fastq.gz',\n",
       " '20200924_MP1_MP8230_S113_R2_001.fastq.gz',\n",
       " '20200924_MP2_MP8230_S114_R1_001.fastq.gz',\n",
       " '20200924_MP2_MP8230_S114_R2_001.fastq.gz',\n",
       " '20200924_MP3_MP8230_S115_R1_001.fastq.gz',\n",
       " '20200924_MP3_MP8230_S115_R2_001.fastq.gz',\n",
       " '20200924_MP4_MP8230_S116_R1_001.fastq.gz',\n",
       " '20200924_MP4_MP8230_S116_R2_001.fastq.gz',\n",
       " '20200924_MP5_MP8230_S117_R1_001.fastq.gz',\n",
       " '20200924_MP5_MP8230_S117_R2_001.fastq.gz',\n",
       " '20200924_MP6_MP8230_S118_R1_001.fastq.gz',\n",
       " '20200924_MP6_MP8230_S118_R2_001.fastq.gz',\n",
       " '20200924_MP7_MP8230_S119_R1_001.fastq.gz',\n",
       " '20200924_MP7_MP8230_S119_R2_001.fastq.gz',\n",
       " '20200924_MP8_MP8230_S120_R1_001.fastq.gz',\n",
       " '20200924_MP8_MP8230_S120_R2_001.fastq.gz']"
      ]
     },
     "execution_count": 276,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "a = ! ls ../../data/$project/fastqs\n",
    "a"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 277,
   "metadata": {},
   "outputs": [],
   "source": [
    "rename = {\n",
    "\"20200924_MP1_MP8230_S113\": \"mp893-MV411_RNP_AAVS1-H3K27AC-r3\",\n",
    "\"20200924_MP2_MP8230_S114\": \"mp894-MV411_RNP_AAVS1-H3K27AC-r4\",\n",
    "\"20200924_MP3_MP8230_S115\": \"mp895-MV411_RNP_MEF2C-H3K27AC-r1\",\n",
    "\"20200924_MP4_MP8230_S116\": \"mp896-MV411_RNP_MEF2C-H3K27AC-r2\",\n",
    "\"20200924_MP5_MP8230_S117\": \"mp897-MV411_RNP_MEF2D-H3K27AC-r3\",\n",
    "\"20200924_MP6_MP8230_S118\": \"mp898-MV411_RNP_MEF2D-H3K27AC-r4\",\n",
    "\"20200924_MP7_MP8230_S119\": \"mp899-MV411_RNP_MEF2C_MEF2D-H3K27AC-r1\",\n",
    "\"20200924_MP8_MP8230_S120\": \"mp900-MV411_RNP_MEF2C_MEF2D-H3K27AC-r2\",}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 278,
   "metadata": {},
   "outputs": [],
   "source": [
    "for val in a:\n",
    "    rep = val\n",
    "    for k,v in rename.items():\n",
    "        rep = rep.replace(k,v)\n",
    "    !mv ../../data/$project/fastqs/$val ../../data/$project/fastqs/$rep"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 279,
   "metadata": {},
   "outputs": [],
   "source": [
    "from gsheets import Sheets\n",
    "sheets = Sheets.from_files('~/.client_secret.json', '~/.storage.json')\n",
    "url=\"https://docs.google.com/spreadsheets/d/1yFLjYB1McU530JnLgL0QIMAKIkVl3kl0_LCHje2gk8U\"\n",
    "gsheet = sheets.get(url).sheets[2].to_frame()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 280,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>id</th>\n",
       "      <th>cell line</th>\n",
       "      <th>replicate</th>\n",
       "      <th>protein</th>\n",
       "      <th>quality</th>\n",
       "      <th>paired_end</th>\n",
       "      <th>matching input name</th>\n",
       "      <th>processed</th>\n",
       "      <th>name</th>\n",
       "      <th>previous name</th>\n",
       "      <th>...</th>\n",
       "      <th>ratio to droso</th>\n",
       "      <th>unique mapped reads(droso)</th>\n",
       "      <th>scaling factor</th>\n",
       "      <th>Total QC</th>\n",
       "      <th>folderNarrow</th>\n",
       "      <th>folderCompensated</th>\n",
       "      <th>folderQC</th>\n",
       "      <th>folderBroad</th>\n",
       "      <th>folder Bigwig</th>\n",
       "      <th>folder diffPeaks</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>mp100</td>\n",
       "      <td>U937</td>\n",
       "      <td>1.0</td>\n",
       "      <td>INPUT</td>\n",
       "      <td>NaN</td>\n",
       "      <td>n</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Y</td>\n",
       "      <td>mp100-U937-INPUT-r1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://storage.cloud.google.com/amlproject/Ch...</td>\n",
       "      <td>https://console.cloud.google.com/storage/brows...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://console.cloud.google.com/storage/brows...</td>\n",
       "      <td>https://console.cloud.google.com/storage/brows...</td>\n",
       "      <td>https://console.cloud.google.com/storage/brows...</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>mp101</td>\n",
       "      <td>NOMO1</td>\n",
       "      <td>1.0</td>\n",
       "      <td>INPUT</td>\n",
       "      <td>NaN</td>\n",
       "      <td>n</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Y</td>\n",
       "      <td>mp101-NOMO1-INPUT-r1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>mp102</td>\n",
       "      <td>UT7</td>\n",
       "      <td>1.0</td>\n",
       "      <td>INPUT</td>\n",
       "      <td>NaN</td>\n",
       "      <td>n</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Y</td>\n",
       "      <td>mp102-UT7-INPUT-r1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>mp106</td>\n",
       "      <td>MV411</td>\n",
       "      <td>1.0</td>\n",
       "      <td>MYB</td>\n",
       "      <td>x</td>\n",
       "      <td>n</td>\n",
       "      <td>INPUT_MV411</td>\n",
       "      <td>Y</td>\n",
       "      <td>mp106-MV411-MYB-r1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>mp109</td>\n",
       "      <td>M6</td>\n",
       "      <td>1.0</td>\n",
       "      <td>CEBPA</td>\n",
       "      <td>x</td>\n",
       "      <td>n</td>\n",
       "      <td>INPUT_M6</td>\n",
       "      <td>Y</td>\n",
       "      <td>mp109-M6-CEBPA-r1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>325</th>\n",
       "      <td>mp897</td>\n",
       "      <td>MV411_RNP_MEF2D</td>\n",
       "      <td>3.0</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>NaN</td>\n",
       "      <td>y</td>\n",
       "      <td>INPUT_MV411</td>\n",
       "      <td>NaN</td>\n",
       "      <td>mp897-MV411_RNP_MEF2D-H3K27AC-r3</td>\n",
       "      <td>20200924_MP5_MP8230_S117</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>326</th>\n",
       "      <td>mp898</td>\n",
       "      <td>MV411_RNP_MEF2D</td>\n",
       "      <td>4.0</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>NaN</td>\n",
       "      <td>y</td>\n",
       "      <td>INPUT_MV411</td>\n",
       "      <td>NaN</td>\n",
       "      <td>mp898-MV411_RNP_MEF2D-H3K27AC-r4</td>\n",
       "      <td>20200924_MP6_MP8230_S118</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>327</th>\n",
       "      <td>mp899</td>\n",
       "      <td>MV411_RNP_MEF2C_MEF2D</td>\n",
       "      <td>1.0</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>NaN</td>\n",
       "      <td>y</td>\n",
       "      <td>INPUT_MV411</td>\n",
       "      <td>NaN</td>\n",
       "      <td>mp899-MV411_RNP_MEF2C_MEF2D-H3K27AC-r1</td>\n",
       "      <td>20200924_MP7_MP8230_S119</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>328</th>\n",
       "      <td>mp900</td>\n",
       "      <td>MV411_RNP_MEF2C_MEF2D</td>\n",
       "      <td>2.0</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>NaN</td>\n",
       "      <td>y</td>\n",
       "      <td>INPUT_MV411</td>\n",
       "      <td>NaN</td>\n",
       "      <td>mp900-MV411_RNP_MEF2C_MEF2D-H3K27AC-r2</td>\n",
       "      <td>20200924_MP8_MP8230_S120</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>329</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>INPUT_MV411</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>330 rows  30 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "        id              cell line  replicate  protein quality paired_end  \\\n",
       "0    mp100                   U937        1.0    INPUT     NaN          n   \n",
       "1    mp101                  NOMO1        1.0    INPUT     NaN          n   \n",
       "2    mp102                    UT7        1.0    INPUT     NaN          n   \n",
       "3    mp106                  MV411        1.0      MYB       x          n   \n",
       "4    mp109                     M6        1.0    CEBPA       x          n   \n",
       "..     ...                    ...        ...      ...     ...        ...   \n",
       "325  mp897        MV411_RNP_MEF2D        3.0  H3K27AC     NaN          y   \n",
       "326  mp898        MV411_RNP_MEF2D        4.0  H3K27AC     NaN          y   \n",
       "327  mp899  MV411_RNP_MEF2C_MEF2D        1.0  H3K27AC     NaN          y   \n",
       "328  mp900  MV411_RNP_MEF2C_MEF2D        2.0  H3K27AC     NaN          y   \n",
       "329    NaN                    NaN        NaN      NaN     NaN        NaN   \n",
       "\n",
       "    matching input name processed                                    name  \\\n",
       "0                   NaN         Y                     mp100-U937-INPUT-r1   \n",
       "1                   NaN         Y                    mp101-NOMO1-INPUT-r1   \n",
       "2                   NaN         Y                      mp102-UT7-INPUT-r1   \n",
       "3           INPUT_MV411         Y                      mp106-MV411-MYB-r1   \n",
       "4              INPUT_M6         Y                       mp109-M6-CEBPA-r1   \n",
       "..                  ...       ...                                     ...   \n",
       "325         INPUT_MV411       NaN        mp897-MV411_RNP_MEF2D-H3K27AC-r3   \n",
       "326         INPUT_MV411       NaN        mp898-MV411_RNP_MEF2D-H3K27AC-r4   \n",
       "327         INPUT_MV411       NaN  mp899-MV411_RNP_MEF2C_MEF2D-H3K27AC-r1   \n",
       "328         INPUT_MV411       NaN  mp900-MV411_RNP_MEF2C_MEF2D-H3K27AC-r2   \n",
       "329         INPUT_MV411       NaN                                     NaN   \n",
       "\n",
       "                previous name  ... ratio to droso unique mapped reads(droso)  \\\n",
       "0                         NaN  ...            NaN                        NaN   \n",
       "1                         NaN  ...            NaN                        NaN   \n",
       "2                         NaN  ...            NaN                        NaN   \n",
       "3                         NaN  ...            NaN                        NaN   \n",
       "4                         NaN  ...            NaN                        NaN   \n",
       "..                        ...  ...            ...                        ...   \n",
       "325  20200924_MP5_MP8230_S117  ...            NaN                        NaN   \n",
       "326  20200924_MP6_MP8230_S118  ...            NaN                        NaN   \n",
       "327  20200924_MP7_MP8230_S119  ...            NaN                        NaN   \n",
       "328  20200924_MP8_MP8230_S120  ...            NaN                        NaN   \n",
       "329                       NaN  ...            NaN                        NaN   \n",
       "\n",
       "    scaling factor                                           Total QC  \\\n",
       "0              NaN  https://storage.cloud.google.com/amlproject/Ch...   \n",
       "1              NaN                                                NaN   \n",
       "2              NaN                                                NaN   \n",
       "3              NaN                                                NaN   \n",
       "4              NaN                                                NaN   \n",
       "..             ...                                                ...   \n",
       "325            NaN                                                NaN   \n",
       "326            NaN                                                NaN   \n",
       "327            NaN                                                NaN   \n",
       "328            NaN                                                NaN   \n",
       "329            NaN                                                NaN   \n",
       "\n",
       "                                          folderNarrow folderCompensated  \\\n",
       "0    https://console.cloud.google.com/storage/brows...               NaN   \n",
       "1                                                  NaN               NaN   \n",
       "2                                                  NaN               NaN   \n",
       "3                                                  NaN               NaN   \n",
       "4                                                  NaN               NaN   \n",
       "..                                                 ...               ...   \n",
       "325                                                NaN               NaN   \n",
       "326                                                NaN               NaN   \n",
       "327                                                NaN               NaN   \n",
       "328                                                NaN               NaN   \n",
       "329                                                NaN               NaN   \n",
       "\n",
       "                                              folderQC  \\\n",
       "0    https://console.cloud.google.com/storage/brows...   \n",
       "1                                                  NaN   \n",
       "2                                                  NaN   \n",
       "3                                                  NaN   \n",
       "4                                                  NaN   \n",
       "..                                                 ...   \n",
       "325                                                NaN   \n",
       "326                                                NaN   \n",
       "327                                                NaN   \n",
       "328                                                NaN   \n",
       "329                                                NaN   \n",
       "\n",
       "                                           folderBroad  \\\n",
       "0    https://console.cloud.google.com/storage/brows...   \n",
       "1                                                  NaN   \n",
       "2                                                  NaN   \n",
       "3                                                  NaN   \n",
       "4                                                  NaN   \n",
       "..                                                 ...   \n",
       "325                                                NaN   \n",
       "326                                                NaN   \n",
       "327                                                NaN   \n",
       "328                                                NaN   \n",
       "329                                                NaN   \n",
       "\n",
       "                                         folder Bigwig  folder diffPeaks  \n",
       "0    https://console.cloud.google.com/storage/brows...               NaN  \n",
       "1                                                  NaN               NaN  \n",
       "2                                                  NaN               NaN  \n",
       "3                                                  NaN               NaN  \n",
       "4                                                  NaN               NaN  \n",
       "..                                                 ...               ...  \n",
       "325                                                NaN               NaN  \n",
       "326                                                NaN               NaN  \n",
       "327                                                NaN               NaN  \n",
       "328                                                NaN               NaN  \n",
       "329                                                NaN               NaN  \n",
       "\n",
       "[330 rows x 30 columns]"
      ]
     },
     "execution_count": 280,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gsheet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 281,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['mp893-MV411_RNP_AAVS1-H3K27AC-r3_R1_001.fastq.gz',\n",
       " 'mp893-MV411_RNP_AAVS1-H3K27AC-r3_R2_001.fastq.gz',\n",
       " 'mp894-MV411_RNP_AAVS1-H3K27AC-r4_R1_001.fastq.gz',\n",
       " 'mp894-MV411_RNP_AAVS1-H3K27AC-r4_R2_001.fastq.gz',\n",
       " 'mp895-MV411_RNP_MEF2C-H3K27AC-r1_R1_001.fastq.gz',\n",
       " 'mp895-MV411_RNP_MEF2C-H3K27AC-r1_R2_001.fastq.gz',\n",
       " 'mp896-MV411_RNP_MEF2C-H3K27AC-r2_R1_001.fastq.gz',\n",
       " 'mp896-MV411_RNP_MEF2C-H3K27AC-r2_R2_001.fastq.gz',\n",
       " 'mp897-MV411_RNP_MEF2D-H3K27AC-r3_R1_001.fastq.gz',\n",
       " 'mp897-MV411_RNP_MEF2D-H3K27AC-r3_R2_001.fastq.gz',\n",
       " 'mp898-MV411_RNP_MEF2D-H3K27AC-r4_R1_001.fastq.gz',\n",
       " 'mp898-MV411_RNP_MEF2D-H3K27AC-r4_R2_001.fastq.gz',\n",
       " 'mp899-MV411_RNP_MEF2C_MEF2D-H3K27AC-r1_R1_001.fastq.gz',\n",
       " 'mp899-MV411_RNP_MEF2C_MEF2D-H3K27AC-r1_R2_001.fastq.gz',\n",
       " 'mp900-MV411_RNP_MEF2C_MEF2D-H3K27AC-r2_R1_001.fastq.gz',\n",
       " 'mp900-MV411_RNP_MEF2C_MEF2D-H3K27AC-r2_R2_001.fastq.gz']"
      ]
     },
     "execution_count": 281,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "a = ! ls ../../data/$project/fastqs\n",
    "a"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 282,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = {\n",
    "\"fastq_1\": [],\n",
    "\"fastq_2\": [],\n",
    "\"antibody\": [],\n",
    "\"group\": [],\n",
    "\"replicate\": [],\n",
    "\"control\": []\n",
    "}\n",
    "for val in h.grouped(a,2):\n",
    "    row = gsheet[gsheet.id==val[0].split('/')[-1].split('-')[0]]\n",
    "    df['group'].append(\"MV4\"+row['name'].values[0].split('-r')[0].split('-MV4')[1])\n",
    "    df['replicate'].append(row['name'].values[0].split('-r')[1])\n",
    "    df['fastq_1'].append(project+\"/fastqs/\"+val[0])\n",
    "    df['fastq_2'].append(project+\"/fastqs/\"+val[1])\n",
    "    df['antibody'].append(row['protein'].values[0])\n",
    "    df['control'].append(\"INPUT\")\n",
    "df['group'].append('INPUT')\n",
    "df['replicate'].append(1)\n",
    "df['fastq_1'].append('ref/mp845-MV411-INPUT-r2_R1.fastq.gz')\n",
    "df['fastq_2'].append('ref/mp845-MV411-INPUT-r2_R2.fastq.gz')\n",
    "df['antibody'].append(\"\")\n",
    "df['control'].append(\"\")\n",
    "df = pd.DataFrame(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 283,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>fastq_1</th>\n",
       "      <th>fastq_2</th>\n",
       "      <th>antibody</th>\n",
       "      <th>group</th>\n",
       "      <th>replicate</th>\n",
       "      <th>control</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>chipseq_MEFF2CMEF2D_degraded/fastqs/mp893-MV41...</td>\n",
       "      <td>chipseq_MEFF2CMEF2D_degraded/fastqs/mp893-MV41...</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>MV411_RNP_AAVS1-H3K27AC</td>\n",
       "      <td>3</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>chipseq_MEFF2CMEF2D_degraded/fastqs/mp894-MV41...</td>\n",
       "      <td>chipseq_MEFF2CMEF2D_degraded/fastqs/mp894-MV41...</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>MV411_RNP_AAVS1-H3K27AC</td>\n",
       "      <td>4</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>chipseq_MEFF2CMEF2D_degraded/fastqs/mp895-MV41...</td>\n",
       "      <td>chipseq_MEFF2CMEF2D_degraded/fastqs/mp895-MV41...</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>MV411_RNP_MEF2C-H3K27AC</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>chipseq_MEFF2CMEF2D_degraded/fastqs/mp896-MV41...</td>\n",
       "      <td>chipseq_MEFF2CMEF2D_degraded/fastqs/mp896-MV41...</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>MV411_RNP_MEF2C-H3K27AC</td>\n",
       "      <td>2</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>chipseq_MEFF2CMEF2D_degraded/fastqs/mp897-MV41...</td>\n",
       "      <td>chipseq_MEFF2CMEF2D_degraded/fastqs/mp897-MV41...</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>MV411_RNP_MEF2D-H3K27AC</td>\n",
       "      <td>3</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>chipseq_MEFF2CMEF2D_degraded/fastqs/mp898-MV41...</td>\n",
       "      <td>chipseq_MEFF2CMEF2D_degraded/fastqs/mp898-MV41...</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>MV411_RNP_MEF2D-H3K27AC</td>\n",
       "      <td>4</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>chipseq_MEFF2CMEF2D_degraded/fastqs/mp899-MV41...</td>\n",
       "      <td>chipseq_MEFF2CMEF2D_degraded/fastqs/mp899-MV41...</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>MV411_RNP_MEF2C_MEF2D-H3K27AC</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>chipseq_MEFF2CMEF2D_degraded/fastqs/mp900-MV41...</td>\n",
       "      <td>chipseq_MEFF2CMEF2D_degraded/fastqs/mp900-MV41...</td>\n",
       "      <td>H3K27AC</td>\n",
       "      <td>MV411_RNP_MEF2C_MEF2D-H3K27AC</td>\n",
       "      <td>2</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>ref/mp845-MV411-INPUT-r2_R1.fastq.gz</td>\n",
       "      <td>ref/mp845-MV411-INPUT-r2_R2.fastq.gz</td>\n",
       "      <td></td>\n",
       "      <td>INPUT</td>\n",
       "      <td>1</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                             fastq_1  \\\n",
       "0  chipseq_MEFF2CMEF2D_degraded/fastqs/mp893-MV41...   \n",
       "1  chipseq_MEFF2CMEF2D_degraded/fastqs/mp894-MV41...   \n",
       "2  chipseq_MEFF2CMEF2D_degraded/fastqs/mp895-MV41...   \n",
       "3  chipseq_MEFF2CMEF2D_degraded/fastqs/mp896-MV41...   \n",
       "4  chipseq_MEFF2CMEF2D_degraded/fastqs/mp897-MV41...   \n",
       "5  chipseq_MEFF2CMEF2D_degraded/fastqs/mp898-MV41...   \n",
       "6  chipseq_MEFF2CMEF2D_degraded/fastqs/mp899-MV41...   \n",
       "7  chipseq_MEFF2CMEF2D_degraded/fastqs/mp900-MV41...   \n",
       "8               ref/mp845-MV411-INPUT-r2_R1.fastq.gz   \n",
       "\n",
       "                                             fastq_2 antibody  \\\n",
       "0  chipseq_MEFF2CMEF2D_degraded/fastqs/mp893-MV41...  H3K27AC   \n",
       "1  chipseq_MEFF2CMEF2D_degraded/fastqs/mp894-MV41...  H3K27AC   \n",
       "2  chipseq_MEFF2CMEF2D_degraded/fastqs/mp895-MV41...  H3K27AC   \n",
       "3  chipseq_MEFF2CMEF2D_degraded/fastqs/mp896-MV41...  H3K27AC   \n",
       "4  chipseq_MEFF2CMEF2D_degraded/fastqs/mp897-MV41...  H3K27AC   \n",
       "5  chipseq_MEFF2CMEF2D_degraded/fastqs/mp898-MV41...  H3K27AC   \n",
       "6  chipseq_MEFF2CMEF2D_degraded/fastqs/mp899-MV41...  H3K27AC   \n",
       "7  chipseq_MEFF2CMEF2D_degraded/fastqs/mp900-MV41...  H3K27AC   \n",
       "8               ref/mp845-MV411-INPUT-r2_R2.fastq.gz            \n",
       "\n",
       "                           group replicate control  \n",
       "0        MV411_RNP_AAVS1-H3K27AC         3   INPUT  \n",
       "1        MV411_RNP_AAVS1-H3K27AC         4   INPUT  \n",
       "2        MV411_RNP_MEF2C-H3K27AC         1   INPUT  \n",
       "3        MV411_RNP_MEF2C-H3K27AC         2   INPUT  \n",
       "4        MV411_RNP_MEF2D-H3K27AC         3   INPUT  \n",
       "5        MV411_RNP_MEF2D-H3K27AC         4   INPUT  \n",
       "6  MV411_RNP_MEF2C_MEF2D-H3K27AC         1   INPUT  \n",
       "7  MV411_RNP_MEF2C_MEF2D-H3K27AC         2   INPUT  \n",
       "8                          INPUT         1          "
      ]
     },
     "execution_count": 283,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 284,
   "metadata": {},
   "outputs": [],
   "source": [
    "df[df.columns[[3,4,0,1,2,5]]].to_csv('../nextflow/'+project+'_design.csv',index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "TIMESTAMP          \tDURATION      \tRUN NAME              \tSTATUS\tREVISION ID\tSESSION ID                          \tCOMMAND                                                                                                                                                                                                                                                          \r\n",
      "2020-02-10 22:39:18\t8m 25s        \tstupefied_crick       \tERR   \t21be314954 \t76ea5df0-153c-4e71-a59d-52c6112fda84\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --narrow_peak --input design.csv --genome GRCh38 --skip_preseq --max_cpus 24 -profile docker -w work                                                                      \r\n",
      "2020-02-10 22:40:37\t1m 6s         \tridiculous_hilbert    \tERR   \t21be314954 \t75004903-035d-4504-ab80-cab74b5acac4\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --narrow_peak --input design.csv --genome GRCh38 --skip_preseq --max_cpus 24 -profile docker -w work                                                                      \r\n",
      "2020-02-10 22:41:52\t1m 18s        \tintergalactic_mercator\tERR   \t21be314954 \tbf13ed87-be57-46c0-8a99-3939bba1a794\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --narrow_peak --input design.csv --genome GRCh38 --skip_preseq --max_cpus 24 -profile docker -w work                                                                      \r\n",
      "2020-02-10 22:52:31\t1d 15h 39m 50s\tgrave_baekeland       \tOK    \t21be314954 \tb554f82a-f7d1-42fd-8421-543518d2aa84\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --narrow_peak --input design.csv --genome GRCh38 --skip_preseq --max_cpus 24 -profile docker -w work                                                                      \r\n",
      "2020-03-10 17:24:20\t4h 47m 8s     \tfestering_lovelace    \tERR   \t21be314954 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --narrow_peak --input design_rep4.csv --genome GRCh38 --skip_preseq --max_cpus 24 -profile docker -w work                                                                 \r\n",
      "2020-03-11 15:16:55\t16s           \tcompassionate_raman   \tERR   \t21be314954 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --narrow_peak --input design_rep4.csv --genome GRCh38 --skip_preseq --max_cpus 24 -profile docker -w work -resume festering_lovelace                                      \r\n",
      "2020-03-11 15:17:44\t19h 57m 30s   \tsilly_thompson        \tOK    \t21be314954 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --narrow_peak --input design_rep4.csv --genome GRCh38 --skip_preseq --max_cpus 24 -profile docker -w work -resume festering_lovelace                                      \r\n",
      "2020-05-27 15:08:20\t-             \tridiculous_bardeen    \t-     \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh37 -resume                                                                                                                                                                        \r\n",
      "2020-05-27 15:09:40\t-             \tmagical_mirzakhani    \t-     \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh37 -resume                                                                                                                                                                        \r\n",
      "2020-05-27 16:54:22\t6m 55s        \tnice_tesla            \tERR   \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh37 -resume                                                                                                                                                                        \r\n",
      "2020-05-27 17:01:54\t1m 32s        \tagitated_bose         \tERR   \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh37 -resume                                                                                                                                                                        \r\n",
      "2020-05-27 17:03:49\t-             \tconfident_pauling     \t-     \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh38 -resume --genome --max_cpus 16                                                                                                                                                 \r\n",
      "2020-05-27 17:04:17\t-             \thopeful_watson        \t-     \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh38 -resume --max_cpus 16                                                                                                                                                          \r\n",
      "2020-05-27 17:42:01\t-             \telegant_liskov        \t-     \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh38 -resume --max_cpus 12                                                                                                                                                          \r\n",
      "2020-05-27 18:23:13\t3h 1m 56s     \tcurious_engelbart     \tERR   \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh38 -resume --max_cpus 12                                                                                                                                                          \r\n",
      "2020-05-27 21:25:55\t4m 43s        \tmad_bell              \tERR   \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh38 -resume --max_cpus 6                                                                                                                                                           \r\n",
      "2020-05-28 13:08:34\t11h 21m 20s   \tspecial_heyrovsky     \tOK    \tfa1e3f8993 \t384099a2-df0f-4a41-b5b0-667436ee09de\tnextflow run nf-core/atacseq -profile docker --input MV4_ATAC.csv --genome GRCh38 -resume --max_cpus 6                                                                                                                                                           \r\n",
      "2020-08-04 17:36:59\t1m 59s        \tbackstabbing_hilbert  \tERR   \t21be314954 \t360ae07b-fb63-4b50-b834-7ba2af75b988\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --broad_peak --input ../AMLproject/nextflow/additional_degraded_v1_design.csv --genome GRCh38 --skip_preseq --max_cpus 8 -profile docker -w work                          \r\n",
      "2020-08-04 17:40:57\t1m 14s        \tfurious_liskov        \tERR   \t21be314954 \tb16ce591-36eb-4bc2-901b-840bbe4c73bd\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --broad_peak --input ../AMLproject/nextflow/additional_degraded_v1_design.csv --genome GRCh38 --skip_preseq --max_cpus 8 -profile docker -w work                          \r\n",
      "2020-08-04 17:42:40\t17.5s         \tridiculous_kilby      \tERR   \t21be314954 \tc3ff2f08-8186-4048-b643-1d615eb7dcc1\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --broad_peak --input ../AMLproject/nextflow/additional_degraded_v1_design.csv --genome GRCh38 --skip_preseq --max_cpus 6 -profile docker -w work                          \r\n",
      "2020-08-04 18:23:46\t6.1s          \tlethal_celsius        \tERR   \t21be314954 \t58d0cab0-e38f-4cd7-97a9-d897c5d2b66d\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --broad_peak --input ../AMLproject/nextflow/additional_degraded_v1_design.csv --genome GRCh38 --skip_preseq --max_cpus 6 -profile docker -w work                          \r\n",
      "2020-08-04 18:23:57\t4h 44m 38s    \tinfallible_woese      \tERR   \t21be314954 \t9d191f62-68e0-4e0b-8c13-2e7c1b32ca2f\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --broad_peak --input ../AMLproject/nextflow/additional_degraded_v1_design.csv --genome GRCh38 --skip_preseq --max_cpus 16 -profile docker -w work                         \r\n",
      "2020-08-04 23:21:36\t16h 4m 40s    \texotic_bartik         \tERR   \t21be314954 \t9d191f62-68e0-4e0b-8c13-2e7c1b32ca2f\tnextflow run nf-core/chipseq --paired_end --seq_center DFCI --email jkobject@gmail.com --broad_peak --input ../AMLproject/nextflow/additional_degraded_v1_design.csv --genome GRCh38 --skip_preseq --max_cpus 16 -profile docker -w work -resume infallible_woese\r\n"
     ]
    }
   ],
   "source": [
    "! cd ../../data/ && sudo ../nextflow log ## to get access to the previous runs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "N E X T F L O W  ~  version 19.10.0\n",
      "Launching `nf-core/chipseq` [wise_legentil] - revision: 21be314954 [master]\n",
      "NOTE: Your local project version looks outdated - a different revision is available in the remote repository [0f487ed76d]\n",
      "-\u001b[2m--------------------------------------------------\u001b[0m-\n",
      "                                        \u001b[0;32m,--.\u001b[0;30m/\u001b[0;32m,-.\u001b[0m\n",
      "\u001b[0;34m        ___     __   __   __   ___     \u001b[0;32m/,-._.--~'\u001b[0m\n",
      "\u001b[0;34m  |\\ | |__  __ /  ` /  \\ |__) |__         \u001b[0;33m}  {\u001b[0m\n",
      "\u001b[0;34m  | \\| |       \\__, \\__/ |  \\ |___     \u001b[0;32m\\`-._,-`-,\u001b[0m\n",
      "                                        \u001b[0;32m`._,._,'\u001b[0m\n",
      "\u001b[0;35m  nf-core/chipseq v1.1.0\u001b[0m\n",
      "-\u001b[2m--------------------------------------------------\u001b[0m-\n",
      "Run Name            : wise_legentil\n",
      "Data Type           : Paired-End\n",
      "Design File         : ../AMLproject/nextflow/additional_degraded_v1_design.csv\n",
      "Genome              : GRCh38\n",
      "Fasta File          : s3://ngi-igenomes/igenomes//Homo_sapiens/NCBI/GRCh38/Sequence/WholeGenomeFasta/genome.fa\n",
      "GTF File            : s3://ngi-igenomes/igenomes//Homo_sapiens/NCBI/GRCh38/Annotation/Genes/genes.gtf\n",
      "Gene BED File       : s3://ngi-igenomes/igenomes//Homo_sapiens/NCBI/GRCh38/Annotation/Genes/genes.bed\n",
      "BWA Index           : s3://ngi-igenomes/igenomes//Homo_sapiens/NCBI/GRCh38/Sequence/BWAIndex/genome.fa\n",
      "Blacklist BED       : /root/.nextflow/assets/nf-core/chipseq/assets/blacklists/hg38-blacklist.bed\n",
      "MACS2 Genome Size   : 2.7e9\n",
      "Min Consensus Reps  : 1\n",
      "MACS2 Narrow Peaks  : No\n",
      "MACS2 Broad Cutoff  : 0.1\n",
      "Trim R1             : 0 bp\n",
      "Trim R2             : 0 bp\n",
      "Trim 3' R1          : 0 bp\n",
      "Trim 3' R2          : 0 bp\n",
      "NextSeq Trim        : 0 bp\n",
      "Sequencing Center   : DFCI\n",
      "Fingerprint Bins    : 500000\n",
      "Save Genome Index   : No\n",
      "Skip Preseq         : Yes\n",
      "Max Resources       : 128 GB memory, 16 cpus, 10d time per job\n",
      "Container           : docker - nfcore/chipseq:1.1.0\n",
      "Output Dir          : ./results\n",
      "Launch Dir          : /home/jeremie/data\n",
      "Working Dir         : /home/jeremie/data/work\n",
      "Script Dir          : /root/.nextflow/assets/nf-core/chipseq\n",
      "User                : root\n",
      "Config Profile      : docker\n",
      "E-mail Address      : jkobject@gmail.com\n",
      "E-mail on failure   : false\n",
      "MultiQC Max Size    : 25 MB\n",
      "-\u001b[2m--------------------------------------------------\u001b[0m-\n",
      "[-        ] process > CheckDesign      -\u001b[K\n",
      "[-        ] process > MakeTSSBED       -\u001b[K\n",
      "[-        ] process > MakeGenomeFilter -\u001b[K\n",
      "[-        ] process > FastQC           -\u001b[K\n",
      "[-        ] process > TrimGalore       -\u001b[K\n",
      "[-        ] process > BWAMem           -\u001b[K\n",
      "\u001b[7A\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[-        ] process > MakeTSSBED                     -\u001b[K\n",
      "[-        ] process > MakeGenomeFilter               -\u001b[K\n",
      "[-        ] process > FastQC                         -\u001b[K\n",
      "[-        ] process > TrimGalore                     -\u001b[K\n",
      "[-        ] process > BWAMem                         -\u001b[K\n",
      "[-        ] process > SortBAM                        -\u001b[K\n",
      "[-        ] process > MergeBAM                       -\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "\u001b[20A\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[-        ] process > MakeGenomeFilter               -\u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[ee/468381] process > TrimGalore (MV411_RNP_RUNX1... [100%] 7 of 7, cached: 7\u001b[K\n",
      "[-        ] process > BWAMem                         -\u001b[K\n",
      "[-        ] process > SortBAM                        -\u001b[K\n",
      "[-        ] process > MergeBAM                       -\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[-        ] process > get_software_versions          -\u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "\u001b[26A\n",
      "executor >  local (1)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[-        ] process > MakeGenomeFilter               -\u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[-        ] process > BWAMem                         -\u001b[K\n",
      "[-        ] process > SortBAM                        -\u001b[K\n",
      "[-        ] process > MergeBAM                       -\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (1)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[-        ] process > MakeGenomeFilter               -\u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[8f/3989b4] process > BWAMem (MV411_RNP_AAVS1-H3K... [100%] 1 of 1, cached: 1\u001b[K\n",
      "[-        ] process > SortBAM                        -\u001b[K\n",
      "[-        ] process > MergeBAM                       -\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (1)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[-        ] process > MergeBAM                       -\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (3)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/8c0a93] process > MergeBAM (MV411_RNP_RUNX2-H... [ 85%] 11 of 13, cached: 11\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (3)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/8c0a93] process > MergeBAM (MV411_RNP_RUNX2-H... [ 85%] 11 of 13, cached: 11\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (4)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/8c0a93] process > MergeBAM (MV411_RNP_RUNX2-H... [ 86%] 12 of 14, cached: 11\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (5)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[c6/4b33eb] process > MergeBAM (MV411_RNP_MEF2D-H... [ 87%] 13 of 15, cached: 11\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (6)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[fc/cf6140] process > MergeBAM (MV411_RNP_SPI1-H3... [ 88%] 14 of 16, cached: 11\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (7)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[c6/4b33eb] process > MergeBAM (MV411_RNP_MEF2D-H... [ 88%] 15 of 17, cached: 11\u001b[K\n",
      "[-        ] process > MergeBAMFilter                 -\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (8)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[12/a07a72] process > MergeBAM (MV411_RNP_MYB-H3K... [ 94%] 16 of 17, cached: 11\u001b[K\n",
      "[84/574208] process > MergeBAMFilter (MV411_RNP_S... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (9)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[66/a24586] process > MergeBAMFilter (MV411_RNP_R... [  0%] 0 of 2\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (10)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/83ab58] process > MergeBAMFilter (MV411_RNP_M... [ 33%] 1 of 3\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (11)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[84/574208] process > MergeBAMFilter (MV411_RNP_S... [ 50%] 2 of 4\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (12)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/83ab58] process > MergeBAMFilter (MV411_RNP_M... [ 60%] 3 of 5\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (13)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[60/4767d1] process > MergeBAMFilter (MV411_RNP_R... [ 67%] 4 of 6\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (14)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[62/3caa87] process > MergeBAMFilter (MV411_RNP_I... [ 57%] 4 of 7\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (14)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[60/4767d1] process > MergeBAMFilter (MV411_RNP_R... [ 71%] 5 of 7\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (15)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[e1/d4ac46] process > MergeBAMFilter (MV411_RNP_A... [ 63%] 5 of 8\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (15)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[9f/a848c2] process > MergeBAMFilter (INPUT_R1)      [ 75%] 6 of 8\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (16)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[ff/e09463] process > MergeBAMFilter (MV411_RNP_M... [ 78%] 7 of 9\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (17)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[77/a464ff] process > MergeBAMFilter (MV411_RNP_I... [ 80%] 8 of 10\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (18)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[ff/e09463] process > MergeBAMFilter (MV411_RNP_M... [ 82%] 9 of 11\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (18)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[77/a464ff] process > MergeBAMFilter (MV411_RNP_I... [ 91%] 10 of 11\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (19)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[e2/12ed6b] process > MergeBAMFilter (MV411_RNP_R... [ 83%] 10 of 12\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (20)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[07/b34e79] process > MergeBAMFilter (MV411_RNP_R... [ 85%] 11 of 13\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (21)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[9a/a02fdf] process > MergeBAMFilter (MV411_RNP_S... [ 86%] 12 of 14\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (22)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[65/4e0731] process > MergeBAMFilter (MV411_RNP_M... [ 87%] 13 of 15\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (23)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[e0/a67acd] process > MergeBAMFilter (MV411_RNP_M... [ 88%] 14 of 16\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (24)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [ 88%] 15 of 17\u001b[K\n",
      "[-        ] process > MergeBAMRemoveOrphan           -\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (25)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[e0/a67acd] process > MergeBAMFilter (MV411_RNP_M... [ 94%] 16 of 17\u001b[K\n",
      "[21/51df00] process > MergeBAMRemoveOrphan (MV411... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (26)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[e0/a67acd] process > MergeBAMFilter (MV411_RNP_M... [ 94%] 16 of 17\u001b[K\n",
      "[21/51df00] process > MergeBAMRemoveOrphan (MV411... [ 50%] 1 of 2\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (27)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[30/1ef981] process > MergeBAMRemoveOrphan (MV411... [ 33%] 1 of 3\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (28)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[57/199f77] process > MergeBAMRemoveOrphan (MV411... [ 50%] 2 of 4\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (29)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[30/1ef981] process > MergeBAMRemoveOrphan (MV411... [ 60%] 3 of 5\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (30)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[bf/2620e0] process > MergeBAMRemoveOrphan (MV411... [ 67%] 4 of 6\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (31)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[c7/9d9b0a] process > MergeBAMRemoveOrphan (MV411... [ 71%] 5 of 7\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (32)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[70/ef513d] process > MergeBAMRemoveOrphan (INPUT... [ 75%] 6 of 8\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (33)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[63/892aee] process > MergeBAMRemoveOrphan (MV411... [ 78%] 7 of 9\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (34)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[b3/9db05d] process > MergeBAMRemoveOrphan (MV411... [ 80%] 8 of 10\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (35)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[9f/c2beee] process > MergeBAMRemoveOrphan (MV411... [ 82%] 9 of 11\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (36)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[22/110bf7] process > MergeBAMRemoveOrphan (MV411... [ 83%] 10 of 12\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (37)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[1b/49d44f] process > MergeBAMRemoveOrphan (MV411... [ 85%] 11 of 13\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (38)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[ba/00ce48] process > MergeBAMRemoveOrphan (MV411... [ 79%] 11 of 14\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (38)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[08/ccd392] process > MergeBAMRemoveOrphan (MV411... [ 86%] 12 of 14\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (39)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[9a/fa4609] process > MergeBAMRemoveOrphan (MV411... [ 87%] 13 of 15\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (40)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[04/2a4108] process > MergeBAMRemoveOrphan (MV411... [ 81%] 13 of 16\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (40)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[ba/00ce48] process > MergeBAMRemoveOrphan (MV411... [ 88%] 14 of 16\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[-        ] process > PhantomPeakQualTools           -\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (41)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[c0/52c2db] process > MergeBAMRemoveOrphan (MV411... [ 94%] 15 of 16\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[-        ] process > BigWig                         -\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[68/d5d1f4] process > PhantomPeakQualTools (MV411... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (42)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[04/2a4108] process > MergeBAMRemoveOrphan (MV411... [100%] 16 of 16\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[-        ] process > CollectMultipleMetrics         -\u001b[K\n",
      "[41/1dc1b3] process > BigWig (MV411_RNP_RUNX1_RUN... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[68/d5d1f4] process > PhantomPeakQualTools (MV411... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (43)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[04/2a4108] process > MergeBAMRemoveOrphan (MV411... [100%] 16 of 16\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[18/e3205c] process > CollectMultipleMetrics (MV4... [  0%] 0 of 1\u001b[K\n",
      "[41/1dc1b3] process > BigWig (MV411_RNP_RUNX1_RUN... [100%] 1 of 1\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[68/d5d1f4] process > PhantomPeakQualTools (MV411... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (44)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [ 94%] 16 of 17\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[18/e3205c] process > CollectMultipleMetrics (MV4... [  0%] 0 of 1\u001b[K\n",
      "[41/1dc1b3] process > BigWig (MV411_RNP_RUNX1_RUN... [100%] 1 of 1\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[68/d5d1f4] process > PhantomPeakQualTools (MV411... [100%] 1 of 1\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (45)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [ 94%] 16 of 17\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[18/e3205c] process > CollectMultipleMetrics (MV4... [100%] 1 of 1\u001b[K\n",
      "[fa/7ca520] process > BigWig (MV411_RNP_SPI1-H3K2... [ 50%] 1 of 2\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[68/d5d1f4] process > PhantomPeakQualTools (MV411... [100%] 1 of 1\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (46)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [ 94%] 16 of 17\u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[18/e3205c] process > CollectMultipleMetrics (MV4... [100%] 1 of 1\u001b[K\n",
      "[fa/7ca520] process > BigWig (MV411_RNP_SPI1-H3K2... [100%] 2 of 2\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[c1/b1a581] process > PhantomPeakQualTools (MV411... [ 50%] 1 of 2\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (47)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[ea/949a3e] process > CollectMultipleMetrics (MV4... [ 50%] 1 of 2\u001b[K\n",
      "[fa/7ca520] process > BigWig (MV411_RNP_SPI1-H3K2... [100%] 2 of 2\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[c1/b1a581] process > PhantomPeakQualTools (MV411... [ 50%] 1 of 2\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (48)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[ea/949a3e] process > CollectMultipleMetrics (MV4... [100%] 2 of 2\u001b[K\n",
      "[55/063141] process > BigWig (MV411_RNP_MEF2D-H3K... [ 67%] 2 of 3\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[c1/b1a581] process > PhantomPeakQualTools (MV411... [ 50%] 1 of 2\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (49)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[ea/949a3e] process > CollectMultipleMetrics (MV4... [100%] 2 of 2\u001b[K\n",
      "[55/063141] process > BigWig (MV411_RNP_MEF2D-H3K... [100%] 3 of 3\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[9d/c498be] process > PhantomPeakQualTools (MV411... [ 33%] 1 of 3\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (50)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[96/83ce50] process > CollectMultipleMetrics (MV4... [ 67%] 2 of 3\u001b[K\n",
      "[55/063141] process > BigWig (MV411_RNP_MEF2D-H3K... [100%] 3 of 3\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[c1/b1a581] process > PhantomPeakQualTools (MV411... [ 67%] 2 of 3\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (51)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[96/83ce50] process > CollectMultipleMetrics (MV4... [100%] 3 of 3\u001b[K\n",
      "[55/063141] process > BigWig (MV411_RNP_MEF2D-H3K... [100%] 3 of 3\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[65/278ab6] process > PhantomPeakQualTools (MV411... [ 50%] 2 of 4\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (52)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[96/83ce50] process > CollectMultipleMetrics (MV4... [100%] 3 of 3\u001b[K\n",
      "[c4/0c2b62] process > BigWig (MV411_RNP_AAVS1-H3K... [ 75%] 3 of 4\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[9d/c498be] process > PhantomPeakQualTools (MV411... [ 75%] 3 of 4\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (53)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[24/913f78] process > CollectMultipleMetrics (MV4... [ 75%] 3 of 4\u001b[K\n",
      "[c4/0c2b62] process > BigWig (MV411_RNP_AAVS1-H3K... [100%] 4 of 4\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[9d/c498be] process > PhantomPeakQualTools (MV411... [ 75%] 3 of 4\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (54)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[24/913f78] process > CollectMultipleMetrics (MV4... [ 75%] 3 of 4\u001b[K\n",
      "[c4/0c2b62] process > BigWig (MV411_RNP_AAVS1-H3K... [100%] 4 of 4\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[65/278ab6] process > PhantomPeakQualTools (MV411... [ 80%] 4 of 5\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (55)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[24/913f78] process > CollectMultipleMetrics (MV4... [100%] 4 of 4\u001b[K\n",
      "[26/8106f4] process > BigWig (MV411_RNP_RUNX1-H3K... [ 80%] 4 of 5\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[65/278ab6] process > PhantomPeakQualTools (MV411... [ 80%] 4 of 5\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (56)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[36/03ab9d] process > CollectMultipleMetrics (MV4... [ 80%] 4 of 5\u001b[K\n",
      "[26/8106f4] process > BigWig (MV411_RNP_RUNX1-H3K... [100%] 5 of 5\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[65/278ab6] process > PhantomPeakQualTools (MV411... [ 80%] 4 of 5\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (57)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[36/03ab9d] process > CollectMultipleMetrics (MV4... [100%] 5 of 5\u001b[K\n",
      "[26/8106f4] process > BigWig (MV411_RNP_RUNX1-H3K... [100%] 5 of 5\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[41/509b9d] process > PhantomPeakQualTools (INPUT... [ 67%] 4 of 6\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (58)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[36/03ab9d] process > CollectMultipleMetrics (MV4... [100%] 5 of 5\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[b6/1c51c9] process > PhantomPeakQualTools (MV411... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[-        ] process > MACSCallPeak                   -\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (59)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[36/03ab9d] process > CollectMultipleMetrics (MV4... [100%] 5 of 5\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[b6/1c51c9] process > PhantomPeakQualTools (MV411... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[df/af3c12] process > MACSCallPeak (MV411_RNP_AAV... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (60)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[36/03ab9d] process > CollectMultipleMetrics (MV4... [100%] 5 of 5\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[b6/1c51c9] process > PhantomPeakQualTools (MV411... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[df/af3c12] process > MACSCallPeak (MV411_RNP_AAV... [ 50%] 1 of 2\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (60)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[36/03ab9d] process > CollectMultipleMetrics (MV4... [100%] 5 of 5\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[b6/1c51c9] process > PhantomPeakQualTools (MV411... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[df/af3c12] process > MACSCallPeak (MV411_RNP_AAV... [ 50%] 1 of 2\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "Staging foreign file: s3://ngi-igenomes/igenomes/Homo_sapiens/NCBI/GRCh38/Annotation/Genes/genes.gtf\u001b[K\n",
      "\u001b[29A\n",
      "executor >  local (61)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[36/03ab9d] process > CollectMultipleMetrics (MV4... [100%] 5 of 5\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[41/509b9d] process > PhantomPeakQualTools (INPUT... [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[ec/449ea1] process > MACSCallPeak (MV411_RNP_RUN... [ 33%] 1 of 3\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[K\n",
      "\u001b[29A\n",
      "executor >  local (62)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[36/03ab9d] process > CollectMultipleMetrics (MV4... [100%] 5 of 5\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[41/509b9d] process > PhantomPeakQualTools (INPUT... [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[ca/eb457d] process > MACSCallPeak (MV411_RNP_MEF... [ 50%] 2 of 4\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (63)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[36/03ab9d] process > CollectMultipleMetrics (MV4... [100%] 5 of 5\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[41/509b9d] process > PhantomPeakQualTools (INPUT... [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[ec/449ea1] process > MACSCallPeak (MV411_RNP_RUN... [ 60%] 3 of 5\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (64)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[6f/0c32f5] process > CollectMultipleMetrics (INP... [ 83%] 5 of 6\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[41/509b9d] process > PhantomPeakQualTools (INPUT... [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[99/4577c5] process > MACSCallPeak (MV411_RNP_RUN... [ 80%] 4 of 5\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (65)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[6f/0c32f5] process > CollectMultipleMetrics (INP... [100%] 6 of 6\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[8c/6222bd] process > PhantomPeakQualTools (MV411... [ 86%] 6 of 7\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[99/4577c5] process > MACSCallPeak (MV411_RNP_RUN... [ 80%] 4 of 5\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (66)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[6f/0c32f5] process > CollectMultipleMetrics (INP... [100%] 6 of 6\u001b[K\n",
      "[2d/982381] process > BigWig (INPUT_R1)              [100%] 6 of 6\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[8c/6222bd] process > PhantomPeakQualTools (MV411... [ 86%] 6 of 7\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[74/d6d890] process > MACSCallPeak (MV411_RNP_SPI... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (67)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[6f/0c32f5] process > CollectMultipleMetrics (INP... [100%] 6 of 6\u001b[K\n",
      "[7f/a5a2c1] process > BigWig (MV411_RNP_AAVS1-H3K... [ 86%] 6 of 7\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[8c/6222bd] process > PhantomPeakQualTools (MV411... [100%] 7 of 7\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[74/d6d890] process > MACSCallPeak (MV411_RNP_SPI... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (68)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[77/85e402] process > CollectMultipleMetrics (MV4... [ 86%] 6 of 7\u001b[K\n",
      "[7f/a5a2c1] process > BigWig (MV411_RNP_AAVS1-H3K... [100%] 7 of 7\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[8c/6222bd] process > PhantomPeakQualTools (MV411... [100%] 7 of 7\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[74/d6d890] process > MACSCallPeak (MV411_RNP_SPI... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (69)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[77/85e402] process > CollectMultipleMetrics (MV4... [100%] 7 of 7\u001b[K\n",
      "[83/f433d6] process > BigWig (MV411_RNP_MYB-H3K27... [ 88%] 7 of 8\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[8c/6222bd] process > PhantomPeakQualTools (MV411... [100%] 7 of 7\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[74/d6d890] process > MACSCallPeak (MV411_RNP_SPI... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (70)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[77/85e402] process > CollectMultipleMetrics (MV4... [100%] 7 of 7\u001b[K\n",
      "[83/f433d6] process > BigWig (MV411_RNP_MYB-H3K27... [100%] 8 of 8\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[83/e5aeda] process > PhantomPeakQualTools (MV411... [ 88%] 7 of 8\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[74/d6d890] process > MACSCallPeak (MV411_RNP_SPI... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (71)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[77/85e402] process > CollectMultipleMetrics (MV4... [100%] 7 of 7\u001b[K\n",
      "[83/f433d6] process > BigWig (MV411_RNP_MYB-H3K27... [100%] 8 of 8\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[83/e5aeda] process > PhantomPeakQualTools (MV411... [ 88%] 7 of 8\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[c5/1a276a] process > MACSCallPeak (MV411_RNP_MYB... [ 71%] 5 of 7\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (71)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[77/85e402] process > CollectMultipleMetrics (MV4... [100%] 7 of 7\u001b[K\n",
      "[83/f433d6] process > BigWig (MV411_RNP_MYB-H3K27... [100%] 8 of 8\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[83/e5aeda] process > PhantomPeakQualTools (MV411... [ 88%] 7 of 8\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[aa/9e4d91] process > MACSCallPeak (MV411_RNP_AAV... [ 86%] 6 of 7\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (72)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[f7/870fa3] process > CollectMultipleMetrics (MV4... [ 88%] 7 of 8\u001b[K\n",
      "[83/f433d6] process > BigWig (MV411_RNP_MYB-H3K27... [100%] 8 of 8\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[83/e5aeda] process > PhantomPeakQualTools (MV411... [100%] 8 of 8\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[aa/9e4d91] process > MACSCallPeak (MV411_RNP_AAV... [ 86%] 6 of 7\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (73)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[f7/870fa3] process > CollectMultipleMetrics (MV4... [100%] 8 of 8\u001b[K\n",
      "[83/f433d6] process > BigWig (MV411_RNP_MYB-H3K27... [100%] 8 of 8\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[bc/873392] process > PhantomPeakQualTools (MV411... [ 89%] 8 of 9\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[aa/9e4d91] process > MACSCallPeak (MV411_RNP_AAV... [ 86%] 6 of 7\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (74)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[f7/870fa3] process > CollectMultipleMetrics (MV4... [100%] 8 of 8\u001b[K\n",
      "[be/8c5392] process > BigWig (MV411_RNP_IRF8-H3K2... [ 89%] 8 of 9\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[bc/873392] process > PhantomPeakQualTools (MV411... [ 89%] 8 of 9\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[c5/1a276a] process > MACSCallPeak (MV411_RNP_MYB... [100%] 7 of 7\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (75)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[f7/870fa3] process > CollectMultipleMetrics (MV4... [100%] 8 of 8\u001b[K\n",
      "[be/8c5392] process > BigWig (MV411_RNP_IRF8-H3K2... [ 89%] 8 of 9\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[bc/873392] process > PhantomPeakQualTools (MV411... [100%] 9 of 9\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[47/040371] process > MACSCallPeak (MV411_RNP_IRF... [ 88%] 7 of 8\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (76)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[eb/85c928] process > CollectMultipleMetrics (MV4... [ 89%] 8 of 9\u001b[K\n",
      "[be/8c5392] process > BigWig (MV411_RNP_IRF8-H3K2... [100%] 9 of 9\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[bc/873392] process > PhantomPeakQualTools (MV411... [100%] 9 of 9\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[47/040371] process > MACSCallPeak (MV411_RNP_IRF... [ 88%] 7 of 8\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (77)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[eb/85c928] process > CollectMultipleMetrics (MV4... [100%] 9 of 9\u001b[K\n",
      "[be/8c5392] process > BigWig (MV411_RNP_IRF8-H3K2... [100%] 9 of 9\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/001577] process > PhantomPeakQualTools (MV411... [ 90%] 9 of 10\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[47/040371] process > MACSCallPeak (MV411_RNP_IRF... [ 88%] 7 of 8\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (78)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[eb/85c928] process > CollectMultipleMetrics (MV4... [100%] 9 of 9\u001b[K\n",
      "[4e/c46492] process > BigWig (MV411_RNP_RUNX1_RUN... [ 90%] 9 of 10\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/001577] process > PhantomPeakQualTools (MV411... [100%] 10 of 10\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[47/040371] process > MACSCallPeak (MV411_RNP_IRF... [ 88%] 7 of 8\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (79)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[eb/85c928] process > CollectMultipleMetrics (MV4... [100%] 9 of 9\u001b[K\n",
      "[4e/c46492] process > BigWig (MV411_RNP_RUNX1_RUN... [100%] 10 of 10\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/001577] process > PhantomPeakQualTools (MV411... [100%] 10 of 10\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[3f/dba7c2] process > MACSCallPeak (MV411_RNP_RUN... [ 78%] 7 of 9\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (80)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[56/e75566] process > CollectMultipleMetrics (MV4... [ 90%] 9 of 10\u001b[K\n",
      "[4e/c46492] process > BigWig (MV411_RNP_RUNX1_RUN... [100%] 10 of 10\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/001577] process > PhantomPeakQualTools (MV411... [100%] 10 of 10\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[47/040371] process > MACSCallPeak (MV411_RNP_IRF... [ 89%] 8 of 9\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (81)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[56/e75566] process > CollectMultipleMetrics (MV4... [100%] 10 of 10\u001b[K\n",
      "[60/3850df] process > BigWig (MV411_RNP_IRF8-H3K2... [ 91%] 10 of 11\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/001577] process > PhantomPeakQualTools (MV411... [100%] 10 of 10\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[47/040371] process > MACSCallPeak (MV411_RNP_IRF... [ 89%] 8 of 9\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (82)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[56/e75566] process > CollectMultipleMetrics (MV4... [100%] 10 of 10\u001b[K\n",
      "[60/3850df] process > BigWig (MV411_RNP_IRF8-H3K2... [100%] 11 of 11\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[1c/07bf5e] process > PhantomPeakQualTools (MV411... [ 91%] 10 of 11\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[47/040371] process > MACSCallPeak (MV411_RNP_IRF... [ 89%] 8 of 9\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (83)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[56/e75566] process > CollectMultipleMetrics (MV4... [100%] 10 of 10\u001b[K\n",
      "[60/3850df] process > BigWig (MV411_RNP_IRF8-H3K2... [100%] 11 of 11\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[1c/07bf5e] process > PhantomPeakQualTools (MV411... [ 91%] 10 of 11\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[3f/dba7c2] process > MACSCallPeak (MV411_RNP_RUN... [ 90%] 9 of 10\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (84)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[af/724a3d] process > CollectMultipleMetrics (MV4... [ 91%] 10 of 11\u001b[K\n",
      "[60/3850df] process > BigWig (MV411_RNP_IRF8-H3K2... [100%] 11 of 11\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[1c/07bf5e] process > PhantomPeakQualTools (MV411... [100%] 11 of 11\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[3f/dba7c2] process > MACSCallPeak (MV411_RNP_RUN... [ 90%] 9 of 10\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (85)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[af/724a3d] process > CollectMultipleMetrics (MV4... [100%] 11 of 11\u001b[K\n",
      "[f7/5fa48c] process > BigWig (MV411_RNP_RUNX2-H3K... [ 92%] 11 of 12\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[1c/07bf5e] process > PhantomPeakQualTools (MV411... [100%] 11 of 11\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[3f/dba7c2] process > MACSCallPeak (MV411_RNP_RUN... [ 90%] 9 of 10\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (86)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[af/724a3d] process > CollectMultipleMetrics (MV4... [100%] 11 of 11\u001b[K\n",
      "[f7/5fa48c] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 12 of 12\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[21/517648] process > PhantomPeakQualTools (MV411... [ 92%] 11 of 12\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[3f/dba7c2] process > MACSCallPeak (MV411_RNP_RUN... [ 90%] 9 of 10\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (87)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[af/724a3d] process > CollectMultipleMetrics (MV4... [100%] 11 of 11\u001b[K\n",
      "[f7/5fa48c] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 12 of 12\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[21/517648] process > PhantomPeakQualTools (MV411... [ 92%] 11 of 12\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[51/147018] process > MACSCallPeak (MV411_RNP_IRF... [ 91%] 10 of 11\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (88)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[be/e4f2a1] process > CollectMultipleMetrics (MV4... [ 92%] 11 of 12\u001b[K\n",
      "[f7/5fa48c] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 12 of 12\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[21/517648] process > PhantomPeakQualTools (MV411... [100%] 12 of 12\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[51/147018] process > MACSCallPeak (MV411_RNP_IRF... [ 91%] 10 of 11\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (89)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[be/e4f2a1] process > CollectMultipleMetrics (MV4... [100%] 12 of 12\u001b[K\n",
      "[eb/3a1856] process > BigWig (MV411_RNP_RUNX1-H3K... [ 92%] 12 of 13\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[21/517648] process > PhantomPeakQualTools (MV411... [100%] 12 of 12\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[51/147018] process > MACSCallPeak (MV411_RNP_IRF... [ 91%] 10 of 11\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (90)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[be/e4f2a1] process > CollectMultipleMetrics (MV4... [100%] 12 of 12\u001b[K\n",
      "[eb/3a1856] process > BigWig (MV411_RNP_RUNX1-H3K... [100%] 13 of 13\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[21/517648] process > PhantomPeakQualTools (MV411... [100%] 12 of 12\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[d8/0be403] process > MACSCallPeak (MV411_RNP_RUN... [ 83%] 10 of 12\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (91)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[be/e4f2a1] process > CollectMultipleMetrics (MV4... [100%] 12 of 12\u001b[K\n",
      "[eb/3a1856] process > BigWig (MV411_RNP_RUNX1-H3K... [100%] 13 of 13\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[ce/6b94f7] process > PhantomPeakQualTools (MV411... [ 92%] 12 of 13\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[99/a486f8] process > MACSCallPeak (MV411_RNP_RUN... [ 92%] 11 of 12\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (92)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[58/9c859d] process > CollectMultipleMetrics (MV4... [ 92%] 12 of 13\u001b[K\n",
      "[eb/3a1856] process > BigWig (MV411_RNP_RUNX1-H3K... [100%] 13 of 13\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[ce/6b94f7] process > PhantomPeakQualTools (MV411... [ 92%] 12 of 13\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[d8/0be403] process > MACSCallPeak (MV411_RNP_RUN... [100%] 12 of 12\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (93)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[58/9c859d] process > CollectMultipleMetrics (MV4... [100%] 13 of 13\u001b[K\n",
      "[eb/3a1856] process > BigWig (MV411_RNP_RUNX1-H3K... [100%] 13 of 13\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[1b/b9c016] process > PhantomPeakQualTools (MV411... [ 86%] 12 of 14\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[d8/0be403] process > MACSCallPeak (MV411_RNP_RUN... [100%] 12 of 12\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (94)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[58/9c859d] process > CollectMultipleMetrics (MV4... [100%] 13 of 13\u001b[K\n",
      "[eb/3a1856] process > BigWig (MV411_RNP_RUNX1-H3K... [100%] 13 of 13\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[ce/6b94f7] process > PhantomPeakQualTools (MV411... [ 93%] 13 of 14\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[46/17f258] process > MACSCallPeak (MV411_RNP_SPI... [ 92%] 12 of 13\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (95)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[58/9c859d] process > CollectMultipleMetrics (MV4... [100%] 13 of 13\u001b[K\n",
      "[e7/594109] process > BigWig (MV411_RNP_SPI1-H3K2... [ 93%] 13 of 14\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[1b/b9c016] process > PhantomPeakQualTools (MV411... [100%] 14 of 14\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[46/17f258] process > MACSCallPeak (MV411_RNP_SPI... [ 92%] 12 of 13\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (96)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[39/7b314d] process > CollectMultipleMetrics (MV4... [ 93%] 13 of 14\u001b[K\n",
      "[e7/594109] process > BigWig (MV411_RNP_SPI1-H3K2... [100%] 14 of 14\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[1b/b9c016] process > PhantomPeakQualTools (MV411... [100%] 14 of 14\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[46/17f258] process > MACSCallPeak (MV411_RNP_SPI... [ 92%] 12 of 13\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (97)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[39/7b314d] process > CollectMultipleMetrics (MV4... [100%] 14 of 14\u001b[K\n",
      "[e7/594109] process > BigWig (MV411_RNP_SPI1-H3K2... [100%] 14 of 14\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[1b/b9c016] process > PhantomPeakQualTools (MV411... [100%] 14 of 14\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[fa/6c4132] process > MACSCallPeak (MV411_RNP_MEF... [ 86%] 12 of 14\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (98)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[39/7b314d] process > CollectMultipleMetrics (MV4... [100%] 14 of 14\u001b[K\n",
      "[e1/c25e9c] process > BigWig (MV411_RNP_MEF2D-H3K... [ 93%] 14 of 15\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[1b/b9c016] process > PhantomPeakQualTools (MV411... [100%] 14 of 14\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[46/17f258] process > MACSCallPeak (MV411_RNP_SPI... [ 93%] 13 of 14\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (99)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[39/7b314d] process > CollectMultipleMetrics (MV4... [100%] 14 of 14\u001b[K\n",
      "[e1/c25e9c] process > BigWig (MV411_RNP_MEF2D-H3K... [100%] 15 of 15\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[d7/e2fcc8] process > PhantomPeakQualTools (MV411... [ 93%] 14 of 15\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[46/17f258] process > MACSCallPeak (MV411_RNP_SPI... [ 93%] 13 of 14\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (100)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[49/670ca2] process > CollectMultipleMetrics (MV4... [ 93%] 14 of 15\u001b[K\n",
      "[e1/c25e9c] process > BigWig (MV411_RNP_MEF2D-H3K... [100%] 15 of 15\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[d7/e2fcc8] process > PhantomPeakQualTools (MV411... [100%] 15 of 15\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[46/17f258] process > MACSCallPeak (MV411_RNP_SPI... [ 93%] 13 of 14\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (101)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[49/670ca2] process > CollectMultipleMetrics (MV4... [ 93%] 14 of 15\u001b[K\n",
      "[e1/c25e9c] process > BigWig (MV411_RNP_MEF2D-H3K... [100%] 15 of 15\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/292df8] process > PhantomPeakQualTools (MV411... [ 94%] 15 of 16\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[fa/6c4132] process > MACSCallPeak (MV411_RNP_MEF... [100%] 14 of 14\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (102)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[49/670ca2] process > CollectMultipleMetrics (MV4... [100%] 15 of 15\u001b[K\n",
      "[9c/f2630b] process > BigWig (MV411_RNP_MYB-H3K27... [ 94%] 15 of 16\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/292df8] process > PhantomPeakQualTools (MV411... [ 94%] 15 of 16\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[fa/6c4132] process > MACSCallPeak (MV411_RNP_MEF... [100%] 14 of 14\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (103)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[49/670ca2] process > CollectMultipleMetrics (MV4... [100%] 15 of 15\u001b[K\n",
      "[9c/f2630b] process > BigWig (MV411_RNP_MYB-H3K27... [100%] 16 of 16\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/292df8] process > PhantomPeakQualTools (MV411... [ 94%] 15 of 16\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 93%] 14 of 15\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (104)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[85/5ed89a] process > CollectMultipleMetrics (MV4... [ 94%] 15 of 16\u001b[K\n",
      "[9c/f2630b] process > BigWig (MV411_RNP_MYB-H3K27... [100%] 16 of 16\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/292df8] process > PhantomPeakQualTools (MV411... [100%] 16 of 16\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 93%] 14 of 15\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (105)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[85/5ed89a] process > CollectMultipleMetrics (MV4... [100%] 16 of 16\u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [ 94%] 16 of 17\u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[52/292df8] process > PhantomPeakQualTools (MV411... [100%] 16 of 16\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 93%] 14 of 15\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (106)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[85/5ed89a] process > CollectMultipleMetrics (MV4... [100%] 16 of 16\u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [ 94%] 16 of 17\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 93%] 14 of 15\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (107)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[85/5ed89a] process > CollectMultipleMetrics (MV4... [100%] 16 of 16\u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [ 94%] 16 of 17\u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 94%] 15 of 16\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (108)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [ 94%] 16 of 17\u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 94%] 15 of 16\u001b[K\n",
      "[-        ] process > AnnotatePeaks                  -\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (109)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 94%] 15 of 16\u001b[K\n",
      "[92/f4e285] process > AnnotatePeaks (MV411_RNP_AA... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (110)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 94%] 15 of 16\u001b[K\n",
      "[3d/7fd60b] process > AnnotatePeaks (MV411_RNP_ME... [ 50%] 1 of 2\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (111)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 94%] 15 of 16\u001b[K\n",
      "[aa/43437e] process > AnnotatePeaks (MV411_RNP_RU... [ 67%] 2 of 3\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (112)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 94%] 15 of 16\u001b[K\n",
      "[a0/3b8b5a] process > AnnotatePeaks (MV411_RNP_RU... [ 75%] 3 of 4\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (113)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 94%] 15 of 16\u001b[K\n",
      "[d9/83574c] process > AnnotatePeaks (MV411_RNP_SP... [ 80%] 4 of 5\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (114)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[5c/97904b] process > MACSCallPeak (MV411_RNP_MYB... [ 94%] 15 of 16\u001b[K\n",
      "[d9/83574c] process > AnnotatePeaks (MV411_RNP_SP... [ 83%] 5 of 6\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (115)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[d5/3a754a] process > AnnotatePeaks (MV411_RNP_MY... [ 71%] 5 of 7\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[-        ] process > ConsensusPeakSet               -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (117)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[46/342adc] process > AnnotatePeaks (MV411_RNP_IR... [ 75%] 6 of 8\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (118)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[48/756ae7] process > AnnotatePeaks (MV411_RNP_RU... [ 78%] 7 of 9\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (119)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[46/342adc] process > AnnotatePeaks (MV411_RNP_IR... [ 80%] 8 of 10\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (120)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[54/d0a95c] process > AnnotatePeaks (MV411_RNP_RU... [ 82%] 9 of 11\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (121)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[08/d35aca] process > AnnotatePeaks (MV411_RNP_RU... [ 83%] 10 of 12\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (122)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[c2/ac16d8] process > AnnotatePeaks (MV411_RNP_SP... [ 85%] 11 of 13\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (123)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[82/d0e0fc] process > AnnotatePeaks (MV411_RNP_ME... [ 86%] 12 of 14\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (124)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[0e/c24b0a] process > AnnotatePeaks (MV411_RNP_MY... [ 87%] 13 of 15\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (125)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [ 88%] 14 of 16\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (125)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotFingerprint                -\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[0e/c24b0a] process > AnnotatePeaks (MV411_RNP_MY... [ 94%] 15 of 16\u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (126)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[bc/e561cf] process > PlotFingerprint (MV411_RNP_... [  0%] 0 of 1\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (127)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[bc/e561cf] process > PlotFingerprint (MV411_RNP_... [ 50%] 1 of 2\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (128)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[3f/9aec63] process > PlotFingerprint (MV411_RNP_... [ 67%] 2 of 3\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (129)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[8c/531d81] process > PlotFingerprint (MV411_RNP_... [ 75%] 3 of 4\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (129)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[8c/531d81] process > PlotFingerprint (MV411_RNP_... [ 75%] 3 of 4\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (130)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[20/afbec0] process > PlotFingerprint (MV411_RNP_... [ 80%] 4 of 5\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (131)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[20/afbec0] process > PlotFingerprint (MV411_RNP_... [ 83%] 5 of 6\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (132)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[5e/ac3cd1] process > PlotFingerprint (MV411_RNP_... [ 86%] 6 of 7\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (133)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[8f/17b4a2] process > PlotFingerprint (MV411_RNP_... [ 88%] 7 of 8\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (134)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[48/6bdc90] process > PlotFingerprint (MV411_RNP_... [ 89%] 8 of 9\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (135)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[48/6bdc90] process > PlotFingerprint (MV411_RNP_... [ 90%] 9 of 10\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (136)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[2b/144a84] process > PlotFingerprint (MV411_RNP_... [ 91%] 10 of 11\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (137)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[5c/b9836e] process > PlotFingerprint (MV411_RNP_... [ 92%] 11 of 12\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (138)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[72/264c6a] process > PlotFingerprint (MV411_RNP_... [ 92%] 12 of 13\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (139)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[8e/ba2fa1] process > PlotFingerprint (MV411_RNP_... [ 93%] 13 of 14\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (140)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > PlotProfile                    -\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[11/798ccb] process > PlotFingerprint (MV411_RNP_... [ 93%] 14 of 15\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (141)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[24/32de0c] process > PlotProfile (MV411_RNP_RUNX... [  0%] 0 of 1\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[11/798ccb] process > PlotFingerprint (MV411_RNP_... [100%] 15 of 15\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (142)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[24/32de0c] process > PlotProfile (MV411_RNP_RUNX... [ 50%] 1 of 2\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[11/798ccb] process > PlotFingerprint (MV411_RNP_... [100%] 15 of 15\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (143)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[e1/6ab2d8] process > PlotProfile (MV411_RNP_SPI1... [100%] 2 of 2\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [ 94%] 15 of 16\u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (144)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[c5/fb716e] process > PlotProfile (MV411_RNP_MEF2... [ 67%] 2 of 3\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (145)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[da/b297d7] process > PlotProfile (MV411_RNP_AAVS... [ 75%] 3 of 4\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (146)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[da/b297d7] process > PlotProfile (MV411_RNP_AAVS... [ 80%] 4 of 5\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (147)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[e2/e0d2e6] process > PlotProfile (MV411_RNP_RUNX... [ 83%] 5 of 6\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (148)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[2c/37fe88] process > PlotProfile (MV411_RNP_AAVS... [ 86%] 6 of 7\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (149)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[2c/37fe88] process > PlotProfile (MV411_RNP_AAVS... [ 88%] 7 of 8\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (150)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[b8/ec5d9c] process > PlotProfile (MV411_RNP_MYB-... [ 89%] 8 of 9\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (151)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[de/2f9acd] process > PlotProfile (MV411_RNP_IRF8... [ 90%] 9 of 10\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (152)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[43/0ba3d9] process > PlotProfile (MV411_RNP_IRF8... [ 91%] 10 of 11\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (153)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[43/0ba3d9] process > PlotProfile (MV411_RNP_IRF8... [ 92%] 11 of 12\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (154)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[78/ac55d3] process > PlotProfile (MV411_RNP_RUNX... [ 92%] 12 of 13\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (155)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[05/0cacbc] process > PlotProfile (MV411_RNP_SPI1... [ 93%] 13 of 14\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (156)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[05/0cacbc] process > PlotProfile (MV411_RNP_SPI1... [ 93%] 14 of 15\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (157)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[28/273a25] process > PlotProfile (MV411_RNP_MEF2... [ 94%] 15 of 16\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (158)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[c2/bcd9cc] process > PlotProfile (MV411_RNP_MYB-... [ 94%] 16 of 17\u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[-        ] process > PeakQC                         -\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[-        ] process > ConsensusPeakSetDESeq          -\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (160)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [  0%] 0 of 1\u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > ConsensusPeakSetAnnotate       -\u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (161)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [100%] 1 of 1 \u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[5e/b631fb] process > ConsensusPeakSetAnnotate (H... [  0%] 0 of 1\u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (161)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [100%] 1 of 1 \u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[5e/b631fb] process > ConsensusPeakSetAnnotate (H... [100%] 1 of 1 \u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [  0%] 0 of 1\u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (161)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [100%] 1 of 1 \u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[5e/b631fb] process > ConsensusPeakSetAnnotate (H... [100%] 1 of 1 \u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > MultiQC                        -\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[28A\n",
      "executor >  local (162)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [100%] 1 of 1 \u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[5e/b631fb] process > ConsensusPeakSetAnnotate (H... [100%] 1 of 1 \u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [100%] 1 of 1 \u001b[K\n",
      "[-        ] process > IGV                            -\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[2b/af351b] process > MultiQC                        [  0%] 0 of 1\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (163)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [100%] 1 of 1 \u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[5e/b631fb] process > ConsensusPeakSetAnnotate (H... [100%] 1 of 1 \u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [100%] 1 of 1 \u001b[K\n",
      "[d4/87dc9d] process > IGV                            [  0%] 0 of 1\u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[2b/af351b] process > MultiQC                        [  0%] 0 of 1\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (163)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [100%] 1 of 1 \u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[5e/b631fb] process > ConsensusPeakSetAnnotate (H... [100%] 1 of 1 \u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [100%] 1 of 1 \u001b[K\n",
      "[d4/87dc9d] process > IGV                            [100%] 1 of 1 \u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[2b/af351b] process > MultiQC                        [  0%] 0 of 1\u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (163)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [100%] 1 of 1 \u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[5e/b631fb] process > ConsensusPeakSetAnnotate (H... [100%] 1 of 1 \u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [100%] 1 of 1 \u001b[K\n",
      "[d4/87dc9d] process > IGV                            [100%] 1 of 1 \u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[2b/af351b] process > MultiQC                        [100%] 1 of 1 \u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[28A\n",
      "executor >  local (163)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [100%] 1 of 1 \u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[5e/b631fb] process > ConsensusPeakSetAnnotate (H... [100%] 1 of 1 \u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [100%] 1 of 1 \u001b[K\n",
      "[d4/87dc9d] process > IGV                            [100%] 1 of 1 \u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[2b/af351b] process > MultiQC                        [100%] 1 of 1 \u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[31mFailed to invoke `workflow.onComplete` event handler\u001b[K\n",
      "\u001b[K\n",
      " -- Check script '/root/.nextflow/assets/nf-core/chipseq/main.nf' at line: 1610 or see '.nextflow.log' file for more details\u001b[39m\u001b[K\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[31A\n",
      "executor >  local (163)\u001b[K\n",
      "[eb/23cd77] process > CheckDesign (additional_deg... [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[c9/ec8de4] process > MakeTSSBED (genes.bed)         [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[0d/f433a4] process > MakeGenomeFilter (genome.fa)   [100%] 1 of 1, cached: 1 \u001b[K\n",
      "[fe/2e0c27] process > FastQC (MV411_RNP_SPI1-H3K2... [100%] 17 of 17, cached:...\u001b[K\n",
      "[68/a565e7] process > TrimGalore (MV411_RNP_MYB-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[a2/0c2388] process > BWAMem (MV411_RNP_RUNX1-H3K... [100%] 17 of 17, cached:...\u001b[K\n",
      "[13/4f49fa] process > SortBAM (INPUT_R1_T1)          [100%] 17 of 17, cached:...\u001b[K\n",
      "[5a/6aaef3] process > MergeBAM (MV411_RNP_RUNX2-H... [100%] 17 of 17, cached:...\u001b[K\n",
      "[48/4b5e62] process > MergeBAMFilter (MV411_RNP_R... [100%] 17 of 17 \u001b[K\n",
      "[dc/6cb451] process > MergeBAMRemoveOrphan (MV411... [100%] 17 of 17 \u001b[K\n",
      "[-        ] process > Preseq                         -\u001b[K\n",
      "[7b/722dae] process > CollectMultipleMetrics (MV4... [100%] 17 of 17 \u001b[K\n",
      "[a1/2654ae] process > BigWig (MV411_RNP_RUNX2-H3K... [100%] 17 of 17 \u001b[K\n",
      "[61/0f1d90] process > PlotProfile (MV411_RNP_RUNX... [100%] 17 of 17 \u001b[K\n",
      "[a0/5324be] process > PhantomPeakQualTools (MV411... [100%] 17 of 17 \u001b[K\n",
      "[e0/988862] process > PlotFingerprint (MV411_RNP_... [100%] 16 of 16 \u001b[K\n",
      "[b4/30f0a0] process > MACSCallPeak (MV411_RNP_RUN... [100%] 16 of 16 \u001b[K\n",
      "[3b/e34578] process > AnnotatePeaks (MV411_RNP_RU... [100%] 16 of 16 \u001b[K\n",
      "[76/5b953a] process > PeakQC                         [100%] 1 of 1 \u001b[K\n",
      "[ee/2435de] process > ConsensusPeakSet (H3K27AC)     [100%] 1 of 1 \u001b[K\n",
      "[5e/b631fb] process > ConsensusPeakSetAnnotate (H... [100%] 1 of 1 \u001b[K\n",
      "[c9/4b85c1] process > ConsensusPeakSetDESeq (H3K2... [100%] 1 of 1 \u001b[K\n",
      "[d4/87dc9d] process > IGV                            [100%] 1 of 1 \u001b[K\n",
      "[7e/9c912a] process > get_software_versions          [100%] 1 of 1 \u001b[K\n",
      "[2b/af351b] process > MultiQC                        [100%] 1 of 1 \u001b[K\n",
      "[8a/7334d7] process > output_documentation           [100%] 1 of 1, cached: 1 \u001b[K\n",
      "\u001b[31mFailed to invoke `workflow.onComplete` event handler\u001b[K\n",
      "\u001b[K\n",
      " -- Check script '/root/.nextflow/assets/nf-core/chipseq/main.nf' at line: 1610 or see '.nextflow.log' file for more details\u001b[39m\u001b[K\n",
      "\u001b[32;1mCompleted at: 06-Aug-2020 19:20:40\n",
      "Duration    : 1d 3h 52m 44s\n",
      "CPU hours   : 538.2 (38.8% cached)\n",
      "Succeeded   : 163\n",
      "Cached      : 83\n",
      "\u001b[22;39m\u001b[K\n",
      "\n"
     ]
    }
   ],
   "source": [
    "#process chips\n",
    "! cd ../../data/ && sudo ../nextflow run nf-core/chipseq --paired_end --seq_center 'DFCI' --email 'jkobject@gmail.com' --input ../AMLproject/nextflow/$project_design.csv --genome GRCh38 --skip_preseq --max_cpus 16 -profile docker -w work -resume exotic_bartik"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! gsutil -m cp -r gs://workamlproject/MEF2CMEF2D/output/* gs://amlproject/Chip/chipseq_MEF2CMEF2D_degraded/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# get scaling values\n",
    "norm, mapped = h.getSpikeInControlScales(refgenome=\"../../data/ref/reference_droso.fna\",\n",
    "fastQfolder='../../data/'+project+'/fastqs/',\n",
    "pairedEnd=True, cores=12,\n",
    "tofilter=True,\n",
    "totrim=True,\n",
    "tomap=True,\n",
    "results=\"../../data/\"+project+\"/\",\n",
    "pathtotrim_galore=\"../../TrimGalore-0.6.5/trim_galore\")\n",
    "norm, mapped"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "computing scales from the excel sheet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [1.00,\n",
    "1.00,\n",
    "0.98,\n",
    "1.08,\n",
    "0.74,\n",
    "0.99,\n",
    "1.13,\n",
    "0.94]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#! mkdir ../../data/$project/bwa \n",
    "#!mkdir ../../data/$project/bwa/mergedLibrary\n",
    "#!gsutil cp gs://amlproject/Chip/$project/bwa/mergedLibrary/*.ba* ../../data/$project/bwa/mergedLibrary/\n",
    "#! gsutil -m cp -r gs://amlproject/Chip/$project/bwa/mergedLibrary/bigwig/ ../../data/$project/bwa/mergedLibrary/\n",
    "! gsutil  -m cp -r gs://amlproject/Chip/$project/bwa/mergedLibrary/macs/ ../../data/$project/bwa/mergedLibrary/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "bams = ! ls ../../data/$project/bwa/mergedLibrary/*.bam\n",
    "bams"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### on scalled data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mkdir ../../data/$project/diffPeaks/\n",
    "! mkdir ../../data/$project/diffData/\n",
    "! mkdir ../../data/$project/droso_aligned\n",
    "! mv ../../data/$project/mp* ../../data/$project/droso_aligned"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "wigs = ! ls ../../data/$project/bwa/mergedLibrary/bigwig/*.bigWig\n",
    "wigs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "# diffPeak on scaled data (full reprocessing)\n",
    "for i in range(len(bams[3:])):\n",
    "    if i < 3:\n",
    "        continue\n",
    "    bam1 = bams[1+(i%2)]\n",
    "    bam2 = bams[3+i]\n",
    "    print(bam1,bam2)\n",
    "    print(chip.fullDiffPeak(bam1,bam2, control1=bams[0], scaling=[1.0, scales[2+i]], directory = \"../../data/\"+project+\"/diffData/\", res_directory = \"../../data/\"+project+\"/diffPeaks/\",pairedend=True))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "initscales = ! cat ../../data/$project/bwa/mergedLibrary/bigwig/scale/*.txt\n",
    "initscales"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "rescales = [val*float(initscales[1+i]) for i, val in enumerate(scales)]\n",
    "rescales"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "chip.bigWigFrom(bams[3:], \n",
    "                genome='GRCh38',scaling=rescales[2:],\n",
    "               numthreads=8)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!mkdir ../../data/$project/recalib_bigwig/ \n",
    "!mv bigwig/* ../../data/$project/recalib_bigwig/\n",
    "!cp ../../data/$project/bwa/mergedLibrary/bigwig/*AAVS1*.bigWig ../../data/$project/recalib_bigwig/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "os.popen('for i in $(ls ../../data/'+project+'/diffPeaks/*.bed); \\\n",
    "            do echo $(wc -l $i); \\\n",
    "            done').read().split('\\n')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "bw = ! ls ../../data/$project/recalib_bigwig/*\n",
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! gsutil -m cp -r ../../data/$project/droso_aligned gs://amlproject/Chip/$project/\n",
    "! gsutil -m cp -r ../../data/$project/recalib_bigwig gs://amlproject/Chip/$project/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!mkdir ../results/$project/\n",
    "!mkdir ../results/$project/plots\n",
    "!mkdir ../results/$project/plots/heatmaps/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# GENOME WIDE comparison"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "peaks = ! ls ../../data/$project/bwa/mergedLibrary/macs/broadPeak/*.broadPeak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"AAVS1\", \"AAVS1_v2\", \"MEF2C\",\"MEF2C_v2\",\"MEF2C-MEF2D\",\"MEF2C-MEF2D_v2\", \"MEF2D\",\"MEF2D_v2\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "for i, val in enumerate(bw):\n",
    "    if i <0:\n",
    "        continue\n",
    "    name = names[i]\n",
    "    print(name)\n",
    "    chip.getPeaksAt(peaks[i], bigwigs = val, bigwignames= name, peaknames=['Macs2_Peaks'], window=3000, folder=\"\", title=name, numthreads=8, refpoint=\"center\", name='../../data/'+project+'/peakplot/'+name+'_mat.pdf', withDeeptools=True, torecompute=True, vmax=2.5, legendLoc=\"lower-left\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "h.createFoldersFor('../results/'+project+'/plots/scaled/heatmaps/')\n",
    "! cp ../../data/$project/*.pdf ../results/$project/plots/scaled/heatmaps/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "cond1peak = ! ls ../../data/$project/diffPeaks/*cond1.bed\n",
    "cond2peak = ! ls ../../data/$project/diffPeaks/*cond2.bed\n",
    "commonpeak = ! ls ../../data/$project/diffPeaks/*common.bed\n",
    "cond1peak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "for i in range(int(len(bw[2:]))):\n",
    "    if i <1:\n",
    "        continue\n",
    "    name1 = bw[i%2]\n",
    "    name2 = bw[i+2]\n",
    "    a = int(i/2)+3 if i%2 else int(i/2)\n",
    "    peak = [cond1peak[a], commonpeak[a], cond2peak[a]]\n",
    "   # for val in peak:\n",
    "      #  chip.dropWeirdChromosomes(val)\n",
    "    name = names[i+2]\n",
    "    print(name,name1,name2,a)\n",
    "    chip.getPeaksAt(peak, [name1, name2], bigwignames=['DMSO', 'VHL'], peaknames=['DMSO_peaks', 'common', 'VHL_peaks'], window=3000, folder=\"\", title=name, numthreads=7, refpoint=\"center\", name='../../data/'+project+'/diffPeaks/'+name+'_mat.pdf', withDeeptools=True, torecompute=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mkdir ../results/additional_degraded_v1/plots/scaled/diffPeaks/\n",
    "! cp ../../data/additional_degraded_v1/diffPeaks/*.pdf ../results/additional_degraded_v1/plots/scaled/diffPeaks/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### on unscalled data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bams = !ls ../../data/$project/bwa/mergedLibrary/mp*.bam\n",
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mkdir ../../data/$project/diffPeaks_unscaled"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#on unscalled data \n",
    "for i in range(int(len(bams[3:]))):\n",
    "    if i < 4:\n",
    "        continue\n",
    "    name1 = bams[1+i%2]\n",
    "    name2 = bams[3+i]\n",
    "    print(name1,name2)\n",
    "    chip.fullDiffPeak(name1,name2, control1='../../data/'+project+'/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam', directory = \"../../data/\"+project+\"/diffData_unscaled/\", res_directory = \"../../data/\"+project+\"/diffPeaks_unscaled/\",pairedend=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bw = ! ls ../../data/$project/bwa/mergedLibrary/bigwig/*.bigWig\n",
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "cond1peak = ! ls ../../data/$project/diffPeaks_unscaled/*cond1.bed\n",
    "cond2peak = ! ls ../../data/$project/diffPeaks_unscaled/*cond2.bed\n",
    "commonpeak = ! ls ../../data/$project/diffPeaks_unscaled/*common.bed\n",
    "commonpeak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for i in range(int(len(bw[2:]))):\n",
    "    name1 = bw[i%2]\n",
    "    name2 = bw[2+i]\n",
    "    a = int(i/2)+3 if i%2 else int(i/2)\n",
    "    peak = [cond1peak[a], commonpeak[a], cond2peak[a]]\n",
    "    #for val in peak:\n",
    "     #   chip.dropWeirdChromosomes(val)\n",
    "    name = names[i+2]\n",
    "    print(name1,name2)\n",
    "    chip.getPeaksAt(peak, [name1, name2], bigwignames=['DMSO', 'VHL'], peaknames=['DMSO_peaks', 'common', 'VHL_peaks'], window=3000, folder=\"\", title=name, numthreads=7, torecompute=True, refpoint='center', name='../../data/'+project+'/diffPeaks_unscaled/'+name+'_mat.pdf', withDeeptools=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "h.createFoldersFor('../results/'+project+'/plots/unscaled/diffPeaks/')\n",
    "! cp ../../data/$project/diffPeaks_unscaled/*.pdf ../results/$project/plots/unscaled/diffPeaks/\n",
    "\n",
    "! gsutil -m cp gs://amlproject/Chip/$project/bwa/mergedLibrary/deepTools/**.pdf ../results/$project/plots/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! gsutil -m cp -r ../../data/$project/diffPeaks gs://amlproject/Chip/$project/\n",
    "! gsutil -m cp -r ../../data/$project/diffData gs://amlproject/Chip/$project/\n",
    "! gsutil -m cp -r ../../data/$project/diffPeaks_unscaled gs://amlproject/Chip/$project/\n",
    "! gsutil -m cp -r ../../data/$project/diffData_unscaled gs://amlproject/Chip/$project/\n",
    "! gsutil -m cp -r ../../data/$project/peakplot gs://amlproject/Chip/$project/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## v8"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 238,
   "metadata": {},
   "outputs": [],
   "source": [
    "project=\"chipseq_IRF8_degraded\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "fastq = ! gsutil ls gs://transfer-amlproject/201023_MP8292_fastq/\n",
    "fastq"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 240,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mkdir: cannot create directory ../data/chipseq_IRF8_degraded/: File exists\r\n"
     ]
    }
   ],
   "source": [
    "mkdir ../data/$project/ && mkdir ../data/$project/qc/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!gsutil -m cp gs://transfer-amlproject/201023_MP8292_fastq/multiqc_report.html ../data/$project/qc/\n",
    "!gsutil -m cp -r gs://transfer-amlproject/201023_MP8292_fastq/Reports/ ../data/$project/qc/\n",
    "!gsutil -m cp -r gs://transfer-amlproject/201023_MP8292_fastq/multiqc_data/ ../data/$project/qc/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! gsutil -m cp gs://transfer-amlproject/201023_MP8292_fastq/*  gs://transfer-amlproject/$project/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "rename = {\n",
    "\"20201023_IRF8_DMSO_MP8292_S119\": \"mp901-MV411_IRF8_NT_DMSO-IRF8-r1\",\n",
    "\"20201023_IRF8_VHL_MP8292_S125\": \"mp902-MV411_IRF8_NT_VHL-IRF8-r1\",\n",
    "\"20201023_MED1_DMSO_MP8292_S122\": \"mp903-MV411_IRF8_NT_DMSO-MED1-r1\",\n",
    "\"20201023_MED1_VHL_MP8292_S128\": \"mp904-MV411_IRF8_NT_VHL-MED1-r1\",\n",
    "\"20201023_MEF2C_DMSO_MP8292_S121\": \"mp905-MV411_IRF8_NT_DMSO-MEF2C-r1\",\n",
    "\"20201023_MEF2C_VHL_MP8292_S127\": \"mp906-MV411_IRF8_NT_VHL-MEF2C-r1\",\n",
    "\"20201023_MEF2D_DMSO_MP8292_S120\": \"mp907-MV411_IRF8_NT_DMSO-MEF2D-r1\",\n",
    "\"20201023_MEF2D_VHL_MP8292_S126\": \"mp908-MV411_IRF8_NT_VHL-MEF2D-r1\",\n",
    "\"20201023_MYC_DMSO_MP8292_S123\": \"mp909-MV411_IRF8_NT_DMSO-MYC-r1\",\n",
    "\"20201023_MYC_VHL_MP8292_S129\": \"mp910-MV411_IRF8_NT_VHL-MYC-r1\",\n",
    "\"20201023_POL_II_total_DMSO_MP8292_S124\": \"mp911-MV411_IRF8_NT_DMSO-POLII_total-r1\",\n",
    "\"20201023_POL_II_total_VHL_MP8292_S130\": \"mp912-MV411_IRF8_NT_VHL-POLII_total-r1\",}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for val in fastq:\n",
    "    rep = val\n",
    "    for k,v in rename.items():\n",
    "        rep = rep.replace(k,v)\n",
    "    rep = rep.replace('transfer-amlproject/201023_MP8292_fastq/',\"amlproject/Chip/\"+project+'/fastqs')\n",
    "    !gsutil cp $val $rep"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from gsheets import Sheets\n",
    "sheets = Sheets.from_files('~/.client_secret.json', '~/.storage.json')\n",
    "url=\"https://docs.google.com/spreadsheets/d/1yFLjYB1McU530JnLgL0QIMAKIkVl3kl0_LCHje2gk8U\"\n",
    "gsheet = sheets.get(url).sheets[2].to_frame()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "fastq = ! gsutil ls gs://amlproject/Chip/$project/fastqs/\n",
    "fastq"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = {\n",
    "\"fastq_1\": [],\n",
    "\"fastq_2\": [],\n",
    "\"antibody\": [],\n",
    "\"group\": [],\n",
    "\"replicate\": [],\n",
    "\"control\": []\n",
    "}\n",
    "for val in h.grouped(fastq,2):\n",
    "    row = gsheet[gsheet.id==val[0].split('/')[-1].split('-')[0]]\n",
    "    df['group'].append(\"MV4\"+row['name'].values[0].split('-r')[0].split('-MV4')[1])\n",
    "    df['replicate'].append(row['name'].values[0].split('-r')[1])\n",
    "    df['fastq_1'].append(val[0])\n",
    "    df['fastq_2'].append(val[1])\n",
    "    df['antibody'].append(row['protein'].values[0])\n",
    "    df['control'].append(\"INPUT\")\n",
    "df['group'].append('INPUT')\n",
    "df['replicate'].append(1)\n",
    "df['fastq_1'].append('gs://amlproject/Chip/IRF2BP2_degraded_rep3/fastqs/mp845-MV411-INPUT-r2_R1.fastq.gz')\n",
    "df['fastq_2'].append('gs://amlproject/Chip/IRF2BP2_degraded_rep3/fastqs/mp845-MV411-INPUT-r2_R2.fastq.gz')\n",
    "df['antibody'].append(\"\")\n",
    "df['control'].append(\"\")\n",
    "df = pd.DataFrame(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df[df.columns[[3,4,0,1,2,5]]].to_csv('../nextflow/'+project+'_design.csv',index=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### about\n",
    "\n",
    "0. you need to have a google project set up with a billing account\n",
    "1. you need to activte your APIs this way: https://cloud.google.com/life-sciences/docs/tutorials/nextflow?hl=fr\n",
    "2. nextflow needs to be installed with this installation command `export NXF_MODE=google && curl https://get.nextflow.io | bash`"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! cd ../../nextflow log ## to get access to the previous runs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#process chips \n",
    "! ../../nextflow run jkobject/chipseq \\\n",
    "    --paired_end \\\n",
    "    --seq_center 'DFCI' \\\n",
    "    --email 'jkobject@gmail.com' \\\n",
    "    --input ../nextflow/chipseq_IRF8_degraded_design.csv \\\n",
    "    --genome GRCh38 \\\n",
    "    --max_cpus 16 \\\n",
    "    -profile jkcloud \\ #my profile for the cloud (to edit for your account)\n",
    "    -w gs://workamlproject/IRF8res \\ #where the withinput will be stored\n",
    "    --spiking \\ #I have spiking using drosophilia genome (default)\n",
    "    --outdir gs://workamlproject/IRF8 \\ #specify aa bucket+folderr where the results will be\n",
    "    --tracedir ../nextflow/IRF8info/ \\ #you need to specify a local place for that"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 243,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam.bai...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-IRF8_R1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-IRF8_R1.mLb.clN.sorted.bam.bai...\n",
      "/ [4 files][  7.5 GiB/  7.5 GiB]  120.9 MiB/s                                   \n",
      "==> NOTE: You are performing a sequence of gsutil operations that may\n",
      "run significantly faster if you instead use gsutil -m cp ... Please\n",
      "see the -m section under \"gsutil help options\" for further information\n",
      "about when gsutil -m can be advantageous.\n",
      "\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MED1_R1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MED1_R1.mLb.clN.sorted.bam.bai...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MEF2C_R1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MEF2C_R1.mLb.clN.sorted.bam.bai...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MEF2D_R1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MEF2D_R1.mLb.clN.sorted.bam.bai...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MYC_R1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MYC_R1.mLb.clN.sorted.bam.bai...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-POLII_total_R1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-POLII_total_R1.mLb.clN.sorted.bam.bai...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-IRF8_R1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-IRF8_R1.mLb.clN.sorted.bam.bai...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MED1_R1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MED1_R1.mLb.clN.sorted.bam.bai...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MEF2C_R1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MEF2C_R1.mLb.clN.sorted.bam.bai...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MEF2D_R1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MEF2D_R1.mLb.clN.sorted.bam.bai...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MYC_R1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MYC_R1.mLb.clN.sorted.bam.bai...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-POLII_total_R1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-POLII_total_R1.mLb.clN.sorted.bam.bai...\n",
      "\\ [26 files][ 32.3 GiB/ 32.3 GiB]   85.2 MiB/s                                  \n",
      "==> NOTE: You are performing a sequence of gsutil operations that may\n",
      "run significantly faster if you instead use gsutil -m cp ... Please\n",
      "see the -m section under \"gsutil help options\" for further information\n",
      "about when gsutil -m can be advantageous.\n",
      "\n",
      "\n",
      "Operation completed over 26 objects/32.3 GiB.                                    \n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/INPUT_R1.bigWig...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-IRF8_R1.bigWig...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-MED1_R1.bigWig...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-MEF2D_R1.bigWig...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-MEF2C_R1.bigWig...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-MYC_R1.bigWig...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_VHL-IRF8_R1.bigWig...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-POLII_total_R1.bigWig...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_VHL-MED1_R1.bigWig...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_VHL-MEF2C_R1.bigWig...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_VHL-MEF2D_R1.bigWig...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_VHL-MYC_R1.bigWig...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_VHL-POLII_total_R1.bigWig...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/scale/INPUT_R1.scale_factor.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/scale/MV411_IRF8_NT_DMSO-IRF8_R1.scale_factor.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/scale/MV411_IRF8_NT_DMSO-MED1_R1.scale_factor.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/scale/MV411_IRF8_NT_DMSO-MEF2C_R1.scale_factor.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/scale/MV411_IRF8_NT_DMSO-MEF2D_R1.scale_factor.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/scale/MV411_IRF8_NT_DMSO-MYC_R1.scale_factor.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/scale/MV411_IRF8_NT_DMSO-POLII_total_R1.scale_factor.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/scale/MV411_IRF8_NT_VHL-IRF8_R1.scale_factor.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/scale/MV411_IRF8_NT_VHL-MED1_R1.scale_factor.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/scale/MV411_IRF8_NT_VHL-MEF2C_R1.scale_factor.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/scale/MV411_IRF8_NT_VHL-MEF2D_R1.scale_factor.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/scale/MV411_IRF8_NT_VHL-MYC_R1.scale_factor.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/scale/MV411_IRF8_NT_VHL-POLII_total_R1.scale_factor.txt...\n",
      "\\ [26/26 files][  2.9 GiB/  2.9 GiB] 100% Done 118.3 MiB/s ETA 00:00:00         \n",
      "Operation completed over 26 objects/2.9 GiB.                                     \n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-IRF8_R1_peaks.annotatePeaks.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-IRF8_R1_peaks.broadPeak...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-IRF8_R1_peaks.gappedPeak...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-IRF8_R1_peaks.xls...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-MED1_R1_peaks.annotatePeaks.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-MED1_R1_peaks.broadPeak...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-MED1_R1_peaks.gappedPeak...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-MED1_R1_peaks.xls...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-MEF2C_R1_peaks.annotatePeaks.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-MEF2C_R1_peaks.broadPeak...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-MEF2C_R1_peaks.gappedPeak...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-MEF2C_R1_peaks.xls...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-MEF2D_R1_peaks.annotatePeaks.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-MEF2D_R1_peaks.broadPeak...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-MEF2D_R1_peaks.gappedPeak...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-MEF2D_R1_peaks.xls...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-MYC_R1_peaks.annotatePeaks.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-MYC_R1_peaks.broadPeak...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-MYC_R1_peaks.gappedPeak...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-MYC_R1_peaks.xls...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-POLII_total_R1_peaks.annotatePeaks.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-POLII_total_R1_peaks.broadPeak...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-POLII_total_R1_peaks.gappedPeak...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-POLII_total_R1_peaks.xls...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-IRF8_R1_peaks.annotatePeaks.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-IRF8_R1_peaks.broadPeak...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-IRF8_R1_peaks.gappedPeak...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-IRF8_R1_peaks.xls...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-MED1_R1_peaks.annotatePeaks.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-MED1_R1_peaks.broadPeak...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-MED1_R1_peaks.gappedPeak...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-MED1_R1_peaks.xls...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-MEF2C_R1_peaks.annotatePeaks.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-MEF2C_R1_peaks.broadPeak...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-MEF2C_R1_peaks.gappedPeak...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-MEF2C_R1_peaks.xls...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-MEF2D_R1_peaks.annotatePeaks.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-MEF2D_R1_peaks.broadPeak...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-MEF2D_R1_peaks.gappedPeak...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-MEF2D_R1_peaks.xls...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-MYC_R1_peaks.annotatePeaks.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-MYC_R1_peaks.broadPeak...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-MYC_R1_peaks.gappedPeak...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-MYC_R1_peaks.xls...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-POLII_total_R1_peaks.broadPeak...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-POLII_total_R1_peaks.annotatePeaks.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-POLII_total_R1_peaks.gappedPeak...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-POLII_total_R1_peaks.xls...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/IRF8/IRF8.consensus_peaks.annotatePeaks.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/IRF8/IRF8.consensus_peaks.bed...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/IRF8/IRF8.consensus_peaks.boolean.annotatePeaks.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/IRF8/IRF8.consensus_peaks.boolean.intersect.plot.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/IRF8/IRF8.consensus_peaks.boolean.intersect.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/IRF8/IRF8.consensus_peaks.boolean.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MYC/MYC.consensus_peaks.bed...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/IRF8/IRF8.consensus_peaks.featureCounts.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MYC/MYC.consensus_peaks.boolean.annotatePeaks.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/IRF8/IRF8.consensus_peaks.featureCounts.txt.summary...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MYC/MYC.consensus_peaks.featureCounts.txt.summary...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/IRF8/IRF8.consensus_peaks.saf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/POLII_total/POLII_total.consensus_peaks.annotatePeaks.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MYC/MYC.consensus_peaks.saf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MED1/MED1.consensus_peaks.annotatePeaks.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/POLII_total/POLII_total.consensus_peaks.bed...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MED1/MED1.consensus_peaks.bed...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MED1/MED1.consensus_peaks.boolean.intersect.plot.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/POLII_total/POLII_total.consensus_peaks.boolean.annotatePeaks.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MED1/MED1.consensus_peaks.boolean.annotatePeaks.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MED1/MED1.consensus_peaks.saf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MED1/MED1.consensus_peaks.featureCounts.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MEF2C/MEF2C.consensus_peaks.annotatePeaks.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MEF2C/MEF2C.consensus_peaks.boolean.intersect.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MED1/MED1.consensus_peaks.featureCounts.txt.summary...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MED1/MED1.consensus_peaks.boolean.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MEF2C/MEF2C.consensus_peaks.bed...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MEF2C/MEF2C.consensus_peaks.saf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MED1/MED1.consensus_peaks.boolean.intersect.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MEF2C/MEF2C.consensus_peaks.boolean.annotatePeaks.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MEF2C/MEF2C.consensus_peaks.featureCounts.txt.summary...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MEF2C/MEF2C.consensus_peaks.boolean.intersect.plot.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MEF2D/MEF2D.consensus_peaks.annotatePeaks.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MEF2C/MEF2C.consensus_peaks.boolean.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MEF2D/MEF2D.consensus_peaks.boolean.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MEF2D/MEF2D.consensus_peaks.boolean.annotatePeaks.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MEF2C/MEF2C.consensus_peaks.featureCounts.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MYC/MYC.consensus_peaks.boolean.intersect.plot.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MEF2D/MEF2D.consensus_peaks.featureCounts.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MEF2D/MEF2D.consensus_peaks.featureCounts.txt.summary...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MEF2D/MEF2D.consensus_peaks.boolean.intersect.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MEF2D/MEF2D.consensus_peaks.boolean.intersect.plot.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MYC/MYC.consensus_peaks.annotatePeaks.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MYC/MYC.consensus_peaks.boolean.intersect.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MEF2D/MEF2D.consensus_peaks.bed...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MYC/MYC.consensus_peaks.boolean.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MEF2D/MEF2D.consensus_peaks.saf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/MYC/MYC.consensus_peaks.featureCounts.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/POLII_total/POLII_total.consensus_peaks.boolean.intersect.plot.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/POLII_total/POLII_total.consensus_peaks.boolean.intersect.txt...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/POLII_total/POLII_total.consensus_peaks.boolean.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/POLII_total/POLII_total.consensus_peaks.featureCounts.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/POLII_total/POLII_total.consensus_peaks.featureCounts.txt.summary...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/consensus/POLII_total/POLII_total.consensus_peaks.saf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/MV411_IRF8_NT_DMSO-IRF8_R1_peaks.FRiP_mqc.tsv...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/MV411_IRF8_NT_DMSO-MED1_R1_peaks.count_mqc.tsv...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/MV411_IRF8_NT_DMSO-IRF8_R1_peaks.count_mqc.tsv...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/MV411_IRF8_NT_DMSO-MED1_R1_peaks.FRiP_mqc.tsv...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/MV411_IRF8_NT_DMSO-MEF2C_R1_peaks.FRiP_mqc.tsv...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/MV411_IRF8_NT_DMSO-MEF2C_R1_peaks.count_mqc.tsv...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/MV411_IRF8_NT_DMSO-MEF2D_R1_peaks.FRiP_mqc.tsv...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/MV411_IRF8_NT_DMSO-MEF2D_R1_peaks.count_mqc.tsv...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/MV411_IRF8_NT_DMSO-MYC_R1_peaks.FRiP_mqc.tsv...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/MV411_IRF8_NT_DMSO-MYC_R1_peaks.count_mqc.tsv...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/MV411_IRF8_NT_DMSO-POLII_total_R1_peaks.FRiP_mqc.tsv...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/MV411_IRF8_NT_DMSO-POLII_total_R1_peaks.count_mqc.tsv...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/MV411_IRF8_NT_VHL-IRF8_R1_peaks.FRiP_mqc.tsv...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/MV411_IRF8_NT_VHL-IRF8_R1_peaks.count_mqc.tsv...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/MV411_IRF8_NT_VHL-MED1_R1_peaks.FRiP_mqc.tsv...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/MV411_IRF8_NT_VHL-MED1_R1_peaks.count_mqc.tsv...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/MV411_IRF8_NT_VHL-MEF2C_R1_peaks.FRiP_mqc.tsv...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/MV411_IRF8_NT_VHL-MEF2C_R1_peaks.count_mqc.tsv...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/MV411_IRF8_NT_VHL-MEF2D_R1_peaks.FRiP_mqc.tsv...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/MV411_IRF8_NT_VHL-MEF2D_R1_peaks.count_mqc.tsv...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/MV411_IRF8_NT_VHL-MYC_R1_peaks.FRiP_mqc.tsv...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/MV411_IRF8_NT_VHL-MYC_R1_peaks.count_mqc.tsv...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/MV411_IRF8_NT_VHL-POLII_total_R1_peaks.FRiP_mqc.tsv...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/MV411_IRF8_NT_VHL-POLII_total_R1_peaks.count_mqc.tsv...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/macs_annotatePeaks.plots.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/macs_annotatePeaks.summary.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/macs_peak.summary.txt...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/macs_annotatePeaks.summary_mqc.tsv...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/qc/macs_peak.plots.pdf...\n",
      "| [131/131 files][270.3 MiB/270.3 MiB] 100% Done                                \n",
      "Operation completed over 131 objects/270.3 MiB.                                  \n"
     ]
    }
   ],
   "source": [
    "h.createFoldersFor('../../data/'+project+'/bwa/mergedLibrary/')\n",
    "!gsutil -m cp gs://amlproject/Chip/$project/bwa/mergedLibrary/*.ba* ../../data/$project/bwa/mergedLibrary/\n",
    "! gsutil -m cp -r gs://amlproject/Chip/$project/bwa/mergedLibrary/bigwig/ ../../data/$project/bwa/mergedLibrary/\n",
    "! gsutil  -m cp -r gs://amlproject/Chip/$project/bwa/mergedLibrary/macs/ ../../data/$project/bwa/mergedLibrary/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "computing scales from the excel sheet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 244,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [\n",
    "        1.00,\n",
    "        0.79,\n",
    "        0.59,\n",
    "        1.00,\n",
    "        0.77,\n",
    "        1.00,\n",
    "        1.00,\n",
    "        0.72,\n",
    "        0.91,\n",
    "        1.00,\n",
    "        1.00,\n",
    "        0.23,\n",
    "         ]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 245,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-IRF8_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MED1_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MEF2C_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MEF2D_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MYC_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-POLII_total_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-IRF8_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MED1_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MEF2C_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MEF2D_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MYC_R1.mLb.clN.sorted.bam',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-POLII_total_R1.mLb.clN.sorted.bam']"
      ]
     },
     "execution_count": 245,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bams = ! ls ../../data/$project/bwa/mergedLibrary/*.bam\n",
    "bams"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### on scalled data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 246,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mkdir ../../data/$project/diffPeaks/ && ! mkdir ../../data/$project/diffData/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 247,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/INPUT_R1.bigWig',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-IRF8_R1.bigWig',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-MED1_R1.bigWig',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-MEF2C_R1.bigWig',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-MEF2D_R1.bigWig',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-MYC_R1.bigWig',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-POLII_total_R1.bigWig',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_VHL-IRF8_R1.bigWig',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_VHL-MED1_R1.bigWig',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_VHL-MEF2C_R1.bigWig',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_VHL-MEF2D_R1.bigWig',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_VHL-MYC_R1.bigWig',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_VHL-POLII_total_R1.bigWig']"
      ]
     },
     "execution_count": 247,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "wigs = ! ls ../../data/$project/bwa/mergedLibrary/bigwig/*.bigWig\n",
    "wigs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 249,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-IRF8_R1.mLb.clN.sorted.bam ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-IRF8_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-IRF8_R1.mLb.clN.sorted.bam and ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-IRF8_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "200\n",
      "computing the scaling values\n",
      "b'INFO  @ Thu, 05 Nov 2020 00:06:25: \\n# Command line: callpeak -B -t ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-IRF8_R1.mLb.clN.sorted.bam -c ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 200 -n MV411_IRF8_NT_DMSO-IRF8_R1 --outdir ../../data/chipseq_IRF8_degraded/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_IRF8_NT_DMSO-IRF8_R1\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-IRF8_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Thu, 05 Nov 2020 00:06:25: #1 read fragment files... \\nINFO  @ Thu, 05 Nov 2020 00:06:25: #1 read treatment fragments... \\nINFO  @ Thu, 05 Nov 2020 00:06:29:  1000000 \\nINFO  @ Thu, 05 Nov 2020 00:06:34:  2000000 \\nINFO  @ Thu, 05 Nov 2020 00:06:39:  3000000 \\nINFO  @ Thu, 05 Nov 2020 00:06:43:  4000000 \\nINFO  @ Thu, 05 Nov 2020 00:06:48:  5000000 \\nINFO  @ Thu, 05 Nov 2020 00:06:53:  6000000 \\nINFO  @ Thu, 05 Nov 2020 00:06:57:  7000000 \\nINFO  @ Thu, 05 Nov 2020 00:07:02:  8000000 \\nINFO  @ Thu, 05 Nov 2020 00:07:06:  9000000 \\nINFO  @ Thu, 05 Nov 2020 00:07:11:  10000000 \\nINFO  @ Thu, 05 Nov 2020 00:07:15:  11000000 \\nINFO  @ Thu, 05 Nov 2020 00:07:20:  12000000 \\nINFO  @ Thu, 05 Nov 2020 00:07:25:  13000000 \\nINFO  @ Thu, 05 Nov 2020 00:07:29:  14000000 \\nINFO  @ Thu, 05 Nov 2020 00:07:34:  15000000 \\nINFO  @ Thu, 05 Nov 2020 00:07:38:  16000000 \\nINFO  @ Thu, 05 Nov 2020 00:07:39: 16213604 fragments have been read. \\nINFO  @ Thu, 05 Nov 2020 00:07:49: #1.2 read input fragments... \\nINFO  @ Thu, 05 Nov 2020 00:07:53:  1000000 \\nINFO  @ Thu, 05 Nov 2020 00:07:56:  2000000 \\nINFO  @ Thu, 05 Nov 2020 00:08:00:  3000000 \\nINFO  @ Thu, 05 Nov 2020 00:08:04:  4000000 \\nINFO  @ Thu, 05 Nov 2020 00:08:08:  5000000 \\nINFO  @ Thu, 05 Nov 2020 00:08:12:  6000000 \\nINFO  @ Thu, 05 Nov 2020 00:08:16:  7000000 \\nINFO  @ Thu, 05 Nov 2020 00:08:20:  8000000 \\nINFO  @ Thu, 05 Nov 2020 00:08:24:  9000000 \\nINFO  @ Thu, 05 Nov 2020 00:08:28:  10000000 \\nINFO  @ Thu, 05 Nov 2020 00:08:32:  11000000 \\nINFO  @ Thu, 05 Nov 2020 00:08:36:  12000000 \\nINFO  @ Thu, 05 Nov 2020 00:08:40:  13000000 \\nINFO  @ Thu, 05 Nov 2020 00:08:44:  14000000 \\nINFO  @ Thu, 05 Nov 2020 00:08:48:  15000000 \\nINFO  @ Thu, 05 Nov 2020 00:08:52:  16000000 \\nINFO  @ Thu, 05 Nov 2020 00:08:56:  17000000 \\nINFO  @ Thu, 05 Nov 2020 00:09:00:  18000000 \\nINFO  @ Thu, 05 Nov 2020 00:09:04:  19000000 \\nINFO  @ Thu, 05 Nov 2020 00:09:08:  20000000 \\nINFO  @ Thu, 05 Nov 2020 00:09:11:  21000000 \\nINFO  @ Thu, 05 Nov 2020 00:09:15:  22000000 \\nINFO  @ Thu, 05 Nov 2020 00:09:19:  23000000 \\nINFO  @ Thu, 05 Nov 2020 00:09:23:  24000000 \\nINFO  @ Thu, 05 Nov 2020 00:09:27:  25000000 \\nINFO  @ Thu, 05 Nov 2020 00:09:31:  26000000 \\nINFO  @ Thu, 05 Nov 2020 00:09:35:  27000000 \\nINFO  @ Thu, 05 Nov 2020 00:09:39:  28000000 \\nINFO  @ Thu, 05 Nov 2020 00:09:43:  29000000 \\nINFO  @ Thu, 05 Nov 2020 00:09:47:  30000000 \\nINFO  @ Thu, 05 Nov 2020 00:09:51:  31000000 \\nINFO  @ Thu, 05 Nov 2020 00:09:54:  32000000 \\nINFO  @ Thu, 05 Nov 2020 00:09:58:  33000000 \\nINFO  @ Thu, 05 Nov 2020 00:10:02:  34000000 \\nINFO  @ Thu, 05 Nov 2020 00:10:06:  35000000 \\nINFO  @ Thu, 05 Nov 2020 00:10:10:  36000000 \\nINFO  @ Thu, 05 Nov 2020 00:10:14:  37000000 \\nINFO  @ Thu, 05 Nov 2020 00:10:18:  38000000 \\nINFO  @ Thu, 05 Nov 2020 00:10:22:  39000000 \\nINFO  @ Thu, 05 Nov 2020 00:10:26:  40000000 \\nINFO  @ Thu, 05 Nov 2020 00:10:30:  41000000 \\nINFO  @ Thu, 05 Nov 2020 00:10:34:  42000000 \\nINFO  @ Thu, 05 Nov 2020 00:10:38:  43000000 \\nINFO  @ Thu, 05 Nov 2020 00:10:42:  44000000 \\nINFO  @ Thu, 05 Nov 2020 00:10:46:  45000000 \\nINFO  @ Thu, 05 Nov 2020 00:10:50:  46000000 \\nINFO  @ Thu, 05 Nov 2020 00:10:53:  47000000 \\nINFO  @ Thu, 05 Nov 2020 00:10:57:  48000000 \\nINFO  @ Thu, 05 Nov 2020 00:11:01:  49000000 \\nINFO  @ Thu, 05 Nov 2020 00:11:05:  50000000 \\nINFO  @ Thu, 05 Nov 2020 00:11:09:  51000000 \\nINFO  @ Thu, 05 Nov 2020 00:11:13:  52000000 \\nINFO  @ Thu, 05 Nov 2020 00:11:17:  53000000 \\nINFO  @ Thu, 05 Nov 2020 00:11:21:  54000000 \\nINFO  @ Thu, 05 Nov 2020 00:11:25:  55000000 \\nINFO  @ Thu, 05 Nov 2020 00:11:29:  56000000 \\nINFO  @ Thu, 05 Nov 2020 00:11:33:  57000000 \\nINFO  @ Thu, 05 Nov 2020 00:11:37:  58000000 \\nINFO  @ Thu, 05 Nov 2020 00:11:40:  59000000 \\nINFO  @ Thu, 05 Nov 2020 00:11:44:  60000000 \\nINFO  @ Thu, 05 Nov 2020 00:11:48:  61000000 \\nINFO  @ Thu, 05 Nov 2020 00:11:52:  62000000 \\nINFO  @ Thu, 05 Nov 2020 00:11:56:  63000000 \\nINFO  @ Thu, 05 Nov 2020 00:12:00:  64000000 \\nINFO  @ Thu, 05 Nov 2020 00:12:04:  65000000 \\nINFO  @ Thu, 05 Nov 2020 00:12:09:  66000000 \\nINFO  @ Thu, 05 Nov 2020 00:12:11: 66660890 fragments have been read. \\nINFO  @ Thu, 05 Nov 2020 00:12:56: #1 mean fragment size is determined as 231.6 bp from treatment \\nINFO  @ Thu, 05 Nov 2020 00:12:56: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Thu, 05 Nov 2020 00:12:56: #1 fragment size = 231.6 \\nINFO  @ Thu, 05 Nov 2020 00:12:56: #1  total fragments in treatment: 16213604 \\nINFO  @ Thu, 05 Nov 2020 00:12:56: #1 user defined the maximum fragments... \\nINFO  @ Thu, 05 Nov 2020 00:12:56: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Thu, 05 Nov 2020 00:13:23: #1  fragments after filtering in treatment: 16198492 \\nINFO  @ Thu, 05 Nov 2020 00:13:23: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Thu, 05 Nov 2020 00:13:23: #1  total fragments in control: 66660890 \\nINFO  @ Thu, 05 Nov 2020 00:13:23: #1 user defined the maximum fragments... \\nINFO  @ Thu, 05 Nov 2020 00:13:23: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Thu, 05 Nov 2020 00:15:16: #1  fragments after filtering in control: 66633066 \\nINFO  @ Thu, 05 Nov 2020 00:15:16: #1  Redundant rate of control: 0.00 \\nINFO  @ Thu, 05 Nov 2020 00:15:16: #1 finished! \\nINFO  @ Thu, 05 Nov 2020 00:15:16: #2 Build Peak Model... \\nINFO  @ Thu, 05 Nov 2020 00:15:16: #2 Skipped... \\nINFO  @ Thu, 05 Nov 2020 00:15:16: #3 Call peaks... \\nINFO  @ Thu, 05 Nov 2020 00:15:16: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 05 Nov 2020 00:24:33: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 05 Nov 2020 00:24:33: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_IRF8_NT_DMSO-IRF8_R1_treat_pileup.bdg \\nINFO  @ Thu, 05 Nov 2020 00:24:33: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_IRF8_NT_DMSO-IRF8_R1_control_lambda.bdg \\nINFO  @ Thu, 05 Nov 2020 00:24:33: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 05 Nov 2020 00:24:33: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 05 Nov 2020 00:30:29: #4 Write output xls file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-IRF8_R1_peaks.xls \\nINFO  @ Thu, 05 Nov 2020 00:30:29: #4 Write peak in narrowPeak format file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-IRF8_R1_peaks.narrowPeak \\nINFO  @ Thu, 05 Nov 2020 00:30:30: #4 Write summits bed file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-IRF8_R1_summits.bed \\nINFO  @ Thu, 05 Nov 2020 00:30:30: Done! \\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'INFO  @ Thu, 05 Nov 2020 00:30:30: \\n# Command line: callpeak -B -t ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-IRF8_R1.mLb.clN.sorted.bam -c ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 200 -n MV411_IRF8_NT_VHL-IRF8_R1 --outdir ../../data/chipseq_IRF8_degraded/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_IRF8_NT_VHL-IRF8_R1\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-IRF8_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Thu, 05 Nov 2020 00:30:30: #1 read fragment files... \\nINFO  @ Thu, 05 Nov 2020 00:30:30: #1 read treatment fragments... \\nINFO  @ Thu, 05 Nov 2020 00:30:35:  1000000 \\nINFO  @ Thu, 05 Nov 2020 00:30:40:  2000000 \\nINFO  @ Thu, 05 Nov 2020 00:30:45:  3000000 \\nINFO  @ Thu, 05 Nov 2020 00:30:50:  4000000 \\nINFO  @ Thu, 05 Nov 2020 00:30:55:  5000000 \\nINFO  @ Thu, 05 Nov 2020 00:31:00:  6000000 \\nINFO  @ Thu, 05 Nov 2020 00:31:05:  7000000 \\nINFO  @ Thu, 05 Nov 2020 00:31:10:  8000000 \\nINFO  @ Thu, 05 Nov 2020 00:31:14:  9000000 \\nINFO  @ Thu, 05 Nov 2020 00:31:19:  10000000 \\nINFO  @ Thu, 05 Nov 2020 00:31:24:  11000000 \\nINFO  @ Thu, 05 Nov 2020 00:31:29:  12000000 \\nINFO  @ Thu, 05 Nov 2020 00:31:34:  13000000 \\nINFO  @ Thu, 05 Nov 2020 00:31:39:  14000000 \\nINFO  @ Thu, 05 Nov 2020 00:31:44:  15000000 \\nINFO  @ Thu, 05 Nov 2020 00:31:48: 15766460 fragments have been read. \\nINFO  @ Thu, 05 Nov 2020 00:31:56: #1.2 read input fragments... \\nINFO  @ Thu, 05 Nov 2020 00:32:00:  1000000 \\nINFO  @ Thu, 05 Nov 2020 00:32:04:  2000000 \\nINFO  @ Thu, 05 Nov 2020 00:32:07:  3000000 \\nINFO  @ Thu, 05 Nov 2020 00:32:11:  4000000 \\nINFO  @ Thu, 05 Nov 2020 00:32:15:  5000000 \\nINFO  @ Thu, 05 Nov 2020 00:32:19:  6000000 \\nINFO  @ Thu, 05 Nov 2020 00:32:22:  7000000 \\nINFO  @ Thu, 05 Nov 2020 00:32:26:  8000000 \\nINFO  @ Thu, 05 Nov 2020 00:32:30:  9000000 \\nINFO  @ Thu, 05 Nov 2020 00:32:34:  10000000 \\nINFO  @ Thu, 05 Nov 2020 00:32:38:  11000000 \\nINFO  @ Thu, 05 Nov 2020 00:32:42:  12000000 \\nINFO  @ Thu, 05 Nov 2020 00:32:45:  13000000 \\nINFO  @ Thu, 05 Nov 2020 00:32:49:  14000000 \\nINFO  @ Thu, 05 Nov 2020 00:32:53:  15000000 \\nINFO  @ Thu, 05 Nov 2020 00:32:57:  16000000 \\nINFO  @ Thu, 05 Nov 2020 00:33:00:  17000000 \\nINFO  @ Thu, 05 Nov 2020 00:33:04:  18000000 \\nINFO  @ Thu, 05 Nov 2020 00:33:08:  19000000 \\nINFO  @ Thu, 05 Nov 2020 00:33:12:  20000000 \\nINFO  @ Thu, 05 Nov 2020 00:33:15:  21000000 \\nINFO  @ Thu, 05 Nov 2020 00:33:19:  22000000 \\nINFO  @ Thu, 05 Nov 2020 00:33:23:  23000000 \\nINFO  @ Thu, 05 Nov 2020 00:33:27:  24000000 \\nINFO  @ Thu, 05 Nov 2020 00:33:31:  25000000 \\nINFO  @ Thu, 05 Nov 2020 00:33:35:  26000000 \\nINFO  @ Thu, 05 Nov 2020 00:33:38:  27000000 \\nINFO  @ Thu, 05 Nov 2020 00:33:42:  28000000 \\nINFO  @ Thu, 05 Nov 2020 00:33:46:  29000000 \\nINFO  @ Thu, 05 Nov 2020 00:33:50:  30000000 \\nINFO  @ Thu, 05 Nov 2020 00:33:54:  31000000 \\nINFO  @ Thu, 05 Nov 2020 00:33:57:  32000000 \\nINFO  @ Thu, 05 Nov 2020 00:34:01:  33000000 \\nINFO  @ Thu, 05 Nov 2020 00:34:05:  34000000 \\nINFO  @ Thu, 05 Nov 2020 00:34:09:  35000000 \\nINFO  @ Thu, 05 Nov 2020 00:34:12:  36000000 \\nINFO  @ Thu, 05 Nov 2020 00:34:16:  37000000 \\nINFO  @ Thu, 05 Nov 2020 00:34:20:  38000000 \\nINFO  @ Thu, 05 Nov 2020 00:34:24:  39000000 \\nINFO  @ Thu, 05 Nov 2020 00:34:27:  40000000 \\nINFO  @ Thu, 05 Nov 2020 00:34:31:  41000000 \\nINFO  @ Thu, 05 Nov 2020 00:34:35:  42000000 \\nINFO  @ Thu, 05 Nov 2020 00:34:39:  43000000 \\nINFO  @ Thu, 05 Nov 2020 00:34:43:  44000000 \\nINFO  @ Thu, 05 Nov 2020 00:34:46:  45000000 \\nINFO  @ Thu, 05 Nov 2020 00:34:50:  46000000 \\nINFO  @ Thu, 05 Nov 2020 00:34:54:  47000000 \\nINFO  @ Thu, 05 Nov 2020 00:34:58:  48000000 \\nINFO  @ Thu, 05 Nov 2020 00:35:02:  49000000 \\nINFO  @ Thu, 05 Nov 2020 00:35:05:  50000000 \\nINFO  @ Thu, 05 Nov 2020 00:35:09:  51000000 \\nINFO  @ Thu, 05 Nov 2020 00:35:13:  52000000 \\nINFO  @ Thu, 05 Nov 2020 00:35:17:  53000000 \\nINFO  @ Thu, 05 Nov 2020 00:35:21:  54000000 \\nINFO  @ Thu, 05 Nov 2020 00:35:25:  55000000 \\nINFO  @ Thu, 05 Nov 2020 00:35:29:  56000000 \\nINFO  @ Thu, 05 Nov 2020 00:35:33:  57000000 \\nINFO  @ Thu, 05 Nov 2020 00:35:36:  58000000 \\nINFO  @ Thu, 05 Nov 2020 00:35:40:  59000000 \\nINFO  @ Thu, 05 Nov 2020 00:35:44:  60000000 \\nINFO  @ Thu, 05 Nov 2020 00:35:48:  61000000 \\nINFO  @ Thu, 05 Nov 2020 00:35:52:  62000000 \\nINFO  @ Thu, 05 Nov 2020 00:35:56:  63000000 \\nINFO  @ Thu, 05 Nov 2020 00:36:00:  64000000 \\nINFO  @ Thu, 05 Nov 2020 00:36:03:  65000000 \\nINFO  @ Thu, 05 Nov 2020 00:36:07:  66000000 \\nINFO  @ Thu, 05 Nov 2020 00:36:10: 66660890 fragments have been read. \\nINFO  @ Thu, 05 Nov 2020 00:36:54: #1 mean fragment size is determined as 213.8 bp from treatment \\nINFO  @ Thu, 05 Nov 2020 00:36:54: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Thu, 05 Nov 2020 00:36:54: #1 fragment size = 213.8 \\nINFO  @ Thu, 05 Nov 2020 00:36:54: #1  total fragments in treatment: 15766460 \\nINFO  @ Thu, 05 Nov 2020 00:36:54: #1 user defined the maximum fragments... \\nINFO  @ Thu, 05 Nov 2020 00:36:54: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Thu, 05 Nov 2020 00:37:22: #1  fragments after filtering in treatment: 15754506 \\nINFO  @ Thu, 05 Nov 2020 00:37:22: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Thu, 05 Nov 2020 00:37:22: #1  total fragments in control: 66660890 \\nINFO  @ Thu, 05 Nov 2020 00:37:22: #1 user defined the maximum fragments... \\nINFO  @ Thu, 05 Nov 2020 00:37:22: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Thu, 05 Nov 2020 00:39:15: #1  fragments after filtering in control: 66633066 \\nINFO  @ Thu, 05 Nov 2020 00:39:15: #1  Redundant rate of control: 0.00 \\nINFO  @ Thu, 05 Nov 2020 00:39:15: #1 finished! \\nINFO  @ Thu, 05 Nov 2020 00:39:15: #2 Build Peak Model... \\nINFO  @ Thu, 05 Nov 2020 00:39:15: #2 Skipped... \\nINFO  @ Thu, 05 Nov 2020 00:39:15: #3 Call peaks... \\nINFO  @ Thu, 05 Nov 2020 00:39:15: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 05 Nov 2020 00:48:22: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 05 Nov 2020 00:48:22: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_IRF8_NT_VHL-IRF8_R1_treat_pileup.bdg \\nINFO  @ Thu, 05 Nov 2020 00:48:22: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_IRF8_NT_VHL-IRF8_R1_control_lambda.bdg \\nINFO  @ Thu, 05 Nov 2020 00:48:22: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 05 Nov 2020 00:48:22: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 05 Nov 2020 00:54:11: #4 Write output xls file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-IRF8_R1_peaks.xls \\nINFO  @ Thu, 05 Nov 2020 00:54:11: #4 Write peak in narrowPeak format file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-IRF8_R1_peaks.narrowPeak \\nINFO  @ Thu, 05 Nov 2020 00:54:11: #4 Write summits bed file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-IRF8_R1_summits.bed \\nINFO  @ Thu, 05 Nov 2020 00:54:11: Done! \\n'\n",
      "16198492 19942412\n",
      "doing differential peak binding\n",
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-IRF8_R1_treat_pileup.bdg --c1 ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-IRF8_R1_control_lambda.bdg --t2 ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-IRF8_R1_treat_pileup.bdg --c2 ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-IRF8_R1_control_lambda.bdg --d1 16198492 --d2 19942412 -g 60 -l 200 --o-prefix MV411_IRF8_NT_DMSO-IRF8_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-IRF8_R1_treat_pileup --outdir ../../data/chipseq_IRF8_degraded/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Thu, 05 Nov 2020 00:54:11: Read and build treatment 1 bedGraph... \\nINFO  @ Thu, 05 Nov 2020 00:54:39: Read and build control 1 bedGraph... \\nINFO  @ Thu, 05 Nov 2020 00:58:20: Read and build treatment 2 bedGraph... \\nINFO  @ Thu, 05 Nov 2020 00:58:49: Read and build control 2 bedGraph... \\nINFO  @ Thu, 05 Nov 2020 01:10:33: Write peaks... \\nINFO  @ Thu, 05 Nov 2020 01:10:33: Done \\n')\n",
      "../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MED1_R1.mLb.clN.sorted.bam ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MED1_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MED1_R1.mLb.clN.sorted.bam and ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MED1_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "212\n",
      "computing the scaling values\n",
      "b'INFO  @ Thu, 05 Nov 2020 01:12:21: \\n# Command line: callpeak -B -t ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MED1_R1.mLb.clN.sorted.bam -c ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 212 -n MV411_IRF8_NT_DMSO-MED1_R1 --outdir ../../data/chipseq_IRF8_degraded/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_IRF8_NT_DMSO-MED1_R1\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MED1_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Thu, 05 Nov 2020 01:12:21: #1 read fragment files... \\nINFO  @ Thu, 05 Nov 2020 01:12:21: #1 read treatment fragments... \\nINFO  @ Thu, 05 Nov 2020 01:12:25:  1000000 \\nINFO  @ Thu, 05 Nov 2020 01:12:30:  2000000 \\nINFO  @ Thu, 05 Nov 2020 01:12:35:  3000000 \\nINFO  @ Thu, 05 Nov 2020 01:12:39:  4000000 \\nINFO  @ Thu, 05 Nov 2020 01:12:44:  5000000 \\nINFO  @ Thu, 05 Nov 2020 01:12:49:  6000000 \\nINFO  @ Thu, 05 Nov 2020 01:12:53:  7000000 \\nINFO  @ Thu, 05 Nov 2020 01:12:58:  8000000 \\nINFO  @ Thu, 05 Nov 2020 01:13:03:  9000000 \\nINFO  @ Thu, 05 Nov 2020 01:13:07:  10000000 \\nINFO  @ Thu, 05 Nov 2020 01:13:12:  11000000 \\nINFO  @ Thu, 05 Nov 2020 01:13:17:  12000000 \\nINFO  @ Thu, 05 Nov 2020 01:13:21:  13000000 \\nINFO  @ Thu, 05 Nov 2020 01:13:26:  14000000 \\nINFO  @ Thu, 05 Nov 2020 01:13:31:  15000000 \\nINFO  @ Thu, 05 Nov 2020 01:13:36:  16000000 \\nINFO  @ Thu, 05 Nov 2020 01:13:41:  17000000 \\nINFO  @ Thu, 05 Nov 2020 01:13:46:  18000000 \\nINFO  @ Thu, 05 Nov 2020 01:13:51:  19000000 \\nINFO  @ Thu, 05 Nov 2020 01:13:55:  20000000 \\nINFO  @ Thu, 05 Nov 2020 01:13:59: 20739878 fragments have been read. \\nINFO  @ Thu, 05 Nov 2020 01:14:10: #1.2 read input fragments... \\nINFO  @ Thu, 05 Nov 2020 01:14:14:  1000000 \\nINFO  @ Thu, 05 Nov 2020 01:14:18:  2000000 \\nINFO  @ Thu, 05 Nov 2020 01:14:22:  3000000 \\nINFO  @ Thu, 05 Nov 2020 01:14:25:  4000000 \\nINFO  @ Thu, 05 Nov 2020 01:14:29:  5000000 \\nINFO  @ Thu, 05 Nov 2020 01:14:33:  6000000 \\nINFO  @ Thu, 05 Nov 2020 01:14:36:  7000000 \\nINFO  @ Thu, 05 Nov 2020 01:14:40:  8000000 \\nINFO  @ Thu, 05 Nov 2020 01:14:44:  9000000 \\nINFO  @ Thu, 05 Nov 2020 01:14:47:  10000000 \\nINFO  @ Thu, 05 Nov 2020 01:14:51:  11000000 \\nINFO  @ Thu, 05 Nov 2020 01:14:55:  12000000 \\nINFO  @ Thu, 05 Nov 2020 01:14:59:  13000000 \\nINFO  @ Thu, 05 Nov 2020 01:15:02:  14000000 \\nINFO  @ Thu, 05 Nov 2020 01:15:06:  15000000 \\nINFO  @ Thu, 05 Nov 2020 01:15:10:  16000000 \\nINFO  @ Thu, 05 Nov 2020 01:15:13:  17000000 \\nINFO  @ Thu, 05 Nov 2020 01:15:17:  18000000 \\nINFO  @ Thu, 05 Nov 2020 01:15:21:  19000000 \\nINFO  @ Thu, 05 Nov 2020 01:15:24:  20000000 \\nINFO  @ Thu, 05 Nov 2020 01:15:28:  21000000 \\nINFO  @ Thu, 05 Nov 2020 01:15:32:  22000000 \\nINFO  @ Thu, 05 Nov 2020 01:15:36:  23000000 \\nINFO  @ Thu, 05 Nov 2020 01:15:39:  24000000 \\nINFO  @ Thu, 05 Nov 2020 01:15:43:  25000000 \\nINFO  @ Thu, 05 Nov 2020 01:15:47:  26000000 \\nINFO  @ Thu, 05 Nov 2020 01:15:50:  27000000 \\nINFO  @ Thu, 05 Nov 2020 01:15:54:  28000000 \\nINFO  @ Thu, 05 Nov 2020 01:15:58:  29000000 \\nINFO  @ Thu, 05 Nov 2020 01:16:02:  30000000 \\nINFO  @ Thu, 05 Nov 2020 01:16:05:  31000000 \\nINFO  @ Thu, 05 Nov 2020 01:16:09:  32000000 \\nINFO  @ Thu, 05 Nov 2020 01:16:13:  33000000 \\nINFO  @ Thu, 05 Nov 2020 01:16:16:  34000000 \\nINFO  @ Thu, 05 Nov 2020 01:16:20:  35000000 \\nINFO  @ Thu, 05 Nov 2020 01:16:24:  36000000 \\nINFO  @ Thu, 05 Nov 2020 01:16:28:  37000000 \\nINFO  @ Thu, 05 Nov 2020 01:16:31:  38000000 \\nINFO  @ Thu, 05 Nov 2020 01:16:35:  39000000 \\nINFO  @ Thu, 05 Nov 2020 01:16:39:  40000000 \\nINFO  @ Thu, 05 Nov 2020 01:16:43:  41000000 \\nINFO  @ Thu, 05 Nov 2020 01:16:47:  42000000 \\nINFO  @ Thu, 05 Nov 2020 01:16:50:  43000000 \\nINFO  @ Thu, 05 Nov 2020 01:16:54:  44000000 \\nINFO  @ Thu, 05 Nov 2020 01:16:58:  45000000 \\nINFO  @ Thu, 05 Nov 2020 01:17:02:  46000000 \\nINFO  @ Thu, 05 Nov 2020 01:17:08:  47000000 \\nINFO  @ Thu, 05 Nov 2020 01:17:11:  48000000 \\nINFO  @ Thu, 05 Nov 2020 01:17:15:  49000000 \\nINFO  @ Thu, 05 Nov 2020 01:17:19:  50000000 \\nINFO  @ Thu, 05 Nov 2020 01:17:23:  51000000 \\nINFO  @ Thu, 05 Nov 2020 01:17:26:  52000000 \\nINFO  @ Thu, 05 Nov 2020 01:17:30:  53000000 \\nINFO  @ Thu, 05 Nov 2020 01:17:34:  54000000 \\nINFO  @ Thu, 05 Nov 2020 01:17:38:  55000000 \\nINFO  @ Thu, 05 Nov 2020 01:17:41:  56000000 \\nINFO  @ Thu, 05 Nov 2020 01:17:45:  57000000 \\nINFO  @ Thu, 05 Nov 2020 01:17:49:  58000000 \\nINFO  @ Thu, 05 Nov 2020 01:17:52:  59000000 \\nINFO  @ Thu, 05 Nov 2020 01:17:56:  60000000 \\nINFO  @ Thu, 05 Nov 2020 01:18:00:  61000000 \\nINFO  @ Thu, 05 Nov 2020 01:18:04:  62000000 \\nINFO  @ Thu, 05 Nov 2020 01:18:08:  63000000 \\nINFO  @ Thu, 05 Nov 2020 01:18:11:  64000000 \\nINFO  @ Thu, 05 Nov 2020 01:18:15:  65000000 \\nINFO  @ Thu, 05 Nov 2020 01:18:19:  66000000 \\nINFO  @ Thu, 05 Nov 2020 01:18:22: 66660890 fragments have been read. \\nINFO  @ Thu, 05 Nov 2020 01:19:06: #1 mean fragment size is determined as 224.2 bp from treatment \\nINFO  @ Thu, 05 Nov 2020 01:19:06: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Thu, 05 Nov 2020 01:19:06: #1 fragment size = 224.2 \\nINFO  @ Thu, 05 Nov 2020 01:19:06: #1  total fragments in treatment: 20739878 \\nINFO  @ Thu, 05 Nov 2020 01:19:06: #1 user defined the maximum fragments... \\nINFO  @ Thu, 05 Nov 2020 01:19:06: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Thu, 05 Nov 2020 01:19:39: #1  fragments after filtering in treatment: 20729873 \\nINFO  @ Thu, 05 Nov 2020 01:19:39: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Thu, 05 Nov 2020 01:19:39: #1  total fragments in control: 66660890 \\nINFO  @ Thu, 05 Nov 2020 01:19:39: #1 user defined the maximum fragments... \\nINFO  @ Thu, 05 Nov 2020 01:19:39: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Thu, 05 Nov 2020 01:21:27: #1  fragments after filtering in control: 66633066 \\nINFO  @ Thu, 05 Nov 2020 01:21:27: #1  Redundant rate of control: 0.00 \\nINFO  @ Thu, 05 Nov 2020 01:21:27: #1 finished! \\nINFO  @ Thu, 05 Nov 2020 01:21:27: #2 Build Peak Model... \\nINFO  @ Thu, 05 Nov 2020 01:21:27: #2 Skipped... \\nINFO  @ Thu, 05 Nov 2020 01:21:27: #3 Call peaks... \\nINFO  @ Thu, 05 Nov 2020 01:21:27: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 05 Nov 2020 01:30:49: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 05 Nov 2020 01:30:49: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_IRF8_NT_DMSO-MED1_R1_treat_pileup.bdg \\nINFO  @ Thu, 05 Nov 2020 01:30:49: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_IRF8_NT_DMSO-MED1_R1_control_lambda.bdg \\nINFO  @ Thu, 05 Nov 2020 01:30:49: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 05 Nov 2020 01:30:49: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 05 Nov 2020 01:36:57: #4 Write output xls file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MED1_R1_peaks.xls \\nINFO  @ Thu, 05 Nov 2020 01:36:57: #4 Write peak in narrowPeak format file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MED1_R1_peaks.narrowPeak \\nINFO  @ Thu, 05 Nov 2020 01:36:57: #4 Write summits bed file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MED1_R1_summits.bed \\nINFO  @ Thu, 05 Nov 2020 01:36:57: Done! \\n'\n",
      "b'INFO  @ Thu, 05 Nov 2020 01:36:58: \\n# Command line: callpeak -B -t ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MED1_R1.mLb.clN.sorted.bam -c ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 212 -n MV411_IRF8_NT_VHL-MED1_R1 --outdir ../../data/chipseq_IRF8_degraded/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_IRF8_NT_VHL-MED1_R1\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MED1_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Thu, 05 Nov 2020 01:36:58: #1 read fragment files... \\nINFO  @ Thu, 05 Nov 2020 01:36:58: #1 read treatment fragments... \\nINFO  @ Thu, 05 Nov 2020 01:37:03:  1000000 \\nINFO  @ Thu, 05 Nov 2020 01:37:07:  2000000 \\nINFO  @ Thu, 05 Nov 2020 01:37:12:  3000000 \\nINFO  @ Thu, 05 Nov 2020 01:37:18:  4000000 \\nINFO  @ Thu, 05 Nov 2020 01:37:23:  5000000 \\nINFO  @ Thu, 05 Nov 2020 01:37:28:  6000000 \\nINFO  @ Thu, 05 Nov 2020 01:37:33:  7000000 \\nINFO  @ Thu, 05 Nov 2020 01:37:38:  8000000 \\nINFO  @ Thu, 05 Nov 2020 01:37:43:  9000000 \\nINFO  @ Thu, 05 Nov 2020 01:37:48:  10000000 \\nINFO  @ Thu, 05 Nov 2020 01:37:52:  11000000 \\nINFO  @ Thu, 05 Nov 2020 01:37:57:  12000000 \\nINFO  @ Thu, 05 Nov 2020 01:38:02:  13000000 \\nINFO  @ Thu, 05 Nov 2020 01:38:07:  14000000 \\nINFO  @ Thu, 05 Nov 2020 01:38:12:  15000000 \\nINFO  @ Thu, 05 Nov 2020 01:38:17:  16000000 \\nINFO  @ Thu, 05 Nov 2020 01:38:22:  17000000 \\nINFO  @ Thu, 05 Nov 2020 01:38:27:  18000000 \\nINFO  @ Thu, 05 Nov 2020 01:38:32:  19000000 \\nINFO  @ Thu, 05 Nov 2020 01:38:36:  20000000 \\nINFO  @ Thu, 05 Nov 2020 01:38:41:  21000000 \\nINFO  @ Thu, 05 Nov 2020 01:38:46:  22000000 \\nINFO  @ Thu, 05 Nov 2020 01:38:51:  23000000 \\nINFO  @ Thu, 05 Nov 2020 01:38:56:  24000000 \\nINFO  @ Thu, 05 Nov 2020 01:38:59: 24566660 fragments have been read. \\nINFO  @ Thu, 05 Nov 2020 01:39:13: #1.2 read input fragments... \\nINFO  @ Thu, 05 Nov 2020 01:39:17:  1000000 \\nINFO  @ Thu, 05 Nov 2020 01:39:20:  2000000 \\nINFO  @ Thu, 05 Nov 2020 01:39:24:  3000000 \\nINFO  @ Thu, 05 Nov 2020 01:39:28:  4000000 \\nINFO  @ Thu, 05 Nov 2020 01:39:32:  5000000 \\nINFO  @ Thu, 05 Nov 2020 01:39:35:  6000000 \\nINFO  @ Thu, 05 Nov 2020 01:39:40:  7000000 \\nINFO  @ Thu, 05 Nov 2020 01:39:43:  8000000 \\nINFO  @ Thu, 05 Nov 2020 01:39:47:  9000000 \\nINFO  @ Thu, 05 Nov 2020 01:39:51:  10000000 \\nINFO  @ Thu, 05 Nov 2020 01:39:55:  11000000 \\nINFO  @ Thu, 05 Nov 2020 01:39:58:  12000000 \\nINFO  @ Thu, 05 Nov 2020 01:40:02:  13000000 \\nINFO  @ Thu, 05 Nov 2020 01:40:06:  14000000 \\nINFO  @ Thu, 05 Nov 2020 01:40:10:  15000000 \\nINFO  @ Thu, 05 Nov 2020 01:40:13:  16000000 \\nINFO  @ Thu, 05 Nov 2020 01:40:17:  17000000 \\nINFO  @ Thu, 05 Nov 2020 01:40:21:  18000000 \\nINFO  @ Thu, 05 Nov 2020 01:40:24:  19000000 \\nINFO  @ Thu, 05 Nov 2020 01:40:28:  20000000 \\nINFO  @ Thu, 05 Nov 2020 01:40:32:  21000000 \\nINFO  @ Thu, 05 Nov 2020 01:40:36:  22000000 \\nINFO  @ Thu, 05 Nov 2020 01:40:39:  23000000 \\nINFO  @ Thu, 05 Nov 2020 01:40:43:  24000000 \\nINFO  @ Thu, 05 Nov 2020 01:40:47:  25000000 \\nINFO  @ Thu, 05 Nov 2020 01:40:51:  26000000 \\nINFO  @ Thu, 05 Nov 2020 01:40:54:  27000000 \\nINFO  @ Thu, 05 Nov 2020 01:40:59:  28000000 \\nINFO  @ Thu, 05 Nov 2020 01:41:02:  29000000 \\nINFO  @ Thu, 05 Nov 2020 01:41:06:  30000000 \\nINFO  @ Thu, 05 Nov 2020 01:41:10:  31000000 \\nINFO  @ Thu, 05 Nov 2020 01:41:14:  32000000 \\nINFO  @ Thu, 05 Nov 2020 01:41:18:  33000000 \\nINFO  @ Thu, 05 Nov 2020 01:41:23:  34000000 \\nINFO  @ Thu, 05 Nov 2020 01:41:26:  35000000 \\nINFO  @ Thu, 05 Nov 2020 01:41:30:  36000000 \\nINFO  @ Thu, 05 Nov 2020 01:41:34:  37000000 \\nINFO  @ Thu, 05 Nov 2020 01:41:37:  38000000 \\nINFO  @ Thu, 05 Nov 2020 01:41:41:  39000000 \\nINFO  @ Thu, 05 Nov 2020 01:41:45:  40000000 \\nINFO  @ Thu, 05 Nov 2020 01:41:49:  41000000 \\nINFO  @ Thu, 05 Nov 2020 01:41:53:  42000000 \\nINFO  @ Thu, 05 Nov 2020 01:41:56:  43000000 \\nINFO  @ Thu, 05 Nov 2020 01:42:00:  44000000 \\nINFO  @ Thu, 05 Nov 2020 01:42:04:  45000000 \\nINFO  @ Thu, 05 Nov 2020 01:42:08:  46000000 \\nINFO  @ Thu, 05 Nov 2020 01:42:12:  47000000 \\nINFO  @ Thu, 05 Nov 2020 01:42:16:  48000000 \\nINFO  @ Thu, 05 Nov 2020 01:42:20:  49000000 \\nINFO  @ Thu, 05 Nov 2020 01:42:23:  50000000 \\nINFO  @ Thu, 05 Nov 2020 01:42:27:  51000000 \\nINFO  @ Thu, 05 Nov 2020 01:42:31:  52000000 \\nINFO  @ Thu, 05 Nov 2020 01:42:35:  53000000 \\nINFO  @ Thu, 05 Nov 2020 01:42:38:  54000000 \\nINFO  @ Thu, 05 Nov 2020 01:42:42:  55000000 \\nINFO  @ Thu, 05 Nov 2020 01:42:46:  56000000 \\nINFO  @ Thu, 05 Nov 2020 01:42:50:  57000000 \\nINFO  @ Thu, 05 Nov 2020 01:42:54:  58000000 \\nINFO  @ Thu, 05 Nov 2020 01:42:57:  59000000 \\nINFO  @ Thu, 05 Nov 2020 01:43:01:  60000000 \\nINFO  @ Thu, 05 Nov 2020 01:43:05:  61000000 \\nINFO  @ Thu, 05 Nov 2020 01:43:09:  62000000 \\nINFO  @ Thu, 05 Nov 2020 01:43:12:  63000000 \\nINFO  @ Thu, 05 Nov 2020 01:43:16:  64000000 \\nINFO  @ Thu, 05 Nov 2020 01:43:20:  65000000 \\nINFO  @ Thu, 05 Nov 2020 01:43:24:  66000000 \\nINFO  @ Thu, 05 Nov 2020 01:43:26: 66660890 fragments have been read. \\nINFO  @ Thu, 05 Nov 2020 01:44:11: #1 mean fragment size is determined as 233.4 bp from treatment \\nINFO  @ Thu, 05 Nov 2020 01:44:11: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Thu, 05 Nov 2020 01:44:11: #1 fragment size = 233.4 \\nINFO  @ Thu, 05 Nov 2020 01:44:11: #1  total fragments in treatment: 24566660 \\nINFO  @ Thu, 05 Nov 2020 01:44:11: #1 user defined the maximum fragments... \\nINFO  @ Thu, 05 Nov 2020 01:44:11: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Thu, 05 Nov 2020 01:44:50: #1  fragments after filtering in treatment: 24558248 \\nINFO  @ Thu, 05 Nov 2020 01:44:50: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Thu, 05 Nov 2020 01:44:50: #1  total fragments in control: 66660890 \\nINFO  @ Thu, 05 Nov 2020 01:44:50: #1 user defined the maximum fragments... \\nINFO  @ Thu, 05 Nov 2020 01:44:50: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Thu, 05 Nov 2020 01:46:36: #1  fragments after filtering in control: 66633066 \\nINFO  @ Thu, 05 Nov 2020 01:46:36: #1  Redundant rate of control: 0.00 \\nINFO  @ Thu, 05 Nov 2020 01:46:36: #1 finished! \\nINFO  @ Thu, 05 Nov 2020 01:46:36: #2 Build Peak Model... \\nINFO  @ Thu, 05 Nov 2020 01:46:36: #2 Skipped... \\nINFO  @ Thu, 05 Nov 2020 01:46:36: #3 Call peaks... \\nINFO  @ Thu, 05 Nov 2020 01:46:36: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 05 Nov 2020 01:56:08: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 05 Nov 2020 01:56:08: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_IRF8_NT_VHL-MED1_R1_treat_pileup.bdg \\nINFO  @ Thu, 05 Nov 2020 01:56:08: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_IRF8_NT_VHL-MED1_R1_control_lambda.bdg \\nINFO  @ Thu, 05 Nov 2020 01:56:08: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 05 Nov 2020 01:56:08: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 05 Nov 2020 02:02:34: #4 Write output xls file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MED1_R1_peaks.xls \\nINFO  @ Thu, 05 Nov 2020 02:02:34: #4 Write peak in narrowPeak format file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MED1_R1_peaks.narrowPeak \\nINFO  @ Thu, 05 Nov 2020 02:02:34: #4 Write summits bed file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MED1_R1_summits.bed \\nINFO  @ Thu, 05 Nov 2020 02:02:34: Done! \\n'\n",
      "35135377 24558248\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MED1_R1_treat_pileup.bdg --c1 ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MED1_R1_control_lambda.bdg --t2 ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MED1_R1_treat_pileup.bdg --c2 ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MED1_R1_control_lambda.bdg --d1 35135377 --d2 24558248 -g 60 -l 212 --o-prefix MV411_IRF8_NT_DMSO-MED1_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MED1_R1_treat_pileup --outdir ../../data/chipseq_IRF8_degraded/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Thu, 05 Nov 2020 02:02:35: Read and build treatment 1 bedGraph... \\nINFO  @ Thu, 05 Nov 2020 02:03:16: Read and build control 1 bedGraph... \\nINFO  @ Thu, 05 Nov 2020 02:07:01: Read and build treatment 2 bedGraph... \\nINFO  @ Thu, 05 Nov 2020 02:07:49: Read and build control 2 bedGraph... \\nINFO  @ Thu, 05 Nov 2020 02:20:00: Write peaks... \\nINFO  @ Thu, 05 Nov 2020 02:20:00: Done \\n')\n",
      "../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MEF2C_R1.mLb.clN.sorted.bam ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MEF2C_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MEF2C_R1.mLb.clN.sorted.bam and ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MEF2C_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "204\n",
      "computing the scaling values\n",
      "b'INFO  @ Thu, 05 Nov 2020 02:21:32: \\n# Command line: callpeak -B -t ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MEF2C_R1.mLb.clN.sorted.bam -c ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 204 -n MV411_IRF8_NT_DMSO-MEF2C_R1 --outdir ../../data/chipseq_IRF8_degraded/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_IRF8_NT_DMSO-MEF2C_R1\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MEF2C_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Thu, 05 Nov 2020 02:21:32: #1 read fragment files... \\nINFO  @ Thu, 05 Nov 2020 02:21:32: #1 read treatment fragments... \\nINFO  @ Thu, 05 Nov 2020 02:21:36:  1000000 \\nINFO  @ Thu, 05 Nov 2020 02:21:41:  2000000 \\nINFO  @ Thu, 05 Nov 2020 02:21:46:  3000000 \\nINFO  @ Thu, 05 Nov 2020 02:21:51:  4000000 \\nINFO  @ Thu, 05 Nov 2020 02:21:57:  5000000 \\nINFO  @ Thu, 05 Nov 2020 02:22:01:  6000000 \\nINFO  @ Thu, 05 Nov 2020 02:22:06:  7000000 \\nINFO  @ Thu, 05 Nov 2020 02:22:11:  8000000 \\nINFO  @ Thu, 05 Nov 2020 02:22:15:  9000000 \\nINFO  @ Thu, 05 Nov 2020 02:22:20:  10000000 \\nINFO  @ Thu, 05 Nov 2020 02:22:25:  11000000 \\nINFO  @ Thu, 05 Nov 2020 02:22:31:  12000000 \\nINFO  @ Thu, 05 Nov 2020 02:22:36:  13000000 \\nINFO  @ Thu, 05 Nov 2020 02:22:41:  14000000 \\nINFO  @ Thu, 05 Nov 2020 02:22:46:  15000000 \\nINFO  @ Thu, 05 Nov 2020 02:22:50:  16000000 \\nINFO  @ Thu, 05 Nov 2020 02:22:55:  17000000 \\nINFO  @ Thu, 05 Nov 2020 02:22:58: 17547907 fragments have been read. \\nINFO  @ Thu, 05 Nov 2020 02:23:07: #1.2 read input fragments... \\nINFO  @ Thu, 05 Nov 2020 02:23:11:  1000000 \\nINFO  @ Thu, 05 Nov 2020 02:23:15:  2000000 \\nINFO  @ Thu, 05 Nov 2020 02:23:19:  3000000 \\nINFO  @ Thu, 05 Nov 2020 02:23:22:  4000000 \\nINFO  @ Thu, 05 Nov 2020 02:23:26:  5000000 \\nINFO  @ Thu, 05 Nov 2020 02:23:29:  6000000 \\nINFO  @ Thu, 05 Nov 2020 02:23:33:  7000000 \\nINFO  @ Thu, 05 Nov 2020 02:23:37:  8000000 \\nINFO  @ Thu, 05 Nov 2020 02:23:41:  9000000 \\nINFO  @ Thu, 05 Nov 2020 02:23:45:  10000000 \\nINFO  @ Thu, 05 Nov 2020 02:23:48:  11000000 \\nINFO  @ Thu, 05 Nov 2020 02:23:52:  12000000 \\nINFO  @ Thu, 05 Nov 2020 02:23:56:  13000000 \\nINFO  @ Thu, 05 Nov 2020 02:23:59:  14000000 \\nINFO  @ Thu, 05 Nov 2020 02:24:03:  15000000 \\nINFO  @ Thu, 05 Nov 2020 02:24:07:  16000000 \\nINFO  @ Thu, 05 Nov 2020 02:24:11:  17000000 \\nINFO  @ Thu, 05 Nov 2020 02:24:14:  18000000 \\nINFO  @ Thu, 05 Nov 2020 02:24:18:  19000000 \\nINFO  @ Thu, 05 Nov 2020 02:24:22:  20000000 \\nINFO  @ Thu, 05 Nov 2020 02:24:25:  21000000 \\nINFO  @ Thu, 05 Nov 2020 02:24:29:  22000000 \\nINFO  @ Thu, 05 Nov 2020 02:24:33:  23000000 \\nINFO  @ Thu, 05 Nov 2020 02:24:36:  24000000 \\nINFO  @ Thu, 05 Nov 2020 02:24:40:  25000000 \\nINFO  @ Thu, 05 Nov 2020 02:24:44:  26000000 \\nINFO  @ Thu, 05 Nov 2020 02:24:47:  27000000 \\nINFO  @ Thu, 05 Nov 2020 02:24:51:  28000000 \\nINFO  @ Thu, 05 Nov 2020 02:24:55:  29000000 \\nINFO  @ Thu, 05 Nov 2020 02:24:59:  30000000 \\nINFO  @ Thu, 05 Nov 2020 02:25:02:  31000000 \\nINFO  @ Thu, 05 Nov 2020 02:25:06:  32000000 \\nINFO  @ Thu, 05 Nov 2020 02:25:10:  33000000 \\nINFO  @ Thu, 05 Nov 2020 02:25:14:  34000000 \\nINFO  @ Thu, 05 Nov 2020 02:25:17:  35000000 \\nINFO  @ Thu, 05 Nov 2020 02:25:21:  36000000 \\nINFO  @ Thu, 05 Nov 2020 02:25:25:  37000000 \\nINFO  @ Thu, 05 Nov 2020 02:25:28:  38000000 \\nINFO  @ Thu, 05 Nov 2020 02:25:32:  39000000 \\nINFO  @ Thu, 05 Nov 2020 02:25:36:  40000000 \\nINFO  @ Thu, 05 Nov 2020 02:25:40:  41000000 \\nINFO  @ Thu, 05 Nov 2020 02:25:44:  42000000 \\nINFO  @ Thu, 05 Nov 2020 02:25:47:  43000000 \\nINFO  @ Thu, 05 Nov 2020 02:25:51:  44000000 \\nINFO  @ Thu, 05 Nov 2020 02:25:55:  45000000 \\nINFO  @ Thu, 05 Nov 2020 02:25:59:  46000000 \\nINFO  @ Thu, 05 Nov 2020 02:26:02:  47000000 \\nINFO  @ Thu, 05 Nov 2020 02:26:06:  48000000 \\nINFO  @ Thu, 05 Nov 2020 02:26:10:  49000000 \\nINFO  @ Thu, 05 Nov 2020 02:26:13:  50000000 \\nINFO  @ Thu, 05 Nov 2020 02:26:17:  51000000 \\nINFO  @ Thu, 05 Nov 2020 02:26:21:  52000000 \\nINFO  @ Thu, 05 Nov 2020 02:26:25:  53000000 \\nINFO  @ Thu, 05 Nov 2020 02:26:28:  54000000 \\nINFO  @ Thu, 05 Nov 2020 02:26:32:  55000000 \\nINFO  @ Thu, 05 Nov 2020 02:26:36:  56000000 \\nINFO  @ Thu, 05 Nov 2020 02:26:40:  57000000 \\nINFO  @ Thu, 05 Nov 2020 02:26:43:  58000000 \\nINFO  @ Thu, 05 Nov 2020 02:26:47:  59000000 \\nINFO  @ Thu, 05 Nov 2020 02:26:51:  60000000 \\nINFO  @ Thu, 05 Nov 2020 02:26:54:  61000000 \\nINFO  @ Thu, 05 Nov 2020 02:26:58:  62000000 \\nINFO  @ Thu, 05 Nov 2020 02:27:02:  63000000 \\nINFO  @ Thu, 05 Nov 2020 02:27:05:  64000000 \\nINFO  @ Thu, 05 Nov 2020 02:27:09:  65000000 \\nINFO  @ Thu, 05 Nov 2020 02:27:13:  66000000 \\nINFO  @ Thu, 05 Nov 2020 02:27:16: 66660890 fragments have been read. \\nINFO  @ Thu, 05 Nov 2020 02:28:00: #1 mean fragment size is determined as 233.8 bp from treatment \\nINFO  @ Thu, 05 Nov 2020 02:28:00: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Thu, 05 Nov 2020 02:28:00: #1 fragment size = 233.8 \\nINFO  @ Thu, 05 Nov 2020 02:28:00: #1  total fragments in treatment: 17547907 \\nINFO  @ Thu, 05 Nov 2020 02:28:00: #1 user defined the maximum fragments... \\nINFO  @ Thu, 05 Nov 2020 02:28:00: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Thu, 05 Nov 2020 02:28:29: #1  fragments after filtering in treatment: 17539361 \\nINFO  @ Thu, 05 Nov 2020 02:28:29: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Thu, 05 Nov 2020 02:28:29: #1  total fragments in control: 66660890 \\nINFO  @ Thu, 05 Nov 2020 02:28:29: #1 user defined the maximum fragments... \\nINFO  @ Thu, 05 Nov 2020 02:28:29: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Thu, 05 Nov 2020 02:30:24: #1  fragments after filtering in control: 66633066 \\nINFO  @ Thu, 05 Nov 2020 02:30:24: #1  Redundant rate of control: 0.00 \\nINFO  @ Thu, 05 Nov 2020 02:30:24: #1 finished! \\nINFO  @ Thu, 05 Nov 2020 02:30:24: #2 Build Peak Model... \\nINFO  @ Thu, 05 Nov 2020 02:30:24: #2 Skipped... \\nINFO  @ Thu, 05 Nov 2020 02:30:24: #3 Call peaks... \\nINFO  @ Thu, 05 Nov 2020 02:30:24: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 05 Nov 2020 02:39:54: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 05 Nov 2020 02:39:54: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_IRF8_NT_DMSO-MEF2C_R1_treat_pileup.bdg \\nINFO  @ Thu, 05 Nov 2020 02:39:54: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_IRF8_NT_DMSO-MEF2C_R1_control_lambda.bdg \\nINFO  @ Thu, 05 Nov 2020 02:39:54: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 05 Nov 2020 02:39:54: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 05 Nov 2020 02:46:17: #4 Write output xls file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MEF2C_R1_peaks.xls \\nINFO  @ Thu, 05 Nov 2020 02:46:17: #4 Write peak in narrowPeak format file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MEF2C_R1_peaks.narrowPeak \\nINFO  @ Thu, 05 Nov 2020 02:46:17: #4 Write summits bed file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MEF2C_R1_summits.bed \\nINFO  @ Thu, 05 Nov 2020 02:46:17: Done! \\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'INFO  @ Thu, 05 Nov 2020 02:46:19: \\n# Command line: callpeak -B -t ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MEF2C_R1.mLb.clN.sorted.bam -c ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 204 -n MV411_IRF8_NT_VHL-MEF2C_R1 --outdir ../../data/chipseq_IRF8_degraded/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_IRF8_NT_VHL-MEF2C_R1\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MEF2C_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Thu, 05 Nov 2020 02:46:19: #1 read fragment files... \\nINFO  @ Thu, 05 Nov 2020 02:46:19: #1 read treatment fragments... \\nINFO  @ Thu, 05 Nov 2020 02:46:24:  1000000 \\nINFO  @ Thu, 05 Nov 2020 02:46:30:  2000000 \\nINFO  @ Thu, 05 Nov 2020 02:46:35:  3000000 \\nINFO  @ Thu, 05 Nov 2020 02:46:40:  4000000 \\nINFO  @ Thu, 05 Nov 2020 02:46:45:  5000000 \\nINFO  @ Thu, 05 Nov 2020 02:46:50:  6000000 \\nINFO  @ Thu, 05 Nov 2020 02:46:55:  7000000 \\nINFO  @ Thu, 05 Nov 2020 02:47:00:  8000000 \\nINFO  @ Thu, 05 Nov 2020 02:47:05:  9000000 \\nINFO  @ Thu, 05 Nov 2020 02:47:10:  10000000 \\nINFO  @ Thu, 05 Nov 2020 02:47:15:  11000000 \\nINFO  @ Thu, 05 Nov 2020 02:47:20:  12000000 \\nINFO  @ Thu, 05 Nov 2020 02:47:25:  13000000 \\nINFO  @ Thu, 05 Nov 2020 02:47:30:  14000000 \\nINFO  @ Thu, 05 Nov 2020 02:47:35:  15000000 \\nINFO  @ Thu, 05 Nov 2020 02:47:40:  16000000 \\nINFO  @ Thu, 05 Nov 2020 02:47:45:  17000000 \\nINFO  @ Thu, 05 Nov 2020 02:47:47: 17440748 fragments have been read. \\nINFO  @ Thu, 05 Nov 2020 02:47:56: #1.2 read input fragments... \\nINFO  @ Thu, 05 Nov 2020 02:48:00:  1000000 \\nINFO  @ Thu, 05 Nov 2020 02:48:04:  2000000 \\nINFO  @ Thu, 05 Nov 2020 02:48:08:  3000000 \\nINFO  @ Thu, 05 Nov 2020 02:48:12:  4000000 \\nINFO  @ Thu, 05 Nov 2020 02:48:16:  5000000 \\nINFO  @ Thu, 05 Nov 2020 02:48:20:  6000000 \\nINFO  @ Thu, 05 Nov 2020 02:48:24:  7000000 \\nINFO  @ Thu, 05 Nov 2020 02:48:27:  8000000 \\nINFO  @ Thu, 05 Nov 2020 02:48:31:  9000000 \\nINFO  @ Thu, 05 Nov 2020 02:48:35:  10000000 \\nINFO  @ Thu, 05 Nov 2020 02:48:39:  11000000 \\nINFO  @ Thu, 05 Nov 2020 02:48:43:  12000000 \\nINFO  @ Thu, 05 Nov 2020 02:48:47:  13000000 \\nINFO  @ Thu, 05 Nov 2020 02:48:50:  14000000 \\nINFO  @ Thu, 05 Nov 2020 02:48:54:  15000000 \\nINFO  @ Thu, 05 Nov 2020 02:48:58:  16000000 \\nINFO  @ Thu, 05 Nov 2020 02:49:02:  17000000 \\nINFO  @ Thu, 05 Nov 2020 02:49:05:  18000000 \\nINFO  @ Thu, 05 Nov 2020 02:49:09:  19000000 \\nINFO  @ Thu, 05 Nov 2020 02:49:13:  20000000 \\nINFO  @ Thu, 05 Nov 2020 02:49:17:  21000000 \\nINFO  @ Thu, 05 Nov 2020 02:49:21:  22000000 \\nINFO  @ Thu, 05 Nov 2020 02:49:25:  23000000 \\nINFO  @ Thu, 05 Nov 2020 02:49:28:  24000000 \\nINFO  @ Thu, 05 Nov 2020 02:49:32:  25000000 \\nINFO  @ Thu, 05 Nov 2020 02:49:36:  26000000 \\nINFO  @ Thu, 05 Nov 2020 02:49:40:  27000000 \\nINFO  @ Thu, 05 Nov 2020 02:49:44:  28000000 \\nINFO  @ Thu, 05 Nov 2020 02:49:48:  29000000 \\nINFO  @ Thu, 05 Nov 2020 02:49:51:  30000000 \\nINFO  @ Thu, 05 Nov 2020 02:49:55:  31000000 \\nINFO  @ Thu, 05 Nov 2020 02:49:59:  32000000 \\nINFO  @ Thu, 05 Nov 2020 02:50:03:  33000000 \\nINFO  @ Thu, 05 Nov 2020 02:50:07:  34000000 \\nINFO  @ Thu, 05 Nov 2020 02:50:11:  35000000 \\nINFO  @ Thu, 05 Nov 2020 02:50:14:  36000000 \\nINFO  @ Thu, 05 Nov 2020 02:50:18:  37000000 \\nINFO  @ Thu, 05 Nov 2020 02:50:22:  38000000 \\nINFO  @ Thu, 05 Nov 2020 02:50:26:  39000000 \\nINFO  @ Thu, 05 Nov 2020 02:50:30:  40000000 \\nINFO  @ Thu, 05 Nov 2020 02:50:34:  41000000 \\nINFO  @ Thu, 05 Nov 2020 02:50:38:  42000000 \\nINFO  @ Thu, 05 Nov 2020 02:50:41:  43000000 \\nINFO  @ Thu, 05 Nov 2020 02:50:46:  44000000 \\nINFO  @ Thu, 05 Nov 2020 02:50:49:  45000000 \\nINFO  @ Thu, 05 Nov 2020 02:50:53:  46000000 \\nINFO  @ Thu, 05 Nov 2020 02:50:57:  47000000 \\nINFO  @ Thu, 05 Nov 2020 02:51:01:  48000000 \\nINFO  @ Thu, 05 Nov 2020 02:51:05:  49000000 \\nINFO  @ Thu, 05 Nov 2020 02:51:09:  50000000 \\nINFO  @ Thu, 05 Nov 2020 02:51:13:  51000000 \\nINFO  @ Thu, 05 Nov 2020 02:51:16:  52000000 \\nINFO  @ Thu, 05 Nov 2020 02:51:20:  53000000 \\nINFO  @ Thu, 05 Nov 2020 02:51:24:  54000000 \\nINFO  @ Thu, 05 Nov 2020 02:51:28:  55000000 \\nINFO  @ Thu, 05 Nov 2020 02:51:32:  56000000 \\nINFO  @ Thu, 05 Nov 2020 02:51:36:  57000000 \\nINFO  @ Thu, 05 Nov 2020 02:51:40:  58000000 \\nINFO  @ Thu, 05 Nov 2020 02:51:44:  59000000 \\nINFO  @ Thu, 05 Nov 2020 02:51:48:  60000000 \\nINFO  @ Thu, 05 Nov 2020 02:51:52:  61000000 \\nINFO  @ Thu, 05 Nov 2020 02:51:56:  62000000 \\nINFO  @ Thu, 05 Nov 2020 02:52:00:  63000000 \\nINFO  @ Thu, 05 Nov 2020 02:52:04:  64000000 \\nINFO  @ Thu, 05 Nov 2020 02:52:07:  65000000 \\nINFO  @ Thu, 05 Nov 2020 02:52:11:  66000000 \\nINFO  @ Thu, 05 Nov 2020 02:52:14: 66660890 fragments have been read. \\nINFO  @ Thu, 05 Nov 2020 02:52:59: #1 mean fragment size is determined as 244.8 bp from treatment \\nINFO  @ Thu, 05 Nov 2020 02:52:59: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Thu, 05 Nov 2020 02:52:59: #1 fragment size = 244.8 \\nINFO  @ Thu, 05 Nov 2020 02:52:59: #1  total fragments in treatment: 17440748 \\nINFO  @ Thu, 05 Nov 2020 02:52:59: #1 user defined the maximum fragments... \\nINFO  @ Thu, 05 Nov 2020 02:52:59: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Thu, 05 Nov 2020 02:53:28: #1  fragments after filtering in treatment: 17430330 \\nINFO  @ Thu, 05 Nov 2020 02:53:28: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Thu, 05 Nov 2020 02:53:28: #1  total fragments in control: 66660890 \\nINFO  @ Thu, 05 Nov 2020 02:53:28: #1 user defined the maximum fragments... \\nINFO  @ Thu, 05 Nov 2020 02:53:28: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Thu, 05 Nov 2020 02:55:19: #1  fragments after filtering in control: 66633066 \\nINFO  @ Thu, 05 Nov 2020 02:55:19: #1  Redundant rate of control: 0.00 \\nINFO  @ Thu, 05 Nov 2020 02:55:19: #1 finished! \\nINFO  @ Thu, 05 Nov 2020 02:55:19: #2 Build Peak Model... \\nINFO  @ Thu, 05 Nov 2020 02:55:19: #2 Skipped... \\nINFO  @ Thu, 05 Nov 2020 02:55:19: #3 Call peaks... \\nINFO  @ Thu, 05 Nov 2020 02:55:19: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 05 Nov 2020 03:04:48: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 05 Nov 2020 03:04:48: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_IRF8_NT_VHL-MEF2C_R1_treat_pileup.bdg \\nINFO  @ Thu, 05 Nov 2020 03:04:48: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_IRF8_NT_VHL-MEF2C_R1_control_lambda.bdg \\nINFO  @ Thu, 05 Nov 2020 03:04:48: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 05 Nov 2020 03:04:48: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 05 Nov 2020 03:10:44: #4 Write output xls file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MEF2C_R1_peaks.xls \\nINFO  @ Thu, 05 Nov 2020 03:10:44: #4 Write peak in narrowPeak format file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MEF2C_R1_peaks.narrowPeak \\nINFO  @ Thu, 05 Nov 2020 03:10:44: #4 Write summits bed file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MEF2C_R1_summits.bed \\nINFO  @ Thu, 05 Nov 2020 03:10:44: Done! \\n'\n",
      "22778390 17430330\n",
      "doing differential peak binding\n",
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MEF2C_R1_treat_pileup.bdg --c1 ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MEF2C_R1_control_lambda.bdg --t2 ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MEF2C_R1_treat_pileup.bdg --c2 ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MEF2C_R1_control_lambda.bdg --d1 22778390 --d2 17430330 -g 60 -l 204 --o-prefix MV411_IRF8_NT_DMSO-MEF2C_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2C_R1_treat_pileup --outdir ../../data/chipseq_IRF8_degraded/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Thu, 05 Nov 2020 03:10:44: Read and build treatment 1 bedGraph... \\nINFO  @ Thu, 05 Nov 2020 03:11:20: Read and build control 1 bedGraph... \\nINFO  @ Thu, 05 Nov 2020 03:15:04: Read and build treatment 2 bedGraph... \\nINFO  @ Thu, 05 Nov 2020 03:15:38: Read and build control 2 bedGraph... \\nINFO  @ Thu, 05 Nov 2020 03:27:01: Write peaks... \\nINFO  @ Thu, 05 Nov 2020 03:27:01: Done \\n')\n",
      "../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MEF2D_R1.mLb.clN.sorted.bam ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MEF2D_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MEF2D_R1.mLb.clN.sorted.bam and ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MEF2D_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "184\n",
      "computing the scaling values\n",
      "b'INFO  @ Thu, 05 Nov 2020 03:28:34: \\n# Command line: callpeak -B -t ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MEF2D_R1.mLb.clN.sorted.bam -c ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 184 -n MV411_IRF8_NT_DMSO-MEF2D_R1 --outdir ../../data/chipseq_IRF8_degraded/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_IRF8_NT_DMSO-MEF2D_R1\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MEF2D_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Thu, 05 Nov 2020 03:28:34: #1 read fragment files... \\nINFO  @ Thu, 05 Nov 2020 03:28:34: #1 read treatment fragments... \\nINFO  @ Thu, 05 Nov 2020 03:28:39:  1000000 \\nINFO  @ Thu, 05 Nov 2020 03:28:44:  2000000 \\nINFO  @ Thu, 05 Nov 2020 03:28:48:  3000000 \\nINFO  @ Thu, 05 Nov 2020 03:28:53:  4000000 \\nINFO  @ Thu, 05 Nov 2020 03:28:58:  5000000 \\nINFO  @ Thu, 05 Nov 2020 03:29:02:  6000000 \\nINFO  @ Thu, 05 Nov 2020 03:29:07:  7000000 \\nINFO  @ Thu, 05 Nov 2020 03:29:12:  8000000 \\nINFO  @ Thu, 05 Nov 2020 03:29:16:  9000000 \\nINFO  @ Thu, 05 Nov 2020 03:29:21:  10000000 \\nINFO  @ Thu, 05 Nov 2020 03:29:26:  11000000 \\nINFO  @ Thu, 05 Nov 2020 03:29:30:  12000000 \\nINFO  @ Thu, 05 Nov 2020 03:29:35:  13000000 \\nINFO  @ Thu, 05 Nov 2020 03:29:40:  14000000 \\nINFO  @ Thu, 05 Nov 2020 03:29:45:  15000000 \\nINFO  @ Thu, 05 Nov 2020 03:29:49:  16000000 \\nINFO  @ Thu, 05 Nov 2020 03:29:54:  17000000 \\nINFO  @ Thu, 05 Nov 2020 03:29:59:  18000000 \\nINFO  @ Thu, 05 Nov 2020 03:29:59: 18046674 fragments have been read. \\nINFO  @ Thu, 05 Nov 2020 03:30:09: #1.2 read input fragments... \\nINFO  @ Thu, 05 Nov 2020 03:30:12:  1000000 \\nINFO  @ Thu, 05 Nov 2020 03:30:17:  2000000 \\nINFO  @ Thu, 05 Nov 2020 03:30:21:  3000000 \\nINFO  @ Thu, 05 Nov 2020 03:30:25:  4000000 \\nINFO  @ Thu, 05 Nov 2020 03:30:28:  5000000 \\nINFO  @ Thu, 05 Nov 2020 03:30:32:  6000000 \\nINFO  @ Thu, 05 Nov 2020 03:30:36:  7000000 \\nINFO  @ Thu, 05 Nov 2020 03:30:39:  8000000 \\nINFO  @ Thu, 05 Nov 2020 03:30:43:  9000000 \\nINFO  @ Thu, 05 Nov 2020 03:30:47:  10000000 \\nINFO  @ Thu, 05 Nov 2020 03:30:50:  11000000 \\nINFO  @ Thu, 05 Nov 2020 03:30:54:  12000000 \\nINFO  @ Thu, 05 Nov 2020 03:30:58:  13000000 \\nINFO  @ Thu, 05 Nov 2020 03:31:02:  14000000 \\nINFO  @ Thu, 05 Nov 2020 03:31:05:  15000000 \\nINFO  @ Thu, 05 Nov 2020 03:31:09:  16000000 \\nINFO  @ Thu, 05 Nov 2020 03:31:13:  17000000 \\nINFO  @ Thu, 05 Nov 2020 03:31:16:  18000000 \\nINFO  @ Thu, 05 Nov 2020 03:31:20:  19000000 \\nINFO  @ Thu, 05 Nov 2020 03:31:24:  20000000 \\nINFO  @ Thu, 05 Nov 2020 03:31:28:  21000000 \\nINFO  @ Thu, 05 Nov 2020 03:31:31:  22000000 \\nINFO  @ Thu, 05 Nov 2020 03:31:35:  23000000 \\nINFO  @ Thu, 05 Nov 2020 03:31:39:  24000000 \\nINFO  @ Thu, 05 Nov 2020 03:31:43:  25000000 \\nINFO  @ Thu, 05 Nov 2020 03:31:46:  26000000 \\nINFO  @ Thu, 05 Nov 2020 03:31:50:  27000000 \\nINFO  @ Thu, 05 Nov 2020 03:31:54:  28000000 \\nINFO  @ Thu, 05 Nov 2020 03:31:57:  29000000 \\nINFO  @ Thu, 05 Nov 2020 03:32:01:  30000000 \\nINFO  @ Thu, 05 Nov 2020 03:32:05:  31000000 \\nINFO  @ Thu, 05 Nov 2020 03:32:08:  32000000 \\nINFO  @ Thu, 05 Nov 2020 03:32:12:  33000000 \\nINFO  @ Thu, 05 Nov 2020 03:32:16:  34000000 \\nINFO  @ Thu, 05 Nov 2020 03:32:19:  35000000 \\nINFO  @ Thu, 05 Nov 2020 03:32:23:  36000000 \\nINFO  @ Thu, 05 Nov 2020 03:32:26:  37000000 \\nINFO  @ Thu, 05 Nov 2020 03:32:30:  38000000 \\nINFO  @ Thu, 05 Nov 2020 03:32:34:  39000000 \\nINFO  @ Thu, 05 Nov 2020 03:32:38:  40000000 \\nINFO  @ Thu, 05 Nov 2020 03:32:41:  41000000 \\nINFO  @ Thu, 05 Nov 2020 03:32:45:  42000000 \\nINFO  @ Thu, 05 Nov 2020 03:32:49:  43000000 \\nINFO  @ Thu, 05 Nov 2020 03:32:52:  44000000 \\nINFO  @ Thu, 05 Nov 2020 03:32:56:  45000000 \\nINFO  @ Thu, 05 Nov 2020 03:33:00:  46000000 \\nINFO  @ Thu, 05 Nov 2020 03:33:03:  47000000 \\nINFO  @ Thu, 05 Nov 2020 03:33:07:  48000000 \\nINFO  @ Thu, 05 Nov 2020 03:33:11:  49000000 \\nINFO  @ Thu, 05 Nov 2020 03:33:14:  50000000 \\nINFO  @ Thu, 05 Nov 2020 03:33:18:  51000000 \\nINFO  @ Thu, 05 Nov 2020 03:33:22:  52000000 \\nINFO  @ Thu, 05 Nov 2020 03:33:25:  53000000 \\nINFO  @ Thu, 05 Nov 2020 03:33:29:  54000000 \\nINFO  @ Thu, 05 Nov 2020 03:33:33:  55000000 \\nINFO  @ Thu, 05 Nov 2020 03:33:37:  56000000 \\nINFO  @ Thu, 05 Nov 2020 03:33:40:  57000000 \\nINFO  @ Thu, 05 Nov 2020 03:33:44:  58000000 \\nINFO  @ Thu, 05 Nov 2020 03:33:48:  59000000 \\nINFO  @ Thu, 05 Nov 2020 03:33:51:  60000000 \\nINFO  @ Thu, 05 Nov 2020 03:33:55:  61000000 \\nINFO  @ Thu, 05 Nov 2020 03:33:59:  62000000 \\nINFO  @ Thu, 05 Nov 2020 03:34:03:  63000000 \\nINFO  @ Thu, 05 Nov 2020 03:34:06:  64000000 \\nINFO  @ Thu, 05 Nov 2020 03:34:10:  65000000 \\nINFO  @ Thu, 05 Nov 2020 03:34:14:  66000000 \\nINFO  @ Thu, 05 Nov 2020 03:34:16: 66660890 fragments have been read. \\nINFO  @ Thu, 05 Nov 2020 03:35:02: #1 mean fragment size is determined as 222.8 bp from treatment \\nINFO  @ Thu, 05 Nov 2020 03:35:02: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Thu, 05 Nov 2020 03:35:02: #1 fragment size = 222.8 \\nINFO  @ Thu, 05 Nov 2020 03:35:02: #1  total fragments in treatment: 18046674 \\nINFO  @ Thu, 05 Nov 2020 03:35:02: #1 user defined the maximum fragments... \\nINFO  @ Thu, 05 Nov 2020 03:35:02: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Thu, 05 Nov 2020 03:35:32: #1  fragments after filtering in treatment: 18035712 \\nINFO  @ Thu, 05 Nov 2020 03:35:32: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Thu, 05 Nov 2020 03:35:32: #1  total fragments in control: 66660890 \\nINFO  @ Thu, 05 Nov 2020 03:35:32: #1 user defined the maximum fragments... \\nINFO  @ Thu, 05 Nov 2020 03:35:32: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Thu, 05 Nov 2020 03:37:18: #1  fragments after filtering in control: 66633066 \\nINFO  @ Thu, 05 Nov 2020 03:37:18: #1  Redundant rate of control: 0.00 \\nINFO  @ Thu, 05 Nov 2020 03:37:18: #1 finished! \\nINFO  @ Thu, 05 Nov 2020 03:37:18: #2 Build Peak Model... \\nINFO  @ Thu, 05 Nov 2020 03:37:18: #2 Skipped... \\nINFO  @ Thu, 05 Nov 2020 03:37:18: #3 Call peaks... \\nINFO  @ Thu, 05 Nov 2020 03:37:18: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 05 Nov 2020 03:46:25: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 05 Nov 2020 03:46:25: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_IRF8_NT_DMSO-MEF2D_R1_treat_pileup.bdg \\nINFO  @ Thu, 05 Nov 2020 03:46:25: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_IRF8_NT_DMSO-MEF2D_R1_control_lambda.bdg \\nINFO  @ Thu, 05 Nov 2020 03:46:25: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 05 Nov 2020 03:46:25: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 05 Nov 2020 03:52:12: #4 Write output xls file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MEF2D_R1_peaks.xls \\nINFO  @ Thu, 05 Nov 2020 03:52:12: #4 Write peak in narrowPeak format file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MEF2D_R1_peaks.narrowPeak \\nINFO  @ Thu, 05 Nov 2020 03:52:12: #4 Write summits bed file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MEF2D_R1_summits.bed \\nINFO  @ Thu, 05 Nov 2020 03:52:12: Done! \\n'\n",
      "b'INFO  @ Thu, 05 Nov 2020 03:52:12: \\n# Command line: callpeak -B -t ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MEF2D_R1.mLb.clN.sorted.bam -c ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 184 -n MV411_IRF8_NT_VHL-MEF2D_R1 --outdir ../../data/chipseq_IRF8_degraded/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_IRF8_NT_VHL-MEF2D_R1\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MEF2D_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Thu, 05 Nov 2020 03:52:12: #1 read fragment files... \\nINFO  @ Thu, 05 Nov 2020 03:52:12: #1 read treatment fragments... \\nINFO  @ Thu, 05 Nov 2020 03:52:17:  1000000 \\nINFO  @ Thu, 05 Nov 2020 03:52:22:  2000000 \\nINFO  @ Thu, 05 Nov 2020 03:52:28:  3000000 \\nINFO  @ Thu, 05 Nov 2020 03:52:33:  4000000 \\nINFO  @ Thu, 05 Nov 2020 03:52:38:  5000000 \\nINFO  @ Thu, 05 Nov 2020 03:52:42:  6000000 \\nINFO  @ Thu, 05 Nov 2020 03:52:47:  7000000 \\nINFO  @ Thu, 05 Nov 2020 03:52:52:  8000000 \\nINFO  @ Thu, 05 Nov 2020 03:52:57:  9000000 \\nINFO  @ Thu, 05 Nov 2020 03:53:02:  10000000 \\nINFO  @ Thu, 05 Nov 2020 03:53:07:  11000000 \\nINFO  @ Thu, 05 Nov 2020 03:53:11:  12000000 \\nINFO  @ Thu, 05 Nov 2020 03:53:16:  13000000 \\nINFO  @ Thu, 05 Nov 2020 03:53:21:  14000000 \\nINFO  @ Thu, 05 Nov 2020 03:53:26:  15000000 \\nINFO  @ Thu, 05 Nov 2020 03:53:31: 15923184 fragments have been read. \\nINFO  @ Thu, 05 Nov 2020 03:53:39: #1.2 read input fragments... \\nINFO  @ Thu, 05 Nov 2020 03:53:43:  1000000 \\nINFO  @ Thu, 05 Nov 2020 03:53:48:  2000000 \\nINFO  @ Thu, 05 Nov 2020 03:53:51:  3000000 \\nINFO  @ Thu, 05 Nov 2020 03:53:55:  4000000 \\nINFO  @ Thu, 05 Nov 2020 03:53:59:  5000000 \\nINFO  @ Thu, 05 Nov 2020 03:54:03:  6000000 \\nINFO  @ Thu, 05 Nov 2020 03:54:06:  7000000 \\nINFO  @ Thu, 05 Nov 2020 03:54:10:  8000000 \\nINFO  @ Thu, 05 Nov 2020 03:54:14:  9000000 \\nINFO  @ Thu, 05 Nov 2020 03:54:18:  10000000 \\nINFO  @ Thu, 05 Nov 2020 03:54:22:  11000000 \\nINFO  @ Thu, 05 Nov 2020 03:54:25:  12000000 \\nINFO  @ Thu, 05 Nov 2020 03:54:29:  13000000 \\nINFO  @ Thu, 05 Nov 2020 03:54:33:  14000000 \\nINFO  @ Thu, 05 Nov 2020 03:54:37:  15000000 \\nINFO  @ Thu, 05 Nov 2020 03:54:41:  16000000 \\nINFO  @ Thu, 05 Nov 2020 03:54:45:  17000000 \\nINFO  @ Thu, 05 Nov 2020 03:54:49:  18000000 \\nINFO  @ Thu, 05 Nov 2020 03:54:52:  19000000 \\nINFO  @ Thu, 05 Nov 2020 03:54:56:  20000000 \\nINFO  @ Thu, 05 Nov 2020 03:55:00:  21000000 \\nINFO  @ Thu, 05 Nov 2020 03:55:04:  22000000 \\nINFO  @ Thu, 05 Nov 2020 03:55:07:  23000000 \\nINFO  @ Thu, 05 Nov 2020 03:55:11:  24000000 \\nINFO  @ Thu, 05 Nov 2020 03:55:15:  25000000 \\nINFO  @ Thu, 05 Nov 2020 03:55:19:  26000000 \\nINFO  @ Thu, 05 Nov 2020 03:55:22:  27000000 \\nINFO  @ Thu, 05 Nov 2020 03:55:26:  28000000 \\nINFO  @ Thu, 05 Nov 2020 03:55:30:  29000000 \\nINFO  @ Thu, 05 Nov 2020 03:55:33:  30000000 \\nINFO  @ Thu, 05 Nov 2020 03:55:37:  31000000 \\nINFO  @ Thu, 05 Nov 2020 03:55:41:  32000000 \\nINFO  @ Thu, 05 Nov 2020 03:55:45:  33000000 \\nINFO  @ Thu, 05 Nov 2020 03:55:48:  34000000 \\nINFO  @ Thu, 05 Nov 2020 03:55:52:  35000000 \\nINFO  @ Thu, 05 Nov 2020 03:55:56:  36000000 \\nINFO  @ Thu, 05 Nov 2020 03:56:00:  37000000 \\nINFO  @ Thu, 05 Nov 2020 03:56:04:  38000000 \\nINFO  @ Thu, 05 Nov 2020 03:56:07:  39000000 \\nINFO  @ Thu, 05 Nov 2020 03:56:11:  40000000 \\nINFO  @ Thu, 05 Nov 2020 03:56:15:  41000000 \\nINFO  @ Thu, 05 Nov 2020 03:56:19:  42000000 \\nINFO  @ Thu, 05 Nov 2020 03:56:23:  43000000 \\nINFO  @ Thu, 05 Nov 2020 03:56:26:  44000000 \\nINFO  @ Thu, 05 Nov 2020 03:56:30:  45000000 \\nINFO  @ Thu, 05 Nov 2020 03:56:34:  46000000 \\nINFO  @ Thu, 05 Nov 2020 03:56:38:  47000000 \\nINFO  @ Thu, 05 Nov 2020 03:56:41:  48000000 \\nINFO  @ Thu, 05 Nov 2020 03:56:45:  49000000 \\nINFO  @ Thu, 05 Nov 2020 03:56:49:  50000000 \\nINFO  @ Thu, 05 Nov 2020 03:56:53:  51000000 \\nINFO  @ Thu, 05 Nov 2020 03:56:57:  52000000 \\nINFO  @ Thu, 05 Nov 2020 03:57:01:  53000000 \\nINFO  @ Thu, 05 Nov 2020 03:57:05:  54000000 \\nINFO  @ Thu, 05 Nov 2020 03:57:08:  55000000 \\nINFO  @ Thu, 05 Nov 2020 03:57:12:  56000000 \\nINFO  @ Thu, 05 Nov 2020 03:57:16:  57000000 \\nINFO  @ Thu, 05 Nov 2020 03:57:20:  58000000 \\nINFO  @ Thu, 05 Nov 2020 03:57:23:  59000000 \\nINFO  @ Thu, 05 Nov 2020 03:57:27:  60000000 \\nINFO  @ Thu, 05 Nov 2020 03:57:31:  61000000 \\nINFO  @ Thu, 05 Nov 2020 03:57:35:  62000000 \\nINFO  @ Thu, 05 Nov 2020 03:57:38:  63000000 \\nINFO  @ Thu, 05 Nov 2020 03:57:42:  64000000 \\nINFO  @ Thu, 05 Nov 2020 03:57:46:  65000000 \\nINFO  @ Thu, 05 Nov 2020 03:57:50:  66000000 \\nINFO  @ Thu, 05 Nov 2020 03:57:52: 66660890 fragments have been read. \\nINFO  @ Thu, 05 Nov 2020 03:58:37: #1 mean fragment size is determined as 220.4 bp from treatment \\nINFO  @ Thu, 05 Nov 2020 03:58:37: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Thu, 05 Nov 2020 03:58:37: #1 fragment size = 220.4 \\nINFO  @ Thu, 05 Nov 2020 03:58:37: #1  total fragments in treatment: 15923184 \\nINFO  @ Thu, 05 Nov 2020 03:58:37: #1 user defined the maximum fragments... \\nINFO  @ Thu, 05 Nov 2020 03:58:37: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Thu, 05 Nov 2020 03:59:04: #1  fragments after filtering in treatment: 15910618 \\nINFO  @ Thu, 05 Nov 2020 03:59:04: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Thu, 05 Nov 2020 03:59:04: #1  total fragments in control: 66660890 \\nINFO  @ Thu, 05 Nov 2020 03:59:04: #1 user defined the maximum fragments... \\nINFO  @ Thu, 05 Nov 2020 03:59:04: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Thu, 05 Nov 2020 04:00:53: #1  fragments after filtering in control: 66633066 \\nINFO  @ Thu, 05 Nov 2020 04:00:53: #1  Redundant rate of control: 0.00 \\nINFO  @ Thu, 05 Nov 2020 04:00:53: #1 finished! \\nINFO  @ Thu, 05 Nov 2020 04:00:53: #2 Build Peak Model... \\nINFO  @ Thu, 05 Nov 2020 04:00:53: #2 Skipped... \\nINFO  @ Thu, 05 Nov 2020 04:00:53: #3 Call peaks... \\nINFO  @ Thu, 05 Nov 2020 04:00:53: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 05 Nov 2020 04:09:59: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 05 Nov 2020 04:09:59: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_IRF8_NT_VHL-MEF2D_R1_treat_pileup.bdg \\nINFO  @ Thu, 05 Nov 2020 04:09:59: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_IRF8_NT_VHL-MEF2D_R1_control_lambda.bdg \\nINFO  @ Thu, 05 Nov 2020 04:09:59: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 05 Nov 2020 04:09:59: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 05 Nov 2020 04:15:48: #4 Write output xls file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MEF2D_R1_peaks.xls \\nINFO  @ Thu, 05 Nov 2020 04:15:48: #4 Write peak in narrowPeak format file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MEF2D_R1_peaks.narrowPeak \\nINFO  @ Thu, 05 Nov 2020 04:15:48: #4 Write summits bed file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MEF2D_R1_summits.bed \\nINFO  @ Thu, 05 Nov 2020 04:15:48: Done! \\n'\n",
      "18035712 22098080\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MEF2D_R1_treat_pileup.bdg --c1 ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MEF2D_R1_control_lambda.bdg --t2 ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MEF2D_R1_treat_pileup.bdg --c2 ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MEF2D_R1_control_lambda.bdg --d1 18035712 --d2 22098080 -g 60 -l 184 --o-prefix MV411_IRF8_NT_DMSO-MEF2D_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2D_R1_treat_pileup --outdir ../../data/chipseq_IRF8_degraded/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Thu, 05 Nov 2020 04:15:49: Read and build treatment 1 bedGraph... \\nINFO  @ Thu, 05 Nov 2020 04:16:26: Read and build control 1 bedGraph... \\nINFO  @ Thu, 05 Nov 2020 04:20:12: Read and build treatment 2 bedGraph... \\nINFO  @ Thu, 05 Nov 2020 04:20:44: Read and build control 2 bedGraph... \\nINFO  @ Thu, 05 Nov 2020 04:32:11: Write peaks... \\nINFO  @ Thu, 05 Nov 2020 04:32:11: Done \\n')\n",
      "../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MYC_R1.mLb.clN.sorted.bam ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MYC_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MYC_R1.mLb.clN.sorted.bam and ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MYC_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "205\n",
      "computing the scaling values\n",
      "b'INFO  @ Thu, 05 Nov 2020 04:33:53: \\n# Command line: callpeak -B -t ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MYC_R1.mLb.clN.sorted.bam -c ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 205 -n MV411_IRF8_NT_DMSO-MYC_R1 --outdir ../../data/chipseq_IRF8_degraded/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_IRF8_NT_DMSO-MYC_R1\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MYC_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Thu, 05 Nov 2020 04:33:53: #1 read fragment files... \\nINFO  @ Thu, 05 Nov 2020 04:33:53: #1 read treatment fragments... \\nINFO  @ Thu, 05 Nov 2020 04:33:58:  1000000 \\nINFO  @ Thu, 05 Nov 2020 04:34:03:  2000000 \\nINFO  @ Thu, 05 Nov 2020 04:34:08:  3000000 \\nINFO  @ Thu, 05 Nov 2020 04:34:12:  4000000 \\nINFO  @ Thu, 05 Nov 2020 04:34:17:  5000000 \\nINFO  @ Thu, 05 Nov 2020 04:34:22:  6000000 \\nINFO  @ Thu, 05 Nov 2020 04:34:27:  7000000 \\nINFO  @ Thu, 05 Nov 2020 04:34:31:  8000000 \\nINFO  @ Thu, 05 Nov 2020 04:34:36:  9000000 \\nINFO  @ Thu, 05 Nov 2020 04:34:41:  10000000 \\nINFO  @ Thu, 05 Nov 2020 04:34:45:  11000000 \\nINFO  @ Thu, 05 Nov 2020 04:34:50:  12000000 \\nINFO  @ Thu, 05 Nov 2020 04:34:55:  13000000 \\nINFO  @ Thu, 05 Nov 2020 04:35:00:  14000000 \\nINFO  @ Thu, 05 Nov 2020 04:35:05:  15000000 \\nINFO  @ Thu, 05 Nov 2020 04:35:10:  16000000 \\nINFO  @ Thu, 05 Nov 2020 04:35:15:  17000000 \\nINFO  @ Thu, 05 Nov 2020 04:35:20:  18000000 \\nINFO  @ Thu, 05 Nov 2020 04:35:26:  19000000 \\nINFO  @ Thu, 05 Nov 2020 04:35:30:  20000000 \\nINFO  @ Thu, 05 Nov 2020 04:35:33: 20590560 fragments have been read. \\nINFO  @ Thu, 05 Nov 2020 04:35:45: #1.2 read input fragments... \\nINFO  @ Thu, 05 Nov 2020 04:35:48:  1000000 \\nINFO  @ Thu, 05 Nov 2020 04:35:52:  2000000 \\nINFO  @ Thu, 05 Nov 2020 04:35:56:  3000000 \\nINFO  @ Thu, 05 Nov 2020 04:36:00:  4000000 \\nINFO  @ Thu, 05 Nov 2020 04:36:04:  5000000 \\nINFO  @ Thu, 05 Nov 2020 04:36:07:  6000000 \\nINFO  @ Thu, 05 Nov 2020 04:36:11:  7000000 \\nINFO  @ Thu, 05 Nov 2020 04:36:15:  8000000 \\nINFO  @ Thu, 05 Nov 2020 04:36:19:  9000000 \\nINFO  @ Thu, 05 Nov 2020 04:36:23:  10000000 \\nINFO  @ Thu, 05 Nov 2020 04:36:27:  11000000 \\nINFO  @ Thu, 05 Nov 2020 04:36:30:  12000000 \\nINFO  @ Thu, 05 Nov 2020 04:36:34:  13000000 \\nINFO  @ Thu, 05 Nov 2020 04:36:38:  14000000 \\nINFO  @ Thu, 05 Nov 2020 04:36:42:  15000000 \\nINFO  @ Thu, 05 Nov 2020 04:36:45:  16000000 \\nINFO  @ Thu, 05 Nov 2020 04:36:49:  17000000 \\nINFO  @ Thu, 05 Nov 2020 04:36:53:  18000000 \\nINFO  @ Thu, 05 Nov 2020 04:36:57:  19000000 \\nINFO  @ Thu, 05 Nov 2020 04:37:01:  20000000 \\nINFO  @ Thu, 05 Nov 2020 04:37:05:  21000000 \\nINFO  @ Thu, 05 Nov 2020 04:37:08:  22000000 \\nINFO  @ Thu, 05 Nov 2020 04:37:12:  23000000 \\nINFO  @ Thu, 05 Nov 2020 04:37:16:  24000000 \\nINFO  @ Thu, 05 Nov 2020 04:37:20:  25000000 \\nINFO  @ Thu, 05 Nov 2020 04:37:24:  26000000 \\nINFO  @ Thu, 05 Nov 2020 04:37:27:  27000000 \\nINFO  @ Thu, 05 Nov 2020 04:37:31:  28000000 \\nINFO  @ Thu, 05 Nov 2020 04:37:35:  29000000 \\nINFO  @ Thu, 05 Nov 2020 04:37:39:  30000000 \\nINFO  @ Thu, 05 Nov 2020 04:37:43:  31000000 \\nINFO  @ Thu, 05 Nov 2020 04:37:46:  32000000 \\nINFO  @ Thu, 05 Nov 2020 04:37:50:  33000000 \\nINFO  @ Thu, 05 Nov 2020 04:37:54:  34000000 \\nINFO  @ Thu, 05 Nov 2020 04:37:58:  35000000 \\nINFO  @ Thu, 05 Nov 2020 04:38:01:  36000000 \\nINFO  @ Thu, 05 Nov 2020 04:38:05:  37000000 \\nINFO  @ Thu, 05 Nov 2020 04:38:09:  38000000 \\nINFO  @ Thu, 05 Nov 2020 04:38:12:  39000000 \\nINFO  @ Thu, 05 Nov 2020 04:38:16:  40000000 \\nINFO  @ Thu, 05 Nov 2020 04:38:20:  41000000 \\nINFO  @ Thu, 05 Nov 2020 04:38:24:  42000000 \\nINFO  @ Thu, 05 Nov 2020 04:38:27:  43000000 \\nINFO  @ Thu, 05 Nov 2020 04:38:31:  44000000 \\nINFO  @ Thu, 05 Nov 2020 04:38:35:  45000000 \\nINFO  @ Thu, 05 Nov 2020 04:38:39:  46000000 \\nINFO  @ Thu, 05 Nov 2020 04:38:42:  47000000 \\nINFO  @ Thu, 05 Nov 2020 04:38:46:  48000000 \\nINFO  @ Thu, 05 Nov 2020 04:38:50:  49000000 \\nINFO  @ Thu, 05 Nov 2020 04:38:54:  50000000 \\nINFO  @ Thu, 05 Nov 2020 04:38:57:  51000000 \\nINFO  @ Thu, 05 Nov 2020 04:39:01:  52000000 \\nINFO  @ Thu, 05 Nov 2020 04:39:05:  53000000 \\nINFO  @ Thu, 05 Nov 2020 04:39:10:  54000000 \\nINFO  @ Thu, 05 Nov 2020 04:39:16:  55000000 \\nINFO  @ Thu, 05 Nov 2020 04:39:19:  56000000 \\nINFO  @ Thu, 05 Nov 2020 04:39:23:  57000000 \\nINFO  @ Thu, 05 Nov 2020 04:39:27:  58000000 \\nINFO  @ Thu, 05 Nov 2020 04:39:31:  59000000 \\nINFO  @ Thu, 05 Nov 2020 04:39:34:  60000000 \\nINFO  @ Thu, 05 Nov 2020 04:39:38:  61000000 \\nINFO  @ Thu, 05 Nov 2020 04:39:42:  62000000 \\nINFO  @ Thu, 05 Nov 2020 04:39:46:  63000000 \\nINFO  @ Thu, 05 Nov 2020 04:39:49:  64000000 \\nINFO  @ Thu, 05 Nov 2020 04:39:54:  65000000 \\nINFO  @ Thu, 05 Nov 2020 04:39:57:  66000000 \\nINFO  @ Thu, 05 Nov 2020 04:40:00: 66660890 fragments have been read. \\nINFO  @ Thu, 05 Nov 2020 04:40:44: #1 mean fragment size is determined as 229.6 bp from treatment \\nINFO  @ Thu, 05 Nov 2020 04:40:44: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Thu, 05 Nov 2020 04:40:44: #1 fragment size = 229.6 \\nINFO  @ Thu, 05 Nov 2020 04:40:44: #1  total fragments in treatment: 20590560 \\nINFO  @ Thu, 05 Nov 2020 04:40:44: #1 user defined the maximum fragments... \\nINFO  @ Thu, 05 Nov 2020 04:40:44: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Thu, 05 Nov 2020 04:41:17: #1  fragments after filtering in treatment: 20578670 \\nINFO  @ Thu, 05 Nov 2020 04:41:17: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Thu, 05 Nov 2020 04:41:17: #1  total fragments in control: 66660890 \\nINFO  @ Thu, 05 Nov 2020 04:41:17: #1 user defined the maximum fragments... \\nINFO  @ Thu, 05 Nov 2020 04:41:17: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Thu, 05 Nov 2020 04:43:02: #1  fragments after filtering in control: 66633066 \\nINFO  @ Thu, 05 Nov 2020 04:43:02: #1  Redundant rate of control: 0.00 \\nINFO  @ Thu, 05 Nov 2020 04:43:02: #1 finished! \\nINFO  @ Thu, 05 Nov 2020 04:43:02: #2 Build Peak Model... \\nINFO  @ Thu, 05 Nov 2020 04:43:02: #2 Skipped... \\nINFO  @ Thu, 05 Nov 2020 04:43:02: #3 Call peaks... \\nINFO  @ Thu, 05 Nov 2020 04:43:02: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 05 Nov 2020 04:52:17: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 05 Nov 2020 04:52:17: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_IRF8_NT_DMSO-MYC_R1_treat_pileup.bdg \\nINFO  @ Thu, 05 Nov 2020 04:52:17: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_IRF8_NT_DMSO-MYC_R1_control_lambda.bdg \\nINFO  @ Thu, 05 Nov 2020 04:52:17: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 05 Nov 2020 04:52:17: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 05 Nov 2020 04:58:26: #4 Write output xls file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MYC_R1_peaks.xls \\nINFO  @ Thu, 05 Nov 2020 04:58:26: #4 Write peak in narrowPeak format file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MYC_R1_peaks.narrowPeak \\nINFO  @ Thu, 05 Nov 2020 04:58:27: #4 Write summits bed file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MYC_R1_summits.bed \\nINFO  @ Thu, 05 Nov 2020 04:58:27: Done! \\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'INFO  @ Thu, 05 Nov 2020 04:58:27: \\n# Command line: callpeak -B -t ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MYC_R1.mLb.clN.sorted.bam -c ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 205 -n MV411_IRF8_NT_VHL-MYC_R1 --outdir ../../data/chipseq_IRF8_degraded/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_IRF8_NT_VHL-MYC_R1\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MYC_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Thu, 05 Nov 2020 04:58:27: #1 read fragment files... \\nINFO  @ Thu, 05 Nov 2020 04:58:27: #1 read treatment fragments... \\nINFO  @ Thu, 05 Nov 2020 04:58:32:  1000000 \\nINFO  @ Thu, 05 Nov 2020 04:58:38:  2000000 \\nINFO  @ Thu, 05 Nov 2020 04:58:43:  3000000 \\nINFO  @ Thu, 05 Nov 2020 04:58:47:  4000000 \\nINFO  @ Thu, 05 Nov 2020 04:58:52:  5000000 \\nINFO  @ Thu, 05 Nov 2020 04:58:57:  6000000 \\nINFO  @ Thu, 05 Nov 2020 04:59:02:  7000000 \\nINFO  @ Thu, 05 Nov 2020 04:59:06:  8000000 \\nINFO  @ Thu, 05 Nov 2020 04:59:11:  9000000 \\nINFO  @ Thu, 05 Nov 2020 04:59:16:  10000000 \\nINFO  @ Thu, 05 Nov 2020 04:59:21:  11000000 \\nINFO  @ Thu, 05 Nov 2020 04:59:25:  12000000 \\nINFO  @ Thu, 05 Nov 2020 04:59:32:  13000000 \\nINFO  @ Thu, 05 Nov 2020 04:59:37:  14000000 \\nINFO  @ Thu, 05 Nov 2020 04:59:41:  15000000 \\nINFO  @ Thu, 05 Nov 2020 04:59:47:  16000000 \\nINFO  @ Thu, 05 Nov 2020 04:59:51:  17000000 \\nINFO  @ Thu, 05 Nov 2020 04:59:56:  18000000 \\nINFO  @ Thu, 05 Nov 2020 04:59:59: 18592701 fragments have been read. \\nINFO  @ Thu, 05 Nov 2020 05:00:09: #1.2 read input fragments... \\nINFO  @ Thu, 05 Nov 2020 05:00:13:  1000000 \\nINFO  @ Thu, 05 Nov 2020 05:00:17:  2000000 \\nINFO  @ Thu, 05 Nov 2020 05:00:21:  3000000 \\nINFO  @ Thu, 05 Nov 2020 05:00:24:  4000000 \\nINFO  @ Thu, 05 Nov 2020 05:00:28:  5000000 \\nINFO  @ Thu, 05 Nov 2020 05:00:32:  6000000 \\nINFO  @ Thu, 05 Nov 2020 05:00:35:  7000000 \\nINFO  @ Thu, 05 Nov 2020 05:00:39:  8000000 \\nINFO  @ Thu, 05 Nov 2020 05:00:43:  9000000 \\nINFO  @ Thu, 05 Nov 2020 05:00:47:  10000000 \\nINFO  @ Thu, 05 Nov 2020 05:00:50:  11000000 \\nINFO  @ Thu, 05 Nov 2020 05:00:54:  12000000 \\nINFO  @ Thu, 05 Nov 2020 05:00:58:  13000000 \\nINFO  @ Thu, 05 Nov 2020 05:01:01:  14000000 \\nINFO  @ Thu, 05 Nov 2020 05:01:05:  15000000 \\nINFO  @ Thu, 05 Nov 2020 05:01:09:  16000000 \\nINFO  @ Thu, 05 Nov 2020 05:01:13:  17000000 \\nINFO  @ Thu, 05 Nov 2020 05:01:16:  18000000 \\nINFO  @ Thu, 05 Nov 2020 05:01:20:  19000000 \\nINFO  @ Thu, 05 Nov 2020 05:01:24:  20000000 \\nINFO  @ Thu, 05 Nov 2020 05:01:28:  21000000 \\nINFO  @ Thu, 05 Nov 2020 05:01:31:  22000000 \\nINFO  @ Thu, 05 Nov 2020 05:01:35:  23000000 \\nINFO  @ Thu, 05 Nov 2020 05:01:39:  24000000 \\nINFO  @ Thu, 05 Nov 2020 05:01:43:  25000000 \\nINFO  @ Thu, 05 Nov 2020 05:01:46:  26000000 \\nINFO  @ Thu, 05 Nov 2020 05:01:50:  27000000 \\nINFO  @ Thu, 05 Nov 2020 05:01:54:  28000000 \\nINFO  @ Thu, 05 Nov 2020 05:01:57:  29000000 \\nINFO  @ Thu, 05 Nov 2020 05:02:01:  30000000 \\nINFO  @ Thu, 05 Nov 2020 05:02:05:  31000000 \\nINFO  @ Thu, 05 Nov 2020 05:02:09:  32000000 \\nINFO  @ Thu, 05 Nov 2020 05:02:12:  33000000 \\nINFO  @ Thu, 05 Nov 2020 05:02:16:  34000000 \\nINFO  @ Thu, 05 Nov 2020 05:02:20:  35000000 \\nINFO  @ Thu, 05 Nov 2020 05:02:23:  36000000 \\nINFO  @ Thu, 05 Nov 2020 05:02:27:  37000000 \\nINFO  @ Thu, 05 Nov 2020 05:02:31:  38000000 \\nINFO  @ Thu, 05 Nov 2020 05:02:34:  39000000 \\nINFO  @ Thu, 05 Nov 2020 05:02:38:  40000000 \\nINFO  @ Thu, 05 Nov 2020 05:02:42:  41000000 \\nINFO  @ Thu, 05 Nov 2020 05:02:46:  42000000 \\nINFO  @ Thu, 05 Nov 2020 05:02:50:  43000000 \\nINFO  @ Thu, 05 Nov 2020 05:02:53:  44000000 \\nINFO  @ Thu, 05 Nov 2020 05:02:57:  45000000 \\nINFO  @ Thu, 05 Nov 2020 05:03:01:  46000000 \\nINFO  @ Thu, 05 Nov 2020 05:03:04:  47000000 \\nINFO  @ Thu, 05 Nov 2020 05:03:08:  48000000 \\nINFO  @ Thu, 05 Nov 2020 05:03:12:  49000000 \\nINFO  @ Thu, 05 Nov 2020 05:03:16:  50000000 \\nINFO  @ Thu, 05 Nov 2020 05:03:20:  51000000 \\nINFO  @ Thu, 05 Nov 2020 05:03:23:  52000000 \\nINFO  @ Thu, 05 Nov 2020 05:03:27:  53000000 \\nINFO  @ Thu, 05 Nov 2020 05:03:31:  54000000 \\nINFO  @ Thu, 05 Nov 2020 05:03:36:  55000000 \\nINFO  @ Thu, 05 Nov 2020 05:03:40:  56000000 \\nINFO  @ Thu, 05 Nov 2020 05:03:44:  57000000 \\nINFO  @ Thu, 05 Nov 2020 05:03:48:  58000000 \\nINFO  @ Thu, 05 Nov 2020 05:03:52:  59000000 \\nINFO  @ Thu, 05 Nov 2020 05:03:55:  60000000 \\nINFO  @ Thu, 05 Nov 2020 05:03:59:  61000000 \\nINFO  @ Thu, 05 Nov 2020 05:04:03:  62000000 \\nINFO  @ Thu, 05 Nov 2020 05:04:07:  63000000 \\nINFO  @ Thu, 05 Nov 2020 05:04:11:  64000000 \\nINFO  @ Thu, 05 Nov 2020 05:04:14:  65000000 \\nINFO  @ Thu, 05 Nov 2020 05:04:18:  66000000 \\nINFO  @ Thu, 05 Nov 2020 05:04:21: 66660890 fragments have been read. \\nINFO  @ Thu, 05 Nov 2020 05:05:05: #1 mean fragment size is determined as 222.0 bp from treatment \\nINFO  @ Thu, 05 Nov 2020 05:05:05: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Thu, 05 Nov 2020 05:05:05: #1 fragment size = 222.0 \\nINFO  @ Thu, 05 Nov 2020 05:05:05: #1  total fragments in treatment: 18592701 \\nINFO  @ Thu, 05 Nov 2020 05:05:05: #1 user defined the maximum fragments... \\nINFO  @ Thu, 05 Nov 2020 05:05:05: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Thu, 05 Nov 2020 05:05:35: #1  fragments after filtering in treatment: 18584773 \\nINFO  @ Thu, 05 Nov 2020 05:05:35: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Thu, 05 Nov 2020 05:05:35: #1  total fragments in control: 66660890 \\nINFO  @ Thu, 05 Nov 2020 05:05:35: #1 user defined the maximum fragments... \\nINFO  @ Thu, 05 Nov 2020 05:05:35: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Thu, 05 Nov 2020 05:07:23: #1  fragments after filtering in control: 66633066 \\nINFO  @ Thu, 05 Nov 2020 05:07:23: #1  Redundant rate of control: 0.00 \\nINFO  @ Thu, 05 Nov 2020 05:07:23: #1 finished! \\nINFO  @ Thu, 05 Nov 2020 05:07:23: #2 Build Peak Model... \\nINFO  @ Thu, 05 Nov 2020 05:07:23: #2 Skipped... \\nINFO  @ Thu, 05 Nov 2020 05:07:23: #3 Call peaks... \\nINFO  @ Thu, 05 Nov 2020 05:07:23: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 05 Nov 2020 05:16:29: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 05 Nov 2020 05:16:29: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_IRF8_NT_VHL-MYC_R1_treat_pileup.bdg \\nINFO  @ Thu, 05 Nov 2020 05:16:29: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_IRF8_NT_VHL-MYC_R1_control_lambda.bdg \\nINFO  @ Thu, 05 Nov 2020 05:16:29: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 05 Nov 2020 05:16:29: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 05 Nov 2020 05:22:48: #4 Write output xls file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MYC_R1_peaks.xls \\nINFO  @ Thu, 05 Nov 2020 05:22:48: #4 Write peak in narrowPeak format file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MYC_R1_peaks.narrowPeak \\nINFO  @ Thu, 05 Nov 2020 05:22:48: #4 Write summits bed file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MYC_R1_summits.bed \\nINFO  @ Thu, 05 Nov 2020 05:22:48: Done! \\n'\n",
      "22613923 18584773\n",
      "doing differential peak binding\n",
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MYC_R1_treat_pileup.bdg --c1 ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MYC_R1_control_lambda.bdg --t2 ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MYC_R1_treat_pileup.bdg --c2 ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MYC_R1_control_lambda.bdg --d1 22613923 --d2 18584773 -g 60 -l 205 --o-prefix MV411_IRF8_NT_DMSO-MYC_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MYC_R1_treat_pileup --outdir ../../data/chipseq_IRF8_degraded/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Thu, 05 Nov 2020 05:22:49: Read and build treatment 1 bedGraph... \\nINFO  @ Thu, 05 Nov 2020 05:23:29: Read and build control 1 bedGraph... \\nINFO  @ Thu, 05 Nov 2020 05:27:19: Read and build treatment 2 bedGraph... \\nINFO  @ Thu, 05 Nov 2020 05:27:56: Read and build control 2 bedGraph... \\nINFO  @ Thu, 05 Nov 2020 05:40:20: Write peaks... \\nINFO  @ Thu, 05 Nov 2020 05:40:20: Done \\n')\n",
      "../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-POLII_total_R1.mLb.clN.sorted.bam ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-POLII_total_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-POLII_total_R1.mLb.clN.sorted.bam and ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-POLII_total_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "236\n",
      "computing the scaling values\n",
      "b'INFO  @ Thu, 05 Nov 2020 05:41:59: \\n# Command line: callpeak -B -t ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-POLII_total_R1.mLb.clN.sorted.bam -c ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 236 -n MV411_IRF8_NT_DMSO-POLII_total_R1 --outdir ../../data/chipseq_IRF8_degraded/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_IRF8_NT_DMSO-POLII_total_R1\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-POLII_total_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Thu, 05 Nov 2020 05:41:59: #1 read fragment files... \\nINFO  @ Thu, 05 Nov 2020 05:41:59: #1 read treatment fragments... \\nINFO  @ Thu, 05 Nov 2020 05:42:04:  1000000 \\nINFO  @ Thu, 05 Nov 2020 05:42:08:  2000000 \\nINFO  @ Thu, 05 Nov 2020 05:42:13:  3000000 \\nINFO  @ Thu, 05 Nov 2020 05:42:18:  4000000 \\nINFO  @ Thu, 05 Nov 2020 05:42:23:  5000000 \\nINFO  @ Thu, 05 Nov 2020 05:42:27:  6000000 \\nINFO  @ Thu, 05 Nov 2020 05:42:32:  7000000 \\nINFO  @ Thu, 05 Nov 2020 05:42:37:  8000000 \\nINFO  @ Thu, 05 Nov 2020 05:42:42:  9000000 \\nINFO  @ Thu, 05 Nov 2020 05:42:47:  10000000 \\nINFO  @ Thu, 05 Nov 2020 05:42:52:  11000000 \\nINFO  @ Thu, 05 Nov 2020 05:42:56:  12000000 \\nINFO  @ Thu, 05 Nov 2020 05:43:01:  13000000 \\nINFO  @ Thu, 05 Nov 2020 05:43:06:  14000000 \\nINFO  @ Thu, 05 Nov 2020 05:43:10:  15000000 \\nINFO  @ Thu, 05 Nov 2020 05:43:15:  16000000 \\nINFO  @ Thu, 05 Nov 2020 05:43:20:  17000000 \\nINFO  @ Thu, 05 Nov 2020 05:43:24:  18000000 \\nINFO  @ Thu, 05 Nov 2020 05:43:29:  19000000 \\nINFO  @ Thu, 05 Nov 2020 05:43:30: 19119331 fragments have been read. \\nINFO  @ Thu, 05 Nov 2020 05:43:40: #1.2 read input fragments... \\nINFO  @ Thu, 05 Nov 2020 05:43:44:  1000000 \\nINFO  @ Thu, 05 Nov 2020 05:43:48:  2000000 \\nINFO  @ Thu, 05 Nov 2020 05:43:51:  3000000 \\nINFO  @ Thu, 05 Nov 2020 05:43:55:  4000000 \\nINFO  @ Thu, 05 Nov 2020 05:43:59:  5000000 \\nINFO  @ Thu, 05 Nov 2020 05:44:03:  6000000 \\nINFO  @ Thu, 05 Nov 2020 05:44:06:  7000000 \\nINFO  @ Thu, 05 Nov 2020 05:44:10:  8000000 \\nINFO  @ Thu, 05 Nov 2020 05:44:15:  9000000 \\nINFO  @ Thu, 05 Nov 2020 05:44:20:  10000000 \\nINFO  @ Thu, 05 Nov 2020 05:44:24:  11000000 \\nINFO  @ Thu, 05 Nov 2020 05:44:27:  12000000 \\nINFO  @ Thu, 05 Nov 2020 05:44:31:  13000000 \\nINFO  @ Thu, 05 Nov 2020 05:44:35:  14000000 \\nINFO  @ Thu, 05 Nov 2020 05:44:38:  15000000 \\nINFO  @ Thu, 05 Nov 2020 05:44:42:  16000000 \\nINFO  @ Thu, 05 Nov 2020 05:44:46:  17000000 \\nINFO  @ Thu, 05 Nov 2020 05:44:49:  18000000 \\nINFO  @ Thu, 05 Nov 2020 05:44:53:  19000000 \\nINFO  @ Thu, 05 Nov 2020 05:44:57:  20000000 \\nINFO  @ Thu, 05 Nov 2020 05:45:01:  21000000 \\nINFO  @ Thu, 05 Nov 2020 05:45:04:  22000000 \\nINFO  @ Thu, 05 Nov 2020 05:45:08:  23000000 \\nINFO  @ Thu, 05 Nov 2020 05:45:12:  24000000 \\nINFO  @ Thu, 05 Nov 2020 05:45:16:  25000000 \\nINFO  @ Thu, 05 Nov 2020 05:45:20:  26000000 \\nINFO  @ Thu, 05 Nov 2020 05:45:24:  27000000 \\nINFO  @ Thu, 05 Nov 2020 05:45:27:  28000000 \\nINFO  @ Thu, 05 Nov 2020 05:45:31:  29000000 \\nINFO  @ Thu, 05 Nov 2020 05:45:35:  30000000 \\nINFO  @ Thu, 05 Nov 2020 05:45:39:  31000000 \\nINFO  @ Thu, 05 Nov 2020 05:45:42:  32000000 \\nINFO  @ Thu, 05 Nov 2020 05:45:46:  33000000 \\nINFO  @ Thu, 05 Nov 2020 05:45:50:  34000000 \\nINFO  @ Thu, 05 Nov 2020 05:45:53:  35000000 \\nINFO  @ Thu, 05 Nov 2020 05:45:57:  36000000 \\nINFO  @ Thu, 05 Nov 2020 05:46:01:  37000000 \\nINFO  @ Thu, 05 Nov 2020 05:46:04:  38000000 \\nINFO  @ Thu, 05 Nov 2020 05:46:08:  39000000 \\nINFO  @ Thu, 05 Nov 2020 05:46:12:  40000000 \\nINFO  @ Thu, 05 Nov 2020 05:46:15:  41000000 \\nINFO  @ Thu, 05 Nov 2020 05:46:19:  42000000 \\nINFO  @ Thu, 05 Nov 2020 05:46:23:  43000000 \\nINFO  @ Thu, 05 Nov 2020 05:46:27:  44000000 \\nINFO  @ Thu, 05 Nov 2020 05:46:30:  45000000 \\nINFO  @ Thu, 05 Nov 2020 05:46:34:  46000000 \\nINFO  @ Thu, 05 Nov 2020 05:46:38:  47000000 \\nINFO  @ Thu, 05 Nov 2020 05:46:41:  48000000 \\nINFO  @ Thu, 05 Nov 2020 05:46:45:  49000000 \\nINFO  @ Thu, 05 Nov 2020 05:46:49:  50000000 \\nINFO  @ Thu, 05 Nov 2020 05:46:53:  51000000 \\nINFO  @ Thu, 05 Nov 2020 05:46:57:  52000000 \\nINFO  @ Thu, 05 Nov 2020 05:47:01:  53000000 \\nINFO  @ Thu, 05 Nov 2020 05:47:04:  54000000 \\nINFO  @ Thu, 05 Nov 2020 05:47:08:  55000000 \\nINFO  @ Thu, 05 Nov 2020 05:47:12:  56000000 \\nINFO  @ Thu, 05 Nov 2020 05:47:16:  57000000 \\nINFO  @ Thu, 05 Nov 2020 05:47:19:  58000000 \\nINFO  @ Thu, 05 Nov 2020 05:47:23:  59000000 \\nINFO  @ Thu, 05 Nov 2020 05:47:27:  60000000 \\nINFO  @ Thu, 05 Nov 2020 05:47:31:  61000000 \\nINFO  @ Thu, 05 Nov 2020 05:47:34:  62000000 \\nINFO  @ Thu, 05 Nov 2020 05:47:38:  63000000 \\nINFO  @ Thu, 05 Nov 2020 05:47:42:  64000000 \\nINFO  @ Thu, 05 Nov 2020 05:47:46:  65000000 \\nINFO  @ Thu, 05 Nov 2020 05:47:49:  66000000 \\nINFO  @ Thu, 05 Nov 2020 05:47:52: 66660890 fragments have been read. \\nINFO  @ Thu, 05 Nov 2020 05:48:36: #1 mean fragment size is determined as 249.8 bp from treatment \\nINFO  @ Thu, 05 Nov 2020 05:48:36: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Thu, 05 Nov 2020 05:48:36: #1 fragment size = 249.8 \\nINFO  @ Thu, 05 Nov 2020 05:48:36: #1  total fragments in treatment: 19119331 \\nINFO  @ Thu, 05 Nov 2020 05:48:36: #1 user defined the maximum fragments... \\nINFO  @ Thu, 05 Nov 2020 05:48:36: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Thu, 05 Nov 2020 05:49:10: #1  fragments after filtering in treatment: 19113190 \\nINFO  @ Thu, 05 Nov 2020 05:49:10: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Thu, 05 Nov 2020 05:49:10: #1  total fragments in control: 66660890 \\nINFO  @ Thu, 05 Nov 2020 05:49:10: #1 user defined the maximum fragments... \\nINFO  @ Thu, 05 Nov 2020 05:49:10: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Thu, 05 Nov 2020 05:51:05: #1  fragments after filtering in control: 66633066 \\nINFO  @ Thu, 05 Nov 2020 05:51:05: #1  Redundant rate of control: 0.00 \\nINFO  @ Thu, 05 Nov 2020 05:51:05: #1 finished! \\nINFO  @ Thu, 05 Nov 2020 05:51:05: #2 Build Peak Model... \\nINFO  @ Thu, 05 Nov 2020 05:51:05: #2 Skipped... \\nINFO  @ Thu, 05 Nov 2020 05:51:05: #3 Call peaks... \\nINFO  @ Thu, 05 Nov 2020 05:51:05: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 05 Nov 2020 06:00:16: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 05 Nov 2020 06:00:16: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_IRF8_NT_DMSO-POLII_total_R1_treat_pileup.bdg \\nINFO  @ Thu, 05 Nov 2020 06:00:16: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_IRF8_NT_DMSO-POLII_total_R1_control_lambda.bdg \\nINFO  @ Thu, 05 Nov 2020 06:00:16: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 05 Nov 2020 06:00:16: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 05 Nov 2020 06:06:29: #4 Write output xls file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-POLII_total_R1_peaks.xls \\nINFO  @ Thu, 05 Nov 2020 06:06:29: #4 Write peak in narrowPeak format file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-POLII_total_R1_peaks.narrowPeak \\nINFO  @ Thu, 05 Nov 2020 06:06:29: #4 Write summits bed file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-POLII_total_R1_summits.bed \\nINFO  @ Thu, 05 Nov 2020 06:06:29: Done! \\n'\n",
      "b'INFO  @ Thu, 05 Nov 2020 06:06:30: \\n# Command line: callpeak -B -t ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-POLII_total_R1.mLb.clN.sorted.bam -c ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 236 -n MV411_IRF8_NT_VHL-POLII_total_R1 --outdir ../../data/chipseq_IRF8_degraded/diffData/ -f BAMPE\\n# ARGUMENTS LIST:\\n# name = MV411_IRF8_NT_VHL-POLII_total_R1\\n# format = BAMPE\\n# ChIP-seq file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-POLII_total_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is on\\n \\nINFO  @ Thu, 05 Nov 2020 06:06:30: #1 read fragment files... \\nINFO  @ Thu, 05 Nov 2020 06:06:30: #1 read treatment fragments... \\nINFO  @ Thu, 05 Nov 2020 06:06:35:  1000000 \\nINFO  @ Thu, 05 Nov 2020 06:06:39:  2000000 \\nINFO  @ Thu, 05 Nov 2020 06:06:45:  3000000 \\nINFO  @ Thu, 05 Nov 2020 06:06:50:  4000000 \\nINFO  @ Thu, 05 Nov 2020 06:06:55:  5000000 \\nINFO  @ Thu, 05 Nov 2020 06:07:00:  6000000 \\nINFO  @ Thu, 05 Nov 2020 06:07:05:  7000000 \\nINFO  @ Thu, 05 Nov 2020 06:07:09:  8000000 \\nINFO  @ Thu, 05 Nov 2020 06:07:14:  9000000 \\nINFO  @ Thu, 05 Nov 2020 06:07:19:  10000000 \\nINFO  @ Thu, 05 Nov 2020 06:07:24:  11000000 \\nINFO  @ Thu, 05 Nov 2020 06:07:29:  12000000 \\nINFO  @ Thu, 05 Nov 2020 06:07:34:  13000000 \\nINFO  @ Thu, 05 Nov 2020 06:07:39:  14000000 \\nINFO  @ Thu, 05 Nov 2020 06:07:44:  15000000 \\nINFO  @ Thu, 05 Nov 2020 06:07:49:  16000000 \\nINFO  @ Thu, 05 Nov 2020 06:07:53:  17000000 \\nINFO  @ Thu, 05 Nov 2020 06:07:58:  18000000 \\nINFO  @ Thu, 05 Nov 2020 06:08:03:  19000000 \\nINFO  @ Thu, 05 Nov 2020 06:08:04: 19180829 fragments have been read. \\nINFO  @ Thu, 05 Nov 2020 06:08:15: #1.2 read input fragments... \\nINFO  @ Thu, 05 Nov 2020 06:08:19:  1000000 \\nINFO  @ Thu, 05 Nov 2020 06:08:22:  2000000 \\nINFO  @ Thu, 05 Nov 2020 06:08:26:  3000000 \\nINFO  @ Thu, 05 Nov 2020 06:08:30:  4000000 \\nINFO  @ Thu, 05 Nov 2020 06:08:34:  5000000 \\nINFO  @ Thu, 05 Nov 2020 06:08:37:  6000000 \\nINFO  @ Thu, 05 Nov 2020 06:08:41:  7000000 \\nINFO  @ Thu, 05 Nov 2020 06:08:45:  8000000 \\nINFO  @ Thu, 05 Nov 2020 06:08:49:  9000000 \\nINFO  @ Thu, 05 Nov 2020 06:08:53:  10000000 \\nINFO  @ Thu, 05 Nov 2020 06:08:57:  11000000 \\nINFO  @ Thu, 05 Nov 2020 06:09:01:  12000000 \\nINFO  @ Thu, 05 Nov 2020 06:09:04:  13000000 \\nINFO  @ Thu, 05 Nov 2020 06:09:08:  14000000 \\nINFO  @ Thu, 05 Nov 2020 06:09:12:  15000000 \\nINFO  @ Thu, 05 Nov 2020 06:09:16:  16000000 \\nINFO  @ Thu, 05 Nov 2020 06:09:19:  17000000 \\nINFO  @ Thu, 05 Nov 2020 06:09:23:  18000000 \\nINFO  @ Thu, 05 Nov 2020 06:09:27:  19000000 \\nINFO  @ Thu, 05 Nov 2020 06:09:30:  20000000 \\nINFO  @ Thu, 05 Nov 2020 06:09:34:  21000000 \\nINFO  @ Thu, 05 Nov 2020 06:09:38:  22000000 \\nINFO  @ Thu, 05 Nov 2020 06:09:42:  23000000 \\nINFO  @ Thu, 05 Nov 2020 06:09:46:  24000000 \\nINFO  @ Thu, 05 Nov 2020 06:09:50:  25000000 \\nINFO  @ Thu, 05 Nov 2020 06:09:53:  26000000 \\nINFO  @ Thu, 05 Nov 2020 06:09:57:  27000000 \\nINFO  @ Thu, 05 Nov 2020 06:10:01:  28000000 \\nINFO  @ Thu, 05 Nov 2020 06:10:05:  29000000 \\nINFO  @ Thu, 05 Nov 2020 06:10:08:  30000000 \\nINFO  @ Thu, 05 Nov 2020 06:10:12:  31000000 \\nINFO  @ Thu, 05 Nov 2020 06:10:16:  32000000 \\nINFO  @ Thu, 05 Nov 2020 06:10:19:  33000000 \\nINFO  @ Thu, 05 Nov 2020 06:10:23:  34000000 \\nINFO  @ Thu, 05 Nov 2020 06:10:27:  35000000 \\nINFO  @ Thu, 05 Nov 2020 06:10:30:  36000000 \\nINFO  @ Thu, 05 Nov 2020 06:10:34:  37000000 \\nINFO  @ Thu, 05 Nov 2020 06:10:38:  38000000 \\nINFO  @ Thu, 05 Nov 2020 06:10:42:  39000000 \\nINFO  @ Thu, 05 Nov 2020 06:10:45:  40000000 \\nINFO  @ Thu, 05 Nov 2020 06:10:50:  41000000 \\nINFO  @ Thu, 05 Nov 2020 06:10:54:  42000000 \\nINFO  @ Thu, 05 Nov 2020 06:10:57:  43000000 \\nINFO  @ Thu, 05 Nov 2020 06:11:01:  44000000 \\nINFO  @ Thu, 05 Nov 2020 06:11:05:  45000000 \\nINFO  @ Thu, 05 Nov 2020 06:11:08:  46000000 \\nINFO  @ Thu, 05 Nov 2020 06:11:12:  47000000 \\nINFO  @ Thu, 05 Nov 2020 06:11:16:  48000000 \\nINFO  @ Thu, 05 Nov 2020 06:11:19:  49000000 \\nINFO  @ Thu, 05 Nov 2020 06:11:23:  50000000 \\nINFO  @ Thu, 05 Nov 2020 06:11:27:  51000000 \\nINFO  @ Thu, 05 Nov 2020 06:11:30:  52000000 \\nINFO  @ Thu, 05 Nov 2020 06:11:34:  53000000 \\nINFO  @ Thu, 05 Nov 2020 06:11:38:  54000000 \\nINFO  @ Thu, 05 Nov 2020 06:11:42:  55000000 \\nINFO  @ Thu, 05 Nov 2020 06:11:45:  56000000 \\nINFO  @ Thu, 05 Nov 2020 06:11:49:  57000000 \\nINFO  @ Thu, 05 Nov 2020 06:11:53:  58000000 \\nINFO  @ Thu, 05 Nov 2020 06:11:56:  59000000 \\nINFO  @ Thu, 05 Nov 2020 06:12:00:  60000000 \\nINFO  @ Thu, 05 Nov 2020 06:12:04:  61000000 \\nINFO  @ Thu, 05 Nov 2020 06:12:08:  62000000 \\nINFO  @ Thu, 05 Nov 2020 06:12:11:  63000000 \\nINFO  @ Thu, 05 Nov 2020 06:12:15:  64000000 \\nINFO  @ Thu, 05 Nov 2020 06:12:19:  65000000 \\nINFO  @ Thu, 05 Nov 2020 06:12:22:  66000000 \\nINFO  @ Thu, 05 Nov 2020 06:12:25: 66660890 fragments have been read. \\nINFO  @ Thu, 05 Nov 2020 06:13:10: #1 mean fragment size is determined as 223.9 bp from treatment \\nINFO  @ Thu, 05 Nov 2020 06:13:10: #1 note: mean fragment size in control is 218.7 bp -- value ignored \\nINFO  @ Thu, 05 Nov 2020 06:13:10: #1 fragment size = 223.9 \\nINFO  @ Thu, 05 Nov 2020 06:13:10: #1  total fragments in treatment: 19180829 \\nINFO  @ Thu, 05 Nov 2020 06:13:10: #1 user defined the maximum fragments... \\nINFO  @ Thu, 05 Nov 2020 06:13:10: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Thu, 05 Nov 2020 06:13:40: #1  fragments after filtering in treatment: 19173520 \\nINFO  @ Thu, 05 Nov 2020 06:13:40: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Thu, 05 Nov 2020 06:13:40: #1  total fragments in control: 66660890 \\nINFO  @ Thu, 05 Nov 2020 06:13:40: #1 user defined the maximum fragments... \\nINFO  @ Thu, 05 Nov 2020 06:13:40: #1 filter out redundant fragments by allowing at most 1 identical fragment(s) \\nINFO  @ Thu, 05 Nov 2020 06:15:28: #1  fragments after filtering in control: 66633066 \\nINFO  @ Thu, 05 Nov 2020 06:15:28: #1  Redundant rate of control: 0.00 \\nINFO  @ Thu, 05 Nov 2020 06:15:28: #1 finished! \\nINFO  @ Thu, 05 Nov 2020 06:15:28: #2 Build Peak Model... \\nINFO  @ Thu, 05 Nov 2020 06:15:28: #2 Skipped... \\nINFO  @ Thu, 05 Nov 2020 06:15:28: #3 Call peaks... \\nINFO  @ Thu, 05 Nov 2020 06:15:28: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 05 Nov 2020 06:24:43: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 05 Nov 2020 06:24:43: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_IRF8_NT_VHL-POLII_total_R1_treat_pileup.bdg \\nINFO  @ Thu, 05 Nov 2020 06:24:43: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_IRF8_NT_VHL-POLII_total_R1_control_lambda.bdg \\nINFO  @ Thu, 05 Nov 2020 06:24:43: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 05 Nov 2020 06:24:43: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 05 Nov 2020 06:31:02: #4 Write output xls file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-POLII_total_R1_peaks.xls \\nINFO  @ Thu, 05 Nov 2020 06:31:02: #4 Write peak in narrowPeak format file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-POLII_total_R1_peaks.narrowPeak \\nINFO  @ Thu, 05 Nov 2020 06:31:02: #4 Write summits bed file... ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-POLII_total_R1_summits.bed \\nINFO  @ Thu, 05 Nov 2020 06:31:02: Done! \\n'\n",
      "19113190 83363130\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-POLII_total_R1_treat_pileup.bdg --c1 ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-POLII_total_R1_control_lambda.bdg --t2 ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-POLII_total_R1_treat_pileup.bdg --c2 ../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-POLII_total_R1_control_lambda.bdg --d1 19113190 --d2 83363130 -g 60 -l 236 --o-prefix MV411_IRF8_NT_DMSO-POLII_total_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-POLII_total_R1_treat_pileup --outdir ../../data/chipseq_IRF8_degraded/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Thu, 05 Nov 2020 06:31:03: Read and build treatment 1 bedGraph... \\nINFO  @ Thu, 05 Nov 2020 06:31:38: Read and build control 1 bedGraph... \\nINFO  @ Thu, 05 Nov 2020 06:35:13: Read and build treatment 2 bedGraph... \\nINFO  @ Thu, 05 Nov 2020 06:35:48: Read and build control 2 bedGraph... \\nINFO  @ Thu, 05 Nov 2020 06:47:37: Write peaks... \\nINFO  @ Thu, 05 Nov 2020 06:47:37: Done \\n')\n"
     ]
    }
   ],
   "source": [
    "# diffPeak on scaled data (full reprocessing)\n",
    "for i in range(int(len(bams)/2)):\n",
    "    if i < 0:\n",
    "        continue\n",
    "    bam1 = bams[1+i]\n",
    "    bam2 = bams[7+i]\n",
    "    print(bam1,bam2)\n",
    "    print(chip.fullDiffPeak(bam1,bam2, control1=bams[0], scaling=[scales[(i*2)], scales[(i*2)+1]], directory = \"../../data/\"+project+\"/diffData/\", res_directory = \"../../data/\"+project+\"/diffPeaks/\",pairedend=True))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 253,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['0.00744797',\n",
       " '0.0305914',\n",
       " '0.0238996',\n",
       " '0.0282355',\n",
       " '0.0274099',\n",
       " '0.0240789',\n",
       " '0.0259882',\n",
       " '0.0313712',\n",
       " '0.0202025',\n",
       " '0.0284237',\n",
       " '0.0310627',\n",
       " '0.0266872',\n",
       " '0.0258712']"
      ]
     },
     "execution_count": 253,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "initscales = ! cat ../../data/$project/bwa/mergedLibrary/bigwig/scale/*.txt\n",
    "initscales"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 252,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[0.0305914,\n",
       " 0.018880684000000002,\n",
       " 0.016658944999999998,\n",
       " 0.0274099,\n",
       " 0.018540753,\n",
       " 0.0259882,\n",
       " 0.0313712,\n",
       " 0.014545800000000001,\n",
       " 0.025865567,\n",
       " 0.0310627,\n",
       " 0.0266872,\n",
       " 0.005950376]"
      ]
     },
     "execution_count": 252,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "rescales = [val*float(initscales[1+i]) for i, val in enumerate(scales)]\n",
    "rescales"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 257,
   "metadata": {},
   "outputs": [],
   "source": [
    "chip.bigWigFrom(bams[1:], \n",
    "                genome='GRCh38',scaling=rescales,\n",
    "               numthreads=8)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 264,
   "metadata": {},
   "outputs": [],
   "source": [
    "!mkdir ../../data/$project/recalib_bigwig/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 265,
   "metadata": {
    "scrolled": false
   },
   "outputs": [],
   "source": [
    "!mv bigwig/* ../../data/$project/recalib_bigwig/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 259,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['696 ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-IRF8_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-IRF8_R1_treat_pileup_c3.0_common.bed',\n",
       " '36431 ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-IRF8_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-IRF8_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '198 ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-IRF8_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-IRF8_R1_treat_pileup_c3.0_cond2.bed',\n",
       " '12195 ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MED1_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MED1_R1_treat_pileup_c3.0_common.bed',\n",
       " '78 ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MED1_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MED1_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '2236 ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MED1_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MED1_R1_treat_pileup_c3.0_cond2.bed',\n",
       " '7037 ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MEF2C_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2C_R1_treat_pileup_c3.0_common.bed',\n",
       " '702 ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MEF2C_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2C_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '1315 ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MEF2C_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2C_R1_treat_pileup_c3.0_cond2.bed',\n",
       " '439 ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MEF2D_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2D_R1_treat_pileup_c3.0_common.bed',\n",
       " '149 ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MEF2D_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2D_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '5 ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MEF2D_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2D_R1_treat_pileup_c3.0_cond2.bed',\n",
       " '24946 ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MYC_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MYC_R1_treat_pileup_c3.0_common.bed',\n",
       " '365 ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MYC_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MYC_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '528 ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MYC_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MYC_R1_treat_pileup_c3.0_cond2.bed',\n",
       " '91 ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-POLII_total_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-POLII_total_R1_treat_pileup_c3.0_common.bed',\n",
       " '27969 ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-POLII_total_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-POLII_total_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '1 ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-POLII_total_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-POLII_total_R1_treat_pileup_c3.0_cond2.bed',\n",
       " '']"
      ]
     },
     "execution_count": 259,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "os.popen('for i in $(ls ../../data/'+project+'/diffPeaks/*.bed); \\\n",
    "            do echo $(wc -l $i); \\\n",
    "            done').read().split('\\n')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 266,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_DMSO-IRF8_R1.bw',\n",
       " '../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_DMSO-MED1_R1.bw',\n",
       " '../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_DMSO-MEF2C_R1.bw',\n",
       " '../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_DMSO-MEF2D_R1.bw',\n",
       " '../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_DMSO-MYC_R1.bw',\n",
       " '../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_DMSO-POLII_total_R1.bw',\n",
       " '../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_VHL-IRF8_R1.bw',\n",
       " '../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_VHL-MED1_R1.bw',\n",
       " '../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_VHL-MEF2C_R1.bw',\n",
       " '../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_VHL-MEF2D_R1.bw',\n",
       " '../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_VHL-MYC_R1.bw',\n",
       " '../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_VHL-POLII_total_R1.bw']"
      ]
     },
     "execution_count": 266,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bw = ! ls ../../data/$project/recalib_bigwig/*\n",
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 261,
   "metadata": {},
   "outputs": [],
   "source": [
    "!mkdir ../results/$project/\n",
    "!mkdir ../results/$project/plots\n",
    "!mkdir ../results/$project/plots/heatmaps/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 262,
   "metadata": {},
   "outputs": [],
   "source": [
    "# GENOME WIDE comparison"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 263,
   "metadata": {},
   "outputs": [],
   "source": [
    "peaks = ! ls ../../data/$project/bwa/mergedLibrary/macs/broadPeak/*.broadPeak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 268,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"IRF8\",\"MED1\",\"MEF2C\",\"MEF2D\",\"MYC\",\"POLII_total\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 270,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "MED1_DMSO\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_DMSO-MED1_R1.bw  --referencePoint center --regionsFileName ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-MED1_R1_peaks.broadPeak --missingDataAsZero --outFileName ../../data/chipseq_IRF8_degraded/MED1_DMSO_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/chipseq_IRF8_degraded/MED1_DMSO_mat.gz --outFileName ../../data/chipseq_IRF8_degraded/MED1_DMSO_mat.pdf --refPointLabel center --legendLocation lower-left --regionsLabel Macs2_Peaks --plotTitle MED1_DMSO', returncode=0, stdout=b'', stderr=b'')\n",
      "MEF2C_DMSO\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_DMSO-MEF2C_R1.bw  --referencePoint center --regionsFileName ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-MEF2C_R1_peaks.broadPeak --missingDataAsZero --outFileName ../../data/chipseq_IRF8_degraded/MEF2C_DMSO_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/chipseq_IRF8_degraded/MEF2C_DMSO_mat.gz --outFileName ../../data/chipseq_IRF8_degraded/MEF2C_DMSO_mat.pdf --refPointLabel center --legendLocation lower-left --regionsLabel Macs2_Peaks --plotTitle MEF2C_DMSO', returncode=0, stdout=b'', stderr=b'')\n",
      "MEF2D_DMSO\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_DMSO-MEF2D_R1.bw  --referencePoint center --regionsFileName ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-MEF2D_R1_peaks.broadPeak --missingDataAsZero --outFileName ../../data/chipseq_IRF8_degraded/MEF2D_DMSO_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/chipseq_IRF8_degraded/MEF2D_DMSO_mat.gz --outFileName ../../data/chipseq_IRF8_degraded/MEF2D_DMSO_mat.pdf --refPointLabel center --legendLocation lower-left --regionsLabel Macs2_Peaks --plotTitle MEF2D_DMSO', returncode=0, stdout=b'', stderr=b'')\n",
      "MYC_DMSO\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_DMSO-MYC_R1.bw  --referencePoint center --regionsFileName ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-MYC_R1_peaks.broadPeak --missingDataAsZero --outFileName ../../data/chipseq_IRF8_degraded/MYC_DMSO_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/chipseq_IRF8_degraded/MYC_DMSO_mat.gz --outFileName ../../data/chipseq_IRF8_degraded/MYC_DMSO_mat.pdf --refPointLabel center --legendLocation lower-left --regionsLabel Macs2_Peaks --plotTitle MYC_DMSO', returncode=0, stdout=b'', stderr=b'')\n",
      "POLII_total_DMSO\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_DMSO-POLII_total_R1.bw  --referencePoint center --regionsFileName ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_DMSO-POLII_total_R1_peaks.broadPeak --missingDataAsZero --outFileName ../../data/chipseq_IRF8_degraded/POLII_total_DMSO_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/chipseq_IRF8_degraded/POLII_total_DMSO_mat.gz --outFileName ../../data/chipseq_IRF8_degraded/POLII_total_DMSO_mat.pdf --refPointLabel center --legendLocation lower-left --regionsLabel Macs2_Peaks --plotTitle POLII_total_DMSO', returncode=0, stdout=b'', stderr=b'')\n",
      "IRF8_wIRF8ko\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_VHL-IRF8_R1.bw  --referencePoint center --regionsFileName ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-IRF8_R1_peaks.broadPeak --missingDataAsZero --outFileName ../../data/chipseq_IRF8_degraded/IRF8_wIRF8ko_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/chipseq_IRF8_degraded/IRF8_wIRF8ko_mat.gz --outFileName ../../data/chipseq_IRF8_degraded/IRF8_wIRF8ko_mat.pdf --refPointLabel center --legendLocation lower-left --regionsLabel Macs2_Peaks --plotTitle IRF8_wIRF8ko', returncode=0, stdout=b'', stderr=b'')\n",
      "MED1_wIRF8ko\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_VHL-MED1_R1.bw  --referencePoint center --regionsFileName ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-MED1_R1_peaks.broadPeak --missingDataAsZero --outFileName ../../data/chipseq_IRF8_degraded/MED1_wIRF8ko_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/chipseq_IRF8_degraded/MED1_wIRF8ko_mat.gz --outFileName ../../data/chipseq_IRF8_degraded/MED1_wIRF8ko_mat.pdf --refPointLabel center --legendLocation lower-left --regionsLabel Macs2_Peaks --plotTitle MED1_wIRF8ko', returncode=0, stdout=b'', stderr=b'')\n",
      "MEF2C_wIRF8ko\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_VHL-MEF2C_R1.bw  --referencePoint center --regionsFileName ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-MEF2C_R1_peaks.broadPeak --missingDataAsZero --outFileName ../../data/chipseq_IRF8_degraded/MEF2C_wIRF8ko_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/chipseq_IRF8_degraded/MEF2C_wIRF8ko_mat.gz --outFileName ../../data/chipseq_IRF8_degraded/MEF2C_wIRF8ko_mat.pdf --refPointLabel center --legendLocation lower-left --regionsLabel Macs2_Peaks --plotTitle MEF2C_wIRF8ko', returncode=0, stdout=b'', stderr=b'')\n",
      "MEF2D_wIRF8ko\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_VHL-MEF2D_R1.bw  --referencePoint center --regionsFileName ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-MEF2D_R1_peaks.broadPeak --missingDataAsZero --outFileName ../../data/chipseq_IRF8_degraded/MEF2D_wIRF8ko_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/chipseq_IRF8_degraded/MEF2D_wIRF8ko_mat.gz --outFileName ../../data/chipseq_IRF8_degraded/MEF2D_wIRF8ko_mat.pdf --refPointLabel center --legendLocation lower-left --regionsLabel Macs2_Peaks --plotTitle MEF2D_wIRF8ko', returncode=0, stdout=b'', stderr=b'')\n",
      "MYC_wIRF8ko\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_VHL-MYC_R1.bw  --referencePoint center --regionsFileName ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-MYC_R1_peaks.broadPeak --missingDataAsZero --outFileName ../../data/chipseq_IRF8_degraded/MYC_wIRF8ko_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/chipseq_IRF8_degraded/MYC_wIRF8ko_mat.gz --outFileName ../../data/chipseq_IRF8_degraded/MYC_wIRF8ko_mat.pdf --refPointLabel center --legendLocation lower-left --regionsLabel Macs2_Peaks --plotTitle MYC_wIRF8ko', returncode=0, stdout=b'', stderr=b'')\n",
      "POLII_total_wIRF8ko\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_VHL-POLII_total_R1.bw  --referencePoint center --regionsFileName ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/macs/broadPeak/MV411_IRF8_NT_VHL-POLII_total_R1_peaks.broadPeak --missingDataAsZero --outFileName ../../data/chipseq_IRF8_degraded/POLII_total_wIRF8ko_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/chipseq_IRF8_degraded/POLII_total_wIRF8ko_mat.gz --outFileName ../../data/chipseq_IRF8_degraded/POLII_total_wIRF8ko_mat.pdf --refPointLabel center --legendLocation lower-left --regionsLabel Macs2_Peaks --plotTitle POLII_total_wIRF8ko', returncode=0, stdout=b'', stderr=b'')\n"
     ]
    }
   ],
   "source": [
    "for i, val in enumerate(bw):\n",
    "    if i <1:\n",
    "        continue\n",
    "    name = names[i-6]+'_wIRF8ko' if i//6 else names[i]+'_DMSO'\n",
    "    print(name)\n",
    "    chip.getPeaksAt(peaks[i], bigwigs = val, bigwignames= name, peaknames=['Macs2_Peaks'], window=3000, folder=\"\", title=name, numthreads=8, refpoint=\"center\", name='../../data/'+project+'/'+name+'_mat.pdf', withDeeptools=True, torecompute=True, legendLoc=\"lower-left\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 271,
   "metadata": {},
   "outputs": [],
   "source": [
    "h.createFoldersFor('../results/'+project+'/plots/scaled/heatmaps/')\n",
    "! cp ../../data/$project/*.pdf ../results/$project/plots/scaled/heatmaps/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 272,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-IRF8_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-IRF8_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MED1_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MED1_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MEF2C_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2C_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MEF2D_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2D_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MYC_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MYC_R1_treat_pileup_c3.0_cond1.bed',\n",
       " '../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-POLII_total_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-POLII_total_R1_treat_pileup_c3.0_cond1.bed']"
      ]
     },
     "execution_count": 272,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "cond1peak = ! ls ../../data/$project/diffPeaks/*cond1.bed\n",
    "cond2peak = ! ls ../../data/$project/diffPeaks/*cond2.bed\n",
    "commonpeak = ! ls ../../data/$project/diffPeaks/*common.bed\n",
    "cond1peak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 273,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_DMSO-IRF8_R1.bw ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_VHL-IRF8_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_DMSO-IRF8_R1.bw ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_VHL-IRF8_R1.bw  --referencePoint center --regionsFileName ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-IRF8_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-IRF8_R1_treat_pileup_c3.0_cond1.bed ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-IRF8_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-IRF8_R1_treat_pileup_c3.0_common.bed ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-IRF8_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-IRF8_R1_treat_pileup_c3.0_cond2.bed --missingDataAsZero --outFileName ../../data/chipseq_IRF8_degraded/diffPeaks/IRF8_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/chipseq_IRF8_degraded/diffPeaks/IRF8_mat.gz --outFileName ../../data/chipseq_IRF8_degraded/diffPeaks/IRF8_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks --plotTitle IRF8', returncode=0, stdout=b'', stderr=b'')\n",
      "../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_DMSO-MED1_R1.bw ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_VHL-MED1_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_DMSO-MED1_R1.bw ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_VHL-MED1_R1.bw  --referencePoint center --regionsFileName ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MED1_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MED1_R1_treat_pileup_c3.0_cond1.bed ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MED1_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MED1_R1_treat_pileup_c3.0_common.bed ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MED1_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MED1_R1_treat_pileup_c3.0_cond2.bed --missingDataAsZero --outFileName ../../data/chipseq_IRF8_degraded/diffPeaks/MED1_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/chipseq_IRF8_degraded/diffPeaks/MED1_mat.gz --outFileName ../../data/chipseq_IRF8_degraded/diffPeaks/MED1_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks --plotTitle MED1', returncode=0, stdout=b'', stderr=b'')\n",
      "../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_DMSO-MEF2C_R1.bw ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_VHL-MEF2C_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_DMSO-MEF2C_R1.bw ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_VHL-MEF2C_R1.bw  --referencePoint center --regionsFileName ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MEF2C_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2C_R1_treat_pileup_c3.0_cond1.bed ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MEF2C_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2C_R1_treat_pileup_c3.0_common.bed ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MEF2C_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2C_R1_treat_pileup_c3.0_cond2.bed --missingDataAsZero --outFileName ../../data/chipseq_IRF8_degraded/diffPeaks/MEF2C_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/chipseq_IRF8_degraded/diffPeaks/MEF2C_mat.gz --outFileName ../../data/chipseq_IRF8_degraded/diffPeaks/MEF2C_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks --plotTitle MEF2C', returncode=0, stdout=b'', stderr=b'')\n",
      "../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_DMSO-MEF2D_R1.bw ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_VHL-MEF2D_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_DMSO-MEF2D_R1.bw ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_VHL-MEF2D_R1.bw  --referencePoint center --regionsFileName ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MEF2D_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2D_R1_treat_pileup_c3.0_cond1.bed ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MEF2D_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2D_R1_treat_pileup_c3.0_common.bed --missingDataAsZero --outFileName ../../data/chipseq_IRF8_degraded/diffPeaks/MEF2D_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/chipseq_IRF8_degraded/diffPeaks/MEF2D_mat.gz --outFileName ../../data/chipseq_IRF8_degraded/diffPeaks/MEF2D_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common --plotTitle MEF2D', returncode=0, stdout=b'', stderr=b'')\n",
      "../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_DMSO-MYC_R1.bw ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_VHL-MYC_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_DMSO-MYC_R1.bw ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_VHL-MYC_R1.bw  --referencePoint center --regionsFileName ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MYC_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MYC_R1_treat_pileup_c3.0_cond1.bed ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MYC_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MYC_R1_treat_pileup_c3.0_common.bed ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MYC_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MYC_R1_treat_pileup_c3.0_cond2.bed --missingDataAsZero --outFileName ../../data/chipseq_IRF8_degraded/diffPeaks/MYC_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/chipseq_IRF8_degraded/diffPeaks/MYC_mat.gz --outFileName ../../data/chipseq_IRF8_degraded/diffPeaks/MYC_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks --plotTitle MYC', returncode=0, stdout=b'', stderr=b'')\n",
      "../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_DMSO-POLII_total_R1.bw ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_VHL-POLII_total_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_DMSO-POLII_total_R1.bw ../../data/chipseq_IRF8_degraded/recalib_bigwig/MV411_IRF8_NT_VHL-POLII_total_R1.bw  --referencePoint center --regionsFileName ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-POLII_total_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-POLII_total_R1_treat_pileup_c3.0_cond1.bed ../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-POLII_total_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-POLII_total_R1_treat_pileup_c3.0_common.bed --missingDataAsZero --outFileName ../../data/chipseq_IRF8_degraded/diffPeaks/POLII_total_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/chipseq_IRF8_degraded/diffPeaks/POLII_total_mat.gz --outFileName ../../data/chipseq_IRF8_degraded/diffPeaks/POLII_total_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common --plotTitle POLII_total', returncode=0, stdout=b'', stderr=b\"WARNING: Group 'MV411_IRF8_NT_DMSO-POLII_total_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-POLII_total_R1_treat_pileup_c3.0_common.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n"
     ]
    }
   ],
   "source": [
    "for i in range(int(len(bw)/2)):\n",
    "    if i <0:\n",
    "        continue\n",
    "    name1 = bw[i]\n",
    "    name2 = bw[i+6]\n",
    "    peak = [cond1peak[i], commonpeak[i], cond2peak[i]]\n",
    "   # for val in peak:\n",
    "      #  chip.dropWeirdChromosomes(val)\n",
    "    name = names[i]\n",
    "    print(name1,name2)\n",
    "    chip.getPeaksAt(peak, [name1, name2], bigwignames=['DMSO', 'VHL'], peaknames=['DMSO_peaks', 'common', 'VHL_peaks'], window=3000, folder=\"\", title=name, numthreads=8, refpoint=\"center\", name='../../data/'+project+'/diffPeaks/'+name+'_mat.pdf', withDeeptools=True, torecompute=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 274,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mkdir ../results/$project/plots/scaled/diffPeaks/\n",
    "! cp ../../data/$project/diffPeaks/*.pdf ../results/$project/plots/scaled/diffPeaks/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### on unscalled data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 275,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mkdir ../../data/$project/diffPeaks_unscaled"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 276,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-POLII_total_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam and ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-POLII_total_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "183\n",
      "computing the scaling values\n",
      "b'INFO  @ Thu, 05 Nov 2020 19:18:41: \\n# Command line: callpeak -B -t ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam -c ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 183 -n INPUT_R1 --outdir ../../data/chipseq_IRF8_degraded/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = INPUT_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Thu, 05 Nov 2020 19:18:41: #1 read tag files... \\nINFO  @ Thu, 05 Nov 2020 19:18:41: #1 read treatment tags... \\nINFO  @ Thu, 05 Nov 2020 19:18:45:  1000000 \\nINFO  @ Thu, 05 Nov 2020 19:18:48:  2000000 \\nINFO  @ Thu, 05 Nov 2020 19:18:52:  3000000 \\nINFO  @ Thu, 05 Nov 2020 19:18:55:  4000000 \\nINFO  @ Thu, 05 Nov 2020 19:18:59:  5000000 \\nINFO  @ Thu, 05 Nov 2020 19:19:03:  6000000 \\nINFO  @ Thu, 05 Nov 2020 19:19:06:  7000000 \\nINFO  @ Thu, 05 Nov 2020 19:19:10:  8000000 \\nINFO  @ Thu, 05 Nov 2020 19:19:13:  9000000 \\nINFO  @ Thu, 05 Nov 2020 19:19:17:  10000000 \\nINFO  @ Thu, 05 Nov 2020 19:19:20:  11000000 \\nINFO  @ Thu, 05 Nov 2020 19:19:24:  12000000 \\nINFO  @ Thu, 05 Nov 2020 19:19:28:  13000000 \\nINFO  @ Thu, 05 Nov 2020 19:19:31:  14000000 \\nINFO  @ Thu, 05 Nov 2020 19:19:35:  15000000 \\nINFO  @ Thu, 05 Nov 2020 19:19:38:  16000000 \\nINFO  @ Thu, 05 Nov 2020 19:19:42:  17000000 \\nINFO  @ Thu, 05 Nov 2020 19:19:46:  18000000 \\nINFO  @ Thu, 05 Nov 2020 19:19:49:  19000000 \\nINFO  @ Thu, 05 Nov 2020 19:19:53:  20000000 \\nINFO  @ Thu, 05 Nov 2020 19:19:56:  21000000 \\nINFO  @ Thu, 05 Nov 2020 19:20:00:  22000000 \\nINFO  @ Thu, 05 Nov 2020 19:20:04:  23000000 \\nINFO  @ Thu, 05 Nov 2020 19:20:07:  24000000 \\nINFO  @ Thu, 05 Nov 2020 19:20:11:  25000000 \\nINFO  @ Thu, 05 Nov 2020 19:20:15:  26000000 \\nINFO  @ Thu, 05 Nov 2020 19:20:18:  27000000 \\nINFO  @ Thu, 05 Nov 2020 19:20:22:  28000000 \\nINFO  @ Thu, 05 Nov 2020 19:20:25:  29000000 \\nINFO  @ Thu, 05 Nov 2020 19:20:29:  30000000 \\nINFO  @ Thu, 05 Nov 2020 19:20:33:  31000000 \\nINFO  @ Thu, 05 Nov 2020 19:20:37:  32000000 \\nINFO  @ Thu, 05 Nov 2020 19:20:40:  33000000 \\nINFO  @ Thu, 05 Nov 2020 19:20:44:  34000000 \\nINFO  @ Thu, 05 Nov 2020 19:20:47:  35000000 \\nINFO  @ Thu, 05 Nov 2020 19:20:51:  36000000 \\nINFO  @ Thu, 05 Nov 2020 19:20:55:  37000000 \\nINFO  @ Thu, 05 Nov 2020 19:20:58:  38000000 \\nINFO  @ Thu, 05 Nov 2020 19:21:02:  39000000 \\nINFO  @ Thu, 05 Nov 2020 19:21:05:  40000000 \\nINFO  @ Thu, 05 Nov 2020 19:21:09:  41000000 \\nINFO  @ Thu, 05 Nov 2020 19:21:13:  42000000 \\nINFO  @ Thu, 05 Nov 2020 19:21:16:  43000000 \\nINFO  @ Thu, 05 Nov 2020 19:21:20:  44000000 \\nINFO  @ Thu, 05 Nov 2020 19:21:24:  45000000 \\nINFO  @ Thu, 05 Nov 2020 19:21:27:  46000000 \\nINFO  @ Thu, 05 Nov 2020 19:21:31:  47000000 \\nINFO  @ Thu, 05 Nov 2020 19:21:34:  48000000 \\nINFO  @ Thu, 05 Nov 2020 19:21:38:  49000000 \\nINFO  @ Thu, 05 Nov 2020 19:21:41:  50000000 \\nINFO  @ Thu, 05 Nov 2020 19:21:45:  51000000 \\nINFO  @ Thu, 05 Nov 2020 19:21:49:  52000000 \\nINFO  @ Thu, 05 Nov 2020 19:21:52:  53000000 \\nINFO  @ Thu, 05 Nov 2020 19:21:56:  54000000 \\nINFO  @ Thu, 05 Nov 2020 19:22:00:  55000000 \\nINFO  @ Thu, 05 Nov 2020 19:22:04:  56000000 \\nINFO  @ Thu, 05 Nov 2020 19:22:08:  57000000 \\nINFO  @ Thu, 05 Nov 2020 19:22:12:  58000000 \\nINFO  @ Thu, 05 Nov 2020 19:22:15:  59000000 \\nINFO  @ Thu, 05 Nov 2020 19:22:19:  60000000 \\nINFO  @ Thu, 05 Nov 2020 19:22:23:  61000000 \\nINFO  @ Thu, 05 Nov 2020 19:22:26:  62000000 \\nINFO  @ Thu, 05 Nov 2020 19:22:30:  63000000 \\nINFO  @ Thu, 05 Nov 2020 19:22:33:  64000000 \\nINFO  @ Thu, 05 Nov 2020 19:22:37:  65000000 \\nINFO  @ Thu, 05 Nov 2020 19:22:41:  66000000 \\nINFO  @ Thu, 05 Nov 2020 19:22:43: 66660890 reads have been read. \\nINFO  @ Thu, 05 Nov 2020 19:22:44: #1.2 read input tags... \\nINFO  @ Thu, 05 Nov 2020 19:22:47:  1000000 \\nINFO  @ Thu, 05 Nov 2020 19:22:51:  2000000 \\nINFO  @ Thu, 05 Nov 2020 19:22:55:  3000000 \\nINFO  @ Thu, 05 Nov 2020 19:22:58:  4000000 \\nINFO  @ Thu, 05 Nov 2020 19:23:02:  5000000 \\nINFO  @ Thu, 05 Nov 2020 19:23:05:  6000000 \\nINFO  @ Thu, 05 Nov 2020 19:23:09:  7000000 \\nINFO  @ Thu, 05 Nov 2020 19:23:12:  8000000 \\nINFO  @ Thu, 05 Nov 2020 19:23:16:  9000000 \\nINFO  @ Thu, 05 Nov 2020 19:23:20:  10000000 \\nINFO  @ Thu, 05 Nov 2020 19:23:23:  11000000 \\nINFO  @ Thu, 05 Nov 2020 19:23:27:  12000000 \\nINFO  @ Thu, 05 Nov 2020 19:23:30:  13000000 \\nINFO  @ Thu, 05 Nov 2020 19:23:34:  14000000 \\nINFO  @ Thu, 05 Nov 2020 19:23:37:  15000000 \\nINFO  @ Thu, 05 Nov 2020 19:23:41:  16000000 \\nINFO  @ Thu, 05 Nov 2020 19:23:45:  17000000 \\nINFO  @ Thu, 05 Nov 2020 19:23:48:  18000000 \\nINFO  @ Thu, 05 Nov 2020 19:23:52:  19000000 \\nINFO  @ Thu, 05 Nov 2020 19:23:55:  20000000 \\nINFO  @ Thu, 05 Nov 2020 19:23:59:  21000000 \\nINFO  @ Thu, 05 Nov 2020 19:24:02:  22000000 \\nINFO  @ Thu, 05 Nov 2020 19:24:06:  23000000 \\nINFO  @ Thu, 05 Nov 2020 19:24:10:  24000000 \\nINFO  @ Thu, 05 Nov 2020 19:24:13:  25000000 \\nINFO  @ Thu, 05 Nov 2020 19:24:17:  26000000 \\nINFO  @ Thu, 05 Nov 2020 19:24:21:  27000000 \\nINFO  @ Thu, 05 Nov 2020 19:24:24:  28000000 \\nINFO  @ Thu, 05 Nov 2020 19:24:28:  29000000 \\nINFO  @ Thu, 05 Nov 2020 19:24:31:  30000000 \\nINFO  @ Thu, 05 Nov 2020 19:24:35:  31000000 \\nINFO  @ Thu, 05 Nov 2020 19:24:38:  32000000 \\nINFO  @ Thu, 05 Nov 2020 19:24:42:  33000000 \\nINFO  @ Thu, 05 Nov 2020 19:24:46:  34000000 \\nINFO  @ Thu, 05 Nov 2020 19:24:49:  35000000 \\nINFO  @ Thu, 05 Nov 2020 19:24:53:  36000000 \\nINFO  @ Thu, 05 Nov 2020 19:24:56:  37000000 \\nINFO  @ Thu, 05 Nov 2020 19:25:00:  38000000 \\nINFO  @ Thu, 05 Nov 2020 19:25:03:  39000000 \\nINFO  @ Thu, 05 Nov 2020 19:25:07:  40000000 \\nINFO  @ Thu, 05 Nov 2020 19:25:11:  41000000 \\nINFO  @ Thu, 05 Nov 2020 19:25:14:  42000000 \\nINFO  @ Thu, 05 Nov 2020 19:25:18:  43000000 \\nINFO  @ Thu, 05 Nov 2020 19:25:21:  44000000 \\nINFO  @ Thu, 05 Nov 2020 19:25:25:  45000000 \\nINFO  @ Thu, 05 Nov 2020 19:25:29:  46000000 \\nINFO  @ Thu, 05 Nov 2020 19:25:32:  47000000 \\nINFO  @ Thu, 05 Nov 2020 19:25:36:  48000000 \\nINFO  @ Thu, 05 Nov 2020 19:25:39:  49000000 \\nINFO  @ Thu, 05 Nov 2020 19:25:43:  50000000 \\nINFO  @ Thu, 05 Nov 2020 19:25:46:  51000000 \\nINFO  @ Thu, 05 Nov 2020 19:25:50:  52000000 \\nINFO  @ Thu, 05 Nov 2020 19:25:54:  53000000 \\nINFO  @ Thu, 05 Nov 2020 19:25:57:  54000000 \\nINFO  @ Thu, 05 Nov 2020 19:26:01:  55000000 \\nINFO  @ Thu, 05 Nov 2020 19:26:04:  56000000 \\nINFO  @ Thu, 05 Nov 2020 19:26:09:  57000000 \\nINFO  @ Thu, 05 Nov 2020 19:26:13:  58000000 \\nINFO  @ Thu, 05 Nov 2020 19:26:16:  59000000 \\nINFO  @ Thu, 05 Nov 2020 19:26:20:  60000000 \\nINFO  @ Thu, 05 Nov 2020 19:26:24:  61000000 \\nINFO  @ Thu, 05 Nov 2020 19:26:27:  62000000 \\nINFO  @ Thu, 05 Nov 2020 19:26:31:  63000000 \\nINFO  @ Thu, 05 Nov 2020 19:26:35:  64000000 \\nINFO  @ Thu, 05 Nov 2020 19:26:38:  65000000 \\nINFO  @ Thu, 05 Nov 2020 19:26:42:  66000000 \\nINFO  @ Thu, 05 Nov 2020 19:26:44: 66660890 reads have been read. \\nINFO  @ Thu, 05 Nov 2020 19:26:45: #1 tag size is determined as 49 bps \\nINFO  @ Thu, 05 Nov 2020 19:26:45: #1 tag size = 49.0 \\nINFO  @ Thu, 05 Nov 2020 19:26:45: #1  total tags in treatment: 66660890 \\nINFO  @ Thu, 05 Nov 2020 19:26:45: #1 user defined the maximum tags... \\nINFO  @ Thu, 05 Nov 2020 19:26:45: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 05 Nov 2020 19:26:46: #1  tags after filtering in treatment: 65725426 \\nINFO  @ Thu, 05 Nov 2020 19:26:46: #1  Redundant rate of treatment: 0.01 \\nINFO  @ Thu, 05 Nov 2020 19:26:46: #1  total tags in control: 66660890 \\nINFO  @ Thu, 05 Nov 2020 19:26:46: #1 user defined the maximum tags... \\nINFO  @ Thu, 05 Nov 2020 19:26:46: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 05 Nov 2020 19:26:46: #1  tags after filtering in control: 65725426 \\nINFO  @ Thu, 05 Nov 2020 19:26:46: #1  Redundant rate of control: 0.01 \\nINFO  @ Thu, 05 Nov 2020 19:26:46: #1 finished! \\nINFO  @ Thu, 05 Nov 2020 19:26:46: #2 Build Peak Model... \\nINFO  @ Thu, 05 Nov 2020 19:26:46: #2 Skipped... \\nINFO  @ Thu, 05 Nov 2020 19:26:46: #2 Use 183 as fragment length \\nINFO  @ Thu, 05 Nov 2020 19:26:46: #3 Call peaks... \\nINFO  @ Thu, 05 Nov 2020 19:26:46: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 05 Nov 2020 19:32:33: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 05 Nov 2020 19:32:33: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... INPUT_R1_treat_pileup.bdg \\nINFO  @ Thu, 05 Nov 2020 19:32:33: #3   Write bedGraph files for control lambda (after scaling if necessary)... INPUT_R1_control_lambda.bdg \\nINFO  @ Thu, 05 Nov 2020 19:32:33: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 05 Nov 2020 19:32:33: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 05 Nov 2020 19:37:32: #4 Write output xls file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/INPUT_R1_peaks.xls \\nINFO  @ Thu, 05 Nov 2020 19:37:32: #4 Write peak in narrowPeak format file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/INPUT_R1_peaks.narrowPeak \\nINFO  @ Thu, 05 Nov 2020 19:37:32: #4 Write summits bed file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/INPUT_R1_summits.bed \\nINFO  @ Thu, 05 Nov 2020 19:37:32: Done! \\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'INFO  @ Thu, 05 Nov 2020 19:37:33: \\n# Command line: callpeak -B -t ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-POLII_total_R1.mLb.clN.sorted.bam -c ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 183 -n MV411_IRF8_NT_DMSO-POLII_total_R1 --outdir ../../data/chipseq_IRF8_degraded/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = MV411_IRF8_NT_DMSO-POLII_total_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-POLII_total_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Thu, 05 Nov 2020 19:37:33: #1 read tag files... \\nINFO  @ Thu, 05 Nov 2020 19:37:33: #1 read treatment tags... \\nINFO  @ Thu, 05 Nov 2020 19:37:38:  1000000 \\nINFO  @ Thu, 05 Nov 2020 19:37:42:  2000000 \\nINFO  @ Thu, 05 Nov 2020 19:37:47:  3000000 \\nINFO  @ Thu, 05 Nov 2020 19:37:52:  4000000 \\nINFO  @ Thu, 05 Nov 2020 19:37:57:  5000000 \\nINFO  @ Thu, 05 Nov 2020 19:38:02:  6000000 \\nINFO  @ Thu, 05 Nov 2020 19:38:07:  7000000 \\nINFO  @ Thu, 05 Nov 2020 19:38:11:  8000000 \\nINFO  @ Thu, 05 Nov 2020 19:38:16:  9000000 \\nINFO  @ Thu, 05 Nov 2020 19:38:22:  10000000 \\nINFO  @ Thu, 05 Nov 2020 19:38:27:  11000000 \\nINFO  @ Thu, 05 Nov 2020 19:38:31:  12000000 \\nINFO  @ Thu, 05 Nov 2020 19:38:36:  13000000 \\nINFO  @ Thu, 05 Nov 2020 19:38:41:  14000000 \\nINFO  @ Thu, 05 Nov 2020 19:38:46:  15000000 \\nINFO  @ Thu, 05 Nov 2020 19:38:50:  16000000 \\nINFO  @ Thu, 05 Nov 2020 19:38:55:  17000000 \\nINFO  @ Thu, 05 Nov 2020 19:39:00:  18000000 \\nINFO  @ Thu, 05 Nov 2020 19:39:04:  19000000 \\nINFO  @ Thu, 05 Nov 2020 19:39:05: 19119331 reads have been read. \\nINFO  @ Thu, 05 Nov 2020 19:39:05: #1.2 read input tags... \\nINFO  @ Thu, 05 Nov 2020 19:39:09:  1000000 \\nINFO  @ Thu, 05 Nov 2020 19:39:13:  2000000 \\nINFO  @ Thu, 05 Nov 2020 19:39:16:  3000000 \\nINFO  @ Thu, 05 Nov 2020 19:39:20:  4000000 \\nINFO  @ Thu, 05 Nov 2020 19:39:24:  5000000 \\nINFO  @ Thu, 05 Nov 2020 19:39:27:  6000000 \\nINFO  @ Thu, 05 Nov 2020 19:39:31:  7000000 \\nINFO  @ Thu, 05 Nov 2020 19:39:35:  8000000 \\nINFO  @ Thu, 05 Nov 2020 19:39:38:  9000000 \\nINFO  @ Thu, 05 Nov 2020 19:39:42:  10000000 \\nINFO  @ Thu, 05 Nov 2020 19:39:46:  11000000 \\nINFO  @ Thu, 05 Nov 2020 19:39:49:  12000000 \\nINFO  @ Thu, 05 Nov 2020 19:39:53:  13000000 \\nINFO  @ Thu, 05 Nov 2020 19:39:56:  14000000 \\nINFO  @ Thu, 05 Nov 2020 19:40:00:  15000000 \\nINFO  @ Thu, 05 Nov 2020 19:40:04:  16000000 \\nINFO  @ Thu, 05 Nov 2020 19:40:07:  17000000 \\nINFO  @ Thu, 05 Nov 2020 19:40:11:  18000000 \\nINFO  @ Thu, 05 Nov 2020 19:40:15:  19000000 \\nINFO  @ Thu, 05 Nov 2020 19:40:18:  20000000 \\nINFO  @ Thu, 05 Nov 2020 19:40:22:  21000000 \\nINFO  @ Thu, 05 Nov 2020 19:40:26:  22000000 \\nINFO  @ Thu, 05 Nov 2020 19:40:29:  23000000 \\nINFO  @ Thu, 05 Nov 2020 19:40:33:  24000000 \\nINFO  @ Thu, 05 Nov 2020 19:40:37:  25000000 \\nINFO  @ Thu, 05 Nov 2020 19:40:41:  26000000 \\nINFO  @ Thu, 05 Nov 2020 19:40:44:  27000000 \\nINFO  @ Thu, 05 Nov 2020 19:40:48:  28000000 \\nINFO  @ Thu, 05 Nov 2020 19:40:52:  29000000 \\nINFO  @ Thu, 05 Nov 2020 19:40:55:  30000000 \\nINFO  @ Thu, 05 Nov 2020 19:40:59:  31000000 \\nINFO  @ Thu, 05 Nov 2020 19:41:03:  32000000 \\nINFO  @ Thu, 05 Nov 2020 19:41:06:  33000000 \\nINFO  @ Thu, 05 Nov 2020 19:41:10:  34000000 \\nINFO  @ Thu, 05 Nov 2020 19:41:14:  35000000 \\nINFO  @ Thu, 05 Nov 2020 19:41:17:  36000000 \\nINFO  @ Thu, 05 Nov 2020 19:41:21:  37000000 \\nINFO  @ Thu, 05 Nov 2020 19:41:24:  38000000 \\nINFO  @ Thu, 05 Nov 2020 19:41:28:  39000000 \\nINFO  @ Thu, 05 Nov 2020 19:41:32:  40000000 \\nINFO  @ Thu, 05 Nov 2020 19:41:35:  41000000 \\nINFO  @ Thu, 05 Nov 2020 19:41:39:  42000000 \\nINFO  @ Thu, 05 Nov 2020 19:41:43:  43000000 \\nINFO  @ Thu, 05 Nov 2020 19:41:46:  44000000 \\nINFO  @ Thu, 05 Nov 2020 19:41:50:  45000000 \\nINFO  @ Thu, 05 Nov 2020 19:41:54:  46000000 \\nINFO  @ Thu, 05 Nov 2020 19:41:57:  47000000 \\nINFO  @ Thu, 05 Nov 2020 19:42:01:  48000000 \\nINFO  @ Thu, 05 Nov 2020 19:42:05:  49000000 \\nINFO  @ Thu, 05 Nov 2020 19:42:08:  50000000 \\nINFO  @ Thu, 05 Nov 2020 19:42:12:  51000000 \\nINFO  @ Thu, 05 Nov 2020 19:42:16:  52000000 \\nINFO  @ Thu, 05 Nov 2020 19:42:20:  53000000 \\nINFO  @ Thu, 05 Nov 2020 19:42:24:  54000000 \\nINFO  @ Thu, 05 Nov 2020 19:42:28:  55000000 \\nINFO  @ Thu, 05 Nov 2020 19:42:32:  56000000 \\nINFO  @ Thu, 05 Nov 2020 19:42:35:  57000000 \\nINFO  @ Thu, 05 Nov 2020 19:42:39:  58000000 \\nINFO  @ Thu, 05 Nov 2020 19:42:43:  59000000 \\nINFO  @ Thu, 05 Nov 2020 19:42:47:  60000000 \\nINFO  @ Thu, 05 Nov 2020 19:42:51:  61000000 \\nINFO  @ Thu, 05 Nov 2020 19:42:54:  62000000 \\nINFO  @ Thu, 05 Nov 2020 19:42:58:  63000000 \\nINFO  @ Thu, 05 Nov 2020 19:43:02:  64000000 \\nINFO  @ Thu, 05 Nov 2020 19:43:05:  65000000 \\nINFO  @ Thu, 05 Nov 2020 19:43:09:  66000000 \\nINFO  @ Thu, 05 Nov 2020 19:43:12: 66660890 reads have been read. \\nINFO  @ Thu, 05 Nov 2020 19:43:12: #1 tag size is determined as 90 bps \\nINFO  @ Thu, 05 Nov 2020 19:43:12: #1 tag size = 90.0 \\nINFO  @ Thu, 05 Nov 2020 19:43:12: #1  total tags in treatment: 19119331 \\nINFO  @ Thu, 05 Nov 2020 19:43:12: #1 user defined the maximum tags... \\nINFO  @ Thu, 05 Nov 2020 19:43:12: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 05 Nov 2020 19:43:12: #1  tags after filtering in treatment: 18826876 \\nINFO  @ Thu, 05 Nov 2020 19:43:12: #1  Redundant rate of treatment: 0.02 \\nINFO  @ Thu, 05 Nov 2020 19:43:12: #1  total tags in control: 66660890 \\nINFO  @ Thu, 05 Nov 2020 19:43:12: #1 user defined the maximum tags... \\nINFO  @ Thu, 05 Nov 2020 19:43:12: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 05 Nov 2020 19:43:13: #1  tags after filtering in control: 65725426 \\nINFO  @ Thu, 05 Nov 2020 19:43:13: #1  Redundant rate of control: 0.01 \\nINFO  @ Thu, 05 Nov 2020 19:43:13: #1 finished! \\nINFO  @ Thu, 05 Nov 2020 19:43:13: #2 Build Peak Model... \\nINFO  @ Thu, 05 Nov 2020 19:43:13: #2 Skipped... \\nINFO  @ Thu, 05 Nov 2020 19:43:13: #2 Use 183 as fragment length \\nINFO  @ Thu, 05 Nov 2020 19:43:13: #3 Call peaks... \\nINFO  @ Thu, 05 Nov 2020 19:43:13: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 05 Nov 2020 19:47:59: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 05 Nov 2020 19:47:59: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_IRF8_NT_DMSO-POLII_total_R1_treat_pileup.bdg \\nINFO  @ Thu, 05 Nov 2020 19:47:59: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_IRF8_NT_DMSO-POLII_total_R1_control_lambda.bdg \\nINFO  @ Thu, 05 Nov 2020 19:47:59: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 05 Nov 2020 19:47:59: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 05 Nov 2020 19:51:31: #4 Write output xls file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-POLII_total_R1_peaks.xls \\nINFO  @ Thu, 05 Nov 2020 19:51:31: #4 Write peak in narrowPeak format file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-POLII_total_R1_peaks.narrowPeak \\nINFO  @ Thu, 05 Nov 2020 19:51:31: #4 Write summits bed file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-POLII_total_R1_summits.bed \\nINFO  @ Thu, 05 Nov 2020 19:51:31: Done! \\n'\n",
      "65725426 18826876\n",
      "doing differential peak binding\n",
      "../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-IRF8_R1.mLb.clN.sorted.bam ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-IRF8_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-IRF8_R1.mLb.clN.sorted.bam and ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-IRF8_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "200\n",
      "computing the scaling values\n",
      "b'INFO  @ Thu, 05 Nov 2020 20:06:14: \\n# Command line: callpeak -B -t ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-IRF8_R1.mLb.clN.sorted.bam -c ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 200 -n MV411_IRF8_NT_DMSO-IRF8_R1 --outdir ../../data/chipseq_IRF8_degraded/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = MV411_IRF8_NT_DMSO-IRF8_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-IRF8_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Thu, 05 Nov 2020 20:06:14: #1 read tag files... \\nINFO  @ Thu, 05 Nov 2020 20:06:14: #1 read treatment tags... \\nINFO  @ Thu, 05 Nov 2020 20:06:19:  1000000 \\nINFO  @ Thu, 05 Nov 2020 20:06:23:  2000000 \\nINFO  @ Thu, 05 Nov 2020 20:06:28:  3000000 \\nINFO  @ Thu, 05 Nov 2020 20:06:34:  4000000 \\nINFO  @ Thu, 05 Nov 2020 20:06:38:  5000000 \\nINFO  @ Thu, 05 Nov 2020 20:06:43:  6000000 \\nINFO  @ Thu, 05 Nov 2020 20:06:47:  7000000 \\nINFO  @ Thu, 05 Nov 2020 20:06:52:  8000000 \\nINFO  @ Thu, 05 Nov 2020 20:06:56:  9000000 \\nINFO  @ Thu, 05 Nov 2020 20:07:01:  10000000 \\nINFO  @ Thu, 05 Nov 2020 20:07:06:  11000000 \\nINFO  @ Thu, 05 Nov 2020 20:07:10:  12000000 \\nINFO  @ Thu, 05 Nov 2020 20:07:15:  13000000 \\nINFO  @ Thu, 05 Nov 2020 20:07:19:  14000000 \\nINFO  @ Thu, 05 Nov 2020 20:07:24:  15000000 \\nINFO  @ Thu, 05 Nov 2020 20:07:28:  16000000 \\nINFO  @ Thu, 05 Nov 2020 20:07:29: 16213604 reads have been read. \\nINFO  @ Thu, 05 Nov 2020 20:07:29: #1.2 read input tags... \\nINFO  @ Thu, 05 Nov 2020 20:07:33:  1000000 \\nINFO  @ Thu, 05 Nov 2020 20:07:37:  2000000 \\nINFO  @ Thu, 05 Nov 2020 20:07:41:  3000000 \\nINFO  @ Thu, 05 Nov 2020 20:07:44:  4000000 \\nINFO  @ Thu, 05 Nov 2020 20:07:48:  5000000 \\nINFO  @ Thu, 05 Nov 2020 20:07:52:  6000000 \\nINFO  @ Thu, 05 Nov 2020 20:07:55:  7000000 \\nINFO  @ Thu, 05 Nov 2020 20:07:59:  8000000 \\nINFO  @ Thu, 05 Nov 2020 20:08:03:  9000000 \\nINFO  @ Thu, 05 Nov 2020 20:08:06:  10000000 \\nINFO  @ Thu, 05 Nov 2020 20:08:10:  11000000 \\nINFO  @ Thu, 05 Nov 2020 20:08:14:  12000000 \\nINFO  @ Thu, 05 Nov 2020 20:08:18:  13000000 \\nINFO  @ Thu, 05 Nov 2020 20:08:21:  14000000 \\nINFO  @ Thu, 05 Nov 2020 20:08:25:  15000000 \\nINFO  @ Thu, 05 Nov 2020 20:08:28:  16000000 \\nINFO  @ Thu, 05 Nov 2020 20:08:32:  17000000 \\nINFO  @ Thu, 05 Nov 2020 20:08:36:  18000000 \\nINFO  @ Thu, 05 Nov 2020 20:08:40:  19000000 \\nINFO  @ Thu, 05 Nov 2020 20:08:43:  20000000 \\nINFO  @ Thu, 05 Nov 2020 20:08:47:  21000000 \\nINFO  @ Thu, 05 Nov 2020 20:08:51:  22000000 \\nINFO  @ Thu, 05 Nov 2020 20:08:55:  23000000 \\nINFO  @ Thu, 05 Nov 2020 20:08:58:  24000000 \\nINFO  @ Thu, 05 Nov 2020 20:09:02:  25000000 \\nINFO  @ Thu, 05 Nov 2020 20:09:06:  26000000 \\nINFO  @ Thu, 05 Nov 2020 20:09:09:  27000000 \\nINFO  @ Thu, 05 Nov 2020 20:09:13:  28000000 \\nINFO  @ Thu, 05 Nov 2020 20:09:16:  29000000 \\nINFO  @ Thu, 05 Nov 2020 20:09:20:  30000000 \\nINFO  @ Thu, 05 Nov 2020 20:09:24:  31000000 \\nINFO  @ Thu, 05 Nov 2020 20:09:28:  32000000 \\nINFO  @ Thu, 05 Nov 2020 20:09:31:  33000000 \\nINFO  @ Thu, 05 Nov 2020 20:09:35:  34000000 \\nINFO  @ Thu, 05 Nov 2020 20:09:39:  35000000 \\nINFO  @ Thu, 05 Nov 2020 20:09:42:  36000000 \\nINFO  @ Thu, 05 Nov 2020 20:09:46:  37000000 \\nINFO  @ Thu, 05 Nov 2020 20:09:49:  38000000 \\nINFO  @ Thu, 05 Nov 2020 20:09:53:  39000000 \\nINFO  @ Thu, 05 Nov 2020 20:09:57:  40000000 \\nINFO  @ Thu, 05 Nov 2020 20:10:00:  41000000 \\nINFO  @ Thu, 05 Nov 2020 20:10:04:  42000000 \\nINFO  @ Thu, 05 Nov 2020 20:10:08:  43000000 \\nINFO  @ Thu, 05 Nov 2020 20:10:11:  44000000 \\nINFO  @ Thu, 05 Nov 2020 20:10:15:  45000000 \\nINFO  @ Thu, 05 Nov 2020 20:10:19:  46000000 \\nINFO  @ Thu, 05 Nov 2020 20:10:22:  47000000 \\nINFO  @ Thu, 05 Nov 2020 20:10:26:  48000000 \\nINFO  @ Thu, 05 Nov 2020 20:10:30:  49000000 \\nINFO  @ Thu, 05 Nov 2020 20:10:33:  50000000 \\nINFO  @ Thu, 05 Nov 2020 20:10:38:  51000000 \\nINFO  @ Thu, 05 Nov 2020 20:10:42:  52000000 \\nINFO  @ Thu, 05 Nov 2020 20:10:46:  53000000 \\nINFO  @ Thu, 05 Nov 2020 20:10:49:  54000000 \\nINFO  @ Thu, 05 Nov 2020 20:10:53:  55000000 \\nINFO  @ Thu, 05 Nov 2020 20:10:57:  56000000 \\nINFO  @ Thu, 05 Nov 2020 20:11:00:  57000000 \\nINFO  @ Thu, 05 Nov 2020 20:11:04:  58000000 \\nINFO  @ Thu, 05 Nov 2020 20:11:08:  59000000 \\nINFO  @ Thu, 05 Nov 2020 20:11:12:  60000000 \\nINFO  @ Thu, 05 Nov 2020 20:11:15:  61000000 \\nINFO  @ Thu, 05 Nov 2020 20:11:19:  62000000 \\nINFO  @ Thu, 05 Nov 2020 20:11:23:  63000000 \\nINFO  @ Thu, 05 Nov 2020 20:11:27:  64000000 \\nINFO  @ Thu, 05 Nov 2020 20:11:30:  65000000 \\nINFO  @ Thu, 05 Nov 2020 20:11:34:  66000000 \\nINFO  @ Thu, 05 Nov 2020 20:11:37: 66660890 reads have been read. \\nINFO  @ Thu, 05 Nov 2020 20:11:37: #1 tag size is determined as 88 bps \\nINFO  @ Thu, 05 Nov 2020 20:11:37: #1 tag size = 88.0 \\nINFO  @ Thu, 05 Nov 2020 20:11:37: #1  total tags in treatment: 16213604 \\nINFO  @ Thu, 05 Nov 2020 20:11:37: #1 user defined the maximum tags... \\nINFO  @ Thu, 05 Nov 2020 20:11:37: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 05 Nov 2020 20:11:37: #1  tags after filtering in treatment: 15127980 \\nINFO  @ Thu, 05 Nov 2020 20:11:37: #1  Redundant rate of treatment: 0.07 \\nINFO  @ Thu, 05 Nov 2020 20:11:37: #1  total tags in control: 66660890 \\nINFO  @ Thu, 05 Nov 2020 20:11:37: #1 user defined the maximum tags... \\nINFO  @ Thu, 05 Nov 2020 20:11:37: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 05 Nov 2020 20:11:38: #1  tags after filtering in control: 65725426 \\nINFO  @ Thu, 05 Nov 2020 20:11:38: #1  Redundant rate of control: 0.01 \\nINFO  @ Thu, 05 Nov 2020 20:11:38: #1 finished! \\nINFO  @ Thu, 05 Nov 2020 20:11:38: #2 Build Peak Model... \\nINFO  @ Thu, 05 Nov 2020 20:11:38: #2 Skipped... \\nINFO  @ Thu, 05 Nov 2020 20:11:38: #2 Use 200 as fragment length \\nINFO  @ Thu, 05 Nov 2020 20:11:38: #3 Call peaks... \\nINFO  @ Thu, 05 Nov 2020 20:11:38: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 05 Nov 2020 20:16:04: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 05 Nov 2020 20:16:04: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_IRF8_NT_DMSO-IRF8_R1_treat_pileup.bdg \\nINFO  @ Thu, 05 Nov 2020 20:16:04: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_IRF8_NT_DMSO-IRF8_R1_control_lambda.bdg \\nINFO  @ Thu, 05 Nov 2020 20:16:04: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 05 Nov 2020 20:16:04: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 05 Nov 2020 20:19:39: #4 Write output xls file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-IRF8_R1_peaks.xls \\nINFO  @ Thu, 05 Nov 2020 20:19:40: #4 Write peak in narrowPeak format file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-IRF8_R1_peaks.narrowPeak \\nINFO  @ Thu, 05 Nov 2020 20:19:40: #4 Write summits bed file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-IRF8_R1_summits.bed \\nINFO  @ Thu, 05 Nov 2020 20:19:40: Done! \\n'\n",
      "b'INFO  @ Thu, 05 Nov 2020 20:19:40: \\n# Command line: callpeak -B -t ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-IRF8_R1.mLb.clN.sorted.bam -c ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 200 -n MV411_IRF8_NT_VHL-IRF8_R1 --outdir ../../data/chipseq_IRF8_degraded/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = MV411_IRF8_NT_VHL-IRF8_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-IRF8_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Thu, 05 Nov 2020 20:19:40: #1 read tag files... \\nINFO  @ Thu, 05 Nov 2020 20:19:40: #1 read treatment tags... \\nINFO  @ Thu, 05 Nov 2020 20:19:45:  1000000 \\nINFO  @ Thu, 05 Nov 2020 20:19:50:  2000000 \\nINFO  @ Thu, 05 Nov 2020 20:19:55:  3000000 \\nINFO  @ Thu, 05 Nov 2020 20:19:59:  4000000 \\nINFO  @ Thu, 05 Nov 2020 20:20:04:  5000000 \\nINFO  @ Thu, 05 Nov 2020 20:20:09:  6000000 \\nINFO  @ Thu, 05 Nov 2020 20:20:14:  7000000 \\nINFO  @ Thu, 05 Nov 2020 20:20:19:  8000000 \\nINFO  @ Thu, 05 Nov 2020 20:20:23:  9000000 \\nINFO  @ Thu, 05 Nov 2020 20:20:28:  10000000 \\nINFO  @ Thu, 05 Nov 2020 20:20:33:  11000000 \\nINFO  @ Thu, 05 Nov 2020 20:20:38:  12000000 \\nINFO  @ Thu, 05 Nov 2020 20:20:43:  13000000 \\nINFO  @ Thu, 05 Nov 2020 20:20:47:  14000000 \\nINFO  @ Thu, 05 Nov 2020 20:20:52:  15000000 \\nINFO  @ Thu, 05 Nov 2020 20:20:56: 15766460 reads have been read. \\nINFO  @ Thu, 05 Nov 2020 20:20:56: #1.2 read input tags... \\nINFO  @ Thu, 05 Nov 2020 20:21:00:  1000000 \\nINFO  @ Thu, 05 Nov 2020 20:21:03:  2000000 \\nINFO  @ Thu, 05 Nov 2020 20:21:07:  3000000 \\nINFO  @ Thu, 05 Nov 2020 20:21:11:  4000000 \\nINFO  @ Thu, 05 Nov 2020 20:21:14:  5000000 \\nINFO  @ Thu, 05 Nov 2020 20:21:18:  6000000 \\nINFO  @ Thu, 05 Nov 2020 20:21:21:  7000000 \\nINFO  @ Thu, 05 Nov 2020 20:21:25:  8000000 \\nINFO  @ Thu, 05 Nov 2020 20:21:29:  9000000 \\nINFO  @ Thu, 05 Nov 2020 20:21:32:  10000000 \\nINFO  @ Thu, 05 Nov 2020 20:21:36:  11000000 \\nINFO  @ Thu, 05 Nov 2020 20:21:40:  12000000 \\nINFO  @ Thu, 05 Nov 2020 20:21:43:  13000000 \\nINFO  @ Thu, 05 Nov 2020 20:21:47:  14000000 \\nINFO  @ Thu, 05 Nov 2020 20:21:51:  15000000 \\nINFO  @ Thu, 05 Nov 2020 20:21:54:  16000000 \\nINFO  @ Thu, 05 Nov 2020 20:21:58:  17000000 \\nINFO  @ Thu, 05 Nov 2020 20:22:01:  18000000 \\nINFO  @ Thu, 05 Nov 2020 20:22:05:  19000000 \\nINFO  @ Thu, 05 Nov 2020 20:22:09:  20000000 \\nINFO  @ Thu, 05 Nov 2020 20:22:12:  21000000 \\nINFO  @ Thu, 05 Nov 2020 20:22:16:  22000000 \\nINFO  @ Thu, 05 Nov 2020 20:22:19:  23000000 \\nINFO  @ Thu, 05 Nov 2020 20:22:23:  24000000 \\nINFO  @ Thu, 05 Nov 2020 20:22:27:  25000000 \\nINFO  @ Thu, 05 Nov 2020 20:22:30:  26000000 \\nINFO  @ Thu, 05 Nov 2020 20:22:34:  27000000 \\nINFO  @ Thu, 05 Nov 2020 20:22:37:  28000000 \\nINFO  @ Thu, 05 Nov 2020 20:22:41:  29000000 \\nINFO  @ Thu, 05 Nov 2020 20:22:45:  30000000 \\nINFO  @ Thu, 05 Nov 2020 20:22:49:  31000000 \\nINFO  @ Thu, 05 Nov 2020 20:22:54:  32000000 \\nINFO  @ Thu, 05 Nov 2020 20:22:58:  33000000 \\nINFO  @ Thu, 05 Nov 2020 20:23:01:  34000000 \\nINFO  @ Thu, 05 Nov 2020 20:23:05:  35000000 \\nINFO  @ Thu, 05 Nov 2020 20:23:08:  36000000 \\nINFO  @ Thu, 05 Nov 2020 20:23:12:  37000000 \\nINFO  @ Thu, 05 Nov 2020 20:23:15:  38000000 \\nINFO  @ Thu, 05 Nov 2020 20:23:19:  39000000 \\nINFO  @ Thu, 05 Nov 2020 20:23:23:  40000000 \\nINFO  @ Thu, 05 Nov 2020 20:23:26:  41000000 \\nINFO  @ Thu, 05 Nov 2020 20:23:30:  42000000 \\nINFO  @ Thu, 05 Nov 2020 20:23:34:  43000000 \\nINFO  @ Thu, 05 Nov 2020 20:23:37:  44000000 \\nINFO  @ Thu, 05 Nov 2020 20:23:41:  45000000 \\nINFO  @ Thu, 05 Nov 2020 20:23:45:  46000000 \\nINFO  @ Thu, 05 Nov 2020 20:23:49:  47000000 \\nINFO  @ Thu, 05 Nov 2020 20:23:52:  48000000 \\nINFO  @ Thu, 05 Nov 2020 20:23:56:  49000000 \\nINFO  @ Thu, 05 Nov 2020 20:23:59:  50000000 \\nINFO  @ Thu, 05 Nov 2020 20:24:03:  51000000 \\nINFO  @ Thu, 05 Nov 2020 20:24:07:  52000000 \\nINFO  @ Thu, 05 Nov 2020 20:24:11:  53000000 \\nINFO  @ Thu, 05 Nov 2020 20:24:14:  54000000 \\nINFO  @ Thu, 05 Nov 2020 20:24:18:  55000000 \\nINFO  @ Thu, 05 Nov 2020 20:24:22:  56000000 \\nINFO  @ Thu, 05 Nov 2020 20:24:25:  57000000 \\nINFO  @ Thu, 05 Nov 2020 20:24:29:  58000000 \\nINFO  @ Thu, 05 Nov 2020 20:24:33:  59000000 \\nINFO  @ Thu, 05 Nov 2020 20:24:36:  60000000 \\nINFO  @ Thu, 05 Nov 2020 20:24:40:  61000000 \\nINFO  @ Thu, 05 Nov 2020 20:24:44:  62000000 \\nINFO  @ Thu, 05 Nov 2020 20:24:47:  63000000 \\nINFO  @ Thu, 05 Nov 2020 20:24:51:  64000000 \\nINFO  @ Thu, 05 Nov 2020 20:24:54:  65000000 \\nINFO  @ Thu, 05 Nov 2020 20:24:58:  66000000 \\nINFO  @ Thu, 05 Nov 2020 20:25:01: 66660890 reads have been read. \\nINFO  @ Thu, 05 Nov 2020 20:25:01: #1 tag size is determined as 91 bps \\nINFO  @ Thu, 05 Nov 2020 20:25:01: #1 tag size = 91.0 \\nINFO  @ Thu, 05 Nov 2020 20:25:01: #1  total tags in treatment: 15766460 \\nINFO  @ Thu, 05 Nov 2020 20:25:01: #1 user defined the maximum tags... \\nINFO  @ Thu, 05 Nov 2020 20:25:01: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 05 Nov 2020 20:25:01: #1  tags after filtering in treatment: 15467624 \\nINFO  @ Thu, 05 Nov 2020 20:25:01: #1  Redundant rate of treatment: 0.02 \\nINFO  @ Thu, 05 Nov 2020 20:25:01: #1  total tags in control: 66660890 \\nINFO  @ Thu, 05 Nov 2020 20:25:01: #1 user defined the maximum tags... \\nINFO  @ Thu, 05 Nov 2020 20:25:01: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 05 Nov 2020 20:25:02: #1  tags after filtering in control: 65725426 \\nINFO  @ Thu, 05 Nov 2020 20:25:02: #1  Redundant rate of control: 0.01 \\nINFO  @ Thu, 05 Nov 2020 20:25:02: #1 finished! \\nINFO  @ Thu, 05 Nov 2020 20:25:02: #2 Build Peak Model... \\nINFO  @ Thu, 05 Nov 2020 20:25:02: #2 Skipped... \\nINFO  @ Thu, 05 Nov 2020 20:25:02: #2 Use 200 as fragment length \\nINFO  @ Thu, 05 Nov 2020 20:25:02: #3 Call peaks... \\nINFO  @ Thu, 05 Nov 2020 20:25:02: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 05 Nov 2020 20:29:45: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 05 Nov 2020 20:29:45: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_IRF8_NT_VHL-IRF8_R1_treat_pileup.bdg \\nINFO  @ Thu, 05 Nov 2020 20:29:45: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_IRF8_NT_VHL-IRF8_R1_control_lambda.bdg \\nINFO  @ Thu, 05 Nov 2020 20:29:45: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 05 Nov 2020 20:29:45: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 05 Nov 2020 20:33:10: #4 Write output xls file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-IRF8_R1_peaks.xls \\nINFO  @ Thu, 05 Nov 2020 20:33:10: #4 Write peak in narrowPeak format file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-IRF8_R1_peaks.narrowPeak \\nINFO  @ Thu, 05 Nov 2020 20:33:10: #4 Write summits bed file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-IRF8_R1_summits.bed \\nINFO  @ Thu, 05 Nov 2020 20:33:10: Done! \\n'\n",
      "15127980 15467624\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MED1_R1.mLb.clN.sorted.bam ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MED1_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MED1_R1.mLb.clN.sorted.bam and ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MED1_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "212\n",
      "computing the scaling values\n",
      "b'INFO  @ Thu, 05 Nov 2020 20:44:32: \\n# Command line: callpeak -B -t ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MED1_R1.mLb.clN.sorted.bam -c ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 212 -n MV411_IRF8_NT_DMSO-MED1_R1 --outdir ../../data/chipseq_IRF8_degraded/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = MV411_IRF8_NT_DMSO-MED1_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MED1_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Thu, 05 Nov 2020 20:44:32: #1 read tag files... \\nINFO  @ Thu, 05 Nov 2020 20:44:32: #1 read treatment tags... \\nINFO  @ Thu, 05 Nov 2020 20:44:36:  1000000 \\nINFO  @ Thu, 05 Nov 2020 20:44:41:  2000000 \\nINFO  @ Thu, 05 Nov 2020 20:44:45:  3000000 \\nINFO  @ Thu, 05 Nov 2020 20:44:50:  4000000 \\nINFO  @ Thu, 05 Nov 2020 20:44:55:  5000000 \\nINFO  @ Thu, 05 Nov 2020 20:44:59:  6000000 \\nINFO  @ Thu, 05 Nov 2020 20:45:04:  7000000 \\nINFO  @ Thu, 05 Nov 2020 20:45:09:  8000000 \\nINFO  @ Thu, 05 Nov 2020 20:45:13:  9000000 \\nINFO  @ Thu, 05 Nov 2020 20:45:18:  10000000 \\nINFO  @ Thu, 05 Nov 2020 20:45:22:  11000000 \\nINFO  @ Thu, 05 Nov 2020 20:45:27:  12000000 \\nINFO  @ Thu, 05 Nov 2020 20:45:32:  13000000 \\nINFO  @ Thu, 05 Nov 2020 20:45:36:  14000000 \\nINFO  @ Thu, 05 Nov 2020 20:45:41:  15000000 \\nINFO  @ Thu, 05 Nov 2020 20:45:46:  16000000 \\nINFO  @ Thu, 05 Nov 2020 20:45:51:  17000000 \\nINFO  @ Thu, 05 Nov 2020 20:45:55:  18000000 \\nINFO  @ Thu, 05 Nov 2020 20:46:00:  19000000 \\nINFO  @ Thu, 05 Nov 2020 20:46:04:  20000000 \\nINFO  @ Thu, 05 Nov 2020 20:46:08: 20739878 reads have been read. \\nINFO  @ Thu, 05 Nov 2020 20:46:08: #1.2 read input tags... \\nINFO  @ Thu, 05 Nov 2020 20:46:12:  1000000 \\nINFO  @ Thu, 05 Nov 2020 20:46:16:  2000000 \\nINFO  @ Thu, 05 Nov 2020 20:46:19:  3000000 \\nINFO  @ Thu, 05 Nov 2020 20:46:23:  4000000 \\nINFO  @ Thu, 05 Nov 2020 20:46:27:  5000000 \\nINFO  @ Thu, 05 Nov 2020 20:46:30:  6000000 \\nINFO  @ Thu, 05 Nov 2020 20:46:34:  7000000 \\nINFO  @ Thu, 05 Nov 2020 20:46:38:  8000000 \\nINFO  @ Thu, 05 Nov 2020 20:46:41:  9000000 \\nINFO  @ Thu, 05 Nov 2020 20:46:45:  10000000 \\nINFO  @ Thu, 05 Nov 2020 20:46:49:  11000000 \\nINFO  @ Thu, 05 Nov 2020 20:46:52:  12000000 \\nINFO  @ Thu, 05 Nov 2020 20:46:56:  13000000 \\nINFO  @ Thu, 05 Nov 2020 20:46:59:  14000000 \\nINFO  @ Thu, 05 Nov 2020 20:47:03:  15000000 \\nINFO  @ Thu, 05 Nov 2020 20:47:07:  16000000 \\nINFO  @ Thu, 05 Nov 2020 20:47:10:  17000000 \\nINFO  @ Thu, 05 Nov 2020 20:47:14:  18000000 \\nINFO  @ Thu, 05 Nov 2020 20:47:18:  19000000 \\nINFO  @ Thu, 05 Nov 2020 20:47:21:  20000000 \\nINFO  @ Thu, 05 Nov 2020 20:47:25:  21000000 \\nINFO  @ Thu, 05 Nov 2020 20:47:29:  22000000 \\nINFO  @ Thu, 05 Nov 2020 20:47:32:  23000000 \\nINFO  @ Thu, 05 Nov 2020 20:47:36:  24000000 \\nINFO  @ Thu, 05 Nov 2020 20:47:40:  25000000 \\nINFO  @ Thu, 05 Nov 2020 20:47:43:  26000000 \\nINFO  @ Thu, 05 Nov 2020 20:47:47:  27000000 \\nINFO  @ Thu, 05 Nov 2020 20:47:50:  28000000 \\nINFO  @ Thu, 05 Nov 2020 20:47:54:  29000000 \\nINFO  @ Thu, 05 Nov 2020 20:47:58:  30000000 \\nINFO  @ Thu, 05 Nov 2020 20:48:01:  31000000 \\nINFO  @ Thu, 05 Nov 2020 20:48:05:  32000000 \\nINFO  @ Thu, 05 Nov 2020 20:48:08:  33000000 \\nINFO  @ Thu, 05 Nov 2020 20:48:12:  34000000 \\nINFO  @ Thu, 05 Nov 2020 20:48:15:  35000000 \\nINFO  @ Thu, 05 Nov 2020 20:48:19:  36000000 \\nINFO  @ Thu, 05 Nov 2020 20:48:23:  37000000 \\nINFO  @ Thu, 05 Nov 2020 20:48:26:  38000000 \\nINFO  @ Thu, 05 Nov 2020 20:48:30:  39000000 \\nINFO  @ Thu, 05 Nov 2020 20:48:33:  40000000 \\nINFO  @ Thu, 05 Nov 2020 20:48:37:  41000000 \\nINFO  @ Thu, 05 Nov 2020 20:48:41:  42000000 \\nINFO  @ Thu, 05 Nov 2020 20:48:44:  43000000 \\nINFO  @ Thu, 05 Nov 2020 20:48:49:  44000000 \\nINFO  @ Thu, 05 Nov 2020 20:48:53:  45000000 \\nINFO  @ Thu, 05 Nov 2020 20:48:57:  46000000 \\nINFO  @ Thu, 05 Nov 2020 20:49:00:  47000000 \\nINFO  @ Thu, 05 Nov 2020 20:49:04:  48000000 \\nINFO  @ Thu, 05 Nov 2020 20:49:07:  49000000 \\nINFO  @ Thu, 05 Nov 2020 20:49:11:  50000000 \\nINFO  @ Thu, 05 Nov 2020 20:49:15:  51000000 \\nINFO  @ Thu, 05 Nov 2020 20:49:19:  52000000 \\nINFO  @ Thu, 05 Nov 2020 20:49:22:  53000000 \\nINFO  @ Thu, 05 Nov 2020 20:49:26:  54000000 \\nINFO  @ Thu, 05 Nov 2020 20:49:30:  55000000 \\nINFO  @ Thu, 05 Nov 2020 20:49:33:  56000000 \\nINFO  @ Thu, 05 Nov 2020 20:49:37:  57000000 \\nINFO  @ Thu, 05 Nov 2020 20:49:41:  58000000 \\nINFO  @ Thu, 05 Nov 2020 20:49:44:  59000000 \\nINFO  @ Thu, 05 Nov 2020 20:49:48:  60000000 \\nINFO  @ Thu, 05 Nov 2020 20:49:52:  61000000 \\nINFO  @ Thu, 05 Nov 2020 20:49:55:  62000000 \\nINFO  @ Thu, 05 Nov 2020 20:49:59:  63000000 \\nINFO  @ Thu, 05 Nov 2020 20:50:02:  64000000 \\nINFO  @ Thu, 05 Nov 2020 20:50:06:  65000000 \\nINFO  @ Thu, 05 Nov 2020 20:50:10:  66000000 \\nINFO  @ Thu, 05 Nov 2020 20:50:12: 66660890 reads have been read. \\nINFO  @ Thu, 05 Nov 2020 20:50:13: #1 tag size is determined as 96 bps \\nINFO  @ Thu, 05 Nov 2020 20:50:13: #1 tag size = 96.0 \\nINFO  @ Thu, 05 Nov 2020 20:50:13: #1  total tags in treatment: 20739878 \\nINFO  @ Thu, 05 Nov 2020 20:50:13: #1 user defined the maximum tags... \\nINFO  @ Thu, 05 Nov 2020 20:50:13: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 05 Nov 2020 20:50:13: #1  tags after filtering in treatment: 20554431 \\nINFO  @ Thu, 05 Nov 2020 20:50:13: #1  Redundant rate of treatment: 0.01 \\nINFO  @ Thu, 05 Nov 2020 20:50:13: #1  total tags in control: 66660890 \\nINFO  @ Thu, 05 Nov 2020 20:50:13: #1 user defined the maximum tags... \\nINFO  @ Thu, 05 Nov 2020 20:50:13: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 05 Nov 2020 20:50:14: #1  tags after filtering in control: 65725426 \\nINFO  @ Thu, 05 Nov 2020 20:50:14: #1  Redundant rate of control: 0.01 \\nINFO  @ Thu, 05 Nov 2020 20:50:14: #1 finished! \\nINFO  @ Thu, 05 Nov 2020 20:50:14: #2 Build Peak Model... \\nINFO  @ Thu, 05 Nov 2020 20:50:14: #2 Skipped... \\nINFO  @ Thu, 05 Nov 2020 20:50:14: #2 Use 212 as fragment length \\nINFO  @ Thu, 05 Nov 2020 20:50:14: #3 Call peaks... \\nINFO  @ Thu, 05 Nov 2020 20:50:14: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 05 Nov 2020 20:54:44: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 05 Nov 2020 20:54:44: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_IRF8_NT_DMSO-MED1_R1_treat_pileup.bdg \\nINFO  @ Thu, 05 Nov 2020 20:54:44: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_IRF8_NT_DMSO-MED1_R1_control_lambda.bdg \\nINFO  @ Thu, 05 Nov 2020 20:54:44: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 05 Nov 2020 20:54:44: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 05 Nov 2020 20:58:12: #4 Write output xls file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MED1_R1_peaks.xls \\nINFO  @ Thu, 05 Nov 2020 20:58:12: #4 Write peak in narrowPeak format file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MED1_R1_peaks.narrowPeak \\nINFO  @ Thu, 05 Nov 2020 20:58:13: #4 Write summits bed file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MED1_R1_summits.bed \\nINFO  @ Thu, 05 Nov 2020 20:58:13: Done! \\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'INFO  @ Thu, 05 Nov 2020 20:58:13: \\n# Command line: callpeak -B -t ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MED1_R1.mLb.clN.sorted.bam -c ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 212 -n MV411_IRF8_NT_VHL-MED1_R1 --outdir ../../data/chipseq_IRF8_degraded/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = MV411_IRF8_NT_VHL-MED1_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MED1_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Thu, 05 Nov 2020 20:58:13: #1 read tag files... \\nINFO  @ Thu, 05 Nov 2020 20:58:13: #1 read treatment tags... \\nINFO  @ Thu, 05 Nov 2020 20:58:18:  1000000 \\nINFO  @ Thu, 05 Nov 2020 20:58:25:  2000000 \\nINFO  @ Thu, 05 Nov 2020 20:58:29:  3000000 \\nINFO  @ Thu, 05 Nov 2020 20:58:34:  4000000 \\nINFO  @ Thu, 05 Nov 2020 20:58:38:  5000000 \\nINFO  @ Thu, 05 Nov 2020 20:58:43:  6000000 \\nINFO  @ Thu, 05 Nov 2020 20:58:49:  7000000 \\nINFO  @ Thu, 05 Nov 2020 20:58:54:  8000000 \\nINFO  @ Thu, 05 Nov 2020 20:58:58:  9000000 \\nINFO  @ Thu, 05 Nov 2020 20:59:03:  10000000 \\nINFO  @ Thu, 05 Nov 2020 20:59:08:  11000000 \\nINFO  @ Thu, 05 Nov 2020 20:59:13:  12000000 \\nINFO  @ Thu, 05 Nov 2020 20:59:17:  13000000 \\nINFO  @ Thu, 05 Nov 2020 20:59:22:  14000000 \\nINFO  @ Thu, 05 Nov 2020 20:59:26:  15000000 \\nINFO  @ Thu, 05 Nov 2020 20:59:31:  16000000 \\nINFO  @ Thu, 05 Nov 2020 20:59:36:  17000000 \\nINFO  @ Thu, 05 Nov 2020 20:59:40:  18000000 \\nINFO  @ Thu, 05 Nov 2020 20:59:45:  19000000 \\nINFO  @ Thu, 05 Nov 2020 20:59:50:  20000000 \\nINFO  @ Thu, 05 Nov 2020 20:59:54:  21000000 \\nINFO  @ Thu, 05 Nov 2020 20:59:59:  22000000 \\nINFO  @ Thu, 05 Nov 2020 21:00:04:  23000000 \\nINFO  @ Thu, 05 Nov 2020 21:00:08:  24000000 \\nINFO  @ Thu, 05 Nov 2020 21:00:11: 24566660 reads have been read. \\nINFO  @ Thu, 05 Nov 2020 21:00:11: #1.2 read input tags... \\nINFO  @ Thu, 05 Nov 2020 21:00:15:  1000000 \\nINFO  @ Thu, 05 Nov 2020 21:00:19:  2000000 \\nINFO  @ Thu, 05 Nov 2020 21:00:22:  3000000 \\nINFO  @ Thu, 05 Nov 2020 21:00:26:  4000000 \\nINFO  @ Thu, 05 Nov 2020 21:00:30:  5000000 \\nINFO  @ Thu, 05 Nov 2020 21:00:33:  6000000 \\nINFO  @ Thu, 05 Nov 2020 21:00:37:  7000000 \\nINFO  @ Thu, 05 Nov 2020 21:00:41:  8000000 \\nINFO  @ Thu, 05 Nov 2020 21:00:44:  9000000 \\nINFO  @ Thu, 05 Nov 2020 21:00:48:  10000000 \\nINFO  @ Thu, 05 Nov 2020 21:00:52:  11000000 \\nINFO  @ Thu, 05 Nov 2020 21:00:55:  12000000 \\nINFO  @ Thu, 05 Nov 2020 21:00:59:  13000000 \\nINFO  @ Thu, 05 Nov 2020 21:01:03:  14000000 \\nINFO  @ Thu, 05 Nov 2020 21:01:06:  15000000 \\nINFO  @ Thu, 05 Nov 2020 21:01:10:  16000000 \\nINFO  @ Thu, 05 Nov 2020 21:01:14:  17000000 \\nINFO  @ Thu, 05 Nov 2020 21:01:17:  18000000 \\nINFO  @ Thu, 05 Nov 2020 21:01:21:  19000000 \\nINFO  @ Thu, 05 Nov 2020 21:01:24:  20000000 \\nINFO  @ Thu, 05 Nov 2020 21:01:28:  21000000 \\nINFO  @ Thu, 05 Nov 2020 21:01:32:  22000000 \\nINFO  @ Thu, 05 Nov 2020 21:01:35:  23000000 \\nINFO  @ Thu, 05 Nov 2020 21:01:39:  24000000 \\nINFO  @ Thu, 05 Nov 2020 21:01:43:  25000000 \\nINFO  @ Thu, 05 Nov 2020 21:01:46:  26000000 \\nINFO  @ Thu, 05 Nov 2020 21:01:50:  27000000 \\nINFO  @ Thu, 05 Nov 2020 21:01:54:  28000000 \\nINFO  @ Thu, 05 Nov 2020 21:01:57:  29000000 \\nINFO  @ Thu, 05 Nov 2020 21:02:01:  30000000 \\nINFO  @ Thu, 05 Nov 2020 21:02:04:  31000000 \\nINFO  @ Thu, 05 Nov 2020 21:02:08:  32000000 \\nINFO  @ Thu, 05 Nov 2020 21:02:12:  33000000 \\nINFO  @ Thu, 05 Nov 2020 21:02:15:  34000000 \\nINFO  @ Thu, 05 Nov 2020 21:02:19:  35000000 \\nINFO  @ Thu, 05 Nov 2020 21:02:22:  36000000 \\nINFO  @ Thu, 05 Nov 2020 21:02:26:  37000000 \\nINFO  @ Thu, 05 Nov 2020 21:02:30:  38000000 \\nINFO  @ Thu, 05 Nov 2020 21:02:34:  39000000 \\nINFO  @ Thu, 05 Nov 2020 21:02:38:  40000000 \\nINFO  @ Thu, 05 Nov 2020 21:02:41:  41000000 \\nINFO  @ Thu, 05 Nov 2020 21:02:45:  42000000 \\nINFO  @ Thu, 05 Nov 2020 21:02:49:  43000000 \\nINFO  @ Thu, 05 Nov 2020 21:02:52:  44000000 \\nINFO  @ Thu, 05 Nov 2020 21:02:56:  45000000 \\nINFO  @ Thu, 05 Nov 2020 21:03:00:  46000000 \\nINFO  @ Thu, 05 Nov 2020 21:03:04:  47000000 \\nINFO  @ Thu, 05 Nov 2020 21:03:07:  48000000 \\nINFO  @ Thu, 05 Nov 2020 21:03:11:  49000000 \\nINFO  @ Thu, 05 Nov 2020 21:03:15:  50000000 \\nINFO  @ Thu, 05 Nov 2020 21:03:18:  51000000 \\nINFO  @ Thu, 05 Nov 2020 21:03:22:  52000000 \\nINFO  @ Thu, 05 Nov 2020 21:03:26:  53000000 \\nINFO  @ Thu, 05 Nov 2020 21:03:29:  54000000 \\nINFO  @ Thu, 05 Nov 2020 21:03:33:  55000000 \\nINFO  @ Thu, 05 Nov 2020 21:03:37:  56000000 \\nINFO  @ Thu, 05 Nov 2020 21:03:40:  57000000 \\nINFO  @ Thu, 05 Nov 2020 21:03:44:  58000000 \\nINFO  @ Thu, 05 Nov 2020 21:03:47:  59000000 \\nINFO  @ Thu, 05 Nov 2020 21:03:51:  60000000 \\nINFO  @ Thu, 05 Nov 2020 21:03:55:  61000000 \\nINFO  @ Thu, 05 Nov 2020 21:03:58:  62000000 \\nINFO  @ Thu, 05 Nov 2020 21:04:02:  63000000 \\nINFO  @ Thu, 05 Nov 2020 21:04:06:  64000000 \\nINFO  @ Thu, 05 Nov 2020 21:04:09:  65000000 \\nINFO  @ Thu, 05 Nov 2020 21:04:13:  66000000 \\nINFO  @ Thu, 05 Nov 2020 21:04:15: 66660890 reads have been read. \\nINFO  @ Thu, 05 Nov 2020 21:04:16: #1 tag size is determined as 89 bps \\nINFO  @ Thu, 05 Nov 2020 21:04:16: #1 tag size = 89.0 \\nINFO  @ Thu, 05 Nov 2020 21:04:16: #1  total tags in treatment: 24566660 \\nINFO  @ Thu, 05 Nov 2020 21:04:16: #1 user defined the maximum tags... \\nINFO  @ Thu, 05 Nov 2020 21:04:16: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 05 Nov 2020 21:04:16: #1  tags after filtering in treatment: 24342695 \\nINFO  @ Thu, 05 Nov 2020 21:04:16: #1  Redundant rate of treatment: 0.01 \\nINFO  @ Thu, 05 Nov 2020 21:04:16: #1  total tags in control: 66660890 \\nINFO  @ Thu, 05 Nov 2020 21:04:16: #1 user defined the maximum tags... \\nINFO  @ Thu, 05 Nov 2020 21:04:16: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 05 Nov 2020 21:04:16: #1  tags after filtering in control: 65725426 \\nINFO  @ Thu, 05 Nov 2020 21:04:16: #1  Redundant rate of control: 0.01 \\nINFO  @ Thu, 05 Nov 2020 21:04:16: #1 finished! \\nINFO  @ Thu, 05 Nov 2020 21:04:16: #2 Build Peak Model... \\nINFO  @ Thu, 05 Nov 2020 21:04:16: #2 Skipped... \\nINFO  @ Thu, 05 Nov 2020 21:04:16: #2 Use 212 as fragment length \\nINFO  @ Thu, 05 Nov 2020 21:04:16: #3 Call peaks... \\nINFO  @ Thu, 05 Nov 2020 21:04:16: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 05 Nov 2020 21:09:01: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 05 Nov 2020 21:09:01: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_IRF8_NT_VHL-MED1_R1_treat_pileup.bdg \\nINFO  @ Thu, 05 Nov 2020 21:09:01: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_IRF8_NT_VHL-MED1_R1_control_lambda.bdg \\nINFO  @ Thu, 05 Nov 2020 21:09:01: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 05 Nov 2020 21:09:01: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 05 Nov 2020 21:12:46: #4 Write output xls file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MED1_R1_peaks.xls \\nINFO  @ Thu, 05 Nov 2020 21:12:46: #4 Write peak in narrowPeak format file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MED1_R1_peaks.narrowPeak \\nINFO  @ Thu, 05 Nov 2020 21:12:46: #4 Write summits bed file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MED1_R1_summits.bed \\nINFO  @ Thu, 05 Nov 2020 21:12:46: Done! \\n'\n",
      "20554431 24342695\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MEF2C_R1.mLb.clN.sorted.bam ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MEF2C_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MEF2C_R1.mLb.clN.sorted.bam and ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MEF2C_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "204\n",
      "computing the scaling values\n",
      "b'INFO  @ Thu, 05 Nov 2020 21:24:16: \\n# Command line: callpeak -B -t ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MEF2C_R1.mLb.clN.sorted.bam -c ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 204 -n MV411_IRF8_NT_DMSO-MEF2C_R1 --outdir ../../data/chipseq_IRF8_degraded/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = MV411_IRF8_NT_DMSO-MEF2C_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MEF2C_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Thu, 05 Nov 2020 21:24:16: #1 read tag files... \\nINFO  @ Thu, 05 Nov 2020 21:24:16: #1 read treatment tags... \\nINFO  @ Thu, 05 Nov 2020 21:24:21:  1000000 \\nINFO  @ Thu, 05 Nov 2020 21:24:25:  2000000 \\nINFO  @ Thu, 05 Nov 2020 21:24:30:  3000000 \\nINFO  @ Thu, 05 Nov 2020 21:24:35:  4000000 \\nINFO  @ Thu, 05 Nov 2020 21:24:39:  5000000 \\nINFO  @ Thu, 05 Nov 2020 21:24:44:  6000000 \\nINFO  @ Thu, 05 Nov 2020 21:24:49:  7000000 \\nINFO  @ Thu, 05 Nov 2020 21:24:53:  8000000 \\nINFO  @ Thu, 05 Nov 2020 21:24:58:  9000000 \\nINFO  @ Thu, 05 Nov 2020 21:25:02:  10000000 \\nINFO  @ Thu, 05 Nov 2020 21:25:07:  11000000 \\nINFO  @ Thu, 05 Nov 2020 21:25:12:  12000000 \\nINFO  @ Thu, 05 Nov 2020 21:25:16:  13000000 \\nINFO  @ Thu, 05 Nov 2020 21:25:21:  14000000 \\nINFO  @ Thu, 05 Nov 2020 21:25:26:  15000000 \\nINFO  @ Thu, 05 Nov 2020 21:25:30:  16000000 \\nINFO  @ Thu, 05 Nov 2020 21:25:35:  17000000 \\nINFO  @ Thu, 05 Nov 2020 21:25:38: 17547907 reads have been read. \\nINFO  @ Thu, 05 Nov 2020 21:25:38: #1.2 read input tags... \\nINFO  @ Thu, 05 Nov 2020 21:25:41:  1000000 \\nINFO  @ Thu, 05 Nov 2020 21:25:45:  2000000 \\nINFO  @ Thu, 05 Nov 2020 21:25:49:  3000000 \\nINFO  @ Thu, 05 Nov 2020 21:25:53:  4000000 \\nINFO  @ Thu, 05 Nov 2020 21:25:56:  5000000 \\nINFO  @ Thu, 05 Nov 2020 21:26:00:  6000000 \\nINFO  @ Thu, 05 Nov 2020 21:26:04:  7000000 \\nINFO  @ Thu, 05 Nov 2020 21:26:07:  8000000 \\nINFO  @ Thu, 05 Nov 2020 21:26:11:  9000000 \\nINFO  @ Thu, 05 Nov 2020 21:26:14:  10000000 \\nINFO  @ Thu, 05 Nov 2020 21:26:18:  11000000 \\nINFO  @ Thu, 05 Nov 2020 21:26:22:  12000000 \\nINFO  @ Thu, 05 Nov 2020 21:26:26:  13000000 \\nINFO  @ Thu, 05 Nov 2020 21:26:29:  14000000 \\nINFO  @ Thu, 05 Nov 2020 21:26:33:  15000000 \\nINFO  @ Thu, 05 Nov 2020 21:26:37:  16000000 \\nINFO  @ Thu, 05 Nov 2020 21:26:40:  17000000 \\nINFO  @ Thu, 05 Nov 2020 21:26:44:  18000000 \\nINFO  @ Thu, 05 Nov 2020 21:26:47:  19000000 \\nINFO  @ Thu, 05 Nov 2020 21:26:51:  20000000 \\nINFO  @ Thu, 05 Nov 2020 21:26:55:  21000000 \\nINFO  @ Thu, 05 Nov 2020 21:26:58:  22000000 \\nINFO  @ Thu, 05 Nov 2020 21:27:02:  23000000 \\nINFO  @ Thu, 05 Nov 2020 21:27:06:  24000000 \\nINFO  @ Thu, 05 Nov 2020 21:27:09:  25000000 \\nINFO  @ Thu, 05 Nov 2020 21:27:13:  26000000 \\nINFO  @ Thu, 05 Nov 2020 21:27:16:  27000000 \\nINFO  @ Thu, 05 Nov 2020 21:27:20:  28000000 \\nINFO  @ Thu, 05 Nov 2020 21:27:24:  29000000 \\nINFO  @ Thu, 05 Nov 2020 21:27:27:  30000000 \\nINFO  @ Thu, 05 Nov 2020 21:27:31:  31000000 \\nINFO  @ Thu, 05 Nov 2020 21:27:35:  32000000 \\nINFO  @ Thu, 05 Nov 2020 21:27:39:  33000000 \\nINFO  @ Thu, 05 Nov 2020 21:27:42:  34000000 \\nINFO  @ Thu, 05 Nov 2020 21:27:46:  35000000 \\nINFO  @ Thu, 05 Nov 2020 21:27:49:  36000000 \\nINFO  @ Thu, 05 Nov 2020 21:27:53:  37000000 \\nINFO  @ Thu, 05 Nov 2020 21:27:56:  38000000 \\nINFO  @ Thu, 05 Nov 2020 21:28:00:  39000000 \\nINFO  @ Thu, 05 Nov 2020 21:28:04:  40000000 \\nINFO  @ Thu, 05 Nov 2020 21:28:07:  41000000 \\nINFO  @ Thu, 05 Nov 2020 21:28:11:  42000000 \\nINFO  @ Thu, 05 Nov 2020 21:28:15:  43000000 \\nINFO  @ Thu, 05 Nov 2020 21:28:18:  44000000 \\nINFO  @ Thu, 05 Nov 2020 21:28:22:  45000000 \\nINFO  @ Thu, 05 Nov 2020 21:28:25:  46000000 \\nINFO  @ Thu, 05 Nov 2020 21:28:29:  47000000 \\nINFO  @ Thu, 05 Nov 2020 21:28:32:  48000000 \\nINFO  @ Thu, 05 Nov 2020 21:28:36:  49000000 \\nINFO  @ Thu, 05 Nov 2020 21:28:39:  50000000 \\nINFO  @ Thu, 05 Nov 2020 21:28:43:  51000000 \\nINFO  @ Thu, 05 Nov 2020 21:28:47:  52000000 \\nINFO  @ Thu, 05 Nov 2020 21:28:51:  53000000 \\nINFO  @ Thu, 05 Nov 2020 21:28:54:  54000000 \\nINFO  @ Thu, 05 Nov 2020 21:28:58:  55000000 \\nINFO  @ Thu, 05 Nov 2020 21:29:01:  56000000 \\nINFO  @ Thu, 05 Nov 2020 21:29:05:  57000000 \\nINFO  @ Thu, 05 Nov 2020 21:29:09:  58000000 \\nINFO  @ Thu, 05 Nov 2020 21:29:12:  59000000 \\nINFO  @ Thu, 05 Nov 2020 21:29:16:  60000000 \\nINFO  @ Thu, 05 Nov 2020 21:29:19:  61000000 \\nINFO  @ Thu, 05 Nov 2020 21:29:23:  62000000 \\nINFO  @ Thu, 05 Nov 2020 21:29:27:  63000000 \\nINFO  @ Thu, 05 Nov 2020 21:29:30:  64000000 \\nINFO  @ Thu, 05 Nov 2020 21:29:34:  65000000 \\nINFO  @ Thu, 05 Nov 2020 21:29:38:  66000000 \\nINFO  @ Thu, 05 Nov 2020 21:29:40: 66660890 reads have been read. \\nINFO  @ Thu, 05 Nov 2020 21:29:41: #1 tag size is determined as 90 bps \\nINFO  @ Thu, 05 Nov 2020 21:29:41: #1 tag size = 90.0 \\nINFO  @ Thu, 05 Nov 2020 21:29:41: #1  total tags in treatment: 17547907 \\nINFO  @ Thu, 05 Nov 2020 21:29:41: #1 user defined the maximum tags... \\nINFO  @ Thu, 05 Nov 2020 21:29:41: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 05 Nov 2020 21:29:41: #1  tags after filtering in treatment: 17352317 \\nINFO  @ Thu, 05 Nov 2020 21:29:41: #1  Redundant rate of treatment: 0.01 \\nINFO  @ Thu, 05 Nov 2020 21:29:41: #1  total tags in control: 66660890 \\nINFO  @ Thu, 05 Nov 2020 21:29:41: #1 user defined the maximum tags... \\nINFO  @ Thu, 05 Nov 2020 21:29:41: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 05 Nov 2020 21:29:41: #1  tags after filtering in control: 65725426 \\nINFO  @ Thu, 05 Nov 2020 21:29:41: #1  Redundant rate of control: 0.01 \\nINFO  @ Thu, 05 Nov 2020 21:29:41: #1 finished! \\nINFO  @ Thu, 05 Nov 2020 21:29:41: #2 Build Peak Model... \\nINFO  @ Thu, 05 Nov 2020 21:29:41: #2 Skipped... \\nINFO  @ Thu, 05 Nov 2020 21:29:41: #2 Use 204 as fragment length \\nINFO  @ Thu, 05 Nov 2020 21:29:41: #3 Call peaks... \\nINFO  @ Thu, 05 Nov 2020 21:29:41: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 05 Nov 2020 21:34:18: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 05 Nov 2020 21:34:18: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_IRF8_NT_DMSO-MEF2C_R1_treat_pileup.bdg \\nINFO  @ Thu, 05 Nov 2020 21:34:18: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_IRF8_NT_DMSO-MEF2C_R1_control_lambda.bdg \\nINFO  @ Thu, 05 Nov 2020 21:34:18: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 05 Nov 2020 21:34:18: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 05 Nov 2020 21:37:56: #4 Write output xls file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MEF2C_R1_peaks.xls \\nINFO  @ Thu, 05 Nov 2020 21:37:56: #4 Write peak in narrowPeak format file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MEF2C_R1_peaks.narrowPeak \\nINFO  @ Thu, 05 Nov 2020 21:37:56: #4 Write summits bed file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MEF2C_R1_summits.bed \\nINFO  @ Thu, 05 Nov 2020 21:37:56: Done! \\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'INFO  @ Thu, 05 Nov 2020 21:37:57: \\n# Command line: callpeak -B -t ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MEF2C_R1.mLb.clN.sorted.bam -c ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 204 -n MV411_IRF8_NT_VHL-MEF2C_R1 --outdir ../../data/chipseq_IRF8_degraded/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = MV411_IRF8_NT_VHL-MEF2C_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MEF2C_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Thu, 05 Nov 2020 21:37:57: #1 read tag files... \\nINFO  @ Thu, 05 Nov 2020 21:37:57: #1 read treatment tags... \\nINFO  @ Thu, 05 Nov 2020 21:38:04:  1000000 \\nINFO  @ Thu, 05 Nov 2020 21:38:09:  2000000 \\nINFO  @ Thu, 05 Nov 2020 21:38:14:  3000000 \\nINFO  @ Thu, 05 Nov 2020 21:38:19:  4000000 \\nINFO  @ Thu, 05 Nov 2020 21:38:23:  5000000 \\nINFO  @ Thu, 05 Nov 2020 21:38:28:  6000000 \\nINFO  @ Thu, 05 Nov 2020 21:38:33:  7000000 \\nINFO  @ Thu, 05 Nov 2020 21:38:38:  8000000 \\nINFO  @ Thu, 05 Nov 2020 21:38:43:  9000000 \\nINFO  @ Thu, 05 Nov 2020 21:38:47:  10000000 \\nINFO  @ Thu, 05 Nov 2020 21:38:52:  11000000 \\nINFO  @ Thu, 05 Nov 2020 21:38:57:  12000000 \\nINFO  @ Thu, 05 Nov 2020 21:39:02:  13000000 \\nINFO  @ Thu, 05 Nov 2020 21:39:07:  14000000 \\nINFO  @ Thu, 05 Nov 2020 21:39:11:  15000000 \\nINFO  @ Thu, 05 Nov 2020 21:39:16:  16000000 \\nINFO  @ Thu, 05 Nov 2020 21:39:21:  17000000 \\nINFO  @ Thu, 05 Nov 2020 21:39:23: 17440748 reads have been read. \\nINFO  @ Thu, 05 Nov 2020 21:39:24: #1.2 read input tags... \\nINFO  @ Thu, 05 Nov 2020 21:39:27:  1000000 \\nINFO  @ Thu, 05 Nov 2020 21:39:31:  2000000 \\nINFO  @ Thu, 05 Nov 2020 21:39:35:  3000000 \\nINFO  @ Thu, 05 Nov 2020 21:39:39:  4000000 \\nINFO  @ Thu, 05 Nov 2020 21:39:43:  5000000 \\nINFO  @ Thu, 05 Nov 2020 21:39:46:  6000000 \\nINFO  @ Thu, 05 Nov 2020 21:39:50:  7000000 \\nINFO  @ Thu, 05 Nov 2020 21:39:54:  8000000 \\nINFO  @ Thu, 05 Nov 2020 21:39:58:  9000000 \\nINFO  @ Thu, 05 Nov 2020 21:40:01:  10000000 \\nINFO  @ Thu, 05 Nov 2020 21:40:05:  11000000 \\nINFO  @ Thu, 05 Nov 2020 21:40:09:  12000000 \\nINFO  @ Thu, 05 Nov 2020 21:40:12:  13000000 \\nINFO  @ Thu, 05 Nov 2020 21:40:16:  14000000 \\nINFO  @ Thu, 05 Nov 2020 21:40:20:  15000000 \\nINFO  @ Thu, 05 Nov 2020 21:40:24:  16000000 \\nINFO  @ Thu, 05 Nov 2020 21:40:27:  17000000 \\nINFO  @ Thu, 05 Nov 2020 21:40:31:  18000000 \\nINFO  @ Thu, 05 Nov 2020 21:40:35:  19000000 \\nINFO  @ Thu, 05 Nov 2020 21:40:39:  20000000 \\nINFO  @ Thu, 05 Nov 2020 21:40:43:  21000000 \\nINFO  @ Thu, 05 Nov 2020 21:40:47:  22000000 \\nINFO  @ Thu, 05 Nov 2020 21:40:50:  23000000 \\nINFO  @ Thu, 05 Nov 2020 21:40:54:  24000000 \\nINFO  @ Thu, 05 Nov 2020 21:40:58:  25000000 \\nINFO  @ Thu, 05 Nov 2020 21:41:01:  26000000 \\nINFO  @ Thu, 05 Nov 2020 21:41:05:  27000000 \\nINFO  @ Thu, 05 Nov 2020 21:41:09:  28000000 \\nINFO  @ Thu, 05 Nov 2020 21:41:13:  29000000 \\nINFO  @ Thu, 05 Nov 2020 21:41:16:  30000000 \\nINFO  @ Thu, 05 Nov 2020 21:41:20:  31000000 \\nINFO  @ Thu, 05 Nov 2020 21:41:24:  32000000 \\nINFO  @ Thu, 05 Nov 2020 21:41:28:  33000000 \\nINFO  @ Thu, 05 Nov 2020 21:41:31:  34000000 \\nINFO  @ Thu, 05 Nov 2020 21:41:35:  35000000 \\nINFO  @ Thu, 05 Nov 2020 21:41:39:  36000000 \\nINFO  @ Thu, 05 Nov 2020 21:41:42:  37000000 \\nINFO  @ Thu, 05 Nov 2020 21:41:46:  38000000 \\nINFO  @ Thu, 05 Nov 2020 21:41:50:  39000000 \\nINFO  @ Thu, 05 Nov 2020 21:41:54:  40000000 \\nINFO  @ Thu, 05 Nov 2020 21:41:57:  41000000 \\nINFO  @ Thu, 05 Nov 2020 21:42:01:  42000000 \\nINFO  @ Thu, 05 Nov 2020 21:42:05:  43000000 \\nINFO  @ Thu, 05 Nov 2020 21:42:09:  44000000 \\nINFO  @ Thu, 05 Nov 2020 21:42:12:  45000000 \\nINFO  @ Thu, 05 Nov 2020 21:42:16:  46000000 \\nINFO  @ Thu, 05 Nov 2020 21:42:20:  47000000 \\nINFO  @ Thu, 05 Nov 2020 21:42:23:  48000000 \\nINFO  @ Thu, 05 Nov 2020 21:42:27:  49000000 \\nINFO  @ Thu, 05 Nov 2020 21:42:31:  50000000 \\nINFO  @ Thu, 05 Nov 2020 21:42:35:  51000000 \\nINFO  @ Thu, 05 Nov 2020 21:42:38:  52000000 \\nINFO  @ Thu, 05 Nov 2020 21:42:42:  53000000 \\nINFO  @ Thu, 05 Nov 2020 21:42:46:  54000000 \\nINFO  @ Thu, 05 Nov 2020 21:42:50:  55000000 \\nINFO  @ Thu, 05 Nov 2020 21:42:54:  56000000 \\nINFO  @ Thu, 05 Nov 2020 21:42:58:  57000000 \\nINFO  @ Thu, 05 Nov 2020 21:43:01:  58000000 \\nINFO  @ Thu, 05 Nov 2020 21:43:05:  59000000 \\nINFO  @ Thu, 05 Nov 2020 21:43:09:  60000000 \\nINFO  @ Thu, 05 Nov 2020 21:43:13:  61000000 \\nINFO  @ Thu, 05 Nov 2020 21:43:16:  62000000 \\nINFO  @ Thu, 05 Nov 2020 21:43:20:  63000000 \\nINFO  @ Thu, 05 Nov 2020 21:43:24:  64000000 \\nINFO  @ Thu, 05 Nov 2020 21:43:28:  65000000 \\nINFO  @ Thu, 05 Nov 2020 21:43:31:  66000000 \\nINFO  @ Thu, 05 Nov 2020 21:43:34: 66660890 reads have been read. \\nINFO  @ Thu, 05 Nov 2020 21:43:35: #1 tag size is determined as 87 bps \\nINFO  @ Thu, 05 Nov 2020 21:43:35: #1 tag size = 87.0 \\nINFO  @ Thu, 05 Nov 2020 21:43:35: #1  total tags in treatment: 17440748 \\nINFO  @ Thu, 05 Nov 2020 21:43:35: #1 user defined the maximum tags... \\nINFO  @ Thu, 05 Nov 2020 21:43:35: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 05 Nov 2020 21:43:35: #1  tags after filtering in treatment: 17238865 \\nINFO  @ Thu, 05 Nov 2020 21:43:35: #1  Redundant rate of treatment: 0.01 \\nINFO  @ Thu, 05 Nov 2020 21:43:35: #1  total tags in control: 66660890 \\nINFO  @ Thu, 05 Nov 2020 21:43:35: #1 user defined the maximum tags... \\nINFO  @ Thu, 05 Nov 2020 21:43:35: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 05 Nov 2020 21:43:35: #1  tags after filtering in control: 65725426 \\nINFO  @ Thu, 05 Nov 2020 21:43:35: #1  Redundant rate of control: 0.01 \\nINFO  @ Thu, 05 Nov 2020 21:43:35: #1 finished! \\nINFO  @ Thu, 05 Nov 2020 21:43:35: #2 Build Peak Model... \\nINFO  @ Thu, 05 Nov 2020 21:43:35: #2 Skipped... \\nINFO  @ Thu, 05 Nov 2020 21:43:35: #2 Use 204 as fragment length \\nINFO  @ Thu, 05 Nov 2020 21:43:35: #3 Call peaks... \\nINFO  @ Thu, 05 Nov 2020 21:43:35: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 05 Nov 2020 21:48:15: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 05 Nov 2020 21:48:15: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_IRF8_NT_VHL-MEF2C_R1_treat_pileup.bdg \\nINFO  @ Thu, 05 Nov 2020 21:48:15: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_IRF8_NT_VHL-MEF2C_R1_control_lambda.bdg \\nINFO  @ Thu, 05 Nov 2020 21:48:15: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 05 Nov 2020 21:48:15: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 05 Nov 2020 21:51:39: #4 Write output xls file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MEF2C_R1_peaks.xls \\nINFO  @ Thu, 05 Nov 2020 21:51:39: #4 Write peak in narrowPeak format file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MEF2C_R1_peaks.narrowPeak \\nINFO  @ Thu, 05 Nov 2020 21:51:39: #4 Write summits bed file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MEF2C_R1_summits.bed \\nINFO  @ Thu, 05 Nov 2020 21:51:39: Done! \\n'\n",
      "17352317 17238865\n",
      "doing differential peak binding\n",
      "../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MEF2D_R1.mLb.clN.sorted.bam ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MEF2D_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MEF2D_R1.mLb.clN.sorted.bam and ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MEF2D_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "184\n",
      "computing the scaling values\n",
      "b'INFO  @ Thu, 05 Nov 2020 22:02:35: \\n# Command line: callpeak -B -t ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MEF2D_R1.mLb.clN.sorted.bam -c ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 184 -n MV411_IRF8_NT_DMSO-MEF2D_R1 --outdir ../../data/chipseq_IRF8_degraded/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = MV411_IRF8_NT_DMSO-MEF2D_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MEF2D_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Thu, 05 Nov 2020 22:02:35: #1 read tag files... \\nINFO  @ Thu, 05 Nov 2020 22:02:35: #1 read treatment tags... \\nINFO  @ Thu, 05 Nov 2020 22:02:39:  1000000 \\nINFO  @ Thu, 05 Nov 2020 22:02:44:  2000000 \\nINFO  @ Thu, 05 Nov 2020 22:02:49:  3000000 \\nINFO  @ Thu, 05 Nov 2020 22:02:53:  4000000 \\nINFO  @ Thu, 05 Nov 2020 22:02:58:  5000000 \\nINFO  @ Thu, 05 Nov 2020 22:03:03:  6000000 \\nINFO  @ Thu, 05 Nov 2020 22:03:07:  7000000 \\nINFO  @ Thu, 05 Nov 2020 22:03:12:  8000000 \\nINFO  @ Thu, 05 Nov 2020 22:03:16:  9000000 \\nINFO  @ Thu, 05 Nov 2020 22:03:21:  10000000 \\nINFO  @ Thu, 05 Nov 2020 22:03:26:  11000000 \\nINFO  @ Thu, 05 Nov 2020 22:03:31:  12000000 \\nINFO  @ Thu, 05 Nov 2020 22:03:35:  13000000 \\nINFO  @ Thu, 05 Nov 2020 22:03:40:  14000000 \\nINFO  @ Thu, 05 Nov 2020 22:03:45:  15000000 \\nINFO  @ Thu, 05 Nov 2020 22:03:49:  16000000 \\nINFO  @ Thu, 05 Nov 2020 22:03:54:  17000000 \\nINFO  @ Thu, 05 Nov 2020 22:03:59:  18000000 \\nINFO  @ Thu, 05 Nov 2020 22:03:59: 18046674 reads have been read. \\nINFO  @ Thu, 05 Nov 2020 22:03:59: #1.2 read input tags... \\nINFO  @ Thu, 05 Nov 2020 22:04:03:  1000000 \\nINFO  @ Thu, 05 Nov 2020 22:04:06:  2000000 \\nINFO  @ Thu, 05 Nov 2020 22:04:10:  3000000 \\nINFO  @ Thu, 05 Nov 2020 22:04:14:  4000000 \\nINFO  @ Thu, 05 Nov 2020 22:04:17:  5000000 \\nINFO  @ Thu, 05 Nov 2020 22:04:21:  6000000 \\nINFO  @ Thu, 05 Nov 2020 22:04:24:  7000000 \\nINFO  @ Thu, 05 Nov 2020 22:04:28:  8000000 \\nINFO  @ Thu, 05 Nov 2020 22:04:32:  9000000 \\nINFO  @ Thu, 05 Nov 2020 22:04:35:  10000000 \\nINFO  @ Thu, 05 Nov 2020 22:04:39:  11000000 \\nINFO  @ Thu, 05 Nov 2020 22:04:42:  12000000 \\nINFO  @ Thu, 05 Nov 2020 22:04:46:  13000000 \\nINFO  @ Thu, 05 Nov 2020 22:04:50:  14000000 \\nINFO  @ Thu, 05 Nov 2020 22:04:53:  15000000 \\nINFO  @ Thu, 05 Nov 2020 22:04:57:  16000000 \\nINFO  @ Thu, 05 Nov 2020 22:05:00:  17000000 \\nINFO  @ Thu, 05 Nov 2020 22:05:04:  18000000 \\nINFO  @ Thu, 05 Nov 2020 22:05:08:  19000000 \\nINFO  @ Thu, 05 Nov 2020 22:05:11:  20000000 \\nINFO  @ Thu, 05 Nov 2020 22:05:15:  21000000 \\nINFO  @ Thu, 05 Nov 2020 22:05:18:  22000000 \\nINFO  @ Thu, 05 Nov 2020 22:05:22:  23000000 \\nINFO  @ Thu, 05 Nov 2020 22:05:25:  24000000 \\nINFO  @ Thu, 05 Nov 2020 22:05:29:  25000000 \\nINFO  @ Thu, 05 Nov 2020 22:05:33:  26000000 \\nINFO  @ Thu, 05 Nov 2020 22:05:36:  27000000 \\nINFO  @ Thu, 05 Nov 2020 22:05:40:  28000000 \\nINFO  @ Thu, 05 Nov 2020 22:05:43:  29000000 \\nINFO  @ Thu, 05 Nov 2020 22:05:47:  30000000 \\nINFO  @ Thu, 05 Nov 2020 22:05:50:  31000000 \\nINFO  @ Thu, 05 Nov 2020 22:05:54:  32000000 \\nINFO  @ Thu, 05 Nov 2020 22:05:58:  33000000 \\nINFO  @ Thu, 05 Nov 2020 22:06:01:  34000000 \\nINFO  @ Thu, 05 Nov 2020 22:06:05:  35000000 \\nINFO  @ Thu, 05 Nov 2020 22:06:08:  36000000 \\nINFO  @ Thu, 05 Nov 2020 22:06:12:  37000000 \\nINFO  @ Thu, 05 Nov 2020 22:06:15:  38000000 \\nINFO  @ Thu, 05 Nov 2020 22:06:19:  39000000 \\nINFO  @ Thu, 05 Nov 2020 22:06:23:  40000000 \\nINFO  @ Thu, 05 Nov 2020 22:06:26:  41000000 \\nINFO  @ Thu, 05 Nov 2020 22:06:30:  42000000 \\nINFO  @ Thu, 05 Nov 2020 22:06:33:  43000000 \\nINFO  @ Thu, 05 Nov 2020 22:06:37:  44000000 \\nINFO  @ Thu, 05 Nov 2020 22:06:40:  45000000 \\nINFO  @ Thu, 05 Nov 2020 22:06:44:  46000000 \\nINFO  @ Thu, 05 Nov 2020 22:06:48:  47000000 \\nINFO  @ Thu, 05 Nov 2020 22:06:51:  48000000 \\nINFO  @ Thu, 05 Nov 2020 22:06:55:  49000000 \\nINFO  @ Thu, 05 Nov 2020 22:06:58:  50000000 \\nINFO  @ Thu, 05 Nov 2020 22:07:02:  51000000 \\nINFO  @ Thu, 05 Nov 2020 22:07:06:  52000000 \\nINFO  @ Thu, 05 Nov 2020 22:07:09:  53000000 \\nINFO  @ Thu, 05 Nov 2020 22:07:13:  54000000 \\nINFO  @ Thu, 05 Nov 2020 22:07:17:  55000000 \\nINFO  @ Thu, 05 Nov 2020 22:07:20:  56000000 \\nINFO  @ Thu, 05 Nov 2020 22:07:24:  57000000 \\nINFO  @ Thu, 05 Nov 2020 22:07:28:  58000000 \\nINFO  @ Thu, 05 Nov 2020 22:07:31:  59000000 \\nINFO  @ Thu, 05 Nov 2020 22:07:35:  60000000 \\nINFO  @ Thu, 05 Nov 2020 22:07:38:  61000000 \\nINFO  @ Thu, 05 Nov 2020 22:07:43:  62000000 \\nINFO  @ Thu, 05 Nov 2020 22:07:46:  63000000 \\nINFO  @ Thu, 05 Nov 2020 22:07:50:  64000000 \\nINFO  @ Thu, 05 Nov 2020 22:07:54:  65000000 \\nINFO  @ Thu, 05 Nov 2020 22:07:57:  66000000 \\nINFO  @ Thu, 05 Nov 2020 22:08:00: 66660890 reads have been read. \\nINFO  @ Thu, 05 Nov 2020 22:08:01: #1 tag size is determined as 95 bps \\nINFO  @ Thu, 05 Nov 2020 22:08:01: #1 tag size = 95.0 \\nINFO  @ Thu, 05 Nov 2020 22:08:01: #1  total tags in treatment: 18046674 \\nINFO  @ Thu, 05 Nov 2020 22:08:01: #1 user defined the maximum tags... \\nINFO  @ Thu, 05 Nov 2020 22:08:01: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 05 Nov 2020 22:08:01: #1  tags after filtering in treatment: 17887183 \\nINFO  @ Thu, 05 Nov 2020 22:08:01: #1  Redundant rate of treatment: 0.01 \\nINFO  @ Thu, 05 Nov 2020 22:08:01: #1  total tags in control: 66660890 \\nINFO  @ Thu, 05 Nov 2020 22:08:01: #1 user defined the maximum tags... \\nINFO  @ Thu, 05 Nov 2020 22:08:01: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 05 Nov 2020 22:08:01: #1  tags after filtering in control: 65725426 \\nINFO  @ Thu, 05 Nov 2020 22:08:01: #1  Redundant rate of control: 0.01 \\nINFO  @ Thu, 05 Nov 2020 22:08:01: #1 finished! \\nINFO  @ Thu, 05 Nov 2020 22:08:01: #2 Build Peak Model... \\nINFO  @ Thu, 05 Nov 2020 22:08:01: #2 Skipped... \\nINFO  @ Thu, 05 Nov 2020 22:08:01: #2 Use 184 as fragment length \\nINFO  @ Thu, 05 Nov 2020 22:08:01: #3 Call peaks... \\nINFO  @ Thu, 05 Nov 2020 22:08:01: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 05 Nov 2020 22:12:34: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 05 Nov 2020 22:12:34: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_IRF8_NT_DMSO-MEF2D_R1_treat_pileup.bdg \\nINFO  @ Thu, 05 Nov 2020 22:12:34: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_IRF8_NT_DMSO-MEF2D_R1_control_lambda.bdg \\nINFO  @ Thu, 05 Nov 2020 22:12:34: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 05 Nov 2020 22:12:34: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 05 Nov 2020 22:16:09: #4 Write output xls file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MEF2D_R1_peaks.xls \\nINFO  @ Thu, 05 Nov 2020 22:16:09: #4 Write peak in narrowPeak format file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MEF2D_R1_peaks.narrowPeak \\nINFO  @ Thu, 05 Nov 2020 22:16:09: #4 Write summits bed file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MEF2D_R1_summits.bed \\nINFO  @ Thu, 05 Nov 2020 22:16:09: Done! \\n'\n",
      "b'INFO  @ Thu, 05 Nov 2020 22:16:10: \\n# Command line: callpeak -B -t ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MEF2D_R1.mLb.clN.sorted.bam -c ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 184 -n MV411_IRF8_NT_VHL-MEF2D_R1 --outdir ../../data/chipseq_IRF8_degraded/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = MV411_IRF8_NT_VHL-MEF2D_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MEF2D_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Thu, 05 Nov 2020 22:16:10: #1 read tag files... \\nINFO  @ Thu, 05 Nov 2020 22:16:10: #1 read treatment tags... \\nINFO  @ Thu, 05 Nov 2020 22:16:17:  1000000 \\nINFO  @ Thu, 05 Nov 2020 22:16:22:  2000000 \\nINFO  @ Thu, 05 Nov 2020 22:16:26:  3000000 \\nINFO  @ Thu, 05 Nov 2020 22:16:32:  4000000 \\nINFO  @ Thu, 05 Nov 2020 22:16:37:  5000000 \\nINFO  @ Thu, 05 Nov 2020 22:16:41:  6000000 \\nINFO  @ Thu, 05 Nov 2020 22:16:46:  7000000 \\nINFO  @ Thu, 05 Nov 2020 22:16:51:  8000000 \\nINFO  @ Thu, 05 Nov 2020 22:16:56:  9000000 \\nINFO  @ Thu, 05 Nov 2020 22:17:00:  10000000 \\nINFO  @ Thu, 05 Nov 2020 22:17:05:  11000000 \\nINFO  @ Thu, 05 Nov 2020 22:17:10:  12000000 \\nINFO  @ Thu, 05 Nov 2020 22:17:15:  13000000 \\nINFO  @ Thu, 05 Nov 2020 22:17:20:  14000000 \\nINFO  @ Thu, 05 Nov 2020 22:17:24:  15000000 \\nINFO  @ Thu, 05 Nov 2020 22:17:29: 15923184 reads have been read. \\nINFO  @ Thu, 05 Nov 2020 22:17:29: #1.2 read input tags... \\nINFO  @ Thu, 05 Nov 2020 22:17:33:  1000000 \\nINFO  @ Thu, 05 Nov 2020 22:17:37:  2000000 \\nINFO  @ Thu, 05 Nov 2020 22:17:40:  3000000 \\nINFO  @ Thu, 05 Nov 2020 22:17:44:  4000000 \\nINFO  @ Thu, 05 Nov 2020 22:17:48:  5000000 \\nINFO  @ Thu, 05 Nov 2020 22:17:51:  6000000 \\nINFO  @ Thu, 05 Nov 2020 22:17:55:  7000000 \\nINFO  @ Thu, 05 Nov 2020 22:17:59:  8000000 \\nINFO  @ Thu, 05 Nov 2020 22:18:02:  9000000 \\nINFO  @ Thu, 05 Nov 2020 22:18:06:  10000000 \\nINFO  @ Thu, 05 Nov 2020 22:18:10:  11000000 \\nINFO  @ Thu, 05 Nov 2020 22:18:14:  12000000 \\nINFO  @ Thu, 05 Nov 2020 22:18:17:  13000000 \\nINFO  @ Thu, 05 Nov 2020 22:18:21:  14000000 \\nINFO  @ Thu, 05 Nov 2020 22:18:25:  15000000 \\nINFO  @ Thu, 05 Nov 2020 22:18:28:  16000000 \\nINFO  @ Thu, 05 Nov 2020 22:18:32:  17000000 \\nINFO  @ Thu, 05 Nov 2020 22:18:36:  18000000 \\nINFO  @ Thu, 05 Nov 2020 22:18:39:  19000000 \\nINFO  @ Thu, 05 Nov 2020 22:18:43:  20000000 \\nINFO  @ Thu, 05 Nov 2020 22:18:47:  21000000 \\nINFO  @ Thu, 05 Nov 2020 22:18:51:  22000000 \\nINFO  @ Thu, 05 Nov 2020 22:18:54:  23000000 \\nINFO  @ Thu, 05 Nov 2020 22:18:58:  24000000 \\nINFO  @ Thu, 05 Nov 2020 22:19:02:  25000000 \\nINFO  @ Thu, 05 Nov 2020 22:19:05:  26000000 \\nINFO  @ Thu, 05 Nov 2020 22:19:09:  27000000 \\nINFO  @ Thu, 05 Nov 2020 22:19:13:  28000000 \\nINFO  @ Thu, 05 Nov 2020 22:19:17:  29000000 \\nINFO  @ Thu, 05 Nov 2020 22:19:20:  30000000 \\nINFO  @ Thu, 05 Nov 2020 22:19:24:  31000000 \\nINFO  @ Thu, 05 Nov 2020 22:19:28:  32000000 \\nINFO  @ Thu, 05 Nov 2020 22:19:32:  33000000 \\nINFO  @ Thu, 05 Nov 2020 22:19:35:  34000000 \\nINFO  @ Thu, 05 Nov 2020 22:19:39:  35000000 \\nINFO  @ Thu, 05 Nov 2020 22:19:43:  36000000 \\nINFO  @ Thu, 05 Nov 2020 22:19:46:  37000000 \\nINFO  @ Thu, 05 Nov 2020 22:19:50:  38000000 \\nINFO  @ Thu, 05 Nov 2020 22:19:54:  39000000 \\nINFO  @ Thu, 05 Nov 2020 22:19:57:  40000000 \\nINFO  @ Thu, 05 Nov 2020 22:20:01:  41000000 \\nINFO  @ Thu, 05 Nov 2020 22:20:05:  42000000 \\nINFO  @ Thu, 05 Nov 2020 22:20:09:  43000000 \\nINFO  @ Thu, 05 Nov 2020 22:20:12:  44000000 \\nINFO  @ Thu, 05 Nov 2020 22:20:16:  45000000 \\nINFO  @ Thu, 05 Nov 2020 22:20:20:  46000000 \\nINFO  @ Thu, 05 Nov 2020 22:20:23:  47000000 \\nINFO  @ Thu, 05 Nov 2020 22:20:27:  48000000 \\nINFO  @ Thu, 05 Nov 2020 22:20:31:  49000000 \\nINFO  @ Thu, 05 Nov 2020 22:20:35:  50000000 \\nINFO  @ Thu, 05 Nov 2020 22:20:38:  51000000 \\nINFO  @ Thu, 05 Nov 2020 22:20:42:  52000000 \\nINFO  @ Thu, 05 Nov 2020 22:20:46:  53000000 \\nINFO  @ Thu, 05 Nov 2020 22:20:50:  54000000 \\nINFO  @ Thu, 05 Nov 2020 22:20:54:  55000000 \\nINFO  @ Thu, 05 Nov 2020 22:20:57:  56000000 \\nINFO  @ Thu, 05 Nov 2020 22:21:01:  57000000 \\nINFO  @ Thu, 05 Nov 2020 22:21:05:  58000000 \\nINFO  @ Thu, 05 Nov 2020 22:21:09:  59000000 \\nINFO  @ Thu, 05 Nov 2020 22:21:12:  60000000 \\nINFO  @ Thu, 05 Nov 2020 22:21:16:  61000000 \\nINFO  @ Thu, 05 Nov 2020 22:21:20:  62000000 \\nINFO  @ Thu, 05 Nov 2020 22:21:24:  63000000 \\nINFO  @ Thu, 05 Nov 2020 22:21:27:  64000000 \\nINFO  @ Thu, 05 Nov 2020 22:21:31:  65000000 \\nINFO  @ Thu, 05 Nov 2020 22:21:35:  66000000 \\nINFO  @ Thu, 05 Nov 2020 22:21:38: 66660890 reads have been read. \\nINFO  @ Thu, 05 Nov 2020 22:21:38: #1 tag size is determined as 90 bps \\nINFO  @ Thu, 05 Nov 2020 22:21:38: #1 tag size = 90.0 \\nINFO  @ Thu, 05 Nov 2020 22:21:38: #1  total tags in treatment: 15923184 \\nINFO  @ Thu, 05 Nov 2020 22:21:38: #1 user defined the maximum tags... \\nINFO  @ Thu, 05 Nov 2020 22:21:38: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 05 Nov 2020 22:21:38: #1  tags after filtering in treatment: 15800645 \\nINFO  @ Thu, 05 Nov 2020 22:21:38: #1  Redundant rate of treatment: 0.01 \\nINFO  @ Thu, 05 Nov 2020 22:21:38: #1  total tags in control: 66660890 \\nINFO  @ Thu, 05 Nov 2020 22:21:38: #1 user defined the maximum tags... \\nINFO  @ Thu, 05 Nov 2020 22:21:38: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 05 Nov 2020 22:21:39: #1  tags after filtering in control: 65725426 \\nINFO  @ Thu, 05 Nov 2020 22:21:39: #1  Redundant rate of control: 0.01 \\nINFO  @ Thu, 05 Nov 2020 22:21:39: #1 finished! \\nINFO  @ Thu, 05 Nov 2020 22:21:39: #2 Build Peak Model... \\nINFO  @ Thu, 05 Nov 2020 22:21:39: #2 Skipped... \\nINFO  @ Thu, 05 Nov 2020 22:21:39: #2 Use 184 as fragment length \\nINFO  @ Thu, 05 Nov 2020 22:21:39: #3 Call peaks... \\nINFO  @ Thu, 05 Nov 2020 22:21:39: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 05 Nov 2020 22:26:12: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 05 Nov 2020 22:26:12: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_IRF8_NT_VHL-MEF2D_R1_treat_pileup.bdg \\nINFO  @ Thu, 05 Nov 2020 22:26:12: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_IRF8_NT_VHL-MEF2D_R1_control_lambda.bdg \\nINFO  @ Thu, 05 Nov 2020 22:26:12: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 05 Nov 2020 22:26:12: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 05 Nov 2020 22:29:43: #4 Write output xls file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MEF2D_R1_peaks.xls \\nINFO  @ Thu, 05 Nov 2020 22:29:43: #4 Write peak in narrowPeak format file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MEF2D_R1_peaks.narrowPeak \\nINFO  @ Thu, 05 Nov 2020 22:29:43: #4 Write summits bed file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MEF2D_R1_summits.bed \\nINFO  @ Thu, 05 Nov 2020 22:29:43: Done! \\n'\n",
      "17887183 15800645\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MYC_R1.mLb.clN.sorted.bam ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MYC_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MYC_R1.mLb.clN.sorted.bam and ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MYC_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "205\n",
      "computing the scaling values\n",
      "b'INFO  @ Thu, 05 Nov 2020 22:40:36: \\n# Command line: callpeak -B -t ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MYC_R1.mLb.clN.sorted.bam -c ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 205 -n MV411_IRF8_NT_DMSO-MYC_R1 --outdir ../../data/chipseq_IRF8_degraded/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = MV411_IRF8_NT_DMSO-MYC_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_DMSO-MYC_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Thu, 05 Nov 2020 22:40:36: #1 read tag files... \\nINFO  @ Thu, 05 Nov 2020 22:40:36: #1 read treatment tags... \\nINFO  @ Thu, 05 Nov 2020 22:40:41:  1000000 \\nINFO  @ Thu, 05 Nov 2020 22:40:45:  2000000 \\nINFO  @ Thu, 05 Nov 2020 22:40:50:  3000000 \\nINFO  @ Thu, 05 Nov 2020 22:40:54:  4000000 \\nINFO  @ Thu, 05 Nov 2020 22:40:59:  5000000 \\nINFO  @ Thu, 05 Nov 2020 22:41:03:  6000000 \\nINFO  @ Thu, 05 Nov 2020 22:41:08:  7000000 \\nINFO  @ Thu, 05 Nov 2020 22:41:12:  8000000 \\nINFO  @ Thu, 05 Nov 2020 22:41:17:  9000000 \\nINFO  @ Thu, 05 Nov 2020 22:41:21:  10000000 \\nINFO  @ Thu, 05 Nov 2020 22:41:26:  11000000 \\nINFO  @ Thu, 05 Nov 2020 22:41:30:  12000000 \\nINFO  @ Thu, 05 Nov 2020 22:41:35:  13000000 \\nINFO  @ Thu, 05 Nov 2020 22:41:39:  14000000 \\nINFO  @ Thu, 05 Nov 2020 22:41:44:  15000000 \\nINFO  @ Thu, 05 Nov 2020 22:41:48:  16000000 \\nINFO  @ Thu, 05 Nov 2020 22:41:53:  17000000 \\nINFO  @ Thu, 05 Nov 2020 22:41:57:  18000000 \\nINFO  @ Thu, 05 Nov 2020 22:42:02:  19000000 \\nINFO  @ Thu, 05 Nov 2020 22:42:06:  20000000 \\nINFO  @ Thu, 05 Nov 2020 22:42:09: 20590560 reads have been read. \\nINFO  @ Thu, 05 Nov 2020 22:42:09: #1.2 read input tags... \\nINFO  @ Thu, 05 Nov 2020 22:42:13:  1000000 \\nINFO  @ Thu, 05 Nov 2020 22:42:16:  2000000 \\nINFO  @ Thu, 05 Nov 2020 22:42:20:  3000000 \\nINFO  @ Thu, 05 Nov 2020 22:42:24:  4000000 \\nINFO  @ Thu, 05 Nov 2020 22:42:27:  5000000 \\nINFO  @ Thu, 05 Nov 2020 22:42:31:  6000000 \\nINFO  @ Thu, 05 Nov 2020 22:42:34:  7000000 \\nINFO  @ Thu, 05 Nov 2020 22:42:38:  8000000 \\nINFO  @ Thu, 05 Nov 2020 22:42:41:  9000000 \\nINFO  @ Thu, 05 Nov 2020 22:42:45:  10000000 \\nINFO  @ Thu, 05 Nov 2020 22:42:48:  11000000 \\nINFO  @ Thu, 05 Nov 2020 22:42:52:  12000000 \\nINFO  @ Thu, 05 Nov 2020 22:42:55:  13000000 \\nINFO  @ Thu, 05 Nov 2020 22:42:59:  14000000 \\nINFO  @ Thu, 05 Nov 2020 22:43:02:  15000000 \\nINFO  @ Thu, 05 Nov 2020 22:43:06:  16000000 \\nINFO  @ Thu, 05 Nov 2020 22:43:09:  17000000 \\nINFO  @ Thu, 05 Nov 2020 22:43:13:  18000000 \\nINFO  @ Thu, 05 Nov 2020 22:43:16:  19000000 \\nINFO  @ Thu, 05 Nov 2020 22:43:20:  20000000 \\nINFO  @ Thu, 05 Nov 2020 22:43:23:  21000000 \\nINFO  @ Thu, 05 Nov 2020 22:43:27:  22000000 \\nINFO  @ Thu, 05 Nov 2020 22:43:31:  23000000 \\nINFO  @ Thu, 05 Nov 2020 22:43:34:  24000000 \\nINFO  @ Thu, 05 Nov 2020 22:43:38:  25000000 \\nINFO  @ Thu, 05 Nov 2020 22:43:41:  26000000 \\nINFO  @ Thu, 05 Nov 2020 22:43:45:  27000000 \\nINFO  @ Thu, 05 Nov 2020 22:43:48:  28000000 \\nINFO  @ Thu, 05 Nov 2020 22:43:52:  29000000 \\nINFO  @ Thu, 05 Nov 2020 22:43:56:  30000000 \\nINFO  @ Thu, 05 Nov 2020 22:44:00:  31000000 \\nINFO  @ Thu, 05 Nov 2020 22:44:04:  32000000 \\nINFO  @ Thu, 05 Nov 2020 22:44:07:  33000000 \\nINFO  @ Thu, 05 Nov 2020 22:44:11:  34000000 \\nINFO  @ Thu, 05 Nov 2020 22:44:14:  35000000 \\nINFO  @ Thu, 05 Nov 2020 22:44:18:  36000000 \\nINFO  @ Thu, 05 Nov 2020 22:44:21:  37000000 \\nINFO  @ Thu, 05 Nov 2020 22:44:25:  38000000 \\nINFO  @ Thu, 05 Nov 2020 22:44:28:  39000000 \\nINFO  @ Thu, 05 Nov 2020 22:44:32:  40000000 \\nINFO  @ Thu, 05 Nov 2020 22:44:35:  41000000 \\nINFO  @ Thu, 05 Nov 2020 22:44:39:  42000000 \\nINFO  @ Thu, 05 Nov 2020 22:44:43:  43000000 \\nINFO  @ Thu, 05 Nov 2020 22:44:46:  44000000 \\nINFO  @ Thu, 05 Nov 2020 22:44:50:  45000000 \\nINFO  @ Thu, 05 Nov 2020 22:44:53:  46000000 \\nINFO  @ Thu, 05 Nov 2020 22:44:57:  47000000 \\nINFO  @ Thu, 05 Nov 2020 22:45:00:  48000000 \\nINFO  @ Thu, 05 Nov 2020 22:45:04:  49000000 \\nINFO  @ Thu, 05 Nov 2020 22:45:08:  50000000 \\nINFO  @ Thu, 05 Nov 2020 22:45:12:  51000000 \\nINFO  @ Thu, 05 Nov 2020 22:45:16:  52000000 \\nINFO  @ Thu, 05 Nov 2020 22:45:19:  53000000 \\nINFO  @ Thu, 05 Nov 2020 22:45:23:  54000000 \\nINFO  @ Thu, 05 Nov 2020 22:45:26:  55000000 \\nINFO  @ Thu, 05 Nov 2020 22:45:30:  56000000 \\nINFO  @ Thu, 05 Nov 2020 22:45:34:  57000000 \\nINFO  @ Thu, 05 Nov 2020 22:45:37:  58000000 \\nINFO  @ Thu, 05 Nov 2020 22:45:41:  59000000 \\nINFO  @ Thu, 05 Nov 2020 22:45:45:  60000000 \\nINFO  @ Thu, 05 Nov 2020 22:45:48:  61000000 \\nINFO  @ Thu, 05 Nov 2020 22:45:52:  62000000 \\nINFO  @ Thu, 05 Nov 2020 22:45:55:  63000000 \\nINFO  @ Thu, 05 Nov 2020 22:45:59:  64000000 \\nINFO  @ Thu, 05 Nov 2020 22:46:03:  65000000 \\nINFO  @ Thu, 05 Nov 2020 22:46:06:  66000000 \\nINFO  @ Thu, 05 Nov 2020 22:46:09: 66660890 reads have been read. \\nINFO  @ Thu, 05 Nov 2020 22:46:10: #1 tag size is determined as 94 bps \\nINFO  @ Thu, 05 Nov 2020 22:46:10: #1 tag size = 94.0 \\nINFO  @ Thu, 05 Nov 2020 22:46:10: #1  total tags in treatment: 20590560 \\nINFO  @ Thu, 05 Nov 2020 22:46:10: #1 user defined the maximum tags... \\nINFO  @ Thu, 05 Nov 2020 22:46:10: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 05 Nov 2020 22:46:10: #1  tags after filtering in treatment: 20358654 \\nINFO  @ Thu, 05 Nov 2020 22:46:10: #1  Redundant rate of treatment: 0.01 \\nINFO  @ Thu, 05 Nov 2020 22:46:10: #1  total tags in control: 66660890 \\nINFO  @ Thu, 05 Nov 2020 22:46:10: #1 user defined the maximum tags... \\nINFO  @ Thu, 05 Nov 2020 22:46:10: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 05 Nov 2020 22:46:10: #1  tags after filtering in control: 65725426 \\nINFO  @ Thu, 05 Nov 2020 22:46:10: #1  Redundant rate of control: 0.01 \\nINFO  @ Thu, 05 Nov 2020 22:46:10: #1 finished! \\nINFO  @ Thu, 05 Nov 2020 22:46:10: #2 Build Peak Model... \\nINFO  @ Thu, 05 Nov 2020 22:46:10: #2 Skipped... \\nINFO  @ Thu, 05 Nov 2020 22:46:10: #2 Use 205 as fragment length \\nINFO  @ Thu, 05 Nov 2020 22:46:10: #3 Call peaks... \\nINFO  @ Thu, 05 Nov 2020 22:46:10: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 05 Nov 2020 22:50:49: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 05 Nov 2020 22:50:49: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_IRF8_NT_DMSO-MYC_R1_treat_pileup.bdg \\nINFO  @ Thu, 05 Nov 2020 22:50:49: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_IRF8_NT_DMSO-MYC_R1_control_lambda.bdg \\nINFO  @ Thu, 05 Nov 2020 22:50:49: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 05 Nov 2020 22:50:49: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 05 Nov 2020 22:54:33: #4 Write output xls file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MYC_R1_peaks.xls \\nINFO  @ Thu, 05 Nov 2020 22:54:33: #4 Write peak in narrowPeak format file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MYC_R1_peaks.narrowPeak \\nINFO  @ Thu, 05 Nov 2020 22:54:33: #4 Write summits bed file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MYC_R1_summits.bed \\nINFO  @ Thu, 05 Nov 2020 22:54:33: Done! \\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'INFO  @ Thu, 05 Nov 2020 22:54:33: \\n# Command line: callpeak -B -t ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MYC_R1.mLb.clN.sorted.bam -c ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 205 -n MV411_IRF8_NT_VHL-MYC_R1 --outdir ../../data/chipseq_IRF8_degraded/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = MV411_IRF8_NT_VHL-MYC_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/MV411_IRF8_NT_VHL-MYC_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Thu, 05 Nov 2020 22:54:33: #1 read tag files... \\nINFO  @ Thu, 05 Nov 2020 22:54:33: #1 read treatment tags... \\nINFO  @ Thu, 05 Nov 2020 22:54:40:  1000000 \\nINFO  @ Thu, 05 Nov 2020 22:54:45:  2000000 \\nINFO  @ Thu, 05 Nov 2020 22:54:49:  3000000 \\nINFO  @ Thu, 05 Nov 2020 22:54:54:  4000000 \\nINFO  @ Thu, 05 Nov 2020 22:54:59:  5000000 \\nINFO  @ Thu, 05 Nov 2020 22:55:03:  6000000 \\nINFO  @ Thu, 05 Nov 2020 22:55:08:  7000000 \\nINFO  @ Thu, 05 Nov 2020 22:55:13:  8000000 \\nINFO  @ Thu, 05 Nov 2020 22:55:17:  9000000 \\nINFO  @ Thu, 05 Nov 2020 22:55:22:  10000000 \\nINFO  @ Thu, 05 Nov 2020 22:55:26:  11000000 \\nINFO  @ Thu, 05 Nov 2020 22:55:31:  12000000 \\nINFO  @ Thu, 05 Nov 2020 22:55:36:  13000000 \\nINFO  @ Thu, 05 Nov 2020 22:55:40:  14000000 \\nINFO  @ Thu, 05 Nov 2020 22:55:45:  15000000 \\nINFO  @ Thu, 05 Nov 2020 22:55:50:  16000000 \\nINFO  @ Thu, 05 Nov 2020 22:55:54:  17000000 \\nINFO  @ Thu, 05 Nov 2020 22:55:59:  18000000 \\nINFO  @ Thu, 05 Nov 2020 22:56:02: 18592701 reads have been read. \\nINFO  @ Thu, 05 Nov 2020 22:56:02: #1.2 read input tags... \\nINFO  @ Thu, 05 Nov 2020 22:56:05:  1000000 \\nINFO  @ Thu, 05 Nov 2020 22:56:09:  2000000 \\nINFO  @ Thu, 05 Nov 2020 22:56:13:  3000000 \\nINFO  @ Thu, 05 Nov 2020 22:56:16:  4000000 \\nINFO  @ Thu, 05 Nov 2020 22:56:20:  5000000 \\nINFO  @ Thu, 05 Nov 2020 22:56:23:  6000000 \\nINFO  @ Thu, 05 Nov 2020 22:56:27:  7000000 \\nINFO  @ Thu, 05 Nov 2020 22:56:31:  8000000 \\nINFO  @ Thu, 05 Nov 2020 22:56:34:  9000000 \\nINFO  @ Thu, 05 Nov 2020 22:56:38:  10000000 \\nINFO  @ Thu, 05 Nov 2020 22:56:42:  11000000 \\nINFO  @ Thu, 05 Nov 2020 22:56:45:  12000000 \\nINFO  @ Thu, 05 Nov 2020 22:56:49:  13000000 \\nINFO  @ Thu, 05 Nov 2020 22:56:53:  14000000 \\nINFO  @ Thu, 05 Nov 2020 22:56:56:  15000000 \\nINFO  @ Thu, 05 Nov 2020 22:57:00:  16000000 \\nINFO  @ Thu, 05 Nov 2020 22:57:03:  17000000 \\nINFO  @ Thu, 05 Nov 2020 22:57:07:  18000000 \\nINFO  @ Thu, 05 Nov 2020 22:57:10:  19000000 \\nINFO  @ Thu, 05 Nov 2020 22:57:14:  20000000 \\nINFO  @ Thu, 05 Nov 2020 22:57:18:  21000000 \\nINFO  @ Thu, 05 Nov 2020 22:57:21:  22000000 \\nINFO  @ Thu, 05 Nov 2020 22:57:25:  23000000 \\nINFO  @ Thu, 05 Nov 2020 22:57:29:  24000000 \\nINFO  @ Thu, 05 Nov 2020 22:57:32:  25000000 \\nINFO  @ Thu, 05 Nov 2020 22:57:36:  26000000 \\nINFO  @ Thu, 05 Nov 2020 22:57:40:  27000000 \\nINFO  @ Thu, 05 Nov 2020 22:57:43:  28000000 \\nINFO  @ Thu, 05 Nov 2020 22:57:47:  29000000 \\nINFO  @ Thu, 05 Nov 2020 22:57:50:  30000000 \\nINFO  @ Thu, 05 Nov 2020 22:57:54:  31000000 \\nINFO  @ Thu, 05 Nov 2020 22:57:58:  32000000 \\nINFO  @ Thu, 05 Nov 2020 22:58:01:  33000000 \\nINFO  @ Thu, 05 Nov 2020 22:58:05:  34000000 \\nINFO  @ Thu, 05 Nov 2020 22:58:08:  35000000 \\nINFO  @ Thu, 05 Nov 2020 22:58:12:  36000000 \\nINFO  @ Thu, 05 Nov 2020 22:58:15:  37000000 \\nINFO  @ Thu, 05 Nov 2020 22:58:19:  38000000 \\nINFO  @ Thu, 05 Nov 2020 22:58:22:  39000000 \\nINFO  @ Thu, 05 Nov 2020 22:58:26:  40000000 \\nINFO  @ Thu, 05 Nov 2020 22:58:29:  41000000 \\nINFO  @ Thu, 05 Nov 2020 22:58:33:  42000000 \\nINFO  @ Thu, 05 Nov 2020 22:58:36:  43000000 \\nINFO  @ Thu, 05 Nov 2020 22:58:40:  44000000 \\nINFO  @ Thu, 05 Nov 2020 22:58:44:  45000000 \\nINFO  @ Thu, 05 Nov 2020 22:58:47:  46000000 \\nINFO  @ Thu, 05 Nov 2020 22:58:51:  47000000 \\nINFO  @ Thu, 05 Nov 2020 22:58:55:  48000000 \\nINFO  @ Thu, 05 Nov 2020 22:58:58:  49000000 \\nINFO  @ Thu, 05 Nov 2020 22:59:02:  50000000 \\nINFO  @ Thu, 05 Nov 2020 22:59:05:  51000000 \\nINFO  @ Thu, 05 Nov 2020 22:59:09:  52000000 \\nINFO  @ Thu, 05 Nov 2020 22:59:13:  53000000 \\nINFO  @ Thu, 05 Nov 2020 22:59:16:  54000000 \\nINFO  @ Thu, 05 Nov 2020 22:59:20:  55000000 \\nINFO  @ Thu, 05 Nov 2020 22:59:24:  56000000 \\nINFO  @ Thu, 05 Nov 2020 22:59:27:  57000000 \\nINFO  @ Thu, 05 Nov 2020 22:59:31:  58000000 \\nINFO  @ Thu, 05 Nov 2020 22:59:34:  59000000 \\nINFO  @ Thu, 05 Nov 2020 22:59:38:  60000000 \\nINFO  @ Thu, 05 Nov 2020 22:59:42:  61000000 \\nINFO  @ Thu, 05 Nov 2020 22:59:45:  62000000 \\nINFO  @ Thu, 05 Nov 2020 22:59:49:  63000000 \\nINFO  @ Thu, 05 Nov 2020 22:59:53:  64000000 \\nINFO  @ Thu, 05 Nov 2020 22:59:56:  65000000 \\nINFO  @ Thu, 05 Nov 2020 23:00:00:  66000000 \\nINFO  @ Thu, 05 Nov 2020 23:00:02: 66660890 reads have been read. \\nINFO  @ Thu, 05 Nov 2020 23:00:03: #1 tag size is determined as 94 bps \\nINFO  @ Thu, 05 Nov 2020 23:00:03: #1 tag size = 94.0 \\nINFO  @ Thu, 05 Nov 2020 23:00:03: #1  total tags in treatment: 18592701 \\nINFO  @ Thu, 05 Nov 2020 23:00:03: #1 user defined the maximum tags... \\nINFO  @ Thu, 05 Nov 2020 23:00:03: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 05 Nov 2020 23:00:03: #1  tags after filtering in treatment: 18406437 \\nINFO  @ Thu, 05 Nov 2020 23:00:03: #1  Redundant rate of treatment: 0.01 \\nINFO  @ Thu, 05 Nov 2020 23:00:03: #1  total tags in control: 66660890 \\nINFO  @ Thu, 05 Nov 2020 23:00:03: #1 user defined the maximum tags... \\nINFO  @ Thu, 05 Nov 2020 23:00:03: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 05 Nov 2020 23:00:03: #1  tags after filtering in control: 65725426 \\nINFO  @ Thu, 05 Nov 2020 23:00:03: #1  Redundant rate of control: 0.01 \\nINFO  @ Thu, 05 Nov 2020 23:00:03: #1 finished! \\nINFO  @ Thu, 05 Nov 2020 23:00:03: #2 Build Peak Model... \\nINFO  @ Thu, 05 Nov 2020 23:00:03: #2 Skipped... \\nINFO  @ Thu, 05 Nov 2020 23:00:03: #2 Use 205 as fragment length \\nINFO  @ Thu, 05 Nov 2020 23:00:03: #3 Call peaks... \\nINFO  @ Thu, 05 Nov 2020 23:00:03: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 05 Nov 2020 23:04:35: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 05 Nov 2020 23:04:35: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... MV411_IRF8_NT_VHL-MYC_R1_treat_pileup.bdg \\nINFO  @ Thu, 05 Nov 2020 23:04:35: #3   Write bedGraph files for control lambda (after scaling if necessary)... MV411_IRF8_NT_VHL-MYC_R1_control_lambda.bdg \\nINFO  @ Thu, 05 Nov 2020 23:04:35: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 05 Nov 2020 23:04:35: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 05 Nov 2020 23:08:14: #4 Write output xls file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MYC_R1_peaks.xls \\nINFO  @ Thu, 05 Nov 2020 23:08:14: #4 Write peak in narrowPeak format file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MYC_R1_peaks.narrowPeak \\nINFO  @ Thu, 05 Nov 2020 23:08:14: #4 Write summits bed file... ../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MYC_R1_summits.bed \\nINFO  @ Thu, 05 Nov 2020 23:08:14: Done! \\n'\n",
      "20358654 18406437\n",
      "doing differential peak binding\n"
     ]
    }
   ],
   "source": [
    "#on unscalled data \n",
    "for i in range(int(len(bams)/2)):\n",
    "    if i < 0:\n",
    "        continue\n",
    "    name1 = bams[i]\n",
    "    name2 = bams[6+i]\n",
    "    print(name1,name2)\n",
    "    chip.fullDiffPeak(name1,name2, control1='../../data/'+project+'/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam', directory = \"../../data/\"+project+\"/diffData_unscaled/\", res_directory = \"../../data/\"+project+\"/diffPeaks_unscaled/\",pairedend=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 277,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/INPUT_R1.bigWig',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-IRF8_R1.bigWig',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-MED1_R1.bigWig',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-MEF2C_R1.bigWig',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-MEF2D_R1.bigWig',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-MYC_R1.bigWig',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-POLII_total_R1.bigWig',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_VHL-IRF8_R1.bigWig',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_VHL-MED1_R1.bigWig',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_VHL-MEF2C_R1.bigWig',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_VHL-MEF2D_R1.bigWig',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_VHL-MYC_R1.bigWig',\n",
       " '../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_VHL-POLII_total_R1.bigWig']"
      ]
     },
     "execution_count": 277,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bw = ! ls ../../data/$project/bwa/mergedLibrary/bigwig/*.bigWig\n",
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 278,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/INPUT_R1_treat_pileup_vs_MV411_IRF8_NT_DMSO-POLII_total_R1_treat_pileup_c3.0_common.bed',\n",
       " '../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-IRF8_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-IRF8_R1_treat_pileup_c3.0_common.bed',\n",
       " '../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-MED1_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MED1_R1_treat_pileup_c3.0_common.bed',\n",
       " '../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-MEF2C_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2C_R1_treat_pileup_c3.0_common.bed',\n",
       " '../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-MEF2D_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2D_R1_treat_pileup_c3.0_common.bed',\n",
       " '../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-MYC_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MYC_R1_treat_pileup_c3.0_common.bed']"
      ]
     },
     "execution_count": 278,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "cond1peak = ! ls ../../data/$project/diffPeaks_unscaled/*cond1.bed\n",
    "cond2peak = ! ls ../../data/$project/diffPeaks_unscaled/*cond2.bed\n",
    "commonpeak = ! ls ../../data/$project/diffPeaks_unscaled/*common.bed\n",
    "commonpeak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 279,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/INPUT_R1.bigWig ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-POLII_total_R1.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/INPUT_R1.bigWig ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-POLII_total_R1.bigWig  --referencePoint center --regionsFileName ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/INPUT_R1_treat_pileup_vs_MV411_IRF8_NT_DMSO-POLII_total_R1_treat_pileup_c3.0_cond2.bed --missingDataAsZero --outFileName ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/IRF8_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/IRF8_mat.gz --outFileName ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/IRF8_mat.pdf --refPointLabel center --regionsLabel VHL_peaks --plotTitle IRF8', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270340v1\\t      1428\\nchrUn_KI270381v1\\t      1930\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270335v1\\t      1048\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270394v1\\t       970\\nchrUn_KI270373v1\\t      1451\\nchrUn_KI270752v1\\t     27745\\n         chrEBV\\t    171823\\nchrUn_KI270374v1\\t      2656\\n')\n",
      "../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-IRF8_R1.bigWig ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_VHL-IRF8_R1.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-IRF8_R1.bigWig ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_VHL-IRF8_R1.bigWig  --referencePoint center --regionsFileName ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-IRF8_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-IRF8_R1_treat_pileup_c3.0_cond1.bed ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-IRF8_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-IRF8_R1_treat_pileup_c3.0_common.bed ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-IRF8_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-IRF8_R1_treat_pileup_c3.0_cond2.bed --missingDataAsZero --outFileName ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MED1_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MED1_mat.gz --outFileName ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MED1_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks --plotTitle MED1', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270390v1\\t      2387\\nchrUn_KI270385v1\\t       990\\nchrUn_KI270329v1\\t      1040\\nchrUn_KI270392v1\\t       971\\nchrUn_KI270381v1\\t      1930\\nchrUn_KI270539v1\\t       993\\nchrUn_KI270411v1\\t      2646\\nchrUn_KI270371v1\\t      2805\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270394v1\\t       970\\nchrUn_KI270414v1\\t      2489\\n')\n",
      "../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-MED1_R1.bigWig ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_VHL-MED1_R1.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-MED1_R1.bigWig ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_VHL-MED1_R1.bigWig  --referencePoint center --regionsFileName ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-MED1_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MED1_R1_treat_pileup_c3.0_cond1.bed ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-MED1_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MED1_R1_treat_pileup_c3.0_common.bed ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-MED1_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MED1_R1_treat_pileup_c3.0_cond2.bed --missingDataAsZero --outFileName ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MEF2C_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MEF2C_mat.gz --outFileName ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MEF2C_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks --plotTitle MEF2C', returncode=0, stdout=b'', stderr=b\"\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270386v1\\t      1788\\nchrUn_KI270340v1\\t      1428\\nchrUn_KI270387v1\\t      1537\\nchrUn_KI270425v1\\t      1884\\nchrUn_KI270539v1\\t       993\\nchrUn_KI270394v1\\t       970\\nchrUn_KI270414v1\\t      2489\\nchrUn_KI270374v1\\t      2656\\nOne of the groups defined in the bed file is too small.\\nGroups that are too small can't be plotted. \\nWARNING: Group 'MV411_IRF8_NT_DMSO-MED1_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MED1_R1_treat_pileup_c3.0_cond2.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n",
      "../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-MEF2C_R1.bigWig ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_VHL-MEF2C_R1.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-MEF2C_R1.bigWig ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_VHL-MEF2C_R1.bigWig  --referencePoint center --regionsFileName ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-MEF2C_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2C_R1_treat_pileup_c3.0_cond1.bed ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-MEF2C_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2C_R1_treat_pileup_c3.0_common.bed ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-MEF2C_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2C_R1_treat_pileup_c3.0_cond2.bed --missingDataAsZero --outFileName ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MEF2D_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MEF2D_mat.gz --outFileName ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MEF2D_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks --plotTitle MEF2D', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270381v1\\t      1930\\n         chrEBV\\t    171823\\nchrUn_KI270371v1\\t      2805\\nchrUn_KI270312v1\\t       998\\nchrUn_KI270425v1\\t      1884\\nchrUn_KI270334v1\\t      1368\\nchrUn_KI270539v1\\t       993\\n')\n",
      "../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-MEF2D_R1.bigWig ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_VHL-MEF2D_R1.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-MEF2D_R1.bigWig ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_VHL-MEF2D_R1.bigWig  --referencePoint center --regionsFileName ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-MEF2D_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2D_R1_treat_pileup_c3.0_common.bed --missingDataAsZero --outFileName ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MYC_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MYC_mat.gz --outFileName ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MYC_mat.pdf --refPointLabel center --regionsLabel common --plotTitle MYC', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270579v1\\t     31033\\nchrUn_KI270388v1\\t      1216\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270334v1\\t      1368\\nchrUn_KI270335v1\\t      1048\\nchrUn_KI270329v1\\t      1040\\nchrUn_KI270414v1\\t      2489\\nchrUn_KI270387v1\\t      1537\\nchrUn_KI270383v1\\t      1750\\nchrUn_KI270371v1\\t      2805\\nchrUn_KI270390v1\\t      2387\\n')\n",
      "../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-MYC_R1.bigWig ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_VHL-MYC_R1.bigWig\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_DMSO-MYC_R1.bigWig ../../data/chipseq_IRF8_degraded/bwa/mergedLibrary/bigwig/MV411_IRF8_NT_VHL-MYC_R1.bigWig  --referencePoint center --regionsFileName ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-MYC_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MYC_R1_treat_pileup_c3.0_cond1.bed ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-MYC_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MYC_R1_treat_pileup_c3.0_common.bed ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-MYC_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MYC_R1_treat_pileup_c3.0_cond2.bed --missingDataAsZero --outFileName ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/POLII_total_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 8 && plotHeatmap --matrixFile ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/POLII_total_mat.gz --outFileName ../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/POLII_total_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks --plotTitle POLII_total', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270381v1\\t      1930\\nchrUn_KI270329v1\\t      1040\\nchrUn_KI270335v1\\t      1048\\nchrUn_KI270334v1\\t      1368\\nchrUn_KI270414v1\\t      2489\\nchrUn_KI270394v1\\t       970\\nchrUn_KI270388v1\\t      1216\\nchrUn_KI270387v1\\t      1537\\nchrUn_KI270412v1\\t      1179\\n')\n"
     ]
    }
   ],
   "source": [
    "for i in range(int((len(bw)-1)/2)):\n",
    "    name1 = bw[i]\n",
    "    name2 = bw[6+i]\n",
    "    peak = [cond1peak[i], commonpeak[i], cond2peak[i]]\n",
    "    #for val in peak:\n",
    "     #   chip.dropWeirdChromosomes(val)\n",
    "    name = names[i]\n",
    "    print(name1,name2)\n",
    "    chip.getPeaksAt(peak, [name1, name2], bigwignames=['DMSO', 'VHL'], peaknames=['DMSO_peaks', 'common', 'VHL_peaks'], window=3000, folder=\"\", title=name, numthreads=8, torecompute=True, refpoint='center', name='../../data/'+project+'/diffPeaks_unscaled/'+name+'_mat.pdf', withDeeptools=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 280,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotFingerprint/MV411_IRF8_NT_DMSO-IRF8_R1.plotFingerprint.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotFingerprint/MV411_IRF8_NT_DMSO-MED1_R1.plotFingerprint.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotFingerprint/MV411_IRF8_NT_DMSO-MEF2C_R1.plotFingerprint.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotFingerprint/MV411_IRF8_NT_DMSO-MEF2D_R1.plotFingerprint.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotFingerprint/MV411_IRF8_NT_DMSO-MYC_R1.plotFingerprint.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotFingerprint/MV411_IRF8_NT_DMSO-POLII_total_R1.plotFingerprint.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotFingerprint/MV411_IRF8_NT_VHL-IRF8_R1.plotFingerprint.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotFingerprint/MV411_IRF8_NT_VHL-MED1_R1.plotFingerprint.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotFingerprint/MV411_IRF8_NT_VHL-MEF2C_R1.plotFingerprint.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotFingerprint/MV411_IRF8_NT_VHL-MEF2D_R1.plotFingerprint.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotFingerprint/MV411_IRF8_NT_VHL-MYC_R1.plotFingerprint.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotFingerprint/MV411_IRF8_NT_VHL-POLII_total_R1.plotFingerprint.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotProfile/INPUT_R1.plotHeatmap.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotProfile/INPUT_R1.plotProfile.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_IRF8_NT_DMSO-IRF8_R1.plotHeatmap.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_IRF8_NT_DMSO-IRF8_R1.plotProfile.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_IRF8_NT_DMSO-MED1_R1.plotHeatmap.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_IRF8_NT_DMSO-MED1_R1.plotProfile.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_IRF8_NT_DMSO-MEF2C_R1.plotHeatmap.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_IRF8_NT_DMSO-MEF2C_R1.plotProfile.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_IRF8_NT_DMSO-MEF2D_R1.plotHeatmap.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_IRF8_NT_DMSO-MEF2D_R1.plotProfile.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_IRF8_NT_DMSO-MYC_R1.plotHeatmap.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_IRF8_NT_DMSO-MYC_R1.plotProfile.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_IRF8_NT_DMSO-POLII_total_R1.plotHeatmap.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_IRF8_NT_DMSO-POLII_total_R1.plotProfile.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_IRF8_NT_VHL-IRF8_R1.plotProfile.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_IRF8_NT_VHL-IRF8_R1.plotHeatmap.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_IRF8_NT_VHL-MEF2C_R1.plotHeatmap.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_IRF8_NT_VHL-MED1_R1.plotProfile.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_IRF8_NT_VHL-MEF2C_R1.plotProfile.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_IRF8_NT_VHL-MED1_R1.plotHeatmap.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_IRF8_NT_VHL-MEF2D_R1.plotHeatmap.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_IRF8_NT_VHL-MEF2D_R1.plotProfile.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_IRF8_NT_VHL-MYC_R1.plotHeatmap.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_IRF8_NT_VHL-POLII_total_R1.plotHeatmap.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_IRF8_NT_VHL-MYC_R1.plotProfile.pdf...\n",
      "Copying gs://amlproject/Chip/chipseq_IRF8_degraded/bwa/mergedLibrary/deepTools/plotProfile/MV411_IRF8_NT_VHL-POLII_total_R1.plotProfile.pdf...\n",
      "- [38/38 files][ 14.5 MiB/ 14.5 MiB] 100% Done                                  \n",
      "Operation completed over 38 objects/14.5 MiB.                                    \n"
     ]
    }
   ],
   "source": [
    "h.createFoldersFor('../results/'+project+'/plots/unscaled/diffPeaks/')\n",
    "! cp ../../data/$project/diffPeaks_unscaled/*.pdf ../results/$project/plots/unscaled/diffPeaks/\n",
    "\n",
    "! gsutil -m cp gs://amlproject/Chip/$project/bwa/mergedLibrary/deepTools/**.pdf ../results/$project/plots/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 281,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MEF2D_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2D_R1_treat_pileup_c3.0_cond1.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MEF2C_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2C_R1_treat_pileup_c3.0_cond2.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/MED1_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/IRF8_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-POLII_total_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-POLII_total_R1_treat_pileup_c3.0_cond2.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-IRF8_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-IRF8_R1_treat_pileup_c3.0_common.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MYC_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MYC_R1_treat_pileup_c3.0_common.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/MEF2C_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MED1_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MED1_R1_treat_pileup_c3.0_cond1.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MEF2D_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2D_R1_treat_pileup_c3.0_cond2.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MYC_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MYC_R1_treat_pileup_c3.0_cond2.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-IRF8_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-IRF8_R1_treat_pileup_c3.0_cond1.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-IRF8_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-IRF8_R1_treat_pileup_c3.0_cond2.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MED1_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MED1_R1_treat_pileup_c3.0_cond2.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/MEF2C_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/POLII_total_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/MEF2D_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MEF2D_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2D_R1_treat_pileup_c3.0_common.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/IRF8_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MEF2C_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2C_R1_treat_pileup_c3.0_cond1.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/POLII_total_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MED1_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MED1_R1_treat_pileup_c3.0_common.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MYC_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MYC_R1_treat_pileup_c3.0_cond1.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/MED1_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-POLII_total_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-POLII_total_R1_treat_pileup_c3.0_common.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-POLII_total_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-POLII_total_R1_treat_pileup_c3.0_cond1.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/MYC_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/MEF2D_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/MV411_IRF8_NT_DMSO-MEF2C_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2C_R1_treat_pileup_c3.0_common.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks/MYC_mat.pdf [Content-Type=application/pdf]...\n",
      "\\ [30/30 files][ 90.4 MiB/ 90.4 MiB] 100% Done                                  \n",
      "Operation completed over 30 objects/90.4 MiB.                                    \n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MED1_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MED1_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "==> NOTE: You are uploading one or more large file(s), which would run\n",
      "significantly faster if you enable parallel composite uploads. This\n",
      "feature can be enabled by editing the\n",
      "\"parallel_composite_upload_threshold\" value in your .boto\n",
      "configuration file. However, note that if you do this large files will\n",
      "be uploaded as `composite objects\n",
      "<https://cloud.google.com/storage/docs/composite-objects>`_,which\n",
      "means that any user who downloads such objects will need to have a\n",
      "compiled crcmod installed (see \"gsutil help crcmod\"). This is because\n",
      "without a compiled crcmod, computing checksums on composite objects is\n",
      "so slow that gsutil disables downloads of composite objects.\n",
      "\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MEF2D_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MEF2C_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MYC_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MED1_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MED1_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-IRF8_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-POLII_total_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MED1_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MEF2C_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MEF2C_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MEF2D_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MEF2C_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MED1_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MEF2D_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-IRF8_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MED1_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MEF2D_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-POLII_total_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-IRF8_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MYC_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MYC_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MEF2D_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-IRF8_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-POLII_total_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MEF2C_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-POLII_total_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MEF2D_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-POLII_total_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-POLII_total_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MEF2D_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MEF2D_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MYC_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MYC_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MEF2C_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MYC_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-POLII_total_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MEF2D_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MED1_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MED1_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MEF2C_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-POLII_total_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-POLII_total_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-POLII_total_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-IRF8_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MYC_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-IRF8_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MYC_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MEF2C_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MEF2C_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MED1_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-MEF2C_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-IRF8_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MEF2D_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-IRF8_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_VHL-IRF8_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-IRF8_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MYC_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData/MV411_IRF8_NT_DMSO-MYC_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "\\ [60/60 files][ 84.3 GiB/ 84.3 GiB] 100% Done 128.5 MiB/s ETA 00:00:00         \n",
      "Operation completed over 60 objects/84.3 GiB.                                    \n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/INPUT_R1_treat_pileup_vs_MV411_IRF8_NT_DMSO-POLII_total_R1_treat_pileup_c3.0_cond1.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-MEF2D_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2D_R1_treat_pileup_c3.0_cond1.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-MYC_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MYC_R1_treat_pileup_c3.0_common.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-MED1_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MED1_R1_treat_pileup_c3.0_cond1.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-MYC_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MYC_R1_treat_pileup_c3.0_cond2.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-MEF2D_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2D_R1_treat_pileup_c3.0_cond2.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MEF2C_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-MEF2C_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2C_R1_treat_pileup_c3.0_cond2.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-MED1_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MED1_R1_treat_pileup_c3.0_cond2.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MED1_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MEF2C_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-IRF8_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-IRF8_R1_treat_pileup_c3.0_common.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/IRF8_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-IRF8_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-IRF8_R1_treat_pileup_c3.0_cond1.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-IRF8_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-IRF8_R1_treat_pileup_c3.0_cond2.bed [Content-Type=application/octet-stream]...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MEF2D_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-MEF2C_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2C_R1_treat_pileup_c3.0_cond1.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-MED1_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MED1_R1_treat_pileup_c3.0_common.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/POLII_total_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-MYC_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MYC_R1_treat_pileup_c3.0_cond1.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/IRF8_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-MEF2D_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2D_R1_treat_pileup_c3.0_common.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MED1_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MYC_mat.gz [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/POLII_total_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/INPUT_R1_treat_pileup_vs_MV411_IRF8_NT_DMSO-POLII_total_R1_treat_pileup_c3.0_cond2.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MEF2D_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MV411_IRF8_NT_DMSO-MEF2C_R1_treat_pileup_vs_MV411_IRF8_NT_VHL-MEF2C_R1_treat_pileup_c3.0_common.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/MYC_mat.pdf [Content-Type=application/pdf]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffPeaks_unscaled/INPUT_R1_treat_pileup_vs_MV411_IRF8_NT_DMSO-POLII_total_R1_treat_pileup_c3.0_common.bed [Content-Type=application/octet-stream]...\n",
      "\\ [30/30 files][132.9 MiB/132.9 MiB] 100% Done                                  \n",
      "Operation completed over 30 objects/132.9 MiB.                                   \n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/INPUT_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MED1_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MED1_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MEF2D_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "==> NOTE: You are uploading one or more large file(s), which would run\n",
      "significantly faster if you enable parallel composite uploads. This\n",
      "feature can be enabled by editing the\n",
      "\"parallel_composite_upload_threshold\" value in your .boto\n",
      "configuration file. However, note that if you do this large files will\n",
      "be uploaded as `composite objects\n",
      "<https://cloud.google.com/storage/docs/composite-objects>`_,which\n",
      "means that any user who downloads such objects will need to have a\n",
      "compiled crcmod installed (see \"gsutil help crcmod\"). This is because\n",
      "without a compiled crcmod, computing checksums on composite objects is\n",
      "so slow that gsutil disables downloads of composite objects.\n",
      "\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MEF2C_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MYC_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MED1_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MED1_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-IRF8_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/INPUT_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MED1_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MEF2C_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MEF2C_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MED1_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MEF2C_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MYC_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MEF2D_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MEF2D_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MYC_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-IRF8_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MEF2D_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-IRF8_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MED1_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MEF2D_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/INPUT_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-POLII_total_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MEF2D_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/INPUT_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-IRF8_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MEF2C_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MEF2D_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MEF2C_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-POLII_total_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MEF2D_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MED1_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MED1_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MYC_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MYC_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MYC_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-POLII_total_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MEF2C_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-POLII_total_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-POLII_total_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MEF2C_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MYC_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MYC_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-IRF8_R1_summits.bed [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MEF2C_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MYC_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MED1_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MEF2D_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-MEF2C_R1_treat_pileup.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-IRF8_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MEF2D_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-IRF8_R1_peaks.narrowPeak [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-MYC_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_VHL-IRF8_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/INPUT_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-IRF8_R1_peaks.xls [Content-Type=application/vnd.ms-excel]...\n",
      "Copying file://../../data/chipseq_IRF8_degraded/diffData_unscaled/MV411_IRF8_NT_DMSO-IRF8_R1_control_lambda.bdg [Content-Type=application/octet-stream]...\n",
      "/ [60/60 files][ 57.3 GiB/ 57.3 GiB] 100% Done 148.6 MiB/s ETA 00:00:00         \n",
      "Operation completed over 60 objects/57.3 GiB.                                    \n",
      "CommandException: No URLs matched: ../../data/chipseq_IRF8_degraded/peakplot\n",
      "CommandException: 1 file/object could not be transferred.\n"
     ]
    }
   ],
   "source": [
    "! gsutil -m cp -r ../../data/$project/diffPeaks gs://amlproject/Chip/$project/\n",
    "! gsutil -m cp -r ../../data/$project/diffData gs://amlproject/Chip/$project/\n",
    "! gsutil -m cp -r ../../data/$project/diffPeaks_unscaled gs://amlproject/Chip/$project/\n",
    "! gsutil -m cp -r ../../data/$project/diffData_unscaled gs://amlproject/Chip/$project/\n",
    "! gsutil -m cp -r ../../data/$project/peakplot gs://amlproject/Chip/$project/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.3rc1"
  },
  "latex_envs": {
   "LaTeX_envs_menu_present": true,
   "autoclose": false,
   "autocomplete": true,
   "bibliofile": "biblio.bib",
   "cite_by": "apalike",
   "current_citInitial": 1,
   "eqLabelWithNumbers": true,
   "eqNumInitial": 1,
   "hotkeys": {
    "equation": "Ctrl-E",
    "itemize": "Ctrl-I"
   },
   "labels_anchors": false,
   "latex_user_defs": false,
   "report_style_numbering": false,
   "user_envs_cfg": false
  },
  "toc": {
   "base_numbering": 1,
   "nav_menu": {},
   "number_sections": false,
   "sideBar": true,
   "skip_h1_title": false,
   "title_cell": "Table of Contents",
   "title_sidebar": "Contents",
   "toc_cell": false,
   "toc_position": {
    "height": "257.4px",
    "left": "127px",
    "top": "111.6px",
    "width": "165px"
   },
   "toc_section_display": true,
   "toc_window_display": true
  },
  "varInspector": {
   "cols": {
    "lenName": 16,
    "lenType": 16,
    "lenVar": 40
   },
   "kernels_config": {
    "python": {
     "delete_cmd_postfix": "",
     "delete_cmd_prefix": "del ",
     "library": "var_list.py",
     "varRefreshCmd": "print(var_dic_list())"
    },
    "r": {
     "delete_cmd_postfix": ") ",
     "delete_cmd_prefix": "rm(",
     "library": "var_list.r",
     "varRefreshCmd": "cat(var_dic_list()) "
    }
   },
   "types_to_exclude": [
    "module",
    "function",
    "builtin_function_or_method",
    "instance",
    "_Feature"
   ],
   "window_display": false
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
