{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 89,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "    <div class=\"bk-root\">\n",
       "        <a href=\"https://bokeh.org\" target=\"_blank\" class=\"bk-logo bk-logo-small bk-logo-notebook\"></a>\n",
       "        <span id=\"1002\">Loading BokehJS ...</span>\n",
       "    </div>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/javascript": [
       "\n",
       "(function(root) {\n",
       "  function now() {\n",
       "    return new Date();\n",
       "  }\n",
       "\n",
       "  var force = true;\n",
       "\n",
       "  if (typeof root._bokeh_onload_callbacks === \"undefined\" || force === true) {\n",
       "    root._bokeh_onload_callbacks = [];\n",
       "    root._bokeh_is_loading = undefined;\n",
       "  }\n",
       "\n",
       "  var JS_MIME_TYPE = 'application/javascript';\n",
       "  var HTML_MIME_TYPE = 'text/html';\n",
       "  var EXEC_MIME_TYPE = 'application/vnd.bokehjs_exec.v0+json';\n",
       "  var CLASS_NAME = 'output_bokeh rendered_html';\n",
       "\n",
       "  /**\n",
       "   * Render data to the DOM node\n",
       "   */\n",
       "  function render(props, node) {\n",
       "    var script = document.createElement(\"script\");\n",
       "    node.appendChild(script);\n",
       "  }\n",
       "\n",
       "  /**\n",
       "   * Handle when an output is cleared or removed\n",
       "   */\n",
       "  function handleClearOutput(event, handle) {\n",
       "    var cell = handle.cell;\n",
       "\n",
       "    var id = cell.output_area._bokeh_element_id;\n",
       "    var server_id = cell.output_area._bokeh_server_id;\n",
       "    // Clean up Bokeh references\n",
       "    if (id != null && id in Bokeh.index) {\n",
       "      Bokeh.index[id].model.document.clear();\n",
       "      delete Bokeh.index[id];\n",
       "    }\n",
       "\n",
       "    if (server_id !== undefined) {\n",
       "      // Clean up Bokeh references\n",
       "      var cmd = \"from bokeh.io.state import curstate; print(curstate().uuid_to_server['\" + server_id + \"'].get_sessions()[0].document.roots[0]._id)\";\n",
       "      cell.notebook.kernel.execute(cmd, {\n",
       "        iopub: {\n",
       "          output: function(msg) {\n",
       "            var id = msg.content.text.trim();\n",
       "            if (id in Bokeh.index) {\n",
       "              Bokeh.index[id].model.document.clear();\n",
       "              delete Bokeh.index[id];\n",
       "            }\n",
       "          }\n",
       "        }\n",
       "      });\n",
       "      // Destroy server and session\n",
       "      var cmd = \"import bokeh.io.notebook as ion; ion.destroy_server('\" + server_id + \"')\";\n",
       "      cell.notebook.kernel.execute(cmd);\n",
       "    }\n",
       "  }\n",
       "\n",
       "  /**\n",
       "   * Handle when a new output is added\n",
       "   */\n",
       "  function handleAddOutput(event, handle) {\n",
       "    var output_area = handle.output_area;\n",
       "    var output = handle.output;\n",
       "\n",
       "    // limit handleAddOutput to display_data with EXEC_MIME_TYPE content only\n",
       "    if ((output.output_type != \"display_data\") || (!output.data.hasOwnProperty(EXEC_MIME_TYPE))) {\n",
       "      return\n",
       "    }\n",
       "\n",
       "    var toinsert = output_area.element.find(\".\" + CLASS_NAME.split(' ')[0]);\n",
       "\n",
       "    if (output.metadata[EXEC_MIME_TYPE][\"id\"] !== undefined) {\n",
       "      toinsert[toinsert.length - 1].firstChild.textContent = output.data[JS_MIME_TYPE];\n",
       "      // store reference to embed id on output_area\n",
       "      output_area._bokeh_element_id = output.metadata[EXEC_MIME_TYPE][\"id\"];\n",
       "    }\n",
       "    if (output.metadata[EXEC_MIME_TYPE][\"server_id\"] !== undefined) {\n",
       "      var bk_div = document.createElement(\"div\");\n",
       "      bk_div.innerHTML = output.data[HTML_MIME_TYPE];\n",
       "      var script_attrs = bk_div.children[0].attributes;\n",
       "      for (var i = 0; i < script_attrs.length; i++) {\n",
       "        toinsert[toinsert.length - 1].firstChild.setAttribute(script_attrs[i].name, script_attrs[i].value);\n",
       "      }\n",
       "      // store reference to server id on output_area\n",
       "      output_area._bokeh_server_id = output.metadata[EXEC_MIME_TYPE][\"server_id\"];\n",
       "    }\n",
       "  }\n",
       "\n",
       "  function register_renderer(events, OutputArea) {\n",
       "\n",
       "    function append_mime(data, metadata, element) {\n",
       "      // create a DOM node to render to\n",
       "      var toinsert = this.create_output_subarea(\n",
       "        metadata,\n",
       "        CLASS_NAME,\n",
       "        EXEC_MIME_TYPE\n",
       "      );\n",
       "      this.keyboard_manager.register_events(toinsert);\n",
       "      // Render to node\n",
       "      var props = {data: data, metadata: metadata[EXEC_MIME_TYPE]};\n",
       "      render(props, toinsert[toinsert.length - 1]);\n",
       "      element.append(toinsert);\n",
       "      return toinsert\n",
       "    }\n",
       "\n",
       "    /* Handle when an output is cleared or removed */\n",
       "    events.on('clear_output.CodeCell', handleClearOutput);\n",
       "    events.on('delete.Cell', handleClearOutput);\n",
       "\n",
       "    /* Handle when a new output is added */\n",
       "    events.on('output_added.OutputArea', handleAddOutput);\n",
       "\n",
       "    /**\n",
       "     * Register the mime type and append_mime function with output_area\n",
       "     */\n",
       "    OutputArea.prototype.register_mime_type(EXEC_MIME_TYPE, append_mime, {\n",
       "      /* Is output safe? */\n",
       "      safe: true,\n",
       "      /* Index of renderer in `output_area.display_order` */\n",
       "      index: 0\n",
       "    });\n",
       "  }\n",
       "\n",
       "  // register the mime type if in Jupyter Notebook environment and previously unregistered\n",
       "  if (root.Jupyter !== undefined) {\n",
       "    var events = require('base/js/events');\n",
       "    var OutputArea = require('notebook/js/outputarea').OutputArea;\n",
       "\n",
       "    if (OutputArea.prototype.mime_types().indexOf(EXEC_MIME_TYPE) == -1) {\n",
       "      register_renderer(events, OutputArea);\n",
       "    }\n",
       "  }\n",
       "\n",
       "  \n",
       "  if (typeof (root._bokeh_timeout) === \"undefined\" || force === true) {\n",
       "    root._bokeh_timeout = Date.now() + 5000;\n",
       "    root._bokeh_failed_load = false;\n",
       "  }\n",
       "\n",
       "  var NB_LOAD_WARNING = {'data': {'text/html':\n",
       "     \"<div style='background-color: #fdd'>\\n\"+\n",
       "     \"<p>\\n\"+\n",
       "     \"BokehJS does not appear to have successfully loaded. If loading BokehJS from CDN, this \\n\"+\n",
       "     \"may be due to a slow or bad network connection. Possible fixes:\\n\"+\n",
       "     \"</p>\\n\"+\n",
       "     \"<ul>\\n\"+\n",
       "     \"<li>re-rerun `output_notebook()` to attempt to load from CDN again, or</li>\\n\"+\n",
       "     \"<li>use INLINE resources instead, as so:</li>\\n\"+\n",
       "     \"</ul>\\n\"+\n",
       "     \"<code>\\n\"+\n",
       "     \"from bokeh.resources import INLINE\\n\"+\n",
       "     \"output_notebook(resources=INLINE)\\n\"+\n",
       "     \"</code>\\n\"+\n",
       "     \"</div>\"}};\n",
       "\n",
       "  function display_loaded() {\n",
       "    var el = document.getElementById(\"1002\");\n",
       "    if (el != null) {\n",
       "      el.textContent = \"BokehJS is loading...\";\n",
       "    }\n",
       "    if (root.Bokeh !== undefined) {\n",
       "      if (el != null) {\n",
       "        el.textContent = \"BokehJS \" + root.Bokeh.version + \" successfully loaded.\";\n",
       "      }\n",
       "    } else if (Date.now() < root._bokeh_timeout) {\n",
       "      setTimeout(display_loaded, 100)\n",
       "    }\n",
       "  }\n",
       "\n",
       "\n",
       "  function run_callbacks() {\n",
       "    try {\n",
       "      root._bokeh_onload_callbacks.forEach(function(callback) {\n",
       "        if (callback != null)\n",
       "          callback();\n",
       "      });\n",
       "    } finally {\n",
       "      delete root._bokeh_onload_callbacks\n",
       "    }\n",
       "    console.debug(\"Bokeh: all callbacks have finished\");\n",
       "  }\n",
       "\n",
       "  function load_libs(css_urls, js_urls, callback) {\n",
       "    if (css_urls == null) css_urls = [];\n",
       "    if (js_urls == null) js_urls = [];\n",
       "\n",
       "    root._bokeh_onload_callbacks.push(callback);\n",
       "    if (root._bokeh_is_loading > 0) {\n",
       "      console.debug(\"Bokeh: BokehJS is being loaded, scheduling callback at\", now());\n",
       "      return null;\n",
       "    }\n",
       "    if (js_urls == null || js_urls.length === 0) {\n",
       "      run_callbacks();\n",
       "      return null;\n",
       "    }\n",
       "    console.debug(\"Bokeh: BokehJS not loaded, scheduling load and callback at\", now());\n",
       "    root._bokeh_is_loading = css_urls.length + js_urls.length;\n",
       "\n",
       "    function on_load() {\n",
       "      root._bokeh_is_loading--;\n",
       "      if (root._bokeh_is_loading === 0) {\n",
       "        console.debug(\"Bokeh: all BokehJS libraries/stylesheets loaded\");\n",
       "        run_callbacks()\n",
       "      }\n",
       "    }\n",
       "\n",
       "    function on_error() {\n",
       "      console.error(\"failed to load \" + url);\n",
       "    }\n",
       "\n",
       "    for (var i = 0; i < css_urls.length; i++) {\n",
       "      var url = css_urls[i];\n",
       "      const element = document.createElement(\"link\");\n",
       "      element.onload = on_load;\n",
       "      element.onerror = on_error;\n",
       "      element.rel = \"stylesheet\";\n",
       "      element.type = \"text/css\";\n",
       "      element.href = url;\n",
       "      console.debug(\"Bokeh: injecting link tag for BokehJS stylesheet: \", url);\n",
       "      document.body.appendChild(element);\n",
       "    }\n",
       "\n",
       "    for (var i = 0; i < js_urls.length; i++) {\n",
       "      var url = js_urls[i];\n",
       "      var element = document.createElement('script');\n",
       "      element.onload = on_load;\n",
       "      element.onerror = on_error;\n",
       "      element.async = false;\n",
       "      element.src = url;\n",
       "      console.debug(\"Bokeh: injecting script tag for BokehJS library: \", url);\n",
       "      document.head.appendChild(element);\n",
       "    }\n",
       "  };var element = document.getElementById(\"1002\");\n",
       "  if (element == null) {\n",
       "    console.error(\"Bokeh: ERROR: autoload.js configured with elementid '1002' but no matching script tag was found. \")\n",
       "    return false;\n",
       "  }\n",
       "\n",
       "  function inject_raw_css(css) {\n",
       "    const element = document.createElement(\"style\");\n",
       "    element.appendChild(document.createTextNode(css));\n",
       "    document.body.appendChild(element);\n",
       "  }\n",
       "\n",
       "  \n",
       "  var js_urls = [\"https://cdn.pydata.org/bokeh/release/bokeh-1.4.0.min.js\", \"https://cdn.pydata.org/bokeh/release/bokeh-widgets-1.4.0.min.js\", \"https://cdn.pydata.org/bokeh/release/bokeh-tables-1.4.0.min.js\", \"https://cdn.pydata.org/bokeh/release/bokeh-gl-1.4.0.min.js\"];\n",
       "  var css_urls = [];\n",
       "  \n",
       "\n",
       "  var inline_js = [\n",
       "    function(Bokeh) {\n",
       "      Bokeh.set_log_level(\"info\");\n",
       "    },\n",
       "    function(Bokeh) {\n",
       "    \n",
       "    \n",
       "    }\n",
       "  ];\n",
       "\n",
       "  function run_inline_js() {\n",
       "    \n",
       "    if (root.Bokeh !== undefined || force === true) {\n",
       "      \n",
       "    for (var i = 0; i < inline_js.length; i++) {\n",
       "      inline_js[i].call(root, root.Bokeh);\n",
       "    }\n",
       "    if (force === true) {\n",
       "        display_loaded();\n",
       "      }} else if (Date.now() < root._bokeh_timeout) {\n",
       "      setTimeout(run_inline_js, 100);\n",
       "    } else if (!root._bokeh_failed_load) {\n",
       "      console.log(\"Bokeh: BokehJS failed to load within specified timeout.\");\n",
       "      root._bokeh_failed_load = true;\n",
       "    } else if (force !== true) {\n",
       "      var cell = $(document.getElementById(\"1002\")).parents('.cell').data().cell;\n",
       "      cell.output_area.append_execute_result(NB_LOAD_WARNING)\n",
       "    }\n",
       "\n",
       "  }\n",
       "\n",
       "  if (root._bokeh_is_loading === 0) {\n",
       "    console.debug(\"Bokeh: BokehJS loaded, going straight to plotting\");\n",
       "    run_inline_js();\n",
       "  } else {\n",
       "    load_libs(css_urls, js_urls, function() {\n",
       "      console.debug(\"Bokeh: BokehJS plotting callback run at\", now());\n",
       "      run_inline_js();\n",
       "    });\n",
       "  }\n",
       "}(window));"
      ],
      "application/vnd.bokehjs_load.v0+json": "\n(function(root) {\n  function now() {\n    return new Date();\n  }\n\n  var force = true;\n\n  if (typeof root._bokeh_onload_callbacks === \"undefined\" || force === true) {\n    root._bokeh_onload_callbacks = [];\n    root._bokeh_is_loading = undefined;\n  }\n\n  \n\n  \n  if (typeof (root._bokeh_timeout) === \"undefined\" || force === true) {\n    root._bokeh_timeout = Date.now() + 5000;\n    root._bokeh_failed_load = false;\n  }\n\n  var NB_LOAD_WARNING = {'data': {'text/html':\n     \"<div style='background-color: #fdd'>\\n\"+\n     \"<p>\\n\"+\n     \"BokehJS does not appear to have successfully loaded. If loading BokehJS from CDN, this \\n\"+\n     \"may be due to a slow or bad network connection. Possible fixes:\\n\"+\n     \"</p>\\n\"+\n     \"<ul>\\n\"+\n     \"<li>re-rerun `output_notebook()` to attempt to load from CDN again, or</li>\\n\"+\n     \"<li>use INLINE resources instead, as so:</li>\\n\"+\n     \"</ul>\\n\"+\n     \"<code>\\n\"+\n     \"from bokeh.resources import INLINE\\n\"+\n     \"output_notebook(resources=INLINE)\\n\"+\n     \"</code>\\n\"+\n     \"</div>\"}};\n\n  function display_loaded() {\n    var el = document.getElementById(\"1002\");\n    if (el != null) {\n      el.textContent = \"BokehJS is loading...\";\n    }\n    if (root.Bokeh !== undefined) {\n      if (el != null) {\n        el.textContent = \"BokehJS \" + root.Bokeh.version + \" successfully loaded.\";\n      }\n    } else if (Date.now() < root._bokeh_timeout) {\n      setTimeout(display_loaded, 100)\n    }\n  }\n\n\n  function run_callbacks() {\n    try {\n      root._bokeh_onload_callbacks.forEach(function(callback) {\n        if (callback != null)\n          callback();\n      });\n    } finally {\n      delete root._bokeh_onload_callbacks\n    }\n    console.debug(\"Bokeh: all callbacks have finished\");\n  }\n\n  function load_libs(css_urls, js_urls, callback) {\n    if (css_urls == null) css_urls = [];\n    if (js_urls == null) js_urls = [];\n\n    root._bokeh_onload_callbacks.push(callback);\n    if (root._bokeh_is_loading > 0) {\n      console.debug(\"Bokeh: BokehJS is being loaded, scheduling callback at\", now());\n      return null;\n    }\n    if (js_urls == null || js_urls.length === 0) {\n      run_callbacks();\n      return null;\n    }\n    console.debug(\"Bokeh: BokehJS not loaded, scheduling load and callback at\", now());\n    root._bokeh_is_loading = css_urls.length + js_urls.length;\n\n    function on_load() {\n      root._bokeh_is_loading--;\n      if (root._bokeh_is_loading === 0) {\n        console.debug(\"Bokeh: all BokehJS libraries/stylesheets loaded\");\n        run_callbacks()\n      }\n    }\n\n    function on_error() {\n      console.error(\"failed to load \" + url);\n    }\n\n    for (var i = 0; i < css_urls.length; i++) {\n      var url = css_urls[i];\n      const element = document.createElement(\"link\");\n      element.onload = on_load;\n      element.onerror = on_error;\n      element.rel = \"stylesheet\";\n      element.type = \"text/css\";\n      element.href = url;\n      console.debug(\"Bokeh: injecting link tag for BokehJS stylesheet: \", url);\n      document.body.appendChild(element);\n    }\n\n    for (var i = 0; i < js_urls.length; i++) {\n      var url = js_urls[i];\n      var element = document.createElement('script');\n      element.onload = on_load;\n      element.onerror = on_error;\n      element.async = false;\n      element.src = url;\n      console.debug(\"Bokeh: injecting script tag for BokehJS library: \", url);\n      document.head.appendChild(element);\n    }\n  };var element = document.getElementById(\"1002\");\n  if (element == null) {\n    console.error(\"Bokeh: ERROR: autoload.js configured with elementid '1002' but no matching script tag was found. \")\n    return false;\n  }\n\n  function inject_raw_css(css) {\n    const element = document.createElement(\"style\");\n    element.appendChild(document.createTextNode(css));\n    document.body.appendChild(element);\n  }\n\n  \n  var js_urls = [\"https://cdn.pydata.org/bokeh/release/bokeh-1.4.0.min.js\", \"https://cdn.pydata.org/bokeh/release/bokeh-widgets-1.4.0.min.js\", \"https://cdn.pydata.org/bokeh/release/bokeh-tables-1.4.0.min.js\", \"https://cdn.pydata.org/bokeh/release/bokeh-gl-1.4.0.min.js\"];\n  var css_urls = [];\n  \n\n  var inline_js = [\n    function(Bokeh) {\n      Bokeh.set_log_level(\"info\");\n    },\n    function(Bokeh) {\n    \n    \n    }\n  ];\n\n  function run_inline_js() {\n    \n    if (root.Bokeh !== undefined || force === true) {\n      \n    for (var i = 0; i < inline_js.length; i++) {\n      inline_js[i].call(root, root.Bokeh);\n    }\n    if (force === true) {\n        display_loaded();\n      }} else if (Date.now() < root._bokeh_timeout) {\n      setTimeout(run_inline_js, 100);\n    } else if (!root._bokeh_failed_load) {\n      console.log(\"Bokeh: BokehJS failed to load within specified timeout.\");\n      root._bokeh_failed_load = true;\n    } else if (force !== true) {\n      var cell = $(document.getElementById(\"1002\")).parents('.cell').data().cell;\n      cell.output_area.append_execute_result(NB_LOAD_WARNING)\n    }\n\n  }\n\n  if (root._bokeh_is_loading === 0) {\n    console.debug(\"Bokeh: BokehJS loaded, going straight to plotting\");\n    run_inline_js();\n  } else {\n    load_libs(css_urls, js_urls, function() {\n      console.debug(\"Bokeh: BokehJS plotting callback run at\", now());\n      run_inline_js();\n    });\n  }\n}(window));"
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The autoreload extension is already loaded. To reload it, use:\n",
      "  %reload_ext autoreload\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "from IPython.display import IFrame\n",
    "import pandas as pd\n",
    "import sys\n",
    "import igv\n",
    "import numpy as np\n",
    "import pyBigWig\n",
    "import sys\n",
    "sys.path.insert(0, '../../')\n",
    "from JKBio import TerraFunction as terra\n",
    "from ccle_processing.src.CCLE_postp_function import *\n",
    "from JKBio import Helper as h\n",
    "from JKBio.epigenetics import ChIP_helper as chip\n",
    "import dalmatian as dm\n",
    "import itertools\n",
    "from sklearn.cluster import AgglomerativeClustering\n",
    "from sklearn.mixture import GaussianMixture\n",
    "from bokeh.plotting import *\n",
    "from sklearn.manifold import MDS, TSNE\n",
    "from sklearn.decomposition import PCA\n",
    "from sklearn.preprocessing import scale\n",
    "output_notebook()\n",
    "%load_ext autoreload\n",
    "%autoreload 2"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# ChIP"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!gsutil ls gs://amlproject/Chip_AML/Chip/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "a = !gsutil ls gs://amlproject/Chip_AML/Chip/IRF2BP2/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# install bwa\n",
    "! mkdir ref\n",
    "! curl ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/001/215/GCF_000001215.4_Release_6_plus_ISO1_MT/GCF_000001215.4_Release_6_plus_ISO1_MT_genomic.fna.gz -o ref/reference_droso.fna.gz\n",
    "! gunzip ref/reference_droso.fna.gz\n",
    "! bwa index -a bwtsw ref/reference_droso.fna"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## V1"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "rename1 = {\n",
    " 'gs://transfer-amlproject/20191211_10_MP7613_S8_R1_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp910_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_10_MP7613_S8_R2_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp910_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_11_MP7613_S9_R1_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp911_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_11_MP7613_S9_R2_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp911_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_12_MP7613_S10_R1_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp912_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_12_MP7613_S10_R2_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp912_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_13_MP7613_S11_R1_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp913_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_13_MP7613_S11_R2_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp913_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_14_MP7613_S12_R1_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp914_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_14_MP7613_S12_R2_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp914_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_15_MP7613_S13_R1_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp915_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_15_MP7613_S13_R2_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp915_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_16_MP7613_S14_R1_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp916_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_16_MP7613_S14_R2_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp916_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_1_MP7613_S1_R1_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp901_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_1_MP7613_S1_R2_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp901_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_2_MP7613_S2_R1_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp902_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_2_MP7613_S2_R2_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp902_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_3_MP7613_S3_R1_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp903_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_3_MP7613_S3_R2_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp903_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_4_MP7613_S4_R1_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp904_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_4_MP7613_S4_R2_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp904_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_5R_MP7613_S15_R1_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp905_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_5R_MP7613_S15_R2_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp905_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_6R_MP7613_S16_R1_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp906_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_6R_MP7613_S16_R2_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp906_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_7_MP7613_S5_R1_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp907_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_7_MP7613_S5_R2_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp907_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_8_MP7613_S6_R1_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp908_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_8_MP7613_S6_R2_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp908_2.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_9_MP7613_S7_R1_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp909_1.fastq.gz\",\n",
    " 'gs://transfer-amlproject/20191211_9_MP7613_S7_R2_001.fastq.gz':\"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/mp909_2.fastq.gz\"}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for k, val in rename1.items():\n",
    "    os.system('gsutil mv '+k+' '+val)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mkdir bams && gsutil -m cp \"gs://amlproject/Chip_AML/IRF2BP2/IRF2BP2/*\" bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "inputfastq=\"gs://amlproject/Chip_AML/Chip/mp99.fastq.gz\"\n",
    "! gsutil cp $inputfastq bams\n",
    "a = ! ls bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from gsheets import Sheets\n",
    "sheets = Sheets.from_files('~/.client_secret.json', '~/.storage.json')\n",
    "url=\"https://docs.google.com/spreadsheets/d/1yFLjYB1McU530JnLgL0QIMAKIkVl3kl0_LCHje2gk8U\"\n",
    "gsheet = sheets.get(url).sheets[2].to_frame()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "gsheet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = {\n",
    "\"fastq1\": [],\n",
    "\"fastq2\": [],\n",
    "\"antibody\": [],\n",
    "\"group\": [],\n",
    "\"replicate\": [],\n",
    "\"control\": []\n",
    "}\n",
    "for val in helper.grouped(a[:-1],2):\n",
    "    row = gsheet[gsheet.id==val[0].split('/')[-1].split('_')[0]]\n",
    "    df['group'].append((row.id.values[0] + '_'+ row.name_replicate.values[0] +\"_\"+row.protein.values[0]))\n",
    "    df['replicate'].append(1)\n",
    "    df['fastq1'].append(val[0])\n",
    "    #df['fastq2'].append(val[1])\n",
    "    df['antibody'].append(row.protein.values[0])\n",
    "    df['control'].append(\"INPUT\")\n",
    "df['group'].append('INPUT')\n",
    "df['replicate'].append(1)\n",
    "df['fastq1'].append(a[-1])\n",
    "#df['fastq2'].append(val[1])\n",
    "df['antibody'].append(\"INPUT\")\n",
    "df['control'].append(\"INPUT\")\n",
    "df = pd.DataFrame(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df[df.columns[[3,4,0,1,2,5]]].to_csv('design.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#process chips\n",
    "! sudo ./nextflow run nf-core/chipseq --single_end --seq_center 'DFCI' --email 'jkobject@gmail.com' --narrow_peak --input design.csv --genome GRCh38 --skip_preseq --max_cpus 24 -profile docker -w work"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#!gsutil cp results gs://amlproject/Chip_AML/Chip/IRF2BP2/ && mv results results1 && sudo rm -r work"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# get scaling values\n",
    "mappedreads, umappedreads_norm, mapped = chip.getSpikeInControlScales(refgenome=\"ref/reference_droso.fna\",FastQfolder='bams',pairedEnd=True, cores=8)\n",
    "mappedreads, umappedreads_norm, mapped"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "computing scales from the excel sheet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [[562285,1496707],\n",
    "[1686168,7198567],\n",
    "[3642441,2612624],\n",
    "[3992589,3474812],\n",
    "[3347901,3829477],\n",
    "[6181136,989703],\n",
    "[7523840,4173047],\n",
    "[922482,1195857]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 135,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [[1.0, 0.3756814126], #IRF2BP2\n",
    "[1.0, 0.2342366196], #MED1\n",
    "[0.5546432407, 1.0], #MYB\n",
    "[0.1601166841, 1.0], #MYC\n",
    "[0.870315477, 1.0], #POL2S2\n",
    "[1.0, 0.8742449687], #POL2S5\n",
    "[0.7172728398, 1.0], #POL2TOT\n",
    "[1.0, 0.7713982525]] #SPI1"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### on scalled data\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/mp784-MV411_IRF2BP2_DMSO_MYC-MYC-r1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/mp785-MV411_IRF2BP2_DMSO_MYB-MYB-r1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/MV411_IRF2BP2_DMSO_R1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/mp780-MV411_IRF2BP2_DMSO_MED1-MED1-r1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/MV411_IRF2BP2_VHL_R1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/mp779-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/mp782-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/mp783-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/mp781-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/mp792-MV411_IRF2BP2_VHL_MYC-MYC-r1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/mp791-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/mp793-MV411_IRF2BP2_VHL_MYB-MYB-r1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/mp794-MV411_IRF2BP2_VHL_SPI1-SPI1-r1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/mp789-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/mp786-MV411_IRF2BP2_DMSO_SPI1-SPI1-r1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/mp790-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/mp788-MV411_IRF2BP2_VHL_MED1-MED1-r1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/mp787-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r1.mLb.clN.sorted.bam...\n",
      "- [19/19 files][ 26.8 GiB/ 26.8 GiB] 100% Done  52.6 MiB/s ETA 00:00:00         \n",
      "Operation completed over 19 objects/26.8 GiB.                                    \n"
     ]
    }
   ],
   "source": [
    "!gsutil -m cp gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/*.bam ../../data/diffBinding_rep1/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 136,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['MV411_IRF2BP2_DMSO_FLAG_IRF2BP2_R1',\n",
       " 'MV411_IRF2BP2_DMSO_MED1_R1',\n",
       " 'MV411_IRF2BP2_DMSO_MYB_R1',\n",
       " 'MV411_IRF2BP2_DMSO_MYC_R1',\n",
       " 'MV411_IRF2BP2_DMSO_POLII_S2_R1',\n",
       " 'MV411_IRF2BP2_DMSO_POLII_S5_R1',\n",
       " 'MV411_IRF2BP2_DMSO_POLII_total_R1',\n",
       " 'MV411_IRF2BP2_DMSO_SPI1_R1',\n",
       " 'MV411_IRF2BP2_VHL_FLAG_IRF2BP2_R1',\n",
       " 'MV411_IRF2BP2_VHL_MED1_R1',\n",
       " 'MV411_IRF2BP2_VHL_MYB_R1',\n",
       " 'MV411_IRF2BP2_VHL_MYC_R1',\n",
       " 'MV411_IRF2BP2_VHL_POLII_S2_R1',\n",
       " 'MV411_IRF2BP2_VHL_POLII_S5_R1',\n",
       " 'MV411_IRF2BP2_VHL_POLII_total_R1',\n",
       " 'MV411_IRF2BP2_VHL_SPI1_R1']"
      ]
     },
     "execution_count": 136,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bams = !ls ../../data/diffBinding_rep1/diffData/*treat_pileup.bdg\n",
    "bams = ['_'.join(i.split('/')[-1].split('_')[:-2]) for i in bams]\n",
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 138,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "MV411_IRF2BP2_DMSO_FLAG_IRF2BP2_R1 MV411_IRF2BP2_VHL_FLAG_IRF2BP2_R1\n",
      "doing differential peak binding\n",
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_DMSO_FLAG_IRF2BP2_R1_treat_pileup.bdg --c1 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_DMSO_FLAG_IRF2BP2_R1_control_lambda.bdg --t2 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_VHL_FLAG_IRF2BP2_R1_treat_pileup.bdg --c2 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_VHL_FLAG_IRF2BP2_R1_control_lambda.bdg --d1 0.3756814126 --d2 1.0 -g 60 -l 206 --o-prefix MV411_IRF2BP2_DMSO_FLAG_IRF2BP2_R1_vs_MV411_IRF2BP2_VHL_FLAG_IRF2BP2_R1 --outdir ../../data/diffBinding_rep1/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Wed, 29 Apr 2020 15:50:06: Read and build treatment 1 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 15:50:58: Read and build control 1 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 15:52:33: Read and build treatment 2 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 15:53:23: Read and build control 2 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 15:58:49: Write peaks... \\nINFO  @ Wed, 29 Apr 2020 15:58:49: Done \\n')\n",
      "MV411_IRF2BP2_DMSO_MED1_R1 MV411_IRF2BP2_VHL_MED1_R1\n",
      "doing differential peak binding\n",
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_DMSO_MED1_R1_treat_pileup.bdg --c1 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_DMSO_MED1_R1_control_lambda.bdg --t2 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_VHL_MED1_R1_treat_pileup.bdg --c2 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_VHL_MED1_R1_control_lambda.bdg --d1 0.2342366196 --d2 1.0 -g 60 -l 213 --o-prefix MV411_IRF2BP2_DMSO_MED1_R1_vs_MV411_IRF2BP2_VHL_MED1_R1 --outdir ../../data/diffBinding_rep1/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Wed, 29 Apr 2020 15:58:50: Read and build treatment 1 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 15:59:43: Read and build control 1 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 16:01:16: Read and build treatment 2 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 16:01:44: Read and build control 2 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 16:06:27: Write peaks... \\nINFO  @ Wed, 29 Apr 2020 16:06:27: Done \\n')\n",
      "MV411_IRF2BP2_DMSO_MYB_R1 MV411_IRF2BP2_VHL_MYB_R1\n",
      "doing differential peak binding\n",
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_DMSO_MYB_R1_treat_pileup.bdg --c1 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_DMSO_MYB_R1_control_lambda.bdg --t2 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_VHL_MYB_R1_treat_pileup.bdg --c2 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_VHL_MYB_R1_control_lambda.bdg --d1 1.0 --d2 0.5546432407 -g 60 -l 47 --o-prefix MV411_IRF2BP2_DMSO_MYB_R1_vs_MV411_IRF2BP2_VHL_MYB_R1 --outdir ../../data/diffBinding_rep1/diffPeaks/', returncode=0, stdout=b'', stderr=b'CRITICAL @ Wed, 29 Apr 2020 16:06:28: MAXGAP should be smaller than MINLEN! Your input is MAXGAP = 60 and MINLEN = 47 \\nINFO  @ Wed, 29 Apr 2020 16:06:28: Read and build treatment 1 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 16:07:11: Read and build control 1 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 16:08:41: Read and build treatment 2 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 16:09:23: Read and build control 2 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 16:14:34: Write peaks... \\nINFO  @ Wed, 29 Apr 2020 16:14:34: Done \\n')\n",
      "MV411_IRF2BP2_DMSO_MYC_R1 MV411_IRF2BP2_VHL_MYC_R1\n",
      "doing differential peak binding\n",
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_DMSO_MYC_R1_treat_pileup.bdg --c1 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_DMSO_MYC_R1_control_lambda.bdg --t2 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_VHL_MYC_R1_treat_pileup.bdg --c2 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_VHL_MYC_R1_control_lambda.bdg --d1 1.0 --d2 0.1601166841 -g 60 -l 272 --o-prefix MV411_IRF2BP2_DMSO_MYC_R1_vs_MV411_IRF2BP2_VHL_MYC_R1 --outdir ../../data/diffBinding_rep1/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Wed, 29 Apr 2020 16:14:35: Read and build treatment 1 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 16:15:11: Read and build control 1 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 16:16:42: Read and build treatment 2 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 16:17:21: Read and build control 2 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 16:22:23: Write peaks... \\nINFO  @ Wed, 29 Apr 2020 16:22:23: Done \\n')\n",
      "MV411_IRF2BP2_DMSO_POLII_S2_R1 MV411_IRF2BP2_VHL_POLII_S2_R1\n",
      "doing differential peak binding\n",
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_DMSO_POLII_S2_R1_treat_pileup.bdg --c1 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_DMSO_POLII_S2_R1_control_lambda.bdg --t2 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_VHL_POLII_S2_R1_treat_pileup.bdg --c2 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_VHL_POLII_S2_R1_control_lambda.bdg --d1 1.0 --d2 0.870315477 -g 60 -l 229 --o-prefix MV411_IRF2BP2_DMSO_POLII_S2_R1_vs_MV411_IRF2BP2_VHL_POLII_S2_R1 --outdir ../../data/diffBinding_rep1/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Wed, 29 Apr 2020 16:22:24: Read and build treatment 1 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 16:23:05: Read and build control 1 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 16:24:37: Read and build treatment 2 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 16:25:11: Read and build control 2 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 16:29:54: Write peaks... \\nINFO  @ Wed, 29 Apr 2020 16:29:54: Done \\n')\n",
      "MV411_IRF2BP2_DMSO_POLII_S5_R1 MV411_IRF2BP2_VHL_POLII_S5_R1\n",
      "doing differential peak binding\n",
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_DMSO_POLII_S5_R1_treat_pileup.bdg --c1 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_DMSO_POLII_S5_R1_control_lambda.bdg --t2 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_VHL_POLII_S5_R1_treat_pileup.bdg --c2 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_VHL_POLII_S5_R1_control_lambda.bdg --d1 0.8742449687 --d2 1.0 -g 60 -l 235 --o-prefix MV411_IRF2BP2_DMSO_POLII_S5_R1_vs_MV411_IRF2BP2_VHL_POLII_S5_R1 --outdir ../../data/diffBinding_rep1/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Wed, 29 Apr 2020 16:29:54: Read and build treatment 1 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 16:30:28: Read and build control 1 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 16:32:00: Read and build treatment 2 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 16:32:37: Read and build control 2 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 16:37:24: Write peaks... \\nINFO  @ Wed, 29 Apr 2020 16:37:24: Done \\n')\n",
      "MV411_IRF2BP2_DMSO_POLII_total_R1 MV411_IRF2BP2_VHL_POLII_total_R1\n",
      "doing differential peak binding\n",
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_DMSO_POLII_total_R1_treat_pileup.bdg --c1 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_DMSO_POLII_total_R1_control_lambda.bdg --t2 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_VHL_POLII_total_R1_treat_pileup.bdg --c2 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_VHL_POLII_total_R1_control_lambda.bdg --d1 1.0 --d2 0.7172728398 -g 60 -l 190 --o-prefix MV411_IRF2BP2_DMSO_POLII_total_R1_vs_MV411_IRF2BP2_VHL_POLII_total_R1 --outdir ../../data/diffBinding_rep1/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Wed, 29 Apr 2020 16:37:24: Read and build treatment 1 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 16:38:29: Read and build control 1 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 16:39:54: Read and build treatment 2 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 16:40:38: Read and build control 2 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 16:46:11: Write peaks... \\nINFO  @ Wed, 29 Apr 2020 16:46:11: Done \\n')\n",
      "MV411_IRF2BP2_DMSO_SPI1_R1 MV411_IRF2BP2_VHL_SPI1_R1\n",
      "doing differential peak binding\n",
      "CompletedProcess(args='macs2 bdgdiff --t1 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_DMSO_SPI1_R1_treat_pileup.bdg --c1 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_DMSO_SPI1_R1_control_lambda.bdg --t2 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_VHL_SPI1_R1_treat_pileup.bdg --c2 ../../data/diffBinding_rep1/diffData/MV411_IRF2BP2_VHL_SPI1_R1_control_lambda.bdg --d1 0.7713982525 --d2 1.0 -g 60 -l 196 --o-prefix MV411_IRF2BP2_DMSO_SPI1_R1_vs_MV411_IRF2BP2_VHL_SPI1_R1 --outdir ../../data/diffBinding_rep1/diffPeaks/', returncode=0, stdout=b'', stderr=b'INFO  @ Wed, 29 Apr 2020 16:46:12: Read and build treatment 1 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 16:47:10: Read and build control 1 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 16:48:36: Read and build treatment 2 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 16:49:20: Read and build control 2 bedGraph... \\nINFO  @ Wed, 29 Apr 2020 16:55:32: Write peaks... \\nINFO  @ Wed, 29 Apr 2020 16:55:32: Done \\n')\n"
     ]
    }
   ],
   "source": [
    "# diffPeak on scalled data\n",
    "sizes = [220, 191, 211, 204, 285, 222, 228, 194]\n",
    "for i in range(int(len(bams)/2)):\n",
    "    name1 = bams[i]\n",
    "    name2 = bams[i+8]\n",
    "    print(name1,name2)\n",
    "    print(chip.diffPeak(name1,name2, directory= \"../../data/diffBinding_rep1/diffData/\", res_directory='../../data/diffBinding_rep1/diffPeaks/', scaling1=scales[i][1], scaling2=scales[i][0], size=size[i]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "doing diff from ../../data/diffBinding_rep1/mp779-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r1.mLb.clN.sorted.bam and ../../data/diffBinding_rep1/mp787-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "220\n",
      "562285 1496707\n",
      "doing differential peak binding\n",
      "doing diff from ../../data/diffBinding_rep1/mp780-MV411_IRF2BP2_DMSO_MED1-MED1-r1.mLb.clN.sorted.bam and ../../data/diffBinding_rep1/mp788-MV411_IRF2BP2_VHL_MED1-MED1-r1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "191\n",
      "1686168 7198567\n",
      "doing differential peak binding\n",
      "doing diff from ../../data/diffBinding_rep1/mp781-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r1.mLb.clN.sorted.bam and ../../data/diffBinding_rep1/mp789-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "228\n",
      "3642441 2612624\n",
      "doing differential peak binding\n",
      "doing diff from ../../data/diffBinding_rep1/mp782-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r1.mLb.clN.sorted.bam and ../../data/diffBinding_rep1/mp790-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "285\n",
      "3992589 3474812\n",
      "doing differential peak binding\n",
      "doing diff from ../../data/diffBinding_rep1/mp783-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r1.mLb.clN.sorted.bam and ../../data/diffBinding_rep1/mp791-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "222\n",
      "3347901 3829477\n",
      "doing differential peak binding\n",
      "doing diff from ../../data/diffBinding_rep1/mp784-MV411_IRF2BP2_DMSO_MYC-MYC-r1.mLb.clN.sorted.bam and ../../data/diffBinding_rep1/mp792-MV411_IRF2BP2_VHL_MYC-MYC-r1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "204\n",
      "6181136 989703\n",
      "doing differential peak binding\n",
      "doing diff from ../../data/diffBinding_rep1/mp785-MV411_IRF2BP2_DMSO_MYB-MYB-r1.mLb.clN.sorted.bam and ../../data/diffBinding_rep1/mp793-MV411_IRF2BP2_VHL_MYB-MYB-r1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "211\n",
      "7523840 4173047\n",
      "doing differential peak binding\n",
      "doing diff from ../../data/diffBinding_rep1/mp786-MV411_IRF2BP2_DMSO_SPI1-SPI1-r1.mLb.clN.sorted.bam and ../../data/diffBinding_rep1/mp794-MV411_IRF2BP2_VHL_SPI1-SPI1-r1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "194\n",
      "922482 1195857\n",
      "doing differential peak binding\n"
     ]
    }
   ],
   "source": [
    "# diffPeak on scalled data\n",
    "for i in range(int(len(bams[1:])/2)-1):\n",
    "    name1 = bams[1+i]\n",
    "    name2 = bams[9+i]\n",
    "    chip.fullDiffPeak(name1,name2, bams[0], scaling = scales[i],compute_size=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [1.0,\n",
    "1.0,\n",
    "0.5546432407,\n",
    "0.1601166841,\n",
    "0.870315477,\n",
    "1.0,\n",
    "0.7172728398,\n",
    "1.0,\n",
    "0.3756814126,\n",
    "0.2342366196,\n",
    "1.0,\n",
    "1.0,\n",
    "1.0,\n",
    "0.8742449687,\n",
    "1.0,\n",
    "0.7713982525]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "chip.bigWigFrom(bams[1:],genome='GRCh38',scaling=scales)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 139,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_FLAG_IRF2BP2_R1.bw',\n",
       " '../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_MED1_R1.bw',\n",
       " '../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_MYB_R1.bw',\n",
       " '../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_MYC_R1.bw',\n",
       " '../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_POLII_S2_R1.bw',\n",
       " '../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_POLII_S5_R1.bw',\n",
       " '../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_POLII_total_R1.bw',\n",
       " '../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_SPI1_R1.bw',\n",
       " '../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_FLAG_IRF2BP2_R1.bw',\n",
       " '../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_MED1_R1.bw',\n",
       " '../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_MYB_R1.bw',\n",
       " '../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_MYC_R1.bw',\n",
       " '../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_POLII_S2_R1.bw',\n",
       " '../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_POLII_S5_R1.bw',\n",
       " '../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_POLII_total_R1.bw',\n",
       " '../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_SPI1_R1.bw']"
      ]
     },
     "execution_count": 139,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bw = ! ls ../../data/recalib_bigwig_rep1/*.bw\n",
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 140,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/diffBinding_rep1/diffPeaks/MV411_IRF2BP2_DMSO_FLAG_IRF2BP2_R1_vs_MV411_IRF2BP2_VHL_FLAG_IRF2BP2_R1_c3.0_common.bed',\n",
       " '../../data/diffBinding_rep1/diffPeaks/MV411_IRF2BP2_DMSO_MED1_R1_vs_MV411_IRF2BP2_VHL_MED1_R1_c3.0_common.bed',\n",
       " '../../data/diffBinding_rep1/diffPeaks/MV411_IRF2BP2_DMSO_MYB_R1_vs_MV411_IRF2BP2_VHL_MYB_R1_c3.0_common.bed',\n",
       " '../../data/diffBinding_rep1/diffPeaks/MV411_IRF2BP2_DMSO_MYC_R1_vs_MV411_IRF2BP2_VHL_MYC_R1_c3.0_common.bed',\n",
       " '../../data/diffBinding_rep1/diffPeaks/MV411_IRF2BP2_DMSO_POLII_S2_R1_vs_MV411_IRF2BP2_VHL_POLII_S2_R1_c3.0_common.bed',\n",
       " '../../data/diffBinding_rep1/diffPeaks/MV411_IRF2BP2_DMSO_POLII_S5_R1_vs_MV411_IRF2BP2_VHL_POLII_S5_R1_c3.0_common.bed',\n",
       " '../../data/diffBinding_rep1/diffPeaks/MV411_IRF2BP2_DMSO_POLII_total_R1_vs_MV411_IRF2BP2_VHL_POLII_total_R1_c3.0_common.bed',\n",
       " '../../data/diffBinding_rep1/diffPeaks/MV411_IRF2BP2_DMSO_SPI1_R1_vs_MV411_IRF2BP2_VHL_SPI1_R1_c3.0_common.bed']"
      ]
     },
     "execution_count": 140,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "cond1peak = ! ls ../../data/diffBinding_rep1/diffPeaks/*cond1.bed\n",
    "cond2peak = ! ls ../../data/diffBinding_rep1/diffPeaks/*cond2.bed\n",
    "commonpeak = ! ls ../../data/diffBinding_rep1/diffPeaks/*common.bed\n",
    "commonpeak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 141,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"FLAG_IRF2BP2\",\"MED1\",\"MYB\",\"MYC\",\"POLII_S2\",\"POLII_S5\",\"POLII_total\",\"SPI1\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 142,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "found 1 header line... removing\n",
      "removed 8 lines\n",
      "found 1 header line... removing\n",
      "found 0 header line... removing\n",
      "removed 1 lines\n",
      "../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_FLAG_IRF2BP2_R1.bw ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_FLAG_IRF2BP2_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_FLAG_IRF2BP2_R1.bw ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_FLAG_IRF2BP2_R1.bw  --referencePoint center --regionsFileName ../../data/diffBinding_rep1/diffPeaks/MV411_IRF2BP2_DMSO_FLAG_IRF2BP2_R1_vs_MV411_IRF2BP2_VHL_FLAG_IRF2BP2_R1_c3.0_cond1.bed ../../data/diffBinding_rep1/diffPeaks/MV411_IRF2BP2_DMSO_FLAG_IRF2BP2_R1_vs_MV411_IRF2BP2_VHL_FLAG_IRF2BP2_R1_c3.0_common.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/FLAG_IRF2BP2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_rep1/diffPeaks/FLAG_IRF2BP2_mat.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/FLAG_IRF2BP2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common  --samplesLabel DMSO VHL  --plotTitle FLAG_IRF2BP2', returncode=0, stdout=b'', stderr=b'')\n",
      "found 1 header line... removing\n",
      "removed 8 lines\n",
      "found 1 header line... removing\n",
      "found 0 header line... removing\n",
      "removed 1 lines\n",
      "../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_MED1_R1.bw ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_MED1_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_MED1_R1.bw ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_MED1_R1.bw  --referencePoint center --regionsFileName ../../data/diffBinding_rep1/diffPeaks/MV411_IRF2BP2_DMSO_MED1_R1_vs_MV411_IRF2BP2_VHL_MED1_R1_c3.0_cond1.bed ../../data/diffBinding_rep1/diffPeaks/MV411_IRF2BP2_DMSO_MED1_R1_vs_MV411_IRF2BP2_VHL_MED1_R1_c3.0_common.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/MED1_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_rep1/diffPeaks/MED1_mat.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/MED1_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common  --samplesLabel DMSO VHL  --plotTitle MED1', returncode=0, stdout=b'', stderr=b'')\n",
      "found 1 header line... removing\n",
      "found 1 header line... removing\n",
      "removed 54 lines\n",
      "found 1 header line... removing\n",
      "removed 67 lines\n",
      "../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_MYB_R1.bw ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_MYB_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_MYB_R1.bw ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_MYB_R1.bw  --referencePoint center --regionsFileName ../../data/diffBinding_rep1/diffPeaks/MV411_IRF2BP2_DMSO_MYB_R1_vs_MV411_IRF2BP2_VHL_MYB_R1_c3.0_cond1.bed ../../data/diffBinding_rep1/diffPeaks/MV411_IRF2BP2_DMSO_MYB_R1_vs_MV411_IRF2BP2_VHL_MYB_R1_c3.0_common.bed ../../data/diffBinding_rep1/diffPeaks/MV411_IRF2BP2_DMSO_MYB_R1_vs_MV411_IRF2BP2_VHL_MYB_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/MYB_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_rep1/diffPeaks/MYB_mat.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/MYB_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle MYB', returncode=0, stdout=b'', stderr=b\"One of the groups defined in the bed file is too small.\\nGroups that are too small can't be plotted. \\nWARNING: Group 'MV411_IRF2BP2_DMSO_MYB_R1_vs_MV411_IRF2BP2_VHL_MYB_R1_c3.0_cond1.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n",
      "found 0 header line... removing\n",
      "removed 1 lines\n",
      "found 1 header line... removing\n",
      "found 1 header line... removing\n",
      "removed 11 lines\n",
      "../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_MYC_R1.bw ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_MYC_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_MYC_R1.bw ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_MYC_R1.bw  --referencePoint center --regionsFileName ../../data/diffBinding_rep1/diffPeaks/MV411_IRF2BP2_DMSO_MYC_R1_vs_MV411_IRF2BP2_VHL_MYC_R1_c3.0_common.bed ../../data/diffBinding_rep1/diffPeaks/MV411_IRF2BP2_DMSO_MYC_R1_vs_MV411_IRF2BP2_VHL_MYC_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/MYC_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_rep1/diffPeaks/MYC_mat.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/MYC_mat.pdf --refPointLabel center --regionsLabel common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle MYC', returncode=0, stdout=b'', stderr=b\"One of the groups defined in the bed file is too small.\\nGroups that are too small can't be plotted. \\nWARNING: Group 'MV411_IRF2BP2_DMSO_MYC_R1_vs_MV411_IRF2BP2_VHL_MYC_R1_c3.0_common.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n",
      "found 1 header line... removing\n",
      "found 1 header line... removing\n",
      "removed 7 lines\n",
      "found 1 header line... removing\n",
      "../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_POLII_S2_R1.bw ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_POLII_S2_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_POLII_S2_R1.bw ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_POLII_S2_R1.bw  --referencePoint center --regionsFileName ../../data/diffBinding_rep1/diffPeaks/MV411_IRF2BP2_DMSO_POLII_S2_R1_vs_MV411_IRF2BP2_VHL_POLII_S2_R1_c3.0_cond1.bed ../../data/diffBinding_rep1/diffPeaks/MV411_IRF2BP2_DMSO_POLII_S2_R1_vs_MV411_IRF2BP2_VHL_POLII_S2_R1_c3.0_common.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_S2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_rep1/diffPeaks/POLII_S2_mat.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_S2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common  --samplesLabel DMSO VHL  --plotTitle POLII_S2', returncode=0, stdout=b'', stderr=b'')\n",
      "found 1 header line... removing\n",
      "found 1 header line... removing\n",
      "removed 6 lines\n",
      "found 1 header line... removing\n",
      "../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_POLII_S5_R1.bw ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_POLII_S5_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_POLII_S5_R1.bw ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_POLII_S5_R1.bw  --referencePoint center --regionsFileName ../../data/diffBinding_rep1/diffPeaks/MV411_IRF2BP2_DMSO_POLII_S5_R1_vs_MV411_IRF2BP2_VHL_POLII_S5_R1_c3.0_cond1.bed ../../data/diffBinding_rep1/diffPeaks/MV411_IRF2BP2_DMSO_POLII_S5_R1_vs_MV411_IRF2BP2_VHL_POLII_S5_R1_c3.0_common.bed ../../data/diffBinding_rep1/diffPeaks/MV411_IRF2BP2_DMSO_POLII_S5_R1_vs_MV411_IRF2BP2_VHL_POLII_S5_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_S5_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_rep1/diffPeaks/POLII_S5_mat.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_S5_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle POLII_S5', returncode=0, stdout=b'', stderr=b\"One of the groups defined in the bed file is too small.\\nGroups that are too small can't be plotted. \\nWARNING: Group 'MV411_IRF2BP2_DMSO_POLII_S5_R1_vs_MV411_IRF2BP2_VHL_POLII_S5_R1_c3.0_cond1.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n",
      "found 1 header line... removing\n",
      "found 1 header line... removing\n",
      "removed 7 lines\n",
      "found 1 header line... removing\n",
      "../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_POLII_total_R1.bw ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_POLII_total_R1.bw\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_POLII_total_R1.bw ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_POLII_total_R1.bw  --referencePoint center --regionsFileName ../../data/diffBinding_rep1/diffPeaks/MV411_IRF2BP2_DMSO_POLII_total_R1_vs_MV411_IRF2BP2_VHL_POLII_total_R1_c3.0_cond1.bed ../../data/diffBinding_rep1/diffPeaks/MV411_IRF2BP2_DMSO_POLII_total_R1_vs_MV411_IRF2BP2_VHL_POLII_total_R1_c3.0_common.bed ../../data/diffBinding_rep1/diffPeaks/MV411_IRF2BP2_DMSO_POLII_total_R1_vs_MV411_IRF2BP2_VHL_POLII_total_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_total_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_rep1/diffPeaks/POLII_total_mat.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_total_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle POLII_total', returncode=0, stdout=b'', stderr=b\"WARNING: Group 'MV411_IRF2BP2_DMSO_POLII_total_R1_vs_MV411_IRF2BP2_VHL_POLII_total_R1_c3.0_cond1.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n",
      "found 1 header line... removing\n",
      "removed 1 lines\n",
      "found 1 header line... removing\n",
      "removed 38 lines\n",
      "found 0 header line... removing\n",
      "removed 1 lines\n",
      "../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_SPI1_R1.bw ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_SPI1_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_SPI1_R1.bw ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_SPI1_R1.bw  --referencePoint center --regionsFileName ../../data/diffBinding_rep1/diffPeaks/MV411_IRF2BP2_DMSO_SPI1_R1_vs_MV411_IRF2BP2_VHL_SPI1_R1_c3.0_cond1.bed ../../data/diffBinding_rep1/diffPeaks/MV411_IRF2BP2_DMSO_SPI1_R1_vs_MV411_IRF2BP2_VHL_SPI1_R1_c3.0_common.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/SPI1_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_rep1/diffPeaks/SPI1_mat.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/SPI1_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common  --samplesLabel DMSO VHL  --plotTitle SPI1', returncode=0, stdout=b'', stderr=b'')\n"
     ]
    }
   ],
   "source": [
    "for i in range(int(len(bw)/2)):\n",
    "    if i<0:\n",
    "        continue\n",
    "    name1 = bw[i]\n",
    "    name2 = bw[8+i]\n",
    "    peak = [cond1peak[i], commonpeak[i], cond2peak[i]]\n",
    "    for val in peak:\n",
    "        chip.dropWeirdChromosomes(val)\n",
    "    name = names[i]\n",
    "    print(name1,name2)\n",
    "    chip.computePeaksAt(peak, [name1, name2], bigwignames=['DMSO', 'VHL'], peaknames=['DMSO_peaks', 'common', 'VHL_peaks'], window=3000, refpoint='center', folder=\"\", title=name, numthreads=7, torecompute=True, name='../../data/diffBinding_rep1/diffPeaks/'+name+'_mat.pdf', withDeeptools=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 107,
   "metadata": {},
   "outputs": [],
   "source": [
    "mkdir ../../data/peaks1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 112,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/macs/narrowPeak/MV411_IRF2BP2_DMSO_R1_peaks.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/macs/narrowPeak/MV411_IRF2BP2_VHL_R1_peaks.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/macs/narrowPeak/mp779-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r1.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/macs/narrowPeak/mp780-MV411_IRF2BP2_DMSO_MED1-MED1-r1.narrowPeak...\n",
      "\\ [4 files][ 17.2 MiB/ 17.2 MiB]                                                \n",
      "==> NOTE: You are performing a sequence of gsutil operations that may\n",
      "run significantly faster if you instead use gsutil -m cp ... Please\n",
      "see the -m section under \"gsutil help options\" for further information\n",
      "about when gsutil -m can be advantageous.\n",
      "\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/macs/narrowPeak/mp781-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r1.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/macs/narrowPeak/mp782-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r1.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/macs/narrowPeak/mp783-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r1.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/macs/narrowPeak/mp784-MV411_IRF2BP2_DMSO_MYC-MYC-r1.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/macs/narrowPeak/mp785-MV411_IRF2BP2_DMSO_MYB-MYB-r1.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/macs/narrowPeak/mp786-MV411_IRF2BP2_DMSO_SPI1-SPI1-r1.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/macs/narrowPeak/mp787-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r1.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/macs/narrowPeak/mp788-MV411_IRF2BP2_VHL_MED1-MED1-r1.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/macs/narrowPeak/mp789-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r1.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/macs/narrowPeak/mp790-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r1.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/macs/narrowPeak/mp791-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r1.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/macs/narrowPeak/mp792-MV411_IRF2BP2_VHL_MYC-MYC-r1.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/macs/narrowPeak/mp793-MV411_IRF2BP2_VHL_MYB-MYB-r1.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/macs/narrowPeak/mp794-MV411_IRF2BP2_VHL_SPI1-SPI1-r1.narrowPeak...\n",
      "/ [18 files][ 99.5 MiB/ 99.5 MiB]                                               \n",
      "Operation completed over 18 objects/99.5 MiB.                                    \n"
     ]
    }
   ],
   "source": [
    "! gsutil cp gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/macs/narrowPeak/*.narrowPeak ../../data/peaks1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/peaks1/mp779-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r1.narrowPeak',\n",
       " '../../data/peaks1/mp780-MV411_IRF2BP2_DMSO_MED1-MED1-r1.narrowPeak',\n",
       " '../../data/peaks1/mp781-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r1.narrowPeak',\n",
       " '../../data/peaks1/mp782-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r1.narrowPeak',\n",
       " '../../data/peaks1/mp783-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r1.narrowPeak',\n",
       " '../../data/peaks1/mp784-MV411_IRF2BP2_DMSO_MYC-MYC-r1.narrowPeak',\n",
       " '../../data/peaks1/mp785-MV411_IRF2BP2_DMSO_MYB-MYB-r1.narrowPeak',\n",
       " '../../data/peaks1/mp786-MV411_IRF2BP2_DMSO_SPI1-SPI1-r1.narrowPeak',\n",
       " '../../data/peaks1/mp787-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r1.narrowPeak',\n",
       " '../../data/peaks1/mp788-MV411_IRF2BP2_VHL_MED1-MED1-r1.narrowPeak',\n",
       " '../../data/peaks1/mp789-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r1.narrowPeak',\n",
       " '../../data/peaks1/mp790-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r1.narrowPeak',\n",
       " '../../data/peaks1/mp791-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r1.narrowPeak',\n",
       " '../../data/peaks1/mp792-MV411_IRF2BP2_VHL_MYC-MYC-r1.narrowPeak',\n",
       " '../../data/peaks1/mp793-MV411_IRF2BP2_VHL_MYB-MYB-r1.narrowPeak',\n",
       " '../../data/peaks1/mp794-MV411_IRF2BP2_VHL_SPI1-SPI1-r1.narrowPeak',\n",
       " '../../data/peaks1/MV411_IRF2BP2_DMSO_R1_peaks.narrowPeak',\n",
       " '../../data/peaks1/MV411_IRF2BP2_VHL_R1_peaks.narrowPeak']"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "peaks = ! ls ../../data/peaks1/*\n",
    "peaks"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['FLAG_IRF2BP2',\n",
       " 'MED1',\n",
       " 'MYB',\n",
       " 'MYC',\n",
       " 'POLII_S2',\n",
       " 'POLII_S5',\n",
       " 'POLII_total',\n",
       " 'SPI1',\n",
       " 'FLAG_IRF2BP2_VHL',\n",
       " 'MED1_VHL',\n",
       " 'MYB_VHL',\n",
       " 'MYC_VHL',\n",
       " 'POLII_S2_VHL',\n",
       " 'POLII_S5_VHL',\n",
       " 'POLII_total_VHL',\n",
       " 'SPI1_VHL']"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "names = [\"FLAG_IRF2BP2\",\"MED1\",\"MYB\",\"MYC\",\"POLII_S2\",\"POLII_S5\",\"POLII_total\",\"SPI1\"]\n",
    "names.extend([i+'_VHL' for i in names])\n",
    "names"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_FLAG_IRF2BP2_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp779-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/FLAG_IRF2BP2_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/FLAG_IRF2BP2_mat_profile.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/FLAG_IRF2BP2_mat_profile.pdf --refPointLabel center --plotTitle FLAG_IRF2BP2', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_FLAG_IRF2BP2_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp779-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/FLAG_IRF2BP2_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/FLAG_IRF2BP2_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/FLAG_IRF2BP2_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle FLAG_IRF2BP2', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_MED1_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp780-MV411_IRF2BP2_DMSO_MED1-MED1-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/MED1_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/MED1_mat_profile.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/MED1_mat_profile.pdf --refPointLabel center --plotTitle MED1', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_MED1_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp780-MV411_IRF2BP2_DMSO_MED1-MED1-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/MED1_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/MED1_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/MED1_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle MED1', returncode=0, stdout=b'', stderr=b\"WARNING: Group 'cluster_1' is too small for plotting, you might want to remove it. \\n\")\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_MYB_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp781-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/MYB_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/MYB_mat_profile.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/MYB_mat_profile.pdf --refPointLabel center --plotTitle MYB', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_MYB_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp781-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/MYB_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/MYB_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/MYB_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle MYB', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_MYC_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp782-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/MYC_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/MYC_mat_profile.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/MYC_mat_profile.pdf --refPointLabel center --plotTitle MYC', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_MYC_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp782-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/MYC_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/MYC_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/MYC_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle MYC', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_POLII_S2_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp783-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_S2_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/POLII_S2_mat_profile.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_S2_mat_profile.pdf --refPointLabel center --plotTitle POLII_S2', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_POLII_S2_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp783-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_S2_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/POLII_S2_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_S2_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle POLII_S2', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_POLII_S5_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp784-MV411_IRF2BP2_DMSO_MYC-MYC-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_S5_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/POLII_S5_mat_profile.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_S5_mat_profile.pdf --refPointLabel center --plotTitle POLII_S5', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_POLII_S5_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp784-MV411_IRF2BP2_DMSO_MYC-MYC-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_S5_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/POLII_S5_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_S5_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle POLII_S5', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_POLII_total_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp785-MV411_IRF2BP2_DMSO_MYB-MYB-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_total_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/POLII_total_mat_profile.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_total_mat_profile.pdf --refPointLabel center --plotTitle POLII_total', returncode=0, stdout=b'', stderr=b'')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_POLII_total_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp785-MV411_IRF2BP2_DMSO_MYB-MYB-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_total_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/POLII_total_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_total_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle POLII_total', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_SPI1_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp786-MV411_IRF2BP2_DMSO_SPI1-SPI1-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/SPI1_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/SPI1_mat_profile.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/SPI1_mat_profile.pdf --refPointLabel center --plotTitle SPI1', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_DMSO_SPI1_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp786-MV411_IRF2BP2_DMSO_SPI1-SPI1-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/SPI1_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/SPI1_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/SPI1_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle SPI1', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_FLAG_IRF2BP2_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp787-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/FLAG_IRF2BP2_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/FLAG_IRF2BP2_VHL_mat_profile.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/FLAG_IRF2BP2_VHL_mat_profile.pdf --refPointLabel center --plotTitle FLAG_IRF2BP2_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_FLAG_IRF2BP2_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp787-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/FLAG_IRF2BP2_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/FLAG_IRF2BP2_VHL_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/FLAG_IRF2BP2_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle FLAG_IRF2BP2_VHL', returncode=0, stdout=b'', stderr=b\"WARNING: Group 'cluster_1' is too small for plotting, you might want to remove it. \\n\")\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_MED1_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp788-MV411_IRF2BP2_VHL_MED1-MED1-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/MED1_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/MED1_VHL_mat_profile.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/MED1_VHL_mat_profile.pdf --refPointLabel center --plotTitle MED1_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_MED1_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp788-MV411_IRF2BP2_VHL_MED1-MED1-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/MED1_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/MED1_VHL_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/MED1_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle MED1_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_MYB_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp789-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/MYB_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/MYB_VHL_mat_profile.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/MYB_VHL_mat_profile.pdf --refPointLabel center --plotTitle MYB_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_MYB_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp789-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/MYB_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/MYB_VHL_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/MYB_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle MYB_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_MYC_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp790-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/MYC_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/MYC_VHL_mat_profile.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/MYC_VHL_mat_profile.pdf --refPointLabel center --plotTitle MYC_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_MYC_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp790-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/MYC_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/MYC_VHL_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/MYC_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle MYC_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_POLII_S2_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp791-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_S2_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/POLII_S2_VHL_mat_profile.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_S2_VHL_mat_profile.pdf --refPointLabel center --plotTitle POLII_S2_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_POLII_S2_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp791-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_S2_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/POLII_S2_VHL_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_S2_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle POLII_S2_VHL', returncode=0, stdout=b'', stderr=b'')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_POLII_S5_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp792-MV411_IRF2BP2_VHL_MYC-MYC-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_S5_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/POLII_S5_VHL_mat_profile.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_S5_VHL_mat_profile.pdf --refPointLabel center --plotTitle POLII_S5_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_POLII_S5_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp792-MV411_IRF2BP2_VHL_MYC-MYC-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_S5_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/POLII_S5_VHL_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_S5_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle POLII_S5_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_POLII_total_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp793-MV411_IRF2BP2_VHL_MYB-MYB-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_total_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/POLII_total_VHL_mat_profile.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_total_VHL_mat_profile.pdf --refPointLabel center --plotTitle POLII_total_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_POLII_total_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp793-MV411_IRF2BP2_VHL_MYB-MYB-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_total_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/POLII_total_VHL_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/POLII_total_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle POLII_total_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_SPI1_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp794-MV411_IRF2BP2_VHL_SPI1-SPI1-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/SPI1_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/SPI1_VHL_mat_profile.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/SPI1_VHL_mat_profile.pdf --refPointLabel center --plotTitle SPI1_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep1/MV411_IRF2BP2_VHL_SPI1_R1.bw  --referencePoint center --regionsFileName ../../data/peaks1/mp794-MV411_IRF2BP2_VHL_SPI1-SPI1-r1.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks/SPI1_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep1/diffPeaks/SPI1_VHL_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep1/diffPeaks/SPI1_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle SPI1_VHL', returncode=0, stdout=b'', stderr=b'')\n"
     ]
    }
   ],
   "source": [
    "for i in range(len(bw)):\n",
    "    chip.computePeaksAt(peaks[i], bw[i], window=3000, folder=\"\", title=names[i], numthreads=7, torecompute=True, onlyProfile=True, refpoint='center', name='../../data/diffBinding_rep1/diffPeaks/'+names[i]+'_mat_profile.pdf', withDeeptools=True)\n",
    "    chip.computePeaksAt(peaks[i], bw[i], window=3000, folder=\"\", title=names[i], numthreads=7, torecompute=True, refpoint='center',onlyProfile=True,name='../../data/diffBinding_rep1/diffPeaks/'+names[i]+'_mat_profile_clust3.pdf', withDeeptools=True, cluster=3)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### on unscalled data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/diffBinding_rep1/mp779-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp780-MV411_IRF2BP2_DMSO_MED1-MED1-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp781-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp782-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp783-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp784-MV411_IRF2BP2_DMSO_MYC-MYC-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp785-MV411_IRF2BP2_DMSO_MYB-MYB-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp786-MV411_IRF2BP2_DMSO_SPI1-SPI1-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp787-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp788-MV411_IRF2BP2_VHL_MED1-MED1-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp789-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp790-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp791-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp792-MV411_IRF2BP2_VHL_MYC-MYC-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp793-MV411_IRF2BP2_VHL_MYB-MYB-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp794-MV411_IRF2BP2_VHL_SPI1-SPI1-r1.mLb.clN.sorted.bam']"
      ]
     },
     "execution_count": 62,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bams = !ls ../../data/diffBinding_rep1/mp*.bam\n",
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/bin/sh: 1: mkdir../../data/diffBinding_rep1/diffPeaks_unscaled: not found\r\n"
     ]
    }
   ],
   "source": [
    "! mkdir ../../data/diffBinding_rep1/diffPeaks_unscaled"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [],
   "source": [
    "bams= list(bams)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp779-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp780-MV411_IRF2BP2_DMSO_MED1-MED1-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp781-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp782-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp783-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp784-MV411_IRF2BP2_DMSO_MYC-MYC-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp785-MV411_IRF2BP2_DMSO_MYB-MYB-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp786-MV411_IRF2BP2_DMSO_SPI1-SPI1-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp787-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp788-MV411_IRF2BP2_VHL_MED1-MED1-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp789-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp790-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp791-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp792-MV411_IRF2BP2_VHL_MYC-MYC-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp793-MV411_IRF2BP2_VHL_MYB-MYB-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/mp794-MV411_IRF2BP2_VHL_SPI1-SPI1-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/MV411_IRF2BP2_DMSO_R1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_rep1/MV411_IRF2BP2_VHL_R1.mLb.clN.sorted.bam']"
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[220, 191, 211, 204, 285, 222, 228, 194]"
      ]
     },
     "execution_count": 36,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "size"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "doing diff from ../../data/diffBinding_rep1/mp779-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r1.mLb.clN.sorted.bam and ../../data/diffBinding_rep1/mp787-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r1.mLb.clN.sorted.bam\n",
      "using default size\n",
      "computing the scaling values\n",
      "b'INFO  @ Mon, 20 Apr 2020 16:15:53: \\n# Command line: callpeak -B -t ../../data/diffBinding_rep1/mp779-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r1.mLb.clN.sorted.bam -c ../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 220 -n mp779-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r1 --outdir ../../data/diffBinding_rep1/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp779-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_rep1/mp779-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Mon, 20 Apr 2020 16:15:53: #1 read tag files... \\nINFO  @ Mon, 20 Apr 2020 16:15:53: #1 read treatment tags... \\nINFO  @ Mon, 20 Apr 2020 16:15:56:  1000000 \\nINFO  @ Mon, 20 Apr 2020 16:15:58:  2000000 \\nINFO  @ Mon, 20 Apr 2020 16:16:00:  3000000 \\nINFO  @ Mon, 20 Apr 2020 16:16:02:  4000000 \\nINFO  @ Mon, 20 Apr 2020 16:16:05:  5000000 \\nINFO  @ Mon, 20 Apr 2020 16:16:07:  6000000 \\nINFO  @ Mon, 20 Apr 2020 16:16:09:  7000000 \\nINFO  @ Mon, 20 Apr 2020 16:16:12:  8000000 \\nINFO  @ Mon, 20 Apr 2020 16:16:14:  9000000 \\nINFO  @ Mon, 20 Apr 2020 16:16:16:  10000000 \\nINFO  @ Mon, 20 Apr 2020 16:16:18:  11000000 \\nINFO  @ Mon, 20 Apr 2020 16:16:21:  12000000 \\nINFO  @ Mon, 20 Apr 2020 16:16:23:  13000000 \\nINFO  @ Mon, 20 Apr 2020 16:16:25:  14000000 \\nINFO  @ Mon, 20 Apr 2020 16:16:28:  15000000 \\nINFO  @ Mon, 20 Apr 2020 16:16:30:  16000000 \\nINFO  @ Mon, 20 Apr 2020 16:16:32:  17000000 \\nINFO  @ Mon, 20 Apr 2020 16:16:35:  18000000 \\nINFO  @ Mon, 20 Apr 2020 16:16:37:  19000000 \\nINFO  @ Mon, 20 Apr 2020 16:16:39:  20000000 \\nINFO  @ Mon, 20 Apr 2020 16:16:42:  21000000 \\nINFO  @ Mon, 20 Apr 2020 16:16:44:  22000000 \\nINFO  @ Mon, 20 Apr 2020 16:16:46:  23000000 \\nINFO  @ Mon, 20 Apr 2020 16:16:49:  24000000 \\nINFO  @ Mon, 20 Apr 2020 16:16:51:  25000000 \\nINFO  @ Mon, 20 Apr 2020 16:16:53:  26000000 \\nINFO  @ Mon, 20 Apr 2020 16:16:55: 26635598 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 16:16:55: #1.2 read input tags... \\nINFO  @ Mon, 20 Apr 2020 16:16:58:  1000000 \\nINFO  @ Mon, 20 Apr 2020 16:17:01:  2000000 \\nINFO  @ Mon, 20 Apr 2020 16:17:04:  3000000 \\nINFO  @ Mon, 20 Apr 2020 16:17:06:  4000000 \\nINFO  @ Mon, 20 Apr 2020 16:17:09:  5000000 \\nINFO  @ Mon, 20 Apr 2020 16:17:12:  6000000 \\nINFO  @ Mon, 20 Apr 2020 16:17:14:  7000000 \\nINFO  @ Mon, 20 Apr 2020 16:17:17:  8000000 \\nINFO  @ Mon, 20 Apr 2020 16:17:20:  9000000 \\nINFO  @ Mon, 20 Apr 2020 16:17:22:  10000000 \\nINFO  @ Mon, 20 Apr 2020 16:17:25:  11000000 \\nINFO  @ Mon, 20 Apr 2020 16:17:27:  12000000 \\nINFO  @ Mon, 20 Apr 2020 16:17:30:  13000000 \\nINFO  @ Mon, 20 Apr 2020 16:17:33:  14000000 \\nINFO  @ Mon, 20 Apr 2020 16:17:35:  15000000 \\nINFO  @ Mon, 20 Apr 2020 16:17:38:  16000000 \\nINFO  @ Mon, 20 Apr 2020 16:17:41:  17000000 \\nINFO  @ Mon, 20 Apr 2020 16:17:44:  18000000 \\nINFO  @ Mon, 20 Apr 2020 16:17:46:  19000000 \\nINFO  @ Mon, 20 Apr 2020 16:17:49:  20000000 \\nINFO  @ Mon, 20 Apr 2020 16:17:52:  21000000 \\nINFO  @ Mon, 20 Apr 2020 16:17:54:  22000000 \\nINFO  @ Mon, 20 Apr 2020 16:17:57:  23000000 \\nINFO  @ Mon, 20 Apr 2020 16:18:00:  24000000 \\nINFO  @ Mon, 20 Apr 2020 16:18:02:  25000000 \\nINFO  @ Mon, 20 Apr 2020 16:18:05:  26000000 \\nINFO  @ Mon, 20 Apr 2020 16:18:08:  27000000 \\nINFO  @ Mon, 20 Apr 2020 16:18:10:  28000000 \\nINFO  @ Mon, 20 Apr 2020 16:18:13:  29000000 \\nINFO  @ Mon, 20 Apr 2020 16:18:16:  30000000 \\nINFO  @ Mon, 20 Apr 2020 16:18:18:  31000000 \\nINFO  @ Mon, 20 Apr 2020 16:18:21:  32000000 \\nINFO  @ Mon, 20 Apr 2020 16:18:24:  33000000 \\nINFO  @ Mon, 20 Apr 2020 16:18:26:  34000000 \\nINFO  @ Mon, 20 Apr 2020 16:18:28: 34520959 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 16:18:28: #1 tag size is determined as 48 bps \\nINFO  @ Mon, 20 Apr 2020 16:18:28: #1 tag size = 48.0 \\nINFO  @ Mon, 20 Apr 2020 16:18:28: #1  total tags in treatment: 26635598 \\nINFO  @ Mon, 20 Apr 2020 16:18:28: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 16:18:28: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 16:18:28: #1  tags after filtering in treatment: 26630518 \\nINFO  @ Mon, 20 Apr 2020 16:18:28: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 20 Apr 2020 16:18:28: #1  total tags in control: 34520959 \\nINFO  @ Mon, 20 Apr 2020 16:18:28: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 16:18:28: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 16:18:28: #1  tags after filtering in control: 34516729 \\nINFO  @ Mon, 20 Apr 2020 16:18:28: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 20 Apr 2020 16:18:28: #1 finished! \\nINFO  @ Mon, 20 Apr 2020 16:18:28: #2 Build Peak Model... \\nINFO  @ Mon, 20 Apr 2020 16:18:28: #2 Skipped... \\nINFO  @ Mon, 20 Apr 2020 16:18:28: #2 Use 220 as fragment length \\nINFO  @ Mon, 20 Apr 2020 16:18:28: #3 Call peaks... \\nINFO  @ Mon, 20 Apr 2020 16:18:28: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 20 Apr 2020 16:21:45: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 20 Apr 2020 16:21:45: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp779-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r1_treat_pileup.bdg \\nINFO  @ Mon, 20 Apr 2020 16:21:45: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp779-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r1_control_lambda.bdg \\nINFO  @ Mon, 20 Apr 2020 16:21:45: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 20 Apr 2020 16:21:45: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 20 Apr 2020 16:24:32: #4 Write output xls file... ../../data/diffBinding_rep1/diffData_unscaled/mp779-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r1_peaks.xls \\nINFO  @ Mon, 20 Apr 2020 16:24:33: #4 Write peak in narrowPeak format file... ../../data/diffBinding_rep1/diffData_unscaled/mp779-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r1_peaks.narrowPeak \\nINFO  @ Mon, 20 Apr 2020 16:24:33: #4 Write summits bed file... ../../data/diffBinding_rep1/diffData_unscaled/mp779-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r1_summits.bed \\nINFO  @ Mon, 20 Apr 2020 16:24:33: Done! \\n'\n",
      "b'INFO  @ Mon, 20 Apr 2020 16:24:33: \\n# Command line: callpeak -B -t ../../data/diffBinding_rep1/mp787-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r1.mLb.clN.sorted.bam -c ../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 220 -n mp787-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r1 --outdir ../../data/diffBinding_rep1/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp787-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_rep1/mp787-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Mon, 20 Apr 2020 16:24:33: #1 read tag files... \\nINFO  @ Mon, 20 Apr 2020 16:24:33: #1 read treatment tags... \\nINFO  @ Mon, 20 Apr 2020 16:24:35:  1000000 \\nINFO  @ Mon, 20 Apr 2020 16:24:38:  2000000 \\nINFO  @ Mon, 20 Apr 2020 16:24:40:  3000000 \\nINFO  @ Mon, 20 Apr 2020 16:24:43:  4000000 \\nINFO  @ Mon, 20 Apr 2020 16:24:45:  5000000 \\nINFO  @ Mon, 20 Apr 2020 16:24:47:  6000000 \\nINFO  @ Mon, 20 Apr 2020 16:24:50:  7000000 \\nINFO  @ Mon, 20 Apr 2020 16:24:52:  8000000 \\nINFO  @ Mon, 20 Apr 2020 16:24:55:  9000000 \\nINFO  @ Mon, 20 Apr 2020 16:24:57:  10000000 \\nINFO  @ Mon, 20 Apr 2020 16:24:59:  11000000 \\nINFO  @ Mon, 20 Apr 2020 16:25:02:  12000000 \\nINFO  @ Mon, 20 Apr 2020 16:25:04:  13000000 \\nINFO  @ Mon, 20 Apr 2020 16:25:07:  14000000 \\nINFO  @ Mon, 20 Apr 2020 16:25:09:  15000000 \\nINFO  @ Mon, 20 Apr 2020 16:25:11:  16000000 \\nINFO  @ Mon, 20 Apr 2020 16:25:14:  17000000 \\nINFO  @ Mon, 20 Apr 2020 16:25:16:  18000000 \\nINFO  @ Mon, 20 Apr 2020 16:25:19:  19000000 \\nINFO  @ Mon, 20 Apr 2020 16:25:21:  20000000 \\nINFO  @ Mon, 20 Apr 2020 16:25:24:  21000000 \\nINFO  @ Mon, 20 Apr 2020 16:25:26:  22000000 \\nINFO  @ Mon, 20 Apr 2020 16:25:28:  23000000 \\nINFO  @ Mon, 20 Apr 2020 16:25:29: 23111379 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 16:25:29: #1.2 read input tags... \\nINFO  @ Mon, 20 Apr 2020 16:25:32:  1000000 \\nINFO  @ Mon, 20 Apr 2020 16:25:34:  2000000 \\nINFO  @ Mon, 20 Apr 2020 16:25:36:  3000000 \\nINFO  @ Mon, 20 Apr 2020 16:25:39:  4000000 \\nINFO  @ Mon, 20 Apr 2020 16:25:41:  5000000 \\nINFO  @ Mon, 20 Apr 2020 16:25:44:  6000000 \\nINFO  @ Mon, 20 Apr 2020 16:25:46:  7000000 \\nINFO  @ Mon, 20 Apr 2020 16:25:49:  8000000 \\nINFO  @ Mon, 20 Apr 2020 16:25:51:  9000000 \\nINFO  @ Mon, 20 Apr 2020 16:25:54:  10000000 \\nINFO  @ Mon, 20 Apr 2020 16:25:56:  11000000 \\nINFO  @ Mon, 20 Apr 2020 16:25:59:  12000000 \\nINFO  @ Mon, 20 Apr 2020 16:26:01:  13000000 \\nINFO  @ Mon, 20 Apr 2020 16:26:03:  14000000 \\nINFO  @ Mon, 20 Apr 2020 16:26:06:  15000000 \\nINFO  @ Mon, 20 Apr 2020 16:26:08:  16000000 \\nINFO  @ Mon, 20 Apr 2020 16:26:11:  17000000 \\nINFO  @ Mon, 20 Apr 2020 16:26:13:  18000000 \\nINFO  @ Mon, 20 Apr 2020 16:26:16:  19000000 \\nINFO  @ Mon, 20 Apr 2020 16:26:18:  20000000 \\nINFO  @ Mon, 20 Apr 2020 16:26:20:  21000000 \\nINFO  @ Mon, 20 Apr 2020 16:26:23:  22000000 \\nINFO  @ Mon, 20 Apr 2020 16:26:25:  23000000 \\nINFO  @ Mon, 20 Apr 2020 16:26:28:  24000000 \\nINFO  @ Mon, 20 Apr 2020 16:26:30:  25000000 \\nINFO  @ Mon, 20 Apr 2020 16:26:33:  26000000 \\nINFO  @ Mon, 20 Apr 2020 16:26:35:  27000000 \\nINFO  @ Mon, 20 Apr 2020 16:26:38:  28000000 \\nINFO  @ Mon, 20 Apr 2020 16:26:40:  29000000 \\nINFO  @ Mon, 20 Apr 2020 16:26:42:  30000000 \\nINFO  @ Mon, 20 Apr 2020 16:26:45:  31000000 \\nINFO  @ Mon, 20 Apr 2020 16:26:47:  32000000 \\nINFO  @ Mon, 20 Apr 2020 16:26:50:  33000000 \\nINFO  @ Mon, 20 Apr 2020 16:26:52:  34000000 \\nINFO  @ Mon, 20 Apr 2020 16:26:54: 34520959 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 16:26:54: #1 tag size is determined as 49 bps \\nINFO  @ Mon, 20 Apr 2020 16:26:54: #1 tag size = 49.0 \\nINFO  @ Mon, 20 Apr 2020 16:26:54: #1  total tags in treatment: 23111379 \\nINFO  @ Mon, 20 Apr 2020 16:26:54: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 16:26:54: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 16:26:54: #1  tags after filtering in treatment: 23108284 \\nINFO  @ Mon, 20 Apr 2020 16:26:54: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 20 Apr 2020 16:26:54: #1  total tags in control: 34520959 \\nINFO  @ Mon, 20 Apr 2020 16:26:54: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 16:26:54: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 16:26:54: #1  tags after filtering in control: 34516729 \\nINFO  @ Mon, 20 Apr 2020 16:26:54: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 20 Apr 2020 16:26:54: #1 finished! \\nINFO  @ Mon, 20 Apr 2020 16:26:54: #2 Build Peak Model... \\nINFO  @ Mon, 20 Apr 2020 16:26:54: #2 Skipped... \\nINFO  @ Mon, 20 Apr 2020 16:26:54: #2 Use 220 as fragment length \\nINFO  @ Mon, 20 Apr 2020 16:26:54: #3 Call peaks... \\nINFO  @ Mon, 20 Apr 2020 16:26:54: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 20 Apr 2020 16:30:03: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 20 Apr 2020 16:30:03: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp787-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r1_treat_pileup.bdg \\nINFO  @ Mon, 20 Apr 2020 16:30:03: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp787-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r1_control_lambda.bdg \\nINFO  @ Mon, 20 Apr 2020 16:30:03: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 20 Apr 2020 16:30:03: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 20 Apr 2020 16:32:42: #4 Write output xls file... ../../data/diffBinding_rep1/diffData_unscaled/mp787-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r1_peaks.xls \\nINFO  @ Mon, 20 Apr 2020 16:32:42: #4 Write peak in narrowPeak format file... ../../data/diffBinding_rep1/diffData_unscaled/mp787-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r1_peaks.narrowPeak \\nINFO  @ Mon, 20 Apr 2020 16:32:42: #4 Write summits bed file... ../../data/diffBinding_rep1/diffData_unscaled/mp787-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r1_summits.bed \\nINFO  @ Mon, 20 Apr 2020 16:32:42: Done! \\n'\n",
      "26630518 23108284\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "doing diff from ../../data/diffBinding_rep1/mp780-MV411_IRF2BP2_DMSO_MED1-MED1-r1.mLb.clN.sorted.bam and ../../data/diffBinding_rep1/mp788-MV411_IRF2BP2_VHL_MED1-MED1-r1.mLb.clN.sorted.bam\n",
      "using default size\n",
      "computing the scaling values\n",
      "b'INFO  @ Mon, 20 Apr 2020 16:40:46: \\n# Command line: callpeak -B -t ../../data/diffBinding_rep1/mp780-MV411_IRF2BP2_DMSO_MED1-MED1-r1.mLb.clN.sorted.bam -c ../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 191 -n mp780-MV411_IRF2BP2_DMSO_MED1-MED1-r1 --outdir ../../data/diffBinding_rep1/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp780-MV411_IRF2BP2_DMSO_MED1-MED1-r1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_rep1/mp780-MV411_IRF2BP2_DMSO_MED1-MED1-r1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Mon, 20 Apr 2020 16:40:46: #1 read tag files... \\nINFO  @ Mon, 20 Apr 2020 16:40:46: #1 read treatment tags... \\nINFO  @ Mon, 20 Apr 2020 16:40:49:  1000000 \\nINFO  @ Mon, 20 Apr 2020 16:40:51:  2000000 \\nINFO  @ Mon, 20 Apr 2020 16:40:54:  3000000 \\nINFO  @ Mon, 20 Apr 2020 16:40:56:  4000000 \\nINFO  @ Mon, 20 Apr 2020 16:40:59:  5000000 \\nINFO  @ Mon, 20 Apr 2020 16:41:01:  6000000 \\nINFO  @ Mon, 20 Apr 2020 16:41:04:  7000000 \\nINFO  @ Mon, 20 Apr 2020 16:41:06:  8000000 \\nINFO  @ Mon, 20 Apr 2020 16:41:08:  9000000 \\nINFO  @ Mon, 20 Apr 2020 16:41:11:  10000000 \\nINFO  @ Mon, 20 Apr 2020 16:41:13:  11000000 \\nINFO  @ Mon, 20 Apr 2020 16:41:16:  12000000 \\nINFO  @ Mon, 20 Apr 2020 16:41:18:  13000000 \\nINFO  @ Mon, 20 Apr 2020 16:41:20:  14000000 \\nINFO  @ Mon, 20 Apr 2020 16:41:23:  15000000 \\nINFO  @ Mon, 20 Apr 2020 16:41:25:  16000000 \\nINFO  @ Mon, 20 Apr 2020 16:41:28:  17000000 \\nINFO  @ Mon, 20 Apr 2020 16:41:30:  18000000 \\nINFO  @ Mon, 20 Apr 2020 16:41:32:  19000000 \\nINFO  @ Mon, 20 Apr 2020 16:41:35:  20000000 \\nINFO  @ Mon, 20 Apr 2020 16:41:37:  21000000 \\nINFO  @ Mon, 20 Apr 2020 16:41:40:  22000000 \\nINFO  @ Mon, 20 Apr 2020 16:41:42:  23000000 \\nINFO  @ Mon, 20 Apr 2020 16:41:44: 23529002 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 16:41:44: #1.2 read input tags... \\nINFO  @ Mon, 20 Apr 2020 16:41:46:  1000000 \\nINFO  @ Mon, 20 Apr 2020 16:41:49:  2000000 \\nINFO  @ Mon, 20 Apr 2020 16:41:51:  3000000 \\nINFO  @ Mon, 20 Apr 2020 16:41:54:  4000000 \\nINFO  @ Mon, 20 Apr 2020 16:41:56:  5000000 \\nINFO  @ Mon, 20 Apr 2020 16:41:59:  6000000 \\nINFO  @ Mon, 20 Apr 2020 16:42:01:  7000000 \\nINFO  @ Mon, 20 Apr 2020 16:42:03:  8000000 \\nINFO  @ Mon, 20 Apr 2020 16:42:06:  9000000 \\nINFO  @ Mon, 20 Apr 2020 16:42:08:  10000000 \\nINFO  @ Mon, 20 Apr 2020 16:42:10:  11000000 \\nINFO  @ Mon, 20 Apr 2020 16:42:13:  12000000 \\nINFO  @ Mon, 20 Apr 2020 16:42:15:  13000000 \\nINFO  @ Mon, 20 Apr 2020 16:42:18:  14000000 \\nINFO  @ Mon, 20 Apr 2020 16:42:20:  15000000 \\nINFO  @ Mon, 20 Apr 2020 16:42:22:  16000000 \\nINFO  @ Mon, 20 Apr 2020 16:42:25:  17000000 \\nINFO  @ Mon, 20 Apr 2020 16:42:27:  18000000 \\nINFO  @ Mon, 20 Apr 2020 16:42:29:  19000000 \\nINFO  @ Mon, 20 Apr 2020 16:42:32:  20000000 \\nINFO  @ Mon, 20 Apr 2020 16:42:34:  21000000 \\nINFO  @ Mon, 20 Apr 2020 16:42:37:  22000000 \\nINFO  @ Mon, 20 Apr 2020 16:42:39:  23000000 \\nINFO  @ Mon, 20 Apr 2020 16:42:42:  24000000 \\nINFO  @ Mon, 20 Apr 2020 16:42:44:  25000000 \\nINFO  @ Mon, 20 Apr 2020 16:42:46:  26000000 \\nINFO  @ Mon, 20 Apr 2020 16:42:49:  27000000 \\nINFO  @ Mon, 20 Apr 2020 16:42:51:  28000000 \\nINFO  @ Mon, 20 Apr 2020 16:42:53:  29000000 \\nINFO  @ Mon, 20 Apr 2020 16:42:56:  30000000 \\nINFO  @ Mon, 20 Apr 2020 16:42:58:  31000000 \\nINFO  @ Mon, 20 Apr 2020 16:43:01:  32000000 \\nINFO  @ Mon, 20 Apr 2020 16:43:03:  33000000 \\nINFO  @ Mon, 20 Apr 2020 16:43:05:  34000000 \\nINFO  @ Mon, 20 Apr 2020 16:43:07: 34520959 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 16:43:07: #1 tag size is determined as 48 bps \\nINFO  @ Mon, 20 Apr 2020 16:43:07: #1 tag size = 48.0 \\nINFO  @ Mon, 20 Apr 2020 16:43:07: #1  total tags in treatment: 23529002 \\nINFO  @ Mon, 20 Apr 2020 16:43:07: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 16:43:07: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 16:43:07: #1  tags after filtering in treatment: 23525351 \\nINFO  @ Mon, 20 Apr 2020 16:43:07: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 20 Apr 2020 16:43:07: #1  total tags in control: 34520959 \\nINFO  @ Mon, 20 Apr 2020 16:43:07: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 16:43:07: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 16:43:07: #1  tags after filtering in control: 34516729 \\nINFO  @ Mon, 20 Apr 2020 16:43:07: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 20 Apr 2020 16:43:07: #1 finished! \\nINFO  @ Mon, 20 Apr 2020 16:43:07: #2 Build Peak Model... \\nINFO  @ Mon, 20 Apr 2020 16:43:07: #2 Skipped... \\nINFO  @ Mon, 20 Apr 2020 16:43:07: #2 Use 191 as fragment length \\nINFO  @ Mon, 20 Apr 2020 16:43:07: #3 Call peaks... \\nINFO  @ Mon, 20 Apr 2020 16:43:07: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 20 Apr 2020 16:46:08: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 20 Apr 2020 16:46:08: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp780-MV411_IRF2BP2_DMSO_MED1-MED1-r1_treat_pileup.bdg \\nINFO  @ Mon, 20 Apr 2020 16:46:08: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp780-MV411_IRF2BP2_DMSO_MED1-MED1-r1_control_lambda.bdg \\nINFO  @ Mon, 20 Apr 2020 16:46:08: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 20 Apr 2020 16:46:08: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 20 Apr 2020 16:48:49: #4 Write output xls file... ../../data/diffBinding_rep1/diffData_unscaled/mp780-MV411_IRF2BP2_DMSO_MED1-MED1-r1_peaks.xls \\nINFO  @ Mon, 20 Apr 2020 16:48:49: #4 Write peak in narrowPeak format file... ../../data/diffBinding_rep1/diffData_unscaled/mp780-MV411_IRF2BP2_DMSO_MED1-MED1-r1_peaks.narrowPeak \\nINFO  @ Mon, 20 Apr 2020 16:48:49: #4 Write summits bed file... ../../data/diffBinding_rep1/diffData_unscaled/mp780-MV411_IRF2BP2_DMSO_MED1-MED1-r1_summits.bed \\nINFO  @ Mon, 20 Apr 2020 16:48:49: Done! \\n'\n",
      "b'INFO  @ Mon, 20 Apr 2020 16:48:50: \\n# Command line: callpeak -B -t ../../data/diffBinding_rep1/mp788-MV411_IRF2BP2_VHL_MED1-MED1-r1.mLb.clN.sorted.bam -c ../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 191 -n mp788-MV411_IRF2BP2_VHL_MED1-MED1-r1 --outdir ../../data/diffBinding_rep1/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp788-MV411_IRF2BP2_VHL_MED1-MED1-r1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_rep1/mp788-MV411_IRF2BP2_VHL_MED1-MED1-r1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Mon, 20 Apr 2020 16:48:50: #1 read tag files... \\nINFO  @ Mon, 20 Apr 2020 16:48:50: #1 read treatment tags... \\nINFO  @ Mon, 20 Apr 2020 16:48:52:  1000000 \\nINFO  @ Mon, 20 Apr 2020 16:48:55:  2000000 \\nINFO  @ Mon, 20 Apr 2020 16:48:57:  3000000 \\nINFO  @ Mon, 20 Apr 2020 16:48:59:  4000000 \\nINFO  @ Mon, 20 Apr 2020 16:49:02:  5000000 \\nINFO  @ Mon, 20 Apr 2020 16:49:04:  6000000 \\nINFO  @ Mon, 20 Apr 2020 16:49:06:  7000000 \\nINFO  @ Mon, 20 Apr 2020 16:49:09:  8000000 \\nINFO  @ Mon, 20 Apr 2020 16:49:12:  9000000 \\nINFO  @ Mon, 20 Apr 2020 16:49:14:  10000000 \\nINFO  @ Mon, 20 Apr 2020 16:49:16:  11000000 \\nINFO  @ Mon, 20 Apr 2020 16:49:19:  12000000 \\nINFO  @ Mon, 20 Apr 2020 16:49:21:  13000000 \\nINFO  @ Mon, 20 Apr 2020 16:49:21: 13086883 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 16:49:21: #1.2 read input tags... \\nINFO  @ Mon, 20 Apr 2020 16:49:24:  1000000 \\nINFO  @ Mon, 20 Apr 2020 16:49:26:  2000000 \\nINFO  @ Mon, 20 Apr 2020 16:49:28:  3000000 \\nINFO  @ Mon, 20 Apr 2020 16:49:31:  4000000 \\nINFO  @ Mon, 20 Apr 2020 16:49:33:  5000000 \\nINFO  @ Mon, 20 Apr 2020 16:49:35:  6000000 \\nINFO  @ Mon, 20 Apr 2020 16:49:38:  7000000 \\nINFO  @ Mon, 20 Apr 2020 16:49:40:  8000000 \\nINFO  @ Mon, 20 Apr 2020 16:49:42:  9000000 \\nINFO  @ Mon, 20 Apr 2020 16:49:45:  10000000 \\nINFO  @ Mon, 20 Apr 2020 16:49:47:  11000000 \\nINFO  @ Mon, 20 Apr 2020 16:49:49:  12000000 \\nINFO  @ Mon, 20 Apr 2020 16:49:52:  13000000 \\nINFO  @ Mon, 20 Apr 2020 16:49:54:  14000000 \\nINFO  @ Mon, 20 Apr 2020 16:49:56:  15000000 \\nINFO  @ Mon, 20 Apr 2020 16:49:59:  16000000 \\nINFO  @ Mon, 20 Apr 2020 16:50:01:  17000000 \\nINFO  @ Mon, 20 Apr 2020 16:50:03:  18000000 \\nINFO  @ Mon, 20 Apr 2020 16:50:05:  19000000 \\nINFO  @ Mon, 20 Apr 2020 16:50:08:  20000000 \\nINFO  @ Mon, 20 Apr 2020 16:50:10:  21000000 \\nINFO  @ Mon, 20 Apr 2020 16:50:13:  22000000 \\nINFO  @ Mon, 20 Apr 2020 16:50:15:  23000000 \\nINFO  @ Mon, 20 Apr 2020 16:50:17:  24000000 \\nINFO  @ Mon, 20 Apr 2020 16:50:20:  25000000 \\nINFO  @ Mon, 20 Apr 2020 16:50:22:  26000000 \\nINFO  @ Mon, 20 Apr 2020 16:50:24:  27000000 \\nINFO  @ Mon, 20 Apr 2020 16:50:27:  28000000 \\nINFO  @ Mon, 20 Apr 2020 16:50:29:  29000000 \\nINFO  @ Mon, 20 Apr 2020 16:50:31:  30000000 \\nINFO  @ Mon, 20 Apr 2020 16:50:34:  31000000 \\nINFO  @ Mon, 20 Apr 2020 16:50:36:  32000000 \\nINFO  @ Mon, 20 Apr 2020 16:50:38:  33000000 \\nINFO  @ Mon, 20 Apr 2020 16:50:41:  34000000 \\nINFO  @ Mon, 20 Apr 2020 16:50:42: 34520959 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 16:50:42: #1 tag size is determined as 49 bps \\nINFO  @ Mon, 20 Apr 2020 16:50:42: #1 tag size = 49.0 \\nINFO  @ Mon, 20 Apr 2020 16:50:42: #1  total tags in treatment: 13086883 \\nINFO  @ Mon, 20 Apr 2020 16:50:42: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 16:50:42: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 16:50:42: #1  tags after filtering in treatment: 13084423 \\nINFO  @ Mon, 20 Apr 2020 16:50:42: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 20 Apr 2020 16:50:42: #1  total tags in control: 34520959 \\nINFO  @ Mon, 20 Apr 2020 16:50:42: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 16:50:42: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 16:50:43: #1  tags after filtering in control: 34516729 \\nINFO  @ Mon, 20 Apr 2020 16:50:43: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 20 Apr 2020 16:50:43: #1 finished! \\nINFO  @ Mon, 20 Apr 2020 16:50:43: #2 Build Peak Model... \\nINFO  @ Mon, 20 Apr 2020 16:50:43: #2 Skipped... \\nINFO  @ Mon, 20 Apr 2020 16:50:43: #2 Use 191 as fragment length \\nINFO  @ Mon, 20 Apr 2020 16:50:43: #3 Call peaks... \\nINFO  @ Mon, 20 Apr 2020 16:50:43: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 20 Apr 2020 16:53:23: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 20 Apr 2020 16:53:23: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp788-MV411_IRF2BP2_VHL_MED1-MED1-r1_treat_pileup.bdg \\nINFO  @ Mon, 20 Apr 2020 16:53:23: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp788-MV411_IRF2BP2_VHL_MED1-MED1-r1_control_lambda.bdg \\nINFO  @ Mon, 20 Apr 2020 16:53:23: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 20 Apr 2020 16:53:23: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 20 Apr 2020 16:55:44: #4 Write output xls file... ../../data/diffBinding_rep1/diffData_unscaled/mp788-MV411_IRF2BP2_VHL_MED1-MED1-r1_peaks.xls \\nINFO  @ Mon, 20 Apr 2020 16:55:44: #4 Write peak in narrowPeak format file... ../../data/diffBinding_rep1/diffData_unscaled/mp788-MV411_IRF2BP2_VHL_MED1-MED1-r1_peaks.narrowPeak \\nINFO  @ Mon, 20 Apr 2020 16:55:44: #4 Write summits bed file... ../../data/diffBinding_rep1/diffData_unscaled/mp788-MV411_IRF2BP2_VHL_MED1-MED1-r1_summits.bed \\nINFO  @ Mon, 20 Apr 2020 16:55:44: Done! \\n'\n",
      "23525351 13084423\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "doing diff from ../../data/diffBinding_rep1/mp781-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r1.mLb.clN.sorted.bam and ../../data/diffBinding_rep1/mp789-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r1.mLb.clN.sorted.bam\n",
      "using default size\n",
      "computing the scaling values\n",
      "b'INFO  @ Mon, 20 Apr 2020 17:02:43: \\n# Command line: callpeak -B -t ../../data/diffBinding_rep1/mp781-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r1.mLb.clN.sorted.bam -c ../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 211 -n mp781-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r1 --outdir ../../data/diffBinding_rep1/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp781-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_rep1/mp781-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Mon, 20 Apr 2020 17:02:43: #1 read tag files... \\nINFO  @ Mon, 20 Apr 2020 17:02:43: #1 read treatment tags... \\nINFO  @ Mon, 20 Apr 2020 17:02:46:  1000000 \\nINFO  @ Mon, 20 Apr 2020 17:02:48:  2000000 \\nINFO  @ Mon, 20 Apr 2020 17:02:51:  3000000 \\nINFO  @ Mon, 20 Apr 2020 17:02:53:  4000000 \\nINFO  @ Mon, 20 Apr 2020 17:02:55:  5000000 \\nINFO  @ Mon, 20 Apr 2020 17:02:58:  6000000 \\nINFO  @ Mon, 20 Apr 2020 17:03:00:  7000000 \\nINFO  @ Mon, 20 Apr 2020 17:03:03:  8000000 \\nINFO  @ Mon, 20 Apr 2020 17:03:05:  9000000 \\nINFO  @ Mon, 20 Apr 2020 17:03:08:  10000000 \\nINFO  @ Mon, 20 Apr 2020 17:03:10:  11000000 \\nINFO  @ Mon, 20 Apr 2020 17:03:12:  12000000 \\nINFO  @ Mon, 20 Apr 2020 17:03:15:  13000000 \\nINFO  @ Mon, 20 Apr 2020 17:03:17:  14000000 \\nINFO  @ Mon, 20 Apr 2020 17:03:19:  15000000 \\nINFO  @ Mon, 20 Apr 2020 17:03:22:  16000000 \\nINFO  @ Mon, 20 Apr 2020 17:03:24:  17000000 \\nINFO  @ Mon, 20 Apr 2020 17:03:27:  18000000 \\nINFO  @ Mon, 20 Apr 2020 17:03:29:  19000000 \\nINFO  @ Mon, 20 Apr 2020 17:03:32:  20000000 \\nINFO  @ Mon, 20 Apr 2020 17:03:34:  21000000 \\nINFO  @ Mon, 20 Apr 2020 17:03:36:  22000000 \\nINFO  @ Mon, 20 Apr 2020 17:03:39:  23000000 \\nINFO  @ Mon, 20 Apr 2020 17:03:41:  24000000 \\nINFO  @ Mon, 20 Apr 2020 17:03:44:  25000000 \\nINFO  @ Mon, 20 Apr 2020 17:03:46:  26000000 \\nINFO  @ Mon, 20 Apr 2020 17:03:49:  27000000 \\nINFO  @ Mon, 20 Apr 2020 17:03:51:  28000000 \\nINFO  @ Mon, 20 Apr 2020 17:03:54:  29000000 \\nINFO  @ Mon, 20 Apr 2020 17:03:56:  30000000 \\nINFO  @ Mon, 20 Apr 2020 17:03:58:  31000000 \\nINFO  @ Mon, 20 Apr 2020 17:04:01:  32000000 \\nINFO  @ Mon, 20 Apr 2020 17:04:02: 32428483 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 17:04:02: #1.2 read input tags... \\nINFO  @ Mon, 20 Apr 2020 17:04:05:  1000000 \\nINFO  @ Mon, 20 Apr 2020 17:04:07:  2000000 \\nINFO  @ Mon, 20 Apr 2020 17:04:10:  3000000 \\nINFO  @ Mon, 20 Apr 2020 17:04:12:  4000000 \\nINFO  @ Mon, 20 Apr 2020 17:04:15:  5000000 \\nINFO  @ Mon, 20 Apr 2020 17:04:17:  6000000 \\nINFO  @ Mon, 20 Apr 2020 17:04:20:  7000000 \\nINFO  @ Mon, 20 Apr 2020 17:04:22:  8000000 \\nINFO  @ Mon, 20 Apr 2020 17:04:25:  9000000 \\nINFO  @ Mon, 20 Apr 2020 17:04:27:  10000000 \\nINFO  @ Mon, 20 Apr 2020 17:04:30:  11000000 \\nINFO  @ Mon, 20 Apr 2020 17:04:32:  12000000 \\nINFO  @ Mon, 20 Apr 2020 17:04:34:  13000000 \\nINFO  @ Mon, 20 Apr 2020 17:04:37:  14000000 \\nINFO  @ Mon, 20 Apr 2020 17:04:39:  15000000 \\nINFO  @ Mon, 20 Apr 2020 17:04:42:  16000000 \\nINFO  @ Mon, 20 Apr 2020 17:04:44:  17000000 \\nINFO  @ Mon, 20 Apr 2020 17:04:47:  18000000 \\nINFO  @ Mon, 20 Apr 2020 17:04:49:  19000000 \\nINFO  @ Mon, 20 Apr 2020 17:04:52:  20000000 \\nINFO  @ Mon, 20 Apr 2020 17:04:54:  21000000 \\nINFO  @ Mon, 20 Apr 2020 17:04:57:  22000000 \\nINFO  @ Mon, 20 Apr 2020 17:04:59:  23000000 \\nINFO  @ Mon, 20 Apr 2020 17:05:02:  24000000 \\nINFO  @ Mon, 20 Apr 2020 17:05:04:  25000000 \\nINFO  @ Mon, 20 Apr 2020 17:05:07:  26000000 \\nINFO  @ Mon, 20 Apr 2020 17:05:10:  27000000 \\nINFO  @ Mon, 20 Apr 2020 17:05:12:  28000000 \\nINFO  @ Mon, 20 Apr 2020 17:05:15:  29000000 \\nINFO  @ Mon, 20 Apr 2020 17:05:17:  30000000 \\nINFO  @ Mon, 20 Apr 2020 17:05:20:  31000000 \\nINFO  @ Mon, 20 Apr 2020 17:05:22:  32000000 \\nINFO  @ Mon, 20 Apr 2020 17:05:24:  33000000 \\nINFO  @ Mon, 20 Apr 2020 17:05:27:  34000000 \\nINFO  @ Mon, 20 Apr 2020 17:05:28: 34520959 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 17:05:29: #1 tag size is determined as 49 bps \\nINFO  @ Mon, 20 Apr 2020 17:05:29: #1 tag size = 49.0 \\nINFO  @ Mon, 20 Apr 2020 17:05:29: #1  total tags in treatment: 32428483 \\nINFO  @ Mon, 20 Apr 2020 17:05:29: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 17:05:29: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 17:05:29: #1  tags after filtering in treatment: 32420265 \\nINFO  @ Mon, 20 Apr 2020 17:05:29: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 20 Apr 2020 17:05:29: #1  total tags in control: 34520959 \\nINFO  @ Mon, 20 Apr 2020 17:05:29: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 17:05:29: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 17:05:29: #1  tags after filtering in control: 34516729 \\nINFO  @ Mon, 20 Apr 2020 17:05:29: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 20 Apr 2020 17:05:29: #1 finished! \\nINFO  @ Mon, 20 Apr 2020 17:05:29: #2 Build Peak Model... \\nINFO  @ Mon, 20 Apr 2020 17:05:29: #2 Skipped... \\nINFO  @ Mon, 20 Apr 2020 17:05:29: #2 Use 211 as fragment length \\nINFO  @ Mon, 20 Apr 2020 17:05:29: #3 Call peaks... \\nINFO  @ Mon, 20 Apr 2020 17:05:29: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 20 Apr 2020 17:08:45: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 20 Apr 2020 17:08:45: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp781-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r1_treat_pileup.bdg \\nINFO  @ Mon, 20 Apr 2020 17:08:45: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp781-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r1_control_lambda.bdg \\nINFO  @ Mon, 20 Apr 2020 17:08:45: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 20 Apr 2020 17:08:45: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 20 Apr 2020 17:11:39: #4 Write output xls file... ../../data/diffBinding_rep1/diffData_unscaled/mp781-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r1_peaks.xls \\nINFO  @ Mon, 20 Apr 2020 17:11:39: #4 Write peak in narrowPeak format file... ../../data/diffBinding_rep1/diffData_unscaled/mp781-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r1_peaks.narrowPeak \\nINFO  @ Mon, 20 Apr 2020 17:11:39: #4 Write summits bed file... ../../data/diffBinding_rep1/diffData_unscaled/mp781-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r1_summits.bed \\nINFO  @ Mon, 20 Apr 2020 17:11:39: Done! \\n'\n",
      "b'INFO  @ Mon, 20 Apr 2020 17:11:40: \\n# Command line: callpeak -B -t ../../data/diffBinding_rep1/mp789-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r1.mLb.clN.sorted.bam -c ../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 211 -n mp789-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r1 --outdir ../../data/diffBinding_rep1/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp789-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_rep1/mp789-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Mon, 20 Apr 2020 17:11:40: #1 read tag files... \\nINFO  @ Mon, 20 Apr 2020 17:11:40: #1 read treatment tags... \\nINFO  @ Mon, 20 Apr 2020 17:11:42:  1000000 \\nINFO  @ Mon, 20 Apr 2020 17:11:45:  2000000 \\nINFO  @ Mon, 20 Apr 2020 17:11:47:  3000000 \\nINFO  @ Mon, 20 Apr 2020 17:11:50:  4000000 \\nINFO  @ Mon, 20 Apr 2020 17:11:53:  5000000 \\nINFO  @ Mon, 20 Apr 2020 17:11:55:  6000000 \\nINFO  @ Mon, 20 Apr 2020 17:11:58:  7000000 \\nINFO  @ Mon, 20 Apr 2020 17:12:00:  8000000 \\nINFO  @ Mon, 20 Apr 2020 17:12:03:  9000000 \\nINFO  @ Mon, 20 Apr 2020 17:12:05:  10000000 \\nINFO  @ Mon, 20 Apr 2020 17:12:07:  11000000 \\nINFO  @ Mon, 20 Apr 2020 17:12:10:  12000000 \\nINFO  @ Mon, 20 Apr 2020 17:12:12:  13000000 \\nINFO  @ Mon, 20 Apr 2020 17:12:15:  14000000 \\nINFO  @ Mon, 20 Apr 2020 17:12:17:  15000000 \\nINFO  @ Mon, 20 Apr 2020 17:12:19:  16000000 \\nINFO  @ Mon, 20 Apr 2020 17:12:22:  17000000 \\nINFO  @ Mon, 20 Apr 2020 17:12:24:  18000000 \\nINFO  @ Mon, 20 Apr 2020 17:12:26:  19000000 \\nINFO  @ Mon, 20 Apr 2020 17:12:29:  20000000 \\nINFO  @ Mon, 20 Apr 2020 17:12:31:  21000000 \\nINFO  @ Mon, 20 Apr 2020 17:12:33: 21557233 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 17:12:33: #1.2 read input tags... \\nINFO  @ Mon, 20 Apr 2020 17:12:35:  1000000 \\nINFO  @ Mon, 20 Apr 2020 17:12:38:  2000000 \\nINFO  @ Mon, 20 Apr 2020 17:12:40:  3000000 \\nINFO  @ Mon, 20 Apr 2020 17:12:43:  4000000 \\nINFO  @ Mon, 20 Apr 2020 17:12:45:  5000000 \\nINFO  @ Mon, 20 Apr 2020 17:12:47:  6000000 \\nINFO  @ Mon, 20 Apr 2020 17:12:50:  7000000 \\nINFO  @ Mon, 20 Apr 2020 17:12:52:  8000000 \\nINFO  @ Mon, 20 Apr 2020 17:12:55:  9000000 \\nINFO  @ Mon, 20 Apr 2020 17:12:57:  10000000 \\nINFO  @ Mon, 20 Apr 2020 17:12:59:  11000000 \\nINFO  @ Mon, 20 Apr 2020 17:13:02:  12000000 \\nINFO  @ Mon, 20 Apr 2020 17:13:04:  13000000 \\nINFO  @ Mon, 20 Apr 2020 17:13:07:  14000000 \\nINFO  @ Mon, 20 Apr 2020 17:13:09:  15000000 \\nINFO  @ Mon, 20 Apr 2020 17:13:11:  16000000 \\nINFO  @ Mon, 20 Apr 2020 17:13:14:  17000000 \\nINFO  @ Mon, 20 Apr 2020 17:13:16:  18000000 \\nINFO  @ Mon, 20 Apr 2020 17:13:19:  19000000 \\nINFO  @ Mon, 20 Apr 2020 17:13:21:  20000000 \\nINFO  @ Mon, 20 Apr 2020 17:13:23:  21000000 \\nINFO  @ Mon, 20 Apr 2020 17:13:26:  22000000 \\nINFO  @ Mon, 20 Apr 2020 17:13:28:  23000000 \\nINFO  @ Mon, 20 Apr 2020 17:13:31:  24000000 \\nINFO  @ Mon, 20 Apr 2020 17:13:33:  25000000 \\nINFO  @ Mon, 20 Apr 2020 17:13:35:  26000000 \\nINFO  @ Mon, 20 Apr 2020 17:13:38:  27000000 \\nINFO  @ Mon, 20 Apr 2020 17:13:40:  28000000 \\nINFO  @ Mon, 20 Apr 2020 17:13:43:  29000000 \\nINFO  @ Mon, 20 Apr 2020 17:13:45:  30000000 \\nINFO  @ Mon, 20 Apr 2020 17:13:47:  31000000 \\nINFO  @ Mon, 20 Apr 2020 17:13:50:  32000000 \\nINFO  @ Mon, 20 Apr 2020 17:13:52:  33000000 \\nINFO  @ Mon, 20 Apr 2020 17:13:55:  34000000 \\nINFO  @ Mon, 20 Apr 2020 17:13:56: 34520959 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 17:13:56: #1 tag size is determined as 49 bps \\nINFO  @ Mon, 20 Apr 2020 17:13:56: #1 tag size = 49.0 \\nINFO  @ Mon, 20 Apr 2020 17:13:56: #1  total tags in treatment: 21557233 \\nINFO  @ Mon, 20 Apr 2020 17:13:56: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 17:13:56: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 17:13:56: #1  tags after filtering in treatment: 21552188 \\nINFO  @ Mon, 20 Apr 2020 17:13:56: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 20 Apr 2020 17:13:56: #1  total tags in control: 34520959 \\nINFO  @ Mon, 20 Apr 2020 17:13:56: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 17:13:56: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 17:13:57: #1  tags after filtering in control: 34516729 \\nINFO  @ Mon, 20 Apr 2020 17:13:57: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 20 Apr 2020 17:13:57: #1 finished! \\nINFO  @ Mon, 20 Apr 2020 17:13:57: #2 Build Peak Model... \\nINFO  @ Mon, 20 Apr 2020 17:13:57: #2 Skipped... \\nINFO  @ Mon, 20 Apr 2020 17:13:57: #2 Use 211 as fragment length \\nINFO  @ Mon, 20 Apr 2020 17:13:57: #3 Call peaks... \\nINFO  @ Mon, 20 Apr 2020 17:13:57: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 20 Apr 2020 17:16:55: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 20 Apr 2020 17:16:55: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp789-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r1_treat_pileup.bdg \\nINFO  @ Mon, 20 Apr 2020 17:16:55: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp789-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r1_control_lambda.bdg \\nINFO  @ Mon, 20 Apr 2020 17:16:55: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 20 Apr 2020 17:16:55: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 20 Apr 2020 17:19:31: #4 Write output xls file... ../../data/diffBinding_rep1/diffData_unscaled/mp789-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r1_peaks.xls \\nINFO  @ Mon, 20 Apr 2020 17:19:31: #4 Write peak in narrowPeak format file... ../../data/diffBinding_rep1/diffData_unscaled/mp789-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r1_peaks.narrowPeak \\nINFO  @ Mon, 20 Apr 2020 17:19:31: #4 Write summits bed file... ../../data/diffBinding_rep1/diffData_unscaled/mp789-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r1_summits.bed \\nINFO  @ Mon, 20 Apr 2020 17:19:31: Done! \\n'\n",
      "32420265 21552188\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "doing diff from ../../data/diffBinding_rep1/mp782-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r1.mLb.clN.sorted.bam and ../../data/diffBinding_rep1/mp790-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r1.mLb.clN.sorted.bam\n",
      "using default size\n",
      "computing the scaling values\n",
      "b'INFO  @ Mon, 20 Apr 2020 17:27:57: \\n# Command line: callpeak -B -t ../../data/diffBinding_rep1/mp782-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r1.mLb.clN.sorted.bam -c ../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 204 -n mp782-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r1 --outdir ../../data/diffBinding_rep1/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp782-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_rep1/mp782-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Mon, 20 Apr 2020 17:27:57: #1 read tag files... \\nINFO  @ Mon, 20 Apr 2020 17:27:57: #1 read treatment tags... \\nINFO  @ Mon, 20 Apr 2020 17:28:00:  1000000 \\nINFO  @ Mon, 20 Apr 2020 17:28:03:  2000000 \\nINFO  @ Mon, 20 Apr 2020 17:28:05:  3000000 \\nINFO  @ Mon, 20 Apr 2020 17:28:08:  4000000 \\nINFO  @ Mon, 20 Apr 2020 17:28:10:  5000000 \\nINFO  @ Mon, 20 Apr 2020 17:28:12:  6000000 \\nINFO  @ Mon, 20 Apr 2020 17:28:15:  7000000 \\nINFO  @ Mon, 20 Apr 2020 17:28:17:  8000000 \\nINFO  @ Mon, 20 Apr 2020 17:28:20:  9000000 \\nINFO  @ Mon, 20 Apr 2020 17:28:22:  10000000 \\nINFO  @ Mon, 20 Apr 2020 17:28:25:  11000000 \\nINFO  @ Mon, 20 Apr 2020 17:28:28:  12000000 \\nINFO  @ Mon, 20 Apr 2020 17:28:30:  13000000 \\nINFO  @ Mon, 20 Apr 2020 17:28:33:  14000000 \\nINFO  @ Mon, 20 Apr 2020 17:28:35:  15000000 \\nINFO  @ Mon, 20 Apr 2020 17:28:37:  16000000 \\nINFO  @ Mon, 20 Apr 2020 17:28:40:  17000000 \\nINFO  @ Mon, 20 Apr 2020 17:28:42: 17744657 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 17:28:42: #1.2 read input tags... \\nINFO  @ Mon, 20 Apr 2020 17:28:44:  1000000 \\nINFO  @ Mon, 20 Apr 2020 17:28:47:  2000000 \\nINFO  @ Mon, 20 Apr 2020 17:28:49:  3000000 \\nINFO  @ Mon, 20 Apr 2020 17:28:52:  4000000 \\nINFO  @ Mon, 20 Apr 2020 17:28:54:  5000000 \\nINFO  @ Mon, 20 Apr 2020 17:28:56:  6000000 \\nINFO  @ Mon, 20 Apr 2020 17:28:59:  7000000 \\nINFO  @ Mon, 20 Apr 2020 17:29:01:  8000000 \\nINFO  @ Mon, 20 Apr 2020 17:29:04:  9000000 \\nINFO  @ Mon, 20 Apr 2020 17:29:06:  10000000 \\nINFO  @ Mon, 20 Apr 2020 17:29:08:  11000000 \\nINFO  @ Mon, 20 Apr 2020 17:29:11:  12000000 \\nINFO  @ Mon, 20 Apr 2020 17:29:13:  13000000 \\nINFO  @ Mon, 20 Apr 2020 17:29:16:  14000000 \\nINFO  @ Mon, 20 Apr 2020 17:29:18:  15000000 \\nINFO  @ Mon, 20 Apr 2020 17:29:20:  16000000 \\nINFO  @ Mon, 20 Apr 2020 17:29:23:  17000000 \\nINFO  @ Mon, 20 Apr 2020 17:29:25:  18000000 \\nINFO  @ Mon, 20 Apr 2020 17:29:28:  19000000 \\nINFO  @ Mon, 20 Apr 2020 17:29:30:  20000000 \\nINFO  @ Mon, 20 Apr 2020 17:29:32:  21000000 \\nINFO  @ Mon, 20 Apr 2020 17:29:35:  22000000 \\nINFO  @ Mon, 20 Apr 2020 17:29:37:  23000000 \\nINFO  @ Mon, 20 Apr 2020 17:29:39:  24000000 \\nINFO  @ Mon, 20 Apr 2020 17:29:42:  25000000 \\nINFO  @ Mon, 20 Apr 2020 17:29:44:  26000000 \\nINFO  @ Mon, 20 Apr 2020 17:29:47:  27000000 \\nINFO  @ Mon, 20 Apr 2020 17:29:49:  28000000 \\nINFO  @ Mon, 20 Apr 2020 17:29:51:  29000000 \\nINFO  @ Mon, 20 Apr 2020 17:29:54:  30000000 \\nINFO  @ Mon, 20 Apr 2020 17:29:56:  31000000 \\nINFO  @ Mon, 20 Apr 2020 17:29:59:  32000000 \\nINFO  @ Mon, 20 Apr 2020 17:30:01:  33000000 \\nINFO  @ Mon, 20 Apr 2020 17:30:03:  34000000 \\nINFO  @ Mon, 20 Apr 2020 17:30:05: 34520959 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 17:30:05: #1 tag size is determined as 49 bps \\nINFO  @ Mon, 20 Apr 2020 17:30:05: #1 tag size = 49.0 \\nINFO  @ Mon, 20 Apr 2020 17:30:05: #1  total tags in treatment: 17744657 \\nINFO  @ Mon, 20 Apr 2020 17:30:05: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 17:30:05: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 17:30:05: #1  tags after filtering in treatment: 17741131 \\nINFO  @ Mon, 20 Apr 2020 17:30:05: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 20 Apr 2020 17:30:05: #1  total tags in control: 34520959 \\nINFO  @ Mon, 20 Apr 2020 17:30:05: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 17:30:05: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 17:30:05: #1  tags after filtering in control: 34516729 \\nINFO  @ Mon, 20 Apr 2020 17:30:05: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 20 Apr 2020 17:30:05: #1 finished! \\nINFO  @ Mon, 20 Apr 2020 17:30:05: #2 Build Peak Model... \\nINFO  @ Mon, 20 Apr 2020 17:30:05: #2 Skipped... \\nINFO  @ Mon, 20 Apr 2020 17:30:05: #2 Use 204 as fragment length \\nINFO  @ Mon, 20 Apr 2020 17:30:05: #3 Call peaks... \\nINFO  @ Mon, 20 Apr 2020 17:30:05: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 20 Apr 2020 17:32:55: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 20 Apr 2020 17:32:55: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp782-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r1_treat_pileup.bdg \\nINFO  @ Mon, 20 Apr 2020 17:32:55: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp782-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r1_control_lambda.bdg \\nINFO  @ Mon, 20 Apr 2020 17:32:55: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 20 Apr 2020 17:32:55: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 20 Apr 2020 17:35:21: #4 Write output xls file... ../../data/diffBinding_rep1/diffData_unscaled/mp782-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r1_peaks.xls \\nINFO  @ Mon, 20 Apr 2020 17:35:21: #4 Write peak in narrowPeak format file... ../../data/diffBinding_rep1/diffData_unscaled/mp782-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r1_peaks.narrowPeak \\nINFO  @ Mon, 20 Apr 2020 17:35:21: #4 Write summits bed file... ../../data/diffBinding_rep1/diffData_unscaled/mp782-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r1_summits.bed \\nINFO  @ Mon, 20 Apr 2020 17:35:21: Done! \\n'\n",
      "b'INFO  @ Mon, 20 Apr 2020 17:35:21: \\n# Command line: callpeak -B -t ../../data/diffBinding_rep1/mp790-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r1.mLb.clN.sorted.bam -c ../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 204 -n mp790-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r1 --outdir ../../data/diffBinding_rep1/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp790-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_rep1/mp790-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Mon, 20 Apr 2020 17:35:21: #1 read tag files... \\nINFO  @ Mon, 20 Apr 2020 17:35:21: #1 read treatment tags... \\nINFO  @ Mon, 20 Apr 2020 17:35:24:  1000000 \\nINFO  @ Mon, 20 Apr 2020 17:35:27:  2000000 \\nINFO  @ Mon, 20 Apr 2020 17:35:29:  3000000 \\nINFO  @ Mon, 20 Apr 2020 17:35:32:  4000000 \\nINFO  @ Mon, 20 Apr 2020 17:35:34:  5000000 \\nINFO  @ Mon, 20 Apr 2020 17:35:37:  6000000 \\nINFO  @ Mon, 20 Apr 2020 17:35:39:  7000000 \\nINFO  @ Mon, 20 Apr 2020 17:35:42:  8000000 \\nINFO  @ Mon, 20 Apr 2020 17:35:44:  9000000 \\nINFO  @ Mon, 20 Apr 2020 17:35:47:  10000000 \\nINFO  @ Mon, 20 Apr 2020 17:35:49:  11000000 \\nINFO  @ Mon, 20 Apr 2020 17:35:51:  12000000 \\nINFO  @ Mon, 20 Apr 2020 17:35:54:  13000000 \\nINFO  @ Mon, 20 Apr 2020 17:35:57:  14000000 \\nINFO  @ Mon, 20 Apr 2020 17:35:57: 14102024 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 17:35:57: #1.2 read input tags... \\nINFO  @ Mon, 20 Apr 2020 17:35:59:  1000000 \\nINFO  @ Mon, 20 Apr 2020 17:36:02:  2000000 \\nINFO  @ Mon, 20 Apr 2020 17:36:04:  3000000 \\nINFO  @ Mon, 20 Apr 2020 17:36:07:  4000000 \\nINFO  @ Mon, 20 Apr 2020 17:36:09:  5000000 \\nINFO  @ Mon, 20 Apr 2020 17:36:12:  6000000 \\nINFO  @ Mon, 20 Apr 2020 17:36:14:  7000000 \\nINFO  @ Mon, 20 Apr 2020 17:36:16:  8000000 \\nINFO  @ Mon, 20 Apr 2020 17:36:19:  9000000 \\nINFO  @ Mon, 20 Apr 2020 17:36:21:  10000000 \\nINFO  @ Mon, 20 Apr 2020 17:36:24:  11000000 \\nINFO  @ Mon, 20 Apr 2020 17:36:26:  12000000 \\nINFO  @ Mon, 20 Apr 2020 17:36:28:  13000000 \\nINFO  @ Mon, 20 Apr 2020 17:36:31:  14000000 \\nINFO  @ Mon, 20 Apr 2020 17:36:33:  15000000 \\nINFO  @ Mon, 20 Apr 2020 17:36:36:  16000000 \\nINFO  @ Mon, 20 Apr 2020 17:36:38:  17000000 \\nINFO  @ Mon, 20 Apr 2020 17:36:40:  18000000 \\nINFO  @ Mon, 20 Apr 2020 17:36:43:  19000000 \\nINFO  @ Mon, 20 Apr 2020 17:36:45:  20000000 \\nINFO  @ Mon, 20 Apr 2020 17:36:48:  21000000 \\nINFO  @ Mon, 20 Apr 2020 17:36:50:  22000000 \\nINFO  @ Mon, 20 Apr 2020 17:36:52:  23000000 \\nINFO  @ Mon, 20 Apr 2020 17:36:55:  24000000 \\nINFO  @ Mon, 20 Apr 2020 17:36:57:  25000000 \\nINFO  @ Mon, 20 Apr 2020 17:37:00:  26000000 \\nINFO  @ Mon, 20 Apr 2020 17:37:02:  27000000 \\nINFO  @ Mon, 20 Apr 2020 17:37:04:  28000000 \\nINFO  @ Mon, 20 Apr 2020 17:37:07:  29000000 \\nINFO  @ Mon, 20 Apr 2020 17:37:09:  30000000 \\nINFO  @ Mon, 20 Apr 2020 17:37:12:  31000000 \\nINFO  @ Mon, 20 Apr 2020 17:37:14:  32000000 \\nINFO  @ Mon, 20 Apr 2020 17:37:16:  33000000 \\nINFO  @ Mon, 20 Apr 2020 17:37:19:  34000000 \\nINFO  @ Mon, 20 Apr 2020 17:37:20: 34520959 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 17:37:21: #1 tag size is determined as 49 bps \\nINFO  @ Mon, 20 Apr 2020 17:37:21: #1 tag size = 49.0 \\nINFO  @ Mon, 20 Apr 2020 17:37:21: #1  total tags in treatment: 14102024 \\nINFO  @ Mon, 20 Apr 2020 17:37:21: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 17:37:21: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 17:37:21: #1  tags after filtering in treatment: 14099396 \\nINFO  @ Mon, 20 Apr 2020 17:37:21: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 20 Apr 2020 17:37:21: #1  total tags in control: 34520959 \\nINFO  @ Mon, 20 Apr 2020 17:37:21: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 17:37:21: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 17:37:21: #1  tags after filtering in control: 34516729 \\nINFO  @ Mon, 20 Apr 2020 17:37:21: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 20 Apr 2020 17:37:21: #1 finished! \\nINFO  @ Mon, 20 Apr 2020 17:37:21: #2 Build Peak Model... \\nINFO  @ Mon, 20 Apr 2020 17:37:21: #2 Skipped... \\nINFO  @ Mon, 20 Apr 2020 17:37:21: #2 Use 204 as fragment length \\nINFO  @ Mon, 20 Apr 2020 17:37:21: #3 Call peaks... \\nINFO  @ Mon, 20 Apr 2020 17:37:21: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 20 Apr 2020 17:40:02: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 20 Apr 2020 17:40:02: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp790-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r1_treat_pileup.bdg \\nINFO  @ Mon, 20 Apr 2020 17:40:02: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp790-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r1_control_lambda.bdg \\nINFO  @ Mon, 20 Apr 2020 17:40:02: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 20 Apr 2020 17:40:02: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 20 Apr 2020 17:42:15: #4 Write output xls file... ../../data/diffBinding_rep1/diffData_unscaled/mp790-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r1_peaks.xls \\nINFO  @ Mon, 20 Apr 2020 17:42:16: #4 Write peak in narrowPeak format file... ../../data/diffBinding_rep1/diffData_unscaled/mp790-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r1_peaks.narrowPeak \\nINFO  @ Mon, 20 Apr 2020 17:42:16: #4 Write summits bed file... ../../data/diffBinding_rep1/diffData_unscaled/mp790-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r1_summits.bed \\nINFO  @ Mon, 20 Apr 2020 17:42:16: Done! \\n'\n",
      "17741131 14099396\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "doing diff from ../../data/diffBinding_rep1/mp783-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r1.mLb.clN.sorted.bam and ../../data/diffBinding_rep1/mp791-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r1.mLb.clN.sorted.bam\n",
      "using default size\n",
      "computing the scaling values\n",
      "b'INFO  @ Mon, 20 Apr 2020 17:48:53: \\n# Command line: callpeak -B -t ../../data/diffBinding_rep1/mp783-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r1.mLb.clN.sorted.bam -c ../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 285 -n mp783-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r1 --outdir ../../data/diffBinding_rep1/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp783-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_rep1/mp783-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Mon, 20 Apr 2020 17:48:53: #1 read tag files... \\nINFO  @ Mon, 20 Apr 2020 17:48:53: #1 read treatment tags... \\nINFO  @ Mon, 20 Apr 2020 17:48:55:  1000000 \\nINFO  @ Mon, 20 Apr 2020 17:48:58:  2000000 \\nINFO  @ Mon, 20 Apr 2020 17:49:00:  3000000 \\nINFO  @ Mon, 20 Apr 2020 17:49:03:  4000000 \\nINFO  @ Mon, 20 Apr 2020 17:49:05:  5000000 \\nINFO  @ Mon, 20 Apr 2020 17:49:07:  6000000 \\nINFO  @ Mon, 20 Apr 2020 17:49:10:  7000000 \\nINFO  @ Mon, 20 Apr 2020 17:49:12:  8000000 \\nINFO  @ Mon, 20 Apr 2020 17:49:14:  9000000 \\nINFO  @ Mon, 20 Apr 2020 17:49:17:  10000000 \\nINFO  @ Mon, 20 Apr 2020 17:49:19:  11000000 \\nINFO  @ Mon, 20 Apr 2020 17:49:21:  12000000 \\nINFO  @ Mon, 20 Apr 2020 17:49:24:  13000000 \\nINFO  @ Mon, 20 Apr 2020 17:49:26:  14000000 \\nINFO  @ Mon, 20 Apr 2020 17:49:28:  15000000 \\nINFO  @ Mon, 20 Apr 2020 17:49:31: 15831177 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 17:49:31: #1.2 read input tags... \\nINFO  @ Mon, 20 Apr 2020 17:49:33:  1000000 \\nINFO  @ Mon, 20 Apr 2020 17:49:35:  2000000 \\nINFO  @ Mon, 20 Apr 2020 17:49:38:  3000000 \\nINFO  @ Mon, 20 Apr 2020 17:49:40:  4000000 \\nINFO  @ Mon, 20 Apr 2020 17:49:42:  5000000 \\nINFO  @ Mon, 20 Apr 2020 17:49:45:  6000000 \\nINFO  @ Mon, 20 Apr 2020 17:49:47:  7000000 \\nINFO  @ Mon, 20 Apr 2020 17:49:49:  8000000 \\nINFO  @ Mon, 20 Apr 2020 17:49:52:  9000000 \\nINFO  @ Mon, 20 Apr 2020 17:49:54:  10000000 \\nINFO  @ Mon, 20 Apr 2020 17:49:56:  11000000 \\nINFO  @ Mon, 20 Apr 2020 17:49:59:  12000000 \\nINFO  @ Mon, 20 Apr 2020 17:50:01:  13000000 \\nINFO  @ Mon, 20 Apr 2020 17:50:03:  14000000 \\nINFO  @ Mon, 20 Apr 2020 17:50:06:  15000000 \\nINFO  @ Mon, 20 Apr 2020 17:50:08:  16000000 \\nINFO  @ Mon, 20 Apr 2020 17:50:10:  17000000 \\nINFO  @ Mon, 20 Apr 2020 17:50:13:  18000000 \\nINFO  @ Mon, 20 Apr 2020 17:50:15:  19000000 \\nINFO  @ Mon, 20 Apr 2020 17:50:17:  20000000 \\nINFO  @ Mon, 20 Apr 2020 17:50:20:  21000000 \\nINFO  @ Mon, 20 Apr 2020 17:50:22:  22000000 \\nINFO  @ Mon, 20 Apr 2020 17:50:24:  23000000 \\nINFO  @ Mon, 20 Apr 2020 17:50:27:  24000000 \\nINFO  @ Mon, 20 Apr 2020 17:50:30:  25000000 \\nINFO  @ Mon, 20 Apr 2020 17:50:33:  26000000 \\nINFO  @ Mon, 20 Apr 2020 17:50:35:  27000000 \\nINFO  @ Mon, 20 Apr 2020 17:50:38:  28000000 \\nINFO  @ Mon, 20 Apr 2020 17:50:40:  29000000 \\nINFO  @ Mon, 20 Apr 2020 17:50:42:  30000000 \\nINFO  @ Mon, 20 Apr 2020 17:50:45:  31000000 \\nINFO  @ Mon, 20 Apr 2020 17:50:47:  32000000 \\nINFO  @ Mon, 20 Apr 2020 17:50:49:  33000000 \\nINFO  @ Mon, 20 Apr 2020 17:50:52:  34000000 \\nINFO  @ Mon, 20 Apr 2020 17:50:53: 34520959 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 17:50:53: #1 tag size is determined as 49 bps \\nINFO  @ Mon, 20 Apr 2020 17:50:53: #1 tag size = 49.0 \\nINFO  @ Mon, 20 Apr 2020 17:50:53: #1  total tags in treatment: 15831177 \\nINFO  @ Mon, 20 Apr 2020 17:50:53: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 17:50:53: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 17:50:54: #1  tags after filtering in treatment: 15827847 \\nINFO  @ Mon, 20 Apr 2020 17:50:54: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 20 Apr 2020 17:50:54: #1  total tags in control: 34520959 \\nINFO  @ Mon, 20 Apr 2020 17:50:54: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 17:50:54: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 17:50:54: #1  tags after filtering in control: 34516729 \\nINFO  @ Mon, 20 Apr 2020 17:50:54: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 20 Apr 2020 17:50:54: #1 finished! \\nINFO  @ Mon, 20 Apr 2020 17:50:54: #2 Build Peak Model... \\nINFO  @ Mon, 20 Apr 2020 17:50:54: #2 Skipped... \\nINFO  @ Mon, 20 Apr 2020 17:50:54: #2 Use 285 as fragment length \\nINFO  @ Mon, 20 Apr 2020 17:50:54: #3 Call peaks... \\nINFO  @ Mon, 20 Apr 2020 17:50:54: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 20 Apr 2020 17:53:42: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 20 Apr 2020 17:53:42: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp783-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r1_treat_pileup.bdg \\nINFO  @ Mon, 20 Apr 2020 17:53:42: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp783-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r1_control_lambda.bdg \\nINFO  @ Mon, 20 Apr 2020 17:53:42: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 20 Apr 2020 17:53:42: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 20 Apr 2020 17:56:03: #4 Write output xls file... ../../data/diffBinding_rep1/diffData_unscaled/mp783-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r1_peaks.xls \\nINFO  @ Mon, 20 Apr 2020 17:56:03: #4 Write peak in narrowPeak format file... ../../data/diffBinding_rep1/diffData_unscaled/mp783-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r1_peaks.narrowPeak \\nINFO  @ Mon, 20 Apr 2020 17:56:03: #4 Write summits bed file... ../../data/diffBinding_rep1/diffData_unscaled/mp783-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r1_summits.bed \\nINFO  @ Mon, 20 Apr 2020 17:56:03: Done! \\n'\n",
      "b'INFO  @ Mon, 20 Apr 2020 17:56:03: \\n# Command line: callpeak -B -t ../../data/diffBinding_rep1/mp791-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r1.mLb.clN.sorted.bam -c ../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 285 -n mp791-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r1 --outdir ../../data/diffBinding_rep1/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp791-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_rep1/mp791-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Mon, 20 Apr 2020 17:56:03: #1 read tag files... \\nINFO  @ Mon, 20 Apr 2020 17:56:03: #1 read treatment tags... \\nINFO  @ Mon, 20 Apr 2020 17:56:06:  1000000 \\nINFO  @ Mon, 20 Apr 2020 17:56:08:  2000000 \\nINFO  @ Mon, 20 Apr 2020 17:56:11:  3000000 \\nINFO  @ Mon, 20 Apr 2020 17:56:13:  4000000 \\nINFO  @ Mon, 20 Apr 2020 17:56:16:  5000000 \\nINFO  @ Mon, 20 Apr 2020 17:56:18:  6000000 \\nINFO  @ Mon, 20 Apr 2020 17:56:21:  7000000 \\nINFO  @ Mon, 20 Apr 2020 17:56:23:  8000000 \\nINFO  @ Mon, 20 Apr 2020 17:56:26:  9000000 \\nINFO  @ Mon, 20 Apr 2020 17:56:28:  10000000 \\nINFO  @ Mon, 20 Apr 2020 17:56:30:  11000000 \\nINFO  @ Mon, 20 Apr 2020 17:56:33:  12000000 \\nINFO  @ Mon, 20 Apr 2020 17:56:35:  13000000 \\nINFO  @ Mon, 20 Apr 2020 17:56:37:  14000000 \\nINFO  @ Mon, 20 Apr 2020 17:56:40:  15000000 \\nINFO  @ Mon, 20 Apr 2020 17:56:42:  16000000 \\nINFO  @ Mon, 20 Apr 2020 17:56:45:  17000000 \\nINFO  @ Mon, 20 Apr 2020 17:56:47:  18000000 \\nINFO  @ Mon, 20 Apr 2020 17:56:50:  19000000 \\nINFO  @ Mon, 20 Apr 2020 17:56:50: 19308753 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 17:56:51: #1.2 read input tags... \\nINFO  @ Mon, 20 Apr 2020 17:56:53:  1000000 \\nINFO  @ Mon, 20 Apr 2020 17:56:55:  2000000 \\nINFO  @ Mon, 20 Apr 2020 17:56:58:  3000000 \\nINFO  @ Mon, 20 Apr 2020 17:57:00:  4000000 \\nINFO  @ Mon, 20 Apr 2020 17:57:02:  5000000 \\nINFO  @ Mon, 20 Apr 2020 17:57:05:  6000000 \\nINFO  @ Mon, 20 Apr 2020 17:57:07:  7000000 \\nINFO  @ Mon, 20 Apr 2020 17:57:09:  8000000 \\nINFO  @ Mon, 20 Apr 2020 17:57:12:  9000000 \\nINFO  @ Mon, 20 Apr 2020 17:57:14:  10000000 \\nINFO  @ Mon, 20 Apr 2020 17:57:16:  11000000 \\nINFO  @ Mon, 20 Apr 2020 17:57:19:  12000000 \\nINFO  @ Mon, 20 Apr 2020 17:57:21:  13000000 \\nINFO  @ Mon, 20 Apr 2020 17:57:23:  14000000 \\nINFO  @ Mon, 20 Apr 2020 17:57:26:  15000000 \\nINFO  @ Mon, 20 Apr 2020 17:57:28:  16000000 \\nINFO  @ Mon, 20 Apr 2020 17:57:30:  17000000 \\nINFO  @ Mon, 20 Apr 2020 17:57:33:  18000000 \\nINFO  @ Mon, 20 Apr 2020 17:57:35:  19000000 \\nINFO  @ Mon, 20 Apr 2020 17:57:37:  20000000 \\nINFO  @ Mon, 20 Apr 2020 17:57:40:  21000000 \\nINFO  @ Mon, 20 Apr 2020 17:57:42:  22000000 \\nINFO  @ Mon, 20 Apr 2020 17:57:45:  23000000 \\nINFO  @ Mon, 20 Apr 2020 17:57:47:  24000000 \\nINFO  @ Mon, 20 Apr 2020 17:57:49:  25000000 \\nINFO  @ Mon, 20 Apr 2020 17:57:52:  26000000 \\nINFO  @ Mon, 20 Apr 2020 17:57:54:  27000000 \\nINFO  @ Mon, 20 Apr 2020 17:57:56:  28000000 \\nINFO  @ Mon, 20 Apr 2020 17:57:59:  29000000 \\nINFO  @ Mon, 20 Apr 2020 17:58:01:  30000000 \\nINFO  @ Mon, 20 Apr 2020 17:58:04:  31000000 \\nINFO  @ Mon, 20 Apr 2020 17:58:06:  32000000 \\nINFO  @ Mon, 20 Apr 2020 17:58:08:  33000000 \\nINFO  @ Mon, 20 Apr 2020 17:58:11:  34000000 \\nINFO  @ Mon, 20 Apr 2020 17:58:12: 34520959 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 17:58:12: #1 tag size is determined as 49 bps \\nINFO  @ Mon, 20 Apr 2020 17:58:12: #1 tag size = 49.0 \\nINFO  @ Mon, 20 Apr 2020 17:58:12: #1  total tags in treatment: 19308753 \\nINFO  @ Mon, 20 Apr 2020 17:58:12: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 17:58:12: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 17:58:12: #1  tags after filtering in treatment: 19301224 \\nINFO  @ Mon, 20 Apr 2020 17:58:12: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 20 Apr 2020 17:58:12: #1  total tags in control: 34520959 \\nINFO  @ Mon, 20 Apr 2020 17:58:12: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 17:58:12: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 17:58:13: #1  tags after filtering in control: 34516729 \\nINFO  @ Mon, 20 Apr 2020 17:58:13: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 20 Apr 2020 17:58:13: #1 finished! \\nINFO  @ Mon, 20 Apr 2020 17:58:13: #2 Build Peak Model... \\nINFO  @ Mon, 20 Apr 2020 17:58:13: #2 Skipped... \\nINFO  @ Mon, 20 Apr 2020 17:58:13: #2 Use 285 as fragment length \\nINFO  @ Mon, 20 Apr 2020 17:58:13: #3 Call peaks... \\nINFO  @ Mon, 20 Apr 2020 17:58:13: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 20 Apr 2020 18:01:08: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 20 Apr 2020 18:01:08: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp791-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r1_treat_pileup.bdg \\nINFO  @ Mon, 20 Apr 2020 18:01:08: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp791-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r1_control_lambda.bdg \\nINFO  @ Mon, 20 Apr 2020 18:01:08: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 20 Apr 2020 18:01:08: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 20 Apr 2020 18:03:33: #4 Write output xls file... ../../data/diffBinding_rep1/diffData_unscaled/mp791-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r1_peaks.xls \\nINFO  @ Mon, 20 Apr 2020 18:03:33: #4 Write peak in narrowPeak format file... ../../data/diffBinding_rep1/diffData_unscaled/mp791-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r1_peaks.narrowPeak \\nINFO  @ Mon, 20 Apr 2020 18:03:33: #4 Write summits bed file... ../../data/diffBinding_rep1/diffData_unscaled/mp791-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r1_summits.bed \\nINFO  @ Mon, 20 Apr 2020 18:03:33: Done! \\n'\n",
      "15827847 19301224\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "doing diff from ../../data/diffBinding_rep1/mp784-MV411_IRF2BP2_DMSO_MYC-MYC-r1.mLb.clN.sorted.bam and ../../data/diffBinding_rep1/mp792-MV411_IRF2BP2_VHL_MYC-MYC-r1.mLb.clN.sorted.bam\n",
      "using default size\n",
      "computing the scaling values\n",
      "b'INFO  @ Mon, 20 Apr 2020 18:10:18: \\n# Command line: callpeak -B -t ../../data/diffBinding_rep1/mp784-MV411_IRF2BP2_DMSO_MYC-MYC-r1.mLb.clN.sorted.bam -c ../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 222 -n mp784-MV411_IRF2BP2_DMSO_MYC-MYC-r1 --outdir ../../data/diffBinding_rep1/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp784-MV411_IRF2BP2_DMSO_MYC-MYC-r1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_rep1/mp784-MV411_IRF2BP2_DMSO_MYC-MYC-r1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Mon, 20 Apr 2020 18:10:18: #1 read tag files... \\nINFO  @ Mon, 20 Apr 2020 18:10:18: #1 read treatment tags... \\nINFO  @ Mon, 20 Apr 2020 18:10:21:  1000000 \\nINFO  @ Mon, 20 Apr 2020 18:10:23:  2000000 \\nINFO  @ Mon, 20 Apr 2020 18:10:25:  3000000 \\nINFO  @ Mon, 20 Apr 2020 18:10:28:  4000000 \\nINFO  @ Mon, 20 Apr 2020 18:10:30:  5000000 \\nINFO  @ Mon, 20 Apr 2020 18:10:33:  6000000 \\nINFO  @ Mon, 20 Apr 2020 18:10:35:  7000000 \\nINFO  @ Mon, 20 Apr 2020 18:10:37:  8000000 \\nINFO  @ Mon, 20 Apr 2020 18:10:40:  9000000 \\nINFO  @ Mon, 20 Apr 2020 18:10:42:  10000000 \\nINFO  @ Mon, 20 Apr 2020 18:10:44:  11000000 \\nINFO  @ Mon, 20 Apr 2020 18:10:47:  12000000 \\nINFO  @ Mon, 20 Apr 2020 18:10:49:  13000000 \\nINFO  @ Mon, 20 Apr 2020 18:10:51:  14000000 \\nINFO  @ Mon, 20 Apr 2020 18:10:54:  15000000 \\nINFO  @ Mon, 20 Apr 2020 18:10:56:  16000000 \\nINFO  @ Mon, 20 Apr 2020 18:10:58:  17000000 \\nINFO  @ Mon, 20 Apr 2020 18:10:59: 17219904 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 18:10:59: #1.2 read input tags... \\nINFO  @ Mon, 20 Apr 2020 18:11:02:  1000000 \\nINFO  @ Mon, 20 Apr 2020 18:11:04:  2000000 \\nINFO  @ Mon, 20 Apr 2020 18:11:06:  3000000 \\nINFO  @ Mon, 20 Apr 2020 18:11:09:  4000000 \\nINFO  @ Mon, 20 Apr 2020 18:11:11:  5000000 \\nINFO  @ Mon, 20 Apr 2020 18:11:13:  6000000 \\nINFO  @ Mon, 20 Apr 2020 18:11:16:  7000000 \\nINFO  @ Mon, 20 Apr 2020 18:11:18:  8000000 \\nINFO  @ Mon, 20 Apr 2020 18:11:21:  9000000 \\nINFO  @ Mon, 20 Apr 2020 18:11:23:  10000000 \\nINFO  @ Mon, 20 Apr 2020 18:11:25:  11000000 \\nINFO  @ Mon, 20 Apr 2020 18:11:28:  12000000 \\nINFO  @ Mon, 20 Apr 2020 18:11:30:  13000000 \\nINFO  @ Mon, 20 Apr 2020 18:11:32:  14000000 \\nINFO  @ Mon, 20 Apr 2020 18:11:35:  15000000 \\nINFO  @ Mon, 20 Apr 2020 18:11:37:  16000000 \\nINFO  @ Mon, 20 Apr 2020 18:11:39:  17000000 \\nINFO  @ Mon, 20 Apr 2020 18:11:42:  18000000 \\nINFO  @ Mon, 20 Apr 2020 18:11:44:  19000000 \\nINFO  @ Mon, 20 Apr 2020 18:11:47:  20000000 \\nINFO  @ Mon, 20 Apr 2020 18:11:49:  21000000 \\nINFO  @ Mon, 20 Apr 2020 18:11:51:  22000000 \\nINFO  @ Mon, 20 Apr 2020 18:11:54:  23000000 \\nINFO  @ Mon, 20 Apr 2020 18:11:56:  24000000 \\nINFO  @ Mon, 20 Apr 2020 18:11:58:  25000000 \\nINFO  @ Mon, 20 Apr 2020 18:12:01:  26000000 \\nINFO  @ Mon, 20 Apr 2020 18:12:03:  27000000 \\nINFO  @ Mon, 20 Apr 2020 18:12:06:  28000000 \\nINFO  @ Mon, 20 Apr 2020 18:12:08:  29000000 \\nINFO  @ Mon, 20 Apr 2020 18:12:10:  30000000 \\nINFO  @ Mon, 20 Apr 2020 18:12:13:  31000000 \\nINFO  @ Mon, 20 Apr 2020 18:12:15:  32000000 \\nINFO  @ Mon, 20 Apr 2020 18:12:17:  33000000 \\nINFO  @ Mon, 20 Apr 2020 18:12:20:  34000000 \\nINFO  @ Mon, 20 Apr 2020 18:12:21: 34520959 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 18:12:22: #1 tag size is determined as 49 bps \\nINFO  @ Mon, 20 Apr 2020 18:12:22: #1 tag size = 49.0 \\nINFO  @ Mon, 20 Apr 2020 18:12:22: #1  total tags in treatment: 17219904 \\nINFO  @ Mon, 20 Apr 2020 18:12:22: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 18:12:22: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 18:12:22: #1  tags after filtering in treatment: 17215930 \\nINFO  @ Mon, 20 Apr 2020 18:12:22: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 20 Apr 2020 18:12:22: #1  total tags in control: 34520959 \\nINFO  @ Mon, 20 Apr 2020 18:12:22: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 18:12:22: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 18:12:22: #1  tags after filtering in control: 34516729 \\nINFO  @ Mon, 20 Apr 2020 18:12:22: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 20 Apr 2020 18:12:22: #1 finished! \\nINFO  @ Mon, 20 Apr 2020 18:12:22: #2 Build Peak Model... \\nINFO  @ Mon, 20 Apr 2020 18:12:22: #2 Skipped... \\nINFO  @ Mon, 20 Apr 2020 18:12:22: #2 Use 222 as fragment length \\nINFO  @ Mon, 20 Apr 2020 18:12:22: #3 Call peaks... \\nINFO  @ Mon, 20 Apr 2020 18:12:22: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 20 Apr 2020 18:15:15: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 20 Apr 2020 18:15:15: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp784-MV411_IRF2BP2_DMSO_MYC-MYC-r1_treat_pileup.bdg \\nINFO  @ Mon, 20 Apr 2020 18:15:15: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp784-MV411_IRF2BP2_DMSO_MYC-MYC-r1_control_lambda.bdg \\nINFO  @ Mon, 20 Apr 2020 18:15:15: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 20 Apr 2020 18:15:15: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 20 Apr 2020 18:17:40: #4 Write output xls file... ../../data/diffBinding_rep1/diffData_unscaled/mp784-MV411_IRF2BP2_DMSO_MYC-MYC-r1_peaks.xls \\nINFO  @ Mon, 20 Apr 2020 18:17:41: #4 Write peak in narrowPeak format file... ../../data/diffBinding_rep1/diffData_unscaled/mp784-MV411_IRF2BP2_DMSO_MYC-MYC-r1_peaks.narrowPeak \\nINFO  @ Mon, 20 Apr 2020 18:17:41: #4 Write summits bed file... ../../data/diffBinding_rep1/diffData_unscaled/mp784-MV411_IRF2BP2_DMSO_MYC-MYC-r1_summits.bed \\nINFO  @ Mon, 20 Apr 2020 18:17:41: Done! \\n'\n",
      "b'INFO  @ Mon, 20 Apr 2020 18:17:41: \\n# Command line: callpeak -B -t ../../data/diffBinding_rep1/mp792-MV411_IRF2BP2_VHL_MYC-MYC-r1.mLb.clN.sorted.bam -c ../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 222 -n mp792-MV411_IRF2BP2_VHL_MYC-MYC-r1 --outdir ../../data/diffBinding_rep1/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp792-MV411_IRF2BP2_VHL_MYC-MYC-r1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_rep1/mp792-MV411_IRF2BP2_VHL_MYC-MYC-r1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Mon, 20 Apr 2020 18:17:41: #1 read tag files... \\nINFO  @ Mon, 20 Apr 2020 18:17:41: #1 read treatment tags... \\nINFO  @ Mon, 20 Apr 2020 18:17:44:  1000000 \\nINFO  @ Mon, 20 Apr 2020 18:17:46:  2000000 \\nINFO  @ Mon, 20 Apr 2020 18:17:48:  3000000 \\nINFO  @ Mon, 20 Apr 2020 18:17:51:  4000000 \\nINFO  @ Mon, 20 Apr 2020 18:17:54:  5000000 \\nINFO  @ Mon, 20 Apr 2020 18:17:56:  6000000 \\nINFO  @ Mon, 20 Apr 2020 18:17:58:  7000000 \\nINFO  @ Mon, 20 Apr 2020 18:18:01:  8000000 \\nINFO  @ Mon, 20 Apr 2020 18:18:03:  9000000 \\nINFO  @ Mon, 20 Apr 2020 18:18:06:  10000000 \\nINFO  @ Mon, 20 Apr 2020 18:18:08:  11000000 \\nINFO  @ Mon, 20 Apr 2020 18:18:10:  12000000 \\nINFO  @ Mon, 20 Apr 2020 18:18:13:  13000000 \\nINFO  @ Mon, 20 Apr 2020 18:18:15:  14000000 \\nINFO  @ Mon, 20 Apr 2020 18:18:18:  15000000 \\nINFO  @ Mon, 20 Apr 2020 18:18:20:  16000000 \\nINFO  @ Mon, 20 Apr 2020 18:18:22:  17000000 \\nINFO  @ Mon, 20 Apr 2020 18:18:24: 17720683 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 18:18:24: #1.2 read input tags... \\nINFO  @ Mon, 20 Apr 2020 18:18:27:  1000000 \\nINFO  @ Mon, 20 Apr 2020 18:18:29:  2000000 \\nINFO  @ Mon, 20 Apr 2020 18:18:32:  3000000 \\nINFO  @ Mon, 20 Apr 2020 18:18:34:  4000000 \\nINFO  @ Mon, 20 Apr 2020 18:18:36:  5000000 \\nINFO  @ Mon, 20 Apr 2020 18:18:39:  6000000 \\nINFO  @ Mon, 20 Apr 2020 18:18:41:  7000000 \\nINFO  @ Mon, 20 Apr 2020 18:18:43:  8000000 \\nINFO  @ Mon, 20 Apr 2020 18:18:46:  9000000 \\nINFO  @ Mon, 20 Apr 2020 18:18:48:  10000000 \\nINFO  @ Mon, 20 Apr 2020 18:18:51:  11000000 \\nINFO  @ Mon, 20 Apr 2020 18:18:53:  12000000 \\nINFO  @ Mon, 20 Apr 2020 18:18:55:  13000000 \\nINFO  @ Mon, 20 Apr 2020 18:18:58:  14000000 \\nINFO  @ Mon, 20 Apr 2020 18:19:00:  15000000 \\nINFO  @ Mon, 20 Apr 2020 18:19:03:  16000000 \\nINFO  @ Mon, 20 Apr 2020 18:19:05:  17000000 \\nINFO  @ Mon, 20 Apr 2020 18:19:07:  18000000 \\nINFO  @ Mon, 20 Apr 2020 18:19:10:  19000000 \\nINFO  @ Mon, 20 Apr 2020 18:19:12:  20000000 \\nINFO  @ Mon, 20 Apr 2020 18:19:14:  21000000 \\nINFO  @ Mon, 20 Apr 2020 18:19:17:  22000000 \\nINFO  @ Mon, 20 Apr 2020 18:19:19:  23000000 \\nINFO  @ Mon, 20 Apr 2020 18:19:21:  24000000 \\nINFO  @ Mon, 20 Apr 2020 18:19:24:  25000000 \\nINFO  @ Mon, 20 Apr 2020 18:19:26:  26000000 \\nINFO  @ Mon, 20 Apr 2020 18:19:28:  27000000 \\nINFO  @ Mon, 20 Apr 2020 18:19:31:  28000000 \\nINFO  @ Mon, 20 Apr 2020 18:19:33:  29000000 \\nINFO  @ Mon, 20 Apr 2020 18:19:36:  30000000 \\nINFO  @ Mon, 20 Apr 2020 18:19:38:  31000000 \\nINFO  @ Mon, 20 Apr 2020 18:19:40:  32000000 \\nINFO  @ Mon, 20 Apr 2020 18:19:43:  33000000 \\nINFO  @ Mon, 20 Apr 2020 18:19:45:  34000000 \\nINFO  @ Mon, 20 Apr 2020 18:19:47: 34520959 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 18:19:47: #1 tag size is determined as 49 bps \\nINFO  @ Mon, 20 Apr 2020 18:19:47: #1 tag size = 49.0 \\nINFO  @ Mon, 20 Apr 2020 18:19:47: #1  total tags in treatment: 17720683 \\nINFO  @ Mon, 20 Apr 2020 18:19:47: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 18:19:47: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 18:19:47: #1  tags after filtering in treatment: 17711846 \\nINFO  @ Mon, 20 Apr 2020 18:19:47: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 20 Apr 2020 18:19:47: #1  total tags in control: 34520959 \\nINFO  @ Mon, 20 Apr 2020 18:19:47: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 18:19:47: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 18:19:47: #1  tags after filtering in control: 34516729 \\nINFO  @ Mon, 20 Apr 2020 18:19:47: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 20 Apr 2020 18:19:47: #1 finished! \\nINFO  @ Mon, 20 Apr 2020 18:19:47: #2 Build Peak Model... \\nINFO  @ Mon, 20 Apr 2020 18:19:47: #2 Skipped... \\nINFO  @ Mon, 20 Apr 2020 18:19:47: #2 Use 222 as fragment length \\nINFO  @ Mon, 20 Apr 2020 18:19:47: #3 Call peaks... \\nINFO  @ Mon, 20 Apr 2020 18:19:47: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 20 Apr 2020 18:22:38: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 20 Apr 2020 18:22:38: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp792-MV411_IRF2BP2_VHL_MYC-MYC-r1_treat_pileup.bdg \\nINFO  @ Mon, 20 Apr 2020 18:22:38: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp792-MV411_IRF2BP2_VHL_MYC-MYC-r1_control_lambda.bdg \\nINFO  @ Mon, 20 Apr 2020 18:22:38: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 20 Apr 2020 18:22:38: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 20 Apr 2020 18:25:10: #4 Write output xls file... ../../data/diffBinding_rep1/diffData_unscaled/mp792-MV411_IRF2BP2_VHL_MYC-MYC-r1_peaks.xls \\nINFO  @ Mon, 20 Apr 2020 18:25:11: #4 Write peak in narrowPeak format file... ../../data/diffBinding_rep1/diffData_unscaled/mp792-MV411_IRF2BP2_VHL_MYC-MYC-r1_peaks.narrowPeak \\nINFO  @ Mon, 20 Apr 2020 18:25:11: #4 Write summits bed file... ../../data/diffBinding_rep1/diffData_unscaled/mp792-MV411_IRF2BP2_VHL_MYC-MYC-r1_summits.bed \\nINFO  @ Mon, 20 Apr 2020 18:25:11: Done! \\n'\n",
      "17215930 17711846\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "doing diff from ../../data/diffBinding_rep1/mp785-MV411_IRF2BP2_DMSO_MYB-MYB-r1.mLb.clN.sorted.bam and ../../data/diffBinding_rep1/mp793-MV411_IRF2BP2_VHL_MYB-MYB-r1.mLb.clN.sorted.bam\n",
      "using default size\n",
      "computing the scaling values\n",
      "b'INFO  @ Mon, 20 Apr 2020 18:32:36: \\n# Command line: callpeak -B -t ../../data/diffBinding_rep1/mp785-MV411_IRF2BP2_DMSO_MYB-MYB-r1.mLb.clN.sorted.bam -c ../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 228 -n mp785-MV411_IRF2BP2_DMSO_MYB-MYB-r1 --outdir ../../data/diffBinding_rep1/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp785-MV411_IRF2BP2_DMSO_MYB-MYB-r1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_rep1/mp785-MV411_IRF2BP2_DMSO_MYB-MYB-r1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Mon, 20 Apr 2020 18:32:36: #1 read tag files... \\nINFO  @ Mon, 20 Apr 2020 18:32:36: #1 read treatment tags... \\nINFO  @ Mon, 20 Apr 2020 18:32:39:  1000000 \\nINFO  @ Mon, 20 Apr 2020 18:32:41:  2000000 \\nINFO  @ Mon, 20 Apr 2020 18:32:43:  3000000 \\nINFO  @ Mon, 20 Apr 2020 18:32:46:  4000000 \\nINFO  @ Mon, 20 Apr 2020 18:32:48:  5000000 \\nINFO  @ Mon, 20 Apr 2020 18:32:51:  6000000 \\nINFO  @ Mon, 20 Apr 2020 18:32:53:  7000000 \\nINFO  @ Mon, 20 Apr 2020 18:32:56:  8000000 \\nINFO  @ Mon, 20 Apr 2020 18:32:58:  9000000 \\nINFO  @ Mon, 20 Apr 2020 18:33:00:  10000000 \\nINFO  @ Mon, 20 Apr 2020 18:33:02:  11000000 \\nINFO  @ Mon, 20 Apr 2020 18:33:05:  12000000 \\nINFO  @ Mon, 20 Apr 2020 18:33:07:  13000000 \\nINFO  @ Mon, 20 Apr 2020 18:33:10:  14000000 \\nINFO  @ Mon, 20 Apr 2020 18:33:12:  15000000 \\nINFO  @ Mon, 20 Apr 2020 18:33:15:  16000000 \\nINFO  @ Mon, 20 Apr 2020 18:33:17:  17000000 \\nINFO  @ Mon, 20 Apr 2020 18:33:19:  18000000 \\nINFO  @ Mon, 20 Apr 2020 18:33:22:  19000000 \\nINFO  @ Mon, 20 Apr 2020 18:33:24:  20000000 \\nINFO  @ Mon, 20 Apr 2020 18:33:24: 20009851 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 18:33:24: #1.2 read input tags... \\nINFO  @ Mon, 20 Apr 2020 18:33:27:  1000000 \\nINFO  @ Mon, 20 Apr 2020 18:33:29:  2000000 \\nINFO  @ Mon, 20 Apr 2020 18:33:31:  3000000 \\nINFO  @ Mon, 20 Apr 2020 18:33:34:  4000000 \\nINFO  @ Mon, 20 Apr 2020 18:33:36:  5000000 \\nINFO  @ Mon, 20 Apr 2020 18:33:38:  6000000 \\nINFO  @ Mon, 20 Apr 2020 18:33:41:  7000000 \\nINFO  @ Mon, 20 Apr 2020 18:33:43:  8000000 \\nINFO  @ Mon, 20 Apr 2020 18:33:45:  9000000 \\nINFO  @ Mon, 20 Apr 2020 18:33:48:  10000000 \\nINFO  @ Mon, 20 Apr 2020 18:33:50:  11000000 \\nINFO  @ Mon, 20 Apr 2020 18:33:52:  12000000 \\nINFO  @ Mon, 20 Apr 2020 18:33:55:  13000000 \\nINFO  @ Mon, 20 Apr 2020 18:33:57:  14000000 \\nINFO  @ Mon, 20 Apr 2020 18:34:00:  15000000 \\nINFO  @ Mon, 20 Apr 2020 18:34:02:  16000000 \\nINFO  @ Mon, 20 Apr 2020 18:34:04:  17000000 \\nINFO  @ Mon, 20 Apr 2020 18:34:06:  18000000 \\nINFO  @ Mon, 20 Apr 2020 18:34:09:  19000000 \\nINFO  @ Mon, 20 Apr 2020 18:34:11:  20000000 \\nINFO  @ Mon, 20 Apr 2020 18:34:14:  21000000 \\nINFO  @ Mon, 20 Apr 2020 18:34:16:  22000000 \\nINFO  @ Mon, 20 Apr 2020 18:34:18:  23000000 \\nINFO  @ Mon, 20 Apr 2020 18:34:21:  24000000 \\nINFO  @ Mon, 20 Apr 2020 18:34:23:  25000000 \\nINFO  @ Mon, 20 Apr 2020 18:34:25:  26000000 \\nINFO  @ Mon, 20 Apr 2020 18:34:28:  27000000 \\nINFO  @ Mon, 20 Apr 2020 18:34:30:  28000000 \\nINFO  @ Mon, 20 Apr 2020 18:34:32:  29000000 \\nINFO  @ Mon, 20 Apr 2020 18:34:35:  30000000 \\nINFO  @ Mon, 20 Apr 2020 18:34:37:  31000000 \\nINFO  @ Mon, 20 Apr 2020 18:34:39:  32000000 \\nINFO  @ Mon, 20 Apr 2020 18:34:42:  33000000 \\nINFO  @ Mon, 20 Apr 2020 18:34:44:  34000000 \\nINFO  @ Mon, 20 Apr 2020 18:34:46: 34520959 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 18:34:46: #1 tag size is determined as 49 bps \\nINFO  @ Mon, 20 Apr 2020 18:34:46: #1 tag size = 49.0 \\nINFO  @ Mon, 20 Apr 2020 18:34:46: #1  total tags in treatment: 20009851 \\nINFO  @ Mon, 20 Apr 2020 18:34:46: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 18:34:46: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 18:34:46: #1  tags after filtering in treatment: 19999482 \\nINFO  @ Mon, 20 Apr 2020 18:34:46: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 20 Apr 2020 18:34:46: #1  total tags in control: 34520959 \\nINFO  @ Mon, 20 Apr 2020 18:34:46: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 18:34:46: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 18:34:47: #1  tags after filtering in control: 34516729 \\nINFO  @ Mon, 20 Apr 2020 18:34:47: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 20 Apr 2020 18:34:47: #1 finished! \\nINFO  @ Mon, 20 Apr 2020 18:34:47: #2 Build Peak Model... \\nINFO  @ Mon, 20 Apr 2020 18:34:47: #2 Skipped... \\nINFO  @ Mon, 20 Apr 2020 18:34:47: #2 Use 228 as fragment length \\nINFO  @ Mon, 20 Apr 2020 18:34:47: #3 Call peaks... \\nINFO  @ Mon, 20 Apr 2020 18:34:47: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 20 Apr 2020 18:37:38: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 20 Apr 2020 18:37:38: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp785-MV411_IRF2BP2_DMSO_MYB-MYB-r1_treat_pileup.bdg \\nINFO  @ Mon, 20 Apr 2020 18:37:38: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp785-MV411_IRF2BP2_DMSO_MYB-MYB-r1_control_lambda.bdg \\nINFO  @ Mon, 20 Apr 2020 18:37:38: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 20 Apr 2020 18:37:38: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 20 Apr 2020 18:40:12: #4 Write output xls file... ../../data/diffBinding_rep1/diffData_unscaled/mp785-MV411_IRF2BP2_DMSO_MYB-MYB-r1_peaks.xls \\nINFO  @ Mon, 20 Apr 2020 18:40:12: #4 Write peak in narrowPeak format file... ../../data/diffBinding_rep1/diffData_unscaled/mp785-MV411_IRF2BP2_DMSO_MYB-MYB-r1_peaks.narrowPeak \\nINFO  @ Mon, 20 Apr 2020 18:40:13: #4 Write summits bed file... ../../data/diffBinding_rep1/diffData_unscaled/mp785-MV411_IRF2BP2_DMSO_MYB-MYB-r1_summits.bed \\nINFO  @ Mon, 20 Apr 2020 18:40:13: Done! \\n'\n",
      "b'INFO  @ Mon, 20 Apr 2020 18:40:13: \\n# Command line: callpeak -B -t ../../data/diffBinding_rep1/mp793-MV411_IRF2BP2_VHL_MYB-MYB-r1.mLb.clN.sorted.bam -c ../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 228 -n mp793-MV411_IRF2BP2_VHL_MYB-MYB-r1 --outdir ../../data/diffBinding_rep1/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp793-MV411_IRF2BP2_VHL_MYB-MYB-r1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_rep1/mp793-MV411_IRF2BP2_VHL_MYB-MYB-r1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Mon, 20 Apr 2020 18:40:13: #1 read tag files... \\nINFO  @ Mon, 20 Apr 2020 18:40:13: #1 read treatment tags... \\nINFO  @ Mon, 20 Apr 2020 18:40:16:  1000000 \\nINFO  @ Mon, 20 Apr 2020 18:40:18:  2000000 \\nINFO  @ Mon, 20 Apr 2020 18:40:20:  3000000 \\nINFO  @ Mon, 20 Apr 2020 18:40:23:  4000000 \\nINFO  @ Mon, 20 Apr 2020 18:40:25:  5000000 \\nINFO  @ Mon, 20 Apr 2020 18:40:28:  6000000 \\nINFO  @ Mon, 20 Apr 2020 18:40:30:  7000000 \\nINFO  @ Mon, 20 Apr 2020 18:40:32:  8000000 \\nINFO  @ Mon, 20 Apr 2020 18:40:35:  9000000 \\nINFO  @ Mon, 20 Apr 2020 18:40:37:  10000000 \\nINFO  @ Mon, 20 Apr 2020 18:40:39:  11000000 \\nINFO  @ Mon, 20 Apr 2020 18:40:42:  12000000 \\nINFO  @ Mon, 20 Apr 2020 18:40:44:  13000000 \\nINFO  @ Mon, 20 Apr 2020 18:40:46:  14000000 \\nINFO  @ Mon, 20 Apr 2020 18:40:49:  15000000 \\nINFO  @ Mon, 20 Apr 2020 18:40:51:  16000000 \\nINFO  @ Mon, 20 Apr 2020 18:40:54:  17000000 \\nINFO  @ Mon, 20 Apr 2020 18:40:56:  18000000 \\nINFO  @ Mon, 20 Apr 2020 18:40:59:  19000000 \\nINFO  @ Mon, 20 Apr 2020 18:41:01:  20000000 \\nINFO  @ Mon, 20 Apr 2020 18:41:03: 20701696 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 18:41:03: #1.2 read input tags... \\nINFO  @ Mon, 20 Apr 2020 18:41:06:  1000000 \\nINFO  @ Mon, 20 Apr 2020 18:41:08:  2000000 \\nINFO  @ Mon, 20 Apr 2020 18:41:11:  3000000 \\nINFO  @ Mon, 20 Apr 2020 18:41:13:  4000000 \\nINFO  @ Mon, 20 Apr 2020 18:41:15:  5000000 \\nINFO  @ Mon, 20 Apr 2020 18:41:18:  6000000 \\nINFO  @ Mon, 20 Apr 2020 18:41:20:  7000000 \\nINFO  @ Mon, 20 Apr 2020 18:41:22:  8000000 \\nINFO  @ Mon, 20 Apr 2020 18:41:25:  9000000 \\nINFO  @ Mon, 20 Apr 2020 18:41:27:  10000000 \\nINFO  @ Mon, 20 Apr 2020 18:41:30:  11000000 \\nINFO  @ Mon, 20 Apr 2020 18:41:32:  12000000 \\nINFO  @ Mon, 20 Apr 2020 18:41:34:  13000000 \\nINFO  @ Mon, 20 Apr 2020 18:41:37:  14000000 \\nINFO  @ Mon, 20 Apr 2020 18:41:39:  15000000 \\nINFO  @ Mon, 20 Apr 2020 18:41:41:  16000000 \\nINFO  @ Mon, 20 Apr 2020 18:41:44:  17000000 \\nINFO  @ Mon, 20 Apr 2020 18:41:46:  18000000 \\nINFO  @ Mon, 20 Apr 2020 18:41:49:  19000000 \\nINFO  @ Mon, 20 Apr 2020 18:41:51:  20000000 \\nINFO  @ Mon, 20 Apr 2020 18:41:53:  21000000 \\nINFO  @ Mon, 20 Apr 2020 18:41:56:  22000000 \\nINFO  @ Mon, 20 Apr 2020 18:41:58:  23000000 \\nINFO  @ Mon, 20 Apr 2020 18:42:00:  24000000 \\nINFO  @ Mon, 20 Apr 2020 18:42:03:  25000000 \\nINFO  @ Mon, 20 Apr 2020 18:42:05:  26000000 \\nINFO  @ Mon, 20 Apr 2020 18:42:08:  27000000 \\nINFO  @ Mon, 20 Apr 2020 18:42:10:  28000000 \\nINFO  @ Mon, 20 Apr 2020 18:42:12:  29000000 \\nINFO  @ Mon, 20 Apr 2020 18:42:15:  30000000 \\nINFO  @ Mon, 20 Apr 2020 18:42:17:  31000000 \\nINFO  @ Mon, 20 Apr 2020 18:42:19:  32000000 \\nINFO  @ Mon, 20 Apr 2020 18:42:22:  33000000 \\nINFO  @ Mon, 20 Apr 2020 18:42:24:  34000000 \\nINFO  @ Mon, 20 Apr 2020 18:42:26: 34520959 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 18:42:26: #1 tag size is determined as 48 bps \\nINFO  @ Mon, 20 Apr 2020 18:42:26: #1 tag size = 48.0 \\nINFO  @ Mon, 20 Apr 2020 18:42:26: #1  total tags in treatment: 20701696 \\nINFO  @ Mon, 20 Apr 2020 18:42:26: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 18:42:26: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 18:42:26: #1  tags after filtering in treatment: 20689943 \\nINFO  @ Mon, 20 Apr 2020 18:42:26: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 20 Apr 2020 18:42:26: #1  total tags in control: 34520959 \\nINFO  @ Mon, 20 Apr 2020 18:42:26: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 18:42:26: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 18:42:26: #1  tags after filtering in control: 34516729 \\nINFO  @ Mon, 20 Apr 2020 18:42:26: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 20 Apr 2020 18:42:26: #1 finished! \\nINFO  @ Mon, 20 Apr 2020 18:42:26: #2 Build Peak Model... \\nINFO  @ Mon, 20 Apr 2020 18:42:26: #2 Skipped... \\nINFO  @ Mon, 20 Apr 2020 18:42:26: #2 Use 228 as fragment length \\nINFO  @ Mon, 20 Apr 2020 18:42:26: #3 Call peaks... \\nINFO  @ Mon, 20 Apr 2020 18:42:26: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 20 Apr 2020 18:45:25: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 20 Apr 2020 18:45:25: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp793-MV411_IRF2BP2_VHL_MYB-MYB-r1_treat_pileup.bdg \\nINFO  @ Mon, 20 Apr 2020 18:45:25: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp793-MV411_IRF2BP2_VHL_MYB-MYB-r1_control_lambda.bdg \\nINFO  @ Mon, 20 Apr 2020 18:45:25: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 20 Apr 2020 18:45:25: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 20 Apr 2020 18:47:59: #4 Write output xls file... ../../data/diffBinding_rep1/diffData_unscaled/mp793-MV411_IRF2BP2_VHL_MYB-MYB-r1_peaks.xls \\nINFO  @ Mon, 20 Apr 2020 18:48:00: #4 Write peak in narrowPeak format file... ../../data/diffBinding_rep1/diffData_unscaled/mp793-MV411_IRF2BP2_VHL_MYB-MYB-r1_peaks.narrowPeak \\nINFO  @ Mon, 20 Apr 2020 18:48:00: #4 Write summits bed file... ../../data/diffBinding_rep1/diffData_unscaled/mp793-MV411_IRF2BP2_VHL_MYB-MYB-r1_summits.bed \\nINFO  @ Mon, 20 Apr 2020 18:48:00: Done! \\n'\n",
      "19999482 20689943\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "doing diff from ../../data/diffBinding_rep1/mp786-MV411_IRF2BP2_DMSO_SPI1-SPI1-r1.mLb.clN.sorted.bam and ../../data/diffBinding_rep1/mp794-MV411_IRF2BP2_VHL_SPI1-SPI1-r1.mLb.clN.sorted.bam\n",
      "using default size\n",
      "computing the scaling values\n",
      "b'INFO  @ Mon, 20 Apr 2020 18:55:42: \\n# Command line: callpeak -B -t ../../data/diffBinding_rep1/mp786-MV411_IRF2BP2_DMSO_SPI1-SPI1-r1.mLb.clN.sorted.bam -c ../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 194 -n mp786-MV411_IRF2BP2_DMSO_SPI1-SPI1-r1 --outdir ../../data/diffBinding_rep1/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp786-MV411_IRF2BP2_DMSO_SPI1-SPI1-r1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_rep1/mp786-MV411_IRF2BP2_DMSO_SPI1-SPI1-r1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Mon, 20 Apr 2020 18:55:42: #1 read tag files... \\nINFO  @ Mon, 20 Apr 2020 18:55:42: #1 read treatment tags... \\nINFO  @ Mon, 20 Apr 2020 18:55:45:  1000000 \\nINFO  @ Mon, 20 Apr 2020 18:55:47:  2000000 \\nINFO  @ Mon, 20 Apr 2020 18:55:49:  3000000 \\nINFO  @ Mon, 20 Apr 2020 18:55:52:  4000000 \\nINFO  @ Mon, 20 Apr 2020 18:55:54:  5000000 \\nINFO  @ Mon, 20 Apr 2020 18:55:56:  6000000 \\nINFO  @ Mon, 20 Apr 2020 18:55:58:  7000000 \\nINFO  @ Mon, 20 Apr 2020 18:56:01:  8000000 \\nINFO  @ Mon, 20 Apr 2020 18:56:03:  9000000 \\nINFO  @ Mon, 20 Apr 2020 18:56:06:  10000000 \\nINFO  @ Mon, 20 Apr 2020 18:56:08:  11000000 \\nINFO  @ Mon, 20 Apr 2020 18:56:10:  12000000 \\nINFO  @ Mon, 20 Apr 2020 18:56:13:  13000000 \\nINFO  @ Mon, 20 Apr 2020 18:56:15:  14000000 \\nINFO  @ Mon, 20 Apr 2020 18:56:17:  15000000 \\nINFO  @ Mon, 20 Apr 2020 18:56:20:  16000000 \\nINFO  @ Mon, 20 Apr 2020 18:56:22:  17000000 \\nINFO  @ Mon, 20 Apr 2020 18:56:24:  18000000 \\nINFO  @ Mon, 20 Apr 2020 18:56:27:  19000000 \\nINFO  @ Mon, 20 Apr 2020 18:56:29:  20000000 \\nINFO  @ Mon, 20 Apr 2020 18:56:31:  21000000 \\nINFO  @ Mon, 20 Apr 2020 18:56:34:  22000000 \\nINFO  @ Mon, 20 Apr 2020 18:56:36:  23000000 \\nINFO  @ Mon, 20 Apr 2020 18:56:39:  24000000 \\nINFO  @ Mon, 20 Apr 2020 18:56:41:  25000000 \\nINFO  @ Mon, 20 Apr 2020 18:56:44:  26000000 \\nINFO  @ Mon, 20 Apr 2020 18:56:46:  27000000 \\nINFO  @ Mon, 20 Apr 2020 18:56:48:  28000000 \\nINFO  @ Mon, 20 Apr 2020 18:56:50:  29000000 \\nINFO  @ Mon, 20 Apr 2020 18:56:53:  30000000 \\nINFO  @ Mon, 20 Apr 2020 18:56:55:  31000000 \\nINFO  @ Mon, 20 Apr 2020 18:56:56: 31548814 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 18:56:57: #1.2 read input tags... \\nINFO  @ Mon, 20 Apr 2020 18:56:59:  1000000 \\nINFO  @ Mon, 20 Apr 2020 18:57:01:  2000000 \\nINFO  @ Mon, 20 Apr 2020 18:57:04:  3000000 \\nINFO  @ Mon, 20 Apr 2020 18:57:06:  4000000 \\nINFO  @ Mon, 20 Apr 2020 18:57:08:  5000000 \\nINFO  @ Mon, 20 Apr 2020 18:57:11:  6000000 \\nINFO  @ Mon, 20 Apr 2020 18:57:13:  7000000 \\nINFO  @ Mon, 20 Apr 2020 18:57:15:  8000000 \\nINFO  @ Mon, 20 Apr 2020 18:57:18:  9000000 \\nINFO  @ Mon, 20 Apr 2020 18:57:20:  10000000 \\nINFO  @ Mon, 20 Apr 2020 18:57:23:  11000000 \\nINFO  @ Mon, 20 Apr 2020 18:57:25:  12000000 \\nINFO  @ Mon, 20 Apr 2020 18:57:27:  13000000 \\nINFO  @ Mon, 20 Apr 2020 18:57:30:  14000000 \\nINFO  @ Mon, 20 Apr 2020 18:57:32:  15000000 \\nINFO  @ Mon, 20 Apr 2020 18:57:34:  16000000 \\nINFO  @ Mon, 20 Apr 2020 18:57:36:  17000000 \\nINFO  @ Mon, 20 Apr 2020 18:57:39:  18000000 \\nINFO  @ Mon, 20 Apr 2020 18:57:41:  19000000 \\nINFO  @ Mon, 20 Apr 2020 18:57:43:  20000000 \\nINFO  @ Mon, 20 Apr 2020 18:57:46:  21000000 \\nINFO  @ Mon, 20 Apr 2020 18:57:48:  22000000 \\nINFO  @ Mon, 20 Apr 2020 18:57:50:  23000000 \\nINFO  @ Mon, 20 Apr 2020 18:57:53:  24000000 \\nINFO  @ Mon, 20 Apr 2020 18:57:55:  25000000 \\nINFO  @ Mon, 20 Apr 2020 18:57:57:  26000000 \\nINFO  @ Mon, 20 Apr 2020 18:58:00:  27000000 \\nINFO  @ Mon, 20 Apr 2020 18:58:02:  28000000 \\nINFO  @ Mon, 20 Apr 2020 18:58:04:  29000000 \\nINFO  @ Mon, 20 Apr 2020 18:58:07:  30000000 \\nINFO  @ Mon, 20 Apr 2020 18:58:09:  31000000 \\nINFO  @ Mon, 20 Apr 2020 18:58:12:  32000000 \\nINFO  @ Mon, 20 Apr 2020 18:58:14:  33000000 \\nINFO  @ Mon, 20 Apr 2020 18:58:16:  34000000 \\nINFO  @ Mon, 20 Apr 2020 18:58:18: 34520959 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 18:58:18: #1 tag size is determined as 49 bps \\nINFO  @ Mon, 20 Apr 2020 18:58:18: #1 tag size = 49.0 \\nINFO  @ Mon, 20 Apr 2020 18:58:18: #1  total tags in treatment: 31548814 \\nINFO  @ Mon, 20 Apr 2020 18:58:18: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 18:58:18: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 18:58:18: #1  tags after filtering in treatment: 31522444 \\nINFO  @ Mon, 20 Apr 2020 18:58:18: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 20 Apr 2020 18:58:18: #1  total tags in control: 34520959 \\nINFO  @ Mon, 20 Apr 2020 18:58:18: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 18:58:18: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 18:58:18: #1  tags after filtering in control: 34516729 \\nINFO  @ Mon, 20 Apr 2020 18:58:18: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 20 Apr 2020 18:58:18: #1 finished! \\nINFO  @ Mon, 20 Apr 2020 18:58:18: #2 Build Peak Model... \\nINFO  @ Mon, 20 Apr 2020 18:58:18: #2 Skipped... \\nINFO  @ Mon, 20 Apr 2020 18:58:18: #2 Use 194 as fragment length \\nINFO  @ Mon, 20 Apr 2020 18:58:18: #3 Call peaks... \\nINFO  @ Mon, 20 Apr 2020 18:58:18: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 20 Apr 2020 19:01:46: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 20 Apr 2020 19:01:46: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp786-MV411_IRF2BP2_DMSO_SPI1-SPI1-r1_treat_pileup.bdg \\nINFO  @ Mon, 20 Apr 2020 19:01:46: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp786-MV411_IRF2BP2_DMSO_SPI1-SPI1-r1_control_lambda.bdg \\nINFO  @ Mon, 20 Apr 2020 19:01:46: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 20 Apr 2020 19:01:46: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 20 Apr 2020 19:04:45: #4 Write output xls file... ../../data/diffBinding_rep1/diffData_unscaled/mp786-MV411_IRF2BP2_DMSO_SPI1-SPI1-r1_peaks.xls \\nINFO  @ Mon, 20 Apr 2020 19:04:46: #4 Write peak in narrowPeak format file... ../../data/diffBinding_rep1/diffData_unscaled/mp786-MV411_IRF2BP2_DMSO_SPI1-SPI1-r1_peaks.narrowPeak \\nINFO  @ Mon, 20 Apr 2020 19:04:47: #4 Write summits bed file... ../../data/diffBinding_rep1/diffData_unscaled/mp786-MV411_IRF2BP2_DMSO_SPI1-SPI1-r1_summits.bed \\nINFO  @ Mon, 20 Apr 2020 19:04:47: Done! \\n'\n",
      "b'INFO  @ Mon, 20 Apr 2020 19:04:47: \\n# Command line: callpeak -B -t ../../data/diffBinding_rep1/mp794-MV411_IRF2BP2_VHL_SPI1-SPI1-r1.mLb.clN.sorted.bam -c ../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 194 -n mp794-MV411_IRF2BP2_VHL_SPI1-SPI1-r1 --outdir ../../data/diffBinding_rep1/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp794-MV411_IRF2BP2_VHL_SPI1-SPI1-r1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_rep1/mp794-MV411_IRF2BP2_VHL_SPI1-SPI1-r1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_rep1/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Mon, 20 Apr 2020 19:04:47: #1 read tag files... \\nINFO  @ Mon, 20 Apr 2020 19:04:47: #1 read treatment tags... \\nINFO  @ Mon, 20 Apr 2020 19:04:50:  1000000 \\nINFO  @ Mon, 20 Apr 2020 19:04:52:  2000000 \\nINFO  @ Mon, 20 Apr 2020 19:04:54:  3000000 \\nINFO  @ Mon, 20 Apr 2020 19:04:56:  4000000 \\nINFO  @ Mon, 20 Apr 2020 19:04:59:  5000000 \\nINFO  @ Mon, 20 Apr 2020 19:05:01:  6000000 \\nINFO  @ Mon, 20 Apr 2020 19:05:03:  7000000 \\nINFO  @ Mon, 20 Apr 2020 19:05:06:  8000000 \\nINFO  @ Mon, 20 Apr 2020 19:05:08:  9000000 \\nINFO  @ Mon, 20 Apr 2020 19:05:10:  10000000 \\nINFO  @ Mon, 20 Apr 2020 19:05:12:  11000000 \\nINFO  @ Mon, 20 Apr 2020 19:05:15:  12000000 \\nINFO  @ Mon, 20 Apr 2020 19:05:17:  13000000 \\nINFO  @ Mon, 20 Apr 2020 19:05:19:  14000000 \\nINFO  @ Mon, 20 Apr 2020 19:05:21:  15000000 \\nINFO  @ Mon, 20 Apr 2020 19:05:23:  16000000 \\nINFO  @ Mon, 20 Apr 2020 19:05:26:  17000000 \\nINFO  @ Mon, 20 Apr 2020 19:05:28:  18000000 \\nINFO  @ Mon, 20 Apr 2020 19:05:30:  19000000 \\nINFO  @ Mon, 20 Apr 2020 19:05:32:  20000000 \\nINFO  @ Mon, 20 Apr 2020 19:05:34:  21000000 \\nINFO  @ Mon, 20 Apr 2020 19:05:37:  22000000 \\nINFO  @ Mon, 20 Apr 2020 19:05:39:  23000000 \\nINFO  @ Mon, 20 Apr 2020 19:05:41:  24000000 \\nINFO  @ Mon, 20 Apr 2020 19:05:42: 24307454 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 19:05:42: #1.2 read input tags... \\nINFO  @ Mon, 20 Apr 2020 19:05:44:  1000000 \\nINFO  @ Mon, 20 Apr 2020 19:05:47:  2000000 \\nINFO  @ Mon, 20 Apr 2020 19:05:49:  3000000 \\nINFO  @ Mon, 20 Apr 2020 19:05:51:  4000000 \\nINFO  @ Mon, 20 Apr 2020 19:05:54:  5000000 \\nINFO  @ Mon, 20 Apr 2020 19:05:56:  6000000 \\nINFO  @ Mon, 20 Apr 2020 19:05:58:  7000000 \\nINFO  @ Mon, 20 Apr 2020 19:06:01:  8000000 \\nINFO  @ Mon, 20 Apr 2020 19:06:03:  9000000 \\nINFO  @ Mon, 20 Apr 2020 19:06:05:  10000000 \\nINFO  @ Mon, 20 Apr 2020 19:06:08:  11000000 \\nINFO  @ Mon, 20 Apr 2020 19:06:10:  12000000 \\nINFO  @ Mon, 20 Apr 2020 19:06:12:  13000000 \\nINFO  @ Mon, 20 Apr 2020 19:06:15:  14000000 \\nINFO  @ Mon, 20 Apr 2020 19:06:17:  15000000 \\nINFO  @ Mon, 20 Apr 2020 19:06:20:  16000000 \\nINFO  @ Mon, 20 Apr 2020 19:06:22:  17000000 \\nINFO  @ Mon, 20 Apr 2020 19:06:24:  18000000 \\nINFO  @ Mon, 20 Apr 2020 19:06:27:  19000000 \\nINFO  @ Mon, 20 Apr 2020 19:06:29:  20000000 \\nINFO  @ Mon, 20 Apr 2020 19:06:31:  21000000 \\nINFO  @ Mon, 20 Apr 2020 19:06:34:  22000000 \\nINFO  @ Mon, 20 Apr 2020 19:06:36:  23000000 \\nINFO  @ Mon, 20 Apr 2020 19:06:38:  24000000 \\nINFO  @ Mon, 20 Apr 2020 19:06:41:  25000000 \\nINFO  @ Mon, 20 Apr 2020 19:06:43:  26000000 \\nINFO  @ Mon, 20 Apr 2020 19:06:45:  27000000 \\nINFO  @ Mon, 20 Apr 2020 19:06:48:  28000000 \\nINFO  @ Mon, 20 Apr 2020 19:06:50:  29000000 \\nINFO  @ Mon, 20 Apr 2020 19:06:53:  30000000 \\nINFO  @ Mon, 20 Apr 2020 19:06:55:  31000000 \\nINFO  @ Mon, 20 Apr 2020 19:06:57:  32000000 \\nINFO  @ Mon, 20 Apr 2020 19:07:00:  33000000 \\nINFO  @ Mon, 20 Apr 2020 19:07:02:  34000000 \\nINFO  @ Mon, 20 Apr 2020 19:07:03: 34520959 reads have been read. \\nINFO  @ Mon, 20 Apr 2020 19:07:04: #1 tag size is determined as 49 bps \\nINFO  @ Mon, 20 Apr 2020 19:07:04: #1 tag size = 49.0 \\nINFO  @ Mon, 20 Apr 2020 19:07:04: #1  total tags in treatment: 24307454 \\nINFO  @ Mon, 20 Apr 2020 19:07:04: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 19:07:04: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 19:07:04: #1  tags after filtering in treatment: 24279763 \\nINFO  @ Mon, 20 Apr 2020 19:07:04: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Mon, 20 Apr 2020 19:07:04: #1  total tags in control: 34520959 \\nINFO  @ Mon, 20 Apr 2020 19:07:04: #1 user defined the maximum tags... \\nINFO  @ Mon, 20 Apr 2020 19:07:04: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Mon, 20 Apr 2020 19:07:04: #1  tags after filtering in control: 34516729 \\nINFO  @ Mon, 20 Apr 2020 19:07:04: #1  Redundant rate of control: 0.00 \\nINFO  @ Mon, 20 Apr 2020 19:07:04: #1 finished! \\nINFO  @ Mon, 20 Apr 2020 19:07:04: #2 Build Peak Model... \\nINFO  @ Mon, 20 Apr 2020 19:07:04: #2 Skipped... \\nINFO  @ Mon, 20 Apr 2020 19:07:04: #2 Use 194 as fragment length \\nINFO  @ Mon, 20 Apr 2020 19:07:04: #3 Call peaks... \\nINFO  @ Mon, 20 Apr 2020 19:07:04: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Mon, 20 Apr 2020 19:09:57: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Mon, 20 Apr 2020 19:09:57: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp794-MV411_IRF2BP2_VHL_SPI1-SPI1-r1_treat_pileup.bdg \\nINFO  @ Mon, 20 Apr 2020 19:09:57: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp794-MV411_IRF2BP2_VHL_SPI1-SPI1-r1_control_lambda.bdg \\nINFO  @ Mon, 20 Apr 2020 19:09:57: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Mon, 20 Apr 2020 19:09:57: #3 Call peaks for each chromosome... \\nINFO  @ Mon, 20 Apr 2020 19:12:34: #4 Write output xls file... ../../data/diffBinding_rep1/diffData_unscaled/mp794-MV411_IRF2BP2_VHL_SPI1-SPI1-r1_peaks.xls \\nINFO  @ Mon, 20 Apr 2020 19:12:35: #4 Write peak in narrowPeak format file... ../../data/diffBinding_rep1/diffData_unscaled/mp794-MV411_IRF2BP2_VHL_SPI1-SPI1-r1_peaks.narrowPeak \\nINFO  @ Mon, 20 Apr 2020 19:12:35: #4 Write summits bed file... ../../data/diffBinding_rep1/diffData_unscaled/mp794-MV411_IRF2BP2_VHL_SPI1-SPI1-r1_summits.bed \\nINFO  @ Mon, 20 Apr 2020 19:12:36: Done! \\n'\n",
      "31522444 24279763\n",
      "doing differential peak binding\n"
     ]
    }
   ],
   "source": [
    "# on unscalled data \n",
    "for i in range(int((len(bams)-1)/2)-1):\n",
    "    if i < 0:\n",
    "        continue\n",
    "    name1 = bams[1+i]\n",
    "    name2 = bams[9+i]\n",
    "    chip.fullDiffPeak(name1,name2, control1=bams[0], size=size[i], compute_size=False, directory = \"../../data/diffBinding_rep1/diffData_unscaled/\", res_directory = \"../../data/diffBinding_rep1/diffPeaks_unscaled/\",pairedend=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "!mkdir ../../data/bigwigrep1unscaled"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/bigwig/MV411_IRF2BP2_DMSO_R1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/bigwig/MV411_IRF2BP2_VHL_R1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/bigwig/mp779-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r1.mLb.clN.bigWig...\n",
      "\\ [4 files][  3.3 GiB/  3.3 GiB]  206.0 MiB/s                                   \n",
      "==> NOTE: You are performing a sequence of gsutil operations that may\n",
      "run significantly faster if you instead use gsutil -m cp ... Please\n",
      "see the -m section under \"gsutil help options\" for further information\n",
      "about when gsutil -m can be advantageous.\n",
      "\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/bigwig/mp780-MV411_IRF2BP2_DMSO_MED1-MED1-r1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/bigwig/mp781-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/bigwig/mp782-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/bigwig/mp783-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/bigwig/mp784-MV411_IRF2BP2_DMSO_MYC-MYC-r1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/bigwig/mp785-MV411_IRF2BP2_DMSO_MYB-MYB-r1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/bigwig/mp786-MV411_IRF2BP2_DMSO_SPI1-SPI1-r1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/bigwig/mp787-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/bigwig/mp788-MV411_IRF2BP2_VHL_MED1-MED1-r1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/bigwig/mp789-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/bigwig/mp790-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/bigwig/mp791-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/bigwig/mp792-MV411_IRF2BP2_VHL_MYC-MYC-r1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/bigwig/mp793-MV411_IRF2BP2_VHL_MYB-MYB-r1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/bigwig/mp794-MV411_IRF2BP2_VHL_SPI1-SPI1-r1.mLb.clN.bigWig...\n",
      "| [19 files][  6.5 GiB/  6.5 GiB]   93.7 MiB/s                                  \n",
      "Operation completed over 19 objects/6.5 GiB.                                     \n"
     ]
    }
   ],
   "source": [
    "! gsutil cp gs://amlproject/Chip/IRF2BP2_degraded_rep1/bwa/mergedLibrary/bigwig/*.bigWig ../../data/bigwigrep1unscaled/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 102,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['INPUT_R1.mLb.clN.bigWig',\n",
       " 'mp779-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r1.mLb.clN.bigWig',\n",
       " 'mp780-MV411_IRF2BP2_DMSO_MED1-MED1-r1.mLb.clN.bigWig',\n",
       " 'mp781-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r1.mLb.clN.bigWig',\n",
       " 'mp782-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r1.mLb.clN.bigWig',\n",
       " 'mp783-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r1.mLb.clN.bigWig',\n",
       " 'mp784-MV411_IRF2BP2_DMSO_MYC-MYC-r1.mLb.clN.bigWig',\n",
       " 'mp785-MV411_IRF2BP2_DMSO_MYB-MYB-r1.mLb.clN.bigWig',\n",
       " 'mp786-MV411_IRF2BP2_DMSO_SPI1-SPI1-r1.mLb.clN.bigWig',\n",
       " 'mp787-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r1.mLb.clN.bigWig',\n",
       " 'mp788-MV411_IRF2BP2_VHL_MED1-MED1-r1.mLb.clN.bigWig',\n",
       " 'mp789-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r1.mLb.clN.bigWig',\n",
       " 'mp790-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r1.mLb.clN.bigWig',\n",
       " 'mp791-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r1.mLb.clN.bigWig',\n",
       " 'mp792-MV411_IRF2BP2_VHL_MYC-MYC-r1.mLb.clN.bigWig',\n",
       " 'mp793-MV411_IRF2BP2_VHL_MYB-MYB-r1.mLb.clN.bigWig',\n",
       " 'mp794-MV411_IRF2BP2_VHL_SPI1-SPI1-r1.mLb.clN.bigWig',\n",
       " 'MV411_IRF2BP2_DMSO_R1.mLb.clN.bigWig',\n",
       " 'MV411_IRF2BP2_VHL_R1.mLb.clN.bigWig']"
      ]
     },
     "execution_count": 102,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bw = ! ls ../../data/bigwigrep1unscaled/\n",
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 103,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/diffBinding_rep1/diffPeaks_unscaled/mp779-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r1_vs_mp787-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r1_c3.0_cond2.bed',\n",
       " '../../data/diffBinding_rep1/diffPeaks_unscaled/mp780-MV411_IRF2BP2_DMSO_MED1-MED1-r1_vs_mp788-MV411_IRF2BP2_VHL_MED1-MED1-r1_c3.0_cond2.bed',\n",
       " '../../data/diffBinding_rep1/diffPeaks_unscaled/mp781-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r1_vs_mp789-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r1_c3.0_cond2.bed',\n",
       " '../../data/diffBinding_rep1/diffPeaks_unscaled/mp782-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r1_vs_mp790-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r1_c3.0_cond2.bed',\n",
       " '../../data/diffBinding_rep1/diffPeaks_unscaled/mp783-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r1_vs_mp791-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r1_c3.0_cond2.bed',\n",
       " '../../data/diffBinding_rep1/diffPeaks_unscaled/mp784-MV411_IRF2BP2_DMSO_MYC-MYC-r1_vs_mp792-MV411_IRF2BP2_VHL_MYC-MYC-r1_c3.0_cond2.bed',\n",
       " '../../data/diffBinding_rep1/diffPeaks_unscaled/mp785-MV411_IRF2BP2_DMSO_MYB-MYB-r1_vs_mp793-MV411_IRF2BP2_VHL_MYB-MYB-r1_c3.0_cond2.bed',\n",
       " '../../data/diffBinding_rep1/diffPeaks_unscaled/mp786-MV411_IRF2BP2_DMSO_SPI1-SPI1-r1_vs_mp794-MV411_IRF2BP2_VHL_SPI1-SPI1-r1_c3.0_cond2.bed']"
      ]
     },
     "execution_count": 103,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "commonpeak = ! ls ../../data/diffBinding_rep1/diffPeaks_unscaled/*common.bed\n",
    "cond1peak = ! ls ../../data/diffBinding_rep1/diffPeaks_unscaled/*cond1.bed\n",
    "cond2peak = ! ls ../../data/diffBinding_rep1/diffPeaks_unscaled/*cond2.bed\n",
    "cond2peak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 104,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"FLAG_IRF2BP2\",\"MED1\",\"POLII_total\",\"POLII_S2\",\"POLII_S5\",\"MYC\",\"MYB\",\"SPI1\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 106,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "found 0 header line... removing\n",
      "found 0 header line... removing\n",
      "found 0 header line... removing\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep1unscaled/mp779-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r1.mLb.clN.bigWig ../../data/bigwigrep1unscaled/mp787-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/diffBinding_rep1/diffPeaks_unscaled/mp779-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r1_vs_mp787-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r1_c3.0_cond1.bed ../../data/diffBinding_rep1/diffPeaks_unscaled/mp779-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r1_vs_mp787-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r1_c3.0_common.bed ../../data/diffBinding_rep1/diffPeaks_unscaled/mp779-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r1_vs_mp787-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks_unscaled/FLAG_IRF2BP2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_rep1/diffPeaks_unscaled/FLAG_IRF2BP2_mat.gz --outFileName ../../data/diffBinding_rep1/diffPeaks_unscaled/FLAG_IRF2BP2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle FLAG_IRF2BP2', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270374v1\\t      2656\\n         chrEBV\\t    171823\\nchrUn_KI270529v1\\t      1899\\nchrUn_KI270383v1\\t      1750\\nchrUn_KI270425v1\\t      1884\\nchrUn_KI270305v1\\t      1472\\nchrUn_KI270414v1\\t      2489\\nchrUn_KI270316v1\\t      1444\\nchrUn_KI270392v1\\t       971\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270364v1\\t      2855\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270386v1\\t      1788\\nchrUn_KI270312v1\\t       998\\nchrUn_KI270302v1\\t      2274\\nchrUn_KI270579v1\\t     31033\\nchrUn_KI270303v1\\t      1942\\nchrUn_KI270530v1\\t      2168\\nchrUn_KI270419v1\\t      1029\\n')\n",
      "empty bed\n",
      "found 0 header line... removing\n",
      "found 0 header line... removing\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep1unscaled/mp780-MV411_IRF2BP2_DMSO_MED1-MED1-r1.mLb.clN.bigWig ../../data/bigwigrep1unscaled/mp788-MV411_IRF2BP2_VHL_MED1-MED1-r1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/diffBinding_rep1/diffPeaks_unscaled/mp780-MV411_IRF2BP2_DMSO_MED1-MED1-r1_vs_mp788-MV411_IRF2BP2_VHL_MED1-MED1-r1_c3.0_common.bed ../../data/diffBinding_rep1/diffPeaks_unscaled/mp780-MV411_IRF2BP2_DMSO_MED1-MED1-r1_vs_mp788-MV411_IRF2BP2_VHL_MED1-MED1-r1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks_unscaled/MED1_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_rep1/diffPeaks_unscaled/MED1_mat.gz --outFileName ../../data/diffBinding_rep1/diffPeaks_unscaled/MED1_mat.pdf --refPointLabel center --regionsLabel common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle MED1', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270384v1\\t      1658\\nchrUn_KI270392v1\\t       971\\nchrUn_KI270529v1\\t      1899\\nchrUn_KI270420v1\\t      2321\\nchrUn_KI270315v1\\t      2276\\nchrUn_GL000226v1\\t     15008\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270539v1\\t       993\\nchrUn_KI270303v1\\t      1942\\nchrUn_KI270579v1\\t     31033\\nchrUn_KI270530v1\\t      2168\\nchrUn_KI270385v1\\t       990\\nchrUn_KI270418v1\\t      2145\\nchrUn_KI270316v1\\t      1444\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270395v1\\t      1143\\n')\n",
      "found 0 header line... removing\n",
      "found 0 header line... removing\n",
      "found 0 header line... removing\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep1unscaled/mp781-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r1.mLb.clN.bigWig ../../data/bigwigrep1unscaled/mp789-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/diffBinding_rep1/diffPeaks_unscaled/mp781-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r1_vs_mp789-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r1_c3.0_cond1.bed ../../data/diffBinding_rep1/diffPeaks_unscaled/mp781-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r1_vs_mp789-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r1_c3.0_common.bed ../../data/diffBinding_rep1/diffPeaks_unscaled/mp781-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r1_vs_mp789-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks_unscaled/POLII_total_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_rep1/diffPeaks_unscaled/POLII_total_mat.gz --outFileName ../../data/diffBinding_rep1/diffPeaks_unscaled/POLII_total_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle POLII_total', returncode=0, stdout=b'', stderr=b\"\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270419v1\\t      1029\\nchrUn_KI270579v1\\t     31033\\nchrUn_KI270422v1\\t      1445\\nchrUn_KI270392v1\\t       971\\nchrUn_KI270384v1\\t      1658\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270316v1\\t      1444\\nchrUn_KI270418v1\\t      2145\\nchrUn_KI270530v1\\t      2168\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270414v1\\t      2489\\nWARNING: Group 'mp781-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r1_vs_mp789-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r1_c3.0_cond1.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n",
      "found 0 header line... removing\n",
      "found 0 header line... removing\n",
      "found 0 header line... removing\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep1unscaled/mp782-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r1.mLb.clN.bigWig ../../data/bigwigrep1unscaled/mp790-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/diffBinding_rep1/diffPeaks_unscaled/mp782-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r1_vs_mp790-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r1_c3.0_cond1.bed ../../data/diffBinding_rep1/diffPeaks_unscaled/mp782-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r1_vs_mp790-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r1_c3.0_common.bed ../../data/diffBinding_rep1/diffPeaks_unscaled/mp782-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r1_vs_mp790-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks_unscaled/POLII_S2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_rep1/diffPeaks_unscaled/POLII_S2_mat.gz --outFileName ../../data/diffBinding_rep1/diffPeaks_unscaled/POLII_S2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle POLII_S2', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270579v1\\t     31033\\nchrUn_KI270539v1\\t       993\\nchrUn_KI270422v1\\t      1445\\nchrUn_KI270302v1\\t      2274\\nchrUn_KI270364v1\\t      2855\\nchrUn_KI270391v1\\t      1484\\nchrUn_KI270411v1\\t      2646\\nchrUn_KI270395v1\\t      1143\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270385v1\\t       990\\nchrUn_KI270305v1\\t      1472\\nchrUn_KI270419v1\\t      1029\\nchrUn_KI270529v1\\t      1899\\nchrUn_KI270530v1\\t      2168\\nchrUn_KI270363v1\\t      1803\\nchrUn_KI270392v1\\t       971\\nchrUn_KI270390v1\\t      2387\\nchrUn_KI270414v1\\t      2489\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270303v1\\t      1942\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270371v1\\t      2805\\n')\n",
      "found 0 header line... removing\n",
      "found 0 header line... removing\n",
      "found 0 header line... removing\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep1unscaled/mp783-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r1.mLb.clN.bigWig ../../data/bigwigrep1unscaled/mp791-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/diffBinding_rep1/diffPeaks_unscaled/mp783-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r1_vs_mp791-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r1_c3.0_cond1.bed ../../data/diffBinding_rep1/diffPeaks_unscaled/mp783-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r1_vs_mp791-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r1_c3.0_common.bed ../../data/diffBinding_rep1/diffPeaks_unscaled/mp783-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r1_vs_mp791-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks_unscaled/POLII_S5_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_rep1/diffPeaks_unscaled/POLII_S5_mat.gz --outFileName ../../data/diffBinding_rep1/diffPeaks_unscaled/POLII_S5_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle POLII_S5', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270418v1\\t      2145\\nchrUn_KI270378v1\\t      1048\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270364v1\\t      2855\\nchrUn_KI270579v1\\t     31033\\nchrUn_KI270419v1\\t      1029\\nchrUn_KI270529v1\\t      1899\\nchrUn_KI270312v1\\t       998\\nchrUn_GL000226v1\\t     15008\\nchrUn_KI270302v1\\t      2274\\nchrUn_KI270304v1\\t      2165\\nchrUn_KI270539v1\\t       993\\nchrUn_KI270384v1\\t      1658\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270422v1\\t      1445\\nchrUn_KI270303v1\\t      1942\\nchrUn_KI270391v1\\t      1484\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270425v1\\t      1884\\n')\n",
      "found 0 header line... removing\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "found 0 header line... removing\n",
      "found 0 header line... removing\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep1unscaled/mp784-MV411_IRF2BP2_DMSO_MYC-MYC-r1.mLb.clN.bigWig ../../data/bigwigrep1unscaled/mp792-MV411_IRF2BP2_VHL_MYC-MYC-r1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/diffBinding_rep1/diffPeaks_unscaled/mp784-MV411_IRF2BP2_DMSO_MYC-MYC-r1_vs_mp792-MV411_IRF2BP2_VHL_MYC-MYC-r1_c3.0_cond1.bed ../../data/diffBinding_rep1/diffPeaks_unscaled/mp784-MV411_IRF2BP2_DMSO_MYC-MYC-r1_vs_mp792-MV411_IRF2BP2_VHL_MYC-MYC-r1_c3.0_common.bed ../../data/diffBinding_rep1/diffPeaks_unscaled/mp784-MV411_IRF2BP2_DMSO_MYC-MYC-r1_vs_mp792-MV411_IRF2BP2_VHL_MYC-MYC-r1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks_unscaled/MYC_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_rep1/diffPeaks_unscaled/MYC_mat.gz --outFileName ../../data/diffBinding_rep1/diffPeaks_unscaled/MYC_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle MYC', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270411v1\\t      2646\\nchrUn_KI270304v1\\t      2165\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270422v1\\t      1445\\n         chrEBV\\t    171823\\nchrUn_KI270335v1\\t      1048\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270390v1\\t      2387\\nchrUn_KI270391v1\\t      1484\\nchrUn_KI270385v1\\t       990\\nchrUn_KI270310v1\\t      1201\\nchrUn_KI270539v1\\t       993\\n')\n",
      "found 0 header line... removing\n",
      "found 0 header line... removing\n",
      "found 0 header line... removing\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep1unscaled/mp785-MV411_IRF2BP2_DMSO_MYB-MYB-r1.mLb.clN.bigWig ../../data/bigwigrep1unscaled/mp793-MV411_IRF2BP2_VHL_MYB-MYB-r1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/diffBinding_rep1/diffPeaks_unscaled/mp785-MV411_IRF2BP2_DMSO_MYB-MYB-r1_vs_mp793-MV411_IRF2BP2_VHL_MYB-MYB-r1_c3.0_cond1.bed ../../data/diffBinding_rep1/diffPeaks_unscaled/mp785-MV411_IRF2BP2_DMSO_MYB-MYB-r1_vs_mp793-MV411_IRF2BP2_VHL_MYB-MYB-r1_c3.0_common.bed ../../data/diffBinding_rep1/diffPeaks_unscaled/mp785-MV411_IRF2BP2_DMSO_MYB-MYB-r1_vs_mp793-MV411_IRF2BP2_VHL_MYB-MYB-r1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks_unscaled/MYB_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_rep1/diffPeaks_unscaled/MYB_mat.gz --outFileName ../../data/diffBinding_rep1/diffPeaks_unscaled/MYB_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle MYB', returncode=0, stdout=b'', stderr=b\"\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270411v1\\t      2646\\nchrUn_KI270419v1\\t      1029\\nchrUn_KI270390v1\\t      2387\\nchrUn_KI270302v1\\t      2274\\nchrUn_KI270391v1\\t      1484\\nchrUn_KI270305v1\\t      1472\\nchrUn_KI270383v1\\t      1750\\nchrUn_KI270392v1\\t       971\\nchrUn_KI270316v1\\t      1444\\nchrUn_KI270363v1\\t      1803\\nchrUn_KI270418v1\\t      2145\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270414v1\\t      2489\\nchrUn_KI270579v1\\t     31033\\nOne of the groups defined in the bed file is too small.\\nGroups that are too small can't be plotted. \\nWARNING: Group 'mp785-MV411_IRF2BP2_DMSO_MYB-MYB-r1_vs_mp793-MV411_IRF2BP2_VHL_MYB-MYB-r1_c3.0_cond1.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n",
      "found 0 header line... removing\n",
      "found 0 header line... removing\n",
      "found 0 header line... removing\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep1unscaled/mp786-MV411_IRF2BP2_DMSO_SPI1-SPI1-r1.mLb.clN.bigWig ../../data/bigwigrep1unscaled/mp794-MV411_IRF2BP2_VHL_SPI1-SPI1-r1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/diffBinding_rep1/diffPeaks_unscaled/mp786-MV411_IRF2BP2_DMSO_SPI1-SPI1-r1_vs_mp794-MV411_IRF2BP2_VHL_SPI1-SPI1-r1_c3.0_common.bed ../../data/diffBinding_rep1/diffPeaks_unscaled/mp786-MV411_IRF2BP2_DMSO_SPI1-SPI1-r1_vs_mp794-MV411_IRF2BP2_VHL_SPI1-SPI1-r1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep1/diffPeaks_unscaled/SPI1_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_rep1/diffPeaks_unscaled/SPI1_mat.gz --outFileName ../../data/diffBinding_rep1/diffPeaks_unscaled/SPI1_mat.pdf --refPointLabel center --regionsLabel common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle SPI1', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270382v1\\t      4215\\nchrUn_KI270419v1\\t      1029\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270414v1\\t      2489\\nchrUn_KI270302v1\\t      2274\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270392v1\\t       971\\nchrUn_KI270311v1\\t     12399\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270322v1\\t     21476\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270391v1\\t      1484\\nchrUn_KI270394v1\\t       970\\nchrUn_KI270305v1\\t      1472\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270411v1\\t      2646\\nchrUn_KI270390v1\\t      2387\\n')\n"
     ]
    }
   ],
   "source": [
    "for i in range(int(len(bw[1:])/2)-1):\n",
    "    name1 = bw[1+i]\n",
    "    name2 = bw[9+i]\n",
    "    peak = [cond1peak[i], commonpeak[i], cond2peak[i]]\n",
    "    for val in peak:\n",
    "        chip.dropWeirdChromosomes(val)\n",
    "    name = names[i]\n",
    "    chip.computePeaksAt(peak, [name1, name2], bigwignames=['DMSO', 'VHL'], peaknames=['DMSO_peaks', 'common', 'VHL_peaks'], window=3000, title=name, numthreads=7, refpoint='center', folder=\"../../data/bigwigrep1unscaled/\", torecompute=True, name='../../data/diffBinding_rep1/diffPeaks_unscaled/'+name+'_mat.pdf', withDeeptools=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! gsutil -m cp -r \"diffPeaks\" gs://amlproject/Chip/IRF2BP2/ &&\\\n",
    "    gsutil -m cp -r \"diffData\" gs://amlproject/Chip/IRF2BP2/ &&\\\n",
    "        gsutil -m cp -r \"bigwigs\" gs://amlproject/Chip/IRF2BP2/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## v2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#! rm -r bams && mkdir bams && gsutil -m cp \"gs://transfer-amlproject/20191219_MP7659*\" bams/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "rename = {\n",
    "\"20191219_MP7659_1_S1_R1_001.fastq.gz\":\"mp795_R1.fastq.gz\",\n",
    "\"20191219_MP7659_1_S1_R2_001.fastq.gz\":\"mp795_R2.fastq.gz\",\n",
    "\"20191219_MP7659_2_S2_R1_001.fastq.gz\":\"mp796_R1.fastq.gz\",\n",
    "\"20191219_MP7659_2_S2_R2_001.fastq.gz\":\"mp796_R2.fastq.gz\",\n",
    "\"20191219_MP7659_3_S3_R1_001.fastq.gz\":\"mp797_R1.fastq.gz\",\n",
    "\"20191219_MP7659_3_S3_R2_001.fastq.gz\":\"mp797_R2.fastq.gz\",\n",
    "\"20191219_MP7659_4_S4_R1_001.fastq.gz\":\"mp798_R1.fastq.gz\",\n",
    "\"20191219_MP7659_4_S4_R2_001.fastq.gz\":\"mp798_R2.fastq.gz\",\n",
    "\"20191219_MP7659_5_S5_R1_001.fastq.gz\":\"mp799_R1.fastq.gz\",\n",
    "\"20191219_MP7659_5_S5_R2_001.fastq.gz\":\"mp799_R2.fastq.gz\",\n",
    "\"20191219_MP7659_6_S6_R1_001.fastq.gz\":\"mp800_R1.fastq.gz\",\n",
    "\"20191219_MP7659_6_S6_R2_001.fastq.gz\":\"mp800_R2.fastq.gz\",\n",
    "\"20191219_MP7659_7_S7_R1_001.fastq.gz\":\"mp801_R1.fastq.gz\",\n",
    "\"20191219_MP7659_7_S7_R2_001.fastq.gz\":\"mp801_R2.fastq.gz\",\n",
    "\"20191219_MP7659_8_S8_R1_001.fastq.gz\":\"mp802_R1.fastq.gz\",\n",
    "\"20191219_MP7659_8_S8_R2_001.fastq.gz\":\"mp802_R2.fastq.gz\",\n",
    "\"20191219_MP7659_9_S9_R1_001.fastq.gz\":\"mp803_R1.fastq.gz\",\n",
    "\"20191219_MP7659_9_S9_R2_001.fastq.gz\":\"mp803_R2.fastq.gz\",\n",
    "\"20191219_MP7659_10_S10_R1_001.fastq.gz\":\"mp804_R1.fastq.gz\",\n",
    "\"20191219_MP7659_10_S10_R2_001.fastq.gz\":\"mp804_R2.fastq.gz\",\n",
    "\"20191219_MP7659_11_S11_R1_001.fastq.gz\":\"mp805_R1.fastq.gz\",\n",
    "\"20191219_MP7659_11_S11_R2_001.fastq.gz\":\"mp805_R2.fastq.gz\",\n",
    "\"20191219_MP7659_12_S12_R1_001.fastq.gz\":\"mp806_R1.fastq.gz\",\n",
    "\"20191219_MP7659_12_S12_R2_001.fastq.gz\":\"mp806_R2.fastq.gz\",\n",
    "\"20191219_MP7659_13_S13_R1_001.fastq.gz\":\"mp807_R1.fastq.gz\",\n",
    "\"20191219_MP7659_13_S13_R2_001.fastq.gz\":\"mp807_R2.fastq.gz\",\n",
    "\"20191219_MP7659_14_S14_R1_001.fastq.gz\":\"mp808_R1.fastq.gz\",\n",
    "\"20191219_MP7659_14_S14_R2_001.fastq.gz\":\"mp808_R2.fastq.gz\",\n",
    "\"20191219_MP7659_15_S15_R1_001.fastq.gz\":\"mp809_R1.fastq.gz\",\n",
    "\"20191219_MP7659_15_S15_R2_001.fastq.gz\":\"mp809_R2.fastq.gz\",\n",
    "\"20191219_MP7659_16_S16_R1_001.fastq.gz\":\"mp810_R1.fastq.gz\",\n",
    "\"20191219_MP7659_16_S16_R2_001.fastq.gz\":\"mp810_R2.fastq.gz\"\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for k,v in rename.items():\n",
    "    ! mv $k $v\n",
    "! gsutil -m cp 'bams/*' gs://amlproject/Chip_AML/IRF2BP2/results_rep2/    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! gsutil ls gs://amlproject/Chip_AML/IRF2BP2/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "inputfastq=\"gs://amlproject/Chip_AML/Chip/mp99.fastq.gz\"\n",
    "! gsutil cp $inputfastq bams\n",
    "a = ! ls bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from gsheets import Sheets\n",
    "sheets = Sheets.from_files('~/.client_secret.json', '~/.storage.json')\n",
    "url=\"https://docs.google.com/spreadsheets/d/1yFLjYB1McU530JnLgL0QIMAKIkVl3kl0_LCHje2gk8U\"\n",
    "gsheet = sheets.get(url).sheets[2].to_frame()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = {\n",
    "\"fastq1\": [],\n",
    "\"fastq2\": [],\n",
    "\"antibody\": [],\n",
    "\"group\": [],\n",
    "\"replicate\": [],\n",
    "\"control\": []\n",
    "}\n",
    "for val in helper.grouped(a[:-1],2):\n",
    "    row = gsheet[gsheet.id==val[0].split('/')[-1].split('_')[0]]\n",
    "    df['group'].append((row.id.values[0] + '_'+ row.name_replicate.values[0] +\"_\"+row.protein.values[0]))\n",
    "    df['replicate'].append(1)\n",
    "    df['fastq1'].append(val[0])\n",
    "    #df['fastq2'].append(val[1])\n",
    "    df['antibody'].append(row.protein.values[0])\n",
    "    df['control'].append(\"INPUT\")\n",
    "df['group'].append('INPUT')\n",
    "df['replicate'].append(1)\n",
    "df['fastq1'].append(a[-1])\n",
    "#df['fastq2'].append(val[1])\n",
    "df['antibody'].append(\"INPUT\")\n",
    "df['control'].append(\"INPUT\")\n",
    "df = pd.DataFrame(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df[df.columns[[3,4,0,1,2,5]]].to_csv('design.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#process chips\n",
    "! sudo ./nextflow run nf-core/chipseq --single_end --seq_center 'DFCI' --email 'jkobject@gmail.com' --narrow_peak --input design.csv --genome GRCh38 --skip_preseq --max_cpus 24 -profile docker -w work"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#!gsutil cp results gs://amlproject/Chip_AML/IRF2BP2/ && sudo rm -r work && mv results results2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# get scaling values\n",
    "mappedreads, umappedreads_norm, mapped = chip.getSpikeInControlScales(refgenome=\"ref/reference_droso.fna\",FastQfolder='bams',pairedEnd=True, cores=8)\n",
    "mappedreads, umappedreads_norm, mapped"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "computing scales from the excel sheet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [[508878,637972],\n",
    "[1929129,11595],\n",
    "[669536,429562],\n",
    "[1272730,810802],\n",
    "[743859,893304],\n",
    "[312888,1154119],\n",
    "[1086031,880901],\n",
    "[850181,1019640]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [[1.0, 0.7976494266],\n",
    "[0.006010484524, 1.0],\n",
    "[0.6415816327, 1.0],\n",
    "[0.6370573492, 1.0],\n",
    "[1.0, 0.8327053276],\n",
    "[1.0, 0.2711054926],\n",
    "[0.8111195721, 1.0],\n",
    "[1.0, 0.8338050685]]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### on scalled data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [],
   "source": [
    "# on scaled data\n",
    "bams = ! ls results2/bwa/mergedLibrary/*.bam\n",
    "for i in range(int(len(bams[1:])/2)-1):\n",
    "    name1 = bams[1+val]\n",
    "    name2 = bams[9+val]\n",
    "    chip.fullDiffPeak(name1,name2,control1='../../data/results3/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam', directory = \"../../data/diffBinding_rep2/diffData_unscaled/\", res_directory = \"../../data/diffBinding_rep2/diffPeaks_unscaled/\", scaling = scales[val][::-1])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bams = ! ls results2/bwa/mergedLibrary/*.bam\n",
    "bams = [i.split('/')[-1].split('.')[0] for i in bams]\n",
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# diffPeak on scalled data\n",
    "sizes = [220, 191, 228, 285, 222, 204, 211, 194]\n",
    "for i in range(int(len(bams[1:])/2)-1):\n",
    "    name1 = bams[1+i]\n",
    "    name2 = bams[9+i]\n",
    "    chip.diffPeak(name1,name2, directory= \"../../data/diffBinding_rep1/diffData/\", res_directory='../../data/diffBinding_rep1/diffPeaks/', scaling1=scales[i][1], scaling2=scales[i][0], size=size[i])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [1.0,\n",
    "0.006010484524,\n",
    "0.6415816327,\n",
    "0.6370573492,\n",
    "1.0,\n",
    "1.0,\n",
    "0.8111195721,\n",
    "1.0,\n",
    "0.7976494266,\n",
    "1.0,\n",
    "1.0,\n",
    "1.0,\n",
    "0.8327053276,\n",
    "0.2711054926,\n",
    "1.0,\n",
    "0.8338050685]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "chip.bigWigFrom(bams[1:],genome='GRCh38',scaling=scales)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 107,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/recalib_bigwig_rep2/mp795_R1.bw',\n",
       " '../../data/recalib_bigwig_rep2/mp796_R1.bw',\n",
       " '../../data/recalib_bigwig_rep2/mp797_R1.bw',\n",
       " '../../data/recalib_bigwig_rep2/mp798_R1.bw',\n",
       " '../../data/recalib_bigwig_rep2/mp799_R1.bw',\n",
       " '../../data/recalib_bigwig_rep2/mp800_R1.bw',\n",
       " '../../data/recalib_bigwig_rep2/mp801_R1.bw',\n",
       " '../../data/recalib_bigwig_rep2/mp802_R1.bw',\n",
       " '../../data/recalib_bigwig_rep2/mp803_R1.bw',\n",
       " '../../data/recalib_bigwig_rep2/mp804_R1.bw',\n",
       " '../../data/recalib_bigwig_rep2/mp805_R1.bw',\n",
       " '../../data/recalib_bigwig_rep2/mp806_R1.bw',\n",
       " '../../data/recalib_bigwig_rep2/mp807_R1.bw',\n",
       " '../../data/recalib_bigwig_rep2/mp808_R1.bw',\n",
       " '../../data/recalib_bigwig_rep2/mp809_R1.bw',\n",
       " '../../data/recalib_bigwig_rep2/mp810_R1.bw']"
      ]
     },
     "execution_count": 107,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bw = ! ls ../../data/recalib_bigwig_rep2/*.bw\n",
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 108,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/diffBinding_rep2/diffPeaks/mp795_R1_vs_mp803_R1_c3.0_common.bed',\n",
       " '../../data/diffBinding_rep2/diffPeaks/mp796_R1_vs_mp804_R1_c3.0_common.bed',\n",
       " '../../data/diffBinding_rep2/diffPeaks/mp797_R1_vs_mp805_R1_c3.0_common.bed',\n",
       " '../../data/diffBinding_rep2/diffPeaks/mp798_R1_vs_mp806_R1_c3.0_common.bed',\n",
       " '../../data/diffBinding_rep2/diffPeaks/mp799_R1_vs_mp807_R1_c3.0_common.bed',\n",
       " '../../data/diffBinding_rep2/diffPeaks/mp800_R1_vs_mp808_R1_c3.0_common.bed',\n",
       " '../../data/diffBinding_rep2/diffPeaks/mp801_R1_vs_mp809_R1_c3.0_common.bed',\n",
       " '../../data/diffBinding_rep2/diffPeaks/mp802_R1_vs_mp810_R1_c3.0_common.bed']"
      ]
     },
     "execution_count": 108,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "cond1peak = ! ls ../../data/diffBinding_rep2/diffPeaks/*cond1.bed\n",
    "cond2peak = ! ls ../../data/diffBinding_rep2/diffPeaks/*cond2.bed\n",
    "commonpeak = ! ls ../../data/diffBinding_rep2/diffPeaks/*common.bed\n",
    "commonpeak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 109,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"FLAG_IRF2BP2\",\"MED1\",\"POLII_total\",\"POLII_S2\",\"POLII_S5\",\"MYC\",\"MYB\",\"SPI1\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 110,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "found 0 header line... removing\n",
      "found 0 header line... removing\n",
      "found 0 header line... removing\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep2/mp795_R1.bw ../../data/recalib_bigwig_rep2/mp803_R1.bw  --referencePoint center --regionsFileName ../../data/diffBinding_rep2/diffPeaks/mp795_R1_vs_mp803_R1_c3.0_cond1.bed ../../data/diffBinding_rep2/diffPeaks/mp795_R1_vs_mp803_R1_c3.0_common.bed ../../data/diffBinding_rep2/diffPeaks/mp795_R1_vs_mp803_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/FLAG_IRF2BP2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 20 && plotHeatmap --matrixFile ../../data/diffBinding_rep2/diffPeaks/FLAG_IRF2BP2_mat.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/FLAG_IRF2BP2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle FLAG_IRF2BP2', returncode=0, stdout=b'', stderr=b'')\n",
      "empty bed\n",
      "empty bed\n",
      "found 0 header line... removing\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep2/mp796_R1.bw ../../data/recalib_bigwig_rep2/mp804_R1.bw  --referencePoint center --regionsFileName ../../data/diffBinding_rep2/diffPeaks/mp796_R1_vs_mp804_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/MED1_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 20 && plotHeatmap --matrixFile ../../data/diffBinding_rep2/diffPeaks/MED1_mat.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/MED1_mat.pdf --refPointLabel center --regionsLabel VHL_peaks  --samplesLabel DMSO VHL  --plotTitle MED1', returncode=0, stdout=b'', stderr=b'')\n",
      "empty bed\n",
      "found 0 header line... removing\n",
      "found 0 header line... removing\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep2/mp797_R1.bw ../../data/recalib_bigwig_rep2/mp805_R1.bw  --referencePoint center --regionsFileName ../../data/diffBinding_rep2/diffPeaks/mp797_R1_vs_mp805_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_total_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 20 && plotHeatmap --matrixFile ../../data/diffBinding_rep2/diffPeaks/POLII_total_mat.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_total_mat.pdf --refPointLabel center --regionsLabel VHL_peaks  --samplesLabel DMSO VHL  --plotTitle POLII_total', returncode=0, stdout=b'', stderr=b'')\n",
      "found 0 header line... removing\n",
      "found 0 header line... removing\n",
      "found 0 header line... removing\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep2/mp798_R1.bw ../../data/recalib_bigwig_rep2/mp806_R1.bw  --referencePoint center --regionsFileName ../../data/diffBinding_rep2/diffPeaks/mp798_R1_vs_mp806_R1_c3.0_cond1.bed ../../data/diffBinding_rep2/diffPeaks/mp798_R1_vs_mp806_R1_c3.0_common.bed ../../data/diffBinding_rep2/diffPeaks/mp798_R1_vs_mp806_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_S2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 20 && plotHeatmap --matrixFile ../../data/diffBinding_rep2/diffPeaks/POLII_S2_mat.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_S2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle POLII_S2', returncode=0, stdout=b'', stderr=b\"WARNING: Group 'mp798_R1_vs_mp806_R1_c3.0_cond1.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n",
      "found 0 header line... removing\n",
      "found 0 header line... removing\n",
      "found 0 header line... removing\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep2/mp799_R1.bw ../../data/recalib_bigwig_rep2/mp807_R1.bw  --referencePoint center --regionsFileName ../../data/diffBinding_rep2/diffPeaks/mp799_R1_vs_mp807_R1_c3.0_cond1.bed ../../data/diffBinding_rep2/diffPeaks/mp799_R1_vs_mp807_R1_c3.0_common.bed ../../data/diffBinding_rep2/diffPeaks/mp799_R1_vs_mp807_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_S5_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 20 && plotHeatmap --matrixFile ../../data/diffBinding_rep2/diffPeaks/POLII_S5_mat.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_S5_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle POLII_S5', returncode=0, stdout=b'', stderr=b\"One of the groups defined in the bed file is too small.\\nGroups that are too small can't be plotted. \\nWARNING: Group 'mp799_R1_vs_mp807_R1_c3.0_cond1.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n",
      "found 0 header line... removing\n",
      "found 0 header line... removing\n",
      "found 0 header line... removing\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep2/mp800_R1.bw ../../data/recalib_bigwig_rep2/mp808_R1.bw  --referencePoint center --regionsFileName ../../data/diffBinding_rep2/diffPeaks/mp800_R1_vs_mp808_R1_c3.0_cond1.bed ../../data/diffBinding_rep2/diffPeaks/mp800_R1_vs_mp808_R1_c3.0_common.bed ../../data/diffBinding_rep2/diffPeaks/mp800_R1_vs_mp808_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/MYC_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 20 && plotHeatmap --matrixFile ../../data/diffBinding_rep2/diffPeaks/MYC_mat.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/MYC_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle MYC', returncode=0, stdout=b'', stderr=b\"WARNING: Group 'mp800_R1_vs_mp808_R1_c3.0_cond2.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n",
      "empty bed\n",
      "found 0 header line... removing\n",
      "found 0 header line... removing\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep2/mp801_R1.bw ../../data/recalib_bigwig_rep2/mp809_R1.bw  --referencePoint center --regionsFileName ../../data/diffBinding_rep2/diffPeaks/mp801_R1_vs_mp809_R1_c3.0_common.bed ../../data/diffBinding_rep2/diffPeaks/mp801_R1_vs_mp809_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/MYB_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 20 && plotHeatmap --matrixFile ../../data/diffBinding_rep2/diffPeaks/MYB_mat.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/MYB_mat.pdf --refPointLabel center --regionsLabel common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle MYB', returncode=0, stdout=b'', stderr=b'')\n",
      "found 0 header line... removing\n",
      "found 0 header line... removing\n",
      "empty bed\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_rep2/mp802_R1.bw ../../data/recalib_bigwig_rep2/mp810_R1.bw  --referencePoint center --regionsFileName ../../data/diffBinding_rep2/diffPeaks/mp802_R1_vs_mp810_R1_c3.0_cond1.bed ../../data/diffBinding_rep2/diffPeaks/mp802_R1_vs_mp810_R1_c3.0_common.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/SPI1_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 20 && plotHeatmap --matrixFile ../../data/diffBinding_rep2/diffPeaks/SPI1_mat.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/SPI1_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common  --samplesLabel DMSO VHL  --plotTitle SPI1', returncode=0, stdout=b'', stderr=b'')\n"
     ]
    }
   ],
   "source": [
    "for i in range(int(len(bw)/2)):\n",
    "    name1 = bw[i]\n",
    "    name2 = bw[8+i]\n",
    "    peak = [cond1peak[i], commonpeak[i], cond2peak[i]]\n",
    "    for val in peak:\n",
    "        chip.dropWeirdChromosomes(val)\n",
    "    name = names[i]\n",
    "    chip.computePeaksAt(peak, [name1, name2], bigwignames=['DMSO', 'VHL'], peaknames=['DMSO_peaks', 'common', 'VHL_peaks'], window=3000, folder=\"\", title=name, numthreads=20, name='../../data/diffBinding_rep2/diffPeaks/'+name+'_mat.pdf', refpoint='center', withDeeptools=True, torecompute= True,)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 119,
   "metadata": {},
   "outputs": [],
   "source": [
    "mkdir ../../data/peaks2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 120,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/macs/narrowPeak/mp795-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r2.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/macs/narrowPeak/mp796-MV411_IRF2BP2_DMSO_MED1-MED1-r2.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/macs/narrowPeak/mp797-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r2.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/macs/narrowPeak/mp798-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r2.narrowPeak...\n",
      "- [4 files][  7.3 MiB/  7.3 MiB]                                                \n",
      "==> NOTE: You are performing a sequence of gsutil operations that may\n",
      "run significantly faster if you instead use gsutil -m cp ... Please\n",
      "see the -m section under \"gsutil help options\" for further information\n",
      "about when gsutil -m can be advantageous.\n",
      "\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/macs/narrowPeak/mp799-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r2.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/macs/narrowPeak/mp800-MV411_IRF2BP2_DMSO_MYC-MYC-r2.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/macs/narrowPeak/mp801-MV411_IRF2BP2_DMSO_MYB-MYB-r2.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/macs/narrowPeak/mp802-MV411_IRF2BP2_DMSO_SPI1-SPI1-r2.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/macs/narrowPeak/mp803-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r2.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/macs/narrowPeak/mp804-MV411_IRF2BP2_VHL_MED1-MED1-r2.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/macs/narrowPeak/mp805-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r2.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/macs/narrowPeak/mp806-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r2.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/macs/narrowPeak/mp807-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r2.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/macs/narrowPeak/mp808-MV411_IRF2BP2_VHL_MYC-MYC-r2.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/macs/narrowPeak/mp809-MV411_IRF2BP2_VHL_MYB-MYB-r2.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/macs/narrowPeak/mp810-MV411_IRF2BP2_VHL_SPI1-SPI1-r2.narrowPeak...\n",
      "- [16 files][ 54.5 MiB/ 54.5 MiB]                                               \n",
      "Operation completed over 16 objects/54.5 MiB.                                    \n"
     ]
    }
   ],
   "source": [
    "! gsutil cp gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/macs/narrowPeak/*.narrowPeak ../../data/peaks2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 119,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/peaks2/mp795-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r2.narrowPeak',\n",
       " '../../data/peaks2/mp796-MV411_IRF2BP2_DMSO_MED1-MED1-r2.narrowPeak',\n",
       " '../../data/peaks2/mp797-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r2.narrowPeak',\n",
       " '../../data/peaks2/mp798-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r2.narrowPeak',\n",
       " '../../data/peaks2/mp799-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r2.narrowPeak',\n",
       " '../../data/peaks2/mp800-MV411_IRF2BP2_DMSO_MYC-MYC-r2.narrowPeak',\n",
       " '../../data/peaks2/mp801-MV411_IRF2BP2_DMSO_MYB-MYB-r2.narrowPeak',\n",
       " '../../data/peaks2/mp802-MV411_IRF2BP2_DMSO_SPI1-SPI1-r2.narrowPeak',\n",
       " '../../data/peaks2/mp803-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r2.narrowPeak',\n",
       " '../../data/peaks2/mp804-MV411_IRF2BP2_VHL_MED1-MED1-r2.narrowPeak',\n",
       " '../../data/peaks2/mp805-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r2.narrowPeak',\n",
       " '../../data/peaks2/mp806-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r2.narrowPeak',\n",
       " '../../data/peaks2/mp807-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r2.narrowPeak',\n",
       " '../../data/peaks2/mp808-MV411_IRF2BP2_VHL_MYC-MYC-r2.narrowPeak',\n",
       " '../../data/peaks2/mp809-MV411_IRF2BP2_VHL_MYB-MYB-r2.narrowPeak',\n",
       " '../../data/peaks2/mp810-MV411_IRF2BP2_VHL_SPI1-SPI1-r2.narrowPeak']"
      ]
     },
     "execution_count": 119,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "peaks = ! ls ../../data/peaks2/*\n",
    "peaks"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 120,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['FLAG_IRF2BP2',\n",
       " 'MED1',\n",
       " 'POLII_total',\n",
       " 'POLII_S2',\n",
       " 'POLII_S5',\n",
       " 'MYC',\n",
       " 'MYB',\n",
       " 'SPI1',\n",
       " 'FLAG_IRF2BP2_VHL',\n",
       " 'MED1_VHL',\n",
       " 'POLII_total_VHL',\n",
       " 'POLII_S2_VHL',\n",
       " 'POLII_S5_VHL',\n",
       " 'MYC_VHL',\n",
       " 'MYB_VHL',\n",
       " 'SPI1_VHL']"
      ]
     },
     "execution_count": 120,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "names = [\"FLAG_IRF2BP2\",\"MED1\",\"POLII_total\",\"POLII_S2\",\"POLII_S5\",\"MYC\",\"MYB\",\"SPI1\"]\n",
    "names.extend([i+'_VHL' for i in names])\n",
    "names"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 128,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/bigwigrep2unscaled/INPUT_R1.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp795-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp796-MV411_IRF2BP2_DMSO_MED1-MED1-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp797-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp798-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp799-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp800-MV411_IRF2BP2_DMSO_MYC-MYC-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp801-MV411_IRF2BP2_DMSO_MYB-MYB-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp802-MV411_IRF2BP2_DMSO_SPI1-SPI1-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp803-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp804-MV411_IRF2BP2_VHL_MED1-MED1-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp805-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp806-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp807-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp808-MV411_IRF2BP2_VHL_MYC-MYC-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp809-MV411_IRF2BP2_VHL_MYB-MYB-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp810-MV411_IRF2BP2_VHL_SPI1-SPI1-r2.mLb.clN.bigWig']"
      ]
     },
     "execution_count": 128,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 130,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp795-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp795-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/FLAG_IRF2BP2_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/FLAG_IRF2BP2_mat_profile.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/FLAG_IRF2BP2_mat_profile.pdf --refPointLabel center --plotTitle FLAG_IRF2BP2', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp795-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp795-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/FLAG_IRF2BP2_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/FLAG_IRF2BP2_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/FLAG_IRF2BP2_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle FLAG_IRF2BP2', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp796-MV411_IRF2BP2_DMSO_MED1-MED1-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp796-MV411_IRF2BP2_DMSO_MED1-MED1-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/MED1_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/MED1_mat_profile.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/MED1_mat_profile.pdf --refPointLabel center --plotTitle MED1', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp796-MV411_IRF2BP2_DMSO_MED1-MED1-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp796-MV411_IRF2BP2_DMSO_MED1-MED1-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/MED1_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/MED1_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/MED1_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle MED1', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp797-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp797-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_total_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/POLII_total_mat_profile.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_total_mat_profile.pdf --refPointLabel center --plotTitle POLII_total', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp797-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp797-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_total_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/POLII_total_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_total_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle POLII_total', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp798-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp798-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_S2_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/POLII_S2_mat_profile.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_S2_mat_profile.pdf --refPointLabel center --plotTitle POLII_S2', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp798-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp798-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_S2_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/POLII_S2_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_S2_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle POLII_S2', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp799-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp799-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_S5_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/POLII_S5_mat_profile.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_S5_mat_profile.pdf --refPointLabel center --plotTitle POLII_S5', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp799-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp799-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_S5_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/POLII_S5_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_S5_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle POLII_S5', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp800-MV411_IRF2BP2_DMSO_MYC-MYC-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp800-MV411_IRF2BP2_DMSO_MYC-MYC-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/MYC_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/MYC_mat_profile.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/MYC_mat_profile.pdf --refPointLabel center --plotTitle MYC', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp800-MV411_IRF2BP2_DMSO_MYC-MYC-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp800-MV411_IRF2BP2_DMSO_MYC-MYC-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/MYC_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/MYC_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/MYC_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle MYC', returncode=0, stdout=b'', stderr=b'')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp801-MV411_IRF2BP2_DMSO_MYB-MYB-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp801-MV411_IRF2BP2_DMSO_MYB-MYB-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/MYB_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/MYB_mat_profile.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/MYB_mat_profile.pdf --refPointLabel center --plotTitle MYB', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp801-MV411_IRF2BP2_DMSO_MYB-MYB-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp801-MV411_IRF2BP2_DMSO_MYB-MYB-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/MYB_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/MYB_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/MYB_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle MYB', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp802-MV411_IRF2BP2_DMSO_SPI1-SPI1-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp802-MV411_IRF2BP2_DMSO_SPI1-SPI1-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/SPI1_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/SPI1_mat_profile.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/SPI1_mat_profile.pdf --refPointLabel center --plotTitle SPI1', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp802-MV411_IRF2BP2_DMSO_SPI1-SPI1-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp802-MV411_IRF2BP2_DMSO_SPI1-SPI1-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/SPI1_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/SPI1_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/SPI1_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle SPI1', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp803-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp803-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/FLAG_IRF2BP2_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/FLAG_IRF2BP2_VHL_mat_profile.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/FLAG_IRF2BP2_VHL_mat_profile.pdf --refPointLabel center --plotTitle FLAG_IRF2BP2_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp803-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp803-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/FLAG_IRF2BP2_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/FLAG_IRF2BP2_VHL_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/FLAG_IRF2BP2_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle FLAG_IRF2BP2_VHL', returncode=0, stdout=b'', stderr=b\"WARNING: Group 'cluster_1' is too small for plotting, you might want to remove it. \\n\")\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp804-MV411_IRF2BP2_VHL_MED1-MED1-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp804-MV411_IRF2BP2_VHL_MED1-MED1-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/MED1_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/MED1_VHL_mat_profile.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/MED1_VHL_mat_profile.pdf --refPointLabel center --plotTitle MED1_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp804-MV411_IRF2BP2_VHL_MED1-MED1-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp804-MV411_IRF2BP2_VHL_MED1-MED1-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/MED1_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/MED1_VHL_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/MED1_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle MED1_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp805-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp805-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_total_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/POLII_total_VHL_mat_profile.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_total_VHL_mat_profile.pdf --refPointLabel center --plotTitle POLII_total_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp805-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp805-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_total_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/POLII_total_VHL_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_total_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle POLII_total_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp806-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp806-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_S2_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/POLII_S2_VHL_mat_profile.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_S2_VHL_mat_profile.pdf --refPointLabel center --plotTitle POLII_S2_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp806-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp806-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_S2_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/POLII_S2_VHL_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_S2_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle POLII_S2_VHL', returncode=0, stdout=b'', stderr=b'')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp807-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp807-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_S5_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/POLII_S5_VHL_mat_profile.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_S5_VHL_mat_profile.pdf --refPointLabel center --plotTitle POLII_S5_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp807-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp807-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_S5_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/POLII_S5_VHL_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/POLII_S5_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle POLII_S5_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp808-MV411_IRF2BP2_VHL_MYC-MYC-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp808-MV411_IRF2BP2_VHL_MYC-MYC-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/MYC_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/MYC_VHL_mat_profile.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/MYC_VHL_mat_profile.pdf --refPointLabel center --plotTitle MYC_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp808-MV411_IRF2BP2_VHL_MYC-MYC-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp808-MV411_IRF2BP2_VHL_MYC-MYC-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/MYC_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/MYC_VHL_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/MYC_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle MYC_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp809-MV411_IRF2BP2_VHL_MYB-MYB-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp809-MV411_IRF2BP2_VHL_MYB-MYB-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/MYB_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/MYB_VHL_mat_profile.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/MYB_VHL_mat_profile.pdf --refPointLabel center --plotTitle MYB_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp809-MV411_IRF2BP2_VHL_MYB-MYB-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp809-MV411_IRF2BP2_VHL_MYB-MYB-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/MYB_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/MYB_VHL_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/MYB_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle MYB_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp810-MV411_IRF2BP2_VHL_SPI1-SPI1-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp810-MV411_IRF2BP2_VHL_SPI1-SPI1-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/SPI1_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/SPI1_VHL_mat_profile.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/SPI1_VHL_mat_profile.pdf --refPointLabel center --plotTitle SPI1_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp810-MV411_IRF2BP2_VHL_SPI1-SPI1-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/peaks2/mp810-MV411_IRF2BP2_VHL_SPI1-SPI1-r2.narrowPeak  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks/SPI1_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_rep2/diffPeaks/SPI1_VHL_mat_profile_clust3.gz --outFileName ../../data/diffBinding_rep2/diffPeaks/SPI1_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle SPI1_VHL', returncode=0, stdout=b'', stderr=b'')\n"
     ]
    }
   ],
   "source": [
    "for i in range(len(bw)-1):\n",
    "    chip.computePeaksAt(peaks[i], bw[1+i], window=3000, folder=\"\", title=names[i], numthreads=7, torecompute=True, onlyProfile=True, name='../../data/diffBinding_rep2/diffPeaks/'+names[i]+'_mat_profile.pdf', refpoint='center', withDeeptools=True)\n",
    "    chip.computePeaksAt(peaks[i], bw[1+i], window=3000, folder=\"\", title=names[i], numthreads=7, torecompute=True, refpoint='center', onlyProfile=True, name='../../data/diffBinding_rep2/diffPeaks/'+names[i]+'_mat_profile_clust3.pdf', withDeeptools=True, cluster=3)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### unscalled"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 73,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[\"ls: cannot access '../../data/diffBinding_rep2/mp*.bam': No such file or directory\"]"
      ]
     },
     "execution_count": 73,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bams = !ls ../../data/diffBinding_rep2/mp*.bam\n",
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 77,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/bin/sh: 1: mkdir../../data/diffBinding_rep2/diffPeaks_unscaled: not found\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/bigwig/mp795-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r2.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/bigwig/mp796-MV411_IRF2BP2_DMSO_MED1-MED1-r2.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/bigwig/mp797-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r2.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/bigwig/mp798-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r2.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/bigwig/mp801-MV411_IRF2BP2_DMSO_MYB-MYB-r2.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/bigwig/mp799-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r2.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/bigwig/mp800-MV411_IRF2BP2_DMSO_MYC-MYC-r2.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/bigwig/mp804-MV411_IRF2BP2_VHL_MED1-MED1-r2.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/bigwig/mp802-MV411_IRF2BP2_DMSO_SPI1-SPI1-r2.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/bigwig/mp805-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r2.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/bigwig/mp806-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r2.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/bigwig/mp807-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r2.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/bigwig/mp803-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r2.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/bigwig/mp809-MV411_IRF2BP2_VHL_MYB-MYB-r2.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/bigwig/mp810-MV411_IRF2BP2_VHL_SPI1-SPI1-r2.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/bigwig/mp808-MV411_IRF2BP2_VHL_MYC-MYC-r2.mLb.clN.bigWig...\n",
      "/ [17/17 files][  5.8 GiB/  5.8 GiB] 100% Done 284.4 MiB/s ETA 00:00:00         \n",
      "Operation completed over 17 objects/5.8 GiB.                                     \n"
     ]
    }
   ],
   "source": [
    "!mkdir  ../../data/diffBinding_rep2/diffPeaks_unscaled\n",
    "!mkdir ../../data/bigwigrep2unscaled/\n",
    "!gsutil -m cp gs://amlproject/Chip/IRF2BP2_degraded_rep2/bwa/mergedLibrary/bigwig/*.bigWig \"../../data/bigwigrep2unscaled/\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 111,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/bigwigrep2unscaled/INPUT_R1.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp795-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp796-MV411_IRF2BP2_DMSO_MED1-MED1-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp797-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp798-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp799-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp800-MV411_IRF2BP2_DMSO_MYC-MYC-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp801-MV411_IRF2BP2_DMSO_MYB-MYB-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp802-MV411_IRF2BP2_DMSO_SPI1-SPI1-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp803-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp804-MV411_IRF2BP2_VHL_MED1-MED1-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp805-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp806-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp807-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp808-MV411_IRF2BP2_VHL_MYC-MYC-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp809-MV411_IRF2BP2_VHL_MYB-MYB-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep2unscaled/mp810-MV411_IRF2BP2_VHL_SPI1-SPI1-r2.mLb.clN.bigWig']"
      ]
     },
     "execution_count": 111,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bw = ! ls ../../data/bigwigrep2unscaled/*\n",
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# on unscalled data \n",
    "for i in range(int(len(bams)/2)):\n",
    "    if i <0:\n",
    "        continue\n",
    "    name1 = bams[i]\n",
    "    name2 = bams[8+i]\n",
    "    chip.fullDiffPeak(name1,name2, control1='../../data/results3/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam', directory = \"../../data/diffBinding_rep2/diffData_unscaled/\", res_directory = \"../../data/diffBinding_rep2/diffPeaks_unscaled/\", pairedend=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 112,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/diffBinding_rep2/diffPeaks_unscaled/mp795-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r2_vs_mp803-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r2_c3.0_cond2.bed',\n",
       " '../../data/diffBinding_rep2/diffPeaks_unscaled/mp796-MV411_IRF2BP2_DMSO_MED1-MED1-r2_vs_mp804-MV411_IRF2BP2_VHL_MED1-MED1-r2_c3.0_cond2.bed',\n",
       " '../../data/diffBinding_rep2/diffPeaks_unscaled/mp797-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r2_vs_mp805-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r2_c3.0_cond2.bed',\n",
       " '../../data/diffBinding_rep2/diffPeaks_unscaled/mp798-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r2_vs_mp806-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r2_c3.0_cond2.bed',\n",
       " '../../data/diffBinding_rep2/diffPeaks_unscaled/mp799-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r2_vs_mp807-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r2_c3.0_cond2.bed',\n",
       " '../../data/diffBinding_rep2/diffPeaks_unscaled/mp800-MV411_IRF2BP2_DMSO_MYC-MYC-r2_vs_mp808-MV411_IRF2BP2_VHL_MYC-MYC-r2_c3.0_cond2.bed',\n",
       " '../../data/diffBinding_rep2/diffPeaks_unscaled/mp801-MV411_IRF2BP2_DMSO_MYB-MYB-r2_vs_mp809-MV411_IRF2BP2_VHL_MYB-MYB-r2_c3.0_cond2.bed',\n",
       " '../../data/diffBinding_rep2/diffPeaks_unscaled/mp802-MV411_IRF2BP2_DMSO_SPI1-SPI1-r2_vs_mp810-MV411_IRF2BP2_VHL_SPI1-SPI1-r2_c3.0_cond2.bed']"
      ]
     },
     "execution_count": 112,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "commonpeak = ! ls ../../data/diffBinding_rep2/diffPeaks_unscaled/*common.bed\n",
    "cond1peak = ! ls ../../data/diffBinding_rep2/diffPeaks_unscaled/*cond1.bed\n",
    "cond2peak = ! ls ../../data/diffBinding_rep2/diffPeaks_unscaled/*cond2.bed\n",
    "cond2peak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 113,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"FLAG_IRF2BP2\",\"MED1\",\"POLII_total\",\"POLII_S2\",\"POLII_S5\",\"MYC\",\"MYB\",\"SPI1\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 114,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "found 0 header line... removing\n",
      "found 1 header line... removing\n",
      "removed 3 lines\n",
      "found 1 header line... removing\n",
      "removed 15 lines\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp795-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r2.mLb.clN.bigWig ../../data/bigwigrep2unscaled/mp803-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/diffBinding_rep2/diffPeaks_unscaled/mp795-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r2_vs_mp803-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r2_c3.0_cond1.bed ../../data/diffBinding_rep2/diffPeaks_unscaled/mp795-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r2_vs_mp803-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r2_c3.0_common.bed ../../data/diffBinding_rep2/diffPeaks_unscaled/mp795-MV411_IRF2BP2_DMSO_FLAG_IRF2BP2-FLAG_IRF2BP2-r2_vs_mp803-MV411_IRF2BP2_VHL_FLAG_IRF2BP2-FLAG_IRF2BP2-r2_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks_unscaled/FLAG_IRF2BP2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_rep2/diffPeaks_unscaled/FLAG_IRF2BP2_mat.gz --outFileName ../../data/diffBinding_rep2/diffPeaks_unscaled/FLAG_IRF2BP2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle FLAG_IRF2BP2', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270393v1\\t      1308\\n         chrEBV\\t    171823\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270364v1\\t      2855\\nchrUn_KI270312v1\\t       998\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270302v1\\t      2274\\nchrUn_KI270425v1\\t      1884\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270379v1\\t      1045\\nchrUn_KI270385v1\\t       990\\nchrUn_KI270378v1\\t      1048\\nchrUn_KI270304v1\\t      2165\\nchrUn_KI270539v1\\t       993\\nchrUn_KI270391v1\\t      1484\\n')\n",
      "found 1 header line... removing\n",
      "removed 14 lines\n",
      "found 1 header line... removing\n",
      "removed 10 lines\n",
      "found 1 header line... removing\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp796-MV411_IRF2BP2_DMSO_MED1-MED1-r2.mLb.clN.bigWig ../../data/bigwigrep2unscaled/mp804-MV411_IRF2BP2_VHL_MED1-MED1-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/diffBinding_rep2/diffPeaks_unscaled/mp796-MV411_IRF2BP2_DMSO_MED1-MED1-r2_vs_mp804-MV411_IRF2BP2_VHL_MED1-MED1-r2_c3.0_cond1.bed ../../data/diffBinding_rep2/diffPeaks_unscaled/mp796-MV411_IRF2BP2_DMSO_MED1-MED1-r2_vs_mp804-MV411_IRF2BP2_VHL_MED1-MED1-r2_c3.0_common.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks_unscaled/MED1_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_rep2/diffPeaks_unscaled/MED1_mat.gz --outFileName ../../data/diffBinding_rep2/diffPeaks_unscaled/MED1_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common  --samplesLabel DMSO VHL  --plotTitle MED1', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270335v1\\t      1048\\nchrUn_KI270581v1\\t      7046\\nchrUn_KI270378v1\\t      1048\\nchrUn_KI270382v1\\t      4215\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270579v1\\t     31033\\nchrUn_KI270530v1\\t      2168\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270384v1\\t      1658\\nchrUn_KI270390v1\\t      2387\\nchrUn_KI270302v1\\t      2274\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270316v1\\t      1444\\nchrUn_KI270419v1\\t      1029\\n')\n",
      "found 1 header line... removing\n",
      "removed 3 lines\n",
      "found 1 header line... removing\n",
      "removed 1 lines\n",
      "found 1 header line... removing\n",
      "removed 16 lines\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp797-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r2.mLb.clN.bigWig ../../data/bigwigrep2unscaled/mp805-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/diffBinding_rep2/diffPeaks_unscaled/mp797-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r2_vs_mp805-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r2_c3.0_common.bed ../../data/diffBinding_rep2/diffPeaks_unscaled/mp797-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r2_vs_mp805-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r2_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks_unscaled/POLII_total_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_rep2/diffPeaks_unscaled/POLII_total_mat.gz --outFileName ../../data/diffBinding_rep2/diffPeaks_unscaled/POLII_total_mat.pdf --refPointLabel center --regionsLabel common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle POLII_total', returncode=0, stdout=b'', stderr=b\"\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270316v1\\t      1444\\nchrUn_KI270395v1\\t      1143\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270752v1\\t     27745\\nchrUn_KI270384v1\\t      1658\\nchrUn_KI270529v1\\t      1899\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270390v1\\t      2387\\nchrUn_KI270528v1\\t      2983\\nOne of the groups defined in the bed file is too small.\\nGroups that are too small can't be plotted. \\nWARNING: Group 'mp797-MV411_IRF2BP2_DMSO_POLII_total-POLII_total-r2_vs_mp805-MV411_IRF2BP2_VHL_POLII_total-POLII_total-r2_c3.0_common.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n",
      "found 1 header line... removing\n",
      "found 1 header line... removing\n",
      "removed 14 lines\n",
      "found 1 header line... removing\n",
      "removed 10 lines\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp798-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r2.mLb.clN.bigWig ../../data/bigwigrep2unscaled/mp806-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/diffBinding_rep2/diffPeaks_unscaled/mp798-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r2_vs_mp806-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r2_c3.0_cond1.bed ../../data/diffBinding_rep2/diffPeaks_unscaled/mp798-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r2_vs_mp806-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r2_c3.0_common.bed ../../data/diffBinding_rep2/diffPeaks_unscaled/mp798-MV411_IRF2BP2_DMSO_POLII_S2-POLII_S2-r2_vs_mp806-MV411_IRF2BP2_VHL_POLII_S2-POLII_S2-r2_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks_unscaled/POLII_S2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_rep2/diffPeaks_unscaled/POLII_S2_mat.gz --outFileName ../../data/diffBinding_rep2/diffPeaks_unscaled/POLII_S2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle POLII_S2', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270582v1\\t      6504\\nchrUn_KI270395v1\\t      1143\\nchrUn_KI270414v1\\t      2489\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270393v1\\t      1308\\n         chrEBV\\t    171823\\nchrUn_KI270425v1\\t      1884\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270364v1\\t      2855\\nchrUn_KI270390v1\\t      2387\\nchrUn_KI270419v1\\t      1029\\nchrUn_KI270422v1\\t      1445\\nchrUn_KI270316v1\\t      1444\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270384v1\\t      1658\\nchrUn_KI270385v1\\t       990\\n')\n",
      "found 1 header line... removing\n",
      "removed 1 lines\n",
      "found 1 header line... removing\n",
      "removed 19 lines\n",
      "found 1 header line... removing\n",
      "removed 1 lines\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp799-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r2.mLb.clN.bigWig ../../data/bigwigrep2unscaled/mp807-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/diffBinding_rep2/diffPeaks_unscaled/mp799-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r2_vs_mp807-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r2_c3.0_common.bed ../../data/diffBinding_rep2/diffPeaks_unscaled/mp799-MV411_IRF2BP2_DMSO_POLII_S5-POLII_S5-r2_vs_mp807-MV411_IRF2BP2_VHL_POLII_S5-POLII_S5-r2_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks_unscaled/POLII_S5_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_rep2/diffPeaks_unscaled/POLII_S5_mat.gz --outFileName ../../data/diffBinding_rep2/diffPeaks_unscaled/POLII_S5_mat.pdf --refPointLabel center --regionsLabel common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle POLII_S5', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_GL000226v1\\t     15008\\nchrUn_KI270581v1\\t      7046\\nchrUn_KI270539v1\\t       993\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270315v1\\t      2276\\nchrUn_KI270422v1\\t      1445\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270316v1\\t      1444\\nchrUn_KI270414v1\\t      2489\\nchrUn_KI270529v1\\t      1899\\nchrUn_KI270385v1\\t       990\\nchrUn_KI270412v1\\t      1179\\n')\n",
      "found 1 header line... removing\n",
      "removed 8 lines\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "found 1 header line... removing\n",
      "removed 22 lines\n",
      "found 1 header line... removing\n",
      "removed 3 lines\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp800-MV411_IRF2BP2_DMSO_MYC-MYC-r2.mLb.clN.bigWig ../../data/bigwigrep2unscaled/mp808-MV411_IRF2BP2_VHL_MYC-MYC-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/diffBinding_rep2/diffPeaks_unscaled/mp800-MV411_IRF2BP2_DMSO_MYC-MYC-r2_vs_mp808-MV411_IRF2BP2_VHL_MYC-MYC-r2_c3.0_common.bed ../../data/diffBinding_rep2/diffPeaks_unscaled/mp800-MV411_IRF2BP2_DMSO_MYC-MYC-r2_vs_mp808-MV411_IRF2BP2_VHL_MYC-MYC-r2_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks_unscaled/MYC_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_rep2/diffPeaks_unscaled/MYC_mat.gz --outFileName ../../data/diffBinding_rep2/diffPeaks_unscaled/MYC_mat.pdf --refPointLabel center --regionsLabel common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle MYC', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270414v1\\t      2489\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270539v1\\t       993\\nchrUn_KI270419v1\\t      1029\\nchrUn_KI270579v1\\t     31033\\nchrUn_KI270425v1\\t      1884\\nchrUn_KI270384v1\\t      1658\\nchrUn_KI270312v1\\t       998\\nchrUn_KI270305v1\\t      1472\\nchrUn_KI270390v1\\t      2387\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270395v1\\t      1143\\nchrUn_KI270530v1\\t      2168\\nchrUn_KI270422v1\\t      1445\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270392v1\\t       971\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270528v1\\t      2983\\n')\n",
      "found 0 header line... removing\n",
      "removed 1 lines\n",
      "found 1 header line... removing\n",
      "removed 16 lines\n",
      "found 1 header line... removing\n",
      "removed 4 lines\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp801-MV411_IRF2BP2_DMSO_MYB-MYB-r2.mLb.clN.bigWig ../../data/bigwigrep2unscaled/mp809-MV411_IRF2BP2_VHL_MYB-MYB-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/diffBinding_rep2/diffPeaks_unscaled/mp801-MV411_IRF2BP2_DMSO_MYB-MYB-r2_vs_mp809-MV411_IRF2BP2_VHL_MYB-MYB-r2_c3.0_common.bed ../../data/diffBinding_rep2/diffPeaks_unscaled/mp801-MV411_IRF2BP2_DMSO_MYB-MYB-r2_vs_mp809-MV411_IRF2BP2_VHL_MYB-MYB-r2_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks_unscaled/MYB_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_rep2/diffPeaks_unscaled/MYB_mat.gz --outFileName ../../data/diffBinding_rep2/diffPeaks_unscaled/MYB_mat.pdf --refPointLabel center --regionsLabel common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle MYB', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270530v1\\t      2168\\nchrUn_KI270419v1\\t      1029\\nchrUn_KI270390v1\\t      2387\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270581v1\\t      7046\\nchrUn_KI270752v1\\t     27745\\nchrUn_KI270383v1\\t      1750\\nchrUn_KI270425v1\\t      1884\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270418v1\\t      2145\\n')\n",
      "found 1 header line... removing\n",
      "removed 19 lines\n",
      "found 1 header line... removing\n",
      "removed 23 lines\n",
      "found 1 header line... removing\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep2unscaled/mp802-MV411_IRF2BP2_DMSO_SPI1-SPI1-r2.mLb.clN.bigWig ../../data/bigwigrep2unscaled/mp810-MV411_IRF2BP2_VHL_SPI1-SPI1-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/diffBinding_rep2/diffPeaks_unscaled/mp802-MV411_IRF2BP2_DMSO_SPI1-SPI1-r2_vs_mp810-MV411_IRF2BP2_VHL_SPI1-SPI1-r2_c3.0_cond1.bed ../../data/diffBinding_rep2/diffPeaks_unscaled/mp802-MV411_IRF2BP2_DMSO_SPI1-SPI1-r2_vs_mp810-MV411_IRF2BP2_VHL_SPI1-SPI1-r2_c3.0_common.bed  --missingDataAsZero --outFileName ../../data/diffBinding_rep2/diffPeaks_unscaled/SPI1_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_rep2/diffPeaks_unscaled/SPI1_mat.gz --outFileName ../../data/diffBinding_rep2/diffPeaks_unscaled/SPI1_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common  --samplesLabel DMSO VHL  --plotTitle SPI1', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270385v1\\t       990\\nchrUn_KI270378v1\\t      1048\\nchrUn_KI270393v1\\t      1308\\nchrUn_GL000226v1\\t     15008\\nchrUn_KI270312v1\\t       998\\nchrUn_KI270364v1\\t      2855\\nchrUn_KI270376v1\\t      1136\\nchrUn_KI270425v1\\t      1884\\nchrUn_KI270529v1\\t      1899\\nchrUn_KI270422v1\\t      1445\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270305v1\\t      1472\\nchrUn_KI270303v1\\t      1942\\nchrUn_KI270316v1\\t      1444\\nchrUn_KI270579v1\\t     31033\\n')\n"
     ]
    }
   ],
   "source": [
    "for i in range(int(len(bw[1:])/2)):\n",
    "    name1 = bw[1+i]\n",
    "    name2 = bw[9+i]\n",
    "    peak = [cond1peak[i], commonpeak[i], cond2peak[i]]\n",
    "    for val in peak:\n",
    "        chip.dropWeirdChromosomes(val)\n",
    "    name = names[i]\n",
    "    chip.computePeaksAt(peak, [name1, name2], bigwignames=['DMSO', 'VHL'], peaknames=['DMSO_peaks', 'common', 'VHL_peaks'], window=3000, title=name, numthreads=7, torecompute=True, refpoint='center', folder=\"\", name='../../data/diffBinding_rep2/diffPeaks_unscaled/'+name+'_mat.pdf', withDeeptools=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! gsutil -m cp -r \"diffPeaks\" gs://amlproject/Chip_AML/IRF2BP2/ &&\\\n",
    "    gsutil -m cp -r \"diffData\" gs://amlproject/Chip_AML/IRF2BP2/ &&\\\n",
    "        gsutil -m cp \"bigwigs/mp*\" gs://amlproject/Chip_AML/IRF2BP2/bigwigs"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## histones V1&2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bamfolder=\"histone_bams/\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mkdir histone_bams && gsutil cp \"gs://transfer-amlproject/*.fastq.gz\" $bamfolder"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! gsutil -m cp gs://transfer-amlproject/*MP7693* gs://transfer-amlproject/IRF2BP2_hist/ && gsutil -m rm gs://transfer-amlproject/*MP7693*"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "\n",
    "### analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "rename = {\n",
    "    \"20200103_1_MP7693_S6_R1_001.fastq.gz\":\"mp811_R1.fastq.gz\",\n",
    "    \"20200103_1_MP7693_S6_R2_001.fastq.gz\":\"mp811_R2.fastq.gz\",\n",
    "    \"20200103_2_MP7693_S7_R1_001.fastq.gz\":\"mp812_R1.fastq.gz\",\n",
    "    \"20200103_2_MP7693_S7_R2_001.fastq.gz\":\"mp812_R2.fastq.gz\",\n",
    "    \"20200103_3_MP7693_S8_R1_001.fastq.gz\":\"mp813_R1.fastq.gz\",\n",
    "    \"20200103_3_MP7693_S8_R2_001.fastq.gz\":\"mp813_R2.fastq.gz\",\n",
    "    \"20200103_4_MP7693_S9_R1_001.fastq.gz\":\"mp814_R1.fastq.gz\",\n",
    "    \"20200103_4_MP7693_S9_R2_001.fastq.gz\":\"mp814_R2.fastq.gz\",\n",
    "    \"20200103_5_MP7693_S10_R1_001.fastq.gz\":\"mp815_R1.fastq.gz\",\n",
    "    \"20200103_5_MP7693_S10_R2_001.fastq.gz\":\"mp815_R2.fastq.gz\",\n",
    "    \"20200103_6_MP7693_S11_R1_001.fastq.gz\":\"mp816_R1.fastq.gz\",\n",
    "    \"20200103_6_MP7693_S11_R2_001.fastq.gz\":\"mp816_R2.fastq.gz\",\n",
    "    \"20200103_7_MP7693_S12_R1_001.fastq.gz\":\"mp817_R1.fastq.gz\",\n",
    "    \"20200103_7_MP7693_S12_R2_001.fastq.gz\":\"mp817_R2.fastq.gz\",\n",
    "    \"20200103_8_MP7693_S13_R1_001.fastq.gz\":\"mp818_R1.fastq.gz\",\n",
    "    \"20200103_8_MP7693_S13_R2_001.fastq.gz\":\"mp818_R2.fastq.gz\",\n",
    "    \"20200103_9_MP7693_S14_R1_001.fastq.gz\":\"mp819_R1.fastq.gz\",\n",
    "    \"20200103_9_MP7693_S14_R2_001.fastq.gz\":\"mp819_R2.fastq.gz\",\n",
    "    \"20200103_10_MP7693_S15_R1_001.fastq.gz\":\"mp820_R1.fastq.gz\",\n",
    "    \"20200103_10_MP7693_S15_R2_001.fastq.gz\":\"mp820_R2.fastq.gz\",\n",
    "    \"20200103_11_MP7693_S16_R1_001.fastq.gz\":\"mp821_R1.fastq.gz\",\n",
    "    \"20200103_11_MP7693_S16_R2_001.fastq.gz\":\"mp821_R2.fastq.gz\",\n",
    "    \"20200103_12_MP7693_S17_R1_001.fastq.gz\":\"mp822_R1.fastq.gz\",\n",
    "    \"20200103_12_MP7693_S17_R2_001.fastq.gz\":\"mp822_R2.fastq.gz\",\n",
    "    \"20200103_13_MP7693_S18_R1_001.fastq.gz\":\"mp823_R1.fastq.gz\",\n",
    "    \"20200103_13_MP7693_S18_R2_001.fastq.gz\":\"mp823_R2.fastq.gz\",\n",
    "    \"20200103_14_MP7693_S19_R1_001.fastq.gz\":\"mp824_R1.fastq.gz\",\n",
    "    \"20200103_14_MP7693_S19_R2_001.fastq.gz\":\"mp824_R2.fastq.gz\",\n",
    "    \"20200103_15_MP7693_S20_R1_001.fastq.gz\":\"mp825_R1.fastq.gz\",\n",
    "    \"20200103_15_MP7693_S20_R2_001.fastq.gz\":\"mp825_R2.fastq.gz\",\n",
    "    \"20200103_16_MP7693_S21_R1_001.fastq.gz\":\"mp826_R1.fastq.gz\",\n",
    "    \"20200103_16_MP7693_S21_R2_001.fastq.gz\":\"mp826_R2.fastq.gz\",\n",
    "    \"20200103_17_MP7693_S22_R1_001.fastq.gz\":\"mp827_R1.fastq.gz\",\n",
    "    \"20200103_17_MP7693_S22_R2_001.fastq.gz\":\"mp827_R2.fastq.gz\",\n",
    "    \"20200103_18_MP7693_S23_R1_001.fastq.gz\":\"mp828_R1.fastq.gz\",\n",
    "    \"20200103_18_MP7693_S23_R2_001.fastq.gz\":\"mp828_R2.fastq.gz\",\n",
    "    \"20200103_19_MP7693_S24_R1_001.fastq.gz\":\"mp829_R1.fastq.gz\",\n",
    "    \"20200103_19_MP7693_S24_R2_001.fastq.gz\":\"mp829_R2.fastq.gz\",\n",
    "    \"20200103_20_MP7693_S25_R1_001.fastq.gz\":\"mp830_R1.fastq.gz\",\n",
    "    \"20200103_20_MP7693_S25_R2_001.fastq.gz\":\"mp830_R2.fastq.gz\"\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for k,v in rename.items():\n",
    "    ! mv $bamfolder$k $bamfolder$v"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "inputfastq=\"gs://amlproject/Chip_AML/Chip/mp99.fastq.gz\"\n",
    "! gsutil cp $inputfastq $bamfolder"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "a = ! ls $bamfolder"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from gsheets import Sheets\n",
    "sheets = Sheets.from_files('~/.client_secret.json', '~/.storage.json')\n",
    "url=\"https://docs.google.com/spreadsheets/d/1yFLjYB1McU530JnLgL0QIMAKIkVl3kl0_LCHje2gk8U\"\n",
    "gsheet = sheets.get(url).sheets[2].to_frame()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = {\n",
    "\"fastq1\": [],\n",
    "\"fastq2\": [],\n",
    "\"antibody\": [],\n",
    "\"group\": [],\n",
    "\"replicate\": [],\n",
    "\"control\": []\n",
    "}\n",
    "for val in helper.grouped(a[:-1],2):\n",
    "    row = gsheet[gsheet.id==val[0].split('/')[-1].split('_')[0]]\n",
    "    df['group'].append((row.id.values[0] + '_'+ row.name_replicate.values[0] +\"_\"+row.protein.values[0]))\n",
    "    df['replicate'].append(1)\n",
    "    df['fastq1'].append(val[0])\n",
    "    #df['fastq2'].append(val[1])\n",
    "    df['antibody'].append(row.protein.values[0])\n",
    "    df['control'].append(\"INPUT\")\n",
    "df['group'].append('INPUT')\n",
    "df['replicate'].append(1)\n",
    "df['fastq1'].append(a[-1])\n",
    "#df['fastq2'].append(val[1])\n",
    "df['antibody'].append(\"INPUT\")\n",
    "df['control'].append(\"INPUT\")\n",
    "df = pd.DataFrame(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df[df.columns[[3,4,0,1,2,5]]].to_csv('design.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#process chips\n",
    " ! sudo ./nextflow run nf-core/chipseq --single_end --seq_center 'DFCI' --email 'jkobject@gmail.com' --narrow_peak --input design.csv --genome GRCh38 --skip_preseq --max_cpus 24 -profile docker -w work"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! gsutil -m cp -r results/* gs://amlproject/Chip/IRF2BP2_degraded_histones/\n",
    "! mv results results_hist"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 171,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/mp811-MV411_IRF2BP2_DMSO_6h-H3K27ac-r1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/mp821-MV411_IRF2BP2_VHL_6h-H3K27ac-r1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1.mLb.clN.sorted.bam...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2.mLb.clN.sorted.bam...\n",
      "| [21/21 files][ 23.8 GiB/ 23.8 GiB] 100% Done  97.5 MiB/s ETA 00:00:00         \n",
      "Operation completed over 21 objects/23.8 GiB.                                    \n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "['../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp811-MV411_IRF2BP2_DMSO_6h-H3K27ac-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp821-MV411_IRF2BP2_VHL_6h-H3K27ac-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2.mLb.clN.sorted.bam']"
      ]
     },
     "execution_count": 171,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "! gsutil -m cp gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/*.bam ../../data/diffBinding_hist/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['INPUT_R1',\n",
       " 'mp811-MV411_IRF2BP2_DMSO_6h-H3K27ac-r1',\n",
       " 'mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2',\n",
       " 'mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1',\n",
       " 'mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2',\n",
       " 'mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1',\n",
       " 'mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2',\n",
       " 'mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1',\n",
       " 'mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2',\n",
       " 'mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1',\n",
       " 'mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2',\n",
       " 'mp821-MV411_IRF2BP2_VHL_6h-H3K27ac-r1',\n",
       " 'mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2',\n",
       " 'mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1',\n",
       " 'mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2',\n",
       " 'mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1',\n",
       " 'mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2',\n",
       " 'mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1',\n",
       " 'mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2',\n",
       " 'mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1',\n",
       " 'mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2']"
      ]
     },
     "execution_count": 38,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bams = ! ls ../../data/diffBinding_hist/*.bam\n",
    "bams = [i.split('/')[-1].split('.')[0] for i in bams]\n",
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 286,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [[191079,278272],\n",
    "[274625,494562],\n",
    "[1094016,2067804],\n",
    "[1328914,969565],\n",
    "[237779,132422],\n",
    "[162971,174092],\n",
    "[115788,187078],\n",
    "[134269,495924],\n",
    "[171890,225315],\n",
    "[188370,199911]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [[1.0,0.6866626897],\n",
    "[1.0,0.5552893267],\n",
    "[1.0,0.5290714207],\n",
    "[0.7295919826,1.0],\n",
    "[0.5569120906,1.0],\n",
    "[1.0,0.9361199825],\n",
    "[1.0,0.6189290029],\n",
    "[1.0,0.2707451142],\n",
    "[1.0,0.762887513],\n",
    "[1.0,0.9422693098]]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### on scalled data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mp811-MV411_IRF2BP2_DMSO_6h-H3K27ac-r1 mp821-MV411_IRF2BP2_VHL_6h-H3K27ac-r1\n",
      "doing differential peak binding\n",
      "mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2 mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2\n",
      "doing differential peak binding\n",
      "mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1 mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1\n",
      "doing differential peak binding\n",
      "mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2 mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2\n",
      "doing differential peak binding\n",
      "mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1 mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1\n",
      "doing differential peak binding\n",
      "mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2 mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2\n",
      "doing differential peak binding\n",
      "mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1 mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1\n",
      "doing differential peak binding\n",
      "mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2 mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2\n",
      "doing differential peak binding\n",
      "mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1 mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1\n",
      "doing differential peak binding\n",
      "mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2 mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2\n",
      "doing differential peak binding\n"
     ]
    }
   ],
   "source": [
    "# diffPeak on scaled data\n",
    "size=[206, 213, 47, 272, 229, 235, 190, 196, 287, 288]\n",
    "for i in range(int(len(bams)/2)):\n",
    "    name1 = bams[1+i]\n",
    "    name2 = bams[11+i]\n",
    "    print(name1,name2)\n",
    "    chip.diffPeak(name1, name2, directory= \"../../data/diffBinding_hist/diffData/\", res_directory='../../data/diffBinding_hist/diffPeaks/', scaling1=scales[i][1], scaling2=scales[i][0], size=size[i])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 277,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/diffBinding_hist/mp811-MV411_IRF2BP2_DMSO_6h-H3K27ac-r1.mLb.clN.sorted.bam ../../data/diffBinding_hist/mp821-MV411_IRF2BP2_VHL_6h-H3K27ac-r1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/diffBinding_hist/mp811-MV411_IRF2BP2_DMSO_6h-H3K27ac-r1.mLb.clN.sorted.bam and ../../data/diffBinding_hist/mp821-MV411_IRF2BP2_VHL_6h-H3K27ac-r1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n"
     ]
    },
    {
     "ename": "KeyboardInterrupt",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-277-afbeef75d510>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m      5\u001b[0m     \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mname1\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0mname2\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      6\u001b[0m     chip.fullDiffPeak(name1,name2, bams[0], scaling = scales[i], directory='../../data/diffBinding_hist/diffData/',\n\u001b[0;32m----> 7\u001b[0;31m res_directory=\"../../data/diffBinding_hist/diffPeaks/\", isTF=False, compute_size=True, pairedend=False)\n\u001b[0m",
      "\u001b[0;32m~/JKBio/epigenetics/ChIP_helper.py\u001b[0m in \u001b[0;36mfullDiffPeak\u001b[0;34m(bam1, bam2, control1, control2, scaling, directory, res_directory, isTF, compute_size, pairedend)\u001b[0m\n\u001b[1;32m    854\u001b[0m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'computing the fragment avg size'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    855\u001b[0m         \u001b[0mcmd\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m\"macs2 predictd -i \"\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mbam1\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 856\u001b[0;31m         \u001b[0mret\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0msubprocess\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrun\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mcmd\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcapture_output\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mTrue\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mshell\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mTrue\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    857\u001b[0m         \u001b[0msize\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mre\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mfindall\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"# predicted fragment length is (\\d+)\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mret\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mstderr\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m0\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    858\u001b[0m         \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0msize\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/lib/python3.7/subprocess.py\u001b[0m in \u001b[0;36mrun\u001b[0;34m(input, capture_output, timeout, check, *popenargs, **kwargs)\u001b[0m\n\u001b[1;32m    488\u001b[0m     \u001b[0;32mwith\u001b[0m \u001b[0mPopen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0mpopenargs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;32mas\u001b[0m \u001b[0mprocess\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    489\u001b[0m         \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 490\u001b[0;31m             \u001b[0mstdout\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mstderr\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mprocess\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mcommunicate\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0minput\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mtimeout\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mtimeout\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    491\u001b[0m         \u001b[0;32mexcept\u001b[0m \u001b[0mTimeoutExpired\u001b[0m \u001b[0;32mas\u001b[0m \u001b[0mexc\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    492\u001b[0m             \u001b[0mprocess\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mkill\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/lib/python3.7/subprocess.py\u001b[0m in \u001b[0;36mcommunicate\u001b[0;34m(self, input, timeout)\u001b[0m\n\u001b[1;32m    962\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    963\u001b[0m             \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 964\u001b[0;31m                 \u001b[0mstdout\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mstderr\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_communicate\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0minput\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mendtime\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mtimeout\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    965\u001b[0m             \u001b[0;32mexcept\u001b[0m \u001b[0mKeyboardInterrupt\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    966\u001b[0m                 \u001b[0;31m# https://bugs.python.org/issue25942\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/lib/python3.7/subprocess.py\u001b[0m in \u001b[0;36m_communicate\u001b[0;34m(self, input, endtime, orig_timeout)\u001b[0m\n\u001b[1;32m   1713\u001b[0m                             'failed to raise TimeoutExpired.')\n\u001b[1;32m   1714\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1715\u001b[0;31m                     \u001b[0mready\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mselector\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mselect\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mtimeout\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1716\u001b[0m                     \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_check_timeout\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mendtime\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0morig_timeout\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mstdout\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mstderr\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1717\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/lib/python3.7/selectors.py\u001b[0m in \u001b[0;36mselect\u001b[0;34m(self, timeout)\u001b[0m\n\u001b[1;32m    413\u001b[0m         \u001b[0mready\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    414\u001b[0m         \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 415\u001b[0;31m             \u001b[0mfd_event_list\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_selector\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mpoll\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mtimeout\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    416\u001b[0m         \u001b[0;32mexcept\u001b[0m \u001b[0mInterruptedError\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    417\u001b[0m             \u001b[0;32mreturn\u001b[0m \u001b[0mready\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
     ]
    }
   ],
   "source": [
    "# diffPeak on scaled data\n",
    "for i in range(int(len(bams)/2)):\n",
    "    name1 = bams[1+i]\n",
    "    name2 = bams[11+i]\n",
    "    print(name1,name2)\n",
    "    chip.fullDiffPeak(name1,name2, bams[0], scaling = scales[i], directory='../../data/diffBinding_hist/diffData/',\n",
    "res_directory=\"../../data/diffBinding_hist/diffPeaks/\", isTF=False, compute_size=True, pairedend=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [1.0,\n",
    "1.0,\n",
    "1.0,\n",
    "0.7295919826,\n",
    "0.5569120906,\n",
    "1.0,\n",
    "1.0,\n",
    "1.0,\n",
    "1.0,\n",
    "1.0,\n",
    "0.6866626897,\n",
    "0.5552893267,\n",
    "0.5290714207,\n",
    "1.0,\n",
    "1.0,\n",
    "0.9361199825,\n",
    "0.6189290029,\n",
    "0.2707451142,\n",
    "0.762887513,\n",
    "0.9422693098,]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "chip.bigWigFrom(bams[1:],genome='GRCh38',scaling=scales)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/recalib_bigwig_hist/mp811_R1.bw',\n",
       " '../../data/recalib_bigwig_hist/mp812_R1.bw',\n",
       " '../../data/recalib_bigwig_hist/mp813_R1.bw',\n",
       " '../../data/recalib_bigwig_hist/mp814_R1.bw',\n",
       " '../../data/recalib_bigwig_hist/mp815_R1.bw',\n",
       " '../../data/recalib_bigwig_hist/mp816_R1.bw',\n",
       " '../../data/recalib_bigwig_hist/mp817_R1.bw',\n",
       " '../../data/recalib_bigwig_hist/mp818_R1.bw',\n",
       " '../../data/recalib_bigwig_hist/mp819_R1.bw',\n",
       " '../../data/recalib_bigwig_hist/mp820_R1.bw',\n",
       " '../../data/recalib_bigwig_hist/mp821_R1.bw',\n",
       " '../../data/recalib_bigwig_hist/mp822_R1.bw',\n",
       " '../../data/recalib_bigwig_hist/mp823_R1.bw',\n",
       " '../../data/recalib_bigwig_hist/mp824_R1.bw',\n",
       " '../../data/recalib_bigwig_hist/mp825_R1.bw',\n",
       " '../../data/recalib_bigwig_hist/mp826_R1.bw',\n",
       " '../../data/recalib_bigwig_hist/mp827_R1.bw',\n",
       " '../../data/recalib_bigwig_hist/mp828_R1.bw',\n",
       " '../../data/recalib_bigwig_hist/mp829_R1.bw',\n",
       " '../../data/recalib_bigwig_hist/mp830_R1.bw']"
      ]
     },
     "execution_count": 58,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bw = ! ls ../../data/recalib_bigwig_hist/*.bw\n",
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 179,
   "metadata": {},
   "outputs": [],
   "source": [
    "! rm ../../data/diffBinding_hist/diffData/mp*_R1*"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/diffBinding_hist/diffPeaks/mp811-MV411_IRF2BP2_DMSO_6h-H3K27ac-r1_vs_mp821-MV411_IRF2BP2_VHL_6h-H3K27ac-r1_c3.0_cond1.bed',\n",
       " '../../data/diffBinding_hist/diffPeaks/mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2_vs_mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2_c3.0_cond1.bed',\n",
       " '../../data/diffBinding_hist/diffPeaks/mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1_vs_mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1_c3.0_cond1.bed',\n",
       " '../../data/diffBinding_hist/diffPeaks/mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2_vs_mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2_c3.0_cond1.bed',\n",
       " '../../data/diffBinding_hist/diffPeaks/mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1_vs_mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1_c3.0_cond1.bed',\n",
       " '../../data/diffBinding_hist/diffPeaks/mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2_vs_mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2_c3.0_cond1.bed',\n",
       " '../../data/diffBinding_hist/diffPeaks/mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1_vs_mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1_c3.0_cond1.bed',\n",
       " '../../data/diffBinding_hist/diffPeaks/mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2_vs_mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2_c3.0_cond1.bed',\n",
       " '../../data/diffBinding_hist/diffPeaks/mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1_vs_mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1_c3.0_cond1.bed',\n",
       " '../../data/diffBinding_hist/diffPeaks/mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2_vs_mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2_c3.0_cond1.bed']"
      ]
     },
     "execution_count": 59,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "cond1peak = ! ls ../../data/diffBinding_hist/diffPeaks/*cond1.bed\n",
    "cond2peak = ! ls ../../data/diffBinding_hist/diffPeaks/*cond2.bed\n",
    "commonpeak = ! ls ../../data/diffBinding_hist/diffPeaks/*common.bed\n",
    "cond1peak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"H3K27ac\", \"H3K27ac_v2\",\"H3K27me3\",\"H3K27me3_v2\",\"H3K4me1\",\"H3K4me1_v2\", \"H3K4me3\", \"H3K4me3_v2\", \"H3K79me2\", \"H3K79me2_v2\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0\n",
      "../../data/recalib_bigwig_hist/mp811_R1.bw ../../data/recalib_bigwig_hist/mp821_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp811_R1.bw ../../data/recalib_bigwig_hist/mp821_R1.bw  --referencePoint center --regionsFileName ../../data/diffBinding_hist/diffPeaks/mp811-MV411_IRF2BP2_DMSO_6h-H3K27ac-r1_vs_mp821-MV411_IRF2BP2_VHL_6h-H3K27ac-r1_c3.0_cond1.bed ../../data/diffBinding_hist/diffPeaks/mp811-MV411_IRF2BP2_DMSO_6h-H3K27ac-r1_vs_mp821-MV411_IRF2BP2_VHL_6h-H3K27ac-r1_c3.0_common.bed ../../data/diffBinding_hist/diffPeaks/mp811-MV411_IRF2BP2_DMSO_6h-H3K27ac-r1_vs_mp821-MV411_IRF2BP2_VHL_6h-H3K27ac-r1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27ac_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K27ac_mat.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27ac_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle H3K27ac', returncode=0, stdout=b'', stderr=b\"WARNING: Group 'mp811-MV411_IRF2BP2_DMSO_6h-H3K27ac-r1_vs_mp821-MV411_IRF2BP2_VHL_6h-H3K27ac-r1_c3.0_cond2.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n",
      "1\n",
      "../../data/recalib_bigwig_hist/mp812_R1.bw ../../data/recalib_bigwig_hist/mp822_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp812_R1.bw ../../data/recalib_bigwig_hist/mp822_R1.bw  --referencePoint center --regionsFileName ../../data/diffBinding_hist/diffPeaks/mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2_vs_mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2_c3.0_cond1.bed ../../data/diffBinding_hist/diffPeaks/mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2_vs_mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2_c3.0_common.bed ../../data/diffBinding_hist/diffPeaks/mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2_vs_mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27ac_v2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K27ac_v2_mat.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27ac_v2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle H3K27ac_v2', returncode=0, stdout=b'', stderr=b\"One of the groups defined in the bed file is too small.\\nGroups that are too small can't be plotted. \\nWARNING: Group 'mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2_vs_mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2_c3.0_cond2.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n",
      "2\n",
      "../../data/recalib_bigwig_hist/mp813_R1.bw ../../data/recalib_bigwig_hist/mp823_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp813_R1.bw ../../data/recalib_bigwig_hist/mp823_R1.bw  --referencePoint center --regionsFileName ../../data/diffBinding_hist/diffPeaks/mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1_vs_mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1_c3.0_cond1.bed  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27me3_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K27me3_mat.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27me3_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks  --samplesLabel DMSO VHL  --plotTitle H3K27me3', returncode=0, stdout=b'', stderr=b'')\n",
      "3\n",
      "../../data/recalib_bigwig_hist/mp814_R1.bw ../../data/recalib_bigwig_hist/mp824_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp814_R1.bw ../../data/recalib_bigwig_hist/mp824_R1.bw  --referencePoint center --regionsFileName ../../data/diffBinding_hist/diffPeaks/mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2_vs_mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2_c3.0_cond1.bed ../../data/diffBinding_hist/diffPeaks/mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2_vs_mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2_c3.0_common.bed ../../data/diffBinding_hist/diffPeaks/mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2_vs_mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27me3_v2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K27me3_v2_mat.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27me3_v2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle H3K27me3_v2', returncode=0, stdout=b'', stderr=b'')\n",
      "4\n",
      "../../data/recalib_bigwig_hist/mp815_R1.bw ../../data/recalib_bigwig_hist/mp825_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp815_R1.bw ../../data/recalib_bigwig_hist/mp825_R1.bw  --referencePoint center --regionsFileName ../../data/diffBinding_hist/diffPeaks/mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1_vs_mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1_c3.0_common.bed ../../data/diffBinding_hist/diffPeaks/mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1_vs_mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me1_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K4me1_mat.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me1_mat.pdf --refPointLabel center --regionsLabel common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle H3K4me1', returncode=0, stdout=b'', stderr=b'')\n",
      "5\n",
      "../../data/recalib_bigwig_hist/mp816_R1.bw ../../data/recalib_bigwig_hist/mp826_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp816_R1.bw ../../data/recalib_bigwig_hist/mp826_R1.bw  --referencePoint center --regionsFileName ../../data/diffBinding_hist/diffPeaks/mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2_vs_mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2_c3.0_cond1.bed ../../data/diffBinding_hist/diffPeaks/mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2_vs_mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2_c3.0_common.bed ../../data/diffBinding_hist/diffPeaks/mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2_vs_mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me1_v2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K4me1_v2_mat.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me1_v2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle H3K4me1_v2', returncode=0, stdout=b'', stderr=b\"One of the groups defined in the bed file is too small.\\nGroups that are too small can't be plotted. \\nWARNING: Group 'mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2_vs_mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2_c3.0_cond1.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n",
      "6\n",
      "../../data/recalib_bigwig_hist/mp817_R1.bw ../../data/recalib_bigwig_hist/mp827_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp817_R1.bw ../../data/recalib_bigwig_hist/mp827_R1.bw  --referencePoint center --regionsFileName ../../data/diffBinding_hist/diffPeaks/mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1_vs_mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1_c3.0_cond1.bed ../../data/diffBinding_hist/diffPeaks/mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1_vs_mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1_c3.0_common.bed  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me3_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K4me3_mat.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me3_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common  --samplesLabel DMSO VHL  --plotTitle H3K4me3', returncode=0, stdout=b'', stderr=b'')\n",
      "7\n",
      "../../data/recalib_bigwig_hist/mp818_R1.bw ../../data/recalib_bigwig_hist/mp828_R1.bw\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp818_R1.bw ../../data/recalib_bigwig_hist/mp828_R1.bw  --referencePoint center --regionsFileName ../../data/diffBinding_hist/diffPeaks/mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2_vs_mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2_c3.0_cond1.bed ../../data/diffBinding_hist/diffPeaks/mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2_vs_mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2_c3.0_common.bed  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me3_v2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K4me3_v2_mat.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me3_v2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common  --samplesLabel DMSO VHL  --plotTitle H3K4me3_v2', returncode=0, stdout=b'', stderr=b'')\n",
      "8\n",
      "../../data/recalib_bigwig_hist/mp819_R1.bw ../../data/recalib_bigwig_hist/mp829_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp819_R1.bw ../../data/recalib_bigwig_hist/mp829_R1.bw  --referencePoint center --regionsFileName ../../data/diffBinding_hist/diffPeaks/mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1_vs_mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1_c3.0_cond1.bed ../../data/diffBinding_hist/diffPeaks/mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1_vs_mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1_c3.0_common.bed ../../data/diffBinding_hist/diffPeaks/mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1_vs_mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K79me2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K79me2_mat.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K79me2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle H3K79me2', returncode=0, stdout=b'', stderr=b\"One of the groups defined in the bed file is too small.\\nGroups that are too small can't be plotted. \\nWARNING: Group 'mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1_vs_mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1_c3.0_cond2.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n",
      "9\n",
      "../../data/recalib_bigwig_hist/mp820_R1.bw ../../data/recalib_bigwig_hist/mp830_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp820_R1.bw ../../data/recalib_bigwig_hist/mp830_R1.bw  --referencePoint center --regionsFileName ../../data/diffBinding_hist/diffPeaks/mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2_vs_mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2_c3.0_cond1.bed ../../data/diffBinding_hist/diffPeaks/mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2_vs_mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2_c3.0_common.bed ../../data/diffBinding_hist/diffPeaks/mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2_vs_mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K79me2_v2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K79me2_v2_mat.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K79me2_v2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle H3K79me2_v2', returncode=0, stdout=b'', stderr=b\"One of the groups defined in the bed file is too small.\\nGroups that are too small can't be plotted. \\nWARNING: Group 'mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2_vs_mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2_c3.0_cond2.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n"
     ]
    }
   ],
   "source": [
    "for i in range(int(len(bw)/2)):\n",
    "    if i < 0 : \n",
    "        continue\n",
    "    print(i)\n",
    "    name1 = bw[i]\n",
    "    name2 = bw[10+i]\n",
    "    peak = [cond1peak[i], commonpeak[i], cond2peak[i]]\n",
    "    #for val in peak:\n",
    "        #chip.dropWeirdChromosomes(val)\n",
    "    name = names[i]\n",
    "    print(name1,name2)\n",
    "    chip.computePeaksAt(peak, [name1, name2], torecompute= True, bigwignames=['DMSO', 'VHL'], peaknames=['DMSO_peaks', 'common', 'VHL_peaks'], window=3000, folder=\"\", title=name, numthreads=7, refpoint='center', name='../../data/diffBinding_hist/diffPeaks/'+name+'_mat.pdf', withDeeptools=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 225,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['H3K27ac',\n",
       " 'H3K27ac_v2',\n",
       " 'H3K27me3',\n",
       " 'H3K27me3_v2',\n",
       " 'H3K4me1',\n",
       " 'H3K4me1_v2',\n",
       " 'H3K4me3',\n",
       " 'H3K4me3_v2',\n",
       " 'H3K79me2',\n",
       " 'H3K79me2_v2',\n",
       " 'H3K27ac_VHL',\n",
       " 'H3K27ac_v2_VHL',\n",
       " 'H3K27me3_VHL',\n",
       " 'H3K27me3_v2_VHL',\n",
       " 'H3K4me1_VHL',\n",
       " 'H3K4me1_v2_VHL',\n",
       " 'H3K4me3_VHL',\n",
       " 'H3K4me3_v2_VHL',\n",
       " 'H3K79me2_VHL',\n",
       " 'H3K79me2_v2_VHL']"
      ]
     },
     "execution_count": 225,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "names = [\"H3K27ac\", \"H3K27ac_v2\",\"H3K27me3\",\"H3K27me3_v2\",\"H3K4me1\",\"H3K4me1_v2\", \"H3K4me3\", \"H3K4me3_v2\", \"H3K79me2\", \"H3K79me2_v2\"]\n",
    "names.extend([i+'_VHL' for i in names])\n",
    "names"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 129,
   "metadata": {},
   "outputs": [],
   "source": [
    "mkdir ../../data/peakshist"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 148,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/macs/broadPeak/mp811-MV411_IRF2BP2_DMSO_6h-H3K27ac-r1.broadPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/macs/broadPeak/mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2.broadPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/macs/broadPeak/mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1.broadPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/macs/broadPeak/mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2.broadPeak...\n",
      "\\ [4 files][ 17.4 MiB/ 17.4 MiB]                                                \n",
      "==> NOTE: You are performing a sequence of gsutil operations that may\n",
      "run significantly faster if you instead use gsutil -m cp ... Please\n",
      "see the -m section under \"gsutil help options\" for further information\n",
      "about when gsutil -m can be advantageous.\n",
      "\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/macs/broadPeak/mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1.broadPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/macs/broadPeak/mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2.broadPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/macs/broadPeak/mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1.broadPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/macs/broadPeak/mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2.broadPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/macs/broadPeak/mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1.broadPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/macs/broadPeak/mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2.broadPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/macs/broadPeak/mp821-MV411_IRF2BP2_VHL_6h-H3K27ac-r1.broadPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/macs/broadPeak/mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2.broadPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/macs/broadPeak/mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1.broadPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/macs/broadPeak/mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2.broadPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/macs/broadPeak/mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1.broadPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/macs/broadPeak/mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2.broadPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/macs/broadPeak/mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1.broadPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/macs/broadPeak/mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2.broadPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/macs/broadPeak/mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1.broadPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/macs/broadPeak/mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2.broadPeak...\n",
      "- [20 files][100.2 MiB/100.2 MiB]                                               \n",
      "Operation completed over 20 objects/100.2 MiB.                                   \n"
     ]
    }
   ],
   "source": [
    "! gsutil cp gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/macs/broadPeak/*.broadPeak ../../data/peakshist"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 222,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/peakshist/mp811-MV411_IRF2BP2_DMSO_6h-H3K27ac-r1.broadPeak',\n",
       " '../../data/peakshist/mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2.broadPeak',\n",
       " '../../data/peakshist/mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1.broadPeak',\n",
       " '../../data/peakshist/mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2.broadPeak',\n",
       " '../../data/peakshist/mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1.broadPeak',\n",
       " '../../data/peakshist/mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2.broadPeak',\n",
       " '../../data/peakshist/mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1.broadPeak',\n",
       " '../../data/peakshist/mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2.broadPeak',\n",
       " '../../data/peakshist/mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1.broadPeak',\n",
       " '../../data/peakshist/mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2.broadPeak',\n",
       " '../../data/peakshist/mp821-MV411_IRF2BP2_VHL_6h-H3K27ac-r1.broadPeak',\n",
       " '../../data/peakshist/mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2.broadPeak',\n",
       " '../../data/peakshist/mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1.broadPeak',\n",
       " '../../data/peakshist/mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2.broadPeak',\n",
       " '../../data/peakshist/mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1.broadPeak',\n",
       " '../../data/peakshist/mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2.broadPeak',\n",
       " '../../data/peakshist/mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1.broadPeak',\n",
       " '../../data/peakshist/mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2.broadPeak',\n",
       " '../../data/peakshist/mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1.broadPeak',\n",
       " '../../data/peakshist/mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2.broadPeak']"
      ]
     },
     "execution_count": 222,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "peaks = ! ls ../../data/peakshist/*\n",
    "peaks"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 226,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp811_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp811-MV411_IRF2BP2_DMSO_6h-H3K27ac-r1.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27ac_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K27ac_mat_profile.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27ac_mat_profile.pdf --refPointLabel center --plotTitle H3K27ac', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp811_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp811-MV411_IRF2BP2_DMSO_6h-H3K27ac-r1.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27ac_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K27ac_mat_profile_clust3.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27ac_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle H3K27ac', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp812_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27ac_v2_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K27ac_v2_mat_profile.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27ac_v2_mat_profile.pdf --refPointLabel center --plotTitle H3K27ac_v2', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp812_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27ac_v2_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K27ac_v2_mat_profile_clust3.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27ac_v2_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle H3K27ac_v2', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp813_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27me3_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K27me3_mat_profile.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27me3_mat_profile.pdf --refPointLabel center --plotTitle H3K27me3', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp813_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27me3_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K27me3_mat_profile_clust3.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27me3_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle H3K27me3', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp814_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27me3_v2_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K27me3_v2_mat_profile.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27me3_v2_mat_profile.pdf --refPointLabel center --plotTitle H3K27me3_v2', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp814_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27me3_v2_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K27me3_v2_mat_profile_clust3.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27me3_v2_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle H3K27me3_v2', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp815_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me1_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K4me1_mat_profile.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me1_mat_profile.pdf --refPointLabel center --plotTitle H3K4me1', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp815_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me1_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K4me1_mat_profile_clust3.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me1_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle H3K4me1', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp816_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me1_v2_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K4me1_v2_mat_profile.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me1_v2_mat_profile.pdf --refPointLabel center --plotTitle H3K4me1_v2', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp816_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me1_v2_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K4me1_v2_mat_profile_clust3.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me1_v2_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle H3K4me1_v2', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp817_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me3_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K4me3_mat_profile.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me3_mat_profile.pdf --refPointLabel center --plotTitle H3K4me3', returncode=0, stdout=b'', stderr=b'')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp817_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me3_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K4me3_mat_profile_clust3.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me3_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle H3K4me3', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp818_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me3_v2_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K4me3_v2_mat_profile.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me3_v2_mat_profile.pdf --refPointLabel center --plotTitle H3K4me3_v2', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp818_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me3_v2_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K4me3_v2_mat_profile_clust3.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me3_v2_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle H3K4me3_v2', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp819_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K79me2_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K79me2_mat_profile.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K79me2_mat_profile.pdf --refPointLabel center --plotTitle H3K79me2', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp819_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K79me2_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K79me2_mat_profile_clust3.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K79me2_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle H3K79me2', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp820_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K79me2_v2_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K79me2_v2_mat_profile.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K79me2_v2_mat_profile.pdf --refPointLabel center --plotTitle H3K79me2_v2', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp820_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K79me2_v2_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K79me2_v2_mat_profile_clust3.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K79me2_v2_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle H3K79me2_v2', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp821_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp821-MV411_IRF2BP2_VHL_6h-H3K27ac-r1.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27ac_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K27ac_VHL_mat_profile.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27ac_VHL_mat_profile.pdf --refPointLabel center --plotTitle H3K27ac_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp821_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp821-MV411_IRF2BP2_VHL_6h-H3K27ac-r1.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27ac_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K27ac_VHL_mat_profile_clust3.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27ac_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle H3K27ac_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp822_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27ac_v2_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K27ac_v2_VHL_mat_profile.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27ac_v2_VHL_mat_profile.pdf --refPointLabel center --plotTitle H3K27ac_v2_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp822_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27ac_v2_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K27ac_v2_VHL_mat_profile_clust3.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27ac_v2_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle H3K27ac_v2_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp823_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27me3_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K27me3_VHL_mat_profile.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27me3_VHL_mat_profile.pdf --refPointLabel center --plotTitle H3K27me3_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp823_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27me3_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K27me3_VHL_mat_profile_clust3.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27me3_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle H3K27me3_VHL', returncode=0, stdout=b'', stderr=b'')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp824_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27me3_v2_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K27me3_v2_VHL_mat_profile.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27me3_v2_VHL_mat_profile.pdf --refPointLabel center --plotTitle H3K27me3_v2_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp824_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27me3_v2_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K27me3_v2_VHL_mat_profile_clust3.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K27me3_v2_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle H3K27me3_v2_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp825_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me1_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K4me1_VHL_mat_profile.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me1_VHL_mat_profile.pdf --refPointLabel center --plotTitle H3K4me1_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp825_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me1_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K4me1_VHL_mat_profile_clust3.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me1_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle H3K4me1_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp826_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me1_v2_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K4me1_v2_VHL_mat_profile.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me1_v2_VHL_mat_profile.pdf --refPointLabel center --plotTitle H3K4me1_v2_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp826_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me1_v2_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K4me1_v2_VHL_mat_profile_clust3.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me1_v2_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle H3K4me1_v2_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp827_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me3_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K4me3_VHL_mat_profile.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me3_VHL_mat_profile.pdf --refPointLabel center --plotTitle H3K4me3_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp827_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me3_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K4me3_VHL_mat_profile_clust3.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me3_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle H3K4me3_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp828_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me3_v2_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K4me3_v2_VHL_mat_profile.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me3_v2_VHL_mat_profile.pdf --refPointLabel center --plotTitle H3K4me3_v2_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp828_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me3_v2_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K4me3_v2_VHL_mat_profile_clust3.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K4me3_v2_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle H3K4me3_v2_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp829_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K79me2_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K79me2_VHL_mat_profile.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K79me2_VHL_mat_profile.pdf --refPointLabel center --plotTitle H3K79me2_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp829_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K79me2_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K79me2_VHL_mat_profile_clust3.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K79me2_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle H3K79me2_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp830_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K79me2_v2_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K79me2_v2_VHL_mat_profile.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K79me2_v2_VHL_mat_profile.pdf --refPointLabel center --plotTitle H3K79me2_v2_VHL', returncode=0, stdout=b'', stderr=b'')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_hist/mp830_R1.bw  --referencePoint center --regionsFileName ../../data/peakshist/mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2.broadPeak  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks/H3K79me2_v2_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/diffBinding_hist/diffPeaks/H3K79me2_v2_VHL_mat_profile_clust3.gz --outFileName ../../data/diffBinding_hist/diffPeaks/H3K79me2_v2_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle H3K79me2_v2_VHL', returncode=0, stdout=b'', stderr=b'')\n"
     ]
    }
   ],
   "source": [
    "for i in range(len(bw)):\n",
    "    chip.computePeaksAt(peaks[i], bw[i], window=3000, folder=\"\", title=names[i], numthreads=7, torecompute=True, onlyProfile=True, refpoint='center', name='../../data/diffBinding_hist/diffPeaks/'+names[i]+'_mat_profile.pdf', withDeeptools=True)\n",
    "    chip.computePeaksAt(peaks[i], bw[i], window=3000, folder=\"\", title=names[i], numthreads=7, torecompute=True, refpoint='center', onlyProfile=True,name='../../data/diffBinding_hist/diffPeaks/'+names[i]+'_mat_profile_clust3.pdf', withDeeptools=True, cluster=3)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### on unscalled data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 86,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/diffBinding_hist/mp811-MV411_IRF2BP2_DMSO_6h-H3K27ac-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp821-MV411_IRF2BP2_VHL_6h-H3K27ac-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2.mLb.clN.sorted.bam']"
      ]
     },
     "execution_count": 86,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bams = !ls ../../data/diffBinding_hist/mp*.bam\n",
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 209,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/bigwig/mp811-MV411_IRF2BP2_DMSO_6h-H3K27ac-r1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/bigwig/mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/bigwig/mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/bigwig/mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/bigwig/mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/bigwig/mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/bigwig/mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/bigwig/mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/bigwig/mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/bigwig/mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/bigwig/mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/bigwig/mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/bigwig/mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/bigwig/mp821-MV411_IRF2BP2_VHL_6h-H3K27ac-r1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/bigwig/mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/bigwig/mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/bigwig/mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/bigwig/mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/bigwig/mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/bigwig/mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2.mLb.clN.bigWig...\n",
      "- [21/21 files][  5.7 GiB/  5.7 GiB] 100% Done 321.9 MiB/s ETA 00:00:00         \n",
      "Operation completed over 21 objects/5.7 GiB.                                     \n"
     ]
    }
   ],
   "source": [
    "! mkdir../../data/diffBinding_hist/diffPeaks_unscaled\n",
    "!mkdir ../../data/bigwigrephistunscaled/\n",
    "!gsutil -m cp gs://amlproject/Chip/IRF2BP2_degraded_histones/bwa/mergedLibrary/bigwig/*.bigWig ../../data/bigwigrephistunscaled/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 191,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "doing diff from ../../data/diffBinding_hist/mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2.mLb.clN.sorted.bam and ../../data/diffBinding_hist/mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "213\n",
      "computing the scaling values\n",
      "b'INFO  @ Tue, 24 Mar 2020 18:33:07: \\n# Command line: callpeak -B -t ../../data/diffBinding_hist/mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 213 -n mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2 --outdir ../../data/diffBinding_hist/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_hist/mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 24 Mar 2020 18:33:07: #1 read tag files... \\nINFO  @ Tue, 24 Mar 2020 18:33:07: #1 read treatment tags... \\nINFO  @ Tue, 24 Mar 2020 18:33:09:  1000000 \\nINFO  @ Tue, 24 Mar 2020 18:33:11:  2000000 \\nINFO  @ Tue, 24 Mar 2020 18:33:13:  3000000 \\nINFO  @ Tue, 24 Mar 2020 18:33:15:  4000000 \\nINFO  @ Tue, 24 Mar 2020 18:33:17:  5000000 \\nINFO  @ Tue, 24 Mar 2020 18:33:19:  6000000 \\nINFO  @ Tue, 24 Mar 2020 18:33:21:  7000000 \\nINFO  @ Tue, 24 Mar 2020 18:33:23:  8000000 \\nINFO  @ Tue, 24 Mar 2020 18:33:25:  9000000 \\nINFO  @ Tue, 24 Mar 2020 18:33:27:  10000000 \\nINFO  @ Tue, 24 Mar 2020 18:33:30:  11000000 \\nINFO  @ Tue, 24 Mar 2020 18:33:32:  12000000 \\nINFO  @ Tue, 24 Mar 2020 18:33:34:  13000000 \\nINFO  @ Tue, 24 Mar 2020 18:33:36:  14000000 \\nINFO  @ Tue, 24 Mar 2020 18:33:38:  15000000 \\nINFO  @ Tue, 24 Mar 2020 18:33:40:  16000000 \\nINFO  @ Tue, 24 Mar 2020 18:33:42:  17000000 \\nINFO  @ Tue, 24 Mar 2020 18:33:44:  18000000 \\nINFO  @ Tue, 24 Mar 2020 18:33:46:  19000000 \\nINFO  @ Tue, 24 Mar 2020 18:33:48:  20000000 \\nINFO  @ Tue, 24 Mar 2020 18:33:50:  21000000 \\nINFO  @ Tue, 24 Mar 2020 18:33:52:  22000000 \\nINFO  @ Tue, 24 Mar 2020 18:33:54:  23000000 \\nINFO  @ Tue, 24 Mar 2020 18:33:56:  24000000 \\nINFO  @ Tue, 24 Mar 2020 18:33:58:  25000000 \\nINFO  @ Tue, 24 Mar 2020 18:34:00:  26000000 \\nINFO  @ Tue, 24 Mar 2020 18:34:02:  27000000 \\nINFO  @ Tue, 24 Mar 2020 18:34:04:  28000000 \\nINFO  @ Tue, 24 Mar 2020 18:34:06:  29000000 \\nINFO  @ Tue, 24 Mar 2020 18:34:08:  30000000 \\nINFO  @ Tue, 24 Mar 2020 18:34:10:  31000000 \\nINFO  @ Tue, 24 Mar 2020 18:34:12:  32000000 \\nINFO  @ Tue, 24 Mar 2020 18:34:14:  33000000 \\nINFO  @ Tue, 24 Mar 2020 18:34:15: 33132240 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 18:34:15: #1.2 read input tags... \\nINFO  @ Tue, 24 Mar 2020 18:34:18:  1000000 \\nINFO  @ Tue, 24 Mar 2020 18:34:20:  2000000 \\nINFO  @ Tue, 24 Mar 2020 18:34:23:  3000000 \\nINFO  @ Tue, 24 Mar 2020 18:34:25:  4000000 \\nINFO  @ Tue, 24 Mar 2020 18:34:28:  5000000 \\nINFO  @ Tue, 24 Mar 2020 18:34:31:  6000000 \\nINFO  @ Tue, 24 Mar 2020 18:34:33:  7000000 \\nINFO  @ Tue, 24 Mar 2020 18:34:36:  8000000 \\nINFO  @ Tue, 24 Mar 2020 18:34:38:  9000000 \\nINFO  @ Tue, 24 Mar 2020 18:34:40:  10000000 \\nINFO  @ Tue, 24 Mar 2020 18:34:43:  11000000 \\nINFO  @ Tue, 24 Mar 2020 18:34:45:  12000000 \\nINFO  @ Tue, 24 Mar 2020 18:34:48:  13000000 \\nINFO  @ Tue, 24 Mar 2020 18:34:50:  14000000 \\nINFO  @ Tue, 24 Mar 2020 18:34:53:  15000000 \\nINFO  @ Tue, 24 Mar 2020 18:34:55:  16000000 \\nINFO  @ Tue, 24 Mar 2020 18:34:58:  17000000 \\nINFO  @ Tue, 24 Mar 2020 18:35:00:  18000000 \\nINFO  @ Tue, 24 Mar 2020 18:35:02:  19000000 \\nINFO  @ Tue, 24 Mar 2020 18:35:05:  20000000 \\nINFO  @ Tue, 24 Mar 2020 18:35:07:  21000000 \\nINFO  @ Tue, 24 Mar 2020 18:35:10:  22000000 \\nINFO  @ Tue, 24 Mar 2020 18:35:12:  23000000 \\nINFO  @ Tue, 24 Mar 2020 18:35:14:  24000000 \\nINFO  @ Tue, 24 Mar 2020 18:35:17:  25000000 \\nINFO  @ Tue, 24 Mar 2020 18:35:19:  26000000 \\nINFO  @ Tue, 24 Mar 2020 18:35:22:  27000000 \\nINFO  @ Tue, 24 Mar 2020 18:35:24:  28000000 \\nINFO  @ Tue, 24 Mar 2020 18:35:27:  29000000 \\nINFO  @ Tue, 24 Mar 2020 18:35:29:  30000000 \\nINFO  @ Tue, 24 Mar 2020 18:35:31:  31000000 \\nINFO  @ Tue, 24 Mar 2020 18:35:34:  32000000 \\nINFO  @ Tue, 24 Mar 2020 18:35:36:  33000000 \\nINFO  @ Tue, 24 Mar 2020 18:35:39:  34000000 \\nINFO  @ Tue, 24 Mar 2020 18:35:40: 34520959 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 18:35:40: #1 tag size is determined as 49 bps \\nINFO  @ Tue, 24 Mar 2020 18:35:40: #1 tag size = 49.0 \\nINFO  @ Tue, 24 Mar 2020 18:35:40: #1  total tags in treatment: 33132240 \\nINFO  @ Tue, 24 Mar 2020 18:35:40: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 18:35:40: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 18:35:41: #1  tags after filtering in treatment: 33109730 \\nINFO  @ Tue, 24 Mar 2020 18:35:41: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Tue, 24 Mar 2020 18:35:41: #1  total tags in control: 34520959 \\nINFO  @ Tue, 24 Mar 2020 18:35:41: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 18:35:41: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 18:35:41: #1  tags after filtering in control: 34516729 \\nINFO  @ Tue, 24 Mar 2020 18:35:41: #1  Redundant rate of control: 0.00 \\nINFO  @ Tue, 24 Mar 2020 18:35:41: #1 finished! \\nINFO  @ Tue, 24 Mar 2020 18:35:41: #2 Build Peak Model... \\nINFO  @ Tue, 24 Mar 2020 18:35:41: #2 Skipped... \\nINFO  @ Tue, 24 Mar 2020 18:35:41: #2 Use 213 as fragment length \\nINFO  @ Tue, 24 Mar 2020 18:35:41: #3 Call peaks... \\nINFO  @ Tue, 24 Mar 2020 18:35:41: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 24 Mar 2020 18:38:42: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 24 Mar 2020 18:38:42: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2_treat_pileup.bdg \\nINFO  @ Tue, 24 Mar 2020 18:38:42: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2_control_lambda.bdg \\nINFO  @ Tue, 24 Mar 2020 18:38:42: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Tue, 24 Mar 2020 18:38:42: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 24 Mar 2020 18:41:29: #4 Write output xls file... ../../data/diffBinding_hist/diffData_unscaled/mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2_peaks.xls \\nINFO  @ Tue, 24 Mar 2020 18:41:29: #4 Write peak in narrowPeak format file... ../../data/diffBinding_hist/diffData_unscaled/mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2_peaks.narrowPeak \\nINFO  @ Tue, 24 Mar 2020 18:41:29: #4 Write summits bed file... ../../data/diffBinding_hist/diffData_unscaled/mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2_summits.bed \\nINFO  @ Tue, 24 Mar 2020 18:41:29: Done! \\n'\n",
      "b'INFO  @ Tue, 24 Mar 2020 18:41:30: \\n# Command line: callpeak -B -t ../../data/diffBinding_hist/mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 213 -n mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2 --outdir ../../data/diffBinding_hist/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_hist/mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 24 Mar 2020 18:41:30: #1 read tag files... \\nINFO  @ Tue, 24 Mar 2020 18:41:30: #1 read treatment tags... \\nINFO  @ Tue, 24 Mar 2020 18:41:32:  1000000 \\nINFO  @ Tue, 24 Mar 2020 18:41:34:  2000000 \\nINFO  @ Tue, 24 Mar 2020 18:41:37:  3000000 \\nINFO  @ Tue, 24 Mar 2020 18:41:39:  4000000 \\nINFO  @ Tue, 24 Mar 2020 18:41:41:  5000000 \\nINFO  @ Tue, 24 Mar 2020 18:41:43:  6000000 \\nINFO  @ Tue, 24 Mar 2020 18:41:45:  7000000 \\nINFO  @ Tue, 24 Mar 2020 18:41:47:  8000000 \\nINFO  @ Tue, 24 Mar 2020 18:41:50:  9000000 \\nINFO  @ Tue, 24 Mar 2020 18:41:52:  10000000 \\nINFO  @ Tue, 24 Mar 2020 18:41:54:  11000000 \\nINFO  @ Tue, 24 Mar 2020 18:41:56:  12000000 \\nINFO  @ Tue, 24 Mar 2020 18:41:58:  13000000 \\nINFO  @ Tue, 24 Mar 2020 18:42:00:  14000000 \\nINFO  @ Tue, 24 Mar 2020 18:42:02:  15000000 \\nINFO  @ Tue, 24 Mar 2020 18:42:04:  16000000 \\nINFO  @ Tue, 24 Mar 2020 18:42:07:  17000000 \\nINFO  @ Tue, 24 Mar 2020 18:42:10:  18000000 \\nINFO  @ Tue, 24 Mar 2020 18:42:12:  19000000 \\nINFO  @ Tue, 24 Mar 2020 18:42:14:  20000000 \\nINFO  @ Tue, 24 Mar 2020 18:42:17:  21000000 \\nINFO  @ Tue, 24 Mar 2020 18:42:19:  22000000 \\nINFO  @ Tue, 24 Mar 2020 18:42:21:  23000000 \\nINFO  @ Tue, 24 Mar 2020 18:42:23:  24000000 \\nINFO  @ Tue, 24 Mar 2020 18:42:25:  25000000 \\nINFO  @ Tue, 24 Mar 2020 18:42:30:  26000000 \\nINFO  @ Tue, 24 Mar 2020 18:42:32:  27000000 \\nINFO  @ Tue, 24 Mar 2020 18:42:34:  28000000 \\nINFO  @ Tue, 24 Mar 2020 18:42:36:  29000000 \\nINFO  @ Tue, 24 Mar 2020 18:42:38:  30000000 \\nINFO  @ Tue, 24 Mar 2020 18:42:40:  31000000 \\nINFO  @ Tue, 24 Mar 2020 18:42:41: 31302154 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 18:42:41: #1.2 read input tags... \\nINFO  @ Tue, 24 Mar 2020 18:42:43:  1000000 \\nINFO  @ Tue, 24 Mar 2020 18:42:46:  2000000 \\nINFO  @ Tue, 24 Mar 2020 18:42:48:  3000000 \\nINFO  @ Tue, 24 Mar 2020 18:42:50:  4000000 \\nINFO  @ Tue, 24 Mar 2020 18:42:53:  5000000 \\nINFO  @ Tue, 24 Mar 2020 18:42:55:  6000000 \\nINFO  @ Tue, 24 Mar 2020 18:42:57:  7000000 \\nINFO  @ Tue, 24 Mar 2020 18:43:00:  8000000 \\nINFO  @ Tue, 24 Mar 2020 18:43:02:  9000000 \\nINFO  @ Tue, 24 Mar 2020 18:43:04:  10000000 \\nINFO  @ Tue, 24 Mar 2020 18:43:06:  11000000 \\nINFO  @ Tue, 24 Mar 2020 18:43:09:  12000000 \\nINFO  @ Tue, 24 Mar 2020 18:43:11:  13000000 \\nINFO  @ Tue, 24 Mar 2020 18:43:13:  14000000 \\nINFO  @ Tue, 24 Mar 2020 18:43:16:  15000000 \\nINFO  @ Tue, 24 Mar 2020 18:43:18:  16000000 \\nINFO  @ Tue, 24 Mar 2020 18:43:20:  17000000 \\nINFO  @ Tue, 24 Mar 2020 18:43:22:  18000000 \\nINFO  @ Tue, 24 Mar 2020 18:43:25:  19000000 \\nINFO  @ Tue, 24 Mar 2020 18:43:27:  20000000 \\nINFO  @ Tue, 24 Mar 2020 18:43:29:  21000000 \\nINFO  @ Tue, 24 Mar 2020 18:43:32:  22000000 \\nINFO  @ Tue, 24 Mar 2020 18:43:34:  23000000 \\nINFO  @ Tue, 24 Mar 2020 18:43:36:  24000000 \\nINFO  @ Tue, 24 Mar 2020 18:43:38:  25000000 \\nINFO  @ Tue, 24 Mar 2020 18:43:41:  26000000 \\nINFO  @ Tue, 24 Mar 2020 18:43:43:  27000000 \\nINFO  @ Tue, 24 Mar 2020 18:43:45:  28000000 \\nINFO  @ Tue, 24 Mar 2020 18:43:48:  29000000 \\nINFO  @ Tue, 24 Mar 2020 18:43:50:  30000000 \\nINFO  @ Tue, 24 Mar 2020 18:43:52:  31000000 \\nINFO  @ Tue, 24 Mar 2020 18:43:55:  32000000 \\nINFO  @ Tue, 24 Mar 2020 18:43:57:  33000000 \\nINFO  @ Tue, 24 Mar 2020 18:43:59:  34000000 \\nINFO  @ Tue, 24 Mar 2020 18:44:01: 34520959 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 18:44:01: #1 tag size is determined as 49 bps \\nINFO  @ Tue, 24 Mar 2020 18:44:01: #1 tag size = 49.0 \\nINFO  @ Tue, 24 Mar 2020 18:44:01: #1  total tags in treatment: 31302154 \\nINFO  @ Tue, 24 Mar 2020 18:44:01: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 18:44:01: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 18:44:01: #1  tags after filtering in treatment: 31281399 \\nINFO  @ Tue, 24 Mar 2020 18:44:01: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Tue, 24 Mar 2020 18:44:01: #1  total tags in control: 34520959 \\nINFO  @ Tue, 24 Mar 2020 18:44:01: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 18:44:01: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 18:44:01: #1  tags after filtering in control: 34516729 \\nINFO  @ Tue, 24 Mar 2020 18:44:01: #1  Redundant rate of control: 0.00 \\nINFO  @ Tue, 24 Mar 2020 18:44:01: #1 finished! \\nINFO  @ Tue, 24 Mar 2020 18:44:01: #2 Build Peak Model... \\nINFO  @ Tue, 24 Mar 2020 18:44:01: #2 Skipped... \\nINFO  @ Tue, 24 Mar 2020 18:44:01: #2 Use 213 as fragment length \\nINFO  @ Tue, 24 Mar 2020 18:44:01: #3 Call peaks... \\nINFO  @ Tue, 24 Mar 2020 18:44:01: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 24 Mar 2020 18:46:51: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 24 Mar 2020 18:46:51: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2_treat_pileup.bdg \\nINFO  @ Tue, 24 Mar 2020 18:46:51: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2_control_lambda.bdg \\nINFO  @ Tue, 24 Mar 2020 18:46:51: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Tue, 24 Mar 2020 18:46:51: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 24 Mar 2020 18:49:34: #4 Write output xls file... ../../data/diffBinding_hist/diffData_unscaled/mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2_peaks.xls \\nINFO  @ Tue, 24 Mar 2020 18:49:35: #4 Write peak in narrowPeak format file... ../../data/diffBinding_hist/diffData_unscaled/mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2_peaks.narrowPeak \\nINFO  @ Tue, 24 Mar 2020 18:49:35: #4 Write summits bed file... ../../data/diffBinding_hist/diffData_unscaled/mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2_summits.bed \\nINFO  @ Tue, 24 Mar 2020 18:49:35: Done! \\n'\n",
      "33109730 31281399\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "doing diff from ../../data/diffBinding_hist/mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1.mLb.clN.sorted.bam and ../../data/diffBinding_hist/mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "47\n",
      "computing the scaling values\n",
      "b'INFO  @ Tue, 24 Mar 2020 19:00:06: \\n# Command line: callpeak -B -t ../../data/diffBinding_hist/mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 47 -n mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1 --outdir ../../data/diffBinding_hist/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_hist/mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 24 Mar 2020 19:00:06: #1 read tag files... \\nINFO  @ Tue, 24 Mar 2020 19:00:06: #1 read treatment tags... \\nINFO  @ Tue, 24 Mar 2020 19:00:08:  1000000 \\nINFO  @ Tue, 24 Mar 2020 19:00:10:  2000000 \\nINFO  @ Tue, 24 Mar 2020 19:00:12:  3000000 \\nINFO  @ Tue, 24 Mar 2020 19:00:14:  4000000 \\nINFO  @ Tue, 24 Mar 2020 19:00:16:  5000000 \\nINFO  @ Tue, 24 Mar 2020 19:00:18:  6000000 \\nINFO  @ Tue, 24 Mar 2020 19:00:21:  7000000 \\nINFO  @ Tue, 24 Mar 2020 19:00:23:  8000000 \\nINFO  @ Tue, 24 Mar 2020 19:00:25:  9000000 \\nINFO  @ Tue, 24 Mar 2020 19:00:27:  10000000 \\nINFO  @ Tue, 24 Mar 2020 19:00:29:  11000000 \\nINFO  @ Tue, 24 Mar 2020 19:00:31:  12000000 \\nINFO  @ Tue, 24 Mar 2020 19:00:33:  13000000 \\nINFO  @ Tue, 24 Mar 2020 19:00:35:  14000000 \\nINFO  @ Tue, 24 Mar 2020 19:00:38:  15000000 \\nINFO  @ Tue, 24 Mar 2020 19:00:40:  16000000 \\nINFO  @ Tue, 24 Mar 2020 19:00:42:  17000000 \\nINFO  @ Tue, 24 Mar 2020 19:00:44:  18000000 \\nINFO  @ Tue, 24 Mar 2020 19:00:46:  19000000 \\nINFO  @ Tue, 24 Mar 2020 19:00:48:  20000000 \\nINFO  @ Tue, 24 Mar 2020 19:00:51:  21000000 \\nINFO  @ Tue, 24 Mar 2020 19:00:53:  22000000 \\nINFO  @ Tue, 24 Mar 2020 19:00:55:  23000000 \\nINFO  @ Tue, 24 Mar 2020 19:00:56: 23387460 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 19:00:56: #1.2 read input tags... \\nINFO  @ Tue, 24 Mar 2020 19:00:58:  1000000 \\nINFO  @ Tue, 24 Mar 2020 19:01:01:  2000000 \\nINFO  @ Tue, 24 Mar 2020 19:01:03:  3000000 \\nINFO  @ Tue, 24 Mar 2020 19:01:05:  4000000 \\nINFO  @ Tue, 24 Mar 2020 19:01:08:  5000000 \\nINFO  @ Tue, 24 Mar 2020 19:01:10:  6000000 \\nINFO  @ Tue, 24 Mar 2020 19:01:12:  7000000 \\nINFO  @ Tue, 24 Mar 2020 19:01:14:  8000000 \\nINFO  @ Tue, 24 Mar 2020 19:01:17:  9000000 \\nINFO  @ Tue, 24 Mar 2020 19:01:19:  10000000 \\nINFO  @ Tue, 24 Mar 2020 19:01:21:  11000000 \\nINFO  @ Tue, 24 Mar 2020 19:01:24:  12000000 \\nINFO  @ Tue, 24 Mar 2020 19:01:26:  13000000 \\nINFO  @ Tue, 24 Mar 2020 19:01:28:  14000000 \\nINFO  @ Tue, 24 Mar 2020 19:01:31:  15000000 \\nINFO  @ Tue, 24 Mar 2020 19:01:33:  16000000 \\nINFO  @ Tue, 24 Mar 2020 19:01:35:  17000000 \\nINFO  @ Tue, 24 Mar 2020 19:01:38:  18000000 \\nINFO  @ Tue, 24 Mar 2020 19:01:40:  19000000 \\nINFO  @ Tue, 24 Mar 2020 19:01:43:  20000000 \\nINFO  @ Tue, 24 Mar 2020 19:01:45:  21000000 \\nINFO  @ Tue, 24 Mar 2020 19:01:47:  22000000 \\nINFO  @ Tue, 24 Mar 2020 19:01:50:  23000000 \\nINFO  @ Tue, 24 Mar 2020 19:01:52:  24000000 \\nINFO  @ Tue, 24 Mar 2020 19:01:54:  25000000 \\nINFO  @ Tue, 24 Mar 2020 19:01:57:  26000000 \\nINFO  @ Tue, 24 Mar 2020 19:01:59:  27000000 \\nINFO  @ Tue, 24 Mar 2020 19:02:02:  28000000 \\nINFO  @ Tue, 24 Mar 2020 19:02:04:  29000000 \\nINFO  @ Tue, 24 Mar 2020 19:02:06:  30000000 \\nINFO  @ Tue, 24 Mar 2020 19:02:09:  31000000 \\nINFO  @ Tue, 24 Mar 2020 19:02:11:  32000000 \\nINFO  @ Tue, 24 Mar 2020 19:02:14:  33000000 \\nINFO  @ Tue, 24 Mar 2020 19:02:16:  34000000 \\nINFO  @ Tue, 24 Mar 2020 19:02:18: 34520959 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 19:02:18: #1 tag size is determined as 49 bps \\nINFO  @ Tue, 24 Mar 2020 19:02:18: #1 tag size = 49.0 \\nINFO  @ Tue, 24 Mar 2020 19:02:18: #1  total tags in treatment: 23387460 \\nINFO  @ Tue, 24 Mar 2020 19:02:18: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 19:02:18: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 19:02:18: #1  tags after filtering in treatment: 23384886 \\nINFO  @ Tue, 24 Mar 2020 19:02:18: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Tue, 24 Mar 2020 19:02:18: #1  total tags in control: 34520959 \\nINFO  @ Tue, 24 Mar 2020 19:02:18: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 19:02:18: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 19:02:18: #1  tags after filtering in control: 34516729 \\nINFO  @ Tue, 24 Mar 2020 19:02:18: #1  Redundant rate of control: 0.00 \\nINFO  @ Tue, 24 Mar 2020 19:02:18: #1 finished! \\nINFO  @ Tue, 24 Mar 2020 19:02:18: #2 Build Peak Model... \\nINFO  @ Tue, 24 Mar 2020 19:02:18: #2 Skipped... \\nINFO  @ Tue, 24 Mar 2020 19:02:18: #2 Use 47 as fragment length \\nINFO  @ Tue, 24 Mar 2020 19:02:18: #3 Call peaks... \\nINFO  @ Tue, 24 Mar 2020 19:02:18: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 24 Mar 2020 19:05:13: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 24 Mar 2020 19:05:13: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1_treat_pileup.bdg \\nINFO  @ Tue, 24 Mar 2020 19:05:13: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1_control_lambda.bdg \\nINFO  @ Tue, 24 Mar 2020 19:05:13: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Tue, 24 Mar 2020 19:05:13: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 24 Mar 2020 19:07:52: #4 Write output xls file... ../../data/diffBinding_hist/diffData_unscaled/mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1_peaks.xls \\nINFO  @ Tue, 24 Mar 2020 19:07:52: #4 Write peak in narrowPeak format file... ../../data/diffBinding_hist/diffData_unscaled/mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1_peaks.narrowPeak \\nINFO  @ Tue, 24 Mar 2020 19:07:52: #4 Write summits bed file... ../../data/diffBinding_hist/diffData_unscaled/mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1_summits.bed \\nINFO  @ Tue, 24 Mar 2020 19:07:52: Done! \\n'\n",
      "b'INFO  @ Tue, 24 Mar 2020 19:07:53: \\n# Command line: callpeak -B -t ../../data/diffBinding_hist/mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 47 -n mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1 --outdir ../../data/diffBinding_hist/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_hist/mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 24 Mar 2020 19:07:53: #1 read tag files... \\nINFO  @ Tue, 24 Mar 2020 19:07:53: #1 read treatment tags... \\nINFO  @ Tue, 24 Mar 2020 19:07:55:  1000000 \\nINFO  @ Tue, 24 Mar 2020 19:07:58:  2000000 \\nINFO  @ Tue, 24 Mar 2020 19:08:00:  3000000 \\nINFO  @ Tue, 24 Mar 2020 19:08:03:  4000000 \\nINFO  @ Tue, 24 Mar 2020 19:08:05:  5000000 \\nINFO  @ Tue, 24 Mar 2020 19:08:08:  6000000 \\nINFO  @ Tue, 24 Mar 2020 19:08:10:  7000000 \\nINFO  @ Tue, 24 Mar 2020 19:08:13:  8000000 \\nINFO  @ Tue, 24 Mar 2020 19:08:15:  9000000 \\nINFO  @ Tue, 24 Mar 2020 19:08:17:  10000000 \\nINFO  @ Tue, 24 Mar 2020 19:08:20:  11000000 \\nINFO  @ Tue, 24 Mar 2020 19:08:22:  12000000 \\nINFO  @ Tue, 24 Mar 2020 19:08:25:  13000000 \\nINFO  @ Tue, 24 Mar 2020 19:08:27:  14000000 \\nINFO  @ Tue, 24 Mar 2020 19:08:29:  15000000 \\nINFO  @ Tue, 24 Mar 2020 19:08:32:  16000000 \\nINFO  @ Tue, 24 Mar 2020 19:08:34:  17000000 \\nINFO  @ Tue, 24 Mar 2020 19:08:36:  18000000 \\nINFO  @ Tue, 24 Mar 2020 19:08:39:  19000000 \\nINFO  @ Tue, 24 Mar 2020 19:08:39: 19097884 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 19:08:39: #1.2 read input tags... \\nINFO  @ Tue, 24 Mar 2020 19:08:42:  1000000 \\nINFO  @ Tue, 24 Mar 2020 19:08:44:  2000000 \\nINFO  @ Tue, 24 Mar 2020 19:08:47:  3000000 \\nINFO  @ Tue, 24 Mar 2020 19:08:49:  4000000 \\nINFO  @ Tue, 24 Mar 2020 19:08:51:  5000000 \\nINFO  @ Tue, 24 Mar 2020 19:08:54:  6000000 \\nINFO  @ Tue, 24 Mar 2020 19:08:56:  7000000 \\nINFO  @ Tue, 24 Mar 2020 19:08:59:  8000000 \\nINFO  @ Tue, 24 Mar 2020 19:09:01:  9000000 \\nINFO  @ Tue, 24 Mar 2020 19:09:03:  10000000 \\nINFO  @ Tue, 24 Mar 2020 19:09:06:  11000000 \\nINFO  @ Tue, 24 Mar 2020 19:09:08:  12000000 \\nINFO  @ Tue, 24 Mar 2020 19:09:11:  13000000 \\nINFO  @ Tue, 24 Mar 2020 19:09:13:  14000000 \\nINFO  @ Tue, 24 Mar 2020 19:09:16:  15000000 \\nINFO  @ Tue, 24 Mar 2020 19:09:18:  16000000 \\nINFO  @ Tue, 24 Mar 2020 19:09:20:  17000000 \\nINFO  @ Tue, 24 Mar 2020 19:09:23:  18000000 \\nINFO  @ Tue, 24 Mar 2020 19:09:25:  19000000 \\nINFO  @ Tue, 24 Mar 2020 19:09:28:  20000000 \\nINFO  @ Tue, 24 Mar 2020 19:09:30:  21000000 \\nINFO  @ Tue, 24 Mar 2020 19:09:33:  22000000 \\nINFO  @ Tue, 24 Mar 2020 19:09:35:  23000000 \\nINFO  @ Tue, 24 Mar 2020 19:09:38:  24000000 \\nINFO  @ Tue, 24 Mar 2020 19:09:40:  25000000 \\nINFO  @ Tue, 24 Mar 2020 19:09:42:  26000000 \\nINFO  @ Tue, 24 Mar 2020 19:09:45:  27000000 \\nINFO  @ Tue, 24 Mar 2020 19:09:47:  28000000 \\nINFO  @ Tue, 24 Mar 2020 19:09:50:  29000000 \\nINFO  @ Tue, 24 Mar 2020 19:09:52:  30000000 \\nINFO  @ Tue, 24 Mar 2020 19:09:55:  31000000 \\nINFO  @ Tue, 24 Mar 2020 19:09:57:  32000000 \\nINFO  @ Tue, 24 Mar 2020 19:10:00:  33000000 \\nINFO  @ Tue, 24 Mar 2020 19:10:02:  34000000 \\nINFO  @ Tue, 24 Mar 2020 19:10:04: 34520959 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 19:10:04: #1 tag size is determined as 48 bps \\nINFO  @ Tue, 24 Mar 2020 19:10:04: #1 tag size = 48.0 \\nINFO  @ Tue, 24 Mar 2020 19:10:04: #1  total tags in treatment: 19097884 \\nINFO  @ Tue, 24 Mar 2020 19:10:04: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 19:10:04: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 19:10:04: #1  tags after filtering in treatment: 19095976 \\nINFO  @ Tue, 24 Mar 2020 19:10:04: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Tue, 24 Mar 2020 19:10:04: #1  total tags in control: 34520959 \\nINFO  @ Tue, 24 Mar 2020 19:10:04: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 19:10:04: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 19:10:04: #1  tags after filtering in control: 34516729 \\nINFO  @ Tue, 24 Mar 2020 19:10:04: #1  Redundant rate of control: 0.00 \\nINFO  @ Tue, 24 Mar 2020 19:10:04: #1 finished! \\nINFO  @ Tue, 24 Mar 2020 19:10:04: #2 Build Peak Model... \\nINFO  @ Tue, 24 Mar 2020 19:10:04: #2 Skipped... \\nINFO  @ Tue, 24 Mar 2020 19:10:04: #2 Use 47 as fragment length \\nINFO  @ Tue, 24 Mar 2020 19:10:04: #3 Call peaks... \\nINFO  @ Tue, 24 Mar 2020 19:10:04: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 24 Mar 2020 19:12:46: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 24 Mar 2020 19:12:46: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1_treat_pileup.bdg \\nINFO  @ Tue, 24 Mar 2020 19:12:46: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1_control_lambda.bdg \\nINFO  @ Tue, 24 Mar 2020 19:12:46: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Tue, 24 Mar 2020 19:12:46: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 24 Mar 2020 19:15:29: #4 Write output xls file... ../../data/diffBinding_hist/diffData_unscaled/mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1_peaks.xls \\nINFO  @ Tue, 24 Mar 2020 19:15:29: #4 Write peak in narrowPeak format file... ../../data/diffBinding_hist/diffData_unscaled/mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1_peaks.narrowPeak \\nINFO  @ Tue, 24 Mar 2020 19:15:29: #4 Write summits bed file... ../../data/diffBinding_hist/diffData_unscaled/mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1_summits.bed \\nINFO  @ Tue, 24 Mar 2020 19:15:29: Done! \\n'\n",
      "23384886 19095976\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "doing diff from ../../data/diffBinding_hist/mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2.mLb.clN.sorted.bam and ../../data/diffBinding_hist/mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "272\n",
      "computing the scaling values\n",
      "b'INFO  @ Tue, 24 Mar 2020 19:25:04: \\n# Command line: callpeak -B -t ../../data/diffBinding_hist/mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 272 -n mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2 --outdir ../../data/diffBinding_hist/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_hist/mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 24 Mar 2020 19:25:04: #1 read tag files... \\nINFO  @ Tue, 24 Mar 2020 19:25:04: #1 read treatment tags... \\nINFO  @ Tue, 24 Mar 2020 19:25:06:  1000000 \\nINFO  @ Tue, 24 Mar 2020 19:25:08:  2000000 \\nINFO  @ Tue, 24 Mar 2020 19:25:10:  3000000 \\nINFO  @ Tue, 24 Mar 2020 19:25:12:  4000000 \\nINFO  @ Tue, 24 Mar 2020 19:25:15:  5000000 \\nINFO  @ Tue, 24 Mar 2020 19:25:17:  6000000 \\nINFO  @ Tue, 24 Mar 2020 19:25:19:  7000000 \\nINFO  @ Tue, 24 Mar 2020 19:25:21:  8000000 \\nINFO  @ Tue, 24 Mar 2020 19:25:23:  9000000 \\nINFO  @ Tue, 24 Mar 2020 19:25:25:  10000000 \\nINFO  @ Tue, 24 Mar 2020 19:25:27:  11000000 \\nINFO  @ Tue, 24 Mar 2020 19:25:29:  12000000 \\nINFO  @ Tue, 24 Mar 2020 19:25:31:  13000000 \\nINFO  @ Tue, 24 Mar 2020 19:25:34:  14000000 \\nINFO  @ Tue, 24 Mar 2020 19:25:36:  15000000 \\nINFO  @ Tue, 24 Mar 2020 19:25:38:  16000000 \\nINFO  @ Tue, 24 Mar 2020 19:25:40:  17000000 \\nINFO  @ Tue, 24 Mar 2020 19:25:42:  18000000 \\nINFO  @ Tue, 24 Mar 2020 19:25:44:  19000000 \\nINFO  @ Tue, 24 Mar 2020 19:25:46:  20000000 \\nINFO  @ Tue, 24 Mar 2020 19:25:49:  21000000 \\nINFO  @ Tue, 24 Mar 2020 19:25:50: 21425292 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 19:25:50: #1.2 read input tags... \\nINFO  @ Tue, 24 Mar 2020 19:25:52:  1000000 \\nINFO  @ Tue, 24 Mar 2020 19:25:54:  2000000 \\nINFO  @ Tue, 24 Mar 2020 19:25:57:  3000000 \\nINFO  @ Tue, 24 Mar 2020 19:25:59:  4000000 \\nINFO  @ Tue, 24 Mar 2020 19:26:01:  5000000 \\nINFO  @ Tue, 24 Mar 2020 19:26:03:  6000000 \\nINFO  @ Tue, 24 Mar 2020 19:26:06:  7000000 \\nINFO  @ Tue, 24 Mar 2020 19:26:08:  8000000 \\nINFO  @ Tue, 24 Mar 2020 19:26:10:  9000000 \\nINFO  @ Tue, 24 Mar 2020 19:26:12:  10000000 \\nINFO  @ Tue, 24 Mar 2020 19:26:15:  11000000 \\nINFO  @ Tue, 24 Mar 2020 19:26:17:  12000000 \\nINFO  @ Tue, 24 Mar 2020 19:26:19:  13000000 \\nINFO  @ Tue, 24 Mar 2020 19:26:22:  14000000 \\nINFO  @ Tue, 24 Mar 2020 19:26:24:  15000000 \\nINFO  @ Tue, 24 Mar 2020 19:26:26:  16000000 \\nINFO  @ Tue, 24 Mar 2020 19:26:28:  17000000 \\nINFO  @ Tue, 24 Mar 2020 19:26:31:  18000000 \\nINFO  @ Tue, 24 Mar 2020 19:26:33:  19000000 \\nINFO  @ Tue, 24 Mar 2020 19:26:35:  20000000 \\nINFO  @ Tue, 24 Mar 2020 19:26:37:  21000000 \\nINFO  @ Tue, 24 Mar 2020 19:26:40:  22000000 \\nINFO  @ Tue, 24 Mar 2020 19:26:42:  23000000 \\nINFO  @ Tue, 24 Mar 2020 19:26:44:  24000000 \\nINFO  @ Tue, 24 Mar 2020 19:26:46:  25000000 \\nINFO  @ Tue, 24 Mar 2020 19:26:49:  26000000 \\nINFO  @ Tue, 24 Mar 2020 19:26:52:  27000000 \\nINFO  @ Tue, 24 Mar 2020 19:26:56:  28000000 \\nINFO  @ Tue, 24 Mar 2020 19:26:59:  29000000 \\nINFO  @ Tue, 24 Mar 2020 19:27:01:  30000000 \\nINFO  @ Tue, 24 Mar 2020 19:27:03:  31000000 \\nINFO  @ Tue, 24 Mar 2020 19:27:06:  32000000 \\nINFO  @ Tue, 24 Mar 2020 19:27:08:  33000000 \\nINFO  @ Tue, 24 Mar 2020 19:27:10:  34000000 \\nINFO  @ Tue, 24 Mar 2020 19:27:12: 34520959 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 19:27:12: #1 tag size is determined as 49 bps \\nINFO  @ Tue, 24 Mar 2020 19:27:12: #1 tag size = 49.0 \\nINFO  @ Tue, 24 Mar 2020 19:27:12: #1  total tags in treatment: 21425292 \\nINFO  @ Tue, 24 Mar 2020 19:27:12: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 19:27:12: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 19:27:12: #1  tags after filtering in treatment: 21423156 \\nINFO  @ Tue, 24 Mar 2020 19:27:12: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Tue, 24 Mar 2020 19:27:12: #1  total tags in control: 34520959 \\nINFO  @ Tue, 24 Mar 2020 19:27:12: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 19:27:12: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 19:27:12: #1  tags after filtering in control: 34516729 \\nINFO  @ Tue, 24 Mar 2020 19:27:12: #1  Redundant rate of control: 0.00 \\nINFO  @ Tue, 24 Mar 2020 19:27:12: #1 finished! \\nINFO  @ Tue, 24 Mar 2020 19:27:12: #2 Build Peak Model... \\nINFO  @ Tue, 24 Mar 2020 19:27:12: #2 Skipped... \\nINFO  @ Tue, 24 Mar 2020 19:27:12: #2 Use 272 as fragment length \\nINFO  @ Tue, 24 Mar 2020 19:27:12: #3 Call peaks... \\nINFO  @ Tue, 24 Mar 2020 19:27:12: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 24 Mar 2020 19:30:00: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 24 Mar 2020 19:30:00: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2_treat_pileup.bdg \\nINFO  @ Tue, 24 Mar 2020 19:30:00: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2_control_lambda.bdg \\nINFO  @ Tue, 24 Mar 2020 19:30:00: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Tue, 24 Mar 2020 19:30:00: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 24 Mar 2020 19:32:22: #4 Write output xls file... ../../data/diffBinding_hist/diffData_unscaled/mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2_peaks.xls \\nINFO  @ Tue, 24 Mar 2020 19:32:23: #4 Write peak in narrowPeak format file... ../../data/diffBinding_hist/diffData_unscaled/mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2_peaks.narrowPeak \\nINFO  @ Tue, 24 Mar 2020 19:32:23: #4 Write summits bed file... ../../data/diffBinding_hist/diffData_unscaled/mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2_summits.bed \\nINFO  @ Tue, 24 Mar 2020 19:32:23: Done! \\n'\n",
      "b'INFO  @ Tue, 24 Mar 2020 19:32:23: \\n# Command line: callpeak -B -t ../../data/diffBinding_hist/mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 272 -n mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2 --outdir ../../data/diffBinding_hist/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_hist/mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 24 Mar 2020 19:32:23: #1 read tag files... \\nINFO  @ Tue, 24 Mar 2020 19:32:23: #1 read treatment tags... \\nINFO  @ Tue, 24 Mar 2020 19:32:26:  1000000 \\nINFO  @ Tue, 24 Mar 2020 19:32:28:  2000000 \\nINFO  @ Tue, 24 Mar 2020 19:32:30:  3000000 \\nINFO  @ Tue, 24 Mar 2020 19:32:33:  4000000 \\nINFO  @ Tue, 24 Mar 2020 19:32:35:  5000000 \\nINFO  @ Tue, 24 Mar 2020 19:32:37:  6000000 \\nINFO  @ Tue, 24 Mar 2020 19:32:40:  7000000 \\nINFO  @ Tue, 24 Mar 2020 19:32:42:  8000000 \\nINFO  @ Tue, 24 Mar 2020 19:32:44:  9000000 \\nINFO  @ Tue, 24 Mar 2020 19:32:46:  10000000 \\nINFO  @ Tue, 24 Mar 2020 19:32:49:  11000000 \\nINFO  @ Tue, 24 Mar 2020 19:32:51:  12000000 \\nINFO  @ Tue, 24 Mar 2020 19:32:53:  13000000 \\nINFO  @ Tue, 24 Mar 2020 19:32:56:  14000000 \\nINFO  @ Tue, 24 Mar 2020 19:32:58:  15000000 \\nINFO  @ Tue, 24 Mar 2020 19:33:00:  16000000 \\nINFO  @ Tue, 24 Mar 2020 19:33:03:  17000000 \\nINFO  @ Tue, 24 Mar 2020 19:33:05:  18000000 \\nINFO  @ Tue, 24 Mar 2020 19:33:07:  19000000 \\nINFO  @ Tue, 24 Mar 2020 19:33:09:  20000000 \\nINFO  @ Tue, 24 Mar 2020 19:33:11: 20684363 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 19:33:11: #1.2 read input tags... \\nINFO  @ Tue, 24 Mar 2020 19:33:14:  1000000 \\nINFO  @ Tue, 24 Mar 2020 19:33:16:  2000000 \\nINFO  @ Tue, 24 Mar 2020 19:33:18:  3000000 \\nINFO  @ Tue, 24 Mar 2020 19:33:21:  4000000 \\nINFO  @ Tue, 24 Mar 2020 19:33:23:  5000000 \\nINFO  @ Tue, 24 Mar 2020 19:33:25:  6000000 \\nINFO  @ Tue, 24 Mar 2020 19:33:27:  7000000 \\nINFO  @ Tue, 24 Mar 2020 19:33:30:  8000000 \\nINFO  @ Tue, 24 Mar 2020 19:33:32:  9000000 \\nINFO  @ Tue, 24 Mar 2020 19:33:34:  10000000 \\nINFO  @ Tue, 24 Mar 2020 19:33:37:  11000000 \\nINFO  @ Tue, 24 Mar 2020 19:33:39:  12000000 \\nINFO  @ Tue, 24 Mar 2020 19:33:41:  13000000 \\nINFO  @ Tue, 24 Mar 2020 19:33:44:  14000000 \\nINFO  @ Tue, 24 Mar 2020 19:33:46:  15000000 \\nINFO  @ Tue, 24 Mar 2020 19:33:48:  16000000 \\nINFO  @ Tue, 24 Mar 2020 19:33:51:  17000000 \\nINFO  @ Tue, 24 Mar 2020 19:33:53:  18000000 \\nINFO  @ Tue, 24 Mar 2020 19:33:55:  19000000 \\nINFO  @ Tue, 24 Mar 2020 19:33:58:  20000000 \\nINFO  @ Tue, 24 Mar 2020 19:34:00:  21000000 \\nINFO  @ Tue, 24 Mar 2020 19:34:03:  22000000 \\nINFO  @ Tue, 24 Mar 2020 19:34:05:  23000000 \\nINFO  @ Tue, 24 Mar 2020 19:34:07:  24000000 \\nINFO  @ Tue, 24 Mar 2020 19:34:10:  25000000 \\nINFO  @ Tue, 24 Mar 2020 19:34:12:  26000000 \\nINFO  @ Tue, 24 Mar 2020 19:34:14:  27000000 \\nINFO  @ Tue, 24 Mar 2020 19:34:17:  28000000 \\nINFO  @ Tue, 24 Mar 2020 19:34:19:  29000000 \\nINFO  @ Tue, 24 Mar 2020 19:34:21:  30000000 \\nINFO  @ Tue, 24 Mar 2020 19:34:24:  31000000 \\nINFO  @ Tue, 24 Mar 2020 19:34:26:  32000000 \\nINFO  @ Tue, 24 Mar 2020 19:34:28:  33000000 \\nINFO  @ Tue, 24 Mar 2020 19:34:31:  34000000 \\nINFO  @ Tue, 24 Mar 2020 19:34:32: 34520959 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 19:34:32: #1 tag size is determined as 49 bps \\nINFO  @ Tue, 24 Mar 2020 19:34:32: #1 tag size = 49.0 \\nINFO  @ Tue, 24 Mar 2020 19:34:32: #1  total tags in treatment: 20684363 \\nINFO  @ Tue, 24 Mar 2020 19:34:32: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 19:34:32: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 19:34:32: #1  tags after filtering in treatment: 20682431 \\nINFO  @ Tue, 24 Mar 2020 19:34:32: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Tue, 24 Mar 2020 19:34:32: #1  total tags in control: 34520959 \\nINFO  @ Tue, 24 Mar 2020 19:34:32: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 19:34:32: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 19:34:32: #1  tags after filtering in control: 34516729 \\nINFO  @ Tue, 24 Mar 2020 19:34:32: #1  Redundant rate of control: 0.00 \\nINFO  @ Tue, 24 Mar 2020 19:34:32: #1 finished! \\nINFO  @ Tue, 24 Mar 2020 19:34:32: #2 Build Peak Model... \\nINFO  @ Tue, 24 Mar 2020 19:34:32: #2 Skipped... \\nINFO  @ Tue, 24 Mar 2020 19:34:32: #2 Use 272 as fragment length \\nINFO  @ Tue, 24 Mar 2020 19:34:32: #3 Call peaks... \\nINFO  @ Tue, 24 Mar 2020 19:34:32: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 24 Mar 2020 19:37:16: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 24 Mar 2020 19:37:16: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2_treat_pileup.bdg \\nINFO  @ Tue, 24 Mar 2020 19:37:16: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2_control_lambda.bdg \\nINFO  @ Tue, 24 Mar 2020 19:37:16: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Tue, 24 Mar 2020 19:37:16: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 24 Mar 2020 19:39:39: #4 Write output xls file... ../../data/diffBinding_hist/diffData_unscaled/mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2_peaks.xls \\nINFO  @ Tue, 24 Mar 2020 19:39:39: #4 Write peak in narrowPeak format file... ../../data/diffBinding_hist/diffData_unscaled/mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2_peaks.narrowPeak \\nINFO  @ Tue, 24 Mar 2020 19:39:39: #4 Write summits bed file... ../../data/diffBinding_hist/diffData_unscaled/mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2_summits.bed \\nINFO  @ Tue, 24 Mar 2020 19:39:39: Done! \\n'\n",
      "21423156 20682431\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "doing diff from ../../data/diffBinding_hist/mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1.mLb.clN.sorted.bam and ../../data/diffBinding_hist/mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "229\n",
      "computing the scaling values\n",
      "b'INFO  @ Tue, 24 Mar 2020 19:48:40: \\n# Command line: callpeak -B -t ../../data/diffBinding_hist/mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 229 -n mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1 --outdir ../../data/diffBinding_hist/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_hist/mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 24 Mar 2020 19:48:40: #1 read tag files... \\nINFO  @ Tue, 24 Mar 2020 19:48:40: #1 read treatment tags... \\nINFO  @ Tue, 24 Mar 2020 19:48:42:  1000000 \\nINFO  @ Tue, 24 Mar 2020 19:48:45:  2000000 \\nINFO  @ Tue, 24 Mar 2020 19:48:47:  3000000 \\nINFO  @ Tue, 24 Mar 2020 19:48:49:  4000000 \\nINFO  @ Tue, 24 Mar 2020 19:48:51:  5000000 \\nINFO  @ Tue, 24 Mar 2020 19:48:53:  6000000 \\nINFO  @ Tue, 24 Mar 2020 19:48:55:  7000000 \\nINFO  @ Tue, 24 Mar 2020 19:48:57:  8000000 \\nINFO  @ Tue, 24 Mar 2020 19:48:59:  9000000 \\nINFO  @ Tue, 24 Mar 2020 19:49:01:  10000000 \\nINFO  @ Tue, 24 Mar 2020 19:49:03:  11000000 \\nINFO  @ Tue, 24 Mar 2020 19:49:05:  12000000 \\nINFO  @ Tue, 24 Mar 2020 19:49:07:  13000000 \\nINFO  @ Tue, 24 Mar 2020 19:49:10:  14000000 \\nINFO  @ Tue, 24 Mar 2020 19:49:12:  15000000 \\nINFO  @ Tue, 24 Mar 2020 19:49:14:  16000000 \\nINFO  @ Tue, 24 Mar 2020 19:49:16:  17000000 \\nINFO  @ Tue, 24 Mar 2020 19:49:18:  18000000 \\nINFO  @ Tue, 24 Mar 2020 19:49:20:  19000000 \\nINFO  @ Tue, 24 Mar 2020 19:49:22:  20000000 \\nINFO  @ Tue, 24 Mar 2020 19:49:24:  21000000 \\nINFO  @ Tue, 24 Mar 2020 19:49:26:  22000000 \\nINFO  @ Tue, 24 Mar 2020 19:49:28:  23000000 \\nINFO  @ Tue, 24 Mar 2020 19:49:30:  24000000 \\nINFO  @ Tue, 24 Mar 2020 19:49:32:  25000000 \\nINFO  @ Tue, 24 Mar 2020 19:49:34:  26000000 \\nINFO  @ Tue, 24 Mar 2020 19:49:36:  27000000 \\nINFO  @ Tue, 24 Mar 2020 19:49:39:  28000000 \\nINFO  @ Tue, 24 Mar 2020 19:49:41:  29000000 \\nINFO  @ Tue, 24 Mar 2020 19:49:43:  30000000 \\nINFO  @ Tue, 24 Mar 2020 19:49:45:  31000000 \\nINFO  @ Tue, 24 Mar 2020 19:49:47:  32000000 \\nINFO  @ Tue, 24 Mar 2020 19:49:49:  33000000 \\nINFO  @ Tue, 24 Mar 2020 19:49:51:  34000000 \\nINFO  @ Tue, 24 Mar 2020 19:49:53:  35000000 \\nINFO  @ Tue, 24 Mar 2020 19:49:55:  36000000 \\nINFO  @ Tue, 24 Mar 2020 19:49:58:  37000000 \\nINFO  @ Tue, 24 Mar 2020 19:50:00:  38000000 \\nINFO  @ Tue, 24 Mar 2020 19:50:02:  39000000 \\nINFO  @ Tue, 24 Mar 2020 19:50:04:  40000000 \\nINFO  @ Tue, 24 Mar 2020 19:50:06:  41000000 \\nINFO  @ Tue, 24 Mar 2020 19:50:08:  42000000 \\nINFO  @ Tue, 24 Mar 2020 19:50:10:  43000000 \\nINFO  @ Tue, 24 Mar 2020 19:50:12:  44000000 \\nINFO  @ Tue, 24 Mar 2020 19:50:14: 44591246 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 19:50:14: #1.2 read input tags... \\nINFO  @ Tue, 24 Mar 2020 19:50:16:  1000000 \\nINFO  @ Tue, 24 Mar 2020 19:50:19:  2000000 \\nINFO  @ Tue, 24 Mar 2020 19:50:21:  3000000 \\nINFO  @ Tue, 24 Mar 2020 19:50:23:  4000000 \\nINFO  @ Tue, 24 Mar 2020 19:50:25:  5000000 \\nINFO  @ Tue, 24 Mar 2020 19:50:28:  6000000 \\nINFO  @ Tue, 24 Mar 2020 19:50:30:  7000000 \\nINFO  @ Tue, 24 Mar 2020 19:50:32:  8000000 \\nINFO  @ Tue, 24 Mar 2020 19:50:34:  9000000 \\nINFO  @ Tue, 24 Mar 2020 19:50:37:  10000000 \\nINFO  @ Tue, 24 Mar 2020 19:50:39:  11000000 \\nINFO  @ Tue, 24 Mar 2020 19:50:41:  12000000 \\nINFO  @ Tue, 24 Mar 2020 19:50:44:  13000000 \\nINFO  @ Tue, 24 Mar 2020 19:50:46:  14000000 \\nINFO  @ Tue, 24 Mar 2020 19:50:48:  15000000 \\nINFO  @ Tue, 24 Mar 2020 19:50:50:  16000000 \\nINFO  @ Tue, 24 Mar 2020 19:50:53:  17000000 \\nINFO  @ Tue, 24 Mar 2020 19:50:55:  18000000 \\nINFO  @ Tue, 24 Mar 2020 19:50:57:  19000000 \\nINFO  @ Tue, 24 Mar 2020 19:50:59:  20000000 \\nINFO  @ Tue, 24 Mar 2020 19:51:02:  21000000 \\nINFO  @ Tue, 24 Mar 2020 19:51:04:  22000000 \\nINFO  @ Tue, 24 Mar 2020 19:51:06:  23000000 \\nINFO  @ Tue, 24 Mar 2020 19:51:08:  24000000 \\nINFO  @ Tue, 24 Mar 2020 19:51:12:  25000000 \\nINFO  @ Tue, 24 Mar 2020 19:51:14:  26000000 \\nINFO  @ Tue, 24 Mar 2020 19:51:16:  27000000 \\nINFO  @ Tue, 24 Mar 2020 19:51:19:  28000000 \\nINFO  @ Tue, 24 Mar 2020 19:51:21:  29000000 \\nINFO  @ Tue, 24 Mar 2020 19:51:23:  30000000 \\nINFO  @ Tue, 24 Mar 2020 19:51:26:  31000000 \\nINFO  @ Tue, 24 Mar 2020 19:51:28:  32000000 \\nINFO  @ Tue, 24 Mar 2020 19:51:30:  33000000 \\nINFO  @ Tue, 24 Mar 2020 19:51:33:  34000000 \\nINFO  @ Tue, 24 Mar 2020 19:51:34: 34520959 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 19:51:34: #1 tag size is determined as 49 bps \\nINFO  @ Tue, 24 Mar 2020 19:51:34: #1 tag size = 49.0 \\nINFO  @ Tue, 24 Mar 2020 19:51:34: #1  total tags in treatment: 44591246 \\nINFO  @ Tue, 24 Mar 2020 19:51:34: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 19:51:34: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 19:51:34: #1  tags after filtering in treatment: 44583884 \\nINFO  @ Tue, 24 Mar 2020 19:51:34: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Tue, 24 Mar 2020 19:51:34: #1  total tags in control: 34520959 \\nINFO  @ Tue, 24 Mar 2020 19:51:34: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 19:51:34: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 19:51:35: #1  tags after filtering in control: 34516729 \\nINFO  @ Tue, 24 Mar 2020 19:51:35: #1  Redundant rate of control: 0.00 \\nINFO  @ Tue, 24 Mar 2020 19:51:35: #1 finished! \\nINFO  @ Tue, 24 Mar 2020 19:51:35: #2 Build Peak Model... \\nINFO  @ Tue, 24 Mar 2020 19:51:35: #2 Skipped... \\nINFO  @ Tue, 24 Mar 2020 19:51:35: #2 Use 229 as fragment length \\nINFO  @ Tue, 24 Mar 2020 19:51:35: #3 Call peaks... \\nINFO  @ Tue, 24 Mar 2020 19:51:35: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 24 Mar 2020 19:54:51: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 24 Mar 2020 19:54:51: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1_treat_pileup.bdg \\nINFO  @ Tue, 24 Mar 2020 19:54:51: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1_control_lambda.bdg \\nINFO  @ Tue, 24 Mar 2020 19:54:51: #3   Pileup will be based on sequencing depth in control. \\nINFO  @ Tue, 24 Mar 2020 19:54:51: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 24 Mar 2020 19:58:55: #4 Write output xls file... ../../data/diffBinding_hist/diffData_unscaled/mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1_peaks.xls \\nINFO  @ Tue, 24 Mar 2020 19:58:55: #4 Write peak in narrowPeak format file... ../../data/diffBinding_hist/diffData_unscaled/mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1_peaks.narrowPeak \\nINFO  @ Tue, 24 Mar 2020 19:58:56: #4 Write summits bed file... ../../data/diffBinding_hist/diffData_unscaled/mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1_summits.bed \\nINFO  @ Tue, 24 Mar 2020 19:58:56: Done! \\n'\n",
      "b'INFO  @ Tue, 24 Mar 2020 19:58:56: \\n# Command line: callpeak -B -t ../../data/diffBinding_hist/mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 229 -n mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1 --outdir ../../data/diffBinding_hist/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_hist/mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 24 Mar 2020 19:58:56: #1 read tag files... \\nINFO  @ Tue, 24 Mar 2020 19:58:56: #1 read treatment tags... \\nINFO  @ Tue, 24 Mar 2020 19:58:59:  1000000 \\nINFO  @ Tue, 24 Mar 2020 19:59:01:  2000000 \\nINFO  @ Tue, 24 Mar 2020 19:59:03:  3000000 \\nINFO  @ Tue, 24 Mar 2020 19:59:05:  4000000 \\nINFO  @ Tue, 24 Mar 2020 19:59:08:  5000000 \\nINFO  @ Tue, 24 Mar 2020 19:59:10:  6000000 \\nINFO  @ Tue, 24 Mar 2020 19:59:12:  7000000 \\nINFO  @ Tue, 24 Mar 2020 19:59:14:  8000000 \\nINFO  @ Tue, 24 Mar 2020 19:59:17:  9000000 \\nINFO  @ Tue, 24 Mar 2020 19:59:19:  10000000 \\nINFO  @ Tue, 24 Mar 2020 19:59:21:  11000000 \\nINFO  @ Tue, 24 Mar 2020 19:59:23:  12000000 \\nINFO  @ Tue, 24 Mar 2020 19:59:26:  13000000 \\nINFO  @ Tue, 24 Mar 2020 19:59:28:  14000000 \\nINFO  @ Tue, 24 Mar 2020 19:59:30:  15000000 \\nINFO  @ Tue, 24 Mar 2020 19:59:32:  16000000 \\nINFO  @ Tue, 24 Mar 2020 19:59:35:  17000000 \\nINFO  @ Tue, 24 Mar 2020 19:59:37:  18000000 \\nINFO  @ Tue, 24 Mar 2020 19:59:39:  19000000 \\nINFO  @ Tue, 24 Mar 2020 19:59:41:  20000000 \\nINFO  @ Tue, 24 Mar 2020 19:59:43:  21000000 \\nINFO  @ Tue, 24 Mar 2020 19:59:46:  22000000 \\nINFO  @ Tue, 24 Mar 2020 19:59:48:  23000000 \\nINFO  @ Tue, 24 Mar 2020 19:59:50:  24000000 \\nINFO  @ Tue, 24 Mar 2020 19:59:52:  25000000 \\nINFO  @ Tue, 24 Mar 2020 19:59:54:  26000000 \\nINFO  @ Tue, 24 Mar 2020 19:59:57:  27000000 \\nINFO  @ Tue, 24 Mar 2020 19:59:59:  28000000 \\nINFO  @ Tue, 24 Mar 2020 20:00:01:  29000000 \\nINFO  @ Tue, 24 Mar 2020 20:00:03:  30000000 \\nINFO  @ Tue, 24 Mar 2020 20:00:05:  31000000 \\nINFO  @ Tue, 24 Mar 2020 20:00:08:  32000000 \\nINFO  @ Tue, 24 Mar 2020 20:00:10:  33000000 \\nINFO  @ Tue, 24 Mar 2020 20:00:11: 33199280 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 20:00:11: #1.2 read input tags... \\nINFO  @ Tue, 24 Mar 2020 20:00:13:  1000000 \\nINFO  @ Tue, 24 Mar 2020 20:00:16:  2000000 \\nINFO  @ Tue, 24 Mar 2020 20:00:18:  3000000 \\nINFO  @ Tue, 24 Mar 2020 20:00:20:  4000000 \\nINFO  @ Tue, 24 Mar 2020 20:00:23:  5000000 \\nINFO  @ Tue, 24 Mar 2020 20:00:25:  6000000 \\nINFO  @ Tue, 24 Mar 2020 20:00:27:  7000000 \\nINFO  @ Tue, 24 Mar 2020 20:00:30:  8000000 \\nINFO  @ Tue, 24 Mar 2020 20:00:32:  9000000 \\nINFO  @ Tue, 24 Mar 2020 20:00:35:  10000000 \\nINFO  @ Tue, 24 Mar 2020 20:00:37:  11000000 \\nINFO  @ Tue, 24 Mar 2020 20:00:39:  12000000 \\nINFO  @ Tue, 24 Mar 2020 20:00:42:  13000000 \\nINFO  @ Tue, 24 Mar 2020 20:00:44:  14000000 \\nINFO  @ Tue, 24 Mar 2020 20:00:46:  15000000 \\nINFO  @ Tue, 24 Mar 2020 20:00:49:  16000000 \\nINFO  @ Tue, 24 Mar 2020 20:00:51:  17000000 \\nINFO  @ Tue, 24 Mar 2020 20:00:54:  18000000 \\nINFO  @ Tue, 24 Mar 2020 20:00:56:  19000000 \\nINFO  @ Tue, 24 Mar 2020 20:00:58:  20000000 \\nINFO  @ Tue, 24 Mar 2020 20:01:01:  21000000 \\nINFO  @ Tue, 24 Mar 2020 20:01:03:  22000000 \\nINFO  @ Tue, 24 Mar 2020 20:01:05:  23000000 \\nINFO  @ Tue, 24 Mar 2020 20:01:08:  24000000 \\nINFO  @ Tue, 24 Mar 2020 20:01:10:  25000000 \\nINFO  @ Tue, 24 Mar 2020 20:01:12:  26000000 \\nINFO  @ Tue, 24 Mar 2020 20:01:15:  27000000 \\nINFO  @ Tue, 24 Mar 2020 20:01:17:  28000000 \\nINFO  @ Tue, 24 Mar 2020 20:01:20:  29000000 \\nINFO  @ Tue, 24 Mar 2020 20:01:22:  30000000 \\nINFO  @ Tue, 24 Mar 2020 20:01:24:  31000000 \\nINFO  @ Tue, 24 Mar 2020 20:01:27:  32000000 \\nINFO  @ Tue, 24 Mar 2020 20:01:29:  33000000 \\nINFO  @ Tue, 24 Mar 2020 20:01:31:  34000000 \\nINFO  @ Tue, 24 Mar 2020 20:01:33: 34520959 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 20:01:33: #1 tag size is determined as 49 bps \\nINFO  @ Tue, 24 Mar 2020 20:01:33: #1 tag size = 49.0 \\nINFO  @ Tue, 24 Mar 2020 20:01:33: #1  total tags in treatment: 33199280 \\nINFO  @ Tue, 24 Mar 2020 20:01:33: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 20:01:33: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 20:01:33: #1  tags after filtering in treatment: 33193829 \\nINFO  @ Tue, 24 Mar 2020 20:01:33: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Tue, 24 Mar 2020 20:01:33: #1  total tags in control: 34520959 \\nINFO  @ Tue, 24 Mar 2020 20:01:33: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 20:01:33: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 20:01:33: #1  tags after filtering in control: 34516729 \\nINFO  @ Tue, 24 Mar 2020 20:01:33: #1  Redundant rate of control: 0.00 \\nINFO  @ Tue, 24 Mar 2020 20:01:33: #1 finished! \\nINFO  @ Tue, 24 Mar 2020 20:01:33: #2 Build Peak Model... \\nINFO  @ Tue, 24 Mar 2020 20:01:33: #2 Skipped... \\nINFO  @ Tue, 24 Mar 2020 20:01:33: #2 Use 229 as fragment length \\nINFO  @ Tue, 24 Mar 2020 20:01:33: #3 Call peaks... \\nINFO  @ Tue, 24 Mar 2020 20:01:33: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 24 Mar 2020 20:04:55: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 24 Mar 2020 20:04:55: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1_treat_pileup.bdg \\nINFO  @ Tue, 24 Mar 2020 20:04:55: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1_control_lambda.bdg \\nINFO  @ Tue, 24 Mar 2020 20:04:55: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Tue, 24 Mar 2020 20:04:55: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 24 Mar 2020 20:08:53: #4 Write output xls file... ../../data/diffBinding_hist/diffData_unscaled/mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1_peaks.xls \\nINFO  @ Tue, 24 Mar 2020 20:08:54: #4 Write peak in narrowPeak format file... ../../data/diffBinding_hist/diffData_unscaled/mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1_peaks.narrowPeak \\nINFO  @ Tue, 24 Mar 2020 20:08:55: #4 Write summits bed file... ../../data/diffBinding_hist/diffData_unscaled/mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1_summits.bed \\nINFO  @ Tue, 24 Mar 2020 20:08:55: Done! \\n'\n",
      "34516729 33193829\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "doing diff from ../../data/diffBinding_hist/mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2.mLb.clN.sorted.bam and ../../data/diffBinding_hist/mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "235\n",
      "computing the scaling values\n",
      "b'INFO  @ Tue, 24 Mar 2020 20:23:27: \\n# Command line: callpeak -B -t ../../data/diffBinding_hist/mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 235 -n mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2 --outdir ../../data/diffBinding_hist/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_hist/mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 24 Mar 2020 20:23:27: #1 read tag files... \\nINFO  @ Tue, 24 Mar 2020 20:23:27: #1 read treatment tags... \\nINFO  @ Tue, 24 Mar 2020 20:23:30:  1000000 \\nINFO  @ Tue, 24 Mar 2020 20:23:32:  2000000 \\nINFO  @ Tue, 24 Mar 2020 20:23:35:  3000000 \\nINFO  @ Tue, 24 Mar 2020 20:23:37:  4000000 \\nINFO  @ Tue, 24 Mar 2020 20:23:40:  5000000 \\nINFO  @ Tue, 24 Mar 2020 20:23:42:  6000000 \\nINFO  @ Tue, 24 Mar 2020 20:23:44:  7000000 \\nINFO  @ Tue, 24 Mar 2020 20:23:47:  8000000 \\nINFO  @ Tue, 24 Mar 2020 20:23:49:  9000000 \\nINFO  @ Tue, 24 Mar 2020 20:23:52:  10000000 \\nINFO  @ Tue, 24 Mar 2020 20:23:54:  11000000 \\nINFO  @ Tue, 24 Mar 2020 20:23:57:  12000000 \\nINFO  @ Tue, 24 Mar 2020 20:24:00:  13000000 \\nINFO  @ Tue, 24 Mar 2020 20:24:02:  14000000 \\nINFO  @ Tue, 24 Mar 2020 20:24:05:  15000000 \\nINFO  @ Tue, 24 Mar 2020 20:24:07:  16000000 \\nINFO  @ Tue, 24 Mar 2020 20:24:10:  17000000 \\nINFO  @ Tue, 24 Mar 2020 20:24:12:  18000000 \\nINFO  @ Tue, 24 Mar 2020 20:24:15:  19000000 \\nINFO  @ Tue, 24 Mar 2020 20:24:17:  20000000 \\nINFO  @ Tue, 24 Mar 2020 20:24:20:  21000000 \\nINFO  @ Tue, 24 Mar 2020 20:24:22:  22000000 \\nINFO  @ Tue, 24 Mar 2020 20:24:25:  23000000 \\nINFO  @ Tue, 24 Mar 2020 20:24:27:  24000000 \\nINFO  @ Tue, 24 Mar 2020 20:24:30:  25000000 \\nINFO  @ Tue, 24 Mar 2020 20:24:33:  26000000 \\nINFO  @ Tue, 24 Mar 2020 20:24:35:  27000000 \\nINFO  @ Tue, 24 Mar 2020 20:24:38:  28000000 \\nINFO  @ Tue, 24 Mar 2020 20:24:40:  29000000 \\nINFO  @ Tue, 24 Mar 2020 20:24:43:  30000000 \\nINFO  @ Tue, 24 Mar 2020 20:24:45:  31000000 \\nINFO  @ Tue, 24 Mar 2020 20:24:48:  32000000 \\nINFO  @ Tue, 24 Mar 2020 20:24:50:  33000000 \\nINFO  @ Tue, 24 Mar 2020 20:24:53:  34000000 \\nINFO  @ Tue, 24 Mar 2020 20:24:56:  35000000 \\nINFO  @ Tue, 24 Mar 2020 20:24:58: 35945515 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 20:24:58: #1.2 read input tags... \\nINFO  @ Tue, 24 Mar 2020 20:25:01:  1000000 \\nINFO  @ Tue, 24 Mar 2020 20:25:04:  2000000 \\nINFO  @ Tue, 24 Mar 2020 20:25:07:  3000000 \\nINFO  @ Tue, 24 Mar 2020 20:25:09:  4000000 \\nINFO  @ Tue, 24 Mar 2020 20:25:12:  5000000 \\nINFO  @ Tue, 24 Mar 2020 20:25:15:  6000000 \\nINFO  @ Tue, 24 Mar 2020 20:25:18:  7000000 \\nINFO  @ Tue, 24 Mar 2020 20:25:21:  8000000 \\nINFO  @ Tue, 24 Mar 2020 20:25:24:  9000000 \\nINFO  @ Tue, 24 Mar 2020 20:25:26:  10000000 \\nINFO  @ Tue, 24 Mar 2020 20:25:29:  11000000 \\nINFO  @ Tue, 24 Mar 2020 20:25:32:  12000000 \\nINFO  @ Tue, 24 Mar 2020 20:25:35:  13000000 \\nINFO  @ Tue, 24 Mar 2020 20:25:37:  14000000 \\nINFO  @ Tue, 24 Mar 2020 20:25:40:  15000000 \\nINFO  @ Tue, 24 Mar 2020 20:25:43:  16000000 \\nINFO  @ Tue, 24 Mar 2020 20:25:46:  17000000 \\nINFO  @ Tue, 24 Mar 2020 20:25:49:  18000000 \\nINFO  @ Tue, 24 Mar 2020 20:25:51:  19000000 \\nINFO  @ Tue, 24 Mar 2020 20:25:54:  20000000 \\nINFO  @ Tue, 24 Mar 2020 20:25:57:  21000000 \\nINFO  @ Tue, 24 Mar 2020 20:26:00:  22000000 \\nINFO  @ Tue, 24 Mar 2020 20:26:02:  23000000 \\nINFO  @ Tue, 24 Mar 2020 20:26:05:  24000000 \\nINFO  @ Tue, 24 Mar 2020 20:26:08:  25000000 \\nINFO  @ Tue, 24 Mar 2020 20:26:11:  26000000 \\nINFO  @ Tue, 24 Mar 2020 20:26:13:  27000000 \\nINFO  @ Tue, 24 Mar 2020 20:26:16:  28000000 \\nINFO  @ Tue, 24 Mar 2020 20:26:19:  29000000 \\nINFO  @ Tue, 24 Mar 2020 20:26:22:  30000000 \\nINFO  @ Tue, 24 Mar 2020 20:26:25:  31000000 \\nINFO  @ Tue, 24 Mar 2020 20:26:27:  32000000 \\nINFO  @ Tue, 24 Mar 2020 20:26:30:  33000000 \\nINFO  @ Tue, 24 Mar 2020 20:26:33:  34000000 \\nINFO  @ Tue, 24 Mar 2020 20:26:35: 34520959 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 20:26:35: #1 tag size is determined as 49 bps \\nINFO  @ Tue, 24 Mar 2020 20:26:35: #1 tag size = 49.0 \\nINFO  @ Tue, 24 Mar 2020 20:26:35: #1  total tags in treatment: 35945515 \\nINFO  @ Tue, 24 Mar 2020 20:26:35: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 20:26:35: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 20:26:35: #1  tags after filtering in treatment: 35939396 \\nINFO  @ Tue, 24 Mar 2020 20:26:35: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Tue, 24 Mar 2020 20:26:35: #1  total tags in control: 34520959 \\nINFO  @ Tue, 24 Mar 2020 20:26:35: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 20:26:35: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 20:26:35: #1  tags after filtering in control: 34516729 \\nINFO  @ Tue, 24 Mar 2020 20:26:35: #1  Redundant rate of control: 0.00 \\nINFO  @ Tue, 24 Mar 2020 20:26:35: #1 finished! \\nINFO  @ Tue, 24 Mar 2020 20:26:35: #2 Build Peak Model... \\nINFO  @ Tue, 24 Mar 2020 20:26:35: #2 Skipped... \\nINFO  @ Tue, 24 Mar 2020 20:26:35: #2 Use 235 as fragment length \\nINFO  @ Tue, 24 Mar 2020 20:26:35: #3 Call peaks... \\nINFO  @ Tue, 24 Mar 2020 20:26:35: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 24 Mar 2020 20:29:45: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 24 Mar 2020 20:29:45: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2_treat_pileup.bdg \\nINFO  @ Tue, 24 Mar 2020 20:29:45: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2_control_lambda.bdg \\nINFO  @ Tue, 24 Mar 2020 20:29:45: #3   Pileup will be based on sequencing depth in control. \\nINFO  @ Tue, 24 Mar 2020 20:29:45: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 24 Mar 2020 20:32:46: #4 Write output xls file... ../../data/diffBinding_hist/diffData_unscaled/mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2_peaks.xls \\nINFO  @ Tue, 24 Mar 2020 20:32:47: #4 Write peak in narrowPeak format file... ../../data/diffBinding_hist/diffData_unscaled/mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2_peaks.narrowPeak \\nINFO  @ Tue, 24 Mar 2020 20:32:47: #4 Write summits bed file... ../../data/diffBinding_hist/diffData_unscaled/mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2_summits.bed \\nINFO  @ Tue, 24 Mar 2020 20:32:47: Done! \\n'\n",
      "b'INFO  @ Tue, 24 Mar 2020 20:32:48: \\n# Command line: callpeak -B -t ../../data/diffBinding_hist/mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 235 -n mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2 --outdir ../../data/diffBinding_hist/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_hist/mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 24 Mar 2020 20:32:48: #1 read tag files... \\nINFO  @ Tue, 24 Mar 2020 20:32:48: #1 read treatment tags... \\nINFO  @ Tue, 24 Mar 2020 20:32:50:  1000000 \\nINFO  @ Tue, 24 Mar 2020 20:32:52:  2000000 \\nINFO  @ Tue, 24 Mar 2020 20:32:55:  3000000 \\nINFO  @ Tue, 24 Mar 2020 20:32:57:  4000000 \\nINFO  @ Tue, 24 Mar 2020 20:32:59:  5000000 \\nINFO  @ Tue, 24 Mar 2020 20:33:02:  6000000 \\nINFO  @ Tue, 24 Mar 2020 20:33:04:  7000000 \\nINFO  @ Tue, 24 Mar 2020 20:33:06:  8000000 \\nINFO  @ Tue, 24 Mar 2020 20:33:08:  9000000 \\nINFO  @ Tue, 24 Mar 2020 20:33:10:  10000000 \\nINFO  @ Tue, 24 Mar 2020 20:33:13:  11000000 \\nINFO  @ Tue, 24 Mar 2020 20:33:15:  12000000 \\nINFO  @ Tue, 24 Mar 2020 20:33:17:  13000000 \\nINFO  @ Tue, 24 Mar 2020 20:33:19:  14000000 \\nINFO  @ Tue, 24 Mar 2020 20:33:21:  15000000 \\nINFO  @ Tue, 24 Mar 2020 20:33:24:  16000000 \\nINFO  @ Tue, 24 Mar 2020 20:33:26:  17000000 \\nINFO  @ Tue, 24 Mar 2020 20:33:28:  18000000 \\nINFO  @ Tue, 24 Mar 2020 20:33:31:  19000000 \\nINFO  @ Tue, 24 Mar 2020 20:33:33:  20000000 \\nINFO  @ Tue, 24 Mar 2020 20:33:35:  21000000 \\nINFO  @ Tue, 24 Mar 2020 20:33:37:  22000000 \\nINFO  @ Tue, 24 Mar 2020 20:33:39:  23000000 \\nINFO  @ Tue, 24 Mar 2020 20:33:42:  24000000 \\nINFO  @ Tue, 24 Mar 2020 20:33:44:  25000000 \\nINFO  @ Tue, 24 Mar 2020 20:33:46:  26000000 \\nINFO  @ Tue, 24 Mar 2020 20:33:48:  27000000 \\nINFO  @ Tue, 24 Mar 2020 20:33:50:  28000000 \\nINFO  @ Tue, 24 Mar 2020 20:33:52:  29000000 \\nINFO  @ Tue, 24 Mar 2020 20:33:55:  30000000 \\nINFO  @ Tue, 24 Mar 2020 20:33:57:  31000000 \\nINFO  @ Tue, 24 Mar 2020 20:33:59:  32000000 \\nINFO  @ Tue, 24 Mar 2020 20:34:01:  33000000 \\nINFO  @ Tue, 24 Mar 2020 20:34:03:  34000000 \\nINFO  @ Tue, 24 Mar 2020 20:34:05:  35000000 \\nINFO  @ Tue, 24 Mar 2020 20:34:08:  36000000 \\nINFO  @ Tue, 24 Mar 2020 20:34:10:  37000000 \\nINFO  @ Tue, 24 Mar 2020 20:34:12:  38000000 \\nINFO  @ Tue, 24 Mar 2020 20:34:14:  39000000 \\nINFO  @ Tue, 24 Mar 2020 20:34:17:  40000000 \\nINFO  @ Tue, 24 Mar 2020 20:34:19:  41000000 \\nINFO  @ Tue, 24 Mar 2020 20:34:21:  42000000 \\nINFO  @ Tue, 24 Mar 2020 20:34:23:  43000000 \\nINFO  @ Tue, 24 Mar 2020 20:34:25:  44000000 \\nINFO  @ Tue, 24 Mar 2020 20:34:27:  45000000 \\nINFO  @ Tue, 24 Mar 2020 20:34:28: 45310470 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 20:34:29: #1.2 read input tags... \\nINFO  @ Tue, 24 Mar 2020 20:34:31:  1000000 \\nINFO  @ Tue, 24 Mar 2020 20:34:33:  2000000 \\nINFO  @ Tue, 24 Mar 2020 20:34:36:  3000000 \\nINFO  @ Tue, 24 Mar 2020 20:34:38:  4000000 \\nINFO  @ Tue, 24 Mar 2020 20:34:40:  5000000 \\nINFO  @ Tue, 24 Mar 2020 20:34:42:  6000000 \\nINFO  @ Tue, 24 Mar 2020 20:34:45:  7000000 \\nINFO  @ Tue, 24 Mar 2020 20:34:47:  8000000 \\nINFO  @ Tue, 24 Mar 2020 20:34:49:  9000000 \\nINFO  @ Tue, 24 Mar 2020 20:34:52:  10000000 \\nINFO  @ Tue, 24 Mar 2020 20:34:54:  11000000 \\nINFO  @ Tue, 24 Mar 2020 20:34:56:  12000000 \\nINFO  @ Tue, 24 Mar 2020 20:34:59:  13000000 \\nINFO  @ Tue, 24 Mar 2020 20:35:01:  14000000 \\nINFO  @ Tue, 24 Mar 2020 20:35:03:  15000000 \\nINFO  @ Tue, 24 Mar 2020 20:35:05:  16000000 \\nINFO  @ Tue, 24 Mar 2020 20:35:08:  17000000 \\nINFO  @ Tue, 24 Mar 2020 20:35:10:  18000000 \\nINFO  @ Tue, 24 Mar 2020 20:35:12:  19000000 \\nINFO  @ Tue, 24 Mar 2020 20:35:15:  20000000 \\nINFO  @ Tue, 24 Mar 2020 20:35:17:  21000000 \\nINFO  @ Tue, 24 Mar 2020 20:35:19:  22000000 \\nINFO  @ Tue, 24 Mar 2020 20:35:21:  23000000 \\nINFO  @ Tue, 24 Mar 2020 20:35:24:  24000000 \\nINFO  @ Tue, 24 Mar 2020 20:35:26:  25000000 \\nINFO  @ Tue, 24 Mar 2020 20:35:28:  26000000 \\nINFO  @ Tue, 24 Mar 2020 20:35:31:  27000000 \\nINFO  @ Tue, 24 Mar 2020 20:35:33:  28000000 \\nINFO  @ Tue, 24 Mar 2020 20:35:35:  29000000 \\nINFO  @ Tue, 24 Mar 2020 20:35:38:  30000000 \\nINFO  @ Tue, 24 Mar 2020 20:35:40:  31000000 \\nINFO  @ Tue, 24 Mar 2020 20:35:42:  32000000 \\nINFO  @ Tue, 24 Mar 2020 20:35:45:  33000000 \\nINFO  @ Tue, 24 Mar 2020 20:35:47:  34000000 \\nINFO  @ Tue, 24 Mar 2020 20:35:48: 34520959 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 20:35:48: #1 tag size is determined as 49 bps \\nINFO  @ Tue, 24 Mar 2020 20:35:48: #1 tag size = 49.0 \\nINFO  @ Tue, 24 Mar 2020 20:35:48: #1  total tags in treatment: 45310470 \\nINFO  @ Tue, 24 Mar 2020 20:35:48: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 20:35:48: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 20:35:49: #1  tags after filtering in treatment: 45301891 \\nINFO  @ Tue, 24 Mar 2020 20:35:49: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Tue, 24 Mar 2020 20:35:49: #1  total tags in control: 34520959 \\nINFO  @ Tue, 24 Mar 2020 20:35:49: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 20:35:49: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 20:35:49: #1  tags after filtering in control: 34516729 \\nINFO  @ Tue, 24 Mar 2020 20:35:49: #1  Redundant rate of control: 0.00 \\nINFO  @ Tue, 24 Mar 2020 20:35:49: #1 finished! \\nINFO  @ Tue, 24 Mar 2020 20:35:49: #2 Build Peak Model... \\nINFO  @ Tue, 24 Mar 2020 20:35:49: #2 Skipped... \\nINFO  @ Tue, 24 Mar 2020 20:35:49: #2 Use 235 as fragment length \\nINFO  @ Tue, 24 Mar 2020 20:35:49: #3 Call peaks... \\nINFO  @ Tue, 24 Mar 2020 20:35:49: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 24 Mar 2020 20:39:01: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 24 Mar 2020 20:39:01: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2_treat_pileup.bdg \\nINFO  @ Tue, 24 Mar 2020 20:39:01: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2_control_lambda.bdg \\nINFO  @ Tue, 24 Mar 2020 20:39:01: #3   Pileup will be based on sequencing depth in control. \\nINFO  @ Tue, 24 Mar 2020 20:39:01: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 24 Mar 2020 20:42:42: #4 Write output xls file... ../../data/diffBinding_hist/diffData_unscaled/mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2_peaks.xls \\nINFO  @ Tue, 24 Mar 2020 20:42:42: #4 Write peak in narrowPeak format file... ../../data/diffBinding_hist/diffData_unscaled/mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2_peaks.narrowPeak \\nINFO  @ Tue, 24 Mar 2020 20:42:43: #4 Write summits bed file... ../../data/diffBinding_hist/diffData_unscaled/mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2_summits.bed \\nINFO  @ Tue, 24 Mar 2020 20:42:43: Done! \\n'\n",
      "34516729 34516729\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "doing diff from ../../data/diffBinding_hist/mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1.mLb.clN.sorted.bam and ../../data/diffBinding_hist/mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "190\n",
      "computing the scaling values\n",
      "b'INFO  @ Tue, 24 Mar 2020 20:55:24: \\n# Command line: callpeak -B -t ../../data/diffBinding_hist/mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 190 -n mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1 --outdir ../../data/diffBinding_hist/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_hist/mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 24 Mar 2020 20:55:24: #1 read tag files... \\nINFO  @ Tue, 24 Mar 2020 20:55:24: #1 read treatment tags... \\nINFO  @ Tue, 24 Mar 2020 20:55:26:  1000000 \\nINFO  @ Tue, 24 Mar 2020 20:55:28:  2000000 \\nINFO  @ Tue, 24 Mar 2020 20:55:30:  3000000 \\nINFO  @ Tue, 24 Mar 2020 20:55:32:  4000000 \\nINFO  @ Tue, 24 Mar 2020 20:55:34:  5000000 \\nINFO  @ Tue, 24 Mar 2020 20:55:36:  6000000 \\nINFO  @ Tue, 24 Mar 2020 20:55:38:  7000000 \\nINFO  @ Tue, 24 Mar 2020 20:55:40:  8000000 \\nINFO  @ Tue, 24 Mar 2020 20:55:42:  9000000 \\nINFO  @ Tue, 24 Mar 2020 20:55:44:  10000000 \\nINFO  @ Tue, 24 Mar 2020 20:55:46:  11000000 \\nINFO  @ Tue, 24 Mar 2020 20:55:48:  12000000 \\nINFO  @ Tue, 24 Mar 2020 20:55:50:  13000000 \\nINFO  @ Tue, 24 Mar 2020 20:55:52:  14000000 \\nINFO  @ Tue, 24 Mar 2020 20:55:54:  15000000 \\nINFO  @ Tue, 24 Mar 2020 20:55:56:  16000000 \\nINFO  @ Tue, 24 Mar 2020 20:55:58:  17000000 \\nINFO  @ Tue, 24 Mar 2020 20:56:00:  18000000 \\nINFO  @ Tue, 24 Mar 2020 20:56:02:  19000000 \\nINFO  @ Tue, 24 Mar 2020 20:56:04:  20000000 \\nINFO  @ Tue, 24 Mar 2020 20:56:04: 20100144 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 20:56:04: #1.2 read input tags... \\nINFO  @ Tue, 24 Mar 2020 20:56:07:  1000000 \\nINFO  @ Tue, 24 Mar 2020 20:56:10:  2000000 \\nINFO  @ Tue, 24 Mar 2020 20:56:12:  3000000 \\nINFO  @ Tue, 24 Mar 2020 20:56:15:  4000000 \\nINFO  @ Tue, 24 Mar 2020 20:56:17:  5000000 \\nINFO  @ Tue, 24 Mar 2020 20:56:19:  6000000 \\nINFO  @ Tue, 24 Mar 2020 20:56:22:  7000000 \\nINFO  @ Tue, 24 Mar 2020 20:56:24:  8000000 \\nINFO  @ Tue, 24 Mar 2020 20:56:26:  9000000 \\nINFO  @ Tue, 24 Mar 2020 20:56:29:  10000000 \\nINFO  @ Tue, 24 Mar 2020 20:56:31:  11000000 \\nINFO  @ Tue, 24 Mar 2020 20:56:33:  12000000 \\nINFO  @ Tue, 24 Mar 2020 20:56:35:  13000000 \\nINFO  @ Tue, 24 Mar 2020 20:56:38:  14000000 \\nINFO  @ Tue, 24 Mar 2020 20:56:40:  15000000 \\nINFO  @ Tue, 24 Mar 2020 20:56:42:  16000000 \\nINFO  @ Tue, 24 Mar 2020 20:56:45:  17000000 \\nINFO  @ Tue, 24 Mar 2020 20:56:47:  18000000 \\nINFO  @ Tue, 24 Mar 2020 20:56:49:  19000000 \\nINFO  @ Tue, 24 Mar 2020 20:56:52:  20000000 \\nINFO  @ Tue, 24 Mar 2020 20:56:54:  21000000 \\nINFO  @ Tue, 24 Mar 2020 20:56:56:  22000000 \\nINFO  @ Tue, 24 Mar 2020 20:56:59:  23000000 \\nINFO  @ Tue, 24 Mar 2020 20:57:01:  24000000 \\nINFO  @ Tue, 24 Mar 2020 20:57:04:  25000000 \\nINFO  @ Tue, 24 Mar 2020 20:57:06:  26000000 \\nINFO  @ Tue, 24 Mar 2020 20:57:08:  27000000 \\nINFO  @ Tue, 24 Mar 2020 20:57:10:  28000000 \\nINFO  @ Tue, 24 Mar 2020 20:57:13:  29000000 \\nINFO  @ Tue, 24 Mar 2020 20:57:15:  30000000 \\nINFO  @ Tue, 24 Mar 2020 20:57:17:  31000000 \\nINFO  @ Tue, 24 Mar 2020 20:57:20:  32000000 \\nINFO  @ Tue, 24 Mar 2020 20:57:22:  33000000 \\nINFO  @ Tue, 24 Mar 2020 20:57:24:  34000000 \\nINFO  @ Tue, 24 Mar 2020 20:57:26: 34520959 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 20:57:26: #1 tag size is determined as 49 bps \\nINFO  @ Tue, 24 Mar 2020 20:57:26: #1 tag size = 49.0 \\nINFO  @ Tue, 24 Mar 2020 20:57:26: #1  total tags in treatment: 20100144 \\nINFO  @ Tue, 24 Mar 2020 20:57:26: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 20:57:26: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 20:57:26: #1  tags after filtering in treatment: 20073492 \\nINFO  @ Tue, 24 Mar 2020 20:57:26: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Tue, 24 Mar 2020 20:57:26: #1  total tags in control: 34520959 \\nINFO  @ Tue, 24 Mar 2020 20:57:26: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 20:57:26: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 20:57:26: #1  tags after filtering in control: 34516729 \\nINFO  @ Tue, 24 Mar 2020 20:57:26: #1  Redundant rate of control: 0.00 \\nINFO  @ Tue, 24 Mar 2020 20:57:26: #1 finished! \\nINFO  @ Tue, 24 Mar 2020 20:57:26: #2 Build Peak Model... \\nINFO  @ Tue, 24 Mar 2020 20:57:26: #2 Skipped... \\nINFO  @ Tue, 24 Mar 2020 20:57:26: #2 Use 190 as fragment length \\nINFO  @ Tue, 24 Mar 2020 20:57:26: #3 Call peaks... \\nINFO  @ Tue, 24 Mar 2020 20:57:26: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 24 Mar 2020 21:00:34: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 24 Mar 2020 21:00:34: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1_treat_pileup.bdg \\nINFO  @ Tue, 24 Mar 2020 21:00:34: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1_control_lambda.bdg \\nINFO  @ Tue, 24 Mar 2020 21:00:34: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Tue, 24 Mar 2020 21:00:34: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 24 Mar 2020 21:03:43: #4 Write output xls file... ../../data/diffBinding_hist/diffData_unscaled/mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1_peaks.xls \\nINFO  @ Tue, 24 Mar 2020 21:03:43: #4 Write peak in narrowPeak format file... ../../data/diffBinding_hist/diffData_unscaled/mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1_peaks.narrowPeak \\nINFO  @ Tue, 24 Mar 2020 21:03:43: #4 Write summits bed file... ../../data/diffBinding_hist/diffData_unscaled/mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1_summits.bed \\nINFO  @ Tue, 24 Mar 2020 21:03:43: Done! \\n'\n",
      "b'INFO  @ Tue, 24 Mar 2020 21:03:44: \\n# Command line: callpeak -B -t ../../data/diffBinding_hist/mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 190 -n mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1 --outdir ../../data/diffBinding_hist/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_hist/mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 24 Mar 2020 21:03:44: #1 read tag files... \\nINFO  @ Tue, 24 Mar 2020 21:03:44: #1 read treatment tags... \\nINFO  @ Tue, 24 Mar 2020 21:03:47:  1000000 \\nINFO  @ Tue, 24 Mar 2020 21:03:50:  2000000 \\nINFO  @ Tue, 24 Mar 2020 21:03:53:  3000000 \\nINFO  @ Tue, 24 Mar 2020 21:03:55:  4000000 \\nINFO  @ Tue, 24 Mar 2020 21:03:57:  5000000 \\nINFO  @ Tue, 24 Mar 2020 21:03:59:  6000000 \\nINFO  @ Tue, 24 Mar 2020 21:04:01:  7000000 \\nINFO  @ Tue, 24 Mar 2020 21:04:03:  8000000 \\nINFO  @ Tue, 24 Mar 2020 21:04:05:  9000000 \\nINFO  @ Tue, 24 Mar 2020 21:04:07:  10000000 \\nINFO  @ Tue, 24 Mar 2020 21:04:09:  11000000 \\nINFO  @ Tue, 24 Mar 2020 21:04:11:  12000000 \\nINFO  @ Tue, 24 Mar 2020 21:04:13:  13000000 \\nINFO  @ Tue, 24 Mar 2020 21:04:15:  14000000 \\nINFO  @ Tue, 24 Mar 2020 21:04:17:  15000000 \\nINFO  @ Tue, 24 Mar 2020 21:04:20:  16000000 \\nINFO  @ Tue, 24 Mar 2020 21:04:22:  17000000 \\nINFO  @ Tue, 24 Mar 2020 21:04:25:  18000000 \\nINFO  @ Tue, 24 Mar 2020 21:04:27:  19000000 \\nINFO  @ Tue, 24 Mar 2020 21:04:29:  20000000 \\nINFO  @ Tue, 24 Mar 2020 21:04:31:  21000000 \\nINFO  @ Tue, 24 Mar 2020 21:04:33:  22000000 \\nINFO  @ Tue, 24 Mar 2020 21:04:35:  23000000 \\nINFO  @ Tue, 24 Mar 2020 21:04:37:  24000000 \\nINFO  @ Tue, 24 Mar 2020 21:04:37: 24095030 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 21:04:38: #1.2 read input tags... \\nINFO  @ Tue, 24 Mar 2020 21:04:40:  1000000 \\nINFO  @ Tue, 24 Mar 2020 21:04:42:  2000000 \\nINFO  @ Tue, 24 Mar 2020 21:04:45:  3000000 \\nINFO  @ Tue, 24 Mar 2020 21:04:47:  4000000 \\nINFO  @ Tue, 24 Mar 2020 21:04:49:  5000000 \\nINFO  @ Tue, 24 Mar 2020 21:04:52:  6000000 \\nINFO  @ Tue, 24 Mar 2020 21:04:54:  7000000 \\nINFO  @ Tue, 24 Mar 2020 21:04:56:  8000000 \\nINFO  @ Tue, 24 Mar 2020 21:04:58:  9000000 \\nINFO  @ Tue, 24 Mar 2020 21:05:01:  10000000 \\nINFO  @ Tue, 24 Mar 2020 21:05:03:  11000000 \\nINFO  @ Tue, 24 Mar 2020 21:05:05:  12000000 \\nINFO  @ Tue, 24 Mar 2020 21:05:08:  13000000 \\nINFO  @ Tue, 24 Mar 2020 21:05:10:  14000000 \\nINFO  @ Tue, 24 Mar 2020 21:05:12:  15000000 \\nINFO  @ Tue, 24 Mar 2020 21:05:14:  16000000 \\nINFO  @ Tue, 24 Mar 2020 21:05:17:  17000000 \\nINFO  @ Tue, 24 Mar 2020 21:05:19:  18000000 \\nINFO  @ Tue, 24 Mar 2020 21:05:21:  19000000 \\nINFO  @ Tue, 24 Mar 2020 21:05:24:  20000000 \\nINFO  @ Tue, 24 Mar 2020 21:05:26:  21000000 \\nINFO  @ Tue, 24 Mar 2020 21:05:28:  22000000 \\nINFO  @ Tue, 24 Mar 2020 21:05:31:  23000000 \\nINFO  @ Tue, 24 Mar 2020 21:05:33:  24000000 \\nINFO  @ Tue, 24 Mar 2020 21:05:35:  25000000 \\nINFO  @ Tue, 24 Mar 2020 21:05:38:  26000000 \\nINFO  @ Tue, 24 Mar 2020 21:05:40:  27000000 \\nINFO  @ Tue, 24 Mar 2020 21:05:42:  28000000 \\nINFO  @ Tue, 24 Mar 2020 21:05:45:  29000000 \\nINFO  @ Tue, 24 Mar 2020 21:05:47:  30000000 \\nINFO  @ Tue, 24 Mar 2020 21:05:49:  31000000 \\nINFO  @ Tue, 24 Mar 2020 21:05:51:  32000000 \\nINFO  @ Tue, 24 Mar 2020 21:05:54:  33000000 \\nINFO  @ Tue, 24 Mar 2020 21:05:56:  34000000 \\nINFO  @ Tue, 24 Mar 2020 21:05:57: 34520959 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 21:05:58: #1 tag size is determined as 49 bps \\nINFO  @ Tue, 24 Mar 2020 21:05:58: #1 tag size = 49.0 \\nINFO  @ Tue, 24 Mar 2020 21:05:58: #1  total tags in treatment: 24095030 \\nINFO  @ Tue, 24 Mar 2020 21:05:58: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 21:05:58: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 21:05:58: #1  tags after filtering in treatment: 24061616 \\nINFO  @ Tue, 24 Mar 2020 21:05:58: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Tue, 24 Mar 2020 21:05:58: #1  total tags in control: 34520959 \\nINFO  @ Tue, 24 Mar 2020 21:05:58: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 21:05:58: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 21:05:58: #1  tags after filtering in control: 34516729 \\nINFO  @ Tue, 24 Mar 2020 21:05:58: #1  Redundant rate of control: 0.00 \\nINFO  @ Tue, 24 Mar 2020 21:05:58: #1 finished! \\nINFO  @ Tue, 24 Mar 2020 21:05:58: #2 Build Peak Model... \\nINFO  @ Tue, 24 Mar 2020 21:05:58: #2 Skipped... \\nINFO  @ Tue, 24 Mar 2020 21:05:58: #2 Use 190 as fragment length \\nINFO  @ Tue, 24 Mar 2020 21:05:58: #3 Call peaks... \\nINFO  @ Tue, 24 Mar 2020 21:05:58: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 24 Mar 2020 21:08:30: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 24 Mar 2020 21:08:30: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1_treat_pileup.bdg \\nINFO  @ Tue, 24 Mar 2020 21:08:30: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1_control_lambda.bdg \\nINFO  @ Tue, 24 Mar 2020 21:08:30: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Tue, 24 Mar 2020 21:08:30: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 24 Mar 2020 21:10:45: #4 Write output xls file... ../../data/diffBinding_hist/diffData_unscaled/mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1_peaks.xls \\nINFO  @ Tue, 24 Mar 2020 21:10:45: #4 Write peak in narrowPeak format file... ../../data/diffBinding_hist/diffData_unscaled/mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1_peaks.narrowPeak \\nINFO  @ Tue, 24 Mar 2020 21:10:45: #4 Write summits bed file... ../../data/diffBinding_hist/diffData_unscaled/mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1_summits.bed \\nINFO  @ Tue, 24 Mar 2020 21:10:45: Done! \\n'\n",
      "20073492 24061616\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "doing diff from ../../data/diffBinding_hist/mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2.mLb.clN.sorted.bam and ../../data/diffBinding_hist/mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "196\n",
      "computing the scaling values\n",
      "b'INFO  @ Tue, 24 Mar 2020 21:18:13: \\n# Command line: callpeak -B -t ../../data/diffBinding_hist/mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 196 -n mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2 --outdir ../../data/diffBinding_hist/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_hist/mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 24 Mar 2020 21:18:13: #1 read tag files... \\nINFO  @ Tue, 24 Mar 2020 21:18:13: #1 read treatment tags... \\nINFO  @ Tue, 24 Mar 2020 21:18:15:  1000000 \\nINFO  @ Tue, 24 Mar 2020 21:18:17:  2000000 \\nINFO  @ Tue, 24 Mar 2020 21:18:19:  3000000 \\nINFO  @ Tue, 24 Mar 2020 21:18:20:  4000000 \\nINFO  @ Tue, 24 Mar 2020 21:18:22:  5000000 \\nINFO  @ Tue, 24 Mar 2020 21:18:24:  6000000 \\nINFO  @ Tue, 24 Mar 2020 21:18:26:  7000000 \\nINFO  @ Tue, 24 Mar 2020 21:18:28:  8000000 \\nINFO  @ Tue, 24 Mar 2020 21:18:30:  9000000 \\nINFO  @ Tue, 24 Mar 2020 21:18:32:  10000000 \\nINFO  @ Tue, 24 Mar 2020 21:18:34:  11000000 \\nINFO  @ Tue, 24 Mar 2020 21:18:36:  12000000 \\nINFO  @ Tue, 24 Mar 2020 21:18:38:  13000000 \\nINFO  @ Tue, 24 Mar 2020 21:18:40:  14000000 \\nINFO  @ Tue, 24 Mar 2020 21:18:42:  15000000 \\nINFO  @ Tue, 24 Mar 2020 21:18:44:  16000000 \\nINFO  @ Tue, 24 Mar 2020 21:18:46:  17000000 \\nINFO  @ Tue, 24 Mar 2020 21:18:48:  18000000 \\nINFO  @ Tue, 24 Mar 2020 21:18:50:  19000000 \\nINFO  @ Tue, 24 Mar 2020 21:18:51:  20000000 \\nINFO  @ Tue, 24 Mar 2020 21:18:53:  21000000 \\nINFO  @ Tue, 24 Mar 2020 21:18:54: 21217359 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 21:18:54: #1.2 read input tags... \\nINFO  @ Tue, 24 Mar 2020 21:18:56:  1000000 \\nINFO  @ Tue, 24 Mar 2020 21:18:59:  2000000 \\nINFO  @ Tue, 24 Mar 2020 21:19:01:  3000000 \\nINFO  @ Tue, 24 Mar 2020 21:19:03:  4000000 \\nINFO  @ Tue, 24 Mar 2020 21:19:06:  5000000 \\nINFO  @ Tue, 24 Mar 2020 21:19:08:  6000000 \\nINFO  @ Tue, 24 Mar 2020 21:19:10:  7000000 \\nINFO  @ Tue, 24 Mar 2020 21:19:13:  8000000 \\nINFO  @ Tue, 24 Mar 2020 21:19:15:  9000000 \\nINFO  @ Tue, 24 Mar 2020 21:19:17:  10000000 \\nINFO  @ Tue, 24 Mar 2020 21:19:20:  11000000 \\nINFO  @ Tue, 24 Mar 2020 21:19:22:  12000000 \\nINFO  @ Tue, 24 Mar 2020 21:19:24:  13000000 \\nINFO  @ Tue, 24 Mar 2020 21:19:27:  14000000 \\nINFO  @ Tue, 24 Mar 2020 21:19:29:  15000000 \\nINFO  @ Tue, 24 Mar 2020 21:19:31:  16000000 \\nINFO  @ Tue, 24 Mar 2020 21:19:34:  17000000 \\nINFO  @ Tue, 24 Mar 2020 21:19:36:  18000000 \\nINFO  @ Tue, 24 Mar 2020 21:19:38:  19000000 \\nINFO  @ Tue, 24 Mar 2020 21:19:40:  20000000 \\nINFO  @ Tue, 24 Mar 2020 21:19:43:  21000000 \\nINFO  @ Tue, 24 Mar 2020 21:19:45:  22000000 \\nINFO  @ Tue, 24 Mar 2020 21:19:47:  23000000 \\nINFO  @ Tue, 24 Mar 2020 21:19:50:  24000000 \\nINFO  @ Tue, 24 Mar 2020 21:19:52:  25000000 \\nINFO  @ Tue, 24 Mar 2020 21:19:54:  26000000 \\nINFO  @ Tue, 24 Mar 2020 21:19:57:  27000000 \\nINFO  @ Tue, 24 Mar 2020 21:19:59:  28000000 \\nINFO  @ Tue, 24 Mar 2020 21:20:01:  29000000 \\nINFO  @ Tue, 24 Mar 2020 21:20:04:  30000000 \\nINFO  @ Tue, 24 Mar 2020 21:20:06:  31000000 \\nINFO  @ Tue, 24 Mar 2020 21:20:08:  32000000 \\nINFO  @ Tue, 24 Mar 2020 21:20:11:  33000000 \\nINFO  @ Tue, 24 Mar 2020 21:20:13:  34000000 \\nINFO  @ Tue, 24 Mar 2020 21:20:14: 34520959 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 21:20:15: #1 tag size is determined as 49 bps \\nINFO  @ Tue, 24 Mar 2020 21:20:15: #1 tag size = 49.0 \\nINFO  @ Tue, 24 Mar 2020 21:20:15: #1  total tags in treatment: 21217359 \\nINFO  @ Tue, 24 Mar 2020 21:20:15: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 21:20:15: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 21:20:15: #1  tags after filtering in treatment: 21185249 \\nINFO  @ Tue, 24 Mar 2020 21:20:15: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Tue, 24 Mar 2020 21:20:15: #1  total tags in control: 34520959 \\nINFO  @ Tue, 24 Mar 2020 21:20:15: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 21:20:15: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 21:20:15: #1  tags after filtering in control: 34516729 \\nINFO  @ Tue, 24 Mar 2020 21:20:15: #1  Redundant rate of control: 0.00 \\nINFO  @ Tue, 24 Mar 2020 21:20:15: #1 finished! \\nINFO  @ Tue, 24 Mar 2020 21:20:15: #2 Build Peak Model... \\nINFO  @ Tue, 24 Mar 2020 21:20:15: #2 Skipped... \\nINFO  @ Tue, 24 Mar 2020 21:20:15: #2 Use 196 as fragment length \\nINFO  @ Tue, 24 Mar 2020 21:20:15: #3 Call peaks... \\nINFO  @ Tue, 24 Mar 2020 21:20:15: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 24 Mar 2020 21:22:46: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 24 Mar 2020 21:22:46: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2_treat_pileup.bdg \\nINFO  @ Tue, 24 Mar 2020 21:22:46: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2_control_lambda.bdg \\nINFO  @ Tue, 24 Mar 2020 21:22:46: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Tue, 24 Mar 2020 21:22:46: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 24 Mar 2020 21:24:59: #4 Write output xls file... ../../data/diffBinding_hist/diffData_unscaled/mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2_peaks.xls \\nINFO  @ Tue, 24 Mar 2020 21:25:00: #4 Write peak in narrowPeak format file... ../../data/diffBinding_hist/diffData_unscaled/mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2_peaks.narrowPeak \\nINFO  @ Tue, 24 Mar 2020 21:25:00: #4 Write summits bed file... ../../data/diffBinding_hist/diffData_unscaled/mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2_summits.bed \\nINFO  @ Tue, 24 Mar 2020 21:25:00: Done! \\n'\n",
      "b'INFO  @ Tue, 24 Mar 2020 21:25:00: \\n# Command line: callpeak -B -t ../../data/diffBinding_hist/mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 196 -n mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2 --outdir ../../data/diffBinding_hist/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_hist/mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 24 Mar 2020 21:25:00: #1 read tag files... \\nINFO  @ Tue, 24 Mar 2020 21:25:00: #1 read treatment tags... \\nINFO  @ Tue, 24 Mar 2020 21:25:02:  1000000 \\nINFO  @ Tue, 24 Mar 2020 21:25:04:  2000000 \\nINFO  @ Tue, 24 Mar 2020 21:25:07:  3000000 \\nINFO  @ Tue, 24 Mar 2020 21:25:09:  4000000 \\nINFO  @ Tue, 24 Mar 2020 21:25:11:  5000000 \\nINFO  @ Tue, 24 Mar 2020 21:25:13:  6000000 \\nINFO  @ Tue, 24 Mar 2020 21:25:15:  7000000 \\nINFO  @ Tue, 24 Mar 2020 21:25:17:  8000000 \\nINFO  @ Tue, 24 Mar 2020 21:25:19:  9000000 \\nINFO  @ Tue, 24 Mar 2020 21:25:21:  10000000 \\nINFO  @ Tue, 24 Mar 2020 21:25:23:  11000000 \\nINFO  @ Tue, 24 Mar 2020 21:25:25:  12000000 \\nINFO  @ Tue, 24 Mar 2020 21:25:27:  13000000 \\nINFO  @ Tue, 24 Mar 2020 21:25:29:  14000000 \\nINFO  @ Tue, 24 Mar 2020 21:25:31:  15000000 \\nINFO  @ Tue, 24 Mar 2020 21:25:33:  16000000 \\nINFO  @ Tue, 24 Mar 2020 21:25:35:  17000000 \\nINFO  @ Tue, 24 Mar 2020 21:25:37:  18000000 \\nINFO  @ Tue, 24 Mar 2020 21:25:39:  19000000 \\nINFO  @ Tue, 24 Mar 2020 21:25:41:  20000000 \\nINFO  @ Tue, 24 Mar 2020 21:25:43:  21000000 \\nINFO  @ Tue, 24 Mar 2020 21:25:45:  22000000 \\nINFO  @ Tue, 24 Mar 2020 21:25:47:  23000000 \\nINFO  @ Tue, 24 Mar 2020 21:25:50:  24000000 \\nINFO  @ Tue, 24 Mar 2020 21:25:51:  25000000 \\nINFO  @ Tue, 24 Mar 2020 21:25:53:  26000000 \\nINFO  @ Tue, 24 Mar 2020 21:25:55:  27000000 \\nINFO  @ Tue, 24 Mar 2020 21:25:57: 27827822 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 21:25:57: #1.2 read input tags... \\nINFO  @ Tue, 24 Mar 2020 21:26:00:  1000000 \\nINFO  @ Tue, 24 Mar 2020 21:26:02:  2000000 \\nINFO  @ Tue, 24 Mar 2020 21:26:05:  3000000 \\nINFO  @ Tue, 24 Mar 2020 21:26:07:  4000000 \\nINFO  @ Tue, 24 Mar 2020 21:26:09:  5000000 \\nINFO  @ Tue, 24 Mar 2020 21:26:11:  6000000 \\nINFO  @ Tue, 24 Mar 2020 21:26:14:  7000000 \\nINFO  @ Tue, 24 Mar 2020 21:26:16:  8000000 \\nINFO  @ Tue, 24 Mar 2020 21:26:18:  9000000 \\nINFO  @ Tue, 24 Mar 2020 21:26:21:  10000000 \\nINFO  @ Tue, 24 Mar 2020 21:26:23:  11000000 \\nINFO  @ Tue, 24 Mar 2020 21:26:25:  12000000 \\nINFO  @ Tue, 24 Mar 2020 21:26:28:  13000000 \\nINFO  @ Tue, 24 Mar 2020 21:26:30:  14000000 \\nINFO  @ Tue, 24 Mar 2020 21:26:32:  15000000 \\nINFO  @ Tue, 24 Mar 2020 21:26:35:  16000000 \\nINFO  @ Tue, 24 Mar 2020 21:26:37:  17000000 \\nINFO  @ Tue, 24 Mar 2020 21:26:39:  18000000 \\nINFO  @ Tue, 24 Mar 2020 21:26:42:  19000000 \\nINFO  @ Tue, 24 Mar 2020 21:26:44:  20000000 \\nINFO  @ Tue, 24 Mar 2020 21:26:46:  21000000 \\nINFO  @ Tue, 24 Mar 2020 21:26:49:  22000000 \\nINFO  @ Tue, 24 Mar 2020 21:26:51:  23000000 \\nINFO  @ Tue, 24 Mar 2020 21:26:53:  24000000 \\nINFO  @ Tue, 24 Mar 2020 21:26:56:  25000000 \\nINFO  @ Tue, 24 Mar 2020 21:26:58:  26000000 \\nINFO  @ Tue, 24 Mar 2020 21:27:00:  27000000 \\nINFO  @ Tue, 24 Mar 2020 21:27:03:  28000000 \\nINFO  @ Tue, 24 Mar 2020 21:27:05:  29000000 \\nINFO  @ Tue, 24 Mar 2020 21:27:07:  30000000 \\nINFO  @ Tue, 24 Mar 2020 21:27:10:  31000000 \\nINFO  @ Tue, 24 Mar 2020 21:27:12:  32000000 \\nINFO  @ Tue, 24 Mar 2020 21:27:14:  33000000 \\nINFO  @ Tue, 24 Mar 2020 21:27:17:  34000000 \\nINFO  @ Tue, 24 Mar 2020 21:27:18: 34520959 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 21:27:18: #1 tag size is determined as 49 bps \\nINFO  @ Tue, 24 Mar 2020 21:27:18: #1 tag size = 49.0 \\nINFO  @ Tue, 24 Mar 2020 21:27:18: #1  total tags in treatment: 27827822 \\nINFO  @ Tue, 24 Mar 2020 21:27:18: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 21:27:18: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 21:27:18: #1  tags after filtering in treatment: 27787785 \\nINFO  @ Tue, 24 Mar 2020 21:27:18: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Tue, 24 Mar 2020 21:27:18: #1  total tags in control: 34520959 \\nINFO  @ Tue, 24 Mar 2020 21:27:18: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 21:27:18: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 21:27:19: #1  tags after filtering in control: 34516729 \\nINFO  @ Tue, 24 Mar 2020 21:27:19: #1  Redundant rate of control: 0.00 \\nINFO  @ Tue, 24 Mar 2020 21:27:19: #1 finished! \\nINFO  @ Tue, 24 Mar 2020 21:27:19: #2 Build Peak Model... \\nINFO  @ Tue, 24 Mar 2020 21:27:19: #2 Skipped... \\nINFO  @ Tue, 24 Mar 2020 21:27:19: #2 Use 196 as fragment length \\nINFO  @ Tue, 24 Mar 2020 21:27:19: #3 Call peaks... \\nINFO  @ Tue, 24 Mar 2020 21:27:19: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 24 Mar 2020 21:29:52: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 24 Mar 2020 21:29:52: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2_treat_pileup.bdg \\nINFO  @ Tue, 24 Mar 2020 21:29:52: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2_control_lambda.bdg \\nINFO  @ Tue, 24 Mar 2020 21:29:52: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Tue, 24 Mar 2020 21:29:52: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 24 Mar 2020 21:32:04: #4 Write output xls file... ../../data/diffBinding_hist/diffData_unscaled/mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2_peaks.xls \\nINFO  @ Tue, 24 Mar 2020 21:32:04: #4 Write peak in narrowPeak format file... ../../data/diffBinding_hist/diffData_unscaled/mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2_peaks.narrowPeak \\nINFO  @ Tue, 24 Mar 2020 21:32:04: #4 Write summits bed file... ../../data/diffBinding_hist/diffData_unscaled/mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2_summits.bed \\nINFO  @ Tue, 24 Mar 2020 21:32:04: Done! \\n'\n",
      "21185249 27787785\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "doing diff from ../../data/diffBinding_hist/mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1.mLb.clN.sorted.bam and ../../data/diffBinding_hist/mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "287\n",
      "computing the scaling values\n",
      "b'INFO  @ Tue, 24 Mar 2020 21:40:17: \\n# Command line: callpeak -B -t ../../data/diffBinding_hist/mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 287 -n mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1 --outdir ../../data/diffBinding_hist/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_hist/mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 24 Mar 2020 21:40:17: #1 read tag files... \\nINFO  @ Tue, 24 Mar 2020 21:40:17: #1 read treatment tags... \\nINFO  @ Tue, 24 Mar 2020 21:40:20:  1000000 \\nINFO  @ Tue, 24 Mar 2020 21:40:22:  2000000 \\nINFO  @ Tue, 24 Mar 2020 21:40:24:  3000000 \\nINFO  @ Tue, 24 Mar 2020 21:40:26:  4000000 \\nINFO  @ Tue, 24 Mar 2020 21:40:28:  5000000 \\nINFO  @ Tue, 24 Mar 2020 21:40:30:  6000000 \\nINFO  @ Tue, 24 Mar 2020 21:40:32:  7000000 \\nINFO  @ Tue, 24 Mar 2020 21:40:34:  8000000 \\nINFO  @ Tue, 24 Mar 2020 21:40:37:  9000000 \\nINFO  @ Tue, 24 Mar 2020 21:40:39:  10000000 \\nINFO  @ Tue, 24 Mar 2020 21:40:41:  11000000 \\nINFO  @ Tue, 24 Mar 2020 21:40:43:  12000000 \\nINFO  @ Tue, 24 Mar 2020 21:40:45:  13000000 \\nINFO  @ Tue, 24 Mar 2020 21:40:47:  14000000 \\nINFO  @ Tue, 24 Mar 2020 21:40:50:  15000000 \\nINFO  @ Tue, 24 Mar 2020 21:40:52:  16000000 \\nINFO  @ Tue, 24 Mar 2020 21:40:54:  17000000 \\nINFO  @ Tue, 24 Mar 2020 21:40:57:  18000000 \\nINFO  @ Tue, 24 Mar 2020 21:40:59:  19000000 \\nINFO  @ Tue, 24 Mar 2020 21:41:02:  20000000 \\nINFO  @ Tue, 24 Mar 2020 21:41:04:  21000000 \\nINFO  @ Tue, 24 Mar 2020 21:41:06:  22000000 \\nINFO  @ Tue, 24 Mar 2020 21:41:08:  23000000 \\nINFO  @ Tue, 24 Mar 2020 21:41:10:  24000000 \\nINFO  @ Tue, 24 Mar 2020 21:41:12:  25000000 \\nINFO  @ Tue, 24 Mar 2020 21:41:15:  26000000 \\nINFO  @ Tue, 24 Mar 2020 21:41:17:  27000000 \\nINFO  @ Tue, 24 Mar 2020 21:41:19:  28000000 \\nINFO  @ Tue, 24 Mar 2020 21:41:21:  29000000 \\nINFO  @ Tue, 24 Mar 2020 21:41:23:  30000000 \\nINFO  @ Tue, 24 Mar 2020 21:41:25:  31000000 \\nINFO  @ Tue, 24 Mar 2020 21:41:28:  32000000 \\nINFO  @ Tue, 24 Mar 2020 21:41:30:  33000000 \\nINFO  @ Tue, 24 Mar 2020 21:41:32:  34000000 \\nINFO  @ Tue, 24 Mar 2020 21:41:34:  35000000 \\nINFO  @ Tue, 24 Mar 2020 21:41:36: 35939092 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 21:41:36: #1.2 read input tags... \\nINFO  @ Tue, 24 Mar 2020 21:41:39:  1000000 \\nINFO  @ Tue, 24 Mar 2020 21:41:41:  2000000 \\nINFO  @ Tue, 24 Mar 2020 21:41:43:  3000000 \\nINFO  @ Tue, 24 Mar 2020 21:41:46:  4000000 \\nINFO  @ Tue, 24 Mar 2020 21:41:48:  5000000 \\nINFO  @ Tue, 24 Mar 2020 21:41:50:  6000000 \\nINFO  @ Tue, 24 Mar 2020 21:41:53:  7000000 \\nINFO  @ Tue, 24 Mar 2020 21:41:55:  8000000 \\nINFO  @ Tue, 24 Mar 2020 21:41:57:  9000000 \\nINFO  @ Tue, 24 Mar 2020 21:42:00:  10000000 \\nINFO  @ Tue, 24 Mar 2020 21:42:02:  11000000 \\nINFO  @ Tue, 24 Mar 2020 21:42:04:  12000000 \\nINFO  @ Tue, 24 Mar 2020 21:42:07:  13000000 \\nINFO  @ Tue, 24 Mar 2020 21:42:09:  14000000 \\nINFO  @ Tue, 24 Mar 2020 21:42:11:  15000000 \\nINFO  @ Tue, 24 Mar 2020 21:42:14:  16000000 \\nINFO  @ Tue, 24 Mar 2020 21:42:16:  17000000 \\nINFO  @ Tue, 24 Mar 2020 21:42:18:  18000000 \\nINFO  @ Tue, 24 Mar 2020 21:42:20:  19000000 \\nINFO  @ Tue, 24 Mar 2020 21:42:23:  20000000 \\nINFO  @ Tue, 24 Mar 2020 21:42:25:  21000000 \\nINFO  @ Tue, 24 Mar 2020 21:42:27:  22000000 \\nINFO  @ Tue, 24 Mar 2020 21:42:30:  23000000 \\nINFO  @ Tue, 24 Mar 2020 21:42:32:  24000000 \\nINFO  @ Tue, 24 Mar 2020 21:42:34:  25000000 \\nINFO  @ Tue, 24 Mar 2020 21:42:37:  26000000 \\nINFO  @ Tue, 24 Mar 2020 21:42:39:  27000000 \\nINFO  @ Tue, 24 Mar 2020 21:42:41:  28000000 \\nINFO  @ Tue, 24 Mar 2020 21:42:44:  29000000 \\nINFO  @ Tue, 24 Mar 2020 21:42:46:  30000000 \\nINFO  @ Tue, 24 Mar 2020 21:42:48:  31000000 \\nINFO  @ Tue, 24 Mar 2020 21:42:51:  32000000 \\nINFO  @ Tue, 24 Mar 2020 21:42:53:  33000000 \\nINFO  @ Tue, 24 Mar 2020 21:42:55:  34000000 \\nINFO  @ Tue, 24 Mar 2020 21:42:57: 34520959 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 21:42:57: #1 tag size is determined as 49 bps \\nINFO  @ Tue, 24 Mar 2020 21:42:57: #1 tag size = 49.0 \\nINFO  @ Tue, 24 Mar 2020 21:42:57: #1  total tags in treatment: 35939092 \\nINFO  @ Tue, 24 Mar 2020 21:42:57: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 21:42:57: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 21:42:57: #1  tags after filtering in treatment: 35930039 \\nINFO  @ Tue, 24 Mar 2020 21:42:57: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Tue, 24 Mar 2020 21:42:57: #1  total tags in control: 34520959 \\nINFO  @ Tue, 24 Mar 2020 21:42:57: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 21:42:57: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 21:42:57: #1  tags after filtering in control: 34516729 \\nINFO  @ Tue, 24 Mar 2020 21:42:57: #1  Redundant rate of control: 0.00 \\nINFO  @ Tue, 24 Mar 2020 21:42:57: #1 finished! \\nINFO  @ Tue, 24 Mar 2020 21:42:57: #2 Build Peak Model... \\nINFO  @ Tue, 24 Mar 2020 21:42:57: #2 Skipped... \\nINFO  @ Tue, 24 Mar 2020 21:42:57: #2 Use 287 as fragment length \\nINFO  @ Tue, 24 Mar 2020 21:42:57: #3 Call peaks... \\nINFO  @ Tue, 24 Mar 2020 21:42:57: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 24 Mar 2020 21:46:06: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 24 Mar 2020 21:46:06: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1_treat_pileup.bdg \\nINFO  @ Tue, 24 Mar 2020 21:46:06: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1_control_lambda.bdg \\nINFO  @ Tue, 24 Mar 2020 21:46:06: #3   Pileup will be based on sequencing depth in control. \\nINFO  @ Tue, 24 Mar 2020 21:46:06: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 24 Mar 2020 21:49:17: #4 Write output xls file... ../../data/diffBinding_hist/diffData_unscaled/mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1_peaks.xls \\nINFO  @ Tue, 24 Mar 2020 21:49:17: #4 Write peak in narrowPeak format file... ../../data/diffBinding_hist/diffData_unscaled/mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1_peaks.narrowPeak \\nINFO  @ Tue, 24 Mar 2020 21:49:17: #4 Write summits bed file... ../../data/diffBinding_hist/diffData_unscaled/mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1_summits.bed \\nINFO  @ Tue, 24 Mar 2020 21:49:17: Done! \\n'\n",
      "b'INFO  @ Tue, 24 Mar 2020 21:49:18: \\n# Command line: callpeak -B -t ../../data/diffBinding_hist/mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 287 -n mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1 --outdir ../../data/diffBinding_hist/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_hist/mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 24 Mar 2020 21:49:18: #1 read tag files... \\nINFO  @ Tue, 24 Mar 2020 21:49:18: #1 read treatment tags... \\nINFO  @ Tue, 24 Mar 2020 21:49:20:  1000000 \\nINFO  @ Tue, 24 Mar 2020 21:49:22:  2000000 \\nINFO  @ Tue, 24 Mar 2020 21:49:24:  3000000 \\nINFO  @ Tue, 24 Mar 2020 21:49:26:  4000000 \\nINFO  @ Tue, 24 Mar 2020 21:49:29:  5000000 \\nINFO  @ Tue, 24 Mar 2020 21:49:31:  6000000 \\nINFO  @ Tue, 24 Mar 2020 21:49:33:  7000000 \\nINFO  @ Tue, 24 Mar 2020 21:49:35:  8000000 \\nINFO  @ Tue, 24 Mar 2020 21:49:37:  9000000 \\nINFO  @ Tue, 24 Mar 2020 21:49:40:  10000000 \\nINFO  @ Tue, 24 Mar 2020 21:49:42:  11000000 \\nINFO  @ Tue, 24 Mar 2020 21:49:44:  12000000 \\nINFO  @ Tue, 24 Mar 2020 21:49:46:  13000000 \\nINFO  @ Tue, 24 Mar 2020 21:49:48:  14000000 \\nINFO  @ Tue, 24 Mar 2020 21:49:50:  15000000 \\nINFO  @ Tue, 24 Mar 2020 21:49:52:  16000000 \\nINFO  @ Tue, 24 Mar 2020 21:49:55:  17000000 \\nINFO  @ Tue, 24 Mar 2020 21:49:57:  18000000 \\nINFO  @ Tue, 24 Mar 2020 21:49:59:  19000000 \\nINFO  @ Tue, 24 Mar 2020 21:50:01:  20000000 \\nINFO  @ Tue, 24 Mar 2020 21:50:03:  21000000 \\nINFO  @ Tue, 24 Mar 2020 21:50:06:  22000000 \\nINFO  @ Tue, 24 Mar 2020 21:50:08:  23000000 \\nINFO  @ Tue, 24 Mar 2020 21:50:10:  24000000 \\nINFO  @ Tue, 24 Mar 2020 21:50:12:  25000000 \\nINFO  @ Tue, 24 Mar 2020 21:50:14:  26000000 \\nINFO  @ Tue, 24 Mar 2020 21:50:16:  27000000 \\nINFO  @ Tue, 24 Mar 2020 21:50:19:  28000000 \\nINFO  @ Tue, 24 Mar 2020 21:50:21:  29000000 \\nINFO  @ Tue, 24 Mar 2020 21:50:23:  30000000 \\nINFO  @ Tue, 24 Mar 2020 21:50:25:  31000000 \\nINFO  @ Tue, 24 Mar 2020 21:50:27:  32000000 \\nINFO  @ Tue, 24 Mar 2020 21:50:30:  33000000 \\nINFO  @ Tue, 24 Mar 2020 21:50:32:  34000000 \\nINFO  @ Tue, 24 Mar 2020 21:50:34:  35000000 \\nINFO  @ Tue, 24 Mar 2020 21:50:36:  36000000 \\nINFO  @ Tue, 24 Mar 2020 21:50:38:  37000000 \\nINFO  @ Tue, 24 Mar 2020 21:50:40:  38000000 \\nINFO  @ Tue, 24 Mar 2020 21:50:43:  39000000 \\nINFO  @ Tue, 24 Mar 2020 21:50:45:  40000000 \\nINFO  @ Tue, 24 Mar 2020 21:50:47:  41000000 \\nINFO  @ Tue, 24 Mar 2020 21:50:49:  42000000 \\nINFO  @ Tue, 24 Mar 2020 21:50:51:  43000000 \\nINFO  @ Tue, 24 Mar 2020 21:50:53:  44000000 \\nINFO  @ Tue, 24 Mar 2020 21:50:56:  45000000 \\nINFO  @ Tue, 24 Mar 2020 21:50:58:  46000000 \\nINFO  @ Tue, 24 Mar 2020 21:51:00:  47000000 \\nINFO  @ Tue, 24 Mar 2020 21:51:02:  48000000 \\nINFO  @ Tue, 24 Mar 2020 21:51:04:  49000000 \\nINFO  @ Tue, 24 Mar 2020 21:51:06:  50000000 \\nINFO  @ Tue, 24 Mar 2020 21:51:08:  51000000 \\nINFO  @ Tue, 24 Mar 2020 21:51:11:  52000000 \\nINFO  @ Tue, 24 Mar 2020 21:51:13:  53000000 \\nINFO  @ Tue, 24 Mar 2020 21:51:15:  54000000 \\nINFO  @ Tue, 24 Mar 2020 21:51:17: 54882374 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 21:51:17: #1.2 read input tags... \\nINFO  @ Tue, 24 Mar 2020 21:51:20:  1000000 \\nINFO  @ Tue, 24 Mar 2020 21:51:22:  2000000 \\nINFO  @ Tue, 24 Mar 2020 21:51:24:  3000000 \\nINFO  @ Tue, 24 Mar 2020 21:51:27:  4000000 \\nINFO  @ Tue, 24 Mar 2020 21:51:29:  5000000 \\nINFO  @ Tue, 24 Mar 2020 21:51:31:  6000000 \\nINFO  @ Tue, 24 Mar 2020 21:51:33:  7000000 \\nINFO  @ Tue, 24 Mar 2020 21:51:36:  8000000 \\nINFO  @ Tue, 24 Mar 2020 21:51:38:  9000000 \\nINFO  @ Tue, 24 Mar 2020 21:51:40:  10000000 \\nINFO  @ Tue, 24 Mar 2020 21:51:43:  11000000 \\nINFO  @ Tue, 24 Mar 2020 21:51:45:  12000000 \\nINFO  @ Tue, 24 Mar 2020 21:51:47:  13000000 \\nINFO  @ Tue, 24 Mar 2020 21:51:49:  14000000 \\nINFO  @ Tue, 24 Mar 2020 21:51:52:  15000000 \\nINFO  @ Tue, 24 Mar 2020 21:51:54:  16000000 \\nINFO  @ Tue, 24 Mar 2020 21:51:56:  17000000 \\nINFO  @ Tue, 24 Mar 2020 21:51:59:  18000000 \\nINFO  @ Tue, 24 Mar 2020 21:52:01:  19000000 \\nINFO  @ Tue, 24 Mar 2020 21:52:03:  20000000 \\nINFO  @ Tue, 24 Mar 2020 21:52:06:  21000000 \\nINFO  @ Tue, 24 Mar 2020 21:52:08:  22000000 \\nINFO  @ Tue, 24 Mar 2020 21:52:10:  23000000 \\nINFO  @ Tue, 24 Mar 2020 21:52:13:  24000000 \\nINFO  @ Tue, 24 Mar 2020 21:52:15:  25000000 \\nINFO  @ Tue, 24 Mar 2020 21:52:17:  26000000 \\nINFO  @ Tue, 24 Mar 2020 21:52:19:  27000000 \\nINFO  @ Tue, 24 Mar 2020 21:52:22:  28000000 \\nINFO  @ Tue, 24 Mar 2020 21:52:24:  29000000 \\nINFO  @ Tue, 24 Mar 2020 21:52:26:  30000000 \\nINFO  @ Tue, 24 Mar 2020 21:52:29:  31000000 \\nINFO  @ Tue, 24 Mar 2020 21:52:31:  32000000 \\nINFO  @ Tue, 24 Mar 2020 21:52:33:  33000000 \\nINFO  @ Tue, 24 Mar 2020 21:52:36:  34000000 \\nINFO  @ Tue, 24 Mar 2020 21:52:37: 34520959 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 21:52:37: #1 tag size is determined as 50 bps \\nINFO  @ Tue, 24 Mar 2020 21:52:37: #1 tag size = 50.0 \\nINFO  @ Tue, 24 Mar 2020 21:52:37: #1  total tags in treatment: 54882374 \\nINFO  @ Tue, 24 Mar 2020 21:52:37: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 21:52:37: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 21:52:38: #1  tags after filtering in treatment: 54859593 \\nINFO  @ Tue, 24 Mar 2020 21:52:38: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Tue, 24 Mar 2020 21:52:38: #1  total tags in control: 34520959 \\nINFO  @ Tue, 24 Mar 2020 21:52:38: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 21:52:38: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 21:52:38: #1  tags after filtering in control: 34516729 \\nINFO  @ Tue, 24 Mar 2020 21:52:38: #1  Redundant rate of control: 0.00 \\nINFO  @ Tue, 24 Mar 2020 21:52:38: #1 finished! \\nINFO  @ Tue, 24 Mar 2020 21:52:38: #2 Build Peak Model... \\nINFO  @ Tue, 24 Mar 2020 21:52:38: #2 Skipped... \\nINFO  @ Tue, 24 Mar 2020 21:52:38: #2 Use 287 as fragment length \\nINFO  @ Tue, 24 Mar 2020 21:52:38: #3 Call peaks... \\nINFO  @ Tue, 24 Mar 2020 21:52:38: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 24 Mar 2020 21:56:05: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 24 Mar 2020 21:56:05: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1_treat_pileup.bdg \\nINFO  @ Tue, 24 Mar 2020 21:56:05: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1_control_lambda.bdg \\nINFO  @ Tue, 24 Mar 2020 21:56:05: #3   Pileup will be based on sequencing depth in control. \\nINFO  @ Tue, 24 Mar 2020 21:56:05: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 24 Mar 2020 22:00:43: #4 Write output xls file... ../../data/diffBinding_hist/diffData_unscaled/mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1_peaks.xls \\nINFO  @ Tue, 24 Mar 2020 22:00:43: #4 Write peak in narrowPeak format file... ../../data/diffBinding_hist/diffData_unscaled/mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1_peaks.narrowPeak \\nINFO  @ Tue, 24 Mar 2020 22:00:43: #4 Write summits bed file... ../../data/diffBinding_hist/diffData_unscaled/mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1_summits.bed \\nINFO  @ Tue, 24 Mar 2020 22:00:43: Done! \\n'\n",
      "34516729 34516729\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "doing diff from ../../data/diffBinding_hist/mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2.mLb.clN.sorted.bam and ../../data/diffBinding_hist/mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "288\n",
      "computing the scaling values\n",
      "b'INFO  @ Tue, 24 Mar 2020 22:15:01: \\n# Command line: callpeak -B -t ../../data/diffBinding_hist/mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 288 -n mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2 --outdir ../../data/diffBinding_hist/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_hist/mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 24 Mar 2020 22:15:01: #1 read tag files... \\nINFO  @ Tue, 24 Mar 2020 22:15:01: #1 read treatment tags... \\nINFO  @ Tue, 24 Mar 2020 22:15:03:  1000000 \\nINFO  @ Tue, 24 Mar 2020 22:15:05:  2000000 \\nINFO  @ Tue, 24 Mar 2020 22:15:07:  3000000 \\nINFO  @ Tue, 24 Mar 2020 22:15:09:  4000000 \\nINFO  @ Tue, 24 Mar 2020 22:15:11:  5000000 \\nINFO  @ Tue, 24 Mar 2020 22:15:13:  6000000 \\nINFO  @ Tue, 24 Mar 2020 22:15:15:  7000000 \\nINFO  @ Tue, 24 Mar 2020 22:15:17:  8000000 \\nINFO  @ Tue, 24 Mar 2020 22:15:19:  9000000 \\nINFO  @ Tue, 24 Mar 2020 22:15:21:  10000000 \\nINFO  @ Tue, 24 Mar 2020 22:15:23:  11000000 \\nINFO  @ Tue, 24 Mar 2020 22:15:25:  12000000 \\nINFO  @ Tue, 24 Mar 2020 22:15:27:  13000000 \\nINFO  @ Tue, 24 Mar 2020 22:15:30:  14000000 \\nINFO  @ Tue, 24 Mar 2020 22:15:32:  15000000 \\nINFO  @ Tue, 24 Mar 2020 22:15:34:  16000000 \\nINFO  @ Tue, 24 Mar 2020 22:15:36:  17000000 \\nINFO  @ Tue, 24 Mar 2020 22:15:38:  18000000 \\nINFO  @ Tue, 24 Mar 2020 22:15:40:  19000000 \\nINFO  @ Tue, 24 Mar 2020 22:15:42:  20000000 \\nINFO  @ Tue, 24 Mar 2020 22:15:44:  21000000 \\nINFO  @ Tue, 24 Mar 2020 22:15:46:  22000000 \\nINFO  @ Tue, 24 Mar 2020 22:15:48:  23000000 \\nINFO  @ Tue, 24 Mar 2020 22:15:50:  24000000 \\nINFO  @ Tue, 24 Mar 2020 22:15:52:  25000000 \\nINFO  @ Tue, 24 Mar 2020 22:15:54:  26000000 \\nINFO  @ Tue, 24 Mar 2020 22:15:56:  27000000 \\nINFO  @ Tue, 24 Mar 2020 22:15:58:  28000000 \\nINFO  @ Tue, 24 Mar 2020 22:16:00:  29000000 \\nINFO  @ Tue, 24 Mar 2020 22:16:02:  30000000 \\nINFO  @ Tue, 24 Mar 2020 22:16:04:  31000000 \\nINFO  @ Tue, 24 Mar 2020 22:16:06:  32000000 \\nINFO  @ Tue, 24 Mar 2020 22:16:08:  33000000 \\nINFO  @ Tue, 24 Mar 2020 22:16:10:  34000000 \\nINFO  @ Tue, 24 Mar 2020 22:16:12:  35000000 \\nINFO  @ Tue, 24 Mar 2020 22:16:14:  36000000 \\nINFO  @ Tue, 24 Mar 2020 22:16:16:  37000000 \\nINFO  @ Tue, 24 Mar 2020 22:16:18:  38000000 \\nINFO  @ Tue, 24 Mar 2020 22:16:20:  39000000 \\nINFO  @ Tue, 24 Mar 2020 22:16:22:  40000000 \\nINFO  @ Tue, 24 Mar 2020 22:16:24:  41000000 \\nINFO  @ Tue, 24 Mar 2020 22:16:26:  42000000 \\nINFO  @ Tue, 24 Mar 2020 22:16:28:  43000000 \\nINFO  @ Tue, 24 Mar 2020 22:16:30:  44000000 \\nINFO  @ Tue, 24 Mar 2020 22:16:31: 44655594 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 22:16:32: #1.2 read input tags... \\nINFO  @ Tue, 24 Mar 2020 22:16:34:  1000000 \\nINFO  @ Tue, 24 Mar 2020 22:16:36:  2000000 \\nINFO  @ Tue, 24 Mar 2020 22:16:39:  3000000 \\nINFO  @ Tue, 24 Mar 2020 22:16:41:  4000000 \\nINFO  @ Tue, 24 Mar 2020 22:16:43:  5000000 \\nINFO  @ Tue, 24 Mar 2020 22:16:45:  6000000 \\nINFO  @ Tue, 24 Mar 2020 22:16:48:  7000000 \\nINFO  @ Tue, 24 Mar 2020 22:16:50:  8000000 \\nINFO  @ Tue, 24 Mar 2020 22:16:52:  9000000 \\nINFO  @ Tue, 24 Mar 2020 22:16:54:  10000000 \\nINFO  @ Tue, 24 Mar 2020 22:16:57:  11000000 \\nINFO  @ Tue, 24 Mar 2020 22:16:59:  12000000 \\nINFO  @ Tue, 24 Mar 2020 22:17:01:  13000000 \\nINFO  @ Tue, 24 Mar 2020 22:17:03:  14000000 \\nINFO  @ Tue, 24 Mar 2020 22:17:06:  15000000 \\nINFO  @ Tue, 24 Mar 2020 22:17:08:  16000000 \\nINFO  @ Tue, 24 Mar 2020 22:17:10:  17000000 \\nINFO  @ Tue, 24 Mar 2020 22:17:12:  18000000 \\nINFO  @ Tue, 24 Mar 2020 22:17:14:  19000000 \\nINFO  @ Tue, 24 Mar 2020 22:17:17:  20000000 \\nINFO  @ Tue, 24 Mar 2020 22:17:19:  21000000 \\nINFO  @ Tue, 24 Mar 2020 22:17:21:  22000000 \\nINFO  @ Tue, 24 Mar 2020 22:17:24:  23000000 \\nINFO  @ Tue, 24 Mar 2020 22:17:26:  24000000 \\nINFO  @ Tue, 24 Mar 2020 22:17:28:  25000000 \\nINFO  @ Tue, 24 Mar 2020 22:17:30:  26000000 \\nINFO  @ Tue, 24 Mar 2020 22:17:33:  27000000 \\nINFO  @ Tue, 24 Mar 2020 22:17:35:  28000000 \\nINFO  @ Tue, 24 Mar 2020 22:17:37:  29000000 \\nINFO  @ Tue, 24 Mar 2020 22:17:39:  30000000 \\nINFO  @ Tue, 24 Mar 2020 22:17:44:  31000000 \\nINFO  @ Tue, 24 Mar 2020 22:17:47:  32000000 \\nINFO  @ Tue, 24 Mar 2020 22:17:49:  33000000 \\nINFO  @ Tue, 24 Mar 2020 22:17:51:  34000000 \\nINFO  @ Tue, 24 Mar 2020 22:17:52: 34520959 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 22:17:53: #1 tag size is determined as 49 bps \\nINFO  @ Tue, 24 Mar 2020 22:17:53: #1 tag size = 49.0 \\nINFO  @ Tue, 24 Mar 2020 22:17:53: #1  total tags in treatment: 44655594 \\nINFO  @ Tue, 24 Mar 2020 22:17:53: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 22:17:53: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 22:17:53: #1  tags after filtering in treatment: 44636592 \\nINFO  @ Tue, 24 Mar 2020 22:17:53: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Tue, 24 Mar 2020 22:17:53: #1  total tags in control: 34520959 \\nINFO  @ Tue, 24 Mar 2020 22:17:53: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 22:17:53: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 22:17:53: #1  tags after filtering in control: 34516729 \\nINFO  @ Tue, 24 Mar 2020 22:17:53: #1  Redundant rate of control: 0.00 \\nINFO  @ Tue, 24 Mar 2020 22:17:53: #1 finished! \\nINFO  @ Tue, 24 Mar 2020 22:17:53: #2 Build Peak Model... \\nINFO  @ Tue, 24 Mar 2020 22:17:53: #2 Skipped... \\nINFO  @ Tue, 24 Mar 2020 22:17:53: #2 Use 288 as fragment length \\nINFO  @ Tue, 24 Mar 2020 22:17:53: #3 Call peaks... \\nINFO  @ Tue, 24 Mar 2020 22:17:53: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 24 Mar 2020 22:21:02: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 24 Mar 2020 22:21:02: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2_treat_pileup.bdg \\nINFO  @ Tue, 24 Mar 2020 22:21:02: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2_control_lambda.bdg \\nINFO  @ Tue, 24 Mar 2020 22:21:02: #3   Pileup will be based on sequencing depth in control. \\nINFO  @ Tue, 24 Mar 2020 22:21:02: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 24 Mar 2020 22:24:20: #4 Write output xls file... ../../data/diffBinding_hist/diffData_unscaled/mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2_peaks.xls \\nINFO  @ Tue, 24 Mar 2020 22:24:20: #4 Write peak in narrowPeak format file... ../../data/diffBinding_hist/diffData_unscaled/mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2_peaks.narrowPeak \\nINFO  @ Tue, 24 Mar 2020 22:24:20: #4 Write summits bed file... ../../data/diffBinding_hist/diffData_unscaled/mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2_summits.bed \\nINFO  @ Tue, 24 Mar 2020 22:24:20: Done! \\n'\n",
      "b'INFO  @ Tue, 24 Mar 2020 22:24:20: \\n# Command line: callpeak -B -t ../../data/diffBinding_hist/mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 288 -n mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2 --outdir ../../data/diffBinding_hist/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/diffBinding_hist/mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 24 Mar 2020 22:24:20: #1 read tag files... \\nINFO  @ Tue, 24 Mar 2020 22:24:20: #1 read treatment tags... \\nINFO  @ Tue, 24 Mar 2020 22:24:23:  1000000 \\nINFO  @ Tue, 24 Mar 2020 22:24:25:  2000000 \\nINFO  @ Tue, 24 Mar 2020 22:24:27:  3000000 \\nINFO  @ Tue, 24 Mar 2020 22:24:30:  4000000 \\nINFO  @ Tue, 24 Mar 2020 22:24:32:  5000000 \\nINFO  @ Tue, 24 Mar 2020 22:24:34:  6000000 \\nINFO  @ Tue, 24 Mar 2020 22:24:36:  7000000 \\nINFO  @ Tue, 24 Mar 2020 22:24:38:  8000000 \\nINFO  @ Tue, 24 Mar 2020 22:24:40:  9000000 \\nINFO  @ Tue, 24 Mar 2020 22:24:43:  10000000 \\nINFO  @ Tue, 24 Mar 2020 22:24:45:  11000000 \\nINFO  @ Tue, 24 Mar 2020 22:24:47:  12000000 \\nINFO  @ Tue, 24 Mar 2020 22:24:49:  13000000 \\nINFO  @ Tue, 24 Mar 2020 22:24:51:  14000000 \\nINFO  @ Tue, 24 Mar 2020 22:24:54:  15000000 \\nINFO  @ Tue, 24 Mar 2020 22:24:56:  16000000 \\nINFO  @ Tue, 24 Mar 2020 22:24:58:  17000000 \\nINFO  @ Tue, 24 Mar 2020 22:25:00:  18000000 \\nINFO  @ Tue, 24 Mar 2020 22:25:02:  19000000 \\nINFO  @ Tue, 24 Mar 2020 22:25:04:  20000000 \\nINFO  @ Tue, 24 Mar 2020 22:25:07:  21000000 \\nINFO  @ Tue, 24 Mar 2020 22:25:09:  22000000 \\nINFO  @ Tue, 24 Mar 2020 22:25:11:  23000000 \\nINFO  @ Tue, 24 Mar 2020 22:25:13:  24000000 \\nINFO  @ Tue, 24 Mar 2020 22:25:16:  25000000 \\nINFO  @ Tue, 24 Mar 2020 22:25:18:  26000000 \\nINFO  @ Tue, 24 Mar 2020 22:25:20:  27000000 \\nINFO  @ Tue, 24 Mar 2020 22:25:22:  28000000 \\nINFO  @ Tue, 24 Mar 2020 22:25:24:  29000000 \\nINFO  @ Tue, 24 Mar 2020 22:25:26:  30000000 \\nINFO  @ Tue, 24 Mar 2020 22:25:28:  31000000 \\nINFO  @ Tue, 24 Mar 2020 22:25:31:  32000000 \\nINFO  @ Tue, 24 Mar 2020 22:25:33:  33000000 \\nINFO  @ Tue, 24 Mar 2020 22:25:35:  34000000 \\nINFO  @ Tue, 24 Mar 2020 22:25:37:  35000000 \\nINFO  @ Tue, 24 Mar 2020 22:25:39:  36000000 \\nINFO  @ Tue, 24 Mar 2020 22:25:41:  37000000 \\nINFO  @ Tue, 24 Mar 2020 22:25:43:  38000000 \\nINFO  @ Tue, 24 Mar 2020 22:25:45:  39000000 \\nINFO  @ Tue, 24 Mar 2020 22:25:47:  40000000 \\nINFO  @ Tue, 24 Mar 2020 22:25:50:  41000000 \\nINFO  @ Tue, 24 Mar 2020 22:25:52:  42000000 \\nINFO  @ Tue, 24 Mar 2020 22:25:54:  43000000 \\nINFO  @ Tue, 24 Mar 2020 22:25:56:  44000000 \\nINFO  @ Tue, 24 Mar 2020 22:25:58:  45000000 \\nINFO  @ Tue, 24 Mar 2020 22:26:01:  46000000 \\nINFO  @ Tue, 24 Mar 2020 22:26:03:  47000000 \\nINFO  @ Tue, 24 Mar 2020 22:26:05:  48000000 \\nINFO  @ Tue, 24 Mar 2020 22:26:06: 48442978 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 22:26:06: #1.2 read input tags... \\nINFO  @ Tue, 24 Mar 2020 22:26:09:  1000000 \\nINFO  @ Tue, 24 Mar 2020 22:26:11:  2000000 \\nINFO  @ Tue, 24 Mar 2020 22:26:13:  3000000 \\nINFO  @ Tue, 24 Mar 2020 22:26:15:  4000000 \\nINFO  @ Tue, 24 Mar 2020 22:26:18:  5000000 \\nINFO  @ Tue, 24 Mar 2020 22:26:20:  6000000 \\nINFO  @ Tue, 24 Mar 2020 22:26:22:  7000000 \\nINFO  @ Tue, 24 Mar 2020 22:26:25:  8000000 \\nINFO  @ Tue, 24 Mar 2020 22:26:27:  9000000 \\nINFO  @ Tue, 24 Mar 2020 22:26:29:  10000000 \\nINFO  @ Tue, 24 Mar 2020 22:26:31:  11000000 \\nINFO  @ Tue, 24 Mar 2020 22:26:34:  12000000 \\nINFO  @ Tue, 24 Mar 2020 22:26:36:  13000000 \\nINFO  @ Tue, 24 Mar 2020 22:26:38:  14000000 \\nINFO  @ Tue, 24 Mar 2020 22:26:41:  15000000 \\nINFO  @ Tue, 24 Mar 2020 22:26:43:  16000000 \\nINFO  @ Tue, 24 Mar 2020 22:26:45:  17000000 \\nINFO  @ Tue, 24 Mar 2020 22:26:48:  18000000 \\nINFO  @ Tue, 24 Mar 2020 22:26:50:  19000000 \\nINFO  @ Tue, 24 Mar 2020 22:26:52:  20000000 \\nINFO  @ Tue, 24 Mar 2020 22:26:54:  21000000 \\nINFO  @ Tue, 24 Mar 2020 22:26:57:  22000000 \\nINFO  @ Tue, 24 Mar 2020 22:26:59:  23000000 \\nINFO  @ Tue, 24 Mar 2020 22:27:01:  24000000 \\nINFO  @ Tue, 24 Mar 2020 22:27:04:  25000000 \\nINFO  @ Tue, 24 Mar 2020 22:27:06:  26000000 \\nINFO  @ Tue, 24 Mar 2020 22:27:08:  27000000 \\nINFO  @ Tue, 24 Mar 2020 22:27:11:  28000000 \\nINFO  @ Tue, 24 Mar 2020 22:27:13:  29000000 \\nINFO  @ Tue, 24 Mar 2020 22:27:15:  30000000 \\nINFO  @ Tue, 24 Mar 2020 22:27:18:  31000000 \\nINFO  @ Tue, 24 Mar 2020 22:27:20:  32000000 \\nINFO  @ Tue, 24 Mar 2020 22:27:22:  33000000 \\nINFO  @ Tue, 24 Mar 2020 22:27:25:  34000000 \\nINFO  @ Tue, 24 Mar 2020 22:27:26: 34520959 reads have been read. \\nINFO  @ Tue, 24 Mar 2020 22:27:26: #1 tag size is determined as 49 bps \\nINFO  @ Tue, 24 Mar 2020 22:27:26: #1 tag size = 49.0 \\nINFO  @ Tue, 24 Mar 2020 22:27:26: #1  total tags in treatment: 48442978 \\nINFO  @ Tue, 24 Mar 2020 22:27:26: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 22:27:26: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 22:27:26: #1  tags after filtering in treatment: 48427565 \\nINFO  @ Tue, 24 Mar 2020 22:27:26: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Tue, 24 Mar 2020 22:27:26: #1  total tags in control: 34520959 \\nINFO  @ Tue, 24 Mar 2020 22:27:26: #1 user defined the maximum tags... \\nINFO  @ Tue, 24 Mar 2020 22:27:26: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 24 Mar 2020 22:27:26: #1  tags after filtering in control: 34516729 \\nINFO  @ Tue, 24 Mar 2020 22:27:26: #1  Redundant rate of control: 0.00 \\nINFO  @ Tue, 24 Mar 2020 22:27:26: #1 finished! \\nINFO  @ Tue, 24 Mar 2020 22:27:26: #2 Build Peak Model... \\nINFO  @ Tue, 24 Mar 2020 22:27:26: #2 Skipped... \\nINFO  @ Tue, 24 Mar 2020 22:27:26: #2 Use 288 as fragment length \\nINFO  @ Tue, 24 Mar 2020 22:27:26: #3 Call peaks... \\nINFO  @ Tue, 24 Mar 2020 22:27:26: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 24 Mar 2020 22:30:40: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 24 Mar 2020 22:30:40: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2_treat_pileup.bdg \\nINFO  @ Tue, 24 Mar 2020 22:30:40: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2_control_lambda.bdg \\nINFO  @ Tue, 24 Mar 2020 22:30:40: #3   Pileup will be based on sequencing depth in control. \\nINFO  @ Tue, 24 Mar 2020 22:30:40: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 24 Mar 2020 22:34:06: #4 Write output xls file... ../../data/diffBinding_hist/diffData_unscaled/mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2_peaks.xls \\nINFO  @ Tue, 24 Mar 2020 22:34:06: #4 Write peak in narrowPeak format file... ../../data/diffBinding_hist/diffData_unscaled/mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2_peaks.narrowPeak \\nINFO  @ Tue, 24 Mar 2020 22:34:06: #4 Write summits bed file... ../../data/diffBinding_hist/diffData_unscaled/mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2_summits.bed \\nINFO  @ Tue, 24 Mar 2020 22:34:06: Done! \\n'\n",
      "34516729 34516729\n",
      "doing differential peak binding\n"
     ]
    }
   ],
   "source": [
    "# on unscalled data \n",
    "for i in range(int((len(bams)-1)/2)):\n",
    "    name1 = bams[1+i]\n",
    "    name2 = bams[11+i]\n",
    "    chip.fullDiffPeak(name1,name2, control1='../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam', directory = \"../../data/diffBinding_hist/diffData_unscaled/\", res_directory = \"../../data/diffBinding_hist/diffPeaks_unscaled/\",isTF=False, compute_size=True, pairedend=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 131,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/bigwigrephistunscaled/INPUT_R1.mLb.clN.bigWig',\n",
       " '../../data/bigwigrephistunscaled/mp811-MV411_IRF2BP2_DMSO_6h-H3K27ac-r1.mLb.clN.bigWig',\n",
       " '../../data/bigwigrephistunscaled/mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrephistunscaled/mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1.mLb.clN.bigWig',\n",
       " '../../data/bigwigrephistunscaled/mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrephistunscaled/mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1.mLb.clN.bigWig',\n",
       " '../../data/bigwigrephistunscaled/mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrephistunscaled/mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1.mLb.clN.bigWig',\n",
       " '../../data/bigwigrephistunscaled/mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrephistunscaled/mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1.mLb.clN.bigWig',\n",
       " '../../data/bigwigrephistunscaled/mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrephistunscaled/mp821-MV411_IRF2BP2_VHL_6h-H3K27ac-r1.mLb.clN.bigWig',\n",
       " '../../data/bigwigrephistunscaled/mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrephistunscaled/mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1.mLb.clN.bigWig',\n",
       " '../../data/bigwigrephistunscaled/mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrephistunscaled/mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1.mLb.clN.bigWig',\n",
       " '../../data/bigwigrephistunscaled/mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrephistunscaled/mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1.mLb.clN.bigWig',\n",
       " '../../data/bigwigrephistunscaled/mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2.mLb.clN.bigWig',\n",
       " '../../data/bigwigrephistunscaled/mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1.mLb.clN.bigWig',\n",
       " '../../data/bigwigrephistunscaled/mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2.mLb.clN.bigWig']"
      ]
     },
     "execution_count": 131,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bw = ! ls ../../data/bigwigrephistunscaled/*.bigWig\n",
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 132,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/diffBinding_hist/diffPeaks_unscaled/mp811-MV411_IRF2BP2_DMSO_6h-H3K27ac-r1_vs_mp821-MV411_IRF2BP2_VHL_6h-H3K27ac-r1_c3.0_cond2.bed',\n",
       " '../../data/diffBinding_hist/diffPeaks_unscaled/mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2_vs_mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2_c3.0_cond2.bed',\n",
       " '../../data/diffBinding_hist/diffPeaks_unscaled/mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1_vs_mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1_c3.0_cond2.bed',\n",
       " '../../data/diffBinding_hist/diffPeaks_unscaled/mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2_vs_mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2_c3.0_cond2.bed',\n",
       " '../../data/diffBinding_hist/diffPeaks_unscaled/mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1_vs_mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1_c3.0_cond2.bed',\n",
       " '../../data/diffBinding_hist/diffPeaks_unscaled/mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2_vs_mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2_c3.0_cond2.bed',\n",
       " '../../data/diffBinding_hist/diffPeaks_unscaled/mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1_vs_mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1_c3.0_cond2.bed',\n",
       " '../../data/diffBinding_hist/diffPeaks_unscaled/mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2_vs_mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2_c3.0_cond2.bed',\n",
       " '../../data/diffBinding_hist/diffPeaks_unscaled/mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1_vs_mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1_c3.0_cond2.bed',\n",
       " '../../data/diffBinding_hist/diffPeaks_unscaled/mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2_vs_mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2_c3.0_cond2.bed']"
      ]
     },
     "execution_count": 132,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "cond1peak = ! ls ../../data/diffBinding_hist/diffPeaks_unscaled/*cond1.bed\n",
    "cond2peak = ! ls ../../data/diffBinding_hist/diffPeaks_unscaled/*cond2.bed\n",
    "commonpeak = ! ls ../../data/diffBinding_hist/diffPeaks_unscaled/*common.bed\n",
    "cond2peak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 133,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"H3K27ac\", \"H3K27ac_v2\",\"H3K27me3\",\"H3K27me3_v2\",\"H3K4me1\",\"H3K4me1_v2\", \"H3K4me3\", \"H3K4me3_v2\", \"H3K79me2\", \"H3K79me2_v2\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 134,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/bigwigrephistunscaled/mp811-MV411_IRF2BP2_DMSO_6h-H3K27ac-r1.mLb.clN.bigWig ../../data/bigwigrephistunscaled/mp821-MV411_IRF2BP2_VHL_6h-H3K27ac-r1.mLb.clN.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrephistunscaled/mp811-MV411_IRF2BP2_DMSO_6h-H3K27ac-r1.mLb.clN.bigWig ../../data/bigwigrephistunscaled/mp821-MV411_IRF2BP2_VHL_6h-H3K27ac-r1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/diffBinding_hist/diffPeaks_unscaled/mp811-MV411_IRF2BP2_DMSO_6h-H3K27ac-r1_vs_mp821-MV411_IRF2BP2_VHL_6h-H3K27ac-r1_c3.0_cond1.bed ../../data/diffBinding_hist/diffPeaks_unscaled/mp811-MV411_IRF2BP2_DMSO_6h-H3K27ac-r1_vs_mp821-MV411_IRF2BP2_VHL_6h-H3K27ac-r1_c3.0_common.bed ../../data/diffBinding_hist/diffPeaks_unscaled/mp811-MV411_IRF2BP2_DMSO_6h-H3K27ac-r1_vs_mp821-MV411_IRF2BP2_VHL_6h-H3K27ac-r1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks_unscaled/H3K27ac_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 10 && plotHeatmap --matrixFile ../../data/diffBinding_hist/diffPeaks_unscaled/H3K27ac_mat.gz --outFileName ../../data/diffBinding_hist/diffPeaks_unscaled/H3K27ac_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle H3K27ac', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270424v1\\t      2140\\nchrUn_GL000213v1\\t    164239\\nchrUn_KI270385v1\\t       990\\nchrUn_KI270382v1\\t      4215\\nchrUn_KI270579v1\\t     31033\\nchrUn_KI270304v1\\t      2165\\nchrUn_KI270303v1\\t      1942\\nchrUn_KI270752v1\\t     27745\\nchrUn_KI270390v1\\t      2387\\nchrUn_KI270316v1\\t      1444\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270378v1\\t      1048\\nchrUn_KI270418v1\\t      2145\\nchrUn_KI270420v1\\t      2321\\nchrUn_KI270317v1\\t     37690\\n')\n",
      "../../data/bigwigrephistunscaled/mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2.mLb.clN.bigWig ../../data/bigwigrephistunscaled/mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2.mLb.clN.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrephistunscaled/mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2.mLb.clN.bigWig ../../data/bigwigrephistunscaled/mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/diffBinding_hist/diffPeaks_unscaled/mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2_vs_mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2_c3.0_cond1.bed ../../data/diffBinding_hist/diffPeaks_unscaled/mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2_vs_mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2_c3.0_common.bed ../../data/diffBinding_hist/diffPeaks_unscaled/mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2_vs_mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks_unscaled/H3K27ac_v2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 10 && plotHeatmap --matrixFile ../../data/diffBinding_hist/diffPeaks_unscaled/H3K27ac_v2_mat.gz --outFileName ../../data/diffBinding_hist/diffPeaks_unscaled/H3K27ac_v2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle H3K27ac_v2', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270411v1\\t      2646\\nchrUn_KI270315v1\\t      2276\\nchrUn_KI270320v1\\t      4416\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270316v1\\t      1444\\nchrUn_KI270302v1\\t      2274\\n         chrEBV\\t    171823\\nchrUn_KI270422v1\\t      1445\\nchrUn_KI270390v1\\t      2387\\nchrUn_KI270418v1\\t      2145\\nchrUn_KI270385v1\\t       990\\nchrUn_KI270364v1\\t      2855\\nchrUn_KI270382v1\\t      4215\\n')\n",
      "../../data/bigwigrephistunscaled/mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1.mLb.clN.bigWig ../../data/bigwigrephistunscaled/mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1.mLb.clN.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrephistunscaled/mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1.mLb.clN.bigWig ../../data/bigwigrephistunscaled/mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/diffBinding_hist/diffPeaks_unscaled/mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1_vs_mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1_c3.0_common.bed ../../data/diffBinding_hist/diffPeaks_unscaled/mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1_vs_mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks_unscaled/H3K27me3_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 10 && plotHeatmap --matrixFile ../../data/diffBinding_hist/diffPeaks_unscaled/H3K27me3_mat.gz --outFileName ../../data/diffBinding_hist/diffPeaks_unscaled/H3K27me3_mat.pdf --refPointLabel center --regionsLabel common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle H3K27me3', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270303v1\\t      1942\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270390v1\\t      2387\\nchrUn_KI270414v1\\t      2489\\nchrUn_KI270316v1\\t      1444\\nchrUn_KI270364v1\\t      2855\\nchrUn_GL000226v1\\t     15008\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270329v1\\t      1040\\nchrUn_KI270539v1\\t       993\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270315v1\\t      2276\\nchrUn_KI270378v1\\t      1048\\nchrUn_KI270382v1\\t      4215\\n')\n",
      "../../data/bigwigrephistunscaled/mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2.mLb.clN.bigWig ../../data/bigwigrephistunscaled/mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2.mLb.clN.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrephistunscaled/mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2.mLb.clN.bigWig ../../data/bigwigrephistunscaled/mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/diffBinding_hist/diffPeaks_unscaled/mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2_vs_mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2_c3.0_cond1.bed ../../data/diffBinding_hist/diffPeaks_unscaled/mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2_vs_mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2_c3.0_common.bed ../../data/diffBinding_hist/diffPeaks_unscaled/mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2_vs_mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks_unscaled/H3K27me3_v2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 10 && plotHeatmap --matrixFile ../../data/diffBinding_hist/diffPeaks_unscaled/H3K27me3_v2_mat.gz --outFileName ../../data/diffBinding_hist/diffPeaks_unscaled/H3K27me3_v2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle H3K27me3_v2', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270411v1\\t      2646\\nchrUn_KI270395v1\\t      1143\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270305v1\\t      1472\\nchrUn_KI270302v1\\t      2274\\nchrUn_KI270392v1\\t       971\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270364v1\\t      2855\\nchrUn_KI270579v1\\t     31033\\nchrUn_KI270425v1\\t      1884\\nchrUn_KI270382v1\\t      4215\\nchrUn_KI270378v1\\t      1048\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270414v1\\t      2489\\nchrUn_KI270384v1\\t      1658\\nchrUn_KI270335v1\\t      1048\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270530v1\\t      2168\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270316v1\\t      1444\\nchrUn_KI270312v1\\t       998\\n')\n",
      "../../data/bigwigrephistunscaled/mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1.mLb.clN.bigWig ../../data/bigwigrephistunscaled/mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1.mLb.clN.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrephistunscaled/mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1.mLb.clN.bigWig ../../data/bigwigrephistunscaled/mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/diffBinding_hist/diffPeaks_unscaled/mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1_vs_mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1_c3.0_common.bed ../../data/diffBinding_hist/diffPeaks_unscaled/mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1_vs_mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks_unscaled/H3K4me1_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 10 && plotHeatmap --matrixFile ../../data/diffBinding_hist/diffPeaks_unscaled/H3K4me1_mat.gz --outFileName ../../data/diffBinding_hist/diffPeaks_unscaled/H3K4me1_mat.pdf --refPointLabel center --regionsLabel common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle H3K4me1', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270385v1\\t       990\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270429v1\\t      1361\\nchrUn_KI270305v1\\t      1472\\nchrUn_KI270390v1\\t      2387\\nchrUn_KI270411v1\\t      2646\\nchrUn_KI270378v1\\t      1048\\nchrUn_KI270752v1\\t     27745\\nchrUn_KI270364v1\\t      2855\\nchrUn_KI270312v1\\t       998\\nchrUn_KI270581v1\\t      7046\\nchrUn_KI270316v1\\t      1444\\nchrUn_KI270422v1\\t      1445\\nchrUn_KI270315v1\\t      2276\\nchrUn_KI270391v1\\t      1484\\nchrUn_KI270539v1\\t       993\\nchrUn_KI270579v1\\t     31033\\nchrUn_KI270384v1\\t      1658\\n')\n",
      "../../data/bigwigrephistunscaled/mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2.mLb.clN.bigWig ../../data/bigwigrephistunscaled/mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2.mLb.clN.bigWig\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrephistunscaled/mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2.mLb.clN.bigWig ../../data/bigwigrephistunscaled/mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/diffBinding_hist/diffPeaks_unscaled/mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2_vs_mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2_c3.0_cond1.bed ../../data/diffBinding_hist/diffPeaks_unscaled/mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2_vs_mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2_c3.0_common.bed ../../data/diffBinding_hist/diffPeaks_unscaled/mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2_vs_mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks_unscaled/H3K4me1_v2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 10 && plotHeatmap --matrixFile ../../data/diffBinding_hist/diffPeaks_unscaled/H3K4me1_v2_mat.gz --outFileName ../../data/diffBinding_hist/diffPeaks_unscaled/H3K4me1_v2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle H3K4me1_v2', returncode=0, stdout=b'', stderr=b\"\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270539v1\\t       993\\nchrUn_KI270392v1\\t       971\\nchrUn_KI270378v1\\t      1048\\nchrUn_KI270752v1\\t     27745\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270305v1\\t      1472\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270581v1\\t      7046\\nchrUn_KI270303v1\\t      1942\\nchrUn_KI270316v1\\t      1444\\nchrUn_KI270425v1\\t      1884\\nchrUn_KI270418v1\\t      2145\\nOne of the groups defined in the bed file is too small.\\nGroups that are too small can't be plotted. \\nWARNING: Group 'mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2_vs_mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2_c3.0_cond1.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n",
      "../../data/bigwigrephistunscaled/mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1.mLb.clN.bigWig ../../data/bigwigrephistunscaled/mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1.mLb.clN.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrephistunscaled/mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1.mLb.clN.bigWig ../../data/bigwigrephistunscaled/mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/diffBinding_hist/diffPeaks_unscaled/mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1_vs_mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1_c3.0_cond1.bed ../../data/diffBinding_hist/diffPeaks_unscaled/mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1_vs_mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1_c3.0_common.bed ../../data/diffBinding_hist/diffPeaks_unscaled/mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1_vs_mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks_unscaled/H3K4me3_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 10 && plotHeatmap --matrixFile ../../data/diffBinding_hist/diffPeaks_unscaled/H3K4me3_mat.gz --outFileName ../../data/diffBinding_hist/diffPeaks_unscaled/H3K4me3_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle H3K4me3', returncode=0, stdout=b'', stderr=b\"\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270530v1\\t      2168\\nchrUn_KI270304v1\\t      2165\\nchrUn_KI270310v1\\t      1201\\nchrY_KI270740v1_random\\t     37240\\nchrUn_KI270593v1\\t      3041\\nchrUn_KI270362v1\\t      3530\\nchrUn_KI270465v1\\t      1774\\nchrUn_KI270748v1\\t     93321\\nchrUn_KI270583v1\\t      1400\\nchrUn_KI270322v1\\t     21476\\nchrUn_KI270391v1\\t      1484\\nchrUn_KI270582v1\\t      6504\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270752v1\\t     27745\\nchrUn_KI270522v1\\t      5674\\nchrUn_KI270510v1\\t      2415\\nchr22_KI270739v1_random\\t     73985\\nchr9_KI270718v1_random\\t     38054\\nchrUn_KI270528v1\\t      2983\\nchrUn_GL000213v1\\t    164239\\nchrUn_KI270390v1\\t      2387\\nchrUn_KI270394v1\\t       970\\nchrUn_KI270330v1\\t      1652\\nchrUn_KI270311v1\\t     12399\\nOne of the groups defined in the bed file is too small.\\nGroups that are too small can't be plotted. \\nWARNING: Group 'mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1_vs_mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1_c3.0_cond1.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n",
      "../../data/bigwigrephistunscaled/mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2.mLb.clN.bigWig ../../data/bigwigrephistunscaled/mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2.mLb.clN.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrephistunscaled/mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2.mLb.clN.bigWig ../../data/bigwigrephistunscaled/mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/diffBinding_hist/diffPeaks_unscaled/mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2_vs_mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2_c3.0_cond1.bed ../../data/diffBinding_hist/diffPeaks_unscaled/mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2_vs_mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2_c3.0_common.bed ../../data/diffBinding_hist/diffPeaks_unscaled/mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2_vs_mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks_unscaled/H3K4me3_v2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 10 && plotHeatmap --matrixFile ../../data/diffBinding_hist/diffPeaks_unscaled/H3K4me3_v2_mat.gz --outFileName ../../data/diffBinding_hist/diffPeaks_unscaled/H3K4me3_v2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle H3K4me3_v2', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270752v1\\t     27745\\nchrUn_KI270363v1\\t      1803\\nchrUn_KI270509v1\\t      2318\\nchrUn_KI270310v1\\t      1201\\nchrUn_KI270422v1\\t      1445\\nchrUn_KI270322v1\\t     21476\\nchrUn_KI270362v1\\t      3530\\nchrUn_KI270516v1\\t      1300\\nchrUn_KI270429v1\\t      1361\\nchrUn_KI270317v1\\t     37690\\nchrUn_KI270304v1\\t      2165\\n         chrEBV\\t    171823\\nchrUn_KI270508v1\\t      1951\\nchrUn_KI270382v1\\t      4215\\nchrUn_KI270582v1\\t      6504\\nchrUn_KI270468v1\\t      4055\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270320v1\\t      4416\\nchrUn_KI270465v1\\t      1774\\n')\n",
      "../../data/bigwigrephistunscaled/mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1.mLb.clN.bigWig ../../data/bigwigrephistunscaled/mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1.mLb.clN.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrephistunscaled/mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1.mLb.clN.bigWig ../../data/bigwigrephistunscaled/mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/diffBinding_hist/diffPeaks_unscaled/mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1_vs_mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1_c3.0_common.bed ../../data/diffBinding_hist/diffPeaks_unscaled/mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1_vs_mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks_unscaled/H3K79me2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 10 && plotHeatmap --matrixFile ../../data/diffBinding_hist/diffPeaks_unscaled/H3K79me2_mat.gz --outFileName ../../data/diffBinding_hist/diffPeaks_unscaled/H3K79me2_mat.pdf --refPointLabel center --regionsLabel common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle H3K79me2', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270579v1\\t     31033\\nchrUn_KI270305v1\\t      1472\\nchrUn_KI270752v1\\t     27745\\nchrUn_KI270384v1\\t      1658\\nchrUn_KI270304v1\\t      2165\\nchrUn_KI270418v1\\t      2145\\nchrUn_KI270414v1\\t      2489\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270539v1\\t       993\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270391v1\\t      1484\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270363v1\\t      1803\\nchrUn_KI270419v1\\t      1029\\nchrUn_KI270316v1\\t      1444\\nchrUn_KI270411v1\\t      2646\\n')\n",
      "../../data/bigwigrephistunscaled/mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2.mLb.clN.bigWig ../../data/bigwigrephistunscaled/mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2.mLb.clN.bigWig\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrephistunscaled/mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2.mLb.clN.bigWig ../../data/bigwigrephistunscaled/mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/diffBinding_hist/diffPeaks_unscaled/mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2_vs_mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2_c3.0_cond1.bed ../../data/diffBinding_hist/diffPeaks_unscaled/mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2_vs_mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2_c3.0_common.bed ../../data/diffBinding_hist/diffPeaks_unscaled/mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2_vs_mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/diffBinding_hist/diffPeaks_unscaled/H3K79me2_v2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 10 && plotHeatmap --matrixFile ../../data/diffBinding_hist/diffPeaks_unscaled/H3K79me2_v2_mat.gz --outFileName ../../data/diffBinding_hist/diffPeaks_unscaled/H3K79me2_v2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle H3K79me2_v2', returncode=0, stdout=b'', stderr=b\"\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270385v1\\t       990\\nchrUn_KI270539v1\\t       993\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270752v1\\t     27745\\nchrUn_KI270395v1\\t      1143\\nchrUn_KI270302v1\\t      2274\\nchr9_KI270717v1_random\\t     40062\\nchrUn_KI270411v1\\t      2646\\nchrUn_KI270304v1\\t      2165\\nchrUn_KI270414v1\\t      2489\\n         chrEBV\\t    171823\\nchrUn_KI270530v1\\t      2168\\nchrUn_KI270316v1\\t      1444\\nchrUn_KI270529v1\\t      1899\\nchrUn_KI270528v1\\t      2983\\nWARNING: Group 'mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2_vs_mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2_c3.0_cond2.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n"
     ]
    }
   ],
   "source": [
    "for i in range(int(len(bw)/2)):\n",
    "    if i < 0:\n",
    "        continue\n",
    "    name1 = bw[1+i]\n",
    "    name2 = bw[11+i]\n",
    "    peak = [cond1peak[i], commonpeak[i], cond2peak[i]]\n",
    "    #for val in peak:\n",
    "        #chip.dropWeirdChromosomes(val)\n",
    "    name = names[i]\n",
    "    print(name1,name2)\n",
    "    chip.computePeaksAt(peak, [name1, name2], bigwignames=['DMSO', 'VHL'], peaknames=['DMSO_peaks', 'common', 'VHL_peaks'], window=3000, folder=\"\", title=name, numthreads=10, refpoint='center', name='../../data/diffBinding_hist/diffPeaks_unscaled/'+name+'_mat.pdf', withDeeptools=True, torecompute=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! gsutil -m cp -r \"diffPeaks\" gs://amlproject/Chip_AML/IRF2BP2/ &&\\\n",
    "    gsutil -m cp -r \"diffData\" gs://amlproject/Chip_AML/IRF2BP2/ &&\\\n",
    "        gsutil -m cp \"bigwigs/mp*\" gs://amlproject/Chip_AML/IRF2BP2/bigwigs"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## v4"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 142,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying gs://transfer-amlproject/20200302_10_MP7868_S60_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/20200302_10_MP7868_S60_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/20200302_11_MP7868_S61_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/20200302_11_MP7868_S61_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/20200302_12_MP7868_S62_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/20200302_12_MP7868_S62_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/20200302_1_MP7868_S51_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/20200302_1_MP7868_S51_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/20200302_2_MP7868_S52_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/20200302_2_MP7868_S52_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/20200302_3_MP7868_S53_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/20200302_3_MP7868_S53_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/20200302_4_MP7868_S54_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/20200302_4_MP7868_S54_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/20200302_5_MP7868_S55_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/20200302_5_MP7868_S55_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/20200302_6_MP7868_S56_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/20200302_6_MP7868_S56_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/20200302_7_MP7868_S57_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/20200302_7_MP7868_S57_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/20200302_8_MP7868_S58_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/20200302_8_MP7868_S58_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/20200302_9_MP7868_S59_R1_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Copying gs://transfer-amlproject/20200302_9_MP7868_S59_R2_001.fastq.gz [Content-Type=application/octet-stream]...\n",
      "Removing gs://transfer-amlproject/20200302_11_MP7868_S61_R1_001.fastq.gz...     \n",
      "Removing gs://transfer-amlproject/20200302_1_MP7868_S51_R1_001.fastq.gz...\n",
      "Removing gs://transfer-amlproject/20200302_12_MP7868_S62_R1_001.fastq.gz...\n",
      "Removing gs://transfer-amlproject/20200302_1_MP7868_S51_R2_001.fastq.gz...\n",
      "Removing gs://transfer-amlproject/20200302_10_MP7868_S60_R1_001.fastq.gz...\n",
      "Removing gs://transfer-amlproject/20200302_11_MP7868_S61_R2_001.fastq.gz...\n",
      "Removing gs://transfer-amlproject/20200302_2_MP7868_S52_R1_001.fastq.gz...\n",
      "Removing gs://transfer-amlproject/20200302_2_MP7868_S52_R2_001.fastq.gz...\n",
      "Removing gs://transfer-amlproject/20200302_10_MP7868_S60_R2_001.fastq.gz...\n",
      "Removing gs://transfer-amlproject/20200302_5_MP7868_S55_R2_001.fastq.gz...      \n",
      "Removing gs://transfer-amlproject/20200302_6_MP7868_S56_R1_001.fastq.gz...      \n",
      "Removing gs://transfer-amlproject/20200302_6_MP7868_S56_R2_001.fastq.gz...      \n",
      "Removing gs://transfer-amlproject/20200302_7_MP7868_S57_R2_001.fastq.gz...      \n",
      "Removing gs://transfer-amlproject/20200302_7_MP7868_S57_R1_001.fastq.gz...      \n",
      "Removing gs://transfer-amlproject/20200302_8_MP7868_S58_R1_001.fastq.gz...      \n",
      "Removing gs://transfer-amlproject/20200302_9_MP7868_S59_R2_001.fastq.gz...      \n",
      "Removing gs://transfer-amlproject/20200302_12_MP7868_S62_R2_001.fastq.gz...     \n",
      "Removing gs://transfer-amlproject/20200302_8_MP7868_S58_R2_001.fastq.gz...      \n",
      "Removing gs://transfer-amlproject/20200302_3_MP7868_S53_R1_001.fastq.gz...      \n",
      "Removing gs://transfer-amlproject/20200302_3_MP7868_S53_R2_001.fastq.gz...\n",
      "Removing gs://transfer-amlproject/20200302_5_MP7868_S55_R1_001.fastq.gz...      \n",
      "Removing gs://transfer-amlproject/20200302_4_MP7868_S54_R2_001.fastq.gz...\n",
      "Removing gs://transfer-amlproject/20200302_9_MP7868_S59_R1_001.fastq.gz...      \n",
      "Removing gs://transfer-amlproject/20200302_4_MP7868_S54_R1_001.fastq.gz...      \n",
      "- [24/24 files][ 17.2 GiB/ 17.2 GiB] 100% Done                                  \n",
      "Operation completed over 24 objects/17.2 GiB.                                    \n"
     ]
    }
   ],
   "source": [
    "! gsutil -m mv gs://transfer-amlproject/*MP7868*  gs://transfer-amlproject/IRF2BP2_v4/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 148,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying gs://transfer-amlproject/IRF2BP2_v4/20200302_10_MP7868_S60_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/IRF2BP2_v4/20200302_10_MP7868_S60_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/IRF2BP2_v4/20200302_11_MP7868_S61_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/IRF2BP2_v4/20200302_11_MP7868_S61_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/IRF2BP2_v4/20200302_12_MP7868_S62_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/IRF2BP2_v4/20200302_12_MP7868_S62_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/IRF2BP2_v4/20200302_1_MP7868_S51_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/IRF2BP2_v4/20200302_1_MP7868_S51_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/IRF2BP2_v4/20200302_2_MP7868_S52_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/IRF2BP2_v4/20200302_2_MP7868_S52_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/IRF2BP2_v4/20200302_3_MP7868_S53_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/IRF2BP2_v4/20200302_3_MP7868_S53_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/IRF2BP2_v4/20200302_4_MP7868_S54_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/IRF2BP2_v4/20200302_4_MP7868_S54_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/IRF2BP2_v4/20200302_5_MP7868_S55_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/IRF2BP2_v4/20200302_5_MP7868_S55_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/IRF2BP2_v4/20200302_6_MP7868_S56_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/IRF2BP2_v4/20200302_6_MP7868_S56_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/IRF2BP2_v4/20200302_7_MP7868_S57_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/IRF2BP2_v4/20200302_7_MP7868_S57_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/IRF2BP2_v4/20200302_8_MP7868_S58_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/IRF2BP2_v4/20200302_8_MP7868_S58_R2_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/IRF2BP2_v4/20200302_9_MP7868_S59_R1_001.fastq.gz...\n",
      "Copying gs://transfer-amlproject/IRF2BP2_v4/20200302_9_MP7868_S59_R2_001.fastq.gz...\n",
      "| [24/24 files][ 17.2 GiB/ 17.2 GiB] 100% Done 279.5 MiB/s ETA 00:00:00         \n",
      "Operation completed over 24 objects/17.2 GiB.                                    \n"
     ]
    }
   ],
   "source": [
    "! mkdir ../../data/bams4 && gsutil -m cp gs://transfer-amlproject/IRF2BP2_v4/* ../../data/bams4/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 149,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[0m\u001b[01;31m20200302_10_MP7868_S60_R1_001.fastq.gz\u001b[0m  \u001b[01;31m20200302_4_MP7868_S54_R1_001.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31m20200302_10_MP7868_S60_R2_001.fastq.gz\u001b[0m  \u001b[01;31m20200302_4_MP7868_S54_R2_001.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31m20200302_11_MP7868_S61_R1_001.fastq.gz\u001b[0m  \u001b[01;31m20200302_5_MP7868_S55_R1_001.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31m20200302_11_MP7868_S61_R2_001.fastq.gz\u001b[0m  \u001b[01;31m20200302_5_MP7868_S55_R2_001.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31m20200302_12_MP7868_S62_R1_001.fastq.gz\u001b[0m  \u001b[01;31m20200302_6_MP7868_S56_R1_001.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31m20200302_12_MP7868_S62_R2_001.fastq.gz\u001b[0m  \u001b[01;31m20200302_6_MP7868_S56_R2_001.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31m20200302_1_MP7868_S51_R1_001.fastq.gz\u001b[0m   \u001b[01;31m20200302_7_MP7868_S57_R1_001.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31m20200302_1_MP7868_S51_R2_001.fastq.gz\u001b[0m   \u001b[01;31m20200302_7_MP7868_S57_R2_001.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31m20200302_2_MP7868_S52_R1_001.fastq.gz\u001b[0m   \u001b[01;31m20200302_8_MP7868_S58_R1_001.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31m20200302_2_MP7868_S52_R2_001.fastq.gz\u001b[0m   \u001b[01;31m20200302_8_MP7868_S58_R2_001.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31m20200302_3_MP7868_S53_R1_001.fastq.gz\u001b[0m   \u001b[01;31m20200302_9_MP7868_S59_R1_001.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31m20200302_3_MP7868_S53_R2_001.fastq.gz\u001b[0m   \u001b[01;31m20200302_9_MP7868_S59_R2_001.fastq.gz\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "ls ../../data/bams4"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 150,
   "metadata": {},
   "outputs": [],
   "source": [
    "rename = {\n",
    "\"20200302_1_MP7868_S51_R1_001.fastq.gz\":\"mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1.fastq.gz\", \n",
    "\"20200302_1_MP7868_S51_R2_001.fastq.gz\":\"mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R2.fastq.gz\", \n",
    "\"20200302_2_MP7868_S52_R1_001.fastq.gz\":\"mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1.fastq.gz\", \n",
    "\"20200302_2_MP7868_S52_R2_001.fastq.gz\":\"mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R2.fastq.gz\", \n",
    "\"20200302_3_MP7868_S53_R1_001.fastq.gz\":\"mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1.fastq.gz\", \n",
    "\"20200302_3_MP7868_S53_R2_001.fastq.gz\":\"mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R2.fastq.gz\",\n",
    "\"20200302_4_MP7868_S54_R1_001.fastq.gz\":\"mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1.fastq.gz\",\n",
    "\"20200302_4_MP7868_S54_R2_001.fastq.gz\":\"mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R2.fastq.gz\",\n",
    "\"20200302_5_MP7868_S55_R1_001.fastq.gz\":\"mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1.fastq.gz\",\n",
    "\"20200302_5_MP7868_S55_R2_001.fastq.gz\":\"mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R2.fastq.gz\",\n",
    "\"20200302_6_MP7868_S56_R1_001.fastq.gz\":\"mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1.fastq.gz\",\n",
    "\"20200302_6_MP7868_S56_R2_001.fastq.gz\":\"mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R2.fastq.gz\",\n",
    "\"20200302_7_MP7868_S57_R1_001.fastq.gz\":\"mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1.fastq.gz\",\n",
    "\"20200302_7_MP7868_S57_R2_001.fastq.gz\":\"mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R2.fastq.gz\",\n",
    "\"20200302_8_MP7868_S58_R1_001.fastq.gz\":\"mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1.fastq.gz\",\n",
    "\"20200302_8_MP7868_S58_R2_001.fastq.gz\":\"mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R2.fastq.gz\",\n",
    "\"20200302_9_MP7868_S59_R1_001.fastq.gz\":\"mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1.fastq.gz\",\n",
    "\"20200302_9_MP7868_S59_R2_001.fastq.gz\":\"mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R2.fastq.gz\", \n",
    "\"20200302_10_MP7868_S60_R1_001.fastq.gz\":\"mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1.fastq.gz\",\n",
    "\"20200302_10_MP7868_S60_R2_001.fastq.gz\":\"mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R2.fastq.gz\",\n",
    "\"20200302_11_MP7868_S61_R1_001.fastq.gz\":\"mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1.fastq.gz\",\n",
    "\"20200302_11_MP7868_S61_R2_001.fastq.gz\":\"mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R2.fastq.gz\",\n",
    "\"20200302_12_MP7868_S62_R1_001.fastq.gz\":\"mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1.fastq.gz\",\n",
    "\"20200302_12_MP7868_S62_R2_001.fastq.gz\":\"mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R2.fastq.gz\"\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 152,
   "metadata": {},
   "outputs": [],
   "source": [
    "for k,v in rename.items():\n",
    "    ! mv ../../data/bams4/$k ../../data/bams4/$v"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 153,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[0m\u001b[01;31mmp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp852-MV411_IRF2BP_VHL_6h-MED1-r4_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp853-MV411_IRF2BP_VHL_6h-MED1-r5_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R2.fastq.gz\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "ls ../../data/bams4"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 154,
   "metadata": {},
   "outputs": [],
   "source": [
    "from gsheets import Sheets\n",
    "sheets = Sheets.from_files('~/.client_secret.json', '~/.storage.json')\n",
    "url=\"https://docs.google.com/spreadsheets/d/1yFLjYB1McU530JnLgL0QIMAKIkVl3kl0_LCHje2gk8U\"\n",
    "gsheet = sheets.get(url).sheets[2].to_frame()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 155,
   "metadata": {},
   "outputs": [],
   "source": [
    "a = ! ls ../../data/bams4"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 156,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>id</th>\n",
       "      <th>cell line</th>\n",
       "      <th>replicate</th>\n",
       "      <th>protein</th>\n",
       "      <th>quality</th>\n",
       "      <th>matching input name</th>\n",
       "      <th>processed</th>\n",
       "      <th>name</th>\n",
       "      <th>previous name</th>\n",
       "      <th>bigwig</th>\n",
       "      <th>...</th>\n",
       "      <th>individual QC</th>\n",
       "      <th>unique mapped reads(droso)</th>\n",
       "      <th>scaling factor</th>\n",
       "      <th>Total QC</th>\n",
       "      <th>folderNarrow</th>\n",
       "      <th>folderCompensated</th>\n",
       "      <th>folderQC</th>\n",
       "      <th>folderBroad</th>\n",
       "      <th>folder Bigwig</th>\n",
       "      <th>folder diffPeaks</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>mp100</td>\n",
       "      <td>U937</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Y</td>\n",
       "      <td>mp100-U937-INPUT-r1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://console.cloud.google.com/storage/brows...</td>\n",
       "      <td>...</td>\n",
       "      <td>https://console.cloud.google.com/storage/brows...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://storage.cloud.google.com/amlproject/Ch...</td>\n",
       "      <td>https://console.cloud.google.com/storage/brows...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://console.cloud.google.com/storage/brows...</td>\n",
       "      <td>https://console.cloud.google.com/storage/brows...</td>\n",
       "      <td>https://console.cloud.google.com/storage/brows...</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>mp101</td>\n",
       "      <td>NOMO1</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Y</td>\n",
       "      <td>mp101-NOMO1-INPUT-r1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://console.cloud.google.com/storage/brows...</td>\n",
       "      <td>...</td>\n",
       "      <td>https://console.cloud.google.com/storage/brows...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>mp102</td>\n",
       "      <td>UT7</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Y</td>\n",
       "      <td>mp102-UT7-INPUT-r1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://console.cloud.google.com/storage/brows...</td>\n",
       "      <td>...</td>\n",
       "      <td>https://console.cloud.google.com/storage/brows...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>mp106</td>\n",
       "      <td>MV411</td>\n",
       "      <td>1</td>\n",
       "      <td>MYB</td>\n",
       "      <td>x</td>\n",
       "      <td>INPUT_MV411</td>\n",
       "      <td>Y</td>\n",
       "      <td>mp106-MV411-MYB-r1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://console.cloud.google.com/storage/brows...</td>\n",
       "      <td>...</td>\n",
       "      <td>https://console.cloud.google.com/storage/brows...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>mp109</td>\n",
       "      <td>M6</td>\n",
       "      <td>1</td>\n",
       "      <td>CEBPA</td>\n",
       "      <td>x</td>\n",
       "      <td>INPUT_M6</td>\n",
       "      <td>Y</td>\n",
       "      <td>mp109-M6-CEBPA-r1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://console.cloud.google.com/storage/brows...</td>\n",
       "      <td>...</td>\n",
       "      <td>https://console.cloud.google.com/storage/brows...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>281</th>\n",
       "      <td>mp853</td>\n",
       "      <td>MV411_IRF2BP_VHL_6h</td>\n",
       "      <td>5</td>\n",
       "      <td>MED1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>mp853-MV411_IRF2BP_VHL_6h-MED1-r5</td>\n",
       "      <td>20200302_8_MP7868_S58_R1_001</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>282</th>\n",
       "      <td>mp854</td>\n",
       "      <td>MV411_IRF2BP_VHL_6h</td>\n",
       "      <td>3</td>\n",
       "      <td>FLAG_IRF2BP2</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3</td>\n",
       "      <td>20200302_9_MP7868_S59_R1_001</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>283</th>\n",
       "      <td>mp855</td>\n",
       "      <td>MV411_IRF2BP_VHL_6h</td>\n",
       "      <td>3</td>\n",
       "      <td>POLII_total</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3</td>\n",
       "      <td>20200302_10_MP7868_S60_R1_001</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>284</th>\n",
       "      <td>mp856</td>\n",
       "      <td>MV411_IRF2BP_VHL_6h</td>\n",
       "      <td>3</td>\n",
       "      <td>POLII_S2</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3</td>\n",
       "      <td>20200302_11_MP7868_S61_R1_001</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>285</th>\n",
       "      <td>mp857</td>\n",
       "      <td>MV411_IRF2BP_VHL_6h</td>\n",
       "      <td>3</td>\n",
       "      <td>POLII_S5</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3</td>\n",
       "      <td>20200302_12_MP7868_S62_R1_001</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>286 rows  24 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "        id            cell line  replicate       protein quality  \\\n",
       "0    mp100                 U937          1         INPUT     NaN   \n",
       "1    mp101                NOMO1          1         INPUT     NaN   \n",
       "2    mp102                  UT7          1         INPUT     NaN   \n",
       "3    mp106                MV411          1           MYB       x   \n",
       "4    mp109                   M6          1         CEBPA       x   \n",
       "..     ...                  ...        ...           ...     ...   \n",
       "281  mp853  MV411_IRF2BP_VHL_6h          5          MED1     NaN   \n",
       "282  mp854  MV411_IRF2BP_VHL_6h          3  FLAG_IRF2BP2     NaN   \n",
       "283  mp855  MV411_IRF2BP_VHL_6h          3   POLII_total     NaN   \n",
       "284  mp856  MV411_IRF2BP_VHL_6h          3      POLII_S2     NaN   \n",
       "285  mp857  MV411_IRF2BP_VHL_6h          3      POLII_S5     NaN   \n",
       "\n",
       "    matching input name processed                                       name  \\\n",
       "0                   NaN         Y                        mp100-U937-INPUT-r1   \n",
       "1                   NaN         Y                       mp101-NOMO1-INPUT-r1   \n",
       "2                   NaN         Y                         mp102-UT7-INPUT-r1   \n",
       "3           INPUT_MV411         Y                         mp106-MV411-MYB-r1   \n",
       "4              INPUT_M6         Y                          mp109-M6-CEBPA-r1   \n",
       "..                  ...       ...                                        ...   \n",
       "281                 NaN       NaN          mp853-MV411_IRF2BP_VHL_6h-MED1-r5   \n",
       "282                 NaN       NaN  mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3   \n",
       "283                 NaN       NaN   mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3   \n",
       "284                 NaN       NaN      mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3   \n",
       "285                 NaN       NaN      mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3   \n",
       "\n",
       "                     previous name  \\\n",
       "0                              NaN   \n",
       "1                              NaN   \n",
       "2                              NaN   \n",
       "3                              NaN   \n",
       "4                              NaN   \n",
       "..                             ...   \n",
       "281   20200302_8_MP7868_S58_R1_001   \n",
       "282   20200302_9_MP7868_S59_R1_001   \n",
       "283  20200302_10_MP7868_S60_R1_001   \n",
       "284  20200302_11_MP7868_S61_R1_001   \n",
       "285  20200302_12_MP7868_S62_R1_001   \n",
       "\n",
       "                                                bigwig  ...  \\\n",
       "0    https://console.cloud.google.com/storage/brows...  ...   \n",
       "1    https://console.cloud.google.com/storage/brows...  ...   \n",
       "2    https://console.cloud.google.com/storage/brows...  ...   \n",
       "3    https://console.cloud.google.com/storage/brows...  ...   \n",
       "4    https://console.cloud.google.com/storage/brows...  ...   \n",
       "..                                                 ...  ...   \n",
       "281                                                NaN  ...   \n",
       "282                                                NaN  ...   \n",
       "283                                                NaN  ...   \n",
       "284                                                NaN  ...   \n",
       "285                                                NaN  ...   \n",
       "\n",
       "                                         individual QC  \\\n",
       "0    https://console.cloud.google.com/storage/brows...   \n",
       "1    https://console.cloud.google.com/storage/brows...   \n",
       "2    https://console.cloud.google.com/storage/brows...   \n",
       "3    https://console.cloud.google.com/storage/brows...   \n",
       "4    https://console.cloud.google.com/storage/brows...   \n",
       "..                                                 ...   \n",
       "281                                                NaN   \n",
       "282                                                NaN   \n",
       "283                                                NaN   \n",
       "284                                                NaN   \n",
       "285                                                NaN   \n",
       "\n",
       "    unique mapped reads(droso) scaling factor  \\\n",
       "0                          NaN            NaN   \n",
       "1                          NaN            NaN   \n",
       "2                          NaN            NaN   \n",
       "3                          NaN            NaN   \n",
       "4                          NaN            NaN   \n",
       "..                         ...            ...   \n",
       "281                        NaN            NaN   \n",
       "282                        NaN            NaN   \n",
       "283                        NaN            NaN   \n",
       "284                        NaN            NaN   \n",
       "285                        NaN            NaN   \n",
       "\n",
       "                                              Total QC  \\\n",
       "0    https://storage.cloud.google.com/amlproject/Ch...   \n",
       "1                                                  NaN   \n",
       "2                                                  NaN   \n",
       "3                                                  NaN   \n",
       "4                                                  NaN   \n",
       "..                                                 ...   \n",
       "281                                                NaN   \n",
       "282                                                NaN   \n",
       "283                                                NaN   \n",
       "284                                                NaN   \n",
       "285                                                NaN   \n",
       "\n",
       "                                          folderNarrow  folderCompensated  \\\n",
       "0    https://console.cloud.google.com/storage/brows...                NaN   \n",
       "1                                                  NaN                NaN   \n",
       "2                                                  NaN                NaN   \n",
       "3                                                  NaN                NaN   \n",
       "4                                                  NaN                NaN   \n",
       "..                                                 ...                ...   \n",
       "281                                                NaN                NaN   \n",
       "282                                                NaN                NaN   \n",
       "283                                                NaN                NaN   \n",
       "284                                                NaN                NaN   \n",
       "285                                                NaN                NaN   \n",
       "\n",
       "                                              folderQC  \\\n",
       "0    https://console.cloud.google.com/storage/brows...   \n",
       "1                                                  NaN   \n",
       "2                                                  NaN   \n",
       "3                                                  NaN   \n",
       "4                                                  NaN   \n",
       "..                                                 ...   \n",
       "281                                                NaN   \n",
       "282                                                NaN   \n",
       "283                                                NaN   \n",
       "284                                                NaN   \n",
       "285                                                NaN   \n",
       "\n",
       "                                           folderBroad  \\\n",
       "0    https://console.cloud.google.com/storage/brows...   \n",
       "1                                                  NaN   \n",
       "2                                                  NaN   \n",
       "3                                                  NaN   \n",
       "4                                                  NaN   \n",
       "..                                                 ...   \n",
       "281                                                NaN   \n",
       "282                                                NaN   \n",
       "283                                                NaN   \n",
       "284                                                NaN   \n",
       "285                                                NaN   \n",
       "\n",
       "                                         folder Bigwig folder diffPeaks  \n",
       "0    https://console.cloud.google.com/storage/brows...              NaN  \n",
       "1                                                  NaN              NaN  \n",
       "2                                                  NaN              NaN  \n",
       "3                                                  NaN              NaN  \n",
       "4                                                  NaN              NaN  \n",
       "..                                                 ...              ...  \n",
       "281                                                NaN              NaN  \n",
       "282                                                NaN              NaN  \n",
       "283                                                NaN              NaN  \n",
       "284                                                NaN              NaN  \n",
       "285                                                NaN              NaN  \n",
       "\n",
       "[286 rows x 24 columns]"
      ]
     },
     "execution_count": 156,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gsheet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 166,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[0m\u001b[01;31mmp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp842-MV411_IRF2BP_VHL_6h-MED1-r3_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp845-MV411_IRF2BP2_-INPUT-r1_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp845-MV411_IRF2BP2_-INPUT-r1_R2.fastq.gz\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "ls ../../data/bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 167,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = {\n",
    "\"fastq_1\": [],\n",
    "\"fastq_2\": [],\n",
    "\"antibody\": [],\n",
    "\"group\": [],\n",
    "\"replicate\": [],\n",
    "\"control\": []\n",
    "}\n",
    "for val in h.grouped(a,2):\n",
    "    row = gsheet[gsheet.id==val[0].split('/')[-1].split('-')[0]]\n",
    "    df['group'].append(row['name'].values[0])\n",
    "    df['replicate'].append(1)\n",
    "    df['fastq_1'].append('bams4/'+val[0])\n",
    "    df['fastq_2'].append('bams4/'+val[1])\n",
    "    df['antibody'].append(row['protein'].values[0])\n",
    "    df['control'].append(\"INPUT\")\n",
    "df['group'].append('INPUT')\n",
    "df['replicate'].append(1)\n",
    "df['fastq_1'].append('bams/mp845-MV411_IRF2BP2_-INPUT-r1_R1.fastq.gz')\n",
    "df['fastq_2'].append('bams/mp845-MV411_IRF2BP2_-INPUT-r1_R2.fastq.gz')\n",
    "df['antibody'].append(\"\")\n",
    "df['control'].append(\"\")\n",
    "df = pd.DataFrame(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 168,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>fastq_1</th>\n",
       "      <th>fastq_2</th>\n",
       "      <th>antibody</th>\n",
       "      <th>group</th>\n",
       "      <th>replicate</th>\n",
       "      <th>control</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>bams4/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1.fa...</td>\n",
       "      <td>bams4/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R2.fa...</td>\n",
       "      <td>MED1</td>\n",
       "      <td>mp846-MV411_IRF2BP_DMSO_6h-MED1-r4</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>bams4/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1.fa...</td>\n",
       "      <td>bams4/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R2.fa...</td>\n",
       "      <td>MED1</td>\n",
       "      <td>mp847-MV411_IRF2BP_DMSO_6h-MED1-r5</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>bams4/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-...</td>\n",
       "      <td>bams4/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-...</td>\n",
       "      <td>FLAG_IRF2BP2</td>\n",
       "      <td>mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>bams4/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r...</td>\n",
       "      <td>bams4/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r...</td>\n",
       "      <td>POLII_total</td>\n",
       "      <td>mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>bams4/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R...</td>\n",
       "      <td>bams4/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R...</td>\n",
       "      <td>POLII_S2</td>\n",
       "      <td>mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>bams4/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R...</td>\n",
       "      <td>bams4/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R...</td>\n",
       "      <td>POLII_S5</td>\n",
       "      <td>mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>bams4/mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1.fas...</td>\n",
       "      <td>bams4/mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R2.fas...</td>\n",
       "      <td>MED1</td>\n",
       "      <td>mp852-MV411_IRF2BP_VHL_6h-MED1-r4</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>bams4/mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1.fas...</td>\n",
       "      <td>bams4/mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R2.fas...</td>\n",
       "      <td>MED1</td>\n",
       "      <td>mp853-MV411_IRF2BP_VHL_6h-MED1-r5</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>bams4/mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r...</td>\n",
       "      <td>bams4/mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r...</td>\n",
       "      <td>FLAG_IRF2BP2</td>\n",
       "      <td>mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>bams4/mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3...</td>\n",
       "      <td>bams4/mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3...</td>\n",
       "      <td>POLII_total</td>\n",
       "      <td>mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>bams4/mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1...</td>\n",
       "      <td>bams4/mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R2...</td>\n",
       "      <td>POLII_S2</td>\n",
       "      <td>mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>bams4/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1...</td>\n",
       "      <td>bams4/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R2...</td>\n",
       "      <td>POLII_S5</td>\n",
       "      <td>mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3</td>\n",
       "      <td>1</td>\n",
       "      <td>INPUT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>bams/mp845-MV411_IRF2BP2_-INPUT-r1_R1.fastq.gz</td>\n",
       "      <td>bams/mp845-MV411_IRF2BP2_-INPUT-r1_R2.fastq.gz</td>\n",
       "      <td></td>\n",
       "      <td>INPUT</td>\n",
       "      <td>1</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                              fastq_1  \\\n",
       "0   bams4/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1.fa...   \n",
       "1   bams4/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1.fa...   \n",
       "2   bams4/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-...   \n",
       "3   bams4/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r...   \n",
       "4   bams4/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R...   \n",
       "5   bams4/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R...   \n",
       "6   bams4/mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1.fas...   \n",
       "7   bams4/mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1.fas...   \n",
       "8   bams4/mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r...   \n",
       "9   bams4/mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3...   \n",
       "10  bams4/mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1...   \n",
       "11  bams4/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1...   \n",
       "12     bams/mp845-MV411_IRF2BP2_-INPUT-r1_R1.fastq.gz   \n",
       "\n",
       "                                              fastq_2      antibody  \\\n",
       "0   bams4/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R2.fa...          MED1   \n",
       "1   bams4/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R2.fa...          MED1   \n",
       "2   bams4/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-...  FLAG_IRF2BP2   \n",
       "3   bams4/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r...   POLII_total   \n",
       "4   bams4/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R...      POLII_S2   \n",
       "5   bams4/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R...      POLII_S5   \n",
       "6   bams4/mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R2.fas...          MED1   \n",
       "7   bams4/mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R2.fas...          MED1   \n",
       "8   bams4/mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r...  FLAG_IRF2BP2   \n",
       "9   bams4/mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3...   POLII_total   \n",
       "10  bams4/mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R2...      POLII_S2   \n",
       "11  bams4/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R2...      POLII_S5   \n",
       "12     bams/mp845-MV411_IRF2BP2_-INPUT-r1_R2.fastq.gz                 \n",
       "\n",
       "                                         group  replicate control  \n",
       "0           mp846-MV411_IRF2BP_DMSO_6h-MED1-r4          1   INPUT  \n",
       "1           mp847-MV411_IRF2BP_DMSO_6h-MED1-r5          1   INPUT  \n",
       "2   mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3          1   INPUT  \n",
       "3    mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3          1   INPUT  \n",
       "4       mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3          1   INPUT  \n",
       "5       mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3          1   INPUT  \n",
       "6            mp852-MV411_IRF2BP_VHL_6h-MED1-r4          1   INPUT  \n",
       "7            mp853-MV411_IRF2BP_VHL_6h-MED1-r5          1   INPUT  \n",
       "8    mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3          1   INPUT  \n",
       "9     mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3          1   INPUT  \n",
       "10       mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3          1   INPUT  \n",
       "11       mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3          1   INPUT  \n",
       "12                                       INPUT          1          "
      ]
     },
     "execution_count": 168,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 169,
   "metadata": {},
   "outputs": [],
   "source": [
    "df[df.columns[[3,4,0,1,2,5]]].to_csv('../../data/design_rep4.csv',index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#process chips\n",
    "! sudo ../nextflow run nf-core/chipseq --paired_end --seq_center 'DFCI' --email \\\n",
    "'jkobject@gmail.com' --narrow_peak --input design_rep4.csv --genome GRCh38 --skip_preseq \\\n",
    "--max_cpus 24 -profile docker -w work"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 238,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[0m\u001b[01;31mmp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp852-MV411_IRF2BP_VHL_6h-MED1-r4_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp853-MV411_IRF2BP_VHL_6h-MED1-r5_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R2.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1.fastq.gz\u001b[0m\r\n",
      "\u001b[01;31mmp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R2.fastq.gz\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "ls ../../data/bams4"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!gsutil -m cp -r results/* gs://amlproject/Chip/IRF2BP2_degraded_rep4/ && sudo rm -r work && mv results results4"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 236,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[0m\u001b[01;32m../../TrimGalore-0.6.5/trim_galore\u001b[0m*\r\n"
     ]
    }
   ],
   "source": [
    "ls ../../TrimGalore-0.6.5/trim_galore"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "if paired_end, need to be name_*1, name_*2\n",
      "using all files from folder\n",
      "[('mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1_val_1.fq.gz', 'mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R2_val_2.fq.gz'), ('mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1_val_1.fq.gz', 'mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R2_val_2.fq.gz'), ('mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1_val_1.fq.gz', 'mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R2_val_2.fq.gz'), ('mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1_val_1.fq.gz', 'mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R2_val_2.fq.gz'), ('mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1_val_1.fq.gz', 'mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R2_val_2.fq.gz'), ('mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_val_1.fq.gz', 'mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R2_val_2.fq.gz'), ('mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1_val_1.fq.gz', 'mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R2_val_2.fq.gz'), ('mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1_val_1.fq.gz', 'mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R2_val_2.fq.gz'), ('mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1_val_1.fq.gz', 'mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R2_val_2.fq.gz'), ('mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1_val_1.fq.gz', 'mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R2_val_2.fq.gz'), ('mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1_val_1.fq.gz', 'mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R2_val_2.fq.gz'), ('mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_val_1.fq.gz', 'mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R2_val_2.fq.gz')]\n",
      "you need to have your files in the ../../data/res4/ folder\n",
      "\n",
      "\n",
      "mapping\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "filtering\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "counting\n",
      "\n",
      "\n",
      "> \u001b[0;32m/home/jeremie/JKBio/epigenetics/ChIP_helper.py\u001b[0m(902)\u001b[0;36mgetSpikeInControlScales\u001b[0;34m()\u001b[0m\n",
      "\u001b[0;32m    901 \u001b[0;31m    \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"\\n\\ncounting\\n\\n\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m--> 902 \u001b[0;31m    \u001b[0;32mfor\u001b[0m \u001b[0mfile\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mfastqs\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m    903 \u001b[0;31m        \u001b[0mipdb\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mset_trace\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\n",
      "ipdb> fastqs\n",
      "['mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1_val_1.sorted.bam', 'mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1_val_1.sorted.bam', 'mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1_val_1.sorted.bam', 'mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1_val_1.sorted.bam', 'mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1_val_1.sorted.bam', 'mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_val_1.sorted.bam', 'mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1_val_1.sorted.bam', 'mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1_val_1.sorted.bam', 'mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1_val_1.sorted.bam', 'mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1_val_1.sorted.bam', 'mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1_val_1.sorted.bam', 'mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_val_1.sorted.bam']\n",
      "ipdb> n\n",
      "> \u001b[0;32m/home/jeremie/JKBio/epigenetics/ChIP_helper.py\u001b[0m(903)\u001b[0;36mgetSpikeInControlScales\u001b[0;34m()\u001b[0m\n",
      "\u001b[0;32m    902 \u001b[0;31m    \u001b[0;32mfor\u001b[0m \u001b[0mfile\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mfastqs\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m--> 903 \u001b[0;31m        \u001b[0mipdb\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mset_trace\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m    904 \u001b[0;31m        mapped[file.split('.')[0]] = int(os.popen(pathtosam + ' view -c -F 0x004 -F 0x0008 -f 0x001 -F 0x0400 -q 1 '+results +\n",
      "\u001b[0m\n",
      "ipdb> n\n",
      "> \u001b[0;32m/home/jeremie/JKBio/epigenetics/ChIP_helper.py\u001b[0m(900)\u001b[0;36mgetSpikeInControlScales\u001b[0;34m()\u001b[0m\n",
      "\u001b[0;32m    899 \u001b[0;31m    \u001b[0mnorm\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m{\u001b[0m\u001b[0;34m}\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m--> 900 \u001b[0;31m    \u001b[0munique_mapped\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m{\u001b[0m\u001b[0;34m}\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m    901 \u001b[0;31m    \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"\\n\\ncounting\\n\\n\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\n",
      "ipdb> mapped[file.split('.')[0]]\n",
      "575250\n",
      "ipdb> mapped\n",
      "{'mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1_val_1': 575250}\n",
      "ipdb> c\n",
      "> \u001b[0;32m/home/jeremie/JKBio/epigenetics/ChIP_helper.py\u001b[0m(902)\u001b[0;36mgetSpikeInControlScales\u001b[0;34m()\u001b[0m\n",
      "\u001b[0;32m    901 \u001b[0;31m        \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"files need to be named: NAME.sorted.bam\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m--> 902 \u001b[0;31m        \u001b[0mfastqs\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mfile\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mfile\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mfastqs\u001b[0m \u001b[0;32mif\u001b[0m \u001b[0;34m'.sorted.bam'\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0mfile\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m11\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m    903 \u001b[0;31m    \u001b[0mmapped\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m{\u001b[0m\u001b[0;34m}\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\n",
      "ipdb> c\n",
      "> \u001b[0;32m/home/jeremie/JKBio/epigenetics/ChIP_helper.py\u001b[0m(901)\u001b[0;36mgetSpikeInControlScales\u001b[0;34m()\u001b[0m\n",
      "\u001b[0;32m    900 \u001b[0;31m    \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m--> 901 \u001b[0;31m        \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"files need to be named: NAME.sorted.bam\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m    902 \u001b[0;31m        \u001b[0mfastqs\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mfile\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mfile\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mfastqs\u001b[0m \u001b[0;32mif\u001b[0m \u001b[0;34m'.sorted.bam'\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0mfile\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m11\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\n",
      "ipdb> mapped\n",
      "{'mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1_val_1': 575250, 'mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1_val_1': 823219}\n",
      "ipdb> unt\n",
      "> \u001b[0;32m/home/jeremie/JKBio/epigenetics/ChIP_helper.py\u001b[0m(902)\u001b[0;36mgetSpikeInControlScales\u001b[0;34m()\u001b[0m\n",
      "\u001b[0;32m    901 \u001b[0;31m        \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"files need to be named: NAME.sorted.bam\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m--> 902 \u001b[0;31m        \u001b[0mfastqs\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mfile\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mfile\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mfastqs\u001b[0m \u001b[0;32mif\u001b[0m \u001b[0;34m'.sorted.bam'\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0mfile\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m11\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m    903 \u001b[0;31m    \u001b[0mmapped\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m{\u001b[0m\u001b[0;34m}\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\n",
      "ipdb> \n",
      "> \u001b[0;32m/home/jeremie/JKBio/epigenetics/ChIP_helper.py\u001b[0m(903)\u001b[0;36mgetSpikeInControlScales\u001b[0;34m()\u001b[0m\n",
      "\u001b[0;32m    902 \u001b[0;31m        \u001b[0mfastqs\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mfile\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mfile\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mfastqs\u001b[0m \u001b[0;32mif\u001b[0m \u001b[0;34m'.sorted.bam'\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0mfile\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m11\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m--> 903 \u001b[0;31m    \u001b[0mmapped\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m{\u001b[0m\u001b[0;34m}\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m    904 \u001b[0;31m    \u001b[0mnorm\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m{\u001b[0m\u001b[0;34m}\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\n",
      "ipdb> \n",
      "> \u001b[0;32m/home/jeremie/JKBio/epigenetics/ChIP_helper.py\u001b[0m(902)\u001b[0;36mgetSpikeInControlScales\u001b[0;34m()\u001b[0m\n",
      "\u001b[0;32m    901 \u001b[0;31m        \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"files need to be named: NAME.sorted.bam\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m--> 902 \u001b[0;31m        \u001b[0mfastqs\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mfile\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mfile\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mfastqs\u001b[0m \u001b[0;32mif\u001b[0m \u001b[0;34m'.sorted.bam'\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0mfile\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m11\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m    903 \u001b[0;31m    \u001b[0mmapped\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m{\u001b[0m\u001b[0;34m}\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\n",
      "ipdb> mapped\n",
      "{'mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1_val_1': 575250, 'mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1_val_1': 823219, 'mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1_val_1': 172050}\n",
      "ipdb> unt\n",
      "> \u001b[0;32m/home/jeremie/JKBio/epigenetics/ChIP_helper.py\u001b[0m(903)\u001b[0;36mgetSpikeInControlScales\u001b[0;34m()\u001b[0m\n",
      "\u001b[0;32m    902 \u001b[0;31m        \u001b[0mfastqs\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mfile\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mfile\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mfastqs\u001b[0m \u001b[0;32mif\u001b[0m \u001b[0;34m'.sorted.bam'\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0mfile\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m11\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m--> 903 \u001b[0;31m    \u001b[0mmapped\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m{\u001b[0m\u001b[0;34m}\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m    904 \u001b[0;31m    \u001b[0mnorm\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m{\u001b[0m\u001b[0;34m}\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\n",
      "ipdb> unt\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "> \u001b[0;32m/home/jeremie/JKBio/epigenetics/ChIP_helper.py\u001b[0m(902)\u001b[0;36mgetSpikeInControlScales\u001b[0;34m()\u001b[0m\n",
      "\u001b[0;32m    901 \u001b[0;31m        \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"files need to be named: NAME.sorted.bam\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m--> 902 \u001b[0;31m        \u001b[0mfastqs\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mfile\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mfile\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mfastqs\u001b[0m \u001b[0;32mif\u001b[0m \u001b[0;34m'.sorted.bam'\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0mfile\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m11\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m    903 \u001b[0;31m    \u001b[0mmapped\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m{\u001b[0m\u001b[0;34m}\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\n",
      "ipdb> unt\n",
      "> \u001b[0;32m/home/jeremie/JKBio/epigenetics/ChIP_helper.py\u001b[0m(903)\u001b[0;36mgetSpikeInControlScales\u001b[0;34m()\u001b[0m\n",
      "\u001b[0;32m    902 \u001b[0;31m        \u001b[0mfastqs\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mfile\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mfile\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mfastqs\u001b[0m \u001b[0;32mif\u001b[0m \u001b[0;34m'.sorted.bam'\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0mfile\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m11\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m--> 903 \u001b[0;31m    \u001b[0mmapped\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m{\u001b[0m\u001b[0;34m}\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m    904 \u001b[0;31m    \u001b[0mnorm\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m{\u001b[0m\u001b[0;34m}\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\n",
      "ipdb> \n",
      "> \u001b[0;32m/home/jeremie/JKBio/epigenetics/ChIP_helper.py\u001b[0m(902)\u001b[0;36mgetSpikeInControlScales\u001b[0;34m()\u001b[0m\n",
      "\u001b[0;32m    901 \u001b[0;31m        \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"files need to be named: NAME.sorted.bam\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m--> 902 \u001b[0;31m        \u001b[0mfastqs\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mfile\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mfile\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mfastqs\u001b[0m \u001b[0;32mif\u001b[0m \u001b[0;34m'.sorted.bam'\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0mfile\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m11\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m    903 \u001b[0;31m    \u001b[0mmapped\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m{\u001b[0m\u001b[0;34m}\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\n",
      "ipdb> unt\n",
      "> \u001b[0;32m/home/jeremie/JKBio/epigenetics/ChIP_helper.py\u001b[0m(903)\u001b[0;36mgetSpikeInControlScales\u001b[0;34m()\u001b[0m\n",
      "\u001b[0;32m    902 \u001b[0;31m        \u001b[0mfastqs\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mfile\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mfile\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mfastqs\u001b[0m \u001b[0;32mif\u001b[0m \u001b[0;34m'.sorted.bam'\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0mfile\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m11\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m--> 903 \u001b[0;31m    \u001b[0mmapped\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m{\u001b[0m\u001b[0;34m}\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m    904 \u001b[0;31m    \u001b[0mnorm\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m{\u001b[0m\u001b[0;34m}\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\n",
      "ipdb> \n",
      "> \u001b[0;32m/home/jeremie/JKBio/epigenetics/ChIP_helper.py\u001b[0m(902)\u001b[0;36mgetSpikeInControlScales\u001b[0;34m()\u001b[0m\n",
      "\u001b[0;32m    901 \u001b[0;31m        \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"files need to be named: NAME.sorted.bam\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m--> 902 \u001b[0;31m        \u001b[0mfastqs\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mfile\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mfile\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mfastqs\u001b[0m \u001b[0;32mif\u001b[0m \u001b[0;34m'.sorted.bam'\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0mfile\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m11\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m    903 \u001b[0;31m    \u001b[0mmapped\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m{\u001b[0m\u001b[0;34m}\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\n",
      "ipdb> unt\n",
      "> \u001b[0;32m/home/jeremie/JKBio/epigenetics/ChIP_helper.py\u001b[0m(903)\u001b[0;36mgetSpikeInControlScales\u001b[0;34m()\u001b[0m\n",
      "\u001b[0;32m    902 \u001b[0;31m        \u001b[0mfastqs\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mfile\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mfile\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mfastqs\u001b[0m \u001b[0;32mif\u001b[0m \u001b[0;34m'.sorted.bam'\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0mfile\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m11\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m--> 903 \u001b[0;31m    \u001b[0mmapped\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m{\u001b[0m\u001b[0;34m}\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m    904 \u001b[0;31m    \u001b[0mnorm\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m{\u001b[0m\u001b[0;34m}\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\n",
      "ipdb> \n",
      "> \u001b[0;32m/home/jeremie/JKBio/epigenetics/ChIP_helper.py\u001b[0m(902)\u001b[0;36mgetSpikeInControlScales\u001b[0;34m()\u001b[0m\n",
      "\u001b[0;32m    901 \u001b[0;31m        \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"files need to be named: NAME.sorted.bam\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m--> 902 \u001b[0;31m        \u001b[0mfastqs\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mfile\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mfile\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mfastqs\u001b[0m \u001b[0;32mif\u001b[0m \u001b[0;34m'.sorted.bam'\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0mfile\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m11\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m    903 \u001b[0;31m    \u001b[0mmapped\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m{\u001b[0m\u001b[0;34m}\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\n",
      "ipdb> c\n",
      "> \u001b[0;32m/home/jeremie/JKBio/epigenetics/ChIP_helper.py\u001b[0m(901)\u001b[0;36mgetSpikeInControlScales\u001b[0;34m()\u001b[0m\n",
      "\u001b[0;32m    900 \u001b[0;31m    \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m--> 901 \u001b[0;31m        \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"files need to be named: NAME.sorted.bam\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m    902 \u001b[0;31m        \u001b[0mfastqs\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mfile\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mfile\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mfastqs\u001b[0m \u001b[0;32mif\u001b[0m \u001b[0;34m'.sorted.bam'\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0mfile\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m11\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\n",
      "ipdb> c\n",
      "> \u001b[0;32m/home/jeremie/JKBio/epigenetics/ChIP_helper.py\u001b[0m(902)\u001b[0;36mgetSpikeInControlScales\u001b[0;34m()\u001b[0m\n",
      "\u001b[0;32m    901 \u001b[0;31m        \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"files need to be named: NAME.sorted.bam\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m--> 902 \u001b[0;31m        \u001b[0mfastqs\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mfile\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mfile\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mfastqs\u001b[0m \u001b[0;32mif\u001b[0m \u001b[0;34m'.sorted.bam'\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0mfile\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m11\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m    903 \u001b[0;31m    \u001b[0mmapped\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m{\u001b[0m\u001b[0;34m}\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\n",
      "ipdb> c\n",
      "> \u001b[0;32m/home/jeremie/JKBio/epigenetics/ChIP_helper.py\u001b[0m(901)\u001b[0;36mgetSpikeInControlScales\u001b[0;34m()\u001b[0m\n",
      "\u001b[0;32m    900 \u001b[0;31m    \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m--> 901 \u001b[0;31m        \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"files need to be named: NAME.sorted.bam\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m    902 \u001b[0;31m        \u001b[0mfastqs\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mfile\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mfile\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mfastqs\u001b[0m \u001b[0;32mif\u001b[0m \u001b[0;34m'.sorted.bam'\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0mfile\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m11\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\n",
      "ipdb> c\n",
      "> \u001b[0;32m/home/jeremie/JKBio/epigenetics/ChIP_helper.py\u001b[0m(902)\u001b[0;36mgetSpikeInControlScales\u001b[0;34m()\u001b[0m\n",
      "\u001b[0;32m    901 \u001b[0;31m        \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"files need to be named: NAME.sorted.bam\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m--> 902 \u001b[0;31m        \u001b[0mfastqs\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mfile\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mfile\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mfastqs\u001b[0m \u001b[0;32mif\u001b[0m \u001b[0;34m'.sorted.bam'\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0mfile\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m11\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\u001b[0;32m    903 \u001b[0;31m    \u001b[0mmapped\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m{\u001b[0m\u001b[0;34m}\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0m\n",
      "ipdb> c\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "({'mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1_val_1': 0.2990873533246415,\n",
       "  'mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1_val_1': 0.20899663394552362,\n",
       "  'mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1_val_1': 1.0,\n",
       "  'mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1_val_1': 0.5203418741380562,\n",
       "  'mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1_val_1': 0.6130608143500059,\n",
       "  'mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_val_1': 0.4283750877664739,\n",
       "  'mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1_val_1': 0.07861534637359585,\n",
       "  'mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1_val_1': 0.18853988706236116,\n",
       "  'mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1_val_1': 0.18696534756754876,\n",
       "  'mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1_val_1': 0.2862047798943014,\n",
       "  'mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1_val_1': 0.37066001396034226,\n",
       "  'mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_val_1': 0.6839730465721839},\n",
       " {'mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1_val_1': 575250,\n",
       "  'mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1_val_1': 823219,\n",
       "  'mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1_val_1': 172050,\n",
       "  'mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1_val_1': 330648,\n",
       "  'mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1_val_1': 280641,\n",
       "  'mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_val_1': 401634,\n",
       "  'mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1_val_1': 2188504,\n",
       "  'mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1_val_1': 912539,\n",
       "  'mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1_val_1': 920224,\n",
       "  'mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1_val_1': 601143,\n",
       "  'mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1_val_1': 464172,\n",
       "  'mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_val_1': 251545})"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# get scaling values\n",
    "mappedreads, umappedreads_norm = chip.getSpikeInControlScales(refgenome=\"../../data/ref/reference_droso.fna\",\n",
    "fastQfolder='../../data/res4/',\n",
    "pairedEnd=True, cores=12,\n",
    "tofilter=True,\n",
    "totrim=False,\n",
    "tomap=True,\n",
    "results=\"../../data/res4/\",\n",
    "pathtotrim_galore=\"../../TrimGalore-0.6.5/trim_galore\")\n",
    "mappedreads, umappedreads_norm"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "computing scales from the excel sheet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [[262850,1000000],\n",
    "[902119,1000000],\n",
    "[186965,1000000],\n",
    "[550032,1000000],\n",
    "[604605,1000000],\n",
    "[1000000,626304]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [[1.0,0.2628507876],\n",
    "[1.0,0.9021192519],\n",
    "[1.0,0.1869653476],\n",
    "[1.0,0.5500321887],\n",
    "[1.0,0.6046056203],\n",
    "[0.626304048,1.0]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam',\n",
       " '../../data/results4/bwa/mergedLibrary/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1.mLb.clN.sorted.bam',\n",
       " '../../data/results4/bwa/mergedLibrary/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1.mLb.clN.sorted.bam',\n",
       " '../../data/results4/bwa/mergedLibrary/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1.mLb.clN.sorted.bam',\n",
       " '../../data/results4/bwa/mergedLibrary/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1.mLb.clN.sorted.bam',\n",
       " '../../data/results4/bwa/mergedLibrary/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1.mLb.clN.sorted.bam',\n",
       " '../../data/results4/bwa/mergedLibrary/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1.mLb.clN.sorted.bam',\n",
       " '../../data/results4/bwa/mergedLibrary/mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1.mLb.clN.sorted.bam',\n",
       " '../../data/results4/bwa/mergedLibrary/mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1.mLb.clN.sorted.bam',\n",
       " '../../data/results4/bwa/mergedLibrary/mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1.mLb.clN.sorted.bam',\n",
       " '../../data/results4/bwa/mergedLibrary/mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1.mLb.clN.sorted.bam',\n",
       " '../../data/results4/bwa/mergedLibrary/mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1.mLb.clN.sorted.bam',\n",
       " '../../data/results4/bwa/mergedLibrary/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1.mLb.clN.sorted.bam']"
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bams = ! ls ../../data/results4/bwa/mergedLibrary/*.bam\n",
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1'"
      ]
     },
     "execution_count": 37,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bams[6]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### on scalled data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "mkdir ../../data/IRF2BP2_degraded_4/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [],
   "source": [
    "bams = [bam1.split('/')[-1].split('.')[0] for bam1 in bams]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1 mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1\n",
      "doing differential peak binding\n",
      "mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1 mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1\n",
      "doing differential peak binding\n",
      "mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1 mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1\n",
      "doing differential peak binding\n",
      "mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1 mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1\n",
      "doing differential peak binding\n",
      "mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1 mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1\n",
      "doing differential peak binding\n"
     ]
    }
   ],
   "source": [
    "# diffPeak on scaled data\n",
    "size=[ 208, 214, 207, 234, 296, 231]\n",
    "for i in range(int(len(bams)/2)-1):\n",
    "    name1 = bams[1+i]\n",
    "    name2 = bams[7+i]\n",
    "    print(name1,name2)\n",
    "    chip.diffPeak(name1, name2, directory= \"../../data/IRF2BP2_degraded_4/diffData/\", res_directory='../../data/IRF2BP2_degraded_4/diffPeaks/', scaling1=scales[i][1], scaling2=scales[i][0], size=size[i])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[autoreload of JKBio.Helper failed: Traceback (most recent call last):\n",
      "  File \"/home/jeremie/.local/lib/python3.7/site-packages/IPython/extensions/autoreload.py\", line 245, in check\n",
      "    superreload(m, reload, self.old_objects)\n",
      "  File \"/home/jeremie/.local/lib/python3.7/site-packages/IPython/extensions/autoreload.py\", line 394, in superreload\n",
      "    module = reload(module)\n",
      "  File \"/usr/lib/python3.7/imp.py\", line 314, in reload\n",
      "    return importlib.reload(module)\n",
      "  File \"/usr/lib/python3.7/importlib/__init__.py\", line 169, in reload\n",
      "    _bootstrap._exec(spec, module)\n",
      "  File \"<frozen importlib._bootstrap>\", line 630, in _exec\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 724, in exec_module\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 860, in get_code\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 791, in source_to_code\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"../../JKBio/Helper.py\", line 569\n",
      "    if len(add) != 0\n",
      "                   ^\n",
      "SyntaxError: invalid syntax\n",
      "]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "doing diff from ../../data/results4/bwa/mergedLibrary/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1.mLb.clN.sorted.bam and ../../data/results4/bwa/mergedLibrary/mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "208\n",
      "computing the scaling values\n",
      "b'INFO  @ Fri, 13 Mar 2020 18:48:38: \\n# Command line: callpeak -B -t ../../data/results4/bwa/mergedLibrary/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1.mLb.clN.sorted.bam -c ../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 208 -n mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1 --outdir ../../data/IRF2BP2_degraded_4/diffData/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/results4/bwa/mergedLibrary/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Fri, 13 Mar 2020 18:48:38: #1 read tag files... \\nINFO  @ Fri, 13 Mar 2020 18:48:38: #1 read treatment tags... \\nINFO  @ Fri, 13 Mar 2020 18:48:43:  1000000 \\nINFO  @ Fri, 13 Mar 2020 18:48:47:  2000000 \\nINFO  @ Fri, 13 Mar 2020 18:48:52:  3000000 \\nINFO  @ Fri, 13 Mar 2020 18:48:58:  4000000 \\nINFO  @ Fri, 13 Mar 2020 18:49:05:  5000000 \\nINFO  @ Fri, 13 Mar 2020 18:49:12:  6000000 \\nINFO  @ Fri, 13 Mar 2020 18:49:19:  7000000 \\nINFO  @ Fri, 13 Mar 2020 18:49:26:  8000000 \\nINFO  @ Fri, 13 Mar 2020 18:49:33:  9000000 \\nINFO  @ Fri, 13 Mar 2020 18:49:40:  10000000 \\nINFO  @ Fri, 13 Mar 2020 18:49:46:  11000000 \\nINFO  @ Fri, 13 Mar 2020 18:49:51:  12000000 \\nINFO  @ Fri, 13 Mar 2020 18:49:55:  13000000 \\nINFO  @ Fri, 13 Mar 2020 18:50:00:  14000000 \\nINFO  @ Fri, 13 Mar 2020 18:50:00: 14007841 reads have been read. \\nINFO  @ Fri, 13 Mar 2020 18:50:00: #1.2 read input tags... \\nINFO  @ Fri, 13 Mar 2020 18:50:05:  1000000 \\nINFO  @ Fri, 13 Mar 2020 18:50:09:  2000000 \\nINFO  @ Fri, 13 Mar 2020 18:50:14:  3000000 \\nINFO  @ Fri, 13 Mar 2020 18:50:18:  4000000 \\nINFO  @ Fri, 13 Mar 2020 18:50:23:  5000000 \\nINFO  @ Fri, 13 Mar 2020 18:50:30:  6000000 \\nINFO  @ Fri, 13 Mar 2020 18:50:37:  7000000 \\nINFO  @ Fri, 13 Mar 2020 18:50:44:  8000000 \\nINFO  @ Fri, 13 Mar 2020 18:50:50:  9000000 \\nINFO  @ Fri, 13 Mar 2020 18:50:59:  10000000 \\nINFO  @ Fri, 13 Mar 2020 18:51:06:  11000000 \\nINFO  @ Fri, 13 Mar 2020 18:51:13:  12000000 \\nINFO  @ Fri, 13 Mar 2020 18:51:20:  13000000 \\nINFO  @ Fri, 13 Mar 2020 18:51:28:  14000000 \\nINFO  @ Fri, 13 Mar 2020 18:51:35:  15000000 \\nINFO  @ Fri, 13 Mar 2020 18:51:42:  16000000 \\nINFO  @ Fri, 13 Mar 2020 18:51:49:  17000000 \\nINFO  @ Fri, 13 Mar 2020 18:51:56:  18000000 \\nINFO  @ Fri, 13 Mar 2020 18:52:04:  19000000 \\nINFO  @ Fri, 13 Mar 2020 18:52:11:  20000000 \\nINFO  @ Fri, 13 Mar 2020 18:52:18:  21000000 \\nINFO  @ Fri, 13 Mar 2020 18:52:25:  22000000 \\nINFO  @ Fri, 13 Mar 2020 18:52:33:  23000000 \\nINFO  @ Fri, 13 Mar 2020 18:52:40:  24000000 \\nINFO  @ Fri, 13 Mar 2020 18:52:47:  25000000 \\nINFO  @ Fri, 13 Mar 2020 18:52:54:  26000000 \\nINFO  @ Fri, 13 Mar 2020 18:53:01:  27000000 \\nINFO  @ Fri, 13 Mar 2020 18:53:09:  28000000 \\nINFO  @ Fri, 13 Mar 2020 18:53:16:  29000000 \\nINFO  @ Fri, 13 Mar 2020 18:53:23:  30000000 \\nINFO  @ Fri, 13 Mar 2020 18:53:30:  31000000 \\nINFO  @ Fri, 13 Mar 2020 18:53:38:  32000000 \\nINFO  @ Fri, 13 Mar 2020 18:53:45:  33000000 \\nINFO  @ Fri, 13 Mar 2020 18:53:52:  34000000 \\nINFO  @ Fri, 13 Mar 2020 18:53:59:  35000000 \\nINFO  @ Fri, 13 Mar 2020 18:54:07:  36000000 \\nINFO  @ Fri, 13 Mar 2020 18:54:14:  37000000 \\nINFO  @ Fri, 13 Mar 2020 18:54:21:  38000000 \\nINFO  @ Fri, 13 Mar 2020 18:54:28:  39000000 \\nINFO  @ Fri, 13 Mar 2020 18:54:36:  40000000 \\nINFO  @ Fri, 13 Mar 2020 18:54:43:  41000000 \\nINFO  @ Fri, 13 Mar 2020 18:54:50:  42000000 \\nINFO  @ Fri, 13 Mar 2020 18:54:57:  43000000 \\nINFO  @ Fri, 13 Mar 2020 18:55:04:  44000000 \\nINFO  @ Fri, 13 Mar 2020 18:55:12:  45000000 \\nINFO  @ Fri, 13 Mar 2020 18:55:19:  46000000 \\nINFO  @ Fri, 13 Mar 2020 18:55:26:  47000000 \\nINFO  @ Fri, 13 Mar 2020 18:55:33:  48000000 \\nINFO  @ Fri, 13 Mar 2020 18:55:41:  49000000 \\nINFO  @ Fri, 13 Mar 2020 18:55:48:  50000000 \\nINFO  @ Fri, 13 Mar 2020 18:55:55:  51000000 \\nINFO  @ Fri, 13 Mar 2020 18:56:02:  52000000 \\nINFO  @ Fri, 13 Mar 2020 18:56:11:  53000000 \\nINFO  @ Fri, 13 Mar 2020 18:56:18:  54000000 \\nINFO  @ Fri, 13 Mar 2020 18:56:25:  55000000 \\nINFO  @ Fri, 13 Mar 2020 18:56:32:  56000000 \\nINFO  @ Fri, 13 Mar 2020 18:56:40:  57000000 \\nINFO  @ Fri, 13 Mar 2020 18:56:47:  58000000 \\nINFO  @ Fri, 13 Mar 2020 18:56:54:  59000000 \\nINFO  @ Fri, 13 Mar 2020 18:57:01:  60000000 \\nINFO  @ Fri, 13 Mar 2020 18:57:09:  61000000 \\nINFO  @ Fri, 13 Mar 2020 18:57:16:  62000000 \\nINFO  @ Fri, 13 Mar 2020 18:57:23:  63000000 \\nINFO  @ Fri, 13 Mar 2020 18:57:30:  64000000 \\nINFO  @ Fri, 13 Mar 2020 18:57:37:  65000000 \\nINFO  @ Fri, 13 Mar 2020 18:57:46:  66000000 \\nINFO  @ Fri, 13 Mar 2020 18:57:51: 66660428 reads have been read. \\nINFO  @ Fri, 13 Mar 2020 18:57:52: #1 tag size is determined as 49 bps \\nINFO  @ Fri, 13 Mar 2020 18:57:52: #1 tag size = 49.0 \\nINFO  @ Fri, 13 Mar 2020 18:57:52: #1  total tags in treatment: 14007841 \\nINFO  @ Fri, 13 Mar 2020 18:57:52: #1 user defined the maximum tags... \\nINFO  @ Fri, 13 Mar 2020 18:57:52: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Fri, 13 Mar 2020 18:57:52: #1  tags after filtering in treatment: 13913704 \\nINFO  @ Fri, 13 Mar 2020 18:57:52: #1  Redundant rate of treatment: 0.01 \\nINFO  @ Fri, 13 Mar 2020 18:57:52: #1  total tags in control: 66660428 \\nINFO  @ Fri, 13 Mar 2020 18:57:52: #1 user defined the maximum tags... \\nINFO  @ Fri, 13 Mar 2020 18:57:52: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Fri, 13 Mar 2020 18:57:53: #1  tags after filtering in control: 65724989 \\nINFO  @ Fri, 13 Mar 2020 18:57:53: #1  Redundant rate of control: 0.01 \\nINFO  @ Fri, 13 Mar 2020 18:57:53: #1 finished! \\nINFO  @ Fri, 13 Mar 2020 18:57:53: #2 Build Peak Model... \\nINFO  @ Fri, 13 Mar 2020 18:57:53: #2 Skipped... \\nINFO  @ Fri, 13 Mar 2020 18:57:53: #2 Use 208 as fragment length \\nINFO  @ Fri, 13 Mar 2020 18:57:53: #3 Call peaks... \\nINFO  @ Fri, 13 Mar 2020 18:57:53: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Fri, 13 Mar 2020 19:06:20: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Fri, 13 Mar 2020 19:06:20: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1_treat_pileup.bdg \\nINFO  @ Fri, 13 Mar 2020 19:06:20: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1_control_lambda.bdg \\nINFO  @ Fri, 13 Mar 2020 19:06:20: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Fri, 13 Mar 2020 19:06:20: #3 Call peaks for each chromosome... \\nINFO  @ Fri, 13 Mar 2020 19:12:25: #4 Write output xls file... ../../data/IRF2BP2_degraded_4/diffData/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1_peaks.xls \\nINFO  @ Fri, 13 Mar 2020 19:12:25: #4 Write peak in narrowPeak format file... ../../data/IRF2BP2_degraded_4/diffData/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1_peaks.narrowPeak \\nINFO  @ Fri, 13 Mar 2020 19:12:25: #4 Write summits bed file... ../../data/IRF2BP2_degraded_4/diffData/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1_summits.bed \\nINFO  @ Fri, 13 Mar 2020 19:12:25: Done! \\n'\n",
      "b'INFO  @ Fri, 13 Mar 2020 19:12:27: \\n# Command line: callpeak -B -t ../../data/results4/bwa/mergedLibrary/mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1.mLb.clN.sorted.bam -c ../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 208 -n mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1 --outdir ../../data/IRF2BP2_degraded_4/diffData/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/results4/bwa/mergedLibrary/mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Fri, 13 Mar 2020 19:12:27: #1 read tag files... \\nINFO  @ Fri, 13 Mar 2020 19:12:27: #1 read treatment tags... \\nINFO  @ Fri, 13 Mar 2020 19:12:34:  1000000 \\nINFO  @ Fri, 13 Mar 2020 19:12:42:  2000000 \\nINFO  @ Fri, 13 Mar 2020 19:12:50:  3000000 \\nINFO  @ Fri, 13 Mar 2020 19:12:58:  4000000 \\nINFO  @ Fri, 13 Mar 2020 19:13:05:  5000000 \\nINFO  @ Fri, 13 Mar 2020 19:13:12:  6000000 \\nINFO  @ Fri, 13 Mar 2020 19:13:20:  7000000 \\nINFO  @ Fri, 13 Mar 2020 19:13:28:  8000000 \\nINFO  @ Fri, 13 Mar 2020 19:13:35:  9000000 \\nINFO  @ Fri, 13 Mar 2020 19:13:42:  10000000 \\nINFO  @ Fri, 13 Mar 2020 19:13:50:  11000000 \\nINFO  @ Fri, 13 Mar 2020 19:13:58:  12000000 \\nINFO  @ Fri, 13 Mar 2020 19:14:03: 12719977 reads have been read. \\nINFO  @ Fri, 13 Mar 2020 19:14:04: #1.2 read input tags... \\nINFO  @ Fri, 13 Mar 2020 19:14:10:  1000000 \\nINFO  @ Fri, 13 Mar 2020 19:14:17:  2000000 \\nINFO  @ Fri, 13 Mar 2020 19:14:24:  3000000 \\nINFO  @ Fri, 13 Mar 2020 19:14:30:  4000000 \\nINFO  @ Fri, 13 Mar 2020 19:14:37:  5000000 \\nINFO  @ Fri, 13 Mar 2020 19:14:44:  6000000 \\nINFO  @ Fri, 13 Mar 2020 19:14:50:  7000000 \\nINFO  @ Fri, 13 Mar 2020 19:14:57:  8000000 \\nINFO  @ Fri, 13 Mar 2020 19:15:04:  9000000 \\nINFO  @ Fri, 13 Mar 2020 19:15:12:  10000000 \\nINFO  @ Fri, 13 Mar 2020 19:15:18:  11000000 \\nINFO  @ Fri, 13 Mar 2020 19:15:25:  12000000 \\nINFO  @ Fri, 13 Mar 2020 19:15:32:  13000000 \\nINFO  @ Fri, 13 Mar 2020 19:15:38:  14000000 \\nINFO  @ Fri, 13 Mar 2020 19:15:45:  15000000 \\nINFO  @ Fri, 13 Mar 2020 19:15:52:  16000000 \\nINFO  @ Fri, 13 Mar 2020 19:15:58:  17000000 \\nINFO  @ Fri, 13 Mar 2020 19:16:05:  18000000 \\nINFO  @ Fri, 13 Mar 2020 19:16:12:  19000000 \\nINFO  @ Fri, 13 Mar 2020 19:16:19:  20000000 \\nINFO  @ Fri, 13 Mar 2020 19:16:25:  21000000 \\nINFO  @ Fri, 13 Mar 2020 19:16:32:  22000000 \\nINFO  @ Fri, 13 Mar 2020 19:16:39:  23000000 \\nINFO  @ Fri, 13 Mar 2020 19:16:46:  24000000 \\nINFO  @ Fri, 13 Mar 2020 19:16:52:  25000000 \\nINFO  @ Fri, 13 Mar 2020 19:16:59:  26000000 \\nINFO  @ Fri, 13 Mar 2020 19:17:05:  27000000 \\nINFO  @ Fri, 13 Mar 2020 19:17:12:  28000000 \\nINFO  @ Fri, 13 Mar 2020 19:17:19:  29000000 \\nINFO  @ Fri, 13 Mar 2020 19:17:26:  30000000 \\nINFO  @ Fri, 13 Mar 2020 19:17:32:  31000000 \\nINFO  @ Fri, 13 Mar 2020 19:17:39:  32000000 \\nINFO  @ Fri, 13 Mar 2020 19:17:46:  33000000 \\nINFO  @ Fri, 13 Mar 2020 19:17:52:  34000000 \\nINFO  @ Fri, 13 Mar 2020 19:17:59:  35000000 \\nINFO  @ Fri, 13 Mar 2020 19:18:06:  36000000 \\nINFO  @ Fri, 13 Mar 2020 19:18:12:  37000000 \\nINFO  @ Fri, 13 Mar 2020 19:18:19:  38000000 \\nINFO  @ Fri, 13 Mar 2020 19:18:26:  39000000 \\nINFO  @ Fri, 13 Mar 2020 19:18:32:  40000000 \\nINFO  @ Fri, 13 Mar 2020 19:18:39:  41000000 \\nINFO  @ Fri, 13 Mar 2020 19:18:46:  42000000 \\nINFO  @ Fri, 13 Mar 2020 19:18:53:  43000000 \\nINFO  @ Fri, 13 Mar 2020 19:18:59:  44000000 \\nINFO  @ Fri, 13 Mar 2020 19:19:06:  45000000 \\nINFO  @ Fri, 13 Mar 2020 19:19:16:  46000000 \\nINFO  @ Fri, 13 Mar 2020 19:19:30:  47000000 \\nINFO  @ Fri, 13 Mar 2020 19:19:40:  48000000 \\nINFO  @ Fri, 13 Mar 2020 19:19:47:  49000000 \\nINFO  @ Fri, 13 Mar 2020 19:19:54:  50000000 \\nINFO  @ Fri, 13 Mar 2020 19:20:04:  51000000 \\nINFO  @ Fri, 13 Mar 2020 19:20:10:  52000000 \\nINFO  @ Fri, 13 Mar 2020 19:20:17:  53000000 \\nINFO  @ Fri, 13 Mar 2020 19:20:24:  54000000 \\nINFO  @ Fri, 13 Mar 2020 19:20:31:  55000000 \\nINFO  @ Fri, 13 Mar 2020 19:20:37:  56000000 \\nINFO  @ Fri, 13 Mar 2020 19:20:44:  57000000 \\nINFO  @ Fri, 13 Mar 2020 19:20:51:  58000000 \\nINFO  @ Fri, 13 Mar 2020 19:20:58:  59000000 \\nINFO  @ Fri, 13 Mar 2020 19:21:04:  60000000 \\nINFO  @ Fri, 13 Mar 2020 19:21:11:  61000000 \\nINFO  @ Fri, 13 Mar 2020 19:21:18:  62000000 \\nINFO  @ Fri, 13 Mar 2020 19:21:24:  63000000 \\nINFO  @ Fri, 13 Mar 2020 19:21:31:  64000000 \\nINFO  @ Fri, 13 Mar 2020 19:21:38:  65000000 \\nINFO  @ Fri, 13 Mar 2020 19:21:45:  66000000 \\nINFO  @ Fri, 13 Mar 2020 19:21:50: 66660428 reads have been read. \\nINFO  @ Fri, 13 Mar 2020 19:21:51: #1 tag size is determined as 48 bps \\nINFO  @ Fri, 13 Mar 2020 19:21:51: #1 tag size = 48.0 \\nINFO  @ Fri, 13 Mar 2020 19:21:51: #1  total tags in treatment: 12719977 \\nINFO  @ Fri, 13 Mar 2020 19:21:51: #1 user defined the maximum tags... \\nINFO  @ Fri, 13 Mar 2020 19:21:51: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Fri, 13 Mar 2020 19:21:51: #1  tags after filtering in treatment: 12638262 \\nINFO  @ Fri, 13 Mar 2020 19:21:51: #1  Redundant rate of treatment: 0.01 \\nINFO  @ Fri, 13 Mar 2020 19:21:51: #1  total tags in control: 66660428 \\nINFO  @ Fri, 13 Mar 2020 19:21:51: #1 user defined the maximum tags... \\nINFO  @ Fri, 13 Mar 2020 19:21:51: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Fri, 13 Mar 2020 19:21:52: #1  tags after filtering in control: 65724989 \\nINFO  @ Fri, 13 Mar 2020 19:21:52: #1  Redundant rate of control: 0.01 \\nINFO  @ Fri, 13 Mar 2020 19:21:52: #1 finished! \\nINFO  @ Fri, 13 Mar 2020 19:21:52: #2 Build Peak Model... \\nINFO  @ Fri, 13 Mar 2020 19:21:52: #2 Skipped... \\nINFO  @ Fri, 13 Mar 2020 19:21:52: #2 Use 208 as fragment length \\nINFO  @ Fri, 13 Mar 2020 19:21:52: #3 Call peaks... \\nINFO  @ Fri, 13 Mar 2020 19:21:52: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Fri, 13 Mar 2020 19:29:37: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Fri, 13 Mar 2020 19:29:37: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1_treat_pileup.bdg \\nINFO  @ Fri, 13 Mar 2020 19:29:37: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1_control_lambda.bdg \\nINFO  @ Fri, 13 Mar 2020 19:29:37: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Fri, 13 Mar 2020 19:29:37: #3 Call peaks for each chromosome... \\nINFO  @ Fri, 13 Mar 2020 19:35:37: #4 Write output xls file... ../../data/IRF2BP2_degraded_4/diffData/mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1_peaks.xls \\nINFO  @ Fri, 13 Mar 2020 19:35:37: #4 Write peak in narrowPeak format file... ../../data/IRF2BP2_degraded_4/diffData/mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1_peaks.narrowPeak \\nINFO  @ Fri, 13 Mar 2020 19:35:37: #4 Write summits bed file... ../../data/IRF2BP2_degraded_4/diffData/mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1_summits.bed \\nINFO  @ Fri, 13 Mar 2020 19:35:37: Done! \\n'\n",
      "13913704 12638262\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "doing diff from ../../data/results4/bwa/mergedLibrary/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1.mLb.clN.sorted.bam and ../../data/results4/bwa/mergedLibrary/mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "214\n",
      "computing the scaling values\n",
      "b'INFO  @ Fri, 13 Mar 2020 19:53:12: \\n# Command line: callpeak -B -t ../../data/results4/bwa/mergedLibrary/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1.mLb.clN.sorted.bam -c ../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 214 -n mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1 --outdir ../../data/IRF2BP2_degraded_4/diffData/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/results4/bwa/mergedLibrary/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Fri, 13 Mar 2020 19:53:12: #1 read tag files... \\nINFO  @ Fri, 13 Mar 2020 19:53:12: #1 read treatment tags... \\nINFO  @ Fri, 13 Mar 2020 19:53:19:  1000000 \\nINFO  @ Fri, 13 Mar 2020 19:53:26:  2000000 \\nINFO  @ Fri, 13 Mar 2020 19:53:32:  3000000 \\nINFO  @ Fri, 13 Mar 2020 19:53:39:  4000000 \\nINFO  @ Fri, 13 Mar 2020 19:53:46:  5000000 \\nINFO  @ Fri, 13 Mar 2020 19:53:52:  6000000 \\nINFO  @ Fri, 13 Mar 2020 19:53:59:  7000000 \\nINFO  @ Fri, 13 Mar 2020 19:54:06:  8000000 \\nINFO  @ Fri, 13 Mar 2020 19:54:13:  9000000 \\nINFO  @ Fri, 13 Mar 2020 19:54:19:  10000000 \\nINFO  @ Fri, 13 Mar 2020 19:54:26:  11000000 \\nINFO  @ Fri, 13 Mar 2020 19:54:33:  12000000 \\nINFO  @ Fri, 13 Mar 2020 19:54:36: 12466587 reads have been read. \\nINFO  @ Fri, 13 Mar 2020 19:54:36: #1.2 read input tags... \\nINFO  @ Fri, 13 Mar 2020 19:54:42:  1000000 \\nINFO  @ Fri, 13 Mar 2020 19:54:49:  2000000 \\nINFO  @ Fri, 13 Mar 2020 19:54:55:  3000000 \\nINFO  @ Fri, 13 Mar 2020 19:55:01:  4000000 \\nINFO  @ Fri, 13 Mar 2020 19:55:07:  5000000 \\nINFO  @ Fri, 13 Mar 2020 19:55:14:  6000000 \\nINFO  @ Fri, 13 Mar 2020 19:55:20:  7000000 \\nINFO  @ Fri, 13 Mar 2020 19:55:26:  8000000 \\nINFO  @ Fri, 13 Mar 2020 19:55:32:  9000000 \\nINFO  @ Fri, 13 Mar 2020 19:55:38:  10000000 \\nINFO  @ Fri, 13 Mar 2020 19:55:44:  11000000 \\nINFO  @ Fri, 13 Mar 2020 19:55:49:  12000000 \\nINFO  @ Fri, 13 Mar 2020 19:55:55:  13000000 \\nINFO  @ Fri, 13 Mar 2020 19:56:00:  14000000 \\nINFO  @ Fri, 13 Mar 2020 19:56:06:  15000000 \\nINFO  @ Fri, 13 Mar 2020 19:56:11:  16000000 \\nINFO  @ Fri, 13 Mar 2020 19:56:16:  17000000 \\nINFO  @ Fri, 13 Mar 2020 19:56:22:  18000000 \\nINFO  @ Fri, 13 Mar 2020 19:56:27:  19000000 \\nINFO  @ Fri, 13 Mar 2020 19:56:33:  20000000 \\nINFO  @ Fri, 13 Mar 2020 19:56:38:  21000000 \\nINFO  @ Fri, 13 Mar 2020 19:56:43:  22000000 \\nINFO  @ Fri, 13 Mar 2020 19:56:49:  23000000 \\nINFO  @ Fri, 13 Mar 2020 19:56:54:  24000000 \\nINFO  @ Fri, 13 Mar 2020 19:56:59:  25000000 \\nINFO  @ Fri, 13 Mar 2020 19:57:04:  26000000 \\nINFO  @ Fri, 13 Mar 2020 19:57:09:  27000000 \\nINFO  @ Fri, 13 Mar 2020 19:57:14:  28000000 \\nINFO  @ Fri, 13 Mar 2020 19:57:18:  29000000 \\nINFO  @ Fri, 13 Mar 2020 19:57:23:  30000000 \\nINFO  @ Fri, 13 Mar 2020 19:57:28:  31000000 \\nINFO  @ Fri, 13 Mar 2020 19:57:33:  32000000 \\nINFO  @ Fri, 13 Mar 2020 19:57:38:  33000000 \\nINFO  @ Fri, 13 Mar 2020 19:57:43:  34000000 \\nINFO  @ Fri, 13 Mar 2020 19:57:47:  35000000 \\nINFO  @ Fri, 13 Mar 2020 19:57:52:  36000000 \\nINFO  @ Fri, 13 Mar 2020 19:57:57:  37000000 \\nINFO  @ Fri, 13 Mar 2020 19:58:02:  38000000 \\nINFO  @ Fri, 13 Mar 2020 19:58:06:  39000000 \\nINFO  @ Fri, 13 Mar 2020 19:58:11:  40000000 \\nINFO  @ Fri, 13 Mar 2020 19:58:16:  41000000 \\nINFO  @ Fri, 13 Mar 2020 19:58:21:  42000000 \\nINFO  @ Fri, 13 Mar 2020 19:58:26:  43000000 \\nINFO  @ Fri, 13 Mar 2020 19:58:31:  44000000 \\nINFO  @ Fri, 13 Mar 2020 19:58:35:  45000000 \\nINFO  @ Fri, 13 Mar 2020 19:58:40:  46000000 \\nINFO  @ Fri, 13 Mar 2020 19:58:45:  47000000 \\nINFO  @ Fri, 13 Mar 2020 19:58:50:  48000000 \\nINFO  @ Fri, 13 Mar 2020 19:58:54:  49000000 \\nINFO  @ Fri, 13 Mar 2020 19:58:58:  50000000 \\nINFO  @ Fri, 13 Mar 2020 19:59:03:  51000000 \\nINFO  @ Fri, 13 Mar 2020 19:59:07:  52000000 \\nINFO  @ Fri, 13 Mar 2020 19:59:12:  53000000 \\nINFO  @ Fri, 13 Mar 2020 19:59:16:  54000000 \\nINFO  @ Fri, 13 Mar 2020 19:59:21:  55000000 \\nINFO  @ Fri, 13 Mar 2020 19:59:25:  56000000 \\nINFO  @ Fri, 13 Mar 2020 19:59:29:  57000000 \\nINFO  @ Fri, 13 Mar 2020 19:59:34:  58000000 \\nINFO  @ Fri, 13 Mar 2020 19:59:38:  59000000 \\nINFO  @ Fri, 13 Mar 2020 19:59:43:  60000000 \\nINFO  @ Fri, 13 Mar 2020 19:59:47:  61000000 \\nINFO  @ Fri, 13 Mar 2020 19:59:51:  62000000 \\nINFO  @ Fri, 13 Mar 2020 19:59:56:  63000000 \\nINFO  @ Fri, 13 Mar 2020 20:00:00:  64000000 \\nINFO  @ Fri, 13 Mar 2020 20:00:04:  65000000 \\nINFO  @ Fri, 13 Mar 2020 20:00:09:  66000000 \\nINFO  @ Fri, 13 Mar 2020 20:00:11: 66660428 reads have been read. \\nINFO  @ Fri, 13 Mar 2020 20:00:12: #1 tag size is determined as 49 bps \\nINFO  @ Fri, 13 Mar 2020 20:00:12: #1 tag size = 49.0 \\nINFO  @ Fri, 13 Mar 2020 20:00:12: #1  total tags in treatment: 12466587 \\nINFO  @ Fri, 13 Mar 2020 20:00:12: #1 user defined the maximum tags... \\nINFO  @ Fri, 13 Mar 2020 20:00:12: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Fri, 13 Mar 2020 20:00:12: #1  tags after filtering in treatment: 12391076 \\nINFO  @ Fri, 13 Mar 2020 20:00:12: #1  Redundant rate of treatment: 0.01 \\nINFO  @ Fri, 13 Mar 2020 20:00:12: #1  total tags in control: 66660428 \\nINFO  @ Fri, 13 Mar 2020 20:00:12: #1 user defined the maximum tags... \\nINFO  @ Fri, 13 Mar 2020 20:00:12: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Fri, 13 Mar 2020 20:00:12: #1  tags after filtering in control: 65724989 \\nINFO  @ Fri, 13 Mar 2020 20:00:12: #1  Redundant rate of control: 0.01 \\nINFO  @ Fri, 13 Mar 2020 20:00:12: #1 finished! \\nINFO  @ Fri, 13 Mar 2020 20:00:12: #2 Build Peak Model... \\nINFO  @ Fri, 13 Mar 2020 20:00:12: #2 Skipped... \\nINFO  @ Fri, 13 Mar 2020 20:00:12: #2 Use 214 as fragment length \\nINFO  @ Fri, 13 Mar 2020 20:00:12: #3 Call peaks... \\nINFO  @ Fri, 13 Mar 2020 20:00:12: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Fri, 13 Mar 2020 20:06:45: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Fri, 13 Mar 2020 20:06:45: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1_treat_pileup.bdg \\nINFO  @ Fri, 13 Mar 2020 20:06:45: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1_control_lambda.bdg \\nINFO  @ Fri, 13 Mar 2020 20:06:45: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Fri, 13 Mar 2020 20:06:45: #3 Call peaks for each chromosome... \\nINFO  @ Fri, 13 Mar 2020 20:13:11: #4 Write output xls file... ../../data/IRF2BP2_degraded_4/diffData/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1_peaks.xls \\nINFO  @ Fri, 13 Mar 2020 20:13:11: #4 Write peak in narrowPeak format file... ../../data/IRF2BP2_degraded_4/diffData/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1_peaks.narrowPeak \\nINFO  @ Fri, 13 Mar 2020 20:13:11: #4 Write summits bed file... ../../data/IRF2BP2_degraded_4/diffData/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1_summits.bed \\nINFO  @ Fri, 13 Mar 2020 20:13:11: Done! \\n'\n",
      "b'INFO  @ Fri, 13 Mar 2020 20:13:12: \\n# Command line: callpeak -B -t ../../data/results4/bwa/mergedLibrary/mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1.mLb.clN.sorted.bam -c ../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 214 -n mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1 --outdir ../../data/IRF2BP2_degraded_4/diffData/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/results4/bwa/mergedLibrary/mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Fri, 13 Mar 2020 20:13:12: #1 read tag files... \\nINFO  @ Fri, 13 Mar 2020 20:13:12: #1 read treatment tags... \\nINFO  @ Fri, 13 Mar 2020 20:13:19:  1000000 \\nINFO  @ Fri, 13 Mar 2020 20:13:28:  2000000 \\nINFO  @ Fri, 13 Mar 2020 20:13:36:  3000000 \\nINFO  @ Fri, 13 Mar 2020 20:13:43:  4000000 \\nINFO  @ Fri, 13 Mar 2020 20:13:51:  5000000 \\nINFO  @ Fri, 13 Mar 2020 20:13:59:  6000000 \\nINFO  @ Fri, 13 Mar 2020 20:14:07:  7000000 \\nINFO  @ Fri, 13 Mar 2020 20:14:15:  8000000 \\nINFO  @ Fri, 13 Mar 2020 20:14:22:  9000000 \\nINFO  @ Fri, 13 Mar 2020 20:14:31:  10000000 \\nINFO  @ Fri, 13 Mar 2020 20:14:31: 10102559 reads have been read. \\nINFO  @ Fri, 13 Mar 2020 20:14:32: #1.2 read input tags... \\nINFO  @ Fri, 13 Mar 2020 20:14:38:  1000000 \\nINFO  @ Fri, 13 Mar 2020 20:14:45:  2000000 \\nINFO  @ Fri, 13 Mar 2020 20:14:52:  3000000 \\nINFO  @ Fri, 13 Mar 2020 20:14:59:  4000000 \\nINFO  @ Fri, 13 Mar 2020 20:15:05:  5000000 \\nINFO  @ Fri, 13 Mar 2020 20:15:12:  6000000 \\nINFO  @ Fri, 13 Mar 2020 20:15:18:  7000000 \\nINFO  @ Fri, 13 Mar 2020 20:15:25:  8000000 \\nINFO  @ Fri, 13 Mar 2020 20:15:32:  9000000 \\nINFO  @ Fri, 13 Mar 2020 20:15:39:  10000000 \\nINFO  @ Fri, 13 Mar 2020 20:15:46:  11000000 \\nINFO  @ Fri, 13 Mar 2020 20:15:52:  12000000 \\nINFO  @ Fri, 13 Mar 2020 20:15:59:  13000000 \\nINFO  @ Fri, 13 Mar 2020 20:16:05:  14000000 \\nINFO  @ Fri, 13 Mar 2020 20:16:12:  15000000 \\nINFO  @ Fri, 13 Mar 2020 20:16:19:  16000000 \\nINFO  @ Fri, 13 Mar 2020 20:16:26:  17000000 \\nINFO  @ Fri, 13 Mar 2020 20:16:32:  18000000 \\nINFO  @ Fri, 13 Mar 2020 20:16:39:  19000000 \\nINFO  @ Fri, 13 Mar 2020 20:16:46:  20000000 \\nINFO  @ Fri, 13 Mar 2020 20:16:53:  21000000 \\nINFO  @ Fri, 13 Mar 2020 20:17:00:  22000000 \\nINFO  @ Fri, 13 Mar 2020 20:17:07:  23000000 \\nINFO  @ Fri, 13 Mar 2020 20:17:14:  24000000 \\nINFO  @ Fri, 13 Mar 2020 20:17:21:  25000000 \\nINFO  @ Fri, 13 Mar 2020 20:17:28:  26000000 \\nINFO  @ Fri, 13 Mar 2020 20:17:34:  27000000 \\nINFO  @ Fri, 13 Mar 2020 20:17:41:  28000000 \\nINFO  @ Fri, 13 Mar 2020 20:17:48:  29000000 \\nINFO  @ Fri, 13 Mar 2020 20:17:55:  30000000 \\nINFO  @ Fri, 13 Mar 2020 20:18:02:  31000000 \\nINFO  @ Fri, 13 Mar 2020 20:18:08:  32000000 \\nINFO  @ Fri, 13 Mar 2020 20:18:15:  33000000 \\nINFO  @ Fri, 13 Mar 2020 20:18:22:  34000000 \\nINFO  @ Fri, 13 Mar 2020 20:18:29:  35000000 \\nINFO  @ Fri, 13 Mar 2020 20:18:35:  36000000 \\nINFO  @ Fri, 13 Mar 2020 20:18:41:  37000000 \\nINFO  @ Fri, 13 Mar 2020 20:18:48:  38000000 \\nINFO  @ Fri, 13 Mar 2020 20:18:56:  39000000 \\nINFO  @ Fri, 13 Mar 2020 20:19:03:  40000000 \\nINFO  @ Fri, 13 Mar 2020 20:19:09:  41000000 \\nINFO  @ Fri, 13 Mar 2020 20:19:16:  42000000 \\nINFO  @ Fri, 13 Mar 2020 20:19:23:  43000000 \\nINFO  @ Fri, 13 Mar 2020 20:19:30:  44000000 \\nINFO  @ Fri, 13 Mar 2020 20:19:36:  45000000 \\nINFO  @ Fri, 13 Mar 2020 20:19:43:  46000000 \\nINFO  @ Fri, 13 Mar 2020 20:19:50:  47000000 \\nINFO  @ Fri, 13 Mar 2020 20:19:57:  48000000 \\nINFO  @ Fri, 13 Mar 2020 20:20:04:  49000000 \\nINFO  @ Fri, 13 Mar 2020 20:20:10:  50000000 \\nINFO  @ Fri, 13 Mar 2020 20:20:17:  51000000 \\nINFO  @ Fri, 13 Mar 2020 20:20:24:  52000000 \\nINFO  @ Fri, 13 Mar 2020 20:20:31:  53000000 \\nINFO  @ Fri, 13 Mar 2020 20:20:38:  54000000 \\nINFO  @ Fri, 13 Mar 2020 20:20:44:  55000000 \\nINFO  @ Fri, 13 Mar 2020 20:20:54:  56000000 \\nINFO  @ Fri, 13 Mar 2020 20:21:07:  57000000 \\nINFO  @ Fri, 13 Mar 2020 20:21:19:  58000000 \\nINFO  @ Fri, 13 Mar 2020 20:21:27:  59000000 \\nINFO  @ Fri, 13 Mar 2020 20:21:34:  60000000 \\nINFO  @ Fri, 13 Mar 2020 20:21:40:  61000000 \\nINFO  @ Fri, 13 Mar 2020 20:21:47:  62000000 \\nINFO  @ Fri, 13 Mar 2020 20:21:54:  63000000 \\nINFO  @ Fri, 13 Mar 2020 20:22:01:  64000000 \\nINFO  @ Fri, 13 Mar 2020 20:22:08:  65000000 \\nINFO  @ Fri, 13 Mar 2020 20:22:15:  66000000 \\nINFO  @ Fri, 13 Mar 2020 20:22:19: 66660428 reads have been read. \\nINFO  @ Fri, 13 Mar 2020 20:22:21: #1 tag size is determined as 48 bps \\nINFO  @ Fri, 13 Mar 2020 20:22:21: #1 tag size = 48.0 \\nINFO  @ Fri, 13 Mar 2020 20:22:21: #1  total tags in treatment: 10102559 \\nINFO  @ Fri, 13 Mar 2020 20:22:21: #1 user defined the maximum tags... \\nINFO  @ Fri, 13 Mar 2020 20:22:21: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Fri, 13 Mar 2020 20:22:21: #1  tags after filtering in treatment: 10044508 \\nINFO  @ Fri, 13 Mar 2020 20:22:21: #1  Redundant rate of treatment: 0.01 \\nINFO  @ Fri, 13 Mar 2020 20:22:21: #1  total tags in control: 66660428 \\nINFO  @ Fri, 13 Mar 2020 20:22:21: #1 user defined the maximum tags... \\nINFO  @ Fri, 13 Mar 2020 20:22:21: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Fri, 13 Mar 2020 20:22:21: #1  tags after filtering in control: 65724989 \\nINFO  @ Fri, 13 Mar 2020 20:22:21: #1  Redundant rate of control: 0.01 \\nINFO  @ Fri, 13 Mar 2020 20:22:21: #1 finished! \\nINFO  @ Fri, 13 Mar 2020 20:22:21: #2 Build Peak Model... \\nINFO  @ Fri, 13 Mar 2020 20:22:21: #2 Skipped... \\nINFO  @ Fri, 13 Mar 2020 20:22:21: #2 Use 214 as fragment length \\nINFO  @ Fri, 13 Mar 2020 20:22:21: #3 Call peaks... \\nINFO  @ Fri, 13 Mar 2020 20:22:21: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Fri, 13 Mar 2020 20:29:52: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Fri, 13 Mar 2020 20:29:52: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1_treat_pileup.bdg \\nINFO  @ Fri, 13 Mar 2020 20:29:52: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1_control_lambda.bdg \\nINFO  @ Fri, 13 Mar 2020 20:29:52: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Fri, 13 Mar 2020 20:29:52: #3 Call peaks for each chromosome... \\nINFO  @ Fri, 13 Mar 2020 20:35:39: #4 Write output xls file... ../../data/IRF2BP2_degraded_4/diffData/mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1_peaks.xls \\nINFO  @ Fri, 13 Mar 2020 20:35:39: #4 Write peak in narrowPeak format file... ../../data/IRF2BP2_degraded_4/diffData/mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1_peaks.narrowPeak \\nINFO  @ Fri, 13 Mar 2020 20:35:39: #4 Write summits bed file... ../../data/IRF2BP2_degraded_4/diffData/mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1_summits.bed \\nINFO  @ Fri, 13 Mar 2020 20:35:39: Done! \\n'\n",
      "12391076 10044508\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "doing diff from ../../data/results4/bwa/mergedLibrary/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1.mLb.clN.sorted.bam and ../../data/results4/bwa/mergedLibrary/mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "207\n",
      "computing the scaling values\n",
      "b'INFO  @ Fri, 13 Mar 2020 20:53:17: \\n# Command line: callpeak -B -t ../../data/results4/bwa/mergedLibrary/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1.mLb.clN.sorted.bam -c ../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 207 -n mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1 --outdir ../../data/IRF2BP2_degraded_4/diffData/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/results4/bwa/mergedLibrary/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Fri, 13 Mar 2020 20:53:17: #1 read tag files... \\nINFO  @ Fri, 13 Mar 2020 20:53:17: #1 read treatment tags... \\nINFO  @ Fri, 13 Mar 2020 20:53:24:  1000000 \\nINFO  @ Fri, 13 Mar 2020 20:53:31:  2000000 \\nINFO  @ Fri, 13 Mar 2020 20:53:45:  3000000 \\nINFO  @ Fri, 13 Mar 2020 20:53:59:  4000000 \\nINFO  @ Fri, 13 Mar 2020 20:54:09:  5000000 \\nINFO  @ Fri, 13 Mar 2020 20:54:17:  6000000 \\nINFO  @ Fri, 13 Mar 2020 20:54:24:  7000000 \\nINFO  @ Fri, 13 Mar 2020 20:54:31:  8000000 \\nINFO  @ Fri, 13 Mar 2020 20:54:38:  9000000 \\nINFO  @ Fri, 13 Mar 2020 20:54:45:  10000000 \\nINFO  @ Fri, 13 Mar 2020 20:54:52:  11000000 \\nINFO  @ Fri, 13 Mar 2020 20:54:59:  12000000 \\nINFO  @ Fri, 13 Mar 2020 20:55:06:  13000000 \\nINFO  @ Fri, 13 Mar 2020 20:55:14:  14000000 \\nINFO  @ Fri, 13 Mar 2020 20:55:21:  15000000 \\nINFO  @ Fri, 13 Mar 2020 20:55:28:  16000000 \\nINFO  @ Fri, 13 Mar 2020 20:55:35:  17000000 \\nINFO  @ Fri, 13 Mar 2020 20:55:39: 17557633 reads have been read. \\nINFO  @ Fri, 13 Mar 2020 20:55:40: #1.2 read input tags... \\nINFO  @ Fri, 13 Mar 2020 20:55:46:  1000000 \\nINFO  @ Fri, 13 Mar 2020 20:55:53:  2000000 \\nINFO  @ Fri, 13 Mar 2020 20:56:00:  3000000 \\nINFO  @ Fri, 13 Mar 2020 20:56:07:  4000000 \\nINFO  @ Fri, 13 Mar 2020 20:56:14:  5000000 \\nINFO  @ Fri, 13 Mar 2020 20:56:20:  6000000 \\nINFO  @ Fri, 13 Mar 2020 20:56:27:  7000000 \\nINFO  @ Fri, 13 Mar 2020 20:56:34:  8000000 \\nINFO  @ Fri, 13 Mar 2020 20:56:41:  9000000 \\nINFO  @ Fri, 13 Mar 2020 20:56:48:  10000000 \\nINFO  @ Fri, 13 Mar 2020 20:56:54:  11000000 \\nINFO  @ Fri, 13 Mar 2020 20:57:01:  12000000 \\nINFO  @ Fri, 13 Mar 2020 20:57:08:  13000000 \\nINFO  @ Fri, 13 Mar 2020 20:57:14:  14000000 \\nINFO  @ Fri, 13 Mar 2020 20:57:21:  15000000 \\nINFO  @ Fri, 13 Mar 2020 20:57:28:  16000000 \\nINFO  @ Fri, 13 Mar 2020 20:57:34:  17000000 \\nINFO  @ Fri, 13 Mar 2020 20:57:41:  18000000 \\nINFO  @ Fri, 13 Mar 2020 20:57:48:  19000000 \\nINFO  @ Fri, 13 Mar 2020 20:57:54:  20000000 \\nINFO  @ Fri, 13 Mar 2020 20:58:01:  21000000 \\nINFO  @ Fri, 13 Mar 2020 20:58:08:  22000000 \\nINFO  @ Fri, 13 Mar 2020 20:58:14:  23000000 \\nINFO  @ Fri, 13 Mar 2020 20:58:21:  24000000 \\nINFO  @ Fri, 13 Mar 2020 20:58:28:  25000000 \\nINFO  @ Fri, 13 Mar 2020 20:58:34:  26000000 \\nINFO  @ Fri, 13 Mar 2020 20:58:41:  27000000 \\nINFO  @ Fri, 13 Mar 2020 20:58:48:  28000000 \\nINFO  @ Fri, 13 Mar 2020 20:58:54:  29000000 \\nINFO  @ Fri, 13 Mar 2020 20:59:01:  30000000 \\nINFO  @ Fri, 13 Mar 2020 20:59:08:  31000000 \\nINFO  @ Fri, 13 Mar 2020 20:59:15:  32000000 \\nINFO  @ Fri, 13 Mar 2020 20:59:22:  33000000 \\nINFO  @ Fri, 13 Mar 2020 20:59:28:  34000000 \\nINFO  @ Fri, 13 Mar 2020 20:59:35:  35000000 \\nINFO  @ Fri, 13 Mar 2020 20:59:42:  36000000 \\nINFO  @ Fri, 13 Mar 2020 20:59:49:  37000000 \\nINFO  @ Fri, 13 Mar 2020 20:59:55:  38000000 \\nINFO  @ Fri, 13 Mar 2020 21:00:02:  39000000 \\nINFO  @ Fri, 13 Mar 2020 21:00:09:  40000000 \\nINFO  @ Fri, 13 Mar 2020 21:00:16:  41000000 \\nINFO  @ Fri, 13 Mar 2020 21:00:23:  42000000 \\nINFO  @ Fri, 13 Mar 2020 21:00:30:  43000000 \\nINFO  @ Fri, 13 Mar 2020 21:00:37:  44000000 \\nINFO  @ Fri, 13 Mar 2020 21:00:44:  45000000 \\nINFO  @ Fri, 13 Mar 2020 21:00:50:  46000000 \\nINFO  @ Fri, 13 Mar 2020 21:00:57:  47000000 \\nINFO  @ Fri, 13 Mar 2020 21:01:04:  48000000 \\nINFO  @ Fri, 13 Mar 2020 21:01:10:  49000000 \\nINFO  @ Fri, 13 Mar 2020 21:01:17:  50000000 \\nINFO  @ Fri, 13 Mar 2020 21:01:24:  51000000 \\nINFO  @ Fri, 13 Mar 2020 21:01:31:  52000000 \\nINFO  @ Fri, 13 Mar 2020 21:01:37:  53000000 \\nINFO  @ Fri, 13 Mar 2020 21:01:44:  54000000 \\nINFO  @ Fri, 13 Mar 2020 21:01:51:  55000000 \\nINFO  @ Fri, 13 Mar 2020 21:01:58:  56000000 \\nINFO  @ Fri, 13 Mar 2020 21:02:05:  57000000 \\nINFO  @ Fri, 13 Mar 2020 21:02:11:  58000000 \\nINFO  @ Fri, 13 Mar 2020 21:02:18:  59000000 \\nINFO  @ Fri, 13 Mar 2020 21:02:25:  60000000 \\nINFO  @ Fri, 13 Mar 2020 21:02:31:  61000000 \\nINFO  @ Fri, 13 Mar 2020 21:02:38:  62000000 \\nINFO  @ Fri, 13 Mar 2020 21:02:45:  63000000 \\nINFO  @ Fri, 13 Mar 2020 21:02:51:  64000000 \\nINFO  @ Fri, 13 Mar 2020 21:02:58:  65000000 \\nINFO  @ Fri, 13 Mar 2020 21:03:05:  66000000 \\nINFO  @ Fri, 13 Mar 2020 21:03:09: 66660428 reads have been read. \\nINFO  @ Fri, 13 Mar 2020 21:03:10: #1 tag size is determined as 49 bps \\nINFO  @ Fri, 13 Mar 2020 21:03:10: #1 tag size = 49.0 \\nINFO  @ Fri, 13 Mar 2020 21:03:10: #1  total tags in treatment: 17557633 \\nINFO  @ Fri, 13 Mar 2020 21:03:10: #1 user defined the maximum tags... \\nINFO  @ Fri, 13 Mar 2020 21:03:10: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Fri, 13 Mar 2020 21:03:10: #1  tags after filtering in treatment: 17473890 \\nINFO  @ Fri, 13 Mar 2020 21:03:10: #1  Redundant rate of treatment: 0.00 \\nINFO  @ Fri, 13 Mar 2020 21:03:10: #1  total tags in control: 66660428 \\nINFO  @ Fri, 13 Mar 2020 21:03:10: #1 user defined the maximum tags... \\nINFO  @ Fri, 13 Mar 2020 21:03:10: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Fri, 13 Mar 2020 21:03:11: #1  tags after filtering in control: 65724989 \\nINFO  @ Fri, 13 Mar 2020 21:03:11: #1  Redundant rate of control: 0.01 \\nINFO  @ Fri, 13 Mar 2020 21:03:11: #1 finished! \\nINFO  @ Fri, 13 Mar 2020 21:03:11: #2 Build Peak Model... \\nINFO  @ Fri, 13 Mar 2020 21:03:11: #2 Skipped... \\nINFO  @ Fri, 13 Mar 2020 21:03:11: #2 Use 207 as fragment length \\nINFO  @ Fri, 13 Mar 2020 21:03:11: #3 Call peaks... \\nINFO  @ Fri, 13 Mar 2020 21:03:11: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Fri, 13 Mar 2020 21:09:24: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Fri, 13 Mar 2020 21:09:24: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1_treat_pileup.bdg \\nINFO  @ Fri, 13 Mar 2020 21:09:24: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1_control_lambda.bdg \\nINFO  @ Fri, 13 Mar 2020 21:09:24: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Fri, 13 Mar 2020 21:09:24: #3 Call peaks for each chromosome... \\nINFO  @ Fri, 13 Mar 2020 21:13:02: #4 Write output xls file... ../../data/IRF2BP2_degraded_4/diffData/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1_peaks.xls \\nINFO  @ Fri, 13 Mar 2020 21:13:02: #4 Write peak in narrowPeak format file... ../../data/IRF2BP2_degraded_4/diffData/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1_peaks.narrowPeak \\nINFO  @ Fri, 13 Mar 2020 21:13:02: #4 Write summits bed file... ../../data/IRF2BP2_degraded_4/diffData/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1_summits.bed \\nINFO  @ Fri, 13 Mar 2020 21:13:02: Done! \\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'INFO  @ Fri, 13 Mar 2020 21:13:03: \\n# Command line: callpeak -B -t ../../data/results4/bwa/mergedLibrary/mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1.mLb.clN.sorted.bam -c ../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 207 -n mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1 --outdir ../../data/IRF2BP2_degraded_4/diffData/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/results4/bwa/mergedLibrary/mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Fri, 13 Mar 2020 21:13:03: #1 read tag files... \\nINFO  @ Fri, 13 Mar 2020 21:13:03: #1 read treatment tags... \\nINFO  @ Fri, 13 Mar 2020 21:13:08:  1000000 \\nINFO  @ Fri, 13 Mar 2020 21:13:14:  2000000 \\nINFO  @ Fri, 13 Mar 2020 21:13:20:  3000000 \\nINFO  @ Fri, 13 Mar 2020 21:13:26:  4000000 \\nINFO  @ Fri, 13 Mar 2020 21:13:31:  5000000 \\nINFO  @ Fri, 13 Mar 2020 21:13:35:  6000000 \\nINFO  @ Fri, 13 Mar 2020 21:13:40:  7000000 \\nINFO  @ Fri, 13 Mar 2020 21:13:44:  8000000 \\nINFO  @ Fri, 13 Mar 2020 21:13:49:  9000000 \\nINFO  @ Fri, 13 Mar 2020 21:13:54:  10000000 \\nINFO  @ Fri, 13 Mar 2020 21:13:59:  11000000 \\nINFO  @ Fri, 13 Mar 2020 21:14:04:  12000000 \\nINFO  @ Fri, 13 Mar 2020 21:14:10: 12927338 reads have been read. \\nINFO  @ Fri, 13 Mar 2020 21:14:10: #1.2 read input tags... \\nINFO  @ Fri, 13 Mar 2020 21:14:18:  1000000 \\nINFO  @ Fri, 13 Mar 2020 21:14:22:  2000000 \\nINFO  @ Fri, 13 Mar 2020 21:14:27:  3000000 \\nINFO  @ Fri, 13 Mar 2020 21:14:31:  4000000 \\nINFO  @ Fri, 13 Mar 2020 21:14:35:  5000000 \\nINFO  @ Fri, 13 Mar 2020 21:14:39:  6000000 \\nINFO  @ Fri, 13 Mar 2020 21:14:43:  7000000 \\nINFO  @ Fri, 13 Mar 2020 21:14:47:  8000000 \\nINFO  @ Fri, 13 Mar 2020 21:14:51:  9000000 \\nINFO  @ Fri, 13 Mar 2020 21:14:55:  10000000 \\nINFO  @ Fri, 13 Mar 2020 21:15:00:  11000000 \\nINFO  @ Fri, 13 Mar 2020 21:15:04:  12000000 \\nINFO  @ Fri, 13 Mar 2020 21:15:08:  13000000 \\nINFO  @ Fri, 13 Mar 2020 21:15:12:  14000000 \\nINFO  @ Fri, 13 Mar 2020 21:15:17:  15000000 \\nINFO  @ Fri, 13 Mar 2020 21:15:24:  16000000 \\nINFO  @ Fri, 13 Mar 2020 21:15:30:  17000000 \\nINFO  @ Fri, 13 Mar 2020 21:15:37:  18000000 \\nINFO  @ Fri, 13 Mar 2020 21:15:44:  19000000 \\nINFO  @ Fri, 13 Mar 2020 21:15:49:  20000000 \\nINFO  @ Fri, 13 Mar 2020 21:15:56:  21000000 \\nINFO  @ Fri, 13 Mar 2020 21:16:03:  22000000 \\nINFO  @ Fri, 13 Mar 2020 21:16:09:  23000000 \\nINFO  @ Fri, 13 Mar 2020 21:16:16:  24000000 \\nINFO  @ Fri, 13 Mar 2020 21:16:22:  25000000 \\nINFO  @ Fri, 13 Mar 2020 21:16:28:  26000000 \\nINFO  @ Fri, 13 Mar 2020 21:16:35:  27000000 \\nINFO  @ Fri, 13 Mar 2020 21:16:41:  28000000 \\nINFO  @ Fri, 13 Mar 2020 21:16:48:  29000000 \\nINFO  @ Fri, 13 Mar 2020 21:16:54:  30000000 \\nINFO  @ Fri, 13 Mar 2020 21:17:01:  31000000 \\nINFO  @ Fri, 13 Mar 2020 21:17:07:  32000000 \\nINFO  @ Fri, 13 Mar 2020 21:17:14:  33000000 \\nINFO  @ Fri, 13 Mar 2020 21:17:20:  34000000 \\nINFO  @ Fri, 13 Mar 2020 21:17:26:  35000000 \\nINFO  @ Fri, 13 Mar 2020 21:17:33:  36000000 \\nINFO  @ Fri, 13 Mar 2020 21:17:39:  37000000 \\nINFO  @ Fri, 13 Mar 2020 21:17:46:  38000000 \\nINFO  @ Fri, 13 Mar 2020 21:17:52:  39000000 \\nINFO  @ Fri, 13 Mar 2020 21:17:58:  40000000 \\nINFO  @ Fri, 13 Mar 2020 21:18:05:  41000000 \\nINFO  @ Fri, 13 Mar 2020 21:18:11:  42000000 \\nINFO  @ Fri, 13 Mar 2020 21:18:18:  43000000 \\nINFO  @ Fri, 13 Mar 2020 21:18:25:  44000000 \\nINFO  @ Fri, 13 Mar 2020 21:18:31:  45000000 \\nINFO  @ Fri, 13 Mar 2020 21:18:37:  46000000 \\nINFO  @ Fri, 13 Mar 2020 21:18:44:  47000000 \\nINFO  @ Fri, 13 Mar 2020 21:18:50:  48000000 \\nINFO  @ Fri, 13 Mar 2020 21:18:56:  49000000 \\nINFO  @ Fri, 13 Mar 2020 21:19:03:  50000000 \\nINFO  @ Fri, 13 Mar 2020 21:19:09:  51000000 \\nINFO  @ Fri, 13 Mar 2020 21:19:16:  52000000 \\nINFO  @ Fri, 13 Mar 2020 21:19:23:  53000000 \\nINFO  @ Fri, 13 Mar 2020 21:19:29:  54000000 \\nINFO  @ Fri, 13 Mar 2020 21:19:35:  55000000 \\nINFO  @ Fri, 13 Mar 2020 21:19:42:  56000000 \\nINFO  @ Fri, 13 Mar 2020 21:19:49:  57000000 \\nINFO  @ Fri, 13 Mar 2020 21:19:55:  58000000 \\nINFO  @ Fri, 13 Mar 2020 21:20:01:  59000000 \\nINFO  @ Fri, 13 Mar 2020 21:20:08:  60000000 \\nINFO  @ Fri, 13 Mar 2020 21:20:14:  61000000 \\nINFO  @ Fri, 13 Mar 2020 21:20:21:  62000000 \\nINFO  @ Fri, 13 Mar 2020 21:20:28:  63000000 \\nINFO  @ Fri, 13 Mar 2020 21:20:33:  64000000 \\nINFO  @ Fri, 13 Mar 2020 21:20:40:  65000000 \\nINFO  @ Fri, 13 Mar 2020 21:20:47:  66000000 \\nINFO  @ Fri, 13 Mar 2020 21:20:51: 66660428 reads have been read. \\nINFO  @ Fri, 13 Mar 2020 21:20:53: #1 tag size is determined as 49 bps \\nINFO  @ Fri, 13 Mar 2020 21:20:53: #1 tag size = 49.0 \\nINFO  @ Fri, 13 Mar 2020 21:20:53: #1  total tags in treatment: 12927338 \\nINFO  @ Fri, 13 Mar 2020 21:20:53: #1 user defined the maximum tags... \\nINFO  @ Fri, 13 Mar 2020 21:20:53: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Fri, 13 Mar 2020 21:20:53: #1  tags after filtering in treatment: 12851447 \\nINFO  @ Fri, 13 Mar 2020 21:20:53: #1  Redundant rate of treatment: 0.01 \\nINFO  @ Fri, 13 Mar 2020 21:20:53: #1  total tags in control: 66660428 \\nINFO  @ Fri, 13 Mar 2020 21:20:53: #1 user defined the maximum tags... \\nINFO  @ Fri, 13 Mar 2020 21:20:53: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Fri, 13 Mar 2020 21:20:53: #1  tags after filtering in control: 65724989 \\nINFO  @ Fri, 13 Mar 2020 21:20:53: #1  Redundant rate of control: 0.01 \\nINFO  @ Fri, 13 Mar 2020 21:20:53: #1 finished! \\nINFO  @ Fri, 13 Mar 2020 21:20:53: #2 Build Peak Model... \\nINFO  @ Fri, 13 Mar 2020 21:20:53: #2 Skipped... \\nINFO  @ Fri, 13 Mar 2020 21:20:53: #2 Use 207 as fragment length \\nINFO  @ Fri, 13 Mar 2020 21:20:53: #3 Call peaks... \\nINFO  @ Fri, 13 Mar 2020 21:20:53: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Fri, 13 Mar 2020 21:28:18: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Fri, 13 Mar 2020 21:28:18: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1_treat_pileup.bdg \\nINFO  @ Fri, 13 Mar 2020 21:28:18: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1_control_lambda.bdg \\nINFO  @ Fri, 13 Mar 2020 21:28:18: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Fri, 13 Mar 2020 21:28:18: #3 Call peaks for each chromosome... \\nINFO  @ Fri, 13 Mar 2020 21:34:13: #4 Write output xls file... ../../data/IRF2BP2_degraded_4/diffData/mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1_peaks.xls \\nINFO  @ Fri, 13 Mar 2020 21:34:13: #4 Write peak in narrowPeak format file... ../../data/IRF2BP2_degraded_4/diffData/mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1_peaks.narrowPeak \\nINFO  @ Fri, 13 Mar 2020 21:34:13: #4 Write summits bed file... ../../data/IRF2BP2_degraded_4/diffData/mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1_summits.bed \\nINFO  @ Fri, 13 Mar 2020 21:34:13: Done! \\n'\n",
      "17473890 12851447\n",
      "doing differential peak binding\n",
      "doing diff from ../../data/results4/bwa/mergedLibrary/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1.mLb.clN.sorted.bam and ../../data/results4/bwa/mergedLibrary/mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "234\n",
      "computing the scaling values\n",
      "b'INFO  @ Fri, 13 Mar 2020 21:52:38: \\n# Command line: callpeak -B -t ../../data/results4/bwa/mergedLibrary/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1.mLb.clN.sorted.bam -c ../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 234 -n mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1 --outdir ../../data/IRF2BP2_degraded_4/diffData/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/results4/bwa/mergedLibrary/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Fri, 13 Mar 2020 21:52:38: #1 read tag files... \\nINFO  @ Fri, 13 Mar 2020 21:52:38: #1 read treatment tags... \\nINFO  @ Fri, 13 Mar 2020 21:52:45:  1000000 \\nINFO  @ Fri, 13 Mar 2020 21:52:52:  2000000 \\nINFO  @ Fri, 13 Mar 2020 21:52:59:  3000000 \\nINFO  @ Fri, 13 Mar 2020 21:53:06:  4000000 \\nINFO  @ Fri, 13 Mar 2020 21:53:12:  5000000 \\nINFO  @ Fri, 13 Mar 2020 21:53:19:  6000000 \\nINFO  @ Fri, 13 Mar 2020 21:53:26:  7000000 \\nINFO  @ Fri, 13 Mar 2020 21:53:33:  8000000 \\nINFO  @ Fri, 13 Mar 2020 21:53:40:  9000000 \\nINFO  @ Fri, 13 Mar 2020 21:53:47:  10000000 \\nINFO  @ Fri, 13 Mar 2020 21:53:54:  11000000 \\nINFO  @ Fri, 13 Mar 2020 21:54:00:  12000000 \\nINFO  @ Fri, 13 Mar 2020 21:54:07:  13000000 \\nINFO  @ Fri, 13 Mar 2020 21:54:14:  14000000 \\nINFO  @ Fri, 13 Mar 2020 21:54:21:  15000000 \\nINFO  @ Fri, 13 Mar 2020 21:54:27:  16000000 \\nINFO  @ Fri, 13 Mar 2020 21:54:34: 16975163 reads have been read. \\nINFO  @ Fri, 13 Mar 2020 21:54:34: #1.2 read input tags... \\nINFO  @ Fri, 13 Mar 2020 21:54:41:  1000000 \\nINFO  @ Fri, 13 Mar 2020 21:54:48:  2000000 \\nINFO  @ Fri, 13 Mar 2020 21:54:54:  3000000 \\nINFO  @ Fri, 13 Mar 2020 21:55:01:  4000000 \\nINFO  @ Fri, 13 Mar 2020 21:55:07:  5000000 \\nINFO  @ Fri, 13 Mar 2020 21:55:14:  6000000 \\nINFO  @ Fri, 13 Mar 2020 21:55:20:  7000000 \\nINFO  @ Fri, 13 Mar 2020 21:55:27:  8000000 \\nINFO  @ Fri, 13 Mar 2020 21:55:33:  9000000 \\nINFO  @ Fri, 13 Mar 2020 21:55:40:  10000000 \\nINFO  @ Fri, 13 Mar 2020 21:55:47:  11000000 \\nINFO  @ Fri, 13 Mar 2020 21:55:53:  12000000 \\nINFO  @ Fri, 13 Mar 2020 21:56:00:  13000000 \\nINFO  @ Fri, 13 Mar 2020 21:56:07:  14000000 \\nINFO  @ Fri, 13 Mar 2020 21:56:13:  15000000 \\nINFO  @ Fri, 13 Mar 2020 21:56:20:  16000000 \\nINFO  @ Fri, 13 Mar 2020 21:56:26:  17000000 \\nINFO  @ Fri, 13 Mar 2020 21:56:33:  18000000 \\nINFO  @ Fri, 13 Mar 2020 21:56:40:  19000000 \\nINFO  @ Fri, 13 Mar 2020 21:56:46:  20000000 \\nINFO  @ Fri, 13 Mar 2020 21:56:53:  21000000 \\nINFO  @ Fri, 13 Mar 2020 21:57:00:  22000000 \\nINFO  @ Fri, 13 Mar 2020 21:57:06:  23000000 \\nINFO  @ Fri, 13 Mar 2020 21:57:13:  24000000 \\nINFO  @ Fri, 13 Mar 2020 21:57:20:  25000000 \\nINFO  @ Fri, 13 Mar 2020 21:57:27:  26000000 \\nINFO  @ Fri, 13 Mar 2020 21:57:33:  27000000 \\nINFO  @ Fri, 13 Mar 2020 21:57:40:  28000000 \\nINFO  @ Fri, 13 Mar 2020 21:57:46:  29000000 \\nINFO  @ Fri, 13 Mar 2020 21:57:53:  30000000 \\nINFO  @ Fri, 13 Mar 2020 21:58:00:  31000000 \\nINFO  @ Fri, 13 Mar 2020 21:58:06:  32000000 \\nINFO  @ Fri, 13 Mar 2020 21:58:13:  33000000 \\nINFO  @ Fri, 13 Mar 2020 21:58:19:  34000000 \\nINFO  @ Fri, 13 Mar 2020 21:58:26:  35000000 \\nINFO  @ Fri, 13 Mar 2020 21:58:32:  36000000 \\nINFO  @ Fri, 13 Mar 2020 21:58:39:  37000000 \\nINFO  @ Fri, 13 Mar 2020 21:58:45:  38000000 \\nINFO  @ Fri, 13 Mar 2020 21:58:57:  39000000 \\nINFO  @ Fri, 13 Mar 2020 21:59:07:  40000000 \\nINFO  @ Fri, 13 Mar 2020 21:59:14:  41000000 \\nINFO  @ Fri, 13 Mar 2020 21:59:20:  42000000 \\nINFO  @ Fri, 13 Mar 2020 21:59:27:  43000000 \\nINFO  @ Fri, 13 Mar 2020 21:59:34:  44000000 \\nINFO  @ Fri, 13 Mar 2020 21:59:40:  45000000 \\nINFO  @ Fri, 13 Mar 2020 21:59:47:  46000000 \\nINFO  @ Fri, 13 Mar 2020 21:59:54:  47000000 \\nINFO  @ Fri, 13 Mar 2020 22:00:00:  48000000 \\nINFO  @ Fri, 13 Mar 2020 22:00:07:  49000000 \\nINFO  @ Fri, 13 Mar 2020 22:00:13:  50000000 \\nINFO  @ Fri, 13 Mar 2020 22:00:20:  51000000 \\nINFO  @ Fri, 13 Mar 2020 22:00:27:  52000000 \\nINFO  @ Fri, 13 Mar 2020 22:00:34:  53000000 \\nINFO  @ Fri, 13 Mar 2020 22:00:40:  54000000 \\nINFO  @ Fri, 13 Mar 2020 22:00:47:  55000000 \\nINFO  @ Fri, 13 Mar 2020 22:00:54:  56000000 \\nINFO  @ Fri, 13 Mar 2020 22:01:01:  57000000 \\nINFO  @ Fri, 13 Mar 2020 22:01:07:  58000000 \\nINFO  @ Fri, 13 Mar 2020 22:01:14:  59000000 \\nINFO  @ Fri, 13 Mar 2020 22:01:20:  60000000 \\nINFO  @ Fri, 13 Mar 2020 22:01:27:  61000000 \\nINFO  @ Fri, 13 Mar 2020 22:01:33:  62000000 \\nINFO  @ Fri, 13 Mar 2020 22:01:40:  63000000 \\nINFO  @ Fri, 13 Mar 2020 22:01:47:  64000000 \\nINFO  @ Fri, 13 Mar 2020 22:01:53:  65000000 \\nINFO  @ Fri, 13 Mar 2020 22:02:00:  66000000 \\nINFO  @ Fri, 13 Mar 2020 22:02:05: 66660428 reads have been read. \\nINFO  @ Fri, 13 Mar 2020 22:02:06: #1 tag size is determined as 49 bps \\nINFO  @ Fri, 13 Mar 2020 22:02:06: #1 tag size = 49.0 \\nINFO  @ Fri, 13 Mar 2020 22:02:06: #1  total tags in treatment: 16975163 \\nINFO  @ Fri, 13 Mar 2020 22:02:06: #1 user defined the maximum tags... \\nINFO  @ Fri, 13 Mar 2020 22:02:06: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Fri, 13 Mar 2020 22:02:06: #1  tags after filtering in treatment: 16547878 \\nINFO  @ Fri, 13 Mar 2020 22:02:06: #1  Redundant rate of treatment: 0.03 \\nINFO  @ Fri, 13 Mar 2020 22:02:06: #1  total tags in control: 66660428 \\nINFO  @ Fri, 13 Mar 2020 22:02:06: #1 user defined the maximum tags... \\nINFO  @ Fri, 13 Mar 2020 22:02:06: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Fri, 13 Mar 2020 22:02:06: #1  tags after filtering in control: 65724989 \\nINFO  @ Fri, 13 Mar 2020 22:02:06: #1  Redundant rate of control: 0.01 \\nINFO  @ Fri, 13 Mar 2020 22:02:06: #1 finished! \\nINFO  @ Fri, 13 Mar 2020 22:02:06: #2 Build Peak Model... \\nINFO  @ Fri, 13 Mar 2020 22:02:06: #2 Skipped... \\nINFO  @ Fri, 13 Mar 2020 22:02:06: #2 Use 234 as fragment length \\nINFO  @ Fri, 13 Mar 2020 22:02:06: #3 Call peaks... \\nINFO  @ Fri, 13 Mar 2020 22:02:06: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Fri, 13 Mar 2020 22:10:00: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Fri, 13 Mar 2020 22:10:00: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1_treat_pileup.bdg \\nINFO  @ Fri, 13 Mar 2020 22:10:00: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1_control_lambda.bdg \\nINFO  @ Fri, 13 Mar 2020 22:10:00: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Fri, 13 Mar 2020 22:10:00: #3 Call peaks for each chromosome... \\nINFO  @ Fri, 13 Mar 2020 22:15:16: #4 Write output xls file... ../../data/IRF2BP2_degraded_4/diffData/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1_peaks.xls \\nINFO  @ Fri, 13 Mar 2020 22:15:17: #4 Write peak in narrowPeak format file... ../../data/IRF2BP2_degraded_4/diffData/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1_peaks.narrowPeak \\nINFO  @ Fri, 13 Mar 2020 22:15:18: #4 Write summits bed file... ../../data/IRF2BP2_degraded_4/diffData/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1_summits.bed \\nINFO  @ Fri, 13 Mar 2020 22:15:18: Done! \\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'INFO  @ Fri, 13 Mar 2020 22:15:19: \\n# Command line: callpeak -B -t ../../data/results4/bwa/mergedLibrary/mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1.mLb.clN.sorted.bam -c ../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 234 -n mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1 --outdir ../../data/IRF2BP2_degraded_4/diffData/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/results4/bwa/mergedLibrary/mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Fri, 13 Mar 2020 22:15:19: #1 read tag files... \\nINFO  @ Fri, 13 Mar 2020 22:15:19: #1 read treatment tags... \\nINFO  @ Fri, 13 Mar 2020 22:15:25:  1000000 \\nINFO  @ Fri, 13 Mar 2020 22:15:31:  2000000 \\nINFO  @ Fri, 13 Mar 2020 22:15:37:  3000000 \\nINFO  @ Fri, 13 Mar 2020 22:15:42:  4000000 \\nINFO  @ Fri, 13 Mar 2020 22:15:48:  5000000 \\nINFO  @ Fri, 13 Mar 2020 22:15:53:  6000000 \\nINFO  @ Fri, 13 Mar 2020 22:16:00:  7000000 \\nINFO  @ Fri, 13 Mar 2020 22:16:05:  8000000 \\nINFO  @ Fri, 13 Mar 2020 22:16:10:  9000000 \\nINFO  @ Fri, 13 Mar 2020 22:16:15:  10000000 \\nINFO  @ Fri, 13 Mar 2020 22:16:20:  11000000 \\nINFO  @ Fri, 13 Mar 2020 22:16:24:  12000000 \\nINFO  @ Fri, 13 Mar 2020 22:16:30:  13000000 \\nINFO  @ Fri, 13 Mar 2020 22:16:34:  14000000 \\nINFO  @ Fri, 13 Mar 2020 22:16:39:  15000000 \\nINFO  @ Fri, 13 Mar 2020 22:16:43:  16000000 \\nINFO  @ Fri, 13 Mar 2020 22:16:47:  17000000 \\nINFO  @ Fri, 13 Mar 2020 22:16:52:  18000000 \\nINFO  @ Fri, 13 Mar 2020 22:16:57:  19000000 \\nINFO  @ Fri, 13 Mar 2020 22:17:02:  20000000 \\nINFO  @ Fri, 13 Mar 2020 22:17:04: 20453626 reads have been read. \\nINFO  @ Fri, 13 Mar 2020 22:17:04: #1.2 read input tags... \\nINFO  @ Fri, 13 Mar 2020 22:17:09:  1000000 \\nINFO  @ Fri, 13 Mar 2020 22:17:13:  2000000 \\nINFO  @ Fri, 13 Mar 2020 22:17:17:  3000000 \\nINFO  @ Fri, 13 Mar 2020 22:17:21:  4000000 \\nINFO  @ Fri, 13 Mar 2020 22:17:25:  5000000 \\nINFO  @ Fri, 13 Mar 2020 22:17:29:  6000000 \\nINFO  @ Fri, 13 Mar 2020 22:17:33:  7000000 \\nINFO  @ Fri, 13 Mar 2020 22:17:37:  8000000 \\nINFO  @ Fri, 13 Mar 2020 22:17:42:  9000000 \\nINFO  @ Fri, 13 Mar 2020 22:17:46:  10000000 \\nINFO  @ Fri, 13 Mar 2020 22:17:50:  11000000 \\nINFO  @ Fri, 13 Mar 2020 22:17:54:  12000000 \\nINFO  @ Fri, 13 Mar 2020 22:17:58:  13000000 \\nINFO  @ Fri, 13 Mar 2020 22:18:03:  14000000 \\nINFO  @ Fri, 13 Mar 2020 22:18:07:  15000000 \\nINFO  @ Fri, 13 Mar 2020 22:18:11:  16000000 \\nINFO  @ Fri, 13 Mar 2020 22:18:15:  17000000 \\nINFO  @ Fri, 13 Mar 2020 22:18:20:  18000000 \\nINFO  @ Fri, 13 Mar 2020 22:18:24:  19000000 \\nINFO  @ Fri, 13 Mar 2020 22:18:28:  20000000 \\nINFO  @ Fri, 13 Mar 2020 22:18:32:  21000000 \\nINFO  @ Fri, 13 Mar 2020 22:18:37:  22000000 \\nINFO  @ Fri, 13 Mar 2020 22:18:41:  23000000 \\nINFO  @ Fri, 13 Mar 2020 22:18:46:  24000000 \\nINFO  @ Fri, 13 Mar 2020 22:18:50:  25000000 \\nINFO  @ Fri, 13 Mar 2020 22:18:54:  26000000 \\nINFO  @ Fri, 13 Mar 2020 22:18:59:  27000000 \\nINFO  @ Fri, 13 Mar 2020 22:19:03:  28000000 \\nINFO  @ Fri, 13 Mar 2020 22:19:07:  29000000 \\nINFO  @ Fri, 13 Mar 2020 22:19:12:  30000000 \\nINFO  @ Fri, 13 Mar 2020 22:19:17:  31000000 \\nINFO  @ Fri, 13 Mar 2020 22:19:21:  32000000 \\nINFO  @ Fri, 13 Mar 2020 22:19:25:  33000000 \\nINFO  @ Fri, 13 Mar 2020 22:19:29:  34000000 \\nINFO  @ Fri, 13 Mar 2020 22:19:33:  35000000 \\nINFO  @ Fri, 13 Mar 2020 22:19:38:  36000000 \\nINFO  @ Fri, 13 Mar 2020 22:19:42:  37000000 \\nINFO  @ Fri, 13 Mar 2020 22:19:46:  38000000 \\nINFO  @ Fri, 13 Mar 2020 22:19:50:  39000000 \\nINFO  @ Fri, 13 Mar 2020 22:19:54:  40000000 \\nINFO  @ Fri, 13 Mar 2020 22:19:59:  41000000 \\nINFO  @ Fri, 13 Mar 2020 22:20:03:  42000000 \\nINFO  @ Fri, 13 Mar 2020 22:20:07:  43000000 \\nINFO  @ Fri, 13 Mar 2020 22:20:11:  44000000 \\nINFO  @ Fri, 13 Mar 2020 22:20:15:  45000000 \\nINFO  @ Fri, 13 Mar 2020 22:20:20:  46000000 \\nINFO  @ Fri, 13 Mar 2020 22:20:24:  47000000 \\nINFO  @ Fri, 13 Mar 2020 22:20:28:  48000000 \\nINFO  @ Fri, 13 Mar 2020 22:20:32:  49000000 \\nINFO  @ Fri, 13 Mar 2020 22:20:36:  50000000 \\nINFO  @ Fri, 13 Mar 2020 22:20:40:  51000000 \\nINFO  @ Fri, 13 Mar 2020 22:20:45:  52000000 \\nINFO  @ Fri, 13 Mar 2020 22:20:49:  53000000 \\nINFO  @ Fri, 13 Mar 2020 22:20:53:  54000000 \\nINFO  @ Fri, 13 Mar 2020 22:20:58:  55000000 \\nINFO  @ Fri, 13 Mar 2020 22:21:02:  56000000 \\nINFO  @ Fri, 13 Mar 2020 22:21:06:  57000000 \\nINFO  @ Fri, 13 Mar 2020 22:21:10:  58000000 \\nINFO  @ Fri, 13 Mar 2020 22:21:14:  59000000 \\nINFO  @ Fri, 13 Mar 2020 22:21:19:  60000000 \\nINFO  @ Fri, 13 Mar 2020 22:21:23:  61000000 \\nINFO  @ Fri, 13 Mar 2020 22:21:27:  62000000 \\nINFO  @ Fri, 13 Mar 2020 22:21:32:  63000000 \\nINFO  @ Fri, 13 Mar 2020 22:21:36:  64000000 \\nINFO  @ Fri, 13 Mar 2020 22:21:40:  65000000 \\nINFO  @ Fri, 13 Mar 2020 22:21:44:  66000000 \\nINFO  @ Fri, 13 Mar 2020 22:21:47: 66660428 reads have been read. \\nINFO  @ Fri, 13 Mar 2020 22:21:48: #1 tag size is determined as 49 bps \\nINFO  @ Fri, 13 Mar 2020 22:21:48: #1 tag size = 49.0 \\nINFO  @ Fri, 13 Mar 2020 22:21:48: #1  total tags in treatment: 20453626 \\nINFO  @ Fri, 13 Mar 2020 22:21:48: #1 user defined the maximum tags... \\nINFO  @ Fri, 13 Mar 2020 22:21:48: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Fri, 13 Mar 2020 22:21:48: #1  tags after filtering in treatment: 19792399 \\nINFO  @ Fri, 13 Mar 2020 22:21:48: #1  Redundant rate of treatment: 0.03 \\nINFO  @ Fri, 13 Mar 2020 22:21:48: #1  total tags in control: 66660428 \\nINFO  @ Fri, 13 Mar 2020 22:21:48: #1 user defined the maximum tags... \\nINFO  @ Fri, 13 Mar 2020 22:21:48: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Fri, 13 Mar 2020 22:21:48: #1  tags after filtering in control: 65724989 \\nINFO  @ Fri, 13 Mar 2020 22:21:48: #1  Redundant rate of control: 0.01 \\nINFO  @ Fri, 13 Mar 2020 22:21:48: #1 finished! \\nINFO  @ Fri, 13 Mar 2020 22:21:48: #2 Build Peak Model... \\nINFO  @ Fri, 13 Mar 2020 22:21:48: #2 Skipped... \\nINFO  @ Fri, 13 Mar 2020 22:21:48: #2 Use 234 as fragment length \\nINFO  @ Fri, 13 Mar 2020 22:21:48: #3 Call peaks... \\nINFO  @ Fri, 13 Mar 2020 22:21:48: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Fri, 13 Mar 2020 22:27:03: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Fri, 13 Mar 2020 22:27:03: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1_treat_pileup.bdg \\nINFO  @ Fri, 13 Mar 2020 22:27:03: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1_control_lambda.bdg \\nINFO  @ Fri, 13 Mar 2020 22:27:03: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Fri, 13 Mar 2020 22:27:03: #3 Call peaks for each chromosome... \\nINFO  @ Fri, 13 Mar 2020 22:33:31: #4 Write output xls file... ../../data/IRF2BP2_degraded_4/diffData/mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1_peaks.xls \\nINFO  @ Fri, 13 Mar 2020 22:33:32: #4 Write peak in narrowPeak format file... ../../data/IRF2BP2_degraded_4/diffData/mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1_peaks.narrowPeak \\nINFO  @ Fri, 13 Mar 2020 22:33:32: #4 Write summits bed file... ../../data/IRF2BP2_degraded_4/diffData/mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1_summits.bed \\nINFO  @ Fri, 13 Mar 2020 22:33:33: Done! \\n'\n",
      "16547878 19792399\n",
      "doing differential peak binding\n",
      "doing diff from ../../data/results4/bwa/mergedLibrary/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1.mLb.clN.sorted.bam and ../../data/results4/bwa/mergedLibrary/mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "296\n",
      "computing the scaling values\n",
      "b'INFO  @ Fri, 13 Mar 2020 22:52:53: \\n# Command line: callpeak -B -t ../../data/results4/bwa/mergedLibrary/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1.mLb.clN.sorted.bam -c ../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 296 -n mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1 --outdir ../../data/IRF2BP2_degraded_4/diffData/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/results4/bwa/mergedLibrary/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Fri, 13 Mar 2020 22:52:53: #1 read tag files... \\nINFO  @ Fri, 13 Mar 2020 22:52:53: #1 read treatment tags... \\nINFO  @ Fri, 13 Mar 2020 22:53:00:  1000000 \\nINFO  @ Fri, 13 Mar 2020 22:53:09:  2000000 \\nINFO  @ Fri, 13 Mar 2020 22:53:16:  3000000 \\nINFO  @ Fri, 13 Mar 2020 22:53:23:  4000000 \\nINFO  @ Fri, 13 Mar 2020 22:53:30:  5000000 \\nINFO  @ Fri, 13 Mar 2020 22:53:37:  6000000 \\nINFO  @ Fri, 13 Mar 2020 22:53:44:  7000000 \\nINFO  @ Fri, 13 Mar 2020 22:53:51:  8000000 \\nINFO  @ Fri, 13 Mar 2020 22:53:58:  9000000 \\nINFO  @ Fri, 13 Mar 2020 22:54:05:  10000000 \\nINFO  @ Fri, 13 Mar 2020 22:54:12:  11000000 \\nINFO  @ Fri, 13 Mar 2020 22:54:19:  12000000 \\nINFO  @ Fri, 13 Mar 2020 22:54:20: 12038784 reads have been read. \\nINFO  @ Fri, 13 Mar 2020 22:54:20: #1.2 read input tags... \\nINFO  @ Fri, 13 Mar 2020 22:54:27:  1000000 \\nINFO  @ Fri, 13 Mar 2020 22:54:33:  2000000 \\nINFO  @ Fri, 13 Mar 2020 22:54:40:  3000000 \\nINFO  @ Fri, 13 Mar 2020 22:54:47:  4000000 \\nINFO  @ Fri, 13 Mar 2020 22:54:54:  5000000 \\nINFO  @ Fri, 13 Mar 2020 22:55:00:  6000000 \\nINFO  @ Fri, 13 Mar 2020 22:55:07:  7000000 \\nINFO  @ Fri, 13 Mar 2020 22:55:14:  8000000 \\nINFO  @ Fri, 13 Mar 2020 22:55:20:  9000000 \\nINFO  @ Fri, 13 Mar 2020 22:55:27:  10000000 \\nINFO  @ Fri, 13 Mar 2020 22:55:34:  11000000 \\nINFO  @ Fri, 13 Mar 2020 22:55:40:  12000000 \\nINFO  @ Fri, 13 Mar 2020 22:55:47:  13000000 \\nINFO  @ Fri, 13 Mar 2020 22:55:54:  14000000 \\nINFO  @ Fri, 13 Mar 2020 22:56:01:  15000000 \\nINFO  @ Fri, 13 Mar 2020 22:56:07:  16000000 \\nINFO  @ Fri, 13 Mar 2020 22:56:14:  17000000 \\nINFO  @ Fri, 13 Mar 2020 22:56:21:  18000000 \\nINFO  @ Fri, 13 Mar 2020 22:56:28:  19000000 \\nINFO  @ Fri, 13 Mar 2020 22:56:34:  20000000 \\nINFO  @ Fri, 13 Mar 2020 22:56:41:  21000000 \\nINFO  @ Fri, 13 Mar 2020 22:56:48:  22000000 \\nINFO  @ Fri, 13 Mar 2020 22:56:55:  23000000 \\nINFO  @ Fri, 13 Mar 2020 22:57:01:  24000000 \\nINFO  @ Fri, 13 Mar 2020 22:57:08:  25000000 \\nINFO  @ Fri, 13 Mar 2020 22:57:15:  26000000 \\nINFO  @ Fri, 13 Mar 2020 22:57:21:  27000000 \\nINFO  @ Fri, 13 Mar 2020 22:57:28:  28000000 \\nINFO  @ Fri, 13 Mar 2020 22:57:35:  29000000 \\nINFO  @ Fri, 13 Mar 2020 22:57:41:  30000000 \\nINFO  @ Fri, 13 Mar 2020 22:57:48:  31000000 \\nINFO  @ Fri, 13 Mar 2020 22:57:55:  32000000 \\nINFO  @ Fri, 13 Mar 2020 22:58:02:  33000000 \\nINFO  @ Fri, 13 Mar 2020 22:58:08:  34000000 \\nINFO  @ Fri, 13 Mar 2020 22:58:15:  35000000 \\nINFO  @ Fri, 13 Mar 2020 22:58:22:  36000000 \\nINFO  @ Fri, 13 Mar 2020 22:58:29:  37000000 \\nINFO  @ Fri, 13 Mar 2020 22:58:35:  38000000 \\nINFO  @ Fri, 13 Mar 2020 22:58:42:  39000000 \\nINFO  @ Fri, 13 Mar 2020 22:58:49:  40000000 \\nINFO  @ Fri, 13 Mar 2020 22:58:56:  41000000 \\nINFO  @ Fri, 13 Mar 2020 22:59:02:  42000000 \\nINFO  @ Fri, 13 Mar 2020 22:59:09:  43000000 \\nINFO  @ Fri, 13 Mar 2020 22:59:16:  44000000 \\nINFO  @ Fri, 13 Mar 2020 22:59:23:  45000000 \\nINFO  @ Fri, 13 Mar 2020 22:59:29:  46000000 \\nINFO  @ Fri, 13 Mar 2020 22:59:36:  47000000 \\nINFO  @ Fri, 13 Mar 2020 22:59:43:  48000000 \\nINFO  @ Fri, 13 Mar 2020 22:59:49:  49000000 \\nINFO  @ Fri, 13 Mar 2020 22:59:56:  50000000 \\nINFO  @ Fri, 13 Mar 2020 23:00:03:  51000000 \\nINFO  @ Fri, 13 Mar 2020 23:00:10:  52000000 \\nINFO  @ Fri, 13 Mar 2020 23:00:17:  53000000 \\nINFO  @ Fri, 13 Mar 2020 23:00:23:  54000000 \\nINFO  @ Fri, 13 Mar 2020 23:00:30:  55000000 \\nINFO  @ Fri, 13 Mar 2020 23:00:37:  56000000 \\nINFO  @ Fri, 13 Mar 2020 23:00:44:  57000000 \\nINFO  @ Fri, 13 Mar 2020 23:00:51:  58000000 \\nINFO  @ Fri, 13 Mar 2020 23:00:57:  59000000 \\nINFO  @ Fri, 13 Mar 2020 23:01:04:  60000000 \\nINFO  @ Fri, 13 Mar 2020 23:01:11:  61000000 \\nINFO  @ Fri, 13 Mar 2020 23:01:18:  62000000 \\nINFO  @ Fri, 13 Mar 2020 23:01:25:  63000000 \\nINFO  @ Fri, 13 Mar 2020 23:01:32:  64000000 \\nINFO  @ Fri, 13 Mar 2020 23:01:38:  65000000 \\nINFO  @ Fri, 13 Mar 2020 23:01:45:  66000000 \\nINFO  @ Fri, 13 Mar 2020 23:01:50: 66660428 reads have been read. \\nINFO  @ Fri, 13 Mar 2020 23:01:51: #1 tag size is determined as 49 bps \\nINFO  @ Fri, 13 Mar 2020 23:01:51: #1 tag size = 49.0 \\nINFO  @ Fri, 13 Mar 2020 23:01:51: #1  total tags in treatment: 12038784 \\nINFO  @ Fri, 13 Mar 2020 23:01:51: #1 user defined the maximum tags... \\nINFO  @ Fri, 13 Mar 2020 23:01:51: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Fri, 13 Mar 2020 23:01:51: #1  tags after filtering in treatment: 11920080 \\nINFO  @ Fri, 13 Mar 2020 23:01:51: #1  Redundant rate of treatment: 0.01 \\nINFO  @ Fri, 13 Mar 2020 23:01:51: #1  total tags in control: 66660428 \\nINFO  @ Fri, 13 Mar 2020 23:01:51: #1 user defined the maximum tags... \\nINFO  @ Fri, 13 Mar 2020 23:01:51: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Fri, 13 Mar 2020 23:01:52: #1  tags after filtering in control: 65724989 \\nINFO  @ Fri, 13 Mar 2020 23:01:52: #1  Redundant rate of control: 0.01 \\nINFO  @ Fri, 13 Mar 2020 23:01:52: #1 finished! \\nINFO  @ Fri, 13 Mar 2020 23:01:52: #2 Build Peak Model... \\nINFO  @ Fri, 13 Mar 2020 23:01:52: #2 Skipped... \\nINFO  @ Fri, 13 Mar 2020 23:01:52: #2 Use 296 as fragment length \\nINFO  @ Fri, 13 Mar 2020 23:01:52: #3 Call peaks... \\nINFO  @ Fri, 13 Mar 2020 23:01:52: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Fri, 13 Mar 2020 23:10:00: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Fri, 13 Mar 2020 23:10:00: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1_treat_pileup.bdg \\nINFO  @ Fri, 13 Mar 2020 23:10:00: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1_control_lambda.bdg \\nINFO  @ Fri, 13 Mar 2020 23:10:00: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Fri, 13 Mar 2020 23:10:00: #3 Call peaks for each chromosome... \\nINFO  @ Fri, 13 Mar 2020 23:16:19: #4 Write output xls file... ../../data/IRF2BP2_degraded_4/diffData/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1_peaks.xls \\nINFO  @ Fri, 13 Mar 2020 23:16:20: #4 Write peak in narrowPeak format file... ../../data/IRF2BP2_degraded_4/diffData/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1_peaks.narrowPeak \\nINFO  @ Fri, 13 Mar 2020 23:16:20: #4 Write summits bed file... ../../data/IRF2BP2_degraded_4/diffData/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1_summits.bed \\nINFO  @ Fri, 13 Mar 2020 23:16:20: Done! \\n'\n",
      "b'INFO  @ Fri, 13 Mar 2020 23:16:21: \\n# Command line: callpeak -B -t ../../data/results4/bwa/mergedLibrary/mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1.mLb.clN.sorted.bam -c ../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 296 -n mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1 --outdir ../../data/IRF2BP2_degraded_4/diffData/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/results4/bwa/mergedLibrary/mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Fri, 13 Mar 2020 23:16:21: #1 read tag files... \\nINFO  @ Fri, 13 Mar 2020 23:16:21: #1 read treatment tags... \\nINFO  @ Fri, 13 Mar 2020 23:16:29:  1000000 \\nINFO  @ Fri, 13 Mar 2020 23:16:38:  2000000 \\nINFO  @ Fri, 13 Mar 2020 23:16:46:  3000000 \\nINFO  @ Fri, 13 Mar 2020 23:16:54:  4000000 \\nINFO  @ Fri, 13 Mar 2020 23:17:02:  5000000 \\nINFO  @ Fri, 13 Mar 2020 23:17:10:  6000000 \\nINFO  @ Fri, 13 Mar 2020 23:17:18:  7000000 \\nINFO  @ Fri, 13 Mar 2020 23:17:25:  8000000 \\nINFO  @ Fri, 13 Mar 2020 23:17:34:  9000000 \\nINFO  @ Fri, 13 Mar 2020 23:17:42:  10000000 \\nINFO  @ Fri, 13 Mar 2020 23:17:50:  11000000 \\nINFO  @ Fri, 13 Mar 2020 23:17:57:  12000000 \\nINFO  @ Fri, 13 Mar 2020 23:17:58: 12039202 reads have been read. \\nINFO  @ Fri, 13 Mar 2020 23:17:58: #1.2 read input tags... \\nINFO  @ Fri, 13 Mar 2020 23:18:05:  1000000 \\nINFO  @ Fri, 13 Mar 2020 23:18:12:  2000000 \\nINFO  @ Fri, 13 Mar 2020 23:18:19:  3000000 \\nINFO  @ Fri, 13 Mar 2020 23:18:25:  4000000 \\nINFO  @ Fri, 13 Mar 2020 23:18:32:  5000000 \\nINFO  @ Fri, 13 Mar 2020 23:18:39:  6000000 \\nINFO  @ Fri, 13 Mar 2020 23:18:46:  7000000 \\nINFO  @ Fri, 13 Mar 2020 23:18:52:  8000000 \\nINFO  @ Fri, 13 Mar 2020 23:18:59:  9000000 \\nINFO  @ Fri, 13 Mar 2020 23:19:06:  10000000 \\nINFO  @ Fri, 13 Mar 2020 23:19:13:  11000000 \\nINFO  @ Fri, 13 Mar 2020 23:19:20:  12000000 \\nINFO  @ Fri, 13 Mar 2020 23:19:26:  13000000 \\nINFO  @ Fri, 13 Mar 2020 23:19:33:  14000000 \\nINFO  @ Fri, 13 Mar 2020 23:19:40:  15000000 \\nINFO  @ Fri, 13 Mar 2020 23:19:47:  16000000 \\nINFO  @ Fri, 13 Mar 2020 23:19:53:  17000000 \\nINFO  @ Fri, 13 Mar 2020 23:20:00:  18000000 \\nINFO  @ Fri, 13 Mar 2020 23:20:07:  19000000 \\nINFO  @ Fri, 13 Mar 2020 23:20:14:  20000000 \\nINFO  @ Fri, 13 Mar 2020 23:20:21:  21000000 \\nINFO  @ Fri, 13 Mar 2020 23:20:28:  22000000 \\nINFO  @ Fri, 13 Mar 2020 23:20:35:  23000000 \\nINFO  @ Fri, 13 Mar 2020 23:20:41:  24000000 \\nINFO  @ Fri, 13 Mar 2020 23:20:48:  25000000 \\nINFO  @ Fri, 13 Mar 2020 23:20:55:  26000000 \\nINFO  @ Fri, 13 Mar 2020 23:21:02:  27000000 \\nINFO  @ Fri, 13 Mar 2020 23:21:08:  28000000 \\nINFO  @ Fri, 13 Mar 2020 23:21:15:  29000000 \\nINFO  @ Fri, 13 Mar 2020 23:21:21:  30000000 \\nINFO  @ Fri, 13 Mar 2020 23:21:28:  31000000 \\nINFO  @ Fri, 13 Mar 2020 23:21:35:  32000000 \\nINFO  @ Fri, 13 Mar 2020 23:21:42:  33000000 \\nINFO  @ Fri, 13 Mar 2020 23:21:48:  34000000 \\nINFO  @ Fri, 13 Mar 2020 23:21:55:  35000000 \\nINFO  @ Fri, 13 Mar 2020 23:22:02:  36000000 \\nINFO  @ Fri, 13 Mar 2020 23:22:08:  37000000 \\nINFO  @ Fri, 13 Mar 2020 23:22:15:  38000000 \\nINFO  @ Fri, 13 Mar 2020 23:22:22:  39000000 \\nINFO  @ Fri, 13 Mar 2020 23:22:28:  40000000 \\nINFO  @ Fri, 13 Mar 2020 23:22:35:  41000000 \\nINFO  @ Fri, 13 Mar 2020 23:22:42:  42000000 \\nINFO  @ Fri, 13 Mar 2020 23:22:49:  43000000 \\nINFO  @ Fri, 13 Mar 2020 23:22:56:  44000000 \\nINFO  @ Fri, 13 Mar 2020 23:23:02:  45000000 \\nINFO  @ Fri, 13 Mar 2020 23:23:09:  46000000 \\nINFO  @ Fri, 13 Mar 2020 23:23:16:  47000000 \\nINFO  @ Fri, 13 Mar 2020 23:23:23:  48000000 \\nINFO  @ Fri, 13 Mar 2020 23:23:29:  49000000 \\nINFO  @ Fri, 13 Mar 2020 23:23:37:  50000000 \\nINFO  @ Fri, 13 Mar 2020 23:23:43:  51000000 \\nINFO  @ Fri, 13 Mar 2020 23:23:50:  52000000 \\nINFO  @ Fri, 13 Mar 2020 23:23:57:  53000000 \\nINFO  @ Fri, 13 Mar 2020 23:24:03:  54000000 \\nINFO  @ Fri, 13 Mar 2020 23:24:10:  55000000 \\nINFO  @ Fri, 13 Mar 2020 23:24:17:  56000000 \\nINFO  @ Fri, 13 Mar 2020 23:24:23:  57000000 \\nINFO  @ Fri, 13 Mar 2020 23:24:33:  58000000 \\nINFO  @ Fri, 13 Mar 2020 23:24:46:  59000000 \\nINFO  @ Fri, 13 Mar 2020 23:24:59:  60000000 \\nINFO  @ Fri, 13 Mar 2020 23:25:05:  61000000 \\nINFO  @ Fri, 13 Mar 2020 23:25:12:  62000000 \\nINFO  @ Fri, 13 Mar 2020 23:25:18:  63000000 \\nINFO  @ Fri, 13 Mar 2020 23:25:25:  64000000 \\nINFO  @ Fri, 13 Mar 2020 23:25:32:  65000000 \\nINFO  @ Fri, 13 Mar 2020 23:25:38:  66000000 \\nINFO  @ Fri, 13 Mar 2020 23:25:43: 66660428 reads have been read. \\nINFO  @ Fri, 13 Mar 2020 23:25:44: #1 tag size is determined as 49 bps \\nINFO  @ Fri, 13 Mar 2020 23:25:44: #1 tag size = 49.0 \\nINFO  @ Fri, 13 Mar 2020 23:25:44: #1  total tags in treatment: 12039202 \\nINFO  @ Fri, 13 Mar 2020 23:25:44: #1 user defined the maximum tags... \\nINFO  @ Fri, 13 Mar 2020 23:25:44: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Fri, 13 Mar 2020 23:25:44: #1  tags after filtering in treatment: 11866599 \\nINFO  @ Fri, 13 Mar 2020 23:25:44: #1  Redundant rate of treatment: 0.01 \\nINFO  @ Fri, 13 Mar 2020 23:25:44: #1  total tags in control: 66660428 \\nINFO  @ Fri, 13 Mar 2020 23:25:44: #1 user defined the maximum tags... \\nINFO  @ Fri, 13 Mar 2020 23:25:44: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Fri, 13 Mar 2020 23:25:44: #1  tags after filtering in control: 65724989 \\nINFO  @ Fri, 13 Mar 2020 23:25:44: #1  Redundant rate of control: 0.01 \\nINFO  @ Fri, 13 Mar 2020 23:25:44: #1 finished! \\nINFO  @ Fri, 13 Mar 2020 23:25:44: #2 Build Peak Model... \\nINFO  @ Fri, 13 Mar 2020 23:25:44: #2 Skipped... \\nINFO  @ Fri, 13 Mar 2020 23:25:44: #2 Use 296 as fragment length \\nINFO  @ Fri, 13 Mar 2020 23:25:44: #3 Call peaks... \\nINFO  @ Fri, 13 Mar 2020 23:25:44: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Fri, 13 Mar 2020 23:31:07: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Fri, 13 Mar 2020 23:31:07: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1_treat_pileup.bdg \\nINFO  @ Fri, 13 Mar 2020 23:31:07: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1_control_lambda.bdg \\nINFO  @ Fri, 13 Mar 2020 23:31:07: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Fri, 13 Mar 2020 23:31:07: #3 Call peaks for each chromosome... \\nINFO  @ Fri, 13 Mar 2020 23:35:37: #4 Write output xls file... ../../data/IRF2BP2_degraded_4/diffData/mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1_peaks.xls \\nINFO  @ Fri, 13 Mar 2020 23:35:37: #4 Write peak in narrowPeak format file... ../../data/IRF2BP2_degraded_4/diffData/mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1_peaks.narrowPeak \\nINFO  @ Fri, 13 Mar 2020 23:35:38: #4 Write summits bed file... ../../data/IRF2BP2_degraded_4/diffData/mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1_summits.bed \\nINFO  @ Fri, 13 Mar 2020 23:35:38: Done! \\n'\n",
      "11920080 11866599\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "doing diff from ../../data/results4/bwa/mergedLibrary/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1.mLb.clN.sorted.bam and ../../data/results4/bwa/mergedLibrary/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "231\n",
      "computing the scaling values\n",
      "b'INFO  @ Fri, 13 Mar 2020 23:52:47: \\n# Command line: callpeak -B -t ../../data/results4/bwa/mergedLibrary/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1.mLb.clN.sorted.bam -c ../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 231 -n mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1 --outdir ../../data/IRF2BP2_degraded_4/diffData/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/results4/bwa/mergedLibrary/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Fri, 13 Mar 2020 23:52:47: #1 read tag files... \\nINFO  @ Fri, 13 Mar 2020 23:52:47: #1 read treatment tags... \\nINFO  @ Fri, 13 Mar 2020 23:52:54:  1000000 \\nINFO  @ Fri, 13 Mar 2020 23:53:01:  2000000 \\nINFO  @ Fri, 13 Mar 2020 23:53:07:  3000000 \\nINFO  @ Fri, 13 Mar 2020 23:53:14:  4000000 \\nINFO  @ Fri, 13 Mar 2020 23:53:21:  5000000 \\nINFO  @ Fri, 13 Mar 2020 23:53:28:  6000000 \\nINFO  @ Fri, 13 Mar 2020 23:53:34:  7000000 \\nINFO  @ Fri, 13 Mar 2020 23:53:41:  8000000 \\nINFO  @ Fri, 13 Mar 2020 23:53:48:  9000000 \\nINFO  @ Fri, 13 Mar 2020 23:53:56:  10000000 \\nINFO  @ Fri, 13 Mar 2020 23:54:03:  11000000 \\nINFO  @ Fri, 13 Mar 2020 23:54:09:  12000000 \\nINFO  @ Fri, 13 Mar 2020 23:54:16:  13000000 \\nINFO  @ Fri, 13 Mar 2020 23:54:19: 13516330 reads have been read. \\nINFO  @ Fri, 13 Mar 2020 23:54:20: #1.2 read input tags... \\nINFO  @ Fri, 13 Mar 2020 23:54:26:  1000000 \\nINFO  @ Fri, 13 Mar 2020 23:54:33:  2000000 \\nINFO  @ Fri, 13 Mar 2020 23:54:39:  3000000 \\nINFO  @ Fri, 13 Mar 2020 23:54:45:  4000000 \\nINFO  @ Fri, 13 Mar 2020 23:54:52:  5000000 \\nINFO  @ Fri, 13 Mar 2020 23:54:58:  6000000 \\nINFO  @ Fri, 13 Mar 2020 23:55:05:  7000000 \\nINFO  @ Fri, 13 Mar 2020 23:55:11:  8000000 \\nINFO  @ Fri, 13 Mar 2020 23:55:17:  9000000 \\nINFO  @ Fri, 13 Mar 2020 23:55:24:  10000000 \\nINFO  @ Fri, 13 Mar 2020 23:55:31:  11000000 \\nINFO  @ Fri, 13 Mar 2020 23:55:37:  12000000 \\nINFO  @ Fri, 13 Mar 2020 23:55:44:  13000000 \\nINFO  @ Fri, 13 Mar 2020 23:55:50:  14000000 \\nINFO  @ Fri, 13 Mar 2020 23:55:56:  15000000 \\nINFO  @ Fri, 13 Mar 2020 23:56:03:  16000000 \\nINFO  @ Fri, 13 Mar 2020 23:56:09:  17000000 \\nINFO  @ Fri, 13 Mar 2020 23:56:16:  18000000 \\nINFO  @ Fri, 13 Mar 2020 23:56:22:  19000000 \\nINFO  @ Fri, 13 Mar 2020 23:56:29:  20000000 \\nINFO  @ Fri, 13 Mar 2020 23:56:35:  21000000 \\nINFO  @ Fri, 13 Mar 2020 23:56:42:  22000000 \\nINFO  @ Fri, 13 Mar 2020 23:56:48:  23000000 \\nINFO  @ Fri, 13 Mar 2020 23:56:55:  24000000 \\nINFO  @ Fri, 13 Mar 2020 23:57:01:  25000000 \\nINFO  @ Fri, 13 Mar 2020 23:57:08:  26000000 \\nINFO  @ Fri, 13 Mar 2020 23:57:14:  27000000 \\nINFO  @ Fri, 13 Mar 2020 23:57:21:  28000000 \\nINFO  @ Fri, 13 Mar 2020 23:57:27:  29000000 \\nINFO  @ Fri, 13 Mar 2020 23:57:34:  30000000 \\nINFO  @ Fri, 13 Mar 2020 23:57:40:  31000000 \\nINFO  @ Fri, 13 Mar 2020 23:57:47:  32000000 \\nINFO  @ Fri, 13 Mar 2020 23:57:53:  33000000 \\nINFO  @ Fri, 13 Mar 2020 23:58:00:  34000000 \\nINFO  @ Fri, 13 Mar 2020 23:58:06:  35000000 \\nINFO  @ Fri, 13 Mar 2020 23:58:12:  36000000 \\nINFO  @ Fri, 13 Mar 2020 23:58:19:  37000000 \\nINFO  @ Fri, 13 Mar 2020 23:58:25:  38000000 \\nINFO  @ Fri, 13 Mar 2020 23:58:32:  39000000 \\nINFO  @ Fri, 13 Mar 2020 23:58:38:  40000000 \\nINFO  @ Fri, 13 Mar 2020 23:58:45:  41000000 \\nINFO  @ Fri, 13 Mar 2020 23:58:51:  42000000 \\nINFO  @ Fri, 13 Mar 2020 23:58:58:  43000000 \\nINFO  @ Fri, 13 Mar 2020 23:59:04:  44000000 \\nINFO  @ Fri, 13 Mar 2020 23:59:11:  45000000 \\nINFO  @ Fri, 13 Mar 2020 23:59:17:  46000000 \\nINFO  @ Fri, 13 Mar 2020 23:59:24:  47000000 \\nINFO  @ Fri, 13 Mar 2020 23:59:30:  48000000 \\nINFO  @ Fri, 13 Mar 2020 23:59:36:  49000000 \\nINFO  @ Fri, 13 Mar 2020 23:59:43:  50000000 \\nINFO  @ Fri, 13 Mar 2020 23:59:49:  51000000 \\nINFO  @ Fri, 13 Mar 2020 23:59:55:  52000000 \\nINFO  @ Sat, 14 Mar 2020 00:00:02:  53000000 \\nINFO  @ Sat, 14 Mar 2020 00:00:08:  54000000 \\nINFO  @ Sat, 14 Mar 2020 00:00:15:  55000000 \\nINFO  @ Sat, 14 Mar 2020 00:00:21:  56000000 \\nINFO  @ Sat, 14 Mar 2020 00:00:28:  57000000 \\nINFO  @ Sat, 14 Mar 2020 00:00:34:  58000000 \\nINFO  @ Sat, 14 Mar 2020 00:00:40:  59000000 \\nINFO  @ Sat, 14 Mar 2020 00:00:47:  60000000 \\nINFO  @ Sat, 14 Mar 2020 00:00:53:  61000000 \\nINFO  @ Sat, 14 Mar 2020 00:00:59:  62000000 \\nINFO  @ Sat, 14 Mar 2020 00:01:06:  63000000 \\nINFO  @ Sat, 14 Mar 2020 00:01:12:  64000000 \\nINFO  @ Sat, 14 Mar 2020 00:01:19:  65000000 \\nINFO  @ Sat, 14 Mar 2020 00:01:25:  66000000 \\nINFO  @ Sat, 14 Mar 2020 00:01:30: 66660428 reads have been read. \\nINFO  @ Sat, 14 Mar 2020 00:01:31: #1 tag size is determined as 49 bps \\nINFO  @ Sat, 14 Mar 2020 00:01:31: #1 tag size = 49.0 \\nINFO  @ Sat, 14 Mar 2020 00:01:31: #1  total tags in treatment: 13516330 \\nINFO  @ Sat, 14 Mar 2020 00:01:31: #1 user defined the maximum tags... \\nINFO  @ Sat, 14 Mar 2020 00:01:31: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Sat, 14 Mar 2020 00:01:31: #1  tags after filtering in treatment: 13104910 \\nINFO  @ Sat, 14 Mar 2020 00:01:31: #1  Redundant rate of treatment: 0.03 \\nINFO  @ Sat, 14 Mar 2020 00:01:31: #1  total tags in control: 66660428 \\nINFO  @ Sat, 14 Mar 2020 00:01:31: #1 user defined the maximum tags... \\nINFO  @ Sat, 14 Mar 2020 00:01:31: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Sat, 14 Mar 2020 00:01:31: #1  tags after filtering in control: 65724989 \\nINFO  @ Sat, 14 Mar 2020 00:01:31: #1  Redundant rate of control: 0.01 \\nINFO  @ Sat, 14 Mar 2020 00:01:31: #1 finished! \\nINFO  @ Sat, 14 Mar 2020 00:01:31: #2 Build Peak Model... \\nINFO  @ Sat, 14 Mar 2020 00:01:31: #2 Skipped... \\nINFO  @ Sat, 14 Mar 2020 00:01:31: #2 Use 231 as fragment length \\nINFO  @ Sat, 14 Mar 2020 00:01:31: #3 Call peaks... \\nINFO  @ Sat, 14 Mar 2020 00:01:31: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Sat, 14 Mar 2020 00:09:12: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Sat, 14 Mar 2020 00:09:12: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_treat_pileup.bdg \\nINFO  @ Sat, 14 Mar 2020 00:09:12: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_control_lambda.bdg \\nINFO  @ Sat, 14 Mar 2020 00:09:12: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Sat, 14 Mar 2020 00:09:12: #3 Call peaks for each chromosome... \\nINFO  @ Sat, 14 Mar 2020 00:15:20: #4 Write output xls file... ../../data/IRF2BP2_degraded_4/diffData/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_peaks.xls \\nINFO  @ Sat, 14 Mar 2020 00:15:20: #4 Write peak in narrowPeak format file... ../../data/IRF2BP2_degraded_4/diffData/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_peaks.narrowPeak \\nINFO  @ Sat, 14 Mar 2020 00:15:20: #4 Write summits bed file... ../../data/IRF2BP2_degraded_4/diffData/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_summits.bed \\nINFO  @ Sat, 14 Mar 2020 00:15:20: Done! \\n'\n",
      "b'INFO  @ Sat, 14 Mar 2020 00:15:21: \\n# Command line: callpeak -B -t ../../data/results4/bwa/mergedLibrary/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1.mLb.clN.sorted.bam -c ../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 231 -n mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1 --outdir ../../data/IRF2BP2_degraded_4/diffData/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/results4/bwa/mergedLibrary/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Sat, 14 Mar 2020 00:15:21: #1 read tag files... \\nINFO  @ Sat, 14 Mar 2020 00:15:21: #1 read treatment tags... \\nINFO  @ Sat, 14 Mar 2020 00:15:29:  1000000 \\nINFO  @ Sat, 14 Mar 2020 00:15:38:  2000000 \\nINFO  @ Sat, 14 Mar 2020 00:15:46:  3000000 \\nINFO  @ Sat, 14 Mar 2020 00:15:54:  4000000 \\nINFO  @ Sat, 14 Mar 2020 00:16:03:  5000000 \\nINFO  @ Sat, 14 Mar 2020 00:16:10:  6000000 \\nINFO  @ Sat, 14 Mar 2020 00:16:19:  7000000 \\nINFO  @ Sat, 14 Mar 2020 00:16:27:  8000000 \\nINFO  @ Sat, 14 Mar 2020 00:16:35:  9000000 \\nINFO  @ Sat, 14 Mar 2020 00:16:42:  10000000 \\nINFO  @ Sat, 14 Mar 2020 00:16:50:  11000000 \\nINFO  @ Sat, 14 Mar 2020 00:16:58:  12000000 \\nINFO  @ Sat, 14 Mar 2020 00:17:06:  13000000 \\nINFO  @ Sat, 14 Mar 2020 00:17:13:  14000000 \\nINFO  @ Sat, 14 Mar 2020 00:17:21:  15000000 \\nINFO  @ Sat, 14 Mar 2020 00:17:26: 15589586 reads have been read. \\nINFO  @ Sat, 14 Mar 2020 00:17:26: #1.2 read input tags... \\nINFO  @ Sat, 14 Mar 2020 00:17:32:  1000000 \\nINFO  @ Sat, 14 Mar 2020 00:17:39:  2000000 \\nINFO  @ Sat, 14 Mar 2020 00:17:45:  3000000 \\nINFO  @ Sat, 14 Mar 2020 00:17:52:  4000000 \\nINFO  @ Sat, 14 Mar 2020 00:17:58:  5000000 \\nINFO  @ Sat, 14 Mar 2020 00:18:04:  6000000 \\nINFO  @ Sat, 14 Mar 2020 00:18:11:  7000000 \\nINFO  @ Sat, 14 Mar 2020 00:18:17:  8000000 \\nINFO  @ Sat, 14 Mar 2020 00:18:23:  9000000 \\nINFO  @ Sat, 14 Mar 2020 00:18:30:  10000000 \\nINFO  @ Sat, 14 Mar 2020 00:18:36:  11000000 \\nINFO  @ Sat, 14 Mar 2020 00:18:43:  12000000 \\nINFO  @ Sat, 14 Mar 2020 00:18:49:  13000000 \\nINFO  @ Sat, 14 Mar 2020 00:18:55:  14000000 \\nINFO  @ Sat, 14 Mar 2020 00:19:01:  15000000 \\nINFO  @ Sat, 14 Mar 2020 00:19:07:  16000000 \\nINFO  @ Sat, 14 Mar 2020 00:19:13:  17000000 \\nINFO  @ Sat, 14 Mar 2020 00:19:20:  18000000 \\nINFO  @ Sat, 14 Mar 2020 00:19:26:  19000000 \\nINFO  @ Sat, 14 Mar 2020 00:19:32:  20000000 \\nINFO  @ Sat, 14 Mar 2020 00:19:38:  21000000 \\nINFO  @ Sat, 14 Mar 2020 00:19:44:  22000000 \\nINFO  @ Sat, 14 Mar 2020 00:19:50:  23000000 \\nINFO  @ Sat, 14 Mar 2020 00:19:56:  24000000 \\nINFO  @ Sat, 14 Mar 2020 00:20:02:  25000000 \\nINFO  @ Sat, 14 Mar 2020 00:20:09:  26000000 \\nINFO  @ Sat, 14 Mar 2020 00:20:15:  27000000 \\nINFO  @ Sat, 14 Mar 2020 00:20:21:  28000000 \\nINFO  @ Sat, 14 Mar 2020 00:20:27:  29000000 \\nINFO  @ Sat, 14 Mar 2020 00:20:33:  30000000 \\nINFO  @ Sat, 14 Mar 2020 00:20:39:  31000000 \\nINFO  @ Sat, 14 Mar 2020 00:20:46:  32000000 \\nINFO  @ Sat, 14 Mar 2020 00:20:52:  33000000 \\nINFO  @ Sat, 14 Mar 2020 00:20:58:  34000000 \\nINFO  @ Sat, 14 Mar 2020 00:21:04:  35000000 \\nINFO  @ Sat, 14 Mar 2020 00:21:10:  36000000 \\nINFO  @ Sat, 14 Mar 2020 00:21:16:  37000000 \\nINFO  @ Sat, 14 Mar 2020 00:21:22:  38000000 \\nINFO  @ Sat, 14 Mar 2020 00:21:28:  39000000 \\nINFO  @ Sat, 14 Mar 2020 00:21:34:  40000000 \\nINFO  @ Sat, 14 Mar 2020 00:21:41:  41000000 \\nINFO  @ Sat, 14 Mar 2020 00:21:46:  42000000 \\nINFO  @ Sat, 14 Mar 2020 00:21:54:  43000000 \\nINFO  @ Sat, 14 Mar 2020 00:22:04:  44000000 \\nINFO  @ Sat, 14 Mar 2020 00:22:09:  45000000 \\nINFO  @ Sat, 14 Mar 2020 00:22:15:  46000000 \\nINFO  @ Sat, 14 Mar 2020 00:22:20:  47000000 \\nINFO  @ Sat, 14 Mar 2020 00:22:26:  48000000 \\nINFO  @ Sat, 14 Mar 2020 00:22:31:  49000000 \\nINFO  @ Sat, 14 Mar 2020 00:22:36:  50000000 \\nINFO  @ Sat, 14 Mar 2020 00:22:42:  51000000 \\nINFO  @ Sat, 14 Mar 2020 00:22:47:  52000000 \\nINFO  @ Sat, 14 Mar 2020 00:22:53:  53000000 \\nINFO  @ Sat, 14 Mar 2020 00:22:58:  54000000 \\nINFO  @ Sat, 14 Mar 2020 00:23:04:  55000000 \\nINFO  @ Sat, 14 Mar 2020 00:23:09:  56000000 \\nINFO  @ Sat, 14 Mar 2020 00:23:15:  57000000 \\nINFO  @ Sat, 14 Mar 2020 00:23:20:  58000000 \\nINFO  @ Sat, 14 Mar 2020 00:23:26:  59000000 \\nINFO  @ Sat, 14 Mar 2020 00:23:32:  60000000 \\nINFO  @ Sat, 14 Mar 2020 00:23:37:  61000000 \\nINFO  @ Sat, 14 Mar 2020 00:23:42:  62000000 \\nINFO  @ Sat, 14 Mar 2020 00:23:47:  63000000 \\nINFO  @ Sat, 14 Mar 2020 00:23:53:  64000000 \\nINFO  @ Sat, 14 Mar 2020 00:23:58:  65000000 \\nINFO  @ Sat, 14 Mar 2020 00:24:04:  66000000 \\nINFO  @ Sat, 14 Mar 2020 00:24:08: 66660428 reads have been read. \\nINFO  @ Sat, 14 Mar 2020 00:24:09: #1 tag size is determined as 49 bps \\nINFO  @ Sat, 14 Mar 2020 00:24:09: #1 tag size = 49.0 \\nINFO  @ Sat, 14 Mar 2020 00:24:09: #1  total tags in treatment: 15589586 \\nINFO  @ Sat, 14 Mar 2020 00:24:09: #1 user defined the maximum tags... \\nINFO  @ Sat, 14 Mar 2020 00:24:09: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Sat, 14 Mar 2020 00:24:09: #1  tags after filtering in treatment: 15378834 \\nINFO  @ Sat, 14 Mar 2020 00:24:09: #1  Redundant rate of treatment: 0.01 \\nINFO  @ Sat, 14 Mar 2020 00:24:09: #1  total tags in control: 66660428 \\nINFO  @ Sat, 14 Mar 2020 00:24:09: #1 user defined the maximum tags... \\nINFO  @ Sat, 14 Mar 2020 00:24:09: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Sat, 14 Mar 2020 00:24:09: #1  tags after filtering in control: 65724989 \\nINFO  @ Sat, 14 Mar 2020 00:24:09: #1  Redundant rate of control: 0.01 \\nINFO  @ Sat, 14 Mar 2020 00:24:09: #1 finished! \\nINFO  @ Sat, 14 Mar 2020 00:24:09: #2 Build Peak Model... \\nINFO  @ Sat, 14 Mar 2020 00:24:09: #2 Skipped... \\nINFO  @ Sat, 14 Mar 2020 00:24:09: #2 Use 231 as fragment length \\nINFO  @ Sat, 14 Mar 2020 00:24:09: #3 Call peaks... \\nINFO  @ Sat, 14 Mar 2020 00:24:09: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Sat, 14 Mar 2020 00:29:54: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Sat, 14 Mar 2020 00:29:54: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_treat_pileup.bdg \\nINFO  @ Sat, 14 Mar 2020 00:29:54: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_control_lambda.bdg \\nINFO  @ Sat, 14 Mar 2020 00:29:54: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Sat, 14 Mar 2020 00:29:54: #3 Call peaks for each chromosome... \\nINFO  @ Sat, 14 Mar 2020 00:33:41: #4 Write output xls file... ../../data/IRF2BP2_degraded_4/diffData/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_peaks.xls \\nINFO  @ Sat, 14 Mar 2020 00:33:42: #4 Write peak in narrowPeak format file... ../../data/IRF2BP2_degraded_4/diffData/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_peaks.narrowPeak \\nINFO  @ Sat, 14 Mar 2020 00:33:42: #4 Write summits bed file... ../../data/IRF2BP2_degraded_4/diffData/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_summits.bed \\nINFO  @ Sat, 14 Mar 2020 00:33:42: Done! \\n'\n",
      "13104910 15378834\n",
      "doing differential peak binding\n"
     ]
    }
   ],
   "source": [
    "# diffPeak on scaled data\n",
    "for i in range(int(len(bams[1:])/2)):\n",
    "    name1 = bams[1+i]\n",
    "    name2 = bams[7+i]\n",
    "    chip.fullDiffPeak(name1,name2, control1='../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam', directory = \"../../data/IRF2BP2_degraded_4/diffData/\", res_directory = \"../../data/IRF2BP2_degraded_4/diffPeaks/\",pairedend=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [1.0,\n",
    "1.0,\n",
    "1.0,\n",
    "1.0,\n",
    "1.0,\n",
    "0.626304048,\n",
    "0.2628507876,\n",
    "0.9021192519,\n",
    "0.1869653476,\n",
    "0.5500321887,\n",
    "0.6046056203,\n",
    "1.0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 227,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/diffBinding_hist/mp811-MV411_IRF2BP2_DMSO_6h-H3K27ac-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp812-MV411_IRF2BP2_DMSO_6h-H3K27ac-r2.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp813-MV411_IRF2BP2_DMSO_6h-H3K27me3-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp814-MV411_IRF2BP2_DMSO_6h-H3K27me3-r2.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp815-MV411_IRF2BP2_DMSO_6h-H3K4me1-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp816-MV411_IRF2BP2_DMSO_6h-H3K4me1-r2.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp817-MV411_IRF2BP2_DMSO_6h-H3K4me3-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp818-MV411_IRF2BP2_DMSO_6h-H3K4me3-r2.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp819-MV411_IRF2BP2_DMSO_6h-H3K79me2-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp820-MV411_IRF2BP2_DMSO_6h-H3K79me2-r2.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp821-MV411_IRF2BP2_VHL_6h-H3K27ac-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp822-MV411_IRF2BP2_VHL_6h-H3K27ac-r2.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp823-MV411_IRF2BP2_VHL_6h-H3K27me3-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp824-MV411_IRF2BP2_VHL_6h-H3K27me3-r2.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp825-MV411_IRF2BP2_VHL_6h-H3K4me1-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp826-MV411_IRF2BP2_VHL_6h-H3K4me1-r2.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp827-MV411_IRF2BP2_VHL_6h-H3K4me3-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp828-MV411_IRF2BP2_VHL_6h-H3K4me3-r2.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp829-MV411_IRF2BP2_VHL_6h-H3K79me2-r1.mLb.clN.sorted.bam',\n",
       " '../../data/diffBinding_hist/mp830-MV411_IRF2BP2_VHL_6h-H3K79me2-r2.mLb.clN.sorted.bam']"
      ]
     },
     "execution_count": 227,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "chip.bigWigFrom(bams[1:], \n",
    "                genome='GRCh38',scaling=scales,\n",
    "               numthreads=8)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [],
   "source": [
    "!mv bigwig ../../data/recalib_bigwig_4 "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['439 ../../data/IRF2BP2_degraded_4/diffPeaks/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1_vs_mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1_c3.0_common.bed',\n",
       " '4403 ../../data/IRF2BP2_degraded_4/diffPeaks/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1_vs_mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1_c3.0_cond1.bed',\n",
       " '1 ../../data/IRF2BP2_degraded_4/diffPeaks/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1_vs_mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1_c3.0_cond2.bed',\n",
       " '2732 ../../data/IRF2BP2_degraded_4/diffPeaks/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1_vs_mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1_c3.0_common.bed',\n",
       " '393 ../../data/IRF2BP2_degraded_4/diffPeaks/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1_vs_mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1_c3.0_cond1.bed',\n",
       " '107 ../../data/IRF2BP2_degraded_4/diffPeaks/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1_vs_mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1_c3.0_cond2.bed',\n",
       " '48 ../../data/IRF2BP2_degraded_4/diffPeaks/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1_vs_mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1_c3.0_common.bed',\n",
       " '10512 ../../data/IRF2BP2_degraded_4/diffPeaks/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1_vs_mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1_c3.0_cond1.bed',\n",
       " '34 ../../data/IRF2BP2_degraded_4/diffPeaks/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1_vs_mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1_c3.0_cond2.bed',\n",
       " '32586 ../../data/IRF2BP2_degraded_4/diffPeaks/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1_vs_mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1_c3.0_common.bed',\n",
       " '5926 ../../data/IRF2BP2_degraded_4/diffPeaks/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1_vs_mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1_c3.0_cond1.bed',\n",
       " '199 ../../data/IRF2BP2_degraded_4/diffPeaks/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1_vs_mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1_c3.0_cond2.bed',\n",
       " '19610 ../../data/IRF2BP2_degraded_4/diffPeaks/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1_vs_mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1_c3.0_common.bed',\n",
       " '1795 ../../data/IRF2BP2_degraded_4/diffPeaks/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1_vs_mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1_c3.0_cond1.bed',\n",
       " '61 ../../data/IRF2BP2_degraded_4/diffPeaks/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1_vs_mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1_c3.0_cond2.bed',\n",
       " '14815 ../../data/IRF2BP2_degraded_4/diffPeaks/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_vs_mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_c3.0_common.bed',\n",
       " '13859 ../../data/IRF2BP2_degraded_4/diffPeaks/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_vs_mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_c3.0_cond1.bed',\n",
       " '101 ../../data/IRF2BP2_degraded_4/diffPeaks/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_vs_mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_c3.0_cond2.bed',\n",
       " '']"
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "os.popen('for i in $(ls ../../data/IRF2BP2_degraded_4/diffPeaks/*.bed); \\\n",
    "            do echo $(wc -l $i); \\\n",
    "            done').read().split('\\n')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/recalib_bigwig_4/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1.bw',\n",
       " '../../data/recalib_bigwig_4/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1.bw',\n",
       " '../../data/recalib_bigwig_4/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1.bw',\n",
       " '../../data/recalib_bigwig_4/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1.bw',\n",
       " '../../data/recalib_bigwig_4/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1.bw',\n",
       " '../../data/recalib_bigwig_4/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1.bw',\n",
       " '../../data/recalib_bigwig_4/mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1.bw',\n",
       " '../../data/recalib_bigwig_4/mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1.bw',\n",
       " '../../data/recalib_bigwig_4/mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1.bw',\n",
       " '../../data/recalib_bigwig_4/mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1.bw',\n",
       " '../../data/recalib_bigwig_4/mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1.bw',\n",
       " '../../data/recalib_bigwig_4/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1.bw']"
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bw = ! ls ../../data/recalib_bigwig_4/*.bw\n",
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/IRF2BP2_degraded_4/diffPeaks/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1_vs_mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1_c3.0_cond1.bed',\n",
       " '../../data/IRF2BP2_degraded_4/diffPeaks/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1_vs_mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1_c3.0_cond1.bed',\n",
       " '../../data/IRF2BP2_degraded_4/diffPeaks/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1_vs_mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1_c3.0_cond1.bed',\n",
       " '../../data/IRF2BP2_degraded_4/diffPeaks/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1_vs_mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1_c3.0_cond1.bed',\n",
       " '../../data/IRF2BP2_degraded_4/diffPeaks/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1_vs_mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1_c3.0_cond1.bed',\n",
       " '../../data/IRF2BP2_degraded_4/diffPeaks/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_vs_mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_c3.0_cond1.bed']"
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "cond1peak = ! ls ../../data/IRF2BP2_degraded_4/diffPeaks/*cond1.bed\n",
    "cond2peak = ! ls ../../data/IRF2BP2_degraded_4/diffPeaks/*cond2.bed\n",
    "commonpeak = ! ls ../../data/IRF2BP2_degraded_4/diffPeaks/*common.bed\n",
    "cond1peak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"MED1\",\"MED1_v2\",\"FLAG_IRF2BP2\",\"POLII_total\",\"POLII_S2\",\"POLII_S5\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/recalib_bigwig_4/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1.bw ../../data/recalib_bigwig_4/mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1.bw ../../data/recalib_bigwig_4/mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1.bw  --referencePoint center --regionsFileName ../../data/IRF2BP2_degraded_4/diffPeaks/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1_vs_mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1_c3.0_cond1.bed ../../data/IRF2BP2_degraded_4/diffPeaks/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1_vs_mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1_c3.0_common.bed  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_mat.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common  --samplesLabel DMSO VHL  --plotTitle MED1', returncode=0, stdout=b'', stderr=b'')\n",
      "../../data/recalib_bigwig_4/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1.bw ../../data/recalib_bigwig_4/mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1.bw ../../data/recalib_bigwig_4/mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1.bw  --referencePoint center --regionsFileName ../../data/IRF2BP2_degraded_4/diffPeaks/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1_vs_mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1_c3.0_cond1.bed ../../data/IRF2BP2_degraded_4/diffPeaks/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1_vs_mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1_c3.0_common.bed ../../data/IRF2BP2_degraded_4/diffPeaks/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1_vs_mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_v2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_v2_mat.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_v2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle MED1_v2', returncode=0, stdout=b'', stderr=b'')\n",
      "../../data/recalib_bigwig_4/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1.bw ../../data/recalib_bigwig_4/mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1.bw ../../data/recalib_bigwig_4/mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1.bw  --referencePoint center --regionsFileName ../../data/IRF2BP2_degraded_4/diffPeaks/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1_vs_mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1_c3.0_cond1.bed ../../data/IRF2BP2_degraded_4/diffPeaks/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1_vs_mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1_c3.0_common.bed ../../data/IRF2BP2_degraded_4/diffPeaks/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1_vs_mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/FLAG_IRF2BP2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/FLAG_IRF2BP2_mat.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/FLAG_IRF2BP2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle FLAG_IRF2BP2', returncode=0, stdout=b'', stderr=b\"WARNING: Group 'mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1_vs_mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1_c3.0_common.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n",
      "../../data/recalib_bigwig_4/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1.bw ../../data/recalib_bigwig_4/mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1.bw ../../data/recalib_bigwig_4/mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1.bw  --referencePoint center --regionsFileName ../../data/IRF2BP2_degraded_4/diffPeaks/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1_vs_mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1_c3.0_cond1.bed ../../data/IRF2BP2_degraded_4/diffPeaks/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1_vs_mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1_c3.0_common.bed ../../data/IRF2BP2_degraded_4/diffPeaks/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1_vs_mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_total_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_total_mat.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_total_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle POLII_total', returncode=0, stdout=b'', stderr=b'')\n",
      "../../data/recalib_bigwig_4/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1.bw ../../data/recalib_bigwig_4/mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1.bw ../../data/recalib_bigwig_4/mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1.bw  --referencePoint center --regionsFileName ../../data/IRF2BP2_degraded_4/diffPeaks/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1_vs_mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1_c3.0_cond1.bed ../../data/IRF2BP2_degraded_4/diffPeaks/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1_vs_mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1_c3.0_common.bed ../../data/IRF2BP2_degraded_4/diffPeaks/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1_vs_mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S2_mat.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle POLII_S2', returncode=0, stdout=b'', stderr=b\"WARNING: Group 'mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1_vs_mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1_c3.0_cond2.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n",
      "../../data/recalib_bigwig_4/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1.bw ../../data/recalib_bigwig_4/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1.bw\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1.bw ../../data/recalib_bigwig_4/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1.bw  --referencePoint center --regionsFileName ../../data/IRF2BP2_degraded_4/diffPeaks/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_vs_mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_c3.0_cond1.bed ../../data/IRF2BP2_degraded_4/diffPeaks/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_vs_mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_c3.0_common.bed ../../data/IRF2BP2_degraded_4/diffPeaks/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_vs_mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S5_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S5_mat.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S5_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle POLII_S5', returncode=0, stdout=b'', stderr=b\"WARNING: Group 'mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_vs_mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_c3.0_cond2.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n"
     ]
    }
   ],
   "source": [
    "for i in range(int(len(bw)/2)):\n",
    "    if i <0:\n",
    "        continue\n",
    "    name1 = bw[i]\n",
    "    name2 = bw[i+6]\n",
    "    peak = [cond1peak[i], commonpeak[i], cond2peak[i]]\n",
    "   # for val in peak:\n",
    "      #  chip.dropWeirdChromosomes(val)\n",
    "    name = names[i]\n",
    "    print(name1,name2)\n",
    "    chip.computePeaksAt(peak, [name1, name2], bigwignames=['DMSO', 'VHL'], peaknames=['DMSO_peaks', 'common', 'VHL_peaks'], window=3000, folder=\"\", title=name, numthreads=7, refpoint=\"center\", name='../../data/IRF2BP2_degraded_4/diffPeaks/'+name+'_mat.pdf', withDeeptools=True, torecompute=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 234,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1_peaks.narrowPeak',\n",
       " '../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1_peaks.narrowPeak',\n",
       " '../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1_peaks.narrowPeak',\n",
       " '../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1_peaks.narrowPeak',\n",
       " '../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1_peaks.narrowPeak',\n",
       " '../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_peaks.narrowPeak',\n",
       " '../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1_peaks.narrowPeak',\n",
       " '../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1_peaks.narrowPeak',\n",
       " '../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1_peaks.narrowPeak',\n",
       " '../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1_peaks.narrowPeak',\n",
       " '../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1_peaks.narrowPeak',\n",
       " '../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_peaks.narrowPeak']"
      ]
     },
     "execution_count": 234,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "peaks = ! ls ../../data/results4/bwa/mergedLibrary/macs/narrowPeak/*.narrowPeak\n",
    "peaks"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 232,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['MED1',\n",
       " 'MED1_v2',\n",
       " 'FLAG_IRF2BP2',\n",
       " 'POLII_total',\n",
       " 'POLII_S2',\n",
       " 'POLII_S5',\n",
       " 'MED1_VHL',\n",
       " 'MED1_v2_VHL',\n",
       " 'FLAG_IRF2BP2_VHL',\n",
       " 'POLII_total_VHL',\n",
       " 'POLII_S2_VHL',\n",
       " 'POLII_S5_VHL']"
      ]
     },
     "execution_count": 232,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "names = [\"MED1\",\"MED1_v2\",\"FLAG_IRF2BP2\",\"POLII_total\",\"POLII_S2\",\"POLII_S5\"]\n",
    "names.extend([i+'_VHL' for i in names])\n",
    "names"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 235,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1.bw  --referencePoint center --regionsFileName ../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_mat_profile.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_mat_profile.pdf --refPointLabel center --plotTitle MED1', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1.bw  --referencePoint center --regionsFileName ../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_mat_profile_clust3.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle MED1', returncode=0, stdout=b'', stderr=b\"WARNING: Group 'cluster_1' is too small for plotting, you might want to remove it. \\n\")\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1.bw  --referencePoint center --regionsFileName ../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_v2_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_v2_mat_profile.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_v2_mat_profile.pdf --refPointLabel center --plotTitle MED1_v2', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1.bw  --referencePoint center --regionsFileName ../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_v2_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_v2_mat_profile_clust3.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_v2_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle MED1_v2', returncode=0, stdout=b'', stderr=b\"WARNING: Group 'cluster_1' is too small for plotting, you might want to remove it. \\n\")\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1.bw  --referencePoint center --regionsFileName ../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/FLAG_IRF2BP2_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/FLAG_IRF2BP2_mat_profile.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/FLAG_IRF2BP2_mat_profile.pdf --refPointLabel center --plotTitle FLAG_IRF2BP2', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1.bw  --referencePoint center --regionsFileName ../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/FLAG_IRF2BP2_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/FLAG_IRF2BP2_mat_profile_clust3.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/FLAG_IRF2BP2_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle FLAG_IRF2BP2', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1.bw  --referencePoint center --regionsFileName ../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_total_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_total_mat_profile.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_total_mat_profile.pdf --refPointLabel center --plotTitle POLII_total', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1.bw  --referencePoint center --regionsFileName ../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_total_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_total_mat_profile_clust3.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_total_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle POLII_total', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1.bw  --referencePoint center --regionsFileName ../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S2_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S2_mat_profile.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S2_mat_profile.pdf --refPointLabel center --plotTitle POLII_S2', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1.bw  --referencePoint center --regionsFileName ../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S2_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S2_mat_profile_clust3.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S2_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle POLII_S2', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1.bw  --referencePoint center --regionsFileName ../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S5_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S5_mat_profile.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S5_mat_profile.pdf --refPointLabel center --plotTitle POLII_S5', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1.bw  --referencePoint center --regionsFileName ../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S5_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S5_mat_profile_clust3.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S5_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle POLII_S5', returncode=0, stdout=b'', stderr=b'')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1.bw  --referencePoint center --regionsFileName ../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_VHL_mat_profile.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_VHL_mat_profile.pdf --refPointLabel center --plotTitle MED1_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1.bw  --referencePoint center --regionsFileName ../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_VHL_mat_profile_clust3.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle MED1_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1.bw  --referencePoint center --regionsFileName ../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_v2_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_v2_VHL_mat_profile.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_v2_VHL_mat_profile.pdf --refPointLabel center --plotTitle MED1_v2_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1.bw  --referencePoint center --regionsFileName ../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_v2_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_v2_VHL_mat_profile_clust3.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/MED1_v2_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle MED1_v2_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1.bw  --referencePoint center --regionsFileName ../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/FLAG_IRF2BP2_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/FLAG_IRF2BP2_VHL_mat_profile.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/FLAG_IRF2BP2_VHL_mat_profile.pdf --refPointLabel center --plotTitle FLAG_IRF2BP2_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1.bw  --referencePoint center --regionsFileName ../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/FLAG_IRF2BP2_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/FLAG_IRF2BP2_VHL_mat_profile_clust3.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/FLAG_IRF2BP2_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle FLAG_IRF2BP2_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1.bw  --referencePoint center --regionsFileName ../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_total_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_total_VHL_mat_profile.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_total_VHL_mat_profile.pdf --refPointLabel center --plotTitle POLII_total_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1.bw  --referencePoint center --regionsFileName ../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_total_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_total_VHL_mat_profile_clust3.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_total_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle POLII_total_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1.bw  --referencePoint center --regionsFileName ../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S2_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S2_VHL_mat_profile.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S2_VHL_mat_profile.pdf --refPointLabel center --plotTitle POLII_S2_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1.bw  --referencePoint center --regionsFileName ../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S2_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S2_VHL_mat_profile_clust3.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S2_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle POLII_S2_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1.bw  --referencePoint center --regionsFileName ../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S5_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S5_VHL_mat_profile.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S5_VHL_mat_profile.pdf --refPointLabel center --plotTitle POLII_S5_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_4/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1.bw  --referencePoint center --regionsFileName ../../data/results4/bwa/mergedLibrary/macs/narrowPeak/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S5_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S5_VHL_mat_profile_clust3.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks/POLII_S5_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle POLII_S5_VHL', returncode=0, stdout=b'', stderr=b'')\n"
     ]
    }
   ],
   "source": [
    "for i in range(len(bw)):\n",
    "    chip.computePeaksAt(peaks[i], bw[i], window=3000, folder=\"\", title=names[i], numthreads=7, torecompute=True, onlyProfile=True, name='../../data/IRF2BP2_degraded_4/diffPeaks/'+names[i]+'_mat_profile.pdf', withDeeptools=True, refpoint=\"center\")\n",
    "    chip.computePeaksAt(peaks[i], bw[i], window=3000, folder=\"\", title=names[i], numthreads=7, torecompute=True, onlyProfile=True,name='../../data/IRF2BP2_degraded_4/diffPeaks/'+names[i]+'_mat_profile_clust3.pdf', withDeeptools=True, cluster=3, refpoint=\"center\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### on unscalled data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/results4/bwa/mergedLibrary/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1.mLb.clN.sorted.bam',\n",
       " '../../data/results4/bwa/mergedLibrary/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1.mLb.clN.sorted.bam',\n",
       " '../../data/results4/bwa/mergedLibrary/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1.mLb.clN.sorted.bam',\n",
       " '../../data/results4/bwa/mergedLibrary/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1.mLb.clN.sorted.bam',\n",
       " '../../data/results4/bwa/mergedLibrary/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1.mLb.clN.sorted.bam',\n",
       " '../../data/results4/bwa/mergedLibrary/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1.mLb.clN.sorted.bam',\n",
       " '../../data/results4/bwa/mergedLibrary/mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1.mLb.clN.sorted.bam',\n",
       " '../../data/results4/bwa/mergedLibrary/mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1.mLb.clN.sorted.bam',\n",
       " '../../data/results4/bwa/mergedLibrary/mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1.mLb.clN.sorted.bam',\n",
       " '../../data/results4/bwa/mergedLibrary/mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1.mLb.clN.sorted.bam',\n",
       " '../../data/results4/bwa/mergedLibrary/mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1.mLb.clN.sorted.bam',\n",
       " '../../data/results4/bwa/mergedLibrary/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1.mLb.clN.sorted.bam']"
      ]
     },
     "execution_count": 59,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bams = !ls ../../data/results4/bwa/mergedLibrary/mp*.bam\n",
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "! mkdir ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 76,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/results4/bwa/mergedLibrary/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1.mLb.clN.sorted.bam ../../data/results4/bwa/mergedLibrary/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/results4/bwa/mergedLibrary/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1.mLb.clN.sorted.bam and ../../data/results4/bwa/mergedLibrary/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "231\n",
      "computing the scaling values\n",
      "b'INFO  @ Tue, 17 Mar 2020 13:57:47: \\n# Command line: callpeak -B -t ../../data/results4/bwa/mergedLibrary/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1.mLb.clN.sorted.bam -c ../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 231 -n mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1 --outdir ../../data/IRF2BP2_degraded_4/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/results4/bwa/mergedLibrary/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 17 Mar 2020 13:57:47: #1 read tag files... \\nINFO  @ Tue, 17 Mar 2020 13:57:47: #1 read treatment tags... \\nINFO  @ Tue, 17 Mar 2020 13:57:52:  1000000 \\nINFO  @ Tue, 17 Mar 2020 13:57:57:  2000000 \\nINFO  @ Tue, 17 Mar 2020 13:58:02:  3000000 \\nINFO  @ Tue, 17 Mar 2020 13:58:07:  4000000 \\nINFO  @ Tue, 17 Mar 2020 13:58:12:  5000000 \\nINFO  @ Tue, 17 Mar 2020 13:58:17:  6000000 \\nINFO  @ Tue, 17 Mar 2020 13:58:22:  7000000 \\nINFO  @ Tue, 17 Mar 2020 13:58:28:  8000000 \\nINFO  @ Tue, 17 Mar 2020 13:58:42:  9000000 \\nINFO  @ Tue, 17 Mar 2020 13:58:48:  10000000 \\nINFO  @ Tue, 17 Mar 2020 13:58:54:  11000000 \\nINFO  @ Tue, 17 Mar 2020 13:58:58:  12000000 \\nINFO  @ Tue, 17 Mar 2020 13:59:03:  13000000 \\nINFO  @ Tue, 17 Mar 2020 13:59:07: 13516330 reads have been read. \\nINFO  @ Tue, 17 Mar 2020 13:59:07: #1.2 read input tags... \\nINFO  @ Tue, 17 Mar 2020 13:59:12:  1000000 \\nINFO  @ Tue, 17 Mar 2020 13:59:18:  2000000 \\nINFO  @ Tue, 17 Mar 2020 13:59:24:  3000000 \\nINFO  @ Tue, 17 Mar 2020 13:59:29:  4000000 \\nINFO  @ Tue, 17 Mar 2020 13:59:37:  5000000 \\nINFO  @ Tue, 17 Mar 2020 13:59:43:  6000000 \\nINFO  @ Tue, 17 Mar 2020 13:59:52:  7000000 \\nINFO  @ Tue, 17 Mar 2020 13:59:58:  8000000 \\nINFO  @ Tue, 17 Mar 2020 14:00:03:  9000000 \\nINFO  @ Tue, 17 Mar 2020 14:00:10:  10000000 \\nINFO  @ Tue, 17 Mar 2020 14:00:15:  11000000 \\nINFO  @ Tue, 17 Mar 2020 14:00:20:  12000000 \\nINFO  @ Tue, 17 Mar 2020 14:00:25:  13000000 \\nINFO  @ Tue, 17 Mar 2020 14:00:31:  14000000 \\nINFO  @ Tue, 17 Mar 2020 14:00:37:  15000000 \\nINFO  @ Tue, 17 Mar 2020 14:00:42:  16000000 \\nINFO  @ Tue, 17 Mar 2020 14:00:47:  17000000 \\nINFO  @ Tue, 17 Mar 2020 14:00:53:  18000000 \\nINFO  @ Tue, 17 Mar 2020 14:00:59:  19000000 \\nINFO  @ Tue, 17 Mar 2020 14:01:05:  20000000 \\nINFO  @ Tue, 17 Mar 2020 14:01:10:  21000000 \\nINFO  @ Tue, 17 Mar 2020 14:01:17:  22000000 \\nINFO  @ Tue, 17 Mar 2020 14:01:22:  23000000 \\nINFO  @ Tue, 17 Mar 2020 14:01:29:  24000000 \\nINFO  @ Tue, 17 Mar 2020 14:01:35:  25000000 \\nINFO  @ Tue, 17 Mar 2020 14:01:40:  26000000 \\nINFO  @ Tue, 17 Mar 2020 14:01:46:  27000000 \\nINFO  @ Tue, 17 Mar 2020 14:01:52:  28000000 \\nINFO  @ Tue, 17 Mar 2020 14:01:58:  29000000 \\nINFO  @ Tue, 17 Mar 2020 14:02:06:  30000000 \\nINFO  @ Tue, 17 Mar 2020 14:02:11:  31000000 \\nINFO  @ Tue, 17 Mar 2020 14:02:17:  32000000 \\nINFO  @ Tue, 17 Mar 2020 14:02:25:  33000000 \\nINFO  @ Tue, 17 Mar 2020 14:02:31:  34000000 \\nINFO  @ Tue, 17 Mar 2020 14:02:37:  35000000 \\nINFO  @ Tue, 17 Mar 2020 14:02:43:  36000000 \\nINFO  @ Tue, 17 Mar 2020 14:02:49:  37000000 \\nINFO  @ Tue, 17 Mar 2020 14:02:54:  38000000 \\nINFO  @ Tue, 17 Mar 2020 14:03:00:  39000000 \\nINFO  @ Tue, 17 Mar 2020 14:03:05:  40000000 \\nINFO  @ Tue, 17 Mar 2020 14:03:10:  41000000 \\nINFO  @ Tue, 17 Mar 2020 14:03:16:  42000000 \\nINFO  @ Tue, 17 Mar 2020 14:03:22:  43000000 \\nINFO  @ Tue, 17 Mar 2020 14:03:27:  44000000 \\nINFO  @ Tue, 17 Mar 2020 14:03:33:  45000000 \\nINFO  @ Tue, 17 Mar 2020 14:03:39:  46000000 \\nINFO  @ Tue, 17 Mar 2020 14:03:45:  47000000 \\nINFO  @ Tue, 17 Mar 2020 14:03:51:  48000000 \\nINFO  @ Tue, 17 Mar 2020 14:03:56:  49000000 \\nINFO  @ Tue, 17 Mar 2020 14:04:02:  50000000 \\nINFO  @ Tue, 17 Mar 2020 14:04:07:  51000000 \\nINFO  @ Tue, 17 Mar 2020 14:04:13:  52000000 \\nINFO  @ Tue, 17 Mar 2020 14:04:20:  53000000 \\nINFO  @ Tue, 17 Mar 2020 14:04:26:  54000000 \\nINFO  @ Tue, 17 Mar 2020 14:04:32:  55000000 \\nINFO  @ Tue, 17 Mar 2020 14:04:38:  56000000 \\nINFO  @ Tue, 17 Mar 2020 14:04:44:  57000000 \\nINFO  @ Tue, 17 Mar 2020 14:04:50:  58000000 \\nINFO  @ Tue, 17 Mar 2020 14:04:57:  59000000 \\nINFO  @ Tue, 17 Mar 2020 14:05:03:  60000000 \\nINFO  @ Tue, 17 Mar 2020 14:05:10:  61000000 \\nINFO  @ Tue, 17 Mar 2020 14:05:16:  62000000 \\nINFO  @ Tue, 17 Mar 2020 14:05:21:  63000000 \\nINFO  @ Tue, 17 Mar 2020 14:05:28:  64000000 \\nINFO  @ Tue, 17 Mar 2020 14:05:33:  65000000 \\nINFO  @ Tue, 17 Mar 2020 14:05:39:  66000000 \\nINFO  @ Tue, 17 Mar 2020 14:05:42: 66660428 reads have been read. \\nINFO  @ Tue, 17 Mar 2020 14:05:43: #1 tag size is determined as 49 bps \\nINFO  @ Tue, 17 Mar 2020 14:05:43: #1 tag size = 49.0 \\nINFO  @ Tue, 17 Mar 2020 14:05:43: #1  total tags in treatment: 13516330 \\nINFO  @ Tue, 17 Mar 2020 14:05:43: #1 user defined the maximum tags... \\nINFO  @ Tue, 17 Mar 2020 14:05:43: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 17 Mar 2020 14:05:43: #1  tags after filtering in treatment: 13104910 \\nINFO  @ Tue, 17 Mar 2020 14:05:43: #1  Redundant rate of treatment: 0.03 \\nINFO  @ Tue, 17 Mar 2020 14:05:43: #1  total tags in control: 66660428 \\nINFO  @ Tue, 17 Mar 2020 14:05:43: #1 user defined the maximum tags... \\nINFO  @ Tue, 17 Mar 2020 14:05:43: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 17 Mar 2020 14:05:43: #1  tags after filtering in control: 65724989 \\nINFO  @ Tue, 17 Mar 2020 14:05:43: #1  Redundant rate of control: 0.01 \\nINFO  @ Tue, 17 Mar 2020 14:05:43: #1 finished! \\nINFO  @ Tue, 17 Mar 2020 14:05:43: #2 Build Peak Model... \\nINFO  @ Tue, 17 Mar 2020 14:05:43: #2 Skipped... \\nINFO  @ Tue, 17 Mar 2020 14:05:43: #2 Use 231 as fragment length \\nINFO  @ Tue, 17 Mar 2020 14:05:43: #3 Call peaks... \\nINFO  @ Tue, 17 Mar 2020 14:05:43: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 17 Mar 2020 14:11:15: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 17 Mar 2020 14:11:15: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_treat_pileup.bdg \\nINFO  @ Tue, 17 Mar 2020 14:11:15: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_control_lambda.bdg \\nINFO  @ Tue, 17 Mar 2020 14:11:15: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Tue, 17 Mar 2020 14:11:15: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 17 Mar 2020 14:15:17: #4 Write output xls file... ../../data/IRF2BP2_degraded_4/diffData_unscaled/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_peaks.xls \\nINFO  @ Tue, 17 Mar 2020 14:15:17: #4 Write peak in narrowPeak format file... ../../data/IRF2BP2_degraded_4/diffData_unscaled/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_peaks.narrowPeak \\nINFO  @ Tue, 17 Mar 2020 14:15:17: #4 Write summits bed file... ../../data/IRF2BP2_degraded_4/diffData_unscaled/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_summits.bed \\nINFO  @ Tue, 17 Mar 2020 14:15:17: Done! \\n'\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'INFO  @ Tue, 17 Mar 2020 14:15:18: \\n# Command line: callpeak -B -t ../../data/results4/bwa/mergedLibrary/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1.mLb.clN.sorted.bam -c ../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 231 -n mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1 --outdir ../../data/IRF2BP2_degraded_4/diffData_unscaled/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/results4/bwa/mergedLibrary/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Tue, 17 Mar 2020 14:15:18: #1 read tag files... \\nINFO  @ Tue, 17 Mar 2020 14:15:18: #1 read treatment tags... \\nINFO  @ Tue, 17 Mar 2020 14:15:26:  1000000 \\nINFO  @ Tue, 17 Mar 2020 14:15:35:  2000000 \\nINFO  @ Tue, 17 Mar 2020 14:15:42:  3000000 \\nINFO  @ Tue, 17 Mar 2020 14:15:49:  4000000 \\nINFO  @ Tue, 17 Mar 2020 14:15:55:  5000000 \\nINFO  @ Tue, 17 Mar 2020 14:16:02:  6000000 \\nINFO  @ Tue, 17 Mar 2020 14:16:08:  7000000 \\nINFO  @ Tue, 17 Mar 2020 14:16:14:  8000000 \\nINFO  @ Tue, 17 Mar 2020 14:16:20:  9000000 \\nINFO  @ Tue, 17 Mar 2020 14:16:27:  10000000 \\nINFO  @ Tue, 17 Mar 2020 14:16:34:  11000000 \\nINFO  @ Tue, 17 Mar 2020 14:16:40:  12000000 \\nINFO  @ Tue, 17 Mar 2020 14:16:46:  13000000 \\nINFO  @ Tue, 17 Mar 2020 14:16:53:  14000000 \\nINFO  @ Tue, 17 Mar 2020 14:16:58:  15000000 \\nINFO  @ Tue, 17 Mar 2020 14:17:01: 15589586 reads have been read. \\nINFO  @ Tue, 17 Mar 2020 14:17:02: #1.2 read input tags... \\nINFO  @ Tue, 17 Mar 2020 14:17:06:  1000000 \\nINFO  @ Tue, 17 Mar 2020 14:17:11:  2000000 \\nINFO  @ Tue, 17 Mar 2020 14:17:15:  3000000 \\nINFO  @ Tue, 17 Mar 2020 14:17:20:  4000000 \\nINFO  @ Tue, 17 Mar 2020 14:17:25:  5000000 \\nINFO  @ Tue, 17 Mar 2020 14:17:29:  6000000 \\nINFO  @ Tue, 17 Mar 2020 14:17:34:  7000000 \\nINFO  @ Tue, 17 Mar 2020 14:17:39:  8000000 \\nINFO  @ Tue, 17 Mar 2020 14:17:43:  9000000 \\nINFO  @ Tue, 17 Mar 2020 14:17:48:  10000000 \\nINFO  @ Tue, 17 Mar 2020 14:17:53:  11000000 \\nINFO  @ Tue, 17 Mar 2020 14:17:57:  12000000 \\nINFO  @ Tue, 17 Mar 2020 14:18:02:  13000000 \\nINFO  @ Tue, 17 Mar 2020 14:18:06:  14000000 \\nINFO  @ Tue, 17 Mar 2020 14:18:11:  15000000 \\nINFO  @ Tue, 17 Mar 2020 14:18:16:  16000000 \\nINFO  @ Tue, 17 Mar 2020 14:18:20:  17000000 \\nINFO  @ Tue, 17 Mar 2020 14:18:25:  18000000 \\nINFO  @ Tue, 17 Mar 2020 14:18:29:  19000000 \\nINFO  @ Tue, 17 Mar 2020 14:18:34:  20000000 \\nINFO  @ Tue, 17 Mar 2020 14:18:39:  21000000 \\nINFO  @ Tue, 17 Mar 2020 14:18:45:  22000000 \\nINFO  @ Tue, 17 Mar 2020 14:18:53:  23000000 \\nINFO  @ Tue, 17 Mar 2020 14:18:57:  24000000 \\nINFO  @ Tue, 17 Mar 2020 14:19:02:  25000000 \\nINFO  @ Tue, 17 Mar 2020 14:19:07:  26000000 \\nINFO  @ Tue, 17 Mar 2020 14:19:11:  27000000 \\nINFO  @ Tue, 17 Mar 2020 14:19:16:  28000000 \\nINFO  @ Tue, 17 Mar 2020 14:19:20:  29000000 \\nINFO  @ Tue, 17 Mar 2020 14:19:25:  30000000 \\nINFO  @ Tue, 17 Mar 2020 14:19:29:  31000000 \\nINFO  @ Tue, 17 Mar 2020 14:19:34:  32000000 \\nINFO  @ Tue, 17 Mar 2020 14:19:39:  33000000 \\nINFO  @ Tue, 17 Mar 2020 14:19:43:  34000000 \\nINFO  @ Tue, 17 Mar 2020 14:19:48:  35000000 \\nINFO  @ Tue, 17 Mar 2020 14:19:52:  36000000 \\nINFO  @ Tue, 17 Mar 2020 14:19:57:  37000000 \\nINFO  @ Tue, 17 Mar 2020 14:20:01:  38000000 \\nINFO  @ Tue, 17 Mar 2020 14:20:06:  39000000 \\nINFO  @ Tue, 17 Mar 2020 14:20:10:  40000000 \\nINFO  @ Tue, 17 Mar 2020 14:20:15:  41000000 \\nINFO  @ Tue, 17 Mar 2020 14:20:20:  42000000 \\nINFO  @ Tue, 17 Mar 2020 14:20:24:  43000000 \\nINFO  @ Tue, 17 Mar 2020 14:20:29:  44000000 \\nINFO  @ Tue, 17 Mar 2020 14:20:34:  45000000 \\nINFO  @ Tue, 17 Mar 2020 14:20:38:  46000000 \\nINFO  @ Tue, 17 Mar 2020 14:20:43:  47000000 \\nINFO  @ Tue, 17 Mar 2020 14:20:47:  48000000 \\nINFO  @ Tue, 17 Mar 2020 14:20:52:  49000000 \\nINFO  @ Tue, 17 Mar 2020 14:20:57:  50000000 \\nINFO  @ Tue, 17 Mar 2020 14:21:01:  51000000 \\nINFO  @ Tue, 17 Mar 2020 14:21:06:  52000000 \\nINFO  @ Tue, 17 Mar 2020 14:21:10:  53000000 \\nINFO  @ Tue, 17 Mar 2020 14:21:15:  54000000 \\nINFO  @ Tue, 17 Mar 2020 14:21:20:  55000000 \\nINFO  @ Tue, 17 Mar 2020 14:21:24:  56000000 \\nINFO  @ Tue, 17 Mar 2020 14:21:29:  57000000 \\nINFO  @ Tue, 17 Mar 2020 14:21:33:  58000000 \\nINFO  @ Tue, 17 Mar 2020 14:21:38:  59000000 \\nINFO  @ Tue, 17 Mar 2020 14:21:43:  60000000 \\nINFO  @ Tue, 17 Mar 2020 14:21:47:  61000000 \\nINFO  @ Tue, 17 Mar 2020 14:21:52:  62000000 \\nINFO  @ Tue, 17 Mar 2020 14:21:56:  63000000 \\nINFO  @ Tue, 17 Mar 2020 14:22:01:  64000000 \\nINFO  @ Tue, 17 Mar 2020 14:22:05:  65000000 \\nINFO  @ Tue, 17 Mar 2020 14:22:10:  66000000 \\nINFO  @ Tue, 17 Mar 2020 14:22:13: 66660428 reads have been read. \\nINFO  @ Tue, 17 Mar 2020 14:22:14: #1 tag size is determined as 49 bps \\nINFO  @ Tue, 17 Mar 2020 14:22:14: #1 tag size = 49.0 \\nINFO  @ Tue, 17 Mar 2020 14:22:14: #1  total tags in treatment: 15589586 \\nINFO  @ Tue, 17 Mar 2020 14:22:14: #1 user defined the maximum tags... \\nINFO  @ Tue, 17 Mar 2020 14:22:14: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 17 Mar 2020 14:22:14: #1  tags after filtering in treatment: 15378834 \\nINFO  @ Tue, 17 Mar 2020 14:22:14: #1  Redundant rate of treatment: 0.01 \\nINFO  @ Tue, 17 Mar 2020 14:22:14: #1  total tags in control: 66660428 \\nINFO  @ Tue, 17 Mar 2020 14:22:14: #1 user defined the maximum tags... \\nINFO  @ Tue, 17 Mar 2020 14:22:14: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Tue, 17 Mar 2020 14:22:15: #1  tags after filtering in control: 65724989 \\nINFO  @ Tue, 17 Mar 2020 14:22:15: #1  Redundant rate of control: 0.01 \\nINFO  @ Tue, 17 Mar 2020 14:22:15: #1 finished! \\nINFO  @ Tue, 17 Mar 2020 14:22:15: #2 Build Peak Model... \\nINFO  @ Tue, 17 Mar 2020 14:22:15: #2 Skipped... \\nINFO  @ Tue, 17 Mar 2020 14:22:15: #2 Use 231 as fragment length \\nINFO  @ Tue, 17 Mar 2020 14:22:15: #3 Call peaks... \\nINFO  @ Tue, 17 Mar 2020 14:22:15: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Tue, 17 Mar 2020 14:27:48: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Tue, 17 Mar 2020 14:27:48: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_treat_pileup.bdg \\nINFO  @ Tue, 17 Mar 2020 14:27:48: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_control_lambda.bdg \\nINFO  @ Tue, 17 Mar 2020 14:27:48: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Tue, 17 Mar 2020 14:27:48: #3 Call peaks for each chromosome... \\nINFO  @ Tue, 17 Mar 2020 14:32:08: #4 Write output xls file... ../../data/IRF2BP2_degraded_4/diffData_unscaled/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_peaks.xls \\nINFO  @ Tue, 17 Mar 2020 14:32:08: #4 Write peak in narrowPeak format file... ../../data/IRF2BP2_degraded_4/diffData_unscaled/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_peaks.narrowPeak \\nINFO  @ Tue, 17 Mar 2020 14:32:08: #4 Write summits bed file... ../../data/IRF2BP2_degraded_4/diffData_unscaled/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_summits.bed \\nINFO  @ Tue, 17 Mar 2020 14:32:08: Done! \\n'\n",
      "13104910 15378834\n",
      "doing differential peak binding\n"
     ]
    }
   ],
   "source": [
    "#on unscalled data \n",
    "for i in range(int(len(bams)/2)):\n",
    "    if i < 5:\n",
    "        continue\n",
    "    name1 = bams[i]\n",
    "    name2 = bams[6+i]\n",
    "    print(name1,name2)\n",
    "    chip.fullDiffPeak(name1,name2, control1='../../data/results4/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam', directory = \"../../data/IRF2BP2_degraded_4/diffData_unscaled/\", res_directory = \"../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/\",pairedend=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 236,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/results4/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bigWig',\n",
       " '../../data/results4/bwa/mergedLibrary/bigwig/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1.mLb.clN.bigWig',\n",
       " '../../data/results4/bwa/mergedLibrary/bigwig/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1.mLb.clN.bigWig',\n",
       " '../../data/results4/bwa/mergedLibrary/bigwig/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1.mLb.clN.bigWig',\n",
       " '../../data/results4/bwa/mergedLibrary/bigwig/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1.mLb.clN.bigWig',\n",
       " '../../data/results4/bwa/mergedLibrary/bigwig/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1.mLb.clN.bigWig',\n",
       " '../../data/results4/bwa/mergedLibrary/bigwig/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1.mLb.clN.bigWig',\n",
       " '../../data/results4/bwa/mergedLibrary/bigwig/mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1.mLb.clN.bigWig',\n",
       " '../../data/results4/bwa/mergedLibrary/bigwig/mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1.mLb.clN.bigWig',\n",
       " '../../data/results4/bwa/mergedLibrary/bigwig/mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1.mLb.clN.bigWig',\n",
       " '../../data/results4/bwa/mergedLibrary/bigwig/mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1.mLb.clN.bigWig',\n",
       " '../../data/results4/bwa/mergedLibrary/bigwig/mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1.mLb.clN.bigWig',\n",
       " '../../data/results4/bwa/mergedLibrary/bigwig/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1.mLb.clN.bigWig']"
      ]
     },
     "execution_count": 236,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bw = ! ls ../../data/results4/bwa/mergedLibrary/bigwig/*.bigWig\n",
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 237,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1_vs_mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1_c3.0_common.bed',\n",
       " '../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1_vs_mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1_c3.0_common.bed',\n",
       " '../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1_vs_mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1_c3.0_common.bed',\n",
       " '../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1_vs_mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1_c3.0_common.bed',\n",
       " '../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1_vs_mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1_c3.0_common.bed',\n",
       " '../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_vs_mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_c3.0_common.bed']"
      ]
     },
     "execution_count": 237,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "cond1peak = ! ls ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/*cond1.bed\n",
    "cond2peak = ! ls ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/*cond2.bed\n",
    "commonpeak = ! ls ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/*common.bed\n",
    "commonpeak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 238,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"MED1\",\"MED1_v2\",\"FLAG_IRF2BP2\",\"POLII_total\",\"POLII_S2\",\"POLII_S5\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 239,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/results4/bwa/mergedLibrary/bigwig/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1.mLb.clN.bigWig ../../data/results4/bwa/mergedLibrary/bigwig/mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1.mLb.clN.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/results4/bwa/mergedLibrary/bigwig/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1.mLb.clN.bigWig ../../data/results4/bwa/mergedLibrary/bigwig/mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1_vs_mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1_c3.0_cond1.bed ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1_vs_mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1_c3.0_common.bed ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/mp846-MV411_IRF2BP_DMSO_6h-MED1-r4_R1_vs_mp852-MV411_IRF2BP_VHL_6h-MED1-r4_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/MED1_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/MED1_mat.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/MED1_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle MED1', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270396v1\\t      1880\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270371v1\\t      2805\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270364v1\\t      2855\\nchrUn_KI270581v1\\t      7046\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270390v1\\t      2387\\nchrUn_KI270544v1\\t      1202\\nchrUn_KI270539v1\\t       993\\nchrUn_KI270385v1\\t       990\\n')\n",
      "../../data/results4/bwa/mergedLibrary/bigwig/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1.mLb.clN.bigWig ../../data/results4/bwa/mergedLibrary/bigwig/mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1.mLb.clN.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/results4/bwa/mergedLibrary/bigwig/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1.mLb.clN.bigWig ../../data/results4/bwa/mergedLibrary/bigwig/mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1_vs_mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1_c3.0_cond1.bed ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1_vs_mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1_c3.0_common.bed ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/mp847-MV411_IRF2BP_DMSO_6h-MED1-r5_R1_vs_mp853-MV411_IRF2BP_VHL_6h-MED1-r5_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/MED1_v2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/MED1_v2_mat.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/MED1_v2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle MED1_v2', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270385v1\\t       990\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270396v1\\t      1880\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270329v1\\t      1040\\nchrUn_KI270539v1\\t       993\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270581v1\\t      7046\\nchrUn_KI270383v1\\t      1750\\n         chrEBV\\t    171823\\nchrUn_KI270414v1\\t      2489\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270387v1\\t      1537\\n')\n",
      "../../data/results4/bwa/mergedLibrary/bigwig/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1.mLb.clN.bigWig ../../data/results4/bwa/mergedLibrary/bigwig/mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1.mLb.clN.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/results4/bwa/mergedLibrary/bigwig/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1.mLb.clN.bigWig ../../data/results4/bwa/mergedLibrary/bigwig/mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1_vs_mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1_c3.0_cond1.bed ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1_vs_mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1_c3.0_common.bed ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/mp848-MV411_IRF2BP_DMSO_6h-FLAG_IRF2BP2-r3_R1_vs_mp854-MV411_IRF2BP_VHL_6h-FLAG_IRF2BP2-r3_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/FLAG_IRF2BP2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/FLAG_IRF2BP2_mat.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/FLAG_IRF2BP2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle FLAG_IRF2BP2', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270335v1\\t      1048\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270425v1\\t      1884\\nchrUn_KI270390v1\\t      2387\\nchrUn_KI270396v1\\t      1880\\nchrUn_KI270386v1\\t      1788\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270381v1\\t      1930\\nchrUn_KI270581v1\\t      7046\\n')\n",
      "../../data/results4/bwa/mergedLibrary/bigwig/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1.mLb.clN.bigWig ../../data/results4/bwa/mergedLibrary/bigwig/mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1.mLb.clN.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/results4/bwa/mergedLibrary/bigwig/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1.mLb.clN.bigWig ../../data/results4/bwa/mergedLibrary/bigwig/mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1_vs_mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1_c3.0_cond1.bed ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1_vs_mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1_c3.0_common.bed ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/mp849-MV411_IRF2BP_DMSO_6h-POLII_total-r3_R1_vs_mp855-MV411_IRF2BP_VHL_6h-POLII_total-r3_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/POLII_total_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/POLII_total_mat.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/POLII_total_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle POLII_total', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270411v1\\t      2646\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270530v1\\t      2168\\nchrUn_KI270329v1\\t      1040\\nchrUn_KI270387v1\\t      1537\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270302v1\\t      2274\\nchrUn_KI270382v1\\t      4215\\nchrUn_KI270581v1\\t      7046\\nchrUn_KI270414v1\\t      2489\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270364v1\\t      2855\\nchrUn_KI270544v1\\t      1202\\nchrUn_KI270425v1\\t      1884\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270378v1\\t      1048\\n')\n",
      "../../data/results4/bwa/mergedLibrary/bigwig/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1.mLb.clN.bigWig ../../data/results4/bwa/mergedLibrary/bigwig/mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1.mLb.clN.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/results4/bwa/mergedLibrary/bigwig/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1.mLb.clN.bigWig ../../data/results4/bwa/mergedLibrary/bigwig/mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1_vs_mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1_c3.0_cond1.bed ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1_vs_mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1_c3.0_common.bed ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1_vs_mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/POLII_S2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/POLII_S2_mat.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/POLII_S2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle POLII_S2', returncode=0, stdout=b'', stderr=b\"\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270425v1\\t      1884\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270390v1\\t      2387\\nchrUn_KI270581v1\\t      7046\\nchrUn_KI270395v1\\t      1143\\nchrUn_KI270387v1\\t      1537\\nchrUn_KI270391v1\\t      1484\\nchrUn_KI270396v1\\t      1880\\nchrUn_KI270544v1\\t      1202\\nchrUn_KI270411v1\\t      2646\\nchrUn_KI270392v1\\t       971\\nchrUn_KI270363v1\\t      1803\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270381v1\\t      1930\\nchrUn_KI270304v1\\t      2165\\nchrUn_KI270364v1\\t      2855\\nWARNING: Group 'mp850-MV411_IRF2BP_DMSO_6h-POLII_S2-r3_R1_vs_mp856-MV411_IRF2BP_VHL_6h-POLII_S2-r3_R1_c3.0_cond1.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n",
      "../../data/results4/bwa/mergedLibrary/bigwig/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1.mLb.clN.bigWig ../../data/results4/bwa/mergedLibrary/bigwig/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1.mLb.clN.bigWig\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/results4/bwa/mergedLibrary/bigwig/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1.mLb.clN.bigWig ../../data/results4/bwa/mergedLibrary/bigwig/mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_vs_mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_c3.0_cond1.bed ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_vs_mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_c3.0_common.bed ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_vs_mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/POLII_S5_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotHeatmap --matrixFile ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/POLII_S5_mat.gz --outFileName ../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/POLII_S5_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle POLII_S5', returncode=0, stdout=b'', stderr=b\"\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270391v1\\t      1484\\nchrUn_KI270414v1\\t      2489\\nchrUn_KI270539v1\\t       993\\nchrUn_KI270395v1\\t      1143\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270530v1\\t      2168\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270382v1\\t      4215\\nchrUn_KI270386v1\\t      1788\\nchrUn_KI270334v1\\t      1368\\nchrUn_KI270422v1\\t      1445\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270544v1\\t      1202\\nchrUn_KI270396v1\\t      1880\\nchrUn_KI270425v1\\t      1884\\nchrUn_KI270305v1\\t      1472\\nchrUn_KI270387v1\\t      1537\\nchrUn_KI270389v1\\t      1298\\nWARNING: Group 'mp851-MV411_IRF2BP_DMSO_6h-POLII_S5-r3_R1_vs_mp857-MV411_IRF2BP_VHL_6h-POLII_S5-r3_R1_c3.0_cond2.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n"
     ]
    }
   ],
   "source": [
    "for i in range(int((len(bw)-1)/2)):\n",
    "    name1 = bw[1+i]\n",
    "    name2 = bw[7+i]\n",
    "    peak = [cond1peak[i], commonpeak[i], cond2peak[i]]\n",
    "    #for val in peak:\n",
    "     #   chip.dropWeirdChromosomes(val)\n",
    "    name = names[i]\n",
    "    print(name1,name2)\n",
    "    chip.computePeaksAt(peak, [name1, name2], bigwignames=['DMSO', 'VHL'], peaknames=['DMSO_peaks', 'common', 'VHL_peaks'], window=3000, folder=\"\", title=name, numthreads=7, torecompute=True, refpoint='center', name='../../data/IRF2BP2_degraded_4/diffPeaks_unscaled/'+name+'_mat.pdf', withDeeptools=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! gsutil -m cp -r \"../../data/recalib_bigwig_3/*\" gs://amlproject/Chip/IRF2BP2_degraded_rep3/recalib_bigwig/ &&\\\n",
    "gsutil -m cp -r \"../../data/IRF2BP2_degraded_3/diffData/*\" gs://amlproject/Chip/IRF2BP2_degraded_rep3/diffData/ &&\\\n",
    "gsutil -m cp -r \"../../data/IRF2BP2_degraded_3/diffPeaks/*\" gs://amlproject/Chip/IRF2BP2_degraded_rep3/diffPeaks/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## v3"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! gsutil mv gs://transfer-amlproject/*MP7781*  gs://transfer-amlproject/IRF2BP2_v3/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "ls"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mkdir bams && gsutil -m cp gs://transfer-amlproject/IRF2BP2_v3/* bams/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "cd bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "rename = {\n",
    "\"20200203_1_MP7781_S67_R1_001.fastq.gz\":\"mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1.fastq.gz\",\n",
    "\"20200203_1_MP7781_S67_R2_001.fastq.gz\":\"mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R2.fastq.gz\",\n",
    "\"20200203_2_MP7781_S68_R1_001.fastq.gz\":\"mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1.fastq.gz\",\n",
    "\"20200203_2_MP7781_S68_R2_001.fastq.gz\":\"mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R2.fastq.gz\",\n",
    "\"20200203_3_MP7781_S69_R1_001.fastq.gz\":\"mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1.fastq.gz\",\n",
    "\"20200203_3_MP7781_S69_R2_001.fastq.gz\":\"mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R2.fastq.gz\",\n",
    "\"20200203_4_MP7781_S70_R1_001.fastq.gz\":\"mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1.fastq.gz\",\n",
    "\"20200203_4_MP7781_S70_R2_001.fastq.gz\":\"mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R2.fastq.gz\",\n",
    "\"20200203_5_MP7781_S71_R2_001.fastq.gz\":\"mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R2.fastq.gz\",\n",
    "\"20200203_5_MP7781_S71_R1_001.fastq.gz\":\"mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1.fastq.gz\",\n",
    "\"20200203_6_MP7781_S72_R1_001.fastq.gz\":\"mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1.fastq.gz\",\n",
    "\"20200203_6_MP7781_S72_R2_001.fastq.gz\":\"mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R2.fastq.gz\",\n",
    "\"20200203_7_MP7781_S73_R1_001.fastq.gz\":\"mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1.fastq.gz\",\n",
    "\"20200203_7_MP7781_S73_R2_001.fastq.gz\":\"mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R2.fastq.gz\",\n",
    "\"20200203_8_MP7781_S74_R1_001.fastq.gz\":\"mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1.fastq.gz\",\n",
    "\"20200203_8_MP7781_S74_R2_001.fastq.gz\":\"mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R2.fastq.gz\",\n",
    "\"20200203_9_MP7781_S75_R1_001.fastq.gz\":\"mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1.fastq.gz\",\n",
    "\"20200203_9_MP7781_S75_R2_001.fastq.gz\":\"mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R2.fastq.gz\",\n",
    "\"20200203_10_MP7781_S76_R2_001.fastq.gz\":\"mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R2.fastq.gz\",\n",
    "\"20200203_10_MP7781_S76_R1_001.fastq.gz\":\"mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1.fastq.gz\",\n",
    "\"20200203_11_MP7781_S77_R1_001.fastq.gz\":\"mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1.fastq.gz\",\n",
    "\"20200203_11_MP7781_S77_R2_001.fastq.gz\":\"mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R2.fastq.gz\",\n",
    "\"20200203_12_MP7781_S78_R1_001.fastq.gz\":\"mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1.fastq.gz\",\n",
    "\"20200203_12_MP7781_S78_R2_001.fastq.gz\":\"mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R2.fastq.gz\",\n",
    "\"20200203_13_MP7781_S79_R1_001.fastq.gz\":\"mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1.fastq.gz\",\n",
    "\"20200203_13_MP7781_S79_R2_001.fastq.gz\":\"mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R2.fastq.gz\",\n",
    "\"20200203_14_MP7781_S80_R2_001.fastq.gz\":\"mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R2.fastq.gz\",\n",
    "\"20200203_14_MP7781_S80_R1_001.fastq.gz\":\"mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1.fastq.gz\",\n",
    "\"20200203_Input_MP7781_S81_R1_001.fastq.gz\":\"mp845-MV411_IRF2BP2_-INPUT-r1_R1.fastq.gz\",\n",
    "\"20200203_Input_MP7781_S81_R2_001.fastq.gz\":\"mp845-MV411_IRF2BP2_-INPUT-r1_R2.fastq.gz\"\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for k,v in rename.items():\n",
    "    ! mv $k $v"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "ls"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "cd .."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! gsutil ls gs://amlproject/Chip_AML/IRF2BP2/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from gsheets import Sheets\n",
    "sheets = Sheets.from_files('~/.client_secret.json', '~/.storage.json')\n",
    "url=\"https://docs.google.com/spreadsheets/d/1yFLjYB1McU530JnLgL0QIMAKIkVl3kl0_LCHje2gk8U\"\n",
    "gsheet = sheets.get(url).sheets[2].to_frame()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "a = ! ls bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "gsheet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = {\n",
    "\"fastq_1\": [],\n",
    "\"fastq_2\": [],\n",
    "\"antibody\": [],\n",
    "\"group\": [],\n",
    "\"replicate\": [],\n",
    "\"control\": []\n",
    "}\n",
    "for val in helper.grouped(a[:-2],2):\n",
    "    row = gsheet[gsheet.id==val[0].split('/')[-1].split('-')[0]]\n",
    "    df['group'].append(row['name'].values[0])\n",
    "    df['replicate'].append(1)\n",
    "    df['fastq_1'].append('bams/'+val[0])\n",
    "    df['fastq_2'].append('bams/'+val[1])\n",
    "    df['antibody'].append(row['protein'].values[0])\n",
    "    df['control'].append(\"INPUT\")\n",
    "df['group'].append('INPUT')\n",
    "df['replicate'].append(1)\n",
    "df['fastq_1'].append('bams/'+a[-2])\n",
    "df['fastq_2'].append('bams/'+a[-1])\n",
    "df['antibody'].append(\"\")\n",
    "df['control'].append(\"\")\n",
    "df = pd.DataFrame(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df[df.columns[[3,4,0,1,2,5]]].to_csv('design.csv',index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#process chips\n",
    "! sudo ../nextflow run nf-core/chipseq --paired_end --seq_center 'DFCI' --email \\\n",
    "'jkobject@gmail.com' --narrow_peak --input design.csv --genome GRCh38 --skip_preseq \\\n",
    "--max_cpus 24 -profile docker -w work"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "ls"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!gsutil -m cp -r results/* gs://amlproject/Chip/IRF2BP2_degraded_rep3/ && sudo rm -r work && mv results results3"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "ls ../TrimGalore-0.6.5/trim_galore"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# get scaling values\n",
    "mappedreads, umappedreads_norm = chip.getSpikeInControlScales(refgenome=\"ref/reference_droso.fna\",\n",
    "                                                                      fastQfolder='res/',\n",
    "                                                                      pairedEnd=True, cores=8,\n",
    "                                                                      tofilter=False,\n",
    "                                                                      totrim=False,\n",
    "                                                                      tomap=False,\n",
    "                                                                      pathtotrim_galore=\"../TrimGalore-0.6.5/trim_galore\")\n",
    "mappedreads, umappedreads_norm"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "computing scales from the excel sheet"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 297,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [[536923,632558],\n",
    "[601370,681405],\n",
    "[2402198,1676203],\n",
    "[417892,216192],\n",
    "[1544590,1350802],\n",
    "[1174994,1376726],\n",
    "[289635,240366]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales= [[1.0, 0.8488122828],\n",
    "[1.0, 0.8825441551],\n",
    "[0.6977788675, 1.0],\n",
    "[0.5173394083, 1.0],\n",
    "[0.8745375796, 1.0],\n",
    "[1.0, 0.8534697536],\n",
    "[0.8298927961, 1.0]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/IRF2BP2_degraded_3/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1.mLb.clN.sorted.bam',\n",
       " '../../data/IRF2BP2_degraded_3/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1.mLb.clN.sorted.bam',\n",
       " '../../data/IRF2BP2_degraded_3/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1.mLb.clN.sorted.bam',\n",
       " '../../data/IRF2BP2_degraded_3/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1.mLb.clN.sorted.bam',\n",
       " '../../data/IRF2BP2_degraded_3/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1.mLb.clN.sorted.bam',\n",
       " '../../data/IRF2BP2_degraded_3/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1.mLb.clN.sorted.bam',\n",
       " '../../data/IRF2BP2_degraded_3/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1.mLb.clN.sorted.bam',\n",
       " '../../data/IRF2BP2_degraded_3/mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1.mLb.clN.sorted.bam',\n",
       " '../../data/IRF2BP2_degraded_3/mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1.mLb.clN.sorted.bam',\n",
       " '../../data/IRF2BP2_degraded_3/mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1.mLb.clN.sorted.bam',\n",
       " '../../data/IRF2BP2_degraded_3/mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1.mLb.clN.sorted.bam',\n",
       " '../../data/IRF2BP2_degraded_3/mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1.mLb.clN.sorted.bam',\n",
       " '../../data/IRF2BP2_degraded_3/mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1.mLb.clN.sorted.bam',\n",
       " '../../data/IRF2BP2_degraded_3/mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1.mLb.clN.sorted.bam']"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bams = ! ls ../../data/IRF2BP2_degraded_3/*.bam #../../data/results3/bwa/mergedLibrary/*.bam\n",
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 241,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CommandException: No URLs matched: gs://amlproject/IRF2BP2_degraded_rep3/bwa/mergedLibrary/*.bam\r\n"
     ]
    }
   ],
   "source": [
    "! gsutil cp gs://amlproject/IRF2BP2_degraded_rep3/bwa/mergedLibrary/*.bam ../../data/IRF2BP2_degraded_3/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1'"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bams[7]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "bams = [bam1.split('/')[-1].split('.')[0] for bam1 in bams]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### on scalled data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1 mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1\n",
      "doing differential peak binding\n",
      "mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1 mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1\n",
      "doing differential peak binding\n",
      "mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1 mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1\n",
      "doing differential peak binding\n",
      "mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1 mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1\n",
      "doing differential peak binding\n",
      "mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1 mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1\n",
      "doing differential peak binding\n",
      "mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1 mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1\n",
      "doing differential peak binding\n",
      "mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1 mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1\n",
      "doing differential peak binding\n"
     ]
    }
   ],
   "source": [
    "# diffPeak on scaled data\n",
    "size=[206, 218, 189, 194, 217, 217, 176]\n",
    "for i in range(int(len(bams)/2)):\n",
    "    if i<0:\n",
    "        continue\n",
    "    name1 = bams[i]\n",
    "    name2 = bams[7+i]\n",
    "    print(name1,name2)\n",
    "    chip.diffPeak(name1, name2, directory= \"../../data/IRF2BP2_degraded_3/diffData/\", res_directory='../../data/IRF2BP2_degraded_3/diffPeaks/', scaling1=scales[i][1], scaling2=scales[i][0], size=size[i])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['9562 ../../data/IRF2BP2_degraded_3/diffPeaks/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1_vs_mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1_c3.0_common.bed',\n",
       " '573 ../../data/IRF2BP2_degraded_3/diffPeaks/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1_vs_mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1_c3.0_cond1.bed',\n",
       " '380 ../../data/IRF2BP2_degraded_3/diffPeaks/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1_vs_mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1_c3.0_cond2.bed',\n",
       " '24160 ../../data/IRF2BP2_degraded_3/diffPeaks/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1_vs_mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1_c3.0_common.bed',\n",
       " '762 ../../data/IRF2BP2_degraded_3/diffPeaks/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1_vs_mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1_c3.0_cond1.bed',\n",
       " '820 ../../data/IRF2BP2_degraded_3/diffPeaks/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1_vs_mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1_c3.0_cond2.bed',\n",
       " '48157 ../../data/IRF2BP2_degraded_3/diffPeaks/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1_vs_mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1_c3.0_common.bed',\n",
       " '9 ../../data/IRF2BP2_degraded_3/diffPeaks/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1_vs_mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1_c3.0_cond1.bed',\n",
       " '771 ../../data/IRF2BP2_degraded_3/diffPeaks/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1_vs_mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1_c3.0_cond2.bed',\n",
       " '32908 ../../data/IRF2BP2_degraded_3/diffPeaks/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1_vs_mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1_c3.0_common.bed',\n",
       " '1 ../../data/IRF2BP2_degraded_3/diffPeaks/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1_vs_mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1_c3.0_cond1.bed',\n",
       " '10638 ../../data/IRF2BP2_degraded_3/diffPeaks/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1_vs_mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1_c3.0_cond2.bed',\n",
       " '8889 ../../data/IRF2BP2_degraded_3/diffPeaks/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1_vs_mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1_c3.0_common.bed',\n",
       " '3 ../../data/IRF2BP2_degraded_3/diffPeaks/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1_vs_mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1_c3.0_cond1.bed',\n",
       " '1132 ../../data/IRF2BP2_degraded_3/diffPeaks/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1_vs_mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1_c3.0_cond2.bed',\n",
       " '16346 ../../data/IRF2BP2_degraded_3/diffPeaks/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1_vs_mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1_c3.0_common.bed',\n",
       " '17 ../../data/IRF2BP2_degraded_3/diffPeaks/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1_vs_mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1_c3.0_cond1.bed',\n",
       " '500 ../../data/IRF2BP2_degraded_3/diffPeaks/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1_vs_mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1_c3.0_cond2.bed',\n",
       " '144324 ../../data/IRF2BP2_degraded_3/diffPeaks/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1_vs_mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1_c3.0_common.bed',\n",
       " '23 ../../data/IRF2BP2_degraded_3/diffPeaks/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1_vs_mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1_c3.0_cond1.bed',\n",
       " '2521 ../../data/IRF2BP2_degraded_3/diffPeaks/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1_vs_mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1_c3.0_cond2.bed',\n",
       " '']"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "os.popen('for i in $(ls ../../data/IRF2BP2_degraded_3/diffPeaks/*.bed); \\\n",
    "            do echo $(wc -l $i); \\\n",
    "            done').read().split('\\n')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 250,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/IRF2BP2_degraded_3/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1.mLb.clN.sorted.bam ../../data/IRF2BP2_degraded_3/mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/IRF2BP2_degraded_3/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1.mLb.clN.sorted.bam and ../../data/IRF2BP2_degraded_3/mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "206\n",
      "computing the scaling values\n",
      "b'INFO  @ Thu, 26 Mar 2020 19:49:51: \\n# Command line: callpeak -B -t ../../data/IRF2BP2_degraded_3/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 206 -n mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1 --outdir ../../data/IRF2BP2_degraded_3/diffData/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/IRF2BP2_degraded_3/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Thu, 26 Mar 2020 19:49:51: #1 read tag files... \\nINFO  @ Thu, 26 Mar 2020 19:49:51: #1 read treatment tags... \\nINFO  @ Thu, 26 Mar 2020 19:49:56:  1000000 \\nINFO  @ Thu, 26 Mar 2020 19:50:00:  2000000 \\nINFO  @ Thu, 26 Mar 2020 19:50:05:  3000000 \\nINFO  @ Thu, 26 Mar 2020 19:50:09:  4000000 \\nINFO  @ Thu, 26 Mar 2020 19:50:13:  5000000 \\nINFO  @ Thu, 26 Mar 2020 19:50:18:  6000000 \\nINFO  @ Thu, 26 Mar 2020 19:50:22:  7000000 \\nINFO  @ Thu, 26 Mar 2020 19:50:27:  8000000 \\nINFO  @ Thu, 26 Mar 2020 19:50:31:  9000000 \\nINFO  @ Thu, 26 Mar 2020 19:50:35:  10000000 \\nINFO  @ Thu, 26 Mar 2020 19:50:40:  11000000 \\nINFO  @ Thu, 26 Mar 2020 19:50:44:  12000000 \\nINFO  @ Thu, 26 Mar 2020 19:50:48:  13000000 \\nINFO  @ Thu, 26 Mar 2020 19:50:53:  14000000 \\nINFO  @ Thu, 26 Mar 2020 19:50:57:  15000000 \\nINFO  @ Thu, 26 Mar 2020 19:51:01:  16000000 \\nINFO  @ Thu, 26 Mar 2020 19:51:06:  17000000 \\nINFO  @ Thu, 26 Mar 2020 19:51:10:  18000000 \\nINFO  @ Thu, 26 Mar 2020 19:51:14:  19000000 \\nINFO  @ Thu, 26 Mar 2020 19:51:19:  20000000 \\nINFO  @ Thu, 26 Mar 2020 19:51:23:  21000000 \\nINFO  @ Thu, 26 Mar 2020 19:51:27:  22000000 \\nINFO  @ Thu, 26 Mar 2020 19:51:31:  23000000 \\nINFO  @ Thu, 26 Mar 2020 19:51:36:  24000000 \\nINFO  @ Thu, 26 Mar 2020 19:51:40:  25000000 \\nINFO  @ Thu, 26 Mar 2020 19:51:44:  26000000 \\nINFO  @ Thu, 26 Mar 2020 19:51:46: 26279587 reads have been read. \\nINFO  @ Thu, 26 Mar 2020 19:51:46: #1.2 read input tags... \\nINFO  @ Thu, 26 Mar 2020 19:51:49:  1000000 \\nINFO  @ Thu, 26 Mar 2020 19:51:51:  2000000 \\nINFO  @ Thu, 26 Mar 2020 19:51:53:  3000000 \\nINFO  @ Thu, 26 Mar 2020 19:51:56:  4000000 \\nINFO  @ Thu, 26 Mar 2020 19:51:58:  5000000 \\nINFO  @ Thu, 26 Mar 2020 19:52:01:  6000000 \\nINFO  @ Thu, 26 Mar 2020 19:52:03:  7000000 \\nINFO  @ Thu, 26 Mar 2020 19:52:06:  8000000 \\nINFO  @ Thu, 26 Mar 2020 19:52:08:  9000000 \\nINFO  @ Thu, 26 Mar 2020 19:52:10:  10000000 \\nINFO  @ Thu, 26 Mar 2020 19:52:13:  11000000 \\nINFO  @ Thu, 26 Mar 2020 19:52:15:  12000000 \\nINFO  @ Thu, 26 Mar 2020 19:52:18:  13000000 \\nINFO  @ Thu, 26 Mar 2020 19:52:20:  14000000 \\nINFO  @ Thu, 26 Mar 2020 19:52:22:  15000000 \\nINFO  @ Thu, 26 Mar 2020 19:52:25:  16000000 \\nINFO  @ Thu, 26 Mar 2020 19:52:27:  17000000 \\nINFO  @ Thu, 26 Mar 2020 19:52:30:  18000000 \\nINFO  @ Thu, 26 Mar 2020 19:52:32:  19000000 \\nINFO  @ Thu, 26 Mar 2020 19:52:35:  20000000 \\nINFO  @ Thu, 26 Mar 2020 19:52:37:  21000000 \\nINFO  @ Thu, 26 Mar 2020 19:52:40:  22000000 \\nINFO  @ Thu, 26 Mar 2020 19:52:42:  23000000 \\nINFO  @ Thu, 26 Mar 2020 19:52:45:  24000000 \\nINFO  @ Thu, 26 Mar 2020 19:52:47:  25000000 \\nINFO  @ Thu, 26 Mar 2020 19:52:49:  26000000 \\nINFO  @ Thu, 26 Mar 2020 19:52:52:  27000000 \\nINFO  @ Thu, 26 Mar 2020 19:52:54:  28000000 \\nINFO  @ Thu, 26 Mar 2020 19:52:57:  29000000 \\nINFO  @ Thu, 26 Mar 2020 19:52:59:  30000000 \\nINFO  @ Thu, 26 Mar 2020 19:53:02:  31000000 \\nINFO  @ Thu, 26 Mar 2020 19:53:04:  32000000 \\nINFO  @ Thu, 26 Mar 2020 19:53:07:  33000000 \\nINFO  @ Thu, 26 Mar 2020 19:53:09:  34000000 \\nINFO  @ Thu, 26 Mar 2020 19:53:11: 34520959 reads have been read. \\nINFO  @ Thu, 26 Mar 2020 19:53:11: #1 tag size is determined as 49 bps \\nINFO  @ Thu, 26 Mar 2020 19:53:11: #1 tag size = 49.0 \\nINFO  @ Thu, 26 Mar 2020 19:53:11: #1  total tags in treatment: 26279587 \\nINFO  @ Thu, 26 Mar 2020 19:53:11: #1 user defined the maximum tags... \\nINFO  @ Thu, 26 Mar 2020 19:53:11: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 26 Mar 2020 19:53:11: #1  tags after filtering in treatment: 25936896 \\nINFO  @ Thu, 26 Mar 2020 19:53:11: #1  Redundant rate of treatment: 0.01 \\nINFO  @ Thu, 26 Mar 2020 19:53:11: #1  total tags in control: 34520959 \\nINFO  @ Thu, 26 Mar 2020 19:53:11: #1 user defined the maximum tags... \\nINFO  @ Thu, 26 Mar 2020 19:53:11: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 26 Mar 2020 19:53:12: #1  tags after filtering in control: 34516729 \\nINFO  @ Thu, 26 Mar 2020 19:53:12: #1  Redundant rate of control: 0.00 \\nINFO  @ Thu, 26 Mar 2020 19:53:12: #1 finished! \\nINFO  @ Thu, 26 Mar 2020 19:53:12: #2 Build Peak Model... \\nINFO  @ Thu, 26 Mar 2020 19:53:12: #2 Skipped... \\nINFO  @ Thu, 26 Mar 2020 19:53:12: #2 Use 206 as fragment length \\nINFO  @ Thu, 26 Mar 2020 19:53:12: #3 Call peaks... \\nINFO  @ Thu, 26 Mar 2020 19:53:12: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 26 Mar 2020 19:56:13: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 26 Mar 2020 19:56:13: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1_treat_pileup.bdg \\nINFO  @ Thu, 26 Mar 2020 19:56:13: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1_control_lambda.bdg \\nINFO  @ Thu, 26 Mar 2020 19:56:13: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 26 Mar 2020 19:56:13: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 26 Mar 2020 19:58:50: #4 Write output xls file... ../../data/IRF2BP2_degraded_3/diffData/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1_peaks.xls \\nINFO  @ Thu, 26 Mar 2020 19:58:50: #4 Write peak in narrowPeak format file... ../../data/IRF2BP2_degraded_3/diffData/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1_peaks.narrowPeak \\nINFO  @ Thu, 26 Mar 2020 19:58:50: #4 Write summits bed file... ../../data/IRF2BP2_degraded_3/diffData/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1_summits.bed \\nINFO  @ Thu, 26 Mar 2020 19:58:50: Done! \\n'\n",
      "b'INFO  @ Thu, 26 Mar 2020 19:58:51: \\n# Command line: callpeak -B -t ../../data/IRF2BP2_degraded_3/mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 206 -n mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1 --outdir ../../data/IRF2BP2_degraded_3/diffData/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/IRF2BP2_degraded_3/mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Thu, 26 Mar 2020 19:58:51: #1 read tag files... \\nINFO  @ Thu, 26 Mar 2020 19:58:51: #1 read treatment tags... \\nINFO  @ Thu, 26 Mar 2020 19:58:56:  1000000 \\nINFO  @ Thu, 26 Mar 2020 19:59:01:  2000000 \\nINFO  @ Thu, 26 Mar 2020 19:59:05:  3000000 \\nINFO  @ Thu, 26 Mar 2020 19:59:10:  4000000 \\nINFO  @ Thu, 26 Mar 2020 19:59:15:  5000000 \\nINFO  @ Thu, 26 Mar 2020 19:59:21:  6000000 \\nINFO  @ Thu, 26 Mar 2020 19:59:26:  7000000 \\nINFO  @ Thu, 26 Mar 2020 19:59:31:  8000000 \\nINFO  @ Thu, 26 Mar 2020 19:59:36:  9000000 \\nINFO  @ Thu, 26 Mar 2020 19:59:41:  10000000 \\nINFO  @ Thu, 26 Mar 2020 19:59:46:  11000000 \\nINFO  @ Thu, 26 Mar 2020 19:59:50:  12000000 \\nINFO  @ Thu, 26 Mar 2020 19:59:55:  13000000 \\nINFO  @ Thu, 26 Mar 2020 20:00:00:  14000000 \\nINFO  @ Thu, 26 Mar 2020 20:00:05:  15000000 \\nINFO  @ Thu, 26 Mar 2020 20:00:10:  16000000 \\nINFO  @ Thu, 26 Mar 2020 20:00:15:  17000000 \\nINFO  @ Thu, 26 Mar 2020 20:00:20:  18000000 \\nINFO  @ Thu, 26 Mar 2020 20:00:25:  19000000 \\nINFO  @ Thu, 26 Mar 2020 20:00:30:  20000000 \\nINFO  @ Thu, 26 Mar 2020 20:00:35:  21000000 \\nINFO  @ Thu, 26 Mar 2020 20:00:41:  22000000 \\nINFO  @ Thu, 26 Mar 2020 20:00:46:  23000000 \\nINFO  @ Thu, 26 Mar 2020 20:00:51:  24000000 \\nINFO  @ Thu, 26 Mar 2020 20:00:56:  25000000 \\nINFO  @ Thu, 26 Mar 2020 20:01:01:  26000000 \\nINFO  @ Thu, 26 Mar 2020 20:01:06:  27000000 \\nINFO  @ Thu, 26 Mar 2020 20:01:11:  28000000 \\nINFO  @ Thu, 26 Mar 2020 20:01:16:  29000000 \\nINFO  @ Thu, 26 Mar 2020 20:01:21:  30000000 \\nINFO  @ Thu, 26 Mar 2020 20:01:26:  31000000 \\nINFO  @ Thu, 26 Mar 2020 20:01:31:  32000000 \\nINFO  @ Thu, 26 Mar 2020 20:01:36:  33000000 \\nINFO  @ Thu, 26 Mar 2020 20:01:41:  34000000 \\nINFO  @ Thu, 26 Mar 2020 20:01:46:  35000000 \\nINFO  @ Thu, 26 Mar 2020 20:01:47: 35073958 reads have been read. \\nINFO  @ Thu, 26 Mar 2020 20:01:47: #1.2 read input tags... \\nINFO  @ Thu, 26 Mar 2020 20:01:50:  1000000 \\nINFO  @ Thu, 26 Mar 2020 20:01:52:  2000000 \\nINFO  @ Thu, 26 Mar 2020 20:01:55:  3000000 \\nINFO  @ Thu, 26 Mar 2020 20:01:57:  4000000 \\nINFO  @ Thu, 26 Mar 2020 20:01:59:  5000000 \\nINFO  @ Thu, 26 Mar 2020 20:02:02:  6000000 \\nINFO  @ Thu, 26 Mar 2020 20:02:04:  7000000 \\nINFO  @ Thu, 26 Mar 2020 20:02:06:  8000000 \\nINFO  @ Thu, 26 Mar 2020 20:02:09:  9000000 \\nINFO  @ Thu, 26 Mar 2020 20:02:11:  10000000 \\nINFO  @ Thu, 26 Mar 2020 20:02:14:  11000000 \\nINFO  @ Thu, 26 Mar 2020 20:02:16:  12000000 \\nINFO  @ Thu, 26 Mar 2020 20:02:18:  13000000 \\nINFO  @ Thu, 26 Mar 2020 20:02:21:  14000000 \\nINFO  @ Thu, 26 Mar 2020 20:02:23:  15000000 \\nINFO  @ Thu, 26 Mar 2020 20:02:26:  16000000 \\nINFO  @ Thu, 26 Mar 2020 20:02:28:  17000000 \\nINFO  @ Thu, 26 Mar 2020 20:02:30:  18000000 \\nINFO  @ Thu, 26 Mar 2020 20:02:33:  19000000 \\nINFO  @ Thu, 26 Mar 2020 20:02:35:  20000000 \\nINFO  @ Thu, 26 Mar 2020 20:02:38:  21000000 \\nINFO  @ Thu, 26 Mar 2020 20:02:40:  22000000 \\nINFO  @ Thu, 26 Mar 2020 20:02:42:  23000000 \\nINFO  @ Thu, 26 Mar 2020 20:02:45:  24000000 \\nINFO  @ Thu, 26 Mar 2020 20:02:47:  25000000 \\nINFO  @ Thu, 26 Mar 2020 20:02:49:  26000000 \\nINFO  @ Thu, 26 Mar 2020 20:02:52:  27000000 \\nINFO  @ Thu, 26 Mar 2020 20:02:54:  28000000 \\nINFO  @ Thu, 26 Mar 2020 20:02:57:  29000000 \\nINFO  @ Thu, 26 Mar 2020 20:02:59:  30000000 \\nINFO  @ Thu, 26 Mar 2020 20:03:01:  31000000 \\nINFO  @ Thu, 26 Mar 2020 20:03:04:  32000000 \\nINFO  @ Thu, 26 Mar 2020 20:03:06:  33000000 \\nINFO  @ Thu, 26 Mar 2020 20:03:09:  34000000 \\nINFO  @ Thu, 26 Mar 2020 20:03:10: 34520959 reads have been read. \\nINFO  @ Thu, 26 Mar 2020 20:03:10: #1 tag size is determined as 48 bps \\nINFO  @ Thu, 26 Mar 2020 20:03:10: #1 tag size = 48.0 \\nINFO  @ Thu, 26 Mar 2020 20:03:10: #1  total tags in treatment: 35073958 \\nINFO  @ Thu, 26 Mar 2020 20:03:10: #1 user defined the maximum tags... \\nINFO  @ Thu, 26 Mar 2020 20:03:10: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 26 Mar 2020 20:03:10: #1  tags after filtering in treatment: 34496952 \\nINFO  @ Thu, 26 Mar 2020 20:03:10: #1  Redundant rate of treatment: 0.02 \\nINFO  @ Thu, 26 Mar 2020 20:03:10: #1  total tags in control: 34520959 \\nINFO  @ Thu, 26 Mar 2020 20:03:10: #1 user defined the maximum tags... \\nINFO  @ Thu, 26 Mar 2020 20:03:10: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 26 Mar 2020 20:03:11: #1  tags after filtering in control: 34516729 \\nINFO  @ Thu, 26 Mar 2020 20:03:11: #1  Redundant rate of control: 0.00 \\nINFO  @ Thu, 26 Mar 2020 20:03:11: #1 finished! \\nINFO  @ Thu, 26 Mar 2020 20:03:11: #2 Build Peak Model... \\nINFO  @ Thu, 26 Mar 2020 20:03:11: #2 Skipped... \\nINFO  @ Thu, 26 Mar 2020 20:03:11: #2 Use 206 as fragment length \\nINFO  @ Thu, 26 Mar 2020 20:03:11: #3 Call peaks... \\nINFO  @ Thu, 26 Mar 2020 20:03:11: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 26 Mar 2020 20:06:10: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 26 Mar 2020 20:06:10: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1_treat_pileup.bdg \\nINFO  @ Thu, 26 Mar 2020 20:06:10: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1_control_lambda.bdg \\nINFO  @ Thu, 26 Mar 2020 20:06:10: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 26 Mar 2020 20:06:10: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 26 Mar 2020 20:08:59: #4 Write output xls file... ../../data/IRF2BP2_degraded_3/diffData/mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1_peaks.xls \\nINFO  @ Thu, 26 Mar 2020 20:08:59: #4 Write peak in narrowPeak format file... ../../data/IRF2BP2_degraded_3/diffData/mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1_peaks.narrowPeak \\nINFO  @ Thu, 26 Mar 2020 20:08:59: #4 Write summits bed file... ../../data/IRF2BP2_degraded_3/diffData/mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1_summits.bed \\nINFO  @ Thu, 26 Mar 2020 20:08:59: Done! \\n'\n",
      "25936896 34496952\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/IRF2BP2_degraded_3/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1.mLb.clN.sorted.bam ../../data/IRF2BP2_degraded_3/mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/IRF2BP2_degraded_3/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1.mLb.clN.sorted.bam and ../../data/IRF2BP2_degraded_3/mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "218\n",
      "computing the scaling values\n",
      "b'INFO  @ Thu, 26 Mar 2020 20:19:37: \\n# Command line: callpeak -B -t ../../data/IRF2BP2_degraded_3/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 218 -n mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1 --outdir ../../data/IRF2BP2_degraded_3/diffData/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/IRF2BP2_degraded_3/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Thu, 26 Mar 2020 20:19:37: #1 read tag files... \\nINFO  @ Thu, 26 Mar 2020 20:19:37: #1 read treatment tags... \\nINFO  @ Thu, 26 Mar 2020 20:19:41:  1000000 \\nINFO  @ Thu, 26 Mar 2020 20:19:45:  2000000 \\nINFO  @ Thu, 26 Mar 2020 20:19:49:  3000000 \\nINFO  @ Thu, 26 Mar 2020 20:19:53:  4000000 \\nINFO  @ Thu, 26 Mar 2020 20:19:58:  5000000 \\nINFO  @ Thu, 26 Mar 2020 20:20:02:  6000000 \\nINFO  @ Thu, 26 Mar 2020 20:20:06:  7000000 \\nINFO  @ Thu, 26 Mar 2020 20:20:10:  8000000 \\nINFO  @ Thu, 26 Mar 2020 20:20:14:  9000000 \\nINFO  @ Thu, 26 Mar 2020 20:20:17:  10000000 \\nINFO  @ Thu, 26 Mar 2020 20:20:21:  11000000 \\nINFO  @ Thu, 26 Mar 2020 20:20:25:  12000000 \\nINFO  @ Thu, 26 Mar 2020 20:20:29:  13000000 \\nINFO  @ Thu, 26 Mar 2020 20:20:33:  14000000 \\nINFO  @ Thu, 26 Mar 2020 20:20:37:  15000000 \\nINFO  @ Thu, 26 Mar 2020 20:20:41:  16000000 \\nINFO  @ Thu, 26 Mar 2020 20:20:46:  17000000 \\nINFO  @ Thu, 26 Mar 2020 20:20:50:  18000000 \\nINFO  @ Thu, 26 Mar 2020 20:20:54:  19000000 \\nINFO  @ Thu, 26 Mar 2020 20:20:59:  20000000 \\nINFO  @ Thu, 26 Mar 2020 20:21:03:  21000000 \\nINFO  @ Thu, 26 Mar 2020 20:21:07:  22000000 \\nINFO  @ Thu, 26 Mar 2020 20:21:11:  23000000 \\nINFO  @ Thu, 26 Mar 2020 20:21:16:  24000000 \\nINFO  @ Thu, 26 Mar 2020 20:21:20:  25000000 \\nINFO  @ Thu, 26 Mar 2020 20:21:24:  26000000 \\nINFO  @ Thu, 26 Mar 2020 20:21:29:  27000000 \\nINFO  @ Thu, 26 Mar 2020 20:21:33:  28000000 \\nINFO  @ Thu, 26 Mar 2020 20:21:38:  29000000 \\nINFO  @ Thu, 26 Mar 2020 20:21:39: 29349401 reads have been read. \\nINFO  @ Thu, 26 Mar 2020 20:21:39: #1.2 read input tags... \\nINFO  @ Thu, 26 Mar 2020 20:21:42:  1000000 \\nINFO  @ Thu, 26 Mar 2020 20:21:44:  2000000 \\nINFO  @ Thu, 26 Mar 2020 20:21:46:  3000000 \\nINFO  @ Thu, 26 Mar 2020 20:21:48:  4000000 \\nINFO  @ Thu, 26 Mar 2020 20:21:51:  5000000 \\nINFO  @ Thu, 26 Mar 2020 20:21:53:  6000000 \\nINFO  @ Thu, 26 Mar 2020 20:21:55:  7000000 \\nINFO  @ Thu, 26 Mar 2020 20:21:57:  8000000 \\nINFO  @ Thu, 26 Mar 2020 20:21:59:  9000000 \\nINFO  @ Thu, 26 Mar 2020 20:22:02:  10000000 \\nINFO  @ Thu, 26 Mar 2020 20:22:04:  11000000 \\nINFO  @ Thu, 26 Mar 2020 20:22:06:  12000000 \\nINFO  @ Thu, 26 Mar 2020 20:22:09:  13000000 \\nINFO  @ Thu, 26 Mar 2020 20:22:11:  14000000 \\nINFO  @ Thu, 26 Mar 2020 20:22:13:  15000000 \\nINFO  @ Thu, 26 Mar 2020 20:22:15:  16000000 \\nINFO  @ Thu, 26 Mar 2020 20:22:18:  17000000 \\nINFO  @ Thu, 26 Mar 2020 20:22:20:  18000000 \\nINFO  @ Thu, 26 Mar 2020 20:22:22:  19000000 \\nINFO  @ Thu, 26 Mar 2020 20:22:24:  20000000 \\nINFO  @ Thu, 26 Mar 2020 20:22:27:  21000000 \\nINFO  @ Thu, 26 Mar 2020 20:22:29:  22000000 \\nINFO  @ Thu, 26 Mar 2020 20:22:32:  23000000 \\nINFO  @ Thu, 26 Mar 2020 20:22:34:  24000000 \\nINFO  @ Thu, 26 Mar 2020 20:22:37:  25000000 \\nINFO  @ Thu, 26 Mar 2020 20:22:39:  26000000 \\nINFO  @ Thu, 26 Mar 2020 20:22:41:  27000000 \\nINFO  @ Thu, 26 Mar 2020 20:22:43:  28000000 \\nINFO  @ Thu, 26 Mar 2020 20:22:46:  29000000 \\nINFO  @ Thu, 26 Mar 2020 20:22:48:  30000000 \\nINFO  @ Thu, 26 Mar 2020 20:22:50:  31000000 \\nINFO  @ Thu, 26 Mar 2020 20:22:52:  32000000 \\nINFO  @ Thu, 26 Mar 2020 20:22:55:  33000000 \\nINFO  @ Thu, 26 Mar 2020 20:22:57:  34000000 \\nINFO  @ Thu, 26 Mar 2020 20:22:58: 34520959 reads have been read. \\nINFO  @ Thu, 26 Mar 2020 20:22:59: #1 tag size is determined as 48 bps \\nINFO  @ Thu, 26 Mar 2020 20:22:59: #1 tag size = 48.0 \\nINFO  @ Thu, 26 Mar 2020 20:22:59: #1  total tags in treatment: 29349401 \\nINFO  @ Thu, 26 Mar 2020 20:22:59: #1 user defined the maximum tags... \\nINFO  @ Thu, 26 Mar 2020 20:22:59: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 26 Mar 2020 20:22:59: #1  tags after filtering in treatment: 29011156 \\nINFO  @ Thu, 26 Mar 2020 20:22:59: #1  Redundant rate of treatment: 0.01 \\nINFO  @ Thu, 26 Mar 2020 20:22:59: #1  total tags in control: 34520959 \\nINFO  @ Thu, 26 Mar 2020 20:22:59: #1 user defined the maximum tags... \\nINFO  @ Thu, 26 Mar 2020 20:22:59: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 26 Mar 2020 20:22:59: #1  tags after filtering in control: 34516729 \\nINFO  @ Thu, 26 Mar 2020 20:22:59: #1  Redundant rate of control: 0.00 \\nINFO  @ Thu, 26 Mar 2020 20:22:59: #1 finished! \\nINFO  @ Thu, 26 Mar 2020 20:22:59: #2 Build Peak Model... \\nINFO  @ Thu, 26 Mar 2020 20:22:59: #2 Skipped... \\nINFO  @ Thu, 26 Mar 2020 20:22:59: #2 Use 218 as fragment length \\nINFO  @ Thu, 26 Mar 2020 20:22:59: #3 Call peaks... \\nINFO  @ Thu, 26 Mar 2020 20:22:59: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 26 Mar 2020 20:25:53: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 26 Mar 2020 20:25:53: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1_treat_pileup.bdg \\nINFO  @ Thu, 26 Mar 2020 20:25:53: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1_control_lambda.bdg \\nINFO  @ Thu, 26 Mar 2020 20:25:53: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 26 Mar 2020 20:25:53: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 26 Mar 2020 20:29:29: #4 Write output xls file... ../../data/IRF2BP2_degraded_3/diffData/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1_peaks.xls \\nINFO  @ Thu, 26 Mar 2020 20:29:29: #4 Write peak in narrowPeak format file... ../../data/IRF2BP2_degraded_3/diffData/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1_peaks.narrowPeak \\nINFO  @ Thu, 26 Mar 2020 20:29:29: #4 Write summits bed file... ../../data/IRF2BP2_degraded_3/diffData/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1_summits.bed \\nINFO  @ Thu, 26 Mar 2020 20:29:29: Done! \\n'\n",
      "b'INFO  @ Thu, 26 Mar 2020 20:29:30: \\n# Command line: callpeak -B -t ../../data/IRF2BP2_degraded_3/mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 218 -n mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1 --outdir ../../data/IRF2BP2_degraded_3/diffData/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/IRF2BP2_degraded_3/mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Thu, 26 Mar 2020 20:29:30: #1 read tag files... \\nINFO  @ Thu, 26 Mar 2020 20:29:30: #1 read treatment tags... \\nINFO  @ Thu, 26 Mar 2020 20:29:36:  1000000 \\nINFO  @ Thu, 26 Mar 2020 20:29:42:  2000000 \\nINFO  @ Thu, 26 Mar 2020 20:29:47:  3000000 \\nINFO  @ Thu, 26 Mar 2020 20:29:53:  4000000 \\nINFO  @ Thu, 26 Mar 2020 20:29:59:  5000000 \\nINFO  @ Thu, 26 Mar 2020 20:30:05:  6000000 \\nINFO  @ Thu, 26 Mar 2020 20:30:11:  7000000 \\nINFO  @ Thu, 26 Mar 2020 20:30:17:  8000000 \\nINFO  @ Thu, 26 Mar 2020 20:30:23:  9000000 \\nINFO  @ Thu, 26 Mar 2020 20:30:29:  10000000 \\nINFO  @ Thu, 26 Mar 2020 20:30:35:  11000000 \\nINFO  @ Thu, 26 Mar 2020 20:30:42:  12000000 \\nINFO  @ Thu, 26 Mar 2020 20:30:48:  13000000 \\nINFO  @ Thu, 26 Mar 2020 20:30:53:  14000000 \\nINFO  @ Thu, 26 Mar 2020 20:30:59:  15000000 \\nINFO  @ Thu, 26 Mar 2020 20:31:05:  16000000 \\nINFO  @ Thu, 26 Mar 2020 20:31:11:  17000000 \\nINFO  @ Thu, 26 Mar 2020 20:31:17:  18000000 \\nINFO  @ Thu, 26 Mar 2020 20:31:24:  19000000 \\nINFO  @ Thu, 26 Mar 2020 20:31:29:  20000000 \\nINFO  @ Thu, 26 Mar 2020 20:31:35:  21000000 \\nINFO  @ Thu, 26 Mar 2020 20:31:41:  22000000 \\nINFO  @ Thu, 26 Mar 2020 20:31:47:  23000000 \\nINFO  @ Thu, 26 Mar 2020 20:31:52:  24000000 \\nINFO  @ Thu, 26 Mar 2020 20:31:58:  25000000 \\nINFO  @ Thu, 26 Mar 2020 20:32:04:  26000000 \\nINFO  @ Thu, 26 Mar 2020 20:32:10:  27000000 \\nINFO  @ Thu, 26 Mar 2020 20:32:16:  28000000 \\nINFO  @ Thu, 26 Mar 2020 20:32:22:  29000000 \\nINFO  @ Thu, 26 Mar 2020 20:32:28:  30000000 \\nINFO  @ Thu, 26 Mar 2020 20:32:33:  31000000 \\nINFO  @ Thu, 26 Mar 2020 20:32:36: 31440893 reads have been read. \\nINFO  @ Thu, 26 Mar 2020 20:32:37: #1.2 read input tags... \\nINFO  @ Thu, 26 Mar 2020 20:32:40:  1000000 \\nINFO  @ Thu, 26 Mar 2020 20:32:42:  2000000 \\nINFO  @ Thu, 26 Mar 2020 20:32:45:  3000000 \\nINFO  @ Thu, 26 Mar 2020 20:32:48:  4000000 \\nINFO  @ Thu, 26 Mar 2020 20:32:51:  5000000 \\nINFO  @ Thu, 26 Mar 2020 20:32:54:  6000000 \\nINFO  @ Thu, 26 Mar 2020 20:32:57:  7000000 \\nINFO  @ Thu, 26 Mar 2020 20:32:59:  8000000 \\nINFO  @ Thu, 26 Mar 2020 20:33:02:  9000000 \\nINFO  @ Thu, 26 Mar 2020 20:33:04:  10000000 \\nINFO  @ Thu, 26 Mar 2020 20:33:06:  11000000 \\nINFO  @ Thu, 26 Mar 2020 20:33:08:  12000000 \\nINFO  @ Thu, 26 Mar 2020 20:33:10:  13000000 \\nINFO  @ Thu, 26 Mar 2020 20:33:12:  14000000 \\nINFO  @ Thu, 26 Mar 2020 20:33:14:  15000000 \\nINFO  @ Thu, 26 Mar 2020 20:33:17:  16000000 \\nINFO  @ Thu, 26 Mar 2020 20:33:19:  17000000 \\nINFO  @ Thu, 26 Mar 2020 20:33:21:  18000000 \\nINFO  @ Thu, 26 Mar 2020 20:33:23:  19000000 \\nINFO  @ Thu, 26 Mar 2020 20:33:25:  20000000 \\nINFO  @ Thu, 26 Mar 2020 20:33:27:  21000000 \\nINFO  @ Thu, 26 Mar 2020 20:33:30:  22000000 \\nINFO  @ Thu, 26 Mar 2020 20:33:32:  23000000 \\nINFO  @ Thu, 26 Mar 2020 20:33:34:  24000000 \\nINFO  @ Thu, 26 Mar 2020 20:33:36:  25000000 \\nINFO  @ Thu, 26 Mar 2020 20:33:39:  26000000 \\nINFO  @ Thu, 26 Mar 2020 20:33:41:  27000000 \\nINFO  @ Thu, 26 Mar 2020 20:33:43:  28000000 \\nINFO  @ Thu, 26 Mar 2020 20:33:45:  29000000 \\nINFO  @ Thu, 26 Mar 2020 20:33:47:  30000000 \\nINFO  @ Thu, 26 Mar 2020 20:33:49:  31000000 \\nINFO  @ Thu, 26 Mar 2020 20:33:51:  32000000 \\nINFO  @ Thu, 26 Mar 2020 20:33:54:  33000000 \\nINFO  @ Thu, 26 Mar 2020 20:33:56:  34000000 \\nINFO  @ Thu, 26 Mar 2020 20:33:57: 34520959 reads have been read. \\nINFO  @ Thu, 26 Mar 2020 20:33:57: #1 tag size is determined as 49 bps \\nINFO  @ Thu, 26 Mar 2020 20:33:57: #1 tag size = 49.0 \\nINFO  @ Thu, 26 Mar 2020 20:33:57: #1  total tags in treatment: 31440893 \\nINFO  @ Thu, 26 Mar 2020 20:33:57: #1 user defined the maximum tags... \\nINFO  @ Thu, 26 Mar 2020 20:33:57: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 26 Mar 2020 20:33:57: #1  tags after filtering in treatment: 31040060 \\nINFO  @ Thu, 26 Mar 2020 20:33:57: #1  Redundant rate of treatment: 0.01 \\nINFO  @ Thu, 26 Mar 2020 20:33:57: #1  total tags in control: 34520959 \\nINFO  @ Thu, 26 Mar 2020 20:33:57: #1 user defined the maximum tags... \\nINFO  @ Thu, 26 Mar 2020 20:33:57: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 26 Mar 2020 20:33:57: #1  tags after filtering in control: 34516729 \\nINFO  @ Thu, 26 Mar 2020 20:33:57: #1  Redundant rate of control: 0.00 \\nINFO  @ Thu, 26 Mar 2020 20:33:57: #1 finished! \\nINFO  @ Thu, 26 Mar 2020 20:33:57: #2 Build Peak Model... \\nINFO  @ Thu, 26 Mar 2020 20:33:57: #2 Skipped... \\nINFO  @ Thu, 26 Mar 2020 20:33:57: #2 Use 218 as fragment length \\nINFO  @ Thu, 26 Mar 2020 20:33:57: #3 Call peaks... \\nINFO  @ Thu, 26 Mar 2020 20:33:57: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 26 Mar 2020 20:36:46: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 26 Mar 2020 20:36:46: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1_treat_pileup.bdg \\nINFO  @ Thu, 26 Mar 2020 20:36:46: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1_control_lambda.bdg \\nINFO  @ Thu, 26 Mar 2020 20:36:46: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 26 Mar 2020 20:36:46: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 26 Mar 2020 20:39:14: #4 Write output xls file... ../../data/IRF2BP2_degraded_3/diffData/mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1_peaks.xls \\nINFO  @ Thu, 26 Mar 2020 20:39:14: #4 Write peak in narrowPeak format file... ../../data/IRF2BP2_degraded_3/diffData/mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1_peaks.narrowPeak \\nINFO  @ Thu, 26 Mar 2020 20:39:14: #4 Write summits bed file... ../../data/IRF2BP2_degraded_3/diffData/mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1_summits.bed \\nINFO  @ Thu, 26 Mar 2020 20:39:14: Done! \\n'\n",
      "29011156 31040060\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/IRF2BP2_degraded_3/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1.mLb.clN.sorted.bam ../../data/IRF2BP2_degraded_3/mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/IRF2BP2_degraded_3/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1.mLb.clN.sorted.bam and ../../data/IRF2BP2_degraded_3/mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "189\n",
      "computing the scaling values\n",
      "b'INFO  @ Thu, 26 Mar 2020 20:49:23: \\n# Command line: callpeak -B -t ../../data/IRF2BP2_degraded_3/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 189 -n mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1 --outdir ../../data/IRF2BP2_degraded_3/diffData/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/IRF2BP2_degraded_3/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Thu, 26 Mar 2020 20:49:23: #1 read tag files... \\nINFO  @ Thu, 26 Mar 2020 20:49:23: #1 read treatment tags... \\nINFO  @ Thu, 26 Mar 2020 20:49:27:  1000000 \\nINFO  @ Thu, 26 Mar 2020 20:49:31:  2000000 \\nINFO  @ Thu, 26 Mar 2020 20:49:35:  3000000 \\nINFO  @ Thu, 26 Mar 2020 20:49:38:  4000000 \\nINFO  @ Thu, 26 Mar 2020 20:49:42:  5000000 \\nINFO  @ Thu, 26 Mar 2020 20:49:46:  6000000 \\nINFO  @ Thu, 26 Mar 2020 20:49:50:  7000000 \\nINFO  @ Thu, 26 Mar 2020 20:49:54:  8000000 \\nINFO  @ Thu, 26 Mar 2020 20:49:58:  9000000 \\nINFO  @ Thu, 26 Mar 2020 20:50:01:  10000000 \\nINFO  @ Thu, 26 Mar 2020 20:50:05:  11000000 \\nINFO  @ Thu, 26 Mar 2020 20:50:09:  12000000 \\nINFO  @ Thu, 26 Mar 2020 20:50:13:  13000000 \\nINFO  @ Thu, 26 Mar 2020 20:50:17:  14000000 \\nINFO  @ Thu, 26 Mar 2020 20:50:20:  15000000 \\nINFO  @ Thu, 26 Mar 2020 20:50:24:  16000000 \\nINFO  @ Thu, 26 Mar 2020 20:50:28:  17000000 \\nINFO  @ Thu, 26 Mar 2020 20:50:32:  18000000 \\nINFO  @ Thu, 26 Mar 2020 20:50:36:  19000000 \\nINFO  @ Thu, 26 Mar 2020 20:50:40:  20000000 \\nINFO  @ Thu, 26 Mar 2020 20:50:43:  21000000 \\nINFO  @ Thu, 26 Mar 2020 20:50:47:  22000000 \\nINFO  @ Thu, 26 Mar 2020 20:50:51:  23000000 \\nINFO  @ Thu, 26 Mar 2020 20:50:55:  24000000 \\nINFO  @ Thu, 26 Mar 2020 20:50:59:  25000000 \\nINFO  @ Thu, 26 Mar 2020 20:51:03:  26000000 \\nINFO  @ Thu, 26 Mar 2020 20:51:07:  27000000 \\nINFO  @ Thu, 26 Mar 2020 20:51:10:  28000000 \\nINFO  @ Thu, 26 Mar 2020 20:51:11: 28088810 reads have been read. \\nINFO  @ Thu, 26 Mar 2020 20:51:11: #1.2 read input tags... \\nINFO  @ Thu, 26 Mar 2020 20:51:13:  1000000 \\nINFO  @ Thu, 26 Mar 2020 20:51:15:  2000000 \\nINFO  @ Thu, 26 Mar 2020 20:51:17:  3000000 \\nINFO  @ Thu, 26 Mar 2020 20:51:19:  4000000 \\nINFO  @ Thu, 26 Mar 2020 20:51:22:  5000000 \\nINFO  @ Thu, 26 Mar 2020 20:51:24:  6000000 \\nINFO  @ Thu, 26 Mar 2020 20:51:26:  7000000 \\nINFO  @ Thu, 26 Mar 2020 20:51:28:  8000000 \\nINFO  @ Thu, 26 Mar 2020 20:51:30:  9000000 \\nINFO  @ Thu, 26 Mar 2020 20:51:32:  10000000 \\nINFO  @ Thu, 26 Mar 2020 20:51:34:  11000000 \\nINFO  @ Thu, 26 Mar 2020 20:51:36:  12000000 \\nINFO  @ Thu, 26 Mar 2020 20:51:38:  13000000 \\nINFO  @ Thu, 26 Mar 2020 20:51:40:  14000000 \\nINFO  @ Thu, 26 Mar 2020 20:51:42:  15000000 \\nINFO  @ Thu, 26 Mar 2020 20:51:44:  16000000 \\nINFO  @ Thu, 26 Mar 2020 20:51:46:  17000000 \\nINFO  @ Thu, 26 Mar 2020 20:51:48:  18000000 \\nINFO  @ Thu, 26 Mar 2020 20:51:50:  19000000 \\nINFO  @ Thu, 26 Mar 2020 20:51:52:  20000000 \\nINFO  @ Thu, 26 Mar 2020 20:51:54:  21000000 \\nINFO  @ Thu, 26 Mar 2020 20:51:56:  22000000 \\nINFO  @ Thu, 26 Mar 2020 20:51:58:  23000000 \\nINFO  @ Thu, 26 Mar 2020 20:52:00:  24000000 \\nINFO  @ Thu, 26 Mar 2020 20:52:02:  25000000 \\nINFO  @ Thu, 26 Mar 2020 20:52:04:  26000000 \\nINFO  @ Thu, 26 Mar 2020 20:52:06:  27000000 \\nINFO  @ Thu, 26 Mar 2020 20:52:08:  28000000 \\nINFO  @ Thu, 26 Mar 2020 20:52:11:  29000000 \\nINFO  @ Thu, 26 Mar 2020 20:52:13:  30000000 \\nINFO  @ Thu, 26 Mar 2020 20:52:15:  31000000 \\nINFO  @ Thu, 26 Mar 2020 20:52:17:  32000000 \\nINFO  @ Thu, 26 Mar 2020 20:52:19:  33000000 \\nINFO  @ Thu, 26 Mar 2020 20:52:21:  34000000 \\nINFO  @ Thu, 26 Mar 2020 20:52:22: 34520959 reads have been read. \\nINFO  @ Thu, 26 Mar 2020 20:52:22: #1 tag size is determined as 49 bps \\nINFO  @ Thu, 26 Mar 2020 20:52:22: #1 tag size = 49.0 \\nINFO  @ Thu, 26 Mar 2020 20:52:22: #1  total tags in treatment: 28088810 \\nINFO  @ Thu, 26 Mar 2020 20:52:22: #1 user defined the maximum tags... \\nINFO  @ Thu, 26 Mar 2020 20:52:22: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 26 Mar 2020 20:52:23: #1  tags after filtering in treatment: 26041296 \\nINFO  @ Thu, 26 Mar 2020 20:52:23: #1  Redundant rate of treatment: 0.07 \\nINFO  @ Thu, 26 Mar 2020 20:52:23: #1  total tags in control: 34520959 \\nINFO  @ Thu, 26 Mar 2020 20:52:23: #1 user defined the maximum tags... \\nINFO  @ Thu, 26 Mar 2020 20:52:23: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 26 Mar 2020 20:52:23: #1  tags after filtering in control: 34516729 \\nINFO  @ Thu, 26 Mar 2020 20:52:23: #1  Redundant rate of control: 0.00 \\nINFO  @ Thu, 26 Mar 2020 20:52:23: #1 finished! \\nINFO  @ Thu, 26 Mar 2020 20:52:23: #2 Build Peak Model... \\nINFO  @ Thu, 26 Mar 2020 20:52:23: #2 Skipped... \\nINFO  @ Thu, 26 Mar 2020 20:52:23: #2 Use 189 as fragment length \\nINFO  @ Thu, 26 Mar 2020 20:52:23: #3 Call peaks... \\nINFO  @ Thu, 26 Mar 2020 20:52:23: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 26 Mar 2020 20:55:04: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 26 Mar 2020 20:55:04: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1_treat_pileup.bdg \\nINFO  @ Thu, 26 Mar 2020 20:55:04: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1_control_lambda.bdg \\nINFO  @ Thu, 26 Mar 2020 20:55:04: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 26 Mar 2020 20:55:04: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 26 Mar 2020 20:57:27: #4 Write output xls file... ../../data/IRF2BP2_degraded_3/diffData/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1_peaks.xls \\nINFO  @ Thu, 26 Mar 2020 20:57:27: #4 Write peak in narrowPeak format file... ../../data/IRF2BP2_degraded_3/diffData/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1_peaks.narrowPeak \\nINFO  @ Thu, 26 Mar 2020 20:57:27: #4 Write summits bed file... ../../data/IRF2BP2_degraded_3/diffData/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1_summits.bed \\nINFO  @ Thu, 26 Mar 2020 20:57:28: Done! \\n'\n",
      "b'INFO  @ Thu, 26 Mar 2020 20:57:28: \\n# Command line: callpeak -B -t ../../data/IRF2BP2_degraded_3/mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 189 -n mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1 --outdir ../../data/IRF2BP2_degraded_3/diffData/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/IRF2BP2_degraded_3/mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Thu, 26 Mar 2020 20:57:28: #1 read tag files... \\nINFO  @ Thu, 26 Mar 2020 20:57:28: #1 read treatment tags... \\nINFO  @ Thu, 26 Mar 2020 20:57:33:  1000000 \\nINFO  @ Thu, 26 Mar 2020 20:57:37:  2000000 \\nINFO  @ Thu, 26 Mar 2020 20:57:42:  3000000 \\nINFO  @ Thu, 26 Mar 2020 20:57:47:  4000000 \\nINFO  @ Thu, 26 Mar 2020 20:57:52:  5000000 \\nINFO  @ Thu, 26 Mar 2020 20:57:56:  6000000 \\nINFO  @ Thu, 26 Mar 2020 20:58:00:  7000000 \\nINFO  @ Thu, 26 Mar 2020 20:58:05:  8000000 \\nINFO  @ Thu, 26 Mar 2020 20:58:09:  9000000 \\nINFO  @ Thu, 26 Mar 2020 20:58:13:  10000000 \\nINFO  @ Thu, 26 Mar 2020 20:58:17:  11000000 \\nINFO  @ Thu, 26 Mar 2020 20:58:21:  12000000 \\nINFO  @ Thu, 26 Mar 2020 20:58:26:  13000000 \\nINFO  @ Thu, 26 Mar 2020 20:58:30:  14000000 \\nINFO  @ Thu, 26 Mar 2020 20:58:35:  15000000 \\nINFO  @ Thu, 26 Mar 2020 20:58:39:  16000000 \\nINFO  @ Thu, 26 Mar 2020 20:58:43:  17000000 \\nINFO  @ Thu, 26 Mar 2020 20:58:48:  18000000 \\nINFO  @ Thu, 26 Mar 2020 20:58:52:  19000000 \\nINFO  @ Thu, 26 Mar 2020 20:58:56:  20000000 \\nINFO  @ Thu, 26 Mar 2020 20:59:00:  21000000 \\nINFO  @ Thu, 26 Mar 2020 20:59:05:  22000000 \\nINFO  @ Thu, 26 Mar 2020 20:59:09:  23000000 \\nINFO  @ Thu, 26 Mar 2020 20:59:13: 23772664 reads have been read. \\nINFO  @ Thu, 26 Mar 2020 20:59:13: #1.2 read input tags... \\nINFO  @ Thu, 26 Mar 2020 20:59:15:  1000000 \\nINFO  @ Thu, 26 Mar 2020 20:59:17:  2000000 \\nINFO  @ Thu, 26 Mar 2020 20:59:19:  3000000 \\nINFO  @ Thu, 26 Mar 2020 20:59:21:  4000000 \\nINFO  @ Thu, 26 Mar 2020 20:59:23:  5000000 \\nINFO  @ Thu, 26 Mar 2020 20:59:25:  6000000 \\nINFO  @ Thu, 26 Mar 2020 20:59:27:  7000000 \\nINFO  @ Thu, 26 Mar 2020 20:59:29:  8000000 \\nINFO  @ Thu, 26 Mar 2020 20:59:31:  9000000 \\nINFO  @ Thu, 26 Mar 2020 20:59:34:  10000000 \\nINFO  @ Thu, 26 Mar 2020 20:59:36:  11000000 \\nINFO  @ Thu, 26 Mar 2020 20:59:38:  12000000 \\nINFO  @ Thu, 26 Mar 2020 20:59:40:  13000000 \\nINFO  @ Thu, 26 Mar 2020 20:59:42:  14000000 \\nINFO  @ Thu, 26 Mar 2020 20:59:44:  15000000 \\nINFO  @ Thu, 26 Mar 2020 20:59:46:  16000000 \\nINFO  @ Thu, 26 Mar 2020 20:59:48:  17000000 \\nINFO  @ Thu, 26 Mar 2020 20:59:50:  18000000 \\nINFO  @ Thu, 26 Mar 2020 20:59:52:  19000000 \\nINFO  @ Thu, 26 Mar 2020 20:59:54:  20000000 \\nINFO  @ Thu, 26 Mar 2020 20:59:56:  21000000 \\nINFO  @ Thu, 26 Mar 2020 20:59:58:  22000000 \\nINFO  @ Thu, 26 Mar 2020 21:00:00:  23000000 \\nINFO  @ Thu, 26 Mar 2020 21:00:02:  24000000 \\nINFO  @ Thu, 26 Mar 2020 21:00:04:  25000000 \\nINFO  @ Thu, 26 Mar 2020 21:00:06:  26000000 \\nINFO  @ Thu, 26 Mar 2020 21:00:08:  27000000 \\nINFO  @ Thu, 26 Mar 2020 21:00:10:  28000000 \\nINFO  @ Thu, 26 Mar 2020 21:00:13:  29000000 \\nINFO  @ Thu, 26 Mar 2020 21:00:15:  30000000 \\nINFO  @ Thu, 26 Mar 2020 21:00:17:  31000000 \\nINFO  @ Thu, 26 Mar 2020 21:00:19:  32000000 \\nINFO  @ Thu, 26 Mar 2020 21:00:21:  33000000 \\nINFO  @ Thu, 26 Mar 2020 21:00:23:  34000000 \\nINFO  @ Thu, 26 Mar 2020 21:00:24: 34520959 reads have been read. \\nINFO  @ Thu, 26 Mar 2020 21:00:24: #1 tag size is determined as 48 bps \\nINFO  @ Thu, 26 Mar 2020 21:00:24: #1 tag size = 48.0 \\nINFO  @ Thu, 26 Mar 2020 21:00:24: #1  total tags in treatment: 23772664 \\nINFO  @ Thu, 26 Mar 2020 21:00:24: #1 user defined the maximum tags... \\nINFO  @ Thu, 26 Mar 2020 21:00:24: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 26 Mar 2020 21:00:25: #1  tags after filtering in treatment: 22438166 \\nINFO  @ Thu, 26 Mar 2020 21:00:25: #1  Redundant rate of treatment: 0.06 \\nINFO  @ Thu, 26 Mar 2020 21:00:25: #1  total tags in control: 34520959 \\nINFO  @ Thu, 26 Mar 2020 21:00:25: #1 user defined the maximum tags... \\nINFO  @ Thu, 26 Mar 2020 21:00:25: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 26 Mar 2020 21:00:25: #1  tags after filtering in control: 34516729 \\nINFO  @ Thu, 26 Mar 2020 21:00:25: #1  Redundant rate of control: 0.00 \\nINFO  @ Thu, 26 Mar 2020 21:00:25: #1 finished! \\nINFO  @ Thu, 26 Mar 2020 21:00:25: #2 Build Peak Model... \\nINFO  @ Thu, 26 Mar 2020 21:00:25: #2 Skipped... \\nINFO  @ Thu, 26 Mar 2020 21:00:25: #2 Use 189 as fragment length \\nINFO  @ Thu, 26 Mar 2020 21:00:25: #3 Call peaks... \\nINFO  @ Thu, 26 Mar 2020 21:00:25: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 26 Mar 2020 21:03:04: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 26 Mar 2020 21:03:04: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1_treat_pileup.bdg \\nINFO  @ Thu, 26 Mar 2020 21:03:04: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1_control_lambda.bdg \\nINFO  @ Thu, 26 Mar 2020 21:03:04: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 26 Mar 2020 21:03:04: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 26 Mar 2020 21:05:29: #4 Write output xls file... ../../data/IRF2BP2_degraded_3/diffData/mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1_peaks.xls \\nINFO  @ Thu, 26 Mar 2020 21:05:30: #4 Write peak in narrowPeak format file... ../../data/IRF2BP2_degraded_3/diffData/mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1_peaks.narrowPeak \\nINFO  @ Thu, 26 Mar 2020 21:05:30: #4 Write summits bed file... ../../data/IRF2BP2_degraded_3/diffData/mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1_summits.bed \\nINFO  @ Thu, 26 Mar 2020 21:05:30: Done! \\n'\n",
      "26041296 22438166\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/IRF2BP2_degraded_3/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1.mLb.clN.sorted.bam ../../data/IRF2BP2_degraded_3/mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/IRF2BP2_degraded_3/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1.mLb.clN.sorted.bam and ../../data/IRF2BP2_degraded_3/mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "194\n",
      "computing the scaling values\n",
      "b'INFO  @ Thu, 26 Mar 2020 21:16:12: \\n# Command line: callpeak -B -t ../../data/IRF2BP2_degraded_3/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 194 -n mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1 --outdir ../../data/IRF2BP2_degraded_3/diffData/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/IRF2BP2_degraded_3/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Thu, 26 Mar 2020 21:16:12: #1 read tag files... \\nINFO  @ Thu, 26 Mar 2020 21:16:12: #1 read treatment tags... \\nINFO  @ Thu, 26 Mar 2020 21:16:16:  1000000 \\nINFO  @ Thu, 26 Mar 2020 21:16:20:  2000000 \\nINFO  @ Thu, 26 Mar 2020 21:16:25:  3000000 \\nINFO  @ Thu, 26 Mar 2020 21:16:29:  4000000 \\nINFO  @ Thu, 26 Mar 2020 21:16:34:  5000000 \\nINFO  @ Thu, 26 Mar 2020 21:16:38:  6000000 \\nINFO  @ Thu, 26 Mar 2020 21:16:43:  7000000 \\nINFO  @ Thu, 26 Mar 2020 21:16:47:  8000000 \\nINFO  @ Thu, 26 Mar 2020 21:16:51:  9000000 \\nINFO  @ Thu, 26 Mar 2020 21:16:56:  10000000 \\nINFO  @ Thu, 26 Mar 2020 21:17:00:  11000000 \\nINFO  @ Thu, 26 Mar 2020 21:17:05:  12000000 \\nINFO  @ Thu, 26 Mar 2020 21:17:09:  13000000 \\nINFO  @ Thu, 26 Mar 2020 21:17:14:  14000000 \\nINFO  @ Thu, 26 Mar 2020 21:17:18:  15000000 \\nINFO  @ Thu, 26 Mar 2020 21:17:23:  16000000 \\nINFO  @ Thu, 26 Mar 2020 21:17:27:  17000000 \\nINFO  @ Thu, 26 Mar 2020 21:17:31:  18000000 \\nINFO  @ Thu, 26 Mar 2020 21:17:36:  19000000 \\nINFO  @ Thu, 26 Mar 2020 21:17:40:  20000000 \\nINFO  @ Thu, 26 Mar 2020 21:17:45:  21000000 \\nINFO  @ Thu, 26 Mar 2020 21:17:49:  22000000 \\nINFO  @ Thu, 26 Mar 2020 21:17:54:  23000000 \\nINFO  @ Thu, 26 Mar 2020 21:17:58:  24000000 \\nINFO  @ Thu, 26 Mar 2020 21:18:03:  25000000 \\nINFO  @ Thu, 26 Mar 2020 21:18:07:  26000000 \\nINFO  @ Thu, 26 Mar 2020 21:18:11:  27000000 \\nINFO  @ Thu, 26 Mar 2020 21:18:16:  28000000 \\nINFO  @ Thu, 26 Mar 2020 21:18:20:  29000000 \\nINFO  @ Thu, 26 Mar 2020 21:18:25:  30000000 \\nINFO  @ Thu, 26 Mar 2020 21:18:29:  31000000 \\nINFO  @ Thu, 26 Mar 2020 21:18:33: 31782919 reads have been read. \\nINFO  @ Thu, 26 Mar 2020 21:18:33: #1.2 read input tags... \\nINFO  @ Thu, 26 Mar 2020 21:18:36:  1000000 \\nINFO  @ Thu, 26 Mar 2020 21:18:38:  2000000 \\nINFO  @ Thu, 26 Mar 2020 21:18:40:  3000000 \\nINFO  @ Thu, 26 Mar 2020 21:18:43:  4000000 \\nINFO  @ Thu, 26 Mar 2020 21:18:45:  5000000 \\nINFO  @ Thu, 26 Mar 2020 21:18:47:  6000000 \\nINFO  @ Thu, 26 Mar 2020 21:18:49:  7000000 \\nINFO  @ Thu, 26 Mar 2020 21:18:52:  8000000 \\nINFO  @ Thu, 26 Mar 2020 21:18:54:  9000000 \\nINFO  @ Thu, 26 Mar 2020 21:18:56:  10000000 \\nINFO  @ Thu, 26 Mar 2020 21:18:59:  11000000 \\nINFO  @ Thu, 26 Mar 2020 21:19:01:  12000000 \\nINFO  @ Thu, 26 Mar 2020 21:19:04:  13000000 \\nINFO  @ Thu, 26 Mar 2020 21:19:07:  14000000 \\nINFO  @ Thu, 26 Mar 2020 21:19:09:  15000000 \\nINFO  @ Thu, 26 Mar 2020 21:19:12:  16000000 \\nINFO  @ Thu, 26 Mar 2020 21:19:14:  17000000 \\nINFO  @ Thu, 26 Mar 2020 21:19:16:  18000000 \\nINFO  @ Thu, 26 Mar 2020 21:19:19:  19000000 \\nINFO  @ Thu, 26 Mar 2020 21:19:21:  20000000 \\nINFO  @ Thu, 26 Mar 2020 21:19:23:  21000000 \\nINFO  @ Thu, 26 Mar 2020 21:19:25:  22000000 \\nINFO  @ Thu, 26 Mar 2020 21:19:28:  23000000 \\nINFO  @ Thu, 26 Mar 2020 21:19:30:  24000000 \\nINFO  @ Thu, 26 Mar 2020 21:19:32:  25000000 \\nINFO  @ Thu, 26 Mar 2020 21:19:35:  26000000 \\nINFO  @ Thu, 26 Mar 2020 21:19:37:  27000000 \\nINFO  @ Thu, 26 Mar 2020 21:19:39:  28000000 \\nINFO  @ Thu, 26 Mar 2020 21:19:42:  29000000 \\nINFO  @ Thu, 26 Mar 2020 21:19:44:  30000000 \\nINFO  @ Thu, 26 Mar 2020 21:19:46:  31000000 \\nINFO  @ Thu, 26 Mar 2020 21:19:49:  32000000 \\nINFO  @ Thu, 26 Mar 2020 21:19:51:  33000000 \\nINFO  @ Thu, 26 Mar 2020 21:19:53:  34000000 \\nINFO  @ Thu, 26 Mar 2020 21:19:55: 34520959 reads have been read. \\nINFO  @ Thu, 26 Mar 2020 21:19:55: #1 tag size is determined as 49 bps \\nINFO  @ Thu, 26 Mar 2020 21:19:55: #1 tag size = 49.0 \\nINFO  @ Thu, 26 Mar 2020 21:19:55: #1  total tags in treatment: 31782919 \\nINFO  @ Thu, 26 Mar 2020 21:19:55: #1 user defined the maximum tags... \\nINFO  @ Thu, 26 Mar 2020 21:19:55: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 26 Mar 2020 21:19:55: #1  tags after filtering in treatment: 31226763 \\nINFO  @ Thu, 26 Mar 2020 21:19:55: #1  Redundant rate of treatment: 0.02 \\nINFO  @ Thu, 26 Mar 2020 21:19:55: #1  total tags in control: 34520959 \\nINFO  @ Thu, 26 Mar 2020 21:19:55: #1 user defined the maximum tags... \\nINFO  @ Thu, 26 Mar 2020 21:19:55: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 26 Mar 2020 21:19:55: #1  tags after filtering in control: 34516729 \\nINFO  @ Thu, 26 Mar 2020 21:19:55: #1  Redundant rate of control: 0.00 \\nINFO  @ Thu, 26 Mar 2020 21:19:55: #1 finished! \\nINFO  @ Thu, 26 Mar 2020 21:19:55: #2 Build Peak Model... \\nINFO  @ Thu, 26 Mar 2020 21:19:55: #2 Skipped... \\nINFO  @ Thu, 26 Mar 2020 21:19:55: #2 Use 194 as fragment length \\nINFO  @ Thu, 26 Mar 2020 21:19:55: #3 Call peaks... \\nINFO  @ Thu, 26 Mar 2020 21:19:55: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 26 Mar 2020 21:22:57: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 26 Mar 2020 21:22:57: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1_treat_pileup.bdg \\nINFO  @ Thu, 26 Mar 2020 21:22:57: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1_control_lambda.bdg \\nINFO  @ Thu, 26 Mar 2020 21:22:57: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 26 Mar 2020 21:22:57: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 26 Mar 2020 21:25:39: #4 Write output xls file... ../../data/IRF2BP2_degraded_3/diffData/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1_peaks.xls \\nINFO  @ Thu, 26 Mar 2020 21:25:40: #4 Write peak in narrowPeak format file... ../../data/IRF2BP2_degraded_3/diffData/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1_peaks.narrowPeak \\nINFO  @ Thu, 26 Mar 2020 21:25:40: #4 Write summits bed file... ../../data/IRF2BP2_degraded_3/diffData/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1_summits.bed \\nINFO  @ Thu, 26 Mar 2020 21:25:40: Done! \\n'\n",
      "b'INFO  @ Thu, 26 Mar 2020 21:25:40: \\n# Command line: callpeak -B -t ../../data/IRF2BP2_degraded_3/mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 194 -n mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1 --outdir ../../data/IRF2BP2_degraded_3/diffData/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/IRF2BP2_degraded_3/mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Thu, 26 Mar 2020 21:25:40: #1 read tag files... \\nINFO  @ Thu, 26 Mar 2020 21:25:40: #1 read treatment tags... \\nINFO  @ Thu, 26 Mar 2020 21:25:45:  1000000 \\nINFO  @ Thu, 26 Mar 2020 21:25:50:  2000000 \\nINFO  @ Thu, 26 Mar 2020 21:25:55:  3000000 \\nINFO  @ Thu, 26 Mar 2020 21:25:59:  4000000 \\nINFO  @ Thu, 26 Mar 2020 21:26:04:  5000000 \\nINFO  @ Thu, 26 Mar 2020 21:26:09:  6000000 \\nINFO  @ Thu, 26 Mar 2020 21:26:14:  7000000 \\nINFO  @ Thu, 26 Mar 2020 21:26:19:  8000000 \\nINFO  @ Thu, 26 Mar 2020 21:26:24:  9000000 \\nINFO  @ Thu, 26 Mar 2020 21:26:28:  10000000 \\nINFO  @ Thu, 26 Mar 2020 21:26:33:  11000000 \\nINFO  @ Thu, 26 Mar 2020 21:26:38:  12000000 \\nINFO  @ Thu, 26 Mar 2020 21:26:42:  13000000 \\nINFO  @ Thu, 26 Mar 2020 21:26:47:  14000000 \\nINFO  @ Thu, 26 Mar 2020 21:26:51:  15000000 \\nINFO  @ Thu, 26 Mar 2020 21:26:56:  16000000 \\nINFO  @ Thu, 26 Mar 2020 21:27:01:  17000000 \\nINFO  @ Thu, 26 Mar 2020 21:27:06:  18000000 \\nINFO  @ Thu, 26 Mar 2020 21:27:12:  19000000 \\nINFO  @ Thu, 26 Mar 2020 21:27:16:  20000000 \\nINFO  @ Thu, 26 Mar 2020 21:27:21:  21000000 \\nINFO  @ Thu, 26 Mar 2020 21:27:26:  22000000 \\nINFO  @ Thu, 26 Mar 2020 21:27:30:  23000000 \\nINFO  @ Thu, 26 Mar 2020 21:27:35:  24000000 \\nINFO  @ Thu, 26 Mar 2020 21:27:40:  25000000 \\nINFO  @ Thu, 26 Mar 2020 21:27:45:  26000000 \\nINFO  @ Thu, 26 Mar 2020 21:27:46: 26283491 reads have been read. \\nINFO  @ Thu, 26 Mar 2020 21:27:46: #1.2 read input tags... \\nINFO  @ Thu, 26 Mar 2020 21:27:49:  1000000 \\nINFO  @ Thu, 26 Mar 2020 21:27:51:  2000000 \\nINFO  @ Thu, 26 Mar 2020 21:27:53:  3000000 \\nINFO  @ Thu, 26 Mar 2020 21:27:56:  4000000 \\nINFO  @ Thu, 26 Mar 2020 21:27:58:  5000000 \\nINFO  @ Thu, 26 Mar 2020 21:28:00:  6000000 \\nINFO  @ Thu, 26 Mar 2020 21:28:02:  7000000 \\nINFO  @ Thu, 26 Mar 2020 21:28:05:  8000000 \\nINFO  @ Thu, 26 Mar 2020 21:28:07:  9000000 \\nINFO  @ Thu, 26 Mar 2020 21:28:11:  10000000 \\nINFO  @ Thu, 26 Mar 2020 21:28:14:  11000000 \\nINFO  @ Thu, 26 Mar 2020 21:28:17:  12000000 \\nINFO  @ Thu, 26 Mar 2020 21:28:19:  13000000 \\nINFO  @ Thu, 26 Mar 2020 21:28:21:  14000000 \\nINFO  @ Thu, 26 Mar 2020 21:28:24:  15000000 \\nINFO  @ Thu, 26 Mar 2020 21:28:26:  16000000 \\nINFO  @ Thu, 26 Mar 2020 21:28:28:  17000000 \\nINFO  @ Thu, 26 Mar 2020 21:28:30:  18000000 \\nINFO  @ Thu, 26 Mar 2020 21:28:33:  19000000 \\nINFO  @ Thu, 26 Mar 2020 21:28:35:  20000000 \\nINFO  @ Thu, 26 Mar 2020 21:28:37:  21000000 \\nINFO  @ Thu, 26 Mar 2020 21:28:40:  22000000 \\nINFO  @ Thu, 26 Mar 2020 21:28:42:  23000000 \\nINFO  @ Thu, 26 Mar 2020 21:28:44:  24000000 \\nINFO  @ Thu, 26 Mar 2020 21:28:46:  25000000 \\nINFO  @ Thu, 26 Mar 2020 21:28:49:  26000000 \\nINFO  @ Thu, 26 Mar 2020 21:28:51:  27000000 \\nINFO  @ Thu, 26 Mar 2020 21:28:53:  28000000 \\nINFO  @ Thu, 26 Mar 2020 21:28:56:  29000000 \\nINFO  @ Thu, 26 Mar 2020 21:28:58:  30000000 \\nINFO  @ Thu, 26 Mar 2020 21:29:00:  31000000 \\nINFO  @ Thu, 26 Mar 2020 21:29:02:  32000000 \\nINFO  @ Thu, 26 Mar 2020 21:29:05:  33000000 \\nINFO  @ Thu, 26 Mar 2020 21:29:07:  34000000 \\nINFO  @ Thu, 26 Mar 2020 21:29:08: 34520959 reads have been read. \\nINFO  @ Thu, 26 Mar 2020 21:29:09: #1 tag size is determined as 49 bps \\nINFO  @ Thu, 26 Mar 2020 21:29:09: #1 tag size = 49.0 \\nINFO  @ Thu, 26 Mar 2020 21:29:09: #1  total tags in treatment: 26283491 \\nINFO  @ Thu, 26 Mar 2020 21:29:09: #1 user defined the maximum tags... \\nINFO  @ Thu, 26 Mar 2020 21:29:09: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 26 Mar 2020 21:29:09: #1  tags after filtering in treatment: 25882131 \\nINFO  @ Thu, 26 Mar 2020 21:29:09: #1  Redundant rate of treatment: 0.02 \\nINFO  @ Thu, 26 Mar 2020 21:29:09: #1  total tags in control: 34520959 \\nINFO  @ Thu, 26 Mar 2020 21:29:09: #1 user defined the maximum tags... \\nINFO  @ Thu, 26 Mar 2020 21:29:09: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 26 Mar 2020 21:29:09: #1  tags after filtering in control: 34516729 \\nINFO  @ Thu, 26 Mar 2020 21:29:09: #1  Redundant rate of control: 0.00 \\nINFO  @ Thu, 26 Mar 2020 21:29:09: #1 finished! \\nINFO  @ Thu, 26 Mar 2020 21:29:09: #2 Build Peak Model... \\nINFO  @ Thu, 26 Mar 2020 21:29:09: #2 Skipped... \\nINFO  @ Thu, 26 Mar 2020 21:29:09: #2 Use 194 as fragment length \\nINFO  @ Thu, 26 Mar 2020 21:29:09: #3 Call peaks... \\nINFO  @ Thu, 26 Mar 2020 21:29:09: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 26 Mar 2020 21:32:08: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 26 Mar 2020 21:32:08: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1_treat_pileup.bdg \\nINFO  @ Thu, 26 Mar 2020 21:32:08: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1_control_lambda.bdg \\nINFO  @ Thu, 26 Mar 2020 21:32:08: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 26 Mar 2020 21:32:08: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 26 Mar 2020 21:34:49: #4 Write output xls file... ../../data/IRF2BP2_degraded_3/diffData/mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1_peaks.xls \\nINFO  @ Thu, 26 Mar 2020 21:34:49: #4 Write peak in narrowPeak format file... ../../data/IRF2BP2_degraded_3/diffData/mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1_peaks.narrowPeak \\nINFO  @ Thu, 26 Mar 2020 21:34:49: #4 Write summits bed file... ../../data/IRF2BP2_degraded_3/diffData/mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1_summits.bed \\nINFO  @ Thu, 26 Mar 2020 21:34:49: Done! \\n'\n",
      "31226763 25882131\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/IRF2BP2_degraded_3/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1.mLb.clN.sorted.bam ../../data/IRF2BP2_degraded_3/mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/IRF2BP2_degraded_3/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1.mLb.clN.sorted.bam and ../../data/IRF2BP2_degraded_3/mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "217\n",
      "computing the scaling values\n",
      "b'INFO  @ Thu, 26 Mar 2020 21:45:08: \\n# Command line: callpeak -B -t ../../data/IRF2BP2_degraded_3/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 217 -n mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1 --outdir ../../data/IRF2BP2_degraded_3/diffData/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/IRF2BP2_degraded_3/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Thu, 26 Mar 2020 21:45:08: #1 read tag files... \\nINFO  @ Thu, 26 Mar 2020 21:45:08: #1 read treatment tags... \\nINFO  @ Thu, 26 Mar 2020 21:45:12:  1000000 \\nINFO  @ Thu, 26 Mar 2020 21:45:17:  2000000 \\nINFO  @ Thu, 26 Mar 2020 21:45:21:  3000000 \\nINFO  @ Thu, 26 Mar 2020 21:45:25:  4000000 \\nINFO  @ Thu, 26 Mar 2020 21:45:29:  5000000 \\nINFO  @ Thu, 26 Mar 2020 21:45:34:  6000000 \\nINFO  @ Thu, 26 Mar 2020 21:45:38:  7000000 \\nINFO  @ Thu, 26 Mar 2020 21:45:42:  8000000 \\nINFO  @ Thu, 26 Mar 2020 21:45:46:  9000000 \\nINFO  @ Thu, 26 Mar 2020 21:45:51:  10000000 \\nINFO  @ Thu, 26 Mar 2020 21:45:55:  11000000 \\nINFO  @ Thu, 26 Mar 2020 21:45:59:  12000000 \\nINFO  @ Thu, 26 Mar 2020 21:46:04:  13000000 \\nINFO  @ Thu, 26 Mar 2020 21:46:08:  14000000 \\nINFO  @ Thu, 26 Mar 2020 21:46:12:  15000000 \\nINFO  @ Thu, 26 Mar 2020 21:46:16:  16000000 \\nINFO  @ Thu, 26 Mar 2020 21:46:21:  17000000 \\nINFO  @ Thu, 26 Mar 2020 21:46:25:  18000000 \\nINFO  @ Thu, 26 Mar 2020 21:46:29:  19000000 \\nINFO  @ Thu, 26 Mar 2020 21:46:29: 19041551 reads have been read. \\nINFO  @ Thu, 26 Mar 2020 21:46:30: #1.2 read input tags... \\nINFO  @ Thu, 26 Mar 2020 21:46:32:  1000000 \\nINFO  @ Thu, 26 Mar 2020 21:46:34:  2000000 \\nINFO  @ Thu, 26 Mar 2020 21:46:36:  3000000 \\nINFO  @ Thu, 26 Mar 2020 21:46:39:  4000000 \\nINFO  @ Thu, 26 Mar 2020 21:46:41:  5000000 \\nINFO  @ Thu, 26 Mar 2020 21:46:43:  6000000 \\nINFO  @ Thu, 26 Mar 2020 21:46:45:  7000000 \\nINFO  @ Thu, 26 Mar 2020 21:46:47:  8000000 \\nINFO  @ Thu, 26 Mar 2020 21:46:50:  9000000 \\nINFO  @ Thu, 26 Mar 2020 21:46:52:  10000000 \\nINFO  @ Thu, 26 Mar 2020 21:46:54:  11000000 \\nINFO  @ Thu, 26 Mar 2020 21:46:56:  12000000 \\nINFO  @ Thu, 26 Mar 2020 21:46:58:  13000000 \\nINFO  @ Thu, 26 Mar 2020 21:47:01:  14000000 \\nINFO  @ Thu, 26 Mar 2020 21:47:03:  15000000 \\nINFO  @ Thu, 26 Mar 2020 21:47:05:  16000000 \\nINFO  @ Thu, 26 Mar 2020 21:47:07:  17000000 \\nINFO  @ Thu, 26 Mar 2020 21:47:09:  18000000 \\nINFO  @ Thu, 26 Mar 2020 21:47:12:  19000000 \\nINFO  @ Thu, 26 Mar 2020 21:47:14:  20000000 \\nINFO  @ Thu, 26 Mar 2020 21:47:16:  21000000 \\nINFO  @ Thu, 26 Mar 2020 21:47:18:  22000000 \\nINFO  @ Thu, 26 Mar 2020 21:47:21:  23000000 \\nINFO  @ Thu, 26 Mar 2020 21:47:23:  24000000 \\nINFO  @ Thu, 26 Mar 2020 21:47:25:  25000000 \\nINFO  @ Thu, 26 Mar 2020 21:47:27:  26000000 \\nINFO  @ Thu, 26 Mar 2020 21:47:29:  27000000 \\nINFO  @ Thu, 26 Mar 2020 21:47:32:  28000000 \\nINFO  @ Thu, 26 Mar 2020 21:47:34:  29000000 \\nINFO  @ Thu, 26 Mar 2020 21:47:36:  30000000 \\nINFO  @ Thu, 26 Mar 2020 21:47:38:  31000000 \\nINFO  @ Thu, 26 Mar 2020 21:47:41:  32000000 \\nINFO  @ Thu, 26 Mar 2020 21:47:43:  33000000 \\nINFO  @ Thu, 26 Mar 2020 21:47:45:  34000000 \\nINFO  @ Thu, 26 Mar 2020 21:47:46: 34520959 reads have been read. \\nINFO  @ Thu, 26 Mar 2020 21:47:47: #1 tag size is determined as 49 bps \\nINFO  @ Thu, 26 Mar 2020 21:47:47: #1 tag size = 49.0 \\nINFO  @ Thu, 26 Mar 2020 21:47:47: #1  total tags in treatment: 19041551 \\nINFO  @ Thu, 26 Mar 2020 21:47:47: #1 user defined the maximum tags... \\nINFO  @ Thu, 26 Mar 2020 21:47:47: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 26 Mar 2020 21:47:47: #1  tags after filtering in treatment: 18902215 \\nINFO  @ Thu, 26 Mar 2020 21:47:47: #1  Redundant rate of treatment: 0.01 \\nINFO  @ Thu, 26 Mar 2020 21:47:47: #1  total tags in control: 34520959 \\nINFO  @ Thu, 26 Mar 2020 21:47:47: #1 user defined the maximum tags... \\nINFO  @ Thu, 26 Mar 2020 21:47:47: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 26 Mar 2020 21:47:47: #1  tags after filtering in control: 34516729 \\nINFO  @ Thu, 26 Mar 2020 21:47:47: #1  Redundant rate of control: 0.00 \\nINFO  @ Thu, 26 Mar 2020 21:47:47: #1 finished! \\nINFO  @ Thu, 26 Mar 2020 21:47:47: #2 Build Peak Model... \\nINFO  @ Thu, 26 Mar 2020 21:47:47: #2 Skipped... \\nINFO  @ Thu, 26 Mar 2020 21:47:47: #2 Use 217 as fragment length \\nINFO  @ Thu, 26 Mar 2020 21:47:47: #3 Call peaks... \\nINFO  @ Thu, 26 Mar 2020 21:47:47: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 26 Mar 2020 21:50:27: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 26 Mar 2020 21:50:27: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1_treat_pileup.bdg \\nINFO  @ Thu, 26 Mar 2020 21:50:27: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1_control_lambda.bdg \\nINFO  @ Thu, 26 Mar 2020 21:50:27: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 26 Mar 2020 21:50:27: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 26 Mar 2020 21:52:55: #4 Write output xls file... ../../data/IRF2BP2_degraded_3/diffData/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1_peaks.xls \\nINFO  @ Thu, 26 Mar 2020 21:52:55: #4 Write peak in narrowPeak format file... ../../data/IRF2BP2_degraded_3/diffData/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1_peaks.narrowPeak \\nINFO  @ Thu, 26 Mar 2020 21:52:55: #4 Write summits bed file... ../../data/IRF2BP2_degraded_3/diffData/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1_summits.bed \\nINFO  @ Thu, 26 Mar 2020 21:52:55: Done! \\n'\n",
      "b'INFO  @ Thu, 26 Mar 2020 21:52:55: \\n# Command line: callpeak -B -t ../../data/IRF2BP2_degraded_3/mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 217 -n mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1 --outdir ../../data/IRF2BP2_degraded_3/diffData/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/IRF2BP2_degraded_3/mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Thu, 26 Mar 2020 21:52:55: #1 read tag files... \\nINFO  @ Thu, 26 Mar 2020 21:52:55: #1 read treatment tags... \\nINFO  @ Thu, 26 Mar 2020 21:53:00:  1000000 \\nINFO  @ Thu, 26 Mar 2020 21:53:05:  2000000 \\nINFO  @ Thu, 26 Mar 2020 21:53:10:  3000000 \\nINFO  @ Thu, 26 Mar 2020 21:53:15:  4000000 \\nINFO  @ Thu, 26 Mar 2020 21:53:20:  5000000 \\nINFO  @ Thu, 26 Mar 2020 21:53:25:  6000000 \\nINFO  @ Thu, 26 Mar 2020 21:53:30:  7000000 \\nINFO  @ Thu, 26 Mar 2020 21:53:35:  8000000 \\nINFO  @ Thu, 26 Mar 2020 21:53:40:  9000000 \\nINFO  @ Thu, 26 Mar 2020 21:53:45:  10000000 \\nINFO  @ Thu, 26 Mar 2020 21:53:50:  11000000 \\nINFO  @ Thu, 26 Mar 2020 21:53:55:  12000000 \\nINFO  @ Thu, 26 Mar 2020 21:54:00:  13000000 \\nINFO  @ Thu, 26 Mar 2020 21:54:05:  14000000 \\nINFO  @ Thu, 26 Mar 2020 21:54:10:  15000000 \\nINFO  @ Thu, 26 Mar 2020 21:54:15:  16000000 \\nINFO  @ Thu, 26 Mar 2020 21:54:20:  17000000 \\nINFO  @ Thu, 26 Mar 2020 21:54:25:  18000000 \\nINFO  @ Thu, 26 Mar 2020 21:54:30:  19000000 \\nINFO  @ Thu, 26 Mar 2020 21:54:35:  20000000 \\nINFO  @ Thu, 26 Mar 2020 21:54:39:  21000000 \\nINFO  @ Thu, 26 Mar 2020 21:54:44:  22000000 \\nINFO  @ Thu, 26 Mar 2020 21:54:45: 22224499 reads have been read. \\nINFO  @ Thu, 26 Mar 2020 21:54:46: #1.2 read input tags... \\nINFO  @ Thu, 26 Mar 2020 21:54:48:  1000000 \\nINFO  @ Thu, 26 Mar 2020 21:54:50:  2000000 \\nINFO  @ Thu, 26 Mar 2020 21:54:53:  3000000 \\nINFO  @ Thu, 26 Mar 2020 21:54:55:  4000000 \\nINFO  @ Thu, 26 Mar 2020 21:54:57:  5000000 \\nINFO  @ Thu, 26 Mar 2020 21:55:00:  6000000 \\nINFO  @ Thu, 26 Mar 2020 21:55:02:  7000000 \\nINFO  @ Thu, 26 Mar 2020 21:55:04:  8000000 \\nINFO  @ Thu, 26 Mar 2020 21:55:06:  9000000 \\nINFO  @ Thu, 26 Mar 2020 21:55:09:  10000000 \\nINFO  @ Thu, 26 Mar 2020 21:55:11:  11000000 \\nINFO  @ Thu, 26 Mar 2020 21:55:13:  12000000 \\nINFO  @ Thu, 26 Mar 2020 21:55:15:  13000000 \\nINFO  @ Thu, 26 Mar 2020 21:55:18:  14000000 \\nINFO  @ Thu, 26 Mar 2020 21:55:20:  15000000 \\nINFO  @ Thu, 26 Mar 2020 21:55:22:  16000000 \\nINFO  @ Thu, 26 Mar 2020 21:55:25:  17000000 \\nINFO  @ Thu, 26 Mar 2020 21:55:27:  18000000 \\nINFO  @ Thu, 26 Mar 2020 21:55:29:  19000000 \\nINFO  @ Thu, 26 Mar 2020 21:55:31:  20000000 \\nINFO  @ Thu, 26 Mar 2020 21:55:34:  21000000 \\nINFO  @ Thu, 26 Mar 2020 21:55:36:  22000000 \\nINFO  @ Thu, 26 Mar 2020 21:55:38:  23000000 \\nINFO  @ Thu, 26 Mar 2020 21:55:41:  24000000 \\nINFO  @ Thu, 26 Mar 2020 21:55:43:  25000000 \\nINFO  @ Thu, 26 Mar 2020 21:55:45:  26000000 \\nINFO  @ Thu, 26 Mar 2020 21:55:48:  27000000 \\nINFO  @ Thu, 26 Mar 2020 21:55:50:  28000000 \\nINFO  @ Thu, 26 Mar 2020 21:55:52:  29000000 \\nINFO  @ Thu, 26 Mar 2020 21:55:55:  30000000 \\nINFO  @ Thu, 26 Mar 2020 21:55:57:  31000000 \\nINFO  @ Thu, 26 Mar 2020 21:55:59:  32000000 \\nINFO  @ Thu, 26 Mar 2020 21:56:02:  33000000 \\nINFO  @ Thu, 26 Mar 2020 21:56:04:  34000000 \\nINFO  @ Thu, 26 Mar 2020 21:56:05: 34520959 reads have been read. \\nINFO  @ Thu, 26 Mar 2020 21:56:05: #1 tag size is determined as 48 bps \\nINFO  @ Thu, 26 Mar 2020 21:56:05: #1 tag size = 48.0 \\nINFO  @ Thu, 26 Mar 2020 21:56:05: #1  total tags in treatment: 22224499 \\nINFO  @ Thu, 26 Mar 2020 21:56:05: #1 user defined the maximum tags... \\nINFO  @ Thu, 26 Mar 2020 21:56:05: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 26 Mar 2020 21:56:06: #1  tags after filtering in treatment: 22035224 \\nINFO  @ Thu, 26 Mar 2020 21:56:06: #1  Redundant rate of treatment: 0.01 \\nINFO  @ Thu, 26 Mar 2020 21:56:06: #1  total tags in control: 34520959 \\nINFO  @ Thu, 26 Mar 2020 21:56:06: #1 user defined the maximum tags... \\nINFO  @ Thu, 26 Mar 2020 21:56:06: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 26 Mar 2020 21:56:06: #1  tags after filtering in control: 34516729 \\nINFO  @ Thu, 26 Mar 2020 21:56:06: #1  Redundant rate of control: 0.00 \\nINFO  @ Thu, 26 Mar 2020 21:56:06: #1 finished! \\nINFO  @ Thu, 26 Mar 2020 21:56:06: #2 Build Peak Model... \\nINFO  @ Thu, 26 Mar 2020 21:56:06: #2 Skipped... \\nINFO  @ Thu, 26 Mar 2020 21:56:06: #2 Use 217 as fragment length \\nINFO  @ Thu, 26 Mar 2020 21:56:06: #3 Call peaks... \\nINFO  @ Thu, 26 Mar 2020 21:56:06: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 26 Mar 2020 21:59:35: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 26 Mar 2020 21:59:35: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1_treat_pileup.bdg \\nINFO  @ Thu, 26 Mar 2020 21:59:35: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1_control_lambda.bdg \\nINFO  @ Thu, 26 Mar 2020 21:59:35: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 26 Mar 2020 21:59:35: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 26 Mar 2020 22:02:28: #4 Write output xls file... ../../data/IRF2BP2_degraded_3/diffData/mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1_peaks.xls \\nINFO  @ Thu, 26 Mar 2020 22:02:28: #4 Write peak in narrowPeak format file... ../../data/IRF2BP2_degraded_3/diffData/mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1_peaks.narrowPeak \\nINFO  @ Thu, 26 Mar 2020 22:02:28: #4 Write summits bed file... ../../data/IRF2BP2_degraded_3/diffData/mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1_summits.bed \\nINFO  @ Thu, 26 Mar 2020 22:02:28: Done! \\n'\n",
      "18902215 22035224\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/IRF2BP2_degraded_3/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1.mLb.clN.sorted.bam ../../data/IRF2BP2_degraded_3/mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/IRF2BP2_degraded_3/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1.mLb.clN.sorted.bam and ../../data/IRF2BP2_degraded_3/mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "217\n",
      "computing the scaling values\n",
      "b'INFO  @ Thu, 26 Mar 2020 22:11:18: \\n# Command line: callpeak -B -t ../../data/IRF2BP2_degraded_3/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 217 -n mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1 --outdir ../../data/IRF2BP2_degraded_3/diffData/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/IRF2BP2_degraded_3/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Thu, 26 Mar 2020 22:11:18: #1 read tag files... \\nINFO  @ Thu, 26 Mar 2020 22:11:18: #1 read treatment tags... \\nINFO  @ Thu, 26 Mar 2020 22:11:22:  1000000 \\nINFO  @ Thu, 26 Mar 2020 22:11:27:  2000000 \\nINFO  @ Thu, 26 Mar 2020 22:11:31:  3000000 \\nINFO  @ Thu, 26 Mar 2020 22:11:35:  4000000 \\nINFO  @ Thu, 26 Mar 2020 22:11:39:  5000000 \\nINFO  @ Thu, 26 Mar 2020 22:11:43:  6000000 \\nINFO  @ Thu, 26 Mar 2020 22:11:47:  7000000 \\nINFO  @ Thu, 26 Mar 2020 22:11:52:  8000000 \\nINFO  @ Thu, 26 Mar 2020 22:11:56:  9000000 \\nINFO  @ Thu, 26 Mar 2020 22:12:00:  10000000 \\nINFO  @ Thu, 26 Mar 2020 22:12:04:  11000000 \\nINFO  @ Thu, 26 Mar 2020 22:12:08:  12000000 \\nINFO  @ Thu, 26 Mar 2020 22:12:13:  13000000 \\nINFO  @ Thu, 26 Mar 2020 22:12:17:  14000000 \\nINFO  @ Thu, 26 Mar 2020 22:12:21:  15000000 \\nINFO  @ Thu, 26 Mar 2020 22:12:25:  16000000 \\nINFO  @ Thu, 26 Mar 2020 22:12:29: 16904143 reads have been read. \\nINFO  @ Thu, 26 Mar 2020 22:12:29: #1.2 read input tags... \\nINFO  @ Thu, 26 Mar 2020 22:12:31:  1000000 \\nINFO  @ Thu, 26 Mar 2020 22:12:33:  2000000 \\nINFO  @ Thu, 26 Mar 2020 22:12:35:  3000000 \\nINFO  @ Thu, 26 Mar 2020 22:12:37:  4000000 \\nINFO  @ Thu, 26 Mar 2020 22:12:40:  5000000 \\nINFO  @ Thu, 26 Mar 2020 22:12:42:  6000000 \\nINFO  @ Thu, 26 Mar 2020 22:12:44:  7000000 \\nINFO  @ Thu, 26 Mar 2020 22:12:46:  8000000 \\nINFO  @ Thu, 26 Mar 2020 22:12:48:  9000000 \\nINFO  @ Thu, 26 Mar 2020 22:12:50:  10000000 \\nINFO  @ Thu, 26 Mar 2020 22:12:52:  11000000 \\nINFO  @ Thu, 26 Mar 2020 22:12:55:  12000000 \\nINFO  @ Thu, 26 Mar 2020 22:12:57:  13000000 \\nINFO  @ Thu, 26 Mar 2020 22:12:59:  14000000 \\nINFO  @ Thu, 26 Mar 2020 22:13:01:  15000000 \\nINFO  @ Thu, 26 Mar 2020 22:13:03:  16000000 \\nINFO  @ Thu, 26 Mar 2020 22:13:05:  17000000 \\nINFO  @ Thu, 26 Mar 2020 22:13:08:  18000000 \\nINFO  @ Thu, 26 Mar 2020 22:13:10:  19000000 \\nINFO  @ Thu, 26 Mar 2020 22:13:12:  20000000 \\nINFO  @ Thu, 26 Mar 2020 22:13:14:  21000000 \\nINFO  @ Thu, 26 Mar 2020 22:13:16:  22000000 \\nINFO  @ Thu, 26 Mar 2020 22:13:18:  23000000 \\nINFO  @ Thu, 26 Mar 2020 22:13:20:  24000000 \\nINFO  @ Thu, 26 Mar 2020 22:13:23:  25000000 \\nINFO  @ Thu, 26 Mar 2020 22:13:25:  26000000 \\nINFO  @ Thu, 26 Mar 2020 22:13:27:  27000000 \\nINFO  @ Thu, 26 Mar 2020 22:13:29:  28000000 \\nINFO  @ Thu, 26 Mar 2020 22:13:31:  29000000 \\nINFO  @ Thu, 26 Mar 2020 22:13:33:  30000000 \\nINFO  @ Thu, 26 Mar 2020 22:13:36:  31000000 \\nINFO  @ Thu, 26 Mar 2020 22:13:38:  32000000 \\nINFO  @ Thu, 26 Mar 2020 22:13:40:  33000000 \\nINFO  @ Thu, 26 Mar 2020 22:13:42:  34000000 \\nINFO  @ Thu, 26 Mar 2020 22:13:43: 34520959 reads have been read. \\nINFO  @ Thu, 26 Mar 2020 22:13:44: #1 tag size is determined as 48 bps \\nINFO  @ Thu, 26 Mar 2020 22:13:44: #1 tag size = 48.0 \\nINFO  @ Thu, 26 Mar 2020 22:13:44: #1  total tags in treatment: 16904143 \\nINFO  @ Thu, 26 Mar 2020 22:13:44: #1 user defined the maximum tags... \\nINFO  @ Thu, 26 Mar 2020 22:13:44: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 26 Mar 2020 22:13:44: #1  tags after filtering in treatment: 16490251 \\nINFO  @ Thu, 26 Mar 2020 22:13:44: #1  Redundant rate of treatment: 0.02 \\nINFO  @ Thu, 26 Mar 2020 22:13:44: #1  total tags in control: 34520959 \\nINFO  @ Thu, 26 Mar 2020 22:13:44: #1 user defined the maximum tags... \\nINFO  @ Thu, 26 Mar 2020 22:13:44: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 26 Mar 2020 22:13:44: #1  tags after filtering in control: 34516729 \\nINFO  @ Thu, 26 Mar 2020 22:13:44: #1  Redundant rate of control: 0.00 \\nINFO  @ Thu, 26 Mar 2020 22:13:44: #1 finished! \\nINFO  @ Thu, 26 Mar 2020 22:13:44: #2 Build Peak Model... \\nINFO  @ Thu, 26 Mar 2020 22:13:44: #2 Skipped... \\nINFO  @ Thu, 26 Mar 2020 22:13:44: #2 Use 217 as fragment length \\nINFO  @ Thu, 26 Mar 2020 22:13:44: #3 Call peaks... \\nINFO  @ Thu, 26 Mar 2020 22:13:44: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 26 Mar 2020 22:16:22: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 26 Mar 2020 22:16:22: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1_treat_pileup.bdg \\nINFO  @ Thu, 26 Mar 2020 22:16:22: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1_control_lambda.bdg \\nINFO  @ Thu, 26 Mar 2020 22:16:22: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 26 Mar 2020 22:16:22: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 26 Mar 2020 22:18:34: #4 Write output xls file... ../../data/IRF2BP2_degraded_3/diffData/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1_peaks.xls \\nINFO  @ Thu, 26 Mar 2020 22:18:35: #4 Write peak in narrowPeak format file... ../../data/IRF2BP2_degraded_3/diffData/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1_peaks.narrowPeak \\nINFO  @ Thu, 26 Mar 2020 22:18:35: #4 Write summits bed file... ../../data/IRF2BP2_degraded_3/diffData/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1_summits.bed \\nINFO  @ Thu, 26 Mar 2020 22:18:35: Done! \\n'\n",
      "b'INFO  @ Thu, 26 Mar 2020 22:18:35: \\n# Command line: callpeak -B -t ../../data/IRF2BP2_degraded_3/mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 217 -n mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1 --outdir ../../data/IRF2BP2_degraded_3/diffData/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/IRF2BP2_degraded_3/mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Thu, 26 Mar 2020 22:18:35: #1 read tag files... \\nINFO  @ Thu, 26 Mar 2020 22:18:35: #1 read treatment tags... \\nINFO  @ Thu, 26 Mar 2020 22:18:40:  1000000 \\nINFO  @ Thu, 26 Mar 2020 22:18:44:  2000000 \\nINFO  @ Thu, 26 Mar 2020 22:18:49:  3000000 \\nINFO  @ Thu, 26 Mar 2020 22:18:53:  4000000 \\nINFO  @ Thu, 26 Mar 2020 22:18:58:  5000000 \\nINFO  @ Thu, 26 Mar 2020 22:19:03:  6000000 \\nINFO  @ Thu, 26 Mar 2020 22:19:07:  7000000 \\nINFO  @ Thu, 26 Mar 2020 22:19:12:  8000000 \\nINFO  @ Thu, 26 Mar 2020 22:19:16:  9000000 \\nINFO  @ Thu, 26 Mar 2020 22:19:20:  10000000 \\nINFO  @ Thu, 26 Mar 2020 22:19:25:  11000000 \\nINFO  @ Thu, 26 Mar 2020 22:19:29:  12000000 \\nINFO  @ Thu, 26 Mar 2020 22:19:34:  13000000 \\nINFO  @ Thu, 26 Mar 2020 22:19:39:  14000000 \\nINFO  @ Thu, 26 Mar 2020 22:19:43:  15000000 \\nINFO  @ Thu, 26 Mar 2020 22:19:48:  16000000 \\nINFO  @ Thu, 26 Mar 2020 22:19:52:  17000000 \\nINFO  @ Thu, 26 Mar 2020 22:19:57:  18000000 \\nINFO  @ Thu, 26 Mar 2020 22:20:01:  19000000 \\nINFO  @ Thu, 26 Mar 2020 22:20:06:  20000000 \\nINFO  @ Thu, 26 Mar 2020 22:20:10:  21000000 \\nINFO  @ Thu, 26 Mar 2020 22:20:14: 21857234 reads have been read. \\nINFO  @ Thu, 26 Mar 2020 22:20:14: #1.2 read input tags... \\nINFO  @ Thu, 26 Mar 2020 22:20:16:  1000000 \\nINFO  @ Thu, 26 Mar 2020 22:20:19:  2000000 \\nINFO  @ Thu, 26 Mar 2020 22:20:21:  3000000 \\nINFO  @ Thu, 26 Mar 2020 22:20:23:  4000000 \\nINFO  @ Thu, 26 Mar 2020 22:20:25:  5000000 \\nINFO  @ Thu, 26 Mar 2020 22:20:27:  6000000 \\nINFO  @ Thu, 26 Mar 2020 22:20:29:  7000000 \\nINFO  @ Thu, 26 Mar 2020 22:20:32:  8000000 \\nINFO  @ Thu, 26 Mar 2020 22:20:34:  9000000 \\nINFO  @ Thu, 26 Mar 2020 22:20:36:  10000000 \\nINFO  @ Thu, 26 Mar 2020 22:20:38:  11000000 \\nINFO  @ Thu, 26 Mar 2020 22:20:40:  12000000 \\nINFO  @ Thu, 26 Mar 2020 22:20:42:  13000000 \\nINFO  @ Thu, 26 Mar 2020 22:20:44:  14000000 \\nINFO  @ Thu, 26 Mar 2020 22:20:47:  15000000 \\nINFO  @ Thu, 26 Mar 2020 22:20:49:  16000000 \\nINFO  @ Thu, 26 Mar 2020 22:20:51:  17000000 \\nINFO  @ Thu, 26 Mar 2020 22:20:53:  18000000 \\nINFO  @ Thu, 26 Mar 2020 22:20:55:  19000000 \\nINFO  @ Thu, 26 Mar 2020 22:20:57:  20000000 \\nINFO  @ Thu, 26 Mar 2020 22:21:00:  21000000 \\nINFO  @ Thu, 26 Mar 2020 22:21:02:  22000000 \\nINFO  @ Thu, 26 Mar 2020 22:21:04:  23000000 \\nINFO  @ Thu, 26 Mar 2020 22:21:06:  24000000 \\nINFO  @ Thu, 26 Mar 2020 22:21:08:  25000000 \\nINFO  @ Thu, 26 Mar 2020 22:21:10:  26000000 \\nINFO  @ Thu, 26 Mar 2020 22:21:12:  27000000 \\nINFO  @ Thu, 26 Mar 2020 22:21:15:  28000000 \\nINFO  @ Thu, 26 Mar 2020 22:21:17:  29000000 \\nINFO  @ Thu, 26 Mar 2020 22:21:19:  30000000 \\nINFO  @ Thu, 26 Mar 2020 22:21:21:  31000000 \\nINFO  @ Thu, 26 Mar 2020 22:21:23:  32000000 \\nINFO  @ Thu, 26 Mar 2020 22:21:25:  33000000 \\nINFO  @ Thu, 26 Mar 2020 22:21:28:  34000000 \\nINFO  @ Thu, 26 Mar 2020 22:21:29: 34520959 reads have been read. \\nINFO  @ Thu, 26 Mar 2020 22:21:29: #1 tag size is determined as 49 bps \\nINFO  @ Thu, 26 Mar 2020 22:21:29: #1 tag size = 49.0 \\nINFO  @ Thu, 26 Mar 2020 22:21:29: #1  total tags in treatment: 21857234 \\nINFO  @ Thu, 26 Mar 2020 22:21:29: #1 user defined the maximum tags... \\nINFO  @ Thu, 26 Mar 2020 22:21:29: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 26 Mar 2020 22:21:29: #1  tags after filtering in treatment: 21182237 \\nINFO  @ Thu, 26 Mar 2020 22:21:29: #1  Redundant rate of treatment: 0.03 \\nINFO  @ Thu, 26 Mar 2020 22:21:29: #1  total tags in control: 34520959 \\nINFO  @ Thu, 26 Mar 2020 22:21:29: #1 user defined the maximum tags... \\nINFO  @ Thu, 26 Mar 2020 22:21:29: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 26 Mar 2020 22:21:29: #1  tags after filtering in control: 34516729 \\nINFO  @ Thu, 26 Mar 2020 22:21:29: #1  Redundant rate of control: 0.00 \\nINFO  @ Thu, 26 Mar 2020 22:21:29: #1 finished! \\nINFO  @ Thu, 26 Mar 2020 22:21:29: #2 Build Peak Model... \\nINFO  @ Thu, 26 Mar 2020 22:21:29: #2 Skipped... \\nINFO  @ Thu, 26 Mar 2020 22:21:29: #2 Use 217 as fragment length \\nINFO  @ Thu, 26 Mar 2020 22:21:29: #3 Call peaks... \\nINFO  @ Thu, 26 Mar 2020 22:21:29: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 26 Mar 2020 22:24:15: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 26 Mar 2020 22:24:15: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1_treat_pileup.bdg \\nINFO  @ Thu, 26 Mar 2020 22:24:15: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1_control_lambda.bdg \\nINFO  @ Thu, 26 Mar 2020 22:24:15: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 26 Mar 2020 22:24:15: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 26 Mar 2020 22:26:39: #4 Write output xls file... ../../data/IRF2BP2_degraded_3/diffData/mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1_peaks.xls \\nINFO  @ Thu, 26 Mar 2020 22:26:39: #4 Write peak in narrowPeak format file... ../../data/IRF2BP2_degraded_3/diffData/mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1_peaks.narrowPeak \\nINFO  @ Thu, 26 Mar 2020 22:26:39: #4 Write summits bed file... ../../data/IRF2BP2_degraded_3/diffData/mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1_summits.bed \\nINFO  @ Thu, 26 Mar 2020 22:26:39: Done! \\n'\n",
      "16490251 21182237\n",
      "doing differential peak binding\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/IRF2BP2_degraded_3/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1.mLb.clN.sorted.bam ../../data/IRF2BP2_degraded_3/mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/IRF2BP2_degraded_3/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1.mLb.clN.sorted.bam and ../../data/IRF2BP2_degraded_3/mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "176\n",
      "computing the scaling values\n",
      "b'INFO  @ Thu, 26 Mar 2020 22:37:13: \\n# Command line: callpeak -B -t ../../data/IRF2BP2_degraded_3/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 176 -n mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1 --outdir ../../data/IRF2BP2_degraded_3/diffData/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/IRF2BP2_degraded_3/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Thu, 26 Mar 2020 22:37:13: #1 read tag files... \\nINFO  @ Thu, 26 Mar 2020 22:37:13: #1 read treatment tags... \\nINFO  @ Thu, 26 Mar 2020 22:37:17:  1000000 \\nINFO  @ Thu, 26 Mar 2020 22:37:21:  2000000 \\nINFO  @ Thu, 26 Mar 2020 22:37:25:  3000000 \\nINFO  @ Thu, 26 Mar 2020 22:37:29:  4000000 \\nINFO  @ Thu, 26 Mar 2020 22:37:33:  5000000 \\nINFO  @ Thu, 26 Mar 2020 22:37:38:  6000000 \\nINFO  @ Thu, 26 Mar 2020 22:37:42:  7000000 \\nINFO  @ Thu, 26 Mar 2020 22:37:46:  8000000 \\nINFO  @ Thu, 26 Mar 2020 22:37:50:  9000000 \\nINFO  @ Thu, 26 Mar 2020 22:37:54:  10000000 \\nINFO  @ Thu, 26 Mar 2020 22:37:58:  11000000 \\nINFO  @ Thu, 26 Mar 2020 22:38:02:  12000000 \\nINFO  @ Thu, 26 Mar 2020 22:38:07:  13000000 \\nINFO  @ Thu, 26 Mar 2020 22:38:11:  14000000 \\nINFO  @ Thu, 26 Mar 2020 22:38:15:  15000000 \\nINFO  @ Thu, 26 Mar 2020 22:38:19:  16000000 \\nINFO  @ Thu, 26 Mar 2020 22:38:23:  17000000 \\nINFO  @ Thu, 26 Mar 2020 22:38:27:  18000000 \\nINFO  @ Thu, 26 Mar 2020 22:38:31:  19000000 \\nINFO  @ Thu, 26 Mar 2020 22:38:36:  20000000 \\nINFO  @ Thu, 26 Mar 2020 22:38:40:  21000000 \\nINFO  @ Thu, 26 Mar 2020 22:38:44:  22000000 \\nINFO  @ Thu, 26 Mar 2020 22:38:48:  23000000 \\nINFO  @ Thu, 26 Mar 2020 22:38:52:  24000000 \\nINFO  @ Thu, 26 Mar 2020 22:38:56:  25000000 \\nINFO  @ Thu, 26 Mar 2020 22:39:00:  26000000 \\nINFO  @ Thu, 26 Mar 2020 22:39:05:  27000000 \\nINFO  @ Thu, 26 Mar 2020 22:39:09:  28000000 \\nINFO  @ Thu, 26 Mar 2020 22:39:13:  29000000 \\nINFO  @ Thu, 26 Mar 2020 22:39:17:  30000000 \\nINFO  @ Thu, 26 Mar 2020 22:39:21:  31000000 \\nINFO  @ Thu, 26 Mar 2020 22:39:25:  32000000 \\nINFO  @ Thu, 26 Mar 2020 22:39:29:  33000000 \\nINFO  @ Thu, 26 Mar 2020 22:39:33:  34000000 \\nINFO  @ Thu, 26 Mar 2020 22:39:37:  35000000 \\nINFO  @ Thu, 26 Mar 2020 22:39:42:  36000000 \\nINFO  @ Thu, 26 Mar 2020 22:39:45: 36655087 reads have been read. \\nINFO  @ Thu, 26 Mar 2020 22:39:45: #1.2 read input tags... \\nINFO  @ Thu, 26 Mar 2020 22:39:47:  1000000 \\nINFO  @ Thu, 26 Mar 2020 22:39:50:  2000000 \\nINFO  @ Thu, 26 Mar 2020 22:39:52:  3000000 \\nINFO  @ Thu, 26 Mar 2020 22:39:54:  4000000 \\nINFO  @ Thu, 26 Mar 2020 22:39:56:  5000000 \\nINFO  @ Thu, 26 Mar 2020 22:40:00:  6000000 \\nINFO  @ Thu, 26 Mar 2020 22:40:03:  7000000 \\nINFO  @ Thu, 26 Mar 2020 22:40:05:  8000000 \\nINFO  @ Thu, 26 Mar 2020 22:40:07:  9000000 \\nINFO  @ Thu, 26 Mar 2020 22:40:10:  10000000 \\nINFO  @ Thu, 26 Mar 2020 22:40:12:  11000000 \\nINFO  @ Thu, 26 Mar 2020 22:40:14:  12000000 \\nINFO  @ Thu, 26 Mar 2020 22:40:16:  13000000 \\nINFO  @ Thu, 26 Mar 2020 22:40:18:  14000000 \\nINFO  @ Thu, 26 Mar 2020 22:40:21:  15000000 \\nINFO  @ Thu, 26 Mar 2020 22:40:23:  16000000 \\nINFO  @ Thu, 26 Mar 2020 22:40:25:  17000000 \\nINFO  @ Thu, 26 Mar 2020 22:40:27:  18000000 \\nINFO  @ Thu, 26 Mar 2020 22:40:29:  19000000 \\nINFO  @ Thu, 26 Mar 2020 22:40:32:  20000000 \\nINFO  @ Thu, 26 Mar 2020 22:40:34:  21000000 \\nINFO  @ Thu, 26 Mar 2020 22:40:36:  22000000 \\nINFO  @ Thu, 26 Mar 2020 22:40:38:  23000000 \\nINFO  @ Thu, 26 Mar 2020 22:40:41:  24000000 \\nINFO  @ Thu, 26 Mar 2020 22:40:43:  25000000 \\nINFO  @ Thu, 26 Mar 2020 22:40:45:  26000000 \\nINFO  @ Thu, 26 Mar 2020 22:40:47:  27000000 \\nINFO  @ Thu, 26 Mar 2020 22:40:49:  28000000 \\nINFO  @ Thu, 26 Mar 2020 22:40:52:  29000000 \\nINFO  @ Thu, 26 Mar 2020 22:40:54:  30000000 \\nINFO  @ Thu, 26 Mar 2020 22:40:56:  31000000 \\nINFO  @ Thu, 26 Mar 2020 22:40:58:  32000000 \\nINFO  @ Thu, 26 Mar 2020 22:41:01:  33000000 \\nINFO  @ Thu, 26 Mar 2020 22:41:03:  34000000 \\nINFO  @ Thu, 26 Mar 2020 22:41:04: 34520959 reads have been read. \\nINFO  @ Thu, 26 Mar 2020 22:41:05: #1 tag size is determined as 49 bps \\nINFO  @ Thu, 26 Mar 2020 22:41:05: #1 tag size = 49.0 \\nINFO  @ Thu, 26 Mar 2020 22:41:05: #1  total tags in treatment: 36655087 \\nINFO  @ Thu, 26 Mar 2020 22:41:05: #1 user defined the maximum tags... \\nINFO  @ Thu, 26 Mar 2020 22:41:05: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 26 Mar 2020 22:41:05: #1  tags after filtering in treatment: 30015739 \\nINFO  @ Thu, 26 Mar 2020 22:41:05: #1  Redundant rate of treatment: 0.18 \\nINFO  @ Thu, 26 Mar 2020 22:41:05: #1  total tags in control: 34520959 \\nINFO  @ Thu, 26 Mar 2020 22:41:05: #1 user defined the maximum tags... \\nINFO  @ Thu, 26 Mar 2020 22:41:05: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 26 Mar 2020 22:41:05: #1  tags after filtering in control: 34516729 \\nINFO  @ Thu, 26 Mar 2020 22:41:05: #1  Redundant rate of control: 0.00 \\nINFO  @ Thu, 26 Mar 2020 22:41:05: #1 finished! \\nINFO  @ Thu, 26 Mar 2020 22:41:05: #2 Build Peak Model... \\nINFO  @ Thu, 26 Mar 2020 22:41:05: #2 Skipped... \\nINFO  @ Thu, 26 Mar 2020 22:41:05: #2 Use 176 as fragment length \\nINFO  @ Thu, 26 Mar 2020 22:41:05: #3 Call peaks... \\nINFO  @ Thu, 26 Mar 2020 22:41:05: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 26 Mar 2020 22:43:58: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 26 Mar 2020 22:43:58: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1_treat_pileup.bdg \\nINFO  @ Thu, 26 Mar 2020 22:43:58: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1_control_lambda.bdg \\nINFO  @ Thu, 26 Mar 2020 22:43:58: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 26 Mar 2020 22:43:58: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 26 Mar 2020 22:46:36: #4 Write output xls file... ../../data/IRF2BP2_degraded_3/diffData/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1_peaks.xls \\nINFO  @ Thu, 26 Mar 2020 22:46:37: #4 Write peak in narrowPeak format file... ../../data/IRF2BP2_degraded_3/diffData/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1_peaks.narrowPeak \\nINFO  @ Thu, 26 Mar 2020 22:46:38: #4 Write summits bed file... ../../data/IRF2BP2_degraded_3/diffData/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1_summits.bed \\nINFO  @ Thu, 26 Mar 2020 22:46:38: Done! \\n'\n",
      "b'INFO  @ Thu, 26 Mar 2020 22:46:39: \\n# Command line: callpeak -B -t ../../data/IRF2BP2_degraded_3/mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1.mLb.clN.sorted.bam -c ../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam --nomodel --extsize 176 -n mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1 --outdir ../../data/IRF2BP2_degraded_3/diffData/ -f BAM\\n# ARGUMENTS LIST:\\n# name = mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1\\n# format = BAM\\n# ChIP-seq file = [\\'../../data/IRF2BP2_degraded_3/mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1.mLb.clN.sorted.bam\\']\\n# control file = [\\'../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam\\']\\n# effective genome size = 2.70e+09\\n# band width = 300\\n# model fold = [5, 50]\\n# qvalue cutoff = 5.00e-02\\n# The maximum gap between significant sites is assigned as the read length/tag size.\\n# The minimum length of peaks is assigned as the predicted fragment length \"d\".\\n# Larger dataset will be scaled towards smaller dataset.\\n# Range for calculating regional lambda is: 1000 bps and 10000 bps\\n# Broad region calling is off\\n# Paired-End mode is off\\n \\nINFO  @ Thu, 26 Mar 2020 22:46:39: #1 read tag files... \\nINFO  @ Thu, 26 Mar 2020 22:46:39: #1 read treatment tags... \\nINFO  @ Thu, 26 Mar 2020 22:46:43:  1000000 \\nINFO  @ Thu, 26 Mar 2020 22:46:47:  2000000 \\nINFO  @ Thu, 26 Mar 2020 22:46:51:  3000000 \\nINFO  @ Thu, 26 Mar 2020 22:46:56:  4000000 \\nINFO  @ Thu, 26 Mar 2020 22:47:00:  5000000 \\nINFO  @ Thu, 26 Mar 2020 22:47:04:  6000000 \\nINFO  @ Thu, 26 Mar 2020 22:47:09:  7000000 \\nINFO  @ Thu, 26 Mar 2020 22:47:13:  8000000 \\nINFO  @ Thu, 26 Mar 2020 22:47:17:  9000000 \\nINFO  @ Thu, 26 Mar 2020 22:47:21:  10000000 \\nINFO  @ Thu, 26 Mar 2020 22:47:25:  11000000 \\nINFO  @ Thu, 26 Mar 2020 22:47:29:  12000000 \\nINFO  @ Thu, 26 Mar 2020 22:47:33:  13000000 \\nINFO  @ Thu, 26 Mar 2020 22:47:37:  14000000 \\nINFO  @ Thu, 26 Mar 2020 22:47:42:  15000000 \\nINFO  @ Thu, 26 Mar 2020 22:47:46:  16000000 \\nINFO  @ Thu, 26 Mar 2020 22:47:50:  17000000 \\nINFO  @ Thu, 26 Mar 2020 22:47:54:  18000000 \\nINFO  @ Thu, 26 Mar 2020 22:47:58:  19000000 \\nINFO  @ Thu, 26 Mar 2020 22:48:02:  20000000 \\nINFO  @ Thu, 26 Mar 2020 22:48:06:  21000000 \\nINFO  @ Thu, 26 Mar 2020 22:48:10:  22000000 \\nINFO  @ Thu, 26 Mar 2020 22:48:14:  23000000 \\nINFO  @ Thu, 26 Mar 2020 22:48:18:  24000000 \\nINFO  @ Thu, 26 Mar 2020 22:48:22:  25000000 \\nINFO  @ Thu, 26 Mar 2020 22:48:26:  26000000 \\nINFO  @ Thu, 26 Mar 2020 22:48:30:  27000000 \\nINFO  @ Thu, 26 Mar 2020 22:48:34:  28000000 \\nINFO  @ Thu, 26 Mar 2020 22:48:38:  29000000 \\nINFO  @ Thu, 26 Mar 2020 22:48:43:  30000000 \\nINFO  @ Thu, 26 Mar 2020 22:48:47:  31000000 \\nINFO  @ Thu, 26 Mar 2020 22:48:50:  32000000 \\nINFO  @ Thu, 26 Mar 2020 22:48:55:  33000000 \\nINFO  @ Thu, 26 Mar 2020 22:48:59:  34000000 \\nINFO  @ Thu, 26 Mar 2020 22:49:03:  35000000 \\nINFO  @ Thu, 26 Mar 2020 22:49:07:  36000000 \\nINFO  @ Thu, 26 Mar 2020 22:49:08: 36211494 reads have been read. \\nINFO  @ Thu, 26 Mar 2020 22:49:08: #1.2 read input tags... \\nINFO  @ Thu, 26 Mar 2020 22:49:11:  1000000 \\nINFO  @ Thu, 26 Mar 2020 22:49:13:  2000000 \\nINFO  @ Thu, 26 Mar 2020 22:49:15:  3000000 \\nINFO  @ Thu, 26 Mar 2020 22:49:17:  4000000 \\nINFO  @ Thu, 26 Mar 2020 22:49:19:  5000000 \\nINFO  @ Thu, 26 Mar 2020 22:49:21:  6000000 \\nINFO  @ Thu, 26 Mar 2020 22:49:23:  7000000 \\nINFO  @ Thu, 26 Mar 2020 22:49:25:  8000000 \\nINFO  @ Thu, 26 Mar 2020 22:49:28:  9000000 \\nINFO  @ Thu, 26 Mar 2020 22:49:30:  10000000 \\nINFO  @ Thu, 26 Mar 2020 22:49:32:  11000000 \\nINFO  @ Thu, 26 Mar 2020 22:49:34:  12000000 \\nINFO  @ Thu, 26 Mar 2020 22:49:36:  13000000 \\nINFO  @ Thu, 26 Mar 2020 22:49:38:  14000000 \\nINFO  @ Thu, 26 Mar 2020 22:49:40:  15000000 \\nINFO  @ Thu, 26 Mar 2020 22:49:42:  16000000 \\nINFO  @ Thu, 26 Mar 2020 22:49:45:  17000000 \\nINFO  @ Thu, 26 Mar 2020 22:49:47:  18000000 \\nINFO  @ Thu, 26 Mar 2020 22:49:49:  19000000 \\nINFO  @ Thu, 26 Mar 2020 22:49:51:  20000000 \\nINFO  @ Thu, 26 Mar 2020 22:49:53:  21000000 \\nINFO  @ Thu, 26 Mar 2020 22:49:55:  22000000 \\nINFO  @ Thu, 26 Mar 2020 22:49:57:  23000000 \\nINFO  @ Thu, 26 Mar 2020 22:49:59:  24000000 \\nINFO  @ Thu, 26 Mar 2020 22:50:01:  25000000 \\nINFO  @ Thu, 26 Mar 2020 22:50:04:  26000000 \\nINFO  @ Thu, 26 Mar 2020 22:50:06:  27000000 \\nINFO  @ Thu, 26 Mar 2020 22:50:08:  28000000 \\nINFO  @ Thu, 26 Mar 2020 22:50:10:  29000000 \\nINFO  @ Thu, 26 Mar 2020 22:50:12:  30000000 \\nINFO  @ Thu, 26 Mar 2020 22:50:15:  31000000 \\nINFO  @ Thu, 26 Mar 2020 22:50:17:  32000000 \\nINFO  @ Thu, 26 Mar 2020 22:50:19:  33000000 \\nINFO  @ Thu, 26 Mar 2020 22:50:21:  34000000 \\nINFO  @ Thu, 26 Mar 2020 22:50:22: 34520959 reads have been read. \\nINFO  @ Thu, 26 Mar 2020 22:50:22: #1 tag size is determined as 49 bps \\nINFO  @ Thu, 26 Mar 2020 22:50:22: #1 tag size = 49.0 \\nINFO  @ Thu, 26 Mar 2020 22:50:22: #1  total tags in treatment: 36211494 \\nINFO  @ Thu, 26 Mar 2020 22:50:22: #1 user defined the maximum tags... \\nINFO  @ Thu, 26 Mar 2020 22:50:22: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 26 Mar 2020 22:50:23: #1  tags after filtering in treatment: 29684545 \\nINFO  @ Thu, 26 Mar 2020 22:50:23: #1  Redundant rate of treatment: 0.18 \\nINFO  @ Thu, 26 Mar 2020 22:50:23: #1  total tags in control: 34520959 \\nINFO  @ Thu, 26 Mar 2020 22:50:23: #1 user defined the maximum tags... \\nINFO  @ Thu, 26 Mar 2020 22:50:23: #1 filter out redundant tags at the same location and the same strand by allowing at most 1 tag(s) \\nINFO  @ Thu, 26 Mar 2020 22:50:23: #1  tags after filtering in control: 34516729 \\nINFO  @ Thu, 26 Mar 2020 22:50:23: #1  Redundant rate of control: 0.00 \\nINFO  @ Thu, 26 Mar 2020 22:50:23: #1 finished! \\nINFO  @ Thu, 26 Mar 2020 22:50:23: #2 Build Peak Model... \\nINFO  @ Thu, 26 Mar 2020 22:50:23: #2 Skipped... \\nINFO  @ Thu, 26 Mar 2020 22:50:23: #2 Use 176 as fragment length \\nINFO  @ Thu, 26 Mar 2020 22:50:23: #3 Call peaks... \\nINFO  @ Thu, 26 Mar 2020 22:50:23: #3 Pre-compute pvalue-qvalue table... \\nINFO  @ Thu, 26 Mar 2020 22:53:11: #3 In the peak calling step, the following will be performed simultaneously: \\nINFO  @ Thu, 26 Mar 2020 22:53:11: #3   Write bedGraph files for treatment pileup (after scaling if necessary)... mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1_treat_pileup.bdg \\nINFO  @ Thu, 26 Mar 2020 22:53:11: #3   Write bedGraph files for control lambda (after scaling if necessary)... mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1_control_lambda.bdg \\nINFO  @ Thu, 26 Mar 2020 22:53:11: #3   Pileup will be based on sequencing depth in treatment. \\nINFO  @ Thu, 26 Mar 2020 22:53:11: #3 Call peaks for each chromosome... \\nINFO  @ Thu, 26 Mar 2020 22:55:44: #4 Write output xls file... ../../data/IRF2BP2_degraded_3/diffData/mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1_peaks.xls \\nINFO  @ Thu, 26 Mar 2020 22:55:45: #4 Write peak in narrowPeak format file... ../../data/IRF2BP2_degraded_3/diffData/mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1_peaks.narrowPeak \\nINFO  @ Thu, 26 Mar 2020 22:55:45: #4 Write summits bed file... ../../data/IRF2BP2_degraded_3/diffData/mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1_summits.bed \\nINFO  @ Thu, 26 Mar 2020 22:55:46: Done! \\n'\n",
      "30015739 29684545\n",
      "doing differential peak binding\n"
     ]
    }
   ],
   "source": [
    "# diffPeak on scaled data\n",
    "for i in range(int(len(bams)/2)):\n",
    "    if i <0:\n",
    "        continue\n",
    "    name1 = bams[i]\n",
    "    name2 = bams[7+i]\n",
    "    print(name1,name2)\n",
    "    chip.fullDiffPeak(name1,name2, control1='../../data/diffBinding_hist/INPUT_R1.mLb.clN.sorted.bam', directory = \"../../data/IRF2BP2_degraded_3/diffData/\", res_directory = \"../../data/IRF2BP2_degraded_3/diffPeaks/\", isTF=True, compute_size=True, pairedend=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "scales = [1.0,\n",
    "1.0,\n",
    "0.6977788675,\n",
    "0.5173394083,\n",
    "0.8745375796,\n",
    "1.0,\n",
    "0.8298927961,\n",
    "0.8488122828,\n",
    "0.8825441551,\n",
    "1.0,\n",
    "1.0,\n",
    "1.0,\n",
    "0.8534697536,\n",
    "1.0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "chip.bigWigFrom(bams[1:], \n",
    "                genome='GRCh38',scaling=scales,\n",
    "               numthreads=8)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "mkdir ../../data/IRF2BP2_degraded_3/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! mv diffPeaks ../../data/IRF2BP2_degraded_3\n",
    "! mv diffData ../../data/IRF2BP2_degraded_3"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "!mv bigwig ../../data/recalib_bigwig_3 "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "os.popen('for i in $(ls ../../data/IRF2BP2_degraded_3/diffPeaks); \\\n",
    "            do echo $(wc -l \"../../data/IRF2BP2_degraded_3/diffPeaks/\"$i); \\\n",
    "            done').read().split('\\n')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/recalib_bigwig_3/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1.bw',\n",
       " '../../data/recalib_bigwig_3/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1.bw',\n",
       " '../../data/recalib_bigwig_3/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1.bw',\n",
       " '../../data/recalib_bigwig_3/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1.bw',\n",
       " '../../data/recalib_bigwig_3/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1.bw',\n",
       " '../../data/recalib_bigwig_3/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1.bw',\n",
       " '../../data/recalib_bigwig_3/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1.bw',\n",
       " '../../data/recalib_bigwig_3/mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1.bw',\n",
       " '../../data/recalib_bigwig_3/mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1.bw',\n",
       " '../../data/recalib_bigwig_3/mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1.bw',\n",
       " '../../data/recalib_bigwig_3/mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1.bw',\n",
       " '../../data/recalib_bigwig_3/mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1.bw',\n",
       " '../../data/recalib_bigwig_3/mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1.bw',\n",
       " '../../data/recalib_bigwig_3/mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1.bw']"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bw = ! ls ../../data/recalib_bigwig_3/*.bw\n",
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/IRF2BP2_degraded_3/diffPeaks/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1_vs_mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1_c3.0_cond1.bed',\n",
       " '../../data/IRF2BP2_degraded_3/diffPeaks/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1_vs_mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1_c3.0_cond1.bed',\n",
       " '../../data/IRF2BP2_degraded_3/diffPeaks/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1_vs_mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1_c3.0_cond1.bed',\n",
       " '../../data/IRF2BP2_degraded_3/diffPeaks/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1_vs_mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1_c3.0_cond1.bed',\n",
       " '../../data/IRF2BP2_degraded_3/diffPeaks/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1_vs_mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1_c3.0_cond1.bed',\n",
       " '../../data/IRF2BP2_degraded_3/diffPeaks/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1_vs_mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1_c3.0_cond1.bed',\n",
       " '../../data/IRF2BP2_degraded_3/diffPeaks/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1_vs_mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1_c3.0_cond1.bed']"
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "cond1peak = ! ls ../../data/IRF2BP2_degraded_3/diffPeaks/*cond1.bed\n",
    "cond2peak = ! ls ../../data/IRF2BP2_degraded_3/diffPeaks/*cond2.bed\n",
    "commonpeak = ! ls ../../data/IRF2BP2_degraded_3/diffPeaks/*common.bed\n",
    "cond1peak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"CDK8\",\"BRD4\",\"IRF8\",\"SMC1\",\"MED1\",\"ZEB2\",\"CEBPA\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1.bw ../../data/recalib_bigwig_3/mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1.bw  --referencePoint center --regionsFileName ../../data/IRF2BP2_degraded_3/diffPeaks/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1_vs_mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1_c3.0_cond1.bed ../../data/IRF2BP2_degraded_3/diffPeaks/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1_vs_mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1_c3.0_common.bed ../../data/IRF2BP2_degraded_3/diffPeaks/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1_vs_mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/CDK8_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 14 && plotHeatmap --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/CDK8_mat.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/CDK8_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle CDK8', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1.bw ../../data/recalib_bigwig_3/mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1.bw  --referencePoint center --regionsFileName ../../data/IRF2BP2_degraded_3/diffPeaks/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1_vs_mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1_c3.0_cond1.bed ../../data/IRF2BP2_degraded_3/diffPeaks/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1_vs_mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1_c3.0_common.bed ../../data/IRF2BP2_degraded_3/diffPeaks/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1_vs_mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/BRD4_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 14 && plotHeatmap --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/BRD4_mat.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/BRD4_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle BRD4', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1.bw ../../data/recalib_bigwig_3/mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1.bw  --referencePoint center --regionsFileName ../../data/IRF2BP2_degraded_3/diffPeaks/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1_vs_mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1_c3.0_common.bed ../../data/IRF2BP2_degraded_3/diffPeaks/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1_vs_mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/IRF8_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 14 && plotHeatmap --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/IRF8_mat.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/IRF8_mat.pdf --refPointLabel center --regionsLabel common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle IRF8', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1.bw ../../data/recalib_bigwig_3/mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1.bw  --referencePoint center --regionsFileName ../../data/IRF2BP2_degraded_3/diffPeaks/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1_vs_mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1_c3.0_common.bed ../../data/IRF2BP2_degraded_3/diffPeaks/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1_vs_mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/SMC1_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 14 && plotHeatmap --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/SMC1_mat.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/SMC1_mat.pdf --refPointLabel center --regionsLabel common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle SMC1', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1.bw ../../data/recalib_bigwig_3/mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1.bw  --referencePoint center --regionsFileName ../../data/IRF2BP2_degraded_3/diffPeaks/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1_vs_mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1_c3.0_common.bed ../../data/IRF2BP2_degraded_3/diffPeaks/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1_vs_mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/MED1_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 14 && plotHeatmap --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/MED1_mat.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/MED1_mat.pdf --refPointLabel center --regionsLabel common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle MED1', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1.bw ../../data/recalib_bigwig_3/mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1.bw  --referencePoint center --regionsFileName ../../data/IRF2BP2_degraded_3/diffPeaks/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1_vs_mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1_c3.0_common.bed ../../data/IRF2BP2_degraded_3/diffPeaks/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1_vs_mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/ZEB2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 14 && plotHeatmap --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/ZEB2_mat.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/ZEB2_mat.pdf --refPointLabel center --regionsLabel common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle ZEB2', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1.bw ../../data/recalib_bigwig_3/mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1.bw  --referencePoint center --regionsFileName ../../data/IRF2BP2_degraded_3/diffPeaks/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1_vs_mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1_c3.0_cond1.bed ../../data/IRF2BP2_degraded_3/diffPeaks/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1_vs_mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1_c3.0_common.bed ../../data/IRF2BP2_degraded_3/diffPeaks/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1_vs_mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/CEBPA_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 14 && plotHeatmap --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/CEBPA_mat.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/CEBPA_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle CEBPA', returncode=0, stdout=b'', stderr=b\"One of the groups defined in the bed file is too small.\\nGroups that are too small can't be plotted. \\nWARNING: Group 'mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1_vs_mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1_c3.0_cond1.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n"
     ]
    }
   ],
   "source": [
    "for i in range(int(len(bw)/2)):\n",
    "    if i<0:\n",
    "        continue\n",
    "    name1 = bw[i]\n",
    "    name2 = bw[7+i]\n",
    "    peak = [cond1peak[i], commonpeak[i], cond2peak[i]]\n",
    "    #for val in peak:\n",
    "        #chip.dropWeirdChromosomes(val)\n",
    "    name = names[i]\n",
    "    chip.computePeaksAt(peak, [name1, name2], bigwignames=['DMSO', 'VHL'], refpoint='center', peaknames=['DMSO_peaks', 'common', 'VHL_peaks'], window=3000, folder=\"\", title=name, numthreads=14, name='../../data/IRF2BP2_degraded_3/diffPeaks/'+name+'_mat.pdf', withDeeptools=True, torecompute=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 256,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['CDK8',\n",
       " 'BRD4',\n",
       " 'IRF8',\n",
       " 'SMC1',\n",
       " 'MED1',\n",
       " 'ZEB2',\n",
       " 'CEBPA',\n",
       " 'CDK8_VHL',\n",
       " 'BRD4_VHL',\n",
       " 'IRF8_VHL',\n",
       " 'SMC1_VHL',\n",
       " 'MED1_VHL',\n",
       " 'ZEB2_VHL',\n",
       " 'CEBPA_VHL']"
      ]
     },
     "execution_count": 256,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "names = [\"CDK8\",\"BRD4\",\"IRF8\",\"SMC1\",\"MED1\",\"ZEB2\",\"CEBPA\"]\n",
    "names.extend([i+'_VHL' for i in names])\n",
    "names"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 155,
   "metadata": {},
   "outputs": [],
   "source": [
    "mkdir ../../data/peaks3"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 156,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/macs/narrowPeak/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1_peaks.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/macs/narrowPeak/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1_peaks.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/macs/narrowPeak/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1_peaks.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/macs/narrowPeak/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1_peaks.narrowPeak...\n",
      "- [4 files][ 21.3 MiB/ 21.3 MiB]                                                \n",
      "==> NOTE: You are performing a sequence of gsutil operations that may\n",
      "run significantly faster if you instead use gsutil -m cp ... Please\n",
      "see the -m section under \"gsutil help options\" for further information\n",
      "about when gsutil -m can be advantageous.\n",
      "\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/macs/narrowPeak/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1_peaks.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/macs/narrowPeak/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1_peaks.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/macs/narrowPeak/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1_peaks.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/macs/narrowPeak/mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1_peaks.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/macs/narrowPeak/mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1_peaks.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/macs/narrowPeak/mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1_peaks.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/macs/narrowPeak/mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1_peaks.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/macs/narrowPeak/mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1_peaks.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/macs/narrowPeak/mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1_peaks.narrowPeak...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/macs/narrowPeak/mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1_peaks.narrowPeak...\n",
      "- [14 files][ 92.8 MiB/ 92.8 MiB]                                               \n",
      "Operation completed over 14 objects/92.8 MiB.                                    \n"
     ]
    }
   ],
   "source": [
    "! gsutil cp gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/macs/narrowPeak/*.narrowPeak ../../data/peaks3"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 257,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/peaks3/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1_peaks.narrowPeak',\n",
       " '../../data/peaks3/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1_peaks.narrowPeak',\n",
       " '../../data/peaks3/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1_peaks.narrowPeak',\n",
       " '../../data/peaks3/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1_peaks.narrowPeak',\n",
       " '../../data/peaks3/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1_peaks.narrowPeak',\n",
       " '../../data/peaks3/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1_peaks.narrowPeak',\n",
       " '../../data/peaks3/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1_peaks.narrowPeak',\n",
       " '../../data/peaks3/mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1_peaks.narrowPeak',\n",
       " '../../data/peaks3/mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1_peaks.narrowPeak',\n",
       " '../../data/peaks3/mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1_peaks.narrowPeak',\n",
       " '../../data/peaks3/mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1_peaks.narrowPeak',\n",
       " '../../data/peaks3/mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1_peaks.narrowPeak',\n",
       " '../../data/peaks3/mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1_peaks.narrowPeak',\n",
       " '../../data/peaks3/mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1_peaks.narrowPeak']"
      ]
     },
     "execution_count": 257,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "peaks = ! ls ../../data/peaks3/*\n",
    "peaks"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1.bw  --referencePoint center --regionsFileName ../../data/peaks3/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/CDK8_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/CDK8_mat_profile.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/CDK8_mat_profile.pdf --refPointLabel center --plotTitle CDK8', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1.bw  --referencePoint center --regionsFileName ../../data/peaks3/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/CDK8_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/CDK8_mat_profile_clust3.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/CDK8_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle CDK8', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1.bw  --referencePoint center --regionsFileName ../../data/peaks3/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/BRD4_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/BRD4_mat_profile.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/BRD4_mat_profile.pdf --refPointLabel center --plotTitle BRD4', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1.bw  --referencePoint center --regionsFileName ../../data/peaks3/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/BRD4_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/BRD4_mat_profile_clust3.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/BRD4_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle BRD4', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1.bw  --referencePoint center --regionsFileName ../../data/peaks3/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/IRF8_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/IRF8_mat_profile.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/IRF8_mat_profile.pdf --refPointLabel center --plotTitle IRF8', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1.bw  --referencePoint center --regionsFileName ../../data/peaks3/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/IRF8_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/IRF8_mat_profile_clust3.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/IRF8_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle IRF8', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1.bw  --referencePoint center --regionsFileName ../../data/peaks3/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/SMC1_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/SMC1_mat_profile.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/SMC1_mat_profile.pdf --refPointLabel center --plotTitle SMC1', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1.bw  --referencePoint center --regionsFileName ../../data/peaks3/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/SMC1_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/SMC1_mat_profile_clust3.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/SMC1_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle SMC1', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1.bw  --referencePoint center --regionsFileName ../../data/peaks3/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/MED1_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/MED1_mat_profile.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/MED1_mat_profile.pdf --refPointLabel center --plotTitle MED1', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1.bw  --referencePoint center --regionsFileName ../../data/peaks3/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/MED1_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/MED1_mat_profile_clust3.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/MED1_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle MED1', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1.bw  --referencePoint center --regionsFileName ../../data/peaks3/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/ZEB2_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/ZEB2_mat_profile.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/ZEB2_mat_profile.pdf --refPointLabel center --plotTitle ZEB2', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1.bw  --referencePoint center --regionsFileName ../../data/peaks3/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/ZEB2_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/ZEB2_mat_profile_clust3.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/ZEB2_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle ZEB2', returncode=0, stdout=b'', stderr=b\"WARNING: Group 'cluster_1' is too small for plotting, you might want to remove it. \\n\")\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1.bw  --referencePoint center --regionsFileName ../../data/peaks3/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/CEBPA_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/CEBPA_mat_profile.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/CEBPA_mat_profile.pdf --refPointLabel center --plotTitle CEBPA', returncode=0, stdout=b'', stderr=b'')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1.bw  --referencePoint center --regionsFileName ../../data/peaks3/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/CEBPA_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/CEBPA_mat_profile_clust3.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/CEBPA_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle CEBPA', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1.bw  --referencePoint center --regionsFileName ../../data/peaks3/mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/CDK8_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/CDK8_VHL_mat_profile.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/CDK8_VHL_mat_profile.pdf --refPointLabel center --plotTitle CDK8_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1.bw  --referencePoint center --regionsFileName ../../data/peaks3/mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/CDK8_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/CDK8_VHL_mat_profile_clust3.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/CDK8_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle CDK8_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1.bw  --referencePoint center --regionsFileName ../../data/peaks3/mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/BRD4_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/BRD4_VHL_mat_profile.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/BRD4_VHL_mat_profile.pdf --refPointLabel center --plotTitle BRD4_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1.bw  --referencePoint center --regionsFileName ../../data/peaks3/mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/BRD4_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/BRD4_VHL_mat_profile_clust3.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/BRD4_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle BRD4_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1.bw  --referencePoint center --regionsFileName ../../data/peaks3/mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/IRF8_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/IRF8_VHL_mat_profile.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/IRF8_VHL_mat_profile.pdf --refPointLabel center --plotTitle IRF8_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1.bw  --referencePoint center --regionsFileName ../../data/peaks3/mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/IRF8_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/IRF8_VHL_mat_profile_clust3.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/IRF8_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle IRF8_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1.bw  --referencePoint center --regionsFileName ../../data/peaks3/mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/SMC1_VHL_mat_profile.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/SMC1_VHL_mat_profile.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/SMC1_VHL_mat_profile.pdf --refPointLabel center --plotTitle SMC1_VHL', returncode=0, stdout=b'', stderr=b'')\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/recalib_bigwig_3/mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1.bw  --referencePoint center --regionsFileName ../../data/peaks3/mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1_peaks.narrowPeak  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/SMC1_VHL_mat_profile_clust3.gz --upstream 3000 --downstream 3000 --numberOfProcessors 7 && plotProfile --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks/SMC1_VHL_mat_profile_clust3.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks/SMC1_VHL_mat_profile_clust3.pdf --refPointLabel center --perGroup --kmeans 3 --plotTitle SMC1_VHL', returncode=0, stdout=b'', stderr=b'')\n"
     ]
    }
   ],
   "source": [
    "for i in range(len(bw)):\n",
    "    chip.computePeaksAt(peaks[i], bw[i], window=3000, folder=\"\", title=names[i], numthreads=7, torecompute=True, onlyProfile=True, refpoint='center', name='../../data/IRF2BP2_degraded_3/diffPeaks/'+names[i]+'_mat_profile.pdf', withDeeptools=True)\n",
    "    chip.computePeaksAt(peaks[i], bw[i], window=3000, folder=\"\", title=names[i], numthreads=7, torecompute=True, refpoint='center', onlyProfile=True,name='../../data/IRF2BP2_degraded_3/diffPeaks/'+names[i]+'_mat_profile_clust3.pdf', withDeeptools=True, cluster=3)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### on unscalled data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 185,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/IRF2BP2_degraded_3/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1.mLb.clN.sorted.bam',\n",
       " '../../data/IRF2BP2_degraded_3/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1.mLb.clN.sorted.bam',\n",
       " '../../data/IRF2BP2_degraded_3/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1.mLb.clN.sorted.bam',\n",
       " '../../data/IRF2BP2_degraded_3/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1.mLb.clN.sorted.bam',\n",
       " '../../data/IRF2BP2_degraded_3/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1.mLb.clN.sorted.bam',\n",
       " '../../data/IRF2BP2_degraded_3/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1.mLb.clN.sorted.bam',\n",
       " '../../data/IRF2BP2_degraded_3/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1.mLb.clN.sorted.bam',\n",
       " '../../data/IRF2BP2_degraded_3/mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1.mLb.clN.sorted.bam',\n",
       " '../../data/IRF2BP2_degraded_3/mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1.mLb.clN.sorted.bam',\n",
       " '../../data/IRF2BP2_degraded_3/mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1.mLb.clN.sorted.bam',\n",
       " '../../data/IRF2BP2_degraded_3/mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1.mLb.clN.sorted.bam',\n",
       " '../../data/IRF2BP2_degraded_3/mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1.mLb.clN.sorted.bam',\n",
       " '../../data/IRF2BP2_degraded_3/mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1.mLb.clN.sorted.bam',\n",
       " '../../data/IRF2BP2_degraded_3/mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1.mLb.clN.sorted.bam']"
      ]
     },
     "execution_count": 185,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bams = !ls ../../data/IRF2BP2_degraded_3/mp*.bam\n",
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "! mkdir ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 186,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/IRF2BP2_degraded_3/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1.mLb.clN.sorted.bam ../../data/IRF2BP2_degraded_3/mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/IRF2BP2_degraded_3/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1.mLb.clN.sorted.bam and ../../data/IRF2BP2_degraded_3/mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "217\n",
      "computing the scaling values\n",
      "16490251 21182237\n",
      "doing differential peak binding\n",
      "../../data/IRF2BP2_degraded_3/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1.mLb.clN.sorted.bam ../../data/IRF2BP2_degraded_3/mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1.mLb.clN.sorted.bam\n",
      "doing diff from ../../data/IRF2BP2_degraded_3/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1.mLb.clN.sorted.bam and ../../data/IRF2BP2_degraded_3/mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1.mLb.clN.sorted.bam\n",
      "computing the fragment avg size\n",
      "176\n",
      "computing the scaling values\n",
      "30015739 29684545\n",
      "doing differential peak binding\n"
     ]
    }
   ],
   "source": [
    "#on unscalled data \n",
    "for i in range(int(len(bams)/2)):\n",
    "    if i < 0:\n",
    "        continue\n",
    "    name1 = bams[i]\n",
    "    name2 = bams[7+i]\n",
    "    print(name1,name2)\n",
    "    chip.fullDiffPeak(name1,name2, control1='../../data/results3/bwa/mergedLibrary/INPUT_R1.mLb.clN.sorted.bam', directory = \"../../data/IRF2BP2_degraded_3/diffData_unscaled/\", res_directory = \"../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/\",pairedend=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 311,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/bigwig/INPUT_R1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/bigwig/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/bigwig/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/bigwig/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/bigwig/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/bigwig/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/bigwig/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/bigwig/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/bigwig/mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/bigwig/mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/bigwig/mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/bigwig/mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/bigwig/mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/bigwig/mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1.mLb.clN.bigWig...\n",
      "Copying gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/bigwig/mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1.mLb.clN.bigWig...\n",
      "/ [15/15 files][  4.3 GiB/  4.3 GiB] 100% Done 330.4 MiB/s ETA 00:00:00         \n",
      "Operation completed over 15 objects/4.3 GiB.                                     \n"
     ]
    }
   ],
   "source": [
    "!mkdir ../../data/bigwigrep3unscaled/\n",
    "!gsutil -m cp gs://amlproject/Chip/IRF2BP2_degraded_rep3/bwa/mergedLibrary/bigwig/*.bigWig ../../data/bigwigrep3unscaled/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 312,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/bigwigrep3unscaled/INPUT_R1.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep3unscaled/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep3unscaled/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep3unscaled/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep3unscaled/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep3unscaled/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep3unscaled/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep3unscaled/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep3unscaled/mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep3unscaled/mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep3unscaled/mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep3unscaled/mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep3unscaled/mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep3unscaled/mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1.mLb.clN.bigWig',\n",
       " '../../data/bigwigrep3unscaled/mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1.mLb.clN.bigWig']"
      ]
     },
     "execution_count": 312,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bw = ! ls ../../data/bigwigrep3unscaled/*.bigWig\n",
    "bw"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 313,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1_vs_mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1_c3.0_cond1.bed',\n",
       " '../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1_vs_mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1_c3.0_cond1.bed',\n",
       " '../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1_vs_mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1_c3.0_cond1.bed',\n",
       " '../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1_vs_mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1_c3.0_cond1.bed',\n",
       " '../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1_vs_mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1_c3.0_cond1.bed',\n",
       " '../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1_vs_mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1_c3.0_cond1.bed',\n",
       " '../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1_vs_mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1_c3.0_cond1.bed']"
      ]
     },
     "execution_count": 313,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "cond1peak = ! ls ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/*cond1.bed\n",
    "cond2peak = ! ls ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/*cond2.bed\n",
    "commonpeak = ! ls ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/*common.bed\n",
    "cond1peak"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 314,
   "metadata": {},
   "outputs": [],
   "source": [
    "names = [\"CDK8\",\"BRD4\",\"IRF8\",\"SMC1\",\"MED1\",\"ZEB2\",\"CEBPA\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 317,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/bigwigrep3unscaled/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1.mLb.clN.bigWig ../../data/bigwigrep3unscaled/mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1.mLb.clN.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep3unscaled/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1.mLb.clN.bigWig ../../data/bigwigrep3unscaled/mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1_vs_mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1_c3.0_cond1.bed ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1_vs_mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1_c3.0_common.bed ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/mp831-MV411_IRF2BP_DMSO_6h-CDK8-r1_R1_vs_mp838-MV411_IRF2BP_VHL_6h-CDK8-r1_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/CDK8_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 14 && plotHeatmap --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/CDK8_mat.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/CDK8_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle CDK8', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270329v1\\t      1040\\nchrUn_KI270414v1\\t      2489\\nchrUn_KI270388v1\\t      1216\\nchrUn_KI270383v1\\t      1750\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270393v1\\t      1308\\n')\n",
      "../../data/bigwigrep3unscaled/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1.mLb.clN.bigWig ../../data/bigwigrep3unscaled/mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1.mLb.clN.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep3unscaled/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1.mLb.clN.bigWig ../../data/bigwigrep3unscaled/mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1_vs_mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1_c3.0_cond1.bed ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1_vs_mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1_c3.0_common.bed ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/mp832-MV411_IRF2BP_DMSO_6h-BRD4-r1_R1_vs_mp839-MV411_IRF2BP_VHL_6h-BRD4-r1_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/BRD4_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 14 && plotHeatmap --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/BRD4_mat.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/BRD4_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle BRD4', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270388v1\\t      1216\\nchrUn_KI270386v1\\t      1788\\nchrUn_KI270390v1\\t      2387\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270394v1\\t       970\\n')\n",
      "../../data/bigwigrep3unscaled/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1.mLb.clN.bigWig ../../data/bigwigrep3unscaled/mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1.mLb.clN.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep3unscaled/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1.mLb.clN.bigWig ../../data/bigwigrep3unscaled/mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1_vs_mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1_c3.0_cond1.bed ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1_vs_mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1_c3.0_common.bed ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1_vs_mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/IRF8_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 14 && plotHeatmap --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/IRF8_mat.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/IRF8_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle IRF8', returncode=0, stdout=b'', stderr=b\"\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270334v1\\t      1368\\nchrUn_KI270412v1\\t      1179\\nchrUn_KI270414v1\\t      2489\\nchrUn_KI270312v1\\t       998\\nchrUn_KI270390v1\\t      2387\\nchrUn_KI270389v1\\t      1298\\nchrUn_KI270388v1\\t      1216\\nchrUn_KI270396v1\\t      1880\\nchrUn_KI270425v1\\t      1884\\nOne of the groups defined in the bed file is too small.\\nGroups that are too small can't be plotted. \\nWARNING: Group 'mp833-MV411_IRF2BP_DMSO_6h-IRF8-r1_R1_vs_mp840-MV411_IRF2BP_VHL_6h-IRF8-r1_R1_c3.0_cond1.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n",
      "../../data/bigwigrep3unscaled/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1.mLb.clN.bigWig ../../data/bigwigrep3unscaled/mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1.mLb.clN.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep3unscaled/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1.mLb.clN.bigWig ../../data/bigwigrep3unscaled/mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1_vs_mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1_c3.0_cond1.bed ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1_vs_mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1_c3.0_common.bed ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1_vs_mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/SMC1_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 14 && plotHeatmap --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/SMC1_mat.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/SMC1_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle SMC1', returncode=0, stdout=b'', stderr=b\"\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270544v1\\t      1202\\nchrUn_KI270376v1\\t      1136\\nchrUn_KI270381v1\\t      1930\\nchrUn_KI270581v1\\t      7046\\nchrUn_KI270395v1\\t      1143\\nchrUn_KI270384v1\\t      1658\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270539v1\\t       993\\nchrUn_KI270371v1\\t      2805\\nchrUn_KI270383v1\\t      1750\\nOne of the groups defined in the bed file is too small.\\nGroups that are too small can't be plotted. \\nWARNING: Group 'mp834-MV411_IRF2BP_DMSO_6h-SMC1-r1_R1_vs_mp841-MV411_IRF2BP_VHL_6h-SMC1-r1_R1_c3.0_cond1.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n",
      "../../data/bigwigrep3unscaled/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1.mLb.clN.bigWig ../../data/bigwigrep3unscaled/mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1.mLb.clN.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep3unscaled/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1.mLb.clN.bigWig ../../data/bigwigrep3unscaled/mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1_vs_mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1_c3.0_cond1.bed ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1_vs_mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1_c3.0_common.bed ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/mp835-MV411_IRF2BP_DMSO_6h-MED1-r3_R1_vs_mp842-MV411_IRF2BP_VHL_6h-MED1-r3_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/MED1_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 14 && plotHeatmap --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/MED1_mat.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/MED1_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle MED1', returncode=0, stdout=b'', stderr=b'\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270381v1\\t      1930\\nchrUn_KI270388v1\\t      1216\\nchrUn_KI270384v1\\t      1658\\nchrUn_KI270414v1\\t      2489\\nchrUn_KI270393v1\\t      1308\\nchrUn_KI270378v1\\t      1048\\nchrUn_KI270424v1\\t      2140\\nchrUn_KI270329v1\\t      1040\\nchrUn_KI270312v1\\t       998\\nchrUn_KI270374v1\\t      2656\\nchrUn_KI270316v1\\t      1444\\n')\n",
      "../../data/bigwigrep3unscaled/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1.mLb.clN.bigWig ../../data/bigwigrep3unscaled/mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1.mLb.clN.bigWig\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep3unscaled/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1.mLb.clN.bigWig ../../data/bigwigrep3unscaled/mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1_vs_mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1_c3.0_cond1.bed ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1_vs_mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1_c3.0_common.bed ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1_vs_mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/ZEB2_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 14 && plotHeatmap --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/ZEB2_mat.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/ZEB2_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle ZEB2', returncode=0, stdout=b'', stderr=b\"\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270371v1\\t      2805\\nchrUn_KI270329v1\\t      1040\\nchrUn_KI270385v1\\t       990\\n         chrEBV\\t    171823\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270383v1\\t      1750\\nchrUn_KI270581v1\\t      7046\\nchrUn_KI270425v1\\t      1884\\nchrUn_KI270544v1\\t      1202\\nchrUn_KI270387v1\\t      1537\\nWARNING: Group 'mp836-MV411_IRF2BP_DMSO_6h-ZEB2-r1_R1_vs_mp843-MV411_IRF2BP_VHL_6h-ZEB2-r1_R1_c3.0_cond1.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n",
      "../../data/bigwigrep3unscaled/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1.mLb.clN.bigWig ../../data/bigwigrep3unscaled/mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1.mLb.clN.bigWig\n",
      "CompletedProcess(args='computeMatrix reference-point -S ../../data/bigwigrep3unscaled/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1.mLb.clN.bigWig ../../data/bigwigrep3unscaled/mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1.mLb.clN.bigWig  --referencePoint center --regionsFileName ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1_vs_mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1_c3.0_cond1.bed ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1_vs_mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1_c3.0_common.bed ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1_vs_mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1_c3.0_cond2.bed  --missingDataAsZero --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/CEBPA_mat.gz --upstream 3000 --downstream 3000 --numberOfProcessors 14 && plotHeatmap --matrixFile ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/CEBPA_mat.gz --outFileName ../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/CEBPA_mat.pdf --refPointLabel center --regionsLabel DMSO_peaks common VHL_peaks  --samplesLabel DMSO VHL  --plotTitle CEBPA', returncode=0, stdout=b'', stderr=b\"\\nThe following chromosome names did not match between the bigwig files\\nchromosome\\tlength\\nchrUn_KI270528v1\\t      2983\\nchrUn_KI270390v1\\t      2387\\nchrUn_KI270382v1\\t      4215\\nchrUn_KI270752v1\\t     27745\\nchrUn_KI270381v1\\t      1930\\nchrUn_KI270425v1\\t      1884\\nchrUn_KI270423v1\\t       981\\nchrUn_KI270529v1\\t      1899\\nchrUn_KI270329v1\\t      1040\\nchrUn_KI270378v1\\t      1048\\nOne of the groups defined in the bed file is too small.\\nGroups that are too small can't be plotted. \\nWARNING: Group 'mp837-MV411_IRF2BP_DMSO_6h-CEBPA-r1_R1_vs_mp844-MV411_IRF2BP_VHL_6h-CEBPA-r1_R1_c3.0_cond1.bed' is too small for plotting, you might want to remove it. There will likely be an error message from matplotlib regarding this below.\\n\")\n"
     ]
    }
   ],
   "source": [
    "for i in range(int((len(bw)-1)/2)):\n",
    "    if i <0:\n",
    "        continue\n",
    "    name1 = bw[1+i]\n",
    "    name2 = bw[8+i]\n",
    "    peak = [cond1peak[i], commonpeak[i], cond2peak[i]]\n",
    "    #for val in peak:\n",
    "        #chip.dropWeirdChromosomes(val)\n",
    "    name = names[i]\n",
    "    print(name1,name2)\n",
    "    chip.computePeaksAt(peak, [name1, name2], bigwignames=['DMSO', 'VHL'], peaknames=['DMSO_peaks', 'common', 'VHL_peaks'], window=3000, folder=\"\", title=name, numthreads=14, torecompute=True, refpoint=\"center\", name='../../data/IRF2BP2_degraded_3/diffPeaks_unscaled/'+name+'_mat.pdf', withDeeptools=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! gsutil -m cp -r \"../../data/recalib_bigwig_3/*\" gs://amlproject/Chip/IRF2BP2_degraded_rep3/recalib_bigwig/ &&\\\n",
    "gsutil -m cp -r \"../../data/IRF2BP2_degraded_3/diffData/*\" gs://amlproject/Chip/IRF2BP2_degraded_rep3/diffData/ &&\\\n",
    "gsutil -m cp -r \"../../data/IRF2BP2_degraded_3/diffPeaks/*\" gs://amlproject/Chip/IRF2BP2_degraded_rep3/diffPeaks/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "## on local\n",
    "! scp runningchip2.us-east1-b.aml-chipseq-rnaseq-259419:\"~/data/IRF2BP2_degraded_3/diffPeaks_unscaled/*.pdf\" unscaled/v3/\n",
    "! scp runningchip2.us-east1-b.aml-chipseq-rnaseq-259419:\"~/data/IRF2BP2_degraded_3/diffPeaks/*.pdf\" scaled/v3/\n",
    "! scp runningchip2.us-east1-b.aml-chipseq-rnaseq-259419:\"~/data/diffBinding_rep1/diffPeaks_unscaled/*.pdf\" unscaled/v3/\n",
    "! scp runningchip2.us-east1-b.aml-chipseq-rnaseq-259419:\"~/data/diffBinding_rep1/diffPeaks/*.pdf\" scaled/v1/\n",
    "! scp runningchip2.us-east1-b.aml-chipseq-rnaseq-259419:\"~/data/diffBinding_hist/diffPeaks_unscaled/*.pdf\" unscaled/v1/\n",
    "! scp runningchip2.us-east1-b.aml-chipseq-rnaseq-259419:\"~/data/diffBinding_hist/diffPeaks/*.pdf\" unscaled/v1/\n",
    "! scp runningchip2.us-east1-b.aml-chipseq-rnaseq-259419:\"~/data/diffBinding_rep2/diffPeaks_unscaled/*.pdf\" unscaled/v2/\n",
    "! scp runningchip2.us-east1-b.aml-chipseq-rnaseq-259419:\"~/data/diffBinding_rep2/diffPeaks/*.pdf\" scaled/v2/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# TODO:\n",
    "\n",
    "- run full diffPeak v1,v2\n",
    "- run v1 v2 unscaled plots\n",
    "- run v1 v2 profiles\n",
    "- run histones unscaled plots\n",
    "- run v3 unscaled plots"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## RNA"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "rename1 = {\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_10_MP7623_S30_R1_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp810_1.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_10_MP7623_S30_R2_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp810_2.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_11_MP7623_S31_R1_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp811_1.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_11_MP7623_S31_R2_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp811_2.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_12_MP7623_S32_R1_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp812_1.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_12_MP7623_S32_R2_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp812_2.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_13_MP7623_S33_R1_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp813_1.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_13_MP7623_S33_R2_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp813_2.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_14_MP7623_S34_R1_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp814_1.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_14_MP7623_S34_R2_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp814_2.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_15_MP7623_S35_R1_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp815_1.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_15_MP7623_S35_R2_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp815_2.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_16_MP7623_S36_R1_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp816_1.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_16_MP7623_S36_R2_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp816_2.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_1_MP7623_S21_R1_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp801_1.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_1_MP7623_S21_R2_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp801_2.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_2_MP7623_S22_R1_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp802_1.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_2_MP7623_S22_R2_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp802_2.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_3_MP7623_S23_R1_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp803_1.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_3_MP7623_S23_R2_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp803_2.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_4_MP7623_S24_R1_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp804_1.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_4_MP7623_S24_R2_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp804_2.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_5_MP7623_S25_R1_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp805_1.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_5_MP7623_S25_R2_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp805_2.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_6_MP7623_S26_R1_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp806_1.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_6_MP7623_S26_R2_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp806_2.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_7_MP7623_S27_R1_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp807_1.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_7_MP7623_S27_R2_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp807_2.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_8_MP7623_S28_R1_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp808_1.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_8_MP7623_S28_R2_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp808_2.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_9_MP7623_S29_R1_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp809_1.fastq.gz',\n",
    "'gs://amlproject/slamseq/IRF2BP2/20191212_9_MP7623_S29_R2_001.fastq.gz':'gs://amlproject/slamseq/IRF2BP2/mp809_2.fastq.gz'}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for k, val in rename1.items():\n",
    "    os.system('gsutil mv '+k+' '+val)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "a"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "terra.list_blobs_with_prefix(\"amlproject\",\"slamseq/IRF2BP2/\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "terra.uploadFromFolder(\"amlproject\", \"slamseq/IRF2BP2/\", workspace=\"broad-firecloud-ccle/hg38_RNAseq\", \n",
    "                       fformat=\"fastq12\", source=\"maximP\",samplesetname=\"slamseq_IRF2BP2\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "samplesetname = 'slamseq_IRF2BP2'\n",
    "refworkspace=\"broad-firecloud-ccle/hg38_RNAseq\"\n",
    "refwm = dm.WorkspaceManager(refworkspace)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "submission_id = refwm.create_submission(\"slamdunk\", samplesetname,'sample_set',expression='this.samples')\n",
    "terra.waitForSubmission(refworkspace, submission_id)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "submission_id1 = refwm.create_submission(\"rsem_v1-0_BETA_cfg\", samplesetname,'sample_set',expression='this.samples')\n",
    "terra.waitForSubmission(refworkspace, submission_id1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "! pip3 install git+https://github.com/jkobject/slamdunk.git --upgrade"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "folder_bams = \"../../data/slam/data/\"\n",
    "bams = ! ls $folder_bams*.fastq.gz\n",
    "bams"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for val in h.grouped(bams[4:],2):\n",
    "    print(val)\n",
    "    res = os.system('../../TrimGalore-0.6.5/trim_galore --paired --cores 12 --retain_unpaired -stringency 3 --illumina '+\n",
    "    val[0] + ' '+val[1]+\" -o \"+folder_bams)\n",
    "    if res!=0:\n",
    "        raise ValueError()\n",
    "    res = os.system('slamdunk all -r ../../data/slam/ref_genome.fa -b ../../data/slam/GSE100708_hg38_refseq_062016_ensemblv84_3UTR.bed \\\n",
    "    -o ../../data/slam/resIRF2BP2/ -t 12 -c 2  '+'.'.join(val[0].split('.')[:-2])+'_val_1.fq.gz \\\n",
    "    '+'.'.join(val[1].split('.')[:-2])+\"_val_2.fq.gz\")\n",
    "    if res!=0:\n",
    "        raise ValueError()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for val in h.grouped(bams[:4],2):\n",
    "    print(val)\n",
    "    res = os.system('slamdunk all -r ../../data/slam/ref_genome.fa -b ../../data/slam/GSE100708_hg38_refseq_062016_ensemblv84_3UTR.bed \\\n",
    "    -o ../../data/slam/resIRF2BP2/ -t 12 -c 2  '+'.'.join(val[0].split('.')[:-2])+'_val_1.fq.gz \\\n",
    "    '+'.'.join(val[1].split('.')[:-2])+\"_val_2.fq.gz\")\n",
    "    if res!=0:\n",
    "        raise ValueError()"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.6"
  },
  "latex_envs": {
   "LaTeX_envs_menu_present": true,
   "autoclose": false,
   "autocomplete": true,
   "bibliofile": "biblio.bib",
   "cite_by": "apalike",
   "current_citInitial": 1,
   "eqLabelWithNumbers": true,
   "eqNumInitial": 1,
   "hotkeys": {
    "equation": "Ctrl-E",
    "itemize": "Ctrl-I"
   },
   "labels_anchors": false,
   "latex_user_defs": false,
   "report_style_numbering": false,
   "user_envs_cfg": false
  },
  "toc": {
   "base_numbering": 1,
   "nav_menu": {},
   "number_sections": false,
   "sideBar": true,
   "skip_h1_title": false,
   "title_cell": "Table of Contents",
   "title_sidebar": "Contents",
   "toc_cell": false,
   "toc_position": {
    "height": "257.4px",
    "left": "127px",
    "top": "111.6px",
    "width": "165px"
   },
   "toc_section_display": true,
   "toc_window_display": true
  },
  "varInspector": {
   "cols": {
    "lenName": 16,
    "lenType": 16,
    "lenVar": 40
   },
   "kernels_config": {
    "python": {
     "delete_cmd_postfix": "",
     "delete_cmd_prefix": "del ",
     "library": "var_list.py",
     "varRefreshCmd": "print(var_dic_list())"
    },
    "r": {
     "delete_cmd_postfix": ") ",
     "delete_cmd_prefix": "rm(",
     "library": "var_list.r",
     "varRefreshCmd": "cat(var_dic_list()) "
    }
   },
   "types_to_exclude": [
    "module",
    "function",
    "builtin_function_or_method",
    "instance",
    "_Feature"
   ],
   "window_display": false
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
