{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "9fa17af9",
   "metadata": {},
   "source": [
    "# Slamseq"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "a1af2fe3",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "    <div class=\"bk-root\">\n",
       "        <a href=\"https://bokeh.org\" target=\"_blank\" class=\"bk-logo bk-logo-small bk-logo-notebook\"></a>\n",
       "        <span id=\"1003\">Loading BokehJS ...</span>\n",
       "    </div>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/javascript": [
       "\n",
       "(function(root) {\n",
       "  function now() {\n",
       "    return new Date();\n",
       "  }\n",
       "\n",
       "  var force = true;\n",
       "\n",
       "  if (typeof root._bokeh_onload_callbacks === \"undefined\" || force === true) {\n",
       "    root._bokeh_onload_callbacks = [];\n",
       "    root._bokeh_is_loading = undefined;\n",
       "  }\n",
       "\n",
       "  var JS_MIME_TYPE = 'application/javascript';\n",
       "  var HTML_MIME_TYPE = 'text/html';\n",
       "  var EXEC_MIME_TYPE = 'application/vnd.bokehjs_exec.v0+json';\n",
       "  var CLASS_NAME = 'output_bokeh rendered_html';\n",
       "\n",
       "  /**\n",
       "   * Render data to the DOM node\n",
       "   */\n",
       "  function render(props, node) {\n",
       "    var script = document.createElement(\"script\");\n",
       "    node.appendChild(script);\n",
       "  }\n",
       "\n",
       "  /**\n",
       "   * Handle when an output is cleared or removed\n",
       "   */\n",
       "  function handleClearOutput(event, handle) {\n",
       "    var cell = handle.cell;\n",
       "\n",
       "    var id = cell.output_area._bokeh_element_id;\n",
       "    var server_id = cell.output_area._bokeh_server_id;\n",
       "    // Clean up Bokeh references\n",
       "    if (id != null && id in Bokeh.index) {\n",
       "      Bokeh.index[id].model.document.clear();\n",
       "      delete Bokeh.index[id];\n",
       "    }\n",
       "\n",
       "    if (server_id !== undefined) {\n",
       "      // Clean up Bokeh references\n",
       "      var cmd = \"from bokeh.io.state import curstate; print(curstate().uuid_to_server['\" + server_id + \"'].get_sessions()[0].document.roots[0]._id)\";\n",
       "      cell.notebook.kernel.execute(cmd, {\n",
       "        iopub: {\n",
       "          output: function(msg) {\n",
       "            var id = msg.content.text.trim();\n",
       "            if (id in Bokeh.index) {\n",
       "              Bokeh.index[id].model.document.clear();\n",
       "              delete Bokeh.index[id];\n",
       "            }\n",
       "          }\n",
       "        }\n",
       "      });\n",
       "      // Destroy server and session\n",
       "      var cmd = \"import bokeh.io.notebook as ion; ion.destroy_server('\" + server_id + \"')\";\n",
       "      cell.notebook.kernel.execute(cmd);\n",
       "    }\n",
       "  }\n",
       "\n",
       "  /**\n",
       "   * Handle when a new output is added\n",
       "   */\n",
       "  function handleAddOutput(event, handle) {\n",
       "    var output_area = handle.output_area;\n",
       "    var output = handle.output;\n",
       "\n",
       "    // limit handleAddOutput to display_data with EXEC_MIME_TYPE content only\n",
       "    if ((output.output_type != \"display_data\") || (!Object.prototype.hasOwnProperty.call(output.data, EXEC_MIME_TYPE))) {\n",
       "      return\n",
       "    }\n",
       "\n",
       "    var toinsert = output_area.element.find(\".\" + CLASS_NAME.split(' ')[0]);\n",
       "\n",
       "    if (output.metadata[EXEC_MIME_TYPE][\"id\"] !== undefined) {\n",
       "      toinsert[toinsert.length - 1].firstChild.textContent = output.data[JS_MIME_TYPE];\n",
       "      // store reference to embed id on output_area\n",
       "      output_area._bokeh_element_id = output.metadata[EXEC_MIME_TYPE][\"id\"];\n",
       "    }\n",
       "    if (output.metadata[EXEC_MIME_TYPE][\"server_id\"] !== undefined) {\n",
       "      var bk_div = document.createElement(\"div\");\n",
       "      bk_div.innerHTML = output.data[HTML_MIME_TYPE];\n",
       "      var script_attrs = bk_div.children[0].attributes;\n",
       "      for (var i = 0; i < script_attrs.length; i++) {\n",
       "        toinsert[toinsert.length - 1].firstChild.setAttribute(script_attrs[i].name, script_attrs[i].value);\n",
       "        toinsert[toinsert.length - 1].firstChild.textContent = bk_div.children[0].textContent\n",
       "      }\n",
       "      // store reference to server id on output_area\n",
       "      output_area._bokeh_server_id = output.metadata[EXEC_MIME_TYPE][\"server_id\"];\n",
       "    }\n",
       "  }\n",
       "\n",
       "  function register_renderer(events, OutputArea) {\n",
       "\n",
       "    function append_mime(data, metadata, element) {\n",
       "      // create a DOM node to render to\n",
       "      var toinsert = this.create_output_subarea(\n",
       "        metadata,\n",
       "        CLASS_NAME,\n",
       "        EXEC_MIME_TYPE\n",
       "      );\n",
       "      this.keyboard_manager.register_events(toinsert);\n",
       "      // Render to node\n",
       "      var props = {data: data, metadata: metadata[EXEC_MIME_TYPE]};\n",
       "      render(props, toinsert[toinsert.length - 1]);\n",
       "      element.append(toinsert);\n",
       "      return toinsert\n",
       "    }\n",
       "\n",
       "    /* Handle when an output is cleared or removed */\n",
       "    events.on('clear_output.CodeCell', handleClearOutput);\n",
       "    events.on('delete.Cell', handleClearOutput);\n",
       "\n",
       "    /* Handle when a new output is added */\n",
       "    events.on('output_added.OutputArea', handleAddOutput);\n",
       "\n",
       "    /**\n",
       "     * Register the mime type and append_mime function with output_area\n",
       "     */\n",
       "    OutputArea.prototype.register_mime_type(EXEC_MIME_TYPE, append_mime, {\n",
       "      /* Is output safe? */\n",
       "      safe: true,\n",
       "      /* Index of renderer in `output_area.display_order` */\n",
       "      index: 0\n",
       "    });\n",
       "  }\n",
       "\n",
       "  // register the mime type if in Jupyter Notebook environment and previously unregistered\n",
       "  if (root.Jupyter !== undefined) {\n",
       "    var events = require('base/js/events');\n",
       "    var OutputArea = require('notebook/js/outputarea').OutputArea;\n",
       "\n",
       "    if (OutputArea.prototype.mime_types().indexOf(EXEC_MIME_TYPE) == -1) {\n",
       "      register_renderer(events, OutputArea);\n",
       "    }\n",
       "  }\n",
       "\n",
       "  \n",
       "  if (typeof (root._bokeh_timeout) === \"undefined\" || force === true) {\n",
       "    root._bokeh_timeout = Date.now() + 5000;\n",
       "    root._bokeh_failed_load = false;\n",
       "  }\n",
       "\n",
       "  var NB_LOAD_WARNING = {'data': {'text/html':\n",
       "     \"<div style='background-color: #fdd'>\\n\"+\n",
       "     \"<p>\\n\"+\n",
       "     \"BokehJS does not appear to have successfully loaded. If loading BokehJS from CDN, this \\n\"+\n",
       "     \"may be due to a slow or bad network connection. Possible fixes:\\n\"+\n",
       "     \"</p>\\n\"+\n",
       "     \"<ul>\\n\"+\n",
       "     \"<li>re-rerun `output_notebook()` to attempt to load from CDN again, or</li>\\n\"+\n",
       "     \"<li>use INLINE resources instead, as so:</li>\\n\"+\n",
       "     \"</ul>\\n\"+\n",
       "     \"<code>\\n\"+\n",
       "     \"from bokeh.resources import INLINE\\n\"+\n",
       "     \"output_notebook(resources=INLINE)\\n\"+\n",
       "     \"</code>\\n\"+\n",
       "     \"</div>\"}};\n",
       "\n",
       "  function display_loaded() {\n",
       "    var el = document.getElementById(\"1003\");\n",
       "    if (el != null) {\n",
       "      el.textContent = \"BokehJS is loading...\";\n",
       "    }\n",
       "    if (root.Bokeh !== undefined) {\n",
       "      if (el != null) {\n",
       "        el.textContent = \"BokehJS \" + root.Bokeh.version + \" successfully loaded.\";\n",
       "      }\n",
       "    } else if (Date.now() < root._bokeh_timeout) {\n",
       "      setTimeout(display_loaded, 100)\n",
       "    }\n",
       "  }\n",
       "\n",
       "\n",
       "  function run_callbacks() {\n",
       "    try {\n",
       "      root._bokeh_onload_callbacks.forEach(function(callback) {\n",
       "        if (callback != null)\n",
       "          callback();\n",
       "      });\n",
       "    } finally {\n",
       "      delete root._bokeh_onload_callbacks\n",
       "    }\n",
       "    console.debug(\"Bokeh: all callbacks have finished\");\n",
       "  }\n",
       "\n",
       "  function load_libs(css_urls, js_urls, callback) {\n",
       "    if (css_urls == null) css_urls = [];\n",
       "    if (js_urls == null) js_urls = [];\n",
       "\n",
       "    root._bokeh_onload_callbacks.push(callback);\n",
       "    if (root._bokeh_is_loading > 0) {\n",
       "      console.debug(\"Bokeh: BokehJS is being loaded, scheduling callback at\", now());\n",
       "      return null;\n",
       "    }\n",
       "    if (js_urls == null || js_urls.length === 0) {\n",
       "      run_callbacks();\n",
       "      return null;\n",
       "    }\n",
       "    console.debug(\"Bokeh: BokehJS not loaded, scheduling load and callback at\", now());\n",
       "    root._bokeh_is_loading = css_urls.length + js_urls.length;\n",
       "\n",
       "    function on_load() {\n",
       "      root._bokeh_is_loading--;\n",
       "      if (root._bokeh_is_loading === 0) {\n",
       "        console.debug(\"Bokeh: all BokehJS libraries/stylesheets loaded\");\n",
       "        run_callbacks()\n",
       "      }\n",
       "    }\n",
       "\n",
       "    function on_error(url) {\n",
       "      console.error(\"failed to load \" + url);\n",
       "    }\n",
       "\n",
       "    for (let i = 0; i < css_urls.length; i++) {\n",
       "      const url = css_urls[i];\n",
       "      const element = document.createElement(\"link\");\n",
       "      element.onload = on_load;\n",
       "      element.onerror = on_error.bind(null, url);\n",
       "      element.rel = \"stylesheet\";\n",
       "      element.type = \"text/css\";\n",
       "      element.href = url;\n",
       "      console.debug(\"Bokeh: injecting link tag for BokehJS stylesheet: \", url);\n",
       "      document.body.appendChild(element);\n",
       "    }\n",
       "\n",
       "    const hashes = {\"https://cdn.bokeh.org/bokeh/release/bokeh-2.3.3.min.js\": \"dM3QQsP+wXdHg42wTqW85BjZQdLNNIXqlPw/BgKoExPmTG7ZLML4EGqLMfqHT6ON\", \"https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.3.3.min.js\": \"8x57I4YuIfu8XyZfFo0XVr2WAT8EK4rh/uDe3wF7YuW2FNUSNEpJbsPaB1nJ2fz2\", \"https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.3.3.min.js\": \"3QTqdz9LyAm2i0sG5XTePsHec3UHWwVsrOL68SYRoAXsafvfAyqtQ+h440+qIBhS\"};\n",
       "\n",
       "    for (let i = 0; i < js_urls.length; i++) {\n",
       "      const url = js_urls[i];\n",
       "      const element = document.createElement('script');\n",
       "      element.onload = on_load;\n",
       "      element.onerror = on_error.bind(null, url);\n",
       "      element.async = false;\n",
       "      element.src = url;\n",
       "      if (url in hashes) {\n",
       "        element.crossOrigin = \"anonymous\";\n",
       "        element.integrity = \"sha384-\" + hashes[url];\n",
       "      }\n",
       "      console.debug(\"Bokeh: injecting script tag for BokehJS library: \", url);\n",
       "      document.head.appendChild(element);\n",
       "    }\n",
       "  };\n",
       "\n",
       "  function inject_raw_css(css) {\n",
       "    const element = document.createElement(\"style\");\n",
       "    element.appendChild(document.createTextNode(css));\n",
       "    document.body.appendChild(element);\n",
       "  }\n",
       "\n",
       "  \n",
       "  var js_urls = [\"https://cdn.bokeh.org/bokeh/release/bokeh-2.3.3.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.3.3.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.3.3.min.js\"];\n",
       "  var css_urls = [];\n",
       "  \n",
       "\n",
       "  var inline_js = [\n",
       "    function(Bokeh) {\n",
       "      Bokeh.set_log_level(\"info\");\n",
       "    },\n",
       "    function(Bokeh) {\n",
       "    \n",
       "    \n",
       "    }\n",
       "  ];\n",
       "\n",
       "  function run_inline_js() {\n",
       "    \n",
       "    if (root.Bokeh !== undefined || force === true) {\n",
       "      \n",
       "    for (var i = 0; i < inline_js.length; i++) {\n",
       "      inline_js[i].call(root, root.Bokeh);\n",
       "    }\n",
       "    if (force === true) {\n",
       "        display_loaded();\n",
       "      }} else if (Date.now() < root._bokeh_timeout) {\n",
       "      setTimeout(run_inline_js, 100);\n",
       "    } else if (!root._bokeh_failed_load) {\n",
       "      console.log(\"Bokeh: BokehJS failed to load within specified timeout.\");\n",
       "      root._bokeh_failed_load = true;\n",
       "    } else if (force !== true) {\n",
       "      var cell = $(document.getElementById(\"1003\")).parents('.cell').data().cell;\n",
       "      cell.output_area.append_execute_result(NB_LOAD_WARNING)\n",
       "    }\n",
       "\n",
       "  }\n",
       "\n",
       "  if (root._bokeh_is_loading === 0) {\n",
       "    console.debug(\"Bokeh: BokehJS loaded, going straight to plotting\");\n",
       "    run_inline_js();\n",
       "  } else {\n",
       "    load_libs(css_urls, js_urls, function() {\n",
       "      console.debug(\"Bokeh: BokehJS plotting callback run at\", now());\n",
       "      run_inline_js();\n",
       "    });\n",
       "  }\n",
       "}(window));"
      ],
      "application/vnd.bokehjs_load.v0+json": "\n(function(root) {\n  function now() {\n    return new Date();\n  }\n\n  var force = true;\n\n  if (typeof root._bokeh_onload_callbacks === \"undefined\" || force === true) {\n    root._bokeh_onload_callbacks = [];\n    root._bokeh_is_loading = undefined;\n  }\n\n  \n\n  \n  if (typeof (root._bokeh_timeout) === \"undefined\" || force === true) {\n    root._bokeh_timeout = Date.now() + 5000;\n    root._bokeh_failed_load = false;\n  }\n\n  var NB_LOAD_WARNING = {'data': {'text/html':\n     \"<div style='background-color: #fdd'>\\n\"+\n     \"<p>\\n\"+\n     \"BokehJS does not appear to have successfully loaded. If loading BokehJS from CDN, this \\n\"+\n     \"may be due to a slow or bad network connection. Possible fixes:\\n\"+\n     \"</p>\\n\"+\n     \"<ul>\\n\"+\n     \"<li>re-rerun `output_notebook()` to attempt to load from CDN again, or</li>\\n\"+\n     \"<li>use INLINE resources instead, as so:</li>\\n\"+\n     \"</ul>\\n\"+\n     \"<code>\\n\"+\n     \"from bokeh.resources import INLINE\\n\"+\n     \"output_notebook(resources=INLINE)\\n\"+\n     \"</code>\\n\"+\n     \"</div>\"}};\n\n  function display_loaded() {\n    var el = document.getElementById(\"1003\");\n    if (el != null) {\n      el.textContent = \"BokehJS is loading...\";\n    }\n    if (root.Bokeh !== undefined) {\n      if (el != null) {\n        el.textContent = \"BokehJS \" + root.Bokeh.version + \" successfully loaded.\";\n      }\n    } else if (Date.now() < root._bokeh_timeout) {\n      setTimeout(display_loaded, 100)\n    }\n  }\n\n\n  function run_callbacks() {\n    try {\n      root._bokeh_onload_callbacks.forEach(function(callback) {\n        if (callback != null)\n          callback();\n      });\n    } finally {\n      delete root._bokeh_onload_callbacks\n    }\n    console.debug(\"Bokeh: all callbacks have finished\");\n  }\n\n  function load_libs(css_urls, js_urls, callback) {\n    if (css_urls == null) css_urls = [];\n    if (js_urls == null) js_urls = [];\n\n    root._bokeh_onload_callbacks.push(callback);\n    if (root._bokeh_is_loading > 0) {\n      console.debug(\"Bokeh: BokehJS is being loaded, scheduling callback at\", now());\n      return null;\n    }\n    if (js_urls == null || js_urls.length === 0) {\n      run_callbacks();\n      return null;\n    }\n    console.debug(\"Bokeh: BokehJS not loaded, scheduling load and callback at\", now());\n    root._bokeh_is_loading = css_urls.length + js_urls.length;\n\n    function on_load() {\n      root._bokeh_is_loading--;\n      if (root._bokeh_is_loading === 0) {\n        console.debug(\"Bokeh: all BokehJS libraries/stylesheets loaded\");\n        run_callbacks()\n      }\n    }\n\n    function on_error(url) {\n      console.error(\"failed to load \" + url);\n    }\n\n    for (let i = 0; i < css_urls.length; i++) {\n      const url = css_urls[i];\n      const element = document.createElement(\"link\");\n      element.onload = on_load;\n      element.onerror = on_error.bind(null, url);\n      element.rel = \"stylesheet\";\n      element.type = \"text/css\";\n      element.href = url;\n      console.debug(\"Bokeh: injecting link tag for BokehJS stylesheet: \", url);\n      document.body.appendChild(element);\n    }\n\n    const hashes = {\"https://cdn.bokeh.org/bokeh/release/bokeh-2.3.3.min.js\": \"dM3QQsP+wXdHg42wTqW85BjZQdLNNIXqlPw/BgKoExPmTG7ZLML4EGqLMfqHT6ON\", \"https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.3.3.min.js\": \"8x57I4YuIfu8XyZfFo0XVr2WAT8EK4rh/uDe3wF7YuW2FNUSNEpJbsPaB1nJ2fz2\", \"https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.3.3.min.js\": \"3QTqdz9LyAm2i0sG5XTePsHec3UHWwVsrOL68SYRoAXsafvfAyqtQ+h440+qIBhS\"};\n\n    for (let i = 0; i < js_urls.length; i++) {\n      const url = js_urls[i];\n      const element = document.createElement('script');\n      element.onload = on_load;\n      element.onerror = on_error.bind(null, url);\n      element.async = false;\n      element.src = url;\n      if (url in hashes) {\n        element.crossOrigin = \"anonymous\";\n        element.integrity = \"sha384-\" + hashes[url];\n      }\n      console.debug(\"Bokeh: injecting script tag for BokehJS library: \", url);\n      document.head.appendChild(element);\n    }\n  };\n\n  function inject_raw_css(css) {\n    const element = document.createElement(\"style\");\n    element.appendChild(document.createTextNode(css));\n    document.body.appendChild(element);\n  }\n\n  \n  var js_urls = [\"https://cdn.bokeh.org/bokeh/release/bokeh-2.3.3.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.3.3.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.3.3.min.js\"];\n  var css_urls = [];\n  \n\n  var inline_js = [\n    function(Bokeh) {\n      Bokeh.set_log_level(\"info\");\n    },\n    function(Bokeh) {\n    \n    \n    }\n  ];\n\n  function run_inline_js() {\n    \n    if (root.Bokeh !== undefined || force === true) {\n      \n    for (var i = 0; i < inline_js.length; i++) {\n      inline_js[i].call(root, root.Bokeh);\n    }\n    if (force === true) {\n        display_loaded();\n      }} else if (Date.now() < root._bokeh_timeout) {\n      setTimeout(run_inline_js, 100);\n    } else if (!root._bokeh_failed_load) {\n      console.log(\"Bokeh: BokehJS failed to load within specified timeout.\");\n      root._bokeh_failed_load = true;\n    } else if (force !== true) {\n      var cell = $(document.getElementById(\"1003\")).parents('.cell').data().cell;\n      cell.output_area.append_execute_result(NB_LOAD_WARNING)\n    }\n\n  }\n\n  if (root._bokeh_is_loading === 0) {\n    console.debug(\"Bokeh: BokehJS loaded, going straight to plotting\");\n    run_inline_js();\n  } else {\n    load_libs(css_urls, js_urls, function() {\n      console.debug(\"Bokeh: BokehJS plotting callback run at\", now());\n      run_inline_js();\n    });\n  }\n}(window));"
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The autoreload extension is already loaded. To reload it, use:\n",
      "  %reload_ext autoreload\n",
      "The rpy2.ipython extension is already loaded. To reload it, use:\n",
      "  %reload_ext rpy2.ipython\n"
     ]
    }
   ],
   "source": [
    "from __future__ import print_function\n",
    "import os.path\n",
    "import pandas as pd\n",
    "import sys\n",
    "sys.path.insert(0, '../../')\n",
    "import seaborn as sns\n",
    "import numpy as np\n",
    "from functools import reduce\n",
    "import glob\n",
    "\n",
    "from genepy.utils import helper as h\n",
    "from genepy.utils import plot as genepyPlot\n",
    "from genepy.rna import pyDESeq2\n",
    "import genepy.rna as rna\n",
    "\n",
    "from bokeh.plotting import *\n",
    "from bokeh.models import HoverTool\n",
    "from bokeh.io import output_notebook\n",
    "\n",
    "import matplotlib.pyplot as plt\n",
    "\n",
    "from sklearn.neighbors import KNeighborsClassifier\n",
    "from sklearn.manifold import MDS, TSNE\n",
    "from sklearn.decomposition import PCA\n",
    "from sklearn.preprocessing import scale\n",
    "#from umap import UMAP\n",
    "\n",
    "output_notebook()\n",
    "%load_ext autoreload\n",
    "%matplotlib inline\n",
    "%autoreload 2\n",
    "%load_ext rpy2.ipython"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2d1e3b4f",
   "metadata": {},
   "source": [
    "## Get files"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "23444349",
   "metadata": {},
   "source": [
    "Expermients\n",
    "- IRF2BP2 degraded v1\n",
    "    - 3h\n",
    "    - 6h\n",
    "- MYB degraded v1\n",
    "    - 1h\n",
    "- 4 degrons v1\n",
    "    - RUNX1\n",
    "    - RUNX2\n",
    "    - SPI1\n",
    "    - GFI1"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "7884232b",
   "metadata": {},
   "source": [
    "DESeq output files\n",
    "- Tccounts ERCC\n",
    "- Readcounts ERCC\n",
    "- Tccounts (unscaled)\n",
    "- Readcounts (unscaled)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "68018965",
   "metadata": {},
   "outputs": [],
   "source": [
    "res_dir = \"../results\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "484971ac",
   "metadata": {},
   "outputs": [],
   "source": [
    "project1 = \"IRF2BP2_degraded_v1\"\n",
    "project2 = \"MYB_degraded_v1\"\n",
    "project3 = \"4_degrons_v1\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "f2bbb5dd",
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_file_info(project, file, conditions):\n",
    "    \n",
    "    project_row = [project]\n",
    "    #name = os.path.basename(file).split(\".\")[0]\n",
    "    file_row = [os.path.basename(file)]\n",
    "    scaling_row = [\"ERCCsamplewise\" if \"ERCCsamplewise\" in file else None]\n",
    "    condition_row = [cond for cond in conditions if cond in file]\n",
    "    count_row = [\"readcounts\" if \"readcounts\" in file else \"tccounts\"]\n",
    "    \n",
    "    return [project_row, file_row, scaling_row, condition_row, count_row]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 142,
   "id": "4d7918d0",
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_deseq_data(file, low_memory=True):\n",
    "    ''' input deseq file & returns df '''\n",
    "    ''' add file name to headers except gene columns '''\n",
    "    df = pd.read_csv(file, header=[0], index_col=None, squeeze=True, low_memory=low_memory)\n",
    "    df.columns = [\"{}_{}\".format(col_name, os.path.basename(file).split(\".\")[0]) if \"gene\" not in col_name\n",
    "                  else col_name for col_name in df.columns.tolist()]\n",
    "    return(df)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "bc487015",
   "metadata": {},
   "source": [
    "# slamseq_MYB_degraded_timecourse_v1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "fb3e6029",
   "metadata": {},
   "outputs": [],
   "source": [
    "project = \"MYB_degraded_timecourse_v1\"\n",
    "location = \"../data/slamseq_\"+project+\"/\"\n",
    "fastq_folder = \"../../data/\"+\"slamseq_\"+project+\"/fastqs\"\n",
    "fastqs = ! ls $fastq_folder/*fastq.gz"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "75c4d4b4",
   "metadata": {},
   "source": [
    "### run trim galore"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "id": "37e5b76a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/slamseq_MYB_degraded_timecourse_v1/fastqs/mr479-MV411-MYB-dTAG-VHL_24h-r3_R1_001.fastq.gz\n",
      "../../data/slamseq_MYB_degraded_timecourse_v1/fastqs/mr479-MV411-MYB-dTAG-VHL_24h-r3_R2_001.fastq.gz\n",
      "\n",
      "\n",
      "../../data/slamseq_MYB_degraded_timecourse_v1/fastqs/mr480-MV411-MYB-dTAG-VHL_24h-r4_R1_001.fastq.gz\n",
      "../../data/slamseq_MYB_degraded_timecourse_v1/fastqs/mr480-MV411-MYB-dTAG-VHL_24h-r4_R2_001.fastq.gz\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "for val1, val2 in h.grouped(fastqs[114:118], 2):\n",
    "    print(val1)\n",
    "    print(val2)\n",
    "    print(\"\\n\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "00513a55",
   "metadata": {},
   "outputs": [],
   "source": [
    "%%time\n",
    "h.parrun(['trim_galore --paired --cores 8 --retain_unpaired -stringency 3\\\n",
    " --illumina '+val1+' '+val2+' -o '+fastq_folder for val1, val2 in h.grouped(fastqs[114:118], 2)], 2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "8c2f5931",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../../data/slamseq_MYB_degraded_timecourse_v1/fastqs/mr450-MV411-MYB-dTAG-VHL_4h-r4_R1_001.fastq.gz\n",
      "../../data/slamseq_MYB_degraded_timecourse_v1/fastqs/mr450-MV411-MYB-dTAG-VHL_4h-r4_R2_001.fastq.gz\n"
     ]
    },
    {
     "ename": "ValueError",
     "evalue": "issue with the command: b'Using an excessive number of cores has a diminishing return! It is recommended not to exceed 8 cores per trimming process (you asked for 16 cores). Please consider re-specifying\\nPath to Cutadapt set as: \\'cutadapt\\' (default)\\nCutadapt seems to be working fine (tested command \\'cutadapt --version\\')\\nCutadapt version: 3.4\\nCould not detect version of Python used by Cutadapt from the first line of Cutadapt (but found this: >>>#!/bin/sh<<<)\\nLetting the (modified) Cutadapt deal with the Python version instead\\nParallel gzip (pigz) detected. Proceeding with multicore (de)compression using 16 cores\\n\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/monika/data/slamseq_MYB_degraded_timecourse_v1/fastqs/\\nWriting report to \\'/home/monika/data/slamseq_MYB_degraded_timecourse_v1/fastqs/mr450-MV411-MYB-dTAG-VHL_4h-r4_R1_001.fastq.gz_trimming_report.txt\\'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: ../../data/slamseq_MYB_degraded_timecourse_v1/fastqs/mr450-MV411-MYB-dTAG-VHL_4h-r4_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.7\\nCutadapt version: 3.4\\nPython version: could not detect\\nNumber of cores used for trimming: 16\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: \\'AGATCGGAAGAGC\\' (Illumina TruSeq, Sanger iPCR; user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 3 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nLength cut-off for read 1: 35 bp (default)\\nLength cut-off for read 2: 35 bb (default)\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be fairly up-to-date (version 3.4). Setting -j 16\\nWriting final adapter and quality trimmed output to mr450-MV411-MYB-dTAG-VHL_4h-r4_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff \\'-q 20\\') and adapter trimming in a single pass for the adapter sequence: \\'AGATCGGAAGAGC\\' from file ../../data/slamseq_MYB_degraded_timecourse_v1/fastqs/mr450-MV411-MYB-dTAG-VHL_4h-r4_R1_001.fastq.gz <<< \\n10000000 sequences processed\\nThis is cutadapt 3.4 with Python 3.8.11\\nCommand line parameters: -j 16 -e 0.1 -q 20 -O 3 -a AGATCGGAAGAGC ../../data/slamseq_MYB_degraded_timecourse_v1/fastqs/mr450-MV411-MYB-dTAG-VHL_4h-r4_R1_001.fastq.gz\\nProcessing reads on 16 cores in single-end mode ...\\nERROR: Traceback (most recent call last):\\n  File \"/home/monika/anaconda3/envs/slamseq/lib/python3.8/site-packages/cutadapt/pipeline.py\", line 559, in run\\n    for chunk_index, chunk in enumerate(dnaio.read_chunks(f, self.buffer_size)):\\n  File \"/home/monika/anaconda3/envs/slamseq/lib/python3.8/site-packages/dnaio/chunks.py\", line 81, in read_chunks\\n    bufend = f.readinto(memoryview(buf)[start:]) + start  # type: ignore\\n  File \"/home/monika/anaconda3/envs/slamseq/lib/python3.8/gzip.py\", line 292, in read\\n    return self._buffer.read(size)\\nMemoryError\\n\\nERROR: Traceback (most recent call last):\\n  File \"/home/monika/anaconda3/envs/slamseq/lib/python3.8/site-packages/cutadapt/pipeline.py\", line 559, in run\\n    for chunk_index, chunk in enumerate(dnaio.read_chunks(f, self.buffer_size)):\\n  File \"/home/monika/anaconda3/envs/slamseq/lib/python3.8/site-packages/dnaio/chunks.py\", line 81, in read_chunks\\n    bufend = f.readinto(memoryview(buf)[start:]) + start  # type: ignore\\n  File \"/home/monika/anaconda3/envs/slamseq/lib/python3.8/gzip.py\", line 292, in read\\n    return self._buffer.read(size)\\nMemoryError\\n\\nTraceback (most recent call last):\\n  File \"/home/monika/anaconda3/envs/slamseq/bin/cutadapt\", line 10, in <module>\\n    sys.exit(main_cli())\\n  File \"/home/monika/anaconda3/envs/slamseq/lib/python3.8/site-packages/cutadapt/__main__.py\", line 848, in main_cli\\n    main(sys.argv[1:])\\n  File \"/home/monika/anaconda3/envs/slamseq/lib/python3.8/site-packages/cutadapt/__main__.py\", line 913, in main\\n    stats = r.run()\\n  File \"/home/monika/anaconda3/envs/slamseq/lib/python3.8/site-packages/cutadapt/pipeline.py\", line 836, in run\\n    data = connection.recv_bytes()\\n  File \"/home/monika/anaconda3/envs/slamseq/lib/python3.8/multiprocessing/connection.py\", line 216, in recv_bytes\\n    buf = self._recv_bytes(maxlength)\\n  File \"/home/monika/anaconda3/envs/slamseq/lib/python3.8/multiprocessing/connection.py\", line 421, in _recv_bytes\\n    return self._recv(size)\\n  File \"/home/monika/anaconda3/envs/slamseq/lib/python3.8/multiprocessing/connection.py\", line 379, in _recv\\n    chunk = read(handle, remaining)\\nMemoryError\\n\\n\\nCutadapt terminated with exit signal: \\'256\\'.\\nTerminating Trim Galore run, please check error message(s) to get an idea what went wrong...\\n\\n'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mValueError\u001b[0m                                Traceback (most recent call last)",
      "\u001b[0;32m<timed exec>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n",
      "\u001b[0;32m~/genepy/genepy/utils/helper.py\u001b[0m in \u001b[0;36mparrun\u001b[0;34m(cmds, cores, add)\u001b[0m\n\u001b[1;32m    298\u001b[0m       \u001b[0mres\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0msubprocess\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mrun\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mexe\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mcapture_output\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mTrue\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mshell\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mTrue\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    299\u001b[0m       \u001b[0;32mif\u001b[0m \u001b[0mres\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mreturncode\u001b[0m \u001b[0;34m!=\u001b[0m \u001b[0;36m0\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 300\u001b[0;31m         \u001b[0;32mraise\u001b[0m \u001b[0mValueError\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'issue with the command: '\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mres\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mstderr\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    301\u001b[0m       \u001b[0mexe\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m''\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    302\u001b[0m       \u001b[0;32mif\u001b[0m \u001b[0mlen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0madd\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m!=\u001b[0m \u001b[0;36m0\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mValueError\u001b[0m: issue with the command: b'Using an excessive number of cores has a diminishing return! It is recommended not to exceed 8 cores per trimming process (you asked for 16 cores). Please consider re-specifying\\nPath to Cutadapt set as: \\'cutadapt\\' (default)\\nCutadapt seems to be working fine (tested command \\'cutadapt --version\\')\\nCutadapt version: 3.4\\nCould not detect version of Python used by Cutadapt from the first line of Cutadapt (but found this: >>>#!/bin/sh<<<)\\nLetting the (modified) Cutadapt deal with the Python version instead\\nParallel gzip (pigz) detected. Proceeding with multicore (de)compression using 16 cores\\n\\nNo quality encoding type selected. Assuming that the data provided uses Sanger encoded Phred scores (default)\\n\\nOutput will be written into the directory: /home/monika/data/slamseq_MYB_degraded_timecourse_v1/fastqs/\\nWriting report to \\'/home/monika/data/slamseq_MYB_degraded_timecourse_v1/fastqs/mr450-MV411-MYB-dTAG-VHL_4h-r4_R1_001.fastq.gz_trimming_report.txt\\'\\n\\nSUMMARISING RUN PARAMETERS\\n==========================\\nInput filename: ../../data/slamseq_MYB_degraded_timecourse_v1/fastqs/mr450-MV411-MYB-dTAG-VHL_4h-r4_R1_001.fastq.gz\\nTrimming mode: paired-end\\nTrim Galore version: 0.6.7\\nCutadapt version: 3.4\\nPython version: could not detect\\nNumber of cores used for trimming: 16\\nQuality Phred score cutoff: 20\\nQuality encoding type selected: ASCII+33\\nAdapter sequence: \\'AGATCGGAAGAGC\\' (Illumina TruSeq, Sanger iPCR; user defined)\\nMaximum trimming error rate: 0.1 (default)\\nMinimum required adapter overlap (stringency): 3 bp\\nMinimum required sequence length for both reads before a sequence pair gets removed: 20 bp\\nLength cut-off for read 1: 35 bp (default)\\nLength cut-off for read 2: 35 bb (default)\\nOutput file(s) will be GZIP compressed\\n\\nCutadapt seems to be fairly up-to-date (version 3.4). Setting -j 16\\nWriting final adapter and quality trimmed output to mr450-MV411-MYB-dTAG-VHL_4h-r4_R1_001_trimmed.fq.gz\\n\\n\\n  >>> Now performing quality (cutoff \\'-q 20\\') and adapter trimming in a single pass for the adapter sequence: \\'AGATCGGAAGAGC\\' from file ../../data/slamseq_MYB_degraded_timecourse_v1/fastqs/mr450-MV411-MYB-dTAG-VHL_4h-r4_R1_001.fastq.gz <<< \\n10000000 sequences processed\\nThis is cutadapt 3.4 with Python 3.8.11\\nCommand line parameters: -j 16 -e 0.1 -q 20 -O 3 -a AGATCGGAAGAGC ../../data/slamseq_MYB_degraded_timecourse_v1/fastqs/mr450-MV411-MYB-dTAG-VHL_4h-r4_R1_001.fastq.gz\\nProcessing reads on 16 cores in single-end mode ...\\nERROR: Traceback (most recent call last):\\n  File \"/home/monika/anaconda3/envs/slamseq/lib/python3.8/site-packages/cutadapt/pipeline.py\", line 559, in run\\n    for chunk_index, chunk in enumerate(dnaio.read_chunks(f, self.buffer_size)):\\n  File \"/home/monika/anaconda3/envs/slamseq/lib/python3.8/site-packages/dnaio/chunks.py\", line 81, in read_chunks\\n    bufend = f.readinto(memoryview(buf)[start:]) + start  # type: ignore\\n  File \"/home/monika/anaconda3/envs/slamseq/lib/python3.8/gzip.py\", line 292, in read\\n    return self._buffer.read(size)\\nMemoryError\\n\\nERROR: Traceback (most recent call last):\\n  File \"/home/monika/anaconda3/envs/slamseq/lib/python3.8/site-packages/cutadapt/pipeline.py\", line 559, in run\\n    for chunk_index, chunk in enumerate(dnaio.read_chunks(f, self.buffer_size)):\\n  File \"/home/monika/anaconda3/envs/slamseq/lib/python3.8/site-packages/dnaio/chunks.py\", line 81, in read_chunks\\n    bufend = f.readinto(memoryview(buf)[start:]) + start  # type: ignore\\n  File \"/home/monika/anaconda3/envs/slamseq/lib/python3.8/gzip.py\", line 292, in read\\n    return self._buffer.read(size)\\nMemoryError\\n\\nTraceback (most recent call last):\\n  File \"/home/monika/anaconda3/envs/slamseq/bin/cutadapt\", line 10, in <module>\\n    sys.exit(main_cli())\\n  File \"/home/monika/anaconda3/envs/slamseq/lib/python3.8/site-packages/cutadapt/__main__.py\", line 848, in main_cli\\n    main(sys.argv[1:])\\n  File \"/home/monika/anaconda3/envs/slamseq/lib/python3.8/site-packages/cutadapt/__main__.py\", line 913, in main\\n    stats = r.run()\\n  File \"/home/monika/anaconda3/envs/slamseq/lib/python3.8/site-packages/cutadapt/pipeline.py\", line 836, in run\\n    data = connection.recv_bytes()\\n  File \"/home/monika/anaconda3/envs/slamseq/lib/python3.8/multiprocessing/connection.py\", line 216, in recv_bytes\\n    buf = self._recv_bytes(maxlength)\\n  File \"/home/monika/anaconda3/envs/slamseq/lib/python3.8/multiprocessing/connection.py\", line 421, in _recv_bytes\\n    return self._recv(size)\\n  File \"/home/monika/anaconda3/envs/slamseq/lib/python3.8/multiprocessing/connection.py\", line 379, in _recv\\n    chunk = read(handle, remaining)\\nMemoryError\\n\\n\\nCutadapt terminated with exit signal: \\'256\\'.\\nTerminating Trim Galore run, please check error message(s) to get an idea what went wrong...\\n\\n'"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "\n",
    "for val1, val2 in h.grouped(fastqs[56:60], 2):\n",
    "    print(val1)\n",
    "    print(val2)\n",
    "    h.parrun(\n",
    "        ['trim_galore --paired --cores 16 --retain_unpaired -stringency 3 \\\n",
    "        --illumina '+val1+' '+val2+' -o '+fastq_folder], 1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 315,
   "id": "dff0931c",
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 124 ms, sys: 68 ms, total: 192 ms\n",
      "Wall time: 1h 27min 36s\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "['pigz 2.6\\n']"
      ]
     },
     "execution_count": 315,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "%%time\n",
    "h.parrun(['trim_galore --paired --cores 8 --retain_unpaired -stringency 3\\\n",
    " --illumina '+val1+' '+val2+' -o '+fastq_folder for val1, val2 in h.grouped(fastqs[18:20], 2)], 2)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8799a605",
   "metadata": {},
   "source": [
    "# IRF2BP2_degraded_v1 (project 1)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "909251cd",
   "metadata": {},
   "source": [
    "__res_v4 no alt__"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 183,
   "id": "97670a73",
   "metadata": {},
   "outputs": [],
   "source": [
    "project = \"IRF2BP2_degraded_v1\"\n",
    "res_v = \"res_v4\"\n",
    "project_dir = \"slamseq_{}_{}\".format(project, res_v)\n",
    "#proj_dir = \"slamseq_{}\".format(project1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 184,
   "id": "e8706dd6",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "9\n"
     ]
    }
   ],
   "source": [
    "files = ! ls $res_dir/$project_dir/*.csv\n",
    "files = [file for file in files if \"deseq\" in file if \"drop\" not in file]\n",
    "# ERCC files first\n",
    "files = [file for file in files if \"ERCCsamplewise\" in file] + \\\n",
    "[file for file in files if \"ERCCsamplewise\" not in file]\n",
    "print(len(files))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 185,
   "id": "d5b207a4",
   "metadata": {},
   "outputs": [],
   "source": [
    "# remove existing deseq merged output\n",
    "files = files[:-1]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 186,
   "id": "8d1d1b84",
   "metadata": {},
   "outputs": [],
   "source": [
    "# project 1\n",
    "conditions = [\"3h\", \"6h\"]\n",
    "scaling = [\"ERCCsamplewise\", None]\n",
    "count_type = [\"readcounts\", \"tccounts\"]"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4cc9a529",
   "metadata": {},
   "source": [
    "### make info rows"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 187,
   "id": "ea7e114c",
   "metadata": {},
   "outputs": [],
   "source": [
    "# df with file info\n",
    "df_info = pd.DataFrame(list(zip([\"project\", \"file\", \"scale\", \"condition\", \"type\"],\n",
    "                                [\"project\", \"file\", \"scale\", \"condition\", \"type\"])))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 188,
   "id": "19fa5387",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(5, 50)"
      ]
     },
     "execution_count": 188,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_info = pd.concat([df_info] + [pd.DataFrame([x*6 for x in get_file_info(project, file, conditions)]) \\\n",
    "                                 for file in files], axis=1)\n",
    "df_info.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 189,
   "id": "a071e6bb",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "      <th>3</th>\n",
       "      <th>4</th>\n",
       "      <th>5</th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>...</th>\n",
       "      <th>2</th>\n",
       "      <th>3</th>\n",
       "      <th>4</th>\n",
       "      <th>5</th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "      <th>3</th>\n",
       "      <th>4</th>\n",
       "      <th>5</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>project</td>\n",
       "      <td>project</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>...</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>file</td>\n",
       "      <td>file</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>...</td>\n",
       "      <td>IRF2BP2_degraded_v1_0_5_6h_deseq_readcounts.csv</td>\n",
       "      <td>IRF2BP2_degraded_v1_0_5_6h_deseq_readcounts.csv</td>\n",
       "      <td>IRF2BP2_degraded_v1_0_5_6h_deseq_readcounts.csv</td>\n",
       "      <td>IRF2BP2_degraded_v1_0_5_6h_deseq_readcounts.csv</td>\n",
       "      <td>IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts.csv</td>\n",
       "      <td>IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts.csv</td>\n",
       "      <td>IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts.csv</td>\n",
       "      <td>IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts.csv</td>\n",
       "      <td>IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts.csv</td>\n",
       "      <td>IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts.csv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>scale</td>\n",
       "      <td>scale</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>condition</td>\n",
       "      <td>condition</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>...</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>type</td>\n",
       "      <td>type</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>...</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows Ã— 50 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "           0          1                                                  0  \\\n",
       "0    project    project                                IRF2BP2_degraded_v1   \n",
       "1       file       file  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2      scale      scale                                     ERCCsamplewise   \n",
       "3  condition  condition                                                 3h   \n",
       "4       type       type                                         readcounts   \n",
       "\n",
       "                                                   1  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                   2  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                   3  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                   4  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                   5  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                   0  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                           tccounts   \n",
       "\n",
       "                                                   1  ...  \\\n",
       "0                                IRF2BP2_degraded_v1  ...   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...  ...   \n",
       "2                                     ERCCsamplewise  ...   \n",
       "3                                                 3h  ...   \n",
       "4                                           tccounts  ...   \n",
       "\n",
       "                                                 2  \\\n",
       "0                              IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_0_5_6h_deseq_readcounts.csv   \n",
       "2                                             None   \n",
       "3                                               6h   \n",
       "4                                       readcounts   \n",
       "\n",
       "                                                 3  \\\n",
       "0                              IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_0_5_6h_deseq_readcounts.csv   \n",
       "2                                             None   \n",
       "3                                               6h   \n",
       "4                                       readcounts   \n",
       "\n",
       "                                                 4  \\\n",
       "0                              IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_0_5_6h_deseq_readcounts.csv   \n",
       "2                                             None   \n",
       "3                                               6h   \n",
       "4                                       readcounts   \n",
       "\n",
       "                                                 5  \\\n",
       "0                              IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_0_5_6h_deseq_readcounts.csv   \n",
       "2                                             None   \n",
       "3                                               6h   \n",
       "4                                       readcounts   \n",
       "\n",
       "                                               0  \\\n",
       "0                            IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts.csv   \n",
       "2                                           None   \n",
       "3                                             6h   \n",
       "4                                       tccounts   \n",
       "\n",
       "                                               1  \\\n",
       "0                            IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts.csv   \n",
       "2                                           None   \n",
       "3                                             6h   \n",
       "4                                       tccounts   \n",
       "\n",
       "                                               2  \\\n",
       "0                            IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts.csv   \n",
       "2                                           None   \n",
       "3                                             6h   \n",
       "4                                       tccounts   \n",
       "\n",
       "                                               3  \\\n",
       "0                            IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts.csv   \n",
       "2                                           None   \n",
       "3                                             6h   \n",
       "4                                       tccounts   \n",
       "\n",
       "                                               4  \\\n",
       "0                            IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts.csv   \n",
       "2                                           None   \n",
       "3                                             6h   \n",
       "4                                       tccounts   \n",
       "\n",
       "                                               5  \n",
       "0                            IRF2BP2_degraded_v1  \n",
       "1  IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts.csv  \n",
       "2                                           None  \n",
       "3                                             6h  \n",
       "4                                       tccounts  \n",
       "\n",
       "[5 rows x 50 columns]"
      ]
     },
     "execution_count": 189,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_info.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ff766e32",
   "metadata": {},
   "source": [
    "### get dfs and merge"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 190,
   "id": "d20142eb",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Get list of deseq dfs\n",
    "deseq_dfs = [get_deseq_data(file) for file in files]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "id": "30435b37",
   "metadata": {},
   "outputs": [],
   "source": [
    "# main df\n",
    "df_merged = pd.DataFrame()\n",
    "df_merged = df_merged.append(deseq_dfs[0][[\"genes\", \"gene_id\"]]) # add gene cols to main df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "id": "ef2b6160",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(15632, 50)"
      ]
     },
     "execution_count": 31,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# join df columns on gene & gene_id columns\n",
    "df_merged = reduce(lambda left, right: pd.merge(left, right, on=['genes', \"gene_id\"], how=\"outer\"), \n",
    "                   [df_merged] + deseq_dfs)\n",
    "df_merged.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "id": "c16bea19",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>genes</th>\n",
       "      <th>gene_id</th>\n",
       "      <th>baseMean_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts</th>\n",
       "      <th>log2FoldChange_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts</th>\n",
       "      <th>lfcSE_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts</th>\n",
       "      <th>stat_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts</th>\n",
       "      <th>pvalue_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts</th>\n",
       "      <th>padj_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts</th>\n",
       "      <th>baseMean_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_tccounts</th>\n",
       "      <th>log2FoldChange_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_tccounts</th>\n",
       "      <th>...</th>\n",
       "      <th>lfcSE_IRF2BP2_degraded_v1_0_5_6h_deseq_readcounts</th>\n",
       "      <th>stat_IRF2BP2_degraded_v1_0_5_6h_deseq_readcounts</th>\n",
       "      <th>pvalue_IRF2BP2_degraded_v1_0_5_6h_deseq_readcounts</th>\n",
       "      <th>padj_IRF2BP2_degraded_v1_0_5_6h_deseq_readcounts</th>\n",
       "      <th>baseMean_IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts</th>\n",
       "      <th>log2FoldChange_IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts</th>\n",
       "      <th>lfcSE_IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts</th>\n",
       "      <th>stat_IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts</th>\n",
       "      <th>pvalue_IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts</th>\n",
       "      <th>padj_IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NAT1</td>\n",
       "      <td>NAT1</td>\n",
       "      <td>95.870596</td>\n",
       "      <td>-0.461812</td>\n",
       "      <td>0.183899</td>\n",
       "      <td>2.511222</td>\n",
       "      <td>0.012031</td>\n",
       "      <td>0.176482</td>\n",
       "      <td>5.033275</td>\n",
       "      <td>-0.560236</td>\n",
       "      <td>...</td>\n",
       "      <td>0.249984</td>\n",
       "      <td>1.345021</td>\n",
       "      <td>0.178618</td>\n",
       "      <td>0.918252</td>\n",
       "      <td>0.752630</td>\n",
       "      <td>2.004205</td>\n",
       "      <td>1.827815</td>\n",
       "      <td>-1.096503</td>\n",
       "      <td>0.272859</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>AAMP</td>\n",
       "      <td>AAMP</td>\n",
       "      <td>842.371621</td>\n",
       "      <td>-0.147606</td>\n",
       "      <td>0.118079</td>\n",
       "      <td>1.250062</td>\n",
       "      <td>0.211277</td>\n",
       "      <td>0.621102</td>\n",
       "      <td>14.553492</td>\n",
       "      <td>0.310173</td>\n",
       "      <td>...</td>\n",
       "      <td>0.143612</td>\n",
       "      <td>-1.174215</td>\n",
       "      <td>0.240309</td>\n",
       "      <td>0.959450</td>\n",
       "      <td>7.433181</td>\n",
       "      <td>0.284259</td>\n",
       "      <td>0.600667</td>\n",
       "      <td>-0.473238</td>\n",
       "      <td>0.636043</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>AARS1</td>\n",
       "      <td>AARS1</td>\n",
       "      <td>1374.176487</td>\n",
       "      <td>-0.025248</td>\n",
       "      <td>0.109373</td>\n",
       "      <td>0.230845</td>\n",
       "      <td>0.817435</td>\n",
       "      <td>0.957162</td>\n",
       "      <td>16.200285</td>\n",
       "      <td>-0.137228</td>\n",
       "      <td>...</td>\n",
       "      <td>0.167173</td>\n",
       "      <td>-0.658551</td>\n",
       "      <td>0.510184</td>\n",
       "      <td>0.996674</td>\n",
       "      <td>11.302474</td>\n",
       "      <td>0.073024</td>\n",
       "      <td>0.490502</td>\n",
       "      <td>-0.148877</td>\n",
       "      <td>0.881651</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>ABAT</td>\n",
       "      <td>ABAT</td>\n",
       "      <td>55.922405</td>\n",
       "      <td>-0.358522</td>\n",
       "      <td>0.211700</td>\n",
       "      <td>1.693536</td>\n",
       "      <td>0.090354</td>\n",
       "      <td>0.438694</td>\n",
       "      <td>1.693745</td>\n",
       "      <td>-0.949985</td>\n",
       "      <td>...</td>\n",
       "      <td>0.268125</td>\n",
       "      <td>0.426446</td>\n",
       "      <td>0.669783</td>\n",
       "      <td>0.996674</td>\n",
       "      <td>0.634574</td>\n",
       "      <td>-1.007275</td>\n",
       "      <td>2.080292</td>\n",
       "      <td>0.484199</td>\n",
       "      <td>0.628245</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>ABCA1</td>\n",
       "      <td>ABCA1</td>\n",
       "      <td>194.049987</td>\n",
       "      <td>0.072710</td>\n",
       "      <td>0.160794</td>\n",
       "      <td>-0.452190</td>\n",
       "      <td>0.651132</td>\n",
       "      <td>0.902410</td>\n",
       "      <td>6.337421</td>\n",
       "      <td>-0.445437</td>\n",
       "      <td>...</td>\n",
       "      <td>0.274569</td>\n",
       "      <td>4.616960</td>\n",
       "      <td>0.000004</td>\n",
       "      <td>0.000465</td>\n",
       "      <td>1.389801</td>\n",
       "      <td>-1.065289</td>\n",
       "      <td>1.377289</td>\n",
       "      <td>0.773468</td>\n",
       "      <td>0.439245</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows Ã— 50 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "   genes gene_id  \\\n",
       "0   NAT1    NAT1   \n",
       "1   AAMP    AAMP   \n",
       "2  AARS1   AARS1   \n",
       "3   ABAT    ABAT   \n",
       "4  ABCA1   ABCA1   \n",
       "\n",
       "   baseMean_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts  \\\n",
       "0                                          95.870596                     \n",
       "1                                         842.371621                     \n",
       "2                                        1374.176487                     \n",
       "3                                          55.922405                     \n",
       "4                                         194.049987                     \n",
       "\n",
       "   log2FoldChange_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts  \\\n",
       "0                                          -0.461812                           \n",
       "1                                          -0.147606                           \n",
       "2                                          -0.025248                           \n",
       "3                                          -0.358522                           \n",
       "4                                           0.072710                           \n",
       "\n",
       "   lfcSE_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts  \\\n",
       "0                                           0.183899                  \n",
       "1                                           0.118079                  \n",
       "2                                           0.109373                  \n",
       "3                                           0.211700                  \n",
       "4                                           0.160794                  \n",
       "\n",
       "   stat_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts  \\\n",
       "0                                           2.511222                 \n",
       "1                                           1.250062                 \n",
       "2                                           0.230845                 \n",
       "3                                           1.693536                 \n",
       "4                                          -0.452190                 \n",
       "\n",
       "   pvalue_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts  \\\n",
       "0                                           0.012031                   \n",
       "1                                           0.211277                   \n",
       "2                                           0.817435                   \n",
       "3                                           0.090354                   \n",
       "4                                           0.651132                   \n",
       "\n",
       "   padj_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts  \\\n",
       "0                                           0.176482                 \n",
       "1                                           0.621102                 \n",
       "2                                           0.957162                 \n",
       "3                                           0.438694                 \n",
       "4                                           0.902410                 \n",
       "\n",
       "   baseMean_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_tccounts  \\\n",
       "0                                           5.033275                   \n",
       "1                                          14.553492                   \n",
       "2                                          16.200285                   \n",
       "3                                           1.693745                   \n",
       "4                                           6.337421                   \n",
       "\n",
       "   log2FoldChange_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_tccounts  \\\n",
       "0                                          -0.560236                         \n",
       "1                                           0.310173                         \n",
       "2                                          -0.137228                         \n",
       "3                                          -0.949985                         \n",
       "4                                          -0.445437                         \n",
       "\n",
       "   ...  lfcSE_IRF2BP2_degraded_v1_0_5_6h_deseq_readcounts  \\\n",
       "0  ...                                           0.249984   \n",
       "1  ...                                           0.143612   \n",
       "2  ...                                           0.167173   \n",
       "3  ...                                           0.268125   \n",
       "4  ...                                           0.274569   \n",
       "\n",
       "   stat_IRF2BP2_degraded_v1_0_5_6h_deseq_readcounts  \\\n",
       "0                                          1.345021   \n",
       "1                                         -1.174215   \n",
       "2                                         -0.658551   \n",
       "3                                          0.426446   \n",
       "4                                          4.616960   \n",
       "\n",
       "   pvalue_IRF2BP2_degraded_v1_0_5_6h_deseq_readcounts  \\\n",
       "0                                           0.178618    \n",
       "1                                           0.240309    \n",
       "2                                           0.510184    \n",
       "3                                           0.669783    \n",
       "4                                           0.000004    \n",
       "\n",
       "   padj_IRF2BP2_degraded_v1_0_5_6h_deseq_readcounts  \\\n",
       "0                                          0.918252   \n",
       "1                                          0.959450   \n",
       "2                                          0.996674   \n",
       "3                                          0.996674   \n",
       "4                                          0.000465   \n",
       "\n",
       "   baseMean_IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts  \\\n",
       "0                                           0.752630    \n",
       "1                                           7.433181    \n",
       "2                                          11.302474    \n",
       "3                                           0.634574    \n",
       "4                                           1.389801    \n",
       "\n",
       "   log2FoldChange_IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts  \\\n",
       "0                                           2.004205          \n",
       "1                                           0.284259          \n",
       "2                                           0.073024          \n",
       "3                                          -1.007275          \n",
       "4                                          -1.065289          \n",
       "\n",
       "   lfcSE_IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts  \\\n",
       "0                                         1.827815   \n",
       "1                                         0.600667   \n",
       "2                                         0.490502   \n",
       "3                                         2.080292   \n",
       "4                                         1.377289   \n",
       "\n",
       "   stat_IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts  \\\n",
       "0                                       -1.096503   \n",
       "1                                       -0.473238   \n",
       "2                                       -0.148877   \n",
       "3                                        0.484199   \n",
       "4                                        0.773468   \n",
       "\n",
       "   pvalue_IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts  \\\n",
       "0                                          0.272859   \n",
       "1                                          0.636043   \n",
       "2                                          0.881651   \n",
       "3                                          0.628245   \n",
       "4                                          0.439245   \n",
       "\n",
       "   padj_IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts  \n",
       "0                                             NaN  \n",
       "1                                             NaN  \n",
       "2                                             NaN  \n",
       "3                                             NaN  \n",
       "4                                             NaN  \n",
       "\n",
       "[5 rows x 50 columns]"
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_merged.iloc[:5]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "id": "db470960",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(15633, 50)"
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# change column names\n",
    "df_merged.columns = [col_name.split(\"_\")[0] if \"gene\" not in col_name else \n",
    "                     col_name for col_name in df_merged.columns.tolist()]\n",
    "df_merged = df_merged.T.reset_index().T.reset_index(drop=True)\n",
    "df_merged.shape"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "cd10bd11",
   "metadata": {
    "heading_collapsed": true
   },
   "source": [
    "### add info columns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "id": "05a61beb",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(5, 50)\n",
      "(15633, 50)\n"
     ]
    }
   ],
   "source": [
    "print(df_info.shape)\n",
    "print(df_merged.shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "id": "3497f580",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "# reset column index\n",
    "df_info = df_info.T.reset_index(drop=True).T"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "id": "f66b264f",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 4 ms, sys: 0 ns, total: 4 ms\n",
      "Wall time: 5.72 ms\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "(15638, 50)"
      ]
     },
     "execution_count": 37,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "%%time\n",
    "\n",
    "# combine info and data\n",
    "df_save = df_info.append(df_merged, ignore_index=True)\n",
    "df_save.shape"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2bf6308c",
   "metadata": {
    "heading_collapsed": true
   },
   "source": [
    "### save merged file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "id": "dc15002c",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved DESeq results to: slamseq_IRF2BP2_degraded_v1_res_v4/slamseq_IRF2BP2_degraded_v1_res_v4_0_5_deseq_output.csv\n"
     ]
    }
   ],
   "source": [
    "# save file\n",
    "res_dir = \"../results\"\n",
    "deseq_res_file = \"slamseq_{}_{}_0_5_deseq_output.csv\".format(project, res_v)\n",
    "\n",
    "df_save.to_csv(\"{}/{}/{}\".format(res_dir, project_dir, deseq_res_file), header=False, index=False)\n",
    "\n",
    "print(\"Saved DESeq results to: {}/{}\".format(project_dir, deseq_res_file))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "68e69906",
   "metadata": {},
   "source": [
    "# MYB_degraded_v1 (project 2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "id": "68809d3d",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(17275, 74)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/monika/anaconda3/envs/slamseq/lib/python3.8/site-packages/IPython/core/interactiveshell.py:3441: DtypeWarning: Columns (0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73) have mixed types.Specify dtype option on import or set low_memory=False.\n",
      "  exec(code_obj, self.user_global_ns, self.user_ns)\n"
     ]
    }
   ],
   "source": [
    "# read in merged deseq file\n",
    "res_dir = \"../results\"\n",
    "deseq_res_file = \"slamseq_merged_deseq_output.csv\"\n",
    "\n",
    "df_save = pd.read_csv(\"{}/{}\".format(res_dir, deseq_res_file), index_col=False, header=None, squeeze=True)\n",
    "print(df_save.shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "id": "089da1c8",
   "metadata": {},
   "outputs": [],
   "source": [
    "project = project2\n",
    "project_dir = \"slamseq_{}\".format(project)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "id": "15cb9725",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "4\n"
     ]
    }
   ],
   "source": [
    "files = ! ls $res_dir/$project_dir/*.csv\n",
    "files = [file for file in files if \"deseq\" in file if \"drop\" not in file]\n",
    "# ERCC files first\n",
    "files = [file for file in files if \"ERCCsamplewise\" in file] + \\\n",
    "[file for file in files if \"ERCCsamplewise\" not in file]\n",
    "print(len(files))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "id": "4d83992e",
   "metadata": {},
   "outputs": [],
   "source": [
    "# project 2\n",
    "conditions = [\"1h\"]\n",
    "scaling = [\"ERCCsamplewise\", None]\n",
    "count_type = [\"readcounts\", \"tccounts\"]"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d0e0ddc6",
   "metadata": {
    "heading_collapsed": true
   },
   "source": [
    "### make info rows"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 164,
   "id": "16a21716",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "# df with file info\n",
    "df_info = pd.DataFrame(list(zip([\"project\", \"file\", \"scale\", \"condition\", \"type\"],\n",
    "                                [\"project\", \"file\", \"scale\", \"condition\", \"type\"])))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 165,
   "id": "f99ac4e8",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(5, 26)"
      ]
     },
     "execution_count": 165,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_info = pd.concat([df_info] + [pd.DataFrame([x*6 for x in get_file_info(project2, file, conditions)]) \\\n",
    "                                 for file in files], axis=1)\n",
    "df_info.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 168,
   "id": "b1e5e401",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "      <th>3</th>\n",
       "      <th>4</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>project</td>\n",
       "      <td>project</td>\n",
       "      <td>MYB_degraded_v1</td>\n",
       "      <td>MYB_degraded_v1</td>\n",
       "      <td>MYB_degraded_v1</td>\n",
       "      <td>MYB_degraded_v1</td>\n",
       "      <td>MYB_degraded_v1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>file</td>\n",
       "      <td>file</td>\n",
       "      <td>MYB_degraded_v1_0_5_1h_deseq_tccounts.csv</td>\n",
       "      <td>MYB_degraded_v1_0_5_1h_deseq_tccounts.csv</td>\n",
       "      <td>MYB_degraded_v1_0_5_1h_deseq_tccounts.csv</td>\n",
       "      <td>MYB_degraded_v1_0_5_1h_deseq_tccounts.csv</td>\n",
       "      <td>MYB_degraded_v1_0_5_1h_deseq_tccounts.csv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>scale</td>\n",
       "      <td>scale</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>condition</td>\n",
       "      <td>condition</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>type</td>\n",
       "      <td>type</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "           0          1                                          0  \\\n",
       "0    project    project                            MYB_degraded_v1   \n",
       "1       file       file  MYB_degraded_v1_0_5_1h_deseq_tccounts.csv   \n",
       "2      scale      scale                                       None   \n",
       "3  condition  condition                                         1h   \n",
       "4       type       type                                   tccounts   \n",
       "\n",
       "                                           1  \\\n",
       "0                            MYB_degraded_v1   \n",
       "1  MYB_degraded_v1_0_5_1h_deseq_tccounts.csv   \n",
       "2                                       None   \n",
       "3                                         1h   \n",
       "4                                   tccounts   \n",
       "\n",
       "                                           2  \\\n",
       "0                            MYB_degraded_v1   \n",
       "1  MYB_degraded_v1_0_5_1h_deseq_tccounts.csv   \n",
       "2                                       None   \n",
       "3                                         1h   \n",
       "4                                   tccounts   \n",
       "\n",
       "                                           3  \\\n",
       "0                            MYB_degraded_v1   \n",
       "1  MYB_degraded_v1_0_5_1h_deseq_tccounts.csv   \n",
       "2                                       None   \n",
       "3                                         1h   \n",
       "4                                   tccounts   \n",
       "\n",
       "                                           4  \n",
       "0                            MYB_degraded_v1  \n",
       "1  MYB_degraded_v1_0_5_1h_deseq_tccounts.csv  \n",
       "2                                       None  \n",
       "3                                         1h  \n",
       "4                                   tccounts  "
      ]
     },
     "execution_count": 168,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_info.iloc[:,[0,1,20,21,22,23,24]]"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "bff6f9ed",
   "metadata": {},
   "source": [
    "### merge dfs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "629c99a9",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>baseMean_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>log2FoldChange_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>lfcSE_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>stat_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>pvalue_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>padj_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>genes</th>\n",
       "      <th>gene_id</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1.877541</td>\n",
       "      <td>-0.045789</td>\n",
       "      <td>1.010641</td>\n",
       "      <td>0.045307</td>\n",
       "      <td>0.963863</td>\n",
       "      <td>NaN</td>\n",
       "      <td>A1BG</td>\n",
       "      <td>A1BG</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   baseMean_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "0                                           1.877541                 \n",
       "\n",
       "   log2FoldChange_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "0                                          -0.045789                       \n",
       "\n",
       "   lfcSE_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "0                                           1.010641              \n",
       "\n",
       "   stat_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "0                                           0.045307             \n",
       "\n",
       "   pvalue_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "0                                           0.963863               \n",
       "\n",
       "   padj_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts genes gene_id  \n",
       "0                                                NaN            A1BG    A1BG  "
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Get list of deseq dfs\n",
    "deseq_dfs = [get_deseq_data(file) for file in files]\n",
    "deseq_dfs[0].iloc[:1]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "4a747a6c",
   "metadata": {},
   "outputs": [],
   "source": [
    "# main df\n",
    "df_merged = pd.DataFrame()\n",
    "df_merged = df_merged.append(deseq_dfs[0][[\"genes\", \"gene_id\"]]) # add gene cols to main df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "ff1c3bf4",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(16558, 26)"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# join df columns on gene & gene_id columns\n",
    "df_merged = reduce(lambda left, right: pd.merge(left, right, on=['genes', \"gene_id\"], how=\"outer\"), \n",
    "                   [df_merged] + deseq_dfs)\n",
    "df_merged.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 146,
   "id": "c8b23ca7",
   "metadata": {},
   "outputs": [],
   "source": [
    "# change column names\n",
    "df_merged.columns = [col_name.split(\"_\")[0] if \"gene\" not in col_name else \n",
    "                     col_name for col_name in df_merged.columns.tolist()]\n",
    "df_merged = df_merged.T.reset_index().T.reset_index(drop=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d143f6a5",
   "metadata": {},
   "source": [
    "### merge info cols"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 147,
   "id": "dc4f8244",
   "metadata": {},
   "outputs": [],
   "source": [
    "df_save_info = df_save.iloc[:5,]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 171,
   "id": "a0ab4c09",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(5, 50)\n",
      "(5, 26)\n"
     ]
    }
   ],
   "source": [
    "print(df_save_info.shape)\n",
    "print(df_info.shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 170,
   "id": "5cd7aaed",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "      <th>3</th>\n",
       "      <th>4</th>\n",
       "      <th>5</th>\n",
       "      <th>6</th>\n",
       "      <th>7</th>\n",
       "      <th>8</th>\n",
       "      <th>9</th>\n",
       "      <th>...</th>\n",
       "      <th>40</th>\n",
       "      <th>41</th>\n",
       "      <th>42</th>\n",
       "      <th>43</th>\n",
       "      <th>44</th>\n",
       "      <th>45</th>\n",
       "      <th>46</th>\n",
       "      <th>47</th>\n",
       "      <th>48</th>\n",
       "      <th>49</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>project</td>\n",
       "      <td>project</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>...</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>file</td>\n",
       "      <td>file</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>...</td>\n",
       "      <td>IRF2BP2_degraded_v1_0_5_6h_deseq_readcounts.csv</td>\n",
       "      <td>IRF2BP2_degraded_v1_0_5_6h_deseq_readcounts.csv</td>\n",
       "      <td>IRF2BP2_degraded_v1_0_5_6h_deseq_readcounts.csv</td>\n",
       "      <td>IRF2BP2_degraded_v1_0_5_6h_deseq_readcounts.csv</td>\n",
       "      <td>IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts.csv</td>\n",
       "      <td>IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts.csv</td>\n",
       "      <td>IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts.csv</td>\n",
       "      <td>IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts.csv</td>\n",
       "      <td>IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts.csv</td>\n",
       "      <td>IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts.csv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>scale</td>\n",
       "      <td>scale</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>condition</td>\n",
       "      <td>condition</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>...</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>type</td>\n",
       "      <td>type</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>...</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows Ã— 50 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "          0          1                                                  2   \\\n",
       "0    project    project                                IRF2BP2_degraded_v1   \n",
       "1       file       file  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2      scale      scale                                     ERCCsamplewise   \n",
       "3  condition  condition                                                 3h   \n",
       "4       type       type                                         readcounts   \n",
       "\n",
       "                                                  3   \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  4   \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  5   \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  6   \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  7   \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  8   \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                           tccounts   \n",
       "\n",
       "                                                  9   ...  \\\n",
       "0                                IRF2BP2_degraded_v1  ...   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...  ...   \n",
       "2                                     ERCCsamplewise  ...   \n",
       "3                                                 3h  ...   \n",
       "4                                           tccounts  ...   \n",
       "\n",
       "                                                40  \\\n",
       "0                              IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_0_5_6h_deseq_readcounts.csv   \n",
       "2                                              NaN   \n",
       "3                                               6h   \n",
       "4                                       readcounts   \n",
       "\n",
       "                                                41  \\\n",
       "0                              IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_0_5_6h_deseq_readcounts.csv   \n",
       "2                                              NaN   \n",
       "3                                               6h   \n",
       "4                                       readcounts   \n",
       "\n",
       "                                                42  \\\n",
       "0                              IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_0_5_6h_deseq_readcounts.csv   \n",
       "2                                              NaN   \n",
       "3                                               6h   \n",
       "4                                       readcounts   \n",
       "\n",
       "                                                43  \\\n",
       "0                              IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_0_5_6h_deseq_readcounts.csv   \n",
       "2                                              NaN   \n",
       "3                                               6h   \n",
       "4                                       readcounts   \n",
       "\n",
       "                                              44  \\\n",
       "0                            IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts.csv   \n",
       "2                                            NaN   \n",
       "3                                             6h   \n",
       "4                                       tccounts   \n",
       "\n",
       "                                              45  \\\n",
       "0                            IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts.csv   \n",
       "2                                            NaN   \n",
       "3                                             6h   \n",
       "4                                       tccounts   \n",
       "\n",
       "                                              46  \\\n",
       "0                            IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts.csv   \n",
       "2                                            NaN   \n",
       "3                                             6h   \n",
       "4                                       tccounts   \n",
       "\n",
       "                                              47  \\\n",
       "0                            IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts.csv   \n",
       "2                                            NaN   \n",
       "3                                             6h   \n",
       "4                                       tccounts   \n",
       "\n",
       "                                              48  \\\n",
       "0                            IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts.csv   \n",
       "2                                            NaN   \n",
       "3                                             6h   \n",
       "4                                       tccounts   \n",
       "\n",
       "                                              49  \n",
       "0                            IRF2BP2_degraded_v1  \n",
       "1  IRF2BP2_degraded_v1_0_5_6h_deseq_tccounts.csv  \n",
       "2                                            NaN  \n",
       "3                                             6h  \n",
       "4                                       tccounts  \n",
       "\n",
       "[5 rows x 50 columns]"
      ]
     },
     "execution_count": 170,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_save_info"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 173,
   "id": "ab4873f2",
   "metadata": {},
   "outputs": [],
   "source": [
    "# reset column index\n",
    "df_info = df_info.T.reset_index(drop=True).T"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 177,
   "id": "1eb7235d",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(5, 74)"
      ]
     },
     "execution_count": 177,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# merge info columns\n",
    "df_info_merged = pd.merge(df_save_info, df_info, left_on=[0,1], right_on=[0,1])\n",
    "df_info_merged = df_info_merged.T.reset_index(drop=True).T\n",
    "df_info_merged.shape"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ab55eb63",
   "metadata": {},
   "source": [
    "### merge gene data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "05a23d83",
   "metadata": {},
   "outputs": [],
   "source": [
    "# main df\n",
    "df_merged = pd.DataFrame()\n",
    "df_merged = df_merged.append(deseq_dfs[0][[\"genes\", \"gene_id\"]]) # add gene cols to main df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 179,
   "id": "ed122ffc",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(16559, 26)\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "      <th>3</th>\n",
       "      <th>4</th>\n",
       "      <th>5</th>\n",
       "      <th>6</th>\n",
       "      <th>7</th>\n",
       "      <th>8</th>\n",
       "      <th>9</th>\n",
       "      <th>...</th>\n",
       "      <th>16</th>\n",
       "      <th>17</th>\n",
       "      <th>18</th>\n",
       "      <th>19</th>\n",
       "      <th>20</th>\n",
       "      <th>21</th>\n",
       "      <th>22</th>\n",
       "      <th>23</th>\n",
       "      <th>24</th>\n",
       "      <th>25</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>genes</td>\n",
       "      <td>gene_id</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>...</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>A1BG</td>\n",
       "      <td>A1BG</td>\n",
       "      <td>1.877541</td>\n",
       "      <td>-0.045789</td>\n",
       "      <td>1.010641</td>\n",
       "      <td>0.045307</td>\n",
       "      <td>0.963863</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.05833</td>\n",
       "      <td>-0.614498</td>\n",
       "      <td>...</td>\n",
       "      <td>0.980153</td>\n",
       "      <td>-0.116824</td>\n",
       "      <td>0.907</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.05833</td>\n",
       "      <td>-0.499954</td>\n",
       "      <td>3.11654</td>\n",
       "      <td>0.16042</td>\n",
       "      <td>0.87255</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>A2MP1</td>\n",
       "      <td>A2MP1</td>\n",
       "      <td>5.229468</td>\n",
       "      <td>-0.26904</td>\n",
       "      <td>0.500213</td>\n",
       "      <td>0.537851</td>\n",
       "      <td>0.59068</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>-0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.478116</td>\n",
       "      <td>0.372684</td>\n",
       "      <td>0.709383</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>-0.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>3 rows Ã— 26 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "      0        1         2               3         4         5         6   \\\n",
       "0  genes  gene_id  baseMean  log2FoldChange     lfcSE      stat    pvalue   \n",
       "1   A1BG     A1BG  1.877541       -0.045789  1.010641  0.045307  0.963863   \n",
       "2  A2MP1    A2MP1  5.229468        -0.26904  0.500213  0.537851   0.59068   \n",
       "\n",
       "     7         8               9   ...        16        17        18    19  \\\n",
       "0  padj  baseMean  log2FoldChange  ...     lfcSE      stat    pvalue  padj   \n",
       "1   NaN   0.05833       -0.614498  ...  0.980153 -0.116824     0.907   NaN   \n",
       "2   NaN       0.0            -0.0  ...  0.478116  0.372684  0.709383   NaN   \n",
       "\n",
       "         20              21       22       23       24    25  \n",
       "0  baseMean  log2FoldChange    lfcSE     stat   pvalue  padj  \n",
       "1   0.05833       -0.499954  3.11654  0.16042  0.87255   NaN  \n",
       "2       0.0            -0.0      NaN      NaN      0.0   NaN  \n",
       "\n",
       "[3 rows x 26 columns]"
      ]
     },
     "execution_count": 179,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "print(df_merged.shape)\n",
    "df_merged.iloc[:3]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 182,
   "id": "5ce9cde4",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(15113, 50)"
      ]
     },
     "execution_count": 182,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# merged data w/o info\n",
    "df_saved_output = df_save.iloc[5:]\n",
    "df_saved_output.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 190,
   "id": "8cff7285",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "15112\n",
      "16558\n",
      "Total unique genes combined: 17269\n"
     ]
    }
   ],
   "source": [
    "saved_genes = df_saved_output.iloc[1:,0].tolist()\n",
    "print(len(saved_genes))\n",
    "new_genes = df_merged.iloc[1:,0].tolist()\n",
    "print(len(new_genes))\n",
    "print(\"Total unique genes combined: {}\".format(len(set(saved_genes+new_genes))))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 194,
   "id": "1b24dde7",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "      <th>3</th>\n",
       "      <th>4</th>\n",
       "      <th>5</th>\n",
       "      <th>6</th>\n",
       "      <th>7</th>\n",
       "      <th>8</th>\n",
       "      <th>9</th>\n",
       "      <th>...</th>\n",
       "      <th>40</th>\n",
       "      <th>41</th>\n",
       "      <th>42</th>\n",
       "      <th>43</th>\n",
       "      <th>44</th>\n",
       "      <th>45</th>\n",
       "      <th>46</th>\n",
       "      <th>47</th>\n",
       "      <th>48</th>\n",
       "      <th>49</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>genes</td>\n",
       "      <td>gene_id</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>...</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>NAT1</td>\n",
       "      <td>NAT1</td>\n",
       "      <td>98.73474046943294</td>\n",
       "      <td>-0.3588695183526639</td>\n",
       "      <td>0.1795865924500918</td>\n",
       "      <td>1.998309080074538</td>\n",
       "      <td>0.0456831617026497</td>\n",
       "      <td>0.5573132426854484</td>\n",
       "      <td>5.021740199533236</td>\n",
       "      <td>-0.5547845714960977</td>\n",
       "      <td>...</td>\n",
       "      <td>0.2491509542089338</td>\n",
       "      <td>1.3371348848527798</td>\n",
       "      <td>0.181178611933578</td>\n",
       "      <td>0.8915384574240992</td>\n",
       "      <td>0.7530778794721894</td>\n",
       "      <td>2.014292355431807</td>\n",
       "      <td>1.8320814934075496</td>\n",
       "      <td>-1.0994556534083848</td>\n",
       "      <td>0.2715693670830352</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>AAMP</td>\n",
       "      <td>AAMP</td>\n",
       "      <td>872.5000270269832</td>\n",
       "      <td>-0.0418951057520827</td>\n",
       "      <td>0.1166181865869584</td>\n",
       "      <td>0.3592501905424756</td>\n",
       "      <td>0.7194079326130601</td>\n",
       "      <td>0.9684493014304824</td>\n",
       "      <td>14.549760596196997</td>\n",
       "      <td>0.3164736874507167</td>\n",
       "      <td>...</td>\n",
       "      <td>0.1441953888640895</td>\n",
       "      <td>-1.1898593830650157</td>\n",
       "      <td>0.2341016648908468</td>\n",
       "      <td>0.9320326022466292</td>\n",
       "      <td>7.440114778273713</td>\n",
       "      <td>0.290637206923639</td>\n",
       "      <td>0.6010687521535898</td>\n",
       "      <td>-0.483534048113972</td>\n",
       "      <td>0.6287165903309597</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>3 rows Ã— 50 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "      0        1                  2                    3                   4   \\\n",
       "5  genes  gene_id           baseMean       log2FoldChange               lfcSE   \n",
       "6   NAT1     NAT1  98.73474046943294  -0.3588695183526639  0.1795865924500918   \n",
       "7   AAMP     AAMP  872.5000270269832  -0.0418951057520827  0.1166181865869584   \n",
       "\n",
       "                   5                   6                   7   \\\n",
       "5                stat              pvalue                padj   \n",
       "6   1.998309080074538  0.0456831617026497  0.5573132426854484   \n",
       "7  0.3592501905424756  0.7194079326130601  0.9684493014304824   \n",
       "\n",
       "                   8                    9   ...                  40  \\\n",
       "5            baseMean       log2FoldChange  ...               lfcSE   \n",
       "6   5.021740199533236  -0.5547845714960977  ...  0.2491509542089338   \n",
       "7  14.549760596196997   0.3164736874507167  ...  0.1441953888640895   \n",
       "\n",
       "                    41                  42                  43  \\\n",
       "5                 stat              pvalue                padj   \n",
       "6   1.3371348848527798   0.181178611933578  0.8915384574240992   \n",
       "7  -1.1898593830650157  0.2341016648908468  0.9320326022466292   \n",
       "\n",
       "                   44                 45                  46  \\\n",
       "5            baseMean     log2FoldChange               lfcSE   \n",
       "6  0.7530778794721894  2.014292355431807  1.8320814934075496   \n",
       "7   7.440114778273713  0.290637206923639  0.6010687521535898   \n",
       "\n",
       "                    47                  48    49  \n",
       "5                 stat              pvalue  padj  \n",
       "6  -1.0994556534083848  0.2715693670830352   NaN  \n",
       "7   -0.483534048113972  0.6287165903309597   NaN  \n",
       "\n",
       "[3 rows x 50 columns]"
      ]
     },
     "execution_count": 194,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_saved_output.iloc[:3]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 201,
   "id": "0d44de12",
   "metadata": {},
   "outputs": [],
   "source": [
    "df_saved_output.columns = df_saved_output.iloc[0]\n",
    "df_saved_output = df_saved_output[1:]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 203,
   "id": "c254d560",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th>5</th>\n",
       "      <th>genes</th>\n",
       "      <th>gene_id</th>\n",
       "      <th>baseMean</th>\n",
       "      <th>log2FoldChange</th>\n",
       "      <th>lfcSE</th>\n",
       "      <th>stat</th>\n",
       "      <th>pvalue</th>\n",
       "      <th>padj</th>\n",
       "      <th>baseMean</th>\n",
       "      <th>log2FoldChange</th>\n",
       "      <th>...</th>\n",
       "      <th>lfcSE</th>\n",
       "      <th>stat</th>\n",
       "      <th>pvalue</th>\n",
       "      <th>padj</th>\n",
       "      <th>baseMean</th>\n",
       "      <th>log2FoldChange</th>\n",
       "      <th>lfcSE</th>\n",
       "      <th>stat</th>\n",
       "      <th>pvalue</th>\n",
       "      <th>padj</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>NAT1</td>\n",
       "      <td>NAT1</td>\n",
       "      <td>98.73474046943294</td>\n",
       "      <td>-0.3588695183526639</td>\n",
       "      <td>0.1795865924500918</td>\n",
       "      <td>1.998309080074538</td>\n",
       "      <td>0.0456831617026497</td>\n",
       "      <td>0.5573132426854484</td>\n",
       "      <td>5.021740199533236</td>\n",
       "      <td>-0.5547845714960977</td>\n",
       "      <td>...</td>\n",
       "      <td>0.2491509542089338</td>\n",
       "      <td>1.3371348848527798</td>\n",
       "      <td>0.181178611933578</td>\n",
       "      <td>0.8915384574240992</td>\n",
       "      <td>0.7530778794721894</td>\n",
       "      <td>2.014292355431807</td>\n",
       "      <td>1.8320814934075496</td>\n",
       "      <td>-1.0994556534083848</td>\n",
       "      <td>0.2715693670830352</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>AAMP</td>\n",
       "      <td>AAMP</td>\n",
       "      <td>872.5000270269832</td>\n",
       "      <td>-0.0418951057520827</td>\n",
       "      <td>0.1166181865869584</td>\n",
       "      <td>0.3592501905424756</td>\n",
       "      <td>0.7194079326130601</td>\n",
       "      <td>0.9684493014304824</td>\n",
       "      <td>14.549760596196997</td>\n",
       "      <td>0.3164736874507167</td>\n",
       "      <td>...</td>\n",
       "      <td>0.1441953888640895</td>\n",
       "      <td>-1.1898593830650157</td>\n",
       "      <td>0.2341016648908468</td>\n",
       "      <td>0.9320326022466292</td>\n",
       "      <td>7.440114778273713</td>\n",
       "      <td>0.290637206923639</td>\n",
       "      <td>0.6010687521535898</td>\n",
       "      <td>-0.483534048113972</td>\n",
       "      <td>0.6287165903309597</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 50 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "5 genes gene_id           baseMean       log2FoldChange               lfcSE  \\\n",
       "6  NAT1    NAT1  98.73474046943294  -0.3588695183526639  0.1795865924500918   \n",
       "7  AAMP    AAMP  872.5000270269832  -0.0418951057520827  0.1166181865869584   \n",
       "\n",
       "5                stat              pvalue                padj  \\\n",
       "6   1.998309080074538  0.0456831617026497  0.5573132426854484   \n",
       "7  0.3592501905424756  0.7194079326130601  0.9684493014304824   \n",
       "\n",
       "5            baseMean       log2FoldChange  ...               lfcSE  \\\n",
       "6   5.021740199533236  -0.5547845714960977  ...  0.2491509542089338   \n",
       "7  14.549760596196997   0.3164736874507167  ...  0.1441953888640895   \n",
       "\n",
       "5                 stat              pvalue                padj  \\\n",
       "6   1.3371348848527798   0.181178611933578  0.8915384574240992   \n",
       "7  -1.1898593830650157  0.2341016648908468  0.9320326022466292   \n",
       "\n",
       "5            baseMean     log2FoldChange               lfcSE  \\\n",
       "6  0.7530778794721894  2.014292355431807  1.8320814934075496   \n",
       "7   7.440114778273713  0.290637206923639  0.6010687521535898   \n",
       "\n",
       "5                 stat              pvalue padj  \n",
       "6  -1.0994556534083848  0.2715693670830352  NaN  \n",
       "7   -0.483534048113972  0.6287165903309597  NaN  \n",
       "\n",
       "[2 rows x 50 columns]"
      ]
     },
     "execution_count": 203,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_saved_output[:2]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 216,
   "id": "e00377c3",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(15112, 50)"
      ]
     },
     "execution_count": 216,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_saved_output.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 223,
   "id": "4035cf13",
   "metadata": {},
   "outputs": [],
   "source": [
    "# change number values to float\n",
    "num_cols = list(range(2,df_saved_output.shape[1]))\n",
    "df_saved_output.iloc[:,num_cols] = df_saved_output.iloc[:,num_cols].astype(float)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d9df20c9",
   "metadata": {},
   "source": [
    "__Make df with all genes__"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 241,
   "id": "336014f4",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved df genes: 15112\n",
      "Merged df genes: 16558\n",
      "Total unique genes: 17269\n"
     ]
    }
   ],
   "source": [
    "print(\"Saved df genes: {}\".format(len(df_saved_output['genes'])))\n",
    "print(\"Merged df genes: {}\".format(len(df_merged['genes'])))\n",
    "merged_genes = list(set(df_saved_output['genes'].tolist() + df_merged['genes'].tolist()))\n",
    "merged_genes.sort()\n",
    "merged_genes = sorted(merged_genes, key=lambda x: (x.isnumeric(),int(x) if x.isnumeric() else x))\n",
    "print(\"Total unique genes: {}\".format(len(merged_genes)))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 243,
   "id": "dd6c3958",
   "metadata": {},
   "outputs": [],
   "source": [
    "df = pd.DataFrame(list(zip(merged_genes, merged_genes)),\n",
    "                  columns=[\"genes\", \"gene_id\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 245,
   "id": "8f515c83",
   "metadata": {},
   "outputs": [],
   "source": [
    "df = reduce(lambda left, right: pd.merge(left, right, on=['genes', \"gene_id\"], how=\"outer\"), \n",
    "                   [df, df_saved_output, df_merged])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 246,
   "id": "abe944a8",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(17269, 74)\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>genes</th>\n",
       "      <th>gene_id</th>\n",
       "      <th>baseMean_x</th>\n",
       "      <th>log2FoldChange_x</th>\n",
       "      <th>lfcSE_x</th>\n",
       "      <th>stat_x</th>\n",
       "      <th>pvalue_x</th>\n",
       "      <th>padj_x</th>\n",
       "      <th>baseMean_x</th>\n",
       "      <th>log2FoldChange_x</th>\n",
       "      <th>...</th>\n",
       "      <th>lfcSE_y</th>\n",
       "      <th>stat_y</th>\n",
       "      <th>pvalue_y</th>\n",
       "      <th>padj_y</th>\n",
       "      <th>baseMean_y</th>\n",
       "      <th>log2FoldChange_y</th>\n",
       "      <th>lfcSE_y</th>\n",
       "      <th>stat_y</th>\n",
       "      <th>pvalue_y</th>\n",
       "      <th>padj_y</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>A1BG</td>\n",
       "      <td>A1BG</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>0.980153</td>\n",
       "      <td>-0.116824</td>\n",
       "      <td>0.907</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.05833</td>\n",
       "      <td>-0.499954</td>\n",
       "      <td>3.11654</td>\n",
       "      <td>0.16042</td>\n",
       "      <td>0.87255</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>A1BG-AS1</td>\n",
       "      <td>A1BG-AS1</td>\n",
       "      <td>25.689167</td>\n",
       "      <td>0.01394</td>\n",
       "      <td>0.311866</td>\n",
       "      <td>-0.044699</td>\n",
       "      <td>0.964347</td>\n",
       "      <td>0.997077</td>\n",
       "      <td>0.209183</td>\n",
       "      <td>-0.545181</td>\n",
       "      <td>...</td>\n",
       "      <td>0.179837</td>\n",
       "      <td>-1.153464</td>\n",
       "      <td>0.24872</td>\n",
       "      <td>0.999857</td>\n",
       "      <td>0.0</td>\n",
       "      <td>-0.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>A1CF</td>\n",
       "      <td>A1CF</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>0.577384</td>\n",
       "      <td>0.735683</td>\n",
       "      <td>0.461924</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>-0.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>A2M-AS1</td>\n",
       "      <td>A2M-AS1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>0.894369</td>\n",
       "      <td>-0.055995</td>\n",
       "      <td>0.955345</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>-0.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>A2ML1</td>\n",
       "      <td>A2ML1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>0.706911</td>\n",
       "      <td>0.342414</td>\n",
       "      <td>0.732039</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>-0.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows Ã— 74 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "      genes   gene_id baseMean_x log2FoldChange_x   lfcSE_x    stat_x  \\\n",
       "0      A1BG      A1BG        NaN              NaN       NaN       NaN   \n",
       "1  A1BG-AS1  A1BG-AS1  25.689167          0.01394  0.311866 -0.044699   \n",
       "2      A1CF      A1CF        NaN              NaN       NaN       NaN   \n",
       "3   A2M-AS1   A2M-AS1        NaN              NaN       NaN       NaN   \n",
       "4     A2ML1     A2ML1        NaN              NaN       NaN       NaN   \n",
       "\n",
       "   pvalue_x    padj_x baseMean_x log2FoldChange_x  ...   lfcSE_y    stat_y  \\\n",
       "0       NaN       NaN        NaN              NaN  ...  0.980153 -0.116824   \n",
       "1  0.964347  0.997077   0.209183        -0.545181  ...  0.179837 -1.153464   \n",
       "2       NaN       NaN        NaN              NaN  ...  0.577384  0.735683   \n",
       "3       NaN       NaN        NaN              NaN  ...  0.894369 -0.055995   \n",
       "4       NaN       NaN        NaN              NaN  ...  0.706911  0.342414   \n",
       "\n",
       "   pvalue_y    padj_y baseMean_y log2FoldChange_y  lfcSE_y   stat_y pvalue_y  \\\n",
       "0     0.907       NaN    0.05833        -0.499954  3.11654  0.16042  0.87255   \n",
       "1   0.24872  0.999857        0.0             -0.0      NaN      NaN      0.0   \n",
       "2  0.461924       NaN        0.0             -0.0      NaN      NaN      0.0   \n",
       "3  0.955345       NaN        0.0             -0.0      NaN      NaN      0.0   \n",
       "4  0.732039       NaN        0.0             -0.0      NaN      NaN      0.0   \n",
       "\n",
       "  padj_y  \n",
       "0    NaN  \n",
       "1    NaN  \n",
       "2    NaN  \n",
       "3    NaN  \n",
       "4    NaN  \n",
       "\n",
       "[5 rows x 74 columns]"
      ]
     },
     "execution_count": 246,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "print(df.shape)\n",
    "df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 247,
   "id": "f0c4f064",
   "metadata": {},
   "outputs": [],
   "source": [
    "# change column names\n",
    "df.columns = [col_name.split(\"_\")[0] if \"gene\" not in col_name else \n",
    "              col_name for col_name in df.columns.tolist()]\n",
    "df = df.T.reset_index().T.reset_index(drop=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9e3eaae2",
   "metadata": {},
   "source": [
    "### add info cols"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 248,
   "id": "63f43be7",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(17270, 74)\n",
      "(5, 74)\n"
     ]
    }
   ],
   "source": [
    "print(df.shape)\n",
    "print(df_info_merged.shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 250,
   "id": "0173f03d",
   "metadata": {},
   "outputs": [],
   "source": [
    "# reset column index\n",
    "df_info_merged = df_info_merged.T.reset_index(drop=True).T"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 251,
   "id": "c20a6cbd",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 12 ms, sys: 8 ms, total: 20 ms\n",
      "Wall time: 22.7 ms\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "(17275, 74)"
      ]
     },
     "execution_count": 251,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "%%time\n",
    "\n",
    "# combine info and data\n",
    "df_save = df_info_merged.append(df, ignore_index=True)\n",
    "df_save.shape"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "69bd0868",
   "metadata": {},
   "source": [
    "### save merged df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 252,
   "id": "873d8c52",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved DESeq results to: ../results/slamseq_merged_deseq_output.csv\n"
     ]
    }
   ],
   "source": [
    "# save file\n",
    "res_dir = \"../results\"\n",
    "deseq_res_file = \"slamseq_merged_deseq_output.csv\"\n",
    "\n",
    "df_save.to_csv(\"{}/{}\".format(res_dir, deseq_res_file), header=False, index=False)\n",
    "\n",
    "print(\"Saved DESeq results to: {}/{}\".format(res_dir, deseq_res_file))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "f4d8b6a6",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "376ce8ec",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "id": "0b179dce",
   "metadata": {},
   "source": [
    "# 4_degrons_v1 (RUNX1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "id": "6b83f212",
   "metadata": {},
   "outputs": [],
   "source": [
    "project = \"4_degrons_v1\"\n",
    "project_dir = \"slamseq_{}\".format(project)\n",
    "gene = \"RUNX1\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 69,
   "id": "ccb5059b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "5\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "['../results/slamseq_4_degrons_v1/4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq_readcounts.csv',\n",
       " '../results/slamseq_4_degrons_v1/4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq_tccounts.csv',\n",
       " '../results/slamseq_4_degrons_v1/4_degrons_v1_RUNX1_1h_0_5_deseq_readcounts.csv',\n",
       " '../results/slamseq_4_degrons_v1/4_degrons_v1_RUNX1_1h_0_5_deseq_tccounts.csv',\n",
       " '../results/slamseq_4_degrons_v1/slamseq_4_degrons_v1_RUNX1_0_5_deseq_output.csv']"
      ]
     },
     "execution_count": 69,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "files = ! ls $res_dir/$project_dir/*.csv\n",
    "files = [file for file in files if \"deseq\" in file and gene in file]\n",
    "# ERCC files first\n",
    "files = [file for file in files if \"ERCCsamplewise\" in file] + \\\n",
    "[file for file in files if \"ERCCsamplewise\" not in file]\n",
    "print(len(files))\n",
    "files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "id": "1b1b0608",
   "metadata": {},
   "outputs": [],
   "source": [
    "files = files[:-1]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 71,
   "id": "e629def9",
   "metadata": {},
   "outputs": [],
   "source": [
    "# project 3\n",
    "conditions = [\"1h\"]\n",
    "scaling = [\"ERCCsamplewise\", None]\n",
    "count_type = [\"readcounts\", \"tccounts\"]"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0c40b4d5",
   "metadata": {},
   "source": [
    "### make info rows"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 72,
   "id": "a6863760",
   "metadata": {},
   "outputs": [],
   "source": [
    "# df with file info\n",
    "df_info = pd.DataFrame(list(zip([\"project\", \"file\", \"scale\", \"condition\", \"type\"],\n",
    "                                [\"project\", \"file\", \"scale\", \"condition\", \"type\"])))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 73,
   "id": "6596b2ec",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(5, 26)"
      ]
     },
     "execution_count": 73,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_info = pd.concat([df_info] + [pd.DataFrame([x*6 for x in get_file_info(project, file, conditions)]) \\\n",
    "                                 for file in files], axis=1)\n",
    "df_info.shape"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9966f051",
   "metadata": {},
   "source": [
    "### get dfs and merge"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 75,
   "id": "9694dbb2",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Get list of deseq dfs\n",
    "deseq_dfs = [get_deseq_data(file) for file in files]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 76,
   "id": "18c8d0bf",
   "metadata": {},
   "outputs": [],
   "source": [
    "# main df\n",
    "df_merged = pd.DataFrame()\n",
    "df_merged = df_merged.append(deseq_dfs[0][[\"genes\", \"gene_id\"]]) # add gene cols to main df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 78,
   "id": "06e67914",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(16744, 26)"
      ]
     },
     "execution_count": 78,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# join df columns on gene & gene_id columns\n",
    "df_merged = reduce(lambda left, right: pd.merge(left, right, on=['genes', \"gene_id\"], how=\"outer\"), \n",
    "                   [df_merged] + deseq_dfs)\n",
    "df_merged.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 79,
   "id": "2f558587",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>genes</th>\n",
       "      <th>gene_id</th>\n",
       "      <th>baseMean_4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq_readcounts</th>\n",
       "      <th>log2FoldChange_4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq_readcounts</th>\n",
       "      <th>lfcSE_4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq_readcounts</th>\n",
       "      <th>stat_4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq_readcounts</th>\n",
       "      <th>pvalue_4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq_readcounts</th>\n",
       "      <th>padj_4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq_readcounts</th>\n",
       "      <th>baseMean_4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq_tccounts</th>\n",
       "      <th>log2FoldChange_4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq_tccounts</th>\n",
       "      <th>...</th>\n",
       "      <th>lfcSE_4_degrons_v1_RUNX1_1h_0_5_deseq_readcounts</th>\n",
       "      <th>stat_4_degrons_v1_RUNX1_1h_0_5_deseq_readcounts</th>\n",
       "      <th>pvalue_4_degrons_v1_RUNX1_1h_0_5_deseq_readcounts</th>\n",
       "      <th>padj_4_degrons_v1_RUNX1_1h_0_5_deseq_readcounts</th>\n",
       "      <th>baseMean_4_degrons_v1_RUNX1_1h_0_5_deseq_tccounts</th>\n",
       "      <th>log2FoldChange_4_degrons_v1_RUNX1_1h_0_5_deseq_tccounts</th>\n",
       "      <th>lfcSE_4_degrons_v1_RUNX1_1h_0_5_deseq_tccounts</th>\n",
       "      <th>stat_4_degrons_v1_RUNX1_1h_0_5_deseq_tccounts</th>\n",
       "      <th>pvalue_4_degrons_v1_RUNX1_1h_0_5_deseq_tccounts</th>\n",
       "      <th>padj_4_degrons_v1_RUNX1_1h_0_5_deseq_tccounts</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>A2M</td>\n",
       "      <td>A2M</td>\n",
       "      <td>2.269069</td>\n",
       "      <td>-0.155731</td>\n",
       "      <td>1.023965</td>\n",
       "      <td>0.152086</td>\n",
       "      <td>0.879119</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>-0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>1.025011</td>\n",
       "      <td>0.157375</td>\n",
       "      <td>0.874950</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>-0.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>A2MP1</td>\n",
       "      <td>A2MP1</td>\n",
       "      <td>2.050300</td>\n",
       "      <td>3.698355</td>\n",
       "      <td>1.401673</td>\n",
       "      <td>-2.638529</td>\n",
       "      <td>0.008327</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>-0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>1.402745</td>\n",
       "      <td>-2.657264</td>\n",
       "      <td>0.007878</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>-0.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 26 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "   genes gene_id  \\\n",
       "0    A2M     A2M   \n",
       "1  A2MP1   A2MP1   \n",
       "\n",
       "   baseMean_4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq_readcounts  \\\n",
       "0                                           2.269069                    \n",
       "1                                           2.050300                    \n",
       "\n",
       "   log2FoldChange_4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq_readcounts  \\\n",
       "0                                          -0.155731                          \n",
       "1                                           3.698355                          \n",
       "\n",
       "   lfcSE_4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq_readcounts  \\\n",
       "0                                           1.023965                 \n",
       "1                                           1.401673                 \n",
       "\n",
       "   stat_4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq_readcounts  \\\n",
       "0                                           0.152086                \n",
       "1                                          -2.638529                \n",
       "\n",
       "   pvalue_4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq_readcounts  \\\n",
       "0                                           0.879119                  \n",
       "1                                           0.008327                  \n",
       "\n",
       "   padj_4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq_readcounts  \\\n",
       "0                                                NaN                \n",
       "1                                                NaN                \n",
       "\n",
       "   baseMean_4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq_tccounts  \\\n",
       "0                                                0.0                  \n",
       "1                                                0.0                  \n",
       "\n",
       "   log2FoldChange_4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq_tccounts  \\\n",
       "0                                               -0.0                        \n",
       "1                                               -0.0                        \n",
       "\n",
       "   ...  lfcSE_4_degrons_v1_RUNX1_1h_0_5_deseq_readcounts  \\\n",
       "0  ...                                          1.025011   \n",
       "1  ...                                          1.402745   \n",
       "\n",
       "   stat_4_degrons_v1_RUNX1_1h_0_5_deseq_readcounts  \\\n",
       "0                                         0.157375   \n",
       "1                                        -2.657264   \n",
       "\n",
       "   pvalue_4_degrons_v1_RUNX1_1h_0_5_deseq_readcounts  \\\n",
       "0                                           0.874950   \n",
       "1                                           0.007878   \n",
       "\n",
       "   padj_4_degrons_v1_RUNX1_1h_0_5_deseq_readcounts  \\\n",
       "0                                              NaN   \n",
       "1                                              NaN   \n",
       "\n",
       "   baseMean_4_degrons_v1_RUNX1_1h_0_5_deseq_tccounts  \\\n",
       "0                                                0.0   \n",
       "1                                                0.0   \n",
       "\n",
       "   log2FoldChange_4_degrons_v1_RUNX1_1h_0_5_deseq_tccounts  \\\n",
       "0                                               -0.0         \n",
       "1                                               -0.0         \n",
       "\n",
       "   lfcSE_4_degrons_v1_RUNX1_1h_0_5_deseq_tccounts  \\\n",
       "0                                             NaN   \n",
       "1                                             NaN   \n",
       "\n",
       "   stat_4_degrons_v1_RUNX1_1h_0_5_deseq_tccounts  \\\n",
       "0                                            NaN   \n",
       "1                                            NaN   \n",
       "\n",
       "   pvalue_4_degrons_v1_RUNX1_1h_0_5_deseq_tccounts  \\\n",
       "0                                              0.0   \n",
       "1                                              0.0   \n",
       "\n",
       "   padj_4_degrons_v1_RUNX1_1h_0_5_deseq_tccounts  \n",
       "0                                            NaN  \n",
       "1                                            NaN  \n",
       "\n",
       "[2 rows x 26 columns]"
      ]
     },
     "execution_count": 79,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_merged.iloc[:2]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 86,
   "id": "4b5fdb14",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(16745, 26)"
      ]
     },
     "execution_count": 86,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# change column names\n",
    "df_merged.columns = [col_name.split(\"_\")[0] if \"gene\" not in col_name else \n",
    "                     col_name for col_name in df_merged.columns.tolist()]\n",
    "df_merged = df_merged.T.reset_index().T.reset_index(drop=True)\n",
    "df_merged.shape"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "69713593",
   "metadata": {},
   "source": [
    "### add info columns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 89,
   "id": "1ee8728d",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(5, 26)\n",
      "(16745, 26)\n"
     ]
    }
   ],
   "source": [
    "print(df_info.shape)\n",
    "print(df_merged.shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 81,
   "id": "8165d6cb",
   "metadata": {},
   "outputs": [],
   "source": [
    "# reset column index\n",
    "df_info = df_info.T.reset_index(drop=True).T"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 82,
   "id": "6b9ddafa",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(5, 26)\n",
      "(16744, 26)\n"
     ]
    }
   ],
   "source": [
    "print(df_info.shape)\n",
    "print(df_merged.shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 90,
   "id": "8c714294",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "      <th>3</th>\n",
       "      <th>4</th>\n",
       "      <th>5</th>\n",
       "      <th>6</th>\n",
       "      <th>7</th>\n",
       "      <th>8</th>\n",
       "      <th>9</th>\n",
       "      <th>...</th>\n",
       "      <th>16</th>\n",
       "      <th>17</th>\n",
       "      <th>18</th>\n",
       "      <th>19</th>\n",
       "      <th>20</th>\n",
       "      <th>21</th>\n",
       "      <th>22</th>\n",
       "      <th>23</th>\n",
       "      <th>24</th>\n",
       "      <th>25</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>project</td>\n",
       "      <td>project</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>...</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>file</td>\n",
       "      <td>file</td>\n",
       "      <td>4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq...</td>\n",
       "      <td>4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq...</td>\n",
       "      <td>4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq...</td>\n",
       "      <td>4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq...</td>\n",
       "      <td>4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq...</td>\n",
       "      <td>4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq...</td>\n",
       "      <td>4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq...</td>\n",
       "      <td>4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq...</td>\n",
       "      <td>...</td>\n",
       "      <td>4_degrons_v1_RUNX1_1h_0_5_deseq_readcounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX1_1h_0_5_deseq_readcounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX1_1h_0_5_deseq_readcounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX1_1h_0_5_deseq_readcounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX1_1h_0_5_deseq_tccounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX1_1h_0_5_deseq_tccounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX1_1h_0_5_deseq_tccounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX1_1h_0_5_deseq_tccounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX1_1h_0_5_deseq_tccounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX1_1h_0_5_deseq_tccounts.csv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>scale</td>\n",
       "      <td>scale</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>condition</td>\n",
       "      <td>condition</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>...</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>type</td>\n",
       "      <td>type</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>...</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows Ã— 26 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "          0          1                                                  2   \\\n",
       "0    project    project                                       4_degrons_v1   \n",
       "1       file       file  4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq...   \n",
       "2      scale      scale                                     ERCCsamplewise   \n",
       "3  condition  condition                                                 1h   \n",
       "4       type       type                                         readcounts   \n",
       "\n",
       "                                                  3   \\\n",
       "0                                       4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 1h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  4   \\\n",
       "0                                       4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 1h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  5   \\\n",
       "0                                       4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 1h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  6   \\\n",
       "0                                       4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 1h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  7   \\\n",
       "0                                       4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 1h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  8   \\\n",
       "0                                       4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 1h   \n",
       "4                                           tccounts   \n",
       "\n",
       "                                                  9   ...  \\\n",
       "0                                       4_degrons_v1  ...   \n",
       "1  4_degrons_v1_RUNX1_1h_ERCCsamplewise_0_5_deseq...  ...   \n",
       "2                                     ERCCsamplewise  ...   \n",
       "3                                                 1h  ...   \n",
       "4                                           tccounts  ...   \n",
       "\n",
       "                                               16  \\\n",
       "0                                    4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX1_1h_0_5_deseq_readcounts.csv   \n",
       "2                                            None   \n",
       "3                                              1h   \n",
       "4                                      readcounts   \n",
       "\n",
       "                                               17  \\\n",
       "0                                    4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX1_1h_0_5_deseq_readcounts.csv   \n",
       "2                                            None   \n",
       "3                                              1h   \n",
       "4                                      readcounts   \n",
       "\n",
       "                                               18  \\\n",
       "0                                    4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX1_1h_0_5_deseq_readcounts.csv   \n",
       "2                                            None   \n",
       "3                                              1h   \n",
       "4                                      readcounts   \n",
       "\n",
       "                                               19  \\\n",
       "0                                    4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX1_1h_0_5_deseq_readcounts.csv   \n",
       "2                                            None   \n",
       "3                                              1h   \n",
       "4                                      readcounts   \n",
       "\n",
       "                                             20  \\\n",
       "0                                  4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX1_1h_0_5_deseq_tccounts.csv   \n",
       "2                                          None   \n",
       "3                                            1h   \n",
       "4                                      tccounts   \n",
       "\n",
       "                                             21  \\\n",
       "0                                  4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX1_1h_0_5_deseq_tccounts.csv   \n",
       "2                                          None   \n",
       "3                                            1h   \n",
       "4                                      tccounts   \n",
       "\n",
       "                                             22  \\\n",
       "0                                  4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX1_1h_0_5_deseq_tccounts.csv   \n",
       "2                                          None   \n",
       "3                                            1h   \n",
       "4                                      tccounts   \n",
       "\n",
       "                                             23  \\\n",
       "0                                  4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX1_1h_0_5_deseq_tccounts.csv   \n",
       "2                                          None   \n",
       "3                                            1h   \n",
       "4                                      tccounts   \n",
       "\n",
       "                                             24  \\\n",
       "0                                  4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX1_1h_0_5_deseq_tccounts.csv   \n",
       "2                                          None   \n",
       "3                                            1h   \n",
       "4                                      tccounts   \n",
       "\n",
       "                                             25  \n",
       "0                                  4_degrons_v1  \n",
       "1  4_degrons_v1_RUNX1_1h_0_5_deseq_tccounts.csv  \n",
       "2                                          None  \n",
       "3                                            1h  \n",
       "4                                      tccounts  \n",
       "\n",
       "[5 rows x 26 columns]"
      ]
     },
     "execution_count": 90,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_info.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 91,
   "id": "2496a7c2",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "      <th>3</th>\n",
       "      <th>4</th>\n",
       "      <th>5</th>\n",
       "      <th>6</th>\n",
       "      <th>7</th>\n",
       "      <th>8</th>\n",
       "      <th>9</th>\n",
       "      <th>...</th>\n",
       "      <th>16</th>\n",
       "      <th>17</th>\n",
       "      <th>18</th>\n",
       "      <th>19</th>\n",
       "      <th>20</th>\n",
       "      <th>21</th>\n",
       "      <th>22</th>\n",
       "      <th>23</th>\n",
       "      <th>24</th>\n",
       "      <th>25</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>genes</td>\n",
       "      <td>gene_id</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>...</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>A2M</td>\n",
       "      <td>A2M</td>\n",
       "      <td>2.269069</td>\n",
       "      <td>-0.155731</td>\n",
       "      <td>1.023965</td>\n",
       "      <td>0.152086</td>\n",
       "      <td>0.879119</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>-0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>1.025011</td>\n",
       "      <td>0.157375</td>\n",
       "      <td>0.87495</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>-0.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>A2MP1</td>\n",
       "      <td>A2MP1</td>\n",
       "      <td>2.0503</td>\n",
       "      <td>3.698355</td>\n",
       "      <td>1.401673</td>\n",
       "      <td>-2.638529</td>\n",
       "      <td>0.008327</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>-0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>1.402745</td>\n",
       "      <td>-2.657264</td>\n",
       "      <td>0.007878</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>-0.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>NAT1</td>\n",
       "      <td>NAT1</td>\n",
       "      <td>93.83574</td>\n",
       "      <td>-0.131532</td>\n",
       "      <td>0.177942</td>\n",
       "      <td>0.739184</td>\n",
       "      <td>0.459795</td>\n",
       "      <td>0.954039</td>\n",
       "      <td>7.47324</td>\n",
       "      <td>0.280315</td>\n",
       "      <td>...</td>\n",
       "      <td>0.179349</td>\n",
       "      <td>0.669795</td>\n",
       "      <td>0.502989</td>\n",
       "      <td>NaN</td>\n",
       "      <td>7.476669</td>\n",
       "      <td>0.284898</td>\n",
       "      <td>0.540413</td>\n",
       "      <td>-0.527186</td>\n",
       "      <td>0.598064</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>AAMP</td>\n",
       "      <td>AAMP</td>\n",
       "      <td>668.981822</td>\n",
       "      <td>-0.004677</td>\n",
       "      <td>0.085659</td>\n",
       "      <td>0.054603</td>\n",
       "      <td>0.956455</td>\n",
       "      <td>0.996275</td>\n",
       "      <td>16.217859</td>\n",
       "      <td>-0.184215</td>\n",
       "      <td>...</td>\n",
       "      <td>0.086942</td>\n",
       "      <td>-0.063919</td>\n",
       "      <td>0.949035</td>\n",
       "      <td>0.993941</td>\n",
       "      <td>16.281909</td>\n",
       "      <td>-0.173311</td>\n",
       "      <td>0.362675</td>\n",
       "      <td>0.477867</td>\n",
       "      <td>0.632745</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows Ã— 26 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "      0        1           2               3         4         5         6   \\\n",
       "0  genes  gene_id    baseMean  log2FoldChange     lfcSE      stat    pvalue   \n",
       "1    A2M      A2M    2.269069       -0.155731  1.023965  0.152086  0.879119   \n",
       "2  A2MP1    A2MP1      2.0503        3.698355  1.401673 -2.638529  0.008327   \n",
       "3   NAT1     NAT1    93.83574       -0.131532  0.177942  0.739184  0.459795   \n",
       "4   AAMP     AAMP  668.981822       -0.004677  0.085659  0.054603  0.956455   \n",
       "\n",
       "         7          8               9   ...        16        17        18  \\\n",
       "0      padj   baseMean  log2FoldChange  ...     lfcSE      stat    pvalue   \n",
       "1       NaN        0.0            -0.0  ...  1.025011  0.157375   0.87495   \n",
       "2       NaN        0.0            -0.0  ...  1.402745 -2.657264  0.007878   \n",
       "3  0.954039    7.47324        0.280315  ...  0.179349  0.669795  0.502989   \n",
       "4  0.996275  16.217859       -0.184215  ...  0.086942 -0.063919  0.949035   \n",
       "\n",
       "         19         20              21        22        23        24    25  \n",
       "0      padj   baseMean  log2FoldChange     lfcSE      stat    pvalue  padj  \n",
       "1       NaN        0.0            -0.0       NaN       NaN       0.0   NaN  \n",
       "2       NaN        0.0            -0.0       NaN       NaN       0.0   NaN  \n",
       "3       NaN   7.476669        0.284898  0.540413 -0.527186  0.598064   NaN  \n",
       "4  0.993941  16.281909       -0.173311  0.362675  0.477867  0.632745   NaN  \n",
       "\n",
       "[5 rows x 26 columns]"
      ]
     },
     "execution_count": 91,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_merged.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 92,
   "id": "cd2f48f6",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 4 ms, sys: 0 ns, total: 4 ms\n",
      "Wall time: 5.29 ms\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "(16750, 26)"
      ]
     },
     "execution_count": 92,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "%%time\n",
    "\n",
    "# combine info and data\n",
    "df_save = df_info.append(df_merged, ignore_index=True)\n",
    "df_save.shape"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "6cdccb98",
   "metadata": {},
   "source": [
    "### save merged file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 94,
   "id": "52872e73",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved DESeq results to: slamseq_4_degrons_v1/slamseq_4_degrons_v1_RUNX1_0_5_deseq_output.csv\n"
     ]
    }
   ],
   "source": [
    "# save file\n",
    "res_dir = \"../results\"\n",
    "deseq_res_file = \"slamseq_{}_{}_0_5_deseq_output.csv\".format(project, gene)\n",
    "\n",
    "df_save.to_csv(\"{}/{}/{}\".format(res_dir, project_dir, deseq_res_file), header=False, index=False)\n",
    "\n",
    "print(\"Saved DESeq results to: {}/{}\".format(project_dir, deseq_res_file))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ac8a0ee2",
   "metadata": {
    "heading_collapsed": true
   },
   "source": [
    "# 4_degrons_v1 (RUNX2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 95,
   "id": "12a6eae6",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "project = \"4_degrons_v1\"\n",
    "project_dir = \"slamseq_{}\".format(project)\n",
    "gene = \"RUNX2\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 96,
   "id": "e4d90fa7",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "5\n"
     ]
    }
   ],
   "source": [
    "files = ! ls $res_dir/$project_dir/*.csv\n",
    "files = [file for file in files if \"deseq\" in file and gene in file]\n",
    "# ERCC files first\n",
    "files = [file for file in files if \"ERCCsamplewise\" in file] + \\\n",
    "[file for file in files if \"ERCCsamplewise\" not in file]\n",
    "print(len(files))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 97,
   "id": "a3e815ef",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "files = files[:-1]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 98,
   "id": "f341288e",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "# project 3\n",
    "conditions = [\"1h\"]\n",
    "scaling = [\"ERCCsamplewise\", None]\n",
    "count_type = [\"readcounts\", \"tccounts\"]"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "46f5a286",
   "metadata": {
    "hidden": true
   },
   "source": [
    "### make info rows"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 99,
   "id": "f51e1f5c",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "# df with file info\n",
    "df_info = pd.DataFrame(list(zip([\"project\", \"file\", \"scale\", \"condition\", \"type\"],\n",
    "                                [\"project\", \"file\", \"scale\", \"condition\", \"type\"])))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 100,
   "id": "1bf0181a",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(5, 26)"
      ]
     },
     "execution_count": 100,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_info = pd.concat([df_info] + [pd.DataFrame([x*6 for x in get_file_info(project, file, conditions)]) \\\n",
    "                                 for file in files], axis=1)\n",
    "df_info.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 101,
   "id": "d2b8c688",
   "metadata": {
    "collapsed": true,
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "      <th>3</th>\n",
       "      <th>4</th>\n",
       "      <th>5</th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>...</th>\n",
       "      <th>2</th>\n",
       "      <th>3</th>\n",
       "      <th>4</th>\n",
       "      <th>5</th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "      <th>3</th>\n",
       "      <th>4</th>\n",
       "      <th>5</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>project</td>\n",
       "      <td>project</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>...</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>file</td>\n",
       "      <td>file</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_ERCCsamplewise_0_5_deseq...</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_ERCCsamplewise_0_5_deseq...</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_ERCCsamplewise_0_5_deseq...</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_ERCCsamplewise_0_5_deseq...</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_ERCCsamplewise_0_5_deseq...</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_ERCCsamplewise_0_5_deseq...</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_ERCCsamplewise_0_5_deseq...</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_ERCCsamplewise_0_5_deseq...</td>\n",
       "      <td>...</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_readcounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_readcounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_readcounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_readcounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>scale</td>\n",
       "      <td>scale</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>condition</td>\n",
       "      <td>condition</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>...</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>type</td>\n",
       "      <td>type</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>...</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows Ã— 26 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "           0          1                                                  0  \\\n",
       "0    project    project                                       4_degrons_v1   \n",
       "1       file       file  4_degrons_v1_RUNX2_1h_ERCCsamplewise_0_5_deseq...   \n",
       "2      scale      scale                                     ERCCsamplewise   \n",
       "3  condition  condition                                                 1h   \n",
       "4       type       type                                         readcounts   \n",
       "\n",
       "                                                   1  \\\n",
       "0                                       4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX2_1h_ERCCsamplewise_0_5_deseq...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 1h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                   2  \\\n",
       "0                                       4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX2_1h_ERCCsamplewise_0_5_deseq...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 1h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                   3  \\\n",
       "0                                       4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX2_1h_ERCCsamplewise_0_5_deseq...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 1h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                   4  \\\n",
       "0                                       4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX2_1h_ERCCsamplewise_0_5_deseq...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 1h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                   5  \\\n",
       "0                                       4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX2_1h_ERCCsamplewise_0_5_deseq...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 1h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                   0  \\\n",
       "0                                       4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX2_1h_ERCCsamplewise_0_5_deseq...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 1h   \n",
       "4                                           tccounts   \n",
       "\n",
       "                                                   1  ...  \\\n",
       "0                                       4_degrons_v1  ...   \n",
       "1  4_degrons_v1_RUNX2_1h_ERCCsamplewise_0_5_deseq...  ...   \n",
       "2                                     ERCCsamplewise  ...   \n",
       "3                                                 1h  ...   \n",
       "4                                           tccounts  ...   \n",
       "\n",
       "                                                2  \\\n",
       "0                                    4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX2_1h_0_5_deseq_readcounts.csv   \n",
       "2                                            None   \n",
       "3                                              1h   \n",
       "4                                      readcounts   \n",
       "\n",
       "                                                3  \\\n",
       "0                                    4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX2_1h_0_5_deseq_readcounts.csv   \n",
       "2                                            None   \n",
       "3                                              1h   \n",
       "4                                      readcounts   \n",
       "\n",
       "                                                4  \\\n",
       "0                                    4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX2_1h_0_5_deseq_readcounts.csv   \n",
       "2                                            None   \n",
       "3                                              1h   \n",
       "4                                      readcounts   \n",
       "\n",
       "                                                5  \\\n",
       "0                                    4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX2_1h_0_5_deseq_readcounts.csv   \n",
       "2                                            None   \n",
       "3                                              1h   \n",
       "4                                      readcounts   \n",
       "\n",
       "                                              0  \\\n",
       "0                                  4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv   \n",
       "2                                          None   \n",
       "3                                            1h   \n",
       "4                                      tccounts   \n",
       "\n",
       "                                              1  \\\n",
       "0                                  4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv   \n",
       "2                                          None   \n",
       "3                                            1h   \n",
       "4                                      tccounts   \n",
       "\n",
       "                                              2  \\\n",
       "0                                  4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv   \n",
       "2                                          None   \n",
       "3                                            1h   \n",
       "4                                      tccounts   \n",
       "\n",
       "                                              3  \\\n",
       "0                                  4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv   \n",
       "2                                          None   \n",
       "3                                            1h   \n",
       "4                                      tccounts   \n",
       "\n",
       "                                              4  \\\n",
       "0                                  4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv   \n",
       "2                                          None   \n",
       "3                                            1h   \n",
       "4                                      tccounts   \n",
       "\n",
       "                                              5  \n",
       "0                                  4_degrons_v1  \n",
       "1  4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv  \n",
       "2                                          None  \n",
       "3                                            1h  \n",
       "4                                      tccounts  \n",
       "\n",
       "[5 rows x 26 columns]"
      ]
     },
     "execution_count": 101,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_info.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2d75a986",
   "metadata": {
    "hidden": true
   },
   "source": [
    "### get dfs and merge"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 102,
   "id": "537570b9",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "# Get list of deseq dfs\n",
    "deseq_dfs = [get_deseq_data(file) for file in files]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 103,
   "id": "a840fd51",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "# main df\n",
    "df_merged = pd.DataFrame()\n",
    "df_merged = df_merged.append(deseq_dfs[0][[\"genes\", \"gene_id\"]]) # add gene cols to main df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 104,
   "id": "0d3238f3",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(16744, 26)"
      ]
     },
     "execution_count": 104,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# join df columns on gene & gene_id columns\n",
    "df_merged = reduce(lambda left, right: pd.merge(left, right, on=['genes', \"gene_id\"], how=\"outer\"), \n",
    "                   [df_merged] + deseq_dfs)\n",
    "df_merged.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 105,
   "id": "b818745f",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(16745, 26)"
      ]
     },
     "execution_count": 105,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# change column names\n",
    "df_merged.columns = [col_name.split(\"_\")[0] if \"gene\" not in col_name else \n",
    "                     col_name for col_name in df_merged.columns.tolist()]\n",
    "df_merged = df_merged.T.reset_index().T.reset_index(drop=True)\n",
    "df_merged.shape"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b8f40258",
   "metadata": {
    "hidden": true
   },
   "source": [
    "### add info columns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 107,
   "id": "262b607f",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(5, 26)\n",
      "(16745, 26)\n"
     ]
    }
   ],
   "source": [
    "print(df_info.shape)\n",
    "print(df_merged.shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 108,
   "id": "9600730a",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "# reset column index\n",
    "df_info = df_info.T.reset_index(drop=True).T"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 109,
   "id": "f2527a1a",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 8 ms, sys: 0 ns, total: 8 ms\n",
      "Wall time: 7.75 ms\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "(16750, 26)"
      ]
     },
     "execution_count": 109,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "%%time\n",
    "\n",
    "# combine info and data\n",
    "df_save = df_info.append(df_merged, ignore_index=True)\n",
    "df_save.shape"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a97a5952",
   "metadata": {
    "hidden": true
   },
   "source": [
    "### save merged file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 110,
   "id": "6dcd29c3",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved DESeq results to: slamseq_4_degrons_v1/slamseq_4_degrons_v1_RUNX2_0_5_deseq_output.csv\n"
     ]
    }
   ],
   "source": [
    "# save file\n",
    "res_dir = \"../results\"\n",
    "deseq_res_file = \"slamseq_{}_{}_0_5_deseq_output.csv\".format(project, gene)\n",
    "\n",
    "df_save.to_csv(\"{}/{}/{}\".format(res_dir, project_dir, deseq_res_file), header=False, index=False)\n",
    "\n",
    "print(\"Saved DESeq results to: {}/{}\".format(project_dir, deseq_res_file))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a7d94e40",
   "metadata": {
    "heading_collapsed": true
   },
   "source": [
    "# 4_degrons_v1 (SPI1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 111,
   "id": "94da8186",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "project = \"4_degrons_v1\"\n",
    "project_dir = \"slamseq_{}\".format(project)\n",
    "gene = \"SPI1\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 112,
   "id": "20d41a07",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "5\n"
     ]
    }
   ],
   "source": [
    "files = ! ls $res_dir/$project_dir/*.csv\n",
    "files = [file for file in files if \"deseq\" in file and gene in file]\n",
    "# ERCC files first\n",
    "files = [file for file in files if \"ERCCsamplewise\" in file] + \\\n",
    "[file for file in files if \"ERCCsamplewise\" not in file]\n",
    "print(len(files))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 114,
   "id": "21510ca2",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "# project 3\n",
    "conditions = [\"1h\"]\n",
    "scaling = [\"ERCCsamplewise\", None]\n",
    "count_type = [\"readcounts\", \"tccounts\"]"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "c0f25182",
   "metadata": {
    "hidden": true
   },
   "source": [
    "### make info rows"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 115,
   "id": "8ff11b46",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "# df with file info\n",
    "df_info = pd.DataFrame(list(zip([\"project\", \"file\", \"scale\", \"condition\", \"type\"],\n",
    "                                [\"project\", \"file\", \"scale\", \"condition\", \"type\"])))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 116,
   "id": "bb1a682f",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(5, 26)"
      ]
     },
     "execution_count": 116,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_info = pd.concat([df_info] + [pd.DataFrame([x*6 for x in get_file_info(project, file, conditions)]) \\\n",
    "                                 for file in files], axis=1)\n",
    "df_info.shape"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "20294a75",
   "metadata": {
    "hidden": true
   },
   "source": [
    "### get dfs and merge"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 117,
   "id": "e2ffa85d",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "# Get list of deseq dfs\n",
    "deseq_dfs = [get_deseq_data(file) for file in files]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 118,
   "id": "4d0c4908",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "# main df\n",
    "df_merged = pd.DataFrame()\n",
    "df_merged = df_merged.append(deseq_dfs[0][[\"genes\", \"gene_id\"]]) # add gene cols to main df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 119,
   "id": "df1e7f18",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(16744, 26)"
      ]
     },
     "execution_count": 119,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# join df columns on gene & gene_id columns\n",
    "df_merged = reduce(lambda left, right: pd.merge(left, right, on=['genes', \"gene_id\"], how=\"outer\"), \n",
    "                   [df_merged] + deseq_dfs)\n",
    "df_merged.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 120,
   "id": "0ecbb535",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(16745, 26)"
      ]
     },
     "execution_count": 120,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# change column names\n",
    "df_merged.columns = [col_name.split(\"_\")[0] if \"gene\" not in col_name else \n",
    "                     col_name for col_name in df_merged.columns.tolist()]\n",
    "df_merged = df_merged.T.reset_index().T.reset_index(drop=True)\n",
    "df_merged.shape"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9349618b",
   "metadata": {
    "hidden": true
   },
   "source": [
    "### add info columns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 121,
   "id": "2288aff7",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(5, 26)\n",
      "(16745, 26)\n"
     ]
    }
   ],
   "source": [
    "print(df_info.shape)\n",
    "print(df_merged.shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 122,
   "id": "1e03aaa3",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "# reset column index\n",
    "df_info = df_info.T.reset_index(drop=True).T"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 123,
   "id": "2c983d07",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 4 ms, sys: 4 ms, total: 8 ms\n",
      "Wall time: 10 ms\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "(16750, 26)"
      ]
     },
     "execution_count": 123,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "%%time\n",
    "\n",
    "# combine info and data\n",
    "df_save = df_info.append(df_merged, ignore_index=True)\n",
    "df_save.shape"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a78bc22e",
   "metadata": {
    "hidden": true
   },
   "source": [
    "### save merged file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 124,
   "id": "90d4e870",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved DESeq results to: slamseq_4_degrons_v1/slamseq_4_degrons_v1_SPI1_0_5_deseq_output.csv\n"
     ]
    }
   ],
   "source": [
    "# save file\n",
    "res_dir = \"../results\"\n",
    "deseq_res_file = \"slamseq_{}_{}_0_5_deseq_output.csv\".format(project, gene)\n",
    "\n",
    "df_save.to_csv(\"{}/{}/{}\".format(res_dir, project_dir, deseq_res_file), header=False, index=False)\n",
    "\n",
    "print(\"Saved DESeq results to: {}/{}\".format(project_dir, deseq_res_file))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4afa5c4b",
   "metadata": {
    "heading_collapsed": true
   },
   "source": [
    "# 4_degrons_v1 (GFI1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 125,
   "id": "9b9e9d42",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "project = \"4_degrons_v1\"\n",
    "project_dir = \"slamseq_{}\".format(project)\n",
    "gene = \"GFI1\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 126,
   "id": "06cd79be",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "5\n"
     ]
    }
   ],
   "source": [
    "files = ! ls $res_dir/$project_dir/*.csv\n",
    "files = [file for file in files if \"deseq\" in file and gene in file]\n",
    "# ERCC files first\n",
    "files = [file for file in files if \"ERCCsamplewise\" in file] + \\\n",
    "[file for file in files if \"ERCCsamplewise\" not in file]\n",
    "print(len(files))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 127,
   "id": "4ebfbcdf",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "files = files[:-1]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 128,
   "id": "1d788d5d",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "# project 3\n",
    "conditions = [\"2h\"]\n",
    "scaling = [\"ERCCsamplewise\", None]\n",
    "count_type = [\"readcounts\", \"tccounts\"]"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ea2115f7",
   "metadata": {
    "hidden": true
   },
   "source": [
    "### make info rows"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 129,
   "id": "f3565937",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "# df with file info\n",
    "df_info = pd.DataFrame(list(zip([\"project\", \"file\", \"scale\", \"condition\", \"type\"],\n",
    "                                [\"project\", \"file\", \"scale\", \"condition\", \"type\"])))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 130,
   "id": "68a9e18c",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(5, 26)"
      ]
     },
     "execution_count": 130,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_info = pd.concat([df_info] + [pd.DataFrame([x*6 for x in get_file_info(project, file, conditions)]) \\\n",
    "                                 for file in files], axis=1)\n",
    "df_info.shape"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d3b79c32",
   "metadata": {
    "hidden": true
   },
   "source": [
    "### get dfs and merge"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 131,
   "id": "b3068dd3",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "# Get list of deseq dfs\n",
    "deseq_dfs = [get_deseq_data(file) for file in files]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 132,
   "id": "c051050e",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "# main df\n",
    "df_merged = pd.DataFrame()\n",
    "df_merged = df_merged.append(deseq_dfs[0][[\"genes\", \"gene_id\"]]) # add gene cols to main df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 133,
   "id": "b3a9c57a",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(16744, 26)"
      ]
     },
     "execution_count": 133,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# join df columns on gene & gene_id columns\n",
    "df_merged = reduce(lambda left, right: pd.merge(left, right, on=['genes', \"gene_id\"], how=\"outer\"), \n",
    "                   [df_merged] + deseq_dfs)\n",
    "df_merged.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 134,
   "id": "b5b0d705",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(16745, 26)"
      ]
     },
     "execution_count": 134,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# change column names\n",
    "df_merged.columns = [col_name.split(\"_\")[0] if \"gene\" not in col_name else \n",
    "                     col_name for col_name in df_merged.columns.tolist()]\n",
    "df_merged = df_merged.T.reset_index().T.reset_index(drop=True)\n",
    "df_merged.shape"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a97e6004",
   "metadata": {
    "hidden": true
   },
   "source": [
    "### add info columns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 135,
   "id": "f234cf0b",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(5, 26)\n",
      "(16745, 26)\n"
     ]
    }
   ],
   "source": [
    "print(df_info.shape)\n",
    "print(df_merged.shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 136,
   "id": "8897609a",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "# reset column index\n",
    "df_info = df_info.T.reset_index(drop=True).T"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 137,
   "id": "afd3e7cf",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 8 ms, sys: 0 ns, total: 8 ms\n",
      "Wall time: 7.52 ms\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "(16750, 26)"
      ]
     },
     "execution_count": 137,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "%%time\n",
    "\n",
    "# combine info and data\n",
    "df_save = df_info.append(df_merged, ignore_index=True)\n",
    "df_save.shape"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "206b88cd",
   "metadata": {
    "hidden": true
   },
   "source": [
    "### save merged file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 138,
   "id": "2bb47072",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved DESeq results to: slamseq_4_degrons_v1/slamseq_4_degrons_v1_GFI1_0_5_deseq_output.csv\n"
     ]
    }
   ],
   "source": [
    "# save file\n",
    "res_dir = \"../results\"\n",
    "deseq_res_file = \"slamseq_{}_{}_0_5_deseq_output.csv\".format(project, gene)\n",
    "\n",
    "df_save.to_csv(\"{}/{}/{}\".format(res_dir, project_dir, deseq_res_file), header=False, index=False)\n",
    "\n",
    "print(\"Saved DESeq results to: {}/{}\".format(project_dir, deseq_res_file))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b2d124a6",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "id": "30200707",
   "metadata": {},
   "source": [
    "# Merge all no alt deseq outputs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 192,
   "id": "781af51f",
   "metadata": {},
   "outputs": [],
   "source": [
    "project_dirs = [\"slamseq_IRF2BP2_degraded_v1_res_v4\",\n",
    "                \"slamseq_MYB_degraded_v1_res_v7\",\n",
    "                \"slamseq_4_degrons_v1\"]\n",
    "\n",
    "deseq_res_dirs = [\"{}/{}\".format(res_dir, project_dir) for project_dir in project_dirs]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 193,
   "id": "d56226ae",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../results/slamseq_IRF2BP2_degraded_v1_res_v4/slamseq_IRF2BP2_degraded_v1_res_v4_0_5_deseq_output.csv',\n",
       " '../results/slamseq_MYB_degraded_v1_res_v7/slamseq_MYB_degraded_v1_res_v7_0_5_deseq_output.csv',\n",
       " '../results/slamseq_4_degrons_v1/slamseq_4_degrons_v1_SPI1_0_5_deseq_output.csv',\n",
       " '../results/slamseq_4_degrons_v1/slamseq_4_degrons_v1_GFI1_0_5_deseq_output.csv',\n",
       " '../results/slamseq_4_degrons_v1/slamseq_4_degrons_v1_RUNX1_0_5_deseq_output.csv',\n",
       " '../results/slamseq_4_degrons_v1/slamseq_4_degrons_v1_RUNX2_0_5_deseq_output.csv']"
      ]
     },
     "execution_count": 193,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "deseq_files = [glob.glob(deseq_res_dir+\"/*output.csv\") for deseq_res_dir in deseq_res_dirs]\n",
    "deseq_files = [item for sublist in deseq_files for item in sublist]\n",
    "deseq_files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 194,
   "id": "6d3545ec",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(15637, 50)"
      ]
     },
     "execution_count": 194,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_res = get_deseq_data(deseq_files[0])\n",
    "df_res.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 195,
   "id": "e2c2a981",
   "metadata": {},
   "outputs": [],
   "source": [
    "# list of deseq dfs\n",
    "#deseq_dfs = [get_deseq_data(deseq_file, low_memory=False) for deseq_file in deseq_files]\n",
    "\n",
    "# list of deseq dfs (no header)\n",
    "deseq_dfs = [pd.read_csv(deseq_file, index_col=False, header=None, squeeze=True) for deseq_file in deseq_files]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 197,
   "id": "2451b4ac",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(15638, 50)\n",
      "(17217, 26)\n",
      "(16750, 26)\n",
      "(16750, 26)\n",
      "(16750, 26)\n",
      "(16750, 26)\n"
     ]
    }
   ],
   "source": [
    "for df in deseq_dfs:\n",
    "    print(df.shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 198,
   "id": "89095d68",
   "metadata": {},
   "outputs": [],
   "source": [
    "# make empty df\n",
    "df_save_data = pd.DataFrame()\n",
    "df_save_info = pd.DataFrame()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e1b6c8b2",
   "metadata": {},
   "source": [
    "## Merge info data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 199,
   "id": "721b4e78",
   "metadata": {},
   "outputs": [],
   "source": [
    "def merge_info_cols(df1, df2):\n",
    "    ''' merged info columns from 2 info dfs '''\n",
    "    df_merged = pd.merge(df1, df2, left_on=[0,1], right_on=[0,1])\n",
    "    df_merged = df_merged.T.reset_index(drop=True).T\n",
    "    if (df1.shape[1]+df2.shape[1]-2) != df_merged.shape[1]:\n",
    "        raise ValueError(\"Incorrect number of columns.\\\n",
    "                          df1: {}\\\n",
    "                          df2: {}\".format(df1.shape, df2.shape))\n",
    "    return(df_merged)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 200,
   "id": "2eabdf3f",
   "metadata": {},
   "outputs": [],
   "source": [
    "df_info_merged = deseq_dfs[0].iloc[:5]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 201,
   "id": "3ff903ff",
   "metadata": {},
   "outputs": [],
   "source": [
    "df_res = deseq_dfs[1]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 202,
   "id": "57a930b8",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(15638, 50)"
      ]
     },
     "execution_count": 202,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "deseq_dfs[0].shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 203,
   "id": "5fbb1029",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(5, 74)"
      ]
     },
     "execution_count": 203,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_info_merged = pd.merge(df_info_merged, df_res.iloc[:5], left_on=[0,1], right_on=[0,1])\n",
    "df_info_merged.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 204,
   "id": "6b5e2af5",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "df1 cols: 50 | df2 cols: 26\n",
      "74\n",
      "df1 cols: 74 | df2 cols: 26\n",
      "98\n",
      "df1 cols: 98 | df2 cols: 26\n",
      "122\n",
      "df1 cols: 122 | df2 cols: 26\n",
      "146\n",
      "df1 cols: 146 | df2 cols: 26\n",
      "170\n"
     ]
    }
   ],
   "source": [
    "# IRF2BP2\n",
    "df_info_merged = deseq_dfs[0].iloc[:5]\n",
    "\n",
    "# merge info columns\n",
    "for df_res in deseq_dfs[1:]:\n",
    "    print(\"df1 cols: {} | df2 cols: {}\".format(df_info_merged.shape[1], df_res.shape[1]))\n",
    "    df_info = df_res.iloc[:5]\n",
    "    df_info_merged = merge_info_cols(df_info_merged, df_info)\n",
    "    print(df_info_merged.shape[1])"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e8d78fce",
   "metadata": {},
   "source": [
    "## Merge gene data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "8b83ae26",
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_deseq_data(file, low_memory=True):\n",
    "    ''' input deseq file & returns df '''\n",
    "    ''' add file name to headers except gene columns '''\n",
    "    df = pd.read_csv(file, header=[0], index_col=None, squeeze=True, low_memory=low_memory)\n",
    "    df.columns = [\"{}_{}\".format(col_name, os.path.basename(file).split(\".\")[0]) if \"gene\" not in col_name\n",
    "                  else col_name for col_name in df.columns.tolist()]\n",
    "    return(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 226,
   "id": "07e46fa8",
   "metadata": {},
   "outputs": [],
   "source": [
    "def rename_deseq_data(df):\n",
    "    ''' renames columns uniquely to deseq output header '''\n",
    "    df.columns = [\"{}_{}\".format(col_name, str(idx)) if \"gene\" not in col_name\n",
    "                  else col_name for idx, col_name in enumerate(df.iloc[0].tolist())]\n",
    "    df = df.iloc[1:]\n",
    "    return(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 233,
   "id": "e5aefe6b",
   "metadata": {},
   "outputs": [],
   "source": [
    "# get df data without info\n",
    "deseq_df_data = [df_res.iloc[5:] for df_res in deseq_dfs]\n",
    "\n",
    "# make first row col names & unique\n",
    "deseq_df_data = [rename_deseq_data(df_data) for df_data in deseq_df_data]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 247,
   "id": "fd19eb42",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(15632, 50)\n",
      "(17211, 26)\n",
      "(16744, 26)\n",
      "(16744, 26)\n",
      "(16744, 26)\n",
      "(16744, 26)\n"
     ]
    }
   ],
   "source": [
    "for df_data in deseq_df_data:\n",
    "    print(df_data.shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 249,
   "id": "b45c233e",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>project_0</th>\n",
       "      <th>project_1</th>\n",
       "      <th>IRF2BP2_degraded_v1_2</th>\n",
       "      <th>IRF2BP2_degraded_v1_3</th>\n",
       "      <th>IRF2BP2_degraded_v1_4</th>\n",
       "      <th>IRF2BP2_degraded_v1_5</th>\n",
       "      <th>IRF2BP2_degraded_v1_6</th>\n",
       "      <th>IRF2BP2_degraded_v1_7</th>\n",
       "      <th>IRF2BP2_degraded_v1_8</th>\n",
       "      <th>IRF2BP2_degraded_v1_9</th>\n",
       "      <th>...</th>\n",
       "      <th>IRF2BP2_degraded_v1_40</th>\n",
       "      <th>IRF2BP2_degraded_v1_41</th>\n",
       "      <th>IRF2BP2_degraded_v1_42</th>\n",
       "      <th>IRF2BP2_degraded_v1_43</th>\n",
       "      <th>IRF2BP2_degraded_v1_44</th>\n",
       "      <th>IRF2BP2_degraded_v1_45</th>\n",
       "      <th>IRF2BP2_degraded_v1_46</th>\n",
       "      <th>IRF2BP2_degraded_v1_47</th>\n",
       "      <th>IRF2BP2_degraded_v1_48</th>\n",
       "      <th>IRF2BP2_degraded_v1_49</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>genes</td>\n",
       "      <td>gene_id</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>...</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>NAT1</td>\n",
       "      <td>NAT1</td>\n",
       "      <td>95.87059628671004</td>\n",
       "      <td>-0.4618119842755553</td>\n",
       "      <td>0.1838993165621769</td>\n",
       "      <td>2.5112218626403395</td>\n",
       "      <td>0.0120314054862605</td>\n",
       "      <td>0.1764824098229201</td>\n",
       "      <td>5.033275352515851</td>\n",
       "      <td>-0.5602362002955328</td>\n",
       "      <td>...</td>\n",
       "      <td>0.2499835106626506</td>\n",
       "      <td>1.345021316023153</td>\n",
       "      <td>0.1786183509749246</td>\n",
       "      <td>0.9182518409151246</td>\n",
       "      <td>0.7526302401777346</td>\n",
       "      <td>2.0042049937800543</td>\n",
       "      <td>1.8278146338544723</td>\n",
       "      <td>-1.0965034181576785</td>\n",
       "      <td>0.2728585282994329</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>AAMP</td>\n",
       "      <td>AAMP</td>\n",
       "      <td>842.3716209229535</td>\n",
       "      <td>-0.1476064480241523</td>\n",
       "      <td>0.1180792991839148</td>\n",
       "      <td>1.2500620264882107</td>\n",
       "      <td>0.2112768900493981</td>\n",
       "      <td>0.621101974228759</td>\n",
       "      <td>14.553492083441196</td>\n",
       "      <td>0.3101726260809158</td>\n",
       "      <td>...</td>\n",
       "      <td>0.1436118349988333</td>\n",
       "      <td>-1.174214545359719</td>\n",
       "      <td>0.2403091013937329</td>\n",
       "      <td>0.9594495599032664</td>\n",
       "      <td>7.433181480720183</td>\n",
       "      <td>0.2842585712308931</td>\n",
       "      <td>0.6006672968377171</td>\n",
       "      <td>-0.4732379683851699</td>\n",
       "      <td>0.6360434123857338</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>AARS1</td>\n",
       "      <td>AARS1</td>\n",
       "      <td>1374.1764868232967</td>\n",
       "      <td>-0.0252481964182913</td>\n",
       "      <td>0.1093729310308428</td>\n",
       "      <td>0.2308450196984433</td>\n",
       "      <td>0.8174352049833612</td>\n",
       "      <td>0.957161656507632</td>\n",
       "      <td>16.200285486653765</td>\n",
       "      <td>-0.1372280711065793</td>\n",
       "      <td>...</td>\n",
       "      <td>0.1671727054204942</td>\n",
       "      <td>-0.6585513747031996</td>\n",
       "      <td>0.5101838964077127</td>\n",
       "      <td>0.9966744113218344</td>\n",
       "      <td>11.302473594467742</td>\n",
       "      <td>0.0730244615436611</td>\n",
       "      <td>0.4905015559836879</td>\n",
       "      <td>-0.1488771251646949</td>\n",
       "      <td>0.8816505915117299</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>ABAT</td>\n",
       "      <td>ABAT</td>\n",
       "      <td>55.9224048891824</td>\n",
       "      <td>-0.3585220629160938</td>\n",
       "      <td>0.2117003248291207</td>\n",
       "      <td>1.6935357241680382</td>\n",
       "      <td>0.0903535409020921</td>\n",
       "      <td>0.4386939785389467</td>\n",
       "      <td>1.693745222699288</td>\n",
       "      <td>-0.9499849425240666</td>\n",
       "      <td>...</td>\n",
       "      <td>0.2681252539633148</td>\n",
       "      <td>0.426446279789382</td>\n",
       "      <td>0.6697826846194971</td>\n",
       "      <td>0.9966744113218344</td>\n",
       "      <td>0.6345738591917904</td>\n",
       "      <td>-1.0072747380631193</td>\n",
       "      <td>2.0802920535413554</td>\n",
       "      <td>0.4841987144778059</td>\n",
       "      <td>0.6282448491442183</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows Ã— 50 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "  project_0 project_1 IRF2BP2_degraded_v1_2 IRF2BP2_degraded_v1_3  \\\n",
       "5     genes   gene_id              baseMean        log2FoldChange   \n",
       "6      NAT1      NAT1     95.87059628671004   -0.4618119842755553   \n",
       "7      AAMP      AAMP     842.3716209229535   -0.1476064480241523   \n",
       "8     AARS1     AARS1    1374.1764868232967   -0.0252481964182913   \n",
       "9      ABAT      ABAT      55.9224048891824   -0.3585220629160938   \n",
       "\n",
       "  IRF2BP2_degraded_v1_4 IRF2BP2_degraded_v1_5 IRF2BP2_degraded_v1_6  \\\n",
       "5                 lfcSE                  stat                pvalue   \n",
       "6    0.1838993165621769    2.5112218626403395    0.0120314054862605   \n",
       "7    0.1180792991839148    1.2500620264882107    0.2112768900493981   \n",
       "8    0.1093729310308428    0.2308450196984433    0.8174352049833612   \n",
       "9    0.2117003248291207    1.6935357241680382    0.0903535409020921   \n",
       "\n",
       "  IRF2BP2_degraded_v1_7 IRF2BP2_degraded_v1_8 IRF2BP2_degraded_v1_9  ...  \\\n",
       "5                  padj              baseMean        log2FoldChange  ...   \n",
       "6    0.1764824098229201     5.033275352515851   -0.5602362002955328  ...   \n",
       "7     0.621101974228759    14.553492083441196    0.3101726260809158  ...   \n",
       "8     0.957161656507632    16.200285486653765   -0.1372280711065793  ...   \n",
       "9    0.4386939785389467     1.693745222699288   -0.9499849425240666  ...   \n",
       "\n",
       "  IRF2BP2_degraded_v1_40 IRF2BP2_degraded_v1_41 IRF2BP2_degraded_v1_42  \\\n",
       "5                  lfcSE                   stat                 pvalue   \n",
       "6     0.2499835106626506      1.345021316023153     0.1786183509749246   \n",
       "7     0.1436118349988333     -1.174214545359719     0.2403091013937329   \n",
       "8     0.1671727054204942    -0.6585513747031996     0.5101838964077127   \n",
       "9     0.2681252539633148      0.426446279789382     0.6697826846194971   \n",
       "\n",
       "  IRF2BP2_degraded_v1_43 IRF2BP2_degraded_v1_44 IRF2BP2_degraded_v1_45  \\\n",
       "5                   padj               baseMean         log2FoldChange   \n",
       "6     0.9182518409151246     0.7526302401777346     2.0042049937800543   \n",
       "7     0.9594495599032664      7.433181480720183     0.2842585712308931   \n",
       "8     0.9966744113218344     11.302473594467742     0.0730244615436611   \n",
       "9     0.9966744113218344     0.6345738591917904    -1.0072747380631193   \n",
       "\n",
       "  IRF2BP2_degraded_v1_46 IRF2BP2_degraded_v1_47 IRF2BP2_degraded_v1_48  \\\n",
       "5                  lfcSE                   stat                 pvalue   \n",
       "6     1.8278146338544723    -1.0965034181576785     0.2728585282994329   \n",
       "7     0.6006672968377171    -0.4732379683851699     0.6360434123857338   \n",
       "8     0.4905015559836879    -0.1488771251646949     0.8816505915117299   \n",
       "9     2.0802920535413554     0.4841987144778059     0.6282448491442183   \n",
       "\n",
       "  IRF2BP2_degraded_v1_49  \n",
       "5                   padj  \n",
       "6                    NaN  \n",
       "7                    NaN  \n",
       "8                    NaN  \n",
       "9                    NaN  \n",
       "\n",
       "[5 rows x 50 columns]"
      ]
     },
     "execution_count": 249,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# get df data without info\n",
    "deseq_df_data = [df_res.iloc[5:] for df_res in deseq_dfs]\n",
    "deseq_df_data[0].iloc[:5]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 251,
   "id": "af6a056d",
   "metadata": {},
   "outputs": [],
   "source": [
    "# main df\n",
    "df_data_merged = pd.DataFrame()\n",
    "df_data_merged = df_data_merged.append(deseq_df_data[0][[\"project_0\", \"project_1\"]]) # add gene cols to main df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 254,
   "id": "4a08b5be",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_7450/3925882540.py:2: FutureWarning: Passing 'suffixes' which cause duplicate columns {'4_degrons_v1_18_x', '4_degrons_v1_5_x', '4_degrons_v1_11_x', '4_degrons_v1_9_x', '4_degrons_v1_7_x', '4_degrons_v1_4_x', '4_degrons_v1_13_x', '4_degrons_v1_19_x', '4_degrons_v1_17_x', '4_degrons_v1_12_x', '4_degrons_v1_15_x', '4_degrons_v1_10_x', '4_degrons_v1_23_x', '4_degrons_v1_21_x', '4_degrons_v1_8_x', '4_degrons_v1_16_x', '4_degrons_v1_22_x', '4_degrons_v1_14_x', '4_degrons_v1_3_x', '4_degrons_v1_25_x', '4_degrons_v1_2_x', '4_degrons_v1_6_x', '4_degrons_v1_24_x', '4_degrons_v1_20_x'} in the result is deprecated and will raise a MergeError in a future version.\n",
      "  df_data_merged = reduce(lambda left, right: pd.merge(left, right, on=[\"project_0\", \"project_1\"], how=\"outer\"),\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "(18323, 170)"
      ]
     },
     "execution_count": 254,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# join df columns on gene & gene_id columns\n",
    "df_data_merged = reduce(lambda left, right: pd.merge(left, right, on=[\"project_0\", \"project_1\"], how=\"outer\"), \n",
    "                        [df_data_merged] + deseq_df_data)\n",
    "df_data_merged.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 256,
   "id": "117c3d9b",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>project_0</th>\n",
       "      <th>project_1</th>\n",
       "      <th>IRF2BP2_degraded_v1_2</th>\n",
       "      <th>IRF2BP2_degraded_v1_3</th>\n",
       "      <th>IRF2BP2_degraded_v1_4</th>\n",
       "      <th>IRF2BP2_degraded_v1_5</th>\n",
       "      <th>IRF2BP2_degraded_v1_6</th>\n",
       "      <th>IRF2BP2_degraded_v1_7</th>\n",
       "      <th>IRF2BP2_degraded_v1_8</th>\n",
       "      <th>IRF2BP2_degraded_v1_9</th>\n",
       "      <th>...</th>\n",
       "      <th>4_degrons_v1_16_y</th>\n",
       "      <th>4_degrons_v1_17_y</th>\n",
       "      <th>4_degrons_v1_18_y</th>\n",
       "      <th>4_degrons_v1_19_y</th>\n",
       "      <th>4_degrons_v1_20_y</th>\n",
       "      <th>4_degrons_v1_21_y</th>\n",
       "      <th>4_degrons_v1_22_y</th>\n",
       "      <th>4_degrons_v1_23_y</th>\n",
       "      <th>4_degrons_v1_24_y</th>\n",
       "      <th>4_degrons_v1_25_y</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>genes</td>\n",
       "      <td>gene_id</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>...</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>NAT1</td>\n",
       "      <td>NAT1</td>\n",
       "      <td>95.87059628671004</td>\n",
       "      <td>-0.4618119842755553</td>\n",
       "      <td>0.1838993165621769</td>\n",
       "      <td>2.5112218626403395</td>\n",
       "      <td>0.0120314054862605</td>\n",
       "      <td>0.1764824098229201</td>\n",
       "      <td>5.033275352515851</td>\n",
       "      <td>-0.5602362002955328</td>\n",
       "      <td>...</td>\n",
       "      <td>0.1910164963313597</td>\n",
       "      <td>1.195738593227763</td>\n",
       "      <td>0.2317985857833831</td>\n",
       "      <td>0.5815447582945407</td>\n",
       "      <td>6.951305773266107</td>\n",
       "      <td>-0.303572495248922</td>\n",
       "      <td>0.6056459757244428</td>\n",
       "      <td>0.5012375338345212</td>\n",
       "      <td>0.6162039617032911</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>AAMP</td>\n",
       "      <td>AAMP</td>\n",
       "      <td>842.3716209229535</td>\n",
       "      <td>-0.1476064480241523</td>\n",
       "      <td>0.1180792991839148</td>\n",
       "      <td>1.2500620264882107</td>\n",
       "      <td>0.2112768900493981</td>\n",
       "      <td>0.621101974228759</td>\n",
       "      <td>14.553492083441196</td>\n",
       "      <td>0.3101726260809158</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0936551287545897</td>\n",
       "      <td>-0.7816068093858641</td>\n",
       "      <td>0.4344456838431942</td>\n",
       "      <td>0.7234391226069221</td>\n",
       "      <td>31.25961097658043</td>\n",
       "      <td>0.3155357818127773</td>\n",
       "      <td>0.2868775385203951</td>\n",
       "      <td>-1.0998971318569954</td>\n",
       "      <td>0.2713769445263918</td>\n",
       "      <td>0.5759060637707974</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>AARS1</td>\n",
       "      <td>AARS1</td>\n",
       "      <td>1374.1764868232967</td>\n",
       "      <td>-0.0252481964182913</td>\n",
       "      <td>0.1093729310308428</td>\n",
       "      <td>0.2308450196984433</td>\n",
       "      <td>0.8174352049833612</td>\n",
       "      <td>0.957161656507632</td>\n",
       "      <td>16.200285486653765</td>\n",
       "      <td>-0.1372280711065793</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0853883738066852</td>\n",
       "      <td>-0.0376794020496949</td>\n",
       "      <td>0.9699432991159118</td>\n",
       "      <td>0.9854767365932272</td>\n",
       "      <td>22.7732265306714</td>\n",
       "      <td>0.3114110726026999</td>\n",
       "      <td>0.3184951940658864</td>\n",
       "      <td>-0.9777575247753314</td>\n",
       "      <td>0.3281942632603161</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>ABAT</td>\n",
       "      <td>ABAT</td>\n",
       "      <td>55.9224048891824</td>\n",
       "      <td>-0.3585220629160938</td>\n",
       "      <td>0.2117003248291207</td>\n",
       "      <td>1.6935357241680382</td>\n",
       "      <td>0.0903535409020921</td>\n",
       "      <td>0.4386939785389467</td>\n",
       "      <td>1.693745222699288</td>\n",
       "      <td>-0.9499849425240666</td>\n",
       "      <td>...</td>\n",
       "      <td>0.1522534347786068</td>\n",
       "      <td>0.3937569945555876</td>\n",
       "      <td>0.6937604579667991</td>\n",
       "      <td>0.8679025465000523</td>\n",
       "      <td>6.911045666718635</td>\n",
       "      <td>-0.1534162228659588</td>\n",
       "      <td>0.5532445763431577</td>\n",
       "      <td>0.2773027146149558</td>\n",
       "      <td>0.7815476753088093</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows Ã— 170 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "  project_0 project_1 IRF2BP2_degraded_v1_2 IRF2BP2_degraded_v1_3  \\\n",
       "0     genes   gene_id              baseMean        log2FoldChange   \n",
       "1      NAT1      NAT1     95.87059628671004   -0.4618119842755553   \n",
       "2      AAMP      AAMP     842.3716209229535   -0.1476064480241523   \n",
       "3     AARS1     AARS1    1374.1764868232967   -0.0252481964182913   \n",
       "4      ABAT      ABAT      55.9224048891824   -0.3585220629160938   \n",
       "\n",
       "  IRF2BP2_degraded_v1_4 IRF2BP2_degraded_v1_5 IRF2BP2_degraded_v1_6  \\\n",
       "0                 lfcSE                  stat                pvalue   \n",
       "1    0.1838993165621769    2.5112218626403395    0.0120314054862605   \n",
       "2    0.1180792991839148    1.2500620264882107    0.2112768900493981   \n",
       "3    0.1093729310308428    0.2308450196984433    0.8174352049833612   \n",
       "4    0.2117003248291207    1.6935357241680382    0.0903535409020921   \n",
       "\n",
       "  IRF2BP2_degraded_v1_7 IRF2BP2_degraded_v1_8 IRF2BP2_degraded_v1_9  ...  \\\n",
       "0                  padj              baseMean        log2FoldChange  ...   \n",
       "1    0.1764824098229201     5.033275352515851   -0.5602362002955328  ...   \n",
       "2     0.621101974228759    14.553492083441196    0.3101726260809158  ...   \n",
       "3     0.957161656507632    16.200285486653765   -0.1372280711065793  ...   \n",
       "4    0.4386939785389467     1.693745222699288   -0.9499849425240666  ...   \n",
       "\n",
       "    4_degrons_v1_16_y    4_degrons_v1_17_y   4_degrons_v1_18_y  \\\n",
       "0               lfcSE                 stat              pvalue   \n",
       "1  0.1910164963313597    1.195738593227763  0.2317985857833831   \n",
       "2  0.0936551287545897  -0.7816068093858641  0.4344456838431942   \n",
       "3  0.0853883738066852  -0.0376794020496949  0.9699432991159118   \n",
       "4  0.1522534347786068   0.3937569945555876  0.6937604579667991   \n",
       "\n",
       "    4_degrons_v1_19_y  4_degrons_v1_20_y    4_degrons_v1_21_y  \\\n",
       "0                padj           baseMean       log2FoldChange   \n",
       "1  0.5815447582945407  6.951305773266107   -0.303572495248922   \n",
       "2  0.7234391226069221  31.25961097658043   0.3155357818127773   \n",
       "3  0.9854767365932272   22.7732265306714   0.3114110726026999   \n",
       "4  0.8679025465000523  6.911045666718635  -0.1534162228659588   \n",
       "\n",
       "    4_degrons_v1_22_y    4_degrons_v1_23_y   4_degrons_v1_24_y  \\\n",
       "0               lfcSE                 stat              pvalue   \n",
       "1  0.6056459757244428   0.5012375338345212  0.6162039617032911   \n",
       "2  0.2868775385203951  -1.0998971318569954  0.2713769445263918   \n",
       "3  0.3184951940658864  -0.9777575247753314  0.3281942632603161   \n",
       "4  0.5532445763431577   0.2773027146149558  0.7815476753088093   \n",
       "\n",
       "    4_degrons_v1_25_y  \n",
       "0                padj  \n",
       "1                 NaN  \n",
       "2  0.5759060637707974  \n",
       "3                 NaN  \n",
       "4                 NaN  \n",
       "\n",
       "[5 rows x 170 columns]"
      ]
     },
     "execution_count": 256,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_data_merged.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 264,
   "id": "786a4670",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total unique genes: 18322\n"
     ]
    }
   ],
   "source": [
    "# total unique genes\n",
    "unique_gene_list = []\n",
    "for df_data in deseq_df_data:\n",
    "    unique_gene_list.extend(df_data.iloc[1:,0].tolist())\n",
    "print(\"Total unique genes: {}\".format(len(list(set(unique_gene_list)))))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 280,
   "id": "9d69de31",
   "metadata": {},
   "outputs": [],
   "source": [
    "df_merged = df_data_merged"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 281,
   "id": "199c3523",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>project_0</th>\n",
       "      <th>project_1</th>\n",
       "      <th>IRF2BP2_degraded_v1_2</th>\n",
       "      <th>IRF2BP2_degraded_v1_3</th>\n",
       "      <th>IRF2BP2_degraded_v1_4</th>\n",
       "      <th>IRF2BP2_degraded_v1_5</th>\n",
       "      <th>IRF2BP2_degraded_v1_6</th>\n",
       "      <th>IRF2BP2_degraded_v1_7</th>\n",
       "      <th>IRF2BP2_degraded_v1_8</th>\n",
       "      <th>IRF2BP2_degraded_v1_9</th>\n",
       "      <th>...</th>\n",
       "      <th>4_degrons_v1_16_y</th>\n",
       "      <th>4_degrons_v1_17_y</th>\n",
       "      <th>4_degrons_v1_18_y</th>\n",
       "      <th>4_degrons_v1_19_y</th>\n",
       "      <th>4_degrons_v1_20_y</th>\n",
       "      <th>4_degrons_v1_21_y</th>\n",
       "      <th>4_degrons_v1_22_y</th>\n",
       "      <th>4_degrons_v1_23_y</th>\n",
       "      <th>4_degrons_v1_24_y</th>\n",
       "      <th>4_degrons_v1_25_y</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>genes</td>\n",
       "      <td>gene_id</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>...</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>NAT1</td>\n",
       "      <td>NAT1</td>\n",
       "      <td>95.87059628671004</td>\n",
       "      <td>-0.4618119842755553</td>\n",
       "      <td>0.1838993165621769</td>\n",
       "      <td>2.5112218626403395</td>\n",
       "      <td>0.0120314054862605</td>\n",
       "      <td>0.1764824098229201</td>\n",
       "      <td>5.033275352515851</td>\n",
       "      <td>-0.5602362002955328</td>\n",
       "      <td>...</td>\n",
       "      <td>0.1910164963313597</td>\n",
       "      <td>1.195738593227763</td>\n",
       "      <td>0.2317985857833831</td>\n",
       "      <td>0.5815447582945407</td>\n",
       "      <td>6.951305773266107</td>\n",
       "      <td>-0.303572495248922</td>\n",
       "      <td>0.6056459757244428</td>\n",
       "      <td>0.5012375338345212</td>\n",
       "      <td>0.6162039617032911</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>AAMP</td>\n",
       "      <td>AAMP</td>\n",
       "      <td>842.3716209229535</td>\n",
       "      <td>-0.1476064480241523</td>\n",
       "      <td>0.1180792991839148</td>\n",
       "      <td>1.2500620264882107</td>\n",
       "      <td>0.2112768900493981</td>\n",
       "      <td>0.621101974228759</td>\n",
       "      <td>14.553492083441196</td>\n",
       "      <td>0.3101726260809158</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0936551287545897</td>\n",
       "      <td>-0.7816068093858641</td>\n",
       "      <td>0.4344456838431942</td>\n",
       "      <td>0.7234391226069221</td>\n",
       "      <td>31.25961097658043</td>\n",
       "      <td>0.3155357818127773</td>\n",
       "      <td>0.2868775385203951</td>\n",
       "      <td>-1.0998971318569954</td>\n",
       "      <td>0.2713769445263918</td>\n",
       "      <td>0.5759060637707974</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>AARS1</td>\n",
       "      <td>AARS1</td>\n",
       "      <td>1374.1764868232967</td>\n",
       "      <td>-0.0252481964182913</td>\n",
       "      <td>0.1093729310308428</td>\n",
       "      <td>0.2308450196984433</td>\n",
       "      <td>0.8174352049833612</td>\n",
       "      <td>0.957161656507632</td>\n",
       "      <td>16.200285486653765</td>\n",
       "      <td>-0.1372280711065793</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0853883738066852</td>\n",
       "      <td>-0.0376794020496949</td>\n",
       "      <td>0.9699432991159118</td>\n",
       "      <td>0.9854767365932272</td>\n",
       "      <td>22.7732265306714</td>\n",
       "      <td>0.3114110726026999</td>\n",
       "      <td>0.3184951940658864</td>\n",
       "      <td>-0.9777575247753314</td>\n",
       "      <td>0.3281942632603161</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>ABAT</td>\n",
       "      <td>ABAT</td>\n",
       "      <td>55.9224048891824</td>\n",
       "      <td>-0.3585220629160938</td>\n",
       "      <td>0.2117003248291207</td>\n",
       "      <td>1.6935357241680382</td>\n",
       "      <td>0.0903535409020921</td>\n",
       "      <td>0.4386939785389467</td>\n",
       "      <td>1.693745222699288</td>\n",
       "      <td>-0.9499849425240666</td>\n",
       "      <td>...</td>\n",
       "      <td>0.1522534347786068</td>\n",
       "      <td>0.3937569945555876</td>\n",
       "      <td>0.6937604579667991</td>\n",
       "      <td>0.8679025465000523</td>\n",
       "      <td>6.911045666718635</td>\n",
       "      <td>-0.1534162228659588</td>\n",
       "      <td>0.5532445763431577</td>\n",
       "      <td>0.2773027146149558</td>\n",
       "      <td>0.7815476753088093</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows Ã— 170 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "  project_0 project_1 IRF2BP2_degraded_v1_2 IRF2BP2_degraded_v1_3  \\\n",
       "0     genes   gene_id              baseMean        log2FoldChange   \n",
       "1      NAT1      NAT1     95.87059628671004   -0.4618119842755553   \n",
       "2      AAMP      AAMP     842.3716209229535   -0.1476064480241523   \n",
       "3     AARS1     AARS1    1374.1764868232967   -0.0252481964182913   \n",
       "4      ABAT      ABAT      55.9224048891824   -0.3585220629160938   \n",
       "\n",
       "  IRF2BP2_degraded_v1_4 IRF2BP2_degraded_v1_5 IRF2BP2_degraded_v1_6  \\\n",
       "0                 lfcSE                  stat                pvalue   \n",
       "1    0.1838993165621769    2.5112218626403395    0.0120314054862605   \n",
       "2    0.1180792991839148    1.2500620264882107    0.2112768900493981   \n",
       "3    0.1093729310308428    0.2308450196984433    0.8174352049833612   \n",
       "4    0.2117003248291207    1.6935357241680382    0.0903535409020921   \n",
       "\n",
       "  IRF2BP2_degraded_v1_7 IRF2BP2_degraded_v1_8 IRF2BP2_degraded_v1_9  ...  \\\n",
       "0                  padj              baseMean        log2FoldChange  ...   \n",
       "1    0.1764824098229201     5.033275352515851   -0.5602362002955328  ...   \n",
       "2     0.621101974228759    14.553492083441196    0.3101726260809158  ...   \n",
       "3     0.957161656507632    16.200285486653765   -0.1372280711065793  ...   \n",
       "4    0.4386939785389467     1.693745222699288   -0.9499849425240666  ...   \n",
       "\n",
       "    4_degrons_v1_16_y    4_degrons_v1_17_y   4_degrons_v1_18_y  \\\n",
       "0               lfcSE                 stat              pvalue   \n",
       "1  0.1910164963313597    1.195738593227763  0.2317985857833831   \n",
       "2  0.0936551287545897  -0.7816068093858641  0.4344456838431942   \n",
       "3  0.0853883738066852  -0.0376794020496949  0.9699432991159118   \n",
       "4  0.1522534347786068   0.3937569945555876  0.6937604579667991   \n",
       "\n",
       "    4_degrons_v1_19_y  4_degrons_v1_20_y    4_degrons_v1_21_y  \\\n",
       "0                padj           baseMean       log2FoldChange   \n",
       "1  0.5815447582945407  6.951305773266107   -0.303572495248922   \n",
       "2  0.7234391226069221  31.25961097658043   0.3155357818127773   \n",
       "3  0.9854767365932272   22.7732265306714   0.3114110726026999   \n",
       "4  0.8679025465000523  6.911045666718635  -0.1534162228659588   \n",
       "\n",
       "    4_degrons_v1_22_y    4_degrons_v1_23_y   4_degrons_v1_24_y  \\\n",
       "0               lfcSE                 stat              pvalue   \n",
       "1  0.6056459757244428   0.5012375338345212  0.6162039617032911   \n",
       "2  0.2868775385203951  -1.0998971318569954  0.2713769445263918   \n",
       "3  0.3184951940658864  -0.9777575247753314  0.3281942632603161   \n",
       "4  0.5532445763431577   0.2773027146149558  0.7815476753088093   \n",
       "\n",
       "    4_degrons_v1_25_y  \n",
       "0                padj  \n",
       "1                 NaN  \n",
       "2  0.5759060637707974  \n",
       "3                 NaN  \n",
       "4                 NaN  \n",
       "\n",
       "[5 rows x 170 columns]"
      ]
     },
     "execution_count": 281,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_merged.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 283,
   "id": "ab780e43",
   "metadata": {},
   "outputs": [],
   "source": [
    "df_merged = df_merged.T.reset_index(drop=True).T"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 284,
   "id": "4c1cf7da",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "      <th>3</th>\n",
       "      <th>4</th>\n",
       "      <th>5</th>\n",
       "      <th>6</th>\n",
       "      <th>7</th>\n",
       "      <th>8</th>\n",
       "      <th>9</th>\n",
       "      <th>...</th>\n",
       "      <th>160</th>\n",
       "      <th>161</th>\n",
       "      <th>162</th>\n",
       "      <th>163</th>\n",
       "      <th>164</th>\n",
       "      <th>165</th>\n",
       "      <th>166</th>\n",
       "      <th>167</th>\n",
       "      <th>168</th>\n",
       "      <th>169</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>genes</td>\n",
       "      <td>gene_id</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>...</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>NAT1</td>\n",
       "      <td>NAT1</td>\n",
       "      <td>95.87059628671004</td>\n",
       "      <td>-0.4618119842755553</td>\n",
       "      <td>0.1838993165621769</td>\n",
       "      <td>2.5112218626403395</td>\n",
       "      <td>0.0120314054862605</td>\n",
       "      <td>0.1764824098229201</td>\n",
       "      <td>5.033275352515851</td>\n",
       "      <td>-0.5602362002955328</td>\n",
       "      <td>...</td>\n",
       "      <td>0.1910164963313597</td>\n",
       "      <td>1.195738593227763</td>\n",
       "      <td>0.2317985857833831</td>\n",
       "      <td>0.5815447582945407</td>\n",
       "      <td>6.951305773266107</td>\n",
       "      <td>-0.303572495248922</td>\n",
       "      <td>0.6056459757244428</td>\n",
       "      <td>0.5012375338345212</td>\n",
       "      <td>0.6162039617032911</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>AAMP</td>\n",
       "      <td>AAMP</td>\n",
       "      <td>842.3716209229535</td>\n",
       "      <td>-0.1476064480241523</td>\n",
       "      <td>0.1180792991839148</td>\n",
       "      <td>1.2500620264882107</td>\n",
       "      <td>0.2112768900493981</td>\n",
       "      <td>0.621101974228759</td>\n",
       "      <td>14.553492083441196</td>\n",
       "      <td>0.3101726260809158</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0936551287545897</td>\n",
       "      <td>-0.7816068093858641</td>\n",
       "      <td>0.4344456838431942</td>\n",
       "      <td>0.7234391226069221</td>\n",
       "      <td>31.25961097658043</td>\n",
       "      <td>0.3155357818127773</td>\n",
       "      <td>0.2868775385203951</td>\n",
       "      <td>-1.0998971318569954</td>\n",
       "      <td>0.2713769445263918</td>\n",
       "      <td>0.5759060637707974</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>AARS1</td>\n",
       "      <td>AARS1</td>\n",
       "      <td>1374.1764868232967</td>\n",
       "      <td>-0.0252481964182913</td>\n",
       "      <td>0.1093729310308428</td>\n",
       "      <td>0.2308450196984433</td>\n",
       "      <td>0.8174352049833612</td>\n",
       "      <td>0.957161656507632</td>\n",
       "      <td>16.200285486653765</td>\n",
       "      <td>-0.1372280711065793</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0853883738066852</td>\n",
       "      <td>-0.0376794020496949</td>\n",
       "      <td>0.9699432991159118</td>\n",
       "      <td>0.9854767365932272</td>\n",
       "      <td>22.7732265306714</td>\n",
       "      <td>0.3114110726026999</td>\n",
       "      <td>0.3184951940658864</td>\n",
       "      <td>-0.9777575247753314</td>\n",
       "      <td>0.3281942632603161</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>ABAT</td>\n",
       "      <td>ABAT</td>\n",
       "      <td>55.9224048891824</td>\n",
       "      <td>-0.3585220629160938</td>\n",
       "      <td>0.2117003248291207</td>\n",
       "      <td>1.6935357241680382</td>\n",
       "      <td>0.0903535409020921</td>\n",
       "      <td>0.4386939785389467</td>\n",
       "      <td>1.693745222699288</td>\n",
       "      <td>-0.9499849425240666</td>\n",
       "      <td>...</td>\n",
       "      <td>0.1522534347786068</td>\n",
       "      <td>0.3937569945555876</td>\n",
       "      <td>0.6937604579667991</td>\n",
       "      <td>0.8679025465000523</td>\n",
       "      <td>6.911045666718635</td>\n",
       "      <td>-0.1534162228659588</td>\n",
       "      <td>0.5532445763431577</td>\n",
       "      <td>0.2773027146149558</td>\n",
       "      <td>0.7815476753088093</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18318</th>\n",
       "      <td>AC113208.3</td>\n",
       "      <td>AC113208.3</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>0.8855085832558085</td>\n",
       "      <td>1.1782373841042888</td>\n",
       "      <td>0.2387019826835846</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>-0.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18319</th>\n",
       "      <td>105377586</td>\n",
       "      <td>105377586</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>0.9973246137985076</td>\n",
       "      <td>0.3124464112381569</td>\n",
       "      <td>0.7547012834221719</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.5719749369309547</td>\n",
       "      <td>-1.908720323682767</td>\n",
       "      <td>1.9634370487950688</td>\n",
       "      <td>0.9721321724341097</td>\n",
       "      <td>0.3309847985506584</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18320</th>\n",
       "      <td>105378663</td>\n",
       "      <td>105378663</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>0.859766767514565</td>\n",
       "      <td>-1.265830008396275</td>\n",
       "      <td>0.2055739490192727</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.783802604256384</td>\n",
       "      <td>1.878134129592003</td>\n",
       "      <td>1.7378394499003909</td>\n",
       "      <td>-1.0807293675486844</td>\n",
       "      <td>0.2798175156218232</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18321</th>\n",
       "      <td>FRGCA</td>\n",
       "      <td>FRGCA</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>-0.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18322</th>\n",
       "      <td>MIR34AHG</td>\n",
       "      <td>MIR34AHG</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>1.857862193029944</td>\n",
       "      <td>1.261161177260749</td>\n",
       "      <td>0.207250782833181</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>-0.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>18323 rows Ã— 170 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "              0           1                   2                    3    \\\n",
       "0           genes     gene_id            baseMean       log2FoldChange   \n",
       "1            NAT1        NAT1   95.87059628671004  -0.4618119842755553   \n",
       "2            AAMP        AAMP   842.3716209229535  -0.1476064480241523   \n",
       "3           AARS1       AARS1  1374.1764868232967  -0.0252481964182913   \n",
       "4            ABAT        ABAT    55.9224048891824  -0.3585220629160938   \n",
       "...           ...         ...                 ...                  ...   \n",
       "18318  AC113208.3  AC113208.3                 NaN                  NaN   \n",
       "18319   105377586   105377586                 NaN                  NaN   \n",
       "18320   105378663   105378663                 NaN                  NaN   \n",
       "18321       FRGCA       FRGCA                 NaN                  NaN   \n",
       "18322    MIR34AHG    MIR34AHG                 NaN                  NaN   \n",
       "\n",
       "                      4                   5                   6    \\\n",
       "0                   lfcSE                stat              pvalue   \n",
       "1      0.1838993165621769  2.5112218626403395  0.0120314054862605   \n",
       "2      0.1180792991839148  1.2500620264882107  0.2112768900493981   \n",
       "3      0.1093729310308428  0.2308450196984433  0.8174352049833612   \n",
       "4      0.2117003248291207  1.6935357241680382  0.0903535409020921   \n",
       "...                   ...                 ...                 ...   \n",
       "18318                 NaN                 NaN                 NaN   \n",
       "18319                 NaN                 NaN                 NaN   \n",
       "18320                 NaN                 NaN                 NaN   \n",
       "18321                 NaN                 NaN                 NaN   \n",
       "18322                 NaN                 NaN                 NaN   \n",
       "\n",
       "                      7                   8                    9    ...  \\\n",
       "0                    padj            baseMean       log2FoldChange  ...   \n",
       "1      0.1764824098229201   5.033275352515851  -0.5602362002955328  ...   \n",
       "2       0.621101974228759  14.553492083441196   0.3101726260809158  ...   \n",
       "3       0.957161656507632  16.200285486653765  -0.1372280711065793  ...   \n",
       "4      0.4386939785389467   1.693745222699288  -0.9499849425240666  ...   \n",
       "...                   ...                 ...                  ...  ...   \n",
       "18318                 NaN                 NaN                  NaN  ...   \n",
       "18319                 NaN                 NaN                  NaN  ...   \n",
       "18320                 NaN                 NaN                  NaN  ...   \n",
       "18321                 NaN                 NaN                  NaN  ...   \n",
       "18322                 NaN                 NaN                  NaN  ...   \n",
       "\n",
       "                      160                  161                 162  \\\n",
       "0                   lfcSE                 stat              pvalue   \n",
       "1      0.1910164963313597    1.195738593227763  0.2317985857833831   \n",
       "2      0.0936551287545897  -0.7816068093858641  0.4344456838431942   \n",
       "3      0.0853883738066852  -0.0376794020496949  0.9699432991159118   \n",
       "4      0.1522534347786068   0.3937569945555876  0.6937604579667991   \n",
       "...                   ...                  ...                 ...   \n",
       "18318  0.8855085832558085   1.1782373841042888  0.2387019826835846   \n",
       "18319  0.9973246137985076   0.3124464112381569  0.7547012834221719   \n",
       "18320   0.859766767514565   -1.265830008396275  0.2055739490192727   \n",
       "18321                 NaN                  NaN                 0.0   \n",
       "18322   1.857862193029944    1.261161177260749   0.207250782833181   \n",
       "\n",
       "                      163                 164                  165  \\\n",
       "0                    padj            baseMean       log2FoldChange   \n",
       "1      0.5815447582945407   6.951305773266107   -0.303572495248922   \n",
       "2      0.7234391226069221   31.25961097658043   0.3155357818127773   \n",
       "3      0.9854767365932272    22.7732265306714   0.3114110726026999   \n",
       "4      0.8679025465000523   6.911045666718635  -0.1534162228659588   \n",
       "...                   ...                 ...                  ...   \n",
       "18318                 NaN                 0.0                 -0.0   \n",
       "18319                 NaN  0.5719749369309547   -1.908720323682767   \n",
       "18320                 NaN   0.783802604256384    1.878134129592003   \n",
       "18321                 NaN                 0.0                 -0.0   \n",
       "18322                 NaN                 0.0                 -0.0   \n",
       "\n",
       "                      166                  167                 168  \\\n",
       "0                   lfcSE                 stat              pvalue   \n",
       "1      0.6056459757244428   0.5012375338345212  0.6162039617032911   \n",
       "2      0.2868775385203951  -1.0998971318569954  0.2713769445263918   \n",
       "3      0.3184951940658864  -0.9777575247753314  0.3281942632603161   \n",
       "4      0.5532445763431577   0.2773027146149558  0.7815476753088093   \n",
       "...                   ...                  ...                 ...   \n",
       "18318                 NaN                  NaN                 0.0   \n",
       "18319  1.9634370487950688   0.9721321724341097  0.3309847985506584   \n",
       "18320  1.7378394499003909  -1.0807293675486844  0.2798175156218232   \n",
       "18321                 NaN                  NaN                 0.0   \n",
       "18322                 NaN                  NaN                 0.0   \n",
       "\n",
       "                      169  \n",
       "0                    padj  \n",
       "1                     NaN  \n",
       "2      0.5759060637707974  \n",
       "3                     NaN  \n",
       "4                     NaN  \n",
       "...                   ...  \n",
       "18318                 NaN  \n",
       "18319                 NaN  \n",
       "18320                 NaN  \n",
       "18321                 NaN  \n",
       "18322                 NaN  \n",
       "\n",
       "[18323 rows x 170 columns]"
      ]
     },
     "execution_count": 284,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_merged"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "00a1522c",
   "metadata": {},
   "source": [
    "__add info columns__"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "63f163e0",
   "metadata": {},
   "outputs": [],
   "source": [
    "# reset column index\n",
    "df_info_merged = df_info_merged.T.reset_index(drop=True).T"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 285,
   "id": "f317d76b",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "      <th>3</th>\n",
       "      <th>4</th>\n",
       "      <th>5</th>\n",
       "      <th>6</th>\n",
       "      <th>7</th>\n",
       "      <th>8</th>\n",
       "      <th>9</th>\n",
       "      <th>...</th>\n",
       "      <th>160</th>\n",
       "      <th>161</th>\n",
       "      <th>162</th>\n",
       "      <th>163</th>\n",
       "      <th>164</th>\n",
       "      <th>165</th>\n",
       "      <th>166</th>\n",
       "      <th>167</th>\n",
       "      <th>168</th>\n",
       "      <th>169</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>project</td>\n",
       "      <td>project</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>...</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>file</td>\n",
       "      <td>file</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>...</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_readcounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_readcounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_readcounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_readcounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>scale</td>\n",
       "      <td>scale</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>condition</td>\n",
       "      <td>condition</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>...</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>type</td>\n",
       "      <td>type</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>...</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows Ã— 170 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "         0          1                                                  2    \\\n",
       "0    project    project                                IRF2BP2_degraded_v1   \n",
       "1       file       file  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2      scale      scale                                     ERCCsamplewise   \n",
       "3  condition  condition                                                 3h   \n",
       "4       type       type                                         readcounts   \n",
       "\n",
       "                                                 3    \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                 4    \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                 5    \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                 6    \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                 7    \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                 8    \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                           tccounts   \n",
       "\n",
       "                                                 9    ...  \\\n",
       "0                                IRF2BP2_degraded_v1  ...   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...  ...   \n",
       "2                                     ERCCsamplewise  ...   \n",
       "3                                                 3h  ...   \n",
       "4                                           tccounts  ...   \n",
       "\n",
       "                                              160  \\\n",
       "0                                    4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX2_1h_0_5_deseq_readcounts.csv   \n",
       "2                                             NaN   \n",
       "3                                              1h   \n",
       "4                                      readcounts   \n",
       "\n",
       "                                              161  \\\n",
       "0                                    4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX2_1h_0_5_deseq_readcounts.csv   \n",
       "2                                             NaN   \n",
       "3                                              1h   \n",
       "4                                      readcounts   \n",
       "\n",
       "                                              162  \\\n",
       "0                                    4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX2_1h_0_5_deseq_readcounts.csv   \n",
       "2                                             NaN   \n",
       "3                                              1h   \n",
       "4                                      readcounts   \n",
       "\n",
       "                                              163  \\\n",
       "0                                    4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX2_1h_0_5_deseq_readcounts.csv   \n",
       "2                                             NaN   \n",
       "3                                              1h   \n",
       "4                                      readcounts   \n",
       "\n",
       "                                            164  \\\n",
       "0                                  4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv   \n",
       "2                                           NaN   \n",
       "3                                            1h   \n",
       "4                                      tccounts   \n",
       "\n",
       "                                            165  \\\n",
       "0                                  4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv   \n",
       "2                                           NaN   \n",
       "3                                            1h   \n",
       "4                                      tccounts   \n",
       "\n",
       "                                            166  \\\n",
       "0                                  4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv   \n",
       "2                                           NaN   \n",
       "3                                            1h   \n",
       "4                                      tccounts   \n",
       "\n",
       "                                            167  \\\n",
       "0                                  4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv   \n",
       "2                                           NaN   \n",
       "3                                            1h   \n",
       "4                                      tccounts   \n",
       "\n",
       "                                            168  \\\n",
       "0                                  4_degrons_v1   \n",
       "1  4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv   \n",
       "2                                           NaN   \n",
       "3                                            1h   \n",
       "4                                      tccounts   \n",
       "\n",
       "                                            169  \n",
       "0                                  4_degrons_v1  \n",
       "1  4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv  \n",
       "2                                           NaN  \n",
       "3                                            1h  \n",
       "4                                      tccounts  \n",
       "\n",
       "[5 rows x 170 columns]"
      ]
     },
     "execution_count": 285,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_info_merged.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 286,
   "id": "3385417a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 28 ms, sys: 0 ns, total: 28 ms\n",
      "Wall time: 28.9 ms\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "(18328, 170)"
      ]
     },
     "execution_count": 286,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "%%time\n",
    "\n",
    "# combine info and data\n",
    "df_save = df_info_merged.append(df_merged, ignore_index=True)\n",
    "df_save.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 287,
   "id": "e285a7b1",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "      <th>3</th>\n",
       "      <th>4</th>\n",
       "      <th>5</th>\n",
       "      <th>6</th>\n",
       "      <th>7</th>\n",
       "      <th>8</th>\n",
       "      <th>9</th>\n",
       "      <th>...</th>\n",
       "      <th>160</th>\n",
       "      <th>161</th>\n",
       "      <th>162</th>\n",
       "      <th>163</th>\n",
       "      <th>164</th>\n",
       "      <th>165</th>\n",
       "      <th>166</th>\n",
       "      <th>167</th>\n",
       "      <th>168</th>\n",
       "      <th>169</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>project</td>\n",
       "      <td>project</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>...</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "      <td>4_degrons_v1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>file</td>\n",
       "      <td>file</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>...</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_readcounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_readcounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_readcounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_readcounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv</td>\n",
       "      <td>4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>scale</td>\n",
       "      <td>scale</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>condition</td>\n",
       "      <td>condition</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>...</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>type</td>\n",
       "      <td>type</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>...</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18323</th>\n",
       "      <td>AC113208.3</td>\n",
       "      <td>AC113208.3</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>0.8855085832558085</td>\n",
       "      <td>1.1782373841042888</td>\n",
       "      <td>0.2387019826835846</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>-0.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18324</th>\n",
       "      <td>105377586</td>\n",
       "      <td>105377586</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>0.9973246137985076</td>\n",
       "      <td>0.3124464112381569</td>\n",
       "      <td>0.7547012834221719</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.5719749369309547</td>\n",
       "      <td>-1.908720323682767</td>\n",
       "      <td>1.9634370487950688</td>\n",
       "      <td>0.9721321724341097</td>\n",
       "      <td>0.3309847985506584</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18325</th>\n",
       "      <td>105378663</td>\n",
       "      <td>105378663</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>0.859766767514565</td>\n",
       "      <td>-1.265830008396275</td>\n",
       "      <td>0.2055739490192727</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.783802604256384</td>\n",
       "      <td>1.878134129592003</td>\n",
       "      <td>1.7378394499003909</td>\n",
       "      <td>-1.0807293675486844</td>\n",
       "      <td>0.2798175156218232</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18326</th>\n",
       "      <td>FRGCA</td>\n",
       "      <td>FRGCA</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>-0.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18327</th>\n",
       "      <td>MIR34AHG</td>\n",
       "      <td>MIR34AHG</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>1.857862193029944</td>\n",
       "      <td>1.261161177260749</td>\n",
       "      <td>0.207250782833181</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>-0.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>18328 rows Ã— 170 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "              0           1    \\\n",
       "0         project     project   \n",
       "1            file        file   \n",
       "2           scale       scale   \n",
       "3       condition   condition   \n",
       "4            type        type   \n",
       "...           ...         ...   \n",
       "18323  AC113208.3  AC113208.3   \n",
       "18324   105377586   105377586   \n",
       "18325   105378663   105378663   \n",
       "18326       FRGCA       FRGCA   \n",
       "18327    MIR34AHG    MIR34AHG   \n",
       "\n",
       "                                                     2    \\\n",
       "0                                    IRF2BP2_degraded_v1   \n",
       "1      IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                         ERCCsamplewise   \n",
       "3                                                     3h   \n",
       "4                                             readcounts   \n",
       "...                                                  ...   \n",
       "18323                                                NaN   \n",
       "18324                                                NaN   \n",
       "18325                                                NaN   \n",
       "18326                                                NaN   \n",
       "18327                                                NaN   \n",
       "\n",
       "                                                     3    \\\n",
       "0                                    IRF2BP2_degraded_v1   \n",
       "1      IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                         ERCCsamplewise   \n",
       "3                                                     3h   \n",
       "4                                             readcounts   \n",
       "...                                                  ...   \n",
       "18323                                                NaN   \n",
       "18324                                                NaN   \n",
       "18325                                                NaN   \n",
       "18326                                                NaN   \n",
       "18327                                                NaN   \n",
       "\n",
       "                                                     4    \\\n",
       "0                                    IRF2BP2_degraded_v1   \n",
       "1      IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                         ERCCsamplewise   \n",
       "3                                                     3h   \n",
       "4                                             readcounts   \n",
       "...                                                  ...   \n",
       "18323                                                NaN   \n",
       "18324                                                NaN   \n",
       "18325                                                NaN   \n",
       "18326                                                NaN   \n",
       "18327                                                NaN   \n",
       "\n",
       "                                                     5    \\\n",
       "0                                    IRF2BP2_degraded_v1   \n",
       "1      IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                         ERCCsamplewise   \n",
       "3                                                     3h   \n",
       "4                                             readcounts   \n",
       "...                                                  ...   \n",
       "18323                                                NaN   \n",
       "18324                                                NaN   \n",
       "18325                                                NaN   \n",
       "18326                                                NaN   \n",
       "18327                                                NaN   \n",
       "\n",
       "                                                     6    \\\n",
       "0                                    IRF2BP2_degraded_v1   \n",
       "1      IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                         ERCCsamplewise   \n",
       "3                                                     3h   \n",
       "4                                             readcounts   \n",
       "...                                                  ...   \n",
       "18323                                                NaN   \n",
       "18324                                                NaN   \n",
       "18325                                                NaN   \n",
       "18326                                                NaN   \n",
       "18327                                                NaN   \n",
       "\n",
       "                                                     7    \\\n",
       "0                                    IRF2BP2_degraded_v1   \n",
       "1      IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                         ERCCsamplewise   \n",
       "3                                                     3h   \n",
       "4                                             readcounts   \n",
       "...                                                  ...   \n",
       "18323                                                NaN   \n",
       "18324                                                NaN   \n",
       "18325                                                NaN   \n",
       "18326                                                NaN   \n",
       "18327                                                NaN   \n",
       "\n",
       "                                                     8    \\\n",
       "0                                    IRF2BP2_degraded_v1   \n",
       "1      IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                         ERCCsamplewise   \n",
       "3                                                     3h   \n",
       "4                                               tccounts   \n",
       "...                                                  ...   \n",
       "18323                                                NaN   \n",
       "18324                                                NaN   \n",
       "18325                                                NaN   \n",
       "18326                                                NaN   \n",
       "18327                                                NaN   \n",
       "\n",
       "                                                     9    ...  \\\n",
       "0                                    IRF2BP2_degraded_v1  ...   \n",
       "1      IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...  ...   \n",
       "2                                         ERCCsamplewise  ...   \n",
       "3                                                     3h  ...   \n",
       "4                                               tccounts  ...   \n",
       "...                                                  ...  ...   \n",
       "18323                                                NaN  ...   \n",
       "18324                                                NaN  ...   \n",
       "18325                                                NaN  ...   \n",
       "18326                                                NaN  ...   \n",
       "18327                                                NaN  ...   \n",
       "\n",
       "                                                  160  \\\n",
       "0                                        4_degrons_v1   \n",
       "1      4_degrons_v1_RUNX2_1h_0_5_deseq_readcounts.csv   \n",
       "2                                                 NaN   \n",
       "3                                                  1h   \n",
       "4                                          readcounts   \n",
       "...                                               ...   \n",
       "18323                              0.8855085832558085   \n",
       "18324                              0.9973246137985076   \n",
       "18325                               0.859766767514565   \n",
       "18326                                             NaN   \n",
       "18327                               1.857862193029944   \n",
       "\n",
       "                                                  161  \\\n",
       "0                                        4_degrons_v1   \n",
       "1      4_degrons_v1_RUNX2_1h_0_5_deseq_readcounts.csv   \n",
       "2                                                 NaN   \n",
       "3                                                  1h   \n",
       "4                                          readcounts   \n",
       "...                                               ...   \n",
       "18323                              1.1782373841042888   \n",
       "18324                              0.3124464112381569   \n",
       "18325                              -1.265830008396275   \n",
       "18326                                             NaN   \n",
       "18327                               1.261161177260749   \n",
       "\n",
       "                                                  162  \\\n",
       "0                                        4_degrons_v1   \n",
       "1      4_degrons_v1_RUNX2_1h_0_5_deseq_readcounts.csv   \n",
       "2                                                 NaN   \n",
       "3                                                  1h   \n",
       "4                                          readcounts   \n",
       "...                                               ...   \n",
       "18323                              0.2387019826835846   \n",
       "18324                              0.7547012834221719   \n",
       "18325                              0.2055739490192727   \n",
       "18326                                             0.0   \n",
       "18327                               0.207250782833181   \n",
       "\n",
       "                                                  163  \\\n",
       "0                                        4_degrons_v1   \n",
       "1      4_degrons_v1_RUNX2_1h_0_5_deseq_readcounts.csv   \n",
       "2                                                 NaN   \n",
       "3                                                  1h   \n",
       "4                                          readcounts   \n",
       "...                                               ...   \n",
       "18323                                             NaN   \n",
       "18324                                             NaN   \n",
       "18325                                             NaN   \n",
       "18326                                             NaN   \n",
       "18327                                             NaN   \n",
       "\n",
       "                                                164  \\\n",
       "0                                      4_degrons_v1   \n",
       "1      4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv   \n",
       "2                                               NaN   \n",
       "3                                                1h   \n",
       "4                                          tccounts   \n",
       "...                                             ...   \n",
       "18323                                           0.0   \n",
       "18324                            0.5719749369309547   \n",
       "18325                             0.783802604256384   \n",
       "18326                                           0.0   \n",
       "18327                                           0.0   \n",
       "\n",
       "                                                165  \\\n",
       "0                                      4_degrons_v1   \n",
       "1      4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv   \n",
       "2                                               NaN   \n",
       "3                                                1h   \n",
       "4                                          tccounts   \n",
       "...                                             ...   \n",
       "18323                                          -0.0   \n",
       "18324                            -1.908720323682767   \n",
       "18325                             1.878134129592003   \n",
       "18326                                          -0.0   \n",
       "18327                                          -0.0   \n",
       "\n",
       "                                                166  \\\n",
       "0                                      4_degrons_v1   \n",
       "1      4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv   \n",
       "2                                               NaN   \n",
       "3                                                1h   \n",
       "4                                          tccounts   \n",
       "...                                             ...   \n",
       "18323                                           NaN   \n",
       "18324                            1.9634370487950688   \n",
       "18325                            1.7378394499003909   \n",
       "18326                                           NaN   \n",
       "18327                                           NaN   \n",
       "\n",
       "                                                167  \\\n",
       "0                                      4_degrons_v1   \n",
       "1      4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv   \n",
       "2                                               NaN   \n",
       "3                                                1h   \n",
       "4                                          tccounts   \n",
       "...                                             ...   \n",
       "18323                                           NaN   \n",
       "18324                            0.9721321724341097   \n",
       "18325                           -1.0807293675486844   \n",
       "18326                                           NaN   \n",
       "18327                                           NaN   \n",
       "\n",
       "                                                168  \\\n",
       "0                                      4_degrons_v1   \n",
       "1      4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv   \n",
       "2                                               NaN   \n",
       "3                                                1h   \n",
       "4                                          tccounts   \n",
       "...                                             ...   \n",
       "18323                                           0.0   \n",
       "18324                            0.3309847985506584   \n",
       "18325                            0.2798175156218232   \n",
       "18326                                           0.0   \n",
       "18327                                           0.0   \n",
       "\n",
       "                                                169  \n",
       "0                                      4_degrons_v1  \n",
       "1      4_degrons_v1_RUNX2_1h_0_5_deseq_tccounts.csv  \n",
       "2                                               NaN  \n",
       "3                                                1h  \n",
       "4                                          tccounts  \n",
       "...                                             ...  \n",
       "18323                                           NaN  \n",
       "18324                                           NaN  \n",
       "18325                                           NaN  \n",
       "18326                                           NaN  \n",
       "18327                                           NaN  \n",
       "\n",
       "[18328 rows x 170 columns]"
      ]
     },
     "execution_count": 287,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_save"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "697e139b",
   "metadata": {},
   "source": [
    "__save composite merged file__"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 288,
   "id": "f9cee3c3",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved DESeq results to: ../results/slamseq_merged_deseq_output_noAlt.csv\n"
     ]
    }
   ],
   "source": [
    "# save file\n",
    "res_dir = \"../results\"\n",
    "deseq_res_file = \"slamseq_merged_deseq_output_noAlt.csv\"\n",
    "\n",
    "df_save.to_csv(\"{}/{}\".format(res_dir, deseq_res_file), header=False, index=False)\n",
    "\n",
    "print(\"Saved DESeq results to: {}/{}\".format(res_dir, deseq_res_file))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4523aa47",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "id": "796d2f53",
   "metadata": {},
   "source": [
    "# Looking into data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 258,
   "id": "79dfe9c6",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 258,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#df_MYB = \n",
    "df_save.loc[:,df_save list(df_save.iloc[0])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 260,
   "id": "e892353d",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['project',\n",
       " 'project',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'IRF2BP2_degraded_v1',\n",
       " 'MYB_degraded_v1',\n",
       " 'MYB_degraded_v1',\n",
       " 'MYB_degraded_v1',\n",
       " 'MYB_degraded_v1',\n",
       " 'MYB_degraded_v1',\n",
       " 'MYB_degraded_v1',\n",
       " 'MYB_degraded_v1',\n",
       " 'MYB_degraded_v1',\n",
       " 'MYB_degraded_v1',\n",
       " 'MYB_degraded_v1',\n",
       " 'MYB_degraded_v1',\n",
       " 'MYB_degraded_v1',\n",
       " 'MYB_degraded_v1',\n",
       " 'MYB_degraded_v1',\n",
       " 'MYB_degraded_v1',\n",
       " 'MYB_degraded_v1',\n",
       " 'MYB_degraded_v1',\n",
       " 'MYB_degraded_v1',\n",
       " 'MYB_degraded_v1',\n",
       " 'MYB_degraded_v1',\n",
       " 'MYB_degraded_v1',\n",
       " 'MYB_degraded_v1',\n",
       " 'MYB_degraded_v1',\n",
       " 'MYB_degraded_v1']"
      ]
     },
     "execution_count": 260,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "list(df_save.iloc[0])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 256,
   "id": "a658a252",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'MYB_degraded_v1'"
      ]
     },
     "execution_count": 256,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "project2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "ac30cc2c",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "id": "980be5dd",
   "metadata": {
    "heading_collapsed": true
   },
   "source": [
    "## Testing"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "29558fde",
   "metadata": {
    "collapsed": true,
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../results/slamseq_IRF2BP2_degraded_v1/IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts.csv\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>baseMean</th>\n",
       "      <th>log2FoldChange</th>\n",
       "      <th>lfcSE</th>\n",
       "      <th>stat</th>\n",
       "      <th>pvalue</th>\n",
       "      <th>padj</th>\n",
       "      <th>genes</th>\n",
       "      <th>gene_id</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>98.734740</td>\n",
       "      <td>-0.358870</td>\n",
       "      <td>0.179587</td>\n",
       "      <td>1.998309</td>\n",
       "      <td>0.045683</td>\n",
       "      <td>0.557313</td>\n",
       "      <td>NAT1</td>\n",
       "      <td>NAT1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>872.500027</td>\n",
       "      <td>-0.041895</td>\n",
       "      <td>0.116618</td>\n",
       "      <td>0.359250</td>\n",
       "      <td>0.719408</td>\n",
       "      <td>0.968449</td>\n",
       "      <td>AAMP</td>\n",
       "      <td>AAMP</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "     baseMean  log2FoldChange     lfcSE      stat    pvalue      padj genes  \\\n",
       "0   98.734740       -0.358870  0.179587  1.998309  0.045683  0.557313  NAT1   \n",
       "1  872.500027       -0.041895  0.116618  0.359250  0.719408  0.968449  AAMP   \n",
       "\n",
       "  gene_id  \n",
       "0    NAT1  \n",
       "1    AAMP  "
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# read in deseq table\n",
    "print(files[0])\n",
    "df1 = pd.read_csv(files[0], header=[0], index_col=None, squeeze=True)\n",
    "df1.iloc[:2]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "089181ff",
   "metadata": {
    "collapsed": true,
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(15112, 8)"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df1.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "9be1f277",
   "metadata": {
    "collapsed": true,
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "15112\n",
      "15112\n",
      "0\n",
      "0\n"
     ]
    }
   ],
   "source": [
    "print(len(df1.genes.unique()))\n",
    "print(len(df1.genes.drop_duplicates(keep=False)))\n",
    "print(df1.shape[0] - len(df1.genes.drop_duplicates(keep=False)))\n",
    "print(len(df1.genes.unique()) - len(df1.genes.drop_duplicates(keep=False)))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "5f587b24",
   "metadata": {
    "collapsed": true,
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['baseMean_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts',\n",
       " 'log2FoldChange_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts',\n",
       " 'lfcSE_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts',\n",
       " 'stat_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts',\n",
       " 'pvalue_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts',\n",
       " 'padj_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts',\n",
       " 'genes',\n",
       " 'gene_id']"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "[\"{}_{}\".format(col_name, os.path.basename(files[0]).split(\".\")[0]) if \"gene\" not in col_name \\\n",
    " else col_name for col_name in df1.columns.tolist()]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "0a1354e8",
   "metadata": {
    "collapsed": true,
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "../results/slamseq_IRF2BP2_degraded_v1/IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_tccounts.csv\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>baseMean</th>\n",
       "      <th>log2FoldChange</th>\n",
       "      <th>lfcSE</th>\n",
       "      <th>stat</th>\n",
       "      <th>pvalue</th>\n",
       "      <th>padj</th>\n",
       "      <th>genes</th>\n",
       "      <th>gene_id</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>5.021740</td>\n",
       "      <td>-0.554785</td>\n",
       "      <td>0.739133</td>\n",
       "      <td>0.750588</td>\n",
       "      <td>0.45290</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NAT1</td>\n",
       "      <td>NAT1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>14.549761</td>\n",
       "      <td>0.316474</td>\n",
       "      <td>0.385427</td>\n",
       "      <td>-0.821099</td>\n",
       "      <td>0.41159</td>\n",
       "      <td>0.642625</td>\n",
       "      <td>AAMP</td>\n",
       "      <td>AAMP</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    baseMean  log2FoldChange     lfcSE      stat   pvalue      padj genes  \\\n",
       "0   5.021740       -0.554785  0.739133  0.750588  0.45290       NaN  NAT1   \n",
       "1  14.549761        0.316474  0.385427 -0.821099  0.41159  0.642625  AAMP   \n",
       "\n",
       "  gene_id  \n",
       "0    NAT1  \n",
       "1    AAMP  "
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# read in deseq table\n",
    "print(files[1])\n",
    "df2 = pd.read_csv(files[1], header=[0], index_col=None, squeeze=True)\n",
    "df2.iloc[:2]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "a7b1412e",
   "metadata": {
    "collapsed": true,
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>baseMean_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts</th>\n",
       "      <th>log2FoldChange_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts</th>\n",
       "      <th>lfcSE_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts</th>\n",
       "      <th>stat_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts</th>\n",
       "      <th>pvalue_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts</th>\n",
       "      <th>padj_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts</th>\n",
       "      <th>genes</th>\n",
       "      <th>gene_id</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>98.73474</td>\n",
       "      <td>-0.35887</td>\n",
       "      <td>0.179587</td>\n",
       "      <td>1.998309</td>\n",
       "      <td>0.045683</td>\n",
       "      <td>0.557313</td>\n",
       "      <td>NAT1</td>\n",
       "      <td>NAT1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   baseMean_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts  \\\n",
       "0                                           98.73474                     \n",
       "\n",
       "   log2FoldChange_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts  \\\n",
       "0                                           -0.35887                           \n",
       "\n",
       "   lfcSE_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts  \\\n",
       "0                                           0.179587                  \n",
       "\n",
       "   stat_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts  \\\n",
       "0                                           1.998309                 \n",
       "\n",
       "   pvalue_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts  \\\n",
       "0                                           0.045683                   \n",
       "\n",
       "   padj_IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts genes  \\\n",
       "0                                           0.557313                NAT1   \n",
       "\n",
       "  gene_id  \n",
       "0    NAT1  "
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#deseq_dfs = map(files, get_deseq_data)\n",
    "deseq_dfs = [get_deseq_data(file) for file in files[:3]]\n",
    "deseq_dfs[0].iloc[:1]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "777ce30a",
   "metadata": {
    "collapsed": true,
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>genes</th>\n",
       "      <th>gene_id</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NAT1</td>\n",
       "      <td>NAT1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>AAMP</td>\n",
       "      <td>AAMP</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>AARS1</td>\n",
       "      <td>AARS1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>ABAT</td>\n",
       "      <td>ABAT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>ABCA1</td>\n",
       "      <td>ABCA1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   genes gene_id\n",
       "0   NAT1    NAT1\n",
       "1   AAMP    AAMP\n",
       "2  AARS1   AARS1\n",
       "3   ABAT    ABAT\n",
       "4  ABCA1   ABCA1"
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# main df\n",
    "df_merged = pd.DataFrame()\n",
    "df_merged = df_merged.append(deseq_dfs[0][[\"genes\", \"gene_id\"]]) # add gene cols to main df\n",
    "df_merged.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "id": "273a8e4b",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(15112, 20)"
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# join df columns on gene & gene_id columns\n",
    "df_merged = reduce(lambda left, right: pd.merge(left, right, on=['genes', \"gene_id\"], how=\"outer\"), \n",
    "                   [df_merged] + deseq_dfs)\n",
    "df_merged.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "id": "8f9f8061",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "# change column names\n",
    "df_merged.columns = [col_name.split(\"_\")[0] if \"gene\" not in col_name else \n",
    "                     col_name for col_name in df_merged.columns.tolist()]\n",
    "df_merged = df_merged.T.reset_index().T.reset_index(drop=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "a3b61d7a",
   "metadata": {
    "collapsed": true,
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "      <th>3</th>\n",
       "      <th>4</th>\n",
       "      <th>5</th>\n",
       "      <th>6</th>\n",
       "      <th>7</th>\n",
       "      <th>8</th>\n",
       "      <th>9</th>\n",
       "      <th>10</th>\n",
       "      <th>11</th>\n",
       "      <th>12</th>\n",
       "      <th>13</th>\n",
       "      <th>14</th>\n",
       "      <th>15</th>\n",
       "      <th>16</th>\n",
       "      <th>17</th>\n",
       "      <th>18</th>\n",
       "      <th>19</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>genes</td>\n",
       "      <td>gene_id</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>NAT1</td>\n",
       "      <td>NAT1</td>\n",
       "      <td>98.73474</td>\n",
       "      <td>-0.35887</td>\n",
       "      <td>0.179587</td>\n",
       "      <td>1.998309</td>\n",
       "      <td>0.045683</td>\n",
       "      <td>0.557313</td>\n",
       "      <td>5.02174</td>\n",
       "      <td>-0.554785</td>\n",
       "      <td>0.739133</td>\n",
       "      <td>0.750588</td>\n",
       "      <td>0.4529</td>\n",
       "      <td>NaN</td>\n",
       "      <td>105.179117</td>\n",
       "      <td>-0.435531</td>\n",
       "      <td>0.261972</td>\n",
       "      <td>1.662513</td>\n",
       "      <td>0.09641</td>\n",
       "      <td>0.616449</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>AAMP</td>\n",
       "      <td>AAMP</td>\n",
       "      <td>872.500027</td>\n",
       "      <td>-0.041895</td>\n",
       "      <td>0.116618</td>\n",
       "      <td>0.35925</td>\n",
       "      <td>0.719408</td>\n",
       "      <td>0.968449</td>\n",
       "      <td>14.549761</td>\n",
       "      <td>0.316474</td>\n",
       "      <td>0.385427</td>\n",
       "      <td>-0.821099</td>\n",
       "      <td>0.41159</td>\n",
       "      <td>0.642625</td>\n",
       "      <td>971.817384</td>\n",
       "      <td>0.058489</td>\n",
       "      <td>0.136273</td>\n",
       "      <td>-0.429207</td>\n",
       "      <td>0.667772</td>\n",
       "      <td>0.989724</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>AARS1</td>\n",
       "      <td>AARS1</td>\n",
       "      <td>1426.956869</td>\n",
       "      <td>0.083048</td>\n",
       "      <td>0.107111</td>\n",
       "      <td>-0.775339</td>\n",
       "      <td>0.438139</td>\n",
       "      <td>0.906009</td>\n",
       "      <td>16.042542</td>\n",
       "      <td>-0.109203</td>\n",
       "      <td>0.395866</td>\n",
       "      <td>0.275859</td>\n",
       "      <td>0.782656</td>\n",
       "      <td>0.886222</td>\n",
       "      <td>890.019221</td>\n",
       "      <td>-0.00236</td>\n",
       "      <td>0.161838</td>\n",
       "      <td>0.014582</td>\n",
       "      <td>0.988365</td>\n",
       "      <td>0.999179</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>ABAT</td>\n",
       "      <td>ABAT</td>\n",
       "      <td>57.515186</td>\n",
       "      <td>-0.258913</td>\n",
       "      <td>0.20369</td>\n",
       "      <td>1.271113</td>\n",
       "      <td>0.203688</td>\n",
       "      <td>0.808742</td>\n",
       "      <td>1.695255</td>\n",
       "      <td>-0.946442</td>\n",
       "      <td>1.260847</td>\n",
       "      <td>0.75064</td>\n",
       "      <td>0.452869</td>\n",
       "      <td>NaN</td>\n",
       "      <td>42.89278</td>\n",
       "      <td>-0.2227</td>\n",
       "      <td>0.265193</td>\n",
       "      <td>0.839767</td>\n",
       "      <td>0.401039</td>\n",
       "      <td>0.944698</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "      0        1            2               3         4         5         6   \\\n",
       "0  genes  gene_id     baseMean  log2FoldChange     lfcSE      stat    pvalue   \n",
       "1   NAT1     NAT1     98.73474        -0.35887  0.179587  1.998309  0.045683   \n",
       "2   AAMP     AAMP   872.500027       -0.041895  0.116618   0.35925  0.719408   \n",
       "3  AARS1    AARS1  1426.956869        0.083048  0.107111 -0.775339  0.438139   \n",
       "4   ABAT     ABAT    57.515186       -0.258913   0.20369  1.271113  0.203688   \n",
       "\n",
       "         7          8               9         10        11        12  \\\n",
       "0      padj   baseMean  log2FoldChange     lfcSE      stat    pvalue   \n",
       "1  0.557313    5.02174       -0.554785  0.739133  0.750588    0.4529   \n",
       "2  0.968449  14.549761        0.316474  0.385427 -0.821099   0.41159   \n",
       "3  0.906009  16.042542       -0.109203  0.395866  0.275859  0.782656   \n",
       "4  0.808742   1.695255       -0.946442  1.260847   0.75064  0.452869   \n",
       "\n",
       "         13          14              15        16        17        18  \\\n",
       "0      padj    baseMean  log2FoldChange     lfcSE      stat    pvalue   \n",
       "1       NaN  105.179117       -0.435531  0.261972  1.662513   0.09641   \n",
       "2  0.642625  971.817384        0.058489  0.136273 -0.429207  0.667772   \n",
       "3  0.886222  890.019221        -0.00236  0.161838  0.014582  0.988365   \n",
       "4       NaN    42.89278         -0.2227  0.265193  0.839767  0.401039   \n",
       "\n",
       "         19  \n",
       "0      padj  \n",
       "1  0.616449  \n",
       "2  0.989724  \n",
       "3  0.999179  \n",
       "4  0.944698  "
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_merged.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "cb962cff",
   "metadata": {
    "collapsed": true,
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "      <th>3</th>\n",
       "      <th>4</th>\n",
       "      <th>5</th>\n",
       "      <th>6</th>\n",
       "      <th>7</th>\n",
       "      <th>8</th>\n",
       "      <th>9</th>\n",
       "      <th>10</th>\n",
       "      <th>11</th>\n",
       "      <th>12</th>\n",
       "      <th>13</th>\n",
       "      <th>14</th>\n",
       "      <th>15</th>\n",
       "      <th>16</th>\n",
       "      <th>17</th>\n",
       "      <th>18</th>\n",
       "      <th>19</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>genes</td>\n",
       "      <td>gene_id</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>NAT1</td>\n",
       "      <td>NAT1</td>\n",
       "      <td>95.678631</td>\n",
       "      <td>-0.463807</td>\n",
       "      <td>0.183351</td>\n",
       "      <td>2.529615</td>\n",
       "      <td>0.011419</td>\n",
       "      <td>0.162526</td>\n",
       "      <td>5.023216</td>\n",
       "      <td>-0.554238</td>\n",
       "      <td>0.738925</td>\n",
       "      <td>0.75006</td>\n",
       "      <td>0.453218</td>\n",
       "      <td>NaN</td>\n",
       "      <td>105.239049</td>\n",
       "      <td>-0.433213</td>\n",
       "      <td>0.261849</td>\n",
       "      <td>1.65444</td>\n",
       "      <td>0.098038</td>\n",
       "      <td>0.646734</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>AAMP</td>\n",
       "      <td>AAMP</td>\n",
       "      <td>841.211188</td>\n",
       "      <td>-0.150545</td>\n",
       "      <td>0.117138</td>\n",
       "      <td>1.285188</td>\n",
       "      <td>0.198727</td>\n",
       "      <td>0.58154</td>\n",
       "      <td>14.561153</td>\n",
       "      <td>0.317738</td>\n",
       "      <td>0.385701</td>\n",
       "      <td>-0.823793</td>\n",
       "      <td>0.410057</td>\n",
       "      <td>0.647203</td>\n",
       "      <td>972.517055</td>\n",
       "      <td>0.060776</td>\n",
       "      <td>0.136377</td>\n",
       "      <td>-0.44565</td>\n",
       "      <td>0.65585</td>\n",
       "      <td>0.995046</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>AARS1</td>\n",
       "      <td>AARS1</td>\n",
       "      <td>1373.801586</td>\n",
       "      <td>-0.025304</td>\n",
       "      <td>0.10876</td>\n",
       "      <td>0.232656</td>\n",
       "      <td>0.816029</td>\n",
       "      <td>0.954353</td>\n",
       "      <td>16.055735</td>\n",
       "      <td>-0.107774</td>\n",
       "      <td>0.395963</td>\n",
       "      <td>0.272182</td>\n",
       "      <td>0.785482</td>\n",
       "      <td>0.888308</td>\n",
       "      <td>890.639308</td>\n",
       "      <td>-0.00008</td>\n",
       "      <td>0.161873</td>\n",
       "      <td>0.000495</td>\n",
       "      <td>0.999605</td>\n",
       "      <td>0.99988</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>ABAT</td>\n",
       "      <td>ABAT</td>\n",
       "      <td>55.753788</td>\n",
       "      <td>-0.362282</td>\n",
       "      <td>0.21001</td>\n",
       "      <td>1.725072</td>\n",
       "      <td>0.084514</td>\n",
       "      <td>0.404263</td>\n",
       "      <td>1.69655</td>\n",
       "      <td>-0.944984</td>\n",
       "      <td>1.260997</td>\n",
       "      <td>0.749394</td>\n",
       "      <td>0.45362</td>\n",
       "      <td>NaN</td>\n",
       "      <td>42.920747</td>\n",
       "      <td>-0.220415</td>\n",
       "      <td>0.265135</td>\n",
       "      <td>0.831329</td>\n",
       "      <td>0.405788</td>\n",
       "      <td>0.961633</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "      0        1            2               3         4         5         6   \\\n",
       "0  genes  gene_id     baseMean  log2FoldChange     lfcSE      stat    pvalue   \n",
       "1   NAT1     NAT1    95.678631       -0.463807  0.183351  2.529615  0.011419   \n",
       "2   AAMP     AAMP   841.211188       -0.150545  0.117138  1.285188  0.198727   \n",
       "3  AARS1    AARS1  1373.801586       -0.025304   0.10876  0.232656  0.816029   \n",
       "4   ABAT     ABAT    55.753788       -0.362282   0.21001  1.725072  0.084514   \n",
       "\n",
       "         7          8               9         10        11        12  \\\n",
       "0      padj   baseMean  log2FoldChange     lfcSE      stat    pvalue   \n",
       "1  0.162526   5.023216       -0.554238  0.738925   0.75006  0.453218   \n",
       "2   0.58154  14.561153        0.317738  0.385701 -0.823793  0.410057   \n",
       "3  0.954353  16.055735       -0.107774  0.395963  0.272182  0.785482   \n",
       "4  0.404263    1.69655       -0.944984  1.260997  0.749394   0.45362   \n",
       "\n",
       "         13          14              15        16        17        18  \\\n",
       "0      padj    baseMean  log2FoldChange     lfcSE      stat    pvalue   \n",
       "1       NaN  105.239049       -0.433213  0.261849   1.65444  0.098038   \n",
       "2  0.647203  972.517055        0.060776  0.136377  -0.44565   0.65585   \n",
       "3  0.888308  890.639308        -0.00008  0.161873  0.000495  0.999605   \n",
       "4       NaN   42.920747       -0.220415  0.265135  0.831329  0.405788   \n",
       "\n",
       "         19  \n",
       "0      padj  \n",
       "1  0.646734  \n",
       "2  0.995046  \n",
       "3   0.99988  \n",
       "4  0.961633  "
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_merged.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "9c16079d",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "id": "dd7e6f15",
   "metadata": {
    "hidden": true
   },
   "source": [
    "which genes are duplicated"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "45f7f93d",
   "metadata": {
    "hidden": true
   },
   "source": [
    "why are the values so different for some unaffected genes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "id": "ff7d7259",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "test_counts = \"../results/slamseq_IRF2BP2_degraded_v1_dup/IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_deseq_readcounts.csv\"\n",
    "df_test = pd.read_csv(test_counts, header=[0], index_col=None, squeeze=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "id": "7213ba57",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "duplicate_genes = list(set(df_test.genes.unique()) - set(df_test.genes.drop_duplicates(keep=False)))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "id": "46ea03d2",
   "metadata": {
    "collapsed": true,
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "27\n",
      "(54, 8)\n"
     ]
    }
   ],
   "source": [
    "print(len(duplicate_genes))\n",
    "df_dup = df_test.loc[df_test['genes'].isin(duplicate_genes)]\n",
    "print(df_dup.shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "id": "2f029b20",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "df_dedup = df1.loc[df1['genes'].isin(duplicate_genes)]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "id": "a806b4c5",
   "metadata": {
    "collapsed": true,
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>baseMean</th>\n",
       "      <th>log2FoldChange</th>\n",
       "      <th>lfcSE</th>\n",
       "      <th>stat</th>\n",
       "      <th>pvalue</th>\n",
       "      <th>padj</th>\n",
       "      <th>genes</th>\n",
       "      <th>gene_id</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>534</th>\n",
       "      <td>8319.176123</td>\n",
       "      <td>-0.093807</td>\n",
       "      <td>0.119186</td>\n",
       "      <td>0.787069</td>\n",
       "      <td>0.431242</td>\n",
       "      <td>0.778920</td>\n",
       "      <td>CHML</td>\n",
       "      <td>CHML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6018</th>\n",
       "      <td>3965.963276</td>\n",
       "      <td>0.065072</td>\n",
       "      <td>0.131960</td>\n",
       "      <td>-0.493116</td>\n",
       "      <td>0.621930</td>\n",
       "      <td>0.878732</td>\n",
       "      <td>CHML</td>\n",
       "      <td>CHML</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "         baseMean  log2FoldChange     lfcSE      stat    pvalue      padj  \\\n",
       "534   8319.176123       -0.093807  0.119186  0.787069  0.431242  0.778920   \n",
       "6018  3965.963276        0.065072  0.131960 -0.493116  0.621930  0.878732   \n",
       "\n",
       "     genes gene_id  \n",
       "534   CHML    CHML  \n",
       "6018  CHML    CHML  "
      ]
     },
     "execution_count": 53,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_dup.loc[df_dup['genes'] == \"CHML\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "id": "ac271204",
   "metadata": {
    "collapsed": true,
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>baseMean</th>\n",
       "      <th>log2FoldChange</th>\n",
       "      <th>lfcSE</th>\n",
       "      <th>stat</th>\n",
       "      <th>pvalue</th>\n",
       "      <th>padj</th>\n",
       "      <th>genes</th>\n",
       "      <th>gene_id</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>534</th>\n",
       "      <td>12746.248337</td>\n",
       "      <td>0.063397</td>\n",
       "      <td>0.114081</td>\n",
       "      <td>-0.555717</td>\n",
       "      <td>0.578405</td>\n",
       "      <td>0.941269</td>\n",
       "      <td>CHML</td>\n",
       "      <td>CHML</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "         baseMean  log2FoldChange     lfcSE      stat    pvalue      padj  \\\n",
       "534  12746.248337        0.063397  0.114081 -0.555717  0.578405  0.941269   \n",
       "\n",
       "    genes gene_id  \n",
       "534  CHML    CHML  "
      ]
     },
     "execution_count": 54,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_dedup.loc[df_dedup['genes'] == \"CHML\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "id": "ec2f8a96",
   "metadata": {
    "collapsed": true,
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>baseMean</th>\n",
       "      <th>log2FoldChange</th>\n",
       "      <th>lfcSE</th>\n",
       "      <th>stat</th>\n",
       "      <th>pvalue</th>\n",
       "      <th>padj</th>\n",
       "      <th>genes</th>\n",
       "      <th>gene_id</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>1790</th>\n",
       "      <td>727.339448</td>\n",
       "      <td>-0.114417</td>\n",
       "      <td>0.113961</td>\n",
       "      <td>1.004001</td>\n",
       "      <td>3.153782e-01</td>\n",
       "      <td>6.937536e-01</td>\n",
       "      <td>MEIS1</td>\n",
       "      <td>MEIS1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1902</th>\n",
       "      <td>3919.460999</td>\n",
       "      <td>-0.100624</td>\n",
       "      <td>0.127026</td>\n",
       "      <td>0.792155</td>\n",
       "      <td>4.282701e-01</td>\n",
       "      <td>7.769807e-01</td>\n",
       "      <td>MYB</td>\n",
       "      <td>MYB</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1906</th>\n",
       "      <td>1757.343901</td>\n",
       "      <td>-0.127836</td>\n",
       "      <td>0.121284</td>\n",
       "      <td>1.054029</td>\n",
       "      <td>2.918698e-01</td>\n",
       "      <td>6.784911e-01</td>\n",
       "      <td>MYC</td>\n",
       "      <td>MYC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12650</th>\n",
       "      <td>15280.599449</td>\n",
       "      <td>0.906230</td>\n",
       "      <td>0.094478</td>\n",
       "      <td>-9.591931</td>\n",
       "      <td>8.645148e-22</td>\n",
       "      <td>1.317088e-18</td>\n",
       "      <td>IRF2BP2</td>\n",
       "      <td>IRF2BP2</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "           baseMean  log2FoldChange     lfcSE      stat        pvalue  \\\n",
       "1790     727.339448       -0.114417  0.113961  1.004001  3.153782e-01   \n",
       "1902    3919.460999       -0.100624  0.127026  0.792155  4.282701e-01   \n",
       "1906    1757.343901       -0.127836  0.121284  1.054029  2.918698e-01   \n",
       "12650  15280.599449        0.906230  0.094478 -9.591931  8.645148e-22   \n",
       "\n",
       "               padj    genes  gene_id  \n",
       "1790   6.937536e-01    MEIS1    MEIS1  \n",
       "1902   7.769807e-01      MYB      MYB  \n",
       "1906   6.784911e-01      MYC      MYC  \n",
       "12650  1.317088e-18  IRF2BP2  IRF2BP2  "
      ]
     },
     "execution_count": 59,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_test.loc[df_test['genes'].isin([\"MYC\", \"MYB\", \"IRF2BP2\", \"MEIS1\", \"MYB\"])]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "id": "2c2f7bb6",
   "metadata": {
    "collapsed": true,
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>baseMean</th>\n",
       "      <th>log2FoldChange</th>\n",
       "      <th>lfcSE</th>\n",
       "      <th>stat</th>\n",
       "      <th>pvalue</th>\n",
       "      <th>padj</th>\n",
       "      <th>genes</th>\n",
       "      <th>gene_id</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>1790</th>\n",
       "      <td>753.976244</td>\n",
       "      <td>-0.008948</td>\n",
       "      <td>0.106854</td>\n",
       "      <td>0.083742</td>\n",
       "      <td>9.332619e-01</td>\n",
       "      <td>9.950914e-01</td>\n",
       "      <td>MEIS1</td>\n",
       "      <td>MEIS1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1902</th>\n",
       "      <td>4062.431874</td>\n",
       "      <td>0.004507</td>\n",
       "      <td>0.120156</td>\n",
       "      <td>-0.037512</td>\n",
       "      <td>9.700771e-01</td>\n",
       "      <td>9.970766e-01</td>\n",
       "      <td>MYB</td>\n",
       "      <td>MYB</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1906</th>\n",
       "      <td>1822.255867</td>\n",
       "      <td>-0.020710</td>\n",
       "      <td>0.117890</td>\n",
       "      <td>0.175672</td>\n",
       "      <td>8.605519e-01</td>\n",
       "      <td>9.866435e-01</td>\n",
       "      <td>MYC</td>\n",
       "      <td>MYC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12647</th>\n",
       "      <td>16046.862685</td>\n",
       "      <td>1.013079</td>\n",
       "      <td>0.087115</td>\n",
       "      <td>-11.629256</td>\n",
       "      <td>2.926355e-31</td>\n",
       "      <td>5.453054e-28</td>\n",
       "      <td>IRF2BP2</td>\n",
       "      <td>IRF2BP2</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "           baseMean  log2FoldChange     lfcSE       stat        pvalue  \\\n",
       "1790     753.976244       -0.008948  0.106854   0.083742  9.332619e-01   \n",
       "1902    4062.431874        0.004507  0.120156  -0.037512  9.700771e-01   \n",
       "1906    1822.255867       -0.020710  0.117890   0.175672  8.605519e-01   \n",
       "12647  16046.862685        1.013079  0.087115 -11.629256  2.926355e-31   \n",
       "\n",
       "               padj    genes  gene_id  \n",
       "1790   9.950914e-01    MEIS1    MEIS1  \n",
       "1902   9.970766e-01      MYB      MYB  \n",
       "1906   9.866435e-01      MYC      MYC  \n",
       "12647  5.453054e-28  IRF2BP2  IRF2BP2  "
      ]
     },
     "execution_count": 60,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df1.loc[df1['genes'].isin([\"MYC\", \"MYB\", \"IRF2BP2\", \"MEIS1\", \"MYB\"])]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "762cbd62",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "edf0ca96",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "24322363",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "099e5e59",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "028e0385",
   "metadata": {
    "collapsed": true,
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(5, 20)\n",
      "(15518, 20)\n"
     ]
    }
   ],
   "source": [
    "print(df_info.shape)\n",
    "print(df_merged.shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "9952c679",
   "metadata": {
    "collapsed": true,
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "15523"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_info.shape[0] + df_merged.shape[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "e8b45099",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "df_info = df_info.T.reset_index(drop=True).T # reset column index"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "id": "85a7dffa",
   "metadata": {
    "collapsed": true,
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "      <th>3</th>\n",
       "      <th>4</th>\n",
       "      <th>5</th>\n",
       "      <th>6</th>\n",
       "      <th>7</th>\n",
       "      <th>8</th>\n",
       "      <th>9</th>\n",
       "      <th>10</th>\n",
       "      <th>11</th>\n",
       "      <th>12</th>\n",
       "      <th>13</th>\n",
       "      <th>14</th>\n",
       "      <th>15</th>\n",
       "      <th>16</th>\n",
       "      <th>17</th>\n",
       "      <th>18</th>\n",
       "      <th>19</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>project</td>\n",
       "      <td>project</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>file</td>\n",
       "      <td>file</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>scale</td>\n",
       "      <td>scale</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>condition</td>\n",
       "      <td>condition</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>type</td>\n",
       "      <td>type</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "          0          1                                                  2   \\\n",
       "0    project    project                                IRF2BP2_degraded_v1   \n",
       "1       file       file  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2      scale      scale                                     ERCCsamplewise   \n",
       "3  condition  condition                                                 3h   \n",
       "4       type       type                                         readcounts   \n",
       "\n",
       "                                                  3   \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  4   \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  5   \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  6   \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  7   \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  8   \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                           tccounts   \n",
       "\n",
       "                                                  9   \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                           tccounts   \n",
       "\n",
       "                                                  10  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                           tccounts   \n",
       "\n",
       "                                                  11  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                           tccounts   \n",
       "\n",
       "                                                  12  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                           tccounts   \n",
       "\n",
       "                                                  13  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                           tccounts   \n",
       "\n",
       "                                                  14  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 6h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  15  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 6h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  16  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 6h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  17  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 6h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  18  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 6h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  19  \n",
       "0                                IRF2BP2_degraded_v1  \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...  \n",
       "2                                     ERCCsamplewise  \n",
       "3                                                 6h  \n",
       "4                                         readcounts  "
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_info"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "7aac3413",
   "metadata": {
    "collapsed": true,
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "      <th>3</th>\n",
       "      <th>4</th>\n",
       "      <th>5</th>\n",
       "      <th>6</th>\n",
       "      <th>7</th>\n",
       "      <th>8</th>\n",
       "      <th>9</th>\n",
       "      <th>10</th>\n",
       "      <th>11</th>\n",
       "      <th>12</th>\n",
       "      <th>13</th>\n",
       "      <th>14</th>\n",
       "      <th>15</th>\n",
       "      <th>16</th>\n",
       "      <th>17</th>\n",
       "      <th>18</th>\n",
       "      <th>19</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>genes</td>\n",
       "      <td>gene_id</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>NAT1</td>\n",
       "      <td>NAT1</td>\n",
       "      <td>95.678631</td>\n",
       "      <td>-0.463807</td>\n",
       "      <td>0.183351</td>\n",
       "      <td>2.529615</td>\n",
       "      <td>0.011419</td>\n",
       "      <td>0.162526</td>\n",
       "      <td>5.023216</td>\n",
       "      <td>-0.554238</td>\n",
       "      <td>0.738925</td>\n",
       "      <td>0.75006</td>\n",
       "      <td>0.453218</td>\n",
       "      <td>NaN</td>\n",
       "      <td>105.239049</td>\n",
       "      <td>-0.433213</td>\n",
       "      <td>0.261849</td>\n",
       "      <td>1.65444</td>\n",
       "      <td>0.098038</td>\n",
       "      <td>0.646734</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>AAMP</td>\n",
       "      <td>AAMP</td>\n",
       "      <td>841.211188</td>\n",
       "      <td>-0.150545</td>\n",
       "      <td>0.117138</td>\n",
       "      <td>1.285188</td>\n",
       "      <td>0.198727</td>\n",
       "      <td>0.58154</td>\n",
       "      <td>14.561153</td>\n",
       "      <td>0.317738</td>\n",
       "      <td>0.385701</td>\n",
       "      <td>-0.823793</td>\n",
       "      <td>0.410057</td>\n",
       "      <td>0.647203</td>\n",
       "      <td>972.517055</td>\n",
       "      <td>0.060776</td>\n",
       "      <td>0.136377</td>\n",
       "      <td>-0.44565</td>\n",
       "      <td>0.65585</td>\n",
       "      <td>0.995046</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>AARS1</td>\n",
       "      <td>AARS1</td>\n",
       "      <td>1373.801586</td>\n",
       "      <td>-0.025304</td>\n",
       "      <td>0.10876</td>\n",
       "      <td>0.232656</td>\n",
       "      <td>0.816029</td>\n",
       "      <td>0.954353</td>\n",
       "      <td>16.055735</td>\n",
       "      <td>-0.107774</td>\n",
       "      <td>0.395963</td>\n",
       "      <td>0.272182</td>\n",
       "      <td>0.785482</td>\n",
       "      <td>0.888308</td>\n",
       "      <td>890.639308</td>\n",
       "      <td>-0.00008</td>\n",
       "      <td>0.161873</td>\n",
       "      <td>0.000495</td>\n",
       "      <td>0.999605</td>\n",
       "      <td>0.99988</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>ABAT</td>\n",
       "      <td>ABAT</td>\n",
       "      <td>55.753788</td>\n",
       "      <td>-0.362282</td>\n",
       "      <td>0.21001</td>\n",
       "      <td>1.725072</td>\n",
       "      <td>0.084514</td>\n",
       "      <td>0.404263</td>\n",
       "      <td>1.69655</td>\n",
       "      <td>-0.944984</td>\n",
       "      <td>1.260997</td>\n",
       "      <td>0.749394</td>\n",
       "      <td>0.45362</td>\n",
       "      <td>NaN</td>\n",
       "      <td>42.920747</td>\n",
       "      <td>-0.220415</td>\n",
       "      <td>0.265135</td>\n",
       "      <td>0.831329</td>\n",
       "      <td>0.405788</td>\n",
       "      <td>0.961633</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "      0        1            2               3         4         5         6   \\\n",
       "0  genes  gene_id     baseMean  log2FoldChange     lfcSE      stat    pvalue   \n",
       "1   NAT1     NAT1    95.678631       -0.463807  0.183351  2.529615  0.011419   \n",
       "2   AAMP     AAMP   841.211188       -0.150545  0.117138  1.285188  0.198727   \n",
       "3  AARS1    AARS1  1373.801586       -0.025304   0.10876  0.232656  0.816029   \n",
       "4   ABAT     ABAT    55.753788       -0.362282   0.21001  1.725072  0.084514   \n",
       "\n",
       "         7          8               9         10        11        12  \\\n",
       "0      padj   baseMean  log2FoldChange     lfcSE      stat    pvalue   \n",
       "1  0.162526   5.023216       -0.554238  0.738925   0.75006  0.453218   \n",
       "2   0.58154  14.561153        0.317738  0.385701 -0.823793  0.410057   \n",
       "3  0.954353  16.055735       -0.107774  0.395963  0.272182  0.785482   \n",
       "4  0.404263    1.69655       -0.944984  1.260997  0.749394   0.45362   \n",
       "\n",
       "         13          14              15        16        17        18  \\\n",
       "0      padj    baseMean  log2FoldChange     lfcSE      stat    pvalue   \n",
       "1       NaN  105.239049       -0.433213  0.261849   1.65444  0.098038   \n",
       "2  0.647203  972.517055        0.060776  0.136377  -0.44565   0.65585   \n",
       "3  0.888308  890.639308        -0.00008  0.161873  0.000495  0.999605   \n",
       "4       NaN   42.920747       -0.220415  0.265135  0.831329  0.405788   \n",
       "\n",
       "         19  \n",
       "0      padj  \n",
       "1  0.646734  \n",
       "2  0.995046  \n",
       "3   0.99988  \n",
       "4  0.961633  "
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_merged.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "id": "68095484",
   "metadata": {
    "hidden": true,
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 4 ms, sys: 0 ns, total: 4 ms\n",
      "Wall time: 4.03 ms\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "\n",
    "# combine info and data\n",
    "df_save = df_info.append(df_merged, ignore_index=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "id": "1796fa14",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "      <th>3</th>\n",
       "      <th>4</th>\n",
       "      <th>5</th>\n",
       "      <th>6</th>\n",
       "      <th>7</th>\n",
       "      <th>8</th>\n",
       "      <th>9</th>\n",
       "      <th>10</th>\n",
       "      <th>11</th>\n",
       "      <th>12</th>\n",
       "      <th>13</th>\n",
       "      <th>14</th>\n",
       "      <th>15</th>\n",
       "      <th>16</th>\n",
       "      <th>17</th>\n",
       "      <th>18</th>\n",
       "      <th>19</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>project</td>\n",
       "      <td>project</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>file</td>\n",
       "      <td>file</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>scale</td>\n",
       "      <td>scale</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>condition</td>\n",
       "      <td>condition</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>type</td>\n",
       "      <td>type</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "          0          1                                                  2   \\\n",
       "0    project    project                                IRF2BP2_degraded_v1   \n",
       "1       file       file  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2      scale      scale                                     ERCCsamplewise   \n",
       "3  condition  condition                                                 3h   \n",
       "4       type       type                                         readcounts   \n",
       "\n",
       "                                                  3   \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  4   \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  5   \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  6   \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  7   \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  8   \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                           tccounts   \n",
       "\n",
       "                                                  9   \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                           tccounts   \n",
       "\n",
       "                                                  10  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                           tccounts   \n",
       "\n",
       "                                                  11  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                           tccounts   \n",
       "\n",
       "                                                  12  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                           tccounts   \n",
       "\n",
       "                                                  13  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                           tccounts   \n",
       "\n",
       "                                                  14  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 6h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  15  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 6h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  16  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 6h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  17  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 6h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  18  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 6h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  19  \n",
       "0                                IRF2BP2_degraded_v1  \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...  \n",
       "2                                     ERCCsamplewise  \n",
       "3                                                 6h  \n",
       "4                                         readcounts  "
      ]
     },
     "execution_count": 36,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_save.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0c303a44",
   "metadata": {
    "heading_collapsed": true
   },
   "source": [
    "## Save grouped data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "id": "2b54721a",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved DESeq results to: ../results/slamseq_merged_deseq.csv\n"
     ]
    }
   ],
   "source": [
    "# save file\n",
    "res_dir = \"../results\"\n",
    "deseq_res_file = \"slamseq_merged_deseq.csv\"\n",
    "\n",
    "df_save.to_csv(\"{}/{}\".format(res_dir, deseq_res_file), header=False, index=False)\n",
    "\n",
    "print(\"Saved DESeq results to: {}/{}\".format(res_dir, deseq_res_file))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a2a4acf1",
   "metadata": {
    "heading_collapsed": true
   },
   "source": [
    "## MYB_degraded_v1 (project 2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 167,
   "id": "2a0b87c2",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(15523, 20)\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "      <th>3</th>\n",
       "      <th>4</th>\n",
       "      <th>5</th>\n",
       "      <th>6</th>\n",
       "      <th>7</th>\n",
       "      <th>8</th>\n",
       "      <th>9</th>\n",
       "      <th>10</th>\n",
       "      <th>11</th>\n",
       "      <th>12</th>\n",
       "      <th>13</th>\n",
       "      <th>14</th>\n",
       "      <th>15</th>\n",
       "      <th>16</th>\n",
       "      <th>17</th>\n",
       "      <th>18</th>\n",
       "      <th>19</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>project</td>\n",
       "      <td>project</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "      <td>IRF2BP2_degraded_v1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>file</td>\n",
       "      <td>file</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...</td>\n",
       "      <td>IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>scale</td>\n",
       "      <td>scale</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>condition</td>\n",
       "      <td>condition</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>3h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "      <td>6h</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>type</td>\n",
       "      <td>type</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "          0          1                                                  2   \\\n",
       "0    project    project                                IRF2BP2_degraded_v1   \n",
       "1       file       file  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2      scale      scale                                     ERCCsamplewise   \n",
       "3  condition  condition                                                 3h   \n",
       "4       type       type                                         readcounts   \n",
       "\n",
       "                                                  3   \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  4   \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  5   \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  6   \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  7   \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  8   \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                           tccounts   \n",
       "\n",
       "                                                  9   \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                           tccounts   \n",
       "\n",
       "                                                  10  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                           tccounts   \n",
       "\n",
       "                                                  11  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                           tccounts   \n",
       "\n",
       "                                                  12  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                           tccounts   \n",
       "\n",
       "                                                  13  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_3h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 3h   \n",
       "4                                           tccounts   \n",
       "\n",
       "                                                  14  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 6h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  15  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 6h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  16  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 6h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  17  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 6h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  18  \\\n",
       "0                                IRF2BP2_degraded_v1   \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 6h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  19  \n",
       "0                                IRF2BP2_degraded_v1  \n",
       "1  IRF2BP2_degraded_v1_ERCCsamplewise_0_5_6h_dese...  \n",
       "2                                     ERCCsamplewise  \n",
       "3                                                 6h  \n",
       "4                                         readcounts  "
      ]
     },
     "execution_count": 167,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# read in merged deseq file\n",
    "res_dir = \"../results\"\n",
    "deseq_res_file = \"slamseq_merged_deseq.csv\"\n",
    "\n",
    "df_save = pd.read_csv(\"{}/{}\".format(res_dir, deseq_res_file), index_col=False, header=None, squeeze=True)\n",
    "print(df_save.shape)\n",
    "df_save.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 255,
   "id": "a2b564c5",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "project = project2\n",
    "project_dir = \"slamseq_{}\".format(project)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 256,
   "id": "11d0867e",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "files = ! ls $res_dir/$project_dir/*.csv\n",
    "files = [file for file in files if \"deseq\" in file if \"drop\" not in file]\n",
    "# ERCC files first\n",
    "files = [file for file in files if \"ERCCsamplewise\" in file] + \\\n",
    "[file for file in files if \"ERCCsamplewise\" not in file]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 257,
   "id": "af8f179e",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['../results/slamseq_MYB_degraded_v1/MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts.csv',\n",
       " '../results/slamseq_MYB_degraded_v1/MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts.csv',\n",
       " '../results/slamseq_MYB_degraded_v1/MYB_degraded_v1_0_5_1h_deseq_readcounts.csv',\n",
       " '../results/slamseq_MYB_degraded_v1/MYB_degraded_v1_0_5_1h_deseq_tccounts.csv']"
      ]
     },
     "execution_count": 257,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 258,
   "id": "2170c61c",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "# project 2\n",
    "conditions = [\"1h\"]\n",
    "scaling = [\"ERCCsamplewise\", None]\n",
    "count_type = [\"readcounts\", \"tccounts\"]"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "67976dae",
   "metadata": {
    "hidden": true
   },
   "source": [
    "Get row info"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 260,
   "id": "54785c90",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "# df with file info\n",
    "df_info = pd.DataFrame(list(zip([\"project\", \"file\", \"scale\", \"condition\", \"type\"],\n",
    "                                [\"project\", \"file\", \"scale\", \"condition\", \"type\"])))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 261,
   "id": "7ca81870",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(5, 26)"
      ]
     },
     "execution_count": 261,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_info = pd.concat([df_info] + [pd.DataFrame([x*6 for x in get_file_info(project, file, conditions)]) \\\n",
    "                                 for file in files], axis=1)\n",
    "df_info = df_info.T.reset_index(drop=True).T # reset column index\n",
    "df_info.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 262,
   "id": "1d5d3cd7",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "      <th>3</th>\n",
       "      <th>4</th>\n",
       "      <th>5</th>\n",
       "      <th>6</th>\n",
       "      <th>7</th>\n",
       "      <th>8</th>\n",
       "      <th>9</th>\n",
       "      <th>...</th>\n",
       "      <th>16</th>\n",
       "      <th>17</th>\n",
       "      <th>18</th>\n",
       "      <th>19</th>\n",
       "      <th>20</th>\n",
       "      <th>21</th>\n",
       "      <th>22</th>\n",
       "      <th>23</th>\n",
       "      <th>24</th>\n",
       "      <th>25</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>project</td>\n",
       "      <td>project</td>\n",
       "      <td>MYB_degraded_v1</td>\n",
       "      <td>MYB_degraded_v1</td>\n",
       "      <td>MYB_degraded_v1</td>\n",
       "      <td>MYB_degraded_v1</td>\n",
       "      <td>MYB_degraded_v1</td>\n",
       "      <td>MYB_degraded_v1</td>\n",
       "      <td>MYB_degraded_v1</td>\n",
       "      <td>MYB_degraded_v1</td>\n",
       "      <td>...</td>\n",
       "      <td>MYB_degraded_v1</td>\n",
       "      <td>MYB_degraded_v1</td>\n",
       "      <td>MYB_degraded_v1</td>\n",
       "      <td>MYB_degraded_v1</td>\n",
       "      <td>MYB_degraded_v1</td>\n",
       "      <td>MYB_degraded_v1</td>\n",
       "      <td>MYB_degraded_v1</td>\n",
       "      <td>MYB_degraded_v1</td>\n",
       "      <td>MYB_degraded_v1</td>\n",
       "      <td>MYB_degraded_v1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>file</td>\n",
       "      <td>file</td>\n",
       "      <td>MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_re...</td>\n",
       "      <td>MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_re...</td>\n",
       "      <td>MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_re...</td>\n",
       "      <td>MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_re...</td>\n",
       "      <td>MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_re...</td>\n",
       "      <td>MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_re...</td>\n",
       "      <td>MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tc...</td>\n",
       "      <td>MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tc...</td>\n",
       "      <td>...</td>\n",
       "      <td>MYB_degraded_v1_0_5_1h_deseq_readcounts.csv</td>\n",
       "      <td>MYB_degraded_v1_0_5_1h_deseq_readcounts.csv</td>\n",
       "      <td>MYB_degraded_v1_0_5_1h_deseq_readcounts.csv</td>\n",
       "      <td>MYB_degraded_v1_0_5_1h_deseq_readcounts.csv</td>\n",
       "      <td>MYB_degraded_v1_0_5_1h_deseq_tccounts.csv</td>\n",
       "      <td>MYB_degraded_v1_0_5_1h_deseq_tccounts.csv</td>\n",
       "      <td>MYB_degraded_v1_0_5_1h_deseq_tccounts.csv</td>\n",
       "      <td>MYB_degraded_v1_0_5_1h_deseq_tccounts.csv</td>\n",
       "      <td>MYB_degraded_v1_0_5_1h_deseq_tccounts.csv</td>\n",
       "      <td>MYB_degraded_v1_0_5_1h_deseq_tccounts.csv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>scale</td>\n",
       "      <td>scale</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>ERCCsamplewise</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>condition</td>\n",
       "      <td>condition</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>...</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "      <td>1h</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>type</td>\n",
       "      <td>type</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>...</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>readcounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "      <td>tccounts</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows Ã— 26 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "          0          1                                                  2   \\\n",
       "0    project    project                                    MYB_degraded_v1   \n",
       "1       file       file  MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_re...   \n",
       "2      scale      scale                                     ERCCsamplewise   \n",
       "3  condition  condition                                                 1h   \n",
       "4       type       type                                         readcounts   \n",
       "\n",
       "                                                  3   \\\n",
       "0                                    MYB_degraded_v1   \n",
       "1  MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_re...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 1h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  4   \\\n",
       "0                                    MYB_degraded_v1   \n",
       "1  MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_re...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 1h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  5   \\\n",
       "0                                    MYB_degraded_v1   \n",
       "1  MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_re...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 1h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  6   \\\n",
       "0                                    MYB_degraded_v1   \n",
       "1  MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_re...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 1h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  7   \\\n",
       "0                                    MYB_degraded_v1   \n",
       "1  MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_re...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 1h   \n",
       "4                                         readcounts   \n",
       "\n",
       "                                                  8   \\\n",
       "0                                    MYB_degraded_v1   \n",
       "1  MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tc...   \n",
       "2                                     ERCCsamplewise   \n",
       "3                                                 1h   \n",
       "4                                           tccounts   \n",
       "\n",
       "                                                  9   ...  \\\n",
       "0                                    MYB_degraded_v1  ...   \n",
       "1  MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tc...  ...   \n",
       "2                                     ERCCsamplewise  ...   \n",
       "3                                                 1h  ...   \n",
       "4                                           tccounts  ...   \n",
       "\n",
       "                                            16  \\\n",
       "0                              MYB_degraded_v1   \n",
       "1  MYB_degraded_v1_0_5_1h_deseq_readcounts.csv   \n",
       "2                                         None   \n",
       "3                                           1h   \n",
       "4                                   readcounts   \n",
       "\n",
       "                                            17  \\\n",
       "0                              MYB_degraded_v1   \n",
       "1  MYB_degraded_v1_0_5_1h_deseq_readcounts.csv   \n",
       "2                                         None   \n",
       "3                                           1h   \n",
       "4                                   readcounts   \n",
       "\n",
       "                                            18  \\\n",
       "0                              MYB_degraded_v1   \n",
       "1  MYB_degraded_v1_0_5_1h_deseq_readcounts.csv   \n",
       "2                                         None   \n",
       "3                                           1h   \n",
       "4                                   readcounts   \n",
       "\n",
       "                                            19  \\\n",
       "0                              MYB_degraded_v1   \n",
       "1  MYB_degraded_v1_0_5_1h_deseq_readcounts.csv   \n",
       "2                                         None   \n",
       "3                                           1h   \n",
       "4                                   readcounts   \n",
       "\n",
       "                                          20  \\\n",
       "0                            MYB_degraded_v1   \n",
       "1  MYB_degraded_v1_0_5_1h_deseq_tccounts.csv   \n",
       "2                                       None   \n",
       "3                                         1h   \n",
       "4                                   tccounts   \n",
       "\n",
       "                                          21  \\\n",
       "0                            MYB_degraded_v1   \n",
       "1  MYB_degraded_v1_0_5_1h_deseq_tccounts.csv   \n",
       "2                                       None   \n",
       "3                                         1h   \n",
       "4                                   tccounts   \n",
       "\n",
       "                                          22  \\\n",
       "0                            MYB_degraded_v1   \n",
       "1  MYB_degraded_v1_0_5_1h_deseq_tccounts.csv   \n",
       "2                                       None   \n",
       "3                                         1h   \n",
       "4                                   tccounts   \n",
       "\n",
       "                                          23  \\\n",
       "0                            MYB_degraded_v1   \n",
       "1  MYB_degraded_v1_0_5_1h_deseq_tccounts.csv   \n",
       "2                                       None   \n",
       "3                                         1h   \n",
       "4                                   tccounts   \n",
       "\n",
       "                                          24  \\\n",
       "0                            MYB_degraded_v1   \n",
       "1  MYB_degraded_v1_0_5_1h_deseq_tccounts.csv   \n",
       "2                                       None   \n",
       "3                                         1h   \n",
       "4                                   tccounts   \n",
       "\n",
       "                                          25  \n",
       "0                            MYB_degraded_v1  \n",
       "1  MYB_degraded_v1_0_5_1h_deseq_tccounts.csv  \n",
       "2                                       None  \n",
       "3                                         1h  \n",
       "4                                   tccounts  \n",
       "\n",
       "[5 rows x 26 columns]"
      ]
     },
     "execution_count": 262,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_info"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ec78bd9d",
   "metadata": {
    "hidden": true
   },
   "source": [
    "Get deseq data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 263,
   "id": "16efe50e",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "def get_deseq_data(file):\n",
    "    ''' input deseq file & returns df '''\n",
    "    ''' add file name to headers except gene columns '''\n",
    "    df = pd.read_csv(file, header=[0], index_col=None, squeeze=True)\n",
    "    df.columns = [\"{}_{}\".format(col_name, os.path.basename(file).split(\".\")[0]) if \"gene\" not in col_name\n",
    "                  else col_name for col_name in df.columns.tolist()]\n",
    "    return(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 266,
   "id": "aeb07d31",
   "metadata": {
    "hidden": true,
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(16558, 8)\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>baseMean_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>log2FoldChange_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>lfcSE_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>stat_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>pvalue_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>padj_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>genes</th>\n",
       "      <th>gene_id</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1.877541</td>\n",
       "      <td>-0.045789</td>\n",
       "      <td>1.010641</td>\n",
       "      <td>0.045307</td>\n",
       "      <td>0.963863</td>\n",
       "      <td>NaN</td>\n",
       "      <td>A1BG</td>\n",
       "      <td>A1BG</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>5.229468</td>\n",
       "      <td>-0.269040</td>\n",
       "      <td>0.500213</td>\n",
       "      <td>0.537851</td>\n",
       "      <td>0.590680</td>\n",
       "      <td>NaN</td>\n",
       "      <td>A2MP1</td>\n",
       "      <td>A2MP1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   baseMean_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "0                                           1.877541                 \n",
       "1                                           5.229468                 \n",
       "\n",
       "   log2FoldChange_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "0                                          -0.045789                       \n",
       "1                                          -0.269040                       \n",
       "\n",
       "   lfcSE_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "0                                           1.010641              \n",
       "1                                           0.500213              \n",
       "\n",
       "   stat_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "0                                           0.045307             \n",
       "1                                           0.537851             \n",
       "\n",
       "   pvalue_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "0                                           0.963863               \n",
       "1                                           0.590680               \n",
       "\n",
       "   padj_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  genes gene_id  \n",
       "0                                                NaN             A1BG    A1BG  \n",
       "1                                                NaN            A2MP1   A2MP1  "
      ]
     },
     "execution_count": 266,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "deseq_dfs = [get_deseq_data(file) for file in files]\n",
    "print(deseq_dfs[0].shape)\n",
    "deseq_dfs[0].iloc[:2]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 267,
   "id": "56e779c9",
   "metadata": {
    "collapsed": true,
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(16558, 8)\n",
      "(16558, 8)\n",
      "(16558, 8)\n",
      "(16558, 8)\n"
     ]
    }
   ],
   "source": [
    "for df in deseq_dfs:\n",
    "    print(df.shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 268,
   "id": "dc2e60a4",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts.csv',\n",
       " 'MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts.csv',\n",
       " 'MYB_degraded_v1_0_5_1h_deseq_readcounts.csv',\n",
       " 'MYB_degraded_v1_0_5_1h_deseq_tccounts.csv']"
      ]
     },
     "execution_count": 268,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "[os.path.basename(x) for x in files]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 269,
   "id": "062a3029",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "df1 = deseq_dfs[0]\n",
    "df2 = deseq_dfs[1]\n",
    "df3 = deseq_dfs[2]\n",
    "df4 = deseq_dfs[3]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 270,
   "id": "d0ec3927",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "genes1 = list(df1[\"genes\"])\n",
    "genes2 = list(df2[\"genes\"])\n",
    "genes3 = list(df3[\"genes\"])\n",
    "genes4 = list(df4[\"genes\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 271,
   "id": "856c43d2",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 271,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "np.logical_and(np.array_equal(df1[\"genes\"].unique(), df2[\"genes\"].unique()), \n",
    "               np.array_equal(df2[\"genes\"].unique(), df3[\"genes\"].unique()))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 272,
   "id": "0e9e3247",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 272,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(list(df1[\"genes\"])) == len(set(df1[\"genes\"].unique()))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 275,
   "id": "3963c28f",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "merge_test = reduce(lambda x, y: pd.merge(x, y, on = [\"genes\", \"gene_id\"], how=\"outer\"), [df1, df2])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 277,
   "id": "4e4c632c",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(16558, 14)\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>baseMean_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>log2FoldChange_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>lfcSE_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>stat_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>pvalue_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>padj_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>genes</th>\n",
       "      <th>gene_id</th>\n",
       "      <th>baseMean_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts</th>\n",
       "      <th>log2FoldChange_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts</th>\n",
       "      <th>lfcSE_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts</th>\n",
       "      <th>stat_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts</th>\n",
       "      <th>pvalue_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts</th>\n",
       "      <th>padj_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1.877541</td>\n",
       "      <td>-0.045789</td>\n",
       "      <td>1.010641</td>\n",
       "      <td>0.045307</td>\n",
       "      <td>0.963863</td>\n",
       "      <td>NaN</td>\n",
       "      <td>A1BG</td>\n",
       "      <td>A1BG</td>\n",
       "      <td>0.058330</td>\n",
       "      <td>-0.614498</td>\n",
       "      <td>3.116540</td>\n",
       "      <td>0.197173</td>\n",
       "      <td>0.843692</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>5.229468</td>\n",
       "      <td>-0.269040</td>\n",
       "      <td>0.500213</td>\n",
       "      <td>0.537851</td>\n",
       "      <td>0.590680</td>\n",
       "      <td>NaN</td>\n",
       "      <td>A2MP1</td>\n",
       "      <td>A2MP1</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>-0.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>58.517384</td>\n",
       "      <td>0.013887</td>\n",
       "      <td>0.186521</td>\n",
       "      <td>-0.074452</td>\n",
       "      <td>0.940651</td>\n",
       "      <td>0.987822</td>\n",
       "      <td>NAT1</td>\n",
       "      <td>NAT1</td>\n",
       "      <td>4.852925</td>\n",
       "      <td>0.551781</td>\n",
       "      <td>0.560071</td>\n",
       "      <td>-0.985199</td>\n",
       "      <td>0.324526</td>\n",
       "      <td>0.837918</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>2.316485</td>\n",
       "      <td>-0.735553</td>\n",
       "      <td>0.776528</td>\n",
       "      <td>0.947233</td>\n",
       "      <td>0.343520</td>\n",
       "      <td>NaN</td>\n",
       "      <td>AADAC</td>\n",
       "      <td>AADAC</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>-0.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>340.671808</td>\n",
       "      <td>-0.130751</td>\n",
       "      <td>0.092915</td>\n",
       "      <td>1.407202</td>\n",
       "      <td>0.159367</td>\n",
       "      <td>0.743707</td>\n",
       "      <td>AAMP</td>\n",
       "      <td>AAMP</td>\n",
       "      <td>9.364142</td>\n",
       "      <td>0.216094</td>\n",
       "      <td>0.341920</td>\n",
       "      <td>-0.632001</td>\n",
       "      <td>0.527386</td>\n",
       "      <td>0.915144</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   baseMean_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "0                                           1.877541                 \n",
       "1                                           5.229468                 \n",
       "2                                          58.517384                 \n",
       "3                                           2.316485                 \n",
       "4                                         340.671808                 \n",
       "\n",
       "   log2FoldChange_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "0                                          -0.045789                       \n",
       "1                                          -0.269040                       \n",
       "2                                           0.013887                       \n",
       "3                                          -0.735553                       \n",
       "4                                          -0.130751                       \n",
       "\n",
       "   lfcSE_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "0                                           1.010641              \n",
       "1                                           0.500213              \n",
       "2                                           0.186521              \n",
       "3                                           0.776528              \n",
       "4                                           0.092915              \n",
       "\n",
       "   stat_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "0                                           0.045307             \n",
       "1                                           0.537851             \n",
       "2                                          -0.074452             \n",
       "3                                           0.947233             \n",
       "4                                           1.407202             \n",
       "\n",
       "   pvalue_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "0                                           0.963863               \n",
       "1                                           0.590680               \n",
       "2                                           0.940651               \n",
       "3                                           0.343520               \n",
       "4                                           0.159367               \n",
       "\n",
       "   padj_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  genes gene_id  \\\n",
       "0                                                NaN             A1BG    A1BG   \n",
       "1                                                NaN            A2MP1   A2MP1   \n",
       "2                                           0.987822             NAT1    NAT1   \n",
       "3                                                NaN            AADAC   AADAC   \n",
       "4                                           0.743707             AAMP    AAMP   \n",
       "\n",
       "   baseMean_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts  \\\n",
       "0                                           0.058330               \n",
       "1                                           0.000000               \n",
       "2                                           4.852925               \n",
       "3                                           0.000000               \n",
       "4                                           9.364142               \n",
       "\n",
       "   log2FoldChange_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts  \\\n",
       "0                                          -0.614498                     \n",
       "1                                          -0.000000                     \n",
       "2                                           0.551781                     \n",
       "3                                          -0.000000                     \n",
       "4                                           0.216094                     \n",
       "\n",
       "   lfcSE_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts  \\\n",
       "0                                           3.116540            \n",
       "1                                                NaN            \n",
       "2                                           0.560071            \n",
       "3                                                NaN            \n",
       "4                                           0.341920            \n",
       "\n",
       "   stat_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts  \\\n",
       "0                                           0.197173           \n",
       "1                                                NaN           \n",
       "2                                          -0.985199           \n",
       "3                                                NaN           \n",
       "4                                          -0.632001           \n",
       "\n",
       "   pvalue_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts  \\\n",
       "0                                           0.843692             \n",
       "1                                           0.000000             \n",
       "2                                           0.324526             \n",
       "3                                           0.000000             \n",
       "4                                           0.527386             \n",
       "\n",
       "   padj_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts  \n",
       "0                                                NaN          \n",
       "1                                                NaN          \n",
       "2                                           0.837918          \n",
       "3                                                NaN          \n",
       "4                                           0.915144          "
      ]
     },
     "execution_count": 277,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "print(merge_test.shape)\n",
    "merge_test.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "95acb794",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "d09b566a",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 196,
   "id": "a1739b41",
   "metadata": {
    "collapsed": true,
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['A1BG',\n",
       " 'A2MP1',\n",
       " 'NAT1',\n",
       " 'AADAC',\n",
       " 'AAMP',\n",
       " 'AARS1',\n",
       " 'ABAT',\n",
       " 'ABCA1',\n",
       " 'ABCA2',\n",
       " 'ABCB7',\n",
       " 'ABL1',\n",
       " 'AOC1',\n",
       " 'ABL2',\n",
       " 'ABO',\n",
       " 'ABR',\n",
       " 'ACAA1',\n",
       " 'ACACB',\n",
       " 'ACADM',\n",
       " 'ACADS',\n",
       " 'ACADSB',\n",
       " 'ACADVL',\n",
       " 'ACAT1',\n",
       " 'ACAT2',\n",
       " 'ASIC2',\n",
       " 'ASIC1',\n",
       " 'ACLY',\n",
       " 'ACO1',\n",
       " 'ACO2',\n",
       " 'ACOX1',\n",
       " 'ACP1',\n",
       " 'ACP2',\n",
       " 'ACP3',\n",
       " 'ACRV1',\n",
       " 'ACTB',\n",
       " 'ACTG1',\n",
       " 'ACTN4',\n",
       " 'ACTL6A',\n",
       " 'ACTN1',\n",
       " 'ACVR1',\n",
       " 'ACVR1B',\n",
       " 'ACVR2A',\n",
       " 'ACVR2B',\n",
       " 'ACVRL1',\n",
       " 'ACY1',\n",
       " 'ACYP1',\n",
       " 'ACYP2',\n",
       " 'ADA',\n",
       " 'ADAM8',\n",
       " 'ADAM10',\n",
       " 'ADAR',\n",
       " 'ADARB1',\n",
       " 'ADCY1',\n",
       " 'ADCY2',\n",
       " 'ADCY3',\n",
       " 'ADCY5',\n",
       " 'ADCY6',\n",
       " 'ADCY7',\n",
       " 'ADCY9',\n",
       " 'ADCYAP1R1',\n",
       " 'ADD1',\n",
       " 'ADD2',\n",
       " 'ADD3',\n",
       " 'PLIN2',\n",
       " 'ADH5',\n",
       " 'ADH6',\n",
       " 'ADK',\n",
       " 'ADORA1',\n",
       " 'ADORA2A',\n",
       " 'ADORA2B',\n",
       " 'ADORA3',\n",
       " 'ADPRH',\n",
       " 'PARP1',\n",
       " 'PARP4',\n",
       " 'ADRA1A',\n",
       " 'ADRA2B',\n",
       " 'ADRA2C',\n",
       " 'ADRB1',\n",
       " 'ADRB2',\n",
       " 'GRK2',\n",
       " 'GRK3',\n",
       " 'ADSL',\n",
       " 'ADSS2',\n",
       " 'AP2A1',\n",
       " 'AP2A2',\n",
       " 'AP1B1',\n",
       " 'AP2B1',\n",
       " 'AP1G1',\n",
       " 'TLE5',\n",
       " 'CRISP1',\n",
       " 'AFG3L1P',\n",
       " 'AGA',\n",
       " 'AGL',\n",
       " 'JAG1',\n",
       " 'AGT',\n",
       " 'AHCY',\n",
       " 'AHR',\n",
       " 'CRYBG1',\n",
       " 'AK1',\n",
       " 'AK2',\n",
       " 'AK4',\n",
       " 'AKT1',\n",
       " 'AKT2',\n",
       " 'ALAD',\n",
       " 'ALAS1',\n",
       " 'ALCAM',\n",
       " 'ABCD1',\n",
       " 'ALDH2',\n",
       " 'ALDH1B1',\n",
       " 'ALDH1A3',\n",
       " 'ALDH3B1',\n",
       " 'ALDH9A1',\n",
       " 'ALDH3A2',\n",
       " 'ABCD2',\n",
       " 'ALDOA',\n",
       " 'ALDOB',\n",
       " 'ALDOC',\n",
       " 'AKR1B1',\n",
       " 'ALOX12',\n",
       " 'ALOX5AP',\n",
       " 'ALOX12B',\n",
       " 'ALOX15B',\n",
       " 'ALPP',\n",
       " 'ALX3',\n",
       " 'AMD1',\n",
       " 'AMFR',\n",
       " 'AMH',\n",
       " 'AMPD2',\n",
       " 'AMPD3',\n",
       " 'AMPH',\n",
       " 'BIN1',\n",
       " 'AMT',\n",
       " 'AMY2B',\n",
       " 'ANGPT1',\n",
       " 'ANGPT2',\n",
       " 'ANK1',\n",
       " 'ANK2',\n",
       " 'ANK3',\n",
       " 'ANPEP',\n",
       " 'SLC25A4',\n",
       " 'SLC25A5',\n",
       " 'SLC25A6',\n",
       " 'ANXA1',\n",
       " 'ANXA2',\n",
       " 'ANXA3',\n",
       " 'ANXA4',\n",
       " 'ANXA5',\n",
       " 'ANXA6',\n",
       " 'ANXA7',\n",
       " 'ANXA11',\n",
       " 'AOAH',\n",
       " 'AOC2',\n",
       " 'APAF1',\n",
       " 'NUDT2',\n",
       " 'APOF',\n",
       " 'APBA2',\n",
       " 'APBB1',\n",
       " 'APBB2',\n",
       " 'APC',\n",
       " 'AIRE',\n",
       " 'APEH',\n",
       " 'APEX1',\n",
       " 'BIRC2',\n",
       " 'BIRC3',\n",
       " 'XIAP',\n",
       " 'BIRC5',\n",
       " 'APLP1',\n",
       " 'APLP2',\n",
       " 'APOA2',\n",
       " 'APOC1',\n",
       " 'APOC2',\n",
       " 'APOC4',\n",
       " 'APP',\n",
       " 'APRT',\n",
       " 'FAS',\n",
       " 'AQP1',\n",
       " 'AQP3',\n",
       " 'AR',\n",
       " 'ARAF',\n",
       " 'ARCN1',\n",
       " 'TRIM23',\n",
       " 'AREG',\n",
       " 'ARF1',\n",
       " 'ARF3',\n",
       " 'ARF4',\n",
       " 'ARL4D',\n",
       " 'ARF5',\n",
       " 'ARF6',\n",
       " 'ARG2',\n",
       " 'RHOA',\n",
       " 'RHOB',\n",
       " 'RHOC',\n",
       " 'RND3',\n",
       " 'RHOG',\n",
       " 'ARHGAP1',\n",
       " 'ARHGAP4',\n",
       " 'ARHGAP5',\n",
       " 'ARHGAP6',\n",
       " 'ARHGDIA',\n",
       " 'ARHGDIB',\n",
       " 'RHOH',\n",
       " 'ARL1',\n",
       " 'PHOX2A',\n",
       " 'ARL2',\n",
       " 'ARL3',\n",
       " 'ARNT',\n",
       " 'ARNTL',\n",
       " 'ARRB1',\n",
       " 'ARRB2',\n",
       " 'ARSA',\n",
       " 'ARSB',\n",
       " 'STS',\n",
       " 'ARSD',\n",
       " 'ARSL',\n",
       " 'ARVCF',\n",
       " 'ASAH1',\n",
       " 'ASCL1',\n",
       " 'ASGR1',\n",
       " 'ASGR2',\n",
       " 'ASL',\n",
       " 'GET3',\n",
       " 'ASNS',\n",
       " 'ASPH',\n",
       " 'ASS1',\n",
       " 'ASTN1',\n",
       " 'ZFHX3',\n",
       " 'ATF1',\n",
       " 'ATF3',\n",
       " 'ATF4',\n",
       " 'ATIC',\n",
       " 'ATM',\n",
       " 'RERE',\n",
       " 'ATOX1',\n",
       " 'ATP1A1',\n",
       " 'ATP1A2',\n",
       " 'ATP1A3',\n",
       " 'ATP1B1',\n",
       " 'ATP1B2',\n",
       " 'ATP1B3',\n",
       " 'ATP2A1',\n",
       " 'ATP2A2',\n",
       " 'ATP2A3',\n",
       " 'ATP2B1',\n",
       " 'ATP2B2',\n",
       " 'ATP2B3',\n",
       " 'ATP2B4',\n",
       " 'ATP4B',\n",
       " 'ATP5F1A',\n",
       " 'ALDH7A1',\n",
       " 'ATP5F1B',\n",
       " 'ATP5F1C',\n",
       " 'ATP5F1D',\n",
       " 'ATP5F1E',\n",
       " 'ATP5PB',\n",
       " 'ATP5MC1',\n",
       " 'ATP5MC2',\n",
       " 'ATP5MC3',\n",
       " 'ATP5ME',\n",
       " 'ATP5PF',\n",
       " 'ATP6V1A',\n",
       " 'ATP6V1B2',\n",
       " 'ATP6V0C',\n",
       " 'ATP6V1C1',\n",
       " 'ATP6V1E1',\n",
       " 'ATP6V0B',\n",
       " 'ATP6V0A1',\n",
       " 'ATP6AP1',\n",
       " 'ATP7A',\n",
       " 'ATP5PO',\n",
       " 'ATP7B',\n",
       " 'ATR',\n",
       " 'ATRX',\n",
       " 'KIF1A',\n",
       " 'AUH',\n",
       " 'AUP1',\n",
       " 'AVPR2',\n",
       " 'AXL',\n",
       " 'B2M',\n",
       " 'BAAT',\n",
       " 'BACH1',\n",
       " 'BAD',\n",
       " 'BAG1',\n",
       " '574',\n",
       " 'ADGRB1',\n",
       " 'BAK1',\n",
       " 'NKX3-2',\n",
       " 'BARD1',\n",
       " 'BAX',\n",
       " 'BBS1',\n",
       " 'BBS2',\n",
       " 'BBS4',\n",
       " 'BCAT1',\n",
       " 'BCAT2',\n",
       " 'BCKDHA',\n",
       " 'BCKDHB',\n",
       " 'CCND1',\n",
       " 'BCL2',\n",
       " 'BCL2A1',\n",
       " 'BCL2L1',\n",
       " 'BCL2L2',\n",
       " 'BCL3',\n",
       " 'BCL6',\n",
       " 'BCL7A',\n",
       " 'BCL9',\n",
       " 'OPN1SW',\n",
       " 'BCR',\n",
       " 'BCS1L',\n",
       " 'BDH1',\n",
       " 'BDKRB2',\n",
       " 'BDNF',\n",
       " 'BFSP1',\n",
       " 'BGLAP',\n",
       " 'CEACAM1',\n",
       " 'BICD1',\n",
       " 'BID',\n",
       " 'BIK',\n",
       " 'PRDM1',\n",
       " 'BLK',\n",
       " 'BLM',\n",
       " 'BLMH',\n",
       " 'CXCR5',\n",
       " 'BLVRB',\n",
       " 'BNC1',\n",
       " 'BMI1',\n",
       " 'BMP1',\n",
       " 'BMP2',\n",
       " 'BMP3',\n",
       " 'BMP4',\n",
       " 'BMP6',\n",
       " 'BMP8B',\n",
       " 'BMPR1A',\n",
       " 'BMPR1B',\n",
       " 'BMPR2',\n",
       " 'POLR3D',\n",
       " 'BNIP1',\n",
       " 'BNIP2',\n",
       " 'BNIP3',\n",
       " 'BNIP3L',\n",
       " 'BOK',\n",
       " 'DST',\n",
       " 'FOXL2',\n",
       " 'BPGM',\n",
       " 'BPHL',\n",
       " 'BRCA1',\n",
       " 'BRAF',\n",
       " 'BRCA2',\n",
       " 'ZFP36L1',\n",
       " 'ZFP36L2',\n",
       " 'BSG',\n",
       " 'BST1',\n",
       " 'BST2',\n",
       " 'BTD',\n",
       " 'KLF9',\n",
       " 'KLF5',\n",
       " 'BTF3',\n",
       " 'BTG1',\n",
       " 'BTK',\n",
       " 'BUB1',\n",
       " 'BUB1B',\n",
       " 'BYSL',\n",
       " 'TSPO',\n",
       " 'C1QBP',\n",
       " 'SERPING1',\n",
       " 'ERC2-IT1',\n",
       " 'C1QA',\n",
       " 'C1QC',\n",
       " 'C1R',\n",
       " 'C1S',\n",
       " 'C3',\n",
       " 'C3AR1',\n",
       " 'CAPN5',\n",
       " 'C5',\n",
       " 'C5AR1',\n",
       " 'OSGIN2',\n",
       " 'C9',\n",
       " 'VPS51',\n",
       " 'MRPL49',\n",
       " 'ZNHIT2',\n",
       " 'MPPED2',\n",
       " 'MYRF',\n",
       " 'TMEM258',\n",
       " 'DAGLA',\n",
       " 'GAS8-AS1',\n",
       " 'FMNL1',\n",
       " 'LDLRAD4',\n",
       " 'PTTG1IP',\n",
       " 'CFAP410',\n",
       " 'TMEM50B',\n",
       " 'MPPED1',\n",
       " 'CA7',\n",
       " 'CA8',\n",
       " 'CA11',\n",
       " 'CA12',\n",
       " 'CACNA1A',\n",
       " 'CACNA1B',\n",
       " 'CACNA1C',\n",
       " 'CACNA1D',\n",
       " 'CACNA1E',\n",
       " 'CACNA2D1',\n",
       " 'CACNB1',\n",
       " 'CACNB2',\n",
       " 'CACNB3',\n",
       " 'CACNB4',\n",
       " 'SLC25A20',\n",
       " 'CAD',\n",
       " 'CALD1',\n",
       " 'CALM1',\n",
       " 'CALM2',\n",
       " 'CALM3',\n",
       " 'CALR',\n",
       " 'CALU',\n",
       " 'CAMK2A',\n",
       " 'CAMK2B',\n",
       " 'CAMK2D',\n",
       " 'CAMK2G',\n",
       " 'CAMLG',\n",
       " 'CANX',\n",
       " 'CAPG',\n",
       " 'CAPN1',\n",
       " 'CAPN2',\n",
       " 'CAPN3',\n",
       " 'CAPNS1',\n",
       " 'CAPN6',\n",
       " 'CAPS',\n",
       " 'CAPZA1',\n",
       " 'CAPZA2',\n",
       " 'CAST',\n",
       " 'CAPZB',\n",
       " 'CARS1',\n",
       " 'CASP1',\n",
       " 'CASP2',\n",
       " 'CASP3',\n",
       " 'CASP4',\n",
       " 'CASP6',\n",
       " 'CASP7',\n",
       " 'CASP8',\n",
       " 'CASP9',\n",
       " 'CASP10',\n",
       " 'CASQ1',\n",
       " 'CAT',\n",
       " 'CAV1',\n",
       " 'CAV2',\n",
       " 'RUNX2',\n",
       " 'RUNX1',\n",
       " 'RUNX1T1',\n",
       " 'CBFA2T3',\n",
       " 'RUNX3',\n",
       " 'CBFB',\n",
       " 'CBL',\n",
       " 'CBLB',\n",
       " 'CBLN1',\n",
       " 'SERPINH1',\n",
       " 'CBR1',\n",
       " 'CBR3',\n",
       " 'KYAT1',\n",
       " 'KRIT1',\n",
       " 'CCNA2',\n",
       " 'CCNB1',\n",
       " 'CCNC',\n",
       " 'CCND2',\n",
       " 'CCND3',\n",
       " 'CCNE1',\n",
       " 'CCNF',\n",
       " 'CCNG1',\n",
       " 'CCNG2',\n",
       " 'CCNH',\n",
       " 'CCNT1',\n",
       " 'CCNT2',\n",
       " 'CCT6A',\n",
       " 'CD1D',\n",
       " 'CD1E',\n",
       " 'CD3G',\n",
       " 'CD4',\n",
       " 'CD5',\n",
       " 'CD5L',\n",
       " 'CD7',\n",
       " 'CD8A',\n",
       " 'CD9',\n",
       " 'CD19',\n",
       " 'MS4A3',\n",
       " 'CD22',\n",
       " 'CD27',\n",
       " 'CD28',\n",
       " 'CD80',\n",
       " 'CD86',\n",
       " 'TNFRSF8',\n",
       " 'TNFSF8',\n",
       " 'CD33',\n",
       " 'SIGLEC6',\n",
       " 'CD36',\n",
       " 'SCARB1',\n",
       " 'SCARB2',\n",
       " 'CD37',\n",
       " 'CD38',\n",
       " 'ENTPD1',\n",
       " 'ENTPD6',\n",
       " 'ENTPD5',\n",
       " 'CD40',\n",
       " 'CD44',\n",
       " 'CD47',\n",
       " 'CD53',\n",
       " 'CD58',\n",
       " 'CD59',\n",
       " 'CD63',\n",
       " 'CD68',\n",
       " 'CD70',\n",
       " 'CD72',\n",
       " 'CD74',\n",
       " 'CD79A',\n",
       " 'CD79B',\n",
       " 'CD81',\n",
       " 'ADGRE5',\n",
       " 'CD151',\n",
       " 'CDA',\n",
       " 'CDK1',\n",
       " 'LRBA',\n",
       " 'CDC5L',\n",
       " 'SEPTIN7',\n",
       " 'CDC6',\n",
       " 'CDC20',\n",
       " 'CDC25A',\n",
       " 'CDC25B',\n",
       " 'CDC25C',\n",
       " 'CDC27',\n",
       " 'CDC34',\n",
       " 'CDC42',\n",
       " 'CDH1',\n",
       " 'CDH2',\n",
       " 'CDH3',\n",
       " 'CDH4',\n",
       " 'CDH6',\n",
       " 'CDH7',\n",
       " 'CDH8',\n",
       " 'CDH12',\n",
       " 'CDH13',\n",
       " 'CDH15',\n",
       " 'CDH18',\n",
       " 'CDK2',\n",
       " 'CDK3',\n",
       " 'CDK4',\n",
       " 'CDK5',\n",
       " 'CDK6',\n",
       " 'CDK8',\n",
       " 'CDK9',\n",
       " 'CDKN1A',\n",
       " 'CDKN1B',\n",
       " 'CDKN2A',\n",
       " 'CDKN2B',\n",
       " 'CDKN2C',\n",
       " 'CDKN2D',\n",
       " 'CDKN3',\n",
       " 'CDR2',\n",
       " 'CDS1',\n",
       " 'CD52',\n",
       " 'CDX1',\n",
       " 'CDX2',\n",
       " 'CEACAM5',\n",
       " 'CEBPA',\n",
       " 'CEBPB',\n",
       " 'CEBPD',\n",
       " 'CEBPE',\n",
       " 'CEBPG',\n",
       " 'CENPA',\n",
       " 'CENPB',\n",
       " 'CENPC',\n",
       " 'CENPE',\n",
       " 'CENPF',\n",
       " 'CETN2',\n",
       " 'CETN3',\n",
       " 'CFL1',\n",
       " 'CFL2',\n",
       " 'CTSC',\n",
       " 'CEACAM8',\n",
       " 'CHAD',\n",
       " 'RCBTB2',\n",
       " 'RCC1',\n",
       " 'CHD1',\n",
       " 'CHD2',\n",
       " 'CHD3',\n",
       " 'CHD4',\n",
       " 'AKR1C4',\n",
       " 'CHEK1',\n",
       " 'FOXN3',\n",
       " 'CHGB',\n",
       " 'CHI3L1',\n",
       " 'CHKA',\n",
       " 'CHKB',\n",
       " 'CHM',\n",
       " 'CHML',\n",
       " 'CHML',\n",
       " 'CHML',\n",
       " 'CHML',\n",
       " 'CHN1',\n",
       " 'LYST',\n",
       " 'CHRM3',\n",
       " 'CHRM5',\n",
       " 'CHRNA2',\n",
       " 'CHRNA3',\n",
       " 'CHRNA5',\n",
       " 'CHRNB1',\n",
       " 'CHRNB2',\n",
       " 'CHRNB4',\n",
       " 'CHRND',\n",
       " 'CHRNE',\n",
       " 'CHRNG',\n",
       " 'CHUK',\n",
       " 'CIDEA',\n",
       " 'CKB',\n",
       " 'CIRBP',\n",
       " 'CISH',\n",
       " 'TBCB',\n",
       " 'CKM',\n",
       " 'ERCC8',\n",
       " 'CKS1B',\n",
       " 'CKS2',\n",
       " 'AP2M1',\n",
       " 'AP1S1',\n",
       " 'AP2S1',\n",
       " 'AP3S1',\n",
       " 'CLCN2',\n",
       " 'CLCN3',\n",
       " 'CLCN4',\n",
       " 'CLCN5',\n",
       " 'CLCN6',\n",
       " 'CLCN7',\n",
       " 'CLU',\n",
       " 'CLIC2',\n",
       " 'CLK1',\n",
       " '1197',\n",
       " 'CLK3',\n",
       " 'TPP1',\n",
       " 'CLN3',\n",
       " 'CLN5',\n",
       " 'CLNS1A',\n",
       " 'CLPTM1',\n",
       " 'CLTA',\n",
       " 'CLTB',\n",
       " 'CLTC',\n",
       " 'CCR1',\n",
       " 'CCR3',\n",
       " 'CCR5',\n",
       " 'CCR6',\n",
       " 'CCR7',\n",
       " 'ACKR2',\n",
       " 'CMKLR1',\n",
       " 'LTB4R',\n",
       " 'ABCC2',\n",
       " 'CNGB1',\n",
       " 'CNGA3',\n",
       " 'CNGA4',\n",
       " 'PLK3',\n",
       " 'CNN2',\n",
       " 'CNN3',\n",
       " 'CNP',\n",
       " 'CNR2',\n",
       " 'CNTF',\n",
       " 'CNTN1',\n",
       " 'COL1A1',\n",
       " 'COL1A2',\n",
       " 'COL4A1',\n",
       " 'COL4A2',\n",
       " 'COL4A3',\n",
       " 'COL4A4',\n",
       " 'COL5A1',\n",
       " 'COL5A2',\n",
       " 'COL6A1',\n",
       " 'COL6A3',\n",
       " 'COL7A1',\n",
       " 'COL8A1',\n",
       " 'COL8A2',\n",
       " 'COL9A1',\n",
       " 'COL9A2',\n",
       " 'COL9A3',\n",
       " 'COL10A1',\n",
       " 'COL11A1',\n",
       " 'COL12A1',\n",
       " 'COL15A1',\n",
       " 'COL16A1',\n",
       " 'COL19A1',\n",
       " 'COMT',\n",
       " 'COPA',\n",
       " 'COPB1',\n",
       " 'KLF6',\n",
       " 'SLC31A1',\n",
       " 'SLC31A2',\n",
       " 'CORT',\n",
       " 'MAP3K8',\n",
       " 'COX4I1',\n",
       " 'COX5B',\n",
       " 'COX6A1',\n",
       " 'COX6B1',\n",
       " 'COX6C',\n",
       " 'COX7A2',\n",
       " 'COX7B',\n",
       " 'COX7C',\n",
       " 'COX8A',\n",
       " 'COX10',\n",
       " 'COX11',\n",
       " 'COX15',\n",
       " 'CP',\n",
       " 'CPB2',\n",
       " 'CPD',\n",
       " 'CLDN4',\n",
       " 'CLDN7',\n",
       " 'CPM',\n",
       " 'CPOX',\n",
       " 'CPS1',\n",
       " 'CPT1A',\n",
       " 'CPT1B',\n",
       " 'CPT2',\n",
       " 'CRABP2',\n",
       " 'CRAT',\n",
       " 'CREB1',\n",
       " 'ATF2',\n",
       " 'CREBBP',\n",
       " 'CREBL2',\n",
       " 'CREM',\n",
       " 'CRH',\n",
       " 'CRIP1',\n",
       " 'CRIP2',\n",
       " 'CRK',\n",
       " 'CRKL',\n",
       " 'CRMP1',\n",
       " 'HAPLN1',\n",
       " 'CRX',\n",
       " 'CRY1',\n",
       " 'CRY2',\n",
       " 'CRYBB1',\n",
       " 'CRYGS',\n",
       " 'CRYZ',\n",
       " 'CS',\n",
       " 'MAPK14',\n",
       " 'CSE1L',\n",
       " 'CSF1',\n",
       " 'CSF1R',\n",
       " 'CSF2RA',\n",
       " 'CSF2RB',\n",
       " 'CSF3R',\n",
       " 'CSK',\n",
       " 'CSNK1A1',\n",
       " 'CSNK1D',\n",
       " 'CSNK1E',\n",
       " 'CSNK1G2',\n",
       " 'CSNK1G3',\n",
       " 'CSNK2A1',\n",
       " 'CSNK2A2',\n",
       " 'VCAN',\n",
       " 'NCAN',\n",
       " 'CSPG4',\n",
       " 'CSRP1',\n",
       " 'SLC25A10',\n",
       " 'CST3',\n",
       " 'CSTA',\n",
       " 'CSTB',\n",
       " 'CSTF1',\n",
       " 'CSTF2',\n",
       " 'CSTF3',\n",
       " 'CTBS',\n",
       " 'CTBP1',\n",
       " 'CTBP2',\n",
       " 'CTF1',\n",
       " 'CTH',\n",
       " 'CTNNA1',\n",
       " 'CTNNA2',\n",
       " 'CTNS',\n",
       " 'CTNNB1',\n",
       " 'CTNND1',\n",
       " 'CTNND2',\n",
       " 'CTPS1',\n",
       " 'CTRL',\n",
       " 'CTSB',\n",
       " 'CTSD',\n",
       " 'CTSG',\n",
       " 'CTSH',\n",
       " 'CTSK',\n",
       " 'CTSV',\n",
       " 'CTSLP3',\n",
       " 'CTSO',\n",
       " 'CTSS',\n",
       " 'CTSW',\n",
       " 'CTSZ',\n",
       " 'CUX1',\n",
       " 'CX3CR1',\n",
       " 'CXADR',\n",
       " 'CYB5A',\n",
       " 'CYB561',\n",
       " 'CYBA',\n",
       " 'CYBB',\n",
       " 'CYC1',\n",
       " 'CYLC2',\n",
       " 'CYLD',\n",
       " 'CYP1A2',\n",
       " 'CYP1B1',\n",
       " 'CYP2B6',\n",
       " 'CYP2B7P',\n",
       " 'CYP2D7',\n",
       " 'CYP2E1',\n",
       " 'CYP2F1',\n",
       " 'CYP3A5',\n",
       " 'CYP17A1',\n",
       " 'CYP27B1',\n",
       " 'CYP51A1',\n",
       " 'DAB2',\n",
       " 'DACH1',\n",
       " 'DAD1',\n",
       " 'CD55',\n",
       " 'DAG1',\n",
       " 'DGKA',\n",
       " 'DGKB',\n",
       " 'DGKG',\n",
       " 'DGKQ',\n",
       " 'DAP',\n",
       " 'DAPK1',\n",
       " 'DAPK3',\n",
       " 'DARS1',\n",
       " 'DAXX',\n",
       " 'DBH',\n",
       " 'DBI',\n",
       " 'DBN1',\n",
       " 'DBP',\n",
       " 'DBT',\n",
       " 'ECI1',\n",
       " 'DCK',\n",
       " 'DCTD',\n",
       " 'DCT',\n",
       " 'DCTN1',\n",
       " 'DCX',\n",
       " 'DDB1',\n",
       " 'DDB2',\n",
       " 'DDC',\n",
       " 'AKR1C1',\n",
       " 'AKR1C2',\n",
       " 'GADD45A',\n",
       " 'DDIT3',\n",
       " 'DDOST',\n",
       " 'DDX1',\n",
       " 'DDX3X',\n",
       " 'DDX5',\n",
       " 'DDX6',\n",
       " 'DMXL1',\n",
       " 'DHX8',\n",
       " 'DHX9',\n",
       " 'DDX10',\n",
       " 'DDX11',\n",
       " 'DHX15',\n",
       " 'DECR1',\n",
       " 'DES',\n",
       " 'CFD',\n",
       " 'DFFA',\n",
       " 'DFFB',\n",
       " 'TIMM8A',\n",
       " 'COCH',\n",
       " 'DGUOK',\n",
       " 'DHCR7',\n",
       " 'DHCR24',\n",
       " 'DHFR',\n",
       " 'DHODH',\n",
       " 'DHPS',\n",
       " 'CYB5R3',\n",
       " 'NQO1',\n",
       " 'DIAPH1',\n",
       " 'DIAPH2',\n",
       " 'SEPTIN1',\n",
       " 'DIO1',\n",
       " 'DIO2',\n",
       " 'DKC1',\n",
       " 'DLAT',\n",
       " 'DLD',\n",
       " 'DLG1',\n",
       " 'DLG2',\n",
       " 'DLG3',\n",
       " 'DLG4',\n",
       " 'DLST',\n",
       " '1744',\n",
       " 'DLX1',\n",
       " 'DLX2',\n",
       " 'DLX3',\n",
       " 'DLX4',\n",
       " 'DLX6',\n",
       " 'DMD',\n",
       " 'SARDH',\n",
       " 'DMP1',\n",
       " 'DNM1',\n",
       " 'DMPK',\n",
       " 'DMWD',\n",
       " 'DNA2',\n",
       " 'DNAH5',\n",
       " 'DNASE1',\n",
       " 'DNASE1L1',\n",
       " 'DNASE1L2',\n",
       " 'DNASE2',\n",
       " 'DYNC1H1',\n",
       " 'DYNC1I2',\n",
       " 'DYNC1LI2',\n",
       " 'DNM2',\n",
       " 'DNMT1',\n",
       " 'TRDMT1',\n",
       " 'DNMT3A',\n",
       " 'DNMT3B',\n",
       " 'DOCK1',\n",
       " 'DOCK2',\n",
       " 'DOCK3',\n",
       " 'DOK1',\n",
       " 'DPAGT1',\n",
       " 'DPH1',\n",
       " 'DPH2',\n",
       " 'DPP4',\n",
       " 'DPP6',\n",
       " 'DPYD',\n",
       " 'DPYSL2',\n",
       " 'DPYSL3',\n",
       " 'DR1',\n",
       " 'DRD5',\n",
       " 'DRG2',\n",
       " 'ARID3A',\n",
       " 'DRP2',\n",
       " 'ATN1',\n",
       " 'DSC1',\n",
       " 'DSC2',\n",
       " 'DSC3',\n",
       " 'DSCAM',\n",
       " 'RCAN1',\n",
       " 'DSG2',\n",
       " 'TSC22D3',\n",
       " 'DSP',\n",
       " 'DSPP',\n",
       " 'SLC26A2',\n",
       " 'DTNA',\n",
       " 'HBEGF',\n",
       " 'DTYMK',\n",
       " 'ECM2',\n",
       " 'DUSP1',\n",
       " 'DUSP2',\n",
       " 'DUSP3',\n",
       " 'DUSP4',\n",
       " 'DUSP5',\n",
       " 'DUSP6',\n",
       " 'DUSP7',\n",
       " 'DUT',\n",
       " 'DVL1',\n",
       " 'DVL2',\n",
       " 'DVL3',\n",
       " 'DYRK1A',\n",
       " 'TOR1A',\n",
       " 'E2F1',\n",
       " 'E2F2',\n",
       " 'E2F3',\n",
       " 'E2F4',\n",
       " 'E2F5',\n",
       " 'E2F6',\n",
       " 'E4F1',\n",
       " 'EBF1',\n",
       " 'GPR183',\n",
       " 'ECE1',\n",
       " 'TYMP',\n",
       " 'ECH1',\n",
       " 'ECHS1',\n",
       " 'ECM1',\n",
       " 'ECT2',\n",
       " 'LPAR1',\n",
       " 'S1PR3',\n",
       " 'EDN1',\n",
       " 'EDNRB',\n",
       " 'PHC1',\n",
       " 'PHC2',\n",
       " 'EEF1A1',\n",
       " 'EEF1A2',\n",
       " 'EEF1B2',\n",
       " 'EEF1G',\n",
       " 'EEF2',\n",
       " 'EIF2D',\n",
       " 'EFNA2',\n",
       " 'EFNA3',\n",
       " 'EFNA4',\n",
       " 'EFNA5',\n",
       " 'EFNB1',\n",
       " 'EFNB2',\n",
       " 'EFNB3',\n",
       " 'EGF',\n",
       " 'CELSR3',\n",
       " 'CELSR2',\n",
       " 'MEGF6',\n",
       " 'MEGF8',\n",
       " 'MEGF9',\n",
       " 'EGFR',\n",
       " 'EGR1',\n",
       " 'EGR2',\n",
       " 'EGR3',\n",
       " 'EGR4',\n",
       " 'EHHADH',\n",
       " 'EIF1AX',\n",
       " 'EIF2S1',\n",
       " 'EIF2B1',\n",
       " 'EIF2S3',\n",
       " 'EPHA2',\n",
       " 'EIF4A1',\n",
       " 'EIF4A2',\n",
       " 'EIF4B',\n",
       " 'EIF4E',\n",
       " 'EIF4EBP1',\n",
       " 'EIF4EBP2',\n",
       " ...]"
      ]
     },
     "execution_count": 196,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "reduce(lambda x, y: pd.merge(x, y, on = \"genes\"), [df1.iloc[:10000], df2.iloc[:10000]])[\"genes\"].tolist()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 197,
   "id": "2db2c891",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "merge_test = reduce(lambda x, y: pd.merge(x, y, on = [\"genes\", \"gene_id\"], how=\"outer\"), [df1, df2])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 198,
   "id": "0a98c3fc",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "test_genes = merge_test[\"genes\"].tolist()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 199,
   "id": "5bc91bb3",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "16648"
      ]
     },
     "execution_count": 199,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(test_genes)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 200,
   "id": "6b566bdc",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "16588"
      ]
     },
     "execution_count": 200,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(genes1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 201,
   "id": "e4c5811c",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "60\n",
      "30\n"
     ]
    }
   ],
   "source": [
    "print(len(list([x for x in genes1 if genes1.count(x) > 1])))\n",
    "print(len(set([x for x in genes1 if genes1.count(x) > 1])))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 152,
   "id": "face4e6e",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "16558"
      ]
     },
     "execution_count": 152,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(set(test_genes))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 153,
   "id": "c21bf7b2",
   "metadata": {
    "collapsed": true,
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'ACBD6',\n",
       " 'ASB3',\n",
       " 'CCDC26',\n",
       " 'CHML',\n",
       " 'FEZ1',\n",
       " 'FNTB',\n",
       " 'GNMT',\n",
       " 'GPRASP2',\n",
       " 'KLHL23',\n",
       " 'KLRK1',\n",
       " 'LINC00511',\n",
       " 'LINC00598',\n",
       " 'LINC01725',\n",
       " 'LIPE-AS1',\n",
       " 'MCL1',\n",
       " 'MEF2B',\n",
       " 'MICAL2',\n",
       " 'MTPN',\n",
       " 'NBL1',\n",
       " 'PAK6',\n",
       " 'PIGY',\n",
       " 'S1PR3',\n",
       " 'SCARNA9',\n",
       " 'SCNM1',\n",
       " 'SMIM8',\n",
       " 'SPSB2',\n",
       " 'STAG3L4',\n",
       " 'TBC1D7',\n",
       " 'TTN-AS1',\n",
       " 'USP9Y'}"
      ]
     },
     "execution_count": 153,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "set([x for x in test_genes if test_genes.count(x) > 1])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 154,
   "id": "b50983fe",
   "metadata": {
    "collapsed": true,
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>baseMean_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>log2FoldChange_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>lfcSE_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>stat_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>pvalue_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>padj_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>genes</th>\n",
       "      <th>gene_id</th>\n",
       "      <th>baseMean_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts</th>\n",
       "      <th>log2FoldChange_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts</th>\n",
       "      <th>lfcSE_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts</th>\n",
       "      <th>stat_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts</th>\n",
       "      <th>pvalue_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts</th>\n",
       "      <th>padj_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>587</th>\n",
       "      <td>1949.022676</td>\n",
       "      <td>0.177454</td>\n",
       "      <td>0.137146</td>\n",
       "      <td>-1.293907</td>\n",
       "      <td>0.195698</td>\n",
       "      <td>0.756115</td>\n",
       "      <td>CHML</td>\n",
       "      <td>CHML</td>\n",
       "      <td>122.062052</td>\n",
       "      <td>0.472268</td>\n",
       "      <td>0.179008</td>\n",
       "      <td>-2.638255</td>\n",
       "      <td>8.333385e-03</td>\n",
       "      <td>1.432467e-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>588</th>\n",
       "      <td>1949.022676</td>\n",
       "      <td>0.177454</td>\n",
       "      <td>0.137146</td>\n",
       "      <td>-1.293907</td>\n",
       "      <td>0.195698</td>\n",
       "      <td>0.756115</td>\n",
       "      <td>CHML</td>\n",
       "      <td>CHML</td>\n",
       "      <td>57.521996</td>\n",
       "      <td>1.500506</td>\n",
       "      <td>0.181365</td>\n",
       "      <td>-8.273385</td>\n",
       "      <td>1.302081e-16</td>\n",
       "      <td>3.976554e-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>589</th>\n",
       "      <td>1221.280092</td>\n",
       "      <td>0.152483</td>\n",
       "      <td>0.090387</td>\n",
       "      <td>-1.687005</td>\n",
       "      <td>0.091602</td>\n",
       "      <td>0.700413</td>\n",
       "      <td>CHML</td>\n",
       "      <td>CHML</td>\n",
       "      <td>122.062052</td>\n",
       "      <td>0.472268</td>\n",
       "      <td>0.179008</td>\n",
       "      <td>-2.638255</td>\n",
       "      <td>8.333385e-03</td>\n",
       "      <td>1.432467e-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>590</th>\n",
       "      <td>1221.280092</td>\n",
       "      <td>0.152483</td>\n",
       "      <td>0.090387</td>\n",
       "      <td>-1.687005</td>\n",
       "      <td>0.091602</td>\n",
       "      <td>0.700413</td>\n",
       "      <td>CHML</td>\n",
       "      <td>CHML</td>\n",
       "      <td>57.521996</td>\n",
       "      <td>1.500506</td>\n",
       "      <td>0.181365</td>\n",
       "      <td>-8.273385</td>\n",
       "      <td>1.302081e-16</td>\n",
       "      <td>3.976554e-14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>959</th>\n",
       "      <td>209.879610</td>\n",
       "      <td>-0.413183</td>\n",
       "      <td>0.124792</td>\n",
       "      <td>3.310983</td>\n",
       "      <td>0.000930</td>\n",
       "      <td>0.058866</td>\n",
       "      <td>S1PR3</td>\n",
       "      <td>S1PR3</td>\n",
       "      <td>35.707272</td>\n",
       "      <td>-0.391967</td>\n",
       "      <td>0.194360</td>\n",
       "      <td>2.016711</td>\n",
       "      <td>4.372564e-02</td>\n",
       "      <td>3.978295e-01</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15759</th>\n",
       "      <td>35.798328</td>\n",
       "      <td>0.089758</td>\n",
       "      <td>0.216731</td>\n",
       "      <td>-0.414143</td>\n",
       "      <td>0.678769</td>\n",
       "      <td>0.918913</td>\n",
       "      <td>LIPE-AS1</td>\n",
       "      <td>LIPE-AS1</td>\n",
       "      <td>1.000995</td>\n",
       "      <td>-0.538208</td>\n",
       "      <td>1.091397</td>\n",
       "      <td>0.493137</td>\n",
       "      <td>6.219158e-01</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15916</th>\n",
       "      <td>1.609619</td>\n",
       "      <td>-0.520566</td>\n",
       "      <td>1.020085</td>\n",
       "      <td>0.510316</td>\n",
       "      <td>0.609830</td>\n",
       "      <td>NaN</td>\n",
       "      <td>LINC01725</td>\n",
       "      <td>LINC01725</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>-0.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15917</th>\n",
       "      <td>1.609619</td>\n",
       "      <td>-0.520566</td>\n",
       "      <td>1.020085</td>\n",
       "      <td>0.510316</td>\n",
       "      <td>0.609830</td>\n",
       "      <td>NaN</td>\n",
       "      <td>LINC01725</td>\n",
       "      <td>LINC01725</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>-0.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15918</th>\n",
       "      <td>8.320226</td>\n",
       "      <td>0.301544</td>\n",
       "      <td>0.436010</td>\n",
       "      <td>-0.691598</td>\n",
       "      <td>0.489190</td>\n",
       "      <td>NaN</td>\n",
       "      <td>LINC01725</td>\n",
       "      <td>LINC01725</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>-0.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15919</th>\n",
       "      <td>8.320226</td>\n",
       "      <td>0.301544</td>\n",
       "      <td>0.436010</td>\n",
       "      <td>-0.691598</td>\n",
       "      <td>0.489190</td>\n",
       "      <td>NaN</td>\n",
       "      <td>LINC01725</td>\n",
       "      <td>LINC01725</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>-0.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>120 rows Ã— 14 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "       baseMean_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "587                                          1949.022676                 \n",
       "588                                          1949.022676                 \n",
       "589                                          1221.280092                 \n",
       "590                                          1221.280092                 \n",
       "959                                           209.879610                 \n",
       "...                                                  ...                 \n",
       "15759                                          35.798328                 \n",
       "15916                                           1.609619                 \n",
       "15917                                           1.609619                 \n",
       "15918                                           8.320226                 \n",
       "15919                                           8.320226                 \n",
       "\n",
       "       log2FoldChange_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "587                                             0.177454                       \n",
       "588                                             0.177454                       \n",
       "589                                             0.152483                       \n",
       "590                                             0.152483                       \n",
       "959                                            -0.413183                       \n",
       "...                                                  ...                       \n",
       "15759                                           0.089758                       \n",
       "15916                                          -0.520566                       \n",
       "15917                                          -0.520566                       \n",
       "15918                                           0.301544                       \n",
       "15919                                           0.301544                       \n",
       "\n",
       "       lfcSE_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "587                                             0.137146              \n",
       "588                                             0.137146              \n",
       "589                                             0.090387              \n",
       "590                                             0.090387              \n",
       "959                                             0.124792              \n",
       "...                                                  ...              \n",
       "15759                                           0.216731              \n",
       "15916                                           1.020085              \n",
       "15917                                           1.020085              \n",
       "15918                                           0.436010              \n",
       "15919                                           0.436010              \n",
       "\n",
       "       stat_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "587                                            -1.293907             \n",
       "588                                            -1.293907             \n",
       "589                                            -1.687005             \n",
       "590                                            -1.687005             \n",
       "959                                             3.310983             \n",
       "...                                                  ...             \n",
       "15759                                          -0.414143             \n",
       "15916                                           0.510316             \n",
       "15917                                           0.510316             \n",
       "15918                                          -0.691598             \n",
       "15919                                          -0.691598             \n",
       "\n",
       "       pvalue_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "587                                             0.195698               \n",
       "588                                             0.195698               \n",
       "589                                             0.091602               \n",
       "590                                             0.091602               \n",
       "959                                             0.000930               \n",
       "...                                                  ...               \n",
       "15759                                           0.678769               \n",
       "15916                                           0.609830               \n",
       "15917                                           0.609830               \n",
       "15918                                           0.489190               \n",
       "15919                                           0.489190               \n",
       "\n",
       "       padj_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts      genes  \\\n",
       "587                                             0.756115                 CHML   \n",
       "588                                             0.756115                 CHML   \n",
       "589                                             0.700413                 CHML   \n",
       "590                                             0.700413                 CHML   \n",
       "959                                             0.058866                S1PR3   \n",
       "...                                                  ...                  ...   \n",
       "15759                                           0.918913             LIPE-AS1   \n",
       "15916                                                NaN            LINC01725   \n",
       "15917                                                NaN            LINC01725   \n",
       "15918                                                NaN            LINC01725   \n",
       "15919                                                NaN            LINC01725   \n",
       "\n",
       "         gene_id  \\\n",
       "587         CHML   \n",
       "588         CHML   \n",
       "589         CHML   \n",
       "590         CHML   \n",
       "959        S1PR3   \n",
       "...          ...   \n",
       "15759   LIPE-AS1   \n",
       "15916  LINC01725   \n",
       "15917  LINC01725   \n",
       "15918  LINC01725   \n",
       "15919  LINC01725   \n",
       "\n",
       "       baseMean_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts  \\\n",
       "587                                           122.062052               \n",
       "588                                            57.521996               \n",
       "589                                           122.062052               \n",
       "590                                            57.521996               \n",
       "959                                            35.707272               \n",
       "...                                                  ...               \n",
       "15759                                           1.000995               \n",
       "15916                                           0.000000               \n",
       "15917                                           0.000000               \n",
       "15918                                           0.000000               \n",
       "15919                                           0.000000               \n",
       "\n",
       "       log2FoldChange_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts  \\\n",
       "587                                             0.472268                     \n",
       "588                                             1.500506                     \n",
       "589                                             0.472268                     \n",
       "590                                             1.500506                     \n",
       "959                                            -0.391967                     \n",
       "...                                                  ...                     \n",
       "15759                                          -0.538208                     \n",
       "15916                                          -0.000000                     \n",
       "15917                                          -0.000000                     \n",
       "15918                                          -0.000000                     \n",
       "15919                                          -0.000000                     \n",
       "\n",
       "       lfcSE_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts  \\\n",
       "587                                             0.179008            \n",
       "588                                             0.181365            \n",
       "589                                             0.179008            \n",
       "590                                             0.181365            \n",
       "959                                             0.194360            \n",
       "...                                                  ...            \n",
       "15759                                           1.091397            \n",
       "15916                                                NaN            \n",
       "15917                                                NaN            \n",
       "15918                                                NaN            \n",
       "15919                                                NaN            \n",
       "\n",
       "       stat_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts  \\\n",
       "587                                            -2.638255           \n",
       "588                                            -8.273385           \n",
       "589                                            -2.638255           \n",
       "590                                            -8.273385           \n",
       "959                                             2.016711           \n",
       "...                                                  ...           \n",
       "15759                                           0.493137           \n",
       "15916                                                NaN           \n",
       "15917                                                NaN           \n",
       "15918                                                NaN           \n",
       "15919                                                NaN           \n",
       "\n",
       "       pvalue_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts  \\\n",
       "587                                         8.333385e-03             \n",
       "588                                         1.302081e-16             \n",
       "589                                         8.333385e-03             \n",
       "590                                         1.302081e-16             \n",
       "959                                         4.372564e-02             \n",
       "...                                                  ...             \n",
       "15759                                       6.219158e-01             \n",
       "15916                                       0.000000e+00             \n",
       "15917                                       0.000000e+00             \n",
       "15918                                       0.000000e+00             \n",
       "15919                                       0.000000e+00             \n",
       "\n",
       "       padj_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts  \n",
       "587                                         1.432467e-01          \n",
       "588                                         3.976554e-14          \n",
       "589                                         1.432467e-01          \n",
       "590                                         3.976554e-14          \n",
       "959                                         3.978295e-01          \n",
       "...                                                  ...          \n",
       "15759                                                NaN          \n",
       "15916                                                NaN          \n",
       "15917                                                NaN          \n",
       "15918                                                NaN          \n",
       "15919                                                NaN          \n",
       "\n",
       "[120 rows x 14 columns]"
      ]
     },
     "execution_count": 154,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "merge_test.loc[merge_test[\"genes\"].isin(set([x for x in test_genes if test_genes.count(x) > 1]))]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "d57b2c0a",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "cb14bc7c",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 114,
   "id": "581bede9",
   "metadata": {
    "collapsed": true,
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>baseMean_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>log2FoldChange_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>lfcSE_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>stat_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>pvalue_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>padj_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>genes</th>\n",
       "      <th>gene_id</th>\n",
       "      <th>baseMean_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts</th>\n",
       "      <th>log2FoldChange_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts</th>\n",
       "      <th>...</th>\n",
       "      <th>lfcSE_MYB_degraded_v1_0_5_1h_deseq_readcounts</th>\n",
       "      <th>stat_MYB_degraded_v1_0_5_1h_deseq_readcounts</th>\n",
       "      <th>pvalue_MYB_degraded_v1_0_5_1h_deseq_readcounts</th>\n",
       "      <th>padj_MYB_degraded_v1_0_5_1h_deseq_readcounts</th>\n",
       "      <th>baseMean_MYB_degraded_v1_0_5_1h_deseq_tccounts</th>\n",
       "      <th>log2FoldChange_MYB_degraded_v1_0_5_1h_deseq_tccounts</th>\n",
       "      <th>lfcSE_MYB_degraded_v1_0_5_1h_deseq_tccounts</th>\n",
       "      <th>stat_MYB_degraded_v1_0_5_1h_deseq_tccounts</th>\n",
       "      <th>pvalue_MYB_degraded_v1_0_5_1h_deseq_tccounts</th>\n",
       "      <th>padj_MYB_degraded_v1_0_5_1h_deseq_tccounts</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1.877539</td>\n",
       "      <td>-0.045684</td>\n",
       "      <td>1.010547</td>\n",
       "      <td>0.045207</td>\n",
       "      <td>0.963942</td>\n",
       "      <td>NaN</td>\n",
       "      <td>A1BG</td>\n",
       "      <td>A1BG</td>\n",
       "      <td>0.058385</td>\n",
       "      <td>-0.615170</td>\n",
       "      <td>...</td>\n",
       "      <td>0.980114</td>\n",
       "      <td>-0.116922</td>\n",
       "      <td>0.906922</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.058385</td>\n",
       "      <td>-0.500626</td>\n",
       "      <td>3.116540</td>\n",
       "      <td>0.160635</td>\n",
       "      <td>0.872381</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>5.229387</td>\n",
       "      <td>-0.269041</td>\n",
       "      <td>0.500087</td>\n",
       "      <td>0.537989</td>\n",
       "      <td>0.590585</td>\n",
       "      <td>NaN</td>\n",
       "      <td>A2MP1</td>\n",
       "      <td>A2MP1</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>-0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.477972</td>\n",
       "      <td>0.372795</td>\n",
       "      <td>0.709301</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>-0.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>58.516396</td>\n",
       "      <td>0.013917</td>\n",
       "      <td>0.186478</td>\n",
       "      <td>-0.074632</td>\n",
       "      <td>0.940507</td>\n",
       "      <td>0.987830</td>\n",
       "      <td>NAT1</td>\n",
       "      <td>NAT1</td>\n",
       "      <td>4.852990</td>\n",
       "      <td>0.551270</td>\n",
       "      <td>...</td>\n",
       "      <td>0.156367</td>\n",
       "      <td>-0.758833</td>\n",
       "      <td>0.447953</td>\n",
       "      <td>0.999815</td>\n",
       "      <td>5.267543</td>\n",
       "      <td>0.666107</td>\n",
       "      <td>0.567477</td>\n",
       "      <td>-1.173804</td>\n",
       "      <td>0.240473</td>\n",
       "      <td>0.774211</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>2.316444</td>\n",
       "      <td>-0.735572</td>\n",
       "      <td>0.776358</td>\n",
       "      <td>0.947465</td>\n",
       "      <td>0.343402</td>\n",
       "      <td>NaN</td>\n",
       "      <td>AADAC</td>\n",
       "      <td>AADAC</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>-0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td>0.748054</td>\n",
       "      <td>0.906933</td>\n",
       "      <td>0.364442</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>-0.000000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>340.667352</td>\n",
       "      <td>-0.130732</td>\n",
       "      <td>0.092874</td>\n",
       "      <td>1.407626</td>\n",
       "      <td>0.159242</td>\n",
       "      <td>0.743147</td>\n",
       "      <td>AAMP</td>\n",
       "      <td>AAMP</td>\n",
       "      <td>9.364263</td>\n",
       "      <td>0.215343</td>\n",
       "      <td>...</td>\n",
       "      <td>0.074923</td>\n",
       "      <td>0.335587</td>\n",
       "      <td>0.737182</td>\n",
       "      <td>0.999815</td>\n",
       "      <td>9.852376</td>\n",
       "      <td>0.296100</td>\n",
       "      <td>0.338771</td>\n",
       "      <td>-0.874041</td>\n",
       "      <td>0.382096</td>\n",
       "      <td>0.864729</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17003</th>\n",
       "      <td>82.429753</td>\n",
       "      <td>0.008600</td>\n",
       "      <td>0.168839</td>\n",
       "      <td>-0.050936</td>\n",
       "      <td>0.959376</td>\n",
       "      <td>0.990346</td>\n",
       "      <td>106736475</td>\n",
       "      <td>106736475</td>\n",
       "      <td>6.090411</td>\n",
       "      <td>0.107138</td>\n",
       "      <td>...</td>\n",
       "      <td>0.142142</td>\n",
       "      <td>-0.825469</td>\n",
       "      <td>0.409105</td>\n",
       "      <td>0.999815</td>\n",
       "      <td>6.362049</td>\n",
       "      <td>0.189380</td>\n",
       "      <td>0.438964</td>\n",
       "      <td>-0.431426</td>\n",
       "      <td>0.666159</td>\n",
       "      <td>0.953370</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17004</th>\n",
       "      <td>13.209258</td>\n",
       "      <td>-0.102026</td>\n",
       "      <td>0.308889</td>\n",
       "      <td>0.330301</td>\n",
       "      <td>0.741173</td>\n",
       "      <td>0.934870</td>\n",
       "      <td>GET1-SH3BGR</td>\n",
       "      <td>GET1-SH3BGR</td>\n",
       "      <td>0.144147</td>\n",
       "      <td>-1.102433</td>\n",
       "      <td>...</td>\n",
       "      <td>0.294363</td>\n",
       "      <td>-0.007729</td>\n",
       "      <td>0.993833</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.144147</td>\n",
       "      <td>-0.987976</td>\n",
       "      <td>2.583608</td>\n",
       "      <td>0.382402</td>\n",
       "      <td>0.702163</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17005</th>\n",
       "      <td>3.067087</td>\n",
       "      <td>-0.096211</td>\n",
       "      <td>0.639843</td>\n",
       "      <td>0.150367</td>\n",
       "      <td>0.880475</td>\n",
       "      <td>NaN</td>\n",
       "      <td>BOLA2-SMG1P6</td>\n",
       "      <td>BOLA2-SMG1P6</td>\n",
       "      <td>0.140468</td>\n",
       "      <td>-0.134282</td>\n",
       "      <td>...</td>\n",
       "      <td>0.619041</td>\n",
       "      <td>-0.022964</td>\n",
       "      <td>0.981679</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.146822</td>\n",
       "      <td>-0.019737</td>\n",
       "      <td>2.568065</td>\n",
       "      <td>0.007686</td>\n",
       "      <td>0.993868</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17006</th>\n",
       "      <td>153.879523</td>\n",
       "      <td>0.023166</td>\n",
       "      <td>0.124540</td>\n",
       "      <td>-0.186016</td>\n",
       "      <td>0.852432</td>\n",
       "      <td>0.968322</td>\n",
       "      <td>107303344</td>\n",
       "      <td>107303344</td>\n",
       "      <td>3.557956</td>\n",
       "      <td>0.031424</td>\n",
       "      <td>...</td>\n",
       "      <td>0.109675</td>\n",
       "      <td>-1.192979</td>\n",
       "      <td>0.232878</td>\n",
       "      <td>0.999815</td>\n",
       "      <td>3.741420</td>\n",
       "      <td>0.118501</td>\n",
       "      <td>0.541308</td>\n",
       "      <td>-0.218916</td>\n",
       "      <td>0.826715</td>\n",
       "      <td>0.974970</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17007</th>\n",
       "      <td>2.831265</td>\n",
       "      <td>-0.606622</td>\n",
       "      <td>0.663979</td>\n",
       "      <td>0.913616</td>\n",
       "      <td>0.360919</td>\n",
       "      <td>NaN</td>\n",
       "      <td>SCARNA18B</td>\n",
       "      <td>SCARNA18B</td>\n",
       "      <td>0.062718</td>\n",
       "      <td>0.346615</td>\n",
       "      <td>...</td>\n",
       "      <td>0.643304</td>\n",
       "      <td>0.813413</td>\n",
       "      <td>0.415981</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.065232</td>\n",
       "      <td>0.461159</td>\n",
       "      <td>3.116540</td>\n",
       "      <td>-0.147972</td>\n",
       "      <td>0.882365</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>17008 rows Ã— 26 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "       baseMean_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "0                                               1.877539                 \n",
       "1                                               5.229387                 \n",
       "2                                              58.516396                 \n",
       "3                                               2.316444                 \n",
       "4                                             340.667352                 \n",
       "...                                                  ...                 \n",
       "17003                                          82.429753                 \n",
       "17004                                          13.209258                 \n",
       "17005                                           3.067087                 \n",
       "17006                                         153.879523                 \n",
       "17007                                           2.831265                 \n",
       "\n",
       "       log2FoldChange_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "0                                              -0.045684                       \n",
       "1                                              -0.269041                       \n",
       "2                                               0.013917                       \n",
       "3                                              -0.735572                       \n",
       "4                                              -0.130732                       \n",
       "...                                                  ...                       \n",
       "17003                                           0.008600                       \n",
       "17004                                          -0.102026                       \n",
       "17005                                          -0.096211                       \n",
       "17006                                           0.023166                       \n",
       "17007                                          -0.606622                       \n",
       "\n",
       "       lfcSE_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "0                                               1.010547              \n",
       "1                                               0.500087              \n",
       "2                                               0.186478              \n",
       "3                                               0.776358              \n",
       "4                                               0.092874              \n",
       "...                                                  ...              \n",
       "17003                                           0.168839              \n",
       "17004                                           0.308889              \n",
       "17005                                           0.639843              \n",
       "17006                                           0.124540              \n",
       "17007                                           0.663979              \n",
       "\n",
       "       stat_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "0                                               0.045207             \n",
       "1                                               0.537989             \n",
       "2                                              -0.074632             \n",
       "3                                               0.947465             \n",
       "4                                               1.407626             \n",
       "...                                                  ...             \n",
       "17003                                          -0.050936             \n",
       "17004                                           0.330301             \n",
       "17005                                           0.150367             \n",
       "17006                                          -0.186016             \n",
       "17007                                           0.913616             \n",
       "\n",
       "       pvalue_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "0                                               0.963942               \n",
       "1                                               0.590585               \n",
       "2                                               0.940507               \n",
       "3                                               0.343402               \n",
       "4                                               0.159242               \n",
       "...                                                  ...               \n",
       "17003                                           0.959376               \n",
       "17004                                           0.741173               \n",
       "17005                                           0.880475               \n",
       "17006                                           0.852432               \n",
       "17007                                           0.360919               \n",
       "\n",
       "       padj_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "0                                                    NaN             \n",
       "1                                                    NaN             \n",
       "2                                               0.987830             \n",
       "3                                                    NaN             \n",
       "4                                               0.743147             \n",
       "...                                                  ...             \n",
       "17003                                           0.990346             \n",
       "17004                                           0.934870             \n",
       "17005                                                NaN             \n",
       "17006                                           0.968322             \n",
       "17007                                                NaN             \n",
       "\n",
       "              genes       gene_id  \\\n",
       "0              A1BG          A1BG   \n",
       "1             A2MP1         A2MP1   \n",
       "2              NAT1          NAT1   \n",
       "3             AADAC         AADAC   \n",
       "4              AAMP          AAMP   \n",
       "...             ...           ...   \n",
       "17003     106736475     106736475   \n",
       "17004   GET1-SH3BGR   GET1-SH3BGR   \n",
       "17005  BOLA2-SMG1P6  BOLA2-SMG1P6   \n",
       "17006     107303344     107303344   \n",
       "17007     SCARNA18B     SCARNA18B   \n",
       "\n",
       "       baseMean_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts  \\\n",
       "0                                               0.058385               \n",
       "1                                               0.000000               \n",
       "2                                               4.852990               \n",
       "3                                               0.000000               \n",
       "4                                               9.364263               \n",
       "...                                                  ...               \n",
       "17003                                           6.090411               \n",
       "17004                                           0.144147               \n",
       "17005                                           0.140468               \n",
       "17006                                           3.557956               \n",
       "17007                                           0.062718               \n",
       "\n",
       "       log2FoldChange_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts  \\\n",
       "0                                              -0.615170                     \n",
       "1                                              -0.000000                     \n",
       "2                                               0.551270                     \n",
       "3                                              -0.000000                     \n",
       "4                                               0.215343                     \n",
       "...                                                  ...                     \n",
       "17003                                           0.107138                     \n",
       "17004                                          -1.102433                     \n",
       "17005                                          -0.134282                     \n",
       "17006                                           0.031424                     \n",
       "17007                                           0.346615                     \n",
       "\n",
       "       ...  lfcSE_MYB_degraded_v1_0_5_1h_deseq_readcounts  \\\n",
       "0      ...                                       0.980114   \n",
       "1      ...                                       0.477972   \n",
       "2      ...                                       0.156367   \n",
       "3      ...                                       0.748054   \n",
       "4      ...                                       0.074923   \n",
       "...    ...                                            ...   \n",
       "17003  ...                                       0.142142   \n",
       "17004  ...                                       0.294363   \n",
       "17005  ...                                       0.619041   \n",
       "17006  ...                                       0.109675   \n",
       "17007  ...                                       0.643304   \n",
       "\n",
       "       stat_MYB_degraded_v1_0_5_1h_deseq_readcounts  \\\n",
       "0                                         -0.116922   \n",
       "1                                          0.372795   \n",
       "2                                         -0.758833   \n",
       "3                                          0.906933   \n",
       "4                                          0.335587   \n",
       "...                                             ...   \n",
       "17003                                     -0.825469   \n",
       "17004                                     -0.007729   \n",
       "17005                                     -0.022964   \n",
       "17006                                     -1.192979   \n",
       "17007                                      0.813413   \n",
       "\n",
       "       pvalue_MYB_degraded_v1_0_5_1h_deseq_readcounts  \\\n",
       "0                                            0.906922   \n",
       "1                                            0.709301   \n",
       "2                                            0.447953   \n",
       "3                                            0.364442   \n",
       "4                                            0.737182   \n",
       "...                                               ...   \n",
       "17003                                        0.409105   \n",
       "17004                                        0.993833   \n",
       "17005                                        0.981679   \n",
       "17006                                        0.232878   \n",
       "17007                                        0.415981   \n",
       "\n",
       "       padj_MYB_degraded_v1_0_5_1h_deseq_readcounts  \\\n",
       "0                                               NaN   \n",
       "1                                               NaN   \n",
       "2                                          0.999815   \n",
       "3                                               NaN   \n",
       "4                                          0.999815   \n",
       "...                                             ...   \n",
       "17003                                      0.999815   \n",
       "17004                                           NaN   \n",
       "17005                                           NaN   \n",
       "17006                                      0.999815   \n",
       "17007                                           NaN   \n",
       "\n",
       "       baseMean_MYB_degraded_v1_0_5_1h_deseq_tccounts  \\\n",
       "0                                            0.058385   \n",
       "1                                            0.000000   \n",
       "2                                            5.267543   \n",
       "3                                            0.000000   \n",
       "4                                            9.852376   \n",
       "...                                               ...   \n",
       "17003                                        6.362049   \n",
       "17004                                        0.144147   \n",
       "17005                                        0.146822   \n",
       "17006                                        3.741420   \n",
       "17007                                        0.065232   \n",
       "\n",
       "       log2FoldChange_MYB_degraded_v1_0_5_1h_deseq_tccounts  \\\n",
       "0                                              -0.500626      \n",
       "1                                              -0.000000      \n",
       "2                                               0.666107      \n",
       "3                                              -0.000000      \n",
       "4                                               0.296100      \n",
       "...                                                  ...      \n",
       "17003                                           0.189380      \n",
       "17004                                          -0.987976      \n",
       "17005                                          -0.019737      \n",
       "17006                                           0.118501      \n",
       "17007                                           0.461159      \n",
       "\n",
       "       lfcSE_MYB_degraded_v1_0_5_1h_deseq_tccounts  \\\n",
       "0                                         3.116540   \n",
       "1                                              NaN   \n",
       "2                                         0.567477   \n",
       "3                                              NaN   \n",
       "4                                         0.338771   \n",
       "...                                            ...   \n",
       "17003                                     0.438964   \n",
       "17004                                     2.583608   \n",
       "17005                                     2.568065   \n",
       "17006                                     0.541308   \n",
       "17007                                     3.116540   \n",
       "\n",
       "       stat_MYB_degraded_v1_0_5_1h_deseq_tccounts  \\\n",
       "0                                        0.160635   \n",
       "1                                             NaN   \n",
       "2                                       -1.173804   \n",
       "3                                             NaN   \n",
       "4                                       -0.874041   \n",
       "...                                           ...   \n",
       "17003                                   -0.431426   \n",
       "17004                                    0.382402   \n",
       "17005                                    0.007686   \n",
       "17006                                   -0.218916   \n",
       "17007                                   -0.147972   \n",
       "\n",
       "       pvalue_MYB_degraded_v1_0_5_1h_deseq_tccounts  \\\n",
       "0                                          0.872381   \n",
       "1                                          0.000000   \n",
       "2                                          0.240473   \n",
       "3                                          0.000000   \n",
       "4                                          0.382096   \n",
       "...                                             ...   \n",
       "17003                                      0.666159   \n",
       "17004                                      0.702163   \n",
       "17005                                      0.993868   \n",
       "17006                                      0.826715   \n",
       "17007                                      0.882365   \n",
       "\n",
       "       padj_MYB_degraded_v1_0_5_1h_deseq_tccounts  \n",
       "0                                             NaN  \n",
       "1                                             NaN  \n",
       "2                                        0.774211  \n",
       "3                                             NaN  \n",
       "4                                        0.864729  \n",
       "...                                           ...  \n",
       "17003                                    0.953370  \n",
       "17004                                         NaN  \n",
       "17005                                         NaN  \n",
       "17006                                    0.974970  \n",
       "17007                                         NaN  \n",
       "\n",
       "[17008 rows x 26 columns]"
      ]
     },
     "execution_count": 114,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "reduce(lambda x, y: pd.merge(x, y, on = [\"genes\", \"gene_id\"], how=\"outer\"), deseq_dfs)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "cdf094d0",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "800b533b",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 90,
   "id": "76c050fe",
   "metadata": {
    "collapsed": true,
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(16588, 2)\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>genes</th>\n",
       "      <th>gene_id</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>A1BG</td>\n",
       "      <td>A1BG</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>A2MP1</td>\n",
       "      <td>A2MP1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>NAT1</td>\n",
       "      <td>NAT1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>AADAC</td>\n",
       "      <td>AADAC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>AAMP</td>\n",
       "      <td>AAMP</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   genes gene_id\n",
       "0   A1BG    A1BG\n",
       "1  A2MP1   A2MP1\n",
       "2   NAT1    NAT1\n",
       "3  AADAC   AADAC\n",
       "4   AAMP    AAMP"
      ]
     },
     "execution_count": 90,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# main df\n",
    "df_merged = pd.DataFrame()\n",
    "df_merged = df_merged.append(deseq_dfs[0][[\"genes\", \"gene_id\"]]) # add gene cols to main df\n",
    "print(df_merged.shape)\n",
    "df_merged.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 91,
   "id": "367d94c5",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(17488, 26)\n"
     ]
    }
   ],
   "source": [
    "# join df columns on gene & gene_id columns\n",
    "df_merged = reduce(lambda left, right: pd.merge(left, right, on=['genes', \"gene_id\"], how=\"outer\"), \n",
    "                   [df_merged] + deseq_dfs)\n",
    "print(df_merged.shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 79,
   "id": "7b4a721e",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "# change column names\n",
    "df_merged.columns = [col_name.split(\"_\")[0] if \"gene\" not in col_name else \n",
    "                     col_name for col_name in df_merged.columns.tolist()]\n",
    "df_merged = df_merged.T.reset_index().T.reset_index(drop=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 80,
   "id": "862e7eaf",
   "metadata": {
    "collapsed": true,
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "      <th>1</th>\n",
       "      <th>2</th>\n",
       "      <th>3</th>\n",
       "      <th>4</th>\n",
       "      <th>5</th>\n",
       "      <th>6</th>\n",
       "      <th>7</th>\n",
       "      <th>8</th>\n",
       "      <th>9</th>\n",
       "      <th>...</th>\n",
       "      <th>16</th>\n",
       "      <th>17</th>\n",
       "      <th>18</th>\n",
       "      <th>19</th>\n",
       "      <th>20</th>\n",
       "      <th>21</th>\n",
       "      <th>22</th>\n",
       "      <th>23</th>\n",
       "      <th>24</th>\n",
       "      <th>25</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>genes</td>\n",
       "      <td>gene_id</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>...</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "      <td>baseMean</td>\n",
       "      <td>log2FoldChange</td>\n",
       "      <td>lfcSE</td>\n",
       "      <td>stat</td>\n",
       "      <td>pvalue</td>\n",
       "      <td>padj</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>A1BG</td>\n",
       "      <td>A1BG</td>\n",
       "      <td>1.877539</td>\n",
       "      <td>-0.045684</td>\n",
       "      <td>1.010547</td>\n",
       "      <td>0.045207</td>\n",
       "      <td>0.963942</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.058385</td>\n",
       "      <td>-0.61517</td>\n",
       "      <td>...</td>\n",
       "      <td>0.980114</td>\n",
       "      <td>-0.116922</td>\n",
       "      <td>0.906922</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.058385</td>\n",
       "      <td>-0.500626</td>\n",
       "      <td>3.11654</td>\n",
       "      <td>0.160635</td>\n",
       "      <td>0.872381</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>A2MP1</td>\n",
       "      <td>A2MP1</td>\n",
       "      <td>5.229387</td>\n",
       "      <td>-0.269041</td>\n",
       "      <td>0.500087</td>\n",
       "      <td>0.537989</td>\n",
       "      <td>0.590585</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>-0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.477972</td>\n",
       "      <td>0.372795</td>\n",
       "      <td>0.709301</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>-0.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>NAT1</td>\n",
       "      <td>NAT1</td>\n",
       "      <td>58.516396</td>\n",
       "      <td>0.013917</td>\n",
       "      <td>0.186478</td>\n",
       "      <td>-0.074632</td>\n",
       "      <td>0.940507</td>\n",
       "      <td>0.98783</td>\n",
       "      <td>4.85299</td>\n",
       "      <td>0.55127</td>\n",
       "      <td>...</td>\n",
       "      <td>0.156367</td>\n",
       "      <td>-0.758833</td>\n",
       "      <td>0.447953</td>\n",
       "      <td>0.999815</td>\n",
       "      <td>5.267543</td>\n",
       "      <td>0.666107</td>\n",
       "      <td>0.567477</td>\n",
       "      <td>-1.173804</td>\n",
       "      <td>0.240473</td>\n",
       "      <td>0.774211</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>AADAC</td>\n",
       "      <td>AADAC</td>\n",
       "      <td>2.316444</td>\n",
       "      <td>-0.735572</td>\n",
       "      <td>0.776358</td>\n",
       "      <td>0.947465</td>\n",
       "      <td>0.343402</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>-0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.748054</td>\n",
       "      <td>0.906933</td>\n",
       "      <td>0.364442</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>-0.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows Ã— 26 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "      0        1          2               3         4         5         6   \\\n",
       "0  genes  gene_id   baseMean  log2FoldChange     lfcSE      stat    pvalue   \n",
       "1   A1BG     A1BG   1.877539       -0.045684  1.010547  0.045207  0.963942   \n",
       "2  A2MP1    A2MP1   5.229387       -0.269041  0.500087  0.537989  0.590585   \n",
       "3   NAT1     NAT1  58.516396        0.013917  0.186478 -0.074632  0.940507   \n",
       "4  AADAC    AADAC   2.316444       -0.735572  0.776358  0.947465  0.343402   \n",
       "\n",
       "        7         8               9   ...        16        17        18  \\\n",
       "0     padj  baseMean  log2FoldChange  ...     lfcSE      stat    pvalue   \n",
       "1      NaN  0.058385        -0.61517  ...  0.980114 -0.116922  0.906922   \n",
       "2      NaN       0.0            -0.0  ...  0.477972  0.372795  0.709301   \n",
       "3  0.98783   4.85299         0.55127  ...  0.156367 -0.758833  0.447953   \n",
       "4      NaN       0.0            -0.0  ...  0.748054  0.906933  0.364442   \n",
       "\n",
       "         19        20              21        22        23        24        25  \n",
       "0      padj  baseMean  log2FoldChange     lfcSE      stat    pvalue      padj  \n",
       "1       NaN  0.058385       -0.500626   3.11654  0.160635  0.872381       NaN  \n",
       "2       NaN       0.0            -0.0       NaN       NaN       0.0       NaN  \n",
       "3  0.999815  5.267543        0.666107  0.567477 -1.173804  0.240473  0.774211  \n",
       "4       NaN       0.0            -0.0       NaN       NaN       0.0       NaN  \n",
       "\n",
       "[5 rows x 26 columns]"
      ]
     },
     "execution_count": 80,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_merged.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 81,
   "id": "d773561d",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(15523, 20)\n",
      "(17489, 26)\n"
     ]
    }
   ],
   "source": [
    "print(df_save.shape)\n",
    "print(df_merged.shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "09d7623a",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "id": "cf57d0bc",
   "metadata": {},
   "source": [
    "# MYB_degraded_v7 (res_v7)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "id": "27a6562b",
   "metadata": {},
   "outputs": [],
   "source": [
    "project = project2\n",
    "res_version = \"res_v7\"\n",
    "project_dir = \"slamseq_{}_{}\".format(project, res_version)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "id": "44dd77cd",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "5\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "['../results/slamseq_MYB_degraded_v1_res_v7/MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts.csv',\n",
       " '../results/slamseq_MYB_degraded_v1_res_v7/MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_tccounts.csv',\n",
       " '../results/slamseq_MYB_degraded_v1_res_v7/MYB_degraded_v1_0_5_1h_deseq_readcounts.csv',\n",
       " '../results/slamseq_MYB_degraded_v1_res_v7/MYB_degraded_v1_0_5_1h_deseq_tccounts.csv',\n",
       " '../results/slamseq_MYB_degraded_v1_res_v7/slamseq_MYB_degraded_v1_res_v7_0_5_deseq_output.csv']"
      ]
     },
     "execution_count": 51,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "files = ! ls $res_dir/$project_dir/*.csv\n",
    "files = [file for file in files if \"deseq\" in file if \"drop\" not in file]\n",
    "# ERCC files first\n",
    "files = [file for file in files if \"ERCCsamplewise\" in file] + \\\n",
    "[file for file in files if \"ERCCsamplewise\" not in file]\n",
    "print(len(files))\n",
    "files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "id": "0087ae31",
   "metadata": {},
   "outputs": [],
   "source": [
    "files = files[:-1]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "id": "bd1526e3",
   "metadata": {},
   "outputs": [],
   "source": [
    "# project 2\n",
    "conditions = [\"1h\"]\n",
    "scaling = [\"ERCCsamplewise\", None]\n",
    "count_type = [\"readcounts\", \"tccounts\"]"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ede1d665",
   "metadata": {},
   "source": [
    "## make info rows"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "id": "c3b25d9f",
   "metadata": {},
   "outputs": [],
   "source": [
    "# df with file info\n",
    "df_info = pd.DataFrame(list(zip([\"project\", \"file\", \"scale\", \"condition\", \"type\"],\n",
    "                                [\"project\", \"file\", \"scale\", \"condition\", \"type\"])))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "id": "bdacf194",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(5, 26)"
      ]
     },
     "execution_count": 55,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_info = pd.concat([df_info] + [pd.DataFrame([x*6 for x in get_file_info(project, file, conditions)]) \\\n",
    "                                 for file in files], axis=1)\n",
    "df_info.shape"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "911feccc",
   "metadata": {},
   "source": [
    "## merge dfs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "id": "67845e8a",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>baseMean_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>log2FoldChange_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>lfcSE_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>stat_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>pvalue_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>padj_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts</th>\n",
       "      <th>genes</th>\n",
       "      <th>gene_id</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1.881134</td>\n",
       "      <td>-0.045298</td>\n",
       "      <td>1.016923</td>\n",
       "      <td>0.044544</td>\n",
       "      <td>0.964471</td>\n",
       "      <td>NaN</td>\n",
       "      <td>A1BG</td>\n",
       "      <td>A1BG</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   baseMean_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "0                                           1.881134                 \n",
       "\n",
       "   log2FoldChange_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "0                                          -0.045298                       \n",
       "\n",
       "   lfcSE_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "0                                           1.016923              \n",
       "\n",
       "   stat_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "0                                           0.044544             \n",
       "\n",
       "   pvalue_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts  \\\n",
       "0                                           0.964471               \n",
       "\n",
       "   padj_MYB_degraded_v1_ERCCsamplewise_0_5_1h_deseq_readcounts genes gene_id  \n",
       "0                                                NaN            A1BG    A1BG  "
      ]
     },
     "execution_count": 56,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Get list of deseq dfs\n",
    "deseq_dfs = [get_deseq_data(file) for file in files]\n",
    "deseq_dfs[0].iloc[:1]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "id": "4360bfa9",
   "metadata": {},
   "outputs": [],
   "source": [
    "# main df\n",
    "df_merged = pd.DataFrame()\n",
    "df_merged = df_merged.append(deseq_dfs[0][[\"genes\", \"gene_id\"]]) # add gene cols to main df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "id": "1b71f3c1",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(17211, 26)"
      ]
     },
     "execution_count": 58,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# join df columns on gene & gene_id columns\n",
    "df_merged = reduce(lambda left, right: pd.merge(left, right, on=['genes', \"gene_id\"], how=\"outer\"), \n",
    "                   [df_merged] + deseq_dfs)\n",
    "df_merged.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "id": "562a7a03",
   "metadata": {},
   "outputs": [],
   "source": [
    "# change column names\n",
    "df_merged.columns = [col_name.split(\"_\")[0] if \"gene\" not in col_name else \n",
    "                     col_name for col_name in df_merged.columns.tolist()]\n",
    "df_merged = df_merged.T.reset_index().T.reset_index(drop=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9debcb21",
   "metadata": {},
   "source": [
    "## add info columns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "id": "b49c363f",
   "metadata": {},
   "outputs": [],
   "source": [
    "# reset column index\n",
    "df_info = df_info.T.reset_index(drop=True).T"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "id": "57b0f907",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 4 ms, sys: 0 ns, total: 4 ms\n",
      "Wall time: 3.44 ms\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "(17217, 26)"
      ]
     },
     "execution_count": 61,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "%%time\n",
    "\n",
    "# combine info and data\n",
    "df_save = df_info.append(df_merged, ignore_index=True)\n",
    "df_save.shape"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e35bc746",
   "metadata": {},
   "source": [
    "## save merged file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "id": "64b0823a",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved DESeq results to: ../results/slamseq_merged_deseq_noAlt.csv\n"
     ]
    }
   ],
   "source": [
    "# save file\n",
    "res_dir = \"../results\"\n",
    "deseq_res_file = \"slamseq_merged_deseq_noAlt.csv\"\n",
    "\n",
    "df_save.to_csv(\"{}/{}\".format(res_dir, deseq_res_file), header=False, index=False)\n",
    "\n",
    "print(\"Saved DESeq results to: {}/{}\".format(res_dir, deseq_res_file))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "865eccaa",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "72a4813e",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.11"
  },
  "toc": {
   "base_numbering": 1,
   "nav_menu": {
    "height": "83px",
    "width": "182px"
   },
   "number_sections": false,
   "sideBar": true,
   "skip_h1_title": false,
   "title_cell": "Table of Contents",
   "title_sidebar": "Contents",
   "toc_cell": false,
   "toc_position": {
    "height": "calc(100% - 180px)",
    "left": "10px",
    "top": "150px",
    "width": "177px"
   },
   "toc_section_display": true,
   "toc_window_display": true
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
